{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "5bddc0fe",
   "metadata": {},
   "source": [
    "# SourceData Augmentation\n",
    "\n",
    "In this notebook we will check how to read the data encoded in XML files, using the different classes, methods and functions already available at soda-roberta."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "33b18032",
   "metadata": {},
   "outputs": [],
   "source": [
    "from smtag.encoder import XMLEncoder\n",
    "import os\n",
    "from lxml.etree import fromstring, Element"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "35b15c07",
   "metadata": {},
   "outputs": [],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_panels/'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "6379891e",
   "metadata": {},
   "outputs": [],
   "source": [
    "subsets = [\"train\", \"eval\", \"test\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "18d62e95",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/app/data/xml/sd_panels/train.txt'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "source_file_path = os.path.join(XML_FOLDER, subsets[0]) + \".txt\"\n",
    "source_file_path"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ebdaf74c",
   "metadata": {},
   "source": [
    "We can see here how the text of a single panel looks like in XML"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "5eb8490f",
   "metadata": {},
   "source": [
    "Taking a single line to ease the process of understanding and trying out the XML encoder"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "de3a0709",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'list'>\n",
      "48771\n"
     ]
    }
   ],
   "source": [
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "    print(type(lines))\n",
    "\n",
    "line = lines[2]\n",
    "print(len(lines))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "07b0aaae",
   "metadata": {},
   "source": [
    "We use the `lxml.etree.fromstring` function to convert the `str` line into an XML element."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 101,
   "id": "067233e0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<Element sd-panel at 0xffff5b4002c0>"
      ]
     },
     "execution_count": 101,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "xml_example = fromstring(line)\n",
    "xml_example"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "eee409e1",
   "metadata": {},
   "source": [
    "The xpath to `sd-tag` will allow to search for the different attributes of the different XML tags in the text. The reason we are interested in `sd-tag` is because it is here where the information to be labeled can be found."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 102,
   "id": "c159ba19",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'id': 'sdTag15', 'source': 'sdapi', 'category': 'None', 'entity_type': 'protein', 'role': 'assayed', 'text': 'ATZ polymers', 'ext_ids': 'P01009', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'SERPINA1', 'ext_tax_ids': '9606', 'ext_tax_names': 'Homo sapiens', 'ext_urls': 'https://www.uniprot.org/uniprot/'}\n",
      "{'id': 'sdTag16', 'source': 'sdapi', 'category': 'assay', 'entity_type': 'None', 'role': 'None', 'text': 'immunoisolated', 'ext_ids': 'BAO_0002508', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'immunoprecipitation', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'https://bioportal.bioontology.org/ontologies/BAO/?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23'}\n",
      "{'id': 'sdTag17', 'source': 'sdapi', 'category': 'None', 'entity_type': 'cell', 'role': 'component', 'text': 'MEF', 'ext_ids': 'CL:2000042', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'embryonic fibroblast', 'ext_tax_ids': '10090', 'ext_tax_names': 'Mus musculus', 'ext_urls': ' https://identifiers.org/'}\n",
      "{'id': 'sdTag18', 'source': 'sdapi', 'category': 'None', 'entity_type': 'molecule', 'role': 'intervention', 'text': 'BafA1', 'ext_ids': 'CHEBI:22689', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'bafilomycin A1', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'http://www.ebi.ac.uk/chebi/searchId.do?chebiId='}\n",
      "{'id': 'sdTag19', 'source': 'sdapi', 'category': 'None', 'entity_type': 'molecule', 'role': 'intervention', 'text': 'BafA1', 'ext_ids': 'CHEBI:22689', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'bafilomycin A1', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'http://www.ebi.ac.uk/chebi/searchId.do?chebiId='}\n",
      "{'id': 'sdTag20', 'source': 'sdapi', 'category': 'assay', 'entity_type': 'None', 'role': 'None', 'text': 'Immunoprecipitation', 'ext_ids': 'BAO_0002508', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'immunoprecipitation', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'https://bioportal.bioontology.org/ontologies/BAO/?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23'}\n",
      "{'id': 'sdTag21', 'source': 'sdapi', 'category': 'assay', 'entity_type': 'None', 'role': 'None', 'text': 'IP', 'ext_ids': 'BAO_0002508', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'immunoprecipitation', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'https://bioportal.bioontology.org/ontologies/BAO/?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23'}\n",
      "{'id': 'sdTag22', 'source': 'sdapi', 'category': 'None', 'entity_type': 'protein', 'role': 'assayed', 'text': 'ATZ polymers', 'ext_ids': 'P01009', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'SERPINA1', 'ext_tax_ids': '9606', 'ext_tax_names': 'Homo sapiens', 'ext_urls': 'https://www.uniprot.org/uniprot/'}\n",
      "{'id': 'sdTag23', 'source': 'sdapi', 'category': 'None', 'entity_type': 'protein', 'role': 'reporter', 'text': 'HA', 'ext_ids': '', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': '', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': ''}\n",
      "{'id': 'sdTag24', 'source': 'sdapi', 'category': 'assay', 'entity_type': 'None', 'role': 'None', 'text': 'western blot', 'ext_ids': 'BAO_0002424', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'western blot', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'https://bioportal.bioontology.org/ontologies/BAO/?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23'}\n",
      "{'id': 'sdTag25', 'source': 'sdapi', 'category': 'assay', 'entity_type': 'None', 'role': 'None', 'text': 'WB', 'ext_ids': 'BAO_0002424', 'ext_dbs': '', 'in_caption': 'True', 'ext_names': 'western blot', 'ext_tax_ids': '', 'ext_tax_names': '', 'ext_urls': 'https://bioportal.bioontology.org/ontologies/BAO/?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23'}\n"
     ]
    }
   ],
   "source": [
    "test = xml_example.xpath(\"sd-tag\")\n",
    "for item in test:\n",
    "    print(item.attrib)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 103,
   "id": "689db3da",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Entity\t\tCategory\t\tRole\t\tText\t\tExtended names\n",
      "prot\t\tNone\t\tassayed\t\tATZ polymers\t\tSERPINA1\n",
      "None\t\tassay\t\tNone\t\timmunoisolated\t\timmunoprecipitation\n",
      "cell\t\tNone\t\tcomponent\t\tMEF\t\tembryonic fibroblast\n",
      "mole\t\tNone\t\tintervention\t\tBafA1\t\tbafilomycin A1\n",
      "mole\t\tNone\t\tintervention\t\tBafA1\t\tbafilomycin A1\n",
      "None\t\tassay\t\tNone\t\tImmunoprecipitation\t\timmunoprecipitation\n",
      "None\t\tassay\t\tNone\t\tIP\t\timmunoprecipitation\n",
      "prot\t\tNone\t\tassayed\t\tATZ polymers\t\tSERPINA1\n",
      "prot\t\tNone\t\treporter\t\tHA\t\t\n",
      "None\t\tassay\t\tNone\t\twestern blot\t\twestern blot\n",
      "None\t\tassay\t\tNone\t\tWB\t\twestern blot\n"
     ]
    }
   ],
   "source": [
    "test = xml_example.xpath(\"sd-tag\")\n",
    "print(f\"{'Entity'}\\t\\t{'Category'}\\t\\t{'Role'}\\t\\t{'Text'}\\t\\t{'Extended names'}\")\n",
    "for item in test:\n",
    "    if item.attrib.get(\"entity_type\", None):\n",
    "        print(f'{item.attrib[\"entity_type\"][0:4]}\\t\\t{item.attrib[\"category\"]}\\t\\t{item.attrib[\"role\"]}\\t\\t{item.attrib[\"text\"]}\\t\\t{item.attrib[\"ext_names\"]}')\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 104,
   "id": "1dc4eaa3",
   "metadata": {},
   "outputs": [],
   "source": [
    "xml_encoder = XMLEncoder(xml_example)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 105,
   "id": "3b84ba27",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "E \n",
      "ATZ polymers\n",
      " \n",
      "immunoisolated\n",
      " from lysates of WT \n",
      "MEF\n",
      " mock treated (lane 1), incubated for 12 h with \n",
      "BafA1\n",
      " (lane 2) and 4 h after \n",
      "BafA1\n",
      " wash-out (lane 3). \n",
      "Immunoprecipitation\n",
      " (\n",
      "IP\n",
      ") of \n",
      "ATZ polymers\n",
      " with polymer-specific 2C1 antibody, transfer on PVDF membrane, revealed with anti-\n",
      "HA\n",
      " antibody on \n",
      "western blot\n",
      " (\n",
      "WB\n",
      "). \n",
      "F\n",
      " Quantification of \n",
      "E\n",
      ", n=3, mean ± SEM. Unpaired two-tailed \n",
      "t\n",
      "-test, ns P>0.05, * P<0.05.\n",
      " Data information: Scale bars: 10 μm. \n"
     ]
    }
   ],
   "source": [
    "for element in xml_encoder.element.itertext():\n",
    "    print(element)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "id": "dd3cc4a7",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'E ATZ polymers immunoisolated from lysates of WT MEF mock treated (lane 1), incubated for 12 h with BafA1 (lane 2) and 4 h after BafA1 wash-out (lane 3). Immunoprecipitation (IP) of ATZ polymers with polymer-specific 2C1 antibody, transfer on PVDF membrane, revealed with anti-HA antibody on western blot (WB). F Quantification of E, n=3, mean ± SEM. Unpaired two-tailed t-test, ns P>0.05, * P<0.05. Data information: Scale bars: 10 μm. '"
      ]
     },
     "execution_count": 106,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text = []\n",
    "for element in xml_encoder.element.itertext():\n",
    "    text.append(element)\n",
    "\"\".join(text)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c5e56e20",
   "metadata": {},
   "source": [
    "At this point we have a text that can be tagged using the information of SDtags and convert into a string of characters. From here on we could begin to generate a proper word labelling algorithm. \n",
    "\n",
    "To be able to generate augmented data we need to go up and change the name of the different entities."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "30b6abdd",
   "metadata": {},
   "source": [
    "## Creating augmented data using the template of a given caption for panelization\n",
    "\n",
    "What we have seen up to know is the summary of how we can get to the point of begin to work with the data from `xml` files. \n",
    "\n",
    "Now we will see how to get the data into a position on which we can edit part of it. Since the XML labels encode the labels we need for the text, in general the entire work will be related to get these lables and modify them. \n",
    "\n",
    "The main problem might arise from the fact that in the case of panels, the characters or words defining different panels are not standarized and not contained into `xml` tags. This will make the data augmentation procedure a bit more complicated. \n",
    "\n",
    "Note that `sd_panels` stores the info of just one panel at a time while `sd_fig` does it for all the panels in a figure. Looks like the second is a better chance to work on the data augmentation for the panelization task."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 116,
   "id": "da352d7a",
   "metadata": {
    "scrolled": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<fig id=\"22315\"><title>.</title><label>Figure 2</label><graphic href=\"https://api.sourcedata.io/file.php?figure_id=22315\"/><sd-panel panel_id=\"65030\">A <sd-tag id=\"sdTag38\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"HEK293\" ext_ids=\"CVCL_0045\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"HEK293\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://identifiers.org/cellosaurus:\">HEK293</sd-tag> cells transfected with empty vector (lanes 1, 6), <sd-tag id=\"sdTag39\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"ATZ\" ext_ids=\"5265\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">ATZ</sd-tag>-<sd-tag id=\"sdTag40\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> (2, 7), <sd-tag id=\"sdTag41\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>-<sd-tag id=\"sdTag42\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> (3 and 8), <sd-tag id=\"sdTag43\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>-<sd-tag id=\"sdTag44\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> and <sd-tag id=\"sdTag45\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"ATZ\" ext_ids=\"5265\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">ATZ</sd-tag>-<sd-tag id=\"sdTag46\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> (4, 9), or <sd-tag id=\"sdTag339\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>LIR-<sd-tag id=\"sdTag48\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> and <sd-tag id=\"sdTag49\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"ATZ\" ext_ids=\"5265\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">ATZ</sd-tag>-<sd-tag id=\"sdTag50\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> (5, 10), incubated for 12 h with 50 nM <sd-tag id=\"sdTag51\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"component\" text=\"BafA1\" ext_ids=\"CHEBI:22689\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"bafilomycin A1\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">BafA1</sd-tag> and then treated with the cross-linker <sd-tag id=\"sdTag52\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"component\" text=\"DSP\" ext_ids=\"93313\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"57757-57-0\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://pubchem.ncbi.nlm.nih.gov/compound/\">DSP</sd-tag> before lysis as described in Methods. Lanes 1-5, <sd-tag id=\"sdTag53\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"WB\" ext_ids=\"BAO_0002424\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"western blot\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">WB</sd-tag> of the total cell extract (TCE); lanes 6-10, <sd-tag id=\"sdTag54\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"WB\" ext_ids=\"BAO_0002424\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"western blot\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">WB</sd-tag> of anti<sd-tag id=\"sdTag55\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> <sd-tag id=\"sdTag56\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"immunoprecipites\" ext_ids=\"BAO_0002508\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"immunoprecipitation\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">immunoprecipites</sd-tag> to isolate complexes containing ectopically expressed <sd-tag id=\"sdTag57\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"FAM134B\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">FAM134B</sd-tag> or <sd-tag id=\"sdTag340\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"FAM134B\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">FAM134B</sd-tag>LIR. The membranes were probed with anti<sd-tag id=\"sdTag59\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> (upper panels), anti<sd-tag id=\"sdTag60\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"CNX\" ext_ids=\"P27824\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"CANX\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">CNX</sd-tag>, anti<sd-tag id=\"sdTag61\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> and antiLC3 antibodies. <p><strong>B</strong> Quantification of LC3 <sd-tag id=\"sdTag64\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"co-precipitating\" ext_ids=\"BAO_0002508\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"immunoprecipitation\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">co-precipitating</sd-tag> with <sd-tag id=\"sdTag65\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"FAM134B\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">FAM134B</sd-tag>-<sd-tag id=\"sdTag66\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> (<strong>A</strong>, lanes 8 and 9). Mean &#177; SEM, n=3, unpaired two-tailed <em>t</em>-test, * P&lt;0.05.</p> <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65030\"/></sd-panel><sd-panel panel_id=\"65031\">WT <sd-tag id=\"sdTag70\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"MEF\" ext_ids=\"CL:2000042\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"embryonic fibroblast\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\" https://identifiers.org/\">MEF</sd-tag>. Scale bar: 10 &#956;m. <p><strong>D</strong> Ectopic <sd-pretag id=\"sdPretag400090983sd\" parent-tag-id=\"108\" class=\"assay\">expression</sd-pretag> of <sd-tag id=\"sdTag341\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>LIR in WT <sd-tag id=\"sdTag73\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"MEF\" ext_ids=\"CL:2000042\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"embryonic fibroblast\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\" https://identifiers.org/\">MEF</sd-tag> inhibits <sd-tag id=\"sdTag74\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"ATZ\" ext_ids=\"P01009\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">ATZ</sd-tag> delivery to <sd-tag id=\"sdTag75\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"endolysosomes\" ext_ids=\"GO:0036019\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"endolysosome\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://amigo.geneontology.org/amigo/term/\">endolysosomes</sd-tag>. Scale bar: 10 &#956;m.</p> <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65031\"/></sd-panel><sd-panel panel_id=\"65032\">E, F <sd-tag id=\"sdTag337\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"Distribution\" ext_ids=\"BAO_0000378\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"protein redistribution assay\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">Distribution</sd-tag> of <sd-tag id=\"sdTag78\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"component\" text=\"gold\" ext_ids=\"CHEBI:29287\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"gold atom\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">gold</sd-tag>-labeled <sd-tag id=\"sdTag79\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"ATZ\" ext_ids=\"P01009\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">ATZ</sd-tag>-<sd-tag id=\"sdTag80\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> by <sd-tag id=\"sdTag81\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"IEM\" ext_ids=\"BAO_0000449\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"electron microscopy\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">IEM</sd-tag> in <sd-tag id=\"sdTag82\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"component\" text=\"BafA1\" ext_ids=\"CHEBI:22689\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"bafilomycin A1\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">BafA1</sd-tag> treated WT <sd-tag id=\"sdTag83\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"MEF\" ext_ids=\"CL:2000042\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"embryonic fibroblast\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\" https://identifiers.org/\">MEF</sd-tag> and WT <sd-tag id=\"sdTag84\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"MEF\" ext_ids=\"CL:2000042\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"embryonic fibroblast\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\" https://identifiers.org/\">MEF</sd-tag> overexpressing <sd-tag id=\"sdTag342\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>LIR, respectively. <sd-tag id=\"sdTag343\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"EV\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">EV</sd-tag>, <sd-tag id=\"sdTag344\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"ER-derived vesicles\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">ER-derived vesicles</sd-tag>, red arrowheads; <sd-tag id=\"sdTag87\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"EL\" ext_ids=\"GO:0036019\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"endolysosome\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://amigo.geneontology.org/amigo/term/\">EL</sd-tag>, <sd-tag id=\"sdTag88\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"endolysosome\" ext_ids=\"GO:0036019\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"endolysosome\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://amigo.geneontology.org/amigo/term/\">endolysosome</sd-tag>. <p><strong>G</strong> Quantification of <sd-tag id=\"sdTag89\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"ATZ\" ext_ids=\"P01009\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">ATZ</sd-tag>-<sd-tag id=\"sdTag90\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"component\" text=\"gold\" ext_ids=\"CHEBI:29287\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"gold atom\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">gold</sd-tag> <sd-tag id=\"sdTag91\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"density\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">density</sd-tag> of <strong>E, F</strong> (n=75 and 79 <sd-tag id=\"sdTag92\" source=\"sdapi\" category=\"None\" entity_type=\"subcellular\" role=\"assayed\" text=\"EL\" ext_ids=\"GO:0036019\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"endolysosome\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://amigo.geneontology.org/amigo/term/\">EL</sd-tag>, respectively). Unpaired two-tailed <em>t</em>-test, **** P &lt;0.0001.</p> <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65032\"/></sd-panel><sd-panel panel_id=\"65033\">H Max projection of the same cell as in <strong>D</strong> after <sd-tag id=\"sdTag99\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"deconvolution\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">deconvolution</sd-tag>. Insets show orthogonal section of select regions. Scale bar: 10 &#956;m. <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65033\"/></sd-panel><sd-panel panel_id=\"65034\">I Decay of <sd-tag id=\"sdTag100\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"ATZ polymers\" ext_ids=\"P01009\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">ATZ polymers</sd-tag> (<sd-tag id=\"sdTag101\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"intervention\" text=\"CHX\" ext_ids=\"CHEBI:27641\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"cycloheximide\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">CHX</sd-tag> chase, upper panel) <sd-tag id=\"sdTag102\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"immunoisolated\" ext_ids=\"BAO_0002508\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"immunoprecipitation\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">immunoisolated</sd-tag> with the polymer-specific 2C1 antibody (visualized with anti<sd-tag id=\"sdTag103\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> in <sd-tag id=\"sdTag104\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"WB\" ext_ids=\"BAO_0002424\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"western blot\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">WB</sd-tag>) in <sd-tag id=\"sdTag105\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"HEK293\" ext_ids=\"CVCL_0045\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"HEK293\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://identifiers.org/cellosaurus:\">HEK293</sd-tag> cells mock-transfected (lanes 1-4) or <sd-pretag id=\"sdPretag1904538393sd\" parent-tag-id=\"108\" class=\"assay\">expressing</sd-pretag> <sd-tag id=\"sdTag346\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134B\" ext_ids=\"66270\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Retreg1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"http://www.ncbi.nlm.nih.gov/gene/\">FAM134B</sd-tag>LIR-<sd-tag id=\"sdTag107\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> (lanes 5-8). Middle panel, <sd-tag id=\"sdTag108\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"expression\" ext_ids=\"OBI_0000615\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"protein expression profiling\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ontobee.org/ontology/OBI?iri=http://purl.obolibrary.org/obo/\">expression</sd-tag> of <sd-tag id=\"sdTag347\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"FAM134B\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">FAM134B</sd-tag>LIR-<sd-tag id=\"sdTag110\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"V5\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">V5</sd-tag> assessed by <sd-tag id=\"sdTag111\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"WB\" ext_ids=\"BAO_0002424\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"western blot\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">WB</sd-tag>; lower panel, loading control. <p>(<strong>J</strong>) Quantification of (<strong>I)</strong> (Mean &#177; SEM, n=3, unpaired two-tailed <em>t</em>-test, * P&lt;0.05).</p> <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65034\"/></sd-panel><sd-panel panel_id=\"65035\">K <sd-tag id=\"sdTag113\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"Flow cytometry\" ext_ids=\"BAO_0000005\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"flow cytometry\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">Flow cytometry</sd-tag> analysis of <sd-tag id=\"sdTag114\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"ATZ\" ext_ids=\"P01009\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"SERPINA1\" ext_tax_ids=\"9606\" ext_tax_names=\"Homo sapiens\" ext_urls=\"https://www.uniprot.org/uniprot/\">ATZ</sd-tag>-<sd-tag id=\"sdTag115\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"HA\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">HA</sd-tag> <sd-tag id=\"sdTag116\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"polymer levels\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">polymer levels</sd-tag> in <sd-tag id=\"sdTag117\" source=\"sdapi\" category=\"None\" entity_type=\"cell\" role=\"component\" text=\"MEFs\" ext_ids=\"CL:2000042\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"embryonic fibroblast\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\" https://identifiers.org/\">MEFs</sd-tag> mock treated, exposed to <sd-tag id=\"sdTag118\" source=\"sdapi\" category=\"None\" entity_type=\"molecule\" role=\"intervention\" text=\"BafA1\" ext_ids=\"CHEBI:22689\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"bafilomycin A1\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ebi.ac.uk/chebi/searchId.do?chebiId=\">BafA1</sd-tag>, and co-<sd-pretag id=\"sdPretag1977576765sd\" parent-tag-id=\"317\" class=\"assay\">expressing</sd-pretag> <sd-tag id=\"sdTag349\" source=\"sdapi\" category=\"None\" entity_type=\"gene\" role=\"intervention\" text=\"FAM134\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">FAM134</sd-tag>LIR. M<sd-tag id=\"sdTag120\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"FI\" ext_ids=\"BAO_0000363\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"fluorescence intensity\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">FI</sd-tag>: mean <sd-tag id=\"sdTag121\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"fluorescence intensity\" ext_ids=\"BAO_0000363\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"fluorescence intensity\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">fluorescence intensity</sd-tag> (Mean &#177; SEM, n=5, unpaired two-tailed <em>t</em>-test, ns P&gt;0.05, ** P&lt;0.01, *** P &lt;0.001). <graphic href=\"https://api.sourcedata.io/file.php?panel_id=65035\"/></sd-panel></fig>\\n'"
      ]
     },
     "execution_count": 116,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "line"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "a1bbbcee",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<class 'list'>\n",
      "12108\n"
     ]
    }
   ],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_fig/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[0]) + \".txt\"\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "    print(type(lines))\n",
    "\n",
    "line = lines[0]\n",
    "\n",
    "xml_example = fromstring(line)\n",
    "print(len(lines))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd6aef93",
   "metadata": {},
   "source": [
    "We try now to see how to get the first characters of a panel. We expect them to be the panel ID in the text. `A`, `B` or so... In this case, unfortunately this is not so easy. We see how from A, it goes all the way down to `E, F`.\n",
    "\n",
    "The reason is that the other tags are encoded into `strong` tags, that make the things difficult. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 143,
   "id": "cacd18cb",
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "A Intracellular \n",
      "E \n",
      "G-K Same as \n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "\".Figure 1A Intracellular localization of total (HA) and polymeric ATZ (2C1) in WT MEF mock-treated, Confocal Laser Scanning Microscopy (CLSM). B Same as A for MEF exposed to 50 nM BafA1 for 12 h. C Same as A, 4 h after BafA1 wash-out. D Quantification of ATZ-positive, LAMP1-positive endolysosomes (EL) (n=13, 10, 11 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. Data information: Scale bars: 10 μm. E ATZ polymers immunoisolated from lysates of WT MEF mock treated (lane 1), incubated for 12 h with BafA1 (lane 2) and 4 h after BafA1 wash-out (lane 3). Immunoprecipitation (IP) of ATZ polymers with polymer-specific 2C1 antibody, transfer on PVDF membrane, revealed with anti-HA antibody on western blot (WB). F Quantification of E, n=3, mean ± SEM. Unpaired two-tailed t-test, ns P>0.05, * P<0.05. Data information: Scale bars: 10 μm. G-K Same as B in WT MEF, in cells exposed to 20 mM CST and in Cnx-, Crt- and ERp57-KO MEF. L Quantification of ATZ-positive EL (n=10, 9, 11, 10, 11 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. Data information: Scale bars: 10 μm. \""
      ]
     },
     "execution_count": 143,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test = xml_example.xpath(\"sd-panel\")\n",
    "for item in test:\n",
    "    print(item.text)\n",
    "\n",
    "    \n",
    "xml_encoder = XMLEncoder(xml_example)    \n",
    "    \n",
    "text = []\n",
    "for element in xml_encoder.element.itertext():\n",
    "    text.append(element)\n",
    "\"\".join(text)    "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f65c5677",
   "metadata": {},
   "source": [
    "We check the elements. We see only a few of them. Thsi means that using sd-panels does not give everything. The reason is that inside sd-panels are other elements. We would need to get the `x-path` to these elements in order to get what we need. That is, the first character of each panel. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 144,
   "id": "34b14cda",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "<Element title at 0xffff5b4b8d80>\n",
      "<Element label at 0xffff5b4b8f80>\n",
      "<Element graphic at 0xffff5b4b8940>\n",
      "<Element sd-panel at 0xffff70073440>\n",
      "<Element sd-panel at 0xffff5b6b9dc0>\n",
      "<Element sd-panel at 0xffff5b19ed40>\n"
     ]
    }
   ],
   "source": [
    "for element in xml_encoder.element:\n",
    "    print(element)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 148,
   "id": "e2f24936",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "B\n",
      "A\n",
      "C\n",
      "A\n",
      "D\n",
      "F\n",
      "E\n",
      "L\n"
     ]
    }
   ],
   "source": [
    "test = xml_example.xpath(\"sd-panel/p/strong\")\n",
    "for item in test:\n",
    "    print(item.text)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9d5d7764",
   "metadata": {},
   "source": [
    "We will check now how easy would be to pick up porgramatically the panel IDs in the figure. I will do a small statistical experiment. We will `.split()` each panel and print to check if this would already give us the panel id."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 192,
   "id": "80270988",
   "metadata": {},
   "outputs": [],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_panels/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[2]) + \".txt\"\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "first_chars = []\n",
    "for line in lines: \n",
    "    xml_example = fromstring(line)  \n",
    "    xml_encoder = XMLEncoder(xml_example)\n",
    "    text = []\n",
    "    for element in xml_encoder.element.itertext():\n",
    "        text.append(element)\n",
    "    ids = \"\".join(text).split()[0]\n",
    "    if len(ids) < 5:\n",
    "        first_chars.append(ids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 193,
   "id": "e30e44a4",
   "metadata": {},
   "outputs": [],
   "source": [
    "from collections import Counter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 194,
   "id": "497c4c53",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "7178 5298\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[(\"'\", 2),\n",
       " ('(A', 10),\n",
       " ('(A)', 369),\n",
       " ('(A),', 1),\n",
       " ('(A,', 35),\n",
       " ('(A1)', 1),\n",
       " ('(A2)', 1),\n",
       " ('(A3)', 1),\n",
       " ('(B', 5),\n",
       " ('(B)', 292),\n",
       " ('(B),', 1),\n",
       " ('(B).', 1),\n",
       " ('(B,', 14),\n",
       " ('(B-', 1),\n",
       " ('(B1)', 1),\n",
       " ('(C', 3),\n",
       " ('(C)', 284),\n",
       " ('(C,', 19),\n",
       " ('(C2)', 1),\n",
       " ('(D', 4),\n",
       " ('(D)', 238),\n",
       " ('(D,', 11),\n",
       " ('(D;', 1),\n",
       " ('(E', 6),\n",
       " ('(E)', 172),\n",
       " ('(E,', 20),\n",
       " ('(E1,', 1),\n",
       " ('(F', 1),\n",
       " ('(F)', 146),\n",
       " ('(F,', 3),\n",
       " ('(G', 2),\n",
       " ('(G)', 107),\n",
       " ('(G,', 8),\n",
       " ('(H)', 66),\n",
       " ('(H,', 5),\n",
       " ('(I', 2),\n",
       " ('(I)', 42),\n",
       " ('(I,', 6),\n",
       " ('(J)', 30),\n",
       " ('(J,', 3),\n",
       " ('(K)', 19),\n",
       " ('(K,', 5),\n",
       " ('(L)', 13),\n",
       " ('(L,', 1),\n",
       " ('(M', 1),\n",
       " ('(M)', 11),\n",
       " ('(M,', 1),\n",
       " ('(N)', 9),\n",
       " ('(N,', 1),\n",
       " ('(O)', 3),\n",
       " ('(O,', 1),\n",
       " ('(P)', 2),\n",
       " ('(P,', 2),\n",
       " ('(Q)', 1),\n",
       " ('(Q,', 1),\n",
       " ('(R)', 1),\n",
       " ('(S)', 2),\n",
       " ('(T)', 1),\n",
       " ('(U)', 1),\n",
       " ('(U,', 1),\n",
       " ('(V)', 1),\n",
       " ('(a)', 90),\n",
       " ('(a,', 5),\n",
       " ('(ai)', 1),\n",
       " ('(b)', 81),\n",
       " ('(b,', 1),\n",
       " ('(c)', 68),\n",
       " ('(c,', 5),\n",
       " ('(d)', 50),\n",
       " ('(d,', 1),\n",
       " ('(e)', 44),\n",
       " ('(e).', 1),\n",
       " ('(e,', 4),\n",
       " ('(f)', 30),\n",
       " ('(f,', 2),\n",
       " ('(g)', 17),\n",
       " ('(g,', 3),\n",
       " ('(h)', 8),\n",
       " ('(h,', 1),\n",
       " ('(i)', 7),\n",
       " ('(i,', 2),\n",
       " ('(j)', 3),\n",
       " ('(j).', 1),\n",
       " ('(k)', 1),\n",
       " ('(k).', 1),\n",
       " ('(k,', 1),\n",
       " ('(l)', 1),\n",
       " ('(l,', 1),\n",
       " ('(m)', 1),\n",
       " ('(n,', 1),\n",
       " ('(p,', 1),\n",
       " (').', 1),\n",
       " (',', 3),\n",
       " ('-E.', 1),\n",
       " ('.', 8),\n",
       " ('.-G', 1),\n",
       " ('3D', 3),\n",
       " ('5.2+', 1),\n",
       " (':', 1),\n",
       " (';UPR', 1),\n",
       " ('A', 175),\n",
       " ('A)', 70),\n",
       " ('A).', 1),\n",
       " ('A,', 30),\n",
       " ('A,B', 4),\n",
       " ('A,C.', 1),\n",
       " ('A-B', 2),\n",
       " ('A-B.', 3),\n",
       " ('A-C', 8),\n",
       " ('A-C,', 1),\n",
       " ('A-C.', 5),\n",
       " ('A-D', 9),\n",
       " ('A-D)', 1),\n",
       " ('A-D.', 4),\n",
       " ('A-E', 1),\n",
       " ('A-F', 4),\n",
       " ('A-F)', 1),\n",
       " ('A-G', 2),\n",
       " ('A-H', 7),\n",
       " ('A-J', 1),\n",
       " ('A-K', 1),\n",
       " ('A-L', 1),\n",
       " ('A-P', 1),\n",
       " ('A.', 162),\n",
       " ('A:', 11),\n",
       " ('ABC.', 1),\n",
       " ('API', 1),\n",
       " ('ATP', 1),\n",
       " ('All', 2),\n",
       " ('An', 1),\n",
       " ('As', 6),\n",
       " ('At', 3),\n",
       " ('Atg8', 1),\n",
       " ('A‐F′', 1),\n",
       " ('B', 128),\n",
       " ('B)', 66),\n",
       " ('B).', 1),\n",
       " ('B,', 20),\n",
       " ('B,C', 4),\n",
       " ('B,C)', 1),\n",
       " ('B-C.', 3),\n",
       " ('B-D', 2),\n",
       " ('B-E', 2),\n",
       " ('B-G', 2),\n",
       " ('B-K.', 1),\n",
       " ('B.', 126),\n",
       " ('B:', 8),\n",
       " ('BAG6', 3),\n",
       " ('BP', 1),\n",
       " ('Bar', 18),\n",
       " ('Both', 1),\n",
       " ('BrdU', 1),\n",
       " ('C', 114),\n",
       " ('C)', 53),\n",
       " ('C).', 3),\n",
       " ('C,', 29),\n",
       " ('C,D', 1),\n",
       " ('C,D)', 1),\n",
       " ('C-D', 1),\n",
       " ('C-D)', 1),\n",
       " ('C-E', 3),\n",
       " ('C-E.', 3),\n",
       " ('C-F', 1),\n",
       " ('C-F.', 2),\n",
       " ('C-F;', 4),\n",
       " ('C-G', 1),\n",
       " ('C.', 110),\n",
       " ('C4da', 2),\n",
       " ('C:', 12),\n",
       " ('CD86', 2),\n",
       " ('CDF1', 3),\n",
       " ('CFC', 1),\n",
       " ('CHX', 1),\n",
       " ('CO', 4),\n",
       " ('CST', 1),\n",
       " ('Cell', 7),\n",
       " ('ChEP', 1),\n",
       " ('ChIP', 3),\n",
       " ('Core', 1),\n",
       " ('CpG', 1),\n",
       " ('D', 100),\n",
       " ('D)', 36),\n",
       " ('D).', 1),\n",
       " ('D,', 17),\n",
       " ('D,E', 5),\n",
       " ('D-E.', 2),\n",
       " ('D-F', 1),\n",
       " ('D-G', 1),\n",
       " ('D-J.', 1),\n",
       " ('D-L', 1),\n",
       " ('D.', 96),\n",
       " ('D:', 10),\n",
       " ('DCs', 4),\n",
       " ('DIA', 1),\n",
       " ('DNA', 7),\n",
       " ('DRiP', 3),\n",
       " ('Data', 1),\n",
       " ('Dose', 2),\n",
       " ('Dot', 11),\n",
       " ('Down', 1),\n",
       " ('D‐G″', 1),\n",
       " ('E', 75),\n",
       " ('E)', 28),\n",
       " ('E).', 1),\n",
       " ('E):', 1),\n",
       " ('E,', 16),\n",
       " ('E,F)', 1),\n",
       " ('E,F.', 2),\n",
       " ('E-F.', 1),\n",
       " ('E-G', 3),\n",
       " ('E-G.', 3),\n",
       " ('E-H', 1),\n",
       " ('E-H.', 6),\n",
       " ('E.', 66),\n",
       " ('E:', 7),\n",
       " ('EAE', 4),\n",
       " ('EMT', 1),\n",
       " ('ER', 1),\n",
       " ('Ear', 2),\n",
       " ('F', 57),\n",
       " ('F)', 18),\n",
       " ('F,', 16),\n",
       " ('F,G', 4),\n",
       " ('F,G)', 1),\n",
       " ('F-G)', 1),\n",
       " ('F-G.', 2),\n",
       " ('F-H', 1),\n",
       " ('F-H.', 1),\n",
       " ('F-I', 1),\n",
       " ('F-I)', 1),\n",
       " ('F-I.', 1),\n",
       " ('F-L', 1),\n",
       " ('F.', 53),\n",
       " ('F:', 4),\n",
       " ('FACS', 10),\n",
       " ('FT', 1),\n",
       " ('Five', 1),\n",
       " ('Flow', 8),\n",
       " ('Fold', 3),\n",
       " ('For', 5),\n",
       " ('G', 41),\n",
       " ('G)', 13),\n",
       " ('G,', 8),\n",
       " ('G,H', 1),\n",
       " ('G,H.', 1),\n",
       " ('G-H', 1),\n",
       " ('G-I', 3),\n",
       " ('G-J', 1),\n",
       " ('G-K', 1),\n",
       " ('G.', 33),\n",
       " ('G:', 2),\n",
       " ('GA', 1),\n",
       " ('GFP', 1),\n",
       " ('GFP+', 1),\n",
       " ('GO', 3),\n",
       " ('GSEA', 7),\n",
       " ('Gene', 9),\n",
       " ('Glia', 1),\n",
       " ('H', 28),\n",
       " ('H)', 11),\n",
       " ('H,', 8),\n",
       " ('H-I.', 1),\n",
       " ('H-J', 1),\n",
       " ('H-J,', 1),\n",
       " ('H-J.', 1),\n",
       " ('H-K.', 1),\n",
       " ('H.', 25),\n",
       " ('H3R', 1),\n",
       " ('H:', 2),\n",
       " ('HOIP', 2),\n",
       " ('HT29', 1),\n",
       " ('Hcp', 1),\n",
       " ('HeLa', 5),\n",
       " ('Heat', 6),\n",
       " ('High', 2),\n",
       " ('I', 19),\n",
       " ('I)', 5),\n",
       " ('I,', 4),\n",
       " ('I,J', 2),\n",
       " ('I-K', 1),\n",
       " ('I-L.', 1),\n",
       " ('I-M', 1),\n",
       " ('I.', 16),\n",
       " ('I:', 1),\n",
       " ('IB', 10),\n",
       " ('IF', 3),\n",
       " ('IFNγ', 1),\n",
       " ('IHC', 1),\n",
       " ('IL-8', 1),\n",
       " ('IP', 1),\n",
       " ('IgA', 1),\n",
       " ('In', 20),\n",
       " ('J', 18),\n",
       " ('J)', 1),\n",
       " ('J,', 1),\n",
       " ('J-K.', 1),\n",
       " ('J-L', 2),\n",
       " ('J-N', 1),\n",
       " ('J.', 8),\n",
       " ('JNK', 1),\n",
       " ('K', 14),\n",
       " ('K+', 1),\n",
       " ('K,', 1),\n",
       " ('K-L.', 1),\n",
       " ('K-N.', 1),\n",
       " ('K.', 5),\n",
       " ('L', 9),\n",
       " ('L)', 1),\n",
       " ('L,', 2),\n",
       " ('L-M', 1),\n",
       " ('L-N', 1),\n",
       " ('L-N.', 1),\n",
       " ('L-O.', 1),\n",
       " ('L.', 3),\n",
       " ('LC3', 1),\n",
       " ('LDH', 6),\n",
       " ('LSK', 1),\n",
       " ('Lack', 2),\n",
       " ('Lam-', 1),\n",
       " ('Left', 6),\n",
       " ('Life', 1),\n",
       " ('Live', 3),\n",
       " ('Loss', 2),\n",
       " ('Lost', 2),\n",
       " ('Low', 1),\n",
       " ('Lung', 4),\n",
       " ('M', 2),\n",
       " ('M,', 1),\n",
       " ('M-N.', 1),\n",
       " ('M-O', 1),\n",
       " ('M.', 2),\n",
       " ('M.tb', 6),\n",
       " ('MEFs', 2),\n",
       " ('MT', 1),\n",
       " ('Male', 24),\n",
       " ('Mass', 3),\n",
       " ('Mavs', 3),\n",
       " ('Mean', 1),\n",
       " ('Mice', 5),\n",
       " ('N', 5),\n",
       " ('N)', 1),\n",
       " ('N,', 1),\n",
       " ('N-O.', 1),\n",
       " ('N.', 1),\n",
       " ('NFAT', 1),\n",
       " ('NOX2', 1),\n",
       " ('Na+', 1),\n",
       " ('O,', 2),\n",
       " ('O-Q', 2),\n",
       " ('O.', 2),\n",
       " ('ONX', 1),\n",
       " ('On', 1),\n",
       " ('One', 4),\n",
       " ('Our', 1),\n",
       " ('P', 1),\n",
       " ('P-Q.', 1),\n",
       " ('P.', 2),\n",
       " ('PCA', 1),\n",
       " ('PCR', 1),\n",
       " ('PI', 2),\n",
       " ('Peak', 2),\n",
       " ('PenG', 1),\n",
       " ('Plot', 3),\n",
       " ('Q', 3),\n",
       " ('R', 2),\n",
       " ('R-T', 1),\n",
       " ('R.', 2),\n",
       " ('RAR', 1),\n",
       " ('RNA', 4),\n",
       " ('RNAi', 1),\n",
       " ('Real', 4),\n",
       " ('Role', 1),\n",
       " ('S', 1),\n",
       " ('S.', 1),\n",
       " ('SDS', 3),\n",
       " ('SIRF', 2),\n",
       " ('SPP1', 5),\n",
       " ('STED', 1),\n",
       " ('Same', 1),\n",
       " ('Sex', 3),\n",
       " ('Six', 1),\n",
       " ('Spot', 1),\n",
       " ('TEM', 6),\n",
       " ('TLN', 3),\n",
       " ('TNC', 1),\n",
       " ('TNF', 1),\n",
       " ('TSLP', 2),\n",
       " ('The', 84),\n",
       " ('Time', 4),\n",
       " ('To', 1),\n",
       " ('Top', 2),\n",
       " ('Top,', 1),\n",
       " ('Top:', 4),\n",
       " ('Two', 4),\n",
       " ('U', 1),\n",
       " ('V', 2),\n",
       " ('VCaP', 2),\n",
       " ('Venn', 3),\n",
       " ('W', 1),\n",
       " ('WB', 2),\n",
       " ('WCLs', 1),\n",
       " ('WT', 13),\n",
       " ('We', 3),\n",
       " ('Week', 6),\n",
       " ('When', 1),\n",
       " ('Wild', 1),\n",
       " ('Wip1', 3),\n",
       " ('ZIKV', 1),\n",
       " ('a)', 8),\n",
       " ('a,', 26),\n",
       " ('a-e)', 5),\n",
       " ('a.', 3),\n",
       " ('b)', 3),\n",
       " ('b,', 12),\n",
       " ('b.', 2),\n",
       " ('c)', 4),\n",
       " ('c,', 15),\n",
       " ('c.', 1),\n",
       " ('d)', 4),\n",
       " ('d,', 13),\n",
       " ('d-f,', 2),\n",
       " ('d-h)', 1),\n",
       " ('d.', 1),\n",
       " ('e)', 1),\n",
       " ('e,', 9),\n",
       " ('e.', 1),\n",
       " ('f)', 2),\n",
       " ('f,', 6),\n",
       " ('fold', 1),\n",
       " ('g)', 1),\n",
       " ('g,', 10),\n",
       " ('h)', 1),\n",
       " ('h,', 2),\n",
       " ('i,', 1),\n",
       " ('in', 1),\n",
       " ('left', 4),\n",
       " ('mRNA', 11),\n",
       " ('mice', 1),\n",
       " ('qPCR', 4),\n",
       " ('s.', 1),\n",
       " ('sulA', 1),\n",
       " ('tSNE', 2),\n",
       " ('the', 2),\n",
       " ('with', 1)]"
      ]
     },
     "execution_count": 194,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import numpy as np\n",
    "counter = Counter(first_chars)\n",
    "print(len(lines), np.array(list(counter.values())).sum())\n",
    "sorted(counter.items())\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "067e90fd",
   "metadata": {},
   "source": [
    "We can see a great variability in the definition of panels in the figure caption. We also see cases on which the text is not available for the panel identification. It will be important to find the best way to train the dataset in this sense. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 196,
   "id": "768488d9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'<sd-panel panel_id=\"60831\">(D-F) <sd-tag id=\"sdTag413\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"Immunohistological\" ext_ids=\"BAO_0000415\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"immunohistochemistry\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">Immunohistological</sd-tag> analysis of <sd-tag id=\"sdTag414\" source=\"sdapi\" category=\"None\" entity_type=\"tissue\" role=\"component\" text=\"splenic\" ext_ids=\"UBERON:0002106\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"spleen\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://www.ebi.ac.uk/ols/ontologies/uberon/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_\">splenic</sd-tag> sections <sd-tag id=\"sdTag415\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"stained\" ext_ids=\"OBI_0302887\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"staining\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ontobee.org/ontology/OBI?iri=http://purl.obolibrary.org/obo/\">stained</sd-tag> with mAb against <sd-tag id=\"sdTag416\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"MOMA1\" ext_ids=\"Q78IK4\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Apool\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"https://www.uniprot.org/uniprot/\">MOMA1</sd-tag>/<sd-tag id=\"sdTag417\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"CD169\" ext_ids=\"Q62230\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Siglec1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"https://www.uniprot.org/uniprot/\">CD169</sd-tag> (blue), <sd-tag id=\"sdTag418\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"GFP\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">GFP</sd-tag>/<sd-tag id=\"sdTag419\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"reporter\" text=\"EYFP\" ext_ids=\"\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"\">EYFP</sd-tag> (green), whose <sd-tag id=\"sdTag420\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"expression\" ext_ids=\"OBI_0000615\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"protein expression profiling\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ontobee.org/ontology/OBI?iri=http://purl.obolibrary.org/obo/\">expression</sd-tag> correlates with that of <sd-tag id=\"sdTag421\" source=\"sdapi\" category=\"None\" entity_type=\"protein\" role=\"assayed\" text=\"IFN-&#946;\" ext_ids=\"P01575\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Ifnb1\" ext_tax_ids=\"10090\" ext_tax_names=\"Mus musculus\" ext_urls=\"https://www.uniprot.org/uniprot/\">IFN-&#946;</sd-tag>, and <sd-tag id=\"sdTag427\" source=\"sdapi\" category=\"None\" entity_type=\"organism\" role=\"assayed\" text=\"MCMV\" ext_ids=\"12138\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Maize chlorotic mottle virus\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ncbi.nlm.nih.gov/taxonomy/\">MCMV</sd-tag> IE-1 (red). Scale bars, 100&#181;m. The <sd-tag id=\"sdTag423\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"image\" ext_ids=\"BAO_0040001\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"image analysis method\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">image</sd-tag> shown is one representative of 15 obtained from 5 different <sd-tag id=\"sdTag424\" source=\"sdapi\" category=\"None\" entity_type=\"organism\" role=\"component\" text=\"mice\" ext_ids=\"10090\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"Mus musculus\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"http://www.ncbi.nlm.nih.gov/taxonomy/\">mice</sd-tag>. Optical magnification was 20x, and <sd-tag id=\"sdTag425\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"images\" ext_ids=\"BAO_0040001\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"image analysis method\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">images</sd-tag> were scanned on the <sd-tag id=\"sdTag426\" source=\"sdapi\" category=\"assay\" entity_type=\"None\" role=\"None\" text=\"microscope\" ext_ids=\"BAO_0002526\" ext_dbs=\"\" in_caption=\"True\" ext_names=\"microscopy\" ext_tax_ids=\"\" ext_tax_names=\"\" ext_urls=\"https://bioportal.bioontology.org/ontologies/BAO/?p=classes&amp;conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23\">microscope</sd-tag> at zoom x2 for (E, F). White dotted lines delimit marginal zone. Yellow squares in (D) defines zoomed region in (E, F). <graphic href=\"https://api.sourcedata.io/file.php?panel_id=60831\"/></sd-panel>\\n'"
      ]
     },
     "execution_count": 196,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "line"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ad2cc9f0",
   "metadata": {},
   "source": [
    "Let us combine the panel and figure information to create a training set where we know that the panels belong to the figures."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 226,
   "id": "52d0af6a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ".\n",
      "Figure 6\n",
      "None\n",
      "A, B Triple \n",
      "Triple \n",
      "Triple \n",
      "G Quantification of \n",
      "H Quantification of double-positive \n"
     ]
    }
   ],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_fig/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[2]) + \".txt\"\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "# first_chars = []\n",
    "# for line in lines: \n",
    "#     xml_example = fromstring(line)  \n",
    "#     xml_encoder = XMLEncoder(xml_example)\n",
    "#     text = []\n",
    "#     for element in xml_encoder.element.itertext():\n",
    "#         text.append(element)\n",
    "#     ids = \"\".join(text).split()[0]\n",
    "#     if len(ids) < 5:\n",
    "#         first_chars.append(ids)\n",
    "\n",
    "line = lines[7]\n",
    "xml_example = fromstring(line)\n",
    "xml_encoder = XMLEncoder(xml_example)\n",
    "test = xml_example.xpath(\"*\")\n",
    "for item in test:\n",
    "    print(item.text)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1a5de610",
   "metadata": {},
   "source": [
    "from collections import Counter"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 245,
   "id": "a38c0c9a",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ".Figure 1A Intracellular localization of total (HA) and polymeric ATZ (2C1) in WT MEF mock-treated, Confocal Laser Scanning Microscopy (CLSM). B Same as A for MEF exposed to 50 nM BafA1 for 12 h. C Same as A, 4 h after BafA1 wash-out. D Quantification of ATZ-positive, LAMP1-positive endolysosomes (EL) (n=13, 10, 11 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. Data information: Scale bars: 10 μm. E ATZ polymers immunoisolated from lysates of WT MEF mock treated (lane 1), incubated for 12 h with BafA1 (lane 2) and 4 h after BafA1 wash-out (lane 3). Immunoprecipitation (IP) of ATZ polymers with polymer-specific 2C1 antibody, transfer on PVDF membrane, revealed with anti-HA antibody on western blot (WB). F Quantification of E, n=3, mean ± SEM. Unpaired two-tailed t-test, ns P>0.05, * P<0.05. Data information: Scale bars: 10 μm. G-K Same as B in WT MEF, in cells exposed to 20 mM CST and in Cnx-, Crt- and ERp57-KO MEF. L Quantification of ATZ-positive EL (n=10, 9, 11, 10, 11 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. Data information: Scale bars: 10 μm. \n",
      ".Figure 3A WB analysis showing KO efficiency for CRISPR134B MEF. WB analysis showing KO efficiency for CRISPR62 MEF. C-E CRISPR WT MEF, CRISPR134B MEF, CRISPR62 MEF, respectively. Scale bars: 10 μm. F Quantification of C-E (n=10, 8, 7 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. \n",
      ".Figure 2A HEK293 cells transfected with empty vector (lanes 1, 6), ATZ-HA (2, 7), FAM134B-V5 (3 and 8), FAM134B-V5 and ATZ-HA (4, 9), or FAM134BLIR-V5 and ATZ-HA (5, 10), incubated for 12 h with 50 nM BafA1 and then treated with the cross-linker DSP before lysis as described in Methods. Lanes 1-5, WB of the total cell extract (TCE); lanes 6-10, WB of antiV5 immunoprecipites to isolate complexes containing ectopically expressed FAM134B or FAM134BLIR. The membranes were probed with antiV5 (upper panels), antiCNX, antiHA and antiLC3 antibodies. B Quantification of LC3 co-precipitating with FAM134B-V5 (A, lanes 8 and 9). Mean ± SEM, n=3, unpaired two-tailed t-test, * P<0.05. WT MEF. Scale bar: 10 μm. D Ectopic expression of FAM134BLIR in WT MEF inhibits ATZ delivery to endolysosomes. Scale bar: 10 μm. E, F Distribution of gold-labeled ATZ-HA by IEM in BafA1 treated WT MEF and WT MEF overexpressing FAM134BLIR, respectively. EV, ER-derived vesicles, red arrowheads; EL, endolysosome. G Quantification of ATZ-gold density of E, F (n=75 and 79 EL, respectively). Unpaired two-tailed t-test, **** P <0.0001. H Max projection of the same cell as in D after deconvolution. Insets show orthogonal section of select regions. Scale bar: 10 μm. I Decay of ATZ polymers (CHX chase, upper panel) immunoisolated with the polymer-specific 2C1 antibody (visualized with antiHA in WB) in HEK293 cells mock-transfected (lanes 1-4) or expressing FAM134BLIR-V5 (lanes 5-8). Middle panel, expression of FAM134BLIR-V5 assessed by WB; lower panel, loading control. (J) Quantification of (I) (Mean ± SEM, n=3, unpaired two-tailed t-test, * P<0.05). K Flow cytometry analysis of ATZ-HA polymer levels in MEFs mock treated, exposed to BafA1, and co-expressing FAM134LIR. MFI: mean fluorescence intensity (Mean ± SEM, n=5, unpaired two-tailed t-test, ns P>0.05, ** P<0.01, *** P <0.001). \n",
      ".Figure 4A-H Intracellular localization of LAMP1-positive EL, total (HA) and polymeric (2C1) ATZ in WT MEF exposed to BafA1 for 12 h (A); in Atg4BKO MEF (B); in Atg7KO MEF (C); in Fip200KO MEF (D); in Ulk1/2 double-KO MEF (E); in Atg13KO MEF (F); in Atg9KO MEF (G); in RubiconKO MEF (H). Scale bars: 10 μm. I Quantifications of A-H (n=10, 9, 9, 10, 7, 15, 9, 12 cells). One-way ANOVA and Dunnett's multiple comparisons test, ns P>0.05, **** P<0.0001. \n",
      "Accumulation of α-Synuclein as a Consequence of Impaired Quality Control.Figure 2(A) Levels of soluble and insoluble α-synuclein from midbrain of P1 gba+/+ and gba−/− mice. TX-100 soluble and insoluble (SDS/Urea) factions were isolated and analyzed via western blotting with α-synuclein antibodies. β-actin was used as a loading control. (B) Sagittal brain stem sections from gba+/+, gba+/−, and gba−/− mice were stained with α-synuclein antibodies. Scale bar, 1,000 μm. \n",
      ".Figure 5A Distribution of gold-labeled ATZ-HA by IEM in WT MEF exposed to BafA1 (the cell on the right is not transfected, as specificity control for gold labeling). ER, endoplasmic reticulum; EV, ER-derived vesicle; EL, endolysosomes. Scale bar: 1 μm. B Inset of A. Red asterisks show site of EV:EL membranes fusion C-E Distribution of gold-labeled ATZ-HA by IEM in WT MEF. ES, ER subdomains. Red asterisk shows site of EV:EL membranes fusion. Scale bar: 1 μm. F 3D visualization by means of electron tomography of organelles containing gold labeled ATZ-HA (visualized as white dots ES, ER subdomains. Scale bar: 500 nm. G CLEM showing ATZ (red), LAMP1-positive EL (cyan) and GFP-derived fluorescence (green) in correspondence to heavily labeled vesicles, which were identified as autophagosomes for their double membrane (insets 1-3). GFP-LC3 was also stained by immunogold. Scale bar: 1 μm. \n",
      "Loss of gba Impairs Both Autophagic and Proteasomal Degradation Machinery.Figure 1(A) Autophagic markers from mixed midbrain gba+/+, gba+/−, and gba−/− cultures were analyzed via western blotting using antibodies as indicated. (B) Autophagic flux was analyzed in gba+/+ and gba−/− midbrain neurons. Neurons were treated with 100 nM bafilomycin A1 and 1 μM rapamycin and flux assayed by comparing LC3I and LC3II levels via western blotting. β-actin was used as a loading control.(C) Densitometry analyzes of autophagy of (B) expressed as a ratio of LC3II/LC3I. Error bars, ± SEM. *p < 0.05. (D) Accumulation of p62/SQSTM1 was analyzed via immunoblotting. β-actin was used as a loading control. (E) Midbrain neurons were treated with either 10 μM MG132 or vehicle control DMSO. Proteins were extracted and analyzed via western blotting and stained with anti-ubiquitin antibodies. (F) Ubiquitin linkage of isolated midbrain analyzed using western blotting with antibodies as indicated. β-actin was used as a loading control. (G) Proteasomal activity (chymotrypsin activity) was measured using the Proteasome Glo assay where activity is proportional to the released luciferase. RFU, relative luciferase units. Data represent the mean ± SEM (n = 3, each experiment contained triplicates for each genotype). *p < 0.05. \n",
      ".Figure 6A, B Triple staining of cells with LAMP1 (cyan), HaloTag-ATZ (red) and the luminal ER marker sfGFP-KDEL (green). Triple staining of cells with LAMP1 (cyan), HaloTag-ATZ (red) and with CNX as ER membranes marker. Triple staining of cells with LAMP1 (cyan), HaloTag-ATZ (red) and with CLIMP63 as ER membranes marker. G Quantification of ER markers and HaloTag-ATZ co-localization in time (sfGFP-KDEL n=10, 10, 12, 13, CNX n=8, 10, 9, 8, CLIMP63 n=8, 10, 6, 10 cells for corresponding time points). Mean ± SEM, unpaired two-tailed t-test, **** P<0.0001. Data information: Scale bars 10 μm. H Quantification of double-positive endolysosomes at 300' (ATZ:sfGFP-KDEL n=14, ATZ:CNX n=8, ATZ:CLIMP63 n=10 cells. One-way ANOVA and Dunnett's multiple comparisons test, ** P<0.01 **** P<0.0001). Data information: Scale bars 10 μm. \n",
      ".Figure 7A KO efficiency in CRISPR17 MEF. KO efficiency in CRISPR8. C-E CLSM analysis of ATZ delivery to LAMP1-positive endolysosomes in BafA1-treated CRISPRWT MEF (C); in CRISPR17 MEF(D); in CRISPR8 MEF (E). Scale bars: 10 μm. Quantification (n=10, 10, 11 cells, respectively). One-way ANOVA and Dunnett's multiple comparisons test, **** P<0.0001. G-I Distribution of gold-labelled ATZ-HA by IEM in CRISPRWT MEF exposed to BafA1 (G); in CRISPR17 (H); in CRISPR8. (I). ER, endoplasmic reticulum; EV, ER-derived vesicles; EL, endolysosomes. Scale bars: 1 μm. GFP reconstitution in ATZ-HA transfected CRISPR17 MEF expressing FAM134B-T11 and GFP1-10-LC3. Scale bar: 10 μm. K Same as J by CLEM. ATZ-HA staining in red, LAMP1-positive endolysosomes in cyan. FAM134B:LC-induce GFP reconstitution in green. FAM134B in gold is shown. Scale bar: 1 μm. L ERLAD schematics. 1. Synthesis and CNX-independent polymers formation; 2. CNX-regulated polymers segregation in ER subdomains (ES); 3. FAM134B-dependent (LIR-independent) formation of ER-derived vesicles (EV); 4. FAM134B:LC3II-mediated EV docking to RAB7/LAMP1-positive endolysosomes (EL); 5. STX17/VAMP8-controlled fusion of EV:EL membranes and polymers release in the EL lumen for clearance. \n",
      "Mitochondrial Physiology Is Affected in gba−/− Neurons and Astrocytes.Figure 3(A) Neurons and astrocytes from gba+/+, gba+/−, and gba−/− mice were stained with TMRM. The mean florescence intensity in mitochondria was analyzed via confocal microscopy (n = 3, >32 cells analyzed/experiment). (B) Neurons from gba+/+ and gba−/− bathed in TMRM containing recording solution and fluorescence intensity analyzed via confocal microscopy. After 1 min, 1 μM oligomycin was added followed by 1 μM FCCP (n = 5, three cells analyzed/experiment). (C) Neurons from gba+/+ and gba−/− incubated with 5 mM methyl pyruvate 5 min prior to imaging. Cells bathed in 25 nM TMRM and 5 mM methyl pyruvate containing recording solution and fluorescence intensity analyzed via confocal microscopy. After 90 s, 1 μM oligomycin was added, followed by 1 μM FCCP (n = 3, three cells analyzed/experiment). (D) As in (C), except cells were bathed in 10 mM methyl succinate, (n = 3, four cells/experiment). All data in this figure represent the mean ± SEM, *p < 0.05. Neurons from gba+/+ and gba−/− incubated with 5 mM methyl pyruvate 5 min prior to imaging. Cells bathed in 25 nM TMRM and 5 mM methyl pyruvate containing recording solution and fluorescence intensity analyzed via confocal microscopy. After 90 s, 1 μM oligomycin was added, followed by 1 μM FCCP (n = 3, three cells analyzed/experiment). \n",
      "Dysfunctional Quality Control in Gaucher Disease With loss of gba, lysosome function is severely impaired, and autophagy does not occur.Figure 7Dysfunctional Quality Control in Gaucher DiseaseWith loss of gba, lysosome function is severely impaired, and autophagy does not occur. Dysfunctional organelles accumulate, including mitochondria. Due to loss of gba and the self-propagation of α-synuclein, the autophagic pathway is disrupted, compromising UPS function. The damaged mitochondrial respiratory chain cannot support the potential, which decreases to a level at which the ATPase reverses. A decreased ΔΨm limits their ability to re-enter the fission-fusion cycle; this low ΔΨm, maintained by the ATPase, appears to be sufficient in maintaining a potential above the critical level required to recruit Parkin, and thus mitochondria are not flagged for turnover, amplifying the defect. \n",
      ".Figure 1(A) rBCV biogenesis in BMMs infected with either wild type (2308), VirB-deficient (ΔvirB11), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria, measured as the percentage of LAMP1-positive BCVs over time. Data are means ± SD of n=3 independent experiments. Asterisks indicate statistically significant differences (P<0.05, two-way ANOVA followed by Dunnett's multiple comparisons test) compared to control (2308). (B) Brucella replication in BMMs infected with either wild type (2308), VirB-deficient (ΔvirB11), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria, measured as number of bacteria per cell at 24 h pi. Data are means ± SD of n=3 independent experiments. Grey dots represent values from individual cells analyzed (n>300); black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, one-way ANOVA followed by Dunnett's multiple comparisons test) compared to control (2308). (C) Brucella replication in BMMs expressing either GFP or GFP-BspF and infected with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria, measured as number of bacteria per cell at 24 h pi. Data are means ± SD of n=3 independent experiments. Grey dots represent values from individual cells analyzed (n>300); black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, two-way ANOVA followed by Dunnett's multiple comparisons test) compared to controls. \n",
      "Mitochondrial Morphology Is Affected in gba−/− Cells.Figure 5(A) Neurons and astrocytes were immunostained for cytochrome c and stained with Hoechst. Far left panels were additionally immunostained for GFAP and MAP2 to mark neurons and astrocytes. \"A\" in nucleus denotes astrocytes. Scale bar, 20 μm.(B) Mitochondrial morphology was blind counted from cells in (A) with neurons scored as MAP2 positive and astrocytes as GFAP positive. A \"normal\" morphology is classified as a mixture of fused and fragmented mitochondria that comprise a wild-type network. (n = 3; 100 cells counted/experiment). (C) Levels of morphology proteins from gba+/+, gba+/−, and gba−/− isolated mitochondria were analyzed via immunoblotting with the indicated antibodies. (D) Midbrain gba+/+, gba+/−, and gba−/− astrocytes expressing GFP-DRP1K38A were immunostained for cytochrome c and mitochondrial morphologies analyzed; concurrently cells were stained with TMRM and fluorescence intensity analyzed via confocal microscopy. Scale bar, 20 μm (n = 3, >15 cells/experiment). All data in this figure represent the mean ± SEM. \n",
      "gba−/− Mitochondria Have a Reduced Respiratory Chain Activity.Figure 4(A) Oxygen consumption rates of gba+/+, gba+/−, and gba−/− mixed midbrain cultures. Basal oxygen consumption was measured over 3 min. The maximal (uncoupled) rate was measured via the addition of FCCP and the nonmitochondrial oxygen consumption analyzed via the addition of antimycin A (n = 3, three runs/experiment). (B) Complex I-NADH: Ubiquinone reductase activity from gba+/+, gba+/−, and gba−/− brains expressed as a ratio to citrate synthase (n = 3). (C) Complex II-III: Succinate dehydeogenase, cytochome c reductase activity from gba+/+, gba+/−, and gba−/− brains expressed as a ratio to citrate synthase (n = 3). (D) Complex IV activity from gba+/+, gba+/−, and gba−/− brains expressed as a ratio to citrate synthase (n = 3). (E) Neurons were treated with 3 nM decylTPP and MitoQ10 for 48 hr and stained with TMRM and fluorescence intensity analyzed via confocal microscopy (n = 3, >16 cells/experiment). (F) gba+/+and gba−/− cells treated with 3 nM decylTPP and MitoQ10 for 48 hr were subjected to immunoblotting using the indicted antibodies. All data in this figure represent the mean ± SEM, *p < 0.05. \n",
      "Impaired Mitophagy in gba−/− Neurons and Astrocytes.Figure 6(A) Midbrain astrocytes were transfected with GFP-LC3, treated with 100 nM bafilomycin A1, and stained with MitoTracker Red and imaged using confocal microscopy. (B) Midbrain gba+/+ and gba−/− neurons and astrocytes were transfected with YFP-Parkin. Half the transfected cells were treated for 1 hr with 10 μM FCCP and all bathed in TMRM containing recording solution and imaged using confocal microscopy. Scale bar, 20 μm.(C) TMRM fluorescence intensity of cells from (B) was analyzed via confocal microscopy. Data represent the mean ± SEM, (n = 3, >4 cells analyzed per experiment). (D) Mitochondria within midbrain neurons were uncoupled by the addition of 10 μM FCCP for 1 and 6 hr. PINK1 expression analyzed via immunoblotting on isolated mitochondria from gba+/+ and gba−/− neurons. Mitochondrial CI subunit NDUFS4 was used as a loading control. \n",
      ".Figure 4(A) Yeast two-hybrid mating screen showing interaction of BspF with a fragment of ACAP1 (amino-acid residues 460-740; ACAP1Lib) or full-length ACAP1 (ACAP1FL), compared to positive (p53/T antigen) and negative (empty vectors) control matings plated on permissive double dropout (DDO) or selective quadruple dropout (QDO) media. ACAP1 schematic indicates the region of interaction initially identified (ACAP1Lib). (B) Representative co-immunoprecipitations of HA-BspF and myc-ACAP1 in HeLa cells. HeLa cells were transfected to either co-produce or individually produce HA-BspF and myc-ACAP1 and either HA-BspF or myc-ACAP1 were immunoprecipitated using either anti-HA-conjugated (upper panel) or anti-myc-conjugated (lower panel) magnetic beads following cross-linking (+DSP) or not (-DSP) with dithiobis[succinimidylpropionate]. Input lysates (10% of the post-nuclear supernatant) and co-immunoprecipitates were separated by SDS-PAGE and probed for HA-BspF and myc-ACAP1 by Western blotting. (C) Representative confocal micrographs of HeLa cells transfected to produce mCherry-BspF and GFP-ACAP1 and treated with Cytochalasin D (200 nM) for 30 min and quantification of colocalization between mCherry-BspF and GFP-ACAP1. Arrows indicate areas of BspF and ACAP1 colocalization. Scale bars: 10 µm and 2 µm (insets). Data are means ± SD from n=3 independent experiments in which 10 cells were analyzed per experiment. Pearson's correlation coefficients were calculated from whole cells using NIH Fiji image analysis software and Coloc_2 plug-in. \n",
      ".Figure 6(A) Representative confocal micrographs of BMMs infected with either wild type (2308), ∆bspF or or complemented ∆bspF (∆bspF::bspF) bacteria (red) for 24 h and immunostained for the TGN vesicular marker Stx6 (green) and GM130 (blue). Scale bars, 10 µm and 2 µm (insets). Magnified insets show the association between Stx6-positive vesicles and rBCVs. (B) Recruitment of Stx6-positive vesicles to rBCVs (expressed as percentage of Stx6-positive BCVs) in BMMs infected for 24 h with either wild type (2308), ∆bspF or complemented ∆bspF (∆bspF::bspF) bacteria. Data are means ± SD from n=3 independent experiments, in which at least 300 BCVs were analyzed per experiment via CellProfiler image analysis. Asterisks indicate statistically significant differences compared to 2308-infected BMMs as determined by one-way ANOVA with Tukey's multiple comparisons test (P < 0.05). (C) Representative Western blot analysis of Arf6, Rab8a, Rab6a/a' and Stx6 depletions in BMMs following siRNA-mediated knockdowns, compared to non-targeting siRNA (siNT) treatments. β-actin was used as loading control. (D-F) Recruitment of Stx6-positive vesicles to rBCVs in BMMs treated with non-targeting siRNAs (siNT), siRNAs against Arf6 (siArf6) (D), Rab8a (siRab8a) (E) or Rab6a/a' (siRab6a/a') (F) and infected for 24 h with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria. Data are means ± SD of n=3 independent experiments, in which 200 BCVs were analyzed per experiment. Asterisks indicate statistically significant differences (P<0.05, one-way ANOVA followed by Dunnett's multiple comparisons test) between test and control conditions; ns, not significant. (G) Quantification of CTxB retrograde transport in BMMs following siRNA-mediated depletion of Stx6 (siStx6) after AlexaFluor™488-CTxB binding on ice followed by 30 min incubation at 37˚C. Data are means ± SD from n=3 independent experiments, in which 100 cells were analyzed per experiment. The asterisk indicates a statistically significant difference (P=0.0114, unpaired Student's t test) compared to the siNT control. (H) Brucella replication in BMMs treated with either non-targeting siRNAs (siNT), or siRNAs against Stx6 (siStx6) and infected for 24 h with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria. Data are means ± SD of n=3 independent experiments, in which at least 100 cells were analyzed per experiment. Grey dots represent individual cells analyzed (n>300); black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, one-way ANOVA followed by Dunnett's multiple comparisons test) between test and control conditions. (I) Model of BspF remodeling of TGN-RE membrane traffic. Bacterially delivered BspF targets RE membranes where it binds ACAP1 and promotes inactivation of Arf6. Increased turnover of active Arf6 results in inhibition of the Arf6/Rab8a cascade and retrograde RE-TGN transport, which alters TGN-derived vesicular traffic and redirects Stx6-positive vesicles to rBCVs in a process that promotes intravacuolar bacterial growth. \n",
      ".Figure 2(A) Quantification of ss-eGFP-FKBPF36M trafficking in HeLa(M)-C1 cells transfected for 24 h with pmCherry (mCherry) or pmCherry-BspF (mCherry-BspF). Rapamycin was added to initiate secretory traffic of ss-eGFP-FKBPF36M and its colocalization with calnexin (ER), ERGIC-53 (ERGIC), GM130 (Golgi), p230 (TGN), or secretory vesicles (SV) was scored over a 60 min time course. Data are means ± SD from n=3 independent experiments. Asterisks indicate statistically significant differences between mCherry- and mCherry-BspF-expressing cells as determined by a two-way ANOVA with Sidak's multiple comparisons test (P<0.05). (B) Representative confocal fluorescence micrographs of HeLa cells co-transfected for 24 h to produce GFP-TGN38 and mCherry-BspF and stained for F-actin with AlexaFluor™647-phalloidin. Cells were left untreated or treated with Cytochalasin D (200 nM) for 30 min prior to fixation. Scale bars: 10 and 1 µm (insets). (C) Quantification of colocalization between mCherry-BspF and GFP-TGN38 in untreated (-CytoD) and Cytochalasin D-treated (+CytoD) HeLa cells. Regions of Interests (ROI, representative shown as insets in panels B) were randomly selected and a Pearson's correlation coefficient was calculated using NIH Fiji image analysis software and Coloc_2 plug-in. Data are means ± SD from n=3 independent experiments in which 2 ROIs from 10 cells (n=20) were analyzed per experiment. The asterisk indicates a statistically significant difference between treatments as determined by a Mann-Whitney test (P<0.05). (D) Representative Western blot analysis of HeLa cells transfected for 24 h to produce HA-BspF, separated into saponin-, Triton X-100- and SDS-soluble fractions and probed for HA-BspF, Hsp27 (cytosol), Calnexin (membranes) and Lamin A/C (nucleus). (E) Schematic depicting key host proteins that control transport pathways associated with the TGN-RE-plasma membrane compartment. Protein colors depict their compartmentalized functions. (F) Representative confocal fluorescence micrographs of HeLa cells co-transfected for 24 h to produce mCherry-BspF and either GFP-Rab6a, GFP-Rab8a or Arf6-GFP and treated with Cytochalasin D (200 nM) for 30 min prior to fixation. Scale bars: 10 and 2 µm (insets). (G) Quantification of localization of GFP-Rab6a, GFP-Rab8a and Arf6-GFP on mCherry-BspF-labelled tubules in transfected HeLa cells. Data are means ± SD from n=3 independent experiments, in which at least 300 individual cells per experiment were analyzed. \n",
      ".Figure 5(A) Representative confocal micrograph of HeLa cells transfected to produce either mCherry (red), GFP-ACAP1 (green) and Arf6-HA (blue; left hand panels) or mCherry-BspF (red), GFP-ACAP1 (green) and HA-Arf6 (blue; right hand panels) and treated with Cytochalasin D (200 nM) for 30 min prior to fixation. Scale bars: 10 µm and 2 µm (insets). (B) Representative Western blot analysis of co-immunoprecipitations of myc-ACAP1 and Arf6-HA in the presence or absence of HA-BspF. HeLa cells were transfected to produce Arf6-HA and combinations of myc-ACAP1 and HA-BspF, or not, and myc-ACAP1 was immunoprecipitated using anti-myc-conjugated magnetic beads. Input lysates (6% of post-nuclear supernatants) and co-immunoprecipitates were separated by SDS-PAGE and probed for Arf6-HA, HA-BspF and myc-ACAP1 by Western blotting. Quantification of the Arf6/ACAP1 ratio was performed by densitometric analysis. Data are means ± SD of 3 independent experiments. The asterisk indicates a statistically significant difference (P = 0.0017, unpaired Student's t test) between BspF-producing and control conditions. (C) Quantification of Arf6 activity (GTP-Arf6) in HeLa cells transfected to produce either mCherry and Arf6-HA or mCherry-BspF and Arf6-HA by G-LISA. Data are means ± SD of n=3 independent experiments, normalized to mCherry-producing controls. The asterisk indicates a statistically significant difference (P = 0.0026, unpaired Student's t test) between BspF-producing and control conditions. (D) Bacterial replication in BMMs transduced to either produce GFP, Arf6Q67L-GFP or Arf6T27N-GFP and infected with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria for 24 h. Data are means ± SD of n=4 independent experiments, in which at least 100 cells were analyzed per experiment. Grey dots represent individual cells analyzed (n>300); black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, two-way ANOVA followed by Dunnett's multiple comparisons test) between test and control conditions. (E) Bacterial replication in BMMs transduced to either produce GFP, GFP-ACAP1 or GFP-ACAP1R448Q and infected with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria for 24 h. Data are means ± SD of n=3 independent experiments, in which at least 100 cells were analyzed per experiment. Grey dots represent individual cells analyzed (n>300); black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, two-way ANOVA followed by Dunnett's multiple comparisons test) between test and control conditions. \n",
      ": USP22 knockout (KO) decreases TBZ-induced necroptotic cell death in HT‑29 cells A-B HT‑29 cells were transfected with non-silencing control siRNA (sictr) or siRNAs against USP22 (siUSP22) for 48 h at 20 nM.Figure 1A-B HT‑29 cells were transfected with non-silencing control siRNA (sictr) or siRNAs against USP22 (siUSP22) for 48 h at 20 nM. After transfection, cells were treated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα either for 6 h and analyzed by Western blotting (A) or for 18 h, and the percentage of PI-positive cells was assessed by fluorescence-based PI staining (B). Vinculin served as a loading control. C HT‑29 CRISPR/Cas9 control (ctr) and USP22 KO cells were analyzed by Western blotting for USP22 expression. GAPDH served as loading control. D HT‑29 control and USP22 KO cells were treated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα after pre-incubation with 30 µM Nec1-s and 20 µM GSK'872 for 1 h and incubated for 18 h before fluorescence-based quantification of PI-positive cells. E HT‑29 control and USP22 KO cells, generated with 2 guide RNAs (2g), expressing empty vector (EV) or PAM mutated 3xFLAG-HA-USP22 WT (USP22 PAM), C185S (USP22 PAM C185S) or C185A (USP22 PAM C185A) were analyzed by Western blotting for USP22 expression levels. β-Actin was used as a loading control. F HT-29 control and USP22 KO cells, generated with 2 guide RNAs (2g), expressing empty vector (EV) or PAM mutated 3xFLAG-HA-USP22 WT (USP22 PAM), C185S (USP22 PAM C185S) or C185A (USP22 PAM C185A) were stimulated with 20 µM zVAD.fmk, 0.5 µM BV6, 1 ng/ml TNFα for 18 h. The percentage of PI-positive cells was assessed by fluorescence-based PI staining. \n",
      ".Figure 3(A) Representative confocal fluorescence micrographs of HeLa cells transfected for 24 h to produce either mCherry or mCherry-BspF (grayscale panels), incubated on ice with AlexaFluor™488-Cholera Toxin subunit B (CTxB; green) and shifted to 37˚C for 20 min to allow for CTxB retrograde transport to the Golgi apparatus (stained using an anti-GM130 antibody; purple). CTxB accumulation within Golgi structures appears white in overlays. Scale bars: 10 and 2 µm (insets). (B) Quantification of CTxB transport to the Golgi apparatus in HeLa cells producing either mCherry, mCherry-BspF or HA-BspF over a 30 min time course, expressed as percentages of cells in which CTxB colocalized with the GM130 Golgi marker, as in (A). Data are means ± SD from n=3 independent experiments, in which 100 cells were analyzed per experiment. Asterisks indicate statistically significant differences compared to mCherry-producing cells as determined by a two-way ANOVA with Tukey's multiple comparisons test (P<0.05). ( (C) Representative Western blot analysis of Arf6, Rab8a and Rab6a/a' depletions in BMMs following siRNA-mediated knockdowns, compared to non-targeting siRNA (siNT) treatments. β-actin was used as loading control. (D) Quantification of CTxB transport to the Golgi apparatus in BMMs following siRNA-mediated depletion of either Arf6 (siArf6), Rab8a (siRab8a) or Rab6a/a' (siRab6a/a') after AlexaFluor™488-CTxB binding on ice followed by 30 min incubation at 37˚C. Data are means ± SD from n=3 independent experiments, in which 100 cells were analyzed per experiment. Asterisks indicate a statistically significant difference compared to siNT control cells as determined by a one-way ANOVA with Tukey's multiple comparisons test (P<0.05). (E) Quantification of CTxB transport to the Golgi apparatus in BMMs that were either mock-infected or infected with wild type (2308), ΔbspF, complemented ∆bspF (ΔbspF::bspF), ∆bspB or complemented ∆bspB (ΔbspB::bspB) bacteria for 24 h, incubated for 30 min with AlexaFluor™488-CTxB on ice for binding followed by 30 min incubation at 37˚C. Data are means ± SD from n=3 independent experiments, in which 100 cells were analyzed per experiment. Asterisks indicate a statistically significant difference compared to mock-infected cells as determined by a one-way ANOVA with Tukey's multiple comparisons test (P<0.05). ( (F) rBCV biogenesis in BMMs treated with either non-targeting siNT, siRab6a/a' or siRab8a siRNAs and infected with wild type (2308) bacteria. Data are means ± SD of n=3 independent experiments, in which 100 BCVs were analyzed per experiment. Asterisks indicate statistically significant differences (P<0.05, two-way ANOVA followed by Dunnett's multiple comparisons test) compared to control (2308). (G-I) Brucella replication in BMMs treated with non-targeting siRNAs (siNT), or siRNAs against Rab6a/a' (siRab6a/a') (G), Rab8a (siRab8a) (H) or Arf6 (siArf6) (I) and infected with either wild type (2308), ΔbspF, or complemented ∆bspF (ΔbspF::bspF) bacteria, measured as number of bacteria per cell at 24 h pi. Data are means ± SD of n=3 independent experiments in which at least 100 cells were analyzed per experiment. Grey dots represent individual cells analyzed; black dots indicate means of individual experiments. Asterisks indicate statistically significant differences (P<0.05, one-way ANOVA followed by Dunnett's multiple comparisons test) between test and control conditions. \n",
      ": USP22 KO leads to increased TBZ-induced RIPK3 phosphorylation in HT‑29 cells A HT‑29 control and USP22 KO cells were stimulated with 20 µM zVAD.Figure 2A HT‑29 control and USP22 KO cells were stimulated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα for the indicated time points. Detection of indicated proteins was carried out by Western blotting. GAPDH served as a loading control. B HT‑29 control and USP22 KO cells were stimulated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα for 4 h. Detection of indicated proteins was carried out by Western blotting. β-Actin served as a loading control. C HT‑29 control and USP22 KO cells were incubated with 30 µM Nec1-s or 20 µM Dab for 18 h, as indicated. Cell were stimulated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα for 5 h. 100 μg of each lysate were incubated with 400 U/μl λ-phosphatase for 30 min at 30 °C. Protein expression of RIPK3 was monitored by Western blotting. β-Actin was used as loading control. High molecular weight RIPK3 'smears' were quantified after λ-phosphatase treatment and normalized to total RIPK3 and β-Actin levels. \n",
      ".Figure 1Apcfl/fl-Cdx2CreERT2 mice were treated with Tamoxifen and, beginning the day following treatment, injected with either anti-CD4 and anti-CD8 neutralizing antibodies (αCD4/αCD8, blue dots) or IgG control (black dots) twice a week for six weeks. FACS analysis of relative TCRβ+ T cell content in blood (left panel) and tumors (right panel) of mice at the end of treatments as indicated in A. αCD4/αCD8: n=10; IgG: n=6. Tumor burden (left panel), tumor number (middle panel) and average tumor size (right panel) per mouse assessed during colon resection at the end of treatments. αCD4/αCD8: n=10; IgG: n=9. \n",
      ": Identification of novel, USP22-regulated RIPK3 ubiquitination sites A Schematic representation of the experimental strategy for quantitative analysis of ubiquitination sites in USP22 KO HT-29 cells versus HT-29 CRISPR/Cas9 control (ctr) cells under untreated and necroptosis-induced conditions.Figure 5A Schematic representation of the experimental strategy for quantitative analysis of ubiquitination sites in USP22 KO HT-29 cells versus HT-29 CRISPR/Cas9 control (ctr) cells under untreated and necroptosis-induced conditions. Control cells were grown in SILAC medium containing Arg0/Lys0 (\"Light\") labeled, whereas USP22 KO cells were labeled in SILAC medium containing Arg10/Lys8 (\"heavy\") labeled. In the treated conditions, cells were incubated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα for 6 h. Cells were lysed, and equal amounts of extracted proteins extracted from \"Light\" or \"Heavy\" were pooled and digested in-solution with trypsin. Ubiquitin remnant peptides were enriched using di‐glycine‐lysine‐specific antibodies, fractionated by Micro‐SCX and analyzed by LC‐MS/MS. B Bar graphs demonstrating the number of up‐, non‐, and downregulated ubiquitination sites in USP22 KO HT-29 cells versus control under TBZ-treatment. C Schematic representation of RIPK3. Indicated are the two major protein domains, the kinase domain (KD), the RHIM and the three different potential ubiquitination sites identified by ubiquitin remnant profiling. D HeLa cells expressing Dox-inducible RIPK3 WT, D160N, K42R, K351, K518R, 2xKR or 3xKR were pre‑incubated with 1 µg/ml Dox, 20 µM Dab, or 10 µM NSA overnight, followed by pre-treatment with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 4 h. Cell death was measured by quantification of PI-positive nuclei. E HeLa cells expressing Dox-inducible RIPK3 mutants were treated with 1 µg/ml Dox overnight. Protein levels of inducible RIPK3 expression were analyzed by Western blotting. GAPDH served as loading control. \n",
      ": RIPK3 K518R hypersensitize HT‑29 cells for TBZ-induced necroptosis A HT‑29 RIPK3 KO cells re-expressing Dox-inducible WT RIPK3 or the indicated RIPK3 mutants were incubated with 1 µg/ml Dox overnight.Figure 7A HT‑29 RIPK3 KO cells re-expressing Dox-inducible WT RIPK3 or the indicated RIPK3 mutants were incubated with 1 µg/ml Dox overnight. Cells were treated with 20 µM zVAD.fmk, 5 µM BV6 and 10 ng/ml TNFα for 4 h. Cell death was measured by analysis of PI-positive nuclei. B HT‑29 RIPK3 KO cells re-expressing Dox-inducible WT RIPK3 or the indicated RIPK3 mutants were incubated with 1 µg/ml Dox overnight, as indicated. Cells were treated with 20 µM zVAD.fmk, 5 µM BV6 and 10 ng/ml TNFα for 2 h and analyzed by Western blotting for RIPK3 and phosphorylated MLKL expression levels. β-Actin served as a loading control. C Schematic overview of the putative mechanistic roles of USP22 in necroptosis. Activation of TNFR1 by TNFα, upon caspase-8 inhibition by zVAD.fmk and cIAP1/2 inactivation by BV6, induces RIPK1/3 activation, necrosome formation and execution of necroptosis. In the absence of USP22 KO, TBZ-induced necroptosis is delayed, accompanied by increased RIPK3 phosphorylation and ubiquitination of RIPK3 at lysine 518. This USP22-mediated necroptotic signaling could be caused either by direct USP22-mediated (de)ubiquitination of RIPK3 and/or indirectly by, for example, regulating RIPK3 autophosphorylation or the activity of RIPK3-associated E3 ligases or kinases. Additionally, USP22-mediated retrograde effects on RIPK1 phosphorylation and RIPK3 oligomerization might be involved as well. \n",
      ": Increased RIPK3 ubiquitination in USP22 KO HT-29 cells during TBZ-induced necroptosis A HT-29 control, USP22 KO and USP22 KO cells re-expressing PAM-mutated 3xFLAG-HA-USP22 WT or C185S were stimulated with 20 µM zVAD.Figure 4A HT-29 control, USP22 KO and USP22 KO cells re-expressing PAM-mutated 3xFLAG-HA-USP22 WT or C185S were stimulated with 20 µM zVAD.fmk, 0.5 µM BV6 and 1 ng/ml TNFα for 4 h. Poly-ubiquitinated proteins were enriched by GST-TUBE pull-down, followed by incubation with the catalytic domain of USP2, as indicated. RIPK3 and USP22 expression and levels of ubiquitinated RIPK3 were monitored using Western blotting with the indicated antibodies. β-Actin served as loading control. Ponceau-staining was used to confirm equal loading of GST-TUBE. B HT-29 control and USP22 KO cells were transfected with His-ubiquitin for 24 h, as indicated. Cells were pre‑stimulated with 20 µM zVAD.fmk, 0.5 µM BV6 for 1 h. Following pre-treatment, 1 ng/ml TNFα was added for 4 h. His-ubiquitin was immunoprecipitated using Ni-NTA beads and detection of indicated proteins was performed by Western blotting. β-Actin served as loading control for the input, whereas His-ubiquitin levels served as loading control for immunoprecipitated ubiquitin. C HeLa TRex RIPK3 control and USP22 KO cells were incubated with 1 µg/ml Dox and transfected with HA-ubiquitin for 24 h, as indicated. Cells were pre-stimulated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. Following pre-treatment, 10 ng/ml TNFα were added for 3 h. HA-ubiquitin was immunoprecipitated using anti-HA-beads and detection of indicated proteins was performed by Western blotting. β-Actin served as loading control for the input, whereas HA-levels served as loading control for immunoprecipitated ubiquitin. \n",
      ".Figure 5Evaluation of TGFβ signaling activity in mouse colon tumors after four consecutive daily injections of hosts with 40mg/kg of the MMP2/9-inhibitor SB3-CT (MMPi). , Co-immunostaining for pSMAD3 (green) and IGFBP7 (magenta) on representative tumor-containing colon sections derived from Apcfl/fl-Cdx2CreERT2 mice treated with vehicle (A) MMPi (B). Evaluation of TGFβ signaling activity in mouse colon tumors after four consecutive daily injections of hosts with 40mg/kg of the MMP2/9-inhibitor SB3-CT (MMPi). Quantification of pSMAD3+ cells (left panel) and pSMAD3 staining intensity per pSMAD3+ cell (right panel) on at least five arbitrarily selected tumor areas per section of MMPi (n=6) or vehicle (n=6) treated mice. effect of in vivo TGFBRi and MMPi treatment of mice for two weeks on colon tumor formation. D, Experimental setup. Apcfl/fl-Cdx2CreERT2 mice were treated with Tamoxifen and, one day post treatment, injected with either MMPi (n=10) or the TGFBRi SB431542 (n=10) at 40mg/kg for two weeks with five injections per week. Vehicle treated mice: n=11. effect of in vivo TGFBRi and MMPi treatment of mice for two weeks on colon tumor formation. Apcfl/fl-Cdx2CreERT2 mice were treated with Tamoxifen and, one day post treatment, injected with either MMPi (n=10) or the TGFBRi SB431542 (n=10) at 40mg/kg for two weeks with five injections per week. Vehicle treated mice: n=11. Average tumor size per mouse was assessed during colon resection one week after the end of treatments (E). Model of activation of latent TGFβ stored in extracellular matrix (ECM) by neutrophil-secreted MMP9, leading to suppression of effector T cells and promotion of Tregs. \n",
      ".Figure 3In vitro co-culture of activated T cells with increasing ratios of neutrophils, monocytes or macrophages. T cell proliferation index is numbers of proliferated T cells after three days of indicated co-culture condition relative to the number of proliferated T cells when cultured alone. CD4+ and CD8+ T cells were derived from lymph nodes of wild-type mice. Neutrophils, monocytes and macrophages were derived from colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Each dot represents an individual neutrophil (n=4), monocyte (n=3) or macrophage (n=4) sample. Bars represent mean +/- SD. Experimental setup. Eight weeks after Tamoxifen treatment of Apcfl/fl-Cdx2CreERT2 mice, animals were treated with anti-Gr1 antibody (αGr1, 3 times/week) plus CXCR2 inhibitor (CXCR2i, 5 times/week) or with IgG (3 times/week) plus DMSO control (5 times/week) for one to three weeks. Tumor neutrophil (left panel) and monocyte (right panel) content after αGr1 + CXCR2i (n=7) or IgG + DMSO (n=10) treatments. Average tumor volume per mouse at the end of αGr1 + CXCR2i (n=9) or IgG + DMSO (n=12) treatments as indicated in B. Anti-CD3 immunostaining on representative tumor sections derived from mice after indicated treatments. FACS analysis of CD45+ TCRβ+ cells, CD45+ TCRβ+ CD8+ cells and CD45+ TCRβ+ CD4+ Foxp3+ cells in extracted tumors after αGr1 + CXCR2i (n=8) or IgG + DMSO (n=7) treatments. FACS analysis of IFNγ (left panel) and IL17A (right panel) expression in tumor extracted T cells after αGr1 + CXCR2i (n=7) or IgG + DMSO (n=7) treatments. \n",
      ".Figure 4Measurement of cytokines present in cell culture supernatants of T cells co-cultured with either tumor-derived neutrophils (T + Ne; n=3) or with tumor-derived monocytes (T + Mo; n=3). Cytokines with a log2 fold change >0.6 or <-0.4 were considered as differentially present in supernatants. Shown is a heatmap of scaled cytokines levels (z-score) differentially present in T + Ne compared to T + Mo. Adjusted p-values were calculated by moderated t-test (limma) and Benjamini post test and were <0.05 only for MMP9 and RELMβ. Supernatants of T cells alone (T only; n=2) served as internal control. T cells cultured with monocytes did not have differential presence of cytokines compared to T cells alone. Co-immunostaining for MMP9 (green) and S100A9 (red) on a representative tumor section derived from an Apcfl/fl-Cdx2CreERT2 mouse. Right panel shows a higher magnification of indicated area of the left panel. mRNA expression levels of Mmp9 (C) or Tgfb1, Tgfb2 and Tgfb3 (D) in indicated FACS sorted cell types isolated from mouse colon tumors (n=4). Data are derived from RNA-sequencing and are displayed as reads per kilo base per million mapped reads (rpkm). Neutrophil - T cell co-cultures treated with vehicle (T + Ne), 10µm Galunisertib (T + Ne + TGFBRi), T cell proliferation after three days of indicated co-culture conditions. T cell proliferation index is numbers of proliferated T cells under indicated condition relative to the number of proliferated T cells when cultured alone. Neutrophil - T cell co-cultures treated with vehicle (T + Ne), 10µm of the MMP inhibitor GM6001 (T + Ne + MMPi) Protein levels of free active TGFβ1 in cell culture supernatants of indicated conditions determined by ELISA. Neutrophil - T cell co-cultures treated with vehicle (T + Ne), 10µm of the MMP inhibitor GM6001 (T + Ne + MMPi) T cell proliferation after three days of indicated co-culture conditions. T cell proliferation index is numbers of proliferated T cells under indicated condition relative to the number of proliferated T cells when cultured alone. Neutrophil - T cell co-cultures treated with vehicle (T + Ne), 10µm Galunisertib (T + Ne + TGFBRi), 10µm of the MMP inhibitor GM6001 (T + Ne + MMPi) or with 10µm Galunisertib plus 10µm GM6001 (T + Ne + TGFBRi + MMPi). T cell proliferation after three days of indicated co-culture conditions. T cell proliferation index is numbers of proliferated T cells under indicated condition relative to the number of proliferated T cells when cultured alone. \n",
      ": USP22 knockdown in RIPK3-expressing HeLa cells induces resistance to TBZ-induced necroptotic cell death A HeLa TRex RIPK3 CRISPR/Cas9 control (ctr) and USP22 KO cells were treated with 1 µg/ml Dox overnight.Figure 3A HeLa TRex RIPK3 CRISPR/Cas9 control (ctr) and USP22 KO cells were treated with 1 µg/ml Dox overnight. Protein expression of induced Strep-RIPK3 was analyzed by Western blotting. GAPDH served as loading control. The asterisk marks an unspecific band. B HeLa TRex RIPK3 control and USP22 KO cells were incubated with 1 µg/ml Dox for 18 h before pre-treatment with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα was added and cell death was measured after 4 and 5 h by analysis of PI-positive nuclei. C HeLa TRex RIPK3 control and USP22 KO cells were pre-treated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 1, 2, 3, 4 and 5 h. Protein expression of phosphorylated RIPK1, total RIPK1, total RIPK3, phosphorylated MLKL, total MLKL and USP22, without (left) or with (right) 1 µg/ml Dox treatment overnight, was monitored by Western blotting. GAPDH was used as a loading control. D HT-29 control cells and RIPK3 KO cells re-expressing PAM-mutated Dox-inducible RIPK3 WT were incubated overnight with 1 µg/ml Dox. Cells were pre-treated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 2 h, as indicated. Strep-RIPK3 was immunoprecipitated using anti‑Strep-beads and the indicated co-immunoprecipitated proteins were analyzed by Western blotting. β-Actin served as a loading control. E USP22 KO HT-29 cells and USP22 KO cells re-expressing PAM-mutated 3xFLAG-HA-USP22 were pre-treated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα was added for 2 h, as indicated. 3xFLAG-HA-USP22 was immunoprecipitated using anti-HA-beads and the indicated co-immunoprecipitated proteins were analyzed by Western blotting. β-Actin served as a loading control. \n",
      ": USP22-regulated modification of RIPK3 lysine residue 518 affects necroptosis in RIPK3-reconstituted HeLa cells A HeLa cells expressing Dox-inducible RIPK3 WT, K518R or 3xKR were treated with 1 µg/ml Dox and/or 10 µM NSA and 20 µM Dab overnight.Figure 6A HeLa cells expressing Dox-inducible RIPK3 WT, K518R or 3xKR were treated with 1 µg/ml Dox and/or 10 µM NSA and 20 µM Dab overnight. Protein levels of inducible RIPK3 expression and phosphorylated MLKL were examined by Western blotting. β-Actin was used as a loading control. B HeLa cells expressing Dox-inducible RIPK3 WT or 3xKR were treated with 1 µg/ml Dox overnight before pre-treatment with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 1, 2, 3 and 4 h. Protein levels of inducible RIPK3 expression and phosphorylated MLKL were analyzed by Western blotting. β-Actin served as loading control. C HeLa cells expressing Dox-inducible RIPK3 WT, 2xKR or 3xKR were treated with 1 µg/ml Dox overnight before pre-treatment with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 2 h. Protein levels of inducible RIPK3 expression were analyzed by Western blotting. β-Actin served as loading control. D HeLa cells expressing Dox-inducible RIPK3 WT, K518R or 3xKR were treated with 1 µg/ml Dox overnight before pre-treatment with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 2 h. Whole cell lysates were generated using RIPA lysis buffer containing 2% SDS. Protein levels of inducible RIPK3 expression were analyzed by Western blotting. β-Actin served as loading control. E HeLa cells expressing Dox-inducible RIPK3 WT, D160N, K518R or 3xKR were incubated overnight with 1 µg/ml Dox and pre-treated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 1 and 2 h. Strep-RIPK3 was immunoprecipitated by using anti-Strep-beads. Co-immunoprecipitated phosphorylated MLKL and RIPK1, as well as protein expression of indicated proteins were analyzed by Western blotting. β-Actin served as a loading control. F HeLa cells expressing Dox-inducible RIPK3 WT, K518R or 3xKR were incubated with 1 µg/ml Dox and transfected with HA-ubiquitin for 24 h, as indicated. Cells were pre-treated with 20 µM zVAD.fmk, 5 µM BV6 for 1 h. After pre-treatment, 10 ng/ml TNFα were added for 2 h. HA-ubiquitin was immunoprecipitated using anti-HA-beads and detection of indicated proteins was performed by Western blotting. β-Actin served as loading control for the input, whereas HA-ubiquitin levels served as loading control for immunoprecipitated ubiquitin. \n",
      ".Figure 2Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Relative TCRβ+ T cell content in colon (n=10) and tumor (n=22). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Anti-CD3 immunostaining on normal colon (left panel) and a representative colon tumor section derived from an Apcfl/fl-Cdx2CreERT2 mouse (right panel: higher magnification of area indicated in middle panel). Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Relative CD11b+ myeloid cell content in colon (n=16) and tumors (n=16). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Anti-Gr1 immunostaining on normal colon (left panel) and a representative colon tumor section derived from an Apcfl/fl-Cdx2CreERT2 mouse (right panel: higher magnification of area indicated in middle panel). Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. Relative CD11b+ MHCII- Gr1hi neutrophil (E) and CD11b+ MHCII- Gr1lo monocyte (F) content in colon (n=13) and tumors (n=16). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. Immune cell contexture in healthy colon and colon tumors of Apcfl/fl-Cdx2CreERT2 mice. , Correlation of tumor T cell numbers to sample-matched tumor-derived neutrophil (left panel) and monocyte (right panel) numbers (n=16). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. Percentage of T cell subsets among total CD45+ hematopoietic cells in colon (n=10) and tumor (n=11). H, CD4+ TCRβ+ T cells. I, CD8+ TCRβ+ T cells. J, Foxp3+ CD4+ TCRβ+ regulatory T cells (Tregs). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. Percentage of Tregs among total T cells (CD45+ TCRβ+) in normal colon (n=10) or colon tumors (n=11). relative content of individual cell types was determined in tissue-derived single cell suspension using cell type specific antibodies and FACS analysis. \n",
      ".Figure 1(A) Sequences of secreted Fc-tagged Nrxn1β variants analyzed. Mutated residues are shown in red, the HS modified serine in green, and cysteines forming the Cys-loop in blue. Right column ('HS?') summarizes the results from (B-C). Asterisks, colons, and periods indicate fully, strongly, or weakly conserved residues, respectively. (B) Representative immunoblot of Nrxn1β-Fc variants harvested from HEK293 cell media and subjected to heparinase treatment ('Heps'). Samples were analyzed by reducing SDS/PAGE and immunoblotting with antibodies against the Fc-tag and the 3G10 epitope (to detect the HS stub which reveal HS after heparinase digestion). (C) Quantification of 3G10 (reflecting HS) signal normalized to that of Fc (Nrxn1). Bar graph shows means ±SEM of 3 independent experiments. \n",
      ".Figure 2HEK293 cells: (A) Schematic outline of the experiment shown in (B). HA-Nrxn1α or HA-Nrxn1α (S>A), a mutant of the modified serine were expressed alone or in combination with V5-tagged CA10 in HEK293 cells. Neurexins were immunoprecipitated for HA and treated with heparinases ('Heps'). HEK293 cells: (B) Representative immunoblots (of three independent experiments) with antibodies against HA, the 3G10 epitope (for the HS stub) and V5. HEK293 cells: (C) Schematic outline of the experiment shown in (D). HA-Nrxn1α was expressed alone or in combination with CA10-V5 in HEK293 cells, lysates were immunoprecipitated for HA (for Nrxn1α) or V5 (for CA10). HEK293 cells: (D) Representative immunoblots (of three independent experiments) analyzed as described for (B). HEK293 cells: (E) Schematic illustration of how endogenous neurexin-HS was analyzed. HS-modified α-neurexin migrates at higher apparent molecular weight (HMW) than non-modified low molecular weight (LMW; white arrowheads) species (left). Following treatment with heparinases (right), both HS-modified and non-modified α-neurexin migrates as LMW species, but HS-modified α-neurexin can be detected by the 3G10 monoclonal. (F-H) Primary cortical neurons: Cortical neurons were infected with control lentivirus (empty vector) or lentivirus expressing FLAG-tagged CA10, and cell lysates subjected to immunoprecipitation for neurexin and heparinase treatment. (F) Samples were analyzed by immunoblotting using antibodies against neurexins, 3G10 (for the HS stub) and FLAG (for CA10). Stars indicate non-neurexin bands (see results). (G) Quantification of low molecular weight neurexin (LMW; white arrowhead) in relation to total neurexin amounts [upper band (line) + LMW]. Data shown as mean ±SEM (n=3 independent experiments); *, p< 0.05 by 2-way ANOVA and Holm-Sidak tests. (H) Quantification of the 3G10 signal normalized to the signal of the corresponding neurexin band. Data shown as mean ±SEM of biological replicates. *, p< 0.05 by 2-way ANOVA and Holm-Sidak tests (n=3). (I-K) Mouse brains: Neurexins and CA10 were immunoprecipitated from total mouse brain lysates and subject to heparinase treatment. (I) Samples were analyzed by immunoblotting using antibodies against neurexins, the 3G10 epitope (for the HS stub) and CA10. Stars indicate non-neurexin bands (see results). (J) Quantification of low molecular weight neurexin (LMW; white arrowhead) in relation to total neurexin amounts [upper band (line) + LMW]. Data shown as mean ±SEM (n=3 independent experiments); *, p< 0.05 by 2-way ANOVA and Holm-Sidak tests. (K) Quantification of the 3G10 signal normalized to the signal of the corresponding neurexin band. Data shown as mean ±SEM of 3 biological replicates. *, p< 0.05 by 2-way ANOVA and Holm-Sidak tests. \n",
      ".Figure 6evaluation of S100A9+ neutrophil and CD8+ T cell infiltration in histologic sections of human CRC tumors. , Anti-S100A9 immunostaining on a representative human CRC section. Arrowheads indicate neutrophil infiltration at the tumor boarder. Co-immunostaining for CD8 (brown) and S100A9 (gray-blue) in the same CRC sample as shown in A. evaluation of S100A9+ neutrophil and CD8+ T cell infiltration in histologic sections of human CRC tumors. Quantification of S100A9+ cells (upper panel) and CD8+ T cells in different areas on CRC tumor sections containing tumor border with adjacent benign mucosa as determined by a specialized pathologist: \"Tumor center\" was tissue within tumor >200μm from tumor border, \"Tumor border\" was tissue within tumor <100μm from tumor border and \"Adjacent benign mucosa\" was mucosa tissue outside tumor <100μm from tumor border. Per section, all identifiable tumor border and adjacent benign areas, as well as three arbitrary tumor center areas were scored. Sections were derived from 10 individual patients. Gene expression scores were generated as described in material and methods. Boxes are lower and upper quartiles with median as solid lines; horizontal lines define minimum and maximum; dots define outliers. , Neutrophil scores in tumor samples categorized according to CMS subtypes. Gene expression scores were generated as described in material and methods. Boxes are lower and upper quartiles with median as solid lines; horizontal lines define minimum and maximum; dots define outliers. Comparison of T cell gene expression scores in CMS4 tumors within two publicly available CRC gene expression datasets (median, lower and upper quartiles; horizontal lines define minimum and maximum; dots define outliers). Samples are categorized according to the medians (high = >median; low = neutrophil and TGFβ gene expression scores. Upper panel: TCGA dataset. Lower panel GSE39582 dataset. Proliferation of activated blood T cells in vitro in co-culture with increasing ratios of CRC patient-derived autologous blood neutrophils. CD3+ T cells and CD66b+ neutrophils were isolated and cultured individually for each patient. Bar graphs display relative numbers of proliferated T cells after three days of co-culture. Each dot per condition represents a neutrophil sample from an individual patient (n=13). Bars represent mean + SD. Proliferation of activated blood T cells in vitro in co-culture with autologous blood neutrophils. Cells were derived from both CRC patients (n=2) and healthy volunteers (n=5). T cells and neutrophils were cultured at a 1:1 ratio and treated with vehicle (T + Ne) or 10µm Galunisertib (T + Ne + TGFBRi). Each line represents a neutrophil sample from an individual healthy volunteer. \n",
      ".Figure 5(A) Schematic representation of the intermediate steps of HS biosynthesis. Heparinase cleavage and the neo-epitope detected by the 3G10 antibody following cleavage et al,(David 1992) are indicated. (B-C) Extracted ion chromatograms (EIC) of DDILVASAECPSDDE peptides and its glycosylated forms, from Nrxn1β expressed in HEK293 cells. (D) Quantification, expressed as % of total occupancy of each glycosylation form, upon control and CA10 expression conditions. Bar graph shows means ±SEM of 4 biological replicates; *, p<0.05 by Mann-Whitney tests comparing the occupancy of each glycosylation with and without CA10 (n=4 biological replicates). \n",
      ".Figure 4(A) Sequence of the Nrxn1 peptide used for the analysis. Residues previously mapped to bind CA10 (et al,Sterky 2017) and the site where alternative splice site 5 (SS5; et al,Ushkaryov 1992) inserts three residues (GGL) are indicated. (B) Solution structure of the Nrxn1 stalk, with cysteines labeled yellow. The 10 models of lowest energy with respect to the target function are shown (ensemble average RMSD 2.95 Å). (C) Chemical shift changes induced by CA10, qualitative mapped onto the Nrxn1 solution structure. Residues with the strongest changes are labeled. (D) Comparison of 1H-13C HSQC spectra of Nrxn1 stalk (red) and Nrxn1 stalk/CA10 (black) at a 2:1 molar ratio (50 mM phosphate buffer, pH 7.4, 0.5 mM d10-DTT, 10% D2O). Nrxn1 residues with stronger chemical shift changes are indicated blue boxes. Asterisk (*) indicates an impurity from glycerol. \n",
      ".Figure 3(A) Sequences of the Nrxn1β mutants used, color-coded as in Fig 1A, with residues previously mapped to bind CA10 indicated. Right, table summarizing properties of the recombinant variants: 'HS?' indicates if HS-modified when expressed alone (Fig 1B), 'Binds CA10?', indicates if found to bind CA10, according to (et al,Sterky 2017), and whether the HS modification is blocked by CA10 (this figure). Asterisk denotes non-covalent interaction (see discussion). (B) Representative immunoblot of the indicated proteins harvested from HEK293 cell media and subjected to heparinase treatment ('Heps'). Samples were analyzed by reducing SDS/PAGE and immunoblotting with antibodies against Fc (for Nrxn1β), the 3G10 epitope (for the HS stub) and V5 (for CA10). Input samples were analyzed separately for CA10 to ensure equal expression (lower panel). (C) Quantification of the 3G10 epitope signal, normalized to Fc (Nrxn1β). Data shown are mean ±SEM of 3 independent replicates. *, p< 0.05; by Mann-Whitney tests comparing each Nrxn1 mutant with and without CA10 (n=3). No significance difference was observed for S&T>G (p=0.1143), GIGG (p=0.9714) or DIRR (p=0.4857) variants. Fc-tagged (secreted) Nrxn1β and HA-tagged (transmembrane) Nrxn1α were co-expressed with CA10 and a CA10 mutant for the reactive cysteine CA10(C310A). (D) Summary of CA10 variants used, and their properties (this figure). Asterisk denotes non-covalent interaction (see discussion). (E) representative immunoblots analyzed using antibodies against the Fc-tag (for Nrxn1β-Fc) the 3G10 epitope (for the HS stub) and V5 (CA10-V5), as indicated. CA10 in input samples is shown for demonstration of equal expression. (F, Quantification of the 3G10 epitope signal normalized to Fc (for Nrxn1β-Fc) signals, Data shown are mean ±SEM of 3 biological replicates. *, p< 0.05; ***, p<0.001 by 2-way ANOVA and Holm-Sidak tests. (G) show representative immunoblots analyzed using antibodies against the HA tag (Nrxn1α-HA), the 3G10 epitope (for the HS stub) and V5 (CA10-V5), as indicated. CA10 in input samples is shown for demonstration of equal expression. H) Quantification of the 3G10 epitope signal normalized to HA (for Nrxn1α-HA) signals, respectively. Data shown are mean ±SEM of 3 biological replicates. *, p< 0.05; ***, p<0.001 by 2-way ANOVA and Holm-Sidak tests. \n",
      ".Figure 8Illustration of how CA10 may control HS specifically on neurexins. CA10 binding to neurexin in the ER along the secretory pathway to the cell surface, which sterically hinder the priming xylose addition by xylosyltransferases in the cis-Golgi. This interaction provides a mechanism to regulate glycan addition at a specific site, to regulate the balance between non-HS- and HS-neurexin in a specific cell type without affecting biogenesis of other HSPGs. Control of neurexin-HS by CA10 affects the affinities of neurexin to postsynaptic ligands. \n",
      ".Figure 6(A) Immunoblot α-showing neurexin isoform distribution in cortex, hippocampus , midbrain and cerebellum dissected from 8-week-old mice. (B) Quantification of lower molecular weight neurexin (LMW; white arrowhead) in relation to total levels of α-neurexins [upper band (line) + LMW]. Bar graphs show mean ± SEM of 3 independent replicates***, p<0.001 by 2-way ANOVA and Holm-Sidak tests, comparing to levels in cerebellum. (C) α-Neurexin immunoprecipitated from forebrain (Fb) and cerebellum (Cb) using an anti-pan-neurexin antibody was heparinase-treated ('Heps') and analyzed by immunoblotting with antibodies against neurexin and the 3G10 monoclonal (to detect the HS stub). Stars indicate non-neurexin bands (see results). (D) Quantification of low molecular weight neurexin (LMW; white arrowhead) in relation to total levels of α-neurexins [upper band (line) + LMW]. Bar graphs show mean ±SEMof 3 biological replicates. ***, p<0.001 by 2-way ANOVA and Holm-Sidak tests comparing forebrain and cerebellum. (E) Quantification of the 3G10 signal normalized to the signal of the corresponding neurexin band. Bar graphs show mean ±SEM of 3 biological replicates. ***, p<0.001 by 2-way ANOVA and Holm-Sidak tests comparing forebrain and cerebellum. \n",
      ".Figure 7(A-B) Cell surface binding to endogenous neurexins. (A) Recombinant Fc-tagged LRRTM2 was added to day 10 hippocampal neurons expressing CA10 or control (empty vector) by lentiviral transduction. Surface-bound protein was quantified as the Fc signal of and normalized to a staining for endogenous neurexin (n=4 cultures; three fields imaged and averaged per well). Scatchard analysis for LRRTM2 revealed an apparent Kd of 29.5 nM and a Bmax of 0.49 nM for control and Kd 35.3 nM, Bmax 0.28 nM upon expression of CA10; significant (p=0.0006) by non-linear regression. Data shown as mean ± SEM of biological replicates (n=4). (B) Increasing amounts of recombinant, Fc-tagged Nlgn1 was added to immature hippocampal neurons expressing CA10 or control (empty vector) by lentiviral transduction, and analyzed as in (A). The apparent Kd for Nlgn1 was 186 nM and Bmax 1.65 nM for control and Kd 223 nM, Bmax 1.49 nM when CA10 was overexpressed. The difference was not significant (p=0.20) by non-linear regression. Data shown as mean ± SEM of biological replicates (n=4). (C-D) Artificial synapse formation assays. (C) Confocal images of hippocampal neurons transduced to express CA10-FLAG or control (empty vector) and co-cultured with HEK293 cells expressing GFP, Nlgn1-YFP, LRRTM2-YFP or TrkC-YFP. Cells were stained with antibodies against the synaptic marker SV2 (red) and somatodendritic MAP2 (blue); GFP/YFP is shown in green. Scale bar 20 μm. (D) Quantifications of SV2 signal overlapping GFP/YFP-positive cells. Data shown as means ±SEM with the n, reflecting number of independent cover slips analyzed, indicated in each bar with the total number of cells imaged in parenthesis. Comparisons were made by one-way ANOVA with Holm-Sidak's posthoc test for multiple comparisons between the indicated groups (GFP condition was omitted from analysis due to unequal variance; ***, p<0.001. (E-F) Analysis of cerebellin(Cbln)-induced artificial synapse formation. (E) Confocal images of hippocampal neurons transduced to express CA10-FLAG or control (empty vector) and co-cultured exposed to anti-V5-coupled beads pre-incubated with recombinant V5-tagged Cbln1 or empty anti-V5 beads. Cells were stained with antibodies against the synaptic marker synapsin (green), somatodendritic MAP2 (blue); beads are colored red and indicated by arrowheads. Scale bar 10 μm. (F) Quantifications of the synapsin signal overlapping red beads. Data shown as means ±SEM (n=3 independent experiments, each data point corresponding to 58-171 analyzed beads), with comparisons made by two-way ANOVA with Holm-Sidak's posthoc test for multiple comparisons between the indicated groups. \n",
      ".Figure 3(A) List of top hits of EZH2-bound proteins identified by TAP-MS. The number of peptides and coverage of each protein are indicated. (B) Reciprocal co-IP of endogenous EZH2 and eIF4G2 or PABP1 in C4-2 cells. (C) Co-IP of endogenous EZH2 with eIF4G2 or PABP1 from C4-2 cell lysate pre-treated with RNaseA in 37oC for 30 min. The effectiveness of RNaseA treatment was monitored by RT-PCR analysis of the presence of GAPDH mRNA. (D) C4-2 cells were transfected control (siC) or EZH2-specific siRNA and lysed for polysome fractionation. RNA was extracted from even-number fractions followed by RT-qPCR analysis for p53 and GAPDH mRNA. The β-ACTIN mRNA was used as an internal control. Data shown as means±SD (n=3). Statistical significance was determined by two-tailed Student's t-test. * P<0.01; NS, no significance. (E) C4-2 cells were transfected with control (siC) or EZH2-specific siRNAs in combination with empty vector, EZH2 WT or different deletion mutants followed by polysome fractionation and RT-qPCR. β-ACTIN mRNA was used as an internal control. Data shown as means±SD (n=3). * P<0.01. \n",
      ".Figure 2(A) Biotin pull down assay by incubating biotin-labelled different fragments of p53 mRNA and HOTAIR (positive control) with C4-2 cell lysate followed by western blot with EZH2 antibody. (B) Biotin pull down assay as in (A) using unmutated and various internally deleted mutants of 5'UTR of p53 mRNA. Top, diagram of different p53 5'UTR deletion mutants. (C) Upper, the linear map of the pRF bicistronic report plasmid. The SV40 promoter (purple box) was used to drive firefly luciferase (Fluc) and Renilla luciferase (Rluc) gene transcription. Different p53 5'UTR fragments were inserted between the Fluc and Rluc genes. Lower, at 24 h after transfection, cells were lysed and luciferase activities were measured using a dual-luciferase kit and the ratio of Fluc/Rluc was calculated. Data shown as means±SD (n=3). Statistical significance was determined by two-tailed Student's t-test. * P<0.01. (D) EZH2 knockdown C4-2 cells were transfected with the bicistronic reporter vector in combination with empty vector, Myc-tagged EZH2 WT or deletion mutants followed by western blot analysis with indicated antibodies (bottom) and luciferase assay (top). Data shown as means±SD (n=3). * P<0.01. (E) C4-2 and U2OS cell lines were transfected with non-specific control (siC) or two independent EZH2-specific siRNAs and harvested for western blot analysis with indicated antibodies. ERK2 and β-TUBULIN, loading controls. (F) U2OS cells were transfected with control (siC) or EZH2-specific siRNAs and treated with 200 nM of CPT followed by western blot analysis for indicated proteins. (G) C4-2 cells were transfected with control (siC) or EZH2-specific siRNAs and plasmids for empty vector, EZH2 WT or deletion mutants followed by western blot analysis for indicated proteins. \n",
      ".Figure 1(A) Heat map showing a subset of mRNAs of genes immunoprecipitated by anti-EZH2 antibody in LNCaP and C4-2 prostate cancer cell lines, which was generated based on the distinct RIP-seq reads on specific gene exons. The units of the heatmap values were reads per kilobase million (RPKM). (B) Screen shots from the UCSC genome browser showing signal profiles of p53 mRNA immunoprecipitated by anti-EZH2 antibody in LNCaP and C4-2 cells. (C) C4-2 cells were subjected to UV-RIP assay. RT-qPCR measurement of p53 mRNA immunoprecipitated by IgG or anti-EZH2 antibody. Data shown as means ± SD (n=3). Statistical significance was determined by two-tailed Student's t-test. ** P<0.01. (D) Schematic diagram of four GST-EZH2 recombinant proteins (EZ1 to EZ4). EBD, EED binding domain; SANT1(/2) L, \"Swi3, Ada2, N-Cor, TFIIIB\"1(/2) like; MCSS, motif connecting SANT1L and SANT2L; CXC, cysteine-rich domain; SET, catalytic domain. (E) Top, RT-qPCR analysis of p53 mRNA in C4-2 cell lysate pulled down by GST or GST-EZH2 recombinant proteins EZ1-EZ4. Bottom, western blotting analysis of GST or GST-EZH2 proteins used for GST pull-down assay. Asterisks indicate the protein bands at expected molecular weight. Data shown as means ± SD (n=3). Statistical significance was determined by two-tailed Student's t-test.** P<0.01. (F) C4-2 cells were transfected with Myc-tagged EZH2-WT or EZH2-∆336-554 for 24 h and cells were harvested for RIP with IgG or anti-Myc tag antibody. Transfected proteins and pull-down p53 mRNAs were analyzed by western blot and RT-qPCR, respectively. Data shown are means ± SD (n=3). * P<0.01. ERK2, a loading control. (G) GST pull-down assay using in vitro transcribed different fragments of p53 mRNA and indicated GST proteins followed by RT-qPCR analysis of pull-down p53 mRNA. FL, full length; ORF, open reading frame; UTR, untranslated region. (H, I) RNA EMSA evaluation of EZh2 binding of p53 mRNA. GST-EZH2 recombinant proteins (EZ1 to EZ4) were incubated with biotin-labeled in vitro transcribed p53 5'UTR (biotin-labeled probe) in the presence or absence of 100-fold unlabeled p53 5'UTR (unlabeled probe), followed by PAGE and immune blotting with HPR-conjugated streptavidin. \n",
      ".Figure 4(A) IHC analysis of Pten, Ezh2 and p53 proteins in prostatic tissues of 4-month-old mice with different genotypes: Ptenpc-/- (n=11); Ezh2pc-/- (n=8); Ptenpc-/-;Ezh2pc-/- (n=10) and 'wild-type' littermate controls (n=10). Scale bars, 50 (B) Western blot analysis of expression of indicated proteins in four different groups of mice as described (n=2 mice/group). (C) Analysis of correlation of Ezh2 and p53 protein levels determined by IHC in Pten knockout (Ptenpc-/-) mice (n=11). (D) RT-qPCR analysis of p53 mRNA expression in four different genotypes of mice as described Statistical significance was determined by two-tailed Student's t-test. * P<0.01. (E) RT-qPCR analysis of mRNA expression of p53 downstream target gene Bax in four different genotypes of mice as indicated. Statistical significance was determined by two-tailed Student's t-test. * P< 0.01. (F) H&E analysis of ventral prostate (VP) of 4-month-old mice with indicated genotypes. The inset shows a high magnification image of the representative (framed) area in each panel. Scale bars, 50 µm. Scale bars in insets, 10 µm. (G) Quantification of nonmalignant, low-grade PIN (LGPIN) and high-grade PIN (HGPIN) or cancerous acini in the prostates (including anterior prostate (AP), ventral prostate (VP) and dorsolateral prostate (DLP)) of mice with the indicated genotypes and number as in (A). \n",
      ".Figure 6(A, B) VCaP cells stably expressing control (sh-control) or p53-specific shRNA were infected with lentivirus for empty vector (EV) or deletion mutants of EZH2. Cell growth in 2D (A) and 3D (B) conditions were determined by MTS assay and measurement of clone size, respectively. Statistical significance was determined by two-tailed Student's t-test. * P < 0.05, *** P < 0.001. (C, D) VCaP cells (1x107) infected with lentivirus were injected subcutaneously into NSG mice (n = 8/group). Tumors were measured by caliper twice a week. Tumors at the end point of measurement were isolated and photographed (C), and data are shown as means ± SD (D). Statistical significance was determined by two-tailed Student's t-test for tumors at day 56. *** P < 0.001. (E-G) Luciferase-expressing MIA PaCa-2 cells (2×106) infected with lentivirus were injected via tail vein into NSG mice (n = 6/group). At 12 weeks after injection, mice were subjected to bioluminescent imaging, and images were recorded (E) and bioluminescent signals were quantified (F). Bioluminescent flux (photons s-1 sr-1 cm-2) was determined for lesions in lung, the ends of the box are the upper and lower quartiles and the box spans the interquartile range; the median is marked by a vertical line inside the box; the whiskers are the two lines outside the box that extend to the highest and lowest observations. Lungs were isolated from mice, stained with Bouin's solution, and photographed (G). The white spots on lungs (stained in yellow) are metastatic tumors. * P < 0.05; NS, no significance. \n",
      ".Figure 5(A) Correlation analysis of EZH2 and p53 mRNA expression measured by RT-qPCR in primary and metastatic prostate cancer specimens of a cohort of patients from University of Washington. (B) IHC analysis of EZH2 and p53 protein expression in human prostate cancer tissues of a cohort of patients from Mayo Clinic. Case 1 and 2 show low and high expression in primary tissues, respectively; case 3 shows high expression in lymph node metastasis. Scale bar, 50 µm. Scale bars in insets, 10 µm. (C) Correlation analysis of EZH2 and p53 protein expression in primary and metastatic prostate cancer specimens of the Mayo Clinic cohort. (D) IHC analysis of EZH2 and p53 protein expression in commercially purchased TMA with patient specimens of brain, colorectal, pancreatic cancer and sarcoma. Scale bars in low magnification images, 200 µm; Scale bars in high magnification images, 50 µm. Arrows, indicated the zoom out-box. (E) IHC data as shown in (D) were quantified and used for correlation analysis of EZH2 and p53 protein expression in patient specimens of brain, colorectal, pancreatic cancer and sarcoma. Pearson correlation r and P values in each cancer type are indicated. \n",
      ".Figure 7(A) Schematic diagram depicting the distinctive impacts of the functional interplay between EZH2 protein and p53 mRNA on progression of cancers expressing WT p53, no p53 or GOF mutated p53. MTase, methyltransferase. (B) Kaplan-Meier plots showing the association of EZH2 overexpression with biochemical recurrence of prostate cancer in patients from the TCGA cohort. Tumors were separated into two groups according to EZH2 expression levels (high or low) and three groups based on the mutation status of p53 (WT, loss or mutated). VCaP and C4-2 cells were treated with GSK-126, DZNep (5 µM for VCaP and 10 µM for C4-2), and EZH2 ASO, followed by western blots with indicated antibodies (C) Data shown as means±SD (n=6). Statistical significance was determined by two-tailed Student's t-test. *** P < 0.001. VCaP and C4-2 cells were treated with GSK-126, DZNep (5 µM for VCaP and 10 µM for C4-2), and EZH2 ASO, followed by MTT assay (D). Data shown as means±SD (n=6). Statistical significance was determined by two-tailed Student's t-test. *** P < 0.001. (E, F) Effect of Ezh2 ASOs on growth of murine prostate cancer allografts. Mice (n=10) bearing the Ptenpc-/-;Trp53pc-/- (E) and Ptenpc-/-;Trp53pc-/R172H allografts (F) were treated with control ASOs (50 mg kg-1) or Ezh2 ASOs at different doses (25 mg kg-1 and 50 mg kg-1). Tumor growth was measured twice a week for 3 weeks and the data are shown in the bottom panels. Each allograft was isolated by the end of ASO treatment and photographed (top panels). Data shown as means±SD (n=10). Statistical significance was determined by two-tailed Student's t-test. *** P < 0.001. \n",
      ": Impaired fear extinction precedes memory decline in Fmn2 mutant mice.Figure 1Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.A. Freezing behavior tested 24h after training was similar in 3-month old APPPS1-21, 5xFAD and Fmn2 -/- mice when compared to corresponding control groups (n = 10/group).E. Freezing behavior was tested 24h after training in 8-month old APPPS1-21, 5xFAD and Fmn2 -/- mice. A significant impairment in associative memory was observed when comparing APPPS1 mice (n=10) to a corresponding control group (n=8; P** < 0,01), when comparing 5xFAD mice (n=10) to a corresponding control group (n=8; P** = 0,01) and when comparing Fmn2 -/- mice (n=10) to corresponding Fmn2 +/+ wild type mice (n=10; P*** < 0,0001). Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.B. Fear extinction in 3-month old APPPS1 mice is similar to a non-transgenic control group (n=9/group). Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.C. Fear extinction in 3-month old 5xFAD mice is similar to a non-transgenic control group (n=9/group). Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.D. Extinction learning in the contextual fear conditioning paradigm is impaired in Fmn2-/- mice (n=9/group; P*** < 0.001, F = 21,77). Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.F. Freezing was analyzed in 3- (n=11), 12- (n=9) and 16- (n=8) month old male wild type mice. A significant impairment (P < 0,0001) was observed in 16-month old animals when compared to either 3- or 12-month old animals. Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.G. Fmn2 expression measured in blood samples via qPCR was reduced in PTSD patients (n=9) when compared to age-matched controls (n=7; P** = 0,0361). Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice.H. Fmn2 mRNA expression is decreased after dexamethasone treatments in human hippocampal progenitor cells (HPCs) during neuronal proliferation (3 days) and differentiation (10 days) ***Padj < 3.27x10-6) Figure 1: Impaired fear extinction precedes memory decline in Fmn2 mutant mice. I. qPCR revealed that Fmn2 expression is decreased (P* = 0,037) in AD patients (n=9) when compared to age-matched controls (n=8). Error bars indicated SEM. \n",
      ": Hippocampal FMN2 is essential for extinction learning and cognitive flexibility.Figure 3Figure 3: Hippocampal FMN2 is essential for extinction learning and cognitive flexibility. A. Experimental design. Anti-Fmn2 or scrambled siRNA oligomers were injected via microcannulae into the hippocampus of 3-month old wild type mice every 12h for 3 days. Figure 3: Hippocampal FMN2 is essential for extinction learning and cognitive flexibility.B. Fmn2 mRNA levels were determined in the hippocampus of mice injected with either scrambled or Fmn2 siRNA via qPCR (n= 4/group). Fmn2 levels are reduced in mice injected with Fmn2 siRNAi when compared to the scrambled control group (*P < 0.05). Error bars indicate SEM. Figure 3: Hippocampal FMN2 is essential for extinction learning and cognitive flexibility. C. Similarly, hippocampal FMN2 protein levels were decreased in mice injected with Fmn2 siRNA when compared to the scramble control group (*P < 0.05).Error bars indicate SEM. Figure 3: Hippocampal FMN2 is essential for extinction learning and cognitive flexibility.D. Experimental Design. Microcannulae were implanted into the hippocampus of wild type mice. All mice were subsequently subjected to contextual fear conditioning training. Fmn2 siRNA or scrambled control RNA was injected into the dentate gyrus 24h later and then every 12h for 3 days. On subsequent days mice were subjected to extinction trial (E) 1, E2, E3 and E4 (Fig 3B). The injection protocol was continued until E3. E. Fear Extinction is significantly impaired (P < 0.001, F = 15.55; * P < 0.05, ***P < 0.001 for post hoc analysis) in 3-month old mice that received intra-hippocampal injections of Fmn2 siRNA when compared to the scrambled control group (n=9/group). When siRNA injection was terminated, all mice exhibited normal extinction behavior. Error bars indicate SEM. Figure 3: Hippocampal FMN2 is essential for extinction learning and cognitive flexibility.F. Left panel: Experimental design. Right panel: Injection of Fmn2 siRNA before fear conditioning training does not affect the acquisition of fear memories. Error bars indicate SEM. \n",
      ": FMN2 is linked to age-associate memory decline A.Figure 4Figure 4: FMN2 is linked to age-associate memory declineA. Escape latency in 3-month old Fmn2 +/+ mice (n=12) compared to 8-month old Fmn2 +/+ (n=20) and Fmn2 -/- mice (n=20). Escape latency is significantly impaired in 8-month old Fmn2 -/- mice (P = 0.001, F = 24.1 vs. age-matched Fmn2 +/+).Error bars indicate SEM. Figure 4: FMN2 is linked to age-associate memory declineB. A probe test revealed that the time spent in the target quadrant significantly differed amongst groups (P = 0.0004, F = 9,4) and that 8-month old Fmn2 -/- mice spent significantly reduced time in the target quadrant when compared directly to age-matched (P = 0,0143) or 3-month old Fmn2 +/+ mice (P < 0,0001).Error bars indicate SEM. Figure 4: FMN2 is linked to age-associate memory declineC. Left panel: The total distance traveled during a 5 min open field exposure did not differ significantly amongst groups (n=8/group). Right panel: The time spent in the center vs. the corners during a 5 min open field exposure was similar amongst groups (n=8/group). Error bars indicate SEM. Figure 4: FMN2 is linked to age-associate memory declineD. Contextual freezing in 3-month old Fmn2 +/+ (n=14), APPPS1-21 (n=13), APPPS1-21_Fmn2 -/- (n=14) and Fmn2 -/- mice (n=14). Freezing behavior differed amongst groups (P = 0.0005, F= 6,947) and was significantly impaired in APPPS1-21_Fmn2 -/- mice when compared to the other groups (***P < 0,001).Error bars indicate SEM. Figure 4: FMN2 is linked to age-associate memory decline E. Escape latency in the water maze test in 3-month old Fmn2 +/+ (n=15), APPPS1-21 (n=15), APPPS1-21_Fmn2 -/- (n=15) and Fmn2 -/- mice (n=15). Comparing the learning curves amongst all groups revealed no significant difference. Error bars indicate SEM. Figure 4: FMN2 is linked to age-associate memory declineF. During the probe test there was a significant difference amongst groups in the time spent in the target quadrant (P = 0.04, F = 3,22). Expect for APPPS1-21_Fmn2 -/- mice, all groups showed a significant preference for the target quadrant (*P < 0.05). A direct comparison between Fmn2 -/- and APPPS1-21_Fmn2 -/- mice revealed a significant reduction in the latter group (*P = 0,017). Error bars indicate SEM. \n",
      ": FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.Figure 2Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.A. Fmn2 expression was analyzed via qPCR in brain regions from 3-month old mice (Left panel, n=5) and in mouse primary hippocampal neurons, astrocytes (n=5) and human postmortem hippocampus (right panel; n=3/group). Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.B. Representative immunoblot images showing FMN2 protein levels in the mouse hippocampus (Hip M), cerebellum (Cere M) and human hippocampus (Hip Hu) Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.C. Representative immunoblot images showing FMN2 levels in the hippocampus of wild type and Fmn2 -/- mice Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation. D. Representative low (upper panel, scale bar 500 µm) and high magnification images (lower panel) showing FMN2 expression (green) in FMN-EGFP knock in mice.For all experiments parasagittal brain slices from 14-21 day old mice were used. Cere, cerebellum; Hip, hippocampus, mf; mossy fiber, str. Luc.; stratum lucidum, cc, corpus callosum. Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.E. Left panel: Representative images showing the co-localization of FMN2 (green) and Synaptoporin in the mossy fiber pathway in low and high magnification. Right panel: Immunoblot analysis of hippocampal synaptosomes (Syn) compared to cell homogenate (H) showing that FMN2 is present in the pre-synaptic compartment. Synaptophysin was used to confirm enrichment of synaptosomes. For all experiments parasagittal brain slices from 14-21 day old mice were used. Cere, cerebellum; Hip, hippocampus, mf; mossy fiber, str. Luc.; stratum lucidum, cc, corpus callosum. Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.F. LTP was similar in Fmn2-/- (n=12) and wild type control mice (n=8).Error bars indicate S.E.M. Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.G. The absolute level of LTD was similar amongst Fmn2-/- and wild type mice (n=10). Error bars indicate S.E.M. Figure 2: FMN2 is highly expressed in mossy fibers and is required for synaptic depotentiation.H. Depotentiation was significantly impaired in Fmn2-/- mice (n=12) when compared to control (n=11; *P < 0,05). Error bars indicate S.E.M. \n",
      ": Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat.Figure 6Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat.A. The total distance traveled during a 5 min open field exposure was similar amongst groups (Fmn2-/- placebo, n=9; Fmn2-/- Vorinostat, n=12). Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat.. B. The time spent in the center and the corners during a 5 min open field exposure was similar amongst groups (Fmn2-/- placebo, n=9; Fmn2-/- Vorinostat, n=12). Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat. C. The time spent in the open (o) and closed arms (c) or the center region (Ce) of an elevated plus maze was similar amongst groups (Fmn2-/- placebo, n=7; Fmn2-/- Vorinostat, n=10). Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat. D. Freezing behavior during the memory test was significantly increased in the Fmn2-/- Vorinostat group when compared to the placebo group (P = 0,04) (Fmn2-/- placebo, n=8; Fmn2-/- Vorinostat, n=9). Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat.. E. Left panel: The escape latency in the water maze test was significantly improved in the Fmn2-/- Vorinostat group when compared to the placebo group (P < 0.0001, F = 4,86; n=10/group). Right panel: Swim speed was not altered amongst groups Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat. F. Mice of the Fmn2-/- Vorinostat group spent significantly more time in the target quadrant (T) compared to the other quadrants (OQ). Please note the values for OQ are presented as average of all 3 other quadrants (P < 0.0001; n=10/group). Figure 6: Cognitive decline in 8-month old Fmn2 -/- mice is rescued by the HDAC inhibitor Vorinostat. G. Expression of selected genes that were down-regulated in 8-month old Fmn2-/- mice was significantly increased in the dentate gyrus from mice of the Fmn2-/- Vorinostat group when compared to placebo (P < 0,05; n=4/group). Error bars indicate SEM. \n",
      ": Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity A.Figure 5Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticityA. Volcano plot showing genes differentially expressed in the dentate gyrus of either 3- (red) or 8-month (black) old Fmn2 -/- mice compared to corresponding wild type control group (FDR < 0,05, log2 fold change < +/- 0,25). The upper panel depicts the number of altered genes. Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity B. Top GO domains and pathways affected in 3-month old Fmn2 -/- mice when compared to wild type control animals. D. Top GO domains and pathways deregulated in 8-month old Fmn2 -/- mice when compared to an age-matched control animals. GO:CC; gene-ontology domain \"cellular compartment\", GO:MF; gene-ontology domain \"molecular function\", GO:BP; gene-ontology domain \"biological process\", pathway \"KEGG pathways\". Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity C. Expression of selected genes affecting the oxidative phosphorylation pathway that are increased in 3-month old Fmn2-/- mice (*P < 0,05; n=5/group). Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity E. Expression of selected genes of the oxidative phosphorylation pathway that were decreased in 8-month old Fmn2-/- mice (*P < 0,05; n=3-5/group). Error bars indicate SEM. Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity F. Cytoscape-generated network of genes deregulated in 8-month old Fmn2-/- mice. Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity G. Volcano plot showing differential exon usage in the dentate gyrus of either 3- (red) or 8-month (black) old Fmn2 -/- mice when compared to the corresponding wild type control group (FDR < 0,05, log2 fold change < +/- 0,2). The upper panel depicts the number of altered genes Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity H. Selected GO domains and pathways affected by differential exons usage in 8-month old Fmn2 -/- mice when compared to a wild type control group. GO:CC; gene-ontology domain \"cellular compartment\", GO:MF; gene-ontology domain \"molecular function\", GO:BP; gene-ontology domain \"biological process\", pathway \"KEGG pathways\". Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity I. Volcano plot showing genes differentially expressed in 3-month old APPPS1-21 mice compared to a wild type control group (green) and APPPS1-21_Fmn2 -/- mice compared to wild type control mice (purple). (FDR < 0,05, log2 fold change < +/- 0,5). The upper panel depicts the number of altered genes. Figure 5: Memory decline in 8-month old Fmn2 -/- mice correlates with deregulated transcriptome plasticity J. Functional pathways affected when comparing APPPS1-21 (green bars) or APPPS1-21_Fmn2 -/- mice (purple bars) to wild type mice. GO:CC; gene-ontology domain \"cellular compartment\", GO:MF; gene-ontology domain \"molecular function\", GO:BP; gene-ontology domain \"biological process\", pathway \"KEGG pathways\". \n",
      ": AD risk factors cause loss of gene-expression control.Figure 7Figure 7: AD risk factors cause loss of gene-expression control.Our central hypothesis is that the various AD risk factors eventually lead to aberrant gene-expression and loss of transcriptional control. This is based on the assumption that proper gene-expression is a core feature of cellular homeostasis and that changes occurring at various compartments of the cell (e.g. at the synapse) will eventually signal to the nucleus and cause gene-expression changes. In the case of FMN2 we provide for example evidence that chronically low levels of FMN2 disturb synaptic actin dynamics and thereby affect ERK1/2-dependent gene-expression programs. Thus, even if an AD risk factor may play no direct role in gene-expression it may eventually contribute to a loss of transcriptional plasticity. We therefore suggest that targeting pathological gene-expression could be a suitable therapeutic approach, especially for multifactorial diseases such as AD, where it is nearly impossible to determine all of the genetic and environmental factors that eventually contribute to clinical phenotypes. \n",
      ".Figure 2Schematic overview of McjD labelling schemes (A) and diffusion-based experiments (B) using confocal microscopy. Stochastic labeling is performed on the NBD (native cysteine C547) and TMD mutants (Y64C; L67C) Confocal single molecule analysis with ALEX of labeled McjD in detergent under different conditions as indicated Apparent ligand binding affinity values of McjD for AMPPNP (top), ATP (middle), and ADP-vanadate (bottom). Data points were obtained from the areas of fits from the E* histograms considering the ratio ligand-free/(ligand-free+ligand-bound) at the indicated substrate concentrations. Extraction of Kd values from a fit to the data is described in the Material and Methods section. Data for detergent-solubilized McjD for additional biochemical conditions is shown in Figure EV2 \n",
      ".Figure 1Schematic representation of the smFRET assay using crystal-structure snapshots of McjD. On the right side McjD is also shown as a cartoon. In the crystal structure nucleotides and ATP-Mg in orange sticks, the peptide MccJ25 in pink. Each half transporter is colored in dark and light grey. McjD is shown in a view along the plane of the membrane which is depicted in light grey. Colored balls show the position of the cysteines used for probing conformational changes. PBD codes for McjD in inward-occluded conformation are 5OFP (left) (Bountra et al, 2017) and in the outward-occluded conformation 4PL0 (right) (Choudhury et al, 2014) Coomassie stained SDS-PAGE of McjD-C547 reconstituted in liposomes. The amount reconstituted was evaluated against detergent purified protein Chromatogram of McjD C547 (black) labelled with Alexa 555 (blue) and Alexa 647 (red) maleimide fluorophore using a Superdex 200 size-exclusion column. The fraction containing the highest degree of labeling is marked by a square The ATPase activity of all labelled variants C547, Y64C, L67C was determined in liposomes and compared to wild-type, wt. All labelled McjD variants show basal- and ligand-induced ATPase activities in liposomes comparable to that of wild type protein, indicating that labelling does not interfere with their activity. Error bars were calculated from two replicates from two independent reconstitutions (mean ± standard deviation, n=2). The ruler character of FRET using the fluorophore pair Alexa555/Cy5 was supported by data of dsDNA in Figure EV1B, where different base-pair distances between both dyes correlate directly with the apparent FRET efficiency \n",
      ".Figure 6  Figure 6. Refined transport mechanism of ABC exporter McjD based on structural, biochemical and biophysical data. A, B) Binding of ATP alone (futile cycle) is not coupled to an opening of the TMD that remains occluded. NBDs can close in the presence of ATP and do not require substrate for closure. On the other hand, the NBDs are more dynamic and they can sample a nearly closed dimer that is driven to full closure in the presence of ATP. Binding of both ATP and MccJ25 are coupled to open up the TMD, which reverts back to an occluded conformation upon substrate release. ATP hydrolysis and ADP release resets the transporter in an inward conformation for a new transport cycle. Expanded View Figure Legends  \n",
      ".Figure 4Representative FRET time traces (black, FRET signal; yellow, fit) of McjD NBD C547 labeled with Alexa 555 and Alexa 647-maleimide under apo conditions at 10 ms time resolution. Approximately ~85% of all apo-McjD traces show no significant fluctuations beyond shot-noise. Apo-McjD is predominantly in the low FRET and thus open state with E* values <0.3 Within the complete data set of apo-McjD (N = 80 traces) comprising donor and acceptor, ~15%, show infrequent fluctuations to a higher FRET efficiency state with a lifetime of (C) 82 ± 25 ms where E* values > 0.5 are observed during short dwells. The data set represents a total recording time of 4.6 mins with a temporal resolution of 10 ms. To exclude unwanted influence on McjD due to surface-immobilization, experimental verification is provided that ATP-induced NBD switching occurs similarly with detergent solubilized McjD on the surface in Figure EV4 \n",
      ".Figure 3schematic showing the immobilization of McjD to the surface: labeled McjD reconstituted in biotinylated DOPE lipid was immobilized to a PEG-biotin-coated surface in a flow cell via a neutravidin-tag on the proteoliposome. A typical surface scan is shown in false-color representation on the right (orange, double-labeled McjD; green, McjD with donor fluorophore only; red, McjD with acceptor fluorophore only) Representative fluorescence time traces (blue, donor signal; red, acceptor signal; black, FRET signal; yellow, fit) of McjD labeled with Alexa555 and Alexa647-maleimide with indicated concentrations of corresponding substrate: (B) McjD L67C with 10 mM ATP and 200 µM MccJ25, McjD Y64C with 10 mM ATP and 200 µM MccJ25, and McjD C547 with 10 mM ATP. The respective right panels are the projections of accumulated time traces in the absence and presence of substrate for each variant. We note that differences between the setup dependent apparent FRET values are not influencing our interpretations since only relative changes of FRET efficiency are interpreted. Experimental support for proper reconstitution is provided in Figure EV3A showing longer bursts and distinct TEV cleavage for liposomes compared to detergent \n",
      ".Figure 5Representative FRET time traces (black, FRET signal; yellow, fit) of McjD NBD C547 labeled with Alexa 555 and Alexa 647-maleimide in apo conditions at 10 ms time resolution. This panel shows stable ATP-loaded (left) and ATP-free (right) McjD Representative FRET time traces that show switching between ATP free McjD (E* ~ 0.3) and ATP-loaded (high FRET >0.5) at ~4s s (left) and ~1.4 s (right). The traces show photobleaching events of the acceptor after ~10 s (left) and ~3.3 s (right) Representative FRET time traces that show switching between ATP-loaded (high FRET >0.5) and ATP free McjD (E* ~ 0.3) at 2 s (left) and 5 s (right). The traces show photobleaching events after ca. 40 s (left, donor-bleaching) and 70 s (right, acceptor-bleaching) \n",
      "a, A TRPML1Va-expressing HEK293T cell showed a large inwardly rectifying whole-cell current elicited by voltage steps (from -140 mV to +80 mV in increments of 20 mV) in the standard extracellular (Tyrode’s) bath solution.figf1a, A TRPML1Va-expressing HEK293T cell showed a large inwardly rectifying whole-cell current elicited by voltage steps (from -140 mV to +80 mV in increments of 20 mV) in the standard extracellular (Tyrode's) bath solution. Step duration, 90 ms; holding potential, 0 mV; Vm, membrane potential. b, No significant inward current was detected in NMDG+ (Na+-free, Ca2+-free, pH 4.6) solution. c, Inwardly rectifying step currents were evoked by 30 mM Fe2+ solution (pH 4.6) in the same cell as shown in a and b. d, pH-dependence of IFe/TRPML1Va. Whole-cell currents were elicited by repeated voltage ramps (-100 to +100 mV; 400 ms) with a 4-s interval between ramps. Only a portion of the voltage protocol is shown. Holding potential, 0 mV. e, Large IFe/TRPML2Va was seen in the presence of 30 and 105 mM Fe2+ (pH 4.6). f, Little or no IFe was seen in a TRPML3Va-expressing cell. \n",
      "a, Current densities (mean ± s.figf2a, Current densities (mean ± s.e.m., n = 4-10) of IFe (30 mM Fe2+, pH 4.6) for TRPML1Va and ML4 mutant TRPML1Va channels. Asterisk indicates statistical difference (P < 0.01) compared to TRPML1Va. b, 55Fe2+uptake (normalized) in HEK293T cells transfected with vector control, with TRPML1Va, with F408D-TRPML1Va and with T232P-TRPML1Va constructs. Error bars indicate the standard deviation on the basis of two independent triplicate experiments. c, [Fe2+]o-dependent quenching of Fura-2 fluorescence in TRPML1Va-transfected cells (arrows), but not in non-transfected control cells (arrowheads) or T232P-TRPML1Va-transfected cells (bottom row). The fluorescence intensity was measured at an excitation wavelength of 360 nm (F360). The original magnification used for all micrographs was ×200. d, Average normalized responses of EGFP-positive TRPML1Va-transfected cells (typically n = 20-40 cells) to 1 or 10 mM Fe2+ (pH 4.6). e, No significant quenching was seen in T232P-TRPML1Va-transfected cells. f, Slightly less quenching was observed for the F408Dgr;-TRPML1Va-expressing cells. g, A small but significant quenching reaction was detected (with 10 mM Fe2+) in R403C-TRPML1Va-expressing cells. \n",
      "a, Co-localization of mCherry–TRPML1 and EGFP–LAMP1 at the membrane of an isolated enlarged LEL (see Methods).figf3a, Co-localization of mCherry-TRPML1 and EGFP-LAMP1 at the membrane of an isolated enlarged LEL (see Methods). The patch pipette was filled with red rhodamine B dye (shown in blue for the purpose of illustration). c, Lysosomal ITRPML1Va. Switching from lysosome-attached to (lysosome) luminal-side-out configuration significantly reduced the amplitude of the current. The luminal-side-out patch was exposed to the Tyrode's solution. A Cs+-based solution (147 mM Cs-methanesulphonate (Cs-MSA)) was used as a pipette solution for both configurations. d, NMDG+-impermeable lysosomal ITRPML1Va was much larger in the absence of divalent cations (nominal divalent-free). e, Lowering pH potentiated lysosomal ITRPML1Va. f, IFe/TRPML1Va induced by 30 mM and 105 mM Fe2+. g, Whole-lysosome current in an enlarged lysosome expressing wild-type TRPML1. The pipette (lumen) solution contained nominal divalent-free Tyrode solution. A Cs+-based bath solution (147 mM Cs-MSA) was used. h, Whole-lysosome ITRPML1Va. i, Whole-lysosome IFe/TRPML1. The pipette (lumen) solution contained 105 mM Fe2+ (pH 4.6). \n",
      "-90 and 1-277 regions of M45 confer protection against TNF-induced necroptosis in human cells.Figure 1Sequence alignment of RHIMs from M45 and human and murine RIPK1, RIPK3 and ZBP1 Schematic representation of M45, RIPK1 and RIPK3 proteins. Grey regions indicate RHIM-encompassing fragments used in this study. Black bar represents 19-residue RHIM. RNR-like: inactive ribonucleotide reductase domain; DD: death domain; Z: double-stranded RNA-specific adenosine deaminase (DRADA) repeat Electron micrograph of fibrils formed by a synthetic peptide encompassing the M45 RHIM. Representative of three samples examined. Scale bar 100 nm HT29 cell survival 18h after treatment with TNF+BV-6+z-VAD-fmk to induce necroptosis. Cells expressed lentivirus constructs encoding 1-90 or 1-277 fragments of M45 and were cultured +/- the addition of coumermycin A1 to induce dimerisation. N=2 for M45-90 gyrase +/- coumermycin A1 and 4 for M45-277 gyrase and for empty vector, +/- coumermycin A1 \n",
      "a, Cultured TRPML1-/- (ML1-/-) skin fibroblasts showed less de-quenching of the iron-sensitive fluorescence than ML1+/- cells did.figf4a, Cultured TRPML1-/- (ML1-/-) skin fibroblasts showed less de-quenching of the iron-sensitive fluorescence than ML1+/- cells did. De-quenching was achieved by preloading the fibroblasts with an iron-sensitive dye, Phen Green SK (PG SK), and then adding the membrane-permeable transition metal chelator, 2,2′-bipyridyl (BPD). 2,2′-BPD is predicted to chelate free cellular iron (also referred to as chelatable or labile iron), which subsequently increases PG SK fluorescence. The original magnification used was ×200 b, The average 2,2′-BPD induced normalized change of fluorescence (Dgr;F/F0) for ML4 cells (ML1-/-; n = 6 experiments) is significantly (asterisk, P < 0.01) lower than for the parental ML1+/- cells. Fibroblast cells (n = 10-20) were analysed for each individual experiment. 4,4′-BPD, a 2,2′-BPD analogue that cannot bind Fe2+, did not induce any significant restoration of PG SK fluorescence. Error bars, s.e.m. c, An ML1-/- skin fibroblast cell showed autofluorescence in LAMP1-positive compartments. Autofluorescence (green) was detected within a range of excitation wavelengths (shown with excitation at 480 nm). No significant autofluorescence was observed for a ML1+/+ cell. Lysosomes were stained with a LAMP1 antibody (red). Differential interference contrast (DIC) images are shown for comparison. \n",
      " Assembly of RHIMs gives rise to homomeric and heteromeric structures.Figure 3Fluorescence polarisation profiles from incorporation of FITC-M451‑90 into excess unlabeled M451‑90 (green), RIPK1497‑583 (red), RIPK3387‑518 (blue) or ZBP1170-355 (purple). Average polarisation profile from smoothed triplicate samples plotted, with s.d. indicated. Experiment performed twice and representative results shown here Schematic representation of fluorescence spectroscopy experiments used to detect the formation of homo- and heteromeric amyloid fibrils. Fusion proteins containing RHIMs and different fluorescent partners are mixed together under assembly-permissive conditions and examined in the confocal volume. Co-assembly of two different fluorophores within one complex results in detection of coincident signals in the two channels Representative fluorescence intensity trace showing homomeric assembly over 40 s in mixtures of (C) mCHERRY-RIPK1497‑583 and YPet-RIPK1497‑583 and (D) mCHERRY-RIPK3387‑518 and YPet-RIPK3387‑518. E, F Representative fluorescence intensity traces showing heteromeric assembly over 40 s in mixtures of (E) mCHERRY-RIPK1497‑583 and YPet-RIPK3387‑518 (inset is expansion of region indicated by *) and (F) mCHERRY-RIPK1497‑583-AAAA and YPet-RIPK3387‑518. All mixtures examined in at least three different experiments on three separate occasions \n",
      " The RHIM-containing fragment forms the amyloid fibril core.Figure 2Thioflavin T fluorescence as a function of time, following dilution of WT or AAAA His‑Ub‑M451‑90, His-Ub-RIPK1497‑583, and His-Ub-RIPK3387‑518 from 8 M to 200 mM urea-containing buffer. Buffer sample contains ThT but no protein. Samples were tested in triplicate and solid line is the average of the triplicates and the error bars (in grey) represent +/- s.d Absorbance spectra from solutions containing Congo red with WT (black lines) or AAAA mutant (dark grey lines) His‑Ub‑M451‑90, His-Ub-RIPK1497‑583, and His-Ub-RIPK3387‑518 that have been dialysed against Assembly Buffer. Congo red sample alone with no protein (light grey line) served as a control for the experiment. Experiment conducted once Electron micrograph of material eluted in peak 1 from the three MBP fusion proteins, as labelled. These are representative micrographs of the material observed. Scale bar 100 nm in all panels Representative micrographs of insoluble material generated after treatment with thrombin removed the MBP. Experiment repeated three times. Scale bar 100 nm in all panel \n",
      " RHIM requires the intact core tetrad.Figure 4Representative fluorescence intensity traces collected over 50 s for mixtures of (A) mCHERRY-M451-90 and YPet‑only; (B) mCHERRY-M451-90 and YPet‑RIPK1497‑583; (C) mCHERRY-M451-90 and YPet-RIPK3387‑518; (D) mCHERRY-M451-90 and YPet-ZBP1170-355; (E) mCHERRY-M451-90AAAA and YPet‑only; (F) mCHERRY-M451-90AAAA and YPet-RIPK1497-583; (G) mCHERRY-M451-90AAAA and YPet-RIPK3387‑518 and (H) mCHERRY-M451-90 and YPet-ZBP1170-355. Independent replicates performed at least twice \n",
      " SDS AGE analysis of RHIM-based assemblies.Figure 6SDS AGE analysis of monomeric or assembled forms of YPet-RIPK3387‑518 and mCHERRY-M451-90WT or mCHERRY-M451-90AAAA. Monomeric forms of the proteins maintained in 8 M urea prior to electrophoresis, as indicated. Assembled samples incubated with 0 or 2% SDS at RT for 10 min before electrophoresis. Protein components as indicated above each lane. SDS AGE analysis of assemblies of mCHERRY-RIPK1497‑583 and YPet-RIPK3387‑518 with or without four times molar equivalents of His-Ub-M45. SDS AGE analysis of assemblies of YPet-RIPK3387‑518 and mCHERRY-ZBP1170-355 with or without four times molar equivalents of His-Ub-M45. \n",
      " Co-assembly of M451-90WT with RIPK1497‑583, His-Ub-RIPK3387‑518 and ZBP1170-355 results in the formation of heteromeric fibrils and dense fibrillar networks.Figure 5Representative electron micrographs of homofibrils formed by M451-90, RIPK1497‑583, His-Ub-RIPK3387‑518 or ZBP1170-355 individually, and heterofibrils formed by indicated mixtures.Samples prepared twice from independent protein preparations Heterofibrils containing mCHERRY-M451-90 and YPet‑RIPK1497‑583 C Heterofibrils containing mCHERRY-M451-90 and YPet-RIPK3387‑518 D Heterofibrils containing mCHERRY-M451-90 and YPet‑ZBP1170‑355 E Heterofibrils containing mCHERRY-RIPK1497‑583 and YPet-RIPK3387‑51 Heterofibrils containing mCHERRY- RIPK3387‑518 and YPet‑ZBP1170‑355 Pearson's correlation coefficients measured from two confocal images of the indicated protein mixtures. Heterofibrils containing ECFP-M451-90 with mCHERRY- RIPK1497‑583 and YPet-RIPK3387‑518 STED images of heteromeric fibrils formed by co-assembly of mCHERRY-M451-90 and YPet-RIPK3387‑518 \n",
      ":RIPK3 amyloid fibril structure A Structure of the RIPK1:RIPK3 RHIM core, prepared using PyMol Graphics System from PDB 5V7Z.Figure 7Structure of the RIPK1:RIPK3 RHIM core, prepared using PyMol Graphics System from PDB 5V7Z. The RIPK3 sequence is coloured in blue and RIPK1 is coloured in green. Tetrad residue side chains are shown as sticks; Tyr, Asn/Gln, Ser and Cys side chains are shown in wheat, red, grey and yellow, respectively. Core tetrad interactions across the opposing β‑sheets are indicated by the dotted box Model of M45:RIPK3 RHIM core, prepared by appropriate mutation of RIPK1. The M45 sequence is coloured in cyan and RIPK3 in blue. Tetrad residue side chains are shown as sticks; Tyr, Asn/Gln, Val and Cys side chains are shown in wheat, red, grey and yellow, respectively. Core tetrad interactions across the opposing sheets are indicated by the dotted box Schematic representation of RHIM fibrils formed by full-length proteins with partner domains (e.g. kinase or RNR, coloured green) flanking either side of the amyloid core formed by RHIMs (coloured grey) Confocal microscopy image of a single fibril composed of YPet-M451-90 \n",
      ".Figure 1(A) Normalized Hi-C contact maps showing contact frequencies between 10-kb bins across the Caulobacter genome during exponential growth (Le et al, 2013) and following nutrient starvation. Axes indicate genome position of each bin. (B-C) Hi-C contact maps for intra-chromosomal arm interactions (dashed box in panel A) rotated 45o clockwise. Images were resized in each dimension after rotation, but otherwise unprocessed relative to panel A. CID boundaries are marked with vertical purple lines and numbered. Directional preference plots were used to assign domain boundaries (see Materials and Methods and Appendix Fig S1B-C). Left- and rightward preferences are shown as green and red bars, respectively. Abrupt transitions from left- to rightward preference correspond to chromosomal interaction domain boundaries. Plots of gene expression 'rpkpm * transcript length' for each 10-kb bin along the chromosome are shown beneath the directional preference plots. Genomic locations of long, highly-expressed genes or operons (see Dataset EV2) are indicated by orange vertical lines. (D-G) Comparison of normalized Hi-C contact maps for regions of the Caulobacterchromosome during exponential growth and following starvation. The black dashed line shows the genomic location of (D) an rRNA locus, (E) a ribosomal protein gene cluster, (F) an ATP synthase gene cluster and (G) starvation-induced genes CCNA03325, CCNA03327, CCNA03328 and CCNA03337. \n",
      ".Figure 3Normalized Hi-C contact maps for ΔrsaAvan::promoterless-rsaA, ΔrsaAvan::PrsaA-rsaA (ATG→TAA), and ΔrsaAvan::PrsaA-rsaA (anti-termination + ATG→TAA) cells. Only the region of the genome containing the van locus (dashed line) is shown. Above each Hi-C contact map is a schematic of the corresponding rsaA construct. The transcription anti-terminator is shown as a red hairpin, the start codon mutated to TAA is in red font, and the transcript produced is shown as a wavy red line. Below each map is the corresponding directional preference plot. \n",
      ".Figure 2Normalized Hi-C contact maps for ΔrsaAvan::promoterless-rsaA and derivatives of ΔrsaAvan::PrsaA-rsaA with transcription terminators inserted 60, 560, 1060, or 2060 bp from the transcription start site. Only the region of the genome containing the van locus (dashed line) is shown. Above each Hi-C contact map is a schematic of the corresponding rsaA construct with the transcription terminator shown as a black hairpin and transcript as a wavy red line. Below each map is the corresponding directional preference plot. Normalized Hi-C contact maps for ΔrsaAvan::promoterless-rsaA and derivatives of ΔrsaAvan::PrsaA-rsaA with transcription terminators inserted 60, 560, 1060, or 2060 bp from the transcription start site. Only the region of the genome containing the van locus (dashed line) is shown. Above each Hi-C contact map is a schematic of the corresponding rsaA construct with the transcription terminator shown as a black hairpin and transcript as a wavy red line. Below each map is the corresponding directional preference plot. \n",
      ".Figure 4(A) Normalized Hi-C contact maps for untreated (left) and chloramphenicol-treated cells (right). (B) Hi-C contact maps for intra-chromosomal arm interactions (dashed box in panel A) rotated 45o clockwise with directional preference plots below. Images were resized in each dimension after rotation, but otherwise unprocessed relative to panel A. Left- and rightward preferences are shown as green and red bars, respectively. Vertical dashed black lines indicate the positions of the largest ribosomal protein cluster and the two rRNA loci in the Caulobacterchromosome. \n",
      ".Figure 5(A) A γδ TnpR recombinase (green) was induced from a cumate-inducible promoter. TnpR recombines two res site (blue and red arrows) that flank a tet resistance cassette (orange arrows) to excise the intervening DNA. Recombination requires supercoiling-dependent juxtaposition of res sites. (B) Induction of TnpR from Pcumate results in a loss of cell viability on tetracycline plates. Each spot is a 10-fold dilution. (C) Percentage of tetracycline-resistant colonies represent a proxy for the extent of supercoil diffusion inhibition by the various PrsaA-rsaA constructs indicated, each inserted between one res site and the tetR cassette. Error bars represent S.D., n=3. The ability of each construct to form a CID boundary is indicated on the left (see Fig 2-3). (D) LacI binding to 10 consecutive lacO sites in between two res sites does not significantly inhibit supercoil diffusion. TnpR and LacI-YFP (brown arrow) were induced from Pcumate and Pxyl, respectively. \n",
      ".Figure 6(A) Pairs of loci were labeled with YFP-ParBpMT1/parSpMT1 and mCherry-ParBP1/parSP1 at genomic positions 400 kb and 600 kb (inter-domain) or at 600 kb and 800 kb (intra-domain). Box plots show the distribution of inter-focus distances for cells of different sizes. Asterisks indicate statistical significance (***, Student's t-test p-value <0.001; ns, not significant). For the region of the chromosome containing the labeled loci (indicated by dashed grey lines), the corresponding region of the Hi-C map generated at each time point indicated is shown, with the same region from wild-type cells shown at the bottom. For complete Hi-C maps, see Fig EV4; for representative microscopy images, see Fig S3. (B) Same as panel A but for a strain producing CtrA(D51E) Δ3Ω. (C-D) Same as panels A-B, but with pairs of loci labeled with YFP-ParBpMT1/parSpMT1 and mCherry-ParBP1/parSP1 at genomic positions 1400 and 1600 kb (inter-domain) or at 1600 and 1800 kb (intra-domain). Note that differences in Hi-C chromosomal domain sharpness reflect differences in interaction patterns as the color scale and image contrast settings are identical across each time series. \n",
      ".Figure 7Compacted chromosomal interaction domains (red and green) are connected by a domain boundary denoted as a blue line. (Top) High rates of transcription that produce a relatively long (> 2 kb) transcript (wavy orange lines) locally decompact the DNA, thereby creating a spatial gap between flanking loci. This spatial separation diminishes the long-range contacts that can occur within a fixed sphere (black dotted circle), leading to the domain boundaries visible by Hi-C analysis. High rates of transcription can also prevent supercoil diffusion, helping to limit DNA-DNA interactions on a shorter length scale (grey dotted circle). (Bottom) In the absence of transcription, DNA at a domain boundary (blue) will become more compacted, reducing the spatial separation between adjacent domains, and enabling supercoil diffusion, thereby promoting long- and short-range contacts, respectively. \n",
      ": Assessment of TRAIL-R2 DISC stoichiometry.Figure 1(A) Western blot analysis of Caspase 8, FLIP and FADD recruitment to the TRAIL-R2 DISC in A549, HCT116 and DU145 cell lines following incubation with increasing concentrations (1x, 2x, or 4x) of anti-TRAIL-R2 (AMG655)-conjugated magnetic beads for 90 minutes. (B) Western blot analysis of the unbound soluble fraction An untreated input of each cell line is included for comparison. (C) Quantification of Western blot analysis for FLIP, FADD, Procaspase 8 and their respective cleavage fragments at the TRAIL-R2 DISC. Proteins were quantified by densitometry and normalised to known protein standards (D) Caspase-8-like (IETDase) activity assay in the unbound soluble fraction from the DISC IP An untreated control (0) was used for each cell line to show basal caspase-8 activity. (E) Ratios of DED proteins quantified at the TRAIL-R2 DISC in A549, HCT116 and DU145 cells; increasing concentrations of AMG655 are indicated by colour intensity. (F) Correlation between caspase-8 activity and ratio of FADD:FLIP(L). \n",
      ": Impact of decreasing procaspase-8 on DISC assembly and caspase activation.Figure 2(A) Western blot analysis of FADD, FLIP and Caspase 8 at the TRAIL-R2 DISC in HCT116 cells treated with escalating doses (0-30nM) of caspase-8 siRNA for 48 hours. Samples were incubated with anti-TRAIL-R2 (AMG655) beads (4x) for 90 minutes prior to collection. A scrambled control siRNA (SC) was transfected (30nM) for comparison. (B) Quantification of Caspase 8 (p55, p43/41 and p26/24) and FLIP (FLIP(L), FLIP(S) and p43-FLIP) at the TRAIL-R2 DISC by densitometry and normalised to known protein standards (C) Ratio of Caspase-8:FLIP calculated from values presented in (B). (D) Caspase-8 (IETDase) and -3/7 (DEVDase) activity assay in the unbound soluble fraction (E) Western blot analysis of FLIP, Caspase 8, and FADD recruitment to the TRAIL-R2 DISC in PC3 cells stably expressing an empty vector (EV) or flag tagged FLIP L (FL) treated with 2.5µM MS-275 for 48 hours, followed by a 90 minute DISC IP. * modified form of FLIP, potentially mono-ubiquitinated p43-FLIP(L). (F) Table of the relative ratios of Caspase-8 (p55, p41/43 and p24/26) and FLIP (endogenous and Flag-tagged) at the TRAIL-R2 DISC in MS-275 treated and untreated cells after a 90 minute DISC IP. (G) Caspase-8 (IETDase) activity assays of the unbound soluble fraction \n",
      ": Although FLIP can interact directly with FADD in a caspase-8-independent manner, its DISC recruitment is highly caspase-8-dependent.Figure 3(A) Western Blot analysis of Caspase 8, Caspase 10, FLIP and FADD recruitment to the TRAIL-R2 DISC in procaspase 8 (WT) or procaspase 8 deficient (Null) A549 cells incubated with AMG655 beads (4x) for either 30, 60 or 180 minutes. (B) Western blot analysis of the recruitment of FLIP, Caspase 8, Caspase 10 and FADD to the TRAIL-R2 DISC in HCT116 Parental, caspase 8 null (CASP8), caspase 10 null (CASP10) or CASP 8/10 cells treated with 4x AMG655 conjugated beads for 90 minutes. (C) AlphaScreen® assessment of the interaction of recombinant FLIP DED1/2 and recombinant FADD DED. (D) AlphaScreen® assessment of the impact on the FLIP-FADD interaction of unstapled and stapled peptides corresponding to the FADD α1 helix. (E) NanoBiT® assay to quantify FLIP/FADD interactions in a U2OS cell line stably expressing the NanoBiT® constructs, FLIP (LgBiT) and FADD (smBiT) following 48 hour silencing of FLIP, FADD, or caspase 8. Western blot analysis was used to confirm successful knockdown of target proteins by siRNA. (F) NanoBiT® assay of HCT116 and U20S wild type (WT) and procaspase 8 null (Null) cells transiently co-transfected with FLIP (LgBiT) and FADD (SmBiT) NanoBiT® constructs for 24 hours. Construct expression was analysed by Western blotting. \n",
      ".Figure 6(A) Schematic representation of a model for DISC structure consistent with the experimental observations in this paper. Key: green chevrons: FADD DED; orange chevrons: FLIP DED1/2; purple chevrons: caspase-8 DED1/2; black triangle: trimeric TRAIL-R2. (B) NanoBiT® assay performed on TRAIL-R2 DISC IPs (90') carried out in U20S Caspase-8 null cells co-transfected with wild-type or F114A FLIP-LgBiT fusion proteins and wild-type or F122A caspase-8-SmBiT fusion proteins. (C) Docking Connolly surface models of interactions between the DEDs of FADD (green), FLIP (orange) and caspase-8 (purple). (D) Potential DISC structure linking two receptor trimers via DED chains. Putative caspase domain interactions are depicted. (E) Western blot analysis of size exclusion chromatography experiments in which HCT116 cells transfected with Flag-tagged wild-type (WT) FLIP(S) were treated with 100ng/mL IZ-TRAIL for 3h prior to cell lysis and separation of low and high MW complexes using a size exclusion column. The fractions eluted at 2MDa and >2MDa are presented. \n",
      ": FADD recruitment to the TRAIL-R2 DISC is impaired in the absence of procaspase-8.Figure 4(A) Western blot analysis of FLIP, Caspase 8 and FADD recruitment to the TRAIL-R2 DISC in U20S parental cells (EV) and 3 independent caspase 8 null clones (#1, #2, #3) after incubation with AMG655 conjugated beads for 30, 60 or 180 minutes. (B) Western blot analysis of FLIP, Caspase 8 and FADD in the soluble unbound fraction (C) Quantification of FADD recruitment to the TRAIL-R2 DISCs FADD levels were normalised to TRAIL-R2 levels in the pull-downs. Densitometry was performed using ImageJ®. Here, we present the Connolly (solvent-excluded) surface, with the positions of the H9 and F25 residues highlighted. The accompanying chevron is a short-hand representation of the 6 α-helices of FADD's DED. (E) Western blot analysis of FADD recruitment to the TRAIL-R2 DISC in HCT116 cells transiently transfected with an empty vector (EV), Flag-tagged FADD (WT), or Flag tagged FADD with point mutations at either Phe25 (F25A) or His9 (H9G). Western blotting analysis of the soluble unbound fraction was used to monitor transfection efficiency. (F) NanoBiT® assay of U20S cells transiently co-transfected with caspase 8 (LgBiT) and either WT FADD smBiT or FADD (smBiT) constructs with a point mutation at either phenylalanine 25 (F25A) or histidine 9 (H9G) for 48 hours. Western blot analysis was used to assess the expression level of each construct. \n",
      ": Impact of FLIP deletion on DISC assembly and caspase activation.Figure 5(A) Western blot analysis of FLIP, caspase 8 and FADD recruitment to the TRAIL-R2 DISC in parental (WT) and CFLAR deficient (KO) HAP-1 cells treated with AMG655 conjugated beads for either 45 minutes (45'), 90 minutes (90'), 180 minutes (180'), 3 hours (3h), 6 hours (6h) or 9 hours (9h). (B) Caspase activity assays of the unbound soluble fraction (C) Caspase-8 (i) and -3/7 (ii) activity assays and high content microscopy assessment of cell death (iii) in Parental (PAR) and FLIP knockout (CFLAR CRISPR) cells treated with escalating doses of isoleucine zipper TRAIL (IZ-TRAIL) for 6 hours. (D) Caspase-8 (i) and -3/7 (ii) activity assays in Parental (PAR) and FLIP knockout (CFLAR CRISPR) cells treated with escalating doses of isoleucine zipper TRAIL (IZ-TRAIL) for 24 hours. (iii) Flow cytometry assessment of cell death induction in Parental (PAR) and FLIP knockout (CFLAR CRISPR) cells treated with IZ-TRAIL for 24h. (E) Western Blot analysis of Parental (WT) and FLIP knockout (KO) HAP-1 cells treated with 10ng/mL IZ-TRAIL for 0, 3, 6 or 24 hours. (F) Cell viability assay of Parental (PAR) and FLIP knockout (CFLAR CRISPR cells) treated with an escalating dose (0-1000ng/mL) of IZ-TRAIL for 72 hours. \n",
      ".Figure 1(A) RPE1 cells were transfected with either scramble or TMEM135 siRNAs and immunostained for LAMP1 as a lysosome marker (red) and PMP70 as a peroxisome marker (green). Scale bar, 10 μm. (B) Quantification of overlap signal between lysosomes and peroxisomes shown in (A). Data represent mean ± SD (n=3 experiments), 50 cells were scored per condition per experiment. *P < 0.05, Student's t-test. (C) Subcellular fractions of a discontinuous sucrose gradient collected from top to bottom, separated by SDS-PAGE, and immunoblotted for LAMP1, EEA1, Rab11, Rab8, PMP70, GM130, and Na+/K+ ATPase. (D) Quantification of total cholesterol in the subcellular fractions Data represent mean ± SD (n=3 experiments), *P < 0.05, Student's t-test. (E) Cells transfected with TMEM135 siRNA were subjected to a SREBP2 cleavage assay for the precursor of SREBP2 and nuclear SREBP2. (F) Cells transfected with TMEM135 siRNA were subjected to qPCR. Data represent mean ± SD (n=3 experiments), *P < 0.05, Student's t-test. \n",
      ".Figure 3(A) RPE1 cells were transfected with siRNA as indicated, and immunostained for Rab8 (red) and γ-tubulin (green) in either 10% FBS medium (upper panel) or serum-starvation medium (lower panel). Scale bar, 10 μm. Arrowhead indicates Rab8 localized to the centrioles. (B) Quantification of the percentage of cells with Rab8 localized to the centriole Data represent mean ± SD (n=3 experiments), 200 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (C) Cells were transfected with siRNA and immunostained for IFT20 (red) and γ-tubulin (green). Scale bar, 10 μm. (D) Quantification of the percentage of IFT20 intensity at the centrioles Data represent mean ± SD (n=3 experiments), 150 cells were scored per condition per experiment, *P < 0.05, Student's t-test. \n",
      ".Figure 2(A) RPE1 cells were transfected with siRNAs as indicated, followed by serum starvation for 24 h, and immunostained for ARL13B (red) and γ-tubulin (green). Scale bar, 10 μm. (B) Quantification of the percentage of ciliated cells shown in (A). Data represent mean ± SD (n=3 experiments), 250 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (C) Cells were transfected with siRNAs as indicated and incubated with HPβCD in serum-starvation media followed by filipin staining. Scale bar, 20 μm. (D) Cells were transfected with siRNAs shown in (C) and immunostained for ARL13B (red) and γ-tubulin (green). Scale bar, 10 μm. (E) Quantification of the percentage of ciliated cells shown in (D). Data represent mean ± SD (n=3 experiments), 250 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (F) Cells were transfected with siRNAs as indicated, incubated with cholesterol-MβCD complex in serum-starvation media, and immunostained for ARL13B (red) and γ-tubulin (green). Scale bar, 10 μm. (G) Quantification of the percentage of ciliated cells shown in (F). Data represent mean ± SD (n=3experiments), 250 cells were scored per condition per experiment, *P < 0.05, Student's t-test. \n",
      ".Figure 4(A) RPE1 cells were transfected with siRNAs as indicated, followed by transfection with wild-type pGFP-Rab8a (WT-Rab8), constitutively active pGFP-Rab8a (Q67L) (CA-Rab8), or DN Rab8 dominant-negative pGFP Rab8a (T22N), incubated in serum-starved media for 12 h, and immunostained for ARL13B (Red), GFP-Rab8 (green), and DAPI (blue). Scale bar, 10 μm. (B) Quantification of the percentage of ciliated cells Data represent mean ± SD (n=3 experiments), 200 GFP-positive cells were scored per condition per experiment, *P < 0.05, Student's t-test. (C) (Upper panel) Cells were transfected as indicated and the cell lysates were incubated with purified proteins, including GST or GST-JCF1 (RBD). The amount of GTP-Rab8 bound to GFT-JCF1(RBD) was analyzed by western blot with Rab8 antibody. (Lower panel) Intensity of the bands was quantified by Image J software. The amount of GTP-Rab8 were normalized to the control level. Bar graph represents mean ± SD (n=3 experiments). *P < 0.05, Student's t-test. (D) Cells were transfected and cell lysates were incubated with purified GST-JCF1 (RBD) fusion protein. The amount of GTP-Rab8 bound to GFT-JCF1(RBD) was analyzed by Western blot with Rab8 antibody. (Lower panel) Cells were transfected and cell lysates were incubated with purified GST- protein. The amount of GTP-Rab8 bound to GFT-JCF1(RBD) was analyzed by Western blot with Rab8 antibody. \n",
      ".Figure 5(A) RPE1 cells were transfected with siRNAs as indicated, followed by 24 h incubation in serum-starved media. Cells were then and subjected to fractionation, and Western blot for IFT20, the Golgi marker GM130, membrane marker UBXD8, and nuclear marker CREB. (B) Efficiency of IFT20 knockdown by western blot. (C) Cells were transfected with siRNAs as indicated, followed by incubation in serum-starved media for 24 h, and immunostained for ARL13B (red). Scale bar, 10 μm. The bar graph represents the quantification of the percentage of ciliated cells. Data represent mean ± SD (n=3 experiments), 250 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (D) Cells were transfected and immunostained for Rab8 and γ-tubulin, followed by quantification of the percentage of cells with Rab8 localized to the centriole. Data represent average (n=2 experiments). (E) Cells were transfected with siRNAs as indicated, followed by transfection with Flag-IFT20, incubation in serum-starvation media for 12 h, and immunostained for ARL13B. Representative fluorescent images of Flag-IFT20 (green), ARL13B (red), and DAPI (blue) are shown. Scale bar, 10 μm. (F) Quantification of the percentage of ciliated cells with both the Flag-IFT20 and ARL13B localized in the cilium. Data represent mean ± SD (n=3 experiments), 150 Flag positive cells were scored per condition per experiment, *P < 0.05, Student's t-test. (G) Cells were transfected with siRNAs as indicated, followed by further transfection with CA-Rab8, incubation in serum-starvation media for 12 h, and immunostained for acetylated-tubulin. Representative fluorescent images of GFP-Rab8 Q67L (green), acetylated-tubulin (red), and DNA (blue) are shown. Scale bar, 10 μm. (H) Quantification of the percentage of GFP -positive ciliated cells (only those cilia having both GFP-Rab8 and acetylated-tubulin on cilium were considered for quantification). Data represent mean ± SD (n = 3 experiments), 200 GFP positive cells were scored per condition per experiment, *P < 0.05, Student's t-test. \n",
      "The Atg12—Atg5–Atg16L1 complex regulates the stability of the ULK1–Atg13–FIP200 complex.Figure 2(A) Immunoblot analysis of Atg proteins in the indicated MEFs. (B) m5‐7 cells were cultured in regular medium containing 10 ng/ml Dox for 8 days, and then cultured in the absence of Dox for a further 2 days. (C, D) m5‐7 cells were cultured in regular medium with or without 10 ng/ml Dox for 4 days, and then cultured in the presence of 50 μg/ml CHX for the indicated time periods (C). mRNA levels of the indicated genes were measured by quantitative PCR following the 4‐day Dox treatment. Data represent mean±s.e. (*P0.05) (D). CHX, cycloheximide; Dox, doxycycline; MEFs, mouse embryonic fibroblasts; mRNA, messenger RNA; WT, wild‐type. \n",
      "Atg16L1 interacts with the ULK1–Atg13–FIP200–Atg101 complex.Figure 1(A) Yeast two‐hybrid screening for Atg16L1‐interacting proteins. The Atg16L1‐interacting FIP200 clones are indicated. (B,C) HEK293T cells were co‐transfected with the indicated constructs. Cell lysates were subjected to IP using anti‐FLAG antibodies. The resulting precipitates were examined by immunoblot analysis with anti‐FLAG and anti‐HA antibodies. (D) HEK293T cells were cultured in regular or starvation medium for 2 h. Cells were harvested and treated with DSP. (E) HEK293T cells were harvested and treated with DSP. (F) HEK293T cells were cultured in regular DMEM or starvation medium in the presence or absence of 200 nM wortmannin for 2 h. After treatment with DSP, cell lysates were subjected to IP analysis. (G) Atg14F/F (undeleted) or Atg14Δ/Δ (deleted) MEFs were harvested and treated with DSP. * (E) and ** (G) indicate the positions of the immunoglobulin light and heavy chains, respectively. aa, amino acid; DSP, dithiobis(succinimidyl propionate); IP, immunoprecipitation; MEFs, mouse embryonic fibroblasts. \n",
      ".Figure 7(A) RPE1 cells were transfected as indicated, followed by incubation in serum-starvation media in the presence or absence of cholesterol/MCD for 24 h, and then immunostained for Rab8 (red) and γ-tubulin (green). Scale bar, 10 μm. Representative magnified images are shown from cells labeled with white asterisks. (B) Quantification of the percentage of cells with Rab8 localized to the centriole Data represent mean ± SD (n=3 experiments), 200 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (C) (Upper panel) Cells were transfected and cell lysate was incubated with purified GST-JCF1 (RBD) fusion protein. The amount of GTP-Rab8 bound to GFT-JCF1(RBD) was analyzed by western blot with Rab8 antibody. (Lower panel) The intensity of bands was quantified by Image J software. The amount of GTP-Rab8 were normalized to the control level. The bar graph represents mean ± SD (n=3 experiments). *P < 0.05, Student's t-test. (D) Cells were transfected and immunostained for IFT20 (red) and γ-tubulin (green). Scale bar, 10 μm. (E) Quantification of IFT20 intensity at the centriole Data represent mean ± SD (n=3 experiments), 150 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (F) A proposed model showing the importance of cholesterol for primary ciliogenesis. \n",
      ".Figure 6(A) Efficiency of Rab8a depletion confirmed by Western blot in RPE1 cells. (B) RPE1 cells were transfected by siRNAs as indicated, followed by incubation in serum-starvation media for 24 h, and immunostained for ARL13B (red) and γ-tubulin (green). Scale bar, 10 μm. (C) Quantification of the percentage of ciliated cells Data represent mean ± SD (n = 3 experiments), 250 GFP positive cells were scored per condition per experiment, *P < 0.05, Student's t-test. (D) Cells were transfected by siRNa as indicated followed by incubation in serum-starvation media for 24 h, and immunostained for EHD1 (red) and γ-tubulin (green). Scale bar, 10 μm. (E) Quantification of the percentage of cells with EHD1 in cilium or in the distal end of basal body as shown in (D). Data represent mean ± SD (n=3 experiments), 150 cells were scored per condition per experiment, *P < 0.05, Student's t-test. (F) Cells were transfected by siRNAs as indicated, followed by incubation in serum-starvation media for 24 h, and immunostained for IFT20 (red) and γ-tubulin (green). Scale bar, 10 μm. (G) Quantification of the percentage of IFT20 fluorescent intensity at the centriole shown in (F). Data represent mean ± SD (n=3 experiments), 150 were scored per condition per experiment, *P < 0.05, Student's t-test. (H, I, J) Cells were transfected by siRNAs as indicated, followed by transfection with GFP-Rab8 WT, GFP-Rab8 Q67L, or GFP-Rab8 T22N, and further incubated in serum-starvation media for 12 h. Cell lysate was subjected to immunoprecipitation with anti-GFP antibody, followed by Western blot with antibody against GFP. \n",
      "The middle region of Atg16L1 is required for interaction with FIP200.Figure 3(A) Schematic representations of the wild‐type Atg16L1 and its mutants. Human Atg16L1α (NP_060444) has an amino‐terminal Atg5‐interacting domain (grey box), a coiled-coil domain (shaded box) and seven WD repeats (black boxes). Star indicates the position of the Crohn's disease‐associated T281A mutation. (B, C) HEK293T cells were co‐transfected with the indicated constructs. Cell lysates were analysed by IP. IP, immunoprecipitation; ND, not determined. \n",
      "Atg16L1Δ(230–300) mutant is impaired in isolation membrane targeting and partially restores the autophagic defects of Atg16L1 KO MEFs.Figure 4(A) WT MEFs, Atg16L1 KO MEFs or Atg16L1 KO MEFs stably expressing either full‐length Atg16L1(1-588), Atg16L1(1-230) or Atg16L1Δ(230-300) were cultured in regular DMEM or starvation medium in the presence or absence of 100 nM BafA1 for 2 h. * indicates nonspecific immunoreactive bands. (B, C) Atg16L1 KO MEFs stably expressing GFP-ULK1 and either full‐length Atg16L1(1-588), Atg16L1(1-230) or Atg16L1Δ(230-300) were cultured in regular DMEM (C) or starvation medium (B,C) for 1 h. Cells were fixed and analysed by immunofluorescence microscopy using anti‐GFP, anti‐Atg16L1 and anti‐LC3 antibodies. Arrowheads indicate the perinuclear localization of Atg16L1(1-230). The number of dots was quantified from more than 30 randomly selected cells from three independent samples as described in the Methods. Data represent mean±s.e. (*P0.001, analysis of variance followed by Bonferroni/Dunn post hoc test). Scale bar, 10 μm, and 2 μm in inset. (D, E) Cells stably expressing GFP-LC3 were cultured in regular DMEM, starvation medium or starvation medium in the presence of 0.2 μM WM or 100 nM BafA1 for 6 h. Total cellular GFP-LC3 signals were analysed by flow cytometry. Representative FACS data were shown (D). The geometric mean of fluorescence intensity was determined. Values are expressed as a percentage of the mean of control cells cultured in regular DMEM. Data represent mean±s.e. (*P0.05) (E). BafA1, bafilomycin A1; GFP, green fluorescent protein; KO, knockout; MEFs, mouse embryonic fibroblasts; NS, not significant; WM, wortmannin; WT, wild‐type. \n",
      "Atg16L1(1–230) localizes to aberrant membranes in a FIP200‐independent manner.Figure 5(A) Atg16L1 KO MEFs stably expressing either full‐length Atg16L1(1-588), Atg16L1(1-230) or Atg16L1Δ(230-300) were cultured in regular DMEM or starvation medium for 1 h. Cells were fixed and subjected to immunofluorescence microscopy using anti‐Atg16L1 antibody. Note that full‐length Atg16L1 and Atg16L1(1-230), but not Atg16L1Δ(230-300), showed punctate structures under starvation conditions (inset). (B) FIP200 KO MEFs stably expressing either CFP-Atg16L1(1-588), CFP-Atg16L1(1-230) or CFP-Atg16L1Δ(230-300) were cultured in regular DMEM or starvation medium for 1 h. Cells were fixed and analysed by immunofluorescence microscopy using anti‐GFP antibody. Arrowheads indicate the perinuclear localization of Atg16L1(1-230). (C, D) FIP200 KO MEFs stably expressing CFP-Atg16L1(1-230) were cultured in regular DMEM. Scale bar, 10 μm, and 2 μm in inset. (E) Hypothetical model of membrane targeting of Atg16L1. Interaction of Atg16L1 with FIP200 leads to proper targeting of this complex to the autophagosome formation site (middle). Without FIP200, puncta formation of Atg16L1 is impaired probably owing to a self‐inhibitory role of the C‐terminal WD‐repeat domain (left). If the WD‐repeat domain is deleted, the N‐terminal half of Atg16L1 localizes to aberrant membranes as well as the autophagosome formation site (right). KO, knockout; MEFs, mouse embryonic fibroblasts. \n",
      "a) Flow cytometric analysis of the survival of CD45.figf3(a) Flow cytometric analysis of the survival of CD45.2+ Ifng−/− (open symbols) or Irgm1−/−Ifng−/− (filled symbols) CD4+ T lymphocytes preactivated with agonistic mAb to CD3 in vitro, then transferred intraperitoneally into congenic CD45.1+ C57BL/6.SJL recipients left untreated or inoculated intraperitoneally with T. gondii or M. avium 3 d earlier, assessed in peritoneal exudate cell samples collected at day 4 after transfer. (b) Survival of M. avium-infected wild-type, Irgm1−/−, Ifng−/− and Irgm1−/−Ifng−/− mice (n = 5 mice per genotype). (c-e) Splenic bacterial loads (c), circulating lymphocytes (d) and splenic lymphocytes (e), assessed at week 4 after M. avium infection (n = 5 mice per genotype). Each symbol in a,c-e represents an individual mouse. Data are representative of two independent experiments. \n",
      "a) [3H]thymidine incorporation at 72 h by wild-type (WT) and Irgm1−/− CD4+ T cells activated with soluble mAb to CD3 and irradiated syngenic wild-type splenocytes.figf1(a) [3H]thymidine incorporation at 72 h by wild-type (WT) and Irgm1-/- CD4+ T cells activated with soluble mAb to CD3 and irradiated syngenic wild-type splenocytes. (b) [3H]thymidine incorporation at 72 h by OT-II and Irgm1-/- OT-II CD4+ T cells stimulated with OVA protein (OVA) or OVA peptide of amino acids 323-339 (OVA(323-339)) in the presence of syngenic splenic DCs. (c,d) Kinetic analysis of [3H]thymidine incorporation (c) and cell numbers (d) in cultures of OT-II and Irgm1−/− OT-II spleen cells stimulated with OVA peptide. (e) Flow cytometric analysis of the activation and proliferation of OVA peptide-stimulated OT-II and Irgm1-/- OT-II CD4+ T cells pulsed with BrdU during the final 4 h of 72 h of culture and then stained with mAb to CD25 and BrdU. Numbers in quadrants indicate percent viable CD4+ cells. (f) Enzyme-linked immunosorbent assay of IL-2 produced by OT-II and Irgm1-/- OT-II CD4+ T cells stimulated with OVA peptide in the presence of splenic DCs, measured at 72 h. Data are representative of more than five (a,b) or three (c-f) independent experiments with similar results (mean and s.d. of triplicate cultures, a-d,f). \n",
      "a) Immunoblot analysis of Irgm1 in lysates of flow cytometry–sorted naive wild-type CD4+ lymphocytes activated for 48 h with immobilized agonistic mAb to CD3 (10 μg/ml) alone or with mAb to CD28 (2 μg/ml) in the presence or absence of IFN-γ-neutralizing mAb.figf2(a) Immunoblot analysis of Irgm1 in lysates of flow cytometry-sorted naive wild-type CD4+ lymphocytes activated for 48 h with immobilized agonistic mAb to CD3 (10 μg/ml) alone or with mAb to CD28 (2 μg/ml) in the presence or absence of IFN-γ-neutralizing mAb. Actin, loading control. (b) Enzyme-linked immunosorbent assay of IFN-γ produced by OT-II and Irgm1−/− OT-II CD4+ T cells stimulated with OVA peptide in the presence of splenic DC in polarized conditions (TH1 or TH2) or nonpolarized conditions (N), assessed in 72-hour cultures. ND, not detected. (c) Accumulation of polarized and nonpolarized OT-II (open symbols) or Irgm1−/− OT-II (filled symbols) CD4+ T cells after peptide stimulation, quantified by counting of viable cells. (d) Viable OT-II and Irgm1−/− OT-II CD4+ T cells 72 h after restimulation with OVA peptide in the presence of T cell-depleted APC samples. (e) Flow cytometry of the viability and division of OT-II and Irgm1−/− OT-II CD4+ T cells stimulated with peptide, allowed to 'rest' for 7 d in IL-2, then labeled with CFSE and restimulated with OVA peptide in the presence of T cell-depleted APC samples and assessed 72 h later. (f) Flow cytometric analysis of the population expansion of CFSE-labeled OT-II and Irgm1−/− OT-II CD4+ T cells activated with OVA peptide in the presence or absence of IFN-γ-neutralizing mAb. Results in e,f obtained after gating on CD4+ T cells; numbers in outlined areas (e) and in top left corners (f) indicate percent CD4+ cells. PI, propidium iodide. (g) [3H]thymidine incorporation by wild-type, Irgm1−/−, Ifng−/− and Irgm1−/−Ifng−/− CD4+ T cells stimulated for 72 h with soluble agonistic mAb to CD3 and irradiated syngenic Ifng−/− splenocytes. Data are representative of two independent experiments with similar results (mean and s.d. of triplicate cultures, b-d,g). \n",
      "a) [3H]thymidine incorporation by naive CD4+ lymphocytes stimulated for 72 h with agonistic mAb to CD3 and IFN-γ-deficient irradiated APCs, with IL-2 (10 U/ml) added 24 h after the initiation of culture.figf4(a) [3H]thymidine incorporation by naive CD4+ lymphocytes stimulated for 72 h with agonistic mAb to CD3 and IFN-γ-deficient irradiated APCs, with IL-2 (10 U/ml) added 24 h after the initiation of culture. (b) Accumulation of viable CD4+ T cells at 72 h in the presence of medium alone (Med) or IFN-γ (5 U/ml). (c) Flow cytometric analysis of the viability and division of CD4+ T cells labeled with CFSE and activated as described in a with T cell-depleted Ifng−/− APC samples in the presence or absence of IFN-γ (5 U/ml). Numbers in plots indicate percent CFSE+CD4+ T cell populations. (d-f) Death of Ifng−/− and Irgm1−/−Ifng−/− T cell populations activated with mAb to CD3 and expanded in IL-2, then treated with IFN-γ (d), agonistic mAb to CD3 (e) or Fas ligand (FasL; f), assessed by propidium iodide staining at 48 h after treatment. Open bars, Ifng−/−; filled bars, Irgm1−/−Ifng−/− (a,b,d-f). Data are representative of three independent experiments with similar results (mean and s.d. of triplicate cultures a,b,d-f). \n",
      "a–f) Electron microscopy analysis of Ifng−/− CD4+ lymphocytes (a) and Irgm1−/−Ifng−/− CD4+ lymphocytes (b–f) exposed for 30 h to IFN-γ (5 U/ml).figf5(a-f) Electron microscopy analysis of Ifng−/− CD4+ lymphocytes (a) and Irgm1−/−Ifng−/− CD4+ lymphocytes (b-f) exposed for 30 h to IFN-γ (5 U/ml). (c) Enlargement of the area outlined in b; arrowheads indicate two autophagic vacuoles. (d) Quantification of vacuoles in 50 randomly selected electron microscopy sections of more than 400 cells. Each symbol represents one section. (e) Irgm1−/− CD4+ T cell with many vacuoles containing dense materials. (f) Two Irgm1−/− CD4+ cells with vacuolated cytoplasma. Original magnification (a-c,e,f), × 1,000. Data are representative of two experiments. (g,h) Death of Ifng−/− and Irgm1−/−Ifng−/− CD4+ T cells in the presence or absence of wortmannin or Ly294002 (g), assessed by flow cytometry with propidium iodide staining at 48 h after IFN-γ exposure. Data are representative of three independent experiments (mean and s.d. of triplicate cultures). (g,h) Death of Ifng−/− and Irgm1−/−Ifng−/− CD4+ T cells transfected with control or beclin1-specific small interfering RNA, assessed by flow cytometry with propidium iodide staining at 48 h after IFN-γ exposure. Right (h), immunoblot analysis of beclin1 expression. C, control small interfering RNA; B, beclin1-specific small interfering RNA. Data are representative of three independent experiments (mean and s.d. of triplicate cultures). \n",
      ".Figure 1(A) Mettl14 cKO leads to perinatal lethality. Newborn cKO pups are smaller with tight and shiny skin. (B) Immunohistochemistry confirms loss of Mettl14 expression in skin in cKO animals. (C) H/E staining of newborn skin sections from WT and Mettl14 cKO mice. (D) Expression of p63 in skin epidermis was examined by immunofluorescence staining in WT and cKO mice. (E) Number of p63-positive cells in WT and cKO skin was quantified and shown as box and whisker plots. The plot indicates the mean (solid diamond within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=6 (biological repeats), P<0.05 (Student's t-test). (F) Skin stratification in WT and cKO skin was determined by immunofluorescence staining with different antibodies as indicated. Krt14: Keratin 14; Krt10: Keratin 10; β4: β4-integrin; DAPI for nucleus staining. The dashed line denotes the basement membrane that separates dermis and epidermis (Epi). (G) Wound healing as monitored by wound size 8 days post injury. n=3; P<0.01 (Student's t-test). Error bar represents S.D. (standard deviation). H) Histological staining of skin sections at the wound edges. Halves of wound sections are shown. Note significant reduction of HPE (hyperproliferative epidermis) in cKO skin. Es: eschar. Dotted lines denote epidermal boundaries. (I) Quantification of Ki67-positive cells present in wound HPE. The plot indicates the mean (open circles within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=19, P<0.01 (Student's t-test). \n",
      ".Figure 3(A) Profiling of RNA methylome by m6A-seq shows significant changes of lncRNA modifications upon epidermal differentiation in vitro. B) Mutations in the predicted m6A modification sites can significantly reduce Pvt1 methylation.n=3. Error bar represents S.D. C) Deletion of endogenous Pvt1 by an inducible Cas9 (iCas9) system can significantly reduce Pvt1 RNA level (quantification from RT-PCR, left panel) and MYC protein level (quantification from immunoblot, right panel) expression. n=3, P<0.01 (Student's t-test). Error bar represents S.D. D) CFE of WT, Pvt1 inducible KO, and Pvt1 inducible KO cells rescued with WT or mutant Pvt1 was quantified and presented as bar graph. n=3, **: P<0.01 (Student's t-test). Error bar represents S.D. E) Skin organoids derived from WT or Pvt1 inducible KO cells were grafted to nude mice. The regenerated skin was analyzed by H/E staining. F) Epidermal thickness of WT and Pvt1 KO skin grafts was quantified and presented as box and whisker plots. The plot indicates the mean (solid diamond within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=6 (biological repeats), P<0.05 (Student's t-test). n=18, P<0.01 (Student's t-test). G) Number of p63-positive cells in WT and cKO skin was quantified and shown as bar graph. n=5, P<0.01 (Student's t-test). Error bar represents S.D. H) Fluorescence microscopy demonstrates different survival capability of WT and Pvt1 inducible KO cells with or without Doxycycline (Dox) treatment. I) Ratio of WT and Pvt1 inducible KO cells in the co-culture model was quantified and shown as dot plots. n=8, P<0.01 (Student's t-test) for KO cells with Dox treatment compared with WT cells or KO cells without Dox stimulation at both Day 7 and 14. Error bar represents S.D. \n",
      ".Figure 2(A) EdU staining of WT or Mettl14 cKO skin after pulse-chase labeling. Skin samples were counterstained with antibody against β4-integrin. Note reduced EdU label-retaining cells in cKO skin epidermis. Arrows indicate Edu-positive cells. (B) Label-retaining cells in WT or Mettl14 cKO skin were quantified and shown as box plots. The plot indicates the mean (solid diamond within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=6 (biological repeats), P<0.05 (Student's t-test). n=19, P<0.01 (Student's t-test). (C) Number of holoclones derived from WT and Mettl14 cKO skin was quantified and shown as box and whisker plots. The plot indicates the mean (solid diamond within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=6 (biological repeats), P<0.05 (Student's t-test). n=24, P<0.01 (Student's t-test). (D) Morphology of primary keratinocytes isolated from WT or Mettl14 cKO skin. (E) Expression of Mettl14 and p63 in WT and KO cells was determined by immunoblots with respective antibodies. Immunoblot for GAPDH was used as loading control. (F) Proliferation of WT and Mettl14 null cells in vitro was quantified and shown as dot plots. n=3, P<0.01 (Student's t-test) for Day 6, 9, and 12, and P<0.05 (Student's t-test) for Day 3. Error bar represents S.D. (G-H) CFE (colony formation efficiency) of WT and Mettl14 null cells was determined in vitro. Results were quantified and shown as bar graph (F). n=19, P<0.01 (Student's t-test). Error bar represents S.D. (I) Fluorescence microscopy demonstrates different survival capability of WT and Mettl14 inducible KO cells with or without Tamoxifen (TAM) treatment. (J) Ratio of WT and Mettl14 inducible KO cells in the co-culture model was quantified and shown as dot plots. n=8, P<0.01 (Student's t-test) for KO cells with TAM treatment compared with WT cells or KO cells without TAM stimulation at both Day 7 and 14. Error bar represents S.D. \n",
      ".Figure 4A) Interaction between Pvt1 and MYC was determined by immunoprecipitation followed with RT-PCR. Note significant decrease in Pvt1 and MYC interaction upon FTO treatment (left panel), calcium shift-induced differentiation (middle panel, Hi: high calcium), and Pvt1 methylation site mutations (right panel). n=3, P<0.01 (Student's t-test). Error bar represents S.D. B) Immunoblots show decreased MYC protein level in differentiated keratinocytes and Mettl14 KO cells. Undifferentiated WT and Mettl14 KO cells (C) were treated with cycloheximide. Cell lysates were collected at 0, 30, 60, 90, 120, 180, and 300 minutes post treatment and subjected to immunoblotting with different antibodies as indicated. undifferentiated WT and Pvt1 KO cells (D) were treated with cycloheximide. Cell lysates were collected at 0, 30, 60, 90, 120, 180, and 300 minutes post treatment and subjected to immunoblotting with different antibodies as indicated. WT cells before and after calcium shift-induced differentiation (E) were treated with cycloheximide. Cell lysates were collected at 0, 30, 60, 90, 120, 180, and 300 minutes post treatment and subjected to immunoblotting with different antibodies as indicated. (F) Skin organoids derived from Mettl14 KO cells or KO cells rescued with exogenous expression of MYC were grafted to nude mice. The regenerated skin was analyzed by H/E staining. (G) Epidermal thickness of KO and KO skin rescued with MYC expression was quantified and presented as box and whisker plots. The plot indicates the mean (solid diamond within the box), 25th percentile (bottom line of the box), median (middle line of the box), 75th percentile (top line of the box), 5th and 95th percentile (whiskers), 1st and 99th percentile (solid triangles) and minimum and maximum measurements (solid squares). n=6 (biological repeats), P<0.05 (Student's t-test). n=18, P<0.01 (Student's t-test). (H) Number of p63-positive cells in KO and KO skin rescued with MYC expression was quantified and shown as bar graph. n=5, P<0.01 (Student's t-test). Error bar represents S.D. (I) Fluorescence microscopy demonstrates different survival capability of WT and Mettl14 KO cells with inducible expression of MYC. (J) Ratio of WT and Mettl14 KO cells with inducible expression of MYC in the co-culture model was quantified and shown as dot plots. n=8, P<0.01 (Student's t-test) for KO cells without Dox treatment (no MYC expression) compared with WT cells or KO cells with Dox stimulation (exogenous MYC expression) at both Day 7 and 14. Error bar represents S.D. \n",
      "Rab25 expression alters cells sensitivity to nutrient stress.Figure 1A. Rab25 expression decreases nutrient withdrawal (24 h) induced cell death in ovarian A2780 and IOSE29 cells measured using the Cell death ELISA plus assay (left); cells were cultured in complete media (containing 5% FBS), serum free media (SF), complete media plus 2DG, serum and glucose free media (SF-glu), or in amino acid, glucose and SF Earls buffer salt solution (EBSS). (a) p < 0.01 versus control and (b) p < 0.05 versus pcDNA. (Right) Percentage of apoptotic cells (sub-G0 population) by flow cytometry in A2780 cells following glucose and serum withdrawal (Glu/SF). (a) p < 0.001 versus A2780pcDNA control and (b) p < 0.05 A2780Rab25 versus A2780pcDNA control. B. Decreasing Rab25 expression increases sensitivity to nutrient stress induced cell death. Expression of Rab25 was down-regulated by siRNA or shRNA specific to Rab25. C. Rab25 regulates autophagy activity in ovarian cancer cells under serum and glucose deprivation conditions. Western blot for the 16kD LC3-II fragment, indicative of autophagy activity, in A2780 cells after 4 and 6 h of serum and glucose withdrawal. D. Electron microscopy of autophagy in ovarian A2780 cells after 4 and 24 h of serum and glucose withdrawal (upper panel, low magnification). High magnification images of boxed areas with arrowheads depicting autophagic vacuoles (lower panel, inset; N: nucleus). Average number of autophagosomes per cell was calculated by counting the number of autophagosomes in 16 individual cells from two-independent experiments. E-H. Expression of Rab25 decreases glucose and serum deprivation induced signalling activation. Protein expression was measured by RPPA or WB analysis.E. RPPA detection of time-dependent activation of AMPK after nutrient withdrawal. E-H. Expression of Rab25 decreases glucose and serum deprivation induced signalling activation. Protein expression was measured by RPPA or WB analysis.F. WB analysis of AMPK and acetyl-CoA carboxylase (ACC) phosphorylation in HEY ovarian cancer cells (upper panel). WB of phospho-ACC levels in A2780, IOSE80ht and SKOV3 ovarian cells after 1 h of nutrient withdrawal (lower panel). E-H. Expression of Rab25 decreases glucose and serum deprivation induced signalling activation. Protein expression was measured by RPPA or WB analysis.G. RPPA detection of phosphorylation of ACC after withdrawal of serum and glucose. E-H. Expression of Rab25 decreases glucose and serum deprivation induced signalling activation. Protein expression was measured by RPPA or WB analysis.H. Effect of Rab25 down-regulation on AMPK (left panel) and ACC (right panel) phosphorylation. Total cellular protein, isolated from A2780pcDNA and Rab25 expressing cells 72 h post-transfection with either non-target (NT) RNAi or Rab25 specific RNAi, was separated by polyacrylamide gel electrophoresis (PAGE) followed by WB analysis. (a) p < 0.001 versus NT RNAi control and (b) p < 0.001, pcDNA versus Rab25. \n",
      "Detection of physical association between AKT and Rab25 by protein complementation assay (PCA).Figure 5Ovarian cancer HEY cells (upper panel) were co-transfected with AKT-IFPN and Rab25-IFPC or with AKT-IFPC and Rab25-IFPN. Cells become fluorescent due to an interaction between AKT and Rab25 bringing the two halves of the fluorescent protein into proximity creating a stable complex and restoring fluorescence. Hela cells stably expressing AKT-IFPN (lower panel) were transfected with Rab25-IFPC or PDK-IFPC as control. A brightfield image corresponding to the fluorescent image is shown and transfected cells indicated by an arrow. Detection of Rab25 and AKT interaction by immunoprecipitation. Ovarian cells expressing HA-tagged Rab25 were IP with anti-HA antibody. The resultant complex was separated by gel electrophoresis and detected by WB using anti-AKT antibody. Generation of Rab25 deletion mutants. Expression of the deletion mutants was detected by immunofluorescence staining (upper panel) and WB (middle panel) using anti-GFP antibody which interacts with YFP. Diagrammatic representation of the structure of the mutants is shown in the lower panel. Rab25 deletion mutants do not alter cellular glycogen and ATP levels. Ovarian cancer cell lysates were collected 24 h post transfection. (a) p < 0.05 versus empty vector transfected cells. \n",
      "Prediction in clinical samples.Figure 6A. Rab25 expression correlates with glycogen levels in patienttumours. Total RNA and total cellular extracts, isolated from 31 ovarian cancer patient specimens, were subjected to Rab25 gene expression and glycogen content analysis using qPCR and glycogen assay, respectively. B-C. The Rab25 expression signature identifies patients with a poor prognosis. Ovarian cancer patients were classified as either mirroring the Rab25-associated gene expression signature (Rab25 signature) or not (non-Rab25 signature) using linear discriminant analysis in BRB tools, in two independent published ovarian datasets. Progression free survival curvesfor patients with advanced disease (Stage II to IV) are shown. Univariate analyses were plotted using Kaplan-Meier method and Coxplots used for multivariate analyses (co-variates included stage, grade, histology and residual disease).B. Tothill et al, 2008.C. Dressman et al, 2007. D. Prediction of breast cancer overall survival in two independent published breast datasets (upper panel) Pawitan et al, 2005 and (lower panel) Chin et al, 2006, based on Rab25-associated gene signature. Breast patient classification was determined by BRB tool class prediction function to identify patient with Rab25-associated gene expression signature (Rab25-signature) or without (non-Rab25 signature). All patients from the datasets were included in class prediction. E. Proposed model for the mechanism by which Rab25 regulates cellular bioenergetics. \n",
      "Rab25 increases glucose uptake.Figure 4A. Rab25 over-expression increases H3-labelled 2DG uptake. (a) p < 0.001 Rab25 versus pcDNA. Results are mean ± s.d. of a triplicate in one of representative experiment. B. Down-regulation of Rab25 or AKT expression, or inhibition AKT activity by MK2206 decreases glucose uptake in HEY cells (a) p < 0.001 versus NT siRNA control, (b) p < 0.05 versus siRab25 and (c) p < 0.05 versus siAKT. C. Effect of down regulation of Rab25 and AKT inhibitor MK2206 (AKTi) on glucose uptake. Cells were either pretreated for 2 h with AKTi before assessing glucose uptake. (a) p < 0.05 versus shRNA control and (b) p < 0.05 versus shRab25. D. Rab25 co-localization with GLUT1 in A2780 cells. Immunofluorescence microscopy staining of GLUT1 (red) and Rab25 (green). Co-localization of GLUT1 and Rab25 (i.e. yellow) is indicated by arrows. Single scale bar, 10 µM. E-G. Resistance to 2DG induced cell death in Rab25 cells is mediated through the PI3K/AKT pathway. Inhibition of the PI3K/AKT pathway was achieved by addition of 10 µM of PI103 or MK2206.E. AKT inhibition increases sensitivity to 2DG induced cell death. *p < 0.001 versus HEYpcDNA cells. E-G. Resistance to 2DG induced cell death in Rab25 cells is mediated through the PI3K/AKT pathway. Inhibition of the PI3K/AKT pathway was achieved by addition of 10 µM of PI103 or MK2206.F. Down-regulation of AKT by siRNA specific to AKT. E-G. Resistance to 2DG induced cell death in Rab25 cells is mediated through the PI3K/AKT pathway. Inhibition of the PI3K/AKT pathway was achieved by addition of 10 µM of PI103 or MK2206.G. Or down-regulation of Rab25 by shRNA specific to Rab25 enhances 2DG induced cell death. *p < 0.001 versus RNAi control cells. \n",
      "Expression of Rab25 regulates cellular ATP and glycogen levels.Figure 2Expression of Rab25 increases endogenous ATP level in ovarian cancer cells. *p < 0.001 Rab25 versus pcDNA. The mean cellular concentration of ATP per mg of protein (based on the standard curve generated using a known amount of ATP) is 2.36−10 moles/mg for IOSE29htpcDNA cells, 6.21−10 moles/mg for IOSE29htRab25 cells, 8.14−11 moles/mg for IOSE80htpcDNA cells, 1.05−10 moles/mg for IOSE80htRab25 cells, 2.85−10 moles/mg for A2780pcDNA cells, 3.81−10 moles/mg for A2780Rab25 cells, 7.23−10 moles/mg for HEYpcDNA cells and 9.82−10 moles/mg for HEYRab25 cells. Rab25 expression increases ATP synthesis and delays fall in total cellular ATP levels after glucose and serum withdrawal, p < 0.001 Rab25 versus control pcDNA. Down-regulation of Rab25 expression decreases ability to maintain ATP after glucose and serum withdrawal (a) p < 0.05 SF-Glu versus FBS and (b) p < 0.05 versus control. Expression of Rab25 increases total cellular glycogen content while down-regulation of Rab25 expression decreases total cellular glycogen content (µg of glycogen/mg protein). Cells were cultured in complete media for 24 h before glycogen measurement. (a) p < 0.01 Rab25 versus pcDNA, (b) p < 0.05 Rab25 RNAi versus NT RNAi and (c) p < 0.05 shRAb25 versus shRNA control. Decrease in cellular glycogen content after glucose and serum withdrawal *p < 0.001 Rab25 versus pcDNA. Effect of GPi (Bay U6571) and oligomycin on ATP production. A2780pcDNA transfected and Rab25 expressing cells were pretreated with vehicle (control), 30 µM of Bay U6571 (GPi) or 25 µg/ml of oligomycin for 2 h. Cells were then cultured in serum- and glucose free RPMI 1640 in the presence or absence of Bay U6571 or oligomycin for the indicated times. (a) p < 0.01 oligomycin time 0 versus control time 0 and (b) p < 0.01 versus time 0 at each treatment group. Effect of Bay U6571 on cellular glycogen (left panel) and ATP (right panel) levels in HEY cells. (a) p < 0.01 versus pcDNA control time 0 at 5% FBS and (b) p < 0.01 GPi versus control at each corresponding treatment. Down regulation of Rab25 expression and inhibition of glycogen breakdown by GPi administration increases 2DG induced cell death *p < 0.05 versus shRNA control. \n",
      "Rab25 regulates bioenergetics in ovarian cancer cell through the AKT pathway.Figure 3A. Rab25 expression increases phospho but not total AKT and GSK3 levels in A2780 cells leading to decreased phosphorylation of GS (pGS). B. WB analysis of AKT and GSK3 phosphorylation confirming the inhibitory effect of PI3K pathways inhibitors PI103, GPi and synthase kinase 3 inhibitor (GSK3i) in HEY cells. C. Inhibition of GSK3 activity increases total cellular glycogen levels. pcDNA transfected and Rab25 expressing ovarian cancer cells were cultured in complete media in the presence of 10 µM GSK3i for the indicated times. Total cellular glycogen content was measured and normalized with total protein content. (a) p < 0.001 versus no GSK3i treated pcDNA cells. D-E. Effect of PI3K inhibitor PI103 on cellular glycogen content and ATP production. Ovarian cancer cells were pretreated with PI103 for at least 2 h in complete media. Cells were then subjected to 2 hglucose and FBS withdrawal to deplete endogenous glycogen followed by culturing cells in RPMI-glucose media in the presence of PI103 for 2 h (2 hSF). After 2 h of nutrient stress, complete media containing PI103 was added to the cells for 30 min (5% FBS) for recovery. A second glucose and FBS withdrawal (15 min SF) followed immediately to examine the effect on glycogen and ATP levels. a, p < 0.001 5% FBS versus 2 hSF, b, p < 0.01 15 min SF versus 5% FBS.D. There is an increase in glycogen content during recovery phase (5% FBS) and a decrease in glycogen when cells are subject to a second nutrient stress (15 min SF). D-E. Effect of PI3K inhibitor PI103 on cellular glycogen content and ATP production. Ovarian cancer cells were pretreated with PI103 for at least 2 h in complete media. Cells were then subjected to 2 hglucose and FBS withdrawal to deplete endogenous glycogen followed by culturing cells in RPMI-glucose media in the presence of PI103 for 2 h (2 hSF). After 2 h of nutrient stress, complete media containing PI103 was added to the cells for 30 min (5% FBS) for recovery. A second glucose and FBS withdrawal (15 min SF) followed immediately to examine the effect on glycogen and ATP levels. a, p < 0.001 5% FBS versus 2 hSF, b, p < 0.01 15 min SF versus 5% FBS.E. Utilization of glycogen to produce ATP. F. Addition of AKT inhibitor MK2206 (AKTi) abolishes Rab25-dependent glycogen storage. (a) p < 0.05 versus shRNA control and (b) p < 0.01 SF + AKTi versus SF. G. Inhibiting AKT pathway and glucose metabolism decreases cell viability. Cells were pretreated with 10 µM of PI103 or AKTi, 100 µM LND or 3BrPy for 2 h before switching to SF media in the presence of inhibitors for an addition16 h before cell titre blue viability assay. *p < 0.001 shRNA Rab25 versus shRNA control. \n",
      ".Figure 1Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (A) Box-plot of the enrichment of the RNAs identified by Mmi1 RNA-IPs coupled to high throughput sequencing. The log of the average enrichments obtained from two independent RNA-IPs is plotted. The enrichment is relative to the no antibody RNA-IPs conducted in parallel of Mmi1 RNA-IPs. The mRNAs and lncRNAs enriched at least two folds in both Mmi1 RNA-IPs are shown in blue and green, respectively. Crosses represent newly identified Mmi1 targets and dots known targets. Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (B) LncRNAs predicted to be targets of Mmi1 with a high confidence by our computational approach. Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (C) Surface representation of Mmi1 YTH domain. Area corresponding to conserved residues (according to the alignment shown in Fig EV 2A) and located at the surface are highlighted in green. Mutated residues in Mmi1 YTH domain are indicated in blue for the ones located within the aromatic cage, in purple for the ones surrounding the cage, and in red for the rest. Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (D) RNA-IPs showing the impact of Mmi1 YTH domain point mutations on Mmi1's binding to ssm4 mRNA. The lower part shows a Western blot monitoring the protein level of WT and mutant Mmi1 proteins in the cells used for the RNA-IPs. Loading was monitored using an anti-Tub1 (tubulin) antibody. Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (E, F) RNA-IPs showing the YTH-dependent association of Mmi1 to the lncRNAs identified in (A) and (B). Average fold enrichment is shown with error bars that indicate mean average deviations for three independent experiments for (D, E and F). Figure 1. Extensive identification of Mmi1 RNA targets by combining RNA-IP sequencing and computational approaches. (E, F) RNA-IPs showing the YTH-dependent association of Mmi1 to the lncRNAs identified in (A) and (B). Average fold enrichment is shown with error bars that indicate mean average deviations for three independent experiments for (D, E and F). \n",
      ".Figure 4Figure 4. Mmi1 drives Rrp6-dependent heterochromatin gene silencing at pericentromeric regions. (A) Upper part, TNAAAC-rich sequence present in nam5, 6 and 7 lncRNAs. Lower part, schematic representation of S. pombe three centromeres showing the different pericentromeric DNA repeats susceptible to produce nam5, 6 and 7 (green arrows). Figure 4. Mmi1 drives Rrp6-dependent heterochromatin gene silencing at pericentromeric regions. (B) Northern blot showing the level of nam5/6/7 lncRNA population in mmi1-ts3, dcr1∆, clr4∆ single mutant cells and in mmi1-ts3 dcr1∆ and mmi1-ts3 clr4∆ double mutant cells, at the permissive (25ºC) and restrictive (36ºC) temperatures. Figure 4. Mmi1 drives Rrp6-dependent heterochromatin gene silencing at pericentromeric regions. (C) RT-PCRs monitoring the accumulation of nam7 lncRNAs and nam7-L read-through transcripts, in the same cells and conditions as in (B). Red arrow heads point to the expected PCR products while the other bands correspond to non-specific PCR products. From the scheme and the agarose gels: black arrows, primers used for the three different reverse transcriptions; black lines and red numbers, expected PCR products of the three different RT-PCRs; M, DNA ladder Markers; tub1, tubulin control. Figure 4. Mmi1 drives Rrp6-dependent heterochromatin gene silencing at pericentromeric regions. (D) RT-qPCRs showing the levels of nam5/6/7 lncRNA population in the double mutant rrp6∆ dcr1∆ cells, relative to the single mutant rrp6∆ and dcr1∆ cells. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (D, E). P value was calculated using a two-tailed Student's t-test. Figure 4. Mmi1 drives Rrp6-dependent heterochromatin gene silencing at pericentromeric regions. (E) RNA-IPs showing that Rrp6-Myc13 binds to nam5/6/7 lncRNAs in a Mmi1-dependent manner. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (D, E). P value was calculated using a two-tailed Student's t-test. \n",
      ".Figure 2Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (A) Upper part, scheme of S. pombe sexual differentiation. Lower part, microscopy images showing WT and mmi1∆ cells, and mmi1∆ cells expressing Mmi1-R351E or Mmi1-R381E mutant proteins. Images were taken after 24 hours of induction of sexual differentiation by growth on SPAS medium. The percentage of cells that underwent differentiation and iodine vapor assays, conducted on the corresponding patches, are shown at the bottom of each image. Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (B) Mating assay showing the percentage of zygotes forming over time in the same cells as in (A). Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation.(C) Scheme of nam1-byr2 locus. Mmi1 UNAAAC binding motifs are depicted by white lines. (D) Scheme of nam1-byr2 locus in nam1-1 cells highlighting the 8 UNAAAC motifs mutated (red lines). Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (E) Mmi1 RNA-IPs showing the specific loss of binding of Mmi1 to nam1-1 lncRNA but not to mei4 mRNA, another target of Mmi1. Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (F) Mating assay showing the percentage of zygotes formed over time in WT and nam1-1 cells. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) (B, E and F). Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (G) Western blots showing the level of Flag-Byr2 protein over the first 4 hours of sexual differentiation in WT and nam1-1 cells. Tubulin (Tub1) level was used as a loading control. Figure 2. Mmi1 binding to nam1 lncRNA controls MAPK-mediated entry into sexual differentiation. (H) Microscopy images of WT (h90) cells transformed with an empty plasmid (Control) and nam1-1 cells transformed with either an empty plasmid (Control) or a plasmid expressing Byr2 protein (Byr2), after 24 hours of induction of sexual differentiation. Scale bar, 10 µm. \n",
      ".Figure 3Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (A) Northern blots showing nam1 and byr2 RNA levels during the first 4 hours of sexual differentiation. Ribosomal RNAs (rRNAs) stained with ethidium bromide were used as loading controls. Black lines indicate probes used to detect nam1 and byr2 RNAs. Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (B) ChIPs showing the occupancy of the elongating RNAPII (RNAPII-S2P) over nam1-byr2 locus, in WT and nam1-1 cells. RNAPII-S2P was immunoprecipitated with an antibody recognizing the heptameric repeats (present in the C-terminal domain of the polymerase) when it is phosphorylated on its serine 2. Black lines, genomic regions investigated. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (B, C, E and F). Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (C) RT-qPCRs showing the accumulation of nam1 read-through transcripts (RT1-qPCR) and nam1 lncRNAs (RT2-qPCR) in cells with or without the transcription terminator Ttef inserted at the 3' end of nam1 (scheme). Black arrow, primer used for the strand-specific reverse transcription (RT); Black line, location of the region amplified by PCR. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (B, C, E and F). Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (D) Microscopy images of, respectively, WT (h90), nam1-1 and nam1-1-Ttef cells, after induction of sexual differentiation for 24 hours. The percentage of sporulation and iodine vapor assays are shown at the bottom of the images. Scale bar, 10 µm. Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (E) RT-qPCRs showing the accumulation of nam1 read-through transcripts in mmi1∆ and rrp6∆ cells. Black arrow and line as in (C). . Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (B, C, E and F). Figure 3. Mmi1 promotes transcription termination of nam1 non-coding gene and prevents nam1 read-through transcription from repressing the downstream MAPKKK gene byr2. (F) ChIPs monitoring the enrichment of H3K9me2 over nam1-byr2 locus and mei4 gene in WT and rrp6∆ cells. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (B, C, E and F). \n",
      ".Figure 6Figure 6. Model for Mmi1/exosome YTH-mediated control of cell differentiation and heterochromatin gene silencing mediated by its targeting of nascent lncRNA and the induction of their transcription termination. The YTH domain of Mmi1 co-transcriptionally binds to specific lncRNAs expressed from either euchromatin or heterochromatin regions. We propose that Mmi1 binding to a euchromatic nascent lncRNA (in green) induces the recruitment of the exosome and together they degrade the lncRNA and promote robust termination of the lncRNA transcription, which otherwise will inhibit the expression of the downstream protein-coding gene (in blue). In the case of nam1-byr2 locus, the efficient transcription termination prevents the occurrence of read-through transcription from nam1 gene, which represses byr2 MAPKKK gene, a critical regulator of the entry into sexual differentiation. In parallel, Mmi1 co-transcriptional binding to the heterochromatic pericentromeric lncRNAs also recruits the exosome and contributes to heterochromatin gene silencing by degrading the nascent lncRNAs as well as by inducing precocious termination of their transcription. STOP sign, site of Mmi1-dependent lncRNA transcription termination. \n",
      ".Figure 5Figure 5. Mmi1- and RNAi-mediated silencing of pericentromeric DNA transcription alternate during the progression of the cell cycle. (A) ChIPs assessing the localization of Mmi1 to pericentromeric DNA in WT and clr4∆ cells. Figure 5. Mmi1- and RNAi-mediated silencing of pericentromeric DNA transcription alternate during the progression of the cell cycle. (B) ChIPs showing the localization of Mmi1 (dashed line) and H3K9me2 (black line) to the pericentromeric nam5/6/7 DNA regions during the progression of the cell cycle. Cell synchronization was achieved by using cdc25-ts cells (see experimental procedures for more details). Cell synchronization was monitored by measuring the percentage of cells with a septum (right part). Figure 5. Mmi1- and RNAi-mediated silencing of pericentromeric DNA transcription alternate during the progression of the cell cycle. (C) RT-qPCRs monitoring the levels of nam5/6/7 (left part) and dg (middle part) lncRNA populations during the cell cycle. Average fold enrichment is shown with error bars that indicate mean average deviations (n=3) for (A, B and C). Figure 5. Mmi1- and RNAi-mediated silencing of pericentromeric DNA transcription alternate during the progression of the cell cycle. (D) RT-PCRs monitoring the accumulation of nam7 lncRNAs and nam7-L read-through transcripts in G2/M and G1/S phases from synchronized cells as in (B). Red arrow heads point to the expected PCR products. The other bands are non-specific PCR products. From the scheme: black arrows, primers used for the reverse transcriptions; black lines and red numbers, regions amplified by PCR. \n",
      " - Chemical genomics screen across four S.Figure 1Core genome phylogeny of part of the Saccharomyces Genome Resequencing Project (SGRP) yeast isolates; coloured dots indicate the four strains whose KOs library was screened in this study. Schematic of the chemical genomics screen; each strain's KO library was robotically plated on 1536 solid agar plates and each KO colony size was used as a proxy for fitness in each condition. Reproducibility of the S-scores using the two batches used in the screening. Reproducibility of the S-scores using genes having multiple independent colonies plated in the screening. Clustered heatmap of the whole chemical genomics screen; each subsection belongs to an individual strain's KO library. Grey cells indicate missing values. \n",
      " - Systematic assessment of genetic background dependencies of gene deletion phenotypes.Figure 2Average S-score Pearson's correlation between the same genes (orthologs, solid line) and random gene pairs (shaded distribution) across all the 38 conditions and four strains. Genes are stratified by the number of conditions in which they show a significant phenotype across the four strains. S-score scatterplots for each pairwise strain comparison, highlighting conserved phenotypes (black points), significant changes (red points) and gene-condition relationships for which no call can reliably be made (grey points). \"r\", pearson's r value. Fraction of deletion phenotypes in each strain conserved with other stains in pairwise, three-way and four-way comparisons. Error bars represent standard deviation for all pairwise and three-way comparisons. Only a four-way comparison is possible for each strain so no error bars are represented in these cases. Gene exclusiveness: a measure of each gene's propensity to change its chemical genomics profile across strains. The top 10 genes' names are reported. Genes with high exclusiveness (> 0) tend to have a higher number of negative genetic and physical interactions (as reported in the biogrid database). \n",
      " - The fitness of the WT strain background is linked to the number of differences in KO phenotypes for a given condition.Figure 3Barplots reporting the average number of gain and sick phenotypes that are specific to each strain across all pairwise comparisons. Wild type fitness of each strain relative to S288C across the same conditions as in panel A; each dot represents a specific replicate where colony sizes were measured. Relationship between the wild type fitness relative to S288C and the number of conditionally essential genes relative to S288C; each dot represents a strain-condition replicate as in panel B. The conditions maltose, glycerol, and NaCl are highlighted. Changes in gene deletion phenotypes for growth on osmotic stress conditions. The top heatmap contains genes belonging to the HOG pathway, while the bottom one those genes whose growth phenotypes varies the most between Y55 and YPS. Significant growth phenotypes are marked with \"-\". Changes in deletion phenotypes for growth on glycerol and maltose. The top heatmap contains the MAL genes, while the bottom one those genes whose growth phenotypes vary the most between S288C and the other three strains. Significant growth phenotypes are marked with \"-\". \n",
      " - Genes linked to growth phenotypes via GWAS analysis in 925 natural yeast isolates.Figure 4S-score heatmap of the yeast natural isolates across 34 conditions that were also used in the KO screening. B) Number of variants significantly associated (p-value < 1E-6) with phenotypic variation in each growth condition. C) Number of associated variants that overlap (i.e. are in a 3kbp window) with a conditionally essential gene in the same condition, in any of the four KO libraries. Manhattan plots showing examples of overlaps between associated variants and the KO screening. The top plot shows the associations between variants and growth in the natural isolates as a function of the -log10 of the association p-value, while the four bottom plots show the strength of the KO phenotypes across the four yeast strains, as a function of the -log10 of the corrected S-score p-value. Sections shaded in grey indicate the overlap between associations and KO data. Position in the yeast chromosome is reported in kilobase units. \n",
      " The non-structural protein NS1 of ZIKV enhances ZIKV-induced NLRP3 inflammasome activation.Figure 1(A) ELISA of supernatant IL-1β for unprimed THP-1 cells (UT) or LPS-primed (500 ng/ml, 3 hrs) THP-1 cells then infected with ZIKV (MOI=1) for 36 hrs. Uninfected cells serves as mock control. (B) Immunoblot analysis of supernatant (Sup) and cell extracts (Lys) of THP-1 cells, left untreated or primed by LPS (500 ng/ml, 3 hrs), after ZIKV infection (MOI=1) for 36 hrs. (C) ELISA of IL-1β in the supernatants of wild type (WT), NLRP3 knockout (KO) #1 or AIM2 KO#1 THP-1 cells left untreated (UT), or pretreated with LPS (500 ng/ml, 3 hrs) followed by ATP (5 mM, 6 hrs) stimulation, or stimulated with poly (dA:dT) (1mg/ml, 6 hrs), or infected with ZIKV (MOI=1, 36 hrs). (D) ELISA of IL-1β in the supernatants of two clones of NLRP3 KO THP-1 cells with or without ZIKV infection (MOI=1) at the indicated time points. (E) ELISA of IL-1β in the supernatants of PBMCs from two donors transfected with control (ctrl) siRNA or NLRP3 siRNA followed by ZIKV infection (MOI=1) for 36 hrs. (F) ELISA of IL-1β in the supernatant of WT, NLRP3 KO or AIM2 KO THP-1 cells with ZIKV (MR766 or GZ01) (MOI=1) or DENV-2 (MOI=1) infection for 36 hrs. (G) ELISA of IL-1β in the supernatants of THP-1 cells pre-infected with ZIKV for 24 hrs or mock control. The cells were then treated with LPS (500 ng/ml, 3 hrs) followed by ATP (5 mM) treatment for 6 hrs. (H) Immunoblot analysis of supernatant (Sup) and cell extracts (Lys) of THP-1 cells pre-infected with ZIKV (MOI=1) or mock control for 24 hrs. The cells were then treated with LPS (500 ng/ml, 3 hrs) followed by ATP (5 mM) stimulation for 6 hrs. (I) Immunoblot analysis of supernatants and cell extracts of 293T cells transfected with plasmids expressing NLRP3, ASC, pro-caspase-1 and pro-IL-1β together with NS1. (J) Immunoblot analysis of protein extracts of Flag-tagged NS1-inducible THP-1 cells treated with increasing doses of doxycycline (Dox) for 24 hrs. (K) ELISA of IL-1β in the supernatants of Flag-NS1-inducible THP-1 cells left unprimed then treated with ATP (5 mM, 6 hrs) or poly (I:C) (2 mg/ml, 6 hrs), or pre-treated with LPS (500 ng/ml, 3 hrs) followed by ATP (5 mM, 6 hrs) or poly (I:C) (2 mg/ml, 6 hrs) stimulation. (L) Immunoblot analysis of supernatants (Sup) and cell extracts (Lys) of Flag-NS1-inducible THP-1 cells pre-treated with LPS (500 ng/ml, 3 hrs) followed by ATP (5 mM, 6 hrs) or poly (I:C) (2 mg/ml, 6 hrs) stimulation. \n",
      " enhances inflammasome activation to benefit ZIKV infection.Figure 2(A) Relative qRT-PCR analysis of ZIKV RNA in THP-1 cells left untreated or treated with LPS (500 ng/ml, 3 hrs) and ATP (5 mM, 6 hrs), followed by ZIKV infection (MOI=1) for the indicated time points. (B) Relative qRT-PCR analysis of ZIKV RNA in wild type (WT) or NLRP3 knockout (KO) THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. (C) Immunoblot analysis of extracts of WT or Nlrp3-/- BMDMs by the indicated antibodies. (D) Relative qRT-PCR analysis of ZIKV RNA in WT or Nlrp3-/- BMDMs infected with ZIKV (MOI=1) for the indicated time points. (E) Plaque titration of ZIKV in supernatants of WT or Nlrp3-/- BMDMs infected with ZIKV (MOI=1) for the indicated time points. (F) Immunoblot analysis of extracts of WT or caspase-1 KO THP-1 cells by the indicated antibodies. (G) Relative qRT-PCR analysis of ZIKV RNA in WT or caspase-1 KO THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. (H) Relative qRT-PCR analysis of ZIKV RNA in NS-1-inducible THP-1 cells treated with or without doxycycline (Dox), then left untreated or treated with Ac-YVAD-cmk (20 μΜ) for 3 hrs followed by ZIKV infection (MOI=1) for the indicated time points. (I) Relative qRT-PCR analysis of NLRP3 knockdown efficiency in PBMCs from two donors. (J) Relative qRT-PCR analysis of ZIKV RNA in PBMCs transfected with control siRNA or NLRP3 siRNA then infected with ZIKV (MOI=1) for 36 hrs. (K) qRT-PCR analysis of ZIKV RNA in PBMCs treated with DMSO or Ac-YVAD-cmk (20μM) for 3hrs followed by ZIKV infection (MOI=1) for 36 hrs. \n",
      " inhibits the proteasomal degradation of caspase-1.Figure 3(A) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with HA-NS1 together with Flag-NLRP3, Flag-ASC, Flag-caspase-1 and Flag-IL-1β. WCL, whole cell lysates. (B) Co-immunoprecipitation and immunoblot analysis of extracts of NS1-inducible THP-1 cells, which expressed Flag-tagged NS1, treated with or without doxycycline (Dox) (100 ng/ml). (C) Immunoblot analysis of extracts of THP-1 cells treated with LPS (500 ng/ml, 3 hrs) and ATP (5 mM, 6 hrs) followed by increasing doses of Dox treatment. (D) Immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 together with increasing amount of HA-NS1. Below, caspase-1 mRNA RT-PCR; RPL13A mRNA RT-PCR serves as a loading control. (E) Immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 together with increasing amount of HA-DENV-2-NS1. (F) Immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 together with empty vector or HA-NS1 followed by cycloheximide (CHX) treatment (100 μg/ml) for the indicated time points. (G) Immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 together with empty vector or HA-NS1 then treated with MG132 (10 mM), Lactacystin (5 μM), Carfilzomib (100 mM) or CQ (50 mM) for 6 hrs. (H) Co-immunoprecipitation and immunoblot analysis of extracts of Flag-NS1-inducible THP-1 cells treated with or without Dox (100 ng/ml). (I) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with plasmids expressing Flag-caspase-1 and indicated HA-tagged WT ubiquitin or mutants (K6-, K11-, K27-, K29-, K33-, K48- and K63-Only), together with empty vector or Myc-NS1, followed by MG132 (10 mM) treatment for 6 hrs. \n",
      "-linked poly-ubiquitin chains from caspase-1.Figure 4(A) Immunoblot analysis of extracts of wild type (WT) and USP8 knockout (KO) 293T cells transfected with Flag-caspase-1 together with empty vector or Myc-NS1. (B) Immunoblot analysis of extracts of transfected with Flag-caspase-1 together with empty vector or Myc-USP8 then treated with or without MG132 (10 mM) for 6 hrs. (C) Co-immunoprecipitation and immunoblot analysis of extracts of Flag-NS1-inducible THP-1 cells treated with or without Dox (100 ng/ml) then transfected with or without poly (I:C) (2 mg/ml, 6 hrs). (D) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 and Myc-USP8 together with HA-NS1 followed by MG132 treatment (10 mM) for 6 hrs. (E) Co-immunoprecipitation and immunoblot analysis of extracts of THP-1 cells infected with ZIKV (MOI=1) for 24 hrs. (F) Co-immunoprecipitation and immunoblot analysis of extracts of WT and USP8 KO THP-1 cells infected with ZIKV (MOI=1) for 24 hrs. (G) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with plasmids expressing Flag-caspase-1 and indicated HA-tagged WT ubiquitin (Ub) or mutants (K6-, K11-, K27-, K29-, K33-, K48- and K63-Only), together with empty vector or Myc-USP8, followed by MG132 (10 mM) treatment for 6 hrs. (H) Immunoblot analysis of extracts of WT or USP8 KO 293T cells transfected with Flag-caspase-1, HA-K11-Ub together with empty vector or Myc-NS1, followed by two-step immunoprecipitation with anti-Flag beads and immunoblot analysis with anti-HA. (I) Co-immunoprecipitation and immunoblot analysis of extracts of USP8 KO 293T cells transfected with Flag-caspase-1 and HA-K11-Ub together with WT or inactive mutant C786A of USP8. \n",
      " stabilizes caspase-1 through the cleavage of poly-ubiquitin chains at Lys134.Figure 5(A) Immunoblot analysis of extracts of 293T cells transfected with empty vector or HA-NS1 together with Flag-caspae-1 wild type (WT) and mutants (K37R, K44R or K134R). Lower panel shows the intensity analysis of the bands from the three independent experiments. (B) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with HA-K11-ubiquitin (Ub) together with Flag-caspase-1 (WT, K37R, K44R, K134R). (C) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 (WT or K134R) and HA-K11-Ub, together with empty vector or Myc-NS1. Lower panel shows the intensity analysis of the bands from the three independent experiments. (D) Co-immunoprecipitation and immunoblot analysis of extracts of 293T cells transfected with Flag-caspase-1 (WT or K134R) and HA-K11-Ub, together with empty vector or Myc-USP8. Lower panel shows the intensity analysis of the bands from the three independent experiments. \n",
      ".Figure 6(A) Relative qRT-PCR analysis of IFNB, IFIT2 and IFIT1 mRNA levels in wild type (WT) or caspase-1 knockout (KO) THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. (B) Relative qRT-PCR analysis of IFNB mRNA levels in THP-1 cells left untreated or treated with LPS (500 ng/ml, 3 hrs) and ATP (5 mM, 6 hrs), followed by ZIKV infection (MOI=1) for the indicated time points. (C) Relative qRT-PCR analysis of IFNB mRNA levels in wild type (WT) or NLRP3 knockout (KO) THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. (D) Relative qRT-PCR analysis of IFNB , IFIT2 and IFIT1 mRNA levels in Flag-NS1-inducible THP-1 cells treated with or without Dox then left untreated or treated with YVAD (20 μM) for 3 hrs, followed by ZIKV infection (MOI=1) for the indicated time points. (E) Relative qRT-PCR analysis of IFNB mRNA in PBMCs transfected with control siRNA or NLRP3 siRNA then infected with ZIKV (MOI=1) for 36 hrs. (F) Relative qRT-PCR analysis of IFNB mRNA in PBMCs treated with DMSO or Ac-YVAD-cmk (20μM) for 3hrs followed by ZIKV infection (MOI=1) for 36 hrs. (G) ELISA of IFN-β in the supernatants of PBMCs treated with DMSO or Ac-YVAD-cmk (20μM) for 3hrs followed by ZIKV infection (MOI=1) for 36 hrs. (H) ELISA of IL-1β production in brain of WT or Nlrp3-/- 1-day-old neonatal mice (n=3 per group) intracerebrally injected with 1×105 plaque-forming units (PFU) of ZIKV for 5 days. (I) Relative qRT-PCR analysis of IFNB mRNA levels in brain tissues of WT or Nlrp3-/- neonatal mice with or without ZIKV infection (J) ELISA of IFN-β production in brains of WT or Nlrp3-/- neonatal mice (K) Relative qRT-PCR analysis of ZIKV RNA levels in brain tissues of WT or Nlrp3-/- neonatal mice with or without ZIKV infection (L) Plaque titration of ZIKV in brain tissues of WT or Nlrp3-/- neonatal mice (M) Body weight gain of WT and Nlrp3-/- neonatal mice (n=4 per group) inoculated with ZIKV (1×105 PFUs). \n",
      " Cleavage of cGAS by caspase-1 inhibits type I IFN signaling and assists ZIKV replication.Figure 7(A) Relative qRT-PCR analysis of ZIKV RNA levels in wild type (WT) or cGAS knockout (KO) THP-1 cells after ZIKV infection (MOI=1) for indicated time points. (B) Relative qRT-PCR analysis of IFNΒ, IFIT2 and IFIT1 mRNA levels in WT or cGAS KO THP-1 cells after ZIKV infection (MOI=1) for indicated time points. (C) Relative qRT-PCR analysis of ZIKV RNA and IFNB mRNA levels in PBMCs transduced with Cas9 and small guide RNA targeting GFP, cGAS or NLRP3 and infected with ZIKV (MOI=1) for 36 hrs. (D) Relative qRT-PCR analysis of ZIKV RNA and IFNΒ mRNA levels in WT or cGAS KO THP-1 cells treated with or without YVAD (20 μM), followed by ZIKV infection (MOI=1) for indicated times points. (E) Immunoblot analysis of extracts of Flag-cGAS-overexpressed THP-1 cells treated with or without YVAD (20 μM) then infected with ZIKV (MOI=5) for 24 hrs or infected with HSV-1 (MOI=1) for 24 hrs. Immunoblot analysis of WT, NLRP3 KO (F) THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. Immunoblot analysis of WT, caspase-1 KO (G) THP-1 cells infected with ZIKV (MOI=1) for the indicated time points. (H) Immunoblot analysis of extracts of WT or Nlrp3-/- BMDMs infected with ZIKV (MOI=1) for the indicated time points. (I) Immunoblot analysis of PBMCs from Donor#1 transfected with control siRNA or NLRP3 siRNA followed by ZIKV infection (MOI=1) for the indicated time points. (J) Immunoblot analysis of extracts of 293T cells transfected with Myc-cGAS (tagged on C-terminal), GFP-ASC, Flag-caspase-1, together with empty vector or HA-NS1. (K) Relative qRT-PCR analysis of ZIKV RNA and IFNB mRNA levels in WT, cGAS KO THP-1 cells as well as cGAS KO THP-1 cells reconstituted with WT or D140A/D157A mutant of cGAS, followed by ZIKV infection (MOI=1) for 24 hrs. \n",
      " Schematic representation of ZIKV by targeting the cross-talk between inflammasome and type I IFN signaling.Figure 8ZIKV nonstructural proteins NS1 enhances inflammasome activation to benefit its infection by recruiting USP8 to cleave K11-linked poly-ubiquitin chains from caspase-1 at Lys134. NS1-mediated stabilization of caspase-1 further promotes the cleavage of cGAS, thus inhibiting type I IFN signaling as well as antiviral innate immunity. \n",
      ".Figure 1A - Distribution of Casp3+ cells in rostral and ventral domains of an E11.5 Rx-Dicer mutant embryo. NCx, neocortex; OB, olfactory bulb; Spt, septum. B,C - Marker analysis of Casp3+ cells in the rostral telencephalon of E12.5 Rx-Dicer mutant embryos. Most apoptotic cells are Pax6+ RGCs (solid arrowheads) and not Tbr1+ neurons (open arrowheads). N = 3 replicates per marker. D,E - Distribution and abundance of apoptotic cells (Casp3+) in the rostral telencephalic primordium of control and Rx-Dicer mutant embryos at the indicated ages. Dotted line indicates basal surface, dashed line apical surface. N = 3 replicates per genotype and age. F,G - Distribution and abundance of mitoses (PH3+) and neurons (Tuj1) in the rostral telencephalic primordium of control and Rx-Dicer mutant embryos at the indicated ages. Dotted line indicates basal surface, dashed line apical surface. N = 3-4 replicates per genotype and age. No significant differences were found between control and mutant embryos in apical nor basal mitoses. \n",
      ".Figure 3A - Coronal section through the rostral telencephalon of an E14.5 Rx-Dicer mutant embryo displaying the three types of rosettes. B - High magnification of rosettes in Rx-Dicer mutants at E12.5 (Type 1), E14.5 (Type 2) and E17.5 (Type 3), immunostained for PH3 and Tuj1. Dashed line indicates ventricular surface; arrowheads indicate basal mitoses; arrows indicate a stream of apical mitoses connecting the lumen of the rosette with the lumen of the telencephalic ventricle. C,D - Abundance of rosette types at the indicated ages (C), and rostro-caudal distribution of total rosette abundance per age, independent of type (D). n ≥ 2 brains per age. E - Schematic of the progression of rosette formation in the rostral telencephalon of Rx-Dicer mutant embryos. F,G - Distribution of neurons (Tuj1+ cells) and apical adherens junction protein Par3 in the OB primordium of control and Rx-Dicer mutant embryos at the indicated ages. Dotted lines indicate basal border, dashed lines indicate apical border. White arrowheads indicate ectopic neurons and/or absence of Par3. Arrow in (G) indicates accumulation of Par3 at the lumen of a nascent rosette. \n",
      ".Figure 2A - DAPI stain of sagittal sections through the rostral telencephalon of E17.5 control and Rx-Dicer mutant littermates showing the neocortex (NCx) and olfactory bulb (OB; dashed line). B - Expression pattern of Grm1 mRNA in OB (dotted line) of control and Rx-Dicer mutants. C - Quantification of OB perimeter (mean ± SEM; symbols indicate values for individual embryos); t-test, ***P < 0.001. N = 14 replicates per genotype. D - Immunostains of E17.5 control and Rx-Dicer mutant brains showing the distribution of progenitor cells (Ki67, red) and neurons (Tuj1, green). Arrowheads indicate rosettes. Sp, septum. E - Detail of a rosette from an E17.5 Rx-Dicer mutant embryo displaying typical features: closed apical surface (dotted line) with PH3+ apical mitoses (white arrowhead) and basal mitoses, surrounded by Tuj1+ neurons (green). Dashed line indicates the basal border of the rosette. F - Rostral half of an Rx-Dicer mutant E17.5 brain immunostained for Pax6, clarified and segmented to reveal rosettes (yellow). C, caudal; R, rostral. G,H - BrdU incorporation and cell cycle re-entry analysis with the progenitor cell marker Ki67 at E17.5, in the rostral cortex of control embryos compared to rostral rosettes of Rx-Dicer mutants. Dotted lines indicate apical surface, dashed lines indicate basal border of VZ. Arrowheads indicate double-positive cells. Data in histograms are mean ± SEM, symbols indicate values for individual embryos; n ≥ 3 embryos; X2-test, *P < 0.05, ***P < 0.001. I - Apical lumen of rosettes immunostained against apical complex proteins (Par3, β-Catenin), primary cilia (Arl13b), apical mitoses (PhVim) and neurons (Tuj1). J - Coronal section through the rostral telencephalon of an E14.5 Rx-Dicer mutant embryo illustrating the high abundance and location of rosettes (arrows), as revealed by the distribution of mitoses (PH3) and neurons (Tuj1). K-S - Analysis of the regional identity of rosettes in E14.5 Rx-Dicer mutants. (K) Schema of normal transcription factor expression patterns defining telencephalic regional identity. Expression of Ngn2, Tbr2 and Pax6 identifies rosettes in the rostro-dorsal telencephalon as having dorsal identity (L-N,P); Gsx2, Dlx2 and Dlx5 identify rosettes in the ventral telencephalon as having ventral identity (O,Q); Nkx2.1 identifies MGE rosettes as being normotopic (R); absence of Gsx2 in LGE rosette cells identifies them as ectopic (S). Tuj1 labels neurons. In (L,M), arrowheads indicate the border between dorsal and ventral territories, and dotted line indicates the outer border of the telencephalon. LGE, lateral ganglionic eminence; MGE, medial ganglionic eminence; P, pallium; SP, subpallium. \n",
      ".Figure 5A-C - Immunostains for activated p53 (red) in the rostral telencephalon of control and Rx-Dicer mutant embryos at E12.5, and quantification of positive cells (C). Dashed lines indicate apical side, dotted lines indicate basal side. N = 4 replicates per genotype. Scale bar, 50µm. D-L - Rostral telencephalon of E12.5 embryos of the indicated genotypes immunostained for the detection of apoptotic cells (Casp3), neurons (Tuj1) and the apical complex protein Par3, and quantification of cells positive for Casp3 (H). N = 4-7 replicates per genotype. Scale bar, 50 µm. M-O - Sagittal sections through the rostral telencephalon of E17.5 embryos of the indicated genotypes. Note the rosettes and general disorganization in Rx-Dicer single mutants, and complete rescue of this phenotype in Rx-Dicer-p53 double mutants. NCx, neocortex; OB, olfactory bulb. Scale bar, 1 mm. \n",
      ".Figure 4A,B - Changes in expression levels of miRNAs between control and Rx-Dicer mutants at E11.5 in the rostral telencephalon. Red dots indicate statistically differentially expressed miRNAs according to FDR correction of P values based on the Wald statistic (DESeq2 analysis) (B). Dashes lines indicate stringency limits in evaluation of results. C - Gene ontology terms (top) and signalling pathways (bottom) associated with target genes of the miRNAs differentially-expressed between control and Rx-Dicer mutants. D,E - Changes in mRNA expression levels between control and Rx-Dicer mutants at E11.5 in the rostral telencephalon. Statistically differentially expressed mRNAs according to FDR correction of P values based on the Wald statistic (DESeq2 analysis; Adj. P < 0.1 and FC > 1.25) are highlighted in red, and the three with the highest Adj. P value are identified by name. Blue dot indicates Irs2: FC=1.748, Adj. P = 5.95e-08. F - Gene ontology terms (top) and signalling pathways (bottom) associated with mRNAs differentially-expressed between control and Rx-Dicer mutants. G - Heatmaps of relative expression levels of DEGs related to apoptosis (top) and proliferation (bottom). Rows correspond to independent biological replicates. Indicated are some example genes. H - Enrichment plots from GSEA for MSigDB Hallmark Neurogenesis (NES = 2.98; p = 0; Adj. P = 0.00045), Regulation of cell proliferation (NES = 2.40; p=0; Adj. P = 0.0033), Positive regulation of cell adhesion (NES = 2.06; p = 0.0058; Adj. P = 0.013), Regulation of response to stress (NES = 2.50; p = 0; Adj. P = 0.0023), Positive regulation of cell death (NES = 1.84; p = 0.0179; Adj. P = 0.0326), Apoptotic signaling pathway (NES = 1.59; p= 0.047; Adj. P = 0.084). I,J - ISH stains of Irs2 mRNA, and qPCR for Irs2 mRNA and let-7-5p miRNA, in the rostral telencephalon of control and Rx-Dicer mutant embryos. Dashed line indicates basal border, dotted line indicates apical surface. Arrowheads indicate area with the greatest increase in Irs2 expression. BG, basal ganglia; H, hippocampus; NCx, neocortex; OB, olfactory bulb; Spt, septum. Histograms represent mean ± SEM (logarithmic scale); symbols in plots indicate values for individual embryos; t-test, *P <0.05, **P <0.01. N = 6-9 replicates per group. Scale bar, 100 µm. K - Working model of two potential genetic mechanisms leading to rosette formation in Dicer mutants: red, direct effect of increased p53 pathway activity upon loss of let-7; green, Irs2 expression is massively increased by the combined effect of let-7 loss and increased p53-related cell stress signals, which enhances proliferation and decreases cell adhesion, leading to rosettes. \n",
      ".Figure 6A - Relative Irs2 mRNA levels in HEK cells upon transfection with Gfp- or Irs2-encoding plasmids. Mean ± SEM; t-test, ****P < 0.0001. N = 6 replicates per group. B-D - Distribution of neurons (Tuj1, red), GFP (green) and mitoses (PH3, white) in the rostral telencephalon of wild-type E14.5 or E17.5 mouse embryos, as indicated, electroporated at E12.5 with the indicated plasmid combinations. Dashed line indicates the apical surface. Insets show the same field of view, with location and extent of electroporations. Note the overlap of rosettes (arrows) with ectopic neurons at the apical surface (arrowheads) and increased basal mitoses, upon overexpression of Irs2 (C). Asterisk indicates absence of malformation in nearby cortex. At E17.5 (D) GFP+ rosettes remain in deep cortical layers (arrowheads). E-H - Details of rosettes (large arrows in E; dashed lines in F-H) from Irs2+Gfp-overexpressing E14.5 embryos immunostained as indicated. Small arrows indicate apical mitoses, arrowheads indicate basal mitoses (F,G) and Par3+ apical surface at the centre of rosettes (H), dotted lines indicate apical surface, dashed lines indicate basal side of rosettes. Inset in (H) is at the same scale. I-M - Distribution of neurons (Tuj1, red), GFP (green) and mitotic (PH3, white) or apoptotic (Casp3, white) cells in the rostral telencephalon of wild-type E13.5 embryos electroporated at E12.5 with the indicated plasmids, and quantification of Casp3+ cells (M). Dashed line indicates apical surface, dotted lines indicate borders of the cortical plate (CP). Nascent rosettes (arrows) are next to ectopic ventricular neurons (arrowheads in K') and basal mitoses, without significant apoptosis in VZ (L',M). Mean ± SEM; t-test; ns, not significant, *P < 0.05. N = 3-6 replicates per group. N - Distribution of neurons (Tuj1, red) in the rostral telencephalon of wild-type E14.5 embryos electroporated at E12.5 with a combination of plasmids encoding Irs2, let-7a, let-7b and let-7c, and Gfp. Dashed line indicates apical surface. Formation of rosettes in Irs2-expressing embryos is rescued by overexpression of let-7. Inset shows the same field of view, with location and extent of electroporation. \n",
      ".Figure 8A - Schema of experimental design. At 40 days in culture, human cerebral organoids were electroporated with one of the indicated DNA plasmid combinations, followed by 7 additional days in culture prior to analysis. B-G - Stains and quantifications of human cerebral organoids electroporated with the indicated construct combinations, to reveal the frequency of electroporated cells remaining as progenitors (%GFP+Ki67+/GFP+). Histograms represent mean ± SEM; symbols in plots indicate values for individual organoids; X2-test, *P < 0.05. N = 8-11 replicates per group. Dashed lines indicate ventricular surface, solid arrowheads indicate Ki67+/GFP+ cells, open arrowheads indicate Ki67-/GFP+ cells. Scale bars, 50 µm. H - Schematic drawing of conclusions from this study. Let-7 expressed in the early telencephalic neuroepithelium represses the expression of p53 and Irs2, which maintains the cellular homeostasis and epithelial integrity. Loss of let-7, as in Dicer1 mutants, de-represses expression of Irs2 and of p53 pathway components, which further increase Irs2 levels. High Irs2 enhances progenitor cell proliferation and loss of apical adhesion, altering neuroepithelial homeostasis and integrity, and leading to the formation of hyperproliferative rosettes. \n",
      ".Figure 7A - Frequency of rosette formation in the experimental conditions indicated. n, number of embryos per condition. B-G - Distribution of neurons (Tuj1, red), GFP (green) and mitoses (PH3, white) in the rostral telencephalon of wild-type E14.5 mouse embryos electroporated at E12.5 with the indicated plasmid combinations. Inset shows the same field of view, with location and extent of electroporations. In (C) and (E), arrows indicate rosettes and arrowheads indicate neuronal ventricular ectopias. (D) and (F) are high magnification details of individual rosettes (dashed lines), where arrowheads indicate apical and basal mitoses. Loss of endogenous let-7 drives the formation of rosettes and ventricular neuronal ectopias (C), which is rescued with the additional loss of Irs2 (G). \n",
      "Fig.Figure 1A) z-stacks were acquired from FM4-64 labeled vacuoles in wildtype cells (BJ3505), using a spinning disc microscope (step size 200 nm). 3D reconstructions from these stacks reveal a peripherally located fusion pore. Scale bar 2 µm. B) Wildtype (BJ3505) and isogenic pmc1∆/vcx1Δ mutant cells were labeled with FM4-64. The number of vacuoles per cell was determined in 3 independent experiments evaluating 200 cells each. Scale bar: 2 µm. C) Frequency of visible fusion pores. Cells were grown as in (B). The percentage of cells showing visible fusion pore was determined from z-stacks. Means s.d. are shown for 100 stained vacuoles from 3 independent experiments. D) Schematic view of Vph1-GFP and FM4-64 distribution over the vacuolar membranes in the presence of a nanoscopic fusion pore or of an extended hemifusion diaphragm. E) Distribution of Vph1-GFP and FM4-64 in vacuole-vacuole contact sites. Vph1-GFP expressing cells, labeled with FM4-64, were analyzed under a spinning disk confocal microscope equipped with two cameras for the simultaneous acquisition of GFP and FM4-64 fluorescence signals. Separate channels are shown on the right. The entire contact surface, but not the pore, is accessible to both probes. Scale bar: 2 µm. F) Expansion of a vacuolar fusion pore in real time. pmc1∆/vcx1Δ cells were labeled with FM4-64 and immobilized in a 50 µL flow chamber (Ibidi). The osmotic value of the medium was changed by perfusion with water, using a pump with a flow of 30 µL/s. A frame was acquired every 2 s for a total period of the 30 s, using the laser at minimal intensity. \n",
      "Fig.Figure 2A) Vacuolar membranes of wildtype (BY4741), nyv1∆ vam3∆, ypt7∆ or vma16F190Y cells were labeled with the vital dye FM4-64. A vacuole that was in a sufficiently peripheral location to be selectively photobleached was exposed to a laser pulse (white arrows indicate the bleaching area). Recovery of fluorescence in this area was assayed 10 seconds later by spinning disc confocal microscopy. Scale bar: 2 µm. B) Kinetics of FM4-64 recovery in (A). Means and s.d. are shown for 20 vacuole clusters from 3 independent experiments. C) In vivo lipid and content mixing. Vacuoles in living yeast were labeled with the indicated lumenal (CDCFDA) and membrane (FM4-64 or PX-GFP) probes. Note that CRY1 cells lack the ADE2 gene and hence naturally accumulate 5-amino-1-(5-phospho-D-ribosyl) imidazole as fluorescent fluid phase marker in the vacuolar lumen. FRAP-experiments were performed (arrows indicate the bleaching area). 1. Non-bleached lumenal area. 2. Bleached lumenal area. 3. Bleached membrane area. Scale bar: 2 µm. D and E) Kinetics of fluorescence recovery after photobleaching of the probes in (C). Numbers denote the areas shown in (C): 1. Non-bleached lumenal area. 2. Bleached lumenal area. 3. Bleached membrane area. Means and s.d. are shown for 20 vacuole clusters from 3 independent experiments. \n",
      "Fig.Figure 4A) Synthetic reporter proteins carrying EGFP either at the lumenal or cytosolic end of the single-spanning transmembrane domain of Nyv1, which carries no hydrophilic extensions at its lumenal C-terminus. B) The topology of these constructs in the vacuolar membrane is indicated. C,D) FRAP assays for fusion pores. The indicated cells expressing (C) yeGFP-LT or (D) LT-yeGFP were labeled with FM4-64 and subjected to FRAP experiments Cells were imaged before and 0 (bleached) or 30 seconds after photobleaching (recovery). The bleached areas are indicated by an arrow. E,F) FRAP assays for fusion pores were performed as in (C, D), but with nyv1∆ cells. Scale bar: 2 µm. G) The histogram reports the fraction of cells showing FRAP. Means and s.d. are shown from 3 independent experiments, with 100 cells analyzed in each. \n",
      "Fig.Figure 5A-F) The indicated yeast strains were transformed with expression plasmids for FYVE2-GFP and their vacuolar membranes were labeled with FM4-64. FRAP was performed using a spinning disk microscope equipped with an emission beam splitter and two synchronized cameras, allowing to simultaneously analyze FM4-64 and GFP fluorescence. The histograms show the kinetics of signal recovery for (A) wildtype (WT), (B) vam3tsf, (C) ypt7T22N, (D) vma16F190Y, (E) vps33P184L, and (F) nyv1∆ cells. Scale bar: 2 µm. Arrows indicate the bleaching area. Means and s.d. are shown for 20 stained vacuoles from 3 independent experiments. \n",
      "Fig.Figure 3A) FM4-64 and GFP excitation/emission spectra. B) FRET-based in vivo strategy to distinguish localization of FM4-64 in the inner or outer leaflets of vacuolar membranes. EGFP tags were added to the cytosolic C-terminus of Vph1 or to the lumenal C-terminus of Nyv1. C) Assay by microscopy. Half of two cultures of cells expressing Vph1-EGFP or Nyv1-EGFP were labeled with FM4-64, washed and mixed with the other, non-labeled half of the same culture. Cells were analyzed by spinning disc confocal microscopy. The intensities of GFP fluorescence were tracked along the dashed lines and compared between neighboring FM4-64-labeled and unlabeled cells. D) FACS analysis. The strains were labeled washed and analyzes by FACS. Shown are side (SSC-A) and forward scatter (FSC-A), as well as the fluorescence intensity distributions for FM4-64 and for GFP in the absence or presence of FM4-64. E) Comparison of the GFP mean fluorescence from FM4-64-stained cells and unstained cells (20'000 each). F) Possibilities for FM4-64 transfer between the inner leaflets of two tethered vacuoles. Flipping between the trans-leaflets of a hemifusion structure; reversible flipping into the outer leaflet, establishing a minor pool that then transfers through the buffer or through the outer leaflet; or passage through a fusion pore. \n",
      "Fig.Figure 8(A) A tension-less protein-lined metastable fusion pore formed between two mixed (3:2, PC:PE) model membranes (POPC head groups colored light brown, POPE head groups colored orange). The radius of the pore is 1.5 nm. The yellow sphere mimics a steric perturbation on the fusion site imposed by HOPS binding to one of the SNARE complexes. The upper panel shows a cross section of this pore in side view. The lower panels show side views of cross sections through this 1.5 nm pore, which is at the free energy minimum, as well as of a 3 nm pore, which is at the threshold where the line tension converges to a constant value Notice that the cross-sectional shape of the pore (shown in top views) remains largely spherical despite the heterogeneity of the fusion site. The top view show also a fluorescent dye molecule (dichlorofluorescein, MW=401 Da, in purple) to illustrate its scale relative to the 1.5 nm pore. (B) Line tension of the fusion pore in pN as function of pore radius and the fusion proteins inserted into it. The black dotted line represents the position of the local minimum. The initial expansion is relatively insensitive to the presence of fusion proteins. Each data point represents the average line tension measured over an ~600 ns equilibrium simulation. Error bars are based on block averaging over the obtained data set (the autocorrelation time is ~30 ns). \n",
      "Fig.Figure 9(A) Asymmetry in lateral leaflet composition and population enforces a fusion pore with a radius of 0.5 nm (POPC head groups colored light brown, POPE colored orange). The panels show cross-sections, viewed from the side (left) or from the top (right). (B) Hemifission event in the presence of a single SNARE complex and in the absence of HOPS. Notice that the SNARE complex is actively expelled from the interior of the pore and bent. (C) The graph shows the probability of hemifission after 2 μs for 9 scenarios differing in lipid and protein content of the pore (10 simulations for each scenario). We only vary the concentration of PE-lipids in the trans-leaflets and the presence of fusion constituents (lipids only: black circles, 1 SNARE complex: red squares, 1 SNARE complex + HOPS: blue stars). The trans-leaflets are always comprised of 5% less lipids than the cis-leaflets. The presence of SNARE complexes and/or HOPS reduces the frequency of hemifission with respect to a system consisting of lipids only. \n",
      "Fig.Figure 6A-D) Wild-type (A, B) or nyv1∆ cells (C,D), which lack the vacuolar R-SNARE Nyv1, were labeled with FM4-64. Cells were immobilized on a chambered slide and subjected to FRAP experiments (C) Vacuole fission induced by hypertonic shift. FRAP experiments were performed before (0') or 15 min after addition of NaCl to 0.5 mol/l. The fluorescence images show the vacuoles inside a single cell. (B, D) Cells in S-phase were selected, which grow a daughter cell and therefore fission their vacuoles into a chain of tubulo-vesicular structures, part of which are transported into the daughter cell. FRAP experiment on vesicles in the inheritance structure were performed. Arrows indicate the bleaching area, dashed orange lines the cell cortex. Scale bar: 2 µm. \n",
      "Fig.Figure 7A) Wildtype (BY4741) cells labeled with FM4-64 show extensive contact zones between vacuoles. Scale bar: 2 µm. B) Schematic view of FM4-64 diffusion through a small fusion pore located centrally (left panel) or laterally (right panel). The laterally positioned fusion pore leads to diffusion paths of unequal length for fluorescence recovery on both sides of the bleached vacuole. The red line indicates the intersections with the membrane where fluorescence recovery is measured. C) FRAP analysis of FM4-64-labeled wildtype cells was performed by acquiring pictures in continuous acquisition mode after bleaching, for a total period of 5 s. Three representative frames after photobleaching are shown. The bars in the magnified parts of frames 2 and 3 illustrate the positions at which the recovery of FM4-64 fluorescence was quantified. Scale bar: 2 µm. D) Quantification of recovery experiments in (C). A delay of >50 ms between the appearance of the signal on both sides of the photobleached vacuole was considered as asymmetric recovery. The histogram shows means and s.d. from 3 independent experiments evaluating 50 stained vacuoles each. A Schematic view of the various possible positions of a fusion pore and its consequences for our ability to detect asymmetric FM4-64 recovery. The contact zone between the two vacuoles is tilted by 90° in the middle panel, which gives four examples for the positioning of the pore (shown as a black dot). Asymmetric recovery could only be detected if the pore is sufficiently far from the center of the contact zone and close to the optical plane that is imaged (the cases marked in red). \n",
      ": Determination of an optimal concentration for each HDACI to induce HIV-1 production in latently-infected cells.Figure 2A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of HDACIs. At 24 h post-treatment, viral production was measured by quantifying p24 antigen production in culture supernatants (A, B) A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of HDACIs. At 24 h post-treatment, metabolic activity was assessed by a WST-1 assay (C, D).Data information: Means and standard errors of the means from three independent biological duplicates (n=6) are indicated. The result obtained with mock-treated cells was arbitrarily set at a value 100% (C, D). The selected doses were indicated by \"an arrow\". Plasmatic concentrations after usual dosage (Cmax) of each drug in human therapy are indicated below the LRA names and by a box in the graph. \n",
      ": Determination of 5-AzadC + SAHA treatment schedule in vitro and ex vivo.Figure 3A-D J-Lat 8.4 cell line was mock-treated or treated with 5-AzadC and/or SAHA for different periods of time as indicated. Samples were harvested at indicated times. Viral production was measured by quantifying p24 antigen production in culture supernatants (A, B) A-D J-Lat 8.4 cell line was mock-treated or treated with 5-AzadC and/or SAHA for different periods of time as indicated. Samples were harvested at indicated times. metabolic activity was assessed by a WST-1 assay (C, D). Means and standard errors of the means from three independent biological duplicates (n=6) are indicated. The result obtained with mock-treated cells was arbitrarily set at a value of 100% (C, D). E From data of ex vivo cultures of CD8+-depleted PBMCs isolated from 24 HIV+ patients presented in Appendix Table S2, the extracellular HIV-1 genomic RNA levels for each LRA treatment are represented. One night after cell purification, cells were mock-treated or simultaneously treated with 5-AzadC (1µM) and/or SAHA (1µM). Six days after treatment, the concentration of viral RNA in culture supernatants was determined (in copies/ml). The results were reported as the actual HIV RNA copy numbers/ml or as an estimated value calculated as 50% of the smallest value when HIV RNA was not detected in order to assign a log value. Means are represented. \n",
      ": The DNA methylation inhibitor 5-AzadC induces HIV-1 expression in latently-infected T cells.Figure 1A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of 5-AzadC or 5-AzaC. At 72 h post-treatment, viral production was measured by quantifying p24 antigen production in culture supernatants (A) A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of 5-AzadC or 5-AzaC. At 72 h post-treatment, metabolic activity was assessed by a WST-1 assay (B) A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of 5-AzadC or 5-AzaC. At 72 h post-treatment viral protein expression was analyzed by FACS (C) A-D J-Lat 8.4 cells were mock-treated or treated with increasing concentrations of 5-AzadC or 5-AzaC. At 72 h post-treatment; and initiated (primers TAR) or elongated (primers tat) transcripts were quantified by RT-qPCR (D). E J-Lat 8.4 cells were mock-treated or treated with 5-AzadC (400nM) or TNFα (10ng/ml) as positive control. At 24 h, 48 h or 72 h post-treatment, initiated (primers TAR) or elongated (primers tat) transcripts were quantified by RT-qPCR. \n",
      ": Sequential 5-AzadC + HDACI treatments induced a variation of cellular metabolic activity depending on the treatment in CD8+-depleted PBMCs from HIV negative donors.Figure 5One night after cell purification, CD8+-depleted PBMCs from 12 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added to the cultures. Six days after 5-AzadC treatment, WST-1 assay was performed. Mock value was arbitrary fixed at 100% for each individual. \n",
      ": Representation of reactivation status of ex vivo cultures of CD8+-depleted PBMCs isolated from HIV+ patients.Figure 6Figure 6: Representation of reactivation status of ex vivo cultures of CD8+-depleted PBMCs isolated from HIV+ patients.A-C From data of ex vivo cultures of CD8+-depleted PBMCs isolated from HIV+ patients (Table 2), the extracellular HIV-1 genomic RNA levels for each LRA treatment are represented from all patient cell cultures (A), from patient cell cultures presenting no viral reactivation in mock condition (C), and from patient cell cultures exhibiting reactivation in mock condition (B). One night after cell purification, cells were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. Six days after 5-AzadC treatment, the concentration of viral RNA in culture supernatants was determined (in copies/ml). The results were reported as the actual HIV RNA copy numbers/ml or as an estimated value calculated as 50% of the smallest value when HIV RNA was not detected in order to assign a log value. Means are represented. \n",
      ": Sequential 5-AzadC + HDACI treatments synergistically activate HIV-1 gene expression and production in latently-infected cells.Figure 4A-F J-Lat 8.4 (A, C, E) and 15.4 (B, D, F) cell lines were mock-treated or treated with 5-AzadC. At 48 h post-treatment, HDACIs were then added for 24 h. At 72 h 5-AzadC post-treatment samples were harvested and analysed as follows: viral p24 production in cell supernatant was measured (A and B) A-F J-Lat 8.4 (A, C, E) and 15.4 (B, D, F) cell lines were mock-treated or treated with 5-AzadC. At 48 h post-treatment, HDACIs were then added for 24 h. At 72 h 5-AzadC post-treatment samples were harvested and analysed as follows:FACS analyses were performed and the percentages of GFP+ cells are presented as histograms (C and D) A-F J-Lat 8.4 (A, C, E) and 15.4 (B, D, F) cell lines were mock-treated or treated with 5-AzadC. At 48 h post-treatment, HDACIs were then added for 24 h. At 72 h 5-AzadC post-treatment samples were harvested and analysed as follows: initiated (primers TAR) or elongated (primers tat) transcripts were quantified by RT-qPCR and results were normalized using the β-actin gene primers and are presented as histograms representing fold-inductions compared to the mock-treated condition (E and F). Means and standard errors of the means from three independent biological duplicates (n=6) are indicated. The result obtained with mock-treated cells was arbitrarily set at a value of 1 (E, F). \n",
      ": Comparison of simultaneous 5-AzadC + SAHA combined treatment with its corresponding sequential treatment in ex vivo cultures of CD8+-depleted PBMCs isolated from aviremic cART-treated HIV+ patients.Figure 7A Reactivation status of patient cell cultures presenting no viral reactivation in mock condition isolated from aviremic cART-treated HIV+ patients (shown in Appendix Table S2).B Reactivation status of patient cell cultures presenting no viral reactivation in mock condition isolated from aviremic cART-treated HIV+ patients (shown in Table 2).Data information: The results were reported as the actual HIV RNA copy numbers/ml or as an estimated value calculated as 50% of the smallest value when HIV RNA was not detected in order to assign a log value. Means are represented by a line. \n",
      ": Representation of reactivation status of ex vivo cultures of resting CD4+ T cells.Figure 8Figure 8: Representation of reactivation status of ex vivo cultures of resting CD4+ T cells.A-C From data of ex vivo cultures of resting CD4+ T cells isolated from HIV+ patients (Table 3), the extracellular HIV-1 genomic RNA levels for each LRA treatment are represented from all patient cell cultures (A), from patient cell cultures presenting no viral reactivation in mock condition (C), and from patient cell cultures exhibiting reactivation in mock condition (B). One night after cell purification, cells were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. Six days after 5-AzadC treatment, the concentration of viral RNA in culture supernatants was determined (in copies/ml). The results were reported as the actual HIV RNA copy numbers/ml or as an estimated value calculated as 50% of the smallest value when HIV RNA was not detected in order to assign a log value. Means are represented. \n",
      ": 5-AzadC + Panobinostat and 5-AzadC + Romidepsin treatments induce a significant decrease in the cell surface CD4 receptor expression.Figure 10One night after cell purification, HLA DR-CD69-CD25-CD4+T cells from 4 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. The median fluorescence intensity of CD4 receptor of viable CD4+T cell subset was assessed 6 days after 5-AzadC treatment by flow cytometry analysis relative to mock treatment before 5-AzadC stimulation corresponding to day 0. Means are represented. \n",
      ": 5-AzadC + Panobinostat and 5-AzadC + Romidepsin treatments do not induce global T-cell activation.Figure 9A-H One night after cell purification, CD8+-depleted PBMCs or HLA DR- CD69-CD25-CD4+T cells from 4 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. The activation status of CD4+T cell subset was assessed 6 days after 5-AzadC treatment by flow cytometry analysis of cellular activation markers relative to mock treatment before 5-AzadC stimulation corresponding to day 0. Means are represented. A-H One night after cell purification, CD8+-depleted PBMCs or HLA DR- CD69-CD25-CD4+T cells from 4 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. The activation status of CD4+T cell subset was assessed 6 days after 5-AzadC treatment by flow cytometry analysis of cellular activation markers relative to mock treatment before 5-AzadC stimulation corresponding to day 0. Means are represented. A-H One night after cell purification, CD8+-depleted PBMCs or HLA DR- CD69-CD25-CD4+T cells from 4 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. The activation status of CD4+T cell subset was assessed 6 days after 5-AzadC treatment by flow cytometry analysis of cellular activation markers relative to mock treatment before 5-AzadC stimulation corresponding to day 0. Means are represented. A-H One night after cell purification, CD8+-depleted PBMCs or HLA DR- CD69-CD25-CD4+T cells from 4 HIV negative donors were mock-treated or treated with 5-AzadC. Three days post-treatment, 1/3 of medium was replaced and HDACIs were added in the cultures. The activation status of CD4+T cell subset was assessed 6 days after 5-AzadC treatment by flow cytometry analysis of cellular activation markers relative to mock treatment before 5-AzadC stimulation corresponding to day 0. Means are represented. \n",
      ": Scalp psoriasis exhibits reduced expression of c-JUN and JUNB in hair follicle stem cells (HF-SCs) A-D.Figure 1Representative composite immune-fluorescent images of whole hair follicle units from non-lesional and lesional scalp psoriasis patients. K15 (green), c-JUN (red A, C) and JUNB (red B, D) and DAPI (blue). Confocal images of the bulge region of human psoriatic hair follicles from non-lesional and lesional regions of the scalp. K15 (green), c-JUN (red E, G) and JUNB (red F, H) and DAPI (blue). Percentage of HF-SCs (K15+) that express c-JUN (I) and JUNB (J) in the bulge region of human psoriatic hair follicles from non-lesional and lesional scalp in comparison with healthy scalp. n=2-6 hair follicles in anagen per group from five psoriatic patients and two healthy patients. \n",
      ": Different molecular signatures of mutant and non-mutant epidermal stem cells (EpSC) A.Figure 5Differentially expressed genes (DEGs) from the four subpopulations (bulge HF-SCGFP; bulge HF-SCTom; b-KCGFP; and b-KCTom) from psoriatic DKO*-mT/mG mice in comparison with their control counterparts (Co). The graphs represent the number of genes differentially up-regulated or down-regulated specifically in each group of interest obtained by Venn diagram analysis Heat map for specific genes relative to biological process enriched based on Gene Ontology IDs for DEGs that are: i) exclusively up-regulated in HF-SCGFP; ii) exclusively up-regulated in HF-SCTom; iii) commonly up-regulated in HF-SCGFP and HF-SCTom. Prediction of secreted proteins enriched in GFP+ subpopulations by the analysis of DEG commonly up-regulated in HF-SCGFP and b-KCGFP by ProteINSIDE analysis of RNA-seq. Gene expression of TSLP in different epidermal cell subpopulations sorted at mid-term (D7) or late-term (D30) of psoriasis-like progression in DKO* mice. protein expression of TSLP in different epidermal cell subpopulations sorted at mid-term (D7) or late-term (D30) of psoriasis-like progression in DKO* mice. n=2-3. Quantification of TSLP production in sera of control, DKO* and DKO*K15 mice at day 30 of psoriasis-like disease by ELISA. n = 4 per group. \n",
      ": Mutant and non-mutant interfollicular keratinocytes have different proliferation and apoptotic rates during psoriasis-like disease progression A.Figure 2Schematic representation of the JunB and c-Jun double knockout mouse model (DKO*) with lineage tracing used to investigate the psoriasis-like disease development in mice. Briefly, DKO* psoriasis-like mouse model was generated by the cross of mice carrying the floxed JunB allele (JunBf/f) and floxed c-Jun allele (c-Junf/f) with the transgenic mice expressing the Cre recombinase-estrogen receptor fusion under the control of the basal keratinocyte specific K5 promoter (K5-Cre-ERT) to obtain JunBf/f c-Junf/f K5-Cre-ERT mice. To trace epidermal specific deletion of JunBf/f and c-Junf/f in DKO* mice, global double-fluorescent Cre reporter mouse Gt(ROSA) 26Sortm4(ACTB-tdTomato,-EGFP) Luo/J (referred as mT/mG) was crossed with DKO* mice (referred as DKO*-mT/mG). Experimental timeline to induce psoriasis-like disease in 8 week-old mice with four consecutive doses of tamoxifen injections (2 mg). Upon induction, non-mutant keratinocytes express Tomato, while mutant keratinocytes express GFP. GFP expression was analyzed at 0, 5, 7, 15 and 30 days post-induction. Composite immunofluorescent images showing GFP expression of whole-ear sections from DKO*-mT/mG mice at day 5, 7, 15 and 30. Increased GFP expression is shown at day 7 followed by progressive decrease of GFP+ keratinocytes (white dotted line represents basal layer, and red dotted line represents outermost skin layer). n=3 per time point. Quantification analysis of GFP expression (top) and epidermal thickness (bottom) of DKO* mice at different time points during psoriasis-like disease progression. n=6 per time point. Statistical significance *p<0.05, **p<0.01, ***p<0.001 (t-student two tailed-test relative to control group). Quantification analysis of cleaved Caspase-3 (cCas3 top) and Ki67 (bottom) in ear skin of DKO* mice at different time points during psoriasis-like disease progression. n=3 per time point. Intravital confocal time-lapse imaging of non-mutant Tomato keratinocytes and mutant GFP keratinocytes of DKO*-mT/mG at day 7 after first tamoxifen injection. Dotted circles emphasize areas in which mutant GFP+ keratinocytes are replaced by non-mutant Tomato+ keratinocytes. Percentage of GFP+ area eliminated during intravital confocal time-lapse imaging at day 5, 7 and 15 in DKO*-mT/mG mice (Time-lapse 1:8h). Positive value: Area of GFP increased. Negative value: Area of GFP reduced. Fluorescence imaging of DKO*-mT/mG ear skin at day 30 after first tamoxifen injection shows that remaining mutant GFP+ keratinocytes reside along the hair follicles. White dotted line separates epidermis and dermis. White arrows represent GFP+ hair follicles. Intravital confocal time-lapse imaging of DKO*-mT/mG at day 15 after first tamoxifen injection. Mutant GFP+ Keratinocytes (white arrows) are maintained around hair follicles. \n",
      ": Inactivation of c-Jun and JunB in bulge HF-SCs is sufficient to initiate psoriasis-like development A.Figure 3Experimental timeline showing the different time points analyzed after mifespristone induction (day 5, 7, 15 and 30, green) to evaluate GFP expression in ear skin from control (CoK15-mT/mG) and mutant (DKO*K15-mT/mG) mice. Ear pictures of control, DKO*, and DKO*K15 mice after 15 days of induction show clear signs of inflammation and psoriatic-like scales in both models. Representative images of hematoxylin and eosin (H&E) staining of ear sections from control, DKO* and DKO*K15 mice. Ear thickness measurement at different time points during psoriasis-like development in control, DKO* and DKO*K15 mice. n > 5 per group. Statistical significance *p<0.05, **p<0.01, ***p<0.001 (t-student two tailed-test relative to control group). FACS quantification of CD45+ cells from ear skin of control, DKO* and DKO*K15 mice at different time points during psoriasis-like development. n > 5 per group. Statistical significance *p<0.05, **p<0.01, ***p<0.001 (t-student two tailed-test relative to controls). Quantification of IL-17A, IL-6 and LCN2 production in sera of control, DKO* and DKO*K15 mice at day 30 of psoriasis-like disease by ELISA. n > 4 per group. Statistical significance *p<0.05, **p<0.01, ***p<0.001 (t-student two tailed-test relative to controls). Quantification of total number of GFP+ epidermal cells by FACS analysis from DKO*-mT/mG and DKO*K15-mT/mG mice at different time points during psoriasis-like progression compared to Co-mT/mG mice. n = 3-6 per group and time point. Representative immunofluorescence images of ear sections from CoK15-mT/mG and DKO*K15-mT/mG mouse models after 15 days of induction. Red (Tomato), green (GFP) and blue (DAPI). Hyperplasia of Tomato+ keratinocytes in psoriatic-like DKO*K15-mT/mG mouse with clusters of GFP+ keratinocytes are derived from hair follicle K15+ GFP+ stem cells. n=3 per group. Dotted lines separate epidermis and dermis. Quantification analysis of cleaved Caspase-3 (cCas3) and Ki67 in ear skin of DKO*K15-mT/mG mice at different time points during psoriasis-like disease progression. \n",
      ": Differential behavior of inter-follicular and follicular stem cells A.Figure 4Schematic representation of cell sorting and single culture conditions. GFP+ bulge hair follicle stem cells (HF-SCs, CD34+ CD49fhigh) and GFP+ basal keratinocytes (b-KC, CD49fhigh) from ear skin of Co-mT/mG and DKO*-mT/mG mice were collected at day 10 after tamoxifen induction, sorted by FACS and cultured at low density (2000 cells/cm2) during 3-5 days as primary cultures. Time lapse imaging of 6 fields was performed during 48 hours, once small colonies of 5-10 cells were observed. Average number of live and dead (TOPRO-3+ Dapi+) primary keratinocytes derived from HF-SCGFP at different time points of time lapse capture during 48 hours. There is a slight increase over time of live HF-SCs from DKO* mice when compared to control cells, and no difference in average of dead cells. n = 4 fields from two independent experiments. Average number of live and dead (TOPRO-3+ DAPI+) primary keratinocytes derived from basal keratinocytesGFP show a significant increase in DKO* basal live keratinocytes at 12 hours, followed by a significant increase in cell death at 48 hours. Schematic representation of cell sorting and co-culture conditions. GFP+ and Tom+ bulge HF-SCs (CD34+ CD49fhigh) and GFP+ and Tom+ basal keratinocytes (CD49fhigh) from ear skin of Co-mT/mG and DKO*-mT/mG collected at day 10 after tamoxifen induction were sorted by FACS-sorting and co-cultured (same proportion of GFP and Tom cells) at low density (2000 cells/cm2) during 3-5 days. When small colonies of 5-10 cells were formed, time lapse imaging of 6 fields was performed during 48 hours. Average number of live primary keratinocytes from co-cultures of GFP+ with Tomato+ bulge HF-SCs at different time points of time lapse capture during 48 hours. Co-culture of HF-SCGFP with HF-SCTom from DKO*-mT/mG mice shows a significant increase of HF-SCTom, while the HF-SCTom from DKO*-mT/mG mice show a normal growth when they are co-culture with HF-SCGFP from control mice. Average number of live primary keratinocytes from co-cultures of GFP+ with Tomato+ basal keratinocytes at different time points of time lapse capture during 48 hours. Co-culture of b-KCGFP with b-KCTom from DKO*-mT/mG mice show significant increase of b-KCTom, while these b-KCTom from DKO*-mT/mG mice show a normal growth when they are co-culture with b-KCGFP from control mice. Average total number of live Tomato+ basal keratinocytes (from DKO* or control mice) at different time points cultured in the bottom part of trans-well system cultures with GFP+ basal keratinocytes (from DKO* or Control mice) cultured in the upper trans-well (Tw). n = 5 fields from three independent experiments. Tom+ basal keratinocytes significantly increased when these were cultured with mutant GFP+ basal keratinocytes in the trans-well in comparison with control GFP+ basal keratinocytes. \n",
      ".Figure 1A-G Hematoxylin and eosin (H&E) staining of the frontal sections of miniature pig mandible slices showing morphological changes during the permanent canine (PC) initiation stage from embryonic day 50 (E50) to postnatal day 10 (PN10); (A'-F') are magnifications of boxed regions in their corresponding figure panel. DC, deciduous canine; PC, permanent canine; SDL, successional dental lamina; n = 3. Scale bars = 100 µm. H In situ hybridization (ISH) showing the expression pattern of Sox2 in the initiation stage from E50 to E90; right figure panels are magnifications of boxed regions in left panels. Dashed lines mark the position of the successional dental lamina (SDL); green arrowhead indicates positive staining of Sox2 at the tip of the SDL. n = 3. Scale bars = 100 µm. I Immunofluorescence (IF) of Ki67 at the SDL at E60, E70, E90; right figure panels are magnifications of boxed regions in left panels. n = 3. Scale bars = 100 µm. J Semi-quantification and comparison of Ki67 expression levels during E60, E70, and E90 stages. n = 3. Data information: Data represent the means ± SEM. *P < 0.05 (one-way ANOVA and Newman-Keuls post-hoc tests). K TUNEL assay showing the apoptosis status of the SDL at E60, E70, E90; right figure panels are magnifications of boxed regions in left panels. n = 3. Scale bars = 100 µm. L Diagram illustrating the initiation stage at E50 (attached SDL), E60 (detached SDL), E90 (bud stage), and PN10 (bell stage). OEE, outer enamel epithelium; IEE, inner enamel epithelium. \n",
      ": Scalp psoriasis patient skin shows increased expression of TSLP in hair follicles and epidermis A.Figure 7Representative composite immune-fluorescent images of whole hair follicle units by confocal from non-lesional and lesional scalp psoriasis patients. c-JUN (purple), K-15 (green), TSLP (red) and DAPI (blue). TSLP intensity measurement in hair follicle units and epidermis by ImageJ from healthy scalp, non-lesional and lesional scalp psoriasis patients. Representation of fold change relative to control healthy scalp. n= 3 samples. Representative composite immune-fluorescent images of whole hair follicle units by confocal from non-lesional and lesional scalp psoriasis patients (D) and magnification images of specific regions from the HFs (E). CD200 (green), c-JUN (white), TSLP (red), DAPI (blue). \n",
      ": TSLP secreted by mutantGFP epidermal cells induces hyper-proliferation of non-mutantTom keratinocytes A.Figure 6Experimental design for induction of mutantGFP KCs in vitro and neutralization of secreted TSLP by anti-TSLP. Primary keratinocytes were cultured from the skin of Junblox/lox; c-Junlox/lox; RosaKi/+; K5cre-ERT+/+ mice. 50% of mutantGFP KCs were induced by infection with Cre Adenovirus, the non-infected cells were non-mutantTom KCs. Adeno-empty was used as control. EdU was added for 5 hours to label proliferating tomato+ cells in 6-day co-culture. Quantification was performed by Confocal imaging analysis. TSLP production in conditioned media quantified by ELISA three days after infection with Ad-empty or Ad-cre. n=3 independent experiments. Percentage of EdU+ Tomato+ keratinocytes after blocking with anti-TSLP. Anti-TSLP neutralizes the hyper-proliferation of non-mutantTom KCs in vitro after infection with Ad-cre. Gene expression of Tslpr after TSLP neutralization is reduced. Gene expression of IL-7ra after TSLP neutralization is increased. Gene expression of VEGFα after TSLP neutralization is reduced. Experimental design for TSLP neutralization in vivo. TSLP neutralization was performed by triple intra-dermal injections of 20 μg of anti-TSLP or IgG to the right ears of DKO* and DKO*15 mice five days after of psoriasis-like induction. EdU was added into mice by IP injection 2 hour before euthanasia. Representative images of treated ears after 9 days of anti-TSLP or IgG treatment. Longitudinal analysis of ear thickness in DKO* mice at different time points during psoriasis-like disease progression Trans-epidermal water loss (TEWL) (J) in DKO* mice at different time points during psoriasis-like disease progression Percentage of EdU+ epidermal cells (CD45-CD49f+) after blocking with anti-TSLP in DKO* and DKO*15 mice Representative immune-fluorescence images of ear sections from control or DKO* mice treated with IgG or anti-TSLP) stained for Ki67. Ki67 expression was reduced in mice treated with anti-TSLP (n =3). \n",
      ".Figure 2A Three-dimensional reconstruction of serial H&E frontal sections of miniature pig mandibles at embryonic day 60 (E60) and E90; deciduous canine (DC) in purple, permanent canine (PC) in yellow, and alveolar socket in green. The red, blue, and green arrows indicate the width of the labial alveolar socket, lingual alveolar socket, and DC, respectively. n = 3. B DC and labial and lingual alveolar socket widths in the horizontal plane at the bottom of the PC during E60 and E90. n = 3. Data information: Data represent the means ± SEM. Unpaired t-tests, *P < .05, **P < .01, ***P < .001. C The proportion of DC width relative to the total alveolar socket width during E60 and E90. n = 3. Data information: Data represent the means ± SEM. Unpaired t-tests, *P < .05, **P < .01, ***P < .001. D Micro-CT imaging of the whole mandible at E60; boxed region is magnified in D' (top panel). (D') Mandible slice isolated with Geomagic software. White dashed lines mark DC. E 3-D color map (left) after alignment of mandible slices before and after (sham) surgery showing a comparison of the surface points. Coronal sections through cusp tips (transparent squares, left) were selected for 2-D comparisons (middle); right panels are magnifications of yellow-boxed regions. The solid purple contour and dotted black contour indicate the pre- and post-surgery shapes, respectively. The distance between the two contours are the colored line segments showing the distance and direction of the movement. The colored ball in the 2-D comparison marks the position of the maximum displacement. The PC position is indicated in pink. n = 3. F Dissected mandible slice with a \"U\" shape. G In the cup model, the mandible slice was simplified as a cup according to its dimensions (unit, mm). Red arrows indicate the uniform force (stress) exerted on the inner wall of the mandible. The bottom of the cup was fixed to avoid rigid body motions. H 3-D color map shows the extent of deformation based on the cup model established with ANSYS software. I Probable range of mechanical stress inside the mandible evaluated by multiple simulation tests in which serial stress and Young's modulus were inputted into the cup model; the cup was set with the outer surface fixed (left, dr = 0 denotes that radial deformation of the outer surface equals 0) or the outer surface free (right). J Deformation of the mandible walls under a series of stress levels with different Young's moduli. Data were obtained from multiple tests as in I. Gray horizontal lines indicate upper and lower limit values of mean mandible wall displacement (79.74 and 36.48 µm, respectively); dashed colored lines indicate results of the free outer surface; solid colored lines indicate results of the fixed outer surface (with consideration of good linearity of the simulation results, the corresponding result points of the simulation series were omitted and replaced by solid or dashed colored lines for clarity); the actual stress value should be between the two extreme boundary conditions. Results show that the probable stress level ranged from 3-20 kPa. n = 3. \n",
      ".Figure 3A Diagram of the dissected miniature pig mandible slice illustrating the deciduous (DC) and permanent (PC) canine and application of compression in vitro using Flexcell FX-5000 Compression System. B-D H&E staining of canine frontal sections from embryonic day 60 (E60) (B), after culturing for two days without stress (0 kPa) (C), or with stress (3 kPa) (D). (B'-D') are magnifications of boxed regions in their corresponding figure panels. Scale bars = 50 µm. n = 6/6 in E60 group; n = 12/12 in 0-kPa group; n = 4/8 in 3-kPa group. E Diagram illustrating the required time during PC initiation; regular process from resting to initiation stage (left) and when the mandible is cultured without stress (0 kPa; middle) or with stress (3 kPa; right). F RT-qPCR of several mechanosensitive proteins (TGFB1, NFKB1, EGR1, and RUNX2) and of four mechanoreceptors (ITGB1, ITGB3, ITGAV and ITGA2) in PC and surrounding soft tissue at E60 and after culturing for two days with (3 kPa) or without extra mechanical stress (0 kPa). n = 3 for each group. Data represent the means ± SEM. One-way ANOVA (Newman-Keuls test for post-hoc comparisons between two groups), *P < .05, **P < .01, ***P < .001; NS, not significant. = \n",
      ".Figure 5A-C H&E staining of miniature pig canine frontal sections from embryonic day 60 (E60 group) (A) and after the mandible slices were infected with RUNX2 overexpression lentiviral vector with a Myc tag (overexpression group) (B) or overexpression control lentiviral vector with a Myc tag (O-control group) (C); (A'-C') are magnifications of boxed regions in the corresponding figure panels. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 50 µm D-F Immunohistochemical (IHC) staining of Myc expression in mandible slices from the E60, overexpression, and O-control groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 100 µm (D-F) G-I IF staining of RUNX2 expression (yellow arrows) in mandible slices of the E60, overexpression, and O-control groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 50 µm J, K Relative expression levels of Myc and RUNX2 in the E60, O-control, and overexpression groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. One-way ANOVA (Newman-Keuls test for post-hoc comparisons between two groups) for J, K , *P < .05, **P < .01, ***P < .001. L, M IF staining of RUNX2 expression after infecting mandible slices with scrambled shRNA lentiviral vector (knockdown control, K-control) (L) or RUNX2 shRNA lentiviral vector (knockdown) (M); (L'-M') are magnifications of boxed regions in the corresponding figure panels. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 50 µm (other panels). N Relative RUNX2 expression levels between K-control and knockdown groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests for N , *P < .05, **P < .01, ***P < .001. O-Q H&E staining of mandible slice sections from the K-control, knockdown + force (3 kPa), and K-control + force (3 kPa) groups. (O'-Q') are magnifications of boxed regions in the corresponding figure panels. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 50 µm R-T IF staining of RUNX2 expression (yellow arrows) of mandible slice sections from the K-control, knockdown + force (3 kPa), and K-control + force (3 kPa) groups. (R'-T') are magnifications of boxed regions in the corresponding figure panels. Data information: Data represent the means ± SEM. n = 3 for all experiments. Scale bars = 25 µm (R'-T') U Relative RUNX2 expression levels among the three groups Data information: Data represent the means ± SEM. n = 3 for all experiments. One-way ANOVA (Newman-Keuls test for post-hoc comparisons between two groups) for U, *P < .05, **P < .01, ***P < .001. V PC epithelium areas in the overexpression, O-control, knockdown, knockdown + force, K-control, and K-control + force groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests for V; , *P < .05, **P < .01, ***P < .001. \n",
      ".Figure 4A-L IF of integrin β1, ERK1, and RUNX2 in miniature pig canine frontal sections at embryonic day 60 (E60), E90, and E60 cultured under 3 kPa stress for two days and E60 cultured under 0 kPa stress for two days; (A'-L') are magnifications of boxed regions in the corresponding figure panels. M Relative IF expression levels of integrin β1, ERK1, and RUNX2 during E60, E90, and E60 cultured under 3 kPa stress for two days and E60 cultured under 0 kPa stress for two days. Data information: Data represent the means ± SEM. n = 3. Scale bars = 25 µm (A', G') and 50 µm (other panels). Unpaired t-tests for M , *P < .05, **P < .01, ***P < .001; NS, not significant. N Morphology of the human PC at weeks 18-19 (H&E staining); (N') is magnification of the boxed region in (N). DC, deciduous canine; PC, permanent canine; IEE, inner enamel epithelium; OEE, outer enamel epithelium. Mesenchyme between DC and PC is indicated with a green arrow. O-Q ISH expression patterns of ITGB1, ERK1, and RUNX2 in human PCs. Green arrows indicate the mesenchyme between DC and PC. R IF of RUNX2 in human PC. Mesenchyme between DC and PC is indicated with a red arrow. Red arrow indicates the mesenchyme between DC and PC. S PC epithelium areas in E60, E90, and E60 cultured under 3 kPa stress for two days and E60 cultured under 0 kPa stress for two days. Data information: Data represent the means ± SEM. n = 3. , *P < .05, **P < .01, ***P < .001; NS, not significant. T Illustration of force exertion on cultured dental follicle cells (DFCs). IF of RUNX2 and phospho-ERK1/2 (p-ERK1/2) upon force loading with 0 or 1.0 g/cm2 for 2 h or after force was removed and the cells cultured for an additional 1, 2, and 4 hours. V Relative IF expression levels of RUNX2 and p-ERK1/2 between groups Data information: Data represent the means ± SEM. n = 3. One-way ANOVA (Newman-Keuls test for post-hoc comparisons between two groups) for V , *P < .05, **P < .01, ***P < .001; NS, not significant. W Western blots of p-ERK1/2 and RUNX2 levels after 1.0 g/cm2 was applied for 2 h, with or without anti-integrin β1 antibody and ERK1 inhibitor (U0126). X Relative expression levels of p-ERK1/2, RUNX2, and Actin in (W). Data information: Data represent the means ± SEM. n = 3. Unpaired t-tests for M; One-way ANOVA (Newman-Keuls test for post-hoc comparisons between two groups) for X, *P < .05, **P < .01, ***P < .001; NS, not significant. \n",
      ".Figure 6A, B ISH of miniature pig canine frontal sections showing expression patterns of RUNX2 at embryonic day 60 (E60) and E90. (A', B') are magnifications of boxed regions in the corresponding figure panels. Data information: Scale bars = 50 µm. C Relative ISH expression levels of RUNX2 in epithelium (Epi.) and mesenchyme (Mes.) at E60 and E90. Data information Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests for C , *P < .05, **P < .01, ***P < .001. D, E ISH of miniature pig canine frontal sections showing expression patterns of β-catenin at E60 and E90. (D', E') are magnifications of boxed regions in the corresponding figure panels. Data information: Scale bars = 50 µm. F Relative ISH expression levels of β-catenin in epithelium (Epi.) and mesenchyme (Mes.) at E60 and E90. Data information: Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests , *P < .05, **P < .01, ***P < .001. G, H IF of Lef1 at E60 and E90. (G, H') are magnifications of boxed regions in the corresponding figure panels. Data information: Scale bars = 50 µm. I Relative IF expression levels of Lef1 in epithelium (Epi.) and mesenchyme (Mes.) at E60 and E90. Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests , *P < .05, **P < .01, ***P < .001. J FISH of β-catenin in E60 and E90 mandible slices and E60 mandible slices after 12 and 24 h in culture via RNAscope. Boxed regions of mesenchyme are magnified in the right panels. Epithelium is indicated with a yellow arrow. Data information: Scale bars = 50 µm. K Relative FISH expression levels of β-catenin in the nucleus and cytoplasm of epithelium and mesenchyme at E60, E90, and E60 cultured for 12 and 24 h. Data information: Data represent the means ± SEM. n = 3 for all experiments. One-way ANOVA for K, *P < .05, **P < .01, ***P < .001. L, M IF of β-catenin at E60 and E90; The yellow arrows indicate the localization of β-catenin in the nucleus. Data information: Scale bars = 50 µm. N Relative IF expression levels of β-catenin in the nucleus of epithelium and mesenchyme at E60 and E90. Data information: Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests , *P < .05, **P < .01, ***P < .001. O, P FISH of β-catenin in mandible slices infected with RUNX2 shRNA lentivirus (knockdown) or scrambled shRNA lentivirus (scramsh) via RNAscope. Data information: Scale bars = 50 µm. Q Relative FISH expression levels of β-catenin in the nucleus and cytoplasm of epithelium and mesenchyme in knockdown and scramsh groups. Data information: Data represent the means ± SEM. n = 3 for all experiments. Unpaired t-tests , *P < .05, **P < .01, ***P < .001. \n",
      "Chimeric GC‐A/G receptor containing the H+CYC domain of GC‐G acquires coolness‐evoked activityIn the chimeric GC‐A/G protein, the H+CYC domain of GC‐A was replaced by that of GC‐G (upper panel).Figure 2In the chimeric GC‐A/G protein, the H+CYC domain of GC‐A was replaced by that of GC‐G (upper panel). HEK‐293T cells expressing the indicated proteins were exposed to ambient temperature of 37 or 15°C for 20 min; then, intracellular cGMP concentration was measured. Data are mean ± SD from three experiments in triplicate. **P 0.01. \n",
      "Guanylyl cyclase (GC) subtype GC‐G activity is stimulated by cool temperatures, and the hinge (H) region and cyclase (CYC) domain are essential for its activation by coolnessCoolness stimulates intracellular cyclic guanosine monophosphate (cGMP) accumulation in HEK‐293T cells expressing GC‐G.Figure 1A Coolness stimulates intracellular cyclic guanosine monophosphate (cGMP) accumulation in HEK‐293T cells expressing GC‐G. Two days after transfection of empty vector or GC‐G‐encoding plasmid, cells were exposed to ambient temperatures (37 or 15°C) for 20 min and cellular cGMP concentration was measured. B Temperature dependence of GC‐G activity. Membrane preparations from HEK‐293T cells transfected with empty vector or GC‐G‐encoding plasmid were used to determine cyclase activity at the indicated temperatures for 20 min (asterisks denote values significantly elevated compared to 37°C). C GC‐G is the only murine transmembrane GC that can be stimulated by cool temperatures. Expression plasmids encoding FLAG‐tagged GC subtypes GC‐A to GC‐G were transfected into HEK‐293T cells. Two days after transfection, membrane protein fractions were prepared and used for cyclase activity assays at 37 or 15°C. Protein expression of each receptor GC was confirmed by Western blot (WB) analysis with anti‐FLAG antibody. D The H+CYC domain is critical for stimulation by coolness. The domain structure of full‐length (FL) GC‐G and its mutant variants is shown in the upper panel. A FLAG tag was added at the N‐terminus of each protein. The ΔECD+KLD mutant protein lacks amino acids 73-455 and 556-833; the ΔCYC variant lacks amino acids 834-1,003. The H+CYC mutant protein contains amino acids 806-1,100. ECD, extracellular domain; KLD, kinase‐like domain; SP, signal peptide; TM, transmembrane region. HEK‐293T cells expressing the indicated GC‐G constructs were incubated at 37 or 15°C for 20 min, then intracellular cGMP concentration was measured (lower panel). \n",
      "The hinge region is critical for activation of GC‐G by cool temperatures A Representation of the domain structure from different C‐terminal fragments of GC‐G analyzed for coolness‐induced activity (amino acids of GC‐G encoded in each expression plasmid are indicated).Figure 3A Representation of the domain structure from different C‐terminal fragments of GC‐G analyzed for coolness‐induced activity (amino acids of GC‐G encoded in each expression plasmid are indicated). B, C Protein expression and coolness‐stimulated cGMP synthesis of the above shown divergent C‐terminal fragments of GC‐G with intact or ablated hinge (H) region. The expression plasmids encoding the H+CYC domain or a deletion construct (Δ1-Δ2) tagged with a FLAG epitope were transfected into HEK‐293T cells. Two days after transfection, cells were exposed to the ambient temperatures (37 or 15°C) for 20 min and cellular cGMP concentration was measured (C). Protein expression of each construct was confirmed by Western blot (WB) analysis with anti‐FLAG antibody (B). D A recombinant protein containing the H+CYC domain of GC‐G acquires coolness‐evoked activity. Diagram of recombinant GST.GC‐G‐CYC protein containing the H+CYC domain of GC‐G (amino acids 849-1,078) compared to the domain structure of full‐length GC‐G (upper panel). The purified GST.GC‐G‐CYC protein was exposed to the indicated temperatures for 20 min, and GST.GC‐G‐CYC cyclase activity was measured (lower panel; asterisks denote values significantly elevated compared to 37°C). \n",
      ".Figure 7A, B IF of Lef1 in mandible slices at embryonic day 60 (E60) infected with RUNX2 overexpression lentiviral vector (overexpression) or overexpression control lentiviral vector (O-control). (A'-B') are magnifications of boxed regions in the corresponding figure panels. Dashed lines mark the epithelium of PC. C Relative IF expression levels of Lef1 in epithelium (Epi.) and mesenchyme (Mes.) of the overexpression and O-control groups. Data information: Data represent the means ± SEM. Scale bars = 100 µm. n = 3 for all experiments. Unpaired t-tests for C , **P < .01, ***P < .001; NS. not significant. D, E IF of Lef1 in E60 mandible slices subjected to 3 or 0 kPa stress for two days. (D', E') are magnifications of boxed regions in the corresponding figure panels. Dashed lines mark the epithelium of PC. F Relative IF expression levels of Lef1 in the epithelium and mesenchyme after applying 3 and 0 kPa pressure. Data information: Data represent the means ± SEM. Scale bars = 100 µm. n = 3 for all experiments. Unpaired t-tests for , *P < .05, **P < .01, ***P < .001; NS. not significant. G Western blots of Lef1, non-phospho-β-catenin, and RUNX2 after DFCs were infected with control lentiviral vector or RUNX2 overexpression lentiviral vector. (G') Relative expression levels between control vector and RUNX2 overexpression groups. Data information: Data represent the means ± SEM. Unpaired t-tests , *P < .05, **P < .01, ***P < .001; NS. not significant. H Western blots of Lef1, non-phospho-β-catenin, and RUNX2 after DFCs were infected with scrambled shRNA (scramsh) or RUNX2 knockdown shRNA (knockdown) lentiviral vectors. (H') Relative expression levels between scramsh and knockdown groups. I Western blots of nuclear non-phospho-β-catenin and Lamin B in DFCs of RUNX2 overexpression (O), control vector (V), RUNX2 knockdown (K), scramsh (S), compressed force (F; 1.0 g/cm2), and control (C; 0 g/cm2) groups. Sample were loaded in a gradient (7.5, 15, 30, and 60 μg) showing a linear relationship for each group. J Western blots of nuclear non-phospho-β-catenin and Lamin B in DFCs treated with IWR-1-endo. Relative expression levels were compared between force and no force groups and between overexpression and control groups with or without IWR-1-endo treatment. K Diagram illustrating the biomechanical stress regulation of Wnt/β-catenin signaling in the mesenchyme between the deciduous (DT) and permanent tooth (PT) via the integrin β1-ERK1-RUNX2-Wnt/β-catenin pathway. L Diagram illustrating the biomechanical stress-associated downregulation of RUNX2-Wnt/β-catenin pathway in the mesenchyme, inducing upregulation of Wnt signaling in the epithelium, which triggers PT development. \n",
      "Increase in intracellular calcium [Ca2+]i concentration in HEK‐293T cells co‐expressing GC‐G and CNGA3 in response to cool temperaturesHEK‐293T cells expressing CNGA3 alone (upper panel) or together with GC‐G (full‐length in the middle panel and H+CYC domain in the lower panel) were loaded with the Ca2+ indicator Fura‐2 to monitor intracellular [Ca2+]i concentration.Figure 5HEK‐293T cells expressing CNGA3 alone (upper panel) or together with GC‐G (full‐length in the middle panel and H+CYC domain in the lower panel) were loaded with the Ca2+ indicator Fura‐2 to monitor intracellular [Ca2+]i concentration. Cells co‐expressing GC‐G and CNGA3 showed a rapid increase in [Ca2+]i when the temperature was lowered from 37 to 15°C (middle and lower panel); cells only expressing CNGA3 are not responsive to cool temperatures (top panel). Removal of extracellular Ca2+ (−Ca2+; middle panel) completely suppressed the response to cooling. ΔF represents changes in the ratio of the fluorescence intensity of Fura‐2 at 340/380 nm excitation. The ratiometric Ca2+ responses are representative of 30 cells recorded from three experiments. \n",
      "Impaired coolness‐evoked ultrasonic vocalization (USV) in GC‐G‐KO mice A Reduced responsive rate of coolness‐stimulated USV in GC‐G‐KO pups.Figure 7A Reduced responsive rate of coolness‐stimulated USV in GC‐G‐KO pups. Pups were removed from their littermates and dams at postnatal (P) day 2, 4, and 6 and exposed to 15°C. USV was measured by use of the Anabat SD1 Bat detector. The number of responsive pups versus total number of pups tested for each WT and GC‐G‐KO group at different ages is indicated on the top of each bar. *P 0.05.B, C Total number of USV calls (B) and response latency (C) in GC‐G‐KO and WT pups at P2. The horizontal bars indicate the mean. *P 0.05; **P 0.01. \n",
      "Cool temperatures enhance dimerization/oligomerization of GC‐G but not GC‐AHEK‐293T cells were co‐transfected with plasmids encoding two differential epitope‐tagged (FLAG or Myc) versions of GC‐G or GC‐A, respectively.Figure 4A HEK‐293T cells were co‐transfected with plasmids encoding two differential epitope‐tagged (FLAG or Myc) versions of GC‐G or GC‐A, respectively. After 2 days, transfected cells were lysed with detergent and lysates were subjected to immunoprecipitation (IP) for 2 h with anti‐Myc antibody at two different temperatures (15 or 25°C), followed by Western blotting (WB) using anti‐FLAG antibody to determine the amount of dimerized/oligomerized GC proteins (upper panel). These approaches demonstrated that in contrast to GC‐A, homomeric protein/protein association of GC‐G seems to be enhanced by cool temperatures (upper panel). Cell lysates were also precipitated and immunoblotted with the anti‐Myc antibody, demonstrating that cooling does not interfere with immunoprecipitation of Myc‐tagged GC‐G via the anti‐Myc antibody (middle panel). Likewise, precipitation and immunoblotting with the anti‐FLAG antibody yielded bands of similar intensity for 15 and 25°C, respectively (lower panel).B Relative quantification of homomeric protein/protein association of GC‐G and GC‐A was performed by densitometric scanning. Each intensity value from the upper panel was subsequently normalized to the total protein levels as determined in the middle and the lower panel of Fig A (intensity of the immunoreactive bands obtained with the anti‐Myc or anti‐FLAG antibody, respectively). Relative dimerization level at 15°C was further calculated by normalizing to values obtained at 25°C; the latter were set as 1. Results are means ± SD of three experiments, **P 0.01 (compared to 25°C). \n",
      "Attenuation of coolness‐induced responses in the GG of GC‐G‐deficient (GC‐G‐KO) mice A–H In situ hybridization experiments with an antisense probe for c‐Fos on coronal sections through the GG of wild‐type (WT; left panel) or GC‐G‐KO (right panel) mouse pups (P0–P4) exposed to warm (35°C; A–D) or cool (22°C; E–H) ambient temperature for 2 h.Figure 6A-H In situ hybridization experiments with an antisense probe for c‐Fos on coronal sections through the GG of wild‐type (WT; left panel) or GC‐G‐KO (right panel) mouse pups (P0-P4) exposed to warm (35°C; A-D) or cool (22°C; E-H) ambient temperature for 2 h. Panels (C, D, G, H) show higher magnifications of the boxed areas in (A, B, E, F). At 35°C (A-D), no c‐Fos expression was detectable in the GG of both WT (A, C) and GC‐G‐KO (B, D) individuals. Upon exposure to 22°C, strong c‐Fos expression was observed in the GG of WT pups (E, G), whereas c‐Fos expression was barely detectable in the GG of GC‐G‐KO animals (F, H). The data shown are representative of 10 independent experiments. Pups originated from six different litters for each genotype. Scale bars: 200 μm in (A, B, E, F); 50 μm in (C, D, G, H).I Quantification of c‐Fos‐positive GG cells in WT and GC‐G‐KO pups on exposure to 22°C for 2 h. All stained cells on every section along the rostrocaudal extent of the GG were counted; the results shown are based on the above‐mentioned 10 experiments. The number of c‐Fos‐positive cells in the GG of a GC‐G‐KO mouse was determined relative to that in a concomitantly processed WT pup; the latter number was set to 100%. Data are mean ± SD (n = 11 in each genotype), **P 0.0001. J-N Calcium imaging of coronal sections through the GG of olfactory marker protein‐green fluorescent protein (OMP‐GFP)/GC‐G+/+ or OMP‐GFP/GC‐G‐KO pups (P1-P4). High magnification image (J) of a tissue slice through the GG of an OMP‐GFP pup with GG neurons labeled by intrinsic GFP fluorescence (GFP fluorescence was merged with the transmitted‐light channel). Cells analyzed are circled in blue (GFP‐negative) or red (GFP‐positive). NC, nasal cavity. Scale bar: 30 μm. (K-M) Representative ratiometric Ca2+ transients induced by cooling from 37 to 15°C in GFP‐negative non‐neuronal cells (K) and in GFP‐positive GG neurons from OMP‐GFP/GC‐G+/+ (L) and OMP‐GFP/GC‐G‐KO pups (M). The numbers in the bottom right hand corners are the number of cells with Ca2+ transient similar to what is shown in the respective graph (left) and total number of measured cells (right). (N) Quantification of coolness‐induced ΔF in GG neurons from OMP‐GFP/GC‐G+/+ and OMP‐GFP/GC‐G‐KO mice. Coolness‐induced ΔF was calculated by subtracting the baseline fluorescence ratio (340/380 nm) at 37°C from the peak fluorescence ratio (340/380 nm) measured at 15°C. For OMP‐GFP/GC‐G+/+, the 29 coolness‐responsive neurons (black bar) of all 54 analyzed neurons (gray bar) from five slices (obtained from different animals) were analyzed. For OMP‐GFP/GC‐G‐KO, the 50 analyzed neurons (shaded bar) from five slices (obtained from different animals) were analyzed. Data are mean ± SD, **P 0.01. \n",
      ".Figure 1A Domain structure of FAK. Important phosphorylation sites are indicated and the FAK region used in the cryo-EM study is boxed with a dashed line. B Negative stain image of 2D crystals containing the FERM-kinase region of FAK bound to a PI(4,5) P2 monolayer. Top right insert: Fourier transform of the 2D crystal image. Bottom right insert: 2.5x zoom of the main image. C Local resolution map of the refined Apo and AMP-PNP particle, as calculated with Blocres (Heymann & Belnap, 2007). D 14 FAK molecules, each containing a FERM and kinase domain are fitted into the EM maps obtained from the AMP-PNP dataset. Each molecule is in a different color. Middle: Membrane distal view; right: Membrane proximal view; left: zoomed view of central symmetric dimer from the membrane distal view with locations of FERM (F) and kinase (K) domains indicated. The equivalent figure for the Apo structure is shown in Fig. EV1d. E Linear oligomeric assembly of FAK formed by three symmetric FAK dimers in the AMP-PNP particle shown from the membrane distal view. FERM (F) and Kinase (K) domains are labelled. FERM and kinase of the same molecule are in the same color with the kinases shaded in a lighter tone. F Model of the FAK oligomeric assembly placed on a lipid bilayer. The membrane is placed by considering the KAKTLRK basic sequence in the FERM F2 lobe as main membrane binding site (Cai et al., 2008; Goni et al., 2014). \n",
      ".Figure 2A Closeup of the central symmetric dimer of the AMP-PNP particle from the membrane distal view. Interfaces within the symmetric dimer are formed intramolecularly between the FERM and kinase domains (F and K; F' and K'). Intermolecular interfaces are formed within the symmetric dimer between F and K', F' and K as well as K and K'. Intermolecular interactions that link symmetric dimers are formed between 2 FERM domains (F and F*; F' and F**). Coloring is as in Fig.1E. The FERM subdomains F1 and F3 (F2 is not visible from this view), kinase N- and C-lobes (KN, KC), AMP-PNP and the N- and C-termini (N, C) are labelled. B Superposition of W266 mediated FERM-FERM 3D-crystallographic dimers (Ceccarelli et al., 2006) (yellow; PDB: 2AEH) and the FERM-FERM contacts seen in the cryo-EM structure to link symmetric dimers (F-F*; F'-F**). The view is 90° rotated from panel (A). Subdomains and termini are labelled and disordered regions in the linker are shown as dashed line. \n",
      ".Figure 3A-C Details of intramolecular F-K (A) and intermolecular F-K' (B) and K-K' (C) interactions as seen in simulations restrained by EM maps. Side chains of main interacting residues are shown in stick representation and hydrogen bonds are shown as orange dashed lines. D The kinase is oriented with the ATP binding site facing the membrane. Key interactions between the kinase and the membrane are made by K621 and K627. The side chains of K621, K627 and K216 (the latter is within the KAKTLRK motif of a neighboring FERM domain) as well as AMP-PNP bound to the ATP binding site are shown in stick representation. E Electrostatic surface potential of the symmetric FAK dimer as seen from the membrane proximal view. Both, the FERM (KAKTLRK) and the kinase (K621, K627) have extensive positive charges on this surface for interaction with negatively charged PI(4,5) P2 lipids. The kinase domains are encircled by a yellow dotted line. \n",
      ".Figure 5Interactions between WT or interface mutant FAK with a PI(4,5) P2 containing membrane was probed using a pulldown assay with membrane coated silica beads. Plotted are bound protein relative to the binding for WT at the highest PI(4,5) P2 concentration. Mutant naming is as in Table 2. For WT the non-specific binding to a PC membrane is shown for comparison. Plotted are mean values from at least three independent experiments. Error bars: s.d. \n",
      ".Figure 4A Autophosphorylation as measured in a ELISA assay is shown for WT and interaction mutants in presence of control vesicles (PC; phosphatidylcholine) or PI(4,5) P2 containing vesicles. Plotted are mean values from two independent autophosphorylation reactions, each determined in duplicates (n=4). Mutant nomenclature is as shown in Table 2. B Catalytic turnover activity as measured in a coupled kinase assay is shown. Plotted are mean values from at least three independent experiments. C To determine whether autophosphorylation occurs in cis or trans, autophosphorylation was measured for WT or a kinase dead form of FAK (KD, contains K454R mutation) in presence or absence of a substrate deficient FAK mutant (Y397F) and/or PI(4,5) P2 vesicles. PI(4,5) P2 negative experiments contain control PC vesicles instead. Note that the scales of the two plots are different, i.e. autophosphorylation is increased in presence of PI(4,5) P2. Plotted are mean values from two independent phosphorylation reactions, each determined in duplicates (n=4). Left: Schematic of possible cis- and trans-autophosphorylation (red arrows) in mixtures of FAK-WT/FAK-Y397F and FAK-KD/FAK-Y397F. D Src phosphorylation of Y577 in FAK in presence of PC or PI(4,5) P2 vesicles as determined by an ELISA method. The plot on the left is obtained in presence Src containing only the kinase domain (Src254-536) and the plot on the right with Src containing additionally the SH3 and SH2 domains (Src84-536). Plotted are mean values from two independent phosphorylation reactions, each determined in duplicates (n=4). \n",
      ".Figure 6A FAK phosphorylation of WT or interface mutant FAK in SCC cells. Whole cell lysates were subjected to western blot analysis with anti-pFAK Y397, anti-pFAK Y576/577, anti-pFAK Y861, anti-pFAK Y925 and anti-FAK. Anti-GAPDH served as a loading control. The graph shows densitometric analysis of relative pFAK/FAK ratios (mean) from three independent experiments. Student's t-test was carried out to calculate the statistical significance. Error bars: s.d. * = p < 0.01, # = p < 0.05. For full blots see source data. B Downstream signaling in SCC cells expressing WT or interface mutant FAK. Whole cell lysates are analyzed by western blot analysis with the indicated antibodies. The graph shows densitometric analysis of relative phospho/total protein ratios (mean) from three independent experiments. Student's t-test was carried out to calculate the statistical significance. Error bars: s.d. * = p < 0.01, # = p < 0.05. For full blots see source data. C Src phosphorylation and interactions between FAK and Src or pSrc Y416 in SCC cells . FAK, Src or pSrc Y416 were immunoprecipitated from whole cell SCC lysates using anti-pSrc Y416, anti-FAK, anti-Src or non-specific IgG-agarose, followed by western blot analysis with anti-FAK, anti-pSrc Y416 and anti-Src. Anti-GAPDH served as a loading control. For full blots see source data. \n",
      ".Figure 9A FERM superposition of autoinhibited FAK (Lietha et al., 2007) (yellow, PDB: 2J0J) with a membrane bound FAK monomer as observed in the cryo-EM structure (green) shown from the membrane proximal view. The KAKTLRK membrane binding site on the FERM domain is colored red and membrane interacting residues K621 and K627 in the kinase domain are in cyan with side chains in space fill representation. B FERM superpositions of autoinhibited FAK (yellow) with a F-F* interacting FERM dimer bound via the KAKTLRK motifs (red) to a lipid membrane. Membrane bound kinase domains connected to the FERM domains F and F* are removed for clarity. C Schematic model for PI(4,5) P2 induced FAK activation in focal adhesions. Locally increased concentration of FAK upon focal adhesion targeting promotes formation of FERM mediated FAK dimers. Binding of FAK dimers via their KAKTLRK basic region (dark blue) to PI(4,5) P2 rich membranes in focal adhesions causes dissociation of the autoinhibitory interaction between the FERM and Kinase (KIN) domains (step 1). The released kinase domains reorient to interact via their own basic residues with the membrane (step 2). This aligns a large surface (F-K', K-K' and F'-K in Fig.2a) for formation of the symmetric dimer and exposes the autophosphorylation site for efficient trans-autophosphorylation across the symmetric dimer (step 3). Src is recruited to the autophosphorylation site and phosphorylates Y576 and Y577 in the activation loop (A-loop) of FAK to induce catalytic activation of FAK (step 4). Possibly an additional mechanism is involved in removing the kinase domain from the membrane to expose the Src phosphorylation sites, thereby potentiating the activating step 4 (see text). \n",
      ".Figure 8A FAK-WT, interface mutant FAK or FAK -/- SCC cells were resuspended in methylcellulose solution in growth medium on a layer of agarose and observed for 3D sphere formation. Images were taken from 10 random fields after nine days (representative images are shown) and the relative colony size (top graph) as well as the number of colonies (bottom graph) were assessed from three independent experiments. Scale bars, 200 μm. Student's t-test was carried out to calculate the statistical significance. Error bars s.d. * = p < 0.01, # = p < 0.05. B FAK-WT, interface mutant FAK or FAK -/- SCC cells were seeded on growth factor-reduced Matrigel in serum-free media. After 72 h, invasion towards a serum gradient was analyzed by staining the cells with calcein and taking images at 10 μm increments through the matrigel. Representative images of the invasion assay are shown. Scale bars, 200 μm. The graph shows relative invasion from five independent experiments. Student's t-test was carried out to calculate the statistical significance. Error bars, s.e.m. * = p < 0.01, # = p < 0.05. \n",
      ".Figure 7FAK-expressing SCC cells were grown on glass coverslips, fixed and stained with anti-FAK anti-Paxillin and DAPI Representative immunofluorescence images are shown. Scale bars, 20 μm. B FAK-expressing SCC cells were grown on glass coverslips, fixed and stained with anti-pSrc Y416 , anti-Paxillin and DAPI Representative immunofluorescence images are shown. Scale bars, 20 μm. The graph (B) shows the quantification of internalized active Src from three independent experiments. Student's t-test was carried out to calculate the statistical significance. Error bars, s.d. * = p < 0.01. C Subcellular fractionation of SCC cells expressing FAK-WT or interface mutant FAK. Whole cell lysates (WCL) and subcellular fractions were subjected to western blot analysis with anti-FAK, anti-Tubulin (marker for cytoplasmic fraction; C), anti-RCAS (marker for perinuclear fraction; PN) and anti-H4 (marker for nuclear fraction; N). The graph shows densitometric analysis of relative FAK amount of the FAK mutants in the different fractions from three independent experiments. Student's t-test was carried out to calculate the statistical significance. Error bars: s.d. * = p < 0.01, # = p < 0.05. For full blots see source data. \n",
      "C cells produce pro-invasive Extracellular Vesicles A.Figure 3A. Left: Cartoon showing the principle behind the Cre-carrying EVs and how recipient cells switch from DsRed+ to eGFP+ expression upon Cre-mediated excision of the DsRed locus. Centre: representative images, and right: quantification of T47DDsRed cells that converted to eGFP+ in the bottom wells of a transwell system in the presence of MDA-MB-231CFP;Cre;Control or MDA‑MB‑231CFP;Cre;HA-RASSF1C seeded in the upper wells. Scale bars represent 100 μm. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, **** p ≤ 0.0001. B. Left: averaged Nanoparticle Tracking Analysis displaying the size distribution of EVs isolated from media of MDA-MB-231 cells expressing Control (pcDNA3) or FLAG-RASSF1C plasmid. Inserts: transmission electron microscopy images of single EVs. Scale bars represent 200 nm. Right: quantification of total secreted EVs (96 h, top) or at indicated times (below), as measured by NTA. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, **** p ≤ 0.0001. C. Western blot analysis of EVs isolated from MDA-MB-231 cells transfected with Control (pcDNA3) or FLAG-RASSF1C with indicated antibodies. Equal amounts of protein were loaded after normalization via microBCA assay. D. Quantification of EVs isolated from MDA-MB-231 cells transfected with Control (pcDNA3), RhoA‑CA, RhoA‑DN, FLAG‑RASSF1C, FLAG-RASSF1C‑R199F, or FLAG‑RASSF1C + RhoA‑DN using Nanoparticle Tracking Analysis. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, **** p ≤ 0.0001. E. Normalized values of real‑time migration rates in xCELLigence plates of MDA-MB-231 cells treated with 20 ng/μl EVs derived from MCF7 cells expressing a Control, FLAG-RASSF1C, or EGFP-RhoA plasmid, left to migrate for 24 h. Statistics were performed with 2-way ANOVA (Dunnett's multiple comparisons test). F. Left: Representative images of Boyden chamber invasion assay in 3D matrigel of MCF7 cells, treated with 20 ng/μl EVs derived from MDA-MB-231 cells, transfected the same constructs used in (e). Right: Quantification of Boyden chamber invasion 24 hours after EV treatment. Scale bars represent 200µm. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, **** p ≤ 0.0001. G. Top: graphical representation of the functional (invasion) assay performed using MDA‑MB‑231 derived EVs on MCF7 recipient cells. Bottom: Boyden chamber invasion assay in 3D matrigel of EVs isolated from different steps of ultracentrifugation. EVs were collected from conditional medium produced by MDA-MB-231 expressing Control (pcDNA3) or FLAG-RASSF1C. The 2K (large EVs), 10K (medium EVs or MVs) and 100K (small EVs, including exosomes) pellets were collected and equal amount of EVs (normalized according to protein content) were incubated with MCF7 cells. MCF7 were evaluated for invasion after 24 hours. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, **** p ≤ 0.0001. \n",
      "C oncogene promotes mesenchymal-amoeboid transition A.Figure 1A. Top: graphs showing the change in cell diameter for MCF7 cells transfected with ZsGreen (left) or ZsRASSF1C (right) plasmids at 0 h and 12 h, connections indicate individual cell tracking. Bottom: example images diameter measurements (red line). Scale 20 μm. B. Single cell morphology of MCF7 and MDA-MB-231 cell lines, expressing or not a RASSF1C construct, cultured in 3D rat tail collagen I and analyzed for their rounded or elongated morphology 24 hours after seeding in 3D matrix. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. C. Confocal images of MDA-MB-231 cells transfected with ZsGreen or ZsRASSF1C grown in 3D-collagen matrix. Cells were imaged and stained with Phalloidin-568 (red), pMLCII/Alexa‑633 (false‑coloured green) or imaged for ZsGreen (false-coloured white). Scale 20 μm. D. Single cell morphology assay of MDA-MB-231 cells cultured in 3D-collagen and analysed for morphology 24 hours after transient knock-down using either a control sequence (NT, non-targeting) or a sequence targeting the RASSF1 locus. RASSF1 knock-down marks a reduction in rounded cells. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. E. Gelatine zymography assay shows downregulation of MMP-9 metalloprotease in MDA-MB-231 cells over‑expressing RASSF1C. F. 3D Matrigel Boyden chamber invasion with or without metalloproteases inhibitor GM6001 and 3D Hydrogel invasion of MDA-MB-231 control cells or cells over‑expressing RASSF1C (24 h). Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. G. Quantification of Boyden chamber invasion after 24 hours in 3D matrigel of MDA-MB-231 cells, transfected with ZsGreen (control) or ZsRASSF1C (RASSF1C) and treated with inhibitors against ROCK (Y27632, 10 μmol/L) or Rho (C3, 2 μg/ml). Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. H. Cartoon summarising intravital imaging experiments. I. Lung colonization of MDA-MB-231 CFP+ cells (MDA-MB-231CFP) stably expressing control or HA-RASSF1C in mammary gland tumors from 5 mice each, measured from intravital imaging time-lapse images. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. J. Average migration speed (expressed as μm/min) of n=30 randomly selected CFP positive cells in intravital images of Control and RASSF1C expressing MDA-MB-231 tumors. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. K. Quantification of CFP+ MDA-MB-231 cells displaying different types of morphology in vivo, expressed as average number of events, observed from a total of 30 positions in n=5 mice/group (Control or RASSF1C). Mesenchymal, amoeboid and hybrid morphology were scored manually based on diameter and circularity (distance between the two furthest points). The higher distance between cell edges was defined as mesenchymal morphology, while the lower distance was considered amoeboid morphology. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. \n",
      "C oncogene upregulates Rho/ROCKI/pMLCII signaling pathway via SRC A.Figure 2A. Western blot analysis of Rho-GTP pull-down assay in MDA-MB-231 and MCF7 cell lines transiently transfected either with control or Flag-RASSF1C plasmid. B. Rho-GTP pull-down binding assay in MDA-MB-231 control or RASSF1C over-expressing cells and co‑transfected either with SRC plasmids, siRNA against SRC or siNT (non-targeting sequence). Western blot analysis shows Rho activity (Rho GTP-double band) and pMCLII and pRhoGDI binding. The ratio below Rho-GTP Western blot shows differences in Rho-GTP protein expression, compared to control condition. Data information: For both Rho-GTP pull-downs and immunoprecipitation assays total proteins were used as loading controls. All data are from n=3 independent experiments. Data are analyzed by Student T-test, represented as mean ± SEM. *** p ≤ 0.001. C. Western blot analysis of proteins in MDA-MB-231 cells transiently transfected with siRNA targeting a control sequence (NT) or targeting RASSF1. GAPDH was used as a loading control. D. Immunoprecipitation from MDA-MB-231 transiently transfected with control (pcDNA3) or HA‑RASSF1C plasmid. Pull‑down was performed using a SRC or same species IgG antibody and blotted with indicated antibodies. E. Schematic cartoon recapitulating the proposed RASSF1C/SRC‑driven mechanism. F. Top, schematic of RASSF1C domain structure indicating putative mutations in the RA domain affecting RhoA activity. Bottom, immunoprecipitation of MDA-MB-231 transiently transfected with either control (DsRed), DsRASSF1C plasmid, DsRASSF1C-R197W or DsRASSF1C-R199F mutants. Pull‑down was performed using a SRC or same species IgG antibody and blotted with indicated antibodies. Data information: For both Rho-GTP pull-downs and immunoprecipitation assays total proteins were used as loading controls. All data are from n=3 independent experiments. Data are analyzed by Student T-test, represented as mean ± SEM. *** p ≤ 0.001. G. Western blot analysis of Rho-GTP pull-down assay in MDA-MB-231 cells transiently transfected with RASSF1C or its mutants (R197W, R199F). The ratio under Rho-GTP Western blot shows differences in active Rho protein expression, compared to control condition. Data information: For both Rho-GTP pull-downs and immunoprecipitation assays total proteins were used as loading controls. All data are from n=3 independent experiments. Data are analyzed by Student T-test, represented as mean ± SEM. *** p ≤ 0.001. H. Quantification of rounded versus elongated cells in single cell morphology assay in 3D‑collagen indicating the degree of mesenchymal-amoeboid transition of MDA-MB-231 cells when RASSF1C or the described mutants are expressed. I. Representative confocal images show elongated or rounded cells per field of view of MDA‑MB-231 cells transfected with DsRed, DsRASSF1C, DsRASSF1C-R197W or DsRASSF1C-R199F, grown in 3D‑collagen and stained with Phalloidin-568 (red) for F‑actin and pMLCII/Alexa 633. Scale bars represent 10 μm. \n",
      "C cells express cancer stemness features A.Figure 4A. Quantification of fluorescent intensity of CD133 and ALDH1 cancer stem cell markers in MDA‑MB‑231 control or RASSF1C transfected spheroids cultured on Matrigel matrix. Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. B. Confocal images of ZsGreen or ZsRASSF1C expressing MDA-MB-231 spheroids embedded in 3D rat tail collagen I. Cells were stained with NANOG (red). Zoom images show that RASSF1C expressing cells are also NANOG positive. Scale 100 μm. C. Western blot analysis of CD133 protein expression level in MDA-MB-231 cells treated with siNT or siRASSF1. D. Representative confocal images (upper row) and quantification of fluorescent intensity of ALDH1 expression in MDA-MB-231 cells expressing RASSF1C (black bar in the graph) or after siRNA treatment against RASSF1 (grey bar). Lower row shows merged images with DAPI (blue), ALDH1 (green) and RASSF1C (red). Scale bars represent 20 μm. Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. E. Quantification of the absolute number of NANOG positive cells (measured as average cell number/spheroid) per MDA-MB-231 ZsGreen (control) or ZsRASSF1C (RASSF1C) spheroids in 3D-collagen, treated with ROCK inhibitor (Y27632, 10 μmol/L) or Rho inhibitor (C3, 2 μg/ml). Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. F. Quantification of the absolute number of NANOG positive cells (measured as average cell number/spheroid) per MDA-MB-231 ZsGreen (control, con) or ZsRASSF1C (RASSF1C) spheroids in 3D-collagen, co‑transfected with either RhoA WT, catalytically active (CA) or dominant negative (DN) constructs. Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. G. Fold change in qRT-PCR of NANOG, OCT4 and SOX2 mRNA from MCF7 cells expressing CRISPR‑dCas9 or dCas9‑VP64 plus single guide RNA for the RASSF1C promoter (sgRASSF1C). Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. H. Quantification of the percentage of double positive pMLCII and NANOG cells in MCF7 cells expressing CRISPR-dCas9, dCas9-VP64, dCas9‑SKD plus single guide RNA for the RASSF1C promoter (sgRASSF1C). At least 300 cells were counted per experiment. Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. I. Quantification of number of invaded MCF7 cells in 3D matrigel, after incubation with EVs from MDA-MB-231 cells sorted for Greenhigh expression. MDA-MB-231 cells stably expressing a pQXCIP‑zsGreen or pQXCIP‑zsGreen-RASSF1C construct were sorted and EVs were harvested 24 h later, prior to seeding with MCF7 cells in Boyden chambers. Invasive potential of MCF7 cells was evaluated 24 h after incubation with EVs. Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. J. qRT‑PCR expression of NANOG, OCT4, SOX2 in MCF7 cells 24 h after exposure to 50 ng/μl EVs derived from MDA‑MB‑231 cells (treated as indicated). Data Information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001. \n",
      "C oncogene induces invasion and metastatic dissemination via EV transfer in vivo A.Figure 5A. Cartoon of in vivo experimental approach. MDA-MB-231CFP;Cre and reported T47DDsRed cells were co‑injected in one mammary gland of 10 mice per experiment. B. Left: representative confocal images of mammary tumor formed by the co-injection of T47DDsRed reporter cells with either MDA-MB-231CFP;Cre;Control or MDA-MB-231CFP;Cre;HA-RASSF1C. Scale 200 μm. Right: averaged values for all mice for the percent of the eGFP+ or CFP+ cells as part of the whole tumor. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001, **** p ≤ 0.0001. C. Representative 3 hour migration paths of either MDA-MB-231CFP;Cre;Control (Control, left) or MDA‑MB‑231CFP;Cre;HA-RASSF1C (RASSF1C, right) within a single intravital imaging field of tumors co‑injected with T47DDsRed reporter cells. D. In vivo migration speed of MDA‑MB‑231CFP;Cre;Control or MDA‑MB‑231CFP;Cre;HA-RASSF1C in a mixed tumor with T47DDsRed cells. Values are an average of the totals from 5 mice each, measured from intravital time-lapse images. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001, **** p ≤ 0.0001. E. The normalized in vivo motility of T47DeGFP and T47DDsRed reporter cells in mixed tumors. A Non‑parametric Mann-Whitney U test was used to derive statistical significance. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001, **** p ≤ 0.0001. F. Cartoon representing the Cre-LoxP model exploited to follow up amoeboid motility in recombined reporter (T47DeGFP) cells. G. Quantification of distinct morphology of T47DeGFP in tumors from observed from a total of 30 positions in n=5 mice/group (Control or RASSF1C). Mesenchymal, Amoeboid or Hybrid motility were scored manually Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001, **** p ≤ 0.0001. H. Quantification of the number of T47DeGFP metastatic events found in the lungs of mice with mammary gland tumors. Whole lung tile images were used for quantification, 6 of 10 μm sections each, 100 μm apart. Data information: All data are from n=3 independent experiments. Data are analyzed by Student T-test and represented as mean ± SEM.* p ≤ 0.05, ** p ≤ 0.01, *** p ≤ 0.001, **** p ≤ 0.0001. \n",
      " gene methylation as a prognostic biomarker in breast cancer patients A.Figure 6A. Schematic representation of the RASSF1 gene showing the domains in RASSF1A and C. Bars shaded dark and light green denote the CpG islands associated with RASSF1A and RASSF1C respectively. B. Kaplan-Meier plots of overall survival of 496 stage I and II breast cancer patients with different levels of promoter/CpG island methylation of RASSF1A, previously reported to be associated with outcome in breast cancer patients. P values were obtained from a log-rank test. C. Left: heatmap depicting methylation patterns across the RASSF1 gene. In light green, RASSF1-2γ CpG island is depicted; in dark green, RASSF1-1α CpG island is depicted. Right: Kaplan‑Meyer survival curves depicting interaction of RASSF1A and RASSF1C methylation to predict the outcome of early stage breast cancer patients. P values were calculated from a log-rank test. Color legend: yellow, low RASSF1A and low RASSF1C; green, low RASSF1A and high RASSF1C; red, high RASSF1A and low RASSF1C; grey, high RASSF1A and high RASSF1C. D. Model representing how RASSF1C expression in RASSF1A‑silenced breast cancer cells (MDA-MB-231) activates a signaling cascade involving SRC kinase, RhoGDI and culminates in RhoA activation and generation of amoeboid contractility and cancer stemness features. RASSF1C-driven amoeboid cancer stem cells contribute to alteration of the tumor microenvironment by rendering recipient cells (MCF7, T47D) more invasive and able to upregulate stem cell markers, due to EV-mediated communication. \n",
      "a,b) Survival of wild-type (Oregon R), yw, PGRP-LE112, PGRP-LC7454 and PGRP-LE112,PGRP-LC7454 flies after injection of wild-type L.figf1(a,b) Survival of wild-type (Oregon R), yw, PGRP-LE112, PGRP-LC7454 and PGRP-LE112,PGRP-LC7454 flies after injection of wild-type L. monocytogenes (a) or Δhly L. monocytogenes (b) at 28 °C. (c) Survival of wild-type, yw and PGRP-LE112 flies, as well as flies treated with RNAi targeting PGRP-LE with the hml-Gal4 driver (hml-Gal4>PGRP-LE RNAi) and PGRP-LE112 flies with hml-Gal4-driven PGRP-LE expression (PGRP-LE112,hml-Gal4>PGRP-LE), after injection of wild-type L. monocytogenes at 28 °C. NS, not significant; *, P 0.01 (Wilcoxon-Mann-Whitney test): P = 0.0006, 0.0043 or 0.0043, for yw versus PGRP-LE112, PGRP-LC7454 or PGRP-LE112,PGRP-LC7454, respectively (a); P = 0.0095, for yw versus RNAi targeting PGRP-LE (c). Data represent the average of four independent experiments with over 30 flies of each genotype examined at the same time. \n",
      "Survival of yw flies, PGRP-LE112 flies, flies treated with Atg5-directed RNAi (hml-Gal4>Atg5IR) and control flies carrying hml-Gal4 (hml-Gal4>GFP), after injection of wild-type L.figf3Survival of yw flies, PGRP-LE112 flies, flies treated with Atg5-directed RNAi (hml-Gal4>Atg5IR) and control flies carrying hml-Gal4 (hml-Gal4>GFP), after injection of wild-type L. monocytogenes (a), Δhly L. monocytogenes (b) or E. carotovora (c) into adult flies. *, P 0.01 (Wilcoxon-Mann-Whitney test): P = 0.0024, Atg5-directed RNAi versus control (a), P = 0.0007, PGRP-LE112 versus control (a); P = 0.0055, PGRP-LC7454 versus yw (c). Data represent the average of four independent experiments. \n",
      "a) Microscopy of hemocytes from third instar larvae cultured ex vivo and infected with wild-type L.figf2(a) Microscopy of hemocytes from third instar larvae cultured ex vivo and infected with wild-type L. monocytogenes. Hemocyte nuclei (filled arrowhead) and DNA of L. monocytogenes (open arrowhead) are visualized by DAPI staining (blue or white); the actin cytoskeleton is stained with rhodamine-labeled phalloidin (red). hml-Gal4>Atg5IR, expression of an inverted repeat sequence of Atg5 to induce RNAi against Atg5. Scale bars, 10 μm. (b) Intracellular wild-type L. monocytogenes (WT Lm) or Δhly L. monocytogenes (Δhly Lm) counted manually in DAPI-stained infected hemocytes. LE112,hml>LE, hml-Gal4-driven expression of PGRP-LE in PGRP-LE112 hemocytes; imd1, imd; RelE20, RelishE20; hml>Atg5IR, inverted repeat sequence described in a; w, hml>Atg1, hml-Gal4-driven expression of Atg1 in w hemocytes; LE112,hml>Atg1, hml-Gal4-driven expression of Atg1 in PGRP-LE112 hemocytes; PGRP-LE112+rap, PGRP-LE112 hemocytes plus rapamycin. *, P 0.001, compared with wild type (t-test). Data are representative of four experiments (a) or at least three independent experiments (b; error bars, s.d. of triplicate measurements). \n",
      "a) Growth of L.figf4(a) Growth of L. monocytogenes in S2 cells (S2) or S2 cells expressing PGRP-LE (S2-LE) transfected with double-stranded RNA (RNAi) specific for Atg5, Rel or Dif and dl (Dif-dl) and infected for 1.5 h with wild-type or Δhly L. monocytogenes, followed by 6 h of incubation in medium containing CuSO4 and gentamicin, quantified by plate assay of colony-forming units. *, P 0.001 (t-test). Data are representative of two independent experiments (error bars, s.d.). (b,c) Fluorescence confocal microscopy of the localization of PGRP-LE together with wild-type L. monocytogenes in S2 cells engineered to express YFP-PGRP-LE (YFP-LE) and infected for 0.5 h with wild-type L. monocytogenes (b) or Δhly L. monocytogenes (c), followed by 1 h of incubation in gentamicin-containing medium, then DAPI staining. Merge, YFP-PGRP-LE (green) and DAPI (magenta). Filled arrowheads indicate L. monocytogenes; open arrowhead indicates accumulation of YFP-PGRP-LE around the bacteria. Scale bars, 5 μm. Data are representative of three experiments. \n",
      "a) GFP-LC3 dots in S2 cells expressing GFP-LC3 alone or PGRP-LE and GFP-LC3, treated for 2 h with TCT (100 nM), highly purified DAP-type PGN from L.figf6(a) GFP-LC3 dots in S2 cells expressing GFP-LC3 alone or PGRP-LE and GFP-LC3, treated for 2 h with TCT (100 nM), highly purified DAP-type PGN from L. plantarum (DAP; 100 μg/ml) or lysine-type PGN from S. epidermidis (Lys). (b) Dot- or ring-shaped GFP-LC3 signals in hemocytes after infection with wild-type or Δhly L. monocytogenes or treatment with rapamycin (5 μM) or with TCT, DAP-type PGN or lysine-type PGN. *, P 0.001 (t-test). Data are representative of two independent experiments (error bars, s.d.). \n",
      " - ATF7IP represses the SETDB1-regulated exogenous provirus reporter and ERVs in mESCs A Atf7ip KO cells show increased expression of SETDB1-regulated ERVs and the provirus reporter, MSCV-GFP (EV1B), as evidenced by RT-qPCR analysis.Figure 1A Atf7ip KO cells show increased expression of SETDB1-regulated ERVs and the provirus reporter, MSCV-GFP as evidenced by RT-qPCR analysis. RNA expression was normalized to Hprt expression and is shown relative to the level in WT cells. Data are mean ± SD; n=3, technical replicates. B Atf7ip KO mESCs (TT#2-12) show decreased H3K9me3 at the LTR of the SETDB1-regulated reporter and the ERVs, as evidenced by Native ChIP followed by qPCR analysis. Gapdh was used as a negative control. Data are mean ± SEM; n=5, independent experiments. *P < 0.05 and **P < 0.01 by paired t-test. C SETDB1 cellular amount is mostly maintained in Atf7ip KO mESCs. Endogenous SETDB1-ATF7IP interaction is validated by anti-ATF7IP antibody co-IP experiment with anti-ATF7IP antibody. \n",
      "a) Dot- or ring-shaped GFP-LC3 (LC3 dot) signals in S2 cells expressing both PGRP-LE and GFP-LC3 under the control of an actin promoter (PGRP-LE GFP-LC3) or GFP-LC3 alone (GFP-LC3) after infection with wild-type or Δhly L.figf5(a) Dot- or ring-shaped GFP-LC3 (LC3 dot) signals in S2 cells expressing both PGRP-LE and GFP-LC3 under the control of an actin promoter (PGRP-LE GFP-LC3) or GFP-LC3 alone (GFP-LC3) after infection with wild-type or Δhly L. monocytogenes or after 1.5 h of incubation with 5 μM rapamycin (rap). -, no infection. (b) Colocalization of GFP-LC3 dots and wild-type L. monocytogenes in S2 cells, quantified by confocal microscopy. (c) Confocal microscopy of S2 cells expressing PGRP-LE and GFP-LC3 (green) infected with wild-type L. monocytogenes and stained with DAPI (magenta). Arrow indicates colocalization of GFP-LC3 and L. monocytogenes. (d) Immunoblot analysis of lysates of S2 cells left untreated (-) or infected with wild-type L. monocytogenes or treated with 5 μM rapamycin. (e-i) Ultrastructural analysis of S2 cells expressing PGRP-LE and GFP-LC3 and infected with wild-type L. monocytogenes. (e,f) Fluorescence microscopy (e) and electron microscopy (f) of cells expressing GFP-LC3 (green) and stained with DAPI (magenta). (g) Enlargement of a bacteria-containing vacuole from f. (h,i) Enlargement of the fields outlined in g. Arrows indicate double-membrane structure that surrounds the bacteria; arrowheads indicate an endoplasmic reticulum-like membrane. (j) Confocal microscopy of hemocytes infected with wild-type L. monocytogenes. Green, GFP-LC3; magenta, DAPI. Arrow indicates colocalization of GFP-LC3 and L. monocytogenes. (k) Dot- or ring-shaped GFP-LC3 signals in ex vivo-cultured hemocytes expressing GFP-LC3 (infection and treatment, horizontal axis). (l) Immunoblot analysis of lysates of hemocytes from third instar larvae cultured ex vivo and infected with L. monocytogenes or treated with rapamycin. Arrowhead indicates processed form of GFP-LC3. (m) GFP-LC3 dots in S2 cells expressing PGRP-LE and GFP-LC3 and treated with RNAi (below graph) and infected for 0.5 h with L. monocytogenes then incubated for 1 h in gentamicin-containing medium; dots quantified by confocal microscopy. Scale bars, 5 μm (c,j), 1 μm (e,f) or 500 nm (g). *, P 0.001 (t-test). Data are representative of three (a,k,m), two (b,e-i), six (c), four (d) or five (j,l) experiments (error bars, s.d. of triplicate measurements (a,b,l) or at least triplicate measurements (m)). \n",
      " - ATF7IP controls SETDB1 nuclear localization A IF analysis shows preferential cytoplasmic localization of SETDB1 in Atf7ip KO mESCs.Figure 2A IF analysis shows preferential cytoplasmic localization of SETDB1 in Atf7ip KO mESCs. Representative projected images are shown, and the quantitative analyses are shown in B-D. Scale bar: 10 µm. B SETDB1 signal in the nucleus that was determined by DAPI staining was calculated. The mean from three independent experiments is shown as a bar-graph with jittered points indicating the average % intensity of each experiment. Over 100 cells were analysed per sample per experiment. ***P < 0.001 by unpaired Student's t-test. C, D SETDB1 foci in the nucleus that was determined by DAPI staining were calculated. Violin plot for SETDB1's foci numbers from all the cells analysed in three independent experiments is shown in c. The mean from the three independent experiments is shown as a bar with jittered points indicating the average number of each experiment in D. Over 100 cells were analysed per sample per experiment. *P < 0.05 by unpaired Student's t-test. E, IF analysis shows preferential cytoplasmic localization of SETDB1 in Atf7ip KO mESCs. Representative projected images are shown. Arrows indicate the region used for line-plot analysis. Cells were treated with 10 ng/mL LMB or mock for 5 h before analysis. Scale bar: 10 µm. F The line-plot analysis for E. Relative intensities of SETDB1 signal were measured by Image J. G The quantitative analysis for E is shown. SETDB1 signal in the nucleus that was determined by DAPI staining was calculated. The mean from three independent experiments is shown as a bar-graph with jittered points indicating the average % intensity of each experiment. Over 100 cells were analysed per sample per experiment. NS: P > 0.05, **P < 0.01 versus \"Mock, WT\" group by Dunnett's test. \n",
      " - ATF7IP may have a role in SETDB1 import in HEK293T cells A ATF7IP KO HEK293T cells shows endogenous SETDB1 cytoplasmic localization phenotype.Figure 3A ATF7IP KO HEK293T cells shows endogenous SETDB1 cytoplasmic localization phenotype. Endogenous SETDB1 in HEK293T cells shows cytoplasm+nuclear localization profile. Scale bar: 10 µm. B Over-production of exogenous mouse SETDB1 induces endogenous SETDB1 nuclear localization in both WT and ATF7IP KO HEK293T cells. The representative data of two independent experiments is shown. Scale bar: 10 µm. C LMB treatment enhances SETDB1 nuclear accumulation in WT HEK293T cells, but no significant impact for ATF7IP KO HEK293T cells. Cells were treated with 10ng/ml of LMB for 5 h and then analysed with immunofluorescent staining. The representative data of three independent experiments is shown. Scale bar: 10 µm. D SETDB1 signal in the nucleus shown in C was calculated. The mean from three independent experiments is shown as a bar-graph ± SEM; n=3. Over 50 cells were analysed per sample per experiment. **P < 0.01 by Student's t-test (two-sided test). E 5 h of proteasome inhibitor bortezomib treatment do not have much impact on SETDB1 relative nuclear accumulation in ATF7IP KO HEK293T cells. The representative data of three independent experiments is shown. Scale bar: 10 µm. F SETDB1 signal in the nucleus shown in E was calculated. The mean from three independent experiments is shown as a bar-graph ± SEM; n=3. Over 50 cells were analysed per sample per experiment. Statistics comparison was only shown between without and with drug treatment in WT and ATF7IP KO cells. *P < 0.05 and **P < 0.01 by Dunnett's method. \n",
      " - N-terminal region of SETDB1 contributes to its cellular localization A HEK293T cells were transfected with V5-SETDB1 and either empty or 3xF-ATF7IP expressing vector.Figure 4A HEK293T cells were transfected with V5-SETDB1 and either empty or 3xF-ATF7IP expressing vector. Over 50 transfected cells analysed per sample were analyzed. The representative data of two independent experiments is shown. Scale bar: 10 µm. B HEK293T cells were transfected with V5-SETDB1 d1 mutant and either empty or 3xF-ATF7IP expressing vector. Over 50 transfected cells per sample were analysed. The representative data of two independent experiments was shown. Scale bar: 10 µm. The yellow arrowheads indicate the cells that show nuclear (and cytoplasmic)-localization of introduced exogenous mSETDB1. \n",
      ".Figure 1Immunoblot analysis of presenilin expression and endoproteolysis in cell lysates of untransfected HEK293/sw cells endogenously expressing presenilin and single cell clones overexpressing PS1 WT or the indicated PS1 mutants. Immunoblot analysis of total Aβ secretion and the production of different Aβ species (shown in low and high immunoblot exposures) in conditioned media of cell lines Mass spectrometry analysis of Aβ species immunoprecipitated from conditioned media of cell lines \n",
      " - ATF7IP-mediated SETDB1 nuclear localization enhances SETDB1 ubiquitination A Co-transfection with ATF7IP augments the K885 ubiquitination of SETDB1.Figure 5A Co-transfection with ATF7IP augments the K885 ubiquitination of SETDB1. HEK293T cells were transfected After the transfection, V5-SETDB1 was immune-purified with anti-V5 antibody and used for WB analysis. B ATF7IP regulates the ubiquitination of SETDB1 in mESCs. Endogenous SETDB1 was immune-purified from WT or Atf7ip KO mESCs cells and used for WB analysis. C LMB treatment increased the relative intensity of the upper ubiquitinated upper band of SETDB1 in Atf7ip KO mESCs. Cells were treated with 10 ng/mL LMB or mock for 5 h before analysis. D HEK293T cells were transfected with either 3xF- or 3xF-NLS-SETDB1. Over 50 transfected cells per sample were analysed. The representative data of two independent experiments is shown. Scale bar: 10 µm. E Nuclear localization itself promotes the ubiquitination of SETDB1. HEK293T cells were transfected with 3xFLAG- or 3xFLAG-NLS-SETDB1. After FLAG-IP, the purified SETDB1s were analyzed by WB. F, G IF analysis was performed with Atf7ip KO mESCs stably expressing 3xF- or 3xF-NLS-SETDB1. Representative images are shown in F, and the quantitative analysis is shown in G. Scale bar: 10 µm. Addition of NLS increases nuclear localization of SETDB1 in Atf7ip KO mESCs. The mean from three independent experiments is shown as a bar-graph with jittered points indicating the average % intensity of each experiment. Over 50 cells were analysed per sample per experiment. ***P < 0.001 by unpaired Student's t-test. H Nuclear localization promotes the ubiquitination of SETDB1 in Atf7ip KO mESCs. Atf7ip KO mESCs expressing 3xF- or 3xF-NLS-SETDB were used for FLAG-IP, and the purified SETDB1s were analyzed by WB. \n",
      ".Figure 2Immunoblot analysis of C99-PS1 NTF or full-length (FL) crosslinks (CL) of the indicated mutants. Samples that were not UV-irradiated were loaded to control for specificity. Changes in the ratio of crosslink efficiencies at positions T48 and L49 compared to corresponding controls (n = 4-5 biological replicates). Changes in the ratio of crosslink efficiencies at positions K54 and V44 compared to corresponding controls (n = 4-5 biological replicates). \n",
      ".Figure 4ELISA analysis of Aβ species in conditioned media of HEK293/sw cells overexpressing WT or mutant PS1 treated with RO7019009 or vehicle (DMSO) showing the secretion of Aβ38 (A), Aβ43 (B) and Aβ42 (C) compared to total Aβ (Aβ38 + Aβ40 + Aβ42 + Aβ43) (n = 5 biological replicates). Note that in some experiments Aβ38 generation was below the detection limit for some mutants. Data are presented as mean ± SD. \n",
      ".Figure 5Mass spectrometry analysis of Aβ species immunoprecipitated from conditioned media of HEK293/sw cells overexpressing WT or mutant PS1 treated with 500 nM RO7019009 or vehicle (DMSO). \n",
      ".Figure 3Structure of RO7019009. ELISA analysis of Aβ species in conditioned medium of HEK293/sw cells treated with RO7019009 or vehicle (DMSO). Aβ levels are shown relative to DMSO vehicle control (n = 4 biological replicates). Dose response curve of RO7019009 in HEK293/sw cells measured using the MSD sandwich-immunoassay (n = 3 biological replicates). Dose response curves for inhibition of Aβ42 generation and Notch cleavage by RO7019009 in different cell lines. Comparison of in vitro potencies for Aβ42 inhibition in HEK293/sw cells, IC50 = 14 nM (n = 3) (as in (C)) with those in H4/sw cells, IC50 = 5 nM (n = 7) and N2a/sw cells, IC50 = 2 nM (n = 6) together with that of Notch cleavage inhibition in a HEK293-based Notch1 luciferase reporter cell line, IC50 > 10 µM (n = 2) (all n numbers represent biological replicates). Left panel: Immunoblot analysis of total Aβ in conditioned media of HEK293/sw cells treated with RO7019009 or vehicle (DMSO). Total APPs levels were analyzed to control for normal APP secretion and equal sample loading. Right panel: Quantification of relative Aβ amounts in (E) (n = 2 biological replicates). Immunoblot analysis of Aβ species in conditioned media of RO7019009 treated HEK293/sw cells after separation by Tris-Bicine urea SDS PAGE; synthetic Aβ peptides were loaded as size markers. \n",
      ".Figure 1(A) Bar chart showing the relative abundance (signal intensity) of the phosphorylated tau species identified using an exploratory IP-MS approach using HT7 antibody on TBS-soluble control and Alzheimer's disease (AD) pools. Blue dashed line indicates the two phosphorylated tau peptides that underwent targeted IP-MS: mono-phosphorylated p-tau231 and double phosphorylated p-tau(231+235). (B) Box-and-whisker plot showing quantification of mono-phosphorylated p-tau231 in control (n=10) and AD (n=10) TBS-soluble fractions using a targeted IP-MS method (P >0.05, non-significant, ns, Mann-Whitney U test). (C) Box-and-whisker plot showing the quantification of double phosphorylated p-tau(231+235) in control (n=10) and AD (n=10) TBS-soluble fractions using a targeted IP-MS method. Data information: Box-and-whisker plots show the median and the 25th and 75th percentiles. Group differences determined using Mann-Whitney U test (****P <0.0001). All samples were run in singlicates. \n",
      ".Figure 2(A) Box-and-whiskers plot showing the levels of CSF p-tau235 in neurological controls (n=21) and biologically defined Alzheimer's disease (AD) cases (n=19). (B) ROC analysis indicating the diagnostic accuracy of CSF p-tau235 when discriminating AD from control. \n",
      ".Figure 6Immunoblot analysis of C99 V44Bpa crosslinking (CL) to PS1 WT or NTFs (circle) of normally endoproteolysed mutants in the presence of vehicle (DMSO) or RO7019009. Quantitation of crosslink efficiencies of C99 V44Bpa to the WT or mutant NTF in (A). Immunoblot analysis of C99 V44Bpa crosslinking to full-length (FL) PS1 (triangle) of endoproteolysis-deficient mutants and the corresponding M292D control in the presence of vehicle (DMSO) or RO7019009. Quantitation of crosslink efficiencies of C99 V44Bpa to the M292 or FAD mutant FL PS1 in (C). \n",
      ".Figure 3(A) Box-and-whiskers plot showing CSF p-tau235 concentrations in the different groups: amyloid-negative cognitively unimpaired (CU-) participants (n=50), participants across the Alzheimer's disease (AD) continuum (amyloid-positive cognitively unimpaired (CU+, n=32); amyloid-positive mild cognitive impairment (MCI+, n=20); AD (n=20) and non-AD cases (n=19). CSF p-tau235 is highly specific for AD pathology. Cut-off value for CSF p-tau235 positivity is displayed with black dashed line (19.92 pg/mL). (B) ROC analysis showing the high diagnostic accuracy of CSF p-tau235 discriminating AD from CU-, CU+, MCI+, and non-AD groups. (C) ROC analysis comparing AUC values of CSF p-tau235, p-tau231 and p-tau217 discriminating CU+ from CU-. CSF p-tau235 discriminating accuracy was statistically lower to that of CSF p-tau231, but not CSF p-tau217. (D) ROC analysis comparing AUC values of CSF p-tau235, p-tau231 and p-tau217 discriminating amyloid-negative cognitively impaired (CI-, non-AD) from amyloid positive cognitively impaired (CI+, MCI+ and AD). CSF p-tau235 performance was statistically similar to that of CSF p-tau217 and p-tau231. \n",
      ".Figure 5Trajectories of the different tau biomarkers in preclinical AD using a local weighted regression method (Loess curve). Changes on biomarkers levels in CSF are represented as z-scores using Aβ PET in centiloid scale (CL) as a proxy of disease progression. Abnormal biomarker levels were determined as two standard deviations (SD) above the mean. Incipient Aβ pathology positivity was determined as Aβ PET higher than CL 12 (Mila-Aloma et al., 2020). All samples were run in singlicates. \n",
      ".Figure 4(A) Box-and-whiskers plot showing CSF p-tau235 concentrations in A-T- participants and across preclinical Alzheimer's disease (AD, dichotomised as A+T- and A+T+). CSF p-tau235 was increased already in A+T-, which represents the early cases within preclinical AD. A prominent increase was observed from A+T- to A+T+ cases, the latter representing late preclinical AD cases. Cut-off value for CSF p-tau235 positivity is displayed with black dashed line (19.92 pg/mL). (B) ROC analysis comparing AUC values of CSF p-tau235, p-tau231 and p-tau217 discriminating A+T- from A-T-. Both CSF p-tau217 and p-tau231 a showed a statically higher accuracy discriminating A-T- from A+T- than CSF p-tau235 (C) ROC analysis comparing AUC values of CSF p-tau235, p-tau231 and p-tau217 discriminating A+T+ from A-T-. However, all three assays had statistically the same accuracy when distinguishing A-T- from A+T+. (D) ROC analysis comparing AUC values of CSF p-tau235, p-tau231 and p-tau217 discriminating A+T+ from A+T-. CSF p-tau235 demonstrated statistically superior diagnostic accuracy than CSF p-tau217 discriminating the two preclinical groups (A+T- and A+T+). In this scenario, no differences in diagnostic performance between CSF p-tau235 and p-tau231 were observed. Data information: Box-and-whisker plots show the median and the 25th and 75th percentiles. \n",
      ".Figure 1A Strains used in the study. Seven wine / European strains, and three other distant ones (all blue) were crossed to the S288C reference (yellow). B B Strategy for identifying suppression by standing variation. A temperature sensitive (TS) allele collection of ~1,100 partial loss-of-function mutants in the reference background (yellow) was crossed to the 10 wild yeast strains with potential suppressor alleles (blue), to produce large segregant populations selected to carry the TS allele. Each cross was performed in one or two biological and four technical replicates. The fitness of the resulting ~70,000 populations was measured at 26°C (permissive temperature) and 34°C (restrictive temperature). A subset of 38 candidate suppression events were used in bulk segregant analysis for linkage mapping of causal loci that display selection at restrictive temperature (black) but not permissive temperature (grey). D \n",
      ".Figure 2A A positive control region shows suppression signal. Average suppression score (y-axis) for TS alleles of query genes located on the left arm of chromosome XVI (TS allele index, sorted by chromosome coordinates, x-axis) across strains with a translocation that generates a duplication of the shaded area (blue markers), and strains without the translocation (grey markers). Dashed line: y = 0.75 (suppression cut-off in screen). B B Genotype and phenotype trees are concordant. Top: Hierarchical clustering (UPGMA) of the ten wild strains used in this study based on sharing segregating sites. Colours: global genetic cluster membership. Bottom: as top, but based on correlation distance between genetic suppression profiles. Strain abbreviations as in Figure 1A, with in addition: NC = NCYC110, UW = UWOPS87-2421, and 27 = 273614N. C C About a third of suppressed alleles can be explained by a deficiency in the reference strain that is not present in the majority of wild strains. The frequency (y-axis) of the number of wild strains that suppress a TS allele (x-axis) for the 187 alleles that could be suppressed by at least one strain, using a more lenient criterion (suppression > 0.5; no z-score cut-off) to avoid edge effects and winner's curse. D D Genetic suppression is consistent across different TS alleles of the same gene. Suppression score (colour scale) in crosses to different wild strains (x-axis) for TS alleles (y-axis) of GAB1 and NSE4 genes. E Genetic suppression is consistent across genes encoding members of the same complex. Strongest suppression score across TS alleles for a gene (colour scale, as in (D)) in crosses to different wild strains (x-axis) for genes (y-axis) that encode members of the GPI-anchor trans-amidase complex (bottom) or the nuclear condensin complex (top). The G GPI16 gene suppression was not estimated in the NCYC110 strain due to chromosome II copy number variation (\"X\"). \n",
      ".Figure 6(A) Stacked bar chart depicting the percentages of [231+/235+], [231+/235-], [231-/235-] and [231-/235+] cases in each preclinical group. Table below indicates the exact percentages of participants in each group (exact number of participants in parenthesis). Discordant cases with the sequential phosphorylation hypothesis [231-/+235] shown in grey. (B) Correlation between CSF p-tau235 and p-tau231 in the whole ALFA+ cohort (Spearman's rank correlation: rS = 0.80, P˂0.0001). Assay cut-offs were determined as the mean + 2 SD of the A-T- group (defining p-tau231 and p-tau235 positivity or negativity in each participant). Cut-off values are indicated in red (19.92 and 9.59 pg/mL for CSF p-tau235 and p-tau231 respectively) and displayed with black dashed lines, resulting in four quadrants, each of them representing the four different positive or negatively status for each biomarker. Discordant cases with the sequential phosphorylation hypothesis ([231-/+235]) showed on the lower right quadrant (2.35% of the total, 9 participants out 383). All samples were run in singlicates. \n",
      ".Figure 4A Mapping results for segregant pools involving the indicated TS alleles and wild strains. The change in S288C allele frequency between the meiotic progeny isolated at 26°C and 34°C is plotted by genomic coordinate. Causal suppressor genes are indicated for regions that show selection for the wild strain sequence. Genes in brackets have not been validated experimentally. B B Comparison of the change in reference allele frequency, either for suppressor loci for which a causal suppressor gene was validated (N=17), or for loci for which we were unable to validate a suppressor gene (N=10). Loci for which all experiments failed due to technical reasons were excluded from the analysis. Statistical significance was determined using a two-tailed Mann-Whitney's U test (** = p < 0.005). Boxes: first, second, and third quartiles; whiskers: 1.5 interquartile range away from the first and third quartiles. C A cartoon of the TORC2 signalling pathway, highlighting suppression of a TORC2 mutant (tsc11) by mutations in ORM2. D D Suppressor prediction within the chromosome XV QTL of rpn11 TS mutants. The functional information prioritisation score (y-axis) for genes in the suppressor region (x-axis) identified RPT4 and RPN8 as the two highest-scoring suppressor candidates. E E Experimental validation of RPN8 as the causal suppressor of the rpn11-8 TS mutant. Cultures of the indicated strains were diluted to an optical density at 600 nm of 0.1 and a series of ten-fold dilutions was spotted on agar plates and incubated for 2-3 days at 34°C. UWOPS = UWOPS87-2421. See also Figure EV4. \n",
      ".Figure 3A Example of the mapping results. Wild allele frequency in progeny of a cross of the gab1-1 temperature sensitive allele to the YJM975 strain (y-axis) along the yeast genome (x-axis) at the permissive 26°C (blue) and restrictive 34°C (TS allele loss-of-function, cyan). The allele frequency change between the two temperatures is used in mapping. Labels: selected loci in the cross. Blue regions: called suppressor loci. B B Suppressors are plentiful. The average number of suppressor loci per cross (y-axis) at a given allele frequency change cut-off (x-axis) with either the wild allele beneficial (blue) or the reference allele beneficial (yellow). Vertical line at an allele frequency change of 0.2: the cut-off used for calling suppressor loci. C Suppressors are reproducible across TS alleles. Allele frequency change of suppressor loci in crosses using different TS alleles of the same gene (x- and y-axis) crossed with the same wild strain. Colours: gene and wild strain combinations. D D Suppressors are reproducible across wild strains. Allele frequency change of suppressor loci at 34°C in crosses using the same TS allele and a different wild strain (x- and y-axis). Colours: TS alleles. \n",
      ".Figure 5A,B Suppression of nse3-ts4 and nse4-ts4 temperature sensitivity (A) and DNA damage sensitivity (B) by the NSE1 allele of UWOPS87-2421. Cultures of the indicated strains were diluted to an optical density at 600 nm of 0.1 and a series of ten-fold dilutions was spotted on agar plates and incubated for 2-3 days. UW = UWOPS87-2421. The plates shown in (B) were incubated at 30°C. C C A cartoon of the SMC5/6 complex. D An illustration of the various types of genetic interactions that have been observed between different alleles of NSE1 and NSE3. E E Recruitment of Smc6-FLAG by ChIP-qPCR at two known SMC5/6 binding sites (pericentromere of chromosome XIV and CEN3) and one negative control locus (arm of chromosome III) in G2/M arrested strains. Relative enrichment corresponds to the ratio of the signal after immunoprecipitation (FLAG) over beads alone, normalised to the ratio in wild-type cells at CEN3. Error bars: standard error of the mean of three independent experiments. Statistical significance was determined using an unpaired, two-tailed t-test (* = p < 0.05, ** = p < 0.005). E \n",
      ".Figure 1A Northern blots comparing GlmY and GlmZ levels in wild type strain Z8 and the ∆rapZ mutant Z28 under normal growth and GlcN6P starvation conditions. Both strains, which also carried a chromosomal glmS'-lacZ fusion, were treated with 60 μg/ml Nva-FMDP or H2O (\"mock\"). Samples were harvested hourly for Northern analysis and determination of β-galactosidase activity. Growth curves are shown in Appendix Fig S1. Blots were re-probed using a 5S rRNA specific probe to provide loading controls. B The purification profile of Strep-RapZ from the ∆glmS strain Z904 under GlcN6P replete and depletion conditions is shown (top). The cleared lysate (CL), flow through (FT), washing steps (W) and the elution fractions (E1-3) from StrepTactin affinity chromatography were separated on 12.5 % SDS-PAA gels and stained with Coomassie blue. Metabolites were extracted from E2 and analyzed by HILIC-MS/MS. The extracted ion chromatograms of the LC-MS analysis targeting GlcN6P (retention time 16.6 minutes) are shown below. The samples derived from purification of Strep-RapZ (panels i and ii) or Strep-KdpE (panels iii and iv) were analyzed with the SRM transition m/z 258.1 to m/z 97 in the negative ion mode. The identity of the metabolite detected in panel i was confirmed by adding chemically pure GlcN6P to a final concentration of 100 pg/μl (panel v). C SPR analysis addressing interaction of RapZ variants with GlcN6P and similar metabolites. The Strep-tagged proteins were captured onto a sensor chip and various concentrations of the respective metabolite (i.e. 100, 500, 1000, 2500 and 5000 nM) were injected using a single cycle kinetics approach. \n",
      ".Figure 4A β-Galactosidase activities of strains Z197 (wild type) and Z1118 (∆glmY ∆glmZ), which carry the chromosomal glmY'-lacZ fusion, were determined during growth. B Strains Z197 (wild type) and Z225 (∆rapZ) were transformed with the following plasmids expressing the mentioned sRNAs: pBR-plac (vector control = VC), pYG83 (glmY), pYG84 (glmZ) and pSD69 (gcvB). sRNA expression was induced with 1 mM IPTG and β-galactosidase activities were determined in the exponential growth phase. C To assess the impact of GlmY* and GlmZ on stimulation of QseE' autophosphorylation by RapZ, 1 μM QseE'-His10 was incubated with [γ-32P]-ATP in absence or presence of 5 μM Strep-RapZ and/or the sRNAs GlmY*, GlmZ and GcvB. In lanes 3-11 QseE' was co-incubated with RapZ as well as the indicated sRNAs provided at 0.5 μM, 1.25 μM and 2.5 μM. In lanes 14-16, QseE' was incubated with 2.5 μM of each sRNA without RapZ. Samples were removed 1 min after addition of [γ-32P]-ATP and separated on 12.5 % SDS-PAA gels, which were analyzed by phospho imaging. \n",
      ".Figure 5Northern blot experiments assessing the half-lives of GlmY* and GlmZ under normal growth and GlcN6P starvation conditions. Bacterial cultures were either treated with 100 μg/ml Nva-FMDP or H2O (mock). Transcription was stopped by rifampicin addition when cultures attained OD600 = 1.0 and samples were removed at indicated times for Northern analysis. A Analysis of GlmY* and GlmZ decay in the wild type strain Z8. B Analysis of GlmY* and GlmZ decay in the ∆rapZ mutant Z28. C Semi-logarithmic plots of GlmY* and full-length GlmZ decay for half-life determination. \n",
      ".Figure 2Reporter gene assays addressing expression of lacZ fusions under GlcN6P replete and depletion conditions. strains were grown in 96-well plates and exposed to various degrees of GlcN6P depletion elicited by Nva-FMDP. Cells were harvested at indicated times and the β-galactosidase activities were determined. Strains Z197 and Z360 were used, which harbor glmY'-lacZ and glmZ'-lacZ fusions, respectively. Reporter gene assays addressing expression of lacZ fusions under GlcN6P replete and depletion conditions. strains were grown in 96-well plates and exposed to various degrees of GlcN6P depletion elicited by Nva-FMDP. Cells were harvested at indicated times and the β-galactosidase activities were determined. Expression of glmY'-lacZ in strain Z197 and the ∆rapZ mutant Z225 is compared. Reporter gene assays addressing expression of lacZ fusions under GlcN6P replete and depletion conditions. strains were grown in 96-well plates and exposed to various degrees of GlcN6P depletion elicited by Nva-FMDP. Cells were harvested at indicated times and the β-galactosidase activities were determined. Strains Z190 and Z201 were addressed, which transcribe glmY'-lacZ either from the σ54-promoter or the σ70 promoter, respectively. Reporter gene assays addressing expression of lacZ fusions under GlcN6P replete and depletion conditions. strains were grown in 96-well plates and exposed to various degrees of GlcN6P depletion elicited by Nva-FMDP. Cells were harvested at indicated times and the β-galactosidase activities were determined. Strains Z190 and the ∆qseF mutant Z196 are compared, both of which transcribe the glmY'-lacZ fusion solely from the σ54 promoter. Reporter gene assays addressing expression of lacZ fusions under GlcN6P replete and depletion conditions. strains were grown in 96-well plates and exposed to various degrees of GlcN6P depletion elicited by Nva-FMDP. Cells were harvested at indicated times and the β-galactosidase activities were determined. Complementation experiment analyzing the requirement of rapZ for glmY expression in cells grown to exponential phase under standard conditions in flask cultures. Strains Z197 and the ∆rapZ mutant Z225 were used. Tested plasmids were pFDX4291 (vector control for pFDX4324 and pYG82 = VC1), pFDX4324 (rapZ), pYG82 (rapZquad), pBGG237 (vector control for pBGG164 = VC2) and pBGG164 (strep-rapZ). \n",
      ".Figure 3A BACTH assay addressing interaction of RapZ with QseF and QseE. The following plasmid combinations were tested using reporter strain RH785 (columns 1-6): pBGG352/pBGG349, pBGG353/pBGG348, pYG199/pBGG349, pYG246/pBGG348, pKT25/pUT18C (negative control), pKT25-zip/pUT18C-zip (positive control). , β-galactosidase activities are presented as mean ± SD. B SPR spectroscopy experiments addressing interaction of RapZ with the cytoplasmic part (aa 196-475) of kinase QseE and response regulator QseF. Strep-RapZ was captured onto a sensor chip and increasing concentrations of QseE'-His10 or QseF-His10 were injected. C To assess the role of RapZ for QseF activity, β-galactosidase activities were determined from strains Z196 (∆qseF) and Z1110 (∆qseF ∆rapZ) at indicated times during growth. Strains harbored the glmY'-lacZ fusion that is solely expressed from the σ54 promoter and the following plasmids: pKESK23 (VC = vector control; black and grey; note that activities are too low for display), pYG89 (qseF, green), pYG90 (qseF-D56E, purple), pYG93 (qseF-D56A, red). , β-galactosidase activities are presented as mean ± SD. D In vitro phosphorylation assays addressing autophosphorylation of 1 μM His10-tagged QseE' (aa 196-475) in presence of various concentrations of RapZ. Samples were removed following [γ-32P]-ATP addition at indicated times and separated on 12.5 % SDS-PAA gels, which were analyzed by phospho imaging. E Analysis of QseE' autophosphorylation (lanes 1-6) and phosphoryl group transfer to QseF (lanes 7-12) in absence or presence of 5 μM Strep-RapZ. To assess phosphoryl group transfer, 1 μM QseF-His10 was added to the assay. F To analyse the role of GlcN6P, 5 μM RapZ or the equivalent volume of buffer was pre-incubated with the indicated GlcN6P concentration for 5 min and subsequently 1 μM QseE'-His10 was added. Following an additional incubation for 5 min, [γ-32P]-ATP was added and the reactions were stopped after 1 min. \n",
      ".Figure 7Cartoon summarizing the current findings. RapZ, presumably in its tetrameric state, binds GlcN6P in its CTD. Upon GlcN6P scarcity, RapZ accumulates in its \"free\" form and activates phosphorylation of QseE/QseF by direct interaction. Activity of the TCS depends on lipoprotein QseG (Göpel & Görke 2018), suggesting that RapZ can only activate those kinases, which are contacted by QseG in the periplasm. QseF~P triggers glmY expression from its σ54 promoter augmenting levels of GlmY*, which subsequently sequesters RapZ into stable complexes. Consequently, RapZ is not available to trigger decay of sRNA GlmZ, which therefore activates synthesis of GlmS, replenishing GlcN6P. Sequestration also precludes RapZ from activating QseE/QseF, providing a negative feedback loop that adjusts GlmY amounts to the required level. GlcN6P releases GlmY* from RapZ, which is then free to promote GlmZ decay repressing glmS. \n",
      ".Figure 6Northern blot experiments (A-C) assessing GlmY* half-life under GlcN6P replete (+GlcN) and depletion conditions (-GlcN). Transcription was stopped by addition of rifampicin and samples were harvested at indicated times for Northern analysis. A Analysis of the ∆glmS strain Z1126. The GlcN6P depleted culture was split 8 min after rifampicin addition and one of the sub-cultures was resupplied with GlcN (indicated by arrow). B The ∆glmS ∆rapZ double mutant Z1127 was tested. C Semi-logarithmic plots of GlmY* decay for half-life determination. D EMSA experiments addressing the role of GlcN6P for GlmY*/RapZ interaction. Radiolabelled GlmY* was incubated with incremental concentrations of RapZ (left panel) or RapZ-CTD (right panel) in absence or presence of 7.5 mM GlcN6P. Binding reactions were separated on native PAA gels and analyzed by phospho-imaging. The RapZ/GlmY* complex is indicated by an arrow. E EMSA following incubation of GlmY* with 1200 nM RapZ in presence of various GlcN6P concentrations ranging from 0 (lane 2) to 8 mM (lane 10). The fraction of GlmY* remaining in the gel pocket is marked with an asterisk. The RapZ/GlmY* complex is indicated by an arrow. F EMSA following incubation of GlmY* with 1200 nM RapZ in absence or presence of 7.5 mM of the indicated metabolite. The RapZ/GlmY* complex is indicated by an arrow. \n",
      ".Figure 1(A) Ventricular weight to tibia length ratio (VW/TL) of littermate control (LC) or cardiomyocyte-specific NCoR1 knockout (CMNKO) mice. n=10:9:12:7. (B) Ventricular weight to body weight ratio (VW/BW) of LC or CMNKO mice. n=10:8:12:7. (C) Representative H&E staining of cross sections of left ventricles. Scale bar: 50μm. (D) Representative wheat germ agglutinin (WGA) staining of cross section of left ventricles. Scale bar: 10μm. (E) Quantification of cardiomyocyte size. n=11:10:8:8. (F) qRT-PCR analysis of hypertrophy-related genes in left ventricles. n=9:9:11:8. (G) Ejection fraction measured by echocardiography. n=9:7:14:10. Student's t test was used for statistical analysis. (H) Fractional shortening measured by echocardiography. n=9:7:14:10. Student's t test was used for statistical analysis. \n",
      ".Figure 2(A) Representative images of hearts from LC or CMNKO mice subjected to sham operation or AAC for 2 weeks. Scale bar: 0.5cm. (B) VW/BW of LC or CMNKO mice 2 weeks after sham operation or AAC. n=9:9:16:15. (C) Representative H&E staining of cross sections of left ventricles. Scale bar: 25μm. (D) Representative WGA staining. Scale bar: 10μm. (E) Quantification of cardiomyocyte size. n=10:10:8:9. (F) qRT-PCR analysis of hypertrophy-related genes in left ventricles. n=8:7:15:15. (G) Representative picrosirius red staining of cross sections of left ventricles. Scale bar: 50μm. (H) Quantification of fibrotic areas. n=11:10:10:9. (I) qRT-PCR analysis of fibrosis-related genes in left ventricles. n=8:7:15:15. (J) Ejection fraction measured by echocardiography. n=9:7:14:12. (K) Fractional shortening measured by echocardiography. n=9:7:14:12. (L) Lung weight to body weight ratio (LW/BW) of LC or CMNKO mice 2 weeks after sham operation or AAC. n=8:7:16:15. (M) Cumulative survival rate of LC or CMNKO mice subjected to AAC. n=12:10. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis. \n",
      ".Figure 3(A) Western blotting analysis of NCoR1 in neonatal rat ventricular myocytes (NRVMs) transfected with siControl or siNCoR1 for 3 days. siControl indicates control siRNA; siNCoR1, NCoR1 siRNA. (B) Representative immunofluorescence staining of α-Actinin in NRVMs transfected with siRNA for 48h and then treated with vehicle (H2O) or phenylephrine (PE) for another 48h. Scale bar: 50μm. (C) Quantification of the surface area of α-Actinin-positive NRVMs with or without NCoR1 knockdown. A total of 40 NRVMs was randomly chosen from 4 replicate coverslips for each group and used for statistical analysis. (D) qRT-PCR analysis of Acta1 and Nppa in NRVMs with or without NCoR1 knockdown. n=4:4. (E) Western blotting analysis of NCoR1-flag in NRVMs infected by control lentivirus (Control) or NCoR1-flag lentivirus (NCoR1-OV) for 4 days. (F) Representative immunofluorescence staining of α-Actinin in NRVMs infected with lentivirus for 48h and then treated with vehicle or PE for another 48h. Scale bar: 50μm. (G) Quantification of the surface area of α-Actinin-positive NRVMs with or without NCoR1 overexpression. A total of 30 NRVMs was randomly chosen from 3 replicate coverslips for each group and used for statistical analysis. (H) qRT-PCR analysis of Acta1 and Nppa in NRVMs with or without NCoR1 overexpression. n=4:3. Data represent three independent experiments. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis. \n",
      ".Figure 4(A) Representative immunofluorescence staining of α-Actinin in NRVMs transfected with siRNA for 48h and then treated with vehicle (H2O) or PE for another 48h. Scale bar: 50μm. siMEF2a indicates MEF2a siRNA. (B) Quantification of the surface area of α-Actinin-positive NRVMs with or without knockdown of NCoR1 and/or MEF2a. A total of 15-20 NRVMs was randomly chosen from 3 replicate coverslips for each group and used for statistical analysis. (C) Representative immunofluorescence staining of α-Actinin in NRVMs transfected with siRNA for 48h and then treated with vehicle or PE for another 48h. Scale bar: 50μm. siMEF2d indicates MEF2d siRNA. (D) Quantification of the surface area of α-Actinin-positive NRVMs with or without knockdown of NCoR1 and/or MEF2d. A total of 15-19 NRVMs was randomly chosen from 3 replicate coverslips for each group and used for statistical analysis. (E) qRT-PCR analysis of Acta1 and Nppa in NRVMs. n=4. Data represent three independent experiments. Student's t test was used for statistical analysis. \n",
      ".Figure 6(A) ChIP analysis showing enrichment of NCoR1 in MEF2 binding regions on the promoters of Acta1 and Nppa in ventricular samples. (B) ChIP analysis of enrichment of MEF2a on promoters of Acta1 and Nppa in ventricular samples. n=3:3. ChIP analysis of enrichment of HDAC4 (C) on promoters of Acta1 and Nppa in ventricular samples. n=4:4 for (C) ChIP analysis of enrichment of HDAC5 (D) on promoters of Acta1 and Nppa in ventricular samples. n=3:3 for (D ChIP analysis of enrichment of , acetylated-Histone 4 (ac-H4) (E) on promoters of Acta1 and Nppa in ventricular samples. , n=3:3 ChIP analysis of enrichment of ac-H3 (F) on promoters of Acta1 and Nppa in ventricular samples. n=3:3 (G) ChIP analysis of RNA Polymerase II enrichment on the transcriptional regions of Acta1 and Nppa in ventricular samples. n=3:3. (H) Working model of NCoR1 function in cardiomyocytes. TSS indicates transcription start site. Student's t test was used for statistical analysis. \n",
      ".Figure 5(A) Luciferase assays of Acta1 and Nppa promoters in HEK293FT cells transfected with NCoR1-Flag and/or MEF2a-HA or with empty plasmids. Student's t test was used for statistical analysis (B) Co-immunoprecipitation (Co-IP) analysis of NCoR1 and MEF2a in HEK293FT cells transfected with full-length NCoR1-Flag and MEF2a-HA. (C) Co-IP analysis of NCoR1 and truncated MEF2a in HEK293FT cells. Schematic illustration of MEF2a-HA construct is shown above the Co-IP results. (D) Co-IP analysis of MEF2a and truncated NCoR1 in HEK293FT cells. Schematic illustration of NCoR1-Flag construct is shown above the Co-IP results. (E) Luciferase assays of Acta1 and Nppa promoters in HEK293FT cells transfected with full-length MEF2a and domain-specific NCoR1. Student's t test was used for statistical analysis (F) Western blotting analysis of NCoR1-flag in NRVMs infected by control lentivirus (Control) or NCoR1 (1939-2453)-flag lentivirus [NCoR1(1939-2453)-OV] for 4 days. (G) Representative immunofluorescence staining of α-Actinin in NRVMs infected with lentivirus for 48h and then treated with vehicle (DMSO) or PE for another 48h. Scale bar: 50μm. (H) Quantification of the surface area of α-Actinin-positive NRVMs. A total of 30 NRVMs was randomly chosen from 3 replicate coverslips for each group and used for statistical analysis. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis (I) qRT-PCR analysis of Acta1 and Nppa in NRVMs. n=4:4. Data represent three independent experiments. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis \n",
      " Heat shock and recovery leading to thermotoleranceA.Figure 1A. Schematic diagram of the temperature shift procedure to render MCF7 cells into thermoresistant MCF7aTT cells. B. Sensitivity of MCF7 and MCF7aTT cells to heat shock at 45 oC as determined by trypan blue staining. The graph shows means +/- SD, n = 3. Numbers indicate p values. C. Lysate of MCF7 and MCF7aTT cells (rendered thermoresistant through the protocol described in Fig. 1A) were separated by sucrose density gradient centrifugation, and P/M ratios were determined by integration of monosomal and polysomal peak areas (bar graph, mean P/M ratios +/- SD, n = 6). Numbers above bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli. \n",
      " Proteomic response to heat stressA.Figure 2A. Heat map summarizing quantitative changes in 1554 proteins in heat stressed naïve MCF7 cells and in thermotolerant MCF7aTT cells. The color bar defines the fold changes in protein expression relative to protein levels of MCF7 cells at 37oC. B. Log2 fold change distribution of proteins in the indicated comparisons. The fractions of up- and downregulated proteins are highlighted in red and blue. The conditions compared are shown in the scheme on the top left. C. Heat map showing the enrichment of Gene Ontology functional terms in up- and downregulated proteins shown in (B). p-values were calculated based on the accumulative hypergeometric distribution (Zhou et al. 2019). D. Heatmap showing changes in the expression of chaperones and proteasome subunits in MCF7 cells during heat shock exposure and recovery (MCF7aTT) as determined by SILAC proteomics. E. Naïve MCF7 cells and MCF7aTT cells were exposed to the indicated stress conditions for 3 h, and cell viability was assessed by staining dead cells with trypan blue. The bar graphs represent the percentage of trypan blue excluding cells. (means +/- SD, n = 3, numbers above bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). \n",
      ".Figure 7(A) Western blotting analysis of RIDs of NCoR1 in left ventricular samples from mice infected with AAV9 GFP or AAV9 NCoR1-RIDs (1939-2453)-Flag. (B) VW/BW of mice 2 weeks after sham operation or AAC. n=7:7:12:12. (C) Representative H&E staining of cross sections of left ventricles. Scale bar: 10μm. (D) Representative WGA staining. Scale bar: 10μm. (E) Quantification of cardiomyocyte size. n=6:6:7:8. (F) qRT-PCR analysis of hypertrophy-related genes in left ventricles. n=8:8:11:11. (G) Representative picrosirius red staining of cross sections of left ventricles. Scale bar: 20μm. (H) Quantification of fibrotic areas. n=5:5:7:7. (I) qRT-PCR analysis of fibrosis-related genes in left ventricles. n=8:8:11:11. (J) Ejection fraction measured before, 2 weeks and 4 weeks after AAC. n=6:6. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis. (K) Fractional shortening measured before, 2 weeks and 4 weeks after AAC. n=6:6. Two-way ANOVA followed by Bonferoni post-tests was used for statistical analysis. \n",
      " activity on the solubility of polyubiquitylated proteinsA.Figure 3A. Whole lysates of MCF7 and MCF7aTT cells exposed to heat shock and/or proteasome (MG132, 20 μM, 1h) and HSP70 (VER-155008, 50 μM, 1h) inhibitors were prepared as described in Materials and Methods. Total lysate, soluble fractions and insoluble fractions were stained by SYPRO Ruby (left) and quantified (right). (means +/- SD, n = 3, numbers above the bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). B. The same total lysates as shown in (A) were assayed for the level of K48-linked polyubiquitin. Actin is shown as a reference. The levels of K48-linked polyubiquitin were quantified from multiple independent repeats. (means +/- SD, n = 5, numbers above the bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). C. The soluble and insoluble fractions shown in (A) were assayed for the level of K48-linked polyubiquitin, and polyubiquitylated proteins were quantified (blue line graph, means +/- SEM, n = 4, numbers above the bars indicate p values). The data for bulk protein obtained in (A) were overlaid onto the graph for comparison (red line, means +/- SEM, n = 3). \n",
      " Ubiquitylation of nascent/newly synthesized proteins during acute heat stress or exposed to the proteasome inhibitor MG132A.Figure 4A. Lysates from MCF7 cells exposed to the proteasome inhibitor MG132 (20 μM, 1 h) or to heat shock (42 oC, 1 h) were subjected to sucrose density gradient centrifugation. Monosomal and polysomal fractions were quantified from the UV traces at 254 nm. Monosomes eluting in fractions 4- 6 and polysomes in fractions 7 - 14 were quantified and polysome to monosome (P/M) ratios were calculated. Average ratios and the individual datapoints (n = 5) are displayed in a bar graph (means +/- SD). Numbers above the bars represent p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli. Gradient fractions were assayed for the presence of lysine (K) 48-linked polyubiquitylated proteins by immunoblotting with K48 linkage-specific antibodies. The line graph on the right is a quantification of K48-linked polyubiquitin across the gradients relative to the levels obtained in MCF7 cells maintained at 37 oC (means +/- SEM, n = 4). B. The same experiment as described in (A) but with MCF7 cells exposed to MG132 (20 μM, 1 h) or puromycin (100 μg/ml, 10 min). Means +/- SEM, n = 2. C. Lysate from MCF7 cells exposed to MG132 (20 μM, 1 h) or puromycin (100 μg/ml, 10 min) as indicated were subjected to immunoprecipitation with anti-puromycin antibodies, followed by assaying for co-precipitation of K48-ubiquitylated proteins. Actin is shown as a reference. \n",
      " activitiesA.Figure 5A. Similar experiment as described in (Figure 4A) with naïve MCF7 and thermotolerant MCF7aTT cells as indicated. means +/- SEM, n = 3, * p<0.05, **p<0.01 (calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). B. Same experiment as in (A) with thermotolerant MCF7aTT cells incubated with the HSP70 inhibitor VER-155008 (50 μM, 1h) or MG132 (20 μM, 1h). The signal obtained for ribosomal protein RPS19 and RPL7 are shown for reference, documenting residual polysomes despite the flat UV trace. Right panels are quantifications of K48-ub (means +/- SEM, n = 2), black vertical lines indicate splicing of lanes that were run on different gels due to limitations in lane capacity. The bar graphs represent the P/M ratios (means +/- SD, n = 3, numbers above bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli). C. Sucrose density gradient fractions described in (B) were assayed for partitioning of HSPA1 and HSPH1 by immunoblotting. HSPA1 data were quantified in a line graph (means +/- SEM, n = 4). D D. Cell lysate was spun through a 30% sucrose cushion to pellet polysomes. Nascent proteins were released from polysomes by dissociating 80S ribosomes with EDTA (see Appendix Figures S3). Soluble and insoluble fractions were prepared by centrifugation and assayed for the level of K48-linked polyubiquitin. The levels of K48-linked polyubiquitin were quantified (means +/- SD, n = 3). Immunoblots within black squares were run on the same gel but irrelevant lanes were removed as indicated by white lines. \n",
      " promotes proteasome recruitment to polysomesA.Figure 6A. Trypsin-like proteasome activity determined in fractions as described in Fig. 4A using an in vitro assay. Results are shown relative to the proteasome activity obtained in MCF7 cells maintained at 37 oC (means +/- SEM, n = 4). B. MCF7 cells were exposed to the proteasome inhibitor MG132 (20 μM) or 42 oC for 1h after which an equal amount of cell lysate was spun through a 30% sucrose cushion. Resulting polysomes were assayed for the levels of 26S proteasome subunits and HSPA1. C. Cell lysates from MCF7 cells maintained under the conditions indicated were fractionated by sucrose density gradient centrifugation. Polysomal fractions were subjected to immunoprecipitation with RPS19 antibody, and precipitated were analyzed by quantitative LC-MS/MS. The heat map represents normalize spectral abundance factors (NSAF). D. Total cell lysates were prepared from naïve MCF7 and thermoresistant MCF7aTT cells, and puromycin labeled peptides were pull down as illustrated in the scheme. Precipitates were assayed for the presence of K48-linked polyubiquitin, chaperones, and proteasome subunits by immunoblotting. E. Partitioning of the 20S proteasome subunit PSMB1 along the sucrose gradient under the indicated conditions was assessed by immunoblotting, black vertical lines indicate splicing of lanes that were run on different gels due to limitations in lane capacity. The line graph on the top shows trypsin-like proteasome activity in the same fractions (means +/- SD, n = 3). \n",
      " Model of the quality control of nascent/newly synthesized proteins and thermotoleranceUnder normal conditions (left panel), cells co-translationally fold newly synthesized proteins with high efficiency.Figure 9Under normal conditions (left panel), cells co-translationally fold newly synthesized proteins with high efficiency. A modest fraction of nascent peptides that cannot fold correctly will be ubiquitylated and targeted for degradation. Proteasomal targeting is aided by the HSPA1, which is present in sufficient amounts to prevent the aggregation of damaged nascent peptides that would impede proteasomal clearance. In response to acute heat shock (middle panel), HSPA1 becomes limiting and aggregation of ubiquitylated nascent polypeptides impedes their degradation by the proteasome. Limitation in HSP70 activity leads to inhibition of global protein synthesis through phosphorylation of eIF2α and mechanisms impinging on translation elongation. Simultaneously, activation of HSF1 leads to transcriptional induction and synthesis of HSPA1. In cells rendered thermotolerant through previous heat exposure and recovery (right panel), increased levels of HSPA1, HSPH1, and DNAJB1 provide increased activity of the HHD chaperone complex to facilitate rapid proteasomal clearance. As a result, eIF2α phosphorylation subsides, and cells maintain protein synthesis despite the environmental challenge. Tumors, including esophageal and oral squamous carcinomas appear to gain a survival advantage by upregulating this pathway. \n",
      " depletion on co-translational quality control in esophageal cancer cellsA.Figure 7A. Immunoblot of individual clones of KYSE150 cells in which HSPH1 was disrupted by CRISPR/Cas9 targeting (KYSE150HSPH1-/- cells). B. Effect of disrupting HSPH1 on stress sensitivity. KYSE150 and KYSE150HSPH1-/- cells were exposed to heat stress (45 oC) (means +/- SD, n = 3). C. KYSE150 and KYSE150HSPH1-/- cells were exposed to the indicated stress conditions and cell viability was assessed by staining dead cells with trypan blue. Graphs represent means +/- SD, n = 3, numbers above bars indicate p values calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli. D. KYSE150 and KYSE150HSPH1-/- cells, either naïve or subjected to the aTT temperature shifting scheme, were exposed to the indicated temperature and cell lysates were separated by sucrose density gradient centrifugation. K48-linked polyubiquitin was detected by immunoblotting of gradient fractions (left panel). Signals were quantified and displayed as a line graph (right panel, means +/- SEM, n = 2). The red boxes highlight the pronounced differences in polysomal polyubiquitylation between KYSE150 and KYSE150HSPH1-/- cells. E. Whole lysates of cells KYSE150 and KYSE150HSPH1-/- cells exposed to the indicated temperature conditions (aTT scheme and heat shock) were fractionated into soluble and insoluble fractions as described in Materials and Methods. Insoluble fractions were assayed for the level of K48-linked polyubiquitin (left panel). Total levels of HSPA1 and HSPH1 are shown for reference. K48 ubiquitin signals were quantified and displayed in a bar graph (means +/- SD, n = 4). F. Lysates of cells KYSE150 and KYSE150HSPH1-/- cells exposed to the indicated temperature conditions (aTT scheme and heat shock) were obtained and polysomes were isolated by centrifugation through a 30% sucrose cushion. Whole cell lysates (left panel) and the polysomal pellets (right panel) were assayed by immunoblotting for the levels of HSPH1, HSPA1 and proteasome subunits. \n",
      " in human carcinomasA.Figure 8A. Representative micrographs of tissue sections derived from oral squamous cell carcinomas (OSCC) and esophageal squamous cell carcinomas (ESCC) stained with the indicated antibodies by immunohistochemistry (IHC). Top panels show routine H&E staining. Black squares point to regions magnified in the adjacent panels to the right. B. IHC staining of HSPA1 and HSPH1 was performed for the indicated numbers of normal, OSCC and ESCC tissues. Staining intensity was scored as described in Materials and Methods. C. Kaplan-Meier plots showing the correlation between HSPH1 mRNA expression level and the survival of patients with ESCC. The expression data was obtained from and visualized with KM Plotter (www.kmplot.com, (Nagy et al, 2018) using the Pan Cancer algorithm at default settings. D. KYSE150 and KYSE150HSPH1-/- cells were injected into nude mice, and xenograft tumor growth was monitored over time (left panel; mean tumor volume +/- SEM, n = 4 with 5 animals per n). Mean tumor weights at the end of the experiments were determined (means +/- SD, n = 4 with 5 animals per n, number above the bar graph represent p values, * p<0.05, **p<0.01 (calculated with the two-stage linear step-up procedure of Benjamini, Krieger and Yekutieli)). E. Macroscopic images of representative tumors derived from KYSE150 and KYSE150HSPH1-/- cells (shown are tumors from two of the four independent experiments). \n",
      ": Blood gene signatures following infant vaccination.Figure 3a) Volcano plot highlighting differentially expressed genes (DEGs, false-discovery rate [FDR] <0.01; red upregulated and blue downregulated) at each study time point versus pre-vaccination (4 months of age) 4 hours post-vaccination (719 DEGs, n=28). P-values were obtained from the moderated t-statistic, after adjustment for multiple testing (Benjamini and Hochberg's method). The top 10 genes, ranked by FDR, are labelled. Volcano plot highlighting differentially expressed genes (DEGs, false-discovery rate [FDR] <0.01; red upregulated and blue downregulated) at each study time point versus pre-vaccination b) Same as (a) but 24 hours post-vaccination (5553 DEGs, n=31) Volcano plot highlighting differentially expressed genes (DEGs, false-discovery rate [FDR] <0.01; red upregulated and blue downregulated) at each study time point versus pre-vaccination c) Same as (a) but 3 days post-vaccination (159 DEGs, n=30) Volcano plot highlighting differentially expressed genes (DEGs, false-discovery rate [FDR] <0.01; red upregulated and blue downregulated) at each study time point versus pre-vaccination d) Same as (a) but 7 days post-vaccination (6 DEGs, n=36). e) Modular signature induced following infant vaccination. Enriched modules (FDR <0.001) are displayed. Segments of the pie charts represent the proportion of upregulated (red) and downregulated (blue) genes (absolute fold-change > 1.25). Enrichment p-values were derived from a hypergeometric test, after adjustment for multiple testing (Benjamini and Hochberg's method). \n",
      ": Plasma proteins that differed in concentrations following vaccination in the test compare with the control vaccine group.Figure 5False-discovery rate (FDR) are displayed above and were derived from a two sample Wilcoxon rank sum test (Benjamini-Hochberg correction). Vertical lines represent the interquartile range around the median. The red dashed horizontal line is upper limit of the normal CRP reference interval (0-5mg/l). The number of individuals in each group is display in the x-axis. \n",
      ": Fever incidence within 24 hours of study vaccines.Figure 2a) Kaplan-Meier to first fever (≥38°C) episode within 24 hours of receiving vaccines administered at 4 months of age; control n=88 and 4CMenB n=89. b) Principal component (PC) analysis of RNA-sequencing data (14837 genes, n= 253) from all study time points. The lines connect participant's pre- and post-vaccination samples. The ellipses are the two-dimensional (PC1 and PC2) 95% confidence intervals for each study time point. A contribution plot (top right) displays the genes contributing most to PC1/PC2, i.e. 0.15 implies 0.15% contribution of that variable to the principal components displayed. c) Plotted are the CIBERSORTx neutrophil fractions from whole blood RNA-sequencing data, with median and interquartile range. P-values were determined from a two sample Wilcoxon rank sum test. The number of individuals in each group is display in the x-axis. d) Spearman's rank correlation between neutrophil counts measure by full blood counts (n= 142) and those estimated by CIBERSORTx, using the LM22 signature matrix. \n",
      ": Differential gene regulation between vaccine groups.Figure 4a) Top genes differentially regulated between vaccine groups 4 hours post-vaccination. Note, only genes differentially expressed in either study group (concomitant 4CMenB or control vaccines alone) were included in intergroup analysis. The plotted lines are the LOESS (locally estimated scatterplot smoothing) regression curves with the 95% confidence intervals in grey. The FDR was derived by comparing fold-changes in gene expression in the control group with the test group, from baseline to the time point designated, and is reported if statistically significant (FDR <0.05). P-values were obtained from the moderated t-statistic, after adjustment for multiple testing (Benjamini and Hochberg's method). Pre-vaccination samples n= 125, 4 hour samples n= 28, 24 hour samples n = 31, 3 day sample n= 30, 7 day samples n= 36. The expression E value, are the gene expression value derived from the voom-limma workflow (Law et al., 2014). Note, only genes differentially expressed in either study group (concomitant 4CMenB or control vaccines alone) were included in intergroup analysis. The plotted lines are the LOESS (locally estimated scatterplot smoothing) regression curves with the 95% confidence intervals in grey. The FDR was derived by comparing fold-changes in gene expression in the control group with the test group, from baseline to the time point designated, and is reported if statistically significant (FDR <0.05). b) Same as (a) but top genes differentially regulated between vaccine groups 24 hours post-vaccination. Note, only genes differentially expressed in either study group (concomitant 4CMenB or control vaccines alone) were included in intergroup analysis. The plotted lines are the LOESS (locally estimated scatterplot smoothing) regression curves with the 95% confidence intervals in grey. The FDR was derived by comparing fold-changes in gene expression in the control group with the test group, from baseline to the time point designated, and is reported if statistically significant (FDR <0.05). P-values were obtained from the moderated t-statistic, after adjustment for multiple testing (Benjamini and Hochberg's method). c) Same as (a) but top genes differentially regulated between vaccine groups 3 days post-vaccination. Note, only genes differentially expressed in either study group (concomitant 4CMenB or control vaccines alone) were included in intergroup analysis. The plotted lines are the LOESS (locally estimated scatterplot smoothing) regression curves with the 95% confidence intervals in grey. The FDR was derived by comparing fold-changes in gene expression in the control group with the test group, from baseline to the time point designated, and is reported if statistically significant (FDR <0.05). P-values were obtained from the moderated t-statistic, after adjustment for multiple testing (Benjamini and Hochberg's method). d) Same as (a) but top genes differentially regulated between vaccine groups 7 days post-vaccination. \n",
      ": Selection of top differentially expressed genes identified in the mouse study.Figure 7Boxplots represent the median with interquartile ranges. False-discovery rate (FDR) are displayed above and were derived from a two sample Wilcoxon rank sum test (Benjamini-Hochberg correction). N = 6 mice per vaccine group. \n",
      ": Comparison of blood gene profiles of infants who experienced post-vaccination fever with those who remained afebrile.Figure 6a, b) Agreement plot. Red = differentially expressed (DE) in febrile infants, blue = DE in afebrile infants, purple =DE in febrile and afebrile infants. c) Exemplar genes differentially expressed at 4 hours and 24 hours, respectively, between infants who experienced a fever within 24 hours of vaccination and afebrile infants. The plotted lines are the LOESS (locally estimated scatterplot smoothing) regression curves with the 95% confidence intervals in grey. The false-discovery rate (FDR) was derived by comparing fold-changes in gene expression between infants who experienced post-vaccination fever and those who remained afebrile. P-values were obtained from the moderated t-statistic, after adjustment for multiple testing (Benjamini and Hochberg's method). Pre-vaccination samples n= 125, 4 hour samples n= 28, 24 hour samples n = 31, 3 day sample n= 30, 7 day samples n= 36. d) Performance of predictive model built with sparse distance weighted discrimination (sdwd) algorithm to predict fever following concomitant 4CMenB vaccine. e) Variable importance score of the features from the sdwd model. f) Expression levels (Limma E value) of the top three transcripts from the sdwd model in the individuals that develop fever (n=33, red circles) and those that don't develop fever (n=21, blue circles); G, H Support Vector Regression (SVR) performance of model to predict post-vaccination MenB-specific SBA titres, g) training dataset (n=36), h) performance of model in the test dataset (n=9) , i) the top 5 genes ranked by importance from the SVR model. \n",
      ": Mouse temperatures following administration of purified preparations of Gram-negative bacteria outer membrane components.Figure 8Boxplots depicting the change in temperature from baseline to 24 hours after the second 1μg dose of each of a variety of Escherichia coli lipopolysaccharide (LPS) and peptidoglycan (PGN) preparations, with and without alum, and combinations compared with 1/5 of the human dose of 4CMenB. Boxplots represent the median with interquartile ranges. Vaccine group medians at each timepoint were compared with baseline for that vaccine using a Wilcoxon signed-rank test. * < 0.05, ** < 0.01, **** < 0.0001. N= 6 mice per vaccine group. \n",
      "A regulates gene expression specifically in ESC+2i A.Figure 2A. Volcano plot showing fold change (log2 values) in transcript level of ESC+2i and ESC+serum with two consecutive treatments with siRNA-control and siRNA-Baz2a, each one lasting for 4 days. Gene expression values of three replicates were averaged and selected for 1.5 fold changes and P < 0.05. Statistical significance (P-values) was calculated using R package DEseq2. B. Validation by qRT-PCR of genes regulated by BAZ2A in ESC+2i but not in ESC+serum (upregulated genes in ESC+2i upon BAZ2A-KD are labeled in blue, downregulated genes are in red). Nanog, Rex1 and Actin B (ActB) are shown as genes not regulated by BAZ2A. mRNA levels were normalized to Rps12 mRNA and to ESCs transfected with siRNA-Control. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (* < 0.05, ** < 0.01, *** < 0.001). Data without P values are statistically non significant. C. Top 10 biological process gene ontology terms as determined using DAVID for genes regulated by BAZ2A in ESC+2i and ESC+serum. \n",
      "A is required for proliferation and differentiation of ESC+2i A.Figure 1A. BAZ2A expression is higher in ESC+2i than in differentiated cells (neural progenitors, NPC). Left panel. BAZ2A mRNA levels were measured by qRT-PCR and normalized to Rps12 mRNA and to ESC+2i. Average values of three independent experiments. Error bars represent s.d. and statistical significance (P-values) was calculated using the paired two-tailed t-test (*** < 0.001). Right panel. Western blot showing BAZ2A protein levels in ESC+2i and NPC. Tubulin is shown as a protein loading control. B. BAZ2A associates with chromatin of ESC and NPC. Chromatin-bound (Chrom.) and soluble (Sol.) fractions of equivalent cell number of ESC+2i and NPCs were analyzed by western blot for BAZ2A levels. Total, total protein. Tubulin and histones are shown as loading and fractionation control. C. siRNA-knockdown efficiency of BAZ2A shown by qRT-PCR and western blot. BAZ2A mRNA levels were measured by qRT-PCR and normalized to Rps12 mRNA and to each ESC line. Average values of three independent experiments. Error bars represent s.d. and statistical significance (P-values) was calculated using the paired two-tailed t-test (**** < 0.0001). Right panel. Western blot showing BAZ2A protein levels. Tubulin is shown as a protein loading control. D. BAZ2A knockdown affects proliferation of ESC+2i but not of ESC+serum. Data represent relative cell numbers after 3 days of siRNA treatment and were normalized to ESC transfected with siRNA-Control. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (** < 0.01; ns, non-significant). E. BAZ2A is expressed at similar levels in both ESC+2i and ESC+serum. Left panel. BAZ2A mRNA levels were measured by qRT-PCR and normalized to Rps12 mRNA and to ESC+2i. Average values of three independent experiments. Error bars represent s.d. Right panel. Western blot showing BAZ2A protein levels in ESC+2i and ESC+serum. Tubulin and histone H3 are shown as a protein loading controls. F. BAZ2A is required for the differentiation of ESC+2i but not of ESC+serum. Representative images of alkaline phosphatase staining of ESC and cells after 3 days of differentiation upon culture in completed medium containing 10% serum and in the absence of LIF. G. Quantification of differentiated cells. Values represent relative number of differentiated cells from three independent experiments relative to control cells. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (*** < 0.001; ns, non-significant). H. Western blot showing BAZ2A protein levels in two BAZ2A-KO ESC lines obtained via CRISPr/Cas9 directly in ESC+serum. PARP1 is shown as a protein loading control. I. Cell proliferation is not affected in BAZ2A-KO ESC+serum. Data represent relative cell numbers 2 days culture starting with the same number of cells. Average values of four independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns, non-significant). J. Representative images of wt and Baz2a-KO ESC+serum before and after transition in 2i conditions. \n",
      "A limits spurious long-distant contacts in ground-state ESCs A.Figure 6A. Hi-C contact matrices for chromosome 1 of ESC+serum, ESC+2i/siRNA-control and ESC+2i/siRNA-Baz2a showing the presence of strong long-distance contacts corresponding to regions in A compartment bound by BAZ2A in ESC+serum. Cells were subjected to two consecutive treatments with siRNA-control and siRNA-Baz2a, each one lasting for 4 days. Eigenvector values are shown. Increase in far-cis contacts in ESC+2i upon BAZ2A-KD are highlighted with a rectangle. B. Relative contact frequencies over genomic distances in ESCs treated with siRNA-control and siRNA-BAZ2A. Statistical significance (P-values) was calculated using the Kolmogorov-Smirnov test. C. Boxplot indicating differences in eigenvector values at boundaries and centers of A and B compartments. Differences in eigenvector values were calculated from HiC data sets from ESC+2i treated with siRNA-Baz2a and siRNA-control. Statistical significance (P-values) for the experiments was calculated using the unpaired two-tailed t-test (*** < 0.001, ns: non significant). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. \n",
      "A-bound regions mark the highly interacting long-distance contacts in ESCs A.Figure 4A. Number of BAZ2A-bound regions in A and B compartments of ESC+serum. B. Hi-C contact matrices for a zoomed in region on chromosome 3 at 100-kb resolution showing the presence of strong long-distance contacts corresponding to regions in A compartment bound by BAZ2A in ESC+serum. Low panels. HiC contacts of a zoomed region at 50-kb resolution in ESC+serum, NPCs and CNs are shown. Data are from (Bonev et al., 2017). HiC contacts were visualized using Juicebox C. BAZ2A-bound regions mark A subcompartments that are characterized by the strongest far-cis contacts in ESC+serum. Boxplots indicating observed/expected contact values between BAZ2A-bound regions and loci not bound by BAZ2A at near-cis and far-cis contacts in all genome or in A compartments. Statistical significance (P-values) was calculated using the unpaired two-tailed t-test (****< 0.0001). D. Long-range contacts marked by BAZ2A binding are specific for ESCs. Boxplots indicating observed/expected contact values between BAZ2A-bound regions and loci not bound by BAZ2A at far-cis contacts in all genome or in A compartments of ESC+serum, NPC and CN. Statistical significance (P-values) was calculated using the unpaired two-tailed t-test (**** < 0.0001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. E. Number of BAZ2A-bound regions in A and B compartments in ESC+2i. F. BAZ2A-bound regions mark A subcompartments that are characterized by the strongest far-cis contacts in ESC+2i. Boxplots indicating observed/expected contact values between BAZ2A-bound regions and loci not bound by BAZ2A at near-cis and far-cis contacts in all genome or in A compartments of ESC+2i. Statistical significance (P-values) was calculated using the unpaired two-tailed t-test (** < 0.001, ****< 0.0001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. G. BAZ2A binding correlates the best with eigenvector values. Heat map showing Pearson correlation values of ChIPseq data sets with the eigenvector values from HiC data in ESC+serum and ESC+2i, respectively. H. Boxplots showing H3K4me1 and H3K27ac content of BAZ2A-bound regions and loci not bound by BAZ2A at far-cis contacts in ESC+2i. Data from regions with lowest 10% and top 10% contact strength are shown. Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. \n",
      ".Figure 1(A) Dynamic regulation of fluorescently tagged Beta-catenin/Armadillo (Arm-GFP) during mitosis in the growing fly wing epithelium. Notice down-regulation of Arm-GFP as cells round up for mitosis (arrow). Scale bar ~2µm. n>10 independent biological replicates. (B) Dynamic regulation of fluorescently tagged E-cadherin (E-cad-GFP) during mitosis in the growing fly wing epithelium. Notice down-regulation of E-cad-GFP as cells round up for mitosis (arrow). Scale bar ~2µm. n>10 independent biological replicates. (C) E-cad-GFP remains downregulated in cells arrested in mitosis with Colchicine. Scale bar ~2µm. n>10 independent biological replicates. (D) Quantification of the degree of down-regulation of Arm-GFP and E-cad-GFP at the junction between mitotic cells and their interphase neighbours (n > 10 independent samples per cell cycle stage). Mean +/- 1 S.D. shown. (E) Two adjacent mitotic cells show no detectable Arm-GFP at the junction between them. Scale bar ~2µm. (F-I) Electron microscopy cross-sections of wing epithelial cells at interphase, prophase, prometaphase and telophase. Notice adherens junctions (dark staining, highlighted in insets) are downregulated as cells round up for mitosis. Scale bars ~1µm (low mag.) and ~10nm (high mag.). n>10 independent biological replicates. Asterisks indicate a single mitotic cell. Green arrowheads indicate adherens junctions. \n",
      "A regulates chromatin accessibility in ground-state ESCs A.Figure 5A. Numbers of ATAC peaks called separately in ESC+2i and ESC+serum including their localization according to compartmentalization. B. Numbers of significant changed ATAC peaks in ESC+2i vs. ESC+serum and their localization according to compartmentalization. Increased chromatin accessibility sites (IAS), decreased accessibility sites (DAS). C. Average profile of normalized ATAC read counts over differential peaks in ESC+2i and ESC+serum changed ATAC peaks. The signal is plotted over +/- 1kb on the center of the ATAC peak. D. H3K27ac correlates with differential accessible sites between ESC+2i and ESC+serum. Boxplot showing the mean values of normalized H3K27ac at ATAC peaks (+/- 500bp) that are increased and decreased in ESC+2i compared to ESC+serum, respectively. ESCs were subjected to two consecutive treatments with siRNA-control and siRNA-Baz2a, each one lasting for 4 days. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (*** < 0.001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. E. BAZ2A depletion increases chromatin accessibility only in ESC+2i. Numbers of significant changed ATAC peaks in ESC+2i and ESC+serum upon BAZ2A knockdown and their localization according to compartmentalization. F. Average density plots of normalized ATAC read counts over differential ATAC peaks of ESC+2i and ESC+serum treated with siRNA-Baz2a and siRNA-control over changed ATAC peaks. The signal is plotted over +/- 1kb centered on the ATAC peak. G. The distances of increased (IAS) and decreased (DAS) chromatin accessibility sites in BAZ2A-depleted ESC+2i to the nearest gene promoter (+/- 5kb of TSS) were calculated. Distances in bp to the nearest promoter of genes upregulated (UP), downregulated (DOWN) or not affected (NOT) upon BAZ2A knockdown in ESC+2i are plotted. Statistical significance (P-values) for the experiments was calculated using the unpaired two-tailed t-test (** < 0.01). Error bars represent s.d. The mean is represented by a horizontal line. H. Genomic annotation of IAS and DAS in ESC+2i depleted of BAZ2A perfomed with ChIPseeker I. Fold-enrichment relative to expected of regions with increased chromatin accessibility (IAS) and upregulated genes upon BAZ2A KD at sub-regions within compartment A and B (center, inside and boundaries). Compartments smaller than 500 kb were excluded from the analysis. Statistical significance (P-values) was calculated with permutation test (ns: non significant, * < 0.05, *** < 0.001). J. BAZ2A-upregulated genes are enriched in the B compartment. Numbers of BAZ2A-regulated genes upregulated and downregulated upon BAZ2A knockdown in ESC+2i including their localization according to compartmentalization. K. Active histone marks increase in the B compartment upon BAZ2A knockdown. Boxplots indicating the log2 fold-changes of normalized H3K27ac and H3K4me3 read counts in ESC+2i treated with siRNA-Baz2a and siRNA-control at boundaries and centers of A and B compartments. ). Statistical significance (P-values) for the experiments was calculated using the unpaired two-tailed t-test (*** < 0.001, * < 0.05). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. \n",
      "A and components of the cohesin complex on chromatin.Figure 8A. Scheme representing the strategy used to identify protein interacting with BAZ2A on chromatin (Chromatin-IP). B. BAZ2A-interacting proteins on chromatin. Mass spectrometry analysis of FLAG immunoprecipitates from wt and F/H-BAZ2A ESC+2i. Values of peptide number are shown as the difference between peptides obtained in FLAG-IP of F/H-BAZ2A and wt ESC chromatin extracts. Data are from three independent experiments. The proteins found enriched in FLAG-IP of F/H-BAZ2A ESCs in all three experiments are shown. Further BAZ2A-interacting proteins identified in only two or one of the immunoprecipitation (IP) experiments Error bars represent s.d. C. Anti-HA IP of wt- and F/H-BAZ2A-ESC+2i and ESC+serum. Western blot shows the interaction of BAZ2A with SNF2H, TOP2A and SMC1a. D. TOP2A and BAZ2A regulate gene expression in ESC+2i via a shared pathway. qRT-PCR of genes regulated by BAZ2A in ESC+2i (left panel) and in ESC+serum (right panel) upon TOP2A or BAZ2A knockdown by siRNA. ESCs were treated with the corresponding RNAs for 4 days. Upregulated genes in ESC+2i upon BAZ2A-KD are labeled in blue, downregulated genes are in red. Nanog, Rex1 and Actin B (ActB) are shown as genes not regulated by BAZ2A. mRNA levels were normalized to Rps12 mRNA and to ESCs transfected with siRNA-Control. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns: non significant, * < 0.05, ** < 0.01, *** < 0.001, **** < 0.0001). E. Inhibition of type 2 topoisomerase activity phenocopies the alterations in gene expression of BAZ2A-regulated genes in ESC+2i observed upon TOP2A or BAZ2A knockdown. ESCs were treated with the TOP2 inhibitor ICF-193 for 24 hours. mRNA levels were normalized to Rps12 mRNA and to ESCs treated with DMSO. Upregulated genes in ESC+2i upon BAZ2A-KD are labeled in blue, downregulated genes are in red. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns: non significant, ** < 0.01, *** < 0.001, **** < 0.0001). F. ChIP analysis showing that TOP2A regulates H3K27me3 occupancy in ESC+2i but not in ESC+serum. Data were measured by qPCR and normalized to input and to Tshz1 value in cells transfected with siRNA-control. #1-#4 indicated sequences with elevated H3K27me3 at CTCF boundary (H3K27me3hi/CTCF). Control represents an intergenic region that does not contain H3K27me3. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns: non significant, * < 0.05, ** < 0.01, *** < 0.001). \n",
      "A associates with large and active chromatin domains in ESCs A.Figure 3A. Western blot showing equal BAZ2A levels in wt-ESC and F/H-BAZ2A ESC line containing FLAG-HA sequences at the N-terminus of both Baz2a alleles. Whole cell lysates from equivalent amounts of cells were analysed. SNF2H, and Tubulin are shown as protein loading controls. B. Anti-FLAG ChIP of wt-ESCs and F/H-BAZ2A-ESCs cultured in 2i or serum. Data were measured by qPCR and normalized to input and a control region that is not bound by BAZ2A. Average values of three independent experiments. Error bars represent s.d. and statistical significance (P-values) was calculated using the paired two-tailed t-test (* < 0.05, ** < 0.01). C. Representative images showing the association of BAZ2A at regions enriched in H3K27ac and low in H3K27me3. The grey rectangles highlighted some of BAZ2A associated regions. D. Pearson correlation heat map for the indicated ChIPseqs in ESC+2i. BAZ2A ChIPseqs were performed with FLAG antibodies in F/H-BAZ2A-ESCs cultured in 2i or serum and with HA antibodies in F/H-BAZ2A-ESC+2i. Data of H3K4me3, H3K27ac and H3K27me3 are from this work. H3K36me3, H3K9me3 and H3K4me1 values in ESC+2i were taken from published data sets E. Boxplot showing the mean of normalized BAZ2A occupancies in ESC+2i and ESC+serum at BAZ2A-bound regions. Statistical significance (P-values) for the experiments was calculated using the unpaired two-tailed t-test (**** < 0.0001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. F. Differential BAZ2A-binding in 2i and serum correlates with H3K27ac occupancies. Boxplot showing the mean of normalized H3K27ac and H3K27me3 read counts in ESC+2i and ESC+serum at all BAZ2A-bound regions and BAZ2A-bound regions only in 2i were calculated and their log2-fold changes are plotted. H3K27ac in ESC+serum Statistical significance (P-values) was calculated using the paired two-tailed t-test (*** < 0.001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. G. Representative images showing 2i-specific BAZ2A-bound regions and H3K27ac and H3K27me3 in ESC+2i and ESC+serum. The grey box highlights strong TIP5-bound regions and corresponding histone modifications. H. Representative images showing that genes regulated by BAZ2A are enriched in H3K27me3, depleted of H3K27ac and not bound by BAZ2A. I. Heat map profiles of H3K27me3 and H3K27ac at ±5 kb from the TSS of genes differentially regulated by BAZ2A in ESC+2i and random genes. Differentially regulated genes are shown as upregulated or downregulated in ESC+2i upon BAZ2A knockdown. Data were ranked by H3K27me3 levels. Transcription start site (TSS). J. Boxplot showing the mean of normalized H3K27me3 and H3K27ac read counts over +/- 1kb of TSSs from upregulated, downregulated and random genes. Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. \n",
      " occupancy in ground-state ESCs A.Figure 7A. Western blot showing total levels of H3K27me3 in ESC+2i transfected with siRNA-control and siRNA-Baz2a. Histone H3 is shown as a protein loading control. B. Scatterplot showing the changes in H3K27me3 upon BAZ2A depletion in ESC+2i and ESC+serum. ESCs were subjected to two consecutive treatments with siRNA-control and siRNA-Baz2a, each one lasting for 4 days. Mean of normalized H3K27me3 read counts in ESC+2i and ESC+serum treated with siRNA-control or siRNA-Baz2a over all TSSs (+/- 1kb) were calculated. Log2-fold changes of (siRNA-Baz2a/siRNA-control) are plotted relative to the mean of normalized H3K27me3 occupancies. The red line represents the mean of H3K27me3 fold-changes. C. Representative images showing the H3K27me3 occupancy. D. Average density plot of ChIPseq read counts of H3K27me3 at -/+ 5 kb from the TSS of refseq genes in ESCs treated with siRNA-control or siRNA-Baz2a Q1: first quartile corresponding to sequences with high H3K27me3 content. Q2-4: quartiles 2-4 corresponding to sequences with low H3K27me3 content. E. ChIP-qPCR of H3K27me3 in ESC+2i and ESC+serum treated with siRNA-control or siRNA-Baz2a. Data were measured by qPCR and normalized to input and to Tshz1 value in control cells. Control represents an intergenic sequence that does not contain H3K27me3. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns: non significant, * < 0.05, ** < 0.01, *** < 0.001). F. Boxplot showing changes in H3K27me3 in ESC+2i upon BAZ2A at BAZ2A-regulated and random genes. Mean of normalized H3K27me3 read counts in ESC+2i treated with siRNA-control or siRNA-Baz2a over TSSs (+/- 1kb) of upregulated, downregulated and random genes were calculated. Log2-fold changes of (siRNA-Baz2a/siRNA-control) are plotted. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (** < 0.01 and *** < 0.001). Box plots depict the minimum and maximum values. The mean is represented by a horizontal line within the boxes. G. Average density and heatmap profile of H3K27me3 transition state regions marked by CTCF sites in ESC+2i or ESC+serum transfected with siRNA-control or siRNA-Baz2a. Data were ranked by H3K27me3 content in the corresponding siRNA-control ESCs. Cluster 1 represents high to low H327me3 transition whereas Cluster 2 shows low to high H3K27me3 levels. H. H3K27me3 ChIP-qPCR in ESC+2i (top panel) or ESC+serum (bottom panel) transfected with siRNA-control or siRNA-Baz2a. Values show H3K27me3 occupancy at four sequences (#1-#4) with elevated H3K27me3 at CTCF boundary (H3K27me3hi/CTCF). Control represents an intergenic sequence that does not contain H3K27me3. Data were measured by qPCR and normalized to input and to sequence #1 value in control cells. Average values of three independent experiments. Error bars represent s.d. Statistical significance (P-values) for the experiments was calculated using the paired two-tailed t-test (ns: non significant, * < 0.05, ** < 0.01). I. Hi-C contact matrices for a zoomed in region on chromosome 6 containing HoxA cluster (10-kb resolution in upper panels, 5 kb resolution in lower panels). Blue arrows (A, B, and C) indicate gain or increase of contacts in ESC+siRNA-Baz2a compared to control cells (siRNA-control). The orange arrow (D) indicates loss or decrease in contacts. \n",
      ".Figure 7(A) Live-imaging of MyoII-GFP in a control wing disc epithelium. Asterisk indicates the location of a single mitotic cell about to initiate rounding. Scale bar ~1µm. n > 5 independent biological replicates. (B) Live-imaging of MyoII-GFP upon treatment with the dual AurA/B inhibitor VX-680 after initiation of mitotic rounding, which rapidly arrests progression of rounding and prevents cytokinesis. Scale bar ~1µm. n > 9 independent biological replicates. (C) Live-imaging of E-cad-GFP upon treatment with the dual AurA/B inhibitor VX-680 after initiation of mitotic rounding, which rapidly arrests progression of rounding and prevents cytokinesis. Scale bar ~1µm. n > 6 independent biological replicates. (D) Working model for mitotic rounding and junctional downregulation in pseudostratified Drosophila epithelial cells. \n",
      ".Figure 3(A) Nuclear localised Pbl-GFP becomes cytoplasmic in mitotic cells, co-incident with Rok-GFP relocalisation to the plasma membrane. Scale bar ~1µm. n > 5 independent biological replicates. (B) Phosphorylated Myosin-II Regulatory Light Chain (p-MLC; purple) detected by antibody staining reveals high levels around the cortex of rounded mitotic cells in wild-type wing epithelial cells, but not those expressing UAS.pbl-RNAi. Phospho-Histone H3 (pH3) staining (green) marks mitotic cell chromosomes. Quantification shown right (n > 10 independent samples per genotype; mean +/- 1 S.D. shown). Scale bar ~1µm. (C) A stripe of expression of ptc.Gal4 UAS.pbl-RNAi in the wing disc leads to both enlarged cells and many basally extruded cells that undergo apoptosis, marked by pyknotic nuclei and loss of aPKC/Dlg staining. Scale bars ~20µm. n > 6 independent biological replicates. \n",
      ".Figure 8(A) Early stage Drosophila egg chambers showing giant germ-line nurse cells surrounded by E-cadherin-positive follicle cell epithelium. A follicle cell undergoing mitotic rounding is highlighted with an asterisk (*). Scale bar ~3µm. n > 3 independent biological replicates. (B) High resolution imaging of the F-actin cortex, E-cadherin, and phospho-MyoII in a mitotically rounded follicle cell. Note maintenance of adherens junctions including both E-cad and the contractile actomyosin ring during mitotic rounding. Asterisks or pH3 staining indicate a single mitotic cell. Scale bar ~3µm. n > 4 independent biological replicates. (C) Schematic diagram of follicle cell rounding, showing the relatively mild shape change that is insufficient to affect E-cadherin levels during mitotic rounding. \n",
      ".Figure 4(A) Nuclear localised Tum-GFP becomes cytoplasmic in mitotic cells, including a distinct plasma membrane localisation (arrows). Note also localisation to the cytokinetic ring (asterisk) and remnant mid-bodies/ring canals in interphase cells. Quantification shown on the right (n > 7 independent samples per genotype; mean +/- 1 S.D. shown). Scale bars ~2µm. (B) A stripe of expression of ptc.Gal4 UAS.tum-RNAi in the wing disc leads to basally extruded cells that undergo apoptosis, marked by Dcp1 immunostaining, similar to UAS.pbl-RNAi. Quantification shown on the right (n > 10 independent samples per genotype; mean +/- 1 S.D. shown). Scale bars ~20µm. (C) Phosphorylated Myosin-II Regulatory Light Chain (p-MLC; red) detected by antibody staining reveals high levels around the cortex of rounded mitotic cells in wild-type wing epithelial cells, but not those cells homozygous mutant for tum347, marked by expression of GFP in single-celled clones (MARCM system). pH3 staining (blue) marks mitotic cell chromosomes. Arrow indicates a GFP-positive single cell tum347 mutant clone. Quantification shown on the right (n > 3 independent samples per genotype; mean +/- 1 S.D. shown). Scale bars ~20µm. \n",
      ".Figure 5(A) Phosphorylated Myosin-II Regulatory Light Chain (p-MLC; purple) detected by antibody staining reveals high levels around the cortex of rounded mitotic cells (pH3, red) in wild-type wing discs, and aurA homozygous mutant wing discs, but not upon inhibition of both AurA and AurB by treatment with VX-680. Note increased numbers of mitotic cells in aurA homozygous mutant wing discs, due to delayed mitotic progression. Scale bars ~20µm (low mag.) and ~1µm (high mag.). n > 15 independent biological replicates. (B) Live-imaging of MyoII-GFP reveals delayed mitosis in aurA homozygous mutant wing discs, while inhibition of both AurA and AurB by treatment with VX-680 leads to complete loss of mitotic rounding and MyoII-GFP membrane localisation. Asterisks indicate a single mitotic cell. Scale bars ~1µm. Quantification shown on the bottom right (n = 5 independent samples per genotype). (C) Live-imaging of E-cad-GFP reveals delayed mitosis in aurA homozygous mutant wing discs, cytokinesis failure in UAS.aurB-RNAi expressing wing discs, and complete failure of mitotic rounding and E-cad-GFP downregulation upon inhibition of both AurA and AurB by treatment with VX-680. Asterisks indicate a single mitotic cell. Scale bars ~1µm. Quantification shown on the bottom right (n = 9 independent samples per genotype). \n",
      ".Figure 6(A) Striped ptc.Gal4-driven expression of UAS.GFP in a control wing imaginal disc. Scale bars ~20µm. n > 6 independent biological replicates. (B) Striped ptc.Gal4-driven expression of UAS.GFP in an aurA homozygous mutant wing imaginal disc leads to increased numbers of pH3 positive mitotic cells. n > 7 independent biological replicates. (C) Striped ptc.Gal4-driven expression of UAS.GFP and UAS.aurB-RNAi in a control wing imaginal disc leads to enlarged cells with large nuclei (marked by DAPI; white arrowhead). n > 6 independent biological replicates. (D) Striped ptc.Gal4-driven expression of UAS.GFP and UAS.aurB -RNAi in an aurA homozygous mutant wing imaginal disc leads to extrusion and apoptosis of cells (marked by Dcp1 and pyknotic nuclei). n > 8 independent biological replicates. (E) Striped ptc.Gal4-driven expression of UAS.GFP and UAS.pbl-RNAi in the wing imaginal disc leads to extrusion and apoptosis of cells (marked by Dcp1 and pyknotic nuclei). n > 8 independent biological replicates. (F) ptc.Gal4-driven expression of UAS.aurB-RNAi in an aurA homozygous mutant endoreplicating salivary gland has no phenotypic consequence. n > 6 independent biological replicates. (G) MARCM clones (GFP-positive) expressing UAS.aurB-RNAi in an aurA homozygous mutant cells result in single cell clones (i.e.: no cell division), while individual loss of AurA or AurB allows some cell proliferation. Quantification shown on the right (n > 7 independent samples per genotype; Mean +/- 1 S.D. shown). Scale bar ~20µm. \n",
      ".Figure 2(A) Phosphorylated Myosin-II Regulatory Light Chain (p-MLC; purple) detected by antibody staining reveals high levels around the cortex of rounded mitotic cells in wild-type wing epithelial cells in the anterior (A) compartment, but not in Rho-kinase (Rok) RNAi expressing cells of the posterior (P) compartment (GFP-positive; green). Scale bar ~10µm. n>10 independent biological replicates. (B) High-magnification view of single mitotic cell stained for p-MLC (purple) and E-cadherin (green), with metaphase plate chromosomes stained by DAPI (blue). Scale bar ~1µm. n>10 independent biological replicates. (C) High-magnification view of single UAS.Rok-RNAi expressing mitotic cell stained for p-MLC (purple) and E-cadherin (green), with metaphase plate chromosomes stained by DAPI (blue). Note loss of cortical p-MLC staining (purple). Scale bar ~1µm. n>10 independent biological replicates. (D) Expression of constitutively active Rho-kinase (UAS.Rok-CA) in the posterior compartment of the wing epithelium is sufficient to elevate p-MLC and reduce E-cadherin immunostaining. Zoom (right) shows high p-MLC and low E-cad in both a mitotic cell and its neighbours. Scale bar ~10µm. n>10 independent biological replicates. (E) MyoII RLC-GFP (MyoII-GFP) accumulates at the cortex as cells round up for mitosis and later appears at the cleavage furrow during cytokinesis (basal section, diagrammed left). Treatment with the Rho-kinase inhibitor Y-27632 inhibits mitotic accumulation of MyoII-GFP. Quantification shown right (n = 8 independent samples per genotype). Scale bar ~1µm. Mean +/- 1 S.D. shown. (F) E-cad-GFP is downregulated as cells round up for mitosis (apical section, diagrammed left). Treatment with the Rho-kinase inhibitor Y-27632 inhibits mitotic downregulation of E-cad-GFP. Extended treatment with Y-27632 leads to a general defect in all interphase and mitotic cells in which apical surface areas enlarge and E-cad-GFP is generally lost. Arrow indicates mitotic cell cortex at maximal rounding. Quantification shown right (n = 6 independent samples per genotype; mean +/- 1 S.D. shown). Scale bar ~1µm. \n",
      "Jumpy siRNA increases recruitment of PI3P‐binding Atg factor WIPI‐1 (Atg18) to autophagic membranes.Figure 5(A-C) C2C12 were transfected for 48 h with Jumpy siRNA, followed by a second transfection overnight with Jumpy siRNA and GFP‐WIPI‐1, incubated for 4 h in starvation media, fixed and immunostained with anti‐Atg16 antibody (red). GFP‐WIPI1‐transfected cell (A), endogenous Atg16 immunostaining (B), colocalization of GFP‐WIPI‐1 and Atg16 is indicated in yellow in merge image (C). Boxed areas are shown at higher magnification as inset. Scale bars, 10 and 5 μm in insets. (D-I) C2C12 cells were transfected for 48 h with control (scramble) (D, E) or Jumpy siRNA (F-H), followed by a second transfection overnight with corresponding siRNA and GFP‐WIPI‐1. Cells were incubated for 4 h in full (D, G) or starvation media (E, F, H) in presence (F) or absence (E-H) of 100 nM wortmannin, fixed and analysed by confocal fluorescence microscopy. Quantitation of WIPI‐1 puncta per cell (I). Bars, s.e.m. (n=3 independent experiments, with an average of 30 cells per experiment). *P0.05, **P0.01 (t‐test). \n",
      "Jumpy localizes to autophagosomes.Figure 3(A) Transient association of Jumpy WT with LC3+ organelles. Time lapse sequence of C2C12 myoblasts transfected for 24 h with GFP‐Jumpy WT and Cherry‐LC3 and analysed by live confocal microscopy in a 5LIVE Zeiss microscope. EBSS was added to the cells and z‐stacks collected at 3‐min intervals for a total of 45 min. The collected images were processed to generate a maximum projection (collapsing a 3D image into an x-y projection) for each time point. Arrows indicate colocalization of GFP‐Jumpy WT with LC3. Graph: Plots of relative fluorescence intensity (GFP fluorescence pixel intensity) over time of GFP‐Jumpy WT colocalizing with Cherry‐LC3+‐positive puncta indicated by arrows. (B-I) Association of Jumpy CS mutant with autophagic organelles. C2C12 cells were transfected for 24 h with GFP‐Jumpy C330S (Jumpy CS) and tdTomato‐LC3 (LC3), fixed and immunostained with anti‐G58K (Alexa 648‐labelled secondary antibody). Jumpy CS (green), LC3 (red) and G58K (white in D, H or blue in E, I). Boxed areas (B-E) are shown at higher magnification in the corresponding panel below (F-I). Scale bars, 2 μm. White arrows indicate colocalization between Jumpy CS, LC3 but not G58K, blue arrow indicates colocalization between Jumpy CS and G58K but not LC3. \n",
      "Jumpy colocalizes with and regulates Atg9 distribution to autophagic organelles.Figure 6(A-T) C2C12 cells were transfected for 24 h with GFP, GFP‐Jumpy WT or GFP‐Jumpy C330S (Jumpy CS) and Tomato‐LC3 (LC3) and HA‐Atg9, fixed and immunostained with anti‐HA. GFP (I), Jumpy WT (M), Jumpy CS (green) (A, E, Q), LC3 (red) (C, G, J, N, R) and Atg9 (white in B, F, K, O, S or blue in D, H, I, P, T). Boxed areas (A-D) are shown at higher magnification in the corresponding panel below (E-H). White arrows indicate colocalization between Jumpy CS, LC3 and Atg9, blue arrow indicates colocalization between Jumpy CS and Atg9 but not LC3 and yellow arrow indicates colocalization between Jumpy CS and LC3 but not Atg9. White boxes (I-T) show LC3 puncta location. Scale bars, 5 μm. (U) Percentage of LC3 puncta colocalizing with Atg9. n, number of puncta counted. \n",
      "Screening of active members of the myotubularin family identifies Jumpy as a negative regulator of autophagy.Figure 1(A) Domains and members of catalytically active myotubularins. Asterisks, Jumpy mutations found in patients with centronuclear myopathy. C330S, R336Q and Y462C, Jumpy mutants used in this study. (B) RAW 264.7 cells transfected for 48 h with control (sc), MTMR6 or Jumpy siRNA, were pretreated for 30 min with 100 nM Bafilomycin A1 (BafA1), 10 μg/ml E64d and 10 μg/ml pepstatin, then incubated for 30 min in full or starvation media in the presence of BafA1, E64d and pepstatin. Cells were lysed and analysed by immunoblotting with anti-LC3 or anti‐actin. Densitometric LC3‐II/actin ratios are shown underneath the blot. Graph: LC3‐II/actin ratio for Jumpy siRNA in full medium is equal to or exceeds LC3‐II/actin ratio for control siRNA (sc) in starvation medium. (C, D) RAW 264.7 cells were transfected for 36 h with control (scramble) or Jumpy siRNA, transfected once more with corresponding siRNA and mRFP‐GFP‐LC3 DNA construct overnight and incubated for 2 h in full or starvation media. Cells were fixed and LC3 puncta analysed by confocal fluorescence microscopy. (C) Representative confocal images of RAW 264.7 cells in full or starvation media after transfection with control (scramble) or Jumpy siRNA (siJumpy). Red and yellow arrows indicate GFP−RFP+ and GFP+RFP+ puncta, respectively. (D) Quantitation of number of LC3 puncta per cell, total puncta per cell (GFP+RFP+ and GFP−RFP+ puncta) GFP+RFP+ puncta per cell and GFP−RFP+ puncta per cell. Data, mean±s.e.m. n=5 independent experiments, 30 cells per experiments. *P0.05, **P0.01, ***P0.001 (t‐test). Scale bars, 5 μm. (E, F) C2C12 myoblasts were transfected for 48 h with control (scramble) or Jumpy siRNA, followed by a second transfection overnight with corresponding siRNA and mRFP‐GFP‐LC3 DNA construct. Cells were fixed and LC3 puncta were counted by confocal fluorescence microscopy. (E) Representative confocal images of C2C12 cells in full media after transfection with control (scramble) or Jumpy siRNA (siJumpy). (F) Quantification of the number of LC3 puncta per cell. Data, mean±SEM for n=3 (independent experiments), 30 cells per experiments. *P 0.05 (t-test). Scale bars, 10 μm. (G, H) C2C12 cells were transfected for 48 h with control (sc) (lanes 1, 2, 5, 6) or Atg5 siRNA (lanes 3, 4, 6, 7) followed by a second transfection for 48 h with same siRNA and control (lanes 1, 3, 5, 7) or Jumpy siRNA (lanes 2, 4, 6, 8). Cells were incubated for 1 h with or without 100 nM Baf A1, 10 μg/ml E64d and 10 μg/ml pepstatin in full media, lysed and analysed by immunoblotting with anti‐LC3 or anti‐actin (G). (H) Densitometric LC3‐II/actin ratios for samples treated with BafA1 and protease inhibitors from G (lanes 5-8). Inset shows Atg5 knock‐down by immunoblotting. \n",
      "Jumpy R336Q mutant associated with centronuclear myopathy is defective in inhibiting autophagy.Figure 7(A-K) C2C12 cells were transfected for 48 h with YFP, YFP‐Jumpy WT (Jumpy WT), YFP‐Jumpy C330S (Jumpy CS), YFP‐Jumpy Y462C (Jumpy YC) or YFP‐Jumpy R336Q (Jumpy RQ), fixed and immunostained with anti‐p62 antibody (red). p62 puncta were quantitated by confocal fluorescence microscopy. Representative confocal images of YFP (A), YFP‐Jumpy WT (B), YFP‐Jumpy C330S (C), YFP‐Jumpy Y462C (D) and YFP‐Jumpy R336Q (E) transfected cells, immunostained for p62 (F), (G), (H), (I) and (J), respectively. Scale bars, 5 μm. Quantitation of p62 puncta per cell (K). Bars, s.e.m. (n=3 independent experiments, 30 cells per experiments). *P0.05 (t‐test), ns: nonsignificant. (L) HeLa cells were transfected for 24 h with YFP, YFP‐Jumpy WT (Jumpy WT) or YFP‐Jumpy R336Q (Jumpy RQ), incubated with or without 50 ng/μl rapamycin (BafA1 (100 nM) was present in both control and rapamycin treated cells) for 2 h, lysed and analysed for LC3, YFP and actin by immunoblotting. Densitometric LC3‐II/actin ratios are shown underneath the blot. \n",
      "Jumpy localizes to autophagic isolation membranes.Figure 4(A-H) C2C12 cells were transfected for 24 h with GFP‐Jumpy C330S (Jumpy CS) and tdTomato‐LC3 (LC3), fixed and immunostained with anti‐Atg16 (Alexa 648‐labelled secondary antibody). Jumpy CS (green), LC3 (red) and Atg16 (white in B and F or blue in D and H). Boxed areas (A-D) are shown at higher magnification in the corresponding panel below (E-H). White arrows indicate colocalization among Jumpy CS, LC3 and Atg16, yellow arrow indicates colocalization between Jumpy CS and LC3 but not Atg16, blue arrow indicates colocalization between Jumpy CS and Atg16 but not LC3. (I, J) C2C12 cells were transfected for 24 h with GFP (I), fixed and immunostained with anti‐Atg16 (J). (K-R) C2C12 cells were transfected for 24 h with GFP‐Jumpy C330S (Jumpy CS) and tdTomato‐LC3 (LC3), fixed and immunostained with anti‐Atg12(M) antibody. Jumpy CS (green), LC3 (red) and Atg12 (white in L, P or blue in N, R). Boxed areas (K-N) are shown at higher magnification in the corresponding panel below (O-R). White arrows indicate colocalization among Jumpy CS, LC3 and Atg12, yellow arrow indicates colocalization between Jumpy CS and LC3 but not Atg12, blue arrow indicates colocalization between Jumpy CS and Atg12 but not LC3. (S-ZA) C2C12 cells were transfected for 24 h with Jumpy CS and Cherry‐LC3 (LC3), fixed and immunostained with anti‐lamp1. Jumpy CS (green), LC3 (red) and Lamp1 (white in T, X or blue in V, ZA). Boxed areas (S-V) are shown at higher magnification in the corresponding panel below (W-ZA). Yellow arrows indicate colocalization between Jumpy CS and LC3 but not Lamp1. Scale bars, 2 μm. \n",
      "Jumpy blocks autophagic degradation.Figure 2(A) Proteolysis of long‐lived proteins in C2C12 myoblasts. C2C12 cells were transfected with control (scramble) or Jumpy siRNA (siJumpy), labelled overnight in media containing [3H] leucine, washed, incubated for 2 h in complete media (containing cold leucine) and incubated for 4 h in full or starvation media. Leucine release was calculated from radioactivity in the tricarboxylic acid‐soluble form relative to total cell radioactivity. Results shown represent mean±s.e.m. for combined data from three independent experiments. (B) Quantitation of p62 protein levels. C2C12 cells were transfected for 48 h with control (sc) or Jumpy siRNA and p62 levels analysed by immunoblotting, quantitated by densitometry and represented as a percentage of control. Data are mean±s.e.m. (n=4 independent experiments). (C) C2C12 cells were transfected for 48 h with control (sc) or Jumpy siRNA, incubated for 4 h with or without 100 nM Bafilomycin A1 in full media, lysed, probed for endogenous p62 and actin by immunoblotting and percentage of p62 were quantitated (mean±s.e.m., n=3). (D) C2C12 cells were transfected for 48 h with control (sc) (lanes 1, 2) or Beclin siRNA (lanes 3, 4) followed by a second transfection for 48 h with same siRNA and control (lanes 1, 3) or Jumpy siRNA (lanes 2, 4). Cells were lysed, probed for p62, Beclin and actin by immunoblotting and percentage of p62 were quantitated (mean±s.e.m., n=5). (E-M) C2C12 cells were transfected for 48 h with GFP, GFP‐Jumpy (Jumpy), GFP‐MTM1 (MTM1) or GFP‐MTMR2 (MTMR2), fixed and immunostained with anti‐p62 antibody (red). p62 puncta were quantitated by confocal fluorescence microscopy. Representative confocal images of GFP (E), GFP‐Jumpy (F), GFP‐MTM1 (G), GFP‐MTMR2 (H) transfected cells, immunostained for p62 (I), (J), (K) and (L), respectively. Scale bars, 5 μm. White lines represent outline of the cells. Quantitation of p62 puncta per cell (M). Bars, SEM (n=3 independent experiments, with an average of 30 cells per experiments). (N, O) C2C12 cells were transfected for 48 h with GFP, GFP‐Jumpy (Jumpy), GFP‐MTM1 (MTM1) or GFP‐MTMR2 (MTMR2), lysed, analysed for p62, GFP and actin by immunoblotting (N) and percentage of p62 were quantitated (mean±s.e.m., n=3) (O). (P, Q) C2C12 cells were transfected for 48 h with GFP or GFP‐Jumpy (Jumpy), incubated with or without 100 nM Bafilomycin A1 (BafA1) for 4 h, lysed, analysed for p62, GFP and actin by immunoblotting (P) and percentage of p62 were quantitated (mean±s.e.m., n=4) (Q). *P0.05, ***P0.001, †ns (t‐test). \n",
      ".Figure 1A. Immunofluorescence of SATB2 (green) and the GSC marker SOX2 or OLIG2 (red) on frozen tissue sections of human GBM surgical specimens. SATB2 is preferentially expressed by GSCs in human GBMs. Scale bar, 25 μm. B. Quantification of the fraction of SATB2+ cells in SOX2+ or OLIG2+ cells in human GBMs. More than 90% SOX2+ or OLIG2+ cells showed SATB2 staining. n=3 GBMs. C. qPCR analysis of SATB2 mRNA expression in GSCs and matched non-stem tumor cells (NSTCs) (n=5). D. Immunoblot analysis of SATB2, SOX2 and OLIG2 expression in cell lysates of GSCs and matched NSTCs. E. Immunofluorescenc of SATB2 (green) and SOX2 (red) in T3359 GSCs and matched NSTCs. Scale bar, 50 μm. F. qPCR analysis of SATB2 mRNA expression in GSCs and neural progenitor cells (NPCs) (n=3). G. Immunoblot analysis of SATB2 and SOX2 expression in cell lysates of GSCs and NPCs. H. Immunoblot analysis of SATB2, GSC marker SOX2 and differentiation marker GFAP expression during serum-induced GSC differentiation. \n",
      ".Figure 8SATB2 binds to the MAR region of its target gene FOXM1 to remodel local chromatin by recruting histone acetyltransferase CBP, which in turn promotes FOXM1 transcription, leading to GSC proliferation and GBM tumor growth. Inhibition of SATB2/CBP transcriptional activity by the CBP inhibitor C646 suppressed GSC proliferation and tumor growth. Reprinted with permission, Cleveland Clinic Center for Medical Art & Photography © 2020. All Rights Reserved. \n",
      ".Figure 2A. Immunoblot analysis of SATB2 expression in GSCs transduced with lentiviral-mediated non-targeting shRNA (shNT) or SATB2 shRNA (shSATB2). B. Cell viability of GSCs transduced with shNT or shSATB2 (n=5). C. EdU incorporation assay of GSCs transduced with shNT or shSATB2. Scale bar: 50 μm. D. Quantification of (C) showing the percentage of EdU+ cells (n=5). E. Tumorsphere images of GSCs transduced with shNTor shSATB2. Scale bar: 100 μm. Quantification of the diameter of tumorspheres formed by GSCs expressing shNT or shSATB2 (F: n=9; G: n=5). G. Quantification of the number of tumorspheres formed by GSCs expressing shNT or shSATB2 (F: n=9; G: n=5). H. In vitro limiting dilution analysis of the tumorsphere formations of GSCs expressing shNT or shSATB2. Silencing SATB2 attenuated the self-renewal capacity of GSCs. \n",
      ".Figure 4A. Heatmap analysis of differentially expressed genes between SATB2 silencing H2S GSCs (shSATB2) and control H2S GSCs (shNT). Differentially expressed genes had a 1.8-fold or greater expression difference. Among differentially expressed genes, 160 are upregulated and 185 are downregulated. B. Gene ontology analysis of genes downregulated in SATB2 silencing GSCs compared with control GSCs. The most significantly down-regulated genes are associated with cell cycle progression and chromosome organization. p values were generated using the PANTHER tool (http://pantherdb.org/). C. qPCR analysis of FOXM1 and SATB2 mRNA expression in GSCs transduced with shNT or shSATB2 (n=3). D. Immunoblot analysis of FOXM1 and SATB2 expression in GSCs transduced with shNT or shSATB2. E. Immunoblot analysis of FOXM1 and SATB2 expression in tumor xenografts-derived from T3359 GSCs expressing shNT or shSATB2. F. Heatmap analysis of FOXM1 downstream targets between SATB2 silencing H2S GSCs (shSATB2) and control H2S GSCs (shNT) from microarray analysis. Differentially expressed genes had a 1.5-fold or greater expression difference. G. qPCR analysis of FOXM1 downstream targets in H2S GSCs transduced with shNT or shSATB2 (n=3). \n",
      ".Figure 3 A. Bioluminescent images of the GBM xenografts derived from the luciferase-labeled T3359 GSCs expressing NT or SATB2 shRNA. Representative images on day 21 posttransplantation are shown (n=5 mice per group). Silencing SATB2 significantly delayed GBM growth .C. Kaplan-Meier survival curves of mice intracranially implanted with T3359 GSCs expressing shNT or shSATB2 (shNT: n=7 mice; shSATB2-1 or shSATB2-2: n=5 mice). Median survival: shNT, 28 days; shSATB2-1, 48 days; shSATB2-2, 45 Days. Animals bearing GSC-derived xenografts expressing SATB2 shRNA survived longer than the control animals. D. Immunofluorescence of SATB2 (Green) in xenografts derived from T3359 GSCs expressing shNT or shSATB2 (n=5 tumors per group). Scale bar: 40 μm.F. Immunofluorescence of Ki67 (Green) in tumor xenografts derived from T3359 GSCs expressing shNT or shSATB2 (n=5 tumors per group). Scale bar: 40 μm.H. Immunofluorescence of SOX2 (Red) in xenografts derived from T3359 GSCs expressing shNT or shSATB2 (n=4 tumors per group). Scale bar: 40 μm.\n",
      ".Figure 1(A) UHMK1 levels in tumor tissues (T) and corresponding nontumor tissues (N) from five GC patients were analyzed using Western blotting and qRT-PCR. The relative mRNA expression of UHMK1 was presented as mean ± standard deviation from three replicates. Pairwise, two-tailed statistical significance was assessed by Student's t‐test. Data information: *P< 0.05, **P< 0.01, ***P< 0.001. (B) Immunoblotting and qRT-PCR were used to measure the levels of UHMK1 in GES-1 and GC cell lines. The relative mRNA expression of UHMK1 was presented as mean ± standard deviation from three replicates. Pairwise, two-tailed statistical significance was assessed by Student's t‐test. Data information: *P< 0.05, **P< 0.01, ***P< 0.001. (C) IHC analysis of UHMK1 expression in a GC patient tissue array (n=102). Representative images are shown. Scale bars=100μm. (D) The IHC signals were scored using the Allred score Data were shown as mean ± standard deviation from three independent experiments. Data information: *P< 0.05, **P< 0.01, ***P< 0.001. (E) UHMK1 expression in 100 GC patients was analyzed based on the following parameters: tumor stage, tumor status, and lymph node invasion status. Fisher's test was applied to calculate the exact P value. Data information: *P< 0.05, **P< 0.01, ***P< 0.001. (F) Patients with GC were stratified by the UHMK1 level (n=100); OS and DFS were assessed by Kaplan-Meier analysis. Data information: *P< 0.05, **P< 0.01, ***P< 0.001. \n",
      ".Figure 7A. Cell viability of T3359 GSCs treated with indicated doses of C646 or the vehicle control (n=5). B. Tumorsphere images of T3359 GSCs treated with indicated doses of C646 or the vehicle control for 6 days. Scale bar: 100 μm. C. qPCR analysis of SATB2, FOXM1 and FOXM1 downstream targets in T3359 GSCs treated with indicated doses of C646 or the vehicle control for 24 hours (n=3). D. Schematic diagram showing the treatment of mice bearing the GSC-derived xenografts with C646. After GSC transplantation for 7 days, mice were treated with C646 or the vehicle control daily. Mice were monitored by IVIS bioluminescent imaging and maintained until neurological signs occur. E. Bioluminescent imaging of tumor growth in mice bearing xenografts derived from the luciferase-labeled T3359 GSCs treated with C646 or the vehicle control at indicated days after GSC transplantation (n=5 mice per group). F. Quantification of tumor growth from (E) (n=5 mice per group). G. Kaplan-Meier survival curves of mice bearing T3359 GSC-derived xenografts treated with C646 or the vehicle control (Vehicle control: n=9 mice; C646: n= 8 mice). Median survival: Vehicle control, 27 days; C646, 33.5 days. H. Immunofluorescence of Ki67 (Green) in T3359 GSC-derived xenografts from mice treated with with C646 or the vehicle control (n=6 tumors per group). Scale bar: 40 μm.\n",
      ".Figure 6A. Schematic representation of MAR (Matrix Attachment Region) within FOXM1 locus. Arrows show the location of PCR primers for ChIP experiments. B. ChIP assays with the SATB2 antibody or IgG using GSCs, NSTCs and NPCs. PCR primers amplified a fragment flanking the MAR of FOXM1 gene locus. Note that abundant SATB2 binds to the MAR of FOXM1 gene locus in GSCs. C. qPCR analysis of ChIP assays with the SATB2 antibody or IgG using GSCs transduced with shNT or shSATB2 (n=3). PCR primers amplified a fragment flanking the MAR of FOXM1 gene locus. Silencing SATB2 decreased its binding amount to the MAR of FOXM1 gene locus. D. CoIP assays of endogenous protein interaction in GSCs. Immunoblots of precipitated proteins or total lysates were performed using indicated antibodies. Note that SATB2 associates with endogenous CBP while not P300. E. qPCR analysis of ChIP assays with the CBP antibody or IgG using GSCs transduced with shNT or shSATB2 (n=3). PCR primers amplified a fragment flanking the MAR of FOXM1 gene locus. Silencing SATB2 reduced the binding of CBP to the MAR of FOXM1 gene locus. F. qPCR analysis of ChIP assays with the indicated antibody (AcH3K18, AcH3K27, AcH4) using T3359 GSCs transduced with shNT or shSATB2 (n=3). PCR primers amplified a fragment flanking the MAR of FOXM1 gene locus. Silencing SATB2 reduced acetylation of H3K18, H3K27 and H4 levels on the MAR of FOXM1 locus. G. qPCR analysis of FOXM1 mRNA expression in T3359 GSCs transduced with shSATB2 or shCBP or both (n=3). H. Immunoblot analysis of FOXM1 expression in T3359 GSCs transduced with shSATB2 or shCBP or both. \n",
      ".Figure 5A. Immunoblot analysis of FOXM1 and SATB2 expression in T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2. B. Cell viability assay of T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (n=5). Ectopic expression of FOXM1 restored the cell proliferation impaired by SATB2 silencing. C. Tumorsphere number of T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (n=5). Ectopic expression of FOXM1 restored the tumorsphere formation of GSCs impaired by SATB2 silencing. In vivo bioluminescent images of the tumor xenografts derived from luciferase-labeled T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (shNT: n=5 mice; shNT+FOXM1: n=4 mice; shSATB2: n=5 mice; shSATB2+ FOXM1: n=5 mice). Representative images on day 21 posttransplantation are shown. Ectopic expression of FOXM1 in GSCs expressing shSATB2 markedly restored GBM tumor growth. E. In vivo bioluminescent images or quantification of the tumor xenografts derived from luciferase-labeled T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (shNT: n=5 mice; shNT+FOXM1: n=4 mice; shSATB2: n=5 mice; shSATB2+ FOXM1: n=5 mice). Representative images on day 21 posttransplantation are shown. Ectopic expression of FOXM1 in GSCs expressing shSATB2 markedly restored GBM tumor growth. F. Kaplan-Meier survival curves of mice intracranially implanted with T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (shNT: n=8 mice; shNT+FOXM1: n=7 mice; shSATB2: n=6 mice; shSATB2+ FOXM1: n=8 mice). Median survival: shNT, 30 days; shNT+FOXM1, 28 days; shSATB2, 50.5 Days; shSATB2+FOXM1, 34 days. Ectopic expression of FOXM1 in GSCs expressing shSATB2 markedly attenuated the increased survival of mice bearing the GSC-derived GBMs. G. Immunofluorescence of Ki67 (Green) in xenografts derived from T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (shNT: n=6 tumors; shNT+FOXM1: n=5 tumors; shSATB2: n=6 tumors; shSATB2+ FOXM1: n=6 tumors). Scale bar: 40 μm.I. Immunofluorescence of SOX2 (Red) in xenografts derived from T3359 GSCs transduced with FOXM1 or vector control in combination with shNT or shSATB2 (shNT: n=6 tumors; shNT+FOXM1: n=5 tumors; shSATB2: n=6 tumors; shSATB2+ FOXM1: n=6 tumors). Scale bar: 40 μm.\n",
      ".Figure 2A) GC cells (SGC7901 and MGC803) were transfected with (A) shUHMK1-#1,-#2, -#3, or control shRNA lentiviral vector, or (B) with a shRNA-resistant expression construct, UHMK1Δ. Western blotting was used to measure the levels of UHMK1. NIH ImageJ software was used to quantify the band intensity. Western blotting assay were conducted for three replicates. SGC7901 cells were transfected with or without shUHMK1-#1 or the shRNA-resistant expression construct UHMK1Δ. CCK-8 assays were conducted. Scale bars=5mm. Data information: **P< 0.01, ***P< 0.001, # marked no significance. SGC7901 cells were transfected with or without shUHMK1-#1 or the shRNA-resistant expression construct UHMK1 colony formation assays were conducted. Scale bars=5mm. Data information: **P< 0.01, ***P< 0.001, # marked no significance. (D and E) The volumes of subcutaneous gastric tumors derived from SGC7901 cells in NOD/SCID mice were determined at different time points. Tumors and representative bioluminescence images are also shown. UHMK1 silencing in the mouse model was confirmed by western blotting (three biological replicates). Data were presented as mean ± standard deviation from shcontrol and shUHMK1#1 mice (n = 10 mice/group). Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Scale bars=1cm.. Data information: **P< 0.01, ***P< 0.001, # marked no significance. (F, G) UHMK1 silencing (F) and overexpression (G) affected GC cell abdominal metastasis. The quantitative number of metastatic nodules (left panel) in nude mice (n = 10 mice/group). Representative images of abdominal metastases are also shown (right panel). Scale bars=1cm. Data information: **P< 0.01, ***P< 0.001, # marked no significance. (H) Heatmap of the top 500 genes upregulated in GC patients with high UHMK1 expression. (I) Gene ontology enrichment analysis was used to analyze the top 20 enriched biological processes among GC patients with high UHMK1 expression. \n",
      ".Figure 3(A) Schematic representation of the main metabolic pathways. (B) LC-MS/MS was used to examine the metabolites in SGC7901 cells with or without UHMK1 knockdown. The data are shown in the heatmap. (C) (upper panel) LC-MS/MS analysis was performed to measure 15N-glutamine-labeled purine synthesis intermediates in BGC823 cells transfected with or without the WT-UHMK1 or UHMK1-K54R constructs. (lower panel) LC-MS/MS was used to analyze metabolites labeled with 13C-glycine in BGC823 cells transfected with or without the WT-UHMK1 or UHMK1-K54R constructs(three biological replicates). Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (D) RNA and DNA with incorporated 14C-glycine in BGC823 cells transfected with or without the WT-UHMK1 or UHMK1-K54R constructs were examined using LC-MS/MS(three biological replicates). Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (E) qRT-PCR assays were used to analyze the effects of WT-UHMK1 or UHMK1-K54R on the genes controlling purine metabolism. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (F) UHMK1 silencing significantly decreased SGC7901 cell proliferation, while ATIC overexpression or purine supplementation markedly reversed this inhibition. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. . Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (G) Treatment with the ATIC inhibitor significantly reversed the proliferation of BGC823 cells induced by UHMK1 overexpression. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. \n",
      ".Figure 7(A) SGC7901 and MGC803 cells were transfected with ATF4 shRNA or shcontrol. Western blottings and qRT-PCR were used to analyze the expression of UHMK1. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data information: Data represent mean ± SD. **P< 0.01, ***P< 0.001. (B) Overexpression of ATF4 enhanced the expression of UHMK1 in BGC823 and HGC27 cells. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data information: Data represent mean ± SD. **P< 0.01, ***P< 0.001. (C) The nucleotide sequences showing the -1348/-948 region in the human UHMK1 gene. The red text indicates the candidate ATF4 binding sites. (D) The effect of ATF4 on the activity of the WT-UHMK1 promoter and its mutations, as evaluated by using luciferase reporter assays. Data information: Data represent mean ± SD. **P< 0.01, ***P< 0.001. (E) ChIP assays were used to examine the binding of ATF4 to the UHMK1 promoter in SGC7901 and MGC803 cells. Input DNA and nonspecific IgG were included as controls. (F) ATF4 deficiency and overexpression regulated the activity of the UHMK1 promoter in GC cells. Data information: Data represent mean ± SD. **P< 0.01, ***P< 0.001. \n",
      ".Figure 5(A) GST-UHMK1 full-length (FL), GST-UHMK1 ΔN (N-terminal truncated, 1-310aa), and GST-UHMK1 ΔC (C-terminal truncated, 311-419aa) were constructed (upper panel). GST labeled FL, ΔN and ΔC protein fragments were incubated with His-NCOA3. Pull-down, Coomassie blue staining, and Western blotting were performed (lower panel). (B) GST-NCOA3 full-length (FL), GST-NCOA3 ΔN (1-408aa), GST-NCOA3 ΔS/T (409-921aa), GST-NCOA3 CBP (922-1161aa), and GST-NCOA3 ΔC (1162-1424aa) were constructed (upper panel).Pulldown assay with GST-NCOA3 FL or its protein fragments (ΔN, ΔS/T, CBP and ΔC) and His-UHMK1 were performed (lower panel). (C) We performed an in vitro kinase assay by incubating purified NCOA3 FL or its protein fragments with purified UHMK1 kinase. These proteins were visualized using Coomassie blue staining. The phosphorylation of the substrates is shown in the autoradiograph. (D) WT-UHMK1 or kinase-dead UHMK1-K54R protein was mixed with GST-CBP-WT or the indicated mutant, and in vitro kinase assays were performed. (E) Silencing UHMK1 in SGC7901 cells via UHMK1-siRNA. NCOA3 phosphorylation at S1062/T1067 was measured with a special Ab that recognizes phosphorylated NCOA3-S1062/T1067. \n",
      ".Figure 9(A) Images showing UHMK1, p-NCOA3 (S1062/T1067), and ATIC protein expression in two human GC specimens. Scale bars=100μm. (B) Pearson correlation analysis about of the levels of UHMK1, p-NCOA3 (S1062/T1067) and ATIC proteins in patients with GC (n=120) by IHC. (C) (Ci)OS data from GC patients stratified by the level of UHMK1. (Cii) Combination of UHMK1 and p-NCOA3 (S1062/T1067) expression predicts more exact OS of GC patients. Survival curves were plotted according to Kaplan-Meier method, with the log-rank test applied for comparison. (D) Schematic depicting the function of UHMK1 in GC. H. pylori-induced upregulation of UHMK1 or human GC-associated UHMK1 mutation enhances the level of NCOA3 phosphorylation at S1062/T1067; this event greatly increases the association of NCOA3 with ATF4, which activates the de novo purine biosynthesis pathway and promotes the development of GC. Interestingly, activated ATF4 transcriptionally upregulated UHMK1 expression. \n",
      ".Figure 8(A) qRT-PCR was performed to analyze UHMK1 expression in BGC823 cells infected with or without H. pylori CagA+ strains 7.13 and J166. MOI, multiplicity of infection. Data were presented as mean ± standard deviation from three replicates. Data information: Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data represent mean ± SD. *P< 0.05, **P< 0.01,. (B) Immunoblotting of UHMK1, NCOA3 phosphorylated at S1062/T1067, NCOA3, and ATF4 in the samples (C) PMSS1, a H. pylori strain, was used to challenge mice orogastrically, with Brucella broth as the control. qRT-PCR was performed to measure UHMK1 (Ci) and PMSS1(Cii) expression in gastric tissues. Data were presented as mean ± standard deviation from three replicates. Data information: Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data represent mean ± SD. *P< 0.05, **P< 0.01,. (D) Immunoblotting of UHMK1, NCOA3 phosphorylated at S1062/T1067, NCOA3, and ATF4 in the samples (E) An ATF4 luciferase reporter assay was carried out in BGC823 cells with or without H. pylori infection. Data were presented as mean ± standard deviation from three replicates. (F) (Fi) The domain architecture of UHMK1 and the positions of human GC-associated mutations. (Fii and Fiii) IP kinase assay. Briefly, UHMK1 from HEK293 cells expressing WT-UHMK1 or its mutants was immunoprecipitated with anti-HA antibodies. The immunoprecipitated proteins were mixed with a synthetic peptide of the CATS protein (a known substrate of UHMK1) and [32P] ATP. A phosphocellulose paper assay was used to measure kinase activity. The results were normalized to a value of 1.0 for WT-UHMK1(three biological replicates). (G) The effects of human GC-associated UHMK1 mutations on the proliferation and colony formation of GC cells (three biological replicates). (H) The effects of human GC-associated UHMK1 mutations on GC cell invasion and migration (three biological replicates). Data information: Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data represent mean ± SD. *P< 0.05, **P< 0.01,. \n",
      ".Figure 6(A) Knockdown of UHMK1 blocked NCOA3 binding to ATF4. SGC7901 cells were treated as indicated. Co-IP was carried out with mouse immunoglobulin G (IgG) or anti-NCOA3 antibody. Then Western blotting was performed. (B) BGC823 cells were transfected with Flag-NCOA3 or NCOA3-S1062A/T1067A with or without His-tagged UHMK1. Co-IP was performed using an anti-Flag antibody. (C) Proximity ligation assay of the interaction between different NCOA3 proteins and ATF4 in BGC823 cells. Confocal images were acquired with a Nikon A1 microscope and the Nikon Elements software suite. Maximum projection images are shown (original magnification ×120). Scale bars=20μm. (D) Computational molecular docking analysis to investigate the molecular mechanism by which NCOA3-S1062/T1067 phosphorylation is involved in the interaction of NCOA3 binding to ATF4. Based on the predictions of ZDOCK software, S1062 and T1067 of NCOA3 are located in the interface between ATF4 and NCOA3, and S1062/T1067 phosphorylation greatly enhances the binding affinity of NCOA3 for ATF4 by increasing NCOA3 hydrophilicity. (E) (Ei) ChIP data from BGC823 cells are presented as indicated. (Eii) NCOA3-S1062/T1067 phosphorylation by UHMK1 enhanced the expression of critical genes as indicated (three biological replicates). (F) (Fi and Fii) The effects of WT-NCOA3 and the NCOA3-S1062A/T1067A, and NCOA3-S1062E/T1067E mutants on BGC823 cell proliferation, invasion, and migration (three biological replicates). Data information: Unpaired, two-tailed statistical significance was assessed by Student's t‐test.Data represent mean ± SD. *P< 0.05, **P< 0.01, ***P< 0.001. (G, H) The effects of WT-NCOA3, and the NCOA3-S1062A/T1067A, and NCOA3-S1062E/T1067E mutants on GC growth and metastasis in the mouse model (n = 10 mice/group). Scale bars=1cm. Data information: Unpaired, two-tailed statistical significance was assessed by Student's t‐test.Data represent mean ± SD. *P< 0.05, **P< 0.01, ***P< 0.001. \n",
      ".Figure 1A. Overview images of hematopoietic sites in an E 9.0 embryo depicting the yolk sac (YS) and para-aortic splanchnopleural region (P-Sp) with phase contrast (upper image) and direct fluorescence microscopy (lower image). Stars indicate CX3CR1+ cells (green) and arrow head points to c-kit+/CD31+ EMPs (blue/white) in the YS. Scale bar represents 100 µm. One representative picture out of five independent experiments is displayed. B. Higher magnifications of the YS and P-Sp. CX3CR1-GFP expression (green) are found in the YS but not in the P-Sp, where only background staining is present. F4/80 (red), CD31 (blue), c-kit (white). Scale bars represent 50 µm. Representative pictures out of five independent experiments are shown. C. Flow cytometry analysis of MΦ precursors in the YS of E 9.0 Cx3cr1GFP/WT animals (A1: CD45+c-kitloCX3CR1loF4/80lo, A2: CD45+c-kit-CX3CR1+F4/80+). Mean fluorescent intensity (MFI) is presented. Four mice were investigated showing similar results. D. Fate-mapping strategy to target the A2 progenitor population preferentially in the YS. Male Cx3cr1CreER:R26-yfpmice were bred to wild-type animals and plug positive mice were injected at E 9.0 with tamoxifen (TAM). Offspring was analyzed at E 16.0, P0 or P42. E. Flow cytometry of F4/80hiCD11blo and F4/80loCD11bhi tissue MΦ in adult (P42) Cx3cr1CreER:R26-yfpmice. Cre negative mice served as a control. Representative pictures from three independent experiments are displayed.F. Quantification of YFP labeling in tissue MΦ at E16.0, P0 and P42 after TAM application at E 9.0 in plug positive Cx3cr1CreER:R26-yfpmice. Data are normalized to YFP+ microglia and presented at mean ± s.e.m. At least one experiment out of two is shown and each symbol presents one mouse. G. Hierarchical clustering of the 1.000 most variable probes within the dataset of embryonic development of yolk sac-derived tissue MΦ (EMP: erythromyeloid precursor, A1/A2: yolk sacMΦ progenitors, embryonic microglia (Emb F4/80 MG), embryonic Kupffer cells (Emb F4/80liver), embryonic (Emb) F4/80kidney cells. Heat map displays z-transformed log2-expression values from red to blue via white. H. Above: Sample-wise co-regulation network of embryonic MΦ development. Pearson correlation threshold of 0.92 was used to generate the network. Below: scheme thereof. \n",
      ".Figure 4(A) (Ai) Western blot analysis was used to confirm UHMK1 overexpression and ATF4 knockdown in BGC823 cells. (Aii) Silencing ATF4 significantly decreased the levels of the indicated metabolites in BGC823 cells overexpressing UHMK1(three biological replicates). (Aiii) Silencing ATF4 significantly decreased the levels of RNA and DNA containing U-14C-glycine in BGC823 cells overexpressing UHMK1(three biological replicates). (B) LC-MS/MS was used to examine the metabolites in SGC7901 cells with or without UHMK1 knockdown or ATF4 reintroduction. The data are shown in the heatmap. (C) Silencing ATF4 significantly decreased the levels of the indicated genes in BGC823 cells overexpressing UHMK1. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. (D) (Di) qRT-PCR was used to measure the expression of ATF4 in BGC823 cells overexpressing UHMK1. (Dii) UHMK1 overexpression enhanced the nuclear translocation of ATF4 in BGC823 cells. (Diii) ChIP assays on the ATIC and PPAT promoters were performed in BGC823 cells with the indicated antibody. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (E) Silencing ATF4 markedly decreased the proliferation of BGC823 cells overexpressing UHMK1. (left panel) The quantitative number of colonies formed in scramble+Flag, scramble+Flag-UHMK1, siATF4+Flag and siATF4+Flag-UHMK1 group. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. (right panel) Representative images of colonies are shown. Scale bars=5mm. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (F) Silencing ATF4 markedly decreased the migration of BGC823 cells with UHMK1 overexpression. (left panel) The quantitative number of migrated cells in scramble+Flag, scramble+Flag-UHMK1, siATF4+Flag and siATF4+Flag-UHMK1 group. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. (right panel) Representative images of migrated cells in each group are shown. Scale bars=100μm. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. (G) Silencing ATF4 markedly decreased the invasion of UHMK1 overexpressing BGC823 cells. (left panel) The quantitative number of invaded cells in scramble+Flag, scramble+Flag-UHMK1, siATF4+Flag and siATF4+Flag-UHMK1 group. Data were presented as mean ± standard deviation from three replicates. Unpaired, two-tailed statistical significance was assessed by Student's t‐test. (right panel) Representative images of invaded cells in each group are shown. Scale bars=100μm. Data information: *P< 0.05, **P< 0.01, ***P< 0.001, # marked no significance. \n",
      ".Figure 2A. Scheme of fate-mapping in Cx3cr1CreER:R26-yfpmice that were either treated at six weeks of age (for B) or at postnatal day 1 (for C) and analyzed at 6 days, 2, 4 and 35 weeks for B and 6 days, 6 and 35 weeks for C. B and C. Flow cytometry quantifications of YFP+ MΦ in the brain, liver and kidney at the indicated time points after TAM application in Cx3cr1CreER:R26-yfpmice. Data are summarized from two independent experiments (7-8 mice each) and represented as mean ± s.e.m. D. CCR2 expression on F4/80hi and CD11bhi MΦ in the liver or kidney of adult Ccr2RFP/WT mice, respectively. Grey area depicts fluorescence signal in MΦ of non-transgenic litter mates. Representative cytometric graphs are shown. Four mice were investigated showing similar results. E. Flow cytometry analysis of MΦ populations in the liver or kidney in adult mice lacking Ccr2 or Nr4a1. Each symbol represents one mouse. Data are summarized from three independent experiments and represent mean ± s.e.m. Significant differences were determined by an unpaired t test. n.s.: not significant. *P < 0.05, **P < 0.01. \n",
      ".Figure 5A-C. Three-dimensional reconstruction (left) and Imaris-based morphometric analysis of CX3CR1+ brain microglia or F4/80+ liver and kidney MΦ in adult Irf8-/- or Irf-8+/+ mice, respectively. Quantification of cell density of Iba1+ microglia and F4/80+ liver and kidney MΦ in adult Irf8-/- or Irf-8+/+ mice, respectively (left). Scale bars = 10 µm. For morphological measurements each symbol represents one mouse with at least six examined cells per mouse. Data represent mean ± s.e.m. Significant differences were examined by an unpaired t-test and marked with asterisks (*P < 0.05, **P < 0.01, ***P < 0.001); n.s.: not significant. \n",
      ".Figure 3A. Hierarchical clustering created on the 1.010 most significantly differentially expressed (DE) genes between embryonic (Emb) F4/80hi and CD11bhi cells. Heat map displays z-transformed log2-expression values from red to blue via white. B. Hierarchical clustering based on the 755 most significantly differentially expressed genes between adult F4/80hi and CD11bhi cells. Heat map shows z-transformed log2-expression values from red to blue via white. C. Scheme describing the compared samples of F4/80hi (microglia, liver, kidney) and CD11bhi (liver, kidney) tissue MΦ populations for analysis shown in D-I. D. Venn diagrams overlaying the genes which are jointly expressed in F4/80hi or in CD11bhi cells at similar levels irrespective of tissue origin ('non-differentially expressed [non-DE] genes') defined by a false discovery rate (FDR) adjusted p-value > 0.95 for both embryonic and adult cells separately. Numbers in parentheses indicate the amount of all genes for each MΦ population. E. Venn diagram shows the differentially expressed (DE) genes between adult Langerhans cells (LC) and adult F4/80hi or CD11bhi cells (functional connectivity |FC| > 1.5, FDR adjusted p-value < 0.05). F. Number of up- and down-regulated genes in F4/80hi versus CD11bhi populations (|FC| > 1.5, FDR adjusted p-value < 0.05) from genes fulfilling the criteria in (D) i.e. non-DE genes between organs for each group. G. Hierarchical clustering of embryonic F4/80hi and CD11bhi cells based on 43 DE-genes in embryonic cells visualized in F. Heat map exhibits z-transformed log2-expression values from red to blue via white. H. Hierarchical clustering of adult F4/80hi and CD11bhi cells based on 209 DE-genes in adult cells visualized in F. Heat map presents z-transformed log2-expression values from red to blue via white. I. Quantitative RT-PCR of differentially expressed genes depicted in G-H. Analyses of differentially expressed genes in embryonic (Hbb-y and Hba-x) or adult (Rnf135, March8, Slc7a11, Ptgs2, Flt3, Ccr2, Maf and Csf1r) tissue macrophages. Data are normalized to β-actin and GAPDH and expressed as ratio of the mRNA expression compared to microglia and show mean ± s.e.m. Three mice per group were analyzed. \n",
      ".Figure 6A. Number of differentially expressed (DE) genes between Irf8+/+ and Irf8-/- mice in several embryonic and adult MΦ cell populations. DE genes were identified by a one-way ANOVA model (|FC| > 1.5, FDR adjusted p-value < 0.05). The absolute numbers were normalized to the number of genes present in embryonic or adult cells. B. Venn diagrams of DE-genes between Irf8+/+ and Irf8-/- CD11bhi cells from adult or embryonic liver and kidney. Commonly up- (red area) and down-regulated (blue area) genes between cells derived from Irf8+/+ and Irf8-/- mice are shown. C. Venn diagrams of DE-genes for F4/80hi cells from adult or embryonic liver, kidney and brain. D. Gene Ontology (GO) network built on common DE genes in embryonic (left panel) and adult (right panel) F4/80hi (upper row) CD11bhi (lower row) cells from microglia, liver and kidney identified in at least two organs (identified in B and C). Red symbols represent GO terms associated with up-regulated genes in Irf8-/- compared to Irf8+/+ mice while blue edges indicate down-regulation. E. Quantitative qRT-PCR of selected genes in tissue MΦ from adult Irf8-/- and Irf8+/+ mice. Data are normalized to β-actin and GAPDH and expressed as ratio of the mRNA expression compared to microglia of Irf8+/+ mice or to liverCD11bhigh cells of Irf8+/+ mice and show mean ± s.e.m. Three mice per group were analyzed. \n",
      ".Figure 4A. Quantitative qRT-PCR of transcription factors during MΦ development. Data are normalized to β-actin and GAPDH and expressed as ratio of the mRNA expression compared to embryonic microglia and show mean ± s.e.m. Two to three mice per group were analyzed. B. Irf8-VENUS expression on A1 and A2 progenitor cells isolated from the E 9.0 yolk sac of Irf8-Venus PAC reporter mice. Representative blots obtained from 17 individual embryos are shown. Mean fluorescence intensity (MFI) of Irf8-VENUS signal in A1 and A2 progenitor cells. Each symbol represents one mouse. Data are from one representative experiment out of two and represented as mean ± s.e.m. C. Flow cytometry measurement of the Irf8-VENUS signal in brain, liver and kidneyMΦ from E 14.0, postnatal (P1) and adult (P42) Irf8-VENUS PAC reporter mice. Representative density blots are shown (left) and quantification thereof (right). Each symbol represents one mouse. Data are from one representative experiment and displayed as mean ± s.e.m. D. Representative immunofluorescence microscopy images of Venus (green) and Iba-1+ (red) MΦ from brain, liver and kidney of adult Irf8-VENUS PAC reporter mice. Stars indicate the cell shown in the insert. Scale bar = 10 µm. \n",
      "a) Induction of alternative macroautophagy in WT and Atg5−/−, but not Ulk1gt/gt and DKO, erythroid cells.f2(a) Induction of alternative macroautophagy in WT and Atg5−/−, but not Ulk1gt/gt and DKO, erythroid cells. Erythroblasts and reticulocytes from the liver of embryonic mice (E18.5) were incubated with or without STS (1 μM) for 24 h, followed by staining with anti-Lamp2 (red), anti-Ter119 (green) and DAPI (blue). DAPI-positive erythroblasts and DAPI-negative reticulocytes are shown. Lamp2 image and merged image (DAPI, Ter119 and Lamp2) are shown. Scale bar, 1 μm. Large dots for Lamp2 are observed in STS-treated WT and Atg5−/− cells, but not STS-treated Ulk1gt/gt or DKO cells. (b) Representative electron micrographs of EC incubated with or without STS. Erythroid cells were harvested from the liver of embryonic mice (E18.5), incubated with or without STS (1 μM) for 24 h, and analysed by electron microscopy (EM). Scale bar, 1 μm. Insets of WT and Atg5−/− cells show mitophagy (Scale bar, 0.5 μm). Inset of a Ulk1gt/gt cell showing mitochondria that have not been engulfed (Scale bar, 0.5 μm). Arrows point to non-engulfed mitochondria and the arrowheads indicate engulfed mitochondria. (c-e) Quantitative analysis of mitophagy after STS treatment, calculated from EM photos. Population of reticulocytes with mitophagy (c), number of mitochondria per reticulocytes (d) and population of reticulocytes showing macroautophagy (e) were calculated (n>35 cells per mouse). The data are shown as mean±s.d. (n=3). *P0.05 versus value of WT (analysis of variance (ANOVA)); #P0.05 versus value of Atg5−/− (ANOVA); 'NS' indicates not significant versus value of WT (ANOVA). \n",
      "a) Expression of Ulk1 and Atg5–12 in the indicated erythroid cells.f1(a) Expression of Ulk1 and Atg5-12 in the indicated erythroid cells. (b-e) Induction of conventional macroautophagy in WT and Ulk1gt/gt, but not in Atg5−/−, embryonic erythroid cells. Ter119+ erythroid cells from WT, Atg5−/−, and Ulk1gt/gt embryonic mice (E18.5) were treated with 1 μM rapamycin (b,c) and 1 μM STS (d,e), and then harvested at the indicated times. (b,d) Representative protein expression of LC3 and p62 measured by western blot. Actin was a loading control. Uncropped images are shown in Supplementary Fig. 10. (c,e) Semi-quantitative analysis of LC3-II/LC3-I and p62 protein expression (n=3, mean±s.d.). Asterisks indicate a significant difference at P0.05 (analysis of variance (ANOVA)). #P0.05 versus value of WT 24 h (ANOVA). 'NS' indicates not significant (ANOVA). (b-e) Induction of conventional macroautophagy in WT and Ulk1gt/gt, but not in Atg5−/−, embryonic erythroid cells. Ter119+ erythroid cells from WT, Atg5−/−, and Ulk1gt/gt embryonic mice (E18.5) were treated with 1 μM rapamycin (b,c) and 1 μM STS (d,e), and then harvested at the indicated times. (b,d) Representative protein expression of LC3 and p62 measured by western blot. Actin was a loading control. Uncropped images are shown in Supplementary Fig. 10. (c,e) Semi-quantitative analysis of LC3-II/LC3-I and p62 protein expression (n=3, mean±s.d.). Asterisks indicate a significant difference at P0.05 (analysis of variance (ANOVA)). #P0.05 versus value of WT 24 h (ANOVA). 'NS' indicates not significant (ANOVA). \n",
      "a) Three-colour flow cytometry of erythroid cells from WT embryonic liver (E14.f3(a) Three-colour flow cytometry of erythroid cells from WT embryonic liver (E14.5) stained with anti-Ter119, anti-CD71 and Mitotracker Deep Red. (Left upper panel) Ter119 versus CD71 fluorescence. (Other panels) FSC versus Mitotracker fluorescence of the R1 (right upper panel), R2 (left lower panel) and R3 (right lower panel) fractions. (b,c) Percentage of cells with a low number of mitochondria (mitochondrialow cells) among CD71−Ter119+ EC. Erythroid cells from the liver (E14.5 and E18.5) and peripheral blood (E18.5) of indicated mice were stained with anti-Ter119, anti-CD71 and Mitotracker Deep Red. (b) Representative histograms of mitochondrial content in CD71−Ter119+ cells. Numbers indicate the population of mitochondrialow cells. (c) The percentage of CD71−Ter119+ cells without mitochondria was determined by gating the mitochondrialow fraction, as shown in (b) (mean±s.d., n=6). *P0.05 versus value of WT (analysis of variance (ANOVA)); #P0.05 versus value of Atg5−/− (ANOVA); 'NS' indicates not significant versus value of WT (ANOVA). (d) Percent mitochondrialow cells in the CD71−Ter119+ EC of Atg7F/FCre embryo. Erythroid cells from the liver and peripheral blood of WT and Atg7F/FCre embryo (E18.5) were stained with anti-Ter119, anti-CD71, and Mitotracker Deep Red (mean±s.d., n=3). *P0.05 versus value of WT (Student's t-test). (e) Percentage of mitochondrialow cells in Syto16low cells. Liver erythroid cells (E18.5) were stained with Syto16 (DNA) and Mitotracker Deep Red. Representative dot plots of the mitochondrial content of Syto16low cells are demonstrated in Supplementary Fig. 5, the percentage of Syto16low cells without mitochondria was determined by gating the mitochondrialow fraction (mean±s.d., n=6). *P0.05 versus value of WT (ANOVA); #P0.05 versus value of Atg5−/− (ANOVA); 'NS' indicates not significant versus value of WT (ANOVA). (f) Immunofluorescent analysis of mitochondrial and lysosomal proteins. Liver Ter119+ cells (E18.5) of the indicated mice were stained with anti-Tom20 (mitochondrial marker) and anti-Lamp2 (lysosomal marker) antibodies, and observed by fluorescent microscopy. Scale bar, 2 μm. Green, red, white, and blue indicate Tom20, Lamp2, Ter119 and DAPI (DNA), respectively. Mitochondrial and lysosomal markers have separate distributions in Ulk1gt/gt cells and DKO cells. \n",
      "a–f) Representative EM of WT (a), ATG5−/− (b), Ulk1gt/gt (c,d) and DKO (e,f) reticulocytes from embryonic liver (E14.f4(a-f) Representative EM of WT (a), ATG5−/− (b), Ulk1gt/gt (c,d) and DKO (e,f) reticulocytes from embryonic liver (E14.5). (a,b) Mitophagy was detected in WT and ATG5−/− cells (scale bar, 1 μm). Insets indicate mitophagy (scale bar, 0.2 μm). (c,e) In Ulk1gt/gt and DKO cells, numerous mitochondria were in contact with membrane structures (scale bar, 1 μm). Enlarged images are shown on the right side (scale bar, 0.2 μm). (d,f) Some Ulk1gt/gt and DKO cells showed plasma membrane blebs containing mitochondria (scale bar, 1 μm). (g,h) Quantitative analysis of mitophagy calculated from EM photos in the embryonic liver at E18.5. Population of reticulocytes with mitophagy (g), and number of mitochondria per reticulocytes (h) were calculated (n>35 cells per mouse). The data are shown as mean±s.d. (n=3). *P0.05 versus value of WT (analysis of variance (ANOVA)); #P0.05 versus value of Atg5−/− (ANOVA); 'NS' indicates not significant versus value of WT (ANOVA). \n",
      ".Figure 1(A) Schematic of low intensity exercise regimen. (B, C) Exercise capacity of MCK-KO and WT mice from the low intensity regiment was conducted. (n = 4 WT, n = 5 KO mice, mean ±SEM, * p < 0.05, two-tailed student's t-test). (D) Schematic of high intensity exercise regimen. (E, F) Exercise capacity of MCK-KO and WT mice from the high intensity regiment (n = 5 mice, means ± SEM, two-tailed student's t-test). \n",
      "a) Erythroblasts (EB; mitochondriahigh Syto16high), reticulocytes (RC; mitochondriahigh Syto16low) and EC (EC; Ter119+ PBC) were collected from the indicated embryos (E18.f5(a) Erythroblasts (EB; mitochondriahigh Syto16high), reticulocytes (RC; mitochondriahigh Syto16low) and EC (EC; Ter119+ PBC) were collected from the indicated embryos (E18.5), and the expression of each protein was examined by western blot. The asterisk indicates a nonspecific band. Uncropped images are shown in Supplementary Figs 11 and 12. (b) Semi-quantitative analysis of the expression level of each protein and the LC3-II/LC3-I ratio (mean±s.d., n=3). *P0.05 versus value of WT EC (analysis of variance (ANOVA)); #P0.05 versus value of Atg5−/− EC (ANOVA); 'NS' indicates not significant versus value of WT EC (ANOVA). (c) Modification of Ulk1 and Atg13 during erythrocyte maturation. Cell lysates were treated with λ-phosphatase (400 U) for 45 min at 30 °C, and band shift was examined by anti-Ulk1 and anti-Atg13 antibodies. Asterisk indicates non-specific band. Uncropped images are shown in Supplementary Fig. 12. \n",
      ".Figure 4(A, B) (A) RNA-Seq was performed in WT and MCK-KO soleus muscle at rest. The scatter plots show differentially expressed genes in MCK-KO soleus muscle at sedentary (A) and after a bout of low-intensity exercise for 50 min (B) (FDR<0.05, p < 0.05). (C) Proposed model for ROS regulation during loss of Dnmt3a. Loss of DNMT3A increases ALDH1L1 expression, thus leading to the increase in NADPH levels and increased activity of NADPH oxidase, leading to increased ROS levels. The increased oxidative stress contributes to mitochondrial dysfunction and muscle fatigue. (D) Q-PCR validation of Aldh1l1 at sedentary and after exercise for 50 min (n = 6, means ± SEM, * p < 0.05, two-tailed student's t-test). (E, F) ALDH1L1 protein expression and the quantification in WT vs. MCK-KO soleus, GA and EDL muscles at sedentary and normalizing to GAPDH using ImageJ. (G) The map of CpG-rich promoter regions of Aldh1l1 and MeDIP and ChIP primers (P1-P5) that cover the CpG rich regions. The numbers correspond to the position from the transcriptional start site of Aldh1l1. (H) MeDIP-qPCR was performed in WT and MCK-KO soleus muscle to assess differential methylation using primer sets from (G) (n = 4, means ± SEM, * p < 0.05, two-tailed student's t-test). (I) DNMT3A ChIP-PCR was conducted in WT and MCK-KO soleus muscles using primer sets from (G) (n = 3, means ± SEM, * p < 0.05, two-tailed student's t-test). (J) H3K27ac ChIP-PCR was conducted in WT and MCK-KO soleus muscles using primer sets from (G) (n = 3, means ± SEM, * p < 0.05, two-tailed student's t-test). \n",
      ".Figure 2(A) The H&E staining of MCK-KO and WT soleus muscle after a bout of low intensity treadmill running (top 40X, bottom 100X magnifications). Black arrow indicates centralized nuclei. (B) The percentage of myofibers with centralized nuclei was determined by manual counting 300 myofibers in 20X magnification. (n = 3, means ± SEM, * p < 0.05, two-tailed student's t-test). (C-D) The analogous set of data is shown with WT and MCK-KO GA muscles. (n = 3, means ± SEM, two-tailed student's t-test. GA: Gastrocnemius) Hydrogen peroxide (H2O2) levels were measured in WT and MCK-KO soleus (E) at rest and after a bout of exercise (n = 6 per group, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). Hydrogen peroxide (H2O2) levels were measured in WT and MCK-KO , GA (F), and EDL (G) at rest and after a bout of exercise (n = 6 per group, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). (H) Serum levels of lactate were measured before and after a bout of low-intense exercise for 50 mins in WT and MCK-KO mice (n = 16 for sedentary and n = 14 for exercise groups, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). \n",
      "a) Morphology of cultured erythroid cells on day 0, 1 and 3 of differentiation.f6(a) Morphology of cultured erythroid cells on day 0, 1 and 3 of differentiation. Erythroblasts were purified from embryonic liver (E14.5) and incubated for the indicated times in erythrocyte differentiation medium. The cells were stained with a Diff-quick kit. (b) Percentage of cells without nuclei (Syto16low cells) on the indicated days of differentiation (mean±s.d., n=6). Loss of nuclei occurred normally in all mice. (c,d) Percentage of Syto16low cells without mitochondria. Differentiated cells were stained with Syto16 and Mitotracker Deep Red. Representative histograms of mitochondrial content in Syto16low cells on day 6 are shown in (c). Numbers indicate the population of mitochondrialow cells. Representative dot plots are also available in Supplementary Fig. 13. (d) Percentage of Syto16low cells lacking mitochondria determined by gating the mitochondrialow fraction as shown in (c) (mean±s.d., n=6). *P0.05 versus value of WT (analysis of variance (ANOVA)); #P0.05 versus value of Atg5−/− (ANOVA); 'NS' indicates not significant versus value of WT (ANOVA). (e-h) Impact of various drugs on mitochondrial clearance during in vitro differentiation. Differentiated cells from WT embryos were incubated with or without wortmannin (e), 3-methyladenine (3-MA) (f), rapamycin (g) or compound C (h) at the indicated concentrations and time periods. Percentage of mitochondrialow cells/Syto16low cells was measured by flow cytometry (mean±s.d., n=3). *P0.05 versus value of no drug (ANOVA). (i) Impact of various drugs on mitochondrial clearance during in vitro differentiation.with cells from Atg5−/− embryos exposed to wortmannin or 3-MA (mean±s.d., n=3). *P0.05 versus value of no drug (ANOVA). \n",
      ".Figure 3(A-D) Succinate dehydrogenase staining was performed in WT and MCK-KO soleus at sedentary (A, B) and after a bout of low-intensity exercise for 50 min (C, D) (10X, 20X magnifications), and the staining intensity was quantified using ImageJ (n = 6, means ± SEM, * p < 0.05, two-tailed student's t-test). (E-H) The analogous set of data is shown with WT and MCK-KO GA muscles. (n = 6, means ± SEM, * p < 0.05, two-tailed student's t-test). (I-L) The analogous set of data is shown with WT and MCK-KO EDL muscles. (n = 6, means ± SEM, two-tailed student's t-test. EDL: Extensor digitorum longus) (M, N) Mitochondrial respiration was measured in WT and MCK-KO soleus tissue after a bout of low-intensity exercise for 50 min under basal conditions and in response to 4 mM oligomycin (complex V inhibitor), 4 mM FCCP (uncoupler), or 4 mM each of rotenone and antimycin A (complex I inhibitor) (n = 8, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). (O, P) Mitochondrial respiration was measured in Dnmt3a knocked down L6 myotubes which were transduced with lentiviral under basal conditions and in response to 4 mM oligomycin (complex V inhibitor), 4 mM FCCP (uncoupler), or 4 mM rotenone and antimycin A (complex I inhibitor) (n = 5, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). \n",
      "a) Number of EC and haemoglobin content in the blood of the indicated mice (10–20 weeks old; mean±s.f7(a) Number of EC and haemoglobin content in the blood of the indicated mice (10-20 weeks old; mean±s.d., n=6). *P0.05 versus value of WT (Student's t-test). (b) Analysis of early erythrocyte development. Bone marrow cells were harvested from the indicated mice (10-20 weeks old) and stained with anti-cKit and anti-Sca1 antibodies. The population of each cell type was counted by FACS (mean±s.d., n=3). *P0.05 versus value of WT (analysis of variance (ANOVA)). (c) Percentage of mitochondria-containing Ter119+ blood cells. Cells in peripheral blood from the indicated mice (10-20 weeks old) were stained with anti-Ter119 antibody together with Mitotracker Deep Red. The population of mitochondriahigh cells among Ter119-positive cells was counted by FACS (mean±s.d., n=3). *P0.05 versus value of WT (ANOVA). (d) Percentage of mitochondria-containing Syto16low blood cells. Cells in peripheral blood from the indicated mice (10-20 weeks old) were stained with Syto16 and Mitotracker Deep Red. The population of mitochondriahigh cells among Syto16-negative cells was counted by FACS (mean±s.d., n=3). *P0.05 versus value of WT (ANOVA). (e) Representative electron micrographs of bone marrow reticulocytes in 10 week-old WT and Ulk1gt/gt mice (scale bar, 1 μm). Insets indicate mitophagy (scale bar, 0.2 μm). (f,g) Erythroblasts (EB; mitochondriahigh Syto16high), reticulocytes (RC; mitochondriahigh Syto16low) and EC (EC; Ter119+ PBC) were collected from the indicated mice (10-20 weeks old), and the expression of each protein was examined by western blot. Uncropped images are shown in Supplementary Fig. 14. (g) Semi-quantitative analysis of the expression level of each protein (mean±s.d., n=3). 'NS' indicates not significant versus value of WT EC (ANOVA). (h,i) The impact of Ulk1 on mitochondrial clearance during stress erythropoiesis. (h) Representative images of erythrocyte in PHZ-treated WT and Ulk1gt/gt mice (14 weeks old). Blood cells were stained with anti-Ter119 (erythroid cell marker) and anti-Tom20 (mitochondrial marker) antibodies and observed by fluorescence microscopy. Green and red indicate Ter119 and Tom20, respectively. Scale bar, 10 μm. (i) Percentage of mitochondrialow cells among Syto16low cells. Erythroid cells were stained with Syto16 (DNA) and Mitotracker Deep Red (mean±s.d., n=3). *P0.05 versus value of WT (Student's t-test). Mitochondrial retention was observed in Ulk1gt/gt mice. \n",
      ".Figure 6(A, B) Exercise capacity of MCK-KO and WT mice treated with PBS (vehicle) or NAC (200mg/kg, i.p). (n = 4 WT and n = 6 MCK-KO , mean ±SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA). (C) Hydrogen peroxide (H2O2) levels (n = 6 for Control groups and Dnmt3a KD with NAC treatment group n = 4 Dnmt3a KD, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). (D, E) mitochondrial respiration in Dnmt3a knockdown and control L6 myotubes treated with NAC (3mM) or vehicle treatment for 24hrs (n = 5 Control groups n = 3 Dnmt3a KD, n = 4 Dnmt3a KD with NAC treatment groups. means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). \n",
      ".Figure 5(A, B) NADPH levels (A) and NOX (B) level was measured in WT and MCK-KO muscles at rest and after a bout of low-intensity exercise for 50 min (n = 6, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). (C-G) L6 myotubes were transduced with lentiviral expression plasmids for Flag-ALDH1L1 and GFP. NADPH levels (C), H2O2 levels (D, E) , Mitochondrial respiration (F, G) were measured from these cells in the presence and absence of NADPH oxidase inhibitor apocynin (n = 3 per group, for (C-E) and n = 5 for (F, G), means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). Single and double knockdowns of Dnmt3a and Aldh1l1 in L6 myotubes were achieved by lentiviral transduction. NADPH levels (H) and NOX activity (I) were measured in single and double knockdowns of Dnmt3a and Aldh1l1 in L6 myotubes. (n = 3, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). (J, K) H2O2 levels were measured in single and double knockdowns of Dnmt3a and Aldh1l1 in L6 myotubes. (J: n = 3 and K: n = 6 for Control, Aldh1l1 KD, Aldh1l1 KD + Dnmt3a KD, n = 5 for Dnmt3a KD, means ± SEM, * p < 0.05, two-tailed student's t-test and two-way ANOVA followed by Bonferroni post-hoc testing). \n",
      ".Figure 1Structure of PRN1126, a reversible covalent selective LMP7 inhibitor. Hydrolysis of fluorogenic substrates (Suc-LLVY-AMC) for the chymotrypsin-like activity of human (B) or mouse (C) 20S constitutive proteasome (CP) or 20S immunoproteasome (IP) at various PRN1126 concentrations. Data are presented as the means ± s.d. from quadruplicate assays. The highest fluorescence value was set to 100%. The experiments were repeated twice with similar results. \n",
      ".Figure 7(A) Aldh1l1 mRNA expression in various muscles in MCK-KO mice that were transfected with gRNA DNAs against Aldh1l1 or control. (n = 4 for Soleus, n = 3 GA gCont and n = 4 GA gAldh1l1, n = 4 EDL gCont and n = 3 EDL gAldh1l1 , means ± SEM, * p < 0.05, two-tailed student's t-test). (B, C) ALDH1L1 protein expression and the quantification in MCK-KO muscles that were transfected with gCont vs. gAldh1l1 using ImageJ. (n = 3, means ± SEM, * p < 0.05, two-tailed student's t-test). (D, E) Exercise capacity of WT and MCK- KO with or without transfection of gCont vs. gAldh1l1 under the low intensity regimen (n = 4 WT + gCont, n = 9 MCK- KO + gCont, and MCK- KO + gAldh1l1, *p<0.05, means ± SEM, two-tailed student's t-test and one-way Anova). \n",
      ".Figure 3Hydrolysis of fluorogenic substrates (Ac-PAL-AMC) for LMP2-activity of human (upper panel) or mouse (lower panel) 20S immunoproteasome at various concentrations of PRN1126 or ONX 0914. Data are presented as the means of fluorescence ± s.d. from quadruplicate assays. The experiments were repeated three times with similar results. Altered electrophoretic mobility of IP subunits by covalent modification with ONX 0914. Ficoll enriched lymphocytes from C57BL/6 mice were treated with DMSO or 300 nM ONX 0914 for 2 h in vitro. SDS-PAGE and immunoblotting against indicated proteins was performed. Shown are representative western blots out of three independent experiments with similar outcome. \n",
      ".Figure 2Presentation of UTY246-254 on splenocytes after exposure to indicated concentrations of PRN1126. Data are presented as the mean absorbance ± s.d. of three replicate cultures. The experiment has been performed twice, yielding similar results. Flow cytometry analysis of H-2Kb surface expression on splenocytes derived from C57BL/6 mice treated with the indicated concentrations of PRN1126 overnight. Pooled data from three independent experiments (n=9) are shown as the means of median fluorescent intensity ± SEM. Splenocytes from C57BL/6 mice (C) were exposed (continuous treatment) to 300 nM PRN1126 or vehicle (DMSO) or 300 nM ONX 0914 and stimulated with LPS overnight. IL-6 concentrations in the supernatant were analyzed by ELISA. (C) IL-6 concentrations are presented as mean and individual data points from triplicate wells. The experiment has been performed twice, yielding similar results. Splenocytes from human PBMCs (D) were exposed (continuous treatment) to 300 nM PRN1126 or vehicle (DMSO) or 300 nM ONX 0914 and stimulated with LPS overnight. IL-6 concentrations in the supernatant were analyzed by ELISA. Data are presented as single dots from three independent donors. The highest cytokine concentration was set to 100%. *P < 0.05. Unpaired student´s t-test. Colitis was induced by oral administration of 3 % DSS for 5 d. Mice were treated daily (s.c.) with either PRN1126 (40 mg/kg), or ONX 0914 (10 mg/kg), or vehicle. Data points represent mean ± SEM of 15 mice pooled from three independent experiments. Mice were immunized with MOG35-55 peptide and were monitored daily for clinical symptoms of EAE. Mice were treated three times a week (s.c.) with either PRN1126 (40 mg/kg), or ONX 0914 (10 mg/kg), or vehicle. All data were statistically compared to the vehicle treated group. *P < 0.05, **P < 0.01, ***P < 0.001. Two-way ANOVA. Shown are the means of the clinical scores ± SEM (n = 6 per group). \n",
      ".Figure 4Flow cytometry analysis of H-2Kb surface expression on splenocytes derived from C57BL/6 wild type (left side) or LMP7-/- (right side) mice treated with vehicle (DMSO), ONX 0914 (300 nM), PRN1126 (300 nM), ML604440 (300 nM), LU-001i (300 nM), PRN1126+ML604440 (300 nM each), or PRN1126+LU-001i (300 nM each) overnight. Unstained splenocytes were used as negative control. Pooled data from three independent experiments (n=9) are shown as the means of median fluorescent intensity ± SEM. All data were statistically compared to the DMSO treated group. **P < 0.01, ***P < 0.001. One-way ANOVA. \n",
      ".Figure 1ChEP analysis using EGFP (control)- or Piwi-KD OSCs. Signals with an adjusted p-value (Bonferroni test) of less than 0.05 are highlighted in red. The signals for Lam, LamC, and H1 are labelled within the plots. Gene ontology terms related to proteins with a significant change upon Piwi-KD using ChEP analysis (adjusted p-value <0.05). Gene ontology terms from the biological process (blue), cellular component (orange), and molecular function (green) categories are listed along with their p-values (Fisher's Exact test, -log10). Western blot showing Piwi, Lam, LamC, and Tubulin (loading control) protein levels upon EGFP (control)-, Piwi-, Lam-, or LamC-KD. Note that KD of Lam or LamC is not complete since their depletion causes serious cell damage. Immunofluorescence of OSCs transfected with indicated siRNAs, using Lam and LamC antibodies (green). DAPI staining (gray) shows the location of nuclei. Scale bar: 5μm. Lam- or LamC-KD followed by qRT-PCR of RNA levels for piRNA target (mdg1 and gypsy) or non-target (roo) TEs. Error bars indicate SD (n=3). * = p-value<0.05 (t-test). Note that the complete depletion of Lam and LamC is impossible since their KD causes severe damage to cells. \n",
      ".Figure 6Colitis was induced by oral administration of 3% DSS. Mice were treated daily (s.c.) with LU-001i (15 mg/kg), PRN1126 (40 mg/kg), PRN1126+ LU-001i (40 mg/kg+15 mg/kg), or vehicle starting from the begin of the experiment. Data points represent means ± SEM of 15 mice pooled from three independent experiments. (A) The body weight of individual mice was monitored daily and the percent weight loss (y-axis) was plotted versus time (x-axis). All data were statistically compared to the vehicle treated group. * Colitis was induced by oral administration of 3% DSS. Mice were treated daily (s.c.) with LU-001i (15 mg/kg), PRN1126 (40 mg/kg), PRN1126+ LU-001i (40 mg/kg+15 mg/kg), or vehicle starting from the begin of the experiment. Data points represent means ± SEM of 15 mice pooled from three independent experiments. On day 9 after initiation of DSS treatment, colon lengths were measured (n=15). Naïve mice (n=5) were used as healthy controls. All data were statistically compared to the vehicle treated group. Mice were immunized with MOG35-55 peptide and were monitored daily for clinical symptoms of EAE. Mice were treated intermittently with three times a week (s.c.) schedule with LU-001i (15 mg/kg), PRN1126 (40 mg/kg), PRN1126+ LU-001i (40 mg/kg+15 mg/kg), or vehicle starting from the start of the experiment. Data points represent the means of the clinical scores ± SEM of 12 mice pooled from two independent experiments. All data were statistically compared to the vehicle treated group. \n",
      ".Figure 3Hi-C interaction matrices of the Control (EGFP)-KD and Piwi-KD OSCs at Chromosome 3L (40k resolution). Differential interaction heatmap for chromosome 3L (40k resolution), showing increased (red) and decreased (blue) interacting bins in log2 fold change upon Piwi-KD. Differential interaction heatmap for 16-19Mbp of chr3L shown in log2 fold change. piRNA target TE insertions are indicated at the bottom (green filled triangles), along with TADs defined using Hi-C interactions of Control (EGFP)-KD OSCs (black triangles). Differential intra-TAD interaction density score upon Piwi-KD (upper), centering TE insertions. Heatmap showing changes in the intra-TAD interaction score of each piRNA target TE insertion sites (lower). The results are shown for piRNA target TE and non-piRNA target TE insertions. Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in intra-TAD interactions upon Piwi-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in inter-TAD interactions upon Piwi-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Oligo-FISH images for changes in the intra-TAD interaction at the TAD described in (C), upon EGFP (control)- or Piwi-KD. Gray indicates DAPI staining for DNA, green and magenta indicates FISH signal at the different positions of TADs (TAD-1 and TAD-2). Details of TAD probe design is described in Fig EV2C. Scale bar, 5μm. Distance between two FISH signals is quantified at right, dot in the box plot indicates mean value; Boxplot whiskers, box, and central band show minimum, first quartile, median, third quartile, and maximum, respectively. P-values calculated with Mann-Whitney U test. Diagram illustrating the possible loss of the intra-TAD interaction for TADs harboring piRNA target TEs. Differential interaction heatmap for 16-19Mbp of chr3L shown in log2 fold change. TADs defined using Hi-C interactions of EGFP-KD (control, upper right) or Piwi-KD (lower left) OSCs are overlaid on the heatmap. Venn diagram displaying the number of TAD boundaries detected using EGFP- or Piwi-KD Hi-C interactions. Dot plot showing the correlation between the change in coverage of LADs for each TAD (x-axis), and changes in intra-TAD interactions upon Piwi-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). \n",
      ".Figure 4Differential interaction heatmap for Piwi- and Control (EGFP)-KD shown in log2 fold change. Hi-C data is shown with RNA levels (RNA-seq signal density), H3K9me3 association (H3K9me3 ChIP-seq signal density), H3K27Ac association (H3K27Ac ChIP-seq signal density), H3K4me3 association (H3K4me3 ChIP-seq signal density), H1 association (H1 ChIP-seq signal density), and lamin association (Lamin DamID-seq signal density). The genomic region flanking the gypsy insertion (red dashed line) is shown for OSCs with EGFP- and Piwi-KD. FlyBase protein-coding genes are indicated, together with chromosome position. Boxplots showing fold changes in the histone mark association of piRNA target or non-target TEs based on ChIP-seq upon Piwi-KD (Definition of piRNA target and non-target TEs are described in Fig EV1E). Boxplot whiskers, box, and central band show maximum, third quartile, median, first quartile, and minimum, respectively (n=10 for piRNA target TEs, n= 80 for non-piRNA target TEs). P-values were calculated by Wilcoxon rank-sum test. Dot plot showing the correlation between changes in Lamin DamID signal upon Piwi-KD (x-axis) and changes of indicated ChIP-seq signal in each LADs upon Piwi-KD (y-axis). The trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Dot plot showing the correlation between changes in intra-TAD interactions upon Piwi-KD (x-axis) and changes of indicated ChIP-seq signal in each TADs upon Piwi-KD (y-axis). The trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). ChIP‐qPCR analysis of H3K27Ac and H3K4me3 occupancy on the reporter gene upon transfection of λN-Nxf2 expression vector or control EGFP expression vector. Bar graph shows the occupancy relative to that of the sample transfected with a control plasmid. Error bars indicate SD (n = 3), * = p-value<0.01 (t-test).. \n",
      ".Figure 2Scatterplot of DamID-seq signals (log2 ratio) for FlyBase genes (left) or genome integrated TEs (right) in EGFP-KD (control, x-axis) and Piwi-KD (y-axis) samples. Both x- and the y-axis is a log2 ratio. Gypsy signal with the greatest decrease upon Piwi-KD is indicated and shown in genome browser view in (B). DamID-seq signal densities are shown for OSCs with EGFP- and Piwi-KD at genomic regions harboring gypsy insertions with the greatest decrease upon Piwi-KD (indicated in (A)). Neighboring regions are annotated with protein-coding genes and LTR TEs. Note that this distribution of the reads includes multi-mapped reads. Boxplots showing fold changes in the expression of piRNA target or non-target TEs based on RNA-seq upon Piwi-KD (left), and the frequency of piRNAs, which target piRNA target TEs or non-piRNA target TEs (right). Definition of piRNA target and non-target TEs is in Fig EV1E. Boxplot whiskers, box, and central band show maximum, third quartile, median, first quartile, and minimum, respectively (n=10 for piRNA target TEs, n= 80 for non-piRNA target TEs). P-values were calculated using the Wilcoxon rank-sum test. Scatterplot as in (A) for EGFP-KD (control, x-axis) or Piwi-KD (y-axis) samples. Values were calculated for 90 TE consensus sequences (one value per one type of TE) (left). Scatterplot of RPM values for the same 90 TE consensus sequences in EGFP-KD (control, x-axis) or Piwi-KD (y-axis) samples examined by RNA-seq (right). The signals for mdg1, gypsy, and roo are labelled within the plots. Boxplot showing DamID-seq signals of piRNA target or non-target TEs upon EGFP (control)- or Piwi-KD. Boxplot whiskers, box, and central band show maximum, third quartile, median, first quartile, and minimum, respectively (n=10 for piRNA target TEs, n= 80 for non-piRNA target TEs). P-values were calculated using the Wilcoxon rank-sum test. Density plots for DamID-seq signals over consensus sequences from piRNA target (gypsy and mdg1, orange) and non-target TEs (roo, blue) in EGFP (control)- or Piwi-KD cells. Dot plot showing the correlation between the normalized density of indicated TEs per LAD (x-axis) and changes in Lam-Dam / Dam signals upon Piwi-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Oligo-FISH images for the representative piRNA target region described in Fig 4A, upon EGFP (control)- or Piwi-KD. Gray indicates DAPI staining for DNA, magenta indicates FISH signal, and green indicates Lamin immunofluorescence staining. Scale bar, 5μm. Ratio of distance from FISH signal to the nuclear periphery (DAPI surface) is quantified at right, dot in the box plot indicates mean value; Boxplot whiskers, box, and central band show minimum, first quartile, median, third quartile, and maximum, respectively. P-values calculated with Mann-Whitney U test. Diagram illustrating relocation of piRNA-target TE coding chromosome regions away from the nuclear periphery. \n",
      ".Figure 5Splenocytes from C57BL/6 mice (A) or human PBMCs (B) were incubated (continuous treatment) with DMSO, ONX 0914 (300 nM), PRN1126 (300 nM), ML604440 (300 nM), LU-001i (300 nM), PRN1126+ML604440 (300 nM each), or PRN1126+LU-001i (300 nM each) and stimulated with LPS overnight. The IL-6 concentration in the supernatant was analyzed by ELISA. The highest IL-6 concentration from each experiment was set to 100%. (A) Data are presented as the mean ± SEM of 4 independent experiments each measured in triplicates. (B) Data are presented as the mean ± SEM of 6 different healthy donors. All data were statistically compared to the DMSO treated group. The differentiation of CD4+ T cells isolated from the spleens of C57BL/6 mice that were exposed (continuous treatment) to DMSO, ONX 0914 (300 nM), PRN1126 (300 nM), ML604440 (300 nM), LU-001i (300 nM), PRN1126+ML604440 (300 nM each), or PRN1126+ LU-001i (300 nM each) and that were stimulated with plate bound antibodies to CD3/CD28 in the presence of TGF-β and IL-6 and scavenging antibodies neutralising IL-4 and IFN-γ was measured in 3 days cultures. IL-17 expression was detected by intracellular cytokine staining after a short restimulation with PMA/Ionomycin. IL-17 expression in activated T cells cultured in the absence of Th17-polarizing conditions is shown as a comparison (no cocktail). Values reflect the percentage of CD4+ cells that are IL-17A+. Data are presented as the means ± SEM of 3 independent experiments each measured in triplicates. All data were statistically compared to the DMSO treated group. \n",
      ".Figure 1(A) HT29 cells were infected with Shigella WT, S325, or ospD deletion mutants, and incubated for 8 h. Aliquots of cellular supernatants were subjected to cytotoxicity assays. *P<0.05 (one-way ANOVA). (B) HT29 cells were infected with Shigella WT or ∆ospD3 and incubated for 8 h. Infected cells were fixed and subjected to TUNEL and PI staining. Percentages of positive cells (TUNEL, green; PI, red) are shown in graph at right. The nuclei were stained with DAPI (blue). Scale bar: 100 μm. n.s., not significant; *P<0.05 (unpaired two-tailed Student's t-test). (C) HT29 cells were infected with Shigella WT, S325, or ∆ospD3 and incubated for 8 h. Infected cells were subjected to Giemsa staining. Arrows indicate cells in which the cytoplasm disappeared. Scale bar: 20 μm. \n",
      ".Figure 6Differential interaction heatmap for chromosome 3L (40k resolution) in log2 fold change, showing increased (red) and decreased (blue) interacting bins upon H1(left)-, Egg(middle)-, or HP1a(right)-KD. Relative contact probability (RCP) plot showing corrected contact counts at different genomic ranges, genome widely. RCP values are compared between EGFP-KD (blue) and H1(left)-, Egg(middle)-, or HP1a(right)-KD OSCs. Differential intra-TAD interaction density score upon H1(left)-, Egg(middle)-, or HP1a(right)-KD (upper), centering TE insertions. Heatmap showing changes in the intra-TAD interaction score of each piRNA target TE insertion sites (lower). The results are shown for piRNA target TE (left) and non-piRNA target TE (right) insertions. Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in intra-TAD interactions upon H1(left)-, Egg(middle)-, or HP1a(right)-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). n.s.; not significant (p > 0.05) Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in inter-TAD interactions upon H1(left)-, Egg(middle)-, or HP1a(right)-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). n.s.; not significant (p > 0.05) Schematic model of stepwise Piwi-piRNA mediated regulation of nuclear architecture and chromatin state. \n",
      ".Figure 5Immunoprecipitation (IP) from OSC lysate using anti-Lam, or myc antibody, followed by WB using indicated antibodies. Mouse immunoglobulin G (n.i.) was used for control IP. Note that OSCs used at the right panel is a line which stably expresses myc-Nxf2. DamID-seq signal densities are shown for OSCs with EGFP (control)- and Nxf2-KD at genomic regions harboring gypsy insertions with the greatest decrease upon Nxf2-KD (locus shown in Fig 2B). Neighboring regions are annotated with protein-coding genes and LTR TEs. Note that this distribution of the reads includes multi-mapped reads. Scatterplot of DamID-seq signals (log2 ratio) for EGFP (control)-KD (x-axis) or Nxf2-KD (y-axis) samples. Values were calculated for 90 TE consensus sequences (one value per one type of TE) (left). Scatterplot of RPKM values for the same 90 TE consensus sequences in EGFP-KD (control, x-axis) or Piwi-KD (y-axis) samples examined by RNA-seq (right). The signals for mdg1, gypsy, and roo are labelled within the plots. Boxplot showing Lamin DamID-seq signals of piRNA target or non-target TEs upon EGFP (control)- or Nxf2-KD. Boxplot whiskers, box, and central band show maximum, third quartile, median, first quartile, and minimum, respectively (n=10 for piRNA target TEs, n= 80 for non-piRNA target TEs). P-values were calculated using the Wilcoxon rank-sum test. Density plots for DamID-seq signals over consensus sequences from piRNA target (gypsy and mdg1, orange) and non-target TEs (roo, blue) in EGFP (control)- or Nxf2-KD cells. Dot plot showing the correlation between the normalized density of indicated TEs per LAD (x-axis) and changes in Lam-Dam / Dam signals upon Nxf2-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Differential interaction heatmap for chromosome 3L (40k resolution) in log2 fold change, showing increased (red) and decreased (blue) interacting bins upon Nxf2-KD. Differential intra-TAD interaction density score upon Nxf2-KD (upper), centering TE insertions. Heatmap showing changes in the intra-TAD interaction score of each piRNA target TE insertion sites (lower). The results are shown for piRNA target TE and non-piRNA target TE insertions. Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in intra-TAD interactions upon Nxf2-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Dot plot showing the correlation between the normalized density of indicated TEs per TAD (x-axis) and changes in inter-TAD interactions upon Nxf2-KD (y-axis). Trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Dot plot showing the correlation between the change of coverage of LAD for each TAD (x-axis), and changes in intra-TAD interactions upon Nxf2-KD (y-axis). The trend lines are in red. Spearman's rank correlation rho is indicated along with the p-value (algorithm AS 89). Oligo-FISH images for the reporter of the tethering system, at 0h (control), 48h hours, 96 hours after transfection of λN-Nxf2 expression vector. Gray indicates DAPI staining for DNA, magenta indicates FISH signal, and green indicates Lamin staining. Scale bar, 5μm. The ratio of distance from FISH signal to the nuclear periphery (DAPI surface) is quantified at right, dot in the box plot indicates mean value; Boxplot whiskers, box, and central band show minimum, first quartile, median, third quartile, and maximum, respectively. P-values calculated with Mann-Whitney U test. \n",
      ".Figure 3(A) HT29 cells were infected with the indicated Shigella strains and incubated for 8 h, and then cell lysates were subjected to immunoblotting. (B) 293T cells were transfected with Myc-tagged ospD expression plasmids. After 24 h, cells were harvested and subjected to immunoblotting. Arrows indicate cleaved RIPK1. C) Multiple sequence alignment of the OspD family: Shigella OspD3, EPEC EspL, and EHEC EspL2. Conserved amino acids are indicated by asterisks. Typical protease catalytic sites are colored in red. D) 293T cells were transfected with a series of plasmids expressing ospD3 point mutants. After 24 h, cells were harvested and subjected to immunoblotting. Arrows indicate cleaved RIPK1. (E HT29 cells were infected with Shigella WT, S325, ∆ospD3, ∆ospD3/D3 (∆ospD3 complemented with wild-type ospD3), or ∆ospD3/D3CS (∆ospD3 complemented with a protease activity-deficient mutant, in which the cysteine residue at position 64 was replaced by serine) strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting F) HT29 cells were infected with Shigella WT, S325, ∆ospD3, ∆ospD3/D3 (∆ospD3 complemented with wild-type ospD3), or ∆ospD3/D3CS (∆ospD3 complemented with a protease activity-deficient mutant, in which the cysteine residue at position 64 was replaced by serine) strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to cytotoxicity assays \n",
      ".Figure 2(A) HT29 cells were infected with Shigella WT, S325, or ospD deletion mutants, and incubated for 8 h. Cell lysates were subjected to immunoblotting. (B HT29 cells were infected with the indicated Shigella strains in the presence or absence of RIPK1 inhibitor, RIPK3 inhibitor, or caspase inhibitor (z-VAD), and then incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting C) HT29 cells were infected with the indicated Shigella strains in the presence or absence of RIPK1 inhibitor, RIPK3 inhibitor, or caspase inhibitor (z-VAD), and then incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to cytotoxicity assay (D HT29 cells treated with the indicated siRNAs were infected with Shigella WT or ∆ospD3. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting (top) and cytotoxicity assays (bottom), respectively. The knockdown efficiency of the indicated siRNAs was assessed by immunoblotting (inset). n.s., not significant; *P<0.05 (unpaired two-tailed Student's t-test). E) HT29 cells treated with the indicated siRNAs were infected with Shigella WT or ∆ospD3. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting (top) and cytotoxicity assays (bottom), respectively. The knockdown efficiency of the indicated siRNAs was assessed by immunoblotting (inset). n.s., not significant; *P<0.05 (unpaired two-tailed Student's t-test). \n",
      ".Figure 4(A) HT29 cells were infected with the indicated Shigella strains and incubated for 8 h. Cells were harvested and subjected to measurement of caspase-8 activation. Caspase-8 activity is reported as relative light units (RLU) of infected samples, minus the value in uninfected samples. *P<0.05 (one-way ANOVA). (B) HT29 cells were infected with the indicated Shigella strains and incubated for 8 h. Cell lysates were subjected to immunoblotting. Arrows indicate cleaved forms of caspase-8, PARP, GSDMD, and IL-18, respectively. (C) HT29 cells were infected with the indicated Shigella strains and incubated for 8 h. Aliquots of cellular supernatants were subjected to cytotoxicity assay. *P<0.05; n.s., not significant (one-way ANOVA). (D) HT29 cells were infected with Shigella WT or ∆ospC1 mutant and incubated for 8 h. Infected cells were then fixed and stained with cleaved caspase-8 (green), rhodamine-phalloidin (red), and DAPI (blue). Percentages of positive cells are shown in the graph at right (*P<0.05; unpaired two-tailed Student's t-test). Scale bar: 100 μm. (E) HT29 cells were infected with the indicated Shigella strains and incubated for 8 h. Cells were harvested and subjected to measurement of caspase-3/7 activation. Caspase-3/7 activity is reported as relative light units (RLU) of infected samples minus uninfected samples. *P<0.05 (one-way ANOVA). (F) HT29 cells were infected with Shigella WT, S325, or ∆ospC1 strains, and incubated for up to 8 h. After incubation for the indicated times, cells were subjected to real-time Annexin V apoptosis assay. Annexin V binding is reported as relative light units (RLU) in infected samples minus the value in uninfected samples. *P<0.05 (one-way ANOVA). \n",
      ".Figure 6(i) When Shigella invades and multiplies within epithelial cells, PAMPS and DAMPs are released. Host cells detect these PAMPs and DAMPs, and subsequently trigger apoptosis as host defense to clear bacterial infection. (ii) To counteract this, Shigella delivers OspC1 effector, and directly or indirectly prevents caspase-8 activation and apoptotic cell death. (iii) On the contrary, host cells detect bacterial disturbance of caspase-8 activation, resulting in induction of necroptosis as a backup host defense. (iv) Again, Shigella subsequently delivers OspD3 effector, which targets RIPK1 and RIPK3 for degradation via its protease activity to prevent necroptosis. \n",
      " Facilitated OGS promotes reprogramming (A-B) 5C or mES medium was used during reprogramming.Figure 15C or mES medium was used during reprogramming. On day 15, Oct4GFP+ colonies were counted (A). 5C or mES medium was used during reprogramming. The percentages of Oct4GFP+ cells were also determined with FACS (B). (C) The expression of several core pluripotency markers was determined with qPCR at different time points during reprogramming and normalized against those in MEFs. (D-E) Energy metabolism was analyzed on day 6 during reprogramming with the Seahorse instrument. Increase in extracellular acidification rate (ECAR) after adding glucose was considered as glycolysis ability of the cells (D), while the decrease in oxygen consumption rate (OCR) was considered as the ATP production ability of the cells (E). (F-G) Expression of Hif1α was modulated with overexpression or sh-RNA-mediated knockdown via a retrovirus system during reprogramming (F). The numbers of Oct4GFP+ colonies were determined on day 15 (G). (H-I) Expression of Pdk1/2 was modulated with a retrovirus system, and oligomycin (1 µM) and 2-DG (5 mM) were used during reprogramming (H). The numbers of Oct4GFP+ colonies were determined on day 15 (I). \n",
      ".Figure 5(A HT29 cells were infected with Shigella WT, ∆ospC1, ∆ospD3, ∆ospC1∆ospD3, or ∆ospC1∆ospD3/ospC1 (∆ospC1∆ospD3 complemented with ospC1) strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting B) HT29 cells were infected with Shigella WT, ∆ospC1, ∆ospD3, ∆ospC1∆ospD3, or ∆ospC1∆ospD3/ospC1 (∆ospC1∆ospD3 complemented with ospC1) strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to cytotoxicity assay (C) HT29 cells were infected with Shigella WT, ∆ospD3, or ∆ospC1. Cell lysates obtained at the indicated time points were subjected to immunoblotting. (D HT29 cells treated with DMSO or caspase-8 inhibitor were infected with Shigella WT, ∆ospC1, ∆ospD3, or ∆ospC1∆ospD3 strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to immunoblotting E) HT29 cells treated with DMSO or caspase-8 inhibitor were infected with Shigella WT, ∆ospC1, ∆ospD3, or ∆ospC1∆ospD3 strains and incubated for 8 h. Cell lysates and aliquots of cellular supernatants were subjected to cytotoxicity assay (F) HT29 cells treated with the control or caspase-8 siRNAs were infected with the indicated Shigella strains and incubated for 8 h. Cell lysates were subjected to immunoblotting. The knockdown efficiency of the indicated siRNAs was assessed by immunoblotting. \n",
      " Facilitated OGS and early EMT form a positive feedback loop (A-B) Expression of Hif1α was modulated via a retrovirus system or small-molecule compounds, oligomycin (1 µM) and 2-DG (5 mM), during reprogramming.Figure 4Expression of Hif1α was modulated via a retrovirus system or small-molecule compounds, oligomycin (1 µM) and 2-DG (5 mM), during reprogramming. The expression of mesenchymal markers (A) Expression of Hif1α was modulated via a retrovirus system or small-molecule compounds, oligomycin (1 µM) and 2-DG (5 mM), during reprogramming. The expression of mesenchymal markers cell migration (B) was determined with qPCR and live-cell imaging, respectively, on day 6. (C) HIF1α induces EMT by transcriptionally activating key mesenchymal transcriptional factors, including Snai1/2, Twist1/2, and Zeb1/2. Although to a less extent, inducing OGS directly with Pdk1/2 or small molecules also activate these factors. Repsox (1 μM) was used on day 2-7 to inhibit early EMT during reprogramming with 5C medium. TGFβ (TGFβ1/2/3, 1ng/ml each) on day 2-7, LSD1i (SP2509, 10 nM), and sequential introduction of Yamanaka factors (OK+M+S) were used to induce early EMT during reprogramming with mES medium (D). Repsox (1 μM) was used on day 2-7 to inhibit early EMT during reprogramming with 5C medium. Except for OK+M+S group, the four Yamanaka factors were delivered simultaneously. Energy metabolism was determined on day 6 with Seahorse instrument (E-F). Repsox (1 μM) was used on day 2-7 to inhibit early EMT during reprogramming with 5C medium. The expression of glycolysis markers was determined on day 6 with qPCR (G). (H) Key mesenchymal transcriptional factors, including Snai1/2, Twist1/2, and Zeb1/2 induce OGS by transcriptionally activating Hif1α and glycolysis genes. (I) Early EMT and facilitated OGS form a positive feedback loop to induce pluripotency. \n",
      "C medium removes epigenetic barriers during reprogramming (A-B) 5C and mES medium were used during reprogramming.Figure 2(A-B) 5C and mES medium were used during reprogramming. Representative alkaline phosphatase (AP) staining on day 15 was provided in (A). AP+Oct4GFP- and AP+Oct4GFP+ colonies were counted on day 15 (B). (C) Thirty-two epigenetic factors were selected because they had higher expression in 5C-Oct4GFP- than in mES-Oct4GFP-. Their expression in ESCs, iPSCs, and pre-iPSCs in two previous reported assay (GSE14012 and GSE10871) was also listed. (D) The expression of Bmi1, Ctcf, Ezh2, Kdm2b, or Wdr5 was modulated with overexpression or sh-RNA-mediated knockdown via a retrovirus system during reprogramming. AP+Oct4GFP- and AP+Oct4GFP+ colonies were counted on day 15. During reprogramming with mES medium, Bmi1, Ctcf, Ezh2, Kdm2b, and Wdr5 were overexpressed simultaneously with the four Yamanaka factors (OKMS+5F) or Oct4 (O+5F). All factors were delivered simultaneously via a retrovirus system on day 0 and 1. Reprogramming with only Yamanaka factors (OKMS) or Oct4 (O) served as control. The expression of pluripotency markers was determined with qPCR on day 6 during reprogramming (E). During reprogramming with mES medium, Bmi1, Ctcf, Ezh2, Kdm2b, and Wdr5 were overexpressed simultaneously with the four Yamanaka factors (OKMS+5F) or Oct4 (O+5F). All factors were delivered simultaneously via a retrovirus system on day 0 and 1. Reprogramming with only Yamanaka factors (OKMS) or Oct4 (O) served as control. The numbers of Oct4GFP+ colonies were determined on day 15 (F). During reprogramming with mES medium, Bmi1, Ctcf, Ezh2, Kdm2b, and Wdr5 were overexpressed simultaneously with the four Yamanaka factors (OKMS+5F) or Oct4 (O+5F). All factors were delivered simultaneously via a retrovirus system on day 0 and 1. Reprogramming with only Yamanaka factors (OKMS) or Oct4 (O) served as control. The histone methylation on core pluripotency loci was determined on day 6 with ChIP-qPCR (G). \n",
      "C state is a distinct pluripotency state (A-E) On day 15 during reprogramming, the expression of E-cadherin (E-CAD) and N-cadherin (N-CAD), was determined with immunofluorescence together with the morphology in Oct4GFP+ colonies (A, Scale bar, 25 μM).Figure 5On day 15 during reprogramming, the expression of E-cadherin (E-CAD) and N-cadherin (N-CAD), was determined with immunofluorescence together with the morphology in Oct4GFP+ colonies (A, Scale bar, 25 μM). After sorting Oct4GFP+ cells out (5C-Oct4GFP+ and mES-Oct4GFP+ cells), the expression of markers of different pluripotent states (B) After sorting Oct4GFP+ cells out (5C-Oct4GFP+ and mES-Oct4GFP+ cells), the expression of markers of Xist (C) After sorting Oct4GFP+ cells out (5C-Oct4GFP+ and mES-Oct4GFP+ cells) their abilities to form chimeras (D) were determined. The characteristics of 5C-Oct4GFP+ cells were summarized and compared to naïve and primed PSCs in (E). (F) The isolated 5C-Oct4GFP+ cells were cultured in naïve or primed medium for three days, and the morphology changed towards R1 ESCs (naïve PSCs) and R1 ESCs-primed (primed PSCs), respectively. The obtained cells were maintained for additional 20 passages. The 5C-Oct4GFP+ cells cultured in primed medium lost GFP fluorescence. R1 ESCs-primed were primed PSCs generated from R1 ESCs. Scale bar, 100 μM. The isolated 5C-Oct4GFP+ cells were cultured in naïve or primed medium for three days, and the morphology changed towards R1 ESCs (naïve PSCs) and R1 ESCs-primed (primed PSCs), respectively. (G) PSCs mentioned in (F) were used for RNA-Seq analysis. Similarity among gene expression profiles was determined. The isolated 5C-Oct4GFP+ cells were cultured in naïve or primed medium for three days to generate 5C-Oct4GFP+ (naïve) and 5C-Oct4GFP+ (primed) cells. The expression of Xist (H) The isolated 5C-Oct4GFP+ cells were cultured in naïve or primed medium for three days to generate 5C-Oct4GFP+ (naïve) and 5C-Oct4GFP+ (primed) cells. the abilities to form chimeras (I-J) were determined. \n",
      " Early EMT and OGS induce the epigenetic changes (A) The enrichment of HIF1α, SNAI2, TWIST1/2, and ZEB1 binding sites on the promoters of Bmi1, Ctcf, Ezh2, Kdm2b, and Wdr5 was determined by Pscan Zambelli et al, 2009().Figure 3(A) The enrichment of HIF1α, SNAI2, TWIST1/2, and ZEB1 binding sites on the promoters of Bmi1, Ctcf, Ezh2, Kdm2b, and Wdr5 was determined by Pscan Zambelli et al, 2009(). The resulted Z-scores were listed. \"Max. M\" suggested the maximum Z-score generated with SNAI2, TWIST1/2, and ZEB1 binding sites on one indicated promoter. \"Average\" suggested the average Z-score of all transcriptional factors tested in the Pscan software on one indicated promoter. The expression of H1f1α, Snai2, Twist1/2, and Zeb1 was modulated with overexpression or sh-RNA-mediated knockdown via a retrovirus system during reprogramming with mES (B) The expression of Bmi1, Ctcf, Ezh2, Kdm2b, Wdr5 was determined on day 6 with qPCR and normalized against those in MEFs. The comparisons were performed between all groups and corresponding Flag groups. The expression of H1f1α, Snai2, Twist1/2, and Zeb1 was modulated with overexpression or sh-RNA-mediated knockdown via a retrovirus system during reprogramming with 5C medium (C). The expression of Bmi1, Ctcf, Ezh2, Kdm2b, Wdr5 was determined on day 6 with qPCR and normalized against those in MEFs. The comparisons were performed between all groups and corresponding Flag groups. pre-iPSCs were isolated during reprogramming with mES-Vc, mES or 5C medium. These pre-iPSCs were further cultured with a different medium (mES-Vc and mES medium) for seven days. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. pre-iPSCs were isolated during reprogramming with mES-Vc, mES or 5C medium. These pre-iPSCs were further cultured with a different medium (mES-Vc and mES medium) for seven days. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. The numbers of Oct4GFP+ colonies were determined on day 7 (E). Reprogramming was also performed with mES-Vc or mES medium. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. AP+Oct4GFP- and AP+Oct4GFP+ colonies were counted on day 15 (G). Reprogramming was also performed with mES-Vc or mES medium. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. In addition, energy metabolism was determined on day 6 with Seahorse instrument Reprogramming was also performed with mES-Vc or mES medium. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. In addition, energy metabolism was determined on day 6 with Seahorse instrument Reprogramming was also performed with mES-Vc or mES medium. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. Cell migration was determined by measuring the migration rate with live-cell imaging on day 6 (J). Reprogramming was also performed with mES-Vc or mES medium. Additional factors (Hif1α, Hif1α overexpression; TGFβ, TGFβ1/2/3 1 ng/ml each; 3F, Ctcf, Kdm2b, and Wdr5 overexpression) were used simultaneously with mES medium. The expression of Bmi1, Ctcf, Ezh2, Kdm2b, or Wdr5 was determined on day 6 with qPCR (K). (L) Early EMT and facilitated OGS cooperate to induce pluripotency by removing the additional epigenetic barrier. \n",
      ".Figure 3(A) Tetrad analysis reveals that the respiration defect of Sc+Sb-CCM1 cells can be rescued by mutations in mitochondrial or nuclear genomes. The suppressor clones were crossed with the Sc+Sb-CCM1-ρ⁰ strain and the tetrads from the cross were analyzed. A non-Mendelian 4:0 segregation pattern for improved respiration is expected if the suppressor mutation is in mtDNA (left panel). Otherwise, a 2:2 segregation pattern would be observed if the rescue effect involved a single nuclear mutation (right panel). (B) Partial sequence alignment of 15S rRNA from S. bayanus, S. cerevisiae and two mitochondrial suppressor clones (#1 and #2) of Sc+Sb-CCM1. A single-nucleotide deletion at position 188 and four nucleotide substitutions at positions 1524-1527 found in both suppressor clones are labeled in red. \n",
      "Beclin localizes to the TGN.Figure 3Beclin localizes to the TGN. HeLa (A, D, E and F), BNL CL.2 (B) and H‐4‐II‐E cells (C) were grown on cover slips. Cells were permeabilized with digitonin before (C, D and E) or after (A, B and F) fixation with 3% paraformaldehyde. Cells were double‐labeled with anti‐Beclin (A-F, left panels) and either anti‐syntaxin 6 (A and B, middle panels), anti‐TGN38 (C, middle panel), anti‐EEA1 (D, middle panel), anti‐LBPA (E, middle panel) or anti‐TfR (F, middle panel) antibodies. In the merged images (right panels), yellow indicates co‐localization. Bar, 10 μm. \n",
      "Immunofluorescence localization of PtdIns 3‐kinase.Figure 4Immunofluorescence localization of PtdIns 3‐kinase. HeLa (A and C) and H‐4‐II‐E cells (B) were permeabilized, fixed, and then subjected to immunostaining. Cells were double‐labeled with anti‐PtdIns 3‐kinase (A-C, left panels) and either anti‐syntaxin 6 (A, middle panel), anti‐TGN38 (B, middle panel) or anti‐LBPA (C, middle panel) antibodies. Bar, 10 μm. \n",
      " EMT and OGS cooperate to inhibit reprogramming at the late stage (A-E) The expression Oct4GFP and E-cadherin was determined with FACS on day 15 during reprogramming (A).Figure 6The expression Oct4GFP and E-cadherin was determined with FACS on day 15 during reprogramming (A). Based on the FACS results, the expression of Oct4GFP and E-cadherin was summarized in Oct4GFP+E-cadherin+ cells (B) and in Oct4GFP-E-cadherin- cells (C). The percentage of E-cadherin+ cells in Oct4GFP+ cells was listed in (D), while the percentage of Oct4GFP+ cells in E-cadherin+ cells was listed in (E). (F-G) Energy metabolism was analyzed in 5C-Oct4GFP+ and mES-Oct4GFP+ cells with Seahorse instrument. (H) 5C-Oct4GFP- cells had higher expression of glycolysis and mesenchymal markers, but lower expression of epithelial markers than mES-Oct4GFP- cells. 5C-Oct4GFP+ cells had higher expression of all three groups of markers than mES-Oct4GFP+ cells. EMT and OGS were modulated with TGFβ (TGFβ1/2/3, 1ng/ml each), Repsox (1 μM), oligomycin (1 µM), and 2-DG (5 mM) at the late stage during reprogramming with 5C medium (I). The numbers of three types of Oct4GFP+ colonies (J) the expression of four epithelial markers in Oct4GFP+ cells (K) were determined on day 15. \"Average\" in (K) suggested the average expression of four epithelia markers. \n",
      "Beclin and PtdIns 3‐kinase form a complex.Figure 1(A) Cell lysates prepared from HeLa cells were solubilized with Triton X-100 and subjected to immunoprecipitation using preimmune sera (lane 1) or anti-Beclin antibodies (lanes 2 and 3). The immunoprecipitates were incubated with phosphatidylinositol, [γ-32P]ATP and 60 μM cold ATP in the presence of Mn2+ (lanes 1 and 2) or Mg2+ (lane 3) for 5 min at 30°C. (B) Immunoprecipitates of anti‐Beclin antibodies prepared as described in (A) were assayed for PI 3‐kinase activity in the presence of wortmannin at concentrations of 0 nM (lane 1), 1 nM (lane 2), 10 nM (lane 3), 100 nM (lane 4). The labeled lipids were extracted and separated by thin layer chromatography, followed by detection by autoradiography using a bioimaging analyzer BAS2000 (Fuji Film). (C) Cell lysates prepared from HeLa cells were solubilized with Triton X‐100 and incubated with protein A‐immobilized anti‐Beclin (lane 2) or anti‐PtdIns 3‐kinase antibodies (lane 4). As controls, preimmune sera of the respective antibodies (lanes 1 and 3) were used. Retained proteins were separated by SDS-PAGE and detected by immunoblotting with anti‐PtdIns 3‐kinase (upper panels) and anti‐Beclin (lower panels) antibodies. \n",
      "Cross‐linking of Beclin and PtdIns 3‐kinase.Figure 2(A) Total cell lysates were treated with DSP (lanes 3, 4, 7 and 8) or its solvent, dimethylsulfoxide (lanes 1, 2, 5 and 6), at 4°C for 2 h. Proteins were solubilized with SDS and subjected to immunoprecipitation using protein A‐immobilized anti‐PtdIns 3‐kinase (PtdIns 3‐K) (lanes 1-4) or Beclin (lanes 5-8) antibodies. Immunoprecipitates were eluted with 1× SDS sample buffer (62.5 mM Tris-HCl pH 6.8, 2% SDS, 10% glycerol, a trace amount of Bromophenol blue). After treatment with 5% 2‐mercaptoethanol, proteins were separated by SDS-PAGE, followed by detection by immunoblotting with anti‐Beclin (upper panels) and anti‐PtdIns 3‐kinase (lower panels) antibodies. (B) Cell lysates prepared from HeLa cells were subjected to subcellular fractionation by centrifugation at 100 000 g for 1 h. The membrane (lanes 1 and 3) and supernatant (lanes 2 and 4) fractions were subjected to SDS-PAGE, followed by immunoblotting with anti‐PtdIns 3‐kinase (lanes 1 and 2) and anti‐Beclin (lanes 3 and 4) antibodies. (C) Membrane (lanes 1-4; 100 000 g, 30 min, pellet) and soluble fractions (lanes 5-8; 100 000 g, 30 min, supernatant) prepared from HeLa cells were treated with DSP (lanes 3, 4, 7 and 8) or dimethylsulfoxide (lanes 1, 2, 5 and 6). Proteins were solubilized with SDS and subjected to immunoprecipitation using protein A-immobilized anti-PtdIns 3-kinase antibodies. Immunoprecipitates were separated by SDS-PAGE under a reducing condition, followed by detection by immunoblotting with Beclin antibodies. F and E indicate flow-through and elute fracions, respectively. \n",
      ".Figure 2(A) Sb-Ccm1 protein is transported efficiently to S. cerevisiae mitochondria. Total cell extracts (total), cytosolic fractions (cyto) or mitochondrial fractions (mito) were hybridized with different antibodies to detect the proteins. G6PDH (glucose-6-phosphate dehydrogenase) and Cox2 served as cytosolic and mitochondrial markers, respectively. Both Sc-Ccm1 and Sb-Ccm1 were fused with c-Myc at the C-terminus and expressed from a single-copy plasmid. The Myc-tagged Ccm1 proteins consistently appeared as double bands, which were probably due to some unknown protein modifications. (B) The level of 15S rRNA is reduced in Sc+Sb-CCM1 cells. RNA isolated from Sc+Sb-CCM1 (labeled as Sb) or Sc+Sc-CCM1 (labeled as Sc) cells was hybridized with gene-specific probes using Northern blots. (C) Sb-Ccm1 interacts weakly with S. cerevisiae 15S rRNA. The level of Ccm1-bound 15S rRNA was measured using quantitative PCR analysis following mitochondrial RNA immunoprecipitation of S. cerevisiae cells expressing Sc-CCM1-13Myc or Sb-CCM1-13Myc. Data was normalized to the control group without the Myc tag to obtain the relative fold enrichment. Graph was plotted using data from three independent repeats for each strain and p value was calculated by unpaired, two-sided student t-test. Error bars indicate S.D. ***, p value < 0.001. (D) Overexpressing Sb-CCM1 rescues the respiration defect of Sc+Sb-CCM1 cells. Cell cultures were serially-diluted and spotted onto YPD (glucose) or YPG (glycerol) plates. The plates were then incubated at 28°C until colonies were easily observed. (E) The steady-state level of mtDNA-encoded proteins is reduced in Sc+Sb-CCM1 cells. Immunoblotting for the mitochondrial proteins in Sc+Sb-CCM1 (labeled as Sb) or Sc+Sc-CCM1 (labeled as Sc) cells. Cox1, Cox2 and Cox3 are mtDNA-encoded complex IV subunits. Tom20 is a nucleus-encoded mitochondrial protein. \n",
      ".Figure 1(A) Chromosome 7 of S. bayanus is incompatible with the S. cerevisiae genome. The chromosome 7 replacement stain (Sc+Sb-chr7) exhibited serious growth defects when grown on the non-fermentable carbon source (glycerol). The respiration defects could be partially rescued when a whole set of S. bayanus chromosomes were provided (Sc+Sb-chr7 x Sb-ρ⁰), or fully rescued with the addition of S. bayanus mtDNA (Sc+Sb-chr7-ρ⁰ x Sb). Cell cultures were serially-diluted and spotted onto YPD (glucose) or YPG (glycerol) plates. The plates were then incubated at 28°C until colonies were easily observed. Sc, S. cerevisiae. Sb, S. bayanus. ρ⁰, mtDNA-less strains. (B) Ectopically-expressing Sc-CCM1 rescues the respiration defect of Sc+Sb-chr7. Spot assays for the wild-type S. cerevisiae (Sc) and the chromosome 7 replacement stain (Sc+Sb-chr7) carrying an empty plasmid (+ vector) or the plasmid with the S. cerevisiae CCM1 gene (+ Sc-CCM1). (C) Sb-CCM1 incompatibility could only be rescued in the presence of S. bayanus mtDNA. The native CCM1 ORF in S. cerevisiae was replaced by either Sb-CCM1 (Sc+Sb-CCM1) or Sc-CCM1 (Sc+Sc-CCM1, as a control) coding regions. Crossing the Sc+Sb-CCM1 strain with the mtDNA-less Sb-ρ⁰ strain (Sc+Sb-CCM1 x Sb-ρ⁰) could not rescue the growth defect. (D) CCM1 incompatibility is symmetric between S. cerevisiae and S. bayanus. The CCM1 ORF in S. bayanus was replaced by either Sc-CCM1 (Sb+Sc-CCM1) or Sb-CCM1 (Sb+Sb-CCM1, as a control) coding regions. Only the Sb+Sc-CCM1 strain exhibited respiration defects and the defect could not be rescued even when a whole set of S. cerevisiae chromosomes were provided (Sb+Sc-CCM1 x Sc-ρ⁰). (E) CCM1 incompatibility probably occurred during the divergence between S. bayanus and the common ancestor of S. cerevisiae, S. paradoxus, S. mikatae and S. kudriavzevii. The endogenous CCM1 in S. cerevisiae was replaced with its orthologous alleles from other Saccharomyces sensu stricto yeasts. Only the strain carrying S. bayanus CCM1 displayed growth defects on glycerol-containing plates. \n",
      ".Figure 7(A-B) Energy metabolism was analyzed with 5C-Oct4GFP+, R1 ESCs (naïve PSCs), and R1 ESCs-primed (primed PSCs) with Seahorse instrument. (C) Expression of epithelial, mesenchymal, glycolysis and oxidative phosphorylation (OXPHOS) markers (listed in Dataset EV4) was summarized in R1 ESCs, 5C-Oct4GFP+ (naïve), 5C-Oct4GFP+, 5C-Oct4GFP+ (primed), and R1 ESCs-primed, based on the current RNA-Seq. For each marker, Log2 values of FKPM were normalized by subtracting the average of five groups. Log2 values of FKPM of each group of markers were then averaged and plotted. 5C-Oct4GFP+ (naïve) and 5C-Oct4GFP+ (primed) were 5C-Oct4GFP+ cultured in naïve and primed medium, respectively, for three days. (D) Genes with higher expression at different developmental stages were selected out from previously reported RNA-seq (Data ref: Bertone, 2015). \"Mor\", \"ICM\", \"pre-EPI\" and \"post-EPI\" represented genes that have high expression only in morula, inner cell mass, pre-implantation epiblast and post-implantation epiblast, respectively. \"Mor+ICM\", \"ICM+pre\", and \"EPI\" indicated genes that have high expression in both morula and inner cell mass, in both inner cell mass and pre-implantation epiblast, and in both pre-implantation and post-implantation epiblast, respectively. The results of previously reported RNA-seq were summarized in the left panel, while the expression of these genes in current RNA-seq was summarized in the right panel. Log2 values of FKPM were presented similarly as in (C). (E) Markers of 5C, naïve and primed states were selected based on the current RNA-Seq (Dataset EV4). The expression of these markers in R1 ESCs, 5C-Oct4GFP+, and R1 ESCs-primed was listed together with the GO enrichment of each group of markers. Log2 values of FKPM were presented similarly as in (C). (F) Markers of 5C, naïve and primed states (Dataset EV4) were used to analyze expression profiles of epiblast during the early development of mouse embryos (Data ref: Mohammed et al, 2017a) (F). Single-cell was plotted based on the average expression of each group of markers. (G) The average expression of both naïve and primed markers was calculated based on GSE100597. The top-2 cells from E4.5 epiblast and top-5 cells from E5.5 epiblast were selected. The expression of several naïve markers (Dppa3, Esrrb, Klf4, Klf5, Tbx3, and Zfp42) and several mesenchymal markers (Fgf5, Lef1, Nodal, and Pou3f1) in these selected cells were listed (upper part). The average expression of these genes in E3.5, 4.5, 5.5 and 6.5 epiblast were also listed (lower part). These results were all normalized against the average expression of these genes in all tested cells. (H) The expression of representative markers of 5C, naïve and primed states was summarized in E3.5, 4.5, 5.5 and 6.5 epiblast. Cells from E5.5 epiblast were separated into 8 groups (I-VIII) basing on the average expression of epithelial, mesenchymal, and glycolysis markers (H). The average expression of three groups of markers were summarized in (J). The expression of six important markers of 5C state was summarized in these eight groups of cells (K) (L) Schematic illustration of different pluripotent states and developmental stages. \n",
      ".Figure 4(A) Reconstituted Sb-CCM1 mutants can rescue the incompatibility of Sb-CCM1. Among 94 suppressor clones, 20 of them carry mutations in the Sb-CCM1 gene. These mutants were reconstructed in Sc+Sb-CCM1 cells to test their function. Superscripts next to Sb-CCM1 indicate the respective amino acid substitutions and numbers in parentheses indicate the number of independent clones in our suppressor screen. Sc, wild-type S. cerevisiae. (B) Sequence alignment shows that most of the Sb-CCM1 suppressor mutants contain de novo amino acid changes and only two mutants have switched their residues from the S. bayanus sequence to the S. cerevisiae sequence (F294L and D400N). Letters beside the phylogenetic tree (in which the line lengths are not proportional to the evolutionary distances) indicate the amino acid state at the 400th residue and parentheses contain the codons encoding for the amino acids. (C) The 400th residue plays a determinant role in the S. cerevisiae-S. bayanus incompatibility. Both Sb-CCM1E375Q and Sb-CCM1D400N mutants relieved the incompatibility in S. cerevisiae, but only Sb-CCM1D400N caused respiration defects in S. bayanus. The Sb-CCM1E375Q mutant was selected as a control in this experiment since it was a charged-to-polar substitution similar to Sb-CCM1D400N and also appeared three times in our screen. Sb, wild-type S. bayanus. (D) Protein structural analysis shows that all intramolecular suppressor mutations were located on the predicted PPR motifs. The predicted protein structure of Ccm1 is shown at left. Blue and green colors indicate the two α-helices in each predicted PPR motif and white color indicates additional α-helices in the N-terminus and the middle region. Red color indicates the positions of the suppressor mutations. Predicted PPR motifs occupy around 52% of the whole protein (453 of 871 residues). The box at right displays a magnified version of some PPR motifs where the substitutions are located. \n",
      ".Figure 5(A) 15S rRNA levels are increased in the suppressor clones. Total RNA was isolated from the replacement strains (Sc+Sc-CCM1 and Sc+Sb-CCM1), two nuclear suppressor strains (Sc+Sb-CCM1D400N and Sc+Sb-CCM1E375Q) and one mitochondrial suppressor strain and then subjected to quantitative PCR analyses. Relative mRNA abundance was calculated by normalizing the 15S rRNA level to the ATP6 mRNA level. Graph was plotted using data from five independent repeats for each strain and p values were calculated by unpaired, two-sided student t-test. Error bars indicate S.D. ***, p value < 0.001. **, p value < 0.01. Asterisks in black, comparing Sc+Sc-CCM1 with others. Asterisks in gray, comparing Sc+Sb-CCM1 with others. (B) The steady-state level of mtDNA-encoded proteins is increased in the suppressor clones. Protein extracts from the replacement and suppressor strains were hybridized with antibodies against G6PDH, Cox1, Cox2 or Cox3. \n",
      ".Figure 6(A) PPR genes show significantly increased evolutionary rates. Box plot of the evolutionary rates (Ka/Ks) of PPR, TPR, RRM, MRP, and PUF genes and the whole genome by comparing S. cerevisiae and S. paradoxus. All these protein families share some features with the PPR proteins. TPR proteins contain TPR motifs that share a similar repetitive structure with the PPR motifs but mainly mediate protein-protein interactions [94]. RRM proteins are RNA recognition motif-containing proteins and function in splicing, RNA stability and translation [95]. MRP proteins comprise ribosomal proteins encoded in the nuclear genome that are imported into the mitochondria [96]. PUF proteins contain RNA-binding motifs similar to the PPR motifs and are involved in mRNA post-transcriptional events, including mitochondrial import of mRNA [97]. p values were calculated by two-sample Kolmogorov-Smirnov test. **, p value = 7.0143E-3. (B) Seven of 12 tested PPR genes exhibit incompatibility between S. cerevisiae and at least one other Saccharomyces sensu stricto species. The doubling time of S. cerevisiae cells carrying the endogenous PPR genes (Sc) or orthologs of other yeast species (Sp, Sm, Sk and Sb) were measured in YPD (glucose) or YPG (glycerol) media. All the measurements were then normalized to the doubling time of Sc to obtain the relative doubling time. Graphs were plotted using data from at least three independent repeats for each strain. Sc, S. cerevisiae. Sp, S. paradoxus. Sm, S. mikatae. Sk, S. kudriavzevii. Sb, S. bayanus. n.d., growth not detected. p values were calculated by unpaired, two-sided student t-test. Error bars indicate S.D. *, p value < 0.05. **, p value < 0.01. ***, p value < 0.001. (B) Seven of 12 tested PPR genes exhibit incompatibility between S. cerevisiae and at least one other Saccharomyces sensu stricto species. The doubling time of S. cerevisiae cells carrying the endogenous PPR genes (Sc) or orthologs of other yeast species (Sp, Sm, Sk and Sb) were measured in YPD (glucose) or YPG (glycerol) media. All the measurements were then normalized to the doubling time of Sc to obtain the relative doubling time. Graphs were plotted using data from at least three independent repeats for each strain. Sc, S. cerevisiae. Sp, S. paradoxus. Sm, S. mikatae. Sk, S. kudriavzevii. Sb, S. bayanus. n.d., growth not detected. p values were calculated by unpaired, two-sided student t-test. Error bars indicate S.D. *, p value < 0.05. **, p value < 0.01. ***, p value < 0.001. \n",
      "-null embryos exhibit altered expression of neural crest genes (A-B) Sox10 in situ hybridization in (A) wildtype and (B) Slc7a5-null E9.Figure 3(A-B) Sox10 in situ hybridization in (A) wildtype and (B) Slc7a5-null E9.5 embryos (n=4 each) with TSs of (a1, b1) the hindbrain at level of trigeminal ganglion V, (a2, b2) otic vesicles and the neural tube at more posterior levels (a3, a4, b3, b4). Scale bars 200 µm, except sections a1 - b4, 100 µm. Closed white arrowheads in (a3, a4) indicate neural crest, open white arrowheads indicate depleted neural crest in (b3) and position where neural crest should be in (b4). (C-G) Immunofluorescence on TSs caudal spinal cord of E9.5 wildtype littermates and Slc7a5-null embryos; Pax3 and FoxA2 were used as indicators for dorso-ventral organisation in (C, F) wildtype, n=2 (for FoxA2) and n= 4 (For Pax3) and (D, G) Slc7a5-null, n=3 (for FoxA2 ) and n = 3 for Pax3) neural tube. Arrowheads indicate border of Pax3 expression domain. Scale bars 50 µm. (E) Percentage of Pax3 expressing cells was determined by counting these cells and all DAPI labelled nuclei in the neural tube and comparison made between wildtype (4 embryos, 6 sections each) and Slc7a5-null (3 embryos, 6 sections each), each dot represents the average for one embryo, unpaired t-test, *p = 0.018 (see original source data). Error bars indicate SEM. \n",
      "-null embryos exhibit overt neural tube closure and limb-bud outgrowth defects, reduced Fgf8 expression domains and aberrant neuronal and neural crest differentiation (A-H) Live wildtype littermate and Slc7a5-null embryos imaged shortly after dissection.Figure 2(A-H) Live wildtype littermate and Slc7a5-null embryos imaged shortly after dissection. (A-D) Wildtype and (E-H) Slc7a5-null E9.75 embryos from lateral (A, D, E, H), frontal (B, F) or dorsal (C, G) views; (e1-h1) higher magnification images of the Slc7a5-null embryo. White arrowheads indicate the smaller limb bud (compare B and F), open/reduced forebrain (compare D and H), open neural tube at posterior (compare C and E, e1) and anterior (compare B and F, f1) regions. Black arrowheads indicate kinked neural tube (compare C and G, g1) and apparently missing optic and smaller otic vesicles (ov) (compare D and h1). Data information: Scale bars 200 µm mRNA in situ hybridization and immunofluorescence in E9.5 or E10.5 wildtype and Slc7a5-null embryos for key marker genes. (I-L) Fgf8 mRNA transcripts were detected in wildtype (I-j1) and in Slc7a5-null (K-l1) E10.5 embryos (n=4 each) (white arrowheads indicate limb buds in K and the isthmus in L) with (j1, l1) sections through isthmus at midbrain/hindbrain border and (i1, k1) lateral views of forelimb buds. Data information: Scale bars 200 µm, except sections 100 µm. mRNA in situ hybridization and immunofluorescence in E9.5 or E10.5 wildtype and Slc7a5-null embryos for key marker genes. (M-N) Wholemount Tubulin-β-III (Tuj1) immunofluorescence performed on (M, m1) wildtype and (N, n1) Slc7a5-null E9.5-E10.5 embryos (n=5 each condition). Dorsal root ganglia (DRG) and Sympathetic Chain Ganglia (SCG) are indicated with open arrowheads together with cranial ganglia IX, X and XI. Images in (m1, n1) show higher magnification of the cranial ganglia V, VII/VIII (open arrowheads). Data information: Scale bars 200 µm mRNA in situ hybridization and immunofluorescence in E9.5 or E10.5 wildtype and Slc7a5-null embryos for key marker genes. (O-P) Neurog2 expression in wildtype (O-o4) and Slc7a5-null embryos (P-p4) (n= 5 and n=6 each), showing reduced expression in forebrain (o2, p2), cranial ganglia (o1, o3, p1, p3, arrowheads) and in spinal cord at level of forelimb (o4, p4, arrowheads indicate position of neural crest). Data information: Scale bars 200 µm, except sections 100 µm. \n",
      " mRNA expression pattern in the early mouse embryo (A) E7.Figure 1(A) E7.0 whole embryo and transverse section (TSs) (dashed lines) through (a1) extra-embryonic tissue showing expression in a ring of extra-embryonic ectoderm (ee) and extraembryonic mesoderm (em) and absence of transcripts in the allantois (al); (a2) revealing expression in epiblast (ep) and in mesendoderm (me) emerging from the primitive streak (ps), dashed box (a2') defines higher magnification region shown in a2. (B) E8.5 whole embryo dorsal view and TSs (dashed lines) showing expression in: (b1,b2,b3) the open anterior neural plate (future fore, mid and hindbrain) including (b3) neural crest (nc) emerging from hindbrain; (b4) in the dorsal neural tube (prospective neural crest) and dorsal somites (s); (b5) in closing preneural tube neural (pnt) and epithelising somites; (b6) in caudal lateral epiblast (cle) adjacent to the node (n) at the anterior tip of the primitive streak, but not in presomitic mesoderm (psm). White arrows indicate open anterior and posterior neural plate . (C) E9.5 whole embryo side view and TSs (white dashed lines) showing expression in: (c1) forebrain (fb) and hindbrain , including optic vesicles (opv), (c1') seen at high magnification (and indicated by box grey dashed line) and otic vesicles (ov) (c1'') seen at high magnification (box grey dashed line); (c2) high expression in dorsal hindbrain (hb) and (c3) spinal cord (sc) and dermamyotome (dm) of the differentiating somite; (c5) throughout open posterior neural plate. (D) High magnification of E9.5 whole embryo showing expression in first branchial arch (ba) and forming cranial ganglia, which are just becoming apparent, (V), (VII), (VIII) (IX) and (X). OV, otic vesicle. (E) Expression in mesenchyme cells underlying the apical ectodermal ridge (aer) in the progress zone (pz) and in the zone of polarising activity (zpa) of the forelimb (fl). Data information: Images representative of n≥ 4 embryos for each stage. Scale bars 200 µm, except sections 100 µm. \n",
      "-null neural tube exhibits slightly reduced cell proliferation and aberrant mTORC1 activity, and qPCR validation of significantly changed genes in Slc7a5-null embryos (A-D) Proliferation was assessed in the spinal cord (forelimb level) (A, B) using a phospho-Histone3 (phospho-H3) antibody (green) to identify mitotic cells and DAPI (blue) to label nuclei; cells were counted in 3 wildtype (18 sections) and 3 Slc7a5-null embryos (18 sections) and mitotic index calculated; (C, D) Proliferation was also assessed in the forebrain in 3 wildtype embryos (17 sections) and 3 Slc7a5-null embryos (16 sections).Figure 4(A-D) Proliferation was assessed in the spinal cord (forelimb level) (A, B) using a phospho-Histone3 (phospho-H3) antibody (green) to identify mitotic cells and DAPI (blue) to label nuclei; cells were counted in 3 wildtype (18 sections) and 3 Slc7a5-null embryos (18 sections) and mitotic index calculated; (C, D) Proliferation was also assessed in the forebrain in 3 wildtype embryos (17 sections) and 3 Slc7a5-null embryos (16 sections). White dashed lines indicate midline. Data information All scale bars 50 µm. (E, F) Comparison of mitotic index in wildtype and Slc7a5-null (E) spinal cord, p = 0.872 ,and (F) forebrain *p = 0.049, each dot represents the average for one embryo, unpaired t-test Data information: *p<0.05, **p<0.01, ***p<0.001 (G, H) mTORC1 activity in the spinal cord (hindlimb level) was assessed by immunofluorescence using an antibody against the phospho-ribosomal protein S6 (phosho-S6 Ser 235/236). Data information All scale bars 50 µm. (I) Labelling intensity was measured and plotted relative to DAPI intensity; 6 wildtype embryos (18 sections) and 5 Slc7a5-null embryos (19 sections). Each dot represents the average for one embryo, (*p = 0.0457) unpaired t-test with Welch correction Data information: *p<0.05, **p<0.01, ***p<0.001 (J) Western blot of individual E9.5 wildtype (n=3) and Slc7a5-null (n=4) embryo lysates immunoblotted using an antibody against phospho-S6K (Thr389) and total P70S6K for loading control. Band intensities were measured with FIJI and an unpaired t-test was performed for statistical analysis Data information: *p<0.05, **p<0.01, ***p<0.001 (K-L) (K) Validation by qPCR of the targets identified by RNAseq; qPCR was performed on individual E8.5 embryos of each genotype (wildtype n=5, heterozygous n=9 and Slc7a5-null n=7). (L) qPCR was performed on individual E9.5 embryos (wildtype n=5, heterozygous n=3 and Slc7a5-null n=4) using primers specific for genes associated with the integrated stress response. A one-way Anova test with a Turkey post-test was performed. Data information: *p<0.05, **p<0.01, ***p<0.001 Error bars indicate SEM. \n",
      ".Figure 4(A) Secondary structures of the anticodon loop of three types of tRNAPhe(GAA) s: tRNAPhe(GAA), △ftsj1_tRNAPhe(GAA), and m1G37_tRNAPhe(GAA). (B, C) Quantification of the Cm/A and Gm/A of tRNAPhe(GAA), △ftsj1_tRNAPhe(GAA), and m1G37_tRNAPhe(GAA) after incubation with FTSJ1, WDR6, and FTSJ1-WDR6 by UPLC-MS/MS analysis, respectively. Error bars represent the standard deviation of three independent experiments. n.d., not detected. (D) UPLC-MS/MS analysis of 2'-O-methylation of m1G37_tRNAPhe(GAA), -G34A, -G34C and G34U after incubation with FTSJ1-WDR6. 2 μL of Gm (1 ng/mL), Am (1 ng/mL), Cm (1 ng/mL) and Um (5 ng/mL) standards were injected to UPLC-MS/MS as control. cps, counts per second. \n",
      ".Figure 5A) The SAM binding affinity of FTSJ1 as measured by ITC. (B The binding affinity of FTSJ1 alone or FTSJ1-WDR6 for tRNA analyzed by the gel mobility shift assay. C) For the reaction in figure C, 0.2 μΜ GST, 0.2 μΜ GST-WDR6, 1 μM FTSJ1, the mixture of 1 μM FTSJ1 with 0.2 μΜ GST-WDR6 or 0.2 μΜ GST, 0.5 mM SAM and 250 nM tRNAPhe(GAA) transcript were incubated. \n",
      ".Figure 2A) Schema showing a series of truncated FTSJ1 mutants used to map FTSJ1 domain(s) responsible for WDR6 binding. B, C) Co-IP assays of the interaction between FTSJ1-Flag or truncated FTSJ1 mutants and WDR6-HA in co-transfected HEK293T cells. Anti-Flag IP (top) were respectively immunoblotted by anti-HA and anti-Flag antibodies. Bottom: cell lysates immunoblotted by anti-HA or anti-Flag antibodies, with GAPDH serving as a loading control. D) The structure of DIZPK and scheme showing the capture of FTSJ1 interacting proteins using DIZPK. E) IP assays of FTSJ1-Asn221DIZPK. The empty vector was transfected as a negative control. (F) A list of top FTSJ1-Asn221DIZPK putative interacting proteins identified by IP-MS in HEK293T cells. \n",
      "-null embryos exhibit localised integrated stress response (A-L) mRNA in situ hybridization was performed at E9.Figure 5mRNA in situ hybridization was performed at E9.5 to detect Chac1 in (A-a6) wildtype and (B-b6) Slc7a5-null littermate embryos (n=3 for each condition); Chac1 expression at E10 in emerging limb buds assessed in (C) wildtype and (D) Slc7a5-null littermate embryos (n=2 for each condition) Scale bars 200 µm for whole embryo images and 100 µm for sections. mRNA in situ hybridization was performed at E9.5 to detect Trib3 mRNA detected in (E-e6) wildtype and (F-f6) Slc7a5-null littermate embryos (n=3 for each condition) and in emerging limb buds at E10.5 assessed in (G) wildtype and (H) Slc7a5-null littermate embryos (n=2 for each condition). Scale bars 200 µm for whole embryo images and 100 µm for sections. mRNA in situ hybridization was performed at E9.5 Chac1 and Trib3 transcripts were also detected in wildtype E9.5 embryos after >3 days in situ hybridisation reaction (n=3 and n=6 respectively). Lateral view (I and K), optic vesicles (white arrowheads), branchial arches (ba) and otic vesicles (ov). Dorsal view (J and L), neural tube (nt) and limb buds (white arrowheads). Scale bars 200 µm for whole embryo images and 100 µm for sections. (M) Schematic indicating regions (purple) in which high levels of Slc7a5 and ISR genes Trib3 and Chac1 are detected in wildtype embryos. \n",
      ".Figure 1(A) Immunofluorescence labeling of FTSJ1-Flag (red) in HEK293T cells. The nucleus was stained by DAPI (blue). Scale bar, 20 μm. (B) Subcellular localization of FTSJ1. Cytoplasmic (C) and nuclear (N) fractions were separated from HEK293T cells expressing FTSJ1-Flag. α-Tubulin and Lamin A/C were used as indicators of the cytoplasmic and nuclear fractions, respectively. (C) Co-IP assays of the interaction between FTSJ1 and WDR6 in co-transfected HEK293T cells. Anti-HA IP (top) and anti-Flag IP (middle) were immunoblotted by anti-HA and anti-Flag antibodies, respectively. Bottom: cell lysates immunoblotted by anti-HA or anti-Flag antibodies, with GAPDH serving as a loading control. (D) Co-IP assays of the interaction between FTSJ1 and WDR6 with RNase A addition. \n",
      "-null embryos and exogenous ER Stress induction in wildtype embryos upregulates Chac1 and Trib3 (A, B) Western blot of individual E9.Figure 6Western blot of individual E9.5 wildtype (n=4) and Slc7a5-null (n=4) embryo lysates immunoblotted using an antibody against phospho-GCN2 (Thr899) and total GCN2 (loading control), ***p = 0.0001 ; band intensities measured with FIJI and analysed with unpaired t-test Data information: All error bars indicate SEM. Western blot of individual E9.5 wildtype (n=4) and Slc7a5-null (n=4) embryo lysates immunoblotted using an antibody against phospho-eIF2α (Ser51) and total eIFα (loading control), *p = 0.0482, band intensities measured with FIJI and analysed with unpaired t-test Data information: All error bars indicate SEM. (C-F) phosphorylated eIFα was assessed by immunofluorescence in E9.5 embryos in the forebrain, wildtype (C,c1) (n=2 embryos, 0/14 sections) and Slc7a5-null (D, d1) (n=2 embryos 16/16 sections), and in hindbrain/anterior spinal cord, wildtype (E-e2) (n= 4 embryos, 0/62 sections) and Slc7a5-null (F-f2) (n= 4 embryos, 54/54 sections) including in the otic vesicle (compare e2, f2). Scale bars in (C-e1, D-f1) 100 µm and in (e2 and f2) 25µm. Arrowheads indicate ventral midline. (G) An increase in Trib3 protein was detected by Western blot in Slc7a5-null embryos, each lane represents an E9.5 embryo lysate (wildtype n=3, Slc7a5-null, n=4) with α-Tubulin loading control, unpaired t-test *p = 0.0153 Data information: All error bars indicate SEM. (H-L) TUNEL assay to detect apoptotic cells in wildtype (n=3 embryos, 32 sections) or Slc7a5-null (n=5 embryos, 47 sections); (H, I) transverse sections through the head at level of the forebrain and (J, K) the spinal cord, (H-K) scale bars 50 µm, arrowheads indicate ventral midline. (L) Quantification of TUNEL positive cells within spinal cord sections was performed Each dot represents the average apoptosis cell count for a single embryo, Welch's correction unpaired t-test, p = 0.250, F-test to compare variances, p = 0.0003 Data information: All error bars indicate SEM. \n",
      ".Figure 3A, B) Schematic depiction of ftsj1 and wdr6, and target sites of mutations introduced by the CRISPR-Cas9 system in HEK293T cells. Shaded and open boxes indicate coding regions and untranslated regions of exons, respectively. Lines indicate introns. For ftsj1, two sgRNA sequences for targeting exons 2 and 3 are noted. For wdr6, two sgRNA sequences for targeting exons 1 and 2 are noted. Sequences of both alleles of ftsj1 or wdr6 in KO1 and KO2 cell lines are aligned. Deleted nucleotides are indicated as dashed lines. (C) Secondary structures of human tRNAPhe(GAA) with modified nucleosides. Mass chromatograms of the nucleosides, Cm (Q1/Q3=258.1/112.1), Gm (Q1/Q3=298.1/152.1) and A (Q1/Q3=268.1/136.2) of tRNAPhe(GAA) isolated from WT, ftsj1 KO and wdr6 KO cells. Target peaks are indicated by black triangles. Cm, 2'-O-methylcytidine; Gm, 2'-O-methylguanosine; Q1/Q3: the mass of the precursor ion and the mass of the product ion. Mass chromatograms of the nucleosides m1G (Q1/Q3=298.1/166.1), m7G (Q1/Q3=298.1/166.1) and m2G (Q1/Q3=298.1/166.1), o2yW (Q1/Q3=541.2/409.0), m5C (Q1/Q3=258.1/126.1) and A (Q1/Q3=268.1/136.2) of tRNAPhe(GAA) isolated from WT, ftsj1 KO and wdr6 KO cells. Target peaks are indicated by black triangles. m2G, 2-methylguanosine; D, dihydrouridine; m22G, N2,N2-dimethylguanosine; m5C, 5-methylcytidine; m7G, 7-methylguanosine; m5U, 5-methyluridine; m1A: 1-methyladenosine; ψ: pseudouridine; o2yW, peroxywybutosine. Q1/Q3: the mass of the precursor ion and the mass of the product ion. Combined the retention time of standard product, we marked the o2yW in Figure 3G. Considering the change of this peak area was consistent with that of o2yW, we speculated the other peak was generated by the intermediate product of o2yW with nature isotope labelled. \n",
      " Wnt β-catenin signalling is unaffected in Slc7a5-null embryos, but is required for Slc7a5 transcription and β-catenin loss induces ISR gene Trib3 (A-B) (A,a1) Wnt β-catenin target Axin2 strongly expressed in littermate, (a2, a3) in TS, and (B, b1) in Scl7a5-null embryos (b2, b3) in TS (n=2 littermates, n=6/6 Slc7a5-null embryos).Figure 7(A-B) (A,a1) Wnt β-catenin target Axin2 strongly expressed in littermate, (a2, a3) in TS, and (B, b1) in Scl7a5-null embryos (b2, b3) in TS (n=2 littermates, n=6/6 Slc7a5-null embryos). Data information: White dashed lines indicate position of sections. Scale bars 200 µm (C, d1) Axin2 transcripts in embryo trunk explant exposed to DMSO (C, c1, n=12/12) were reduced in Wnt-c59 exposed explants (D, d1, n=14/14). Images in (c1, d1) show TS through explants (data from 3 independent embryo explant experiments). (E, F) Slc7a5 transcripts in embryo trunk explant exposed to DMSO (E, e1, n=9/10) or Wnt-c59 (F, f1, n=15/15). Images in (e1, f1) show TS through explants (data from 3 independent embryo explant experiments). Data information: Scale bars = 100 µm (G, H) Slc7a5 expression in control T-Cre;Ctnnb1flLOF/Δ heterozygous littermate embryo (G, g1) and in TS (g2, g3), and in T-Cre;Ctnnb1flLOF/Δ mutant embryo where Slc7a5 transcripts are absent in posterior neural tube (the region of T-cre recombination, indicated with white arrowhead in H) and shown in TS in (h2, h3) (n= 0/6 littermate and n=6/6 mutant embryos). Data information: White dashed lines indicate position of sections. Scale bars, 200 µm (I, J) Trib3 is lacking in T-Cre;Ctnnb1flLOF/Δ heterozygous control embryos at E7.75 (I), caudal region dissected from the same embryo (white dashed box in I) (i1) and viewed in TS (i2); Trib3 detected in homozygous T-Cre;Ctnnb1flLOF/Δ embryo (J), in caudal region dissected from the same embryo (white dashed box in J) (j1) and shown in TS in caudal epiblast and remnant primitive streak (ps) (j2) (n= 3/3 T-Cre;Ctnnb1flLOF/Δ embryos and 14 littermate controls). Trib3 was also detected in the more rostral forming neural tube at this early stage (seen in J) and so in cells not directly experiencing β-catenin loss; this may reflect the failure to form paraxial mesoderm, which provides signals that promote and support neurogenesis. Data information: White dashed lines indicate position of sections. Scale bars all sections 50 µm. (K-M) At E9.5 Trib3 was not detected in T-Cre;Ctnnb1flLOF/Δ heterozygous littermate embryos (K, k1) and in TS ( k2, k3), but was detected in homozygous T-Cre;Ctnnb1flLOF/Δ mutant embryos (L-m3) in patches of cells in neural tube (nt) (L - l2), gut (g) (l3) and somites (s) (m2, arrowheads) (n=9/9 T-Cre;Ctnnb1flLOF/Δ embryos and 12 littermate controls). Data information: White dashed lines indicate position of sections. Scale bars, all sections 50 µm. \n",
      ".Figure 6The representative graphs for quantification of the expected Nm32/A% and Nm34/A% ratios of G37-tRNAPhe(GAA), tRNALeu(UAA), tRNATrp(CCA), tRNAArg(ACG), tRNAArg(UCG), and A37-tRNAMet(CAU), which were isolated from WT, ftsj1 KO, and wdr6 KO cells by UPLC-MS/MS analysis. Error bars represent the standard deviation of three independent experiments. p values were determined using two-tailed Student's t-test for paired samples. ***p < 0.001. n.s., no significance. n.d., not detected. \n",
      ".Figure 8(A) Schema showing the identified tRNA substrates and recognition mechanism of FTSJ1. (B) The translation efficiency of UUU and UUC codons regulated by the interdependence of tRNAPhe(GAA) modifications at positions 32, 34, and 37. \n",
      ".Figure 3(A) Meta-analysis of footprint size of all 17 inhibitory pairs identified aligning the first codon of the pair in the ribosomal P site. (B) Metacodon analysis of 21 nt RPFs centered at the first codon of each inhibitory pair (red) compared to their corresponding optimal pair (green). P-values were determined by Student's t-test between each inhibitory and the corresponding optimal pair (** p < 0.01, n=2). (C) Comparison of 21 nt RPFs aligned at all 17 inhibitory codons from libraries made with CHX/ANS (blue) and CHX/TIG (black) (left) to their corresponding optimal pairs with the same antibiotic combination (right). No significant difference found (n.s.: p=0.9961, n=2) between CHX-TIG and CHX-ANS. \n",
      ".Figure 1(A) Inhibitory pairs showing the inhibitory mRNA codons (red) and the optimal codons (green). Schematic representation of the in vitro elongation reactions performed using the reconstituted yeast translation system. Pelleted 80S initiation complexes (3 nM) are incubated with aminoacyl-tRNAs (15 to 25 nM), elongation factors (1 µM), GTP, and ATP. Reactions are quenched with KOH and the products resolved by electrophoretic TLCs. (B) Representative eTLCs (left) and corresponding elongation kinetics (right) for the CGA-CGA inhibitory pair (red) and the CGC optimal pair (green). Product formation is normalized to the fraction of Met ICs that form Met-Puro when reacted with puromycin (Fig EV1A). (C) Representative eTLCs (left) and corresponding elongation kinetics (right) for the CGA-CCG inhibitory pair (red) and the CGC-CCA optimal pair (green). Product formation is normalized to the fraction of Met ICs that form Met-Puro when reacted with puromycin (Fig EV1A). \n",
      ".Figure 7(A, B) The growth curve of WT and two ftsj1 KO HEK293T cell lines under normal culture or in the culture condition with 0.2 mg/mL paromomycin assayed by cell counting kit-8 proliferation analysis. (C, D) The growth curve of WT with or without overexpressing tRNAPhe(GAA) under normal culture or with 0.2 mg/mL paromomycin. (E, F) The growth curve of ftsj1 KO cells with or without overexpressing tRNAPhe(GAA) under normal culture or with 0.2 mg/mL paromomycin. (G) The RNA reporter vector encodes F-luc as the primary reporter and R-luc on the same plasmid as the internal transfection control. The effect of tRNAPhe(GAA) in protein translation was revealed by a reporter assay. 6×TTC (Phe) or 6×TTT (Phe) were inserted after the PGK promoter region of F-luc as the positive reporter. (H, I) The translation efficiency of F-luc with the 6×UUC codons or 6×UUU codons. (J, K) The translation efficiency of F-luc with all UUC codons or all UUU codons. (L) The frequency usage analysis of UUU and UUC codons showed that in 488 genes at least one transcript was found to use the UUU codon for more than 80% of Phe codons. Among the 488 genes, 189 genes are involved in the nervous system and brain functions. \n",
      ".Figure 2(A) Comparison of observed rates of elongation for inhibitory pairs (red) and their optimal controls (green) at limiting tRNA concentrations (hatched bars, 15 to 25 nM aa-tRNA) and saturating tRNA concentrations (solid bars, 150 to 250 nM aa-tRNA). (B) Comparison of total peptide formation for inhibitory pairs (red) and their optimal controls (green) at limiting tRNA concentrations (hatched bars, 15 to 25 nM aa-tRNA) and saturating tRNA concentrations (solid bars, 150 to 250 nM aa-tRNA). (C) Elongation kinetics for the CGA-CGA inhibitory codon pair with the native arginine ICGtRNA (red) or the non-native arginine UCGtRNA (pink) and for the CGC-CGC optimal control pair with the native arginine ICGtRNA (green). \n",
      ": Ribosomes stalled on poly(A) stretches reveal alterations in both the peptidyl-transferase and decoding centers.Figure 5(A, B) Schematic representation (A) of the stalling situation on poly(A) tract mRNA. Cryo-EM density map of the poly(A) stalled 80S RNC filtered according to local resolution and used to build the molecular model (B). (C) Cryo-EM density (mesh) and stick model with cartoon phosphate backbone representing the peptidyl-tRNA in the peptidyl-transferase center (PTC). (D) General overview of the A, P, and E sites with the P/P tRNA and mRNA. (E, F) Detailed view of the mRNA in the A site using sticks model with cartoon phosphate backbone representation and cryo-EM density (mesh). The poly adenine mRNA sequence forms a π-stacking array between positions +4 and +7, which is stabilized from both sides by stacking of 18S rRNA bases C1634 and A1756. \n",
      ": CGA-CCG and CGA-CGA induce stalling through decoding-incompatible mRNA conformations in the A site (A-C) Cryo-EM structural characterization of the pre-state 80S RNC with A site tRNA in the decoding center.Figure 4(A-C) Cryo-EM structural characterization of the pre-state 80S RNC with A site tRNA in the decoding center. (A) Schematic representation of the decoding situation (top) and molecular model for the pre-state RNC with A site tRNA in the decoding center. (B) General overview of the A, P and E sites with A/A and P/P tRNAs and mRNA. (C) Detailed view of the mRNA in the A site using sticks model with cartoon phosphate backbone representation. The 18S rRNA bases A1755 and A1756 recognize the minor groove of A site tRNA - mRNA interaction during tRNA decoding. (D-F) Cryo-EM structural characterization of the CGA-CCG stalled 80S RNC. (D) Schematic representation of the stalling situation (top) and molecular model of the CGA-CCG stalled RNC (bottom). (E) General overview of the A, P, and E sites with P/P and E/E tRNAs and mRNA. (F) Detailed view of the mRNA in the A site using sticks model with cartoon phosphate backbone representation. The mRNA positions +2 to +5 and their interactions are shown. The C +4 is flipped by approximately 95° degrees towards the wobble A:I base pair in the P site and stabilized by interaction with the C1637 of the 18S rRNA helix 44. The C +5 is stabilized by stacking interaction with the A1756 of the 18S rRNA which normally recognizes the minor groove of A site tRNA - mRNA interaction during decoding (C). (G-I) Cryo-EM structural characterization of the CGA-CGA stalled 80S RNC. (G) Schematic representation of the stalling situation (top) and molecular model of the CGA-CGA stalled RNC (bottom). (H) General overview of the A, P and E sites with P/P and E/E tRNAs and mRNA. (I) Detailed view of the mRNA in the A site as in (F). Downstream mRNA is indicated by the dotted line. Note the rotation of the C+4 base compared to the CGA-CCG mRNA. \n",
      "Sequence features of SARS-CoV-2 nucleocapsid proteinFigure legend: A.Figure 1:Sequence features of SARS-CoV-2 nucleocapsid proteinFigure legend: A. The complete whole genomic features of SARS-CoV-2 isolate Wuhan-Hu-1 (Genebank: MN908947). UTR: untranslated region; orf/ORF: open reading frame; TRS: transcriptional regulatory sequences; S: spike glycoprotein encoding region; E: envelope protein encoding region; M: membrane protein encoding region; N: nucleocapsid protein encoding region. The figure is illustrated by SnapGene Viewer; B. Domain architectures of coronavirus nucleocapsid protein. NTD: N-terminal RNA-binding domain; CTD: C-terminal dimerization domain; C. Multiple sequence alignment of SARS-CoV-2 N-NTD with SARS-CoV N-NTD (UniProtKB: P59595), MERS-CoV N-NTD (UniProtKB: R9UM87), HCoV-OC43 N-NTD (UniProtKB: P33469). Red arrows indicate conserved residues for ribonucleotide binding site, dash boxes indicate variably residues in the structural comparisons. \n",
      ": Disomes stalled on poly(A) tracts form a novel POST-POST assembly.Figure 6(A, B) Composite cryo-EM density map (A) of the POST-POST disome stalled on the poly(A) mRNA reporter filtered according to local resolution and used to build the molecular model (B). (C) Cut top views of both the first (stalling) and the second (colliding) ribosomes forming the disome. Observed ribosomal and tRNA translocation states are indicated. (D) Comparison of ribosomal assemblies between the previously described CGA-CCG stalled yeast disome in pink (EMD-4427, Ikeuchi, Tesina et al., 2019a) and the novel POST-POST assembly observed in poly(A) stalling. The EMD-4427 density map was fitted into the density of the first stalling ribosome on the poly(A) reporter. The indicated rotation was calculated using the 60S subunit, as the compared colliding ribosomes are not in the same translocation state (PRE vs. POST). \n",
      "Comparison of SARS-CoV-2 N-NTD with related viral N-NTD structuresFigure legend: A.Figure 3:A. Mapping the conserved surfaces of four CoV N-NTDs in SARS-CoV-2 N-NTD structure. The multiple sequence alignment used for mapping is shown in Fig. 1C. Blue denotes conserved residues among 4 CoVs N-NTD; green denotes one variation among 4 CoVs N-NTD; pink denotes two variations among 4 CoVs N-NTD; red denotes three variations among 4 CoVs N-NTD B. Electrostatic surface of the SARS-CoV-2 N-NTD, SARS-CoV N-NTD, MERS-CoV N-NTD, HCoV-OC43 N-NTD. Blue denotes positive charge potential, while red indicates negative charge potential; C. Overall structural comparison of SARS-CoV-2 N-NTD with related viral N-NTD structures. Left: superimposition of SARS-CoV-2 N-NTD (blue) to SARS-CoV N-NTD (green); middle: superimposition of SARS-CoV-2 N-NTD (blue) to MERS-CoV N-NTD (brown); right: superimposition of SARS-CoV-2 N-NTD (blue) to HCoV-OC43 N-NTD (orange). \n",
      "Structural overview of SARS-CoV-2 N-NTDFigure legend: A.Figure 2:A. Ribbon representation of SARS-CoV-2 N-NTD molecules in one asymmetric unit. The four molecules are highlighted with different color respectively B. Superimpositions of four molecules in one asymmetric unit. The dash circles indicate the sandwiched structure composed of Loop region 1, β-sheet core, and Loop region 2. The β-strand is labeled with β1 to β7, the 310 helix is labeled with η1 C. Topological style illustration of SARS-CoV-2 N-NTD structure D. Electrostatic surface of the SARS-CoV-2 N-NTD. Blue denotes positive charge potential, while red indicates negative charge potential. The potential distribution was calculated by Pymol. The values range from −5 kT (red) to 0 (white) and to +5 kT, where k is the Boltzmann constant and T is the temperature. \n",
      "A potential unique drug target pocket in SARS-CoV-2 N-NTDFigure legend: A.Figure 4:A potential unique drug target pocket in SARS-CoV-2 N-NTDFigure legend: A. Detailed view of ribonucleotide binding pocket in superimposition structures between SARS-CoV-2 N-NTD with HCoV-OC43 N-NTD AMP complex. AMP, interacting residues and equivalents are highlighted with stick representation; B. Electrostatic surface of the potential ribonucleotide binding pocket on SARS-CoV-2 N-NTD; C. Electrostatic surface of the ribonucleotide binding pocket on HCoV-OC43 N-NTD; D. Detailed view of phosphate group binding site in superimposition structures between SARS-CoV-2 N-NTD with HCoV-OC43 N-NTD AMP complex; E. Dot representation of SARS-CoV-2 residues Thr 55 and Ala 56, which indicates potential steric clashes with the ribonucleotide phosphate group; F. Dot representation of HCoV-OC43 N-NTD residues Ser 67 and Gly 68; G. Detailed view of nitrogenous base binding site in superimposition structures between SARS-CoV-2 N-NTD with HCoV-OC43 N-NTD AMP complex; H. Electrostatic surface of the potential ribonucleotide nitrogenous base binding pocket on SARS-CoV-2 N-NTD; I. Electrostatic surface of the ribonucleotide nitrogenous base binding pocket on HCoV-OC43 N-NTD. In electrostatic surface potential panels, blue denotes positive charge potential, while red indicates negative charge potential. The potential distribution was calculated by Pymol. The values range from −5 kT (red) to 0 (white) and to +5 kT, where k is the Boltzmann constant and T is the temperature. \n",
      " - ER whorls arise from ER stacks and undergo microautophagy.Figure 2A Individual frames from time-lapse imaging of a cell expressing GAL promoter-driven GFP-Pho8 and stained with the vacuole membrane dye FM4-64. Numbers indicate the time in minutes after the start of the image sequence. Scale bar: 2 µm. B Electron micrograph of a cell expressing GPD promoter-driven GFP-Pho8. Arrows indicate the edges of ER cisternae that are part of a bent stack in a vacuole membrane invagination. Black arrowheads point to vacuole membrane that encases the bent stack and the whorl. C Fraction of uptake intermediates that were converted into vacuolar whorls or stacks. n = 43. The image sequence shows formation of a vacuolar whorl that becomes a cytosolic whorl. Scale bar: 2 µm. \n",
      " - Macro- and micro-ER-phagy are parallel pathways with distinct morphologies and molecular requirements.Figure 4A Electron micrograph of wild-type yeast treated with DTT for 3 h. Inset 1 shows an ER whorl and inset 2 shows ER fragments in macroautophagic bodies (white arrows). Cy, cytosol; V, vacuole. B Relative activity of cyto-Pho8∆60 after nitrogen starvation, which reports on non-selective macroautophagy of cytosol. Mean + SEM, n ≥ 3. WT, wild-type. Relative activity of the ER-phagy reporter Yop1-Pho8∆60 after nitrogen starvation (C) Mean + SEM, n ≥ 3. Relative activity of the ER-phagy reporter Yop1-Pho8∆60 after tunicamycin treatment (D, E). Mean + SEM, n ≥ 3. F, G Growth assays of untreated and tunicamycin-treated WT, ∆atg39/40, ∆snf7 and ∆snf7 ∆atg39/40 cells. Numbers indicate growth relative to WT cells based on the areas under the curves. Mean ± SEM, n = 3. \n",
      " - Expression of ER-resident transmembrane proteins can induce autophagy of ER whorls.Figure 1A Freeze-fracture electron micrograph of wild-type yeast treated with DTT for 3 h. The left panel shows two spherical structures, one in a vacuole membrane invagination and one inside the vacuole. The middle panel shows the boxed area of the left panel at higher magnification. Arrows indicate membrane layers that are free of intramembrane particles. The right panel shows the protoplasmic fracture face (P face) of cortical ER with a high density of intramembrane particles. Cy, cytosol; N, nucleus; V, vacuole. B Fluorescence images of cells expressing GPD promoter-driven GFP-Pho8 and the ER marker Sec63-mCherry and stained with CMAC to label the vacuole. The upper and lower panel show vacuole-associated and perinuclear GFP-Pho8 stretches and an intravacuolar GFP-Pho8 ring, respectively. Scale bars: 2 µm. C Correlative light and electron microscopy of cells expressing GPD promoter-driven GFP-Pho8 and Sec63-mCherry. The lower panel shows the boxed area in the electron micrograph at higher magnification. The ER ring corresponds to a vacuolar whorl and the ER stretch to a perinuclear stack. D Fluorescence images of cells expressing Hmg2-GFP and either additionally expressing Sec63-mCherry or stained with CMAC. The upper and lower panel show cytosolic and vacuolar Hmg2-GFP rings, respectively. Scale bars: 2 µm. \n",
      " - The Nem1-Spo7 complex and ESCRTs are key components for micro-ER-phagy.Figure 3Fluorescence images of ∆atg7 and ∆atg8 cells (A) expressing GAL promoter-driven GFP-Pho8 and the ER marker Sec63-mCherry and stained with CMAC to label the vacuole. Scale bar: 2 µm. Fluorescence images of ∆snf7 and ∆vps4 cells (B) expressing GAL promoter-driven GFP-Pho8 and the ER marker Sec63-mCherry and stained with CMAC to label the vacuole. Scale bar: 2 µm. C Relative activity of the ER-phagy reporter Yop1-Pho8∆60 in tunicamycin-treated cells. The dotted line indicates the activity in ∆atg7 cells. Mean + SEM, n ≥ 3. D, E Relative activity of endogenous Pho8 in untreated cells (D) or after tunicamycin treatment (E). Background activity determined in ∆pho8 mutants was subtracted from the activity in all other strains. Mean + SEM, n = 3. WT, wild-type. F Western blot of GFP from cells expressing Sec63-GFP and treated with tunicamycin (Tm) where indicated. The two Atg7-independent cleavage products of around 55 kDa were used as readout for micro-ER-phagy. Pgk1 served as a loading control. G Quantification of Sec63-GFP cleavage normalized to the WT from western blots as shown in panel (F). Mean + SEM, n = 4. \n",
      " - ESCRT machinery mediates vacuole membrane dynamics during micro-ER-phagy of GFP-Pho8 structures.Figure 5A Fraction of wild-type (WT), ∆vps23, ∆snf7 and ∆vps4 cells with GFP-Pho8 structures. Mean + SEM, n ≥ 3. B Fraction of GFP-Pho8 structures classified as stacks, cytosolic whorls, uptake intermediates or vacuolar whorls. The number of GFP-Pho8 structures in WT, ∆vps23, ∆snf7 and ∆vps4 cells was 562, 345, 367 and 449, respectively. The table shows the results of Chi-Square tests for independence for all pairs of strains. ***, highly significant (P < 0.00001); ns, not significant (P > 0.05). C Fraction of stacks classified as peripheral, perinuclear or vacuole-associated. The number of stacks in WT and ∆snf7 cells was 252 and 351, respectively. The observed distributions in the two strains were analyzed with a Chi-Square test for independence. ***, highly significant (P < 0.00001). Fluorescence images of a cell expressing non-fluorescent GFP(Y66F)-Pho8, Snf7-mNeonGreen and the ER marker mCherry-Ubc6, and stained with the vacuole dye CMAC. Numbers indicate the time in minutes after the start of the image sequence. White arrows indicate a Snf7-mNeonGreen punctum adjacent to an ER whorl. Black arrow point to a vacuole membrane invagination. In this series, mNeonGreen fluorescence was adjusted differently for different time points to compensate signal decay over time. Scale bar: 2 µm. Fluorescence images of a cell expressing non-fluorescent GFP(Y66F)-Pho8, Snf7-mNeonGreen and the ER marker mCherry-Ubc6, and stained with the vacuole dye CMAC. Numbers indicate the time in minutes after the start of the image sequence. White arrows indicate a Snf7-mNeonGreen punctum adjacent to an ER whorl. Black arrow point to a vacuole membrane invagination. Panel (E) illustrates transient appearance of Snf7 at an ER whorl. Panel (F) illustrates orientation of a Snf7 punctum towards the neck of a vacuole membrane invagination. In this series, mNeonGreen fluorescence was adjusted differently for different time points to compensate signal decay over time. Scale bar: 2 µm. \n",
      " - Model of ER-phagy.Figure 7During micro-ER-phagy, portions of the ER (blue) are reorganized to form stacks (1). ER stacks are bent, recognized by the vacuole, taken up into vacuole membrane invaginations, converted into whorls and detached from the remainder of the ER (2). The order of these events is uncertain. ESCRT machinery is recruited to sites of ER whorl uptake and mediates scission of the vacuole membrane (3). In parallel, macro-ER-phagy can also selectively target the ER, which is mediated by the core autophagy machinery. \n",
      " - ESCRT machinery is required for completion of microautophagy of ER whorls during ER stress.Figure 6A Electron micrographs of wild-type and ∆snf7 cells treated with DTT for 3 h. Cy, cytosol; V, vacuole. Scale bars: 300 nm. B Quantification of vacuolar whorls for which the uptake process was complete or incomplete. The observed distributions in the two strains were analyzed with a Fisher's exact test. C, D Electron micrographs of serial thin sections of a ∆snf7 cell (C) or a wild-type cell (D) treated with DTT for 3 h. Note that the whorl appears separated from the cytosol in some sections (e.g. section 300 nm) but is still in contact with the cytosol (see sections 600 nm and 700 nm). In the wild-type cell (D), the vacuolar whorl is completely separated from the cytosol. Scale bar: 300 nm. \n",
      ".Figure 1A, B Immunoblot analyses of the protein abundance of the METTL3/METTL14/WTAP heterotrimer in PBMCs (A) and BMDMs (B) upon challenge with VSV-eGFP, poly(I:C), HSV-1 or HSV60 at the indicated time points. C, D Immunoblot analyses of the protein abundance of the METTL3/METTL14/WTAP heterotrimer in A549 cells upon challenge with VSV-eGFP or SeV (C) and poly (I:C) or poly (dA:dT) (D) at the indicated time points. \n",
      ".Figure 2A, B Real-time PCR (qRT-PCR) analyses of WTAP mRNA expression in PBMCs (A) or BMDMs (B) upon challenge with VSV-eGFP, poly(I:C), HSV-1 or HSV60 at indicated time points. n = 3 independent biological replicates, and error bars represent standard deviations. C qRT-PCR analyses of WTAP mRNA expression in A549 cells upon challenge with VSV-eGFP, poly(I:C), HSV-1 or poly (dA:dT) at the indicated time points. D qRT-PCR analyses of WTAP mRNA expression in A549 cells stimulated with IFN-α or -β at the indicated time points. E A549 cells were stimulated with poly (I:C) for 16 hrs and then treated with PR-171 (10 μM) or PS-341 (10 μM) for additional 4 hrs and 8 hrs. The cell lysates were analyzed by immunoblotting. Representative images from 3 independent biological experiments are shown (left), and the relative levels of WTAP are presented as the mean ± SD (right). F A549 cells were stimulated with poly (I:C) for 16 hrs and then treated with CQ (50 mM) or Baf A1 (0.2 mM) for additional 4 hrs and 8 hrs. The cell lysates were analyzed by immunoblotting. LC3B served as a good marker of the autophagy process. Representative images from 3 independent biological experiments are shown (left), and the relative levels of WTAP are presented as the mean ± SD (right). G Immunoassays of extracts of A549 cells infected with VSV-GFP for 16 hrs and then treated with PR-171 (10 μM) or DMSO for additional 4 and 8 hrs, followed by immunoprecipitation with anti-WTAP antibody and immunoblot analysis with anti-Ub antibody. H Coimmunoprecipitation and immunoblot analyses of extracts of 293T cells transfected with various combinations of plasmids encoding FLAG-tagged WTAP, and HA-tagged K48-linked, K63-linked or wild-type ubiquitin and treated with DMSO or PR171 (10 μM). \n",
      ".Figure 3A WTAP knockout efficiency in 293T cells detected by immunoblotting. B Luciferase activity analyses in wild-type (WT), or WTAP-/- 293T cells, transfected with a luciferase reporter for IFN-β (IFN-β-luc) or ISRE (ISRE-luc) followed by treatments with intracellular poly(I:C), poly(dA:dT), or VSV-eGFP. The results are expressed relative to Renilla luciferase activity. C qRT-PCR analyses to detect the mRNA abundances of IFNB1, RANTES, DDX58, and ISG15 in WT and WTAP-/- 293T cells followed by treatments with intracellular poly(I:C), poly(dA:dT), or VSV-eGFP. D Immunoblot analyses to detect the WTAP knockdown efficiency in A549 cells after transfection with Ctrl or WTAP-specific siRNAs for 48 hrs. E, F qRT-PCR analyses to detect the mRNA abundance of IFNB1 (E), RANTES and ISG15 (F) in A549 cells transfected with Ctrl or WTAP-specific siRNA, followed by treatment with poly(I:C)-HMW at the indicated time points. G, H ELISA analyses to detect IFN-β (G) and CCL5 (H) secretion in supernatants at 16 h. I, J Immunoblot analyses to detect the protein expression of ISGs in shCtrl and shWTAP A549 cells that were stimulated with poly(I:C)-LMW (I) or poly(dA:dT) (J) at different time points with the indicated antibodies. K, L qRT-PCR analyses to detect the mRNA abundances of IFNB1 (K) and RANTES (L) in A549 cells transfected with empty vector (EV) or WTAP-HA plasmid, followed by treatment with poly(I:C)-HMW at the indicated time points. M, N ELISA analyses to detect IFN-β (M) and CCL5 (N) secretion in supernatants \n",
      ".Figure 4A, B Immunoblot analyses of total and phosphorylated (p-) TBK1 and IRF3 in A549 cells transfected with Ctrl or WTAP-specific siRNA, followed by stimulation with poly (I:C) (A) or poly (dA:dT) (B) at different time points. C Immunoblot analyses of IRF3 dimerization in A549 cells transfected with Ctrl or WTAP-specific siRNA, followed by stimulation with poly (I:C) or poly (dA:dT) for 6 hrs. D Immunoblot analyses to detect the protein expression of ISGs in A549 cells transfected with Ctrl or WTAP-specific siRNA, followed by stimulation with IFN-α or -β (50 U/ml) for 4 hrs. E, F Immunoblot analyses of total and phosphorylated (p-) STAT1 in shCtrl and shWTAP A549 cells, followed by stimulation with IFN-α (E) or -β (F) at different time points. \n",
      ".Figure 5A, B Immunoblot analyses to detect the protein expression of critical adaptors in the IFN-I and Jak-STAT signaling pathway in A549 cells after transfection with Ctrl or WTAP-specific siRNAs for 48 hrs. C ELISA quantification of m6A levels in mRNA extracted from shCtrl and shWTAP A549 cells. D, E Schematic representation of the position of m6A motifs within the IRF3 (D) and IFNAR1 (E) transcripts. The sites of the m6A RNA modification element 'DRACH' in IRF3 mRNA 5'-UTRs or IFNAR1 mRNA 3'-UTRs were predicted using RMBase v2.0 F1/R1 represents the detection site-1 containing three (IRF3) or four (IFNAR1) 'DRACH' elements and F2/R2 represents the detection site-2 containing two (IRF3) or three (IFNAR1) 'DRACH' elements. F, G Abundance of IRF3 (F) and IFNAR1 (G) transcripts (detection site-1 and -2) among mRNA immunoprecipitated with anti-m6A antibody from shCtrl and shWTAP A549 cells. n = 3 independent biological replicates, and error bars represent standard deviations. H Abundance of IRF3 and IFNAR1 transcripts (detection site-1) among mRNAs immunoprecipitated with anti-m6A antibody from A549 cells that were transfected with empty vector (EV) or HA-tagged WTAP plasmid for 24 hrs. I qRT-PCR analyses of the mRNA expression of IRF3 and IFNAR1 in shNC and shWTAP A549 cells. J Semi-qRT-PCR analyses of IRF3 and IFNAR1 mRNA and RNU6 and GAPDH mRNA (loading controls throughout) in the nucleus (Nu) and cytoplasm (Cyto) of A549 cells transfected with Ctrl or WTAP-specific siRNA for 48 hrs. K qRT-PCR analyses of IFNAR1 mRNA expression in shCtrl and shWTAP A549 cells, followed by treatment with Act D (5 μg/mL) for the indicated times. L Immunoblot analyses of IFNAR1 in shCtrl and shWTAP A549 cells, followed by treatment with Act D (5 μg/mL) for the indicated times. M qRT-PCR analyses of the proportion of IRF3 mRNA in polysome fractions from A549 cells transfected with Ctrl or WTAP-specific siRNAs. N, O Wild-type or mutated m6A consensus sequences (A-to-T mutation) of IFNAR1-3' UTR (N) and IRF3-5' UTR (O) were fused with Renilla luciferase reporter in psiCHECKTM-2 vector. P Relative luciferase activities in 293T cells after transfection with reporter vectors bearing IFNAR1-3' UTR or IRF3-5' UTR with wild-type (WT) or mutated (Mut) m6A sites. Renilla luciferase activity was measured and normalized to firefly luciferase activity. Q Relative luciferase activities of reporter vectors bearing IFNAR1-3' UTR or IRF3-5' UTR with wild-type (WT) m6A sites after cotransfection with empty vector (EV) or HA-tagged WTAP into shWTAP A549 cells. Renilla luciferase activity was measured and normalized to firefly luciferase activity. \n",
      " Loss of the TNC protein in NeuNT tumors impacts gene expression and the immune cell infiltrate(A-C) Display of differentially expressed genes in NeuNT tumors (WT, TNCKO) as volcano plot (A) and heat maps (B, C) representing all (A) and immunity associated genes to be low (B) or high (C) in WT tumors.Figure 1Display of differentially expressed genes in NeuNT tumors (WT, TNCKO) as volcano plot representing all associated genes in WT tumors. N = 3 tumors per genotype. (A) Log2 Fold Change (X-axis) and the negative log10 (P.Value) (Y-axis) is displayed, the blue line indicating p = 0.05 (Y-axis), points above the line, p < 0.05. The dashed red lines indicate the log 2 Fold Change cutoff of -0.25 and 0.25, respectively (X-axis). Green dots represent genes that display both statistical significance (p < 0.05) and fold changes -0.25 and 0.25, respectively. Selected genes regulated by TNC are marked in red (type I interferon response) or pink (Cxcl12) dot. Display of differentially expressed genes in NeuNT tumors (WT, TNCKO) as heat maps (B, C) representing immunity associated genes to be low (B) or high (C) in WT tumors. N = 3 tumors per genotype. (D, E) Infiltration of F4/80+ macrophages determined by flow cytometry. (WT, N = 10, TNCKO, N = 7 tumors), * p = 0.0304, unpaired t-test and iNOS+ macrophages (WT, N = 9, TNCKO, N = 7 tumors) * p = 0.0197, unpaired t-test. Mean ± SEM. \n",
      " breast cancer model(A) Representative staining images (N = 5) of tissue from TNC high (WT/shC) and TNC low (KO/shTNC) tumors (N = 5 per condition) with the indicated antibodies.Figure 2(A) Representative staining images (N = 5) of tissue from TNC high (WT/shC) and TNC low (KO/shTNC) tumors (N = 5 per condition) with the indicated antibodies. Arrows point at the respective immune subtype. Scale bar, 50 µm. (B, C) Quantification of CD8 TIL upon tissue staining as total number per field (B) and in the stroma or tumor cell nest (C), expressed as relative ratio, *** p = 0.0002, Fisher's exact test and in TNC high (N = 5) and TNC low tumors (N = 5). 3 sections per tumor, 8 random fields per section. Mean ± SEM. (D, E) Apoptosis measurement upon tumor staining for cl. casp3 (D) and signal quantification per field (E) in TNC high and TNC low tumors (N = 7 per condition), 2 sections per tumor, 5 random fields. *** p = 0.0006, Mann-Whitney test, Mean ± SEM. Scale bar, 50 µm. (F) Representative staining images (N = 5 tumors, 3 sections per tumor) for the indicated molecules. Scale bar, 50µm. Arrows point at CD8 TIL. (G, Gzmb expression as determined by qRT-PCR. N = 6 tumors per condition. ** p = 0.0022 for both graphs, Mann Whitney test. Mean ± SEM. H) Perforin expression as determined by qRT-PCR. N = 6 tumors per condition. ** p = 0.0022 for both graphs, Mann Whitney test. Mean ± SEM. (I, J) Detection (I) and quantification (J) of CD8 TIL in lung metastasis of NeuNT WT and TNCKO mice (N = 3 mice, n = 7 and 12 lung metastases, respectively). Scale bar, 100 µm. ** p = 0.0097, Mann Whitney test. Mean ± SEM. \n",
      "(A) Boyden chamber experiment to measure attraction and retention of CD8 T cells towards factors in the lower chamber.Figure 4(A) Boyden chamber experiment to measure attraction and retention of CD8 T cells towards factors in the lower chamber. Cells in the lower chamber were measured by flow cytometry (attraction) or on the matrix coatings (retention) by counting Dapi stained nuclei 5 hours after plating. Attraction and retention were determined towards coatings with collagen I (COL), fibronectin (FN) or TNC in the presence of CXCL10, CCL21, CXCL12 (B, C) 3 independent experiments in duplicates, Mann-Whitney test, ns > 0.05; * p <0.05; **p < 0.01; ***p < 0.005. Mean ± SEM. retention were determined towards coatings with collagen I (COL), fibronectin (FN) or TNC in the presence of CXCL12 and, upon AMD3100 (AMD, 5 µg/mL) (D) CXCL12 (indicated concentrations) was added overnight at 4°C during the coating followed by washing (2 times with PBS) and plating CD8 T cells for 5 hours before fixation and counting Dapi stained nuclei. 3 independent experiments in duplicates, Mann-Whitney test, ns > 0.05; * p <0.05; **p < 0.01; ***p < 0.005. Mean ± SEM. retention were determined towards coatings with collagen I (COL), fibronectin (FN) or TNC in the presence of CXCL10 and, upon AMD3100 (AMD, 5 µg/mL) 3 independent experiments in duplicates, Mann-Whitney test, ns > 0.05; * p <0.05; **p < 0.01; ***p < 0.005. Mean ± SEM. Attraction were determined towards coatings with collagen I (COL), fibronectin (FN) or TNC in the presence of CM from shC cells (CM:shC) and, upon AMD3100 (AMD, 5 µg/mL) 3 independent experiments in duplicates, Mann-Whitney test, ns > 0.05; * p <0.05; **p < 0.01; ***p < 0.005. Mean ± SEM. retention were determined towards coatings with collagen I (COL), fibronectin (FN) or TNC in the presence of CM from shC cells (CM:shC) and, upon AMD3100 (AMD, 5 µg/mL) 3 independent experiments in duplicates, Mann-Whitney test, ns > 0.05; * p <0.05; **p < 0.01; ***p < 0.005. Mean ± SEM. \n",
      " expression and binding to TNC(A) Heat map representation of the most 17 deregulated Panther identified immune related genes in TNC high (WT/shC) and TNC low (KO/shTNC) tumors (N = 2 tumors each).Figure 3(A) Heat map representation of the most 17 deregulated Panther identified immune related genes in TNC high (WT/shC) and TNC low (KO/shTNC) tumors (N = 2 tumors each). (B) CXCL12 protein levels in NT193 tumors (ELISA). N = 7 tumors per group, ** p = 0.0035 (WT host, shC versus sh1TNC), ** p = 0.0070 (WT host, shC versus sh2TNC), * p = 0.0379 (TNCKO host, shC versus sh1TNC), * p = 0.0262 (TNCKO host, shC versus sh2TNC), Mann-Whitney test. Mean ± SEM. (C) Cxcl12 levels in cultured tumor cells (qRT-PCR). N = 5 independent experiments, ** p = 0.0079, Mann Whitney test. Mean ± SEM. (D) CXCL12 protein levels (ELISA) in conditioned medium (CM) from cultured tumor cells, N = 6 independent experiments, ** p = 0.0022, Mann Whitney test. Mean ± SEM. (E) Analysis of Cxcl12 (qRT-PCR) in tumor cells upon pretreatment with the indicated inhibitors for 45 minutes (GW, BOP, DMSO and PBS as control respectively) or 60 minutes (Sunitinib) and 6 hours (Cli95) (6 hours DMSO as control) before incubation with TNC for 24 hours. N = 6 independent experiments, ** p = 0.0022, Mann Whitney test. Mean ± SEM. CXCL12 binding to TNC as determined by SPR measurement CXCL12 binding to TNC as determined by negative EM imaging (G), followed by quantification (H). (G) scale bars, 100 nm (left), 50 nm (right), the arrowheads point at gold labeled CXCL12 and the asterisk points at the fibrinogen globe of the murine TNC monomer. (H) Representation of TNC monomer, oligomerization domain (triangle) to form hexamers as seen in (G), FNIII repeats (grey boxes, contant domains, white boxes, alternative domains), fibrinogen like domain (circle). Representative result (3 independent experiments). \n",
      ".Figure 6A Immunoblot analyses to detect IRF3 and IFNAR1 protein expression in A549 cells stimulated with poly (I:C) or poly (dA:dT) at the indicated time points. B Immunoblot analyses to detect IRF3 and IFNAR1 protein expression in A549 and L929 cells infected with VSV-eGFP at the indicated time points. C, D qRT-PCR analyses of the mRNA abundance of IFNAR1 and IRF3 in A549 cells transfected with poly (I:C)-LMW (C) or infected with VSV-eGFP (D) for 16 hrs, followed by treatment with Act D (5 μg/mL) for the indicated times. E qRT-PCR analyses of the proportion of IRF3 and IFNAR1 mRNAs in polysome fractions from A549 cells transfected with poly (I:C)-LMW for 16 hrs. F Relative luciferase activities of IFNAR1-3' UTR or IRF3-5' UTR with wild-type (WT) m6A sites in 293T cells that were transfected with poly (I:C)-LMW or infected with VSV-eGFP for 24 hrs. Renilla luciferase activity was measured and normalized to firefly luciferase activity. G Immunoblot analyses of IRF3 and IFNAR1 in shCtrl and shWTAP A549 cells, followed by infection with VSV-eGFP at the indicated time points. H, Immunoblot analysis of IRF3 and IFNAR1 in A549 cells transfected with poly (I:C)-LMW or infected with VSV-GFP for 16 hrs and then treated with PR-171 or DMSO for additional 4 and 8 hrs. Representative images from 3 independent biological experiments are shown (H), protein abundance of IRF3 and IFNAR1 in A549 cells transfected with poly (I:C)-LMW or infected with VSV-GFP for 16 hrs and then treated with PR-171 or DMSO for additional 4 and 8 hrs. the relative levels of IFNAR1 (left) and IRF3 (right) are presented as the mean ± SD (I).  \n",
      " leukocytes and macrophages, and reduces tumor cell death and tumor growth (7 weeks model)(A) Growth of TNC-high (WT/shC) tumors upon cell engraftment and 5 weeks AMD treatment.Figure 5(A) Growth of TNC-high (WT/shC) tumors upon cell engraftment and 5 weeks AMD treatment. N = 20 tumors per condition, *** p = 0.0007, unpaired t-test.The central band represents the median, the ends of the box the first and the third quartiles, and the whiskers reach from each quartile to the minimum or maximum. (B, C) Imaging and signal quantification upon staining for cl. casp3 and DAPI in PBS and AMD treated tumors, N = 5 tumors per condition, 2 sections per tumor, 5 random fields, ** p = 0.0079, Mann-Whitney test. Mean ± SEM. Scale bar, 50µm, arrows point at cl. Casp3. (D, E) CD8 TIL abundance and localization in PBS and AMD treated tumors, arrow points at CD8 TIL, scale bar, 50µm (D). (E) Quantification of CD8 T cells per randomly chosen fields. N = 5 tumors per condition, 2 sections per tumor, 5 random fields, ** p = 0.0079, Mann-Whitney test. Mean ± SEM. (F) Gzmb levels in PBS and AMD treated tumors. N = 10 tumors, **** p < 0.0001, unpaired t-test. Mean ± SEM. ( (G, H) Heatmap of tumor cell apoptosis (G) and lymphocyte apoptosis (H) associated genes differentially expressed in PBS and AMD treated TNC-high (WT/shC) tumors, N = 2. (I, J) Abundance and quantification of F4/80+ macrophages in PBS and AMD treated tumors, (I) staining for the indicated molecules, (arrow, macrophages in the stroma; asterisk, macrophages in the tumor nest), (J) quantification of F4/80+ cells inside the tumor nest. N = 5 tumors per condition, 2 sections per tumor, 5 random fields, ** p = 0.0079, Mann-Whitney test. Mean ± SEM. Scale bar, 50 µm. \n",
      " \"TIL-matrix-release-and-reactivate\" strategy through targeting TNC and CXCR4TNC contributes to an immune suppressive TME in breast cancer by retaining CD8 TIL in the stroma.Figure 9TNC contributes to an immune suppressive TME in breast cancer by retaining CD8 TIL in the stroma. TNC induces CXCL12 in a TLR4 dependent manner. The TNC protein binds CXCL12 generating a substratum that tethers CD8 TIL in the stroma thus preventing infiltration of CD8 TIL into the tumor nests and subsequent tumor cell killing. This mechanism could be relevant in human breast cancer as patients with tumors of high TNC expression show high CD8 TIL in the (TNC-rich) stroma which correlates with shorter MFS. (Lower left) Upon removal of TNC anti-tumor immunity is activated and characterized by lower CXCL12 expression, higher CD8 TIL nest infiltration and, a reduced metastasis rate as previously shown (Sun et al., 2019). In human breast cancer low TNC and high CD8 TIL correlate with longest MFS. (Lower right) TNC impacts CD8 TIL through CXCL12/CXCR4. Inhibition of CXCR4 led to increased CD8 TIL infiltration, higher tumor cell apoptosis, and reduced tumor growth and subsequent lung metastasis. Also macrophages, loosing their M2 phenotype, infiltrate the tumor nests more numerously contributing to the observed activated CD8 TIL phenotype (e.g. less PD-L1, CTLA4). Targeting TNC and CXCL12 could direct a novel \"TIL-matrix-release-and-reactivate\" strategy to improve ICT in breast cancer.  \n",
      "ER Proliferation under UPR-Inducing Conditions.Figure 1ER Proliferation under UPR-Inducing Conditions(A) Determination of ER abundance in control and UPR-induced cells. Representative cells are shown. The UPR was induced in wild-type cells by addition of DTT. Ultrastructure of control cells and UPR-induced cells was analyzed using ImageJ. The lower images show traces of cortical ER (represented in magenta) and the nuclear envelope (NE, in blue). Vacuoles, nuclei, and mitochondria are indicated as V, N, and M, respectively.(B) Quantification of the ER proliferation during the UPR. UPR was induced and cells were collected for EM at the indicated time points. Length of the ER (as traced in [A]) was measured and divided by the area of the section. Data are plotted relative to time 0. Measurements for each time point correspond to the mean of 25 independent cell images.(C) Expression of HAC1i was induced by addition of 100 μM DOC for 3 h. ER was quantified as described above in (B). \n",
      " expression in human breast cancer and patient survival(A) Representative IHC images of CXCL12 intensity and staining pattern in the TMA.Figure 8(A) Representative IHC images of CXCL12 intensity and staining pattern in the TMA. N = 219 tumors. Upper panel, scale bar, 400 μm; lower panel, scale bar, 100 μm. Arrows point at punctate staining in the stroma. (B, C) Histological score of CXCL12 staining in TNC+ and TNC- tumors in combination with CD8+ (above median) and CD8- (below the median). * p = 0.0206, ** p = 0,0040, Unpaired t-test. The central band represents the median, the ends of the box the first and the third quartiles, and the whiskers reach from each quartile to the minimum or maximum. (D) Kaplan Meier survival analysis of breast cancer patients upon stratification into tumors with a CXCL12 Hscore below or above the median, p = 0.002. Mann-Whitney test. (E) Survival analysis of breast cancer patients in the GSE 19783-GPL6580 cohort upon stratification into tumors with high or low expression of TNC and CXCL12. Log-rank test. HR and p values are indicated, \n",
      " inhibition on lung metastasis(A) Schematic depiction of the AMD treatment protocol before sacrifice and analysis of the lungs.Figure 6(A) Schematic depiction of the AMD treatment protocol before sacrifice and analysis of the lungs. Intra peritoneal (ip) injection of 5 mg/kg/day AMD into NeuNT mice (daily for 2 weeks, week 27 - 29). (B) IF tissue staining identifying blood vessel invasions (BVI), surrounded by an EC monolayer (arrow) and expanded EC (asterisk), scale bar, 50µm. (C) Quantification of BVI with normal (EC monolayer) and aberrant (expanded EC) phenotype expressed as ratio. N = 5 mice (PBS), n = 3 sections, n = 15 BVI, n = 7 PM (parenchymal metastasis); N = 5 mice (AMD), n = 3 sections, n = 12 BVI, n = 2 PM, ****, p < 0.0001, Fisher's exact test. ( (D) Schematic depiction of the AMD treatment protocol before sacrifice and analysis of the lungs. Tail vein (iv) tumor cell engrafted mice (WT/shC, WT/shTNC) were ip injected (daily for one week, week 4-5) with AMD 7.5 mg/kg/day. Metastasis assessment in the 5 weeks iv orthotopic grafting model (D-G). (E) Ratio of mice with and without lung metastasis determined by H&E tissue staining. N = 8 mice (shC, PBS), N = 7 mice (shC, AMD), N = 7 mice (shTNC, PBS), N = 7 mice (shTNC, AMD). p < 0,0001, Chi square test. Fisher's exact test was used to compare the conditions pairwise *, p < 0,05, *** p < 0,001. p = 0,0109 (shC/PBS versus shTNC/PBS), p = 0.0001 (shC/AMD versus shTNC/AMD). (F, G) Ratio of BVI and PM in lungs from shC (F) and shTNC (G) engrafted mice. N = 8 mice (shC, PBS), n = 4 BVI, n = 13 PM; N = 7 mice (shC, AMD), n = 15 BVI, n = 21 PM; N = 7 mice (shTNC, PBS), n = 8 BVI, n = 9 PM; N = 7 mice (shTNC, AMD). n = 5 BVI, n = 1 PM, *** p < 0,001, **** p < 0,0001. p = 0.0009 (shC/PBS versus shC/AMD) and p = 0.0001 (shTNC/PBS versus shTNC/AMD), Fisher's exact test. (H) Schematic depiction of the AMD treatment protocol before sacrifice and analysis of the lungs. Mice with mammary gland (MG) engrafted tumor cells (WT/shC) were daily injected peritumorally (pt) for 5 weeks with 5 mg/kg/day (week 2-7). Metastasis assessment in the 7 weeks orthotopic grafting model (H-L). (I-L) qRT-PCR assessment of Neu (I), Cd8a (J), Ifnγ (K) and Gzmb (L) by qRT-PCR. PBS (N = 16) and AMD (N = 19) treated mice, *, p = 0.0364, Mann whitney test (I), **, p = 0.0086 unpaired t-test (J), **, p = 0.0055, unpaired t-test (K), ***, p = 0.0003 unpaired t-test (L), respectively. The central band represents the median, the ends of the box the first and the third quartiles, and the whiskers reach from each quartile to the minimum or maximum. \n",
      " TNC expression, abundance and localization of CD8 TIL in breast tumors and survival of human breast cancer patients(A-C) Adjacent human breast cancer tissue microarray (TMA) sections were stained with the indicated antibodies, scale bar, 50 μm, arrows point at CD8 TIL (A) and, CD8 TIL were counted per field (B), or in the stroma and nest, respectively (C), N = 103 TNC+ tumors, N = 116 TNC- tumors.Figure 7(A-C) Adjacent human breast cancer tissue microarray (TMA) sections were stained with the indicated antibodies, scale bar, 50 μm, arrows point at CD8 TIL (A) and, CD8 TIL were counted per field (B), or in the stroma and nest, respectively (C), N = 103 TNC+ tumors, N = 116 TNC- tumors.. (B) ** p = 0.0055, Fisher's exact test. The central band represents the median, the ends of the box the first and the third quartiles, and the whiskers reach from each quartile to the minimum or maximum. (C) ** p = 0,0055, Mann-Whitney test. (D-G) Kaplan Meier analysis to address correlation of TNC expression (D), total numbers of CD8 TIL (E), combined high (above median) CD8 TIL and high or low (above/below median) TNC (F) and stromal CD8 cells (G) with metastasis-free survival (MFS) of breast cancer patients. Stromal CD8- or CD8+ is defined as below or above median stromal CD8 TIL per area. Log-rank test. (H, I) Correlation of TNC and CD8a levels with overall survival (OS) using survival analysis on breast cancer patient cohort GSE 19783-GPL6580. Hazard ratio (HR) and p values are indicated in Appendix Fig. S5N. Log-rank test. \n",
      "The ER Morphologically Changes during the UPR.Figure 2The ER Morphologically Changes during the UPR(A) Control cells and UPR-induced cells were used to analyze and follow the ER within a single cell using EM. Boxes indicate the areas magnified in (B). Cells shown here correspond to the full section of the images labeled \"+140 nm\" in (B).(B) Serial section of control and UPR-induced cells. Sections are separated by 70 nm on the z-axis. ER is represented in magenta and NE in blue.(C) Electro micrographs from control and UPR-induced cells showing that the distance between ER membranes increases during the UPR. For a better preservation of the ultrastructure, samples for this experiment were prepared using high-pressure freezing/freeze substitution techniques (see Material and Methods). \n",
      "Fluorescence Visualization of an ER Marker after UPR Induction.Figure 4Fluorescence Visualization of an ER Marker after UPR Induction(A) Cells treated with the UPR-inducing drug DTT (+DTT) or with no drug were visualized using a fusion protein between the translocon component Sec61 and the red-fluorescent protein \"cherry.\" Top panels show untreated cells, and bottom panels show representative UPR-induced cells.(B) Representative images showing UPR-induced cells that contain ERAs (indicated by arrows). \n",
      "Characterization of ER-Containing Autophagosomes (ERAs) during the UPR.Figure 3Characterization of ER-Containing Autophagosomes (ERAs) during the UPR(A) Images of representative DTT-treated wild-type cells that contain ERAs. Nuclei and cytoplasm are indicated as N and C, respectively.(B) Enlargement of representative images of ERAs from different cells. The bottom right image is likely to show a section through a cup-shaped ERA. Note that there are no connections between the stacked cisternae and the envelope.(C) High magnification of the ERA double membrane envelope.(D) Some ERAs are found attached to or are in close proximity to ER tubules/sheets (indicated by the arrow). Note that the section in (A) includes two such junctions.(E) High-pressure freezing/freeze substitution image of an ERA linked to an ER tubule/sheet. The osmium/lead staining used in this technique visualizes ribosomes and demonstrates that the outer ERA envelope membrane, but not the stacked internal cisternae, are tightly studded with ribosomes, indicating that they originate from ER membranes.(F) High-pressure freezing/freeze substitution image of an ER-ERA junction using an improved protocol to visualize membranes.(G) Using the same technique as in (F), we visualized the internal membrane content of an ERA. Note that both portions of the internal membranes and of the sequestering double membrane envelope contain bound ribosomes, and hence are likely derived from the ER. \n",
      "Immunogold Labeling of ERAs with an Antibody Directed against an ER Membrane Marker.Figure 5Immunogold Labeling of ERAs with an Antibody Directed against an ER Membrane Marker(A) Representative section of a cell immunolabeled against a myc-tagged Sec63, an integral ER membrane protein. As a primary antibody, we used a rabbit polyclonal anti-myc and, as a secondary, we used 15-nm gold particles-conjugated anti-rabbit antibody. Nucleus, nuclear envelope, ER, and ERA are indicated as N, NE, ER, and ERA, respectively.(B) High magnification of an electron micrograph of a section of ER. Quantification showed that there are 5 ± 2 gold particles per linear micrometer of ER.(C) High magnification of ERAs. To predict how many gold particles one should expect in a particular ERA, we first calculated and averaged the amount of ER (expressed as length in linear micrometers) present in an ERA (similar to the ones shown in Figure 3B), and normalized the value for its area. These calculations determined that there are 20.8 ± 3.3 μm of ER per μm2 inside the ERAs. These values allowed us to predict how many gold particles would be expected over a section of an ERA if it were packed with ER membranes. Two representative ERAs are shown. The ERA shown in the middle picture should hold 2.4 μm of ER inside and, therefore, should have 12 gold particles. We counted 12 gold particles. The ERA on the right could contain 2.7 μm of ER and should contain14 gold particles; we counted 16 gold particles. The image on the right shows a representative view of a nucleoplasmic region.(D) Quantification of gold-labeling density per area. To assess the signal-to-noise ratio of our immunogold-labeling procedure, we assessed background labeling by counting the number of gold particles over an areas of nucleoplasm (N) and over ERAs, and normalized the counts to the respective areas. \n",
      "UPR-Induction of the Autophagy Marker GFP-Atg8.Figure 6(A) Wild-type cells transformed with a plasmid containing GFP-Atg8 were grown for 4 h in synthetic media with no drug, with UPR-inducing conditions (+DTT and +TM), or under nitrogen starvation conditions (N starv), and then harvested for protein preparation. Protein extracts were analyzed by Western blotting using antibodies against GFP (top panel) or Hac1 (bottom panel). Total protein concentration was measured by BCA protein assay. Same concentration of protein was loaded in each lane, and transfer efficiency was checked by Ponceau staining. The identities of the different bands are indicated. (B) Wild-type cells expressing GFP-Atg8 grown under the conditions described above were visualized by fluorescence microscopy. (C) GFP-Atg8 was detected in extracts from untreated hac1Δ cells or cells expressing HAC1i (+DOC) by Western blotting using antibodies against GFP. (D) Western blot using antibodies against GFP of extracts from hac1Δ, ire1Δ, or vps4Δ pep4Δ cells expressing GFP-Atg8. Mutant cells were grown under regular conditions, UPR-inducing conditions (+DTT), or nitrogen starvation conditions (N starv). \n",
      "Localization of GFP-Atg8 during UPR Induction.Figure 7Localization of GFP-Atg8 during UPR InductionSome of the DTT-treated cells shown in Figure 4B expressing GFP-Atg8 and Sec61-cherry (as an ER marker) were visualized using fluorescence microscopy. GFP-Atg8 localizes in close proximity to the ERAs detected by the ER marker. \n",
      "Atg8 and Other ATG Genes Are Necessary during UPR Induction.Figure 8Atg8 and Other ATG Genes Are Necessary during UPR InductionSerial dilutions for wild-type, hac1Δ, atg1Δ, atg8Δ, atg9Δ, atg16Δ, and atg20Δ deletion cells and vps4Δ pep4Δ double deletion cells were grown on rich-media plates with no drug (YPD) or with different concentrations of tunicamycin (TM; 0.2 or 1.0 μg/ml). atg19Δ gave an identical result to the other autophagy genes shown here (unpublished data). \n",
      ".Figure 1A Localisation of transiently expressed GFP-Rab35 in IMCD3 cells after 48h serum starvation and staining for acetylated tubulin (acetyl. tub.) and DNA. Insets show higher magnification images of the cilia region. Scale bars, 10 µm. B Localisation of transiently expressed GFP-RAB35 in hTERT-RPE1 cells after 24 h serum starvation and staining for polyglutamylated tubulin (polyglu. tub.) and DNA. Insets show higher magnification images of the cilia region. Scale bars, 10 µm. C Super resolution (FV-OSR) imaging of hTERT-RPE1 stably expressing GFP-RAB35 after 24 h serum starvation and staining for GFP, ARL13B (cilia membrane), IFT88 (axoneme), acetylated tubulin (acetyl. tub.; axoneme), and γ-tubulin (γ-tub; centrosome). Top panels show representative images and bottom panel shows line profile plots of fluorescence intensity (arbitrary units; a. u.) in the distal (dotted line) and proximal (dashed line) cilia regions. Data are mean ± S.E.M. (n = 5 cilia), with the solid line in the line profile plots indicating mean and the dotted lines indicating S.E.M. values. Scale bars, 1 µm. D Representative FV-OSR images of 24 h serum-starved hTERT-RPE1 stably expressing GFP-RAB35 and stained for GFP, ARL13B and acetylated tubulin. Right panels show a cilium with proximal enrichment of GFP-RAB35 and left panels a cilium with even GFP-RAB35 localisation along the full length. Scale bars; 1 µm. E Representative FV-OSR images of 24 h serum-starved hTERT-RPE1 stably expressing GFP-RAB35 and stained for GFP, EHD1 and ARL13B. Line profile plots of fluorescence intensity in the proximal cilia region (dashed line) are shown to the right. Data are mean ± S.E.M. (n = 6 cilia), with the solid line in the line profile plots indicating mean and the dotted lines indicating S.E.M. values. Scale bars; 1 µm. \n",
      ".Figure 3A Representative images of hTERT-RPE1 cells transiently expressing wild type (WT), GDP-bound (S22N) or GTP-bound (Q67L) GFP-tagged RAB35. Cells were serum-starved for 24 h and stained for polyglutamylated tubulin (polyglu. tub.) and DNA. Higher magnification images of the cilia region shown in smaller panels. Scale bars; 10 µm. Quantification of GFP-RAB35 ciliary localisation (B) in hTERT-RPE1 cells transiently expressing indicated GFP-RAB35 constructs or GFP. Data are mean ± S.E.M. of 3 independent experiments. Statistical significance according to ANOVA followed by Bonferroni post-hoc test ( ** P < 0.01; P-values: (B) P = 0.0015 Quantification of ciliation (C) in hTERT-RPE1 cells transiently expressing indicated GFP-RAB35 constructs or GFP. Data are mean ± S.E.M. of 3 independent experiments. Statistical significance according to ANOVA followed by Bonferroni post-hoc test ( ** P < 0.01; P-values: (C) P = 0.0062). Quantification of ciliary length in hTERT-RPE1 cells transiently expressing indicated GFP-RAB35 constructs or GFP. Cilia length quantification in (D) is shown as box-and-whisker plots. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n ≥ 50 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P < 0.05, ** P < 0.01; P-values: GFP vs. GFP-RAB35-S22N P = 0.0015, GFP vs. GFP-RAB35-Q67L P = 0.0158) Analysis of cilia length in hTERT-RPE1 cells depleted of GEF DENND1B) and GAP (TBC1D10A regulators of RAB35. Cells transfected with indicated siRNAs were serum-starved for 48 h and stained for acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (E). Regions within white boxes shown at higher magnifications to the right. Scale bars; 10 µm. Analysis of cilia length in hTERT-RPE1 cells depleted of GEF (DENND1A, DENND1B) and GAP (TBC1D10A, TBC1D10B) regulators of RAB35. Cells transfected with indicated siRNAs were serum-starved for 48 h and stained for acetylated tubulin (acetyl. tub.) and DNA. Cilia length quantification in (F) is shown as box-and-whisker plots. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n ≥ 50 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (** P < 0.01, *** P < 0.001; P-values: Neg vs. DENND1B P = 0.0033, Neg vs. TBC1D10A P = 0.0008) G Quantification of ciliary localisation of GFP-RAB35 in hTERT-RPE1 cells stably expressing GFP-RAB35 treated Data are mean ± S.E.M. of 5 independent experiments. Statistical significance according to ANOVA followed by Bonferroni post-hoc test (* P < 0.05, *** P < 0.001; P-values: Neg vs. DENND1B P < 0.0001, Neg vs. TBC1D10A P = 0.0322). H-J Localisation of transiently expressed GFP-DENND1B (H) or GFP-TBC1D10A (I) in hTERT-RPE1 cells after 24 h serum starvation and staining with indicated antibodies. Higher magnification images of the cilia region are shown in smaller panels. Scale bars; 5 µm. Quantification of localisation to ciliary base or along axoneme in (J). Data are mean ± S.E.M. of 3 independent experiments (n ≥ 30 cilia per experimental condition). \n",
      ".Figure 2A Immunoblot analysis of RAB35 depletion in hTERT-RPE1 cells transfected with two independent siRNAs targeting RAB35 (RAB35-1, RAB35-2) or a non-targeting siRNA control (Neg). 24 h after transfection, cells were serum-starved for further 48 h. β-tubulin served as a loading control. B Quantification of ciliation of hTERT-RPE1 cells treated and using acetylated tubulin staining as a cilia marker. Data are mean ± S.E.M. of 4 independent experiments (n ≥ 50 cilia per experimental condition). C, D hTERT-RPE1 cells and hTERT-RPE1 cells stably expressing siRNA-resistant GFP-RAB35 were treated and stained for GFP, acetylated tubulin (acetyl. tub.) and DNA. Representative images in (C) of cells treated with Neg or RAB35 siRNA. Regions within white boxes shown at higher magnifications to the right. Scale bars, 10 µm. Cilia length quantifications in (D) are shown as box-and-whisker plots. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment of three is shown (n ≥ 50 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P < 0.05, ** P < 0.01, *** P < 0.001, n.s.: non-significant; P-values: hTERT-RPE1-Neg vs. hTERT-RPE1-RAB35-1 P < 0.0001, hTERT-RPE1-Neg vs. hTERT-RPE1-RAB35-1 P < 0.0001, hTERT-RPE1-RAB35-1 vs. GFP-RAB35-RAB35-1 P = 0.0249, hTERT-RPE1-RAB35-2 vs. GFP-RAB35-RAB35-2 P = 0.0056) E Immunoblot analysis of Rab35 depletion in IMCD3 cells transfected with pool of siRNAs targeting mouse Rab35 or a non-targeting siRNA control (Neg). 24 h after transfection, cells were serum-starved for further 48 h. GAPDH served as a loading control. F Quantification of ciliation of IMCD3 cells treated and using acetylated tubulin staining as a cilia marker. Data are mean ± S.E.M. of 3 independent experiments (n ≥ 50 cilia per experimental condition). G-H Immunofluorescence of IMCD3 representative images treated cells are stained for acetylated tubulin (acetyl. tub.) and DNA. Cilia length quantification in (H) is shown as box-and-whisker plots and is the result of 3 independent experiments (n ≥ 50 cilia per experimental condition). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. Statistical significance according to unpaired t-test with Mann-Whitney test (*** P = 0.0002) I Immunoblot analysis of NIH3T3 wild type (WT) and Rab35 knockout (KO) cell lines. β-tubulin served as a loading control. NIH3T3 WT and Rab35 KO cell lines were serum-starved for 24h and stained for acetylated tubulin (acetyl. tub.) and DNA. Quantification of ciliation in (J). Data are mean ± S.E.M. of 3 independent experiments (n ≥ 100 cilia per experimental condition). NIH3T3 WT and Rab35 KO cell lines were serum-starved for 24h and stained for acetylated tubulin (acetyl. tub.) and DNA. Representative images in (K). Regions within white boxes shown at higher magnifications at the bottom. Scale bars, 10 µm. Cilia length quantifications in (L) are shown as box-and-whisker plots. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment of three is shown (n ≥ 100 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (**** P < 0.0001). \n",
      ".Figure 4A Immunostaining of representative images of KV embryos at 8 somite stage in which mRNA encoding for mCherry or mCherry-Rab35 WT was injected. Staining of acetylated tubulin was used as cilia marker. Inset shows a higher magnification image of the cilia region, displaying colocalization of mCherry-Rab35 and acetylated tubulin. Scale bar, 10 µm. B,C Immunoblot analysis with anti-Rab35 and anti-GAPDH antibodies of 40 µg of total lysates of zebrafish embryos injected with Rab35 morpholino (MO) at 8 somite stage, compared with non-injected embryos. The plot in (C) shows Rab35 protein level in non-injected and Rab35 morphants, normalized to GAPDH protein level. Data are mean ± S.E.M. of 3 independent experiments (n > 50 embryos per condition). D Organ situs was determined by observing the heart position at 30 hours post-fertilisation (hpf), and the liver position at 53 hpf, by in situ hybridisation with a probe for foxa3, in controls (Mismatch MO and non-injected [WT]) and Rab35 MO. Situs solitus means left heart and liver; situs inversus means right heart and liver; and heterotaxia means any other possible combination of the heart and liver position. E Whole-mount in situ hybridization with the dand5 probe at 8 somite stage (13 hpf) in wild type non-injected embryos and mismatch morphants, both presenting right-sided distribution of dand5, and Rab35 morphants presenting bilateral localization of the dand5 probe. Scale bar, 50 µm. Graph shows quantification of embryos (%) with bilateral or right-sided dand5 localisation from both conditions. Statistical significance according to Fisher Exact test (** P = 0.0002) F-I Immunostaining of representative images of KV embryos at 8 somite stage in which: (F) Mismatch or Rab35 MO was injected, as well as a co-injection of Rab35 MO with mRNA encoding mCherry-Rab35 WT [Rescue]; (H) mRNA encoding mCherry or mCherry-tagged Rab35 WT, Rab35-Q67L or Rab35-S22N were injected. Acetylated tubulin is shown in green as cilia marker. Quantification of KV cilia length in (F) and (H) is shown respectively in (G) and (I) as box-and-whisker plots of 3 independent experiments. N ≥ 8 embryos and n > 300 cilia per experimental condition. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (** P < 0.01, *** P < 0.001, ns. not significant; Mismatch vs Rab35 MO, P < 0.001; Cherry vs Rab35-SN, P = 0.0063; ). Scale bars, 10 µm. \n",
      ": CRP-mediated transcriptional control of vcdRPA.Figure 2A. Northern blot analysis of the expression of VcdRP and Spot42 sRNA monitored at low cell density (OD600 of 0.1) and high cell density (OD600 of 2.0), in the absence (-) or presence (+) of cAMP (5mM f.c.) supplemented externally in LB medium. The solid triangle indicates the band corresponding to the full-length primary transcript. Probing with 5S rRNA confirmed equal loading. B. Relative fluorescence intensities of V. cholerae wild-type or mutants lacking crp or cyaA harboring a transcriptional reporter plasmid of vcdRP fused to mKate2. Cells were grown in LB to OD600 of 1.0 and fluorophore production was measured. The fluorescence of WT was set to 1. C. Electrophoretic mobility shift assay (EMSA) showing that CRP protein binds to the vcdRP promoter sequence. Migration of the [32P] end-labeled vcdRP promoter fragment in the absence and presence of increasing concentrations of purified CRP protein (indicated by the black triangle above the gel) was determined by native polyacrylamide gel electrophoresis and autoradiography. The open triangle indicates free DNA whereas the filled triangle indicates VcdRP in complex with CRP. Final concentrations of CRP used in each reaction from left to right (lanes 1-8) were: 0, 0.35, 0.7, 1.4, 2.1, 2.8, 3.5, and 4.2 µM. Radiolabelled DNA and CRP were incubated at 37°C for 20 minutes in buffer containing 0.2 mM cAMP, before being loaded on a 7.5 % non-denaturing polyacrylamide gel. \n",
      ": The VcdRP dual-function RNA regulator inhibits CTX productionA.Figure 1A. Violin plot of V. cholerae ΔhapR cells expressing the indicated sRNAs (x-axis) plotted as relative fold-repression (y-axis) of secreted CTX protein fractions in comparison with a control strain (pCMW) grown under AKI conditions. Indicated in grey is the only known repressor of CTX to date (VqmR) and indicated in pink is the yet uncharacterized sRNA, Vcr082 (VcdRP). B. Gene synteny analysis of the genomic loci encoding vcdRP (formerly known as vcr082) in various Vibrio strains. Homologous genes are depicted using the same colors. C. Alignment of vcdRP sequences, including the promoter regions from various Vibrio species: V. cholerae (Vch), V. furnissi (Vfu), V. vulnificus (Vvu), V. harveyi (Vha), V. parahaemolyticus (Vpa), V. anguillarum (Van) and V. splendidus (Vsp). The -35 box, -10 box, transcriptional start site (TSS; arrow), the start and stop positions of VcdP (blue box), the highly conserved base-pairing region (four consecutive cytosines) and rho-independent terminator (brackets) are indicated. D. Northern blot analysis of V. cholerae wild-type strain examined for the expression of VcdRP monitored over bacterial growth in the indicated media. The solid triangle represents the band corresponding to the full length primary vcdRP transcript, whereas the open triangles correspond to the different processed isoforms. Probing with 5S rRNA confirmed equal loading. E. Sequence conservation of the 29 amino acid small protein, VcdP, in different Vibrio species. F. Western blot analysis of the production of chromosomally-encoded VcdP::SPA, corresponding to the same growth conditions as the Northern blot in (D). The SPA epitope contains the 3×FLAG and the calmodulin binding peptide sequences separated by a TEV protease cleavage site (Zeghouf et al, 2004). Therefore, anti-Flag antibody was used to examine the levels of VcdP. RNAP served as a loading control. \n",
      ".Figure 9A Model of how activated (GTP-bound) Rab35 at the ciliary and/or periciliary (pocket) membrane limits Arl13b ciliary levels. In one scenario, Rab35 inhibits Arl13b entry into cilia, possibly through an IFT pathway. In a second scenario, Rab35 promotes Arl13b exit/retrieval from ciliary (and periciliary) membranes via: (i) interaction with retrograde IFT, (ii) modulation of the transition zone membrane diffusion barriers that prevent Arl13b from exiting the cilum, and/or (iii) endocytic processes at or near the ciliary pocket that target Arl13b for degradation or recycling to non-ciliary plasma membrane destinations. B Ciliary membrane and ciliary length phenotypes following Rab35 disruption. In wild type cells, Arl13b, INPP5E and Smoothened (SMO; under conditions of Shh signal activation) all localise to the ciliary membrane. Arl13b promotes the ciliary localisation of INPP5E whilst limiting (or altering) the ciliary distribution of SMO ]. PI(4,5) P2 (blue line) is enriched within the proximal portion of the ciliary membrane.In Rab35-depleted cells, ciliary Arl13b levels are elevated, causing a concomitant increase and decrease in the ciliary levels of INPP5E and SMO, respectively. Rab35 disruption also leads to a reduction in the length of the proximal ciliary PI(4,5) P2 signal and a reduction in the frequency of PI(4,5) P2-positive cilia (thin blue line). Misregulation of ciliary membrane protein composition in Rab35-disrupted cells leads to a decrease in cilium length. \n",
      ": VcdRP regulates central metabolism genesA.Figure 3A. A schematic representation of the three VcdRP and VcdR/P variants used for the transcriptome analysis. Left to right: first, the native VcdRP variant which harbors both the small protein as well as the base-pairing region. Next, a STOP codon was introduced in the 3rd codon of vcdP ORF as a readout for VcdR devoid of VcdP. Finally, the sequence of the VcdP codons was scrambled such that the correct small protein is translated but the nucleotide sequence of the scrambled vcdP variant is significantly altered when compared to native vcdRP. This variant also carries an artificial 5'UTR and a terminator. The arrows indicate the TSS. B. Western blot analysis (top) of CTX levels detected in secreted protein fractions of V. cholerae ΔhapR cells grown under AKI conditions and carrying the indicated plasmids. Coomassie-stained SDS gel (bottom) confirmed equal loading of the protein fractions. The normalized band intensities are indicated for each expression plasmid relative to pCtrl, which was set to 100%. C. A Venn diagram of the differentially expressed genes (DEG) among VcdRP (red), VcdR (yellow) and VcdP (blue). RNA samples were harvested from V. cholerae ΔvcdRP strains carrying pBAD-based plasmids depicted in (A), including an empty vector control. Genes with a total count cutoff >10 in all samples, with an absolute fold-change ≥ 2.0 and a FDR adjusted p-value ≤ 0.05 were considered to be differentially expressed. D. Gene enrichment analysis of the DEG shown in (C) using gene ontology analysis. \n",
      ".Figure 6A HEK293T cells were transiently co-transfected with ARL13B-FLAG and GFP or GFP-RAB35. Cells were cultured either in full growth medium (+serum) or serum-starved for the final 8 h before harvesting. Immunoprecipitations (IP) were performed using anti-GFP beads. Cell lysates (2.5% of input) and immunoprecipitates were loaded on the same gel, subjected to immunoblot (I) analysis with indicated antibodies and detected with the same exposure. B Schematic representation of the human ARL13B structure depicting the guanine nucleotide-binding (G), coiled coil (CC) and proline-rich (PR) domains as well as the point mutations affecting N-terminal palmitoylation or C-terminal ciliary targeting RVEP motif. C HEK293T cells were transiently co-transfected with mCherry-RAB35 together with GFP or indicated ARL13B-GFP wild type, mutant or truncation constructs. Cells were serum-starved for the final 8 h before harvesting. IPs were performed using anti-GFP beads and interacting proteins detected by immunoblotting (IB). D HEK293T cells were transiently transfected with ARL13B-FLAG and serum-starved for the final 8 h before harvesting. HEK293T cell lysates expressing ARL13B-FLAG were subjected to pull-down with GFP-RAB35 bound to anti-GFP beads and preloaded with either no nucleotide, GTPγS or GDP. Bound proteins and cell lysates (1% of input) were analysed by immunoblotting. The graph shows the ratio of precipitated ARL13B and RAB35, normalized to the no nucleotide control. Data are mean ± S.E.M. of 3 independent experiments. Statistical significance according ANOVA followed by Tukey post-hoc test (* P =0.0333). E HEK293T cells were transiently transfected with mCherry-RAB35 and serum-starved for the final 8 h before harvesting. HEK293T cell lysates expressing ARL13B-FLAG were subjected to pull-down with ARL13B-GFP bound to anti-GFP beads and preloaded with either no nucleotide, GTPγS or GDP. Bound proteins and cell lysates (1% of input) were analysed by immunoblotting. The graph shows the ratio of precipitated ARL13B and RAB35, normalized to the no nucleotide control. Data are mean ± S.E.M. of 3 independent experiments. Statistical significance according ANOVA followed by Tukey post-hoc test (* P =0.0466) \n",
      ".Figure 7A, B hTERT-RPE1 cells transfected with non-targeting siRNA control (Neg) or RAB35 siRNA, were serum-starved for 48 h and stained for INPP5E, ARL13B, acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (A). Regions within white boxes shown at higher magnifications to the right. Scale bars, 10 µm. (B) Box-and-whisker plots show quantification of ciliary INPP5E intensity in arbitrary units (a. u.). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 50 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (*** P < 0.001; P-values: Neg vs. RAB35-1 P < 0.0001, Neg vs. RAB35-2 P = 0.0002). C For co-depletion experiments, hTERT-RPE1 cells were first transfected with Neg or RAB35 siRNA. After 24h, a second transfection with Neg or ARL13B siRNA was performed and cells were serum starved for 48 h and stained as in (A). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 40 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (** P = 0.0017). D, E NIH3T3 wild type (WT) and Rab35 knockout (KO) cell lines were serum-starved for 24h and stained for Innpp5e, acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (D). Higher magnifications of regions within white boxes shown at the bottom. Scale bars; 10 µm. (E) Box-and-whisker plots show quantification of ciliary Inpp5e intensity in arbitrary units (a. u.). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 25 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P < 0.05, *** P < 0.001; NIH3T3 WT vs. Rab35 KO#1 P < 0.0001, NIH3T3 WT vs. Rab35 KO#2 P = 0.0345). F, G Localisation of the PI(4,5) P2 sensor PH-PLCδ1-GFP to cilia in live NIH3T3 WT and Rab35 KO cells. Cilia are marked with HTR6-RFP. Representative images are shown in (F). Higher magnification images of the cilia region shown to the right. Scale bars; 10 µm. Quantification of localisation in (G). Data are mean ± S.E.M. of 3 independent experiments (n > 100 cilia per experimental condition). Statistical significance according to two-way ANOVA (* P = 0.0242). \n",
      ".Figure 5A, B hTERT-RPE1 cells and hTERT-RPE1 cells stably expressing siRNA-resistant GFP-RAB35 transfected with non-targeting siRNA control (Neg) or Rab35 siRNA, were serum-starved for 48 h and stained for ARL13B, GFP, polyglutamylated tubulin (polyglu. tub.) and DNA. Representative images are shown in (A). Regions within white boxes shown at higher magnifications to the right. Scale bars; 10 µm. (B) Box-and-whisker plots show quantification of ciliary ARL13B intensity in arbitrary units (a. u.). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 25 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (** P < 0.01, *** P < 0.001, n.s.: non-significant; P-values: hTERT-RPE1-Neg vs. hTERT-RPE1-RAB35-1 P < 0.0001, hTERT-RPE1-Neg vs. hTERT-RPE1-RAB35-1 P = 0.0001, hTERT-RPE1-RAB35-1 vs. GFP-RAB35-RAB35-1 P = 0.0014, hTERT-RPE1-RAB35-2 vs. GFP-RAB35-RAB35-2 P = 0.0081). C Immunoblot analysis of total ARL13B protein levels in control and RAB35-depleted cells. β-tubulin served as a loading control. The graph shows ARL13B protein levels normalized (norm.) to β-tubulin. Data are mean ± S.E.M. of 4 independent experiments. D,E NIH3T3 wild type (WT) and Rab35 knockout (KO) cell lines were serum-starved for 24h and stained for Arl13b, acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (D). Higher magnifications of regions within white boxes shown at the bottom. Scale bars; 10 µm. (E) Box-and-whisker plots show quantification of ciliary Arl13b intensity in arbitrary units (a. u.). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 25 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P < 0.05; P-values: NIH3T3 WT vs. Rab35 KO#1 P = 0.0118, NIH3T3 WT vs. Rab35 KO#2 P = 0.0111). F Immunoblot analysis of total ARL13B protein levels in serum-starved NIH3T3 WT and Rab35 KO cell lines. β-tubulin served as a loading control. Quantification of ciliary ARL13B intensity in hTERT-RPE1 cells depleted of GEF (G) regulators of RAB35. Cells transfected with indicated siRNAs were serum-starved for 48 h and stained for ARL13B, acetylated tubulin and DNA. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 30 cilia per experimental condition). Statistical significance according ANOVA followed by Bonferroni post-hoc test (*** P < 0.001; P-values: (G) P = 0.0005 Quantification of ciliary ARL13B intensity in hTERT-RPE1 cells depleted of GAP (H) regulators of RAB35. Cells transfected with indicated siRNAs were serum-starved for 48 h and stained for ARL13B, acetylated tubulin and DNA. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 30 cilia per experimental condition). Statistical significance according ANOVA followed by Bonferroni post-hoc test (*** P < 0.001; P-values: (H) P < 0.0001). I, J Ciliary ARL13B intensity quantification of hTERT-RPE1 cells transiently overexpressing GFP or GFP-RAB35. Cells were serum-starved for 24 h and stained for ARL13B, acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (I). Higher magnification images of the cilia region are shown at the bottom. Scale bars; 5 µm. Cells with (RAB35+ cilia) and without (RAB35- cilia) GFP-RAB35 ciliary localisation are compared in (J). Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n > 25 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (*** P =0.0004). \n",
      ".Figure 8hTERT-RPE1 cells transfected with indicated siRNAs were serum-starved for 48 h and treated with SMO agonist (SAG) or vehicle control (DMSO) for the last 24 h. Cells were stained for SMO, acetylated tubulin and DNA. Representative images are shown in (A). Regions within white boxes shown at higher magnifications to the right. Scale bars, 10 µm. Graph in (B) shows the percentages of SMO-positive (SMO+) cilia. Data are mean ± S.E.M. of 5 independent experiments. Statistical significance according to ANOVA followed by Bonferroni post-hoc test (*** P < 0.001; P-values: Neg+SAG vs. RAB35-1+SAG P < 0.0001, Neg+SAG vs. RAB35-2+SAG P = 0.0033) hTERT-RPE1 cells transfected with indicated siRNAs were serum-starved for 48 h and treated with SMO agonist (SAG) or vehicle control (DMSO) for the last 24 h. (C) Box-and-whisker plots show quantification of the average SMO intensity in the ciliary area marked by acetylated tubulin staining. All cilia: cilia identified with acetylated tubulin staining. SMO+ cilia: cilia with discernible SMO localisation. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n ≥ 30 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P < 0.05, *** P < 0.001; P-values: Neg+SAG vs. RAB35-1+SAG (all cilia) P < 0.0001, Neg+SAG vs. RAB35-2+SAG (all cilia) P = 0.0024, Neg+SAG vs. RAB35-1+SAG (SMO+ cilia) P < 0.0007, Neg+SAG vs. RAB35-2+SAG (SMO+ cilia) P = 0.0205) For co-depletion experiments, hTERT-RPE1 cells were first transfected with Neg or RAB35 siRNA. After 24h, a second transfection with Neg or ARL13B siRNA was performed and cells were treated as in (A). Representative images are shown in (D). Regions within white boxes shown at higher magnifications as insets. Scale bars, 10 µm. Graph in (E) shows the percentages of SMO-positive cilia. Data are mean ± S.E.M. of 5 independent experiments. Statistical significance according to ANOVA followed by Bonferroni post-hoc test (* P = 0.0151, n.s.; non-significant). For co-depletion experiments, hTERT-RPE1 cells were first transfected with Neg or RAB35 siRNA. After 24h, a second transfection with Neg or ARL13B siRNA was performed and cells were treated (F) show quantification of ciliary SMO intensity. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n ≥ 30 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (* P = 0.0142, n.s.; non-significant). Ciliary SMO intensity quantification in hTERT-RPE1 cells transiently expressing GFP, wild type ARL13B-GFP (WT) or the ciliary-targeting defective mutant ARL13B-V358A. Cells were serum-starved and treated with SAG for 24 h and stained for SMO, acetylated tubulin (acetyl. tub.) and DNA. Representative images are shown in (G). Ciliary SMO intensity quantification in hTERT-RPE1 cells transiently expressing GFP, wild type ARL13B-GFP (WT) or the ciliary-targeting defective mutant ARL13B-V358A. Cells were serum-starved and treated with SAG for 24 h and stained for SMO Box-and-whisker plots in (H) show quantification of ciliary SMO intensity. Horizontal lines show 25, 50 and 75th percentiles; whiskers extend to minimum and maximum values. One representative experiment out of three is shown (n ≥ 25 cilia per experimental condition). Statistical significance according to Kruskal-Wallis followed by Dunn's post-hoc test (** P = 0.0046). \n",
      ": Glycolytic and citric acid cycle metabolites are regulated by VcdR/PMetabolite abundance of glycolytic and citric acid cycle intermediates in exponential (OD600 of 0.Figure 6Metabolite abundance of glycolytic and citric acid cycle intermediates in exponential (OD600 of 0.5) and stationary phase (OD600 of 3.0) of growth were measured for V. cholerae wild-type and ΔvcdRP, each harboring an empty vector control (pCtrl) or ΔvcdRP with vcdRP expression plasmids (pVcdRP and pVcdP). The y-axis represents the peak area of each metabolite determined by mass spectrometry. The main operative pathways including the fates of certain amino acids are color coded (yellow for glycolysis and blue for citric acid cycle). \n",
      ": VcdR is a direct inhibitor of the synthesis of PTS sugar transport proteinsA-D.Figure 4A-D. Relative fluorescence intensities (y-axis) of E. coli strains harboring the gene-specific translational reporters for ptsG (A), treB (B), nagE (C) and ptsHI (D), fused to sfGFP, combined with either an empty vector control (pCtrl) or vcdRP expression plasmids (pVcdRP, pVcdR, pVcdP and pVcdRP Δ4C, x-axis). Cells were grown in LB to OD600 of 1.0 and fluorophore production was measured. The fluorescence of pCtrl (for each reporter fusion) was set to 1. E-H. Predicted base-pairing sequences for ptsG (E), treB (F), nagE (G) and ptsHI (H) to VcdRP. The numbers indicate the distances relative to the TSS for VcdRP, and the start codons of the respective mRNA sequences. Underlined is the conserved stretch of four consecutive cytosines that corresponds to the seed region indicated in (1C). The M2* and M2 mutations refer to the site-directed mutagenesis of two guanines/cytosines in positions 2 and 3 of the seed region to two cytosines/guanines in the corresponding mRNA/VcdRP, respectively. I-L. Relative fluorescence intensities (y-axis) of E. coli strains harboring the gene-specific translational reporters as in (A-D) or the corresponding M2* variant for each gene, combined with an empty control plasmid or vcdRP expression plasmids (pVcdRP and pVcdRP M2) measured using plate reader (I-K) or by Western blot analysis (L). The fluorescence of pCtrl (for each reporter fusion) was set to 1. \n",
      ": Model depicting the dual function of VcdRP in balancing carbon metabolismVcdRP is a dual-function regulator that represses CTX production in V.Figure 7VcdRP is a dual-function regulator that represses CTX production in V. cholerae. Additionally, CRP and cAMP act on the regulator at the transcriptional level and suppress the production of VcdRP. The sRNA element (VcdR) acts via a conserved stretch of four consecutive cytosines to base pair with the translation initiation sites of PTS-specific and non-specific sugars, resulting in the inhibition of carbon uptake. The peptide VcdP binds to and increases the activity of citrate synthase enzyme, thereby modulating the levels of metabolites within the cycle. Thus, VcdR and VcdP together synchronize the carbon uptake and its subsequent utilization, thereby balancing carbon metabolism. \n",
      " Three-dimensional analysis of interaction between the peroxisome and the ciliary pocket.Figure 4Maximum projection of z-stack sections of hTERT-RPE1 cells expressing MCHR1-GFP (blue), DsRed2-PACT (red) and ECFP-SKL (green) 48 h after serum starvation. The ECFP-SKL-positive cell with primary cilia, indicated by the red box, was analyzed by FIB-SEM. Top, differential interference contrast (DIC); bottom, merged image. Scale bar, 10 µm. The fluorescent image of primary cilia, enlarged the area boxed in red in panel (A). Arrow indicates the ECFP-SKL positive-peroxisome associated with primary cilium. Scale bar, 5 µm. Single FIB-SEM image (left) showing a cross-section the ciliary pocket membrane, and 3D surface model reconstructed from segmented FIB-SEM images (right). Region of interest was coated by a protection layer of platinum before FIB milling. Plasma membrane (PM), yellow (surface view 75% transmission); cilia, green (surface view); ciliary pocket membrane (CPM), red (surface view 70% transmission); mitochondria (Mito), orange (surface view); peroxisome (PO), purple (surface view); basal body (BB), white (surface view). Scale bar, 1 µm. Enlarged 3D view of peroxisome interacting with ciliary pocket and mitochondria. Left, x-y view; right, oblique view. Scale bar, 1 µm. \n",
      ": VcdP interacts with and increases the activity of citrate synthaseA.Figure 5A. Immunoprecipitation of chromosomally tagged GltA::HA combined with either an empty vector control (pCtrl) or SPA-tagged vcdP over-expression plasmid, grown in LB medium to exponential phase (OD600 of 0.5). The SPA epitope contains the 3×FLAG and the calmodulin binding peptide sequences separated by a TEV protease cleavage site (Zeghouf et al, 2004). Therefore, protein samples corresponding to the total input and cell lysates before and after subjecting to immunoprecipitation with anti-Flag antibody were loaded on a SDS-PAGE gel. Western blotting with anti-HA and anti-Flag antibodies confirmed the interaction of GltA with VcdP in vivo. RNAP served as loading control. The solid triangles indicate the corresponding protein sizes. B. Citrate synthase enzyme activity measurements performed on cellular lysates. An empty vector control (pCtrl) or vcdP expression plasmids (pVcdP and pVcdP*) were conjugated into V. cholerae wild-type, ΔvcdRP, ΔgltA or F383A GltA backgrounds. Cells were grown in LB medium to stationary phase and subsequently lysed. The protein concentrations of these cellular extracts were measured, and served as input for the assay. The resulting colorimetric product at 412nm was proportional to the enzymatic activity of citrate synthase present. The enzyme kinetics were assayed at 15 second intervals for at least 30 minutes and the activity per milligram of lysate was determined from the initial velocities. C. Citrate synthase enzyme activity measurements performed in vitro. V. cholerae GltA and F383A GltA proteins were purified using a 6xHis tag in E. coli. Both the proteins were treated without the synthesized VcdP or VcdP* peptide, or with increasing concentrations of the two peptides (1x, 2x, 5x, 10x, 15x or 20x with respect to the protein monomer). The citrate synthase activity per milligram of protein as a result of its interaction with the peptide was determined from the initial velocities. \n",
      " is involved in the peroxisome-mediated cholesterol supply into the ciliary membrane (A) Schematic of ORP3 structure.Figure 5Schematic of ORP3 structure. ORP3 contains an N-terminal pleckstrin homology (PH) domain that interacts with phospholipids, an FFAT motif that specifically interacts with VAP-A/B, and a highly conserved C-terminal OSBP-homology domain (OHD). Western blot analysis showing depletion of ORP3 in the ORP3−/− hTERT-RPE1 cell clones. GAPDH served as a loading control. ORP3+/+ and ORP3−/− hTERT-RPE1 cells incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (green). Arrows indicate primary cilia. Scale bar, 5 μm. Quantification of the Filipin III intensity at primary cilia from (C). ORP3−/− hTERT-RPE1 cells exhibited a significant reduction of ciliary cholesterol (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Three-dimensional reconstitution of the quiescent G0-phase ORP3+/+ and ORP3−/− hTERT-RPE1 cells immunostained with anti-ARL13B (blue), anti-phospho-S473-Akt (red), anti-ORP3 (green), and anti-PMP70 (white) antibodies indicates that ORP3 at the ciliary pocket (arrow and arrowhead) mediates the membrane regions of ciliary pocket and peroxisomes (arrowhead). The scale bars indicate 5 μm. Quantification of proportion of primary cilia interacting with peroxisomes from (E). Depletion of ORP3 significantly interfered with the spatial interaction between peroxisomes and primary cilia (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 45-50 cells per experiment). 3×FLAG-tagged ORP3 and AcGFP1-tagged PEX14, EHD1 or EHD3 were coexpressed in HEK293T cells and then immunoprecipitated from whole-cell lysates using the anti-FLAG antibody. 3×FLAG-tagged ORP3 and AcGFP1-tagged PEX14, EHD1 or EHD3 fragments in the IP fractions and inputs were detected by western blotting. ORP3−/− hTERT-RPE1 cells were transfected with AcGFP1-tagged ORP3, PH domain deleted-ORP3 mutant (ΔPH), FFAT motif-deleted-ORP3 mutant (ΔFFAT), or OHD domain deleted-ORP3 mutant (ΔOHD) and cultured without serum for 24 h before Filipin-III (green)-mediated cholesterol staining and immunostaining with anti-GFP (red), anti-pericentrin (white), and anti-acetylated-tubulin (blue) antibodies. Arrows represent AcGFP1-tagged ORP3 or the mutants localized to the ciliary pocket. AcGFP1-tagged ORP3ΔPH mutant mis-localized to the ciliary pocket. The scale bars indicate 5 μm. Quantification of the Filipin III intensity at primary cilia from (H). AcGFP1-tagged ORP3 deletion mutants did not restore the ciliary cholesterol insufficiency in the ORP3−/− hTERT-RPE1 cells (*p<0.05, ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. \n",
      " Cells from ZS patients show defects in cholesterol enrichment in the ciliary membrane and Shh signal transduction (A) Primary skin fibroblasts from a normal individual, SLOS patient, ZS patients, X-ALD patient, and NPC patient incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies.Figure 1Primary skin fibroblasts from a normal individual, SLOS patient, ZS patients, X-ALD patient, and NPC patient incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (green). Arrows indicate primary cilia. Scale bar, 5 μm. The intensity of Filipin III signal at primary cilia from (a) was remarkably reduced in SLOS and ZS patient cells (**p<0.01: one-way ANOVA with Tukey's HSD, n=3: 45-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Immunofluorescent staining of primary skin fibroblasts in quiescent G0 phase treated with 50 nM Shh-N for 24 h using anti-Smo (green), anti-acetylated-tubulin (blue), and anti-γ-tubulin (red) antibodies. Arrows indicate primary cilia. Scale bar, 2.5 μm. Quantification of proportion of Smo-positive ciliated cells upon Shh-N stimulation from (C). The populations of Smo-positive ciliated fibroblasts from the SLOS and ZS patients were significantly reduced (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 100 cells per experiment). The Smo intensity at primary cilia in the fibroblasts from (C). The ciliary localization of Smo upon Shh-N stimulation was significantly impaired in fibroblasts from the SLOS and ZS patients (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 90-100 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Primary fibroblasts from a normal individual, and SLOS and ZS patients were treated with 1.5% methyl-β-cyclodextrin for 45 min to remove cellular cholesterol, and then incubated with or without cholesterol (cholesterol/methyl-β-cyclodextrin complex) for 1 h. After washing out the exogenous cholesterol, the fibroblasts were stimulated with 50 nM Shh-N for 24 h in the presence of pravastatin, and then immunostained with anti-Smo (green), anti-acetylated-tubulin (blue), and anti-pericentrin (red) antibodies. For the alternative cholesterol complementation, LDL (0.06 mg/ml) was co-incubated with Shh-N and pravastatin for 24 h after methyl-β-cyclodextrin-mediated cholesterol depletion. Scale bar, 2.5 μm. Quantification of the Smo intensity at primary cilia in the fibroblasts from (F). The ciliary accumulation of Smo upon the Shh-N stimulation of fibroblasts from an SLO patient was restored by the complementation of both exogenous cholesterol/methyl-β-cyclodextrin complex and LDL, while that of cells from ZS patients was efficiently rescued by not LDL but cholesterol/methyl-β-cyclodextrin complex (*p<0.05, **p<0.01, ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 90-100 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. \n",
      " CRISPR/Cas9-mediated disruption of the PEX genes phenocopies the dysfunction of cholesterol trafficking to ciliary membranes (A) Western blot analysis showing the depletion of PEX1 and PEX14 and the normal processing of Shh protein in the PEX1−/− and PEX14−/− hTERT-RPE1 cell clones.Figure 2Western blot analysis showing the depletion of PEX1 and PEX14 and the normal processing of Shh protein in the PEX1−/− and PEX14−/− hTERT-RPE1 cell clones. GAPDH served as a loading control. Immunostaining with anti-PEX1 (red) or PEX14 (red), anti-PMP70 (green), anti-ninein (blue), and anti-acetylated tubulin (white) in wild-type, PEX1−/−, and PEX14−/− hTERT-RPE1 cells in quiescent G0 phase. Arrowheads indicate primary cilia. Scale bar, 10 μm. Quantification of the number of PMP70-positive peroxisomes per cell from (B). Peroxisome formation in PEX1−/− and PEX14−/− hTERT-RPE1 cells was significantly impaired compared with that of the parental cells (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 20-25 cells per experiment). Quantification of proportion of ciliated cells from (B). Ciliogenesis in PEX1−/− and PEX14−/− hTERT-RPE1 cells was not significantly altered compared with that of the parental cells (mean ± s.d.: one-way ANOVA with Tukey's HSD, n=3: 190-200 cells per experiment). Quiescent G0-phase wild-type, PEX1−/−, and PEX14−/− hTERT-RPE1 cells transfected with AcGFP1-tagged D4 as a cholesterol probe were immunostained with anti-pericentrin (white) and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (red). Arrows and arrowheads indicate primary cilia and cytosolic accumulations of AcGFP1-tagged D4, respectively. Scale bar, 5 μm. Quantification of the Filipin III intensity at primary cilia from (E). PEX1−/− and PEX14−/− hTERT-RPE1 cells had significant reductions in the ciliary signal of Filipin III (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Quantification of the AcGFP1-tagged D4 intensity at primary cilia from (E). The ciliary signal of AcGFP1-tagged D4 in PEX1−/− and PEX14−/− hTERT-RPE1 cells was significantly diminished (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 25-30 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Quiescent G0-phase wild-type, PEX1−/−, and PEX14−/− hTERT-RPE1 cells were treated with or without 1.5% methyl-β-cyclodextrin for 45 min, and then incubated with or without cholesterol (cholesterol/methyl-β-cyclodextrin complex) for 1 h. After removing exogenous cholesterol, they were stimulated with 50 nM Shh-N for 24 h in the presence of pravastatin, and then immunostained with anti-Smo (green), anti-acetylated-tubulin (blue), and anti-γ-tubulin (red) antibodies. For the alternative cholesterol complementation, LDL (0.06 mg/ml) was co-incubated with Shh-N and pravastatin for 24 h after methyl-β-cyclodextrin-mediated cholesterol depletion. Scale bar, 2.5 μm. Quantification of the Smo intensity at primary cilia in wild-type, PEX1−/−, and PEX14−/− hTERT-RPE1 cells from (H). PEX1−/− and PEX14−/− hTERT-RPE1 cells exhibited the dampened Shh-N ligand-induced ciliary accumulation of Smo. The complementation of exogenous cholesterol (cholesterol/methyl-β-cyclodextrin complex) restored the ciliary accumulation of Smo in both PEX-knockout cells, while the LDL complementation did not rescue the ciliary phenotypes efficiently (*p<0.05, **p<0.01, ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 90-100 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. \n",
      " Peroxisomes accompanied by cholesterol move along the microtubules to interact with the ciliary pocket structure (A) Quiescent G0-phase hTERT-RPE1 cells were immunostained with anti-PEX14 (red), anti-ninein (white), and anti-acetylated tubulin (blue) antibodies.Figure 3Quiescent G0-phase hTERT-RPE1 cells were immunostained with anti-PEX14 (red), anti-ninein (white), and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (Green). Magnified images of the boxed regions showing peroxisome accompanied by cholesterol (arrows). Three-dimensional reconstitution of the same cell indicates that Filipin III stains the membrane regions of ciliary axonemes and peroxisomes. The scale bars indicate 2.5 μm and 1.25 μm in lower- and higher-magnified images, respectively. Quiescent G0-phase wild-type hTERT-RPE1 cells treated with Cytochalasin-D (200 nM), colcemid (50 nM), or Ciliobrevin-D (10 μM) for 6 h were immunostained with anti-ARL13B (blue), anti-phospho-S473-Akt (red), anti-ninein (green), and anti-PMP70 (white) antibodies. Arrows indicate the peroxisomes interacting with the ciliary pocket. Scale bar, 5 μm. Quantification of proportion of primary cilia interacting with peroxisomes from (B). Colcemid (50 nM) and Ciliobrevin-D (10 μM) significantly inhibited the spatial interaction between peroxisomes and primary cilia (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 45-50 cells per experiment). Quiescent G0-phase wild-type hTERT-RPE1 cells were treated with Cytochalasin-D (200 nM), Colcemid (50 nM), or Ciliobrevin-D (10 μM) for 6 h and then immunostained with anti-acetylated tubulin (blue) and anti-pericentrin (green) antibodies. Cholesterol was stained with Filipin III (green). Arrows indicate primary cilia. Scale bar, 2.5 μm. Quantification of the Filipin III intensity at primary cilia from (D). Colcemid and Ciliobrevin-D interfered with the distribution of cholesterol in the ciliary membrane (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. hTERT-RPE1 cells were transfected with AcGFP1-tagged EHD3 and cultured without serum for 24 h before immunostaining with anti-GFP (green), anti-ARL13B (blue), anti-PMP70 (white), and anti-phospho-S473-Akt (red) antibodies. The scale bars represent 2.5 μm. 3×FLAG-tagged EHD1 or EHD3 and AcGFP1-tagged PEX14 were coexpressed in HEK293T cells and then immunoprecipitated from whole-cell lysates using the anti-FLAG antibody. AcGFP1-tagged PEX14 and 3×FLAG-tagged EHD1 or EHD3 fragments in the IP fractions and inputs were detected by western blotting. Recombinant GST (0.2 μg) or GST fused to PEX14 proteins (1 μg) and 6×His-EHD3 protein (1 μg) were pulled down using glutathione-Sepharose beads. GST-tagged PEX14 and 6×His-tagged EHD3 proteins in the pull-down fractions and inputs were detected by western blotting. 3D reconstitution of the quiescent G0-phase hTERT-RPE1 cells transfected with AcGFP1-tagged EHD3 (white) immunostained with anti-acetylated-tubulin (blue) and anti-PEX14 (red) antibodies. Cholesterol was stained with Filipin III (green). Cholesterol-containing peroxisome interacted with the ciliary pocket (arrow). The scale bars indicate 2 μm. \n",
      ".Figure 4(A - C) Fluorescence images (upper) and quantifications (lower) of GFP-tagged CST-1 (A  B) or GFP-labeled F-actin (C) and mCherry tagged plasma membrane and histone in WT and mig-2 null mutant animals. The membrane protrusions in the white boxes are enlarged 3 times at the bottom. Yellow dashed lines indicate the cell periphery. White lines indicate the line to measure the fluorescence intensity. Green lines denote the CST-1 fluorescence intensity; red lines denote the plasma membrane fluorescence intensity; double-headed arrows denote the distance between the GFP signal and cell periphery. Scale bars, 5 μm. (D) Distance between GFP::CST-1 or GFP::Moesin ABD and the plasma membrane at the leading edge in WT and mig-2 null mutants. Error bars indicate SD. N = 9 - 14. ***, P  0.001 by Student's t test. (E - F) Fluorescence images (left) and quantifications (right) of GFP-tagged CST-1 and mCherry tagged plasma membrane and histone in WT and mig-2 null mutant animals. Blue lines, sites of measurements; dotted lines, cell periphery; Green lines; CST-1 fluorescence intensity; red lines, mCherry fluorescence intensity. Scale bars, 5 μm. (G) Quantification of the fluorescence intensity ratio of GFP::CST-1 and mCherry-membrane between the leading edge and the lagging edge of AQR in WT and mig-2 null mutant animals. Error bars indicate SD. N = 20. ***, P < 0.001 by Student's t test. (H) Proposed model of Hippo-RhoG signaling negative feedback loop in the prevention of secondary fronts during directional cell migration. \n",
      ".Figure 1(A) Model of Q cell position and migration direction in C. elegans L1 Larvae. Green arrow indicates that Q cells migrate along the AP body axis, whereas the dotted red arrows indicate that Q cells are potent to move towards the DV direction. (B) Schematics of the cst-1 and cst-2 gene models and DNA sequences of the cst-1 and/or cst-2 mutants that were generated via CRISPR-Cas9. Purple bar, Serine/Threonine kinase domain; blue bar, SARAH (Salvador-Rassf-Hippo homolog) domain. Arrow, sgRNA target site; green, target sequence; red, PAM sequence; blue, inserted nucleotide; dashed lines, deleted nucleotides. (C) Quantification of AQR position in adult WT, cst-1 and/or cst-2 mutant animals. The full length between the URXs and the end of the nematode tail is divided into 10 blocks, and the percent of AQR that stopped within each block is indicated. ***, P < 0.001 by Fisher's exact test. N = 50 - 109. (D) Quantification scheme of the angle (α) between the longitudinal axis of a migrating cell and the AP body axis. (E - F) Fluorescence time-lapse images of F-actin, plasma membrane and histone during AQR/QR.ap migration in WT (E) or cst-1; cst-2 double mutant (F) animals. Merged images, left; inverted fluorescence images of F-actin, right; white arrows, migration direction; asterisks, nucleus; double headed arrows, migration distance. F-actin is labeled with GFP-tagged with the actin-binding domain of Moesin (GFP::Moesin ABD), and the plasma membrane and Histone are labeled with mCherry-tagged with a myristoylation signal and a histone. The dotted lines indicate the cell periphery. Time is presented in minutes. Scale bars, 5 μm. (G - J) Quantification of the QR.ap migration angle (G H), distance (I) and speed (J). Each line in (G I) represents the measurement from one time-lapse movie. Error bars indicate SD. N = 10 - 17. Statistical significance is based on Student's t-tests, **, P 0.01; ***, P  0.001. \n",
      ".Figure 2(A) Fluorescence time-lapse images of F-actin, plasma membrane and histone during QR.ap migration in mig-2(rh17) gain-of-function mutant animals. Merged images, left; inverted fluorescence images of F-actin, right; white arrows, migration direction; asterisks, nucleus; double headed arrow, migration distance. The dotted lines indicate the cell periphery. Time is presented in minutes. Scale bar, 5 μm. (B - C) Quantification of the QR.ap migration angle (B) and distance (C). N = 10. (D) Mass spectrum of a MIG-2 peptide that was phosphorylated at Serine 139 in C. elegans. The peptide was identified from the GFP::MIG-2 immunoprecipitate. (E) In vitro phosphorylation assay of CST-1/2 to MIG-2. Bacterially expressed His-tagged MIG-2 or His-tagged MIG-2S139A were incubated with the GST-tagged kinase domain of GST-CST-1/2 (1-318 aa). The kinase may use ATP-γ-S as a phosphodonor to thiophosphorylate its substrate; thus, the phosphorylation reaction was performed with ATP-γ-S, and anti-thiophosphate ester antibody was used to detect substrate phosphorylation (Allen et al, 2005; Allen et al, 2007). The final reaction was probed via immunoblotting with antibodies as indicated. Thio, thiophosphate ester. (F) Sequence alignment of MIG-2. Serine 139 (red) is conserved among the Rac1 homologs across species. The accession version is as follows: C. elegans, CAB01691.1; Drosophila, CAC88352.1, AAA67041.1; Human, CAB53579.5; Rat, NP_599193.1; Mouse, AAH51053.1; Chicken, NP_990348.1, NP_001188381.1; Zebrafish, AAV85902.1; Xenopus, NP_001089332.1. (G H) Three-dimensional ribbon (PDB 3TH5) representations of the human Rac1 polypeptide which indicate the position of the T135 (S139 in MIG-2) located between the last two α-helixes (Krauthammer et al, 2012). \n",
      ".Figure 3(A) Quantification of AQR position in mig-2(mu28) null mutant animals with different rescue transgenes. N = 108 - 114. *, P < 0.05; ##, P < 0.01, phenotype enhanced. P values are based on Fisher's exact tests. (B) Quantification of AQR position in WT and knock-in animals that harbor mig-2S139 point mutations. N = 100 - 111. ***, P < 0.001 by Fisher's exact test. (C - F) Quantification of the QR.ap migration angle (C D), distance (E) and speed (F) in mig-2S139A/E mutant animals. Each line in (C  E) represents the measurement from one movie of QR.ap migration. Error bars indicate SD. N = 12 -17. **, P 0.01; ***, P  0.001 by Student's t tests. (G) Inverted fluorescence images (up) and schematics (down) of QR.ap in WT, cst-1; cst-2 double or mig-2 mutant animals. The plasma membrane and histones were labeled with mCherry. Scale bar, 5 μm. (H) Area ratio between the cell body and the nucleus in WT and mutant animals. Error bars indicate SD. N = 17 - 22. *, P < 0.05; **, P < 0.01; ***, P < 0.001 by Student's t tests. (I - K) Representative frames (upper) and quantification (lower) of F-actin enrichment at the QR.ap periphery in WT (I) and mig-2S139A/E mutant animals (J  K). The trace starts from the rear of the QR.ap and moves along the cell periphery to the leading edge (L) and back to the rear (R). F-actin was labeled with GFP::Moesin ABD; the plasma membrane and histone were marked with mCherry fluorescence. Green lines, F-actin fluorescence intensity; red lines, plasma membrane fluorescence intensity; dotted lines, cell periphery. Double-headed arrows indicate the F-actin enrichment region, in which the fluorescence intensity is 2-fold higher than the background. Scale bars, 5 μm. (L) Quantification of F-actin enrichment ratio in WT and mig-2 mutants. Error bars indicate SD. N = 28 - 68. **, P  0.01; ***, P  0.001 by Student's t-tests. \n",
      " Peroxisomes directly contribute to the cholesterol supply into ciliary membranes (A) Quiescent G0-phase wild-type, Rab10−/−, KIFC3−/−, ORP3−/−, and EHD3−/− hTERT-RPE1 cells transfected with PEX3-GFP-FRB (red) and tdTomato-BicD2-FKBP (white) were treated with or without 100 μM rapamycin for 30 min, and then immunostained with anti-acetylated-tubulin (blue) antibody.Figure 8Quiescent G0-phase wild-type, Rab10−/−, KIFC3−/−, ORP3−/−, and EHD3−/− hTERT-RPE1 cells transfected with PEX3-GFP-FRB (red) and tdTomato-BicD2-FKBP (white) were treated with or without 100 μM rapamycin for 30 min, and then immunostained with anti-acetylated-tubulin (blue) antibody. Cholesterol was stained with Filipin III (Green). Arrows indicate the enrichment of ciliary cholesterol. The scale bars represent 5 μm. Quantification of the Filipin III intensity at primary cilia from (A). Rapamycin-induced peroxisome targeting to the ciliary pocket restored the insufficiency of ciliary cholesterol in the Rab10−/−and KIFC3−/−hTERT-RPE1 cells, but not in ORP3−/−and EHD3−/− hTERT-RPE1 cells (*p<0.05, **p<0.01, ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 35-40 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Model for the molecular machinery for the supply of cholesterol into the ciliary membrane. The Rabin8-Rab10-KIFC3 complex associates peroxisomes with cholesterol to load them on microtubule arrays. KIFC3-mediated microtubule minus-end-directed movement guides peroxisomes to the ciliary pocket for the supply of cholesterol to the ciliary membrane. The ORP3-EHD1-EHD3 complex is involved in cholesterol trafficking at the ciliary pocket. In the pathological context of ZS, insufficient ciliary cholesterol might impair the cilium-related Shh signal competence, causing ciliopathy-related symptoms. \n",
      "a) EGF enhances colocalization of phosphorylated (P)-bRAF, P-MEK and P-ERK with LC3.f1a) EGF enhances colocalization of phosphorylated (P)-bRAF, P-MEK and P-ERK with LC3. Immunofluorescence (IF) showing colocalization of P-bRAF (green), P-MEK (green) and P-ERK (green) with LC3 (red) in NIH/3T3 cells in presence or absence of EGF (10 min as shown in scheme). The bars represent mean±s.e.m. **P0.01, ***P0.001; Student's t-test, 60 cells analysed from two experiments. Scale bars, 10 μm. (b) P-ERK colocalizes with pre-autophagosomal and autophagosomal structures. IF depicting colocalization of P-ERK (green or red) with LC3 (red), ATG5-ATG12 (red), ATG16 (green), vps34 (red), WIPI1 (red), WIPI2 (red) and P-ULK1 (red) in EGF-treated NIH/3T3 cells. Scale bars, 10 μm. The bars represent mean±s.e.m. 50 cells analysed from two experiments. (c) Growth factors in serum maintain P-ERK/LC3 colocalization. IF depicting colocalization of P-ERK (green) with LC3 (red) in 2 h serum-fed NIH/3T3 cells. (a-c) Native merged images or images with colocalization highlighted in white pixels are shown. Nuclei are blue (DAPI). Scale bar, 10 μm. \n",
      "a) P- and total MEK and ERK are enriched in APh in vivo.f2(a) P- and total MEK and ERK are enriched in APh in vivo. Immunoblots for indicated proteins in homogenate (Hom), APh, APL and Lys fractions from livers of fed mice, n=4. (b) EGF enhances enrichment of P-MEK and ERK in APh fractions in vitro. Immunoblots for indicated proteins in Hom, APh and APL fractions from NIH/3T3 cells in presence or absence of EGF (10 min). The bars represent mean±s.e.m. *P0.05, **P0.01; Student's t-test, n=3. (c) P- and total MEK and ERK localize on to the cytoplasmic/extra-luminal face of APh in vivo. Left; model depicting localization of MEK and ERK on the extra-luminal face of autophagic vesicles and the ability of trypsin to degrade extra-luminal MEK and ERK, and right; immunoblots for indicated proteins in APh fractions from mice livers untreated (−) or treated with increasing amounts of trypsin for 15 min. The bars represent mean±s.e.m. *P0.05, **P0.01, ***P0.001 compared with corresponding trypsin-untreated value; ANOVA-Bonferroni post hoc test, n=3. The p44, p42 forms of ERK, LC3-I and membrane-associated LC3-II, and 37 kDa and 25 kDa forms of cathepsin (Cath) B are indicated. \n",
      " to control the peroxisome-mediated supply of cholesterol to the ciliary membrane (A) Rabin8+/+ and Rabin8−/− hTERT-RPE1 cells incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies.Figure 6Rabin8+/+ and Rabin8−/− hTERT-RPE1 cells incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (green). Scale bar, 5 μm. Quantification of the Filipin III intensity at primary cilia from (A). Rabin8−/− hTERT-RPE1 cells exhibited a significant reduction of ciliary cholesterol (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Quiescent G0-phase Rabin8+/+ and Rabin8−/− hTERT-RPE1 cells were immunostained with anti-ARL13B (blue), anti-phospho-S473-Akt (red), anti-ninein (green), and anti-PMP70 (white) antibodies. Arrow indicates the peroxisomes contacting the ciliary pocket. Scale bar, 5 μm. Quantification of (C) indicating that disruption of the Rabin8 gene significantly inhibited the spatial interaction between peroxisomes and primary cilia (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 45-50 cells per experiment). Schematic representation of the rapamycin-inducible peroxisome targeting to the ciliary pocket system. Quiescent G0-phase hTERT-RPE1 cells transiently expressing PEX3-GFP-FRB (green) and tdTomato-BicD2-FKBP (red) treated with rapamycin were imaged live for 10 min by confocal microscopy (Movie EV10). Scale bar, 5 μm. Quiescent G0-phase Rabin8+/+ and Rabin8−/− hTERT-RPE1 cells transfected with PEX3-GFP-FRB (red) and tdTomato-BicD2-FKBP (white) were treated with or without 100 μM rapamycin for 30 min, and then immunostained with anti-acetylated-tubulin (blue) antibody. Cholesterol was stained with Filipin III (Green). Arrows indicate the enrichment of ciliary cholesterol. The scale bars represent 5 μm. Quantification of the Filipin III intensity at primary cilia from (G). Rapamycin-induced peroxisome targeting to the ciliary pocket restored the reduction of ciliary cholesterol in the Rabin8−/− hTERT-RPE1 cells (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Rabin8−/− hTERT-RPE1 cells were transfected with AcGFP1-tagged Rabin8, GFP-tagged Rab8A, GFP-tagged Rab8A-Q67L (constitutively active form), GFP-Rab8B, GFP-Rab8B-Q67L (constitutively active form), AcGFP1-tagged Rab10, or AcGFP1-tagged Rab10-Q68L (constitutively active form), and then immunostained with anti-GFP (red), anti-acetylated-tubulin (blue), and anti-ninein (white) antibodies. Cholesterol was stained with Filipin III (Green). Arrows indicate the enrichment of ciliary cholesterol. The scale bars represent 5 μm. Quantification of (I) showing that AcGFP1-tagged Rabin8 and AcGFP1-tagged Rab10-Q68L effectively restored the defect in ciliary enrichment of cholesterol in Rabin8−/− hTERT-RPE1 cells (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 35-40 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. \n",
      "-Rab10-KIFC3 complex is implicated in the peroxisome-mediated supply of cholesterol into the ciliary membrane (A) Whole-cell lysates from HEK293T cells expressing AcGFP1 or AcGFP1-tagged PEX14 and 3×FLAG-tagged KIFC3 were immunoprecipitated with anti-FLAG antibody and immunoblotted with anti-GFP or anti-FLAG antibody.Figure 7Whole-cell lysates from HEK293T cells expressing AcGFP1 or AcGFP1-tagged PEX14 and 3×FLAG-tagged KIFC3 were immunoprecipitated with anti-FLAG antibody and immunoblotted with anti-GFP or anti-FLAG antibody. Whole-cell lysates from HEK293T cells expressing AcGFP1-tagged KIFC3 and 3×FLAG-tagged Rab10 or Rab10-Q68L were immunoprecipitated with anti-FLAG antibody and immunoblotted with anti-GFP or anti-FLAG antibody. Whole-cell lysates from HEK293T cells expressing AcGFP1-tagged Pex14 and 3×FLAG-tagged Rab10 or Rab10-Q68L were immunoprecipitated with anti-FLAG antibody and immunoblotted with anti-GFP or anti-FLAG antibody. Western blot analysis of the ciliary cholesterol trafficking-associated components in Pex1+/+ and Pex1−/− hTERT-RPE1 cells. Total cell lysates were separated to crude peroxisomal (Crude Pex), lysosomal and mitochondrial (Lyso/Mito), and peroxisomal (PEX) fractions. CYPOR, a lysosomal and mitochondrial protein, served as a positive control for the Lyso/Mito fractionation. Total cell lysates were gradually injected at 20 µg and 5 µg into a gel for SDS-PAGE, while equal amounts (5 µg) of protein from each fraction were loaded. Rabin8, Rab10, and KIFC3 were concentrated in the PEX fraction in Pex1+/+ hTERT-RPE1 cells. CYPOR (cytochrome P450 reductase) is a mitochondrial protein. Quiescent G0-phase hTERT-RPE1 cells transfected with 3×FLAG-tagged KIFC3 were immunostained with anti-FLAG (green), anti-PEX14 (red), and anti-α-tubulin (white) antibodies. Magnified 3D-constituted images of the boxed regions showing peroxisomes are located on the microtubule arrays via KIFC3 (arrows). The scale bars indicate 5 μm. Western blot analysis showing depletion of KIFC3 in the KIFC3−/− hTERT-RPE1 cell clones. GAPDH served as a loading control. KIFC3+/+ and KIFC3−/− hTERT-RPE1 cells incubated for 24 h without serum were immunostained with anti-pericentrin (red) and anti-acetylated tubulin (blue) antibodies. Cholesterol was stained with Filipin III (green). Scale bar, 5 μm. Quantification of (G) indicating that KIFC3−/− hTERT-RPE1 cells significantly reduced the ciliary accumulation of cholesterol (***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. Quiescent G0-phase KIFC3+/+ and KIFC3−/− hTERT-RPE1 cells were immunostained with anti-ARL13B (blue), anti-phospho-S473-Akt (red), anti-ninein (green), and anti-PMP70 (white) antibodies. Arrows represent the peroxisomes contacting the ciliary pocket. Scale bar, 5 μm. Quantification of (I) showing that disruption of the KIFC3 gene significantly interfered with the contact between peroxisomes and primary cilia (mean ± s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 45-50 cells per experiment). KIFC3−/− hTERT-RPE1 cells were transfected with AcGFP1, AcGFP1-tagged KIFC3, rod domain-deleted KIFC3 mutant (ΔRod), or motor domain-deleted KIFC3 mutant (ΔMotor) and cultured without serum for 24 h before Filipin-III (green)-mediated cholesterol staining and immunostaining with anti-GFP (red), anti-pericentrin (white), and anti-acetylated-tubulin (blue) antibodies. Arrows represent ciliary localization of cholesterol. The scale bars indicate 5 μm. Quantification of the Filipin III intensity at primary cilia from (K). AcGFP1-tagged KIFC3 deletion mutants did not restore the ciliary cholesterol insufficiency in the KIFC3−/− hTERT-RPE1 cells (mean +/- s.d.: ***p<0.001: one-way ANOVA with Tukey's HSD, n=3: 40-50 cells per experiment). In the boxplot, medians, 25th/75th percentile and min/max were represented by the central lines, the box limits, and the whiskers/error bars, representatively. \n",
      ".Figure 2A. Cells were pulse-labelled with BrdU for 1.5 h and sorted by flow cytometry in two fractions, S1 and S2, corresponding to early and late S phase fractions, respectively. Neo-synthesized DNA was immunoprecipitated with BrdU antibodies. Early and late neo-synthesized DNAs were labeled with Cy3 and Cy5, respectively and hybridized on microarrays. After processing analysis with the START-R software, replication-timing profiles were obtained from two replicates Shown are the zoomed microarray profiles of the timing of replication on chromosome 1 (136.6-143.5 Mb), chromosome 5 (127.3-134.7 Mb) and chromosome 9 (46.1-51.3 Mb) from Rev3l+/+ and Rev3l-/- MEFs overlaid. Blue lines represent replication timing from Rev3l+/+ MEFs and red lines represent replication timing from Rev3l-/- MEFs. Genomic regions displaying significant difference between Rev3l+/+ and Rev3l-/- MEFs by START-R are indicated by a pink line (p<0.01). B. Analysis of GC and Line-1 content in early, mid, late and TTR compared to disturbed replicating regions found in Rev3l-/- MEFs. Bar in boxplot represents the median, and red points represents the mean. The limit of the boxes corresponds to the 0.25 to 0.75 quartiles with whiskers extending to the maximum value of 1.5 times the interquartile range. Graphs show data from two biological experiments. C. Analysis of the active marks H3K27ac and H3K4me3 and facultative heterochromatin mark H3K27me3 content in early, mid, late and TTR compared to disturbed replicating regions found in Rev3l-/- MEFs. Bar in boxplot represents the median, and red points represents the mean. The limit of the boxes corresponds to the 0.25 to 0.75 quartiles with whiskers extending to the maximum value of 1.5 times the interquartile range. Graphs show data from two biological experiments. \n",
      "We propose that growth factor stimulation results in the docking of the ERK cascade components, Raf, MEK and ERK onto the cytoplasmic face of autophagic structures.f8We propose that growth factor stimulation results in the docking of the ERK cascade components, Raf, MEK and ERK onto the cytoplasmic face of autophagic structures. ATG5-ATG12-positive preautophagosomes and LC3-II-positive membranes serve as scaffolds or cellular signalling platforms that facilitate efficient spatial coordination of the Raf-MEK-ERK cascade and thus facilitate growth factor-induced ERK phosphorylation. \n",
      ".Figure 1A. Rev3l+/+ and Rev3l-/- MEFs were pulse-labelled with BrdU prior to harvesting and analyzed by flow cytometry at different time points. The analysis was focused on S phase divided in 3 parts: G1/early S, middle S and late S/G2 BrdU+ cells. Red arrow represents BrdU+ cells that re-entry in S phase and green arrow shows BrdU- cells in G2 phase. Histograms represent the percentage of cells in G1/early and late S/G2 phase after BrdU pulse. Error bars represent standard error for three independent experiments. ns: not significant, * p<0.05 by Student's t-test. B. Rev3l+/+ and Rev3l-/- MEFs were pulse-labelled with EdU for 15min, permeabilized, fixed and stained for EdU incorporation (green). S phase sub-stages from I to IV were evaluated by visual inspection of the cycling population (>300 EdU+ cells, top panel). Scale bar= 5 μm. Dot plots and pie charts show the relative proportion (percentage of total S) of each sub-stage from I to IV (middle and bottom panel, respectively). Each dot represents the mean of two technical replicates. C. Seventy-two hours after transfection with non-targeting (NT) or Rev3l siRNA, HeLa cells were pulse-labelled with BrdU for 15 min, permeabilized, fixed and stained for BrdU (red). As in (B), S phase sub-stages were evaluated by visual inspection of the cycling population (>300 BrdU+ cells, top panel). Scale bar= 5 μm. Dot plots and pie charts show the relative proportion (percentage of total S) of early, middle-late and late S phase (middle and bottom panel, respectively). Each dot represents the mean of two technical replicates. \n",
      ".Figure 3A. Heat map showing log2 fold change in differentially expressed genes in Rev3l+/+ and Rev3l-/- MEFs from 2 independent biological experiments (Rev3l+/+: samples 1 and 2, Rev3l-/-: samples 5 and 6) with 2 technical replicates (samples 3,4,7,8: technical repeats from samples 1,2,5,6 respectively). Several developmentally regulated genes and imprinting genes (paternally in blue and maternally in pink) downregulated in Rev3l-/- MEFs are indicated on the right. Yellow and blue indicate high and low mRNA expression levels, respectively. B. Top five biological process gene ontology (GO) terms of genes found downregulated in Rev3l-/- MEFs transcriptome analysis. C. Volcano plot shows in red genes involved in development and imprinting with high fold changes ≥ 3 (FDR<0.05 D. Relative mRNA levels of four genes involved in development (Hoxb2, 8, 9 and WT1) were validated through qRT-PCR from Rev3l+/+ and Rev3l-/- MEFs samples. The data were normalized to the amount of GAPDH mRNA. Error bars indicate standard error of the mean for three independent experiments. ***, P<0.001 and **, P<0.005 by Student's t-test. E. Relative mRNA levels of genes involved in imprinting (paternal expressed genes: Dlk1and Slc18a4 and maternal expressed genes: Dcn1 and H19) were validated through qRT-PCR from Rev3l+/+ and Rev3l-/- MEFs samples. The data were normalized to the amount of GAPDH mRNA. Error bars indicate standard error of the mean for three independent experiments. *** P<0.001 by Student's t-test. \n",
      "a) Atg7−/− livers display decreased ERK phosphorylation.f4(a) Atg7−/− livers display decreased ERK phosphorylation. Immunoblots for the indicated proteins in liver homogenate (Hom) and cytosolic (Cyt) fractions of control (Con) and liver-specific Atg7−/− mice are shown. The bars represent mean±s.e.m. *P0.05, **P0.01, ***P0.001 compared with Con; Student's t-test, n=4. (b) Atg7−/− livers display decreased ERK2 dimers. Immunoblots show the indicated proteins in Hom from Con and Atg7−/− livers. The bars represent mean±s.e.m. *P0.05, ***P0.001 compared to Con; Student's t-test, n=4-5. Arrows indicate p42 monomers and dimers. (c) Atg7−/− brown adipose tissues (BAT) display decreased ERK phosphorylation. Immunoblots for indicated proteins in Hom from Con and Atg7−/−BAT. The bars represent mean±s.e.m. **P0.01 compared with Con; Student's t-test, n=4-6. (d) Atg7−/− livers exhibit decreased nuclear ERK phosphorylation. Immunoblots show the indicated proteins in homogenate (Hom, lane 1) and nuclear fractions (lanes 2-5) from Con and Atg7−/− livers. The bars represent mean±s.e.m. ***P0.001 compared with Con; Student's t-test, n=4-5. (e) LC3 C terminus glycine-deleted (ΔG) mutants exhibit decreased ERK phosphorylation. Immunoblots for indicated proteins in total lysates from CFP-LC3- and CFP-LC3ΔG-transfected NIH/3T3 cells exposed or not to EGF (10 min). The bars represent mean±s.e.m. ***P0.001 compared with Con; Student's t-test, n=8. \n",
      "a) Atg5−/− MEFs display reduced ERK phosphorylation during nutrient deprivation.f5(a) Atg5−/− MEFs display reduced ERK phosphorylation during nutrient deprivation. Quantification of P-ERK levels normalized to total ERK (detected by immunoblotting) in WT and Atg5−/− MEFs cultured in the absence of serum for indicated times. The bars represent mean±s.e.m. *P0.05 compared with corresponding WT value; ***P0.001, ****P0.0001 compared with 10 min serum-starved Atg5−/− MEFs; ANOVA-Bonferroni post hoc test, n=3. (b) Serum-deprived Atg5−/− MEFs display decreased ERK and p90RSK phosphorylation. Immunoblots for the indicated proteins in total lysates from 2 h serum-deprived WT and Atg5−/− MEFs. The bars represent mean±s.e.m. *P0.05, ***P0.001 compared with Con; Student's t-test, n=3. (c) Atg5−/− MEFs display decreased nuclear ERK activity. Immunoblots for phosphorylated HA (haemagglutinin)-tagged Elk1 in total lysates from 2 h serum-deprived WT and Atg5−/− MEFs cells exposed or not to EGF (10 min). (d) Atg5−/− MEFs display decreased Elk1-driven gene expression. Elk1-driven ZFP36 mRNA levels from 2 hserum-deprived WT and Atg5−/− MEFs cells exposed or not to EGF (10 min). The bars represent mean±s.e.m. ***P0.001 compared with Con; Student's t-test, n=3. ( (e) Decreased ERK phosphorylation in Atg5−/− MEFs occurs independently of changes in ERK phosphatases. Immunoblots for the indicated proteins in 2 h serum-deprived WT MEFs transfected with scrambled siRNA (scr), and Atg5−/− MEFs transfected with scr or siRNAs against MKP3 or PP2A in the presence or absence of EGF. The bars represent mean±s.e.m., n=3. \n",
      "a) Mutations in FRS on ERK2 decrease colocalization of ERK2 with ATG5–ATG12, LC3 and WIPI1.f6(a) Mutations in FRS on ERK2 decrease colocalization of ERK2 with ATG5-ATG12, LC3 and WIPI1. Immunofluorescence (IF) showing colocalization (depicted as white pixels by 'colocalization finder application') of WT-ERK2-HA, FRS ERK2 mutants (L198A-, L232A-, L198A/L232A-, Y261A-ERK2-HA) or common docking (CD) mutant (D319N-ERK2-HA) with ATG5-ATG12 (panels 1-6), LC3 (panels 7-12) or WIPI1 (panels 13-18) in EGF-treated NIH/3T3 cells. ERK2 is stained in red, and autophagy proteins are stained in green. Scale bar, 10 μm. The bars represent mean±s.e.m. **P0.01, ****P0.0001 compared with WT-ERK2-transfected cells; Student's t-test, 50 cells analysed from n=2. ( (b) Mutations in FRS on ERK2 decrease colocalization of ERK2 with nuclearATG5-ATG12 or LC3. IF showing colocalization (white pixels) of WT-ERK2-HA (red), FRS ERK2 mutants (L198A-, L232A-, L198A/L232A-, Y261A-ERK2-HA) or common docking (CD) mutant (D319N-ERK2-HA) with ATG5-ATG12 (panels 1-6) or LC3-II (panels 7-12) in EGF-treated NIH/3T3 cells. Scale bar, 5 μm. The bars represent mean±s.e.m. **P0.01, ***P0.001, ****P0.0001 compared with WT-ERK2-transfected cells; Student's t-test, 50 cells analysed from n=2. Arrows depict colocalization. (c) ATG5-ATG12 conjugation is required for ERK phosphorylation. Immunoblots for P-ERK, total ERK and β-actin in NIH/3T3 cells transfected with WT ATG5 or the conjugation-defective ATG5 K130R mutant and treated with EGF (10 min). The bars represent mean±s.e.m. *P0.05 compared with WT ATG5-transfected cells; Student's t-test, n=3. \n",
      "a) EGF enhances nuclear LC3-II content in hepatocytes.f3(a) EGF enhances nuclear LC3-II content in hepatocytes. Immunofluorescence (IF) depicting nuclear LC3-II in RALA hepatocytes in presence or absence of EGF (10 min). Native/inverted images are shown. Scale bar, 5 μm. Bars represent mean±s.e.m. ***P0.001 compared with control (Con); Student's t-test, 50 cells from two experiments. (b) EGF enhances nuclear P-ERK/LC3-II colocalization. IF depicting P-ERK (green)/LC3-II (red) colocalization in untreated (Con) or EGF-treated NIH/3T3 cells. Native images (top)/images with colocalization in white pixels (bottom) are shown. Scale bar, 5 μm. Bars represent mean±s.e.m. ***P0.001 compared with Con; Student's t-test, 50 cells from two experiments. (c) Adipogenic differentiation increases nuclear LC3-II. Images depict nuclear LC3-II in 3T3-L1 preadipocytes in maintenance or differentiation medium (2 h). Scale bar, 5 μm. Bars represent mean±s.e.m. **P0.01 compared with Con; Student's t-test, 50 cells from two experiments. (d) Nuclear P-ERK/LC3-II colocalization in serum-fed cells. IF showing P-ERK (green)/LC3 (red) colocalization in 2 h serum-fed NIH/3T3 cells. Native images (top)/images with colocalization in white pixels (bottom) are shown. Scale bar, 5 μm. (e) LC3 interacts with ERK in vivo. Immunoblots showing co-immunoprecipitation of LC3 with ERK, MEK and bRAF in homogenate (Hom) (e, top), and of LC3 with P- and total ERK in nuclear fractions from mouse livers (e, bottom). (f) Blocking nuclear transport decreases EGF-induced increase in nuclear LC3-II. LC3 IF (red) in EGF-treated NIH/3T3 cells pre-exposed (30 min) or not to WGA. Bars represent mean±s.e.m. ***P0.001 compared to with; Student's t-test, 60 cells from n=3. Scale bar, 10 μm. (g) Blocking nuclear transport decreases nuclear ERK content. ERK IF (green) in EGF-treated NIH/3T3 cells pre-exposed (30 min) or not to WGA. Bars represent mean±s.e.m. ***P0.001 compared with Con; Student's t-test. (h) Blocking nuclear transport does not modify P-ERK/LC3-II colocalization. IF depicting nuclear P-ERK (green)/LC3 (red) colocalization in EGF-treated NIH/3T3 cells pre-exposed or not to WGA. For (g) and (h): Scale bar, 10 μm, bars are mean±s.e.m. 50 cells from n=2. Nuclei are blue (DAPI). Arrows indicate LC3 puncta, ERK or colocalization. \n",
      "a) ATG4B depletion increases steady-state LC3-II levels.f7a) ATG4B depletion increases steady-state LC3-II levels. Immunoblots for indicated proteins in NIH/3T3 cells transfected with scrambled siRNAs (scr) or siRNAs against ATG4B (siATG4B) in presence/absence of EGF. Bars represent mean±s.e.m. *P0.05, ***P0.001 compared with scr; Student's t-test, n=4. (b) siATG4B cells exhibit increased autophagic vesicles. Representative electron micrographs depicting autophagic vesicles (indicated by black arrows) in scr and siATG4B NIH/3T3 cells. (c) ATG4B depletion does not increase autophagic flux in serum-fed cells. Immunoblots for indicated proteins in lysates from scr or siATG4B NIH/3T3 cells in presence/absence of lysosomal inhibitors, ammonium chloride and leupeptin (Inh) for 2 h. Scale bars, 1 μm. Bars represent mean±s.e.m., n=3. (d) siATG4B cells display increased nuclear LC3-II. Images for LC3-II (red) in scr or siATG4B NIH/3T3 cells in presence/absence of EGF (10 min). Scale bar, 5 μm. Bars represent mean±s.e.m. ***P0.001 compared with scr; Student's t-test, 60 cells from n=2. (e) ATG4B deficiency increases P-ERK/LC3-II colocalization. Immunofluorescence (IF) for P-ERK (green)/LC3 (red) colocalization in scr or siATG4B NIH/3T3 cells in presence/absence of EGF (10 min). Scale bars, 10 μm. Bars represent mean±s.e.m. **P0.01, ***P0.001 compared with scr; Student's t-test, 60 cells from n=2. (f) ATG4B-deficient cells display increased nuclear P-ERK/LC3 colocalization. IF depicting P-ERK (green)/LC3 (red) colocalization in nuclei of scr or siATG4B NIH/3T3 cells in presence/absence of EGF. Scale bars, 5 μm. Bars represent mean±s.e.m. **P0.01, ***P0.001 compared with scr; Student's t-test, 60 cells from n=2. (g) ATG4B deficiency augments EGF-induced MEK and ERK phosphorylation. Immunoblots for P- and total MEK and ERK, LC3 and GAPDH in total lysates from scr or siATG4B NIH/3T3 cells in presence/absence of EGF. Bars represent mean±s.e.m., n=3. h) ATG4B deficiency increases nuclear P-ERK levels. Native (top)/inverted images (bottom) showing nuclear P-ERK content. P-ERK fluorescence in untreated (Con)/EGF-treated scr or siATG4B NIH/3T3 cells is shown. Scale bars, 5 μm. Bars represent mean±s.e.m. **P0.01, ***P0.001 compared with scr; Student's t-test, 60 cells from n=2. Nuclei are blue (DAPI). \n",
      ".Figure 4A. Total histones were acid extracted from Rev3l+/+ and Rev3l-/- MEFs. Samples were analyzed by western-blot using indicated antibodies specific to histone marks (right panel). An anti-pan H4 or H3 was used as loading control (left panel). Histograms show WT/KO Rev3l fold change relative to the immunoblot shown on the top panel (using 1X sample intensity). B. Histone mark levels were examined at selected genes repressed in Rev3l-/- MEFs (Hoxb2, Hoxb8, WT1). Histograms represent enrichment of H3K9me3 and H3K27me3 at indicated loci assessed by ChIP-qPCR in Rev3l+/+ and Rev3l-/- MEFs. Error bars indicate standard error of the mean for three independent experiments. C. H3K9me3 and H3K27me3 mark levels were evaluated at selected genes that localized in genomic loci displaying replication timing delay and downregulated in Rev3l-/- MEFs (FoxG1, GATA6). Histograms represent enrichment of H3K9me3 and H3K27me3 assessed by ChIP-qPCR at indicated loci in Rev3l+/+ and Rev3l-/- MEFs. Error bars indicate standard error of the mean for three independent experiments. D. Southern blot analysis of genomic DNA extracted from Rev3l+/+ and Rev3l-/- MEFs and digested with the CpG methylation-sensitive enzyme MaeII (5' -ACGT- 3'). The membrane was hybridized with radiolabeled probes specific to major satellites. The presented data are representative of two repeats \n",
      ": Rev3l-deficient cells show numerous genomic deletionsA.Figure 7A. Experimental design for the whole genome sequencing of Rev3l-deficient cells. Populations of mouse embryonic fibroblasts Rev3l+/+ and Rev3l-/- were isolated from Rev3l+/+ and Rev3lflox/flox littermates and Rev3l was excised using CRE recombinase. The resulting Rev3l-deficient cells and Rev3l+/+ cells were expanded for 90 doublings. Rev3l+/+ and Rev3l−/− subclones were then isolated for deep sequencing. B. Percentage of rearrangements (deletions, duplications and inversions) identified in Rev3l+/+ and Rev3l−/− subclones. C. Size distribution of the deletions detected in Rev3l+/+ and Rev3l−/− subclones. Poisson test. Deletions were counted from four independent clones. D. Distribution of the deletions detected in Rev3l−/− subclones related to the replication timing in deletions per megabase. Difference between late and early regions is statistically significant (p=0.01454, Poisson test with the correction for different length of intervals through the ratio). Data were calculated from two independent clones. \n",
      ": REV3L facilitates replication in pericentromeric heterochromatin, preventing DNA double-strand breaks.Figure 6A. Representative fibers of newly synthesized DNA labeled with IdU (red) for 30 minutes and CldU (green) for 30 minutes in HeLa cells transfected with non-targeting siRNA (siNT) or siRNA against Rev3l. SatIII probe is visualized in blue. Scale bar: 10 μm = 20kb. Relative Rev3l mRNA level normalized to GAPDH mRNA level is shown (top panel). Distribution of fork speeds (kb/min) and fork ratios (IdU/CldU track length) are shown in dot plots for SatIII and global DNA (bottom panel). The number of fibers analyzed is indicated in (n). Bars represent the median ± interquartile range (Mann-Whitney test. ns: not significant, * p<0.05; ** p<0.01, *** p< 0.001 and ****: p<0.0001). The presented data are representative of 3 biological repeats. B. Asynchronous Rev3l+/+ and Rev3l-/- MEFs were fixed with PFA then subjected to in situ proximity ligation assay (PLA) using 53BP1 and HP1α antibodies then PLA foci were counted in both cell lines (more than 150 nuclei for each condition were counted). p-values were calculated by Mann-Whitney test ( ****: p<0.0001). Red lines indicate the mean values. Error bars: SEM. Controls with a single antibody are also shown. Experiments were repeated three times. Representative images are shown. Scale bar= 5 μm. C. Asynchronous Rev3l+/+ and Rev3l-/- MEFs were subjected to PLA as by using γH2AX and HP1β antibodies and processed as in (B). p-values were calculated by Mann-Whitney test ( *: p<0.05). Red lines indicate the mean values. Error bars: SEM. Controls with a single antibody are also shown. Three independent experiments were performed. D. RPE cells were transfected with non-targeting siRNA (siNT) or siRNA against Rev3l then 72h later cells were subjected to PLA by using 53BP1 and HP1α antibodies and processed as in (B). p-values were calculated by Mann-Whitney test ( ****: p<0.0001). Red lines indicate the mean values. Error bars: SEM. Three independent experiments were performed. E. HeLa cells were transfected with non-targeting siRNA (siNT) or siRNA against Rev3l then 72 h later cells were subjected to PLA by using 53BP1 and HP1α antibodies and processed as in (B). p-values were calculated by Mann-Whitney test ( ***: p<0.001). Red lines indicate the mean values. Error bars: SEM. Three independent experiments were performed. Rev3l+/+ and Rev3l-/- MEFs were incubated with or without 0.23 μM aphidicolin for 24h before metaphase spreading. FISH was performed using major satellite probe to quantify breaks in pericentromeric regions (F). Representative images of chromosomes showing abnormalities (see arrows) in pericentromeric regions from Rev3l-/- MEFs. Chromosomes were labelled with DAPI Error bars indicate standard error of the mean for three independent experiments. Mann-Whitney test (ns: not significant, * p<0.05; *** p< 0.001). G. breaks in non-pericentromic regions were quantified (G). Error bars indicate standard error of the mean for three independent experiments. Mann-Whitney test (ns: not significant, * p<0.05; *** p< 0.001). \n",
      ": REV3L loss compromises heterochromatin-associated DSB repair.Figure 8A. Asynchronous Rev3l+/+ and Rev3l-/- MEFs were UV-irradiated at the indicated doses and harvested 6 h or 24 h later. Cell lysates were analyzed by immunoblotting with indicated antibodies. B. Asynchronous Rev3l-/- MEFs complemented with empty vector (EV) or REV3L were UV-irradiated at the indicated doses and harvested 6 h or 24 h later. Cell lysates were analyzed by immunoblotting with indicated antibodies. C Asynchronous Rev3l+/+ and Rev3l-/- MEFs were pulse labelled for 15 min using 10 μM EdU then UV-irradiated at 4 J/m2. Cells pre-treated with CSK buffer were fixed and 53BP1, RPA and RAD51 were detected by immunofluorescence. Representative images in non-irradiated cells (NI) and quantification of 53BP1 foci in EdU positive cells at different time points are shown Scale bar= 10 μm The number of cells analyzed is indicated in (n) on each graph. ***p < 0.001 (Kruskal-Wallis test). The presented data are representative of 2 repeats). Bars represent the median ± interquartile range Quantification of the intensity of chromatin-bound RPA2 (D) and RAD51 foci (E) 24 hours after UV-irradiation was performed. The number of cells analyzed is indicated in (n) on each graph. ***p < 0.001 (Kruskal-Wallis test). The presented data are representative of 2 repeats). Bars represent the median ± interquartile range F-H. Asynchronous Rev3l-/- MEFs complemented with empty vector (EV) or REV3L were processed as in (C). Quantification of 53BP1 foci in EdU positive cells at different time points is shown (F). Quantification of the intensity of chromatin-bound RPA2 (G) and RAD51 foci (H) 24 hours after UV-irradiation was performed. The number of cells analyzed is indicated in (n) on each graph. ***p < 0.001 (Kruskal-Wallis test). The presented data are representative of 2 repeats. Bars represent the median ± interquartile range. I. Model that summarizes the results obtained in this study. \n",
      ": REV3L localizes in heterochromatin through a direct interaction with HP1 dimer.Figure 5A. Asynchronous MEFs expressing Flag-tagged REV3L from the endogenous locus were lysed and REV3L was immunoprecipitated using anti-Flag (M2) antibodies. Co-immunoprecipitated proteins were analyzed by immunoblotting using the indicated antibodies. *: IgG light chain. This experiment was repeated 2-4 times. B. Asynchronous MEFs expressing Flag-tagged REV3L were subjected to in situ proximity ligation assay (PLA) to test the interactions REV3L-REV7 and REV3L-HP1α. Nuclear foci were quantified (more than 150 nuclei for each condition were counted). Reactions omitting one of the primary antibodies were used as negative controls. Horizontal bars show the mean. Mann-Whitney test, ns: not significant, *** p<0.001. Experiments were repeated three times. C. Asynchronous MEFs expressing Flag-tagged REV3L were subjected to PLA to test the interactions REV3L-H4K20me3, REV3L-H3K9me3, REV3L-H3K27me3, REV3L-H3K4me3 and REV3L-H3K9ac. Nuclear foci were quantified as in (B). Horizontal bars show the mean. Mann-Whitney test, ns: not significant, **** p<0.0001. Three independent experiments were performed. D. Schematic representation of human REV3L and truncated constructs. Conserved domains between yeast and human REV3L are in black blue (the N-terminal domain, 1-333 aa) and in green (the catalytic domain, 2276-3130 aa), the REV7 interacting domains are in red (1880-2001 aa). A large region in royal blue not found in yeast protein is almost exclusively encoded by exon 14. All the truncated constructs lack the NTD domain and were fused to GFP. MEFs cells were transiently transfected with various GFP-REV3L constructs and fixed with 4% formaldehyde. The distribution of the GFP-REV3L mutants was detected by autofluorescence and nuclei were visualized using DAPI staining. Scale bar= 5 μm E. MEFs cells were transiently transfected with GFP-REV3L761-1029 and fixed with 4% formaldehyde. The distribution of GFP- REV3L761-1029 was detected by autofluorescence (green), chromocenters were visualized by H3K9me3 immunostaining (red, top panel) or HP1α (red, bottom panel) and DNA was counterstained with DAPI. Line scans represent the colocalization of proteins within each image (right panels). Scale bar= 5 μm F. Sequence alignment of proteins containing the PxVxL motif important for an interaction with HP1, with canonical residues shown in red. G. MEFs cells were transfected with GFP-REV3L761-1029 WT PSVVL or ASAVA mutant and fixed with 4% formaldehyde. The distribution of GFP-REV3L constructs was detected by autofluorescence (green), chromocenters were visualized by HP1α immunostaining (red) and DNA was counterstained with DAPI. Scale bar= 10 μm H. Human 293T cells were transfected with either FH-REV3L700-900, mutant V802D F-H-REV3L700- 900 or empty vector. Forty-eight hours after transfection cell lysates were made and used for immunoprecipitation with HP1 antibody. Western blot was processed and membranes were immunoblotted with the indicated antibodies. I. 293T cells were co-transfected with F-H-REV3L700-900 or empty vector and V5-HP1γ, mutant IY/EE V5-HP1γ or empty vector. Forty-eight hours after transfection cell lysates were made and used for immunoprecipitation with α-V5 antibody. After electrophoresis, samples were immunoblotted with anti-HA, anti-HP1γ or anti-α-Tubulin as indicated. J. 293 cells were transfected with various GFP-REV3L constructs or empty vector (GFP). Twenty-four hours after transfection cell lysates were made and GFP-REV3L was affinity-purified on GFP-Trap beads. After electrophoresis, samples were analyzed by immunoblotting with antibodies against SCAI or HP1β as indicated. K. MEFFlag-REV3L were transfected with various GFP-SCAI constructs, empty vector (GFP) or not transfected (Ø). Twenty-four hours after transfection cell lysates were made and Flag-REV3L was affinity-purified on M2 beads. After electrophoresis, samples were analyzed by immunoblotting with antibodies against GFP or Flag (M2). *: non-specific bands \n",
      ".Figure 1A. Binding of ubiquitin chains to immobilized GST-fusion proteins. Indicated proteins were purified from E. coli and incubated with equal amounts of preformed fluorescently labeled ubiquitin chains, which were either K48- or K63-linked. The fraction of ubiquitin chains interacting with the bait protein was analyzed by SDS‑PAGE and fluorescence scan. Total fluorescence was quantified per lane and normalized to the first lane (rel. fluo.). Asterisk (*) indicates lane from the same gel that was cropped and moved. B. A structural model of Ube2K in complex with two ubiquitin molecules The UBC domain associates via its active site cysteine with the side chain of K48 in the immediate acceptor ubiquitin molecule (UbA‑prox). A second ubiquitin molecule (UbA‑dist) binds the UBA domain through the conserved hydrophobic patch. Selected atoms were highlighted as spheres: C-atom in the carboxy group (CC) of UbA‑prox-G76, S‑atom in E2 active site Cys, CC in UbA‑dist-L71, N-atoms (blue) in UbA‑prox of ε‑amino groups of lysine residues (Nε) and in the backbone of M1 (Nα). A structure of Ubc1 was superimposed onto the model Key residues were highlighted as sticks for the binding interfaces (C) between the UBC domain and UbA‑prox, Additionally, amino acids whose side chains contribute to the UBC/UBA binding interface were determined and classified into \"hot spot\" or \"contributing\" residues using the algorithm SpotOn -E. A structure of Ubc1 was superimposed onto the model Key residues were highlighted as sticks for the binding interfaces (D) between the UBA domain and UbA‑dist, Additionally, amino acids whose side chains contribute to the UBC/UBA binding interface were determined and classified into \"hot spot\" or \"contributing\" residues using the algorithm SpotOn A structure of Ubc1 was superimposed onto the model Key residues were highlighted as sticks for the binding interfaces (E) between the UBC domain and UBA domain. Additionally, amino acids whose side chains contribute to the UBC/UBA binding interface were determined and classified into \"hot spot\" or \"contributing\" residues using the algorithm SpotOn F. Interatomic distances from L71‑CC in UbA‑dist to G76-CC in UbA-dist or to putative attachment sites in UbA-prox were measured for various known ubiquitin conformations. The six closest attachment sites are listed in ascending order. \n",
      ".Figure 2A. Reaction scheme for single turnover ubiquitination experiments with Ubc1 in presence of fluorescently labeled monoubiquitin as donor and various C-terminally blocked acceptor ubiquitin molecules. B. At indicated time points, samples were taken from the reaction mix and analyzed by SDS‑PAGE and fluorescence scan. Product fluorescence over total fluorescence per lane was quantified (rel. fluo.). D. Initial reaction rates were determined for a panel of acceptor ubiquitin molecules in reactions with either full‑length Ubc1 or Ubc1-ΔUBA (aa1‑150). The binding of the UBA domain to individual ubiquitin molecules was disrupted by introducing an R42A amino acid exchange in ubiquitin, while acceptor sites were blocked by introducing the K48R substitution. Reactions were performed in triplicate. C. The relative fluorescence was plotted over time for reactions with different ubiquitin acceptor molecules - for example in presence of K63-linked diubiquitin (K63‑Ub2) or K48‑linked diubiquitin. Initial reaction rates were determined as the slope of a linear fir of the fluorescence increase over 20 min. E. Schematic transition state for the reaction of Ubc1 with K63-linked diubiquitin. Binding of the distal moiety in K63-Ub2 to the UBA domain involves R42 in ubiquitin and enhances targeting of K48 in the proximal moiety for discharge of donor ubiquitin by the UBC domain. This results in the assembly of a K48/K63-branched chain. \n",
      ".Figure 4A.-D. 15N-labeled diubiquitin probes were titrated with the UBA domain up to molar ratio of 8:1 (UBA : Ub2). NMR spectra were recorded for (A) the distal moiety in K63-Ub2, (B) the proximal moiety in K63-Ub2, (C) the distal moiety in K48-Ub2 or (D) the proximal moiety in K48-Ub2. e. CSPs of I44 were plotted over UBA domain concentration. f. KD values for binding of the UBA domain were determined from results in (E) assuming two independent but different binding sites. \n",
      ".Figure 3a. Residues associated with the binding of the UBA domain to the UBC domain were analyzed through sequence alignment of Ubc1 with other E2 enzymes from S. cerevisiae. Y76 and W144 in Ubc1 were predicted to be hot spot residues of the interface, while L143 and L147 were classified as contributing amino acids b. L143E and/or L147E amino acid substitutions were introduced into Ubc1 to disrupt the hydrophobic UBC/UBA domain interface. Single turnover ubiquitination experiments were performed with these Ubc1 variants in presence of K63‑Ub2 with Ub(K48R) at the distal position or monoubiquitin as acceptor. Experiment and initial rate determination c. Ubc1 variants were generated with deletions in the linker region (aa152‑167), which connects the UBA domain and the UBC domain. Single turnover ubiquitination reaction mixes were incubated for 5 min with K63-Ub2 or for 20 min with monoubiquitin as acceptor. Final product formation was analyzed by SDS-PAGE and fluorescence scan. Product fluorescence over total fluorescence (rel. fluo.) was quantified. d. C-terminally blocked M1-linked diubiquitin with or without a K48R amino acid substitution at the distal or proximal position were employed in single turnover experiments with full-length Ubc1 or Ubc1-ΔUBA. Reactions with acceptor K63-Ub2 were carried out controls. Initial rate determination \n",
      ".Figure 5a. The activity of Ubc1 was compared with the K48-linked chain building enzyme Ubc7/Cue1 in single turnover ubiquitination experiments in presence of either monoubiquitin, K48‑Ub2 or K63-Ub2. Initial rates were determined b. The K63-linked chain building enzyme Ubc13 with its cofactor Uev1a or the Ubc1 homologue Ube2K (h. sapiens) were employed in single turnover ubiquitination experiments in presence of K63-Ub2 with Ub(K48R) at the proximal or distal position or in presence of K48‑Ub2. Initial rate determination c. Experiment as in (B) but with the Ubc1 homologue ubc-20 (c. elegans). \n",
      ".Figure 6A. Cartoon representation of reaction conditions used in (B-D), which simultaneously enable de novo chain assembly as well as the formation of K48/K63-branched chains. B. In vitro ubiquitination reactions with full‑length Ubc1 or Ubc1-ΔUBA were performed. Samples were taken at indicated time points and analyzed by SDS-PAGE and fluorescence scan. Free chain synthesis from Alexa488-labeled monoubiquitin was observed over time. Fluorescence of all reaction products was quantified and normalized to number of fluorophores per product (\"de novo\"). Double cross (‡) indicates ubiquitinated E2 enzyme. C. Reactions as in (B) were performed in presence of C‑terminally capped and Alexa647-labeled K63‑linked Ub3. Monoubiquitin and K63-Ub3 were used in a 3:1 molar ratio. Analysis as in (B). Modification of K63-Ub3 was identified through correlation of fluorescence channels and their fluorescence quantified in the 488 channel (\"branched\"). D. Alexa488-labeled monoubiquitin and C-terminally capped K63-Ub4 were used in a 4:1 molar ratio for in vitro ubiquitination reactions with full‑length Ubc1 or Ubc1-ΔUBA. Where indicated, Alexa488-labeled monoubiquitin was methylated prior to the reaction (met-Ub) to block de novo chain assembly. Reactions were incubated overnight, quenched and half the reaction mix from each condition was treated with the K63 specific deubiquitinating enzyme AMSH (lanes 4-6). Samples were analyzed by SDS-PAGE and fluorescence scan. \n",
      ".Figure 7a. A flow-chart showing the sample preparation for the SILAC-based relative quantification of K48/K63-branched ubiquitin chains from yeast by mass spectrometry. 10xHis-Ub(R54A) was overexpressed in wildtype cells (wt) and either in cells deleted for UBC1 (Δubc1) or cells expressing ubiquitin-binding-deficient Ubc1-EA. The R54A substitution in ubiquitin enables the production of a marker peptide for K48/K63-branched ubiquitin by trypsin digest, which can be detected by targeted proteomics (PRM). B. Log2 transformed fold changes of peptides between the Δubc1 strain and wildtype were determined. The forward and reverse SILAC experiments are shown on the x or y axis, respectively. The different ubiquitin marker peptides are indicated in the figure. C. Experiment as in (B) but for the UBC1-EA strain and wildtype. D. A flow-chart depicting the sample preparation for a SILAC-based proteomic shotgun experiment aimed to explore changes in protein abundance in Ubc1-EA expressing yeast cells. E. Log2 transformed fold changes of proteins between the UBC1-EA strain and wildtype were determined. The forward and reverse SILAC experiments are shown on the x or y axis, respectively. Genes, which were determined to be significantly different in their protein abundance, are marked in red and labeled. Asterisks (*) indicate genes, which are upregulated at a transcriptional and/or translational level upon heat stress A double cross (‡) marks genes, for which the protein product changes in abundance and/or localization in response to DNA replication stress F. Ubc1-EA was genomically integrated into various yeast strains. Exponentially growing cells of the indicated genotype were spotted onto YPD plates in serial 10‑fold dilutions. Where noted cells were incubated at elevated temperature (37 °C) or in presence of hydroxyurea (HU) to induce DNA replication stress. \n",
      "FT cells to apoptosis induction by stimulation with PMA.Figure 3A 293FT-TetR-3xFlag-Usp27x or 3xFlag-Usp27xC87A, a polyclonal derivative where the Bim-locus had been targeted by CRISPR/Cas9- and four single clones obtained by serial dilution from the polyclonal line were treated as indicated (PMA [16.2 nM], QVD [10 µM]). Apoptosis was measured after 24 h of treatment by staining for active caspase-3, followed by flow cytometric analysis. Data (means/SEM) are from n=14 (3xFlag-Usp27x line and 3xFlag-Usp27xBim2KO polyclonal line), n=5 (3xFlag-Usp27xC87A, and 3xFlag-Usp27xBim2KO clone #14), n=7 (3xFlag-Usp27xBim2KO clone #11) or n=4 (3xFlag-Usp27xBim2KO clone #4 and #7) separate experiments. P-values (t-test) for statistically significant differences are shown. B The same cells as in A (except Usp27xC87A mutant) were treated as in A and stained for active Bax. Data (means/SEM) are from n=8 (3xFlag-Usp27x line and 3xFlag-Usp27xBim2KO polyclonal line), n=5 (3xFlag-Usp27xBim2KO clone #11 and #14) or n=4 (3xFlag-Usp27xBim2KO clone #4 and #7) separate experiments. P-values (t-test) for statistically significant differences are shown. Expression levels of Bim, β-TrCP and Flag-Usp27x of the different cell lines are shown in Appendix Fig. S5E. \n",
      "x stabilizes the expression of the phosphorylated form of BimEL and reduces Bim-ubiquitination in PMA-stimulated 293FT cells.Figure 2A Usp27x inhibits the PMA-induced degradation of BimEL in 293FT cells. 293FT-TetR-3xFlagUsp27x or TetR-3xFlag-Usp27xC87A cells were treated with dox to induce 3xFlag-Usp27x or Usp27xC87A, or with PMA [16.2 nM] (to induce BimEL degradation), or with the combination of dox, PMA and QVD [10µM] (to inhibit apoptosis) as indicated for 24 h (left) or 48 h (right). Cell lysates were analysed by Western blotting. The blots are representative of n=4 similar experiments. See also Appendix Fig. S5B. * Probably modified form of wild-type 3xFlag-Usp27x (such as dimer, ubiquitinated Usp27x or a stable complex of Usp27x with another interacting partner); this form is only seen if high expression levels of catalytically active Usp27x are reached (see Appendix Fig. S5A where Usp27x was transfected into 293FT cells) or if wt Usp27x is enriched during immunoprecipitation.** Probably degradation product of 3xFlag-Usp27x. These forms of Usp27x were not seen in every experiment. B 293FT-TetR-3xFlagUsp27x or TetR-3xFlag-Usp27xC87A cells treated as in A were analysed using conditions of higher resolution. A band very likely corresponding to phosphorylated BimEL is detectable. Similar results were obtained in n=2 separate experiments. C Usp27x stabilizes expression of p-BimEL (Ser69) in Bim-degrading conditions without reducing β-TrCP levels293FT-TetR-3xFlagUsp27x were treated with dox, PMA [16.2 nM], UO126 [10µM] or QVD [10µM] as indicated for 24h, and levels of phosphorylated BimEL at serine69 were analysed by Western blotting using a phospho-Bim (Ser69) specific antibody. Scans are from the same membrane and exposure times. Similar results were obtained in n=2 separate experiments. D UO126 reduces phosphorylation-associated shift of BimEL293FT-TetR-3xFlagUsp27x were treated for 24h with the indicated drug combinations and Bim was detected by Western blotting. UO126 was added 30 minutes prior other stimulation. Similar results were obtained in n=3 separate experiments (see Appendix Fig. S5D). E Usp27x-expression reduces ubiquitination of Bim293FT-TetR-3xFlagUsp27x cells were transfected with a vector coding for 6His-ubiquitin (left) or GFP (right). After 24h cells were treated with PMA and QVD, and 3xFlag-Usp27x was induced by dox at the same time. After additional 20h cells were treated for 4h with MG132 (40 µM) to block proteasomal degradation of ubiquitinated proteins. Cells were lysed under denaturing conditions and His-ubiquitin-labelled proteins or proteins from only GFP expressing cells were purified by Ni2+-NTAaffinity chromatography. Ubiquitinated Bim (Bim-Ubn) was detected by Western blot using anti-Bim antibodies (left). Similar results were obtained in n=3 separate experiments (another experiment including a Coomassie stain is shown in Appendix Fig. S3E). *Indicates non-His-ubiquitinated BimEL that was bound unspecifically to the Ni2+-agarose beads \n",
      "x interacts with BimEL.Figure 1A 293FT cells were transfected with 3xFlag-Usp27x (pFCMV7.1 vector backbone) together with a construct driving expression of untagged BimEL or 3xHA-BimEL(both pMIG vector backbone). Cells were lysed, and 3xHA-BimEL was immunoprecipitated with anti-HA-antibodies. IP products were tested by Western blotting for the presence of Bim and FLAG-Usp27x probing with antibodies against Bim or against the FLAG-peptide. See also Appendix Fig. S1B. Western blots show representative of n≥3 independent experiments. B Usp27x binds a mutant of Bim incapable of binding to anti-apoptotic Bcl-2-proteins. 293FT cells transfected with constructs encoding 3xFLAG-Usp27x and 3xHA-tagged BimEL (see A, 2µg each) or 3xHA-tagged BimEL∆∆(a mutant with two mutations in the BH3-domain, incapable of binding antiapoptotic Bcl-2proteins [50]) were immunoprecipitated from whole cell extracts using anti-HA-resin. Bim and Usp27x were detected with anti-HA- and anti-FLAG-antibodies as indicated; antiapoptotic proteins: Mcl-1, Bcl-XL, Bcl-2. The caspase-inhibitor Q-VD-OPh (QVD) was added to the cultures described in A and B to inhibit Bim-induced apoptosis. Western blots are representative of n=3 independent experiments. C Usp27x interacts with endogenous BimEL independently of its catalytic activity293FT cells either carrying 3xFlag-Usp27x (293FT-TetR-3xFlag-usp27x) or the catalytic inactive mutant 3xFlag-Usp27xC87A under the control of the Tet-repressor were treated for 24h with dox to induce expression of Usp27x or Usp27xC87A. In all conditions PMA (to induce Bim-ubiquitination, 16.2 nM), and Q-VD-OPh (to inhibit apoptosis, 10 µM, see Fig. 3) were added at the time of Usp27x-induction. MG132 (to prevent Bim-degradation, 40 µM) was added in all conditions 4h prior to cell lysis. 3xFlag tagged Usp27x or Usp27xC87A was immunoprecipitated from whole cell lysates using anti-Flag resin. Interaction with BimEL or β-TrCP was detected by Western blotting using anti-Bim or anti-β-TrCP antibodies. Western blots are representative of n≥3 independent experiments. D Usp27x expression does not inhibit interaction of BimEL to β-TrCP293FT-TetR-3xFlag-Usp27x cells were transfected with pMIG-3xHA-BimEL in the presence of both PMA and QVD. At the same time dox (to induce 3xFlag-Usp27x) was added as indicated and 20h later cells were treated with MG132 [40µM] for additional 4h.Cells were lysed and 3xHA-BimEL was immunoprecipitated as described above. As a control HA-matrix was used without lysate to rule out any unspecific signal that might come from the immobilized anti HA-antibody (beads). Western blots show representative of n=2 independent experiments. E Binding of Usp27x to BimEL can be blocked by the MEK-inhibitor UO126293FT-TetR-3xFlag-Usp27x cells (see C) were treated for 24h with dox to induce expression of Usp27x. In all conditions PMA [16.2 nM], and Q-VD-OPh [10 µM] were added at the time of Usp27x-induction. As a control, cells were pretreated with UO126 [10µM] for 30 minutes to block the PMA stimulated ERK pathway before the addition of dox plus PMA plus QVD. 3xFlag tagged Usp27x was immuno-precipitated from whole cell lysates using anti-Flag resin. Interaction with BimEL was detected by Western blotting using anti-Bim antibodies. As a control Flag-matrix (beads) was used alongside the IP under same conditions but without addition of protein lysates. Western blots are representative of n=4 independent experiments (see also Appendix Fig. S2B). F Binding of Usp27xto BimEL does not require β-TrCP293FT-TetR-3xFlag-Usp27x cells were transfected with control siRNA (siCo3) or siRNA specific for β-TrCP. 48h later cells were stimulated with PMA plus dox plus QVD for additional 24h. After cell lysis, Flag-Usp27x was immuno-precipitated using anti-Flag-matrix, and BimEL and β-TrCP bound to Usp27x were identified by Western blotting. Blots are representative of n=2 independent experiments. G Usp27x binds specifically to BimEL293FT-TetR-3xFlag-Usp27x cells or 293FT-3xFlag-Usp27x cells with a specific deletion of the BimELprotein were treated with dox, PMA and QVD as in C. Cells were lysed, and lysates were subjected to anti-Flagimmunoprecipitation. 3xFlag-Usp27x was detected using anti-FLAG antibodies. Blots are representative of n=3 independent experiments. \n",
      "FT cells.Figure 5A Loss of endogenous Usp27x enhances destabilization of BimEL in response to BRAF-V600ECaco2 cells carrying inducible HA-BRAF-V600E were transfected with control siRNA (siCo3) or siRNA directed against Usp27x (combination of three siRNAs targeting Usp27xmRNA) for 24h prior to BRAF-V600E expression with dox for 2h (left) or 4h (right). BimEL was detected by Western blotting. Similar results were also observed in n=2 more experiments after 3h induction (not shown, see Appendix Fig. S6E for siRNA efficacy). B 293FT cells deficient for Usp27x show enhanced destabilization of BimEL in response to PMA293FT wt cells or 293FT-Usp27xKO (clone 2/10, generated using CRISPR-Cas-9) were treated with PMA [16.2 nM] for 16h or 24h to induce BimEL degradation. Western blots show protein levels of n=3 independently performed experiments (similar results were obtained in one more experiment, not shown). See Appendix Fig. S6G for verification of the Usp27xKO cell line by DNA sequence analysis. \n",
      " can counter Bim destabilization through the Raf-ERK pathway .Figure 4A Lines based on the human melanoma cell line 1205Lu (BRAF-V600E-positive) were made to carry GFP-Usp27x, GFP-Usp27xC87A or GFP-Usp22 under the control of a doxycycline (dox)-inducible promoter. GFP-Usp27x, GFP-Usp27xC87A or GFP-Usp22 were induced with dox for 24 or 48h. Levels of Bim were determined by Western blotting. The two left panels are from the same membrane and exposure times. Similar results were obtained in n=4 separate experiments (left) and n=3 experiments (right). See also Appendix Fig. S6A where 3xFlag-Usp22 is also shown to be unable to increase Bim levels. B A cell line derived from the BRAF-V600E-positive human melanoma line WM1158 was generated to carry inducible GFP-Usp27x. GFP-Usp27x was induced, and Bim was detected as in A. Similar results were obtained in n=3 separate experiments. C Derivatives of the NSCLC cell line HCC827 (expressing constitutively active EGFR) were generated that only express the TetR-repressor or carrying in addition either GFP-Usp27x or GFP-Usp27xC87A. Cells were treated with dox for 72h. Bim was detected by Western blotting. Similar results were obtained in n=2 separate experiments. GFP-Usp27x, GFP-Usp27xC87A, or GFP-Usp22 expression (A-C) was detected using an antibody against GFP. D Caco2 cells carrying a dox inducible HA-BRAF-V600E [37] (left two lanes) or the same cells stably expressing 3xFlag-Usp27x were treated with dox for 48h to induce expression of HA-BRAF-V600E. Bim was detected by Western blotting. The amount of BimEL was quantified from the shown immunoblot using the expression levels of the uninduced control cells set to 100% (normalized to the tubulin signal for each condition). Similar results were obtained in n=3 separate experiments with varying induction times. See also Appendix Fig. S6D. \n",
      "x stabilizes Bim protein levels in the ERK-degradation pathway in 293FT cells, 1205Lu melanoma and HCC827 NSCLC cells.Figure 6A 293FT-TetR-3xFlagUsp27x cells were treated with PMA [16.2 nM] plus QVD [10 µM] and 3xFlag-Usp27x was induced by dox at the same time for 24h followed by addition of cycloheximide (CHX). Cells were harvested at the indicated time points, and levels of BimEL were determined by Western blotting. Similar results were obtained in n=3 separate experiments. Cut membranes shown were from one membrane with same exposure time. For each condition BimEL-levels were quantified and normalized to the tubulin-signal. Percent BimEL gives the expression relative to the starting point. B 1205Lu melanoma cells carrying dox-inducible GFP-Usp27x were treated with dox for 24h. Cycloheximide (CHX, 1µg/ml) was then added. Cells were harvested at the indicated time points, and levels of BimEL were determined by Western blotting. GFP-Usp27x was detected with anti-GFP-antibodies. Similar results were obtained in n=3 separate experiments. A second experiment is shown in the right panel also including 1205Lu melanoma carrying dox-inducible GFP-Usp27xC87A mutant. For each condition BimEL-levels were quantified and normalized to the tubulin-signal. Percent BimEL gives the expression relative to the starting point set to 100%. The starting point for the non-Usp27x/C87A inducing cells (-dox) is shown in lane 1 for experiment one and two (-dox, - CHX). Cut membranes shown were from one membrane with same exposure time. Same results were obtained for WM1158 melanoma cells (data not shown). C HCC827 cells were induced to express GFP-Usp27x with dox as indicated. 24h later cycloheximide (CHX, 1µg/ml) was added for the indicated time. BimEL was detected after the indicated times of treatment by Western blotting. For each condition BimEL-levels were quantified and normalized to the tubulin-signal. Percent BimEL gives the expression relative to the starting point. Similar results were obtained in n=2 separate experiments. Cut membranes shown were from one membrane with same exposure time. \n",
      " NSCLC cells to apoptosis induction by inhibition of the Raf-ERK-pathway.Figure 7A 1205Lu melanoma cells carrying dox-inducible GFP, GFP-Usp27x or GFP-Usp27xC87A were treated with doxycycline (dox) as indicated. After 48 hUO126 (10µM) was added for another 48 h. Apoptosis was measured by staining for active caspase-3, followed by flow cytometric analysis. Data (means/SEM) are from n=3 (GFP) or from n≥6 separate experiments (GFP-Usp27x and GFP-Usp27xC87A). P-values (t-test) for statistically significant differences are shown. N.s., p>0.05. The addition of QVD inhibited cells from active Caspase-3 positive staining (not shown). A stain for active Bax is shown in Appendix Fig. S7B. B HCC827 NSCLC cells carrying dox-inducible GFP-Usp27xC87A, GFP-Usp27x, or two separate lines established from these GFP-Usp27x-cells where the Bim-locus had been targeted for deletion using CRISPR/Cas9 (Bim-KO), were treated with combinations of dox and the EGFR-inhibitor gefitinib as indicated. Apoptosis was measured after 72h of treatment by staining for active caspase-3, followed by flow cytometric analysis. Data (means/SEM) are from n=≥15 (maternal GFP-Usp27x line) or from n≥5 (for Bim1KO) or from n≥6 (for Bim2KO), or n=4 (for GFP-Usp27xC87A) separate experiments. P-values (t-test) for statistically significant differences are shown. Again, the addition of QVD inhibited cells from active Caspase-3 positive staining (not shown). The inset shows Bim-knock-out-efficiency (n=2). \n",
      "x reduces gefitinib-induced apoptosis in HCC827 NSCLC.Figure 8Figure 8 Loss of Usp27xreduces gefitinib-induced apoptosis in HCC827 NSCLCHCC827 NSCLC cells and two separate polyclonal lines established from these cells where the Usp27x-locus had been targeted for deletion using CRISPR/Cas9 with two different guide RNA against Usp27x (Usp27xKO-1 and Usp27xKO-4), were treated with gefitinib [10µM] for 48h. Apoptosis was measured by staining for active caspase-3 (A) or by staining for activated Bax (antibody: 6A7 clone) followed by flow cytometric analysis. Data shown in A, B (means/SEM) are from n=4 separate experiments. P-values (t-test) for statistically significant differences are shown. \n",
      ".Figure 1A) Representative images of HeLa cells treated with Control, H2A.Z.1 or H2A.Z.2 siRNA for 72 h, fixed, and stained with DAPI. White arrows point at micronuclei. Scale bar: 10 μm. B) Quantification of the percentage of cells with micronuclei from experiment in (A). Three biological replicates were analysed for each condition (Control si N=1271; H2A.Z.1 si N=1091; H2A.Z.2#1 si N=1188; H2A.Z.2#2 si N=1193). The error bars represent the SD. Data sets were statistically analysed using Chi-square test. ***=p<0.001. C) Quantification of the percentage of cells with micronuclei from HeLa cells co-transfected with H2A.Z.2 siRNA and GFP:H2A.Z.2.1WT with either single or double mutations. Error bars represent the SD of three biological replicates (Control si N=1316; H2A.Z.2 si N=1877; H2A.Z.2 si + wt N=1973; H2A.Z.2 si + A14T N=618; H2A.Z.2 si + S38T N=558; H2A.Z.2 si + A127V N=649; H2A.Z.2 si + A14T/S38T N=1090; H2A.Z.2 si + A14T/A127V N=1038; H2A.Z.2 si + S38T/A127V N=1150). Data sets were statistically analysed using Chi-square test. ***=p<0.001; black stars refer to the comparison with the H2A.Z.2si data and blue stars refer to the comparison with the H2A.Z.2si + wt data; no stars = non significant. D) Schemes of the GFP constructs used for the rescue experiments in (E) and (F). Green boxes represent the GFP, brown boxes represent the H2A.Z.2.1 isoform. Solid fill represents the WT construct whereas striped box represents the KR mutant form where the residues indicate the mutations performed. Dots represent the possible post-translational modifications in the mutated amino acids: acetylation (ac, dark blue), methylation (me, pink) and ubiquitination (ub, light blue). E) Representative images of prometaphase chromosomes from HeLa cells co-transfected with H2A.Z.2 siRNA and each of the constructs in D (green). Scale bar: 5μm. F) Quantification of the percentage of cells with micronuclei from experiment (E). The error bars represent the SD of three biological replicates (Control si N=887; H2A.Z.2 si N=1005; H2A.Z.2 si + wt N=760; H2A.Z.2 si + KR N=479). Data sets were statistically analysed using Chi-square test. \n",
      ".Figure 3A) Gene expression of three biological replicates of HeLa cells transfected with Control, H2A.Z.1 or H2A.Z.2 siRNA was analysed by RNA sequencing. Venn diagram shows the number of significant (p<0.01) differentially expressed genes and the overlap between each set of genes. B) Pie charts displaying the percentages of H2A.Z peaks at or nearby transcription start sites (TSS) (dark blue), within the gene body (grey) or elsewhere in the whole genome (purple) for the differentially expressed genes following H2A.Z.1 or H2A.Z.2 depletion. A 2-sample test for equality of proportions was used for the statistical analyses. *=p<0.05; ns=not significant. C) Downregulated genes following H2A.Z.1 depletion were analysed by STRING. The image shows the cluster of genes with gene onthology (GO) terms related to cell cycle. PPI=Protein-protein interaction. D) IGV analyses of H2A.Z localisation (from ENCODE) on Aurora B (green) and c-MYC (blue) genes showing H2A.Z enrichment at the TSS and at the upstream region respectively (bottom panels). IGV analyses of ATAC seq peaks for Control si and H2A.Z.1 si for Aurora B and MYC genes. The number in red represents the log2Ratio H2AZ.1 si/Control si. E) Plot representing the average value for all the target regions around the TSS obtained by ATAC seq after Control, H2A.Z.1 or H2A.Z.2 siRNAi. F) Plot representing the number of differential chromatin accessibility regions in pairwise comparison. Green (up) represents regions with increased accessibility and red (down) with decreased accessibility. G) Distribution of the position within the nucleus of the centromere of Chr17 from the experiment in (H) calculated as the ratio d2/d1. The ratio between d2 and d1 gives the position of the centromere relative to the center of the nucleus. The graph represents the percentages of centromeres with distances following within the 5 binning categories. At least 500 nuclei were analysed per condition. Data sets were statistically analysed using Chi-square test for the distribution of signals among the 5 categories. ***=p<0.001. H) Representative image of a HeLa nucleus after FISH with Chr17 centromeric probe (green). The distance of the FISH signals from the nucleus periphery was calculated as follows: the distance from the centre of the nucleus to the periphery (d1) and the distance from the centre of the nucleus to the FISH signal (d2) were measured. (scale bar 5 μm). \n",
      ".Figure 2A) Quantification of micronuclei containing 0, 1 or 2 CENP-A signals in HeLa YFP:CENP-A cells transfected with control and H2A.Z.2 siRNA for 72 h. 160 micronuclei were analysed. Error bars indicate SD of three biological replicates. The image represents an example of a micronucleus containing a single CENP-A signal. The insets are enlargements of the MN. Scale bar 5 μm. B) Representative images of metaphases from YFP:CENP-A (green) HeLa cells treated with Control (top) or H2A.Z.2 (bottom) siRNA. The white arrowheads indicate the mis-aligned chromosomes. Scale bar: 10 μm. C) Representative images of metaphase spreads from Control (top) or H2A.Z.2 (bottom) siRNA-transfected HeLa cells after FISH with a Chr17 centromeric probe (green). The arrowheads indicate the separated sister chromatids. Scale bar: 10 μm. D) Representative images of HeLa cells transfected with Control (top) or H2A.Z.2 (bottom) siRNA, fixed, and stained for Sgo1 (green). Scale bar: 10 μm. E) Quantification of Sgo1 localisation in pro-metaphase cells from the experiment in (D). Error bar represents SD of two biological replicates. 35 pro-metaphase cells were analysed. F) Representative images of HeLa cells treated as in (D) and stained for Aurora B (green) and alpha tubulin (grey). Scale bar: 10 μm. G) Mitotic index of HeLa cells transfected with Control (grey), H2A.Z.1 (blue) or H2A.Z.2 (red) siRNA and treated with nocodazole for 0, 2, 4 or 6 h. At least 800 cells were analysed for each category. Error bar represents SD of three biological replicates. **=p<0.01; ***=p<0.001 ns=not significant (Chi square test). H) Left panels: Representative images of HeLa cells treated as in (D) and stained for HP1α (green). Scale bar: 10 μm. Right panels: Representative images of HeLa cells treated as in (D) and stained for h3T3ph (green). Scale bar 5 μm I) Violin plot of centromeric H3T3ph intensity of pro-metaphase/metaphase cells from the experiment in (H). The median is shown as bar. Data sets were statistically analysed using the Wilcoxon rank test in R. **=p<0.01. J) Representative images of HeLa YFP:CENP-A (green) mitotic cells after Control (top) or H2A.Z.2 (bottom) siRNA treatment. Scale bar: 5 μm. K) Violin plot of centromeric CENP-A intensity of pro-metaphase/metaphase cells from the experiments in J. (Control si N=197 H2A.Z.2 si N=260). The bar represents the median. Data sets were statistically analysed using the Wilcoxon rank test in R. **=p<0.01. L) Representative images of HeLa mitotic cells stained for CENP-C after Control (top) or H2A.Z.2 (bottom) siRNA treatment. Scale bar: 5 μm. M) Violin plot of centromeric CENP-C intensity of pro-metaphase/metaphase cells from the experiments in L. (Control si N=1083 H2A.Z.2 si N=686, from 3 biological replicas). The bars represent the median. Data sets were statistically analysed using the Wilcoxon rank test in R. ***=p<0.0001. \n",
      ".Figure 5A) Mitotic index (number of mitotic cells/mitotic + interphase cells) of HeLa cells after Control si, H2A.Z.1 si or H2A.Z.1 si + GFP:cMYC . Error bars represent standard deviation from 2 biological replica (Control si N= 1033; H2A.Z.1 si N=1042; H2A.Z.1 si + GFP:cMYC N=1024). Data sets were statistically analysed using Chi-square test. *=p0.05 **=p<0.01; ***=p<0.001. B) Correlation of H2AFZ (H2A.Z.1) expression levels with MYCN status, cancer stage and risk incidence in neuroblastoma patients obtained from R2 genomics. C) Kaplan Meier survival curve for neuroblastoma patients with low (red) or high (blue) expression of H2AFZ (H2A.Z.1) obtained from R2 genomics. \n",
      ".Figure 4A) Flow cytometry analysis profiles of control (top) and H2A.Z.1 (bottom) siRNA-treated HeLa cells. Percentages represent the mean of two biological replicates. Data sets were statistically analysed using Chi-square test. ***=p<0.001. B) CDKN1A (top) and CDKN1B (bottom) average expression values obtained by RNA sequencing of three biological replicates after Control, H2A.Z.1 (Z1) or H2A.Z.2 (Z2) siRNA treatment. Error bars show the standard deviation (SD). ***=p<0.001 (Student's t Test). C) GSEA analyses of the upregulated (light blue upwards arrow) and downregulated (light green downwards arrow) genes upon H2A.Z.1 siRNA indicating the number of genes for category and the relative p values. D) Representative images of HeLa cells transfected with control, H2A.Z.1 or H2A.Z.2 siRNA and stained for senescence-associated β-galactosidase (blue) Sclae bar 200μm. At the bottom-left, quantification of the percentage of β-galactosidase-positive cells. At least 500 cells were counted for each experiment. Error bars show the standard deviation (SD) from two biological replicates. **=p<0.01; ***=p<0.001 (Student's t Test). E) Representative images of Hela cell stained for Ki67. The arrows indicate the different staining patterns quantified: strong signal (grey arrow), weak signal (light brown arrow) and no signal (dark brown arrow (scale bar 10μm). F) Quantification of the experiment in (E). Error bars indicate the standard deviation from 3 biological replicates Data sets were statistically analysed using a Chi square test. ***=p<0.001. G) Representative images of HeLa cells transfected with control (top) or H2A.Z.1 (bottom) siRNA and stained for Lamin A/C (green). Scale bar = 10μm. H) Violin plots of the nuclear circularity index of HeLa cells transfected with control (grey), H2A.Z.1 (blue) or H2A.Z.2 (red) siRNA. The nuclear circularity was analysed using the NIS Elements AR analysis software (NIKON). At least 140 nuclei from three biological replicates were analysed for each condition. Mean and SD are shown. Data sets were statistically analysed using the Wilcoxon rank test in R. ***=p<0.001. I) Violin plots of the nuclear circularity index of HeLa cells transfected with Control (grey) or H2A.Z.1 (blue) siRNA either alone or in combination with siRNA-resistant GFP:H2A.Z.1WT (WT) or GFP:H2A.Z.1KR (KR) plasmids. The nuclear circularity was analysed using the NIS Elements AR analysis software (NIKON). At least 140 nuclei from three biological replicates were analysed for each condition. Mean and SD are shown. Data sets were statistically analysed using the Wilcoxon rank test in R. ***=p<0.001. \n",
      ".Figure 6Left panel) H2A.Z.1 promotes the expression of several cell cycle genes whilst repressing the G1/M checkpoint proteins p21 and p27 thus favouring cell cycle progression. Upon downregulation of H2A.Z.1, the expression of these genes is altered leading to G1/S cell cycle arrest and cellular senescence. Right panel) H2A.Z.2, through its post-translational modifications, regulates the recruitment of several centromeric proteins and contributes to the regulation of the spindle assembly checkpoint. H2A.Z.2 depletion leads to a defective SAC, premature sister chromatid separation and chromosome segregation errors. \n",
      ".Figure 1The syntheses of the ligands PR-924, 9, 14, 16, 17 and 18 have been described previously (de Bruin et al, 2014). All ligands share the peptide scaffold shown in the upper left corner with P1, P2 and P3 side chains and an N-cap; they are equipped with a l-Ala (stereochemistry (S)) or d-Ala (stereochemistry (R)) residue at P3 (red) and either an N-acylmorpholine or a 3-methyl-1H-indene group (green). Furthermore, compounds differ in their P2 residue (either methoxytyrosine or tryptophane; blue). Inhibitor 9, also termed LU-015i, is endowed with an exceptional cyclohexyl residue (gray), which is associated with increased selectivity for human β5i. \n",
      ".Figure 2A. Inhibition of human β5c and β5i by ONX 0914 and PR-924. IC50 values are deduced from the fitted data. B. IC50 values [µM] for all compounds towards the yeast, human and mouse β5 subunits. \n",
      ".Figure 3A. Schematic representation of yeast and mammalian β5 subunits and their propeptides. Secondary structure elements, helices (H) and sheets (S), are numbered. The full length human hβ5i (yellow) and β5c (red) as well as the mouse mβ5i (green) subunits cannot substitute the endogenous yeast yβ5 subunit (gray). Expression of the mammalian subunits with their natural propeptide (pp; colored) or with the yβ5 propeptide (pp; gray) is lethal in a yeastpre2Δ knock-out strain (for details see Appendix Supplementary Methods). B. Chimeric β5 subunits are schematically illustrated according to panel A. The N-terminal part of the subunits, which forms the primed and non-primed substrate binding pockets and the active site surroundings, was taken from humanβ5i and the C-terminus from yeast. A construct featuring residues 1-173 from humanβ5i was still lethal. Replacement of the residues 1-139 of yβ5 by the humanβ5i counterpart (aa 1-138; see Appendix Supplementary Note) however was compatible with yeastsurvival. Based on this observation, analogous fragments of the humanβ5c and mouseβ5i subunits were incorporated into the yβ5 subunit to create the respective hβ5c and mβ5i chimeras. C. Subunit β6 significantly contributes to the S3 and S4 pockets of the ChT-L active site. In order to mimic the entire mammalian β5/6 substrate binding channel in yeast we mutated five residues of the yβ6 subunit as well (G89S, K90R, H98Y, T99N and E120Q). These mutations correspond to both the human and the mouse β6 sequence and create two mammalian segments aa 87-101 and aa 108-121 (blue; see also Appendix Supplementary Note). D. The mouse specific point mutation V31M was inserted into the hβ5i/hβ6 chimera to investigate its impact on inhibitor binding. E. Serial dilutions of yeast cells were spotted on YPD plates and grown for 2 days either at 30 °C or 37 °C. Mutant yeasts are significantly retarded in growth and show increased temperature sensitivity. F. Surface illustration of the yCP. The β5 and β6 subunits are depicted in dark gray, with the residue ranges that correspond to mammalian β5 and β6 subunits in the chimeric proteasomes highlighted in gold and blue, respectively. The β5' and β6' subunits of the second β-ring are hidden on the back of 20S proteasome. \n",
      ".Figure 4A. Superposition of the β5i/β6 active sites of genuine mouse iCP and the hβ5i/hβ6 human/yeast chimera illustrates that Met45 adopts the same 'open' conformation in both structures. Amino acid side chains important for substrate binding are depicted as sticks. The catalytic Thr1 is colored in black. Met31, the single amino acid difference between mouse and human β5i active sites, is highlighted in green. B. Superposition of the genuine mouse and the human chimeric β5c/β6 active sites according to panel A proves the proper orientation of Met45 in the 'closed' conformation. C. Superposition of the β5c/β6 and β5i/β6 active sites of the natural mouse cCP and iCP according to panel A. Note the distinct conformations of Met45 (Huber et al, 2012). D. Superposition of the hβ5c/hβ6 and the hβ5i/hβ6 chimeric proteasomes confirm the distinct positions of Met45 analogously to the natural mouse subunits shown in panel C. \n",
      ".Figure 5A. Complex structures of the human β5c/β6 (left) and β5i/β6 (right) chimeric proteasomes with PR-924. The ligand binds in a linear (l) manner to the β5c/β6 substrate binding channel, but in a kinked (k) conformation to the β5i/β6 active site. The 2FO-FC electron density maps for the ligands bound to Thr1 of the chimeric β5 active sites are shown as blue meshes contoured at 1σ. The inhibitor and Thr1 have been omitted for phasing. B. Comparison of the conformations of ONX 0914, PR-924, 14, 16, 17 and 18 bound to the chimeric hβ5/hβ6 substrate binding channels. The l amino acid compounds ONX 0914 and 14 display a linear (l) binding mode to the hβ5c/hβ6 (left) and the hβ5i/hβ6 (right) chimeras, whereas the ligands 16 and 17 (P3-d-Ala and morpholine cap) adopt a kinked (k) conformation in both β5 substrate binding channels. Remarkably, the orientation of PR-924 and its analogue 18 (P3-d-Ala and 3-methyl-1H-indene cap) in the hβ5c/hβ6 chimera (and in WT yβ5; see Appendix Figure S5)) differs from that observed with the hβ5i/hβ6 chimera. Note that the inhibitors are rotated by about 45 ° compared to panel A. \n",
      ".Figure 6A. Complex structures of the mouse chimeric proteasome (left) and the hβ5i-V31M/hβ6 mutant (right) with PR-924 visualize the kinked (k) orientation. The 2FO-FC electron density maps for the ligands bound to Thr1 of the chimeric β5 active sites are depicted according to Fig 5A. Steric hindrance of Met31 (green) with the bulky 3-methyl-1H-indene cap is indicated by black double arrows. B. Superposition of inhibitors bound to the mouse chimeric proteasome (left) and the hβ5i-V31M/hβ6 mutant (right). Compounds display the same binding modes (kinked (k) or linear (l)) to the mβ5i/mβ6 active site as to the corresponding human β5i/β6 subunit (Fig. 5) and to the hβ5i-V31M/hβ6 mutant. Note that the inhibitors are rotated by about 45° compared to panel A. C. Superposition of chimeric mammalian and WT yeast β5 active sites bound to the P3-d-Ala compounds PR-924 and 18. The nucleophilic Thr1 is depicted as well as the S1 pocket forming residues 31 and 45. Residue 31 impacts on the position of the 3-methyl-1H-indene cap of PR-924 and 18. Due to steric hindrance, the N-cap is shifted in the mouse and in the hβ5i-V31M/hβ6 chimera compared to the hβ5i/hβ6 mutant proteasome crystal structures (black double arrow). Note that the structures are rotated by 90° compared to panel B. \n",
      ".Figure 7ONX 0914 binds in a linear mode to all active sites. Changing the stereochemistry at the P3 site from l-Ala (S) to d-Ala (R) leads to a kinked conformation in the β5 substrate binding channels (compound 16). Substitution of the morpholine cap of 16 by a larger 3-metyhl-1H-indene moiety yields PR-924 and results in different inhibitor conformations at the yβ5/hβ5c and the mammalian β5i active sites. Hereby, the selectivity for hβ5i over hβ5c is drastically increased (IC50 ratio hβ5c/hβ5i: 255) compared to ONX 0914 (IC50 ratio hβ5c/hβ5i: 20) and compound 16 (IC50 ratio hβ5c/hβ5i: 5.2, Fig 2). Residue Met31 of mβ5i sterically hinders the kinked binding mode of PR-924, thereby creating 7-fold selectivity for human β5i. \n",
      ".Figure 1A. Congo Red depletion assay. Fn 12230 (black), the fadA-deletion mutant US1 (∆fadA) (orange) and the spontaneous lam mutant (blue) (top panel) were grown to different OD600. E. coli MC4100, ∆curli, and ∆csgA (bottom panel) were grown on TSA-blood agar plate at 26oC for 48hr. The bacteria were suspended in PBS to OD600 of 0.5, 1.0, 1.5 and 2.0, followed by incubation in 10 μg/ml Congo Red (CR) for 10 min. After centrifugation, the supernatants were measured at OD500. The results shown are the average of five independent experiments performed in duplicate. The error bars indicate SD. *p<0.05, ** p<0.01, ***p<0.001 (compared to wild type, t-test). B. Kinetics of FadA production in Fn. Fn 12230 and US1 were inoculated to OD600 = 0.1. 10 OD600 units of bacteria were harvested by centrifugation at 12-hr intervals. Bacteria were lysed with 2 mg/ml lysozyme, followed by incubation in 1% sarkosyl at 4° C for 20 minutes, which became the 'total' sample. Lysates were then centrifuged at 100,000 g for 20 minutes and the supernatants and pellets were collected. An aliquot of 5 μl is loaded onto each lane, followed by immunoblot using anti-FadA monoclonal antibody (mAb) 7H7 at 1:4,000 dilution to detect FadA protein in total, supernatant, and pellet fractions. Ponceau staining of lysozyme is shown as a loading control. Pre-FadA and mFadA are pointed by arrows. Note pre-FadA migrates faster than mFadA on SDS-PAGE as previously reported (Han et al., 2005). C. Analysis of detergent-resistant FadA polymers in wild-type Fn and its mutants by semi-denaturing detergent agarose gel electrophoresis (SDD-AGE) and Western blot analysis using mAb 7H7. Fn12230 (wt), fadA-deletion mutant US1 (∆fadA) and spontaneous mutant lam (lam) were grown to log (OD600 0.3-0.4) or stationary phase (OD600 >0.8), E. coli MC4100, ∆curli, and ∆csgA were grown to stationary phase. The bacteria were harvested by centrifugation. Following sequential incubation in lysis buffer containing 2 mg/ml lysozyme and 1% sarkosyl, the insoluble pellets were obtained by centrifugation. An aliquot of 100 μg of each pellet was loaded onto 1.7% agarose gel followed by overnight electrophoresis in 0.5xTAE and 0.1% SDS. Following transfer to nitrocellulose membrane, Western blot was performed using mAb 7H7 at 1:4,000 dilution to detect the FadA proteins or anti-CsgA antibody at 1:15,000 dilution. The heterogeneously large detergent resistant FadA polymers were detected in wild-type Fn in the stationary phase, defective in US1 and lam, and absent in all in log phase. D. Immunohistochemical analysis of FadA in log and stationary phases. Fn12230 (Fn), fadA-deletion mutant US1 (∆fadA) and spontaneous mutant lam in log and stationary phase were fixed before being placed onto glass slides and stained by mAb 7H7 at 1:800 dilution, followed by incubation of HRP-conjugated anti-mouse IgG and developed by DAB. The large FadA aggregates are detected specifically in wild-type Fn 12230 in the stationary but not in log phase. FadA was detected on lam, but without secreted aggregates. No FadA was detected in US1. The imagines were taken using 40X objective. Scale bar equals 20 μm. E. Double immunofluorescent staining of Fn12230, fadA-deletion mutant US1 (∆fadA) and spontaneous mutant lam in log and stationary phases using mAb 7H7 at 1:800 dilution and anti-human β-amyloid polyclonal antibody A11 at 1:500 dilution, followed by incubation with Alexa Fluor 680-conjugated donkey anti-rabbit and Alexa Fluor 555-conjugated goat anti-mouse, both at 1:1,000 dilution. Co-staining of FadA and A11 was observed in Fn in stationary phase, not in log phase, or in US1 or lam. The imagines were taken under 60X objective. Scale bar equals 5 μm. F. Scanning electron microscopy of Fn 12230, US1 (∆fadA) and lam in log and stationary phase at 10,000x magnification. Note the fibrous structure coating Fn in stationary phase, pointed by the clear arrows, but not in log phase. Scale bar equals 1 μm. \n",
      ".Figure 3A. Detection of FadA in subgingival plaque samples by IHC. Three patients with periodontitis are shown here who provided plaque samples from their healthy sites (probing depth ≤ 3 mm, left panels) as well as periodontal diseased sites (probing depth ≥ 7 mm, middle panels). IHC was performed as described using mAb 7H7 at 1:800 dilution. Anti-mouse IgG isotype control of the diseased site is shown (right panels). The images were taken using 100X objective. Scale bar equals 50 μm. B. Double immunofluorescence staining of paired normal and carcinoma tissues from two CRC patients. The frozen tissue sections were incubated with mAb 7H7 at 1:50 dilution and anti-amyloid antibody A11 at 1:25 dilution, or mouse and rabbit IgG control, followed by incubation with Alexa Fluor 555-conjugated goat anti-mouse and Alexa Fluor 680-conjugated donkey anti-rabbit, both at 1:1000 dilution. Co-staining of FadA (red) and amyloid oligomers (green) was observed in the carcinoma but not normal tissues. The images were taken under 60X objective. Scale bar equals 10 μm. \n",
      ".Figure 2A. Single-letter amino acid (aa) sequence of FadA. The intact pre-FadA consists of 129 aa, with the first 18-aa (top row) constitute the signal peptide. The remaining 111 aa constitute mFadA. The residues that are replaced in the mutant proteins used in this study are shown in red. B. Analysis of detergent resistant recombinant FadA polymers expressed in E. coli. An aliquot of 50 μg detergent-resistant pellets prepared from E. coli BL21(DE3) carrying the cloning vector pET21(b), pYWH417-6 (p_FadAc), or pYWH418 (p_mFadA) were analyzed by SDD-AGE, followed by Western blot analysis. FadA polymers were only detected in E. coli expressing FadAc, but not in mFadA or the vector control. C. Reactivity of recombinant FadA and its variants with anti-amyloid fibril antibody OC. A total of 10 μg each of recombinant protein mFadA, FadAc, FadA-L-9A, FadA-L14A, FadA-S71A, and FadA-L76A purified from E.coli was slot-blotted onto nitrocellulose and incubated with polyclonal OC antibody at 1:5,000 dilution. Amyloid-like recombinant Rim4 (Berchowitz et al., 2015) was used as a positive control and histone H1 was used as a negative control. Ponceau stain (pink color) is shown as a loading control. D. Thioflavin-T binding assay. FadA proteins at indicated concentrations were incubated with 10 μM Thioflavin-T at room temperature for 10 min. The fluorescent intensity was measured at excitation wavelength of 440 nm and emission wavelength of 500 nm. The experiment was performed in duplicate and repeated three times. The error bars indicate SD. *p<0.05, **p< 0.01, ***p<0.001 (compared to BSA, t-test). E. Thioflavin-T binding assay in the presence of 0.1% SDS. FadA proteins (0.8 mg/ml) were incubated with 10 μM Thioflavin-T at room temperature for 10 min. The fluorescent intensity was measured at excitation wavelength of 440 nm and emission wavelength of 500 nm. The experiment was performed in duplicate and repeated three times. The error bars indicate SD. *p<0.05, **p< 0.01 (compared to BSA, t-test). F. Western blot analysis of Fn12230 (Fn), fadA-deletion mutant US1 (∆fadA) and spontaneous mutant lam following SDS-PAGE. A total of 400 μl of each culture grown to OD600 of 1.0 was pelleted and loaded onto 12% SDS-PAGE. FadA protein was detected using anti-FadA mAb 7H7 at 1:4,000 dilution. Compared to the wild type, lam produced significantly less pre-FadA, but not mFadA. G. Analysis of amyloid-like FadA produced by Fn 23726 and its ∆fap2 mutant. An aliquot of 50 μg detergent-resistant pellets was loaded onto each lane, followed by SDD-AGE and Western blot analysis as described above. The mutant produced significantly less amyloid-like polymers than the wild type. \n",
      ".Figure 5A. Analysis of Fn colonization in mouse xenograft tumors by IHC. An inoculum of 5×106 HCT116 cells was injected subcutaneously and bilaterally into the nude mice. Once the tumors became visible (after 3-4 days), approximately 5×106 CFU of Fn 12230 washed in PBS or 1 mg/ml Congo Red (CR) were injected into the tumors. The tumors were extracted one week later, and the formalin-fixed and embedded (FFPE) tissues were stained with mAb 7H7. Note the brown stains in tumor injected with PBS-washed tumor, but not in CR-washed tumor. Scale bar = 20 μM. B. Effects of secreted amyloid-like FadA on tumor growth. HCT116 cells were injected into the nude mice as described above, followed by injection of 5×106 CFU of Fn 12230 (Fn, n=7), lam (n=3) or US1 (∆fadA, n=4). The tumor volumes were measured immediately before bacteria injection (Day 0, designated as 100%), and at 3 days after injection (Day 3). The horizontal lines represent averages of tumor volumes (left panel). The individual tumors are shown (right panel). *p < 0.05, **p < 0.01, ***p< 0.001 (t-test). C. Effect of anti-amyloid antibody OC on FadAc-mediated tumor growth. HCT116 cells were injected into the nude mice (n=6) as described above, followed by injection of 4 μl of 2 mg/ml FadAc mixed with 6 μl OC antibody or rabbit IgG control (rIgG) into tumors on the opposite sides. The tumor volumes were measured on Day 0 and Day 5 as described above. The horizontal lines represent the averages (left panel). The individual tumor pairs are shown (right panel, n=6). *p < 0.05, **p < 0.01 (t-test). D. Effect of detergent-resistant pellets prepared from Fn 12230 (Fn) and US1 (∆fadA) on tumor growth. HCT116 cells were inoculated into the nude mice as described above. In one group of mice (n=3), sarkosyl-resistant pellets prepared from Fn 12230 and US1 (∆fadA) were injected into the tumors on opposite sides. In another group (n=5), sarkosyl-resistant pellets prepared from Fn 12230 were mixed with OC or rabbit IgG control (rIgG) before injecting into the tumors on opposite sides. The tumor volumes were measured described above. The vertical lines represent the standard deviations. The individual tumor pairs are shown (right panel). *p < 0.05, **p < 0.01, ***p< 0.001 [compared to US1 (∆fadA), t-test]. \n",
      ".Figure 4A. Facilitation of Fn biofilm formation by amyloid-like FadA. An aliquot of 10 μg each of the detergent-resistant pellets prepared from of Fn 12230 (wild type), US1 (∆fadA) and lam, or PBS, was added to the 96-well plates and incubated at 370C for 5 hours. Fresh culture of Fn 12230 was diluted to OD600 of 0.2 and an aliquot of 200 μl was added to the wells, followed by incubation at 37 0C for 48 or 72 hours under anaerobic conditions. Following washes, the biofilms were incubated with 100 μl 0.1% crystal violet for 15 min. After washing, 100 μl 95% ethanol was added to each well and the optical density at OD550 was measured in a microplate reader. The results shown are the average of four experiments each performed in duplicate. The error bars indicate SD. A representative image following 72 hours of incubation is shown in the top panel. ** p<0.01. ***p<0.001 (t-test). B. Inhibition of biofilm formation mediated by detergent-resistant Fn pellets by Congo Red. An aliquot of 10 μg detergent-resistant pellets prepared from Fn 12230 (pellet) was added in 96-well plates in the presence of 0, 10 or 50 µg/ml Congo Red (CR), followed by addition of Fn 12230 and incubated as described above. The results shown are the average of four experiments each performed in duplicate. The error bars indicate SD. A representative image following 72 hours of incubation is shown in the top panel. ** p<0.01, ***p<0.001 (t-test). C. Effects of pH on Fn survival. Wild type Fn 12230 (Fn) or its fadA deletion mutant US1 (∆fadA) grown to log or stationary phase were washed and incubated in PBS at pH7 or pH4 for 1 or 2 hours. The live bacterial counts at time 0 were designated as 100%, and those after 1 or 2 hrs of incubation were expressed relative to time 0. The results shown are the average of 5 (for log) or 4 (for stationary) independent experiments, each performed in triplicate. The error bars indicate SD. *p<0.05, ** p<0.01, ***p<0.001 (One-way ANOVA). D. Effects of Congo Red on Fn survival in acidic environment. Wild type Fn 12230 (Fn) or its fadA deletion mutant US1 (∆fadA) grown to stationary phase were washed and incubated in PBS at pH4 in the presence of absence of 50 μg/ml Congo Red (CR) for 1 or 2 hours. The live bacterial counts at time 0 were designated as 100%, and those after 1 or 2 hrs of incubation were expressed relative to time 0. The results shown are the average of 5 independent experiments, each performed in duplicate. The error bars indicate SD. *p<0.05, ** p<0.01, ***p<0.001 (One-way ANOVA). E. Attachment of Fn 12230 grown to log or stationary phase to CRC cells DLD1. Fn 12230 (Fn) was either unwashed or washed twice with PBS or 1 mg/ml Congo Red (CR), or mixed with 20 μl mAb 7H7, before adding to the monolayers at multiplicity of infection (MOI) of 50:1. US1 (∆fadA) was included as a negative control. The attachment level by untreated log phase Fn was designated as 100%, which all other values were expressed relative to. Data shown are mean values ± SEM. The experiment was performed in duplicate and repeated four times. *p < 0.05, **p < 0.01, ***p < 0.001 (t-test). F. Inhibition of Fn attachment by FadAc in the presence or absence of anti-amyloid antibody OC. An aliquot of 50 μg purified recombinant FadAc was pre-incubated with DLD1 for 45 min either alone, or mixed with 20 μl OC or rabbit IgG control, before Fn was added and the attachment assay performed as above. The attachment value by Fn alone was designated as 100%, with all other values expressed relative to it. FadA mutant L14A was included as a negative control. Data shown are mean values ± SEM. The experiment was performed in duplicate and repeated three times. **p< 0.01 (One-way ANOVA). \n",
      ".Figure 6A. Approximately 1x109 CFU of Fn 12230, US1 (∆fadA), and lam suspended in carboxymethylcellulose (CMC) were orally administered to C57BL/6 mice three times a week for 10 weeks. CMC alone was administered as a control. Maxillae from mice inoculated with CMC alone (n=5), Fn 12230 (n=5), US1 (∆fadA; n=5), and lam (n=4) were harvested and fixed in 4% paraformaldehyde and stored in 70% ethanol, followed by microCT (μCT) scanning using a Scanco vivaCT 80 system at 55 kVp, 145 µA, and 250 ms integration time. Shown in the figure are reconstructed grayscale images. ImageJ was used to measure difference in bone height from the cementoenamel junction (CEJ) to the alveolar crest between palatal roots of first and second molars (see arrows). B. The average bone loss of each group shown in (A), with the lines above each bar representing standard deviations. *p< 0.05. **p< 0.01 (t-test). \n",
      ".Figure 7The FadA filaments were based on the crystal structure of mFadA (Nithianantham et al., 2009), with the addition of intercalating pre-FadA. The signal peptide pairs from two neighboring filaments (one clear bar and one shaded bar) bind to each other through hydrophobic interactions, serving as \"hooks\" crosslinking the filaments so that the filaments are stacked perpendicular to the fibril axis into fibrous sheets. \n",
      ".Figure 2(A-C) Three different mediation models are shown, testing plasma P-tau217 as a statistical mediator of plaque density on tangle density. Panel A shows the mediation of total tangle density (significant mediation, 60% of the direct effect of plaque density on tangle density was explained by plasma P-tau217). Panel B shows the mediation of medial temporal lobe (MTL, entorhinal cortex and hippocampus) tangle density (no mediation). Panel C shows the mediation of tangle density in all regions except the MTL (strongest mediation, 77%). The P-values are from linear regression models used to estimate the mediation effects. \n",
      ".Figure 1Panel A shows R2 and AIC for different regression models, using either only amyloid plaque density (\"P\"), only tangle density (\"T\"), both plaque and tangle density (\"P+T\"), or both plaque and tangle density including their interaction term (\"PxT\"). All models included age and sex as covariates. The panel shows adjusted R2, together with AIC (above the bars) for each model. We compared R2 between the models using a bootstrap procedure (N=1000 iterations), which showed that the R2 for the \"PxT\" model was marginally higher than for the \"P+T\" model (∆R2=0.027, 95% CI -0.0034-0.087), and significantly higher than for the \"T\" model (∆R2=0.12, 95% CI 0.04-0.24) and the \"P\" model (∆R2=0.18, 95% CI 0.055-0.31). Panel B shows associations between plasma P-tau217 and plaque density (stratified by tertiles [T1-3] of tangle density). Panel C shows associations between plasma P-tau217 and tangle count (stratified by tertiles [T1-3] of plaque density). \n",
      ".Figure 3(A-C) Individuals in the neuropathology cohort without tangles in the parietal or frontal lobe and no more than minimal tangle load in the temporal lobe were included in this analysis (N=42). MTL tangles were defined as tangles in entorhinal cortex plus hippocampus. Relationships between variables were tested in linear regression models, adjusted for age and sex. In these models, plasma P-tau217 was significantly associated with Aβ plaques (β=1.64, P<0.0001) (panel A) but not with MTL tangles (panel B). MTL tangles were not associated with Aβ plaques (panel C). The solid line in panel A is the mean effect from the regression model, and the shaded are is the 95% confidence interval of the mean. \n",
      ".Figure 6(A, B) Plasma P-tau217 partly mediated the effect of Aβ PET on tau PET, when using tau PET quantified in Braak stages I-VI (A, \"Tau PET global\", 66% of the direct effect of Aβ PET on tau PET was explained by plasma P-tau217) or in the medial temporal lobe (B, MTL, entorhinal cortex and hippocampus, 29%). The P-values are from linear regression models used to estimate the mediation effects. \n",
      ".Figure 4(A-C) Individuals in the neuropathology cohort were divided in groups of no significant primary pathology (N=31), AD (N=36), PD (N=13), and others (N=8), and analyzed together with N=9 additional subjects with primary non-AD tauopathies (PSP/CBD). Data is shown for plasma P-tau217 (panel A), plaque density (panel B), and tangle density (panel C). Plasma P-tau217 was not elevated in PSP/CBD compared to subjects without significant primary pathology. AD, Alzheimer's disease; CBD, corticobasal degeneration; PSP, progressive supranuclear palsy; PD, Parkinson's disease. The central band of the boxes are medians and the boxes show inter-quartile ranges. The whiskers are defined as the smallest (largest) observation greater (less) than or equal to the first (third) quartile minus (plus) 1.5 times the inter-quartile range. \n",
      ".Figure 5Panel A shows R2 and AIC for different regression models, using either only Aβ PET (\"A\"), only tau PET (\"T(I-VI)\"), both Aβ and tau PET (\"A+T\"), or both Aβ and tau PET including their interaction term (\"AxT\"). All models included age and sex as covariates. The panel shows adjusted R2, together with AIC (above the bars) for each model. We compared R2 between the models using a bootstrap procedure (N=1000 iterations), verifying that the R2 for the \"AxT\" model was higher than for the \"A+T\" model (∆R2=0.026, 95% CI 0.010-0.058), the \"T(I-VI)\" model (∆R2=0.17, 95% CI 0.11-0.26) and the \"A\" model (∆R2=0.14, 95% CI 0.060-0.22). Panel B shows associations between plasma P-tau217 and Aβ PET stratified by tertiles (T) of tau PET (T1: SUVR ≤ 1.05, T2: 1.05 < SUVR ≤ 1.11, T3: 1.11 < SUVR ≤ 3.03).   Panel C shows associations between plasma P-tau217 and tau PET stratified by tertiles (T) of Aβ PET (T1: SUVR ≤ 0.468, T2: 0.468 < SUVR ≤ 0.579, T3: 0.579 < SUVR ≤ 1.08).   (D-I) Panels D-I show associations with P-tau217 across groups of Aβ- and tau-PET positivity. Cut-points for Aβ-PET (> 0.533 SUVR) and tau-PET (> 1.36 SUVR in a ROI corresponding to Braak stages I-IV) were used to define the groups. Associations are shown for global cortical Aβ PET (panels D-F) and global cortical tau PET (corresponding to Braak Stages I-VI) (panels G-I) in individuals classified as negative for both Aβ and tau (N=264, panels D and G), positive for Aβ only (N=104, panels E and H), or positive for both Aβ and tau (N=58, panels F and I). The plots show R2 and P-values from linear regression models. No covariates were included in these models, in order to generate R2-values for the PET measures alone. \n",
      ".Figure 7The figure shows hypothetical pathways, supported by our results, which link deposition of Aβ pathology to development of neocortical tangles through increased phosphorylation and/or release of soluble tau. We also include a hypothetical feedback loop, where tangle pathology also may drive increased levels of soluble P-tau, possible due to tau propagation between affected neurons. We also acknowledge that other alternative explanations may be considered for our findings. One possibility is that lack of sensitivity of our methods to accurately quantify aggregated tau makes us unable to detect very early associations between aggregated tau and plasma P-tau217. \n",
      "Vacuoles and autophagosome-like structures in the degenerating epithelial glia.Fig 2(A-E) Horizontal sections of the adult head lamina cartridge. (A) In repo>GFP.nls flies, two lamina neurons L1, L2 axons (L) and R1-6 axons (blue area) were surrounded by the electron-dense cytoplasm of the epithelial glia (G). The section was examined at three different depths, and the size of a single cartridge is not significantly different at different depths. (B-D) repo>Shits1 adults maintained at 29°C for 4 days. Axons were enlarged but intact. A large number of small and medium vacuoles were identified in the epithelial glia. (C) Small vacuoles in the glia that contained a double-membrane structure (arrowhead). (C') Higher magnification of the boxed area in (C). (D) A double-membrane autophagosome-like structure (arrowhead) in the glialcytoplasm. (E) In repots>DERDN adults maintained at 29°C for 2 days, large vacuoles were identified in between the lamina cartridges. N: glialnuclei. (E') Higher magnification of the boxed area in (E), which shows autophagosome-like vesicles (arrowhead) in the gliacytoplasm. \n",
      "Shits induced cell-autonomous glia degeneration in optic lamina.Fig 1H&amp;amp;E staining of adult head sections of (A) repo>GFP.nls and (B) repo>Shits1 at 29°C for 3 days. Lamina degeneration was identified as vacuoles in (B). Cryosection of adult (C) repo>H2B-RFP exhibited the expression of the nuclear red fluorescent protein (RFP) in the glia (epithelialglialnucleus: arrowhead) and (D) repo>Shits1 at 28°C for 12 days. Vacuoles were identified in the lamina neuropile (D). Epithelial (arrowhead) (E), marginal (arrow) and distal satellite (arrowhead) glianuclei (F) are labeled by HisCl-GAL4 and NP2109-GAL4, respectively. Note that HisCl-GAL4 is not expressed in all epithelialglia. (G, H) Weak lamina vacuolization was identified in HisCl>Shits1 (G, 0.92%) and NP2109>Shits1 (H, 1.46%) at 29°C for 14 days. (I, J) A single MARCMglia clone (GFP, green) expressed Shits1 at 21°C (I) and 29°C (J). (J) A vacuole occurred within a glial clone. DAPI (white) stains the nuclei (C, D, and G-J). (K) The percentage of the vacuole area in the lamina of the repo>Shits1flies at 28°C progressively increased. n indicated in each column. P-values were calculated using one-way ANOVA with Bonferroni's post-test. (L) When repo>Shits1flies were shifted to 28°C for 12 days and then shifted to 17°C for 9 additional days, the vacuolization was not alleviated. P-values were calculated using one-way ANOVA with Tukey's post-test. n indicated in each column. \n",
      "R1-6 photoreceptors are required for lamina glia maintenance.Fig 3(A) Rh1>lacZ+H2B-RFP exhibited nuclearRFP expression in the R1-6 photoreceptors (red). (B) Rh1>lacZ+Shits1 maintained at 29°C for 14 days resulted in lamina vacuolization. (C) Pan-Rh7>H2B-RFP exhibited expression in R7 (arrow) and R8 (arrowhead) (red). (D) Pan-Rh7>Shits1 incubated at 29°C for 14 days did not exhibit lamina degeneration. (E) Lamina monopolar (L) neurons (red) were selectively labeled by the Ln-GAL4 driven H2B-RFP with repo-GAL80. (F) Ln>Shits1, repoGAL80 shifted to 29°C for 14 days did not exhibit lamina degeneration. (G) R1-6 photoreceptors were killed in Rh1ts>lacZ+Hid shifted to 29°C for 14 days. Rh1ts indicates tubGAL80ts; Rh1-GAL4. Degeneration was induced in the lamina in addition to the retina. (H) rdgC306/+ mutant exposed to constant light for 14 days exhibited degeneration in the retina, as well as the lamina. DAPI: nuclei (white in A-H). (I, K) Rh1>lacZ+Shits1 at 29°C for 4 days exhibited vacuoles in the electron dense cytoplasm of epithelialglia or near the glianucleus. Arrow: vacuole with internal debris. Arrowhead: large vacuole. (J) Lamina cartridge of Rh1>lacZ+H2B-RFP. N indicates glialnuclei. (L) The percentage of the vacuole area in the lamina at 29°C for 14 days was examined. All P-values were calculated using one-way ANOVA with Tukey's post-test. Scale bar: 20 μm. \n",
      "EGFR is required and sufficient in the lamina glia to maintain glia integrity.Fig 4(A) repots>Shits1+H2B-RFP, (B) repots>Shits1+Egfrλtop and (C) repots>Shits1+RlSem incubated for 12 days at 28°C. (D) repots>DERDN for 7 days at 28°C. (E) Epithelial glia MARCM clone and (F) Egfrco mutant glia MARCM clone (labeled with Tomato, green). The penetrance is indicated as the number of samples with vacuoles over the total number of samples. DAPI: nuclei (white in A-F). (G) Percentage of vacuole area in the lamina neuropile of (A-D). The P-values were calculated using one-way ANOVA with Dunnett's post-test. (H) Percentage of vacuole area in the lamina of repots>Shits1 flies in the indicated genetic background. The adults were shifted to 28°C for 12 days. The P-values were calculated using one-way ANOVA with Tukey's post-test. Scale bar: 20 μm. \n",
      "EGFR signaling in the lamina glia is dependent on Spitz from the R1-6 photoreceptors.Fig 6(A, A') Anti-Spitz detected Spitz (magenta) colocalizing (arrowhead) with epithelialgliacytoplasm marked by anti-Black (green). Scale bar: 5 μm. (B-B') EGFR reporter pointed-lacZ (pnt-lacZ) exhibited expression in the epithelial (arrow) and marginal glia (arrowhead). (C, C') Dominant-negative form of EGFR (DERDN) expressed in glia at 28°C for 3 days inhibited pnt-lacZ expression. (D, D') pnt-laZ expression was lost in Shits-expressing glia at 28°C for 3 days. (E, E') The knockdown of Spi in R1-6 photoreceptors at 28°C for 12 days also inhibited pnt-lacZ expression in the glia. The penetrances of (B-E) are shown in the upper right corner of each panel. DAPI: nuclei (white in B'-E'). Scale bar: 20 μm. \n",
      "Spitz from photoreceptors is transported to the lamina and is required for lamina glia maintenance.Fig 5(A, A') Anti-Spitz (red) immunostaining of w1118 adult head. Spitz can be detected in the retina and lamina. (B, B') The full-length transmembrane form of Spitz-GFP (mSpi, green) expressed in the retina in GMR>mSpiGFP flies was predominantly identified in the photoreceptor soma and terminally localized in the lamina cartridge. mSpitz-GFP requires processing by Rho and Star to become a secreted form. Overexpressed mSpitzGFP has been demonstrated to be retained in the perinuclear ER even in the presence of endogenous Rho/Star [56] (C) Knockdown of both EGFR ligands Spi and Krn in R1-6 photoreceptors in Rh1>Dcr2+Spi-RNAi+Krn-RNAi and (D) blockade of Spi processing in Rh1>lacZ+iRhom exhibited lamina degeneration after shifting to 29°C for 14 days. (E-E') Lamina degeneration in whole eye rho7M43ru1 double mutant clones at 28°C for 12 days. The clone is labeled by DsRed (red). (F) Spi secretion is inhibited in Rh1>lacZ+Rab11S25N. (G) The percentages of the vacuole areas of (C, D, F) in lamina at 29°C for 14 days were examined. All P-values were calculated using one-way ANOVA with Tukey's post-test. DAPI: nuclei (white in A', B', C-F). Scale bar: 20 μm. \n",
      "Defect in early endocytic steps and lysosomal trafficking caused lamina vacuolization.Fig 10(A) repots>Rab5S43N. (B) repots>Rab7T22N. (C) repots>Rab11S25N. (D-H) MARCM clones labeled by RFP or GFP (green) of control (D), Rab52 (E), α-Ada3 (F), HrsD28 (G), Rab7KO (H), dor8 (I) and carΔ146 (J). The penetrance (number of samples with vacuole over the number of samples examined) is indicated in each panel. Adults of all genotypes were incubated at 28°C for 14 days. DAPI: nuclei (white in A-J). Scale bar: 20 μm. (K) The percentage of the vacuole area in (A-C) was summarized. Adults of these genotypes were incubated at 28°C for 12 days. P-values were calculated using Kruskal-Wallis with Dunn's post-tests. \n",
      "EGFR signaling is required for endo-lysosomal trafficking.Fig 9(A-C) repots>GFP-LAMP1. (D-F) repots>GFP-LAMP1+Shits1. (G-I) repots>GFP-LAMP1+DERDN. Adults were incubated at 28°C for 12 h (A, D, G), 24 h (B, E, H), and 48 h (C, F, I), respectively. The GFP-LAMP1 signal (green) was induced at 12 h and progressively enhanced in (D-F) and (G-I). (J) repots>Rab7-mCherry+H2B-RFP. (K) repots>Rab7-mCherry+Shits1. (L) repots>Rab7-mCherry+DERDN. The adults were shifted to 28°C for 2 days. Rab7-mCherry puncta (red) were increased in (K, L). DAPI: nuclei (white in J'-L'). Scale bar: 20 μm. \n",
      "Apoptosis is not involved in the epithelial glia degeneration mediated by blockade of EGFR signaling.Fig 7(A-C) Immunostaining of TUNEL assay (red), active Caspase-3 (magenta, A'-C') and Repo (green, A\"-C\"). (A) GMR-wIR; repots>lacZ. (B) GMR-wIR; repots>Shits1. (C) GMR-wIR; repots>DERDN. GMR-wIR was used to reduce the autofluorescence of the eye pigments. Scale bar: 20 μm. (D-F) The in vivo fluorescent sensor of caspase activity (Apoliner) indicated there was no active caspase activity in the glia. (D) repots>Apoliner+lacZ. (E) repots>Apoliner+Shits1. (F) repots>Apoliner+DERDN. All adults were shifted to 28°C for 5 days. nls-GFP (green, D-F); merge of GFP (green) and mRFP (red) in (D'-F'); merge of nls-GFP and DAPI (magenta) in (D\"-F\"). Scale bar: 10 μm. (G) Percentage of vacuole area in the lamina in repots>Shits1flies that coexpressed the anti-apoptotic factors P35 and Diap1. The adults were shifted to 28°C for 21 days. The P-values were calculated using one-way ANOVA with Tukey's post-test. (H) The cell number of the epithelialglia was not reduced in the vacuolated lamina. The numbers of epithelial glia in the lamina of repo>H2B-RFP and repo>H2B-RFP+Shits1 female adults incubated at 28°C for 14 days were examined by counting the nuclear RFP at the epithelial layer from the entire Z-stacks of confocal images. P-values were calculated via Mann-Whitney tests. \n",
      "EGFR signaling is required for autophagosome-lysosomal trafficking.Fig 8In repots>GFP-LC3+DERDN shifted to 28°C, the GFP signal (green) was weak on day 1 (A-A\") and progressively increased on days 2 (B-B\") and 3 (C-C\") and colocalized with Ref(2) P (magenta). (D) Ref(2) P (magenta) accumulated within the Egfrco mutant MARCM clone (green). The double-tagged GFP-mCherry-Atg8a is used to distinguish the autophagosomes (GFP and mCherry, yellow) and autolysosomes (mCherry, red) in autophagic flux. (E) In repots>GFP-mCherry-Atg8a+dAtg1, autophagosomes (arrowhead) and autolysosomes (arrow) were induced in the glia with normal autophagic flux. (F) repots>GFP-mCherry-Atg8a. (G) repots>GFP-mCherry-Atg8a+Shits1. Epithelialglial nuclei are indicated (arrow). (H) repots>GFP-mCherry-Atg8a+DERDN. The adults were shifted to 28°C for 3 days. GFP: green (E-H); mCherry: red (E'-H'); merge (E\"-H\"). Scale bar: 10 μm. \n",
      "Vps-C complex components Dor and Car affected lamina glial vacuolization.Fig 11repots>DERDN was coexpressed with (A) H2B-RFP, (B) Dorwt, (C) Dor-RNAi, (D) H2B-RFP, (E) Car, (F) Car-RNAi, (G) lacZ, (H) Car-RNAi+Dor-RNAi and (I) Car+Dorwt. The autophagy reporter Ref(2) P (green) was stained (G'-H'). Scale bar: 20 μm. (J, K, L) The percentages of the vacuole areas in the lamina of (A-C, D-F, G-I) were summarized, respectively. The adults were shifted to 28°C for 7 days. DAPI: nuclei (white in A-I). All P-values were calculated using Kruskal-Wallis tests with Dunn's post-tests. \n",
      "a–d) Degradation of long-lived proteins in MEFs from wild-type (18Q-htt) and mutant huntingtin knock-in (111Q-htt) mice.figf1(a-d) Degradation of long-lived proteins in MEFs from wild-type (18Q-htt) and mutant huntingtin knock-in (111Q-htt) mice. (a,b) Rates of protein degradation after serum removal (a) (a-d) Degradation of long-lived proteins in MEFs from wild-type (18Q-htt) and mutant huntingtin knock-in (111Q-htt) mice. (a,b) Rates of protein degradation after treatment with rapamycin (rapam) or thapsigargin (thapsig) (b). (c) Lysosomal degradation calculated as percentage of protein degradation sensitive to NH4Cl. (d) Contribution of macroautophagy calculated as percentage of protein degradation sensitive to 3-methyladenine (3-MA). (e) Degradation of long-lived proteins in striatal cells from wild-type (7Q-htt) and mutant huntingtin knock-in mice (111Q-htt) in response to different autophagic stimuli. (f,g) Rates of protein degradation in wild-type and HD MEFs (f) and striatal cells (g) in the absence (control) or presence (Atg7−) of Atg7 RNA interference. (h) Contribution of macroautophagy (macroaut) to the degradation of long-lived proteins in lymphoblasts from unaffected controls (UC) and people with HD. Mean + s.d. of values from two to four different individuals and three or four different experiments. *P 0.05 versus untreated, #P 0.05 versus control. \n",
      "a) Top: LC3 immunostaining of 18Q-htt and 111Q-htt MEFs maintained in the presence (+) or absence (−) of serum and inhibitors of lysosomal proteolysis (inhib+).figf2(a) Top: LC3 immunostaining of 18Q-htt and 111Q-htt MEFs maintained in the presence (+) or absence (-) of serum and inhibitors of lysosomal proteolysis (inhib+). Bottom: mean number of LC3+ vesicles per cell in cells maintained in the presence (left) or absence (right) of serum. n = 4. (b) Top: LC3 immunoblot of the same cells after serum (S) removal or thapsigargin (TG) treatment. PI, protease inhibitors. Bottom: LC3-II levels and LC3-II flux (calculated as a multiple of LC3-II value in absence of PI; lateral numbers). n = 4. (c) LC3-II values and LC3-II flux in neuronal cultures from wild-type (WT) or HD94 mice (HD) grown on wild-type rat astrocyte monolayers, analyzed as in b. n = 4. (d) LC3 staining of lymphoblasts, from three unaffected controls (UC) and people with HD, maintained in the presence or absence of serum. Accession numbers are indicated above panels. Right: mean number of LC3+ vesicles. Extended study in Supplementary Figure 8b,c. (e) Immunoblots for LC3-II in UC and HD lymphoblasts treated or not treated with protease inhibitors. Right: LC3-II values and LC3-II flux. Extended study in Supplementary Figure 8a. Mean + s.d.; #P 0.05. Full-length blots in Supplementary Figure 20. \n",
      "a) Electron micrographs of fractions enriched in autophagosomes and autophagolysosomes isolated from liver of 18Q-htt and 111Q-htt mice.figf4(a) Electron micrographs of fractions enriched in autophagosomes and autophagolysosomes isolated from liver of 18Q-htt and 111Q-htt mice. Insets: higher magnification of single vesicles. Right: percentage of vesicles with electron-clear (light), vesiculated (multivesicular) or electron-dense (dark) contents. Mean + s.e.m. of three different isolations (> ∼1,000 autophagic vacuoles). (b) Bidimensional electrophoresis and Sypro Ruby staining of these fractions. Left, intact vacuoles; right, vacuole contents. Horizontal dimension, isoelectric point (pH); vertical, molecular mass (kDa). *P 0.05. \n",
      "a) Electron micrographs of striatal neurons from control and HD94 mice grown over a wild-type rat astrocyte monolayer.figf3(a) Electron microscopy of striatal neurons from control and HD94 mice grown over a wild-type ratastrocyte monolayer. Higher magnification fields show the double membrane and clear content of the cytosolic vesicles. Top right: number of vacuoles per cell profile (13-17 cell profiles in triplicate experiments; *P 0.05). Arrows, enlarged electron-clear vesicles. (b) Electron micrographs of striatal cells from mutant huntingtin knock-in (111Q-htt) and wild-type (7Q-htt) mice. (c) Electron micrographs of lymphoblasts from unaffected controls (UC) and people with HD, maintained in the presence or absence of serum. Arrows, enlarged electron-clear vesicles. (d) Immunogold for LC3 in striatal cells from mutant huntingtin knock-in mice (111Q-htt), striatal neurons from HD94 mice and lymphoblasts (lymph) from people with HD. Full fields and more details of autophagic vacuoles in Supplementary Figures 10 and 11. \n",
      "a) Immunoblot for the indicated proteins in fractions enriched in APHs and autophagolysosomes (APHL) isolated from livers of 18Q-htt (18Q) and 111Q-htt (111Q) mice.figf5(a) Immunoblot for the indicated proteins in fractions enriched in APHs and autophagolysosomes (APHL) isolated from livers of 18Q-htt (18Q) and 111Q-htt (111Q) mice. Cyt, cytochrome; Dyn IC, dynein intermediate chain; ADRP, adipose differentiation-related protein; Polyubq, polyubiquitin. Bottom: amount of each protein in 111Q-htt sample, as a multiple of its amount in 18Q-htt sample. Mean + s.d.; n = 4. (b) Immunoblots for htt and LC3 in homogenates (Homog), cytosol and fractions enriched in APHs and enriched in autophagolysosomes (APHL) isolated from livers of wild-type (18Q-htt) and mutant huntingtin knock-in mice (111Q-htt). Representative one of four experiments with duplicated samples. *P 0.05. Full-length blots in Supplementary Figure 20. \n",
      "a) Immunoblot for htt or p62 from total APHs (T) and their corresponding membranes (Mbr) and matrices (Mtx) isolated from livers of wild-type htt (18Q) and 111Q-htt mutant huntingtin knock-in mice (111Q).figf6(a) Immunoblot for htt or p62 from total APHs (T) and their corresponding membranes (Mbr) and matrices (Mtx) isolated from livers of wild-type htt (18Q) and 111Q-htt mutant huntingtin knock-in mice (111Q). Left, representative immunoblots. Right, distribution of htt and p62 between Mbr and Mtx calculated by densitometric quantification in six such immunoblots. AV, autophagic vacuoles. Mean + s.d. *P 0.05 compared to wild-type values. (b) Filter retardation analysis of the same fractions as in a, blotted for htt. Negative signal indicated absence of aggregates retained in the filter. (c) Immunoblot for ubiquitin in homogenates (Homog), cytosol and the autophagic fractions described in a, using 6% (top) and 16% (bottom) gels. Dotted line separates stacking and running parts of gel. (d) Immunoblot for htt, LC3 and p62, of brain homogenates (Homog) and fractions enriched in APHs from the same mouse groups as in a. (e) Immunoblot for htt and p62 of membranes from autophagic vacuoles shown in d subjected to co-immunoprecipitation for htt under low stringency. Input for 111Q-htt autophagic vacuoles was one-third of that for 18Q-htt to avoid antibody saturation. Levels of htt (top) and p62 (bottom) in the input, immunoprecipitate (IP) and flow through (FT) are shown. Full-length blots in Supplementary Figure 21. \n",
      ": Destabilizing mutations in DHFR induce filamentous phenotype due to loss of DHFR activity.Figure 1Growth rates of mutant DHFR strains at 30°C, 37°C and 42°C. While most mutants grow well at 30°C, they grow very poorly at high temperatures. Error bars represent SEM of three biological replicates. Live cells DIC images and DAPI nucleoids staining of WT DHFR and I91L+W133V DHFR strains after being grown at 30°C, 37°C, or 42°C (I91L+W133V was grown at 40°C) in M9 medium supplemented with amino acids for 4 hours (see Methods). Cell lengths were measured from the obtained DIC images (see Methods) and their distribution at 37°C and 40°C/42°C is shown in (C) and (D) as box-plots (see Methods). Images of other mutant DHFR strains W133V, V75H+I155A and V75H+I91L+I155A are presented in related Figure EV1A. Distribution of cell lengths of WT and mutants W133V and V75H+I155A at 42°C (gray box) and I91L+W133V at 40°C (represented as white box) after being grown in M9 minimal medium (without amino acids) for 4 hours. Median cell length of W133V and V75H+I155A is significantly smaller than WT (Mann-Whitney test, p-value <0.001). Functional complementation of WT DHFR from an arabinose inducible pBAD plasmid rescues filamentation of mutant strains grown at 42°C (for WT, W133V and V75H+I155A strains) or at 40°C (for I91L+W133V and V75H+I91L+I155A) in M9 medium supplemented with amino acids. For the boxplots in panels , F the central band represents the median of the distribution, the box ends represent the 25th and 75th percentile, the whiskers represent the 10th and 90th percentile, while the dots represent the 5th and 95th percentile. Data was usually obtained from 2-3 biological replicates. The number of cells used to derive the boxplot distributions in the different panels range usually between 200 to 450 (please refer to Figure 1 source data for exact number of cells for each dataset). See related Figure EV1B. Functional complementation of WT DHFR from an arabinose inducible pBAD plasmid rescues growth defects of mutant strains grown at 42°C (for WT, W133V and V75H+I155A strains) or at 40°C (for I91L+W133V and V75H+I91L+I155A) in M9 medium supplemented with amino acids. Expression of mutant proteins from pBAD plasmid on the WT background does not result in filamentation For the boxplots in panels the central band represents the median of the distribution, the box ends represent the 25th and 75th percentile, the whiskers represent the 10th and 90th percentile, while the dots represent the 5th and 95th percentile. Data was usually obtained from 2-3 biological replicates. The number of cells used to derive the boxplot distributions in the different panels range usually between 200 to 450 (please refer to Figure 1 source data for exact number of cells for each dataset). See related Figure EV1B. Expression of mutant proteins from pBAD plasmid on the WT background does not result in growth defects. \n",
      "a) Top: immunofluorescence for the mitochondrial marker COX IV in 18Q-htt and 111Q-htt MEFs maintained in the presence or absence of serum.figf8(a) Top: immunofluorescence for the mitochondrial marker COX IV in 18Q-htt and 111Q-htt MEFs maintained in the presence or absence of serum. Bottom: number of mitochondria per cell. Mean + s.d. of 10-20 cells in three different experiments. (b) Electron micrographs of striatal neurons from wild-type (Ctr) and HD94 mice grown over a wild-type ratastrocyte monolayer. Arrows, abnormally short (black) or abnormally long (green) mitochondria. Right: number of mitochondria per cell profile. Mean + s.d. of ten cells per group in triplicates. (c) Left: electron micrographs of lymphoblasts from controls (UC) or people with HD. Arrows as in b. Right: percentage of cellular area occupied by mitochondria in different individuals. Line indicates population mean. (d,e) Striatal cells from 7Q-htt and 111Q-htt mice (d) stained with Mitotracker and Mito-ROS. Right, merged images. Percentage of colocalization is indicated at the bottom in d MEFs from 18Q-htt and 111Q-httmice (e) stained with Mitotracker and Mito-ROS. Right, merged images. Percentage of colocalization is indicated at the bottom in d and is displayed in the graph at the bottom in e. CCCP was added to control cells in e as a positive control for depolarization. *P 0.05. (f,g) Striatal cells from 7Q-htt and 111Q-htt mice untreated or treated with vinblastine were co-stained for LC3 and Mitotracker (f). Arrows, colocalization events. Extended study in Supplementary Figure 16. (f,g) Striatal cells from 7Q-htt and 111Q-httmice untreated or treated with vinblastine were co-stained for LC3 and BODIPY 493/503 (g). Arrows, colocalization events. Extended study in Supplementary Figure 16. \n",
      ": Metabolomics of mutant DHFR strains in minimal media without or with added amino acids.Figure 2(A) and (B) shows abundance of selected nucleotides and amino acids for mutants I91L+W133V and W133V as well as WT strain treated with 0.5μg/ml of Tmp (Trimethoprim) after 4 hours of growth at 40/42°C in M9 minimal medium (condition of no filamentation), Concentration of all metabolites were normalized to WT levels at 4 hours when grown under similar conditions. In minimal medium (B) Methionine levels are extremely low Levels of purines (IMP, AMP) are also largely rescued with amino acid supplementation, however dTMP, dTDP and dTTP levels remain extremely low. Error bars represent SEM of at least three biological replicates (see Methods). The dashed line in each plot represents value of 1 for WT. (C) and (D) represents nucleotide and amino acid abundances after 4 hours of growth in amino acid supplemented M9 medium at 40/42°C (condition of filamentation). Concentration of all metabolites were normalized to WT levels at 4 hours when grown under similar conditions. Methionine levels are extremely low, which recover largely in panel (D). Levels of purines (IMP, AMP) are also largely rescued with amino acid supplementation, however dTMP, dTDP and dTTP levels remain extremely low. Error bars represent SEM of at least three biological replicates (see Methods). The dashed line in each plot represents value of 1 for WT. \n",
      "a) Neutral lipids in MEFs from 18Q-htt and 111Q-htt mice stained with BODIPY 493/503.figf7(a) Neutral lipids in MEFs from 18Q-htt and 111Q-htt mice stained with BODIPY 493/503. (b) Electron micrographs of livers from 18Q-htt and 111Q-htt mice. Right: number of lipid droplets (LD) per cell profile (prof.), mean area of LDs and percentage of cellular area occupied by LDs. Mean + s.d.; n = 3. (c-e) Neutral lipids in striatal cells from 7Q-htt and 111Q-httmice (c), grown on a monolayer of their own astrocytes (c-e) Neutral lipids in primary striatal neurons from 18Q-htt and 111Q-httmice grown on a monolayer of their own astrocytes (d) (c-e) Neutral lipids in lymphoblasts from a control (UC) and HD-affected human (e) stained with BODIPY 493/503. MAP2 staining highlights neurons. Extended study in Supplementary Figure 15b. (f) Electron micrographs of a lymphoblast from an individual with HD (with 78 and 15 polyglutamine repeats in the two alleles). Green arrows, LDs. (g) Fraction of cellular cytosol occupied by LDs quantified in lymphoblasts from three UC and four HD subjects. Numbers of polyglutamine repeats are shown below. Mean + s.d. of >100 cell profiles. Dotted red line, mean value of the respective samples. *P 0.05. (h) Oil Red O staining of striatal tissue from brain of two different unaffected control individuals (top) and two individuals with HD (bottom). Nuclei are highlighted with hematoxylin. Lipid droplets are visible as red puncta. Subject identifiers are shown in each panel. (i) The percentage of total cellular area occupied by lipid droplets (left) and the average number of lipid droplets per cell (right) was calculated for each of the samples by quantification of eight or nine different fields. *P 0.001; analysis of variance, P 0.01 for each person with HD versus each control individual by Tukey post hoc test. Subject identifiers are as in h. \n",
      ": Filamentation in mutant DHFR strains is associated with strong SOS response.Figure 3(A) recA (B) recN genes measured by quantitative PCR when WT and mutant strains are grown in M9 medium with or without supplementation of amino acids or dTMP. WT and mutant strains were grown for 4 hours of growth in the indicated medium at 37°C, while WT treated with different concentrations of Tmp were grown for 4 hours at 42°C. Brown bars (M9+AA) in the gray shaded area correspond to filamentation conditions and these are associated with pronounced upregulation of all three SOS genes. On the other hand, conditions with loss of filamentation (with dTMP or no supplementation) show much less expression. Error bars represent SD of 2-3 biological replicates (see Methods). sulA genes measured by quantitative PCR when WT and mutant strains are grown in M9 medium with or without supplementation of amino acids or dTMP. WT and mutant strains were grown for 4 hours of growth in the indicated medium at 37°C, while WT treated with different concentrations of Tmp were grown for 4 hours at 42°C. Brown bars (M9+AA) in the gray shaded area correspond to filamentation conditions and these are associated with pronounced upregulation of all three SOS genes. On the other hand, conditions with loss of filamentation (with dTMP or no supplementation) show much less expression. Error bars represent SD of 2-3 biological replicates (see Methods). Treatment of WT E. coli cells with sub-MIC concentration of Tmp (0.5μg/ml) leads to filamentation at 42°C when grown in amino acid supplemented medium. However, a recA knock-out strain under similar condition shows no elongation, indicating the role of SOS pathway in filamentation. A sulA knock-out continues to elongate, indicating the role of sulA-independent pathways. An E. coli strain deleted for the e14 prophage region however showed no filamentation upon Tmp treatment, indicating that sfiC gene in the e14 region might be one such sulA independent player. The central band in the box plots represents the median of the distribution, the box ends represent the 25th and 75th percentile, the whiskers represent the 10th and 90th percentile, while the dots represent the 5th and 95th percentile. Data was usually obtained from 2-3 biological replicates. The number of cells used to derive the boxplot distributions in the different panels range usually between 100 to 200 (please refer to Figure 3 source data for exact number of cells for each dataset). \n",
      ": Filamentation in mutant DHFR strains is completely reversible and does not represent TLD.Figure 4Mutant W133V was grown in amino acid supplemented M9 medium (M9+AA) for 4 hours at 42°C, and subsequently placed on M9 agar pads and their growth was monitored at room temperature. Shown are phase contrast images taken from different time points throughout the time-lapse experiment. Unlike cells experiencing TLD, an irreversible phenomenon, W133V DHFR cells recover and resume growth at low temperature. WT cells were treated with different concentrations of Tmp at 42°C for varying amounts of time in amino acid supplemented M9 medium (panel B) or in M9 media supplemented with both amino acids and GMP (panel C), following which they were spotted on M9+AA plates and allowed to grow at 30°C. Colonies were counted next day. In the presence of only amino acids, there was no loss in viability for any concentration of Tmp (panel B), despite extensive filamentation (Figure EV1E). In contrast, in the presence of amino acids and GMP, the cells showed sharp loss in viability when grown at high Tmp concentrations. In both panels, error bars represent SD of two biological replicates. The dashed line represents cfu at one-hour timepoint. WT (with and without 5μg/ml Tmp) and mutants were grown for 6 hours at 42°C in M9+AA medium without or with GMP, and subsequently diluted serially and spotted on M9+AA agar plates and allowed to grow at 30°C till visible colonies were formed. While WT treated with high Tmp shows loss in cfu, indicating TLD, no loss in viability was observed for WT or mutants. Expression of SOS response genes recA, recN and sulA when mutant strains and Tmp treated WT cells are grown in M9 medium supplemented with amino acids (black bars) or in the presence of both amino acids and GMP (gray bars) at 42°C (I91L+W133V was grown at 40°C). Error bars represent SD of 2-3 biological replicates. \n",
      ": Supplementation of dTMP alleviates dTDP/dTTP levels and rescues filamentation.Figure 5Distribution of cell length of WT and mutant strains when grown in M9 medium supplemented with amino acids (gray) or with both amino acids and 1mM dTMP (pink). WT, W133V and V75H+I155A strains were grown at 42°C while I91L+W133V and V75H+I91L+I155A mutants were grown at 40°C. 1mM dTMP largely rescues filamentation of mutant strains (* indicates the median cell lengths were significantly different, Mann-Whitney test, p-value <0.001). The central band in the box plots represents the median of the distribution, the box ends represent the 25th and 75th percentile, the whiskers represent the 10th and 90th percentile, while the dots represent the 5th and 95th percentile. Data was usually obtained from 2-3 biological replicates. The number of cells used to derive the boxplot distributions in the different panels range usually between 150 to 300 (please refer to Figure 5 source data for exact number of cells for each dataset). Abundances of selected nucleotides in I91L+W133V mutant when grown for 4 hours at 40°C in M9 medium supplemented with both amino acids and 1mM dTMP. Metabolite levels were normalized to those of WT grown under similar conditions. dTDP and dTTP levels recover and are now comparable to dTMP levels. Error bars represent SEM of 3 biological replicates. The dashed line in represents value of 1 for WT. Mutant V75H+ I91L+I155A grows very poorly (in terms of colony forming units, cfu) on a minimal media agar plate supplemented with amino acids (M9+AA) at 40°C. Supplementation of additional dTMP increases the cfu by several orders at the same temperature, while supplementation with both pyrimidine (dTMP) and purine (GMP) allows it to grow as good as WT. In comparison, WT was grown at 42°C under different supplementation conditions. In all cases, cultures were 7-fold serially diluted for the next spot. The three rows (two rows for WT) in each condition represent biological replicates. Comparison of growth rates of WT and mutant DHFR strains at 42°C (40°C for I91L+W133V and V75H+I91L+I155A) in minimal medium that is supplemented with amino acids and/or 1mM dTMP. Except for W133V and to a lesser extent for V75H+I155A, the effect of dTMP on growth rates is only modest. Error bars represent SEM of 3 biological replicates. \n",
      ": In vivo enzyme kinetics of Tmk is highly cooperative and different from that in vitro.Figure 6in vitro activity of purified Thymidylate Kinase (Tmk) as a function of dTMP concentration shows Michaelis-Menten (MM) like kinetics. ATP concentration is saturating at 1mM. The KM of dTMP is 13μM. The error bars represent SD of three technical replicates. For a 10-fold drop in intracellular dTMP concentration in mutant relative to WT (as seen in I91L+W133V mutant), we calculate dTDP levels in mutant (relative to WT) as a function of various assumed intracellular concentrations of dTMP in WT (shown along x-axis), as absolute value of this is not known experimentally, assuming MM kinetics (black line) as shown in panel (A) or Hill kinetics with coefficient of 2.5 (red line) as shown in panel (C). The dotted line corresponds to the experimentally observed dTDP ratio of 0.01 for I91L+W133V mutant. This ratio is realized only for the Hill-like curve. Apparent in vivo activity kinetics of Tmk enzyme using steady state dTMP and dTDP levels obtained from metabolomics. The plot includes data from WT, mutants W133V and I91L+W133V, as well as WT treated with 0.5μg/ml Trimethoprim, obtained at different time points during growth. Data points represent metabolite levels for all individual biological replicates without averaging. The black data points were acquired during growth in the presence of different concentrations of external dTMP (0.25, 0.5, 1, 2 and 5mM), while red points were from conditions with no external dTMP. Both black and red solid lines represent fit to Hill equation. The green points were acquired during growth of WT, W133V and I91L+W133V mutants in the presence of different concentrations of thymidine, which follow the red curve. The inset plot shows the same graph with selected datapoints for I91L+W133V mutant. The cyan circle shows I91L+W133V mutant in the absence of any metabolite supplementation, while red and black points indicate metabolite levels following thymidine and dTMP supplementation respectively. A flow chart summary of the actual chain of events triggered by destabilizing mutations in DHFR that eventually lead to bacterial filamentation, as revealed in this study. \n",
      ": In vivo activity curves of representative E.Figure 7Steady state levels of different metabolites obtained from metabolomics were used to derive activity curves for (A) Adenylate Kinase (AK) (B) Uridylate Kinase (PyrH) (C) Adenylosuccinate Lyase (PurB) and (D) GAR Transformylase (PurN). AK, PyrH and PurB appear to follow hyperbolic MM kinetics in vivo, while PurN shows Hill-like dependence with cooperativity. In panel (A), the red data points were acquired during growth in amino acid supplemented M9 medium in the absence of external AMP and are derived from WT and various stabilizing and destabilizing point mutations of AK (Adkar et al., 2019) as well as DHFR mutants used in the present study, while black points were derived exclusively from cultures of AK mutants supplemented with 1mM external AMP (Adkar et al., 2019). Data points for all other panels (B-D) are derived from mutant DHFR strains in the present study. Since SAICAR, AICAR, GAR and FGAR levels were undetectable in WT, all data points in (C-D) are relative to those observed in I91L+W133V mutant following 4 hours of growth at 40°C. Data points in all panels represent metabolite levels for individual biological replicates without averaging. \n",
      ".Figure 1Schematic representation of BioID proximity biotinylation in HeLa-T-REx cells stably expressing H3.1-BirA*-HA. Randomly cycling cells were treated with biotin to label proteins proximal to nucleosomal (nucH3.1) or non-nucleosomal soluble (solH3.1). H3.1-BirA*-HA, and biotin-labeled proteins were purified on streptavidin-conjugated beads. Representative images of parental cells or cells stably expressing H3.1-BirA*-HA after incubation with or without biotin as in (A). Coverslips were pre-extracted to visualize chromatin-bound proteins. DNA is visualized via DAPI staining shown in cyan, immunofluorescence for H3.1-BirA*-HA is shown in green, and biotinylated proteins are shown in red. Scale bar = 5 μM. Representative images of HeLa cells pre-extracted as in (B). DNA is visualized as in (B). Immunofluorescence for endogenous H3 is shown in green. Scale bar = 10μM. Immunoblot analysis of proteins biotinylated and isolated as in (A). Parental HeLa-T-REx cells or cells expressing H3.1-BirA*-HA and treated with biotin for indicated times. Immunoblot analysis of TAP-H3.1 or -CENP-A pulldown of UBR7-LAP (schematic, upper). 293T cells were co-transfected with UBR7-LAP and TAP-tagged constructs, which were purified on IgG-conjugated sepharose beads. \n",
      ".Figure 2Schematic representation of cellular fractionation to generate non-nucleosomal and nucleosomal fractions. WCE: whole cell extract, NF1: nuclear fraction 1, NF2: nuclear fraction 2. Representative immunoblot analysis of NASP, UBR7, and ASF1 distribution in cellular fractions. Relative loading factors are marked e.g. 1x, 0.2x, etc. Numbers below blot indicate UBR7 immunoblot signal relative to 1x loading of non-nucleosomal fraction. Enrichment of non-nucleosomal:nucleosomal UBR7 as estimated from 5x, 10x, and 20x non-nucleosomal loading conditions in Fig 2B (see Fig EV2). Error bars represent mean +/- standard deviation. A two-tailed one sample t-test was used to calculate p-value (italics) associated with mean difference from a hypothetical value of 1.0 (represented by dashed line). Immunoblot of anti-FLAG IP of 3x-FLAG tagged GFP, or FLAG-HA tagged H3.1 or H2B, from the non-nucleosomal fraction of 293T cells. Representative images of HeLa cells transfected with UBR7-FL. DNA is visualized via DAPI staining shown in magenta and immunofluorescence for 3xFLAG is shown in green. The area within yellow dotted box in (E) is magnified 4x in (E'). Scale bar = 5 μm.   Quantification of the nuclear anti-FLAG immunofluorescence intensity in (E). Error bars represent mean +/- standard deviation. One way ANOVA was used to calculate p-values (italics) associated with differences between treatments. Experiment in (E-F) was performed twice.   \n",
      ".Figure 3Immunoblot analysis of anti-H3.1-HA pulldown of UBR7-LAP after deletion of H3.1 N-terminal tail (schematic, upper). 293T cells were co-transfected with UBR7-LAP and HA-tagged constructs. Numbers below blot indicate HA-normalized anti-GFP immunoblot signal relative to full-length H3.1-HA. Representative images of HeLa cells transfected with indicated constructs. DNA is visualized via DAPI staining shown in cyan and immunofluorescence for endogenous H3 is shown in green. Scale bar = 10 μm. Experiment was performed three times. Immunoblot analysis of anti-FLAG pulldown of full length UBR7-FLAG (residues 1-425) mutant UBR7 bearing deletion of UBR7 C-terminus (∆C-term, residues 1-215), UBR box (∆UBR, residues 117-425), UBR box and PHD (∆UBR∆PHD, residues 216-425), or PHD and C-terminus (∆PHD∆C-term, residues 1-116). 293T cells were co-transfected with H3.1-HA and UBR7-FLAG constructs (schematic, upper). Asterisk represents non-specific band. Numbers below blot indicate FLAG-normalized anti-HA immunoblot signal relative to full length UBR7-FLAG. Immunoblot analysis of anti-H3.1-HA pulldown of full-length UBR7-LAP and UBR7PHD fragment (residues 117-215) (schematic, upper). 293T cells were transfected as in (A). Values below pulldown represent H3.1-HA-normalized GFP-UBR7 immunoblot signal relative to UBR7-FL. Data were acquired from a continuous membrane, and the image is cropped to omit irrelevant samples. Numbers below blot indicate HA-normalized anti-GFP immunoblot signal relative to full length UBR7-LAP. \n",
      ".Figure 4Histone peptide array identified modified histone H3 N-terminal peptides bound by purified His-MBP-UBR7. Boxplots represent Z-scores of the local background corrected peptide intensity values on the array. Box limits represent 25th and 75th percentiles. Whiskers extend to 1.5x the interquartile range. Experiment was performed twice. Left: Peptides (un) modified at H3K4. Right: Peptides (un) modified at H3K9 and H3S10. Anti-GFP immunoblot analysis of streptavidin pulldown of biotin-conjugated H3 and H4 peptides following incubation with 293T-GFP-UBR7 cell lysate. HP1α serves as a positive control for binding to H3K9me3. Predicted structure of UBR7 PHD (a.a.133-190) using Phyre2 (Kelley et al, 2015) overlayed on the BPTF PHD2 structure (RSCB PDB: 2F6N) as reference. UBR7 PHD aromatic cage residues mutated in (D) and corresponding residues in BPTF PHD2 are labeled. Anti-GFP immunoblot analysis of streptavidin pulldown of the indicated biotin-conjugated H3 peptides following incubation with 293T-GFP-UBR7 cell lysate after mutation of residues highlighted in (C). Immunoblot analysis for total and modified histones in anti-FLAG pulldown of constructs from soluble cellular fractions demonstrates that UBR7 UBR box and PHD are necessary for binding to post-nucleosomal H3/H4. Quantification of the immunoblot data in (E). Immunoblot signals were normalized to anti-FLAG signal. Error bars represent mean +/- standard deviation. A two-tailed one sample t-test was used to calculate p-values (italics) associated with group means difference from a hypothetical value of 1.0 (represented by dashed line). Schematic representation of proximity biotinylation in stable inducible H3.1-APEX2-3xFLAG cells. To perform proximity biotinylation with newly synthesized H3.1-APEX2-3xFLAG, after doxycycline induction followed by washout, cells were immediately subjected to labeling reaction. To perform proximity biotinylation with pre-existing H3.1-APEX2-3xFLAG, following washout cells were grown for 48 h before labeling reaction. Biotin-labeled proteins were purified on streptavidin-conjugated beads. Immunoblot analysis of streptavidin pulldown from proximity biotinylation described in (G). 4.5 h represent newly synthesized histones and 48 h represent pre-existing histones. Wedge represents loading titration: 1x > 0.1x > 0.01x > 0.001x. All other lanes are loaded at 1x concentration. Immunoblot signals were normalized to anti-FLAG signal.   Quantification of anti-FLAG (H3.1-APEX2-3xFLAG) and anti-UBR7 immunoblot represented in (G-H). Lanes 8 and 11 were quantified. Error bar represents mean +/- standard deviation. A two-tailed one-sample t-test was used to calculate p-value (italics) associated with group mean difference from a hypothetical value of 1.0 (represented by dashed line).   \n",
      ".Figure 5Immunoblot analysis of parental HeLa cells or cells expressing UBR7-BirA*-HA confirms specific expression of UBR7-BirA*-HA. Representative images of parental cells or cells stably expressing UBR7 fused to BirA* ligase and HA tag (UBR7-BirA*-HA) after incubation with or without biotin. DNA is visualized via DAPI staining shown in cyan, immunofluorescence for UBR7-BirA*-HA is shown in green, and biotinylated proteins are shown in red. Scale bar = 5 μm. Quantification of log2 fold change and -log10 p-value of MS spectral counts from UBR7-BirA*-HA (heavy) compared to parental (light) cells. Positive log2 fold changes are presented. Full dataset displayed as inset bordered by dotted line. The two proteins most abundantly labeled by UBR7-BirA*-HA are UBR7 and NASP. Experiment was performed twice. Immunoblot analysis of anti-FLAG IP. Cells were fractionated 72 h post-transfection, and 3xFLAG constructs were immunoprecipitated from non-nucleosomal fraction and eluted with FLAG peptide. Immunoblot analysis of anti-FLAG IP. Immunoblot signals were normalized to anti-FLAG signal. Experiment was performed as in (D). Asterisk indicates non-specific bands in input. Quantification appears on the right. A two-tailed one sample t-test was used to calculate p-value (italics) associated with group mean difference from a hypothetical value of 1.0 (represented by dashed line). Experiment was performed twice. Immunoblot analysis for total and modified H3 and H4 in anti-FLAG IP. Experiment was performed as in (D). Dot plot error bars represent mean +/- standard deviation. Immunoblot signals were normalized to anti-H4 signal. A two-tailed one sample t-test was used to calculate p-value associated with group mean difference from a hypothetical value of 1.0. Experiment was performed twice. Representative images of HeLa cells transfected with sNASP-FL. DNA is visualized via DAPI staining shown in magenta and immunofluorescence for 3xFLAG is shown in green. Scale bar = 5 μm.   Quantification of the nuclear anti-FLAG immunofluorescence intensity in (G) demonstrating that sNASP-FL is highly susceptible to pre-extraction with Triton X-100. One way ANOVA was used to determine p-values (italics) associated with differences between treatments. Error bars represent mean +/- standard deviation. Experiment in (G-H) was performed twice.   \n",
      ".Figure 6A,C. Immunoblot analysis for modified histone in anti-FLAG pulldown of UBR7-FL (A) or sNASP-FL (C) from non-nucleosomal fractions of untreated cells or cells treated with DNA replication inhibitors (hydroxyurea, aphidicolin) or transcription inhibitors (α-amanitin, triptolide). B,D. Plots represent either normalized immunoblot signals or cell counts prior to fractionation. Immunoblot signals were normalized to anti-FLAG signal. Error bars represent mean +/- standard deviation, with grey dotted line marking mean of No Treat condition. A two-tailed one sample t-test was used to calculate p-values (italics) associated with group means difference from a hypothetical value of 1.0. Experiments were performed three times. Cells numbers were counted in two of the three experiments. \n",
      ".Figure 7Immunoblot analysis of H3, UBR7, and GAPDH in the non-nucleosomal fraction of parental and 293T-UBR7KO cells.   Quantification of anti-H3 immunoblot represented in (A). Immunoblot signals were normalized to anti-GAPDH signal. Error bars represent mean +/- standard deviation. A two-tailed one-sample t-test was used to calculate p-value (italics) associated with group mean difference from a hypothetical value of 1.0 (represented by dashed line). Experiment in (A-B) was performed in three times.   Immunoblot analysis of anti-FLAG pulldown of sNASP-FL in parental or 293T-UBR7KO cells. 3xFLAG tagged sNASP-FL was immunoprecipitated from non-nucleosomal fraction and eluted with FLAG peptide. Experiment was performed three times.   Quantification of anti-FLAG (sNASP-FL), and anti-histone immunoblots represented in (C). Immunoblot signals were normalized to anti-FLAG signal. Error bars represent mean +/- standard deviation. A two-tailed one sample t-test was used to calculate p-values (italics) associated with group means difference from a hypothetical value of 1.0 (represented by dashed line). Experiments were performed two (sNASP, H4) to three (all other targets) times.   \n",
      ".Figure 8Changes in LAMP2A expression. RT-qPCR was performed to detect actin and LAMP2A mRNA following treatement with siCTRL or siLAMP2A in sNASP-FL and UBR7-FL 293T cells. Grey dotted line represents expression level in siCTRL treatment. Error bar represents mean +/- standard deviation. A two-tailed one sample t-test was used to calculate p-values (italics) associated with group means difference from a hypothetical value of 1.0. Experiment was performed twice. Immunoblot for LAMP2A in parental 293T cells or cells expressing UBR7-FL (left) or sNASP-FL (right) following treatment with siCTRL, siLAMP2A, or MG-132. Immunoblot for total and modified histones following anti-FLAG IP from the soluble fraction of parental 293T cells, or sNASP-FL treated with siCTRL, siLAMP2A, or MG-132. D. Immunoblot for total and modified histones following anti-FLAG IP from the soluble fraction of UBR7-FL 293T cells treated with siCTRL, siLAMP2A, or MG-132. Quantification of data represented in (C, D). Error bars represent mean +/- standard deviation. A two-tailed one sample t-test was used to calculate p-values (italics) associated with group means difference from a hypothetical value of 1.0 (represented by dashed line). Experiments in (C-E) were performed three times. \n",
      ".Figure 10Model of UBR7 function. In the absence of UBR7, sNASP-bound H3K4me3 is not deposited, and total non-nucleosomal H3 and sNASP-bound H3K4me3 accumulate, leading to depleted H3K4me3 at UBR7 binding sites in chromatin. \n",
      ".Figure 9Heatmaps of ATAC-seq, anti-GFP (GFP-UBR7) ChIP-seq, and anti-H3K4me3 CUT&RUN in parental and UBR7KO cells. Plots are centered on TSS that­­­­­ show changes in ATAC-seq signal in UBR7KO versus parental cells. Sites are clustered by unsupervised k-means analysis. Cluster color key is defined by cluster labels. Representative genome track examples of clusters 1-4 as defined in (A). Antibody 1 H3K4me3 data are shown. C, D. Metaplots (left) of mean values per bin centered on all TSS, TSS not bound by GFP-UBR7, and GFP-UBR7 bound TSS. Dot plots (right) represent H3K4me3 (Antibody 1) from parental 293T cells (C) or loss of H3K4me3 in UBR7KO versus parental cells (D). Box limits represent 25th and 75th percentiles. Whiskers extend to 1.5x the interquartile range. Outliers are represented as black dots. A Kruskal-Wallis multiple comparisons test with Dunn post-hoc test was used to calculate Benjamini-Hochberg adjusted p-values (italics). Values p<0.05 are pictured. Experiment was performed with two independent anti-H3K4me3 antibodies \n",
      ".Figure 2(A) Schematic of the ANKRD26 domain identified as PIDD1 interactor by yeast-two-hybrid screen. PMID = PIDD1 Minimal Interaction Domain; CCDC144C-like = Coiled-Coil Domain similar to CCDC144C; DUF: Domain of Unknown Function. (B) Representative fluorescent micrographs of RPE1 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). Cells were co-stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. (C) Dot plot showing the average pixel intensities of the Myc signal at individual parent centrioles in RPE1 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). Mean values (red lines) ± s.e.m. are reported. N > 50 centrosomes were assessed for each condition in as many individual cells; a.u. = arbitrary units. Kruskal-Wallis test (****P < 0.0001; n.s. = non-significant). (D) Dot plot showing the average pixel intensities of the PIDD1 signal at individual parent centrioles in RPE1 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). Mean values (red lines) ± s.e.m. are reported. N > 50 centrosomes were assessed for each condition in as many individual cells; a.u. = arbitrary units. Kruskal-Wallis test (****P < 0.0001; n.s. = non-significant). (E) Representative fluorescent micrographs of RPE1 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). Cells were co-stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. \n",
      ".Figure 1(A) 2D STED micrographs of RPE1 cells co-stained with the indicated antibodies. ­Scale bar: 200 nm. (B) Dot plots showing the average pixel intensities at individual parent centrioles expressed as the PIDD1/ODF2 fluorescence ratio in the indicated cell lines and genotypes. Mean values (red lines) ± s.e.m. are reported. N ≥ 50 centrosomes were assessed for each condition, a.u. = arbitrary units. Unpaired Mann-Whitney test (****P < 0.0001). (C) Representative fluorescence micrographs from the indicated cell lines co-stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. (D) Co-localization between ODF2 and PIDD1 (n = 11), FBF1 and PIDD1 (n = 21), and ANKRD26 and PIDD1 (n = 17) assessed on images The value represents the fraction PIDD1 objects touching either ODF2, FBF1 or ANKRD26 objects. (E) 2D STED micrographs of RPE1 cells co-stained with the indicated antibodies, scale bar: 200 nm. (F) Representative fluorescence micrographs of RPE1 cells of the indicated genotypes co-stained with indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. (G) Dot plot showing the average pixel intensities at individual parent centrioles expressed as the PIDD1/CEP128 fluorescence ratio in RPE1 of the indicated genotypes. Mean values (red lines) ± s.e.m. are reported. N ≥ 50 centrosomes were assessed from as many individual cells for each condition, a.u. = arbitrary units. Statistical significance was assessed by Kruskal-Wallis test, comparing each sample to the wild type (****P < 0.0001). (H) Scheme summarizing the epistatic interdependencies between DAPs emerging from work displayed in this Figure. \n",
      ".Figure 3(A) RPE1 cells of the indicated genotypes were treated either with DMSO or with DHCB for 24h. A fraction of DHCB treated cells were released into fresh medium for other 24h (release). Samples were subjected to immunoblotting; n = 3 independent experiments. (B) Fluorescence micrographs of RPE1 cells of the indicated genotypes, either treated for 24h with DHCB or with vehicle alone (DMSO). Blow-up without Hoechst 33342 is magnified 2.5X. Scale bar: 5 μm. (C) Immunofluorescence micrographs of RPE1 as in (B) were used to visually assess the percentage of cells presenting one or two nuclei. N = 3, ≥ 50 cells from each independent experiment. Mean values ± s.e.m. are reported. The increase in the number of binucleated cells between the wild type sample and all the other genotypes was assessed (ANOVA test; *P < 0.05). (D) Immunofluorescence micrographs of RPE1 as in (B) were used to visual score the number of centrosomes per cell by counting the number of γ-tubulin-positive centrioles. N = 3, ≥ 50 cells from each independent experiment. Mean values ± s.e.m. are reported. The increase in the number of cells with > 2 centrosomes between the wild type sample and all the other genotypes was assessed (ANOVA test; **P < 0.01). (E) Quantitative assessment of the fraction of RPE1 cells undergoing cytokinesis failure upon DHCB treatment, inferred on the basis of the increase of ploidies ≥ 4C (upper panel), and of the fraction of the abovementioned cells undergoing genome reduplication upon release after DHCB treatment (lower panel). Individual values of biological replicates, their mean and standard deviations are reported. (F) RPE1 cells were either left untransduced (mock) or transduced with lentiviral vectors expressing the indicated Myc-ANKRD26 constructs. Subsequently, cells were treated either with DMSO or ZM447439 for 24h and subjected to immunoblotting; n = 3 independent experiments. \n",
      ".Figure 5(A) A549 cells of the indicated genotypes were either left untransduced or transduced with a lentiviral vector expressing PIDD1-V5 in its wild type form. Cells were treated either with DMSO or with ZM447439 for 24h and subjected to immunoblotting. N = 2 independent experiments. (B,C) A549 cells of the indicated genotypes were either left untransduced (mock), or transduced with lentiviral vectors expressing PIDD1-V5 in its wild type form or carrying the indicated point mutations. Cells were treated either with DMSO or with ZM447439 for 24h and subjected to immunoblotting. N = 2 independent experiments. (D) Fluorescence micrographs of A549 cells of the indicated genotypes. Cells were either left untransduced (mock) or transduced with PIDD1-V5 lentiviral vectors carrying the indicated point mutations, and stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. \n",
      ".Figure 4(A) Schematic of the PIDD1 domain structure and of the different PIDD1 species generated by autoproteolysis. LRR = Leucin Rich Repeat Domain; ZU5 (N and C) domains = Domain present in ZO-1, Unc5-like netrin receptors and in ankyrins; UPA domain = conserved in UNC5, PIDD, and Ankyrins; DD = death domain. (B) A549 cells of the indicated genotypes were either left untransduced (mock) or transduced with lentiviral vectors expressing the indicated PIDD1-V5 derivatives and subjected to immunoblotting. N = 2 independent experiments. (C) Fluorescence micrographs of A549 cells of the indicated genotypes either left untransduced (mock), or transduced with PIDD1-V5 lentiviral vectors and co- stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. (D) Dot plot showing the average V5 pixel intensities at individual parent centrioles in A549 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). N > 50 centrosomes were assessed for each condition in as many individual cells; a.u. = arbitrary units. (E) Dot plot showing the average V5 pixel intensities at individual parent centrioles in A549 cells of the indicated genotypes, either transduced with the indicated lentiviral vectors or left untransduced (mock). N > 50 centrosomes were assessed for each condition in as many individual cells; a.u. = arbitrary units. (F) Representative fluorescence micrographs of A549 cells of the indicated genotypes. Cells were either left untransduced (mock) or transduced with PIDD1-V5 lentiviral vectors expressing the PIDD1 non-cleavable derivative (PIDD1S446A-S588A) or truncations thereof. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. \n",
      ".Figure 7(A) Dot plot showing the distance between parent centrioles pairs in A549 cells following the indicated treatments (ZM = ZM447439; REV = reversine). Nocodazole concentrations are 0.03 μM, 0.1 μM, 0.33 μM, 1 μM, 3.3 μM. Median (red) and 95% confidence interval thereof (black) are shown. N > 50 cells were analysed. (B) Immunoblot of A549 cells subjected to the indicated treatments for 24h as in (A). N = 3 independent experiments. (C) DNA content analysis of A549 cells subjected to the indicated treatments as in (A) either for 24h (left panels) or 48h (right panels). N = 2 independent experiments. (D) Schematic of the experimental conditions utilized to synchronize RPE1 cells and to specifically interfere with centrosome clustering after telophase. (E) Immunoblots of RPE1 cells synchronized (F) Representative fluorescence micrographs of RPE1 cells synchronized Centrosomal antigens were stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2X. Scale bar: 5 μm. \n",
      ".Figure 6(A) Representative fluorescence micrographs across the indicated cell cycle phases from RPE1 cells stably expressing CETN1-GFP. Centrosomal antigens were stained with the indicated antibodies. Blow-ups without Hoechst 33342 are magnified 2.5X. Scale bar: 5 μm. The centrioles subjected to fluorescence intensity measurement in (B) are labelled with numbers. (B) Quantification of the PIDD1 average pixel intensity at individual centrioles across the indicated cell cycle phases of RPE1 cells stained N > 50 centrioles were assessed for each phase, a.u. = arbitrary units. (C) Fluorescence micrographs of RPE1 cells stably expressing CETN1-GFP treated either with DMSO or with DHCB for 24h. Blow-ups 33342 are magnified 2.5X. Scale bar: 5 μm. (D) Dot plot showing PIDD1 average pixel intensities at individual parent centrioles calculated from images as in (C). Mean values (red lines) ± s.e.m. are reported. N > 50 centrosomes were assessed for each condition, a.u. = arbitrary units. (E) Movie stills of a representative RPE1 cell stably expressing CETN1-GFP treated with DHCB and subjected to time-lapse video microscopy in the presence of SiR-DNA. Time is expressed in minutes, relative to anaphase onset. The dashed line indicates the plasma membrane of the cell of interest, arrowheads indicate the centrosomes' position. (F) Centrosomal distance over time in RPE1 cells stably expressing CETN1-GFP and treated with DHCB. Time zero corresponds to the frame preceding anaphase onset. Coloured dots (lower panel) summarize the clustering time for each cell, mean ± standard deviation in black. Data calculated from four-dimensional imaging as in (E). N = 10 cells. \n",
      ".Figure 8(A) A549 cells of the indicated genotypes were treated for 24h as indicated (CPT = camptothecin, ZM = ZM447439). Samples were subjected to immunoblotting; n = 3 independent experiments. (B) A549 cells treated as in (A) were subjected to fluorescence microscopy and centrosome abundance was assessed by visually scoring γ-tubulin-positive centrioles per cell. N = 3, ≥ 50 cells from each independent experiment. (C) RT-qPCR analysis of PIDD1 mRNA expression in A549 cells upon treatment with increasing doses of Nutlin 3a (i.e. 3.3 μM or 10 μM) using two independent probes. The average fold of induction ± standard deviation is shown. N = 3 biological replicates with two technical replicates each. Comparisons were performed between treatments of every genotype and the corresponding treatment of wild type (WT) cells with increasing doses of Nutlin 3a (i.e. 3.3 μM or 10 μM) using two independent probes. (D) Immunoblot analysis of samples treated N = 3 independent experiments. (E) Proposed model for the centrosome-dependent PIDDosome activation upon different stimuli. The centrosome constitutively acts as PIDD1 centralizer. A local increase in PIDD1 concentration (achieved either by centrosome clustering or upon p53 activation) triggers PIDDosome activation. In ANKRD26-deficient cells, the inability of the centrosome to generate a local increase in PIDD1 concentration hinders the activation of the complex in response to both stimuli. \n",
      ".Figure 1 A. RT-PCR of CNPase (left) and MBP (right) mRNAs expression in MO3.13 precursors under control conditions and following 10-200 μM D-Asp exposure for 3 days. Graphs show quantification of ratio of CNPase, and MBP to L19 Data information: The values represent the means ± S.E.M. Level of significance was determined by using: in A, left panel, one-way ANOVA P<0.0001 followed by Tukey's post hoc test, *P< 0.05 versus control (n=3); A, right panel, one-way ANOVA P<0.0001 followed by Tukey's post hoc test, *P<0.05 versus control (n=3  B. Western blotting (left) and densitometric analysis (right) of MBP expression in absence or in presence of 10-200 μΜ D-Asp exposure for 5 days Data information: The values represent the means ± S.E.M. Level of significance was determined by using ); B, one-way ANOVA P=0.0001 followed by Bonferroni post hoc test, *P<0.05 versus control (n=3  C. Cell growth analysis of human MO3.13 oligodendrocytes in absence or in presence of 200 μM D-Asp for 1-5 days. The density of M03.13 oligodendrocytes was daily recorded through trypan blue dye exclusion. Mean of daily measurements were recorded. The data of each experimental group were normalized to the density of cells plated at day 0 and expressed as percentage of ctrlday0 Data information: The values represent the means ± S.E.M. Level of significance was determined by using C, for each day, two-tailed Student's t test, *P<0.05 versus control at 3 days (n=5)  D. FACS-based cell cycle distribution analysis after PI incorporation of MO3.13 oligodendrocytes in absence or in presence of 200 μM D-Asp for 3 days. Representative FACS plots of biological replicates are shown (n=5 independent experimental sessions)  E. FACS-based cell cycle distribution analysis after PI incorporation of rat primary OPC in absence or in presence of 100 μM D-Asp for 1 day. Representative FACS plots of biological replicates are shown (n=3 independent experimental sessions)  F. Confocal microscopic images displaying the coexpression of MAG or MBP with Phalloidin-594 (a-b and c-d, respectively) in rat primary OPC cultured in absence or in presence of 100 μM D-Asp for 4 days. (e), Single representative MBP++ cell. Scale bars: 50 μm in a-d; 10 μm in e. (f-g); Quantitative analysis of Alexa-594-phalloidin-positive oligodendrocytes (f) or MAG+ and MBP++ oligodendrocytes (g) scored in control and D-Asp exposed cells. Alexa-594-phalloidin-positive oligodendrocytes were scored in 4 categories, according to their morphological complexity. In each category, data were normalized on the total number of oligodendrocytes Data information: The values represent the means ± S.E.M. Level of significance was determined by usin F, for each category two-tailed Student's t test, *P<0.05 versus control (n=3) \n",
      ".Figure 2 A. Schematic diagram showing D-Asp exposure protocol in organotypic slices  B. Maximum intensity projection of z-stack confocal images displaying MBP (red) and NF200 (green) immunoreactivities in 7 DIV cerebellar organotypic slices cultured in absence (b-d) or in presence of 100 μM D-Asp (f-h). Panels d and h display colocalized points (white). Panels a and e show representative low magnification images of 7 DIV cerebellar slices cultured cultures in absence (a) or in presence of 100 μM D-Asp (e). Scale bars in a and e: 200 μm; in b-d and f-h: 50 μm  C. Scatter plot histogram analysis of the myelination index in 7 DIV cerebellar slices cultured in absence or in presence of 100 μΜ D-Asp Data information: The values represent the means ± S.E.M. Level of significance was determined by using: in C, two-tailed Student's t test, *P<0.05 versus control (n=4 animals, 3-4 slices per group  D. Western blotting analysis of MBP protein levels from homogenates of organotypic cerebellar slices culture in absence or in presence of 100 μΜ D-Asp. Data were normalized on the basis of β-actin and expressed as percentage of controls Data information: The values represent the means ± S.E.M. Level of significance was determined by using D, two-tailed Student's t test, *P<0.05 versus controls (n=3  E. Schematic diagram showing D-Asp exposure protocol in organotypic slices after LPC exposure F. Maximum intensity projection of z-stack confocal images displaying MBP (red) and NF200 (green) immunoreactivities in control cerebellar organotypic slices (b-d), and in cerebellar slices at 6 days after LPC exposure (dpl) in absence (f-h) or in presence 100 μM D-Asp (j-l). Panels d, h, and l display colocalized points (white). Panels a, e, and i show representative low magnification images of 7 DIV cerebellar slices cultured cultures in absence (a) or in presence of 100 μM D-Asp (e). Scale bars in a, e ,i: 200 μm; in b-d, f-h, and j-l: 50 μm  G. Western blotting and quantitative analysis of MBP protein levels from homogenates of organotypic cerebellar slices at 6 dpl cultured in absence or in presence of 100 μΜ D-Asp Data information: The values represent the means ± S.E.M. Level of significance was determined by usin G, one-way ANOVA P=0.003 followed by Bonferroni post hoc test, *P<0.05 versus control, ˄P< 0.05 versus LPC (n=3  H. Scatter plot histogram analysis of the remyelination index in cerebellar explants at 6dpl after LPC exposure cultured in absence or in presence of 100 μΜ D-Asp. Data were normalized to vehicle control Data information: The values represent the means ± S.E.M. Level of significance was determined by using ); H, one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P<0.05 versus controls, ˄P< 0.05 versus LPC (n=4 animals, 3-4 slices per group)  I. Maximum intensity projection of z-stack confocal images displaying MBP (red) and NF200 (green) immunoreactivities in control cerebellar organotypic slices (b-d), and in cerebellar slices at 10 dpl in absence (f-h) or in presence of 100 μM D-Asp (j-l). Panels d, h, and l display colocalized points (white). Panels a, e, and i show representative low magnification images of 7 DIV cerebellar slices cultured cultures in absence (a) or in presence of 100 μM D-Asp (e). Scale bars in a, e ,i: 200 μm; in b-d, f-h,and j-l: 50 μm  J. Scatter plot histogram analysis of the remyelination index in cerebellar explants at 10 dpl cultured in absence or in presence of 100 μΜ D-Asp. Data were normalized to vehicle control Data information: The values represent the means ± S.E.M. Level of significance was determined by usin J, one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P<0.05 versus controls, ˄P< 0.05 versus LPC (n=4 animals, 3-4 slices per group) \n",
      ".Figure 3 A. RT-PCR of CNPase mRNA expression in oligodendrocyte MO3.13 progenitors under control conditions and following 200 μM D-Asp exposure for 3 days, in absence or in presence of 10 μM MK-801 (left panel), or 30 nM YM-244769 or 100 nM BED (right panel). Graphs show quantification of ratio of CNPase to L19 Data information: The values represent the means ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: in A, left, one-way ANOVA P=0.009 followed by Tukey's post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3); A, right, one-way ANOVA P=0.0001 followed by Tukey's post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3  B. RT-PCR of MBP mRNA expression in MO3.13 cells under control conditions and following D-Asp exposure for 3 days, in absence or in presence of 10 μΜ MK-801 (left panel), or 30 nM YM-244769 or 100 nM BED (right panel) Graphs show quantification of ratio of MBP to L19 Data information: The values represent the means ± S.E.M from 3 independent experimental sessions. Level of significance was determined by usin B, left, one-way ANOVA P=0.0001 followed by Tukey's post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3); B, right, one-way ANOVA P=0.0004 followed by Tukey's post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3  C. (a-j); Confocal images displaying the expression of NG2 and MAG proteins in rat primary OPC cultured in presence of D-Asp for 3 days, in absence (a-c) or in presence of 10 μM MK-801 (d-f), or 30 nM YM-244769 (g-i) or 100 nM BED (j-l) C. (m-n); Quantitative analysis of NG2+ and MAG+ cells in rat primary OPC cultured in presence of D-Asp for 3 days, in absence or in presence of 10 μM MK-801, 30 nM YM-244769 or 100 nM BED Data information: The values represent the means ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using C, m and C, n, one-way ANOVA P<0.0001 followed by Bonferroni post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3)  D. RT-PCR of NCX3 mRNA expression under control conditions and following 10-200 μM D-Asp exposure or 100 nM PMA for 3 days Data information: The values represent the means ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: i D, one-way ANOVA P=0.0001 followed by Tukey's post hoc test, *P< 0.05 versus control, ˄P< 0.05 versus D-Asp (n=3)  E. RT-PCR of NCX3 mRNA expression following 200 μM D-Asp exposure, in absence or in presence of 10 μM MK-801 Data information: The values represent the means ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using E, one-way ANOVA P=0.0083 followed by Tukey's post hoc test, *P<0.05 versus control, ˄P< 0.05 versus D-Asp (n=3), n.s, not significant  F. RT-PCR of NCX1 mRNA expression following 200 μM D-Asp exposure, in absence or in presence of 10 μM MK-801. Graphs show quantification of ratio of NCX1 and NCX3 to L19 \n",
      ".Figure 5 A. Representative inward current traces elicited by 10μM AMPA in human oligodendrocyte MO3.13 progenitor cells in the absence or in the presence of 10μM DNQX. The concentration-dependent curve of AMPA (1-100 μM) on inward currents is showed at the bottom of panel A Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: in A, one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus 1μM AMPA, n=3 biological replicate  B. Representative inward current traces elicited by 100μM D-Asp application in MO3.13 cells in the absence or in the presence of 10μM DNQX  C. Representative traces of the inward current (upper panel) and the corresponding quantification (bottom panel) observed in human oligodendrocyte MO3.13 progenitors after: (I) the first application of 10μM AMPA alone; (II) the second application of 10μM AMPA alone; (III) the third application of 10μM AMPA in the presence of 10μM DNQX; (IV) the fourth application of 10μM AMPA alone. Data are expressed as percentage of control Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by usin i one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus versus AMPA(I). Data are reported as mean of at least 25 cells in each group, n=3 biological replicate  D. Representative traces of the inward current (upper panel) and the corresponding quantification (bottom panel) observed in human oligodendrocyte MO3.13 progenitors after: (I) the first application of 100μM D-Asp alone; (II) the second application of 100μM D-Asp alone; (III) the third application of 100μM D-Asp in presence of 10μM DNQX; (IV) the fourth application of 100μM D-Asp alone. Data are expressed as percentage of control Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: i one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus D-Asp (I). Data are reported as mean of at least 25 cells for each group, n=3 biological replicates  E. Representative inward current traces elicited by 10μM AMPA in rat primary OPC in the absence or in the presence of 10μM DNQX  F, Representative inward current traces elicited by 100μM D-Asp in rat primary OPC in the absence or in presence of 10μM DNQX or 10μM DNQX + 20μM PDC  G. Representative traces of the inward current (upper panel) and the corresponding quantification (lower panel) observed in rat primary OPC after: (I) the first application of 10μM AMPA alone; (II) the second application of 10μM AMPA alone; (III) the third application of 10μM AMPA in the presence of 10μM DNQX; (IV) the fourth application of 10μM AMPA alone. Data are expressed as percentage of control Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: i G, one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus versus AMPA(I). Data are reported as mean of at least 25 cells in each group, n=3 biological replicate  H, Representative traces of the inward current (upper panel) and the corresponding quantification (lower panel) observed in rat primary OPC after: (I) the first application of 100μM D-Asp alone; (II) the second application of 100μM D-Asp alone; (III) the third application of 100μM D-Asp in presence of 10μM DNQX; (IV) the fourth application of 100μM D-Asp in presence of 10μM DNQX + 10μM MK801; (V) the fifth application of 100μM D-Asp in presence of 10μM DNQX + 10μM MK-801+ 20μM PDC; (VI) the sixth application of 100μM D-Asp alone. Data are expressed as percentage of control Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: i H, one-way ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus D-Asp (I). Data are reported as mean of at least 25 cells for each group, n=3 biological replicates \n",
      ".Figure 6 A. Schematic representation of the administration protocols for D-Asp treatment in mice. Cuprizone was administered for weeks 1 to 5 (blue) to induce demyelination (DEM) in the corpus callosum. Then, mice returned to normal diet for weeks 6 to 7.5 (red), allowing recovery and remyelination (REM). D-Asp (20 mM in drinking solution) was given to the mice for 5 weeks concomitantly the cuprizone treatment (DEM); D-Asp treatment was initiated after cuprizone withdrawal and maintained for additional 2.5 weeks (D-Asp I); D-Asp treatment was initiated one week before cuprizone withdrawal and maintained for additional 2.5 weeks (D-Asp II)  B-C. Beam crossing latency during daily training (left panel, averaged across 3 trials per day) and average latency to cross the beam over 3 consecutive days (right panel), recorded in control (open circles), cuprizone- (filled black circles) and in cuprizone + D-Asp-treated mice (filled red triangles) during 3 weeks (B), and 5 weeks (C) of cuprizone feeding The values represent the means ± S.E.M (n=8 mice for each group) Level of significance was determined by using left panels (daily training), for each day (d) one-way ANOVA P=0.0013 (B, 1d), P=0.00 followed by Bonferroni post hoc test, #P<0.05 versus cpz at day 1, §P<0.05 versus cpz at day 2, *P<0.05 versus cpz at day 3 right panels (average): one-way ANOVA P=0.0001 (B), P<0.0001 (C ), followed by Bonferroni post hoc test *P<0.05 versus control; ^P<0.05 versus cpz or vehicle (veh)  D. Beam crossing latency during daily training (left panel, averaged across 3 trials per day) and average latency to cross the beam over 3 consecutive days (middle and right panels) recorded after 2.5 weeks of remyelination in control (open circles), vehicle- (filled black circles) and in D-Asp- treated mice. D-Asp (I) (filled grey triangles) refers to the group of mice which received D-Asp only for 2.5 weeks after cuprizone withdrawal; D-Asp (II) (filled red triangles) refers to the group of mice which received D-Asp during the last week of cuprizone feeding and for 2.5 additional weeks after cuprizone withdrawal The values represent the means ± S.E.M (n=8 mice for each group) Level of significance was determined by using left panels (daily training), for each day (d) one-way ANOV P=0.0027 (D, 3d), P=0.0025 (D, 2d followed by Bonferroni post hoc test, #P<0.05 versus cpz at day 1, §P<0.05 versus cpz at day 2, *P<0.05 versus cpz at day 3 right panels (average): one-way ANOV P<0.0001 (D, for both middle and right panels followed by Bonferroni post hoc test *P<0.05 versus control; ^P<0.05 versus cpz or vehicle (veh)  E-F. Latency to fall during daily training in accelerating rotarod test (left panel, averaged across 3 trials per day) and average over 3 consecutive days (right panel) recorded in control (open circles), cuprizone- (filled black circles) and D-Asp- treated (filled red triangles) mice at 3 weeks (E), and 5 weeks (F) of cuprizone feeding The values represent the means ± S.E.M (n=8 mice for each group) Level of significance was determined by usin left panels (daily training), for each day (d) one-way ANOV P<0.0001 (E, 3d), P=0.0082 (F, 3d followed by Bonferroni post hoc test, #P<0.05 versus cpz at day 1, §P<0.05 versus cpz at day 2, *P<0.05 versus cpz at day 3 right panels (average): one-way ANOV ), P=0.0002 (E), P<0.0001 (F) followed by Bonferroni post hoc test *P<0.05 versus control; ^P<0.05 versus cpz or vehicle (veh)  G. Latency to fall during daily training in accelerating rotarod test (left panel, averaged across 3 trials per day) and average over 3 consecutive days (right panel) recorded after 2.5 weeks of remyelination (REM) in control, vehicle- and in D-Asp- treated mice. D-Asp (II) refers to the group of mice which received D-Asp during the last week of cuprizone feeding and 2.5 additional weeks after cuprizone withdrawal The values represent the means ± S.E.M (n=8 mice for each group) Level of significance was determined by using left panels (daily training), for each day (d) one-way ANOV P<0.0001 (G, 1d), P=0.0044 (G, 2d), P=0.0038 (G, 3d), followed by Bonferroni post hoc test, #P<0.05 versus cpz at day 1, §P<0.05 versus cpz at day 2, *P<0.05 versus cpz at day 3 right panels (average): one-way ANOV P<0.0001 (G), followed by Bonferroni post hoc test *P<0.05 versus control; ^P<0.05 versus cpz or vehicle (veh) \n",
      ".Figure 7 A. Representative confocal double immunofluorescence images displaying MBP (red) and NF200 (green) distribution and their coexpression (white) in the middle corpus callosum of septostriatal sections of control mice (a-c), and of mice fed with cuprizone for 5 weeks in the absence (d-f) or in the presence of D-Asp (g-i). Scale bars in a-i: 20 μm. (j), Densitometric analysis of MBP immunofluorescence signal and (k) quantification of MBP-NF200 colocalized points in the middle corpus callosum of septostriatal sections from control mice, and from mice fed with cuprizone for 5 weeks in the absence or in the presence of D-Asp Data information The values represent the means ± S.E.M. Level of significance was determined by using in: A, j, one-way ANOVA P<0.0001 followed by Bonferroni post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=4 mice for each group); A, k, one-way ANOVA P=0.0162 followed by Newman-Keuls post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=4 mice for each group  B. Western blot and densitometric analysis of MBP levels in corpus callosum lysates from control and cuprizone treated-mice for 5 weeks in the absence or in the presence of D-Asp, respectively. Data were normalized on the basis of α-tubulin and expressed as percentage of controls Data information The values represent the means ± S.E.M. Level of significance was determined by using i B, one-way ANOVA P<0.0001 followed by Bonferroni post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=3 mice for each group  C. Representative confocal double immunofluorescence images displaying GFAP (green) and Iba1 (red) distribution in the middle corpus callosum of septostriatal sections of control mice (a-c), and of mice fed with cuprizone for 5 weeks in the absence (d-f) or in the presence of D-Asp (g-i). Scale bars in a-i: 50 μm. (j-k) Quantification of GFAP+ and Iba1+ cells in the middle corpus callosum of septostriatal sections of control mice and of mice fed with cuprizone for 5 weeks in the absence or in the presence of D-Asp. Data were normalized to the total cell number (Hoechst signal) and expressed as percentage of controls Data information The values represent the means ± S.E.M. Level of significance was determined by using i C, j-k, one-way ANOVA P<0.0001 followed by Bonferroni post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=4 mice for each group  D. Western blot of GFAP and Iba1 protein levels in corpus callosum lysates obtained from control or cuprizone treated-mice for 5 weeks in the absence or in the presence of D-Asp, respectively Data information The values represent the means ± S.E.M. Level of significance was determined by using i D middle panel, one-way ANOVA P=0.0146 followed by Bonferroni post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=3 mice for each group); D right panel, one-way ANOVA P= 0.0026 followed by Bonferroni post hoc test, *P<0.05 versus control, ^P<0.05 versus cpz (n=3 mice for each group) \n",
      ".Figure 4left panels. Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected in MO3.13 cells (A in absence or in presence of 10 μM MK-801 or 150 μM APV right panels. Quantification of the oscillation index in MO3.13 cells (A in absence or in presence of 10 μM MK-801 or 150 μM APV Quantification of the initial [Ca2+]i increase elicited by D-Asp measured as ∆% of peak versus basal values in absence or in presence of 10 μM MK-801 or 150 μM AP in M03.13 cells (a MK-801 and APV were preincubated 10 minutes before registration Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using: in A and a, one way-ANOVA P<0.001 followed by Bonferroni post hoc test, *P<0.05 versus control (basal value), ˄P<0.05 versus D-Asp. Data are reported as mean of 23-30 cells in each group, n=3 biological replicate left panels. Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected i primary OP (B) in absence or in presence of 10 μM MK-801 or 150 μM APV right panels. Quantification of the oscillation index i primary OPC (B) in absence or in presence of 10 μM MK-801 or 150 μM APV Quantification of the initial [Ca2+]i increase elicited by D-Asp measured as ∆% of peak versus basal values in absence or in presence of 10 μM MK-801 or 150 μM APV -i primary OPC (b). MK-801 and APV were preincubated 10 minutes before registration Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using B and b, one way-ANOVA, P<0.001 followed by Bonferroni post hoc test, *P<0.05 versus control (basal value), ˄P< 0.05 versus D-Asp. Data are reported as mean of 20-23 cells in each group, n=3 biological replicate left panels; Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected in MO3.13 cells (C in absence or in presence of 30 nM YM-244769 or 100 nM BED right panels; Quantification of the oscillation index in MO3.13 cells (C in absence or in presence of 30 nM YM-244769 or 100 nM BED d, Quantification of the initial [Ca2+]i increase elicited by D-Asp measured as ∆% of peak versus basal values in absence or in presence of 30 nM YM-244769 or 100 nM BED in M03.13 cells (c YM-244769 or BED were preincubated 10 minutes before registration Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using C and c, one way-ANOVA P<0.001 followed by Bonferroni post hoc test, *p< 0.05 versus control (basal value), ˄ P< 0.05 versus D-Asp. Data are reported as mean of 19-30 cells in each group, n=3 biological replicate left panels; Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected i primary OPC (D) in absence or in presence of 30 nM YM-244769 or 100 nM BED D, right panels; Quantification of the oscillation index i primary OPC (D) in absence or in presence of 30 nM YM-244769 or 100 nM BED Quantification of the initial [Ca2+]i increase elicited by D-Asp measured as ∆% of peak versus basal values in absence or in presence of 30 nM YM-244769 or 10 nM BED -i primary OPC (d). YM-244769 or BED were preincubated 10 minutes before registration Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by usin D and d, one way-ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus control (basal value), ˄P< 0.05 versus D-Asp. Data are reported as mean of 18-20 cells in each group, n=3 biological replicate  E. left panel; Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected in MO3.13 cells in presence of siCtl or sincx3 silencing. E, right panel; Quantification of the oscillation index in MO3.13 cells in absence or in presence of sincx3. e, Quantification of the initial [Ca2+]i increase elicited by D-Asp and measured as ∆% of peak versus basal values in absence or in presence of sincx3 Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by usin E and e, one way-ANOVA P<0.001 followed by Bonferroni post hoc test, *P< 0.05 versus sictl, ˄P< 0.05 versus D-Asp + sictl. Data are reported as mean of 25-30 cells in each group, n=3 biological replicate  F. left panel. Superimposed single-cell traces representative of the effect of 100 μM D-Asp on [Ca2+]i detected in primary OPC obtained from wild-type ncx3+/+, heterozygous ncx3 +/-, and knock-out ncx3-/- mice. F. right panel. Quantification of the oscillation index elicited by D-Asp in primary mouse OPC obtained from ncx3+/+, ncx3 +/-, and ncx3-/- mice. f, Quantification of the initial [Ca2+]i increase measured as ∆% of peak versus basal values Data information: The values represent the mean ± S.E.M from 3 independent experimental sessions. Level of significance was determined by using F and f, one way-ANOVA P<0.001 and P=0.003, respectively, followed by Bonferroni post hoc test, *P< 0.05 versus basal value, ˄P< 0.05 versus ncx3+/+ and ncx3+/-. Data are reported as mean of 10-19 cells in each group, n=3 biological replicates \n",
      ".Figure 2Average gene profiles of normalized RNA-seq and RNAP ChIPseq coverages from wt (solid lines) and ∆rnjA (dashed lines) strains were plotted for gene classes I-IV (n = 762, 315, 553 and 1654 genes, respectively). Open reading frames were rescaled to 1 kb, upstream and downstream regions of 0.2 kb were also included in the plots. Data represent mean values of 3 independent experiments. The pie-chart shows the overall distribution of class I-IV and other genes. \n",
      ".Figure 1A, B. Comparisons of RNAseq and ChIPseq data. Wt strain (A) and ∆rnjA (B). The x-axis shows the relative RNAP occupancy at a given gene (normalized ChIPseq coverage). The y-axis shows the relative abundance of the transcript (normalized RNAseq coverage; each dot is one gene). The genes are color-coded, ranging from yellow (low RNA abundance) to red (high RNA abundance) in wt. The ∆rnjA strain has higher RNAP occupancy mainly among the genes with less abundant transcripts. The color coding in (B) reveals no dramatic overall changes in the vertical direction (RNA abundance). If anything, some of the low abundance transcripts decreased further in level in the mutant strain. The main difference (mostly among the low abundance transcripts) is their shift in the horizontal direction to the right (towards higher occupancy with RNAP). Data represent mean values of three independent experiments. C. Relative expression of all sigma factors in wt (LK1371) versus ΔrnjA (LK1381). Wt levels of each sigma factor were set as 1 (indicated with the horizontal line). D. Sigma-dependent genes and correlation with expression of sigma factors (significantly changed) in ΔrnjA (LK1381). The most down-regulated sigma factor was sigD and almost all sigD-dependent genes were downregulated. A similar trend is visible for the rest of sigma factors and their respective dependent genes. The x-axis shows expression of each sigma factor, the y-axis shows expression of genes for each sigma regulon. The violet (dark blue) line: regression line. \n",
      ".Figure 3A. SIM of exponential B. subtilis cells. RNase J1 was fused to GFP (green), RNAP to mCherry (red). The graph shows relative fluorescence intensities at the cell midsection (along the long axis); SIM of the cell is below. Yellow indicates colocalization of the two proteins. B. Pull-down with RNase J1-8xHis tag and RNAP-10xHis tag and detection of the proteins with antibodies. Lanes 1, 2 - RNase J1-8xHis was used to pull down RNAP; lanes 3, 4 - RNAP-10xHis was used to pull down RNase J1; lanes 5, 6 - purified proteins were used as markers; lanes 7, 8 - strains without His-tagged proteins were used as negative controls to demonstrate the specificity of the interaction. M, molecular size marker; EX, exponential phase; STA, stationary phase. C. Pull-down with RNase J1-8xHis tag from stationary phase cells The samples then either were (lane 9) or were not (lane 10) treated with RNase A to detect whether the interaction was via RNA. Lanes 11, 12 - purified proteins were used as markers. The experiment was performed three times (biological replicates) with the same result. \n",
      ".Figure 8 A. Confocal double immunofluorescence images depicting Olig2+ (a-e) and Olig2+-CC1+ (f-j) double labeled cells in the middle corpus callosum of control, cuprizone-treated mice at 5 weeks, in absence (cpz) or in presence of D-Asp (cpz + D-Asp), and 2.5 weeks after cuprizone withdrawal, in absence (vehicle) or in presence of D-Asp (D-Asp). (k, left panel), Quantitative analysis of Olig2+ cells (red) and Olig2+-CC1+ (blue) double labeled cells in the middle corpus callosum of control, cuprizone-treated mice, and cuprizone+D-Asp-treated mice at 5 weeks. (k, right panel), Quantitative analysis of Olig2+ cells (red) and Olig2+-CC1+ (blue) double labeled cells in the middle corpus callosum of control, vehicle, and D-Asp-treated mice 2.5 weeks after cuprizone withdrawal. Arrows point to Olig2+-CC1+ cells. Arrowheads point to Olig2+-CC1+ cells. Scale bar in a-j: 50 μm Data information: The values represent the means ± S.E.M. Level of significance was determined by using: in A, k left panel, one-way ANOVA P<0.0001 followed by Newman-Keuls post hoc test, *P<0.05 versus ctrl; ^P<0.05 versus cpz (n=3 mice for each group). A, k right panel, one-way ANOVA P<0.0001 followed by Newman-Keuls post hoc test, *P<0.05 versus ctrl; ^P<0.05 versus vehicle (n=3 mice for each group  B. Representative electron microscopy images of middle corpus callosum sections from control mice (a), cuprizone treated-mice for 5 weeks in the absence (b) or in the presence of D-Asp (c), and from mice 2.5 weeks after cuprizone withdrawal in the absence (d, vehicle) or in presence of D-Asp (e). Scale bars in a-e: 1 μ  C. Percentage of myelinated axons in the middle corpus callosum of cuprizone (blue) and cuprizone + D-Asp-treated mice (red) for 5 weeks Data information: The values represent the means ± S.E.M. Level of significance was determined by using C, two-tailed Student's t test, *P<0.05 versus cpz (n=4 mice for each group  D. Scatter plot analysis of myelin thickness, expressed as individual G-ratios against axon diameters, after 2.5 weeks of cuprizone withdrawal in the middle corpus callosum of vehicle (blue circle) or D-Asp-treated mice (red triangle). Linear regression was used to indicate the differences between the groups in myelin thickness across the range of axon diameters  E. Histogram analysis displaying the G-ratio values of myelinated fibers across the range of axon diameters. Data represent the mean S.E.M. for at least 100 axons from each group Data information: The values represent the means ± S.E.M. Level of significance was determined by usin E, for each axon diameter group, Student's t test, *P<0.05 versus vehicle (n=4 mice for each group  F. Percentage of myelinated axons after 2.5 weeks of cuprizone withdrawal in the middle corpus callosum of vehicle (blue) or D-Asp- treated mice (red). Abbreviations: DEM, demyelination; REM, remyelination Data information: The values represent the means ± S.E.M. Level of significance was determined by usin F, for each axon diameter group, Student's t test, *P<0.05 versus vehicle (n=4 mice for each group) \n",
      ".Figure 4A. Schematic representation of the flow of the experiment. TCs were assembled on DNA-RNA scaffolds (DNA was biotinylated); RNA was labeled at 3' with radioactive UTPs (asterisks). The RNA length (including label) was 33 nt. B. Representative primary data - polyacrylamide gel (the experiment was performed 3x with the same results). Lane 1, the full length (33 nt) labeled RNA; lane 2, the same as lane 1 but it included also incubation with RNase J1 (J1), 17-18 nt long fragments (RNase J1 stopped by RNAP; indicated with asterisks) and <5 nt fragments (RNA released from TC into buffer - indicative of TC disassembly) are shown; lane 3, the same as lane 1 but included also incubation with RNase R (R); lanes 4 and 5, TCs were denatured by heat prior to RNase addition to demonstrate the activity and cleavage patterns of both enzymes; lanes 6 and 7, the same as lanes 2 and 3 but the buffer was washed off (TCs were retained by streptavidin beads) to demonstrate which RNA fragments were associated with TC; lanes 8, 9 (M1, M2) Mw marker generated by treating the 30 nt RNA with alkali and formamide (M1-4 min treatment, M2-7 min treatment). As reported in (Costanzo et al., 2016) [and references therein], the cleavage by alkali or formamide leaves the phosphate group of the attacked phosphodiester bond bound at 3', initially in the 2',3' cyclic form (upper band in the band couples). This successively opens (lower band in the band couples) yielding a double-banded pattern for short oligoribonucleotides. \n",
      ".Figure 5A. Primary data - polyacrylamide gel - a representative result. For description of bands/fragments see next panel legend. Asterisks indicate STUBS (16-19 nt). B. A schematic representation of quantitation of fragments. OUTSIDE (16-30 nt) are RNA fragments that originated by digestion of the full length RNA by RNases that were stopped either before reaching or at the point of reaching RNAP; STUBS (16-19 nt) are RNA fragments that originated by digestion of the full length RNA that were stopped at the point of reaching RNAP; INSIDE (< 16 nt) are RNA fragments (oligonucleotides) that could be only generated after the disassembly of the complex by the torpedo mechanism. The red color indicates parts of RNA that were digested. C. Quantitation of three independent experiments. \"Full length\" indicates the remaining undigested RNA. \"OUTSIDE\", \"INSIDE\" are fragments as explained in (B) and indicated in (A). The bars represent 100 % (all fragments). The black-grey-white boxes indicate the percentage of each fragment group (in %). The error bars indicate ± SEM for each group of fragments calculated from three biological replicates. \n",
      ".Figure 7A. Exponential cells of indicated strains (below the bars) were plated onto LB agar and either were, or were not UV-irradiated. After overnight incubation, CFU were counted. UV sensitivity of the mutant strains (KO) was then calculated as the ratio between irradiated vs non-irradiated cells and normalized to this ratio from the wt strain As a consequence, the wt ratio is 1. The experiment was conducted 4x (biological replicates) and the bars show the geometric mean. The p-values, shown above the bars, were computed using two-tailed unpaired t-test for logarithms of the ratios. The error bars show ±SEM (computed on the log scale). B. Relative expression (mRNA) of helD, mfd, rho, greA, nusA, nusB, and nusG in the ∆rnjA strain (normalized to wt [set as 1]). \n",
      ".Figure 6A. TCs were assembled on DNA-RNA scaffolds (DNA was biotinylated and attached to streptavidin coated magnetic beads); TCs were then divided into three tubes and challenged with buffer (\"-\", mock treatment) or RNase J1 of Xrn1. Dissociation of RNAP from DNA was monitored by detecting RNAP with anti-β antibody in two fractions: B - beads (RNAP still bound to DNA), R - released (free in buffer). The gel strip shows representative primary data (Western blot). The graph shows averages (the bars) of two experiments (biological replicates) and the error bars show the range. The combined signal for B+R for each treatment was set as 100 %. B. Primary data - polyacrylamide gel. RNAP from E. coli. Lane 1, the full length (33 nt) labeled RNA; lane 2, the same as lane 1 but it included also incubation with RNase J1 (J1), lane 3, the same as lane 1 but included also incubation with Xrn1. C. Quantitation of three independent experiments. \"Full length\" indicates the remaining undigested RNA. \"OUTSIDE\", \"INSIDE\" are fragments The bars represent 100 % (all fragments). The black-grey-white boxes indicate the percentage of each fragment group (in %). The error bars indicate ± SEM for each group of fragments \n",
      ".Figure 8RNase J1 is depicted as a dimer. RNase J1, after the nascent transcript has been endonucleolytically cleaved, degrades RNA in the 5'-to-3' direction. Upon encounter with RNAP, conformational changes are induced, likely resulting in dissociation of the complex, and liberating RNAP from DNA thereby removing obstacles potentially causing transcription-replication clashes. \n",
      ".Figure 1(A) Quantitative analysis of Rnf114 mRNA by realtime-PCR in various mouse tissues and organs shows that Rnf114 was predominately expressed in oocytes. (B) Immunohistochemistry analysis of RNF114 in mouse ovary shows that RNF114 was mainly expressed in the oocytes (Scale bar = 100um). (C) Knockdown efficiency of Rnf114 in the 2-cell stage embryos derived from Rnf114 siRNA #1 or siRNA #2 injected zygotes was verified by realtime-PCR and western blotting. NC indicates negative control injected with non-silencing siRNA. Three independent experiment replicates were performed for realtime-PCR, error bars represent s.e.m.; **:P<0.01, ***:P<0.01 in unpaired two-tailed t-test. (D) RNF114 depletion impaired the early embryonic development. Rnf114 siRNA #1 or siRNA #2 injected zygotes were cultured and quantified (siRNA1#, n = 120; siRNA2#, n = 128; three independent experiments); NC indicates negative control (n = 193; six independent experiments). The error bars represent s.d., *:P<0.05, **:P<0.01, ***:P<0.001 in unpaired two-tailed t-test. (E) Representative images to show early embryonic development defects. The embryos arrested at two-cell stage are indicated by arrows (Scale bar = 100um). \n",
      ".Figure 3(A) RNF114 substrate validation. HEK293 cells were co-transfected with constructs expressing FLAG-tagged substrates, wild-type or mutated His-RNF114 and EGFP, and western blot was performed with anti-FLAG and anti-His antibodies. The results show that the protein levels of CD74, TAB1, OPTN, TNIP1, APPL1, PSAT1, and RALGPS1 were reduced in the presence of wild-type RNF114 compared to co-transfecting with mutant RNF114. (B) RNF114-dependent in vitro ubiquitination of substrates. Substrate ubiquitination assays were performed with purified RNF114, the reaction products were immunoblotted using anti-His antibody to detect the ubiquitin (upper panel) or anti-FLAG antibody detecting the substrate protein (middle panel), and the lower panel showed the merged signals. The results show that clear polyubiquitin chains were formed in the systems containing recombinant CD74, TAB1, TNIP1, PSAT1 or RALGPS1. \n",
      ".Figure 2(A) In vitro ubiquitination assay. Purified His-RNF114 was subjected to ubiquitination assays with the indicated components and the samples were analyzed by immunoblotting with anti-FLAG (Ub) antibody. Lanes 1-4 indicate negative controls. The results show that recombinant RNF114 efficiently catalyzed the formation of polyubiquitin chains. (B, C) Purified wild-type and RING mutated His-RNF114 were stained by Coomassie Blue (B). Purified wild-type and RING mutated His-RNF114 were subjected to ubiquitination assays together with E1, Ubc4, FLAG-Ub, and ATP. The results show that mutant RNF114 lacked ubiquitination activity (C). (D) Screening for proteins ubiquitinated by RNF114 on a proteome array. Putative substrates of RNF114 were identified on Ubiquitin Ligase Substrate Identification on ProtoArray® Human Protein Microarrays v5.0. Ubiquitination reactions were performed on quintuplicate arrays using the indicated components, the five sub-array are respectively negative controls, E1+E2 (no E3), WT RNF114 (two arrays, duplicate), and mutated RNF114. (E) A heat map comparing hits between E1+E2, E1+E2+WT RNF114, and E1+E2+mutated RNF114 reactions. Candidate substrates of RNF114 selected for further characterization are indicated in red font; HIP1, indicated by blue font, was considered to be a false positive because it was also detected in the mutated RNF114 reaction. Black font indicates proteins with ubiquitin-binding domains such as UIM, UBA or CUE or proteins without clearly reported functions, which were excluded from further investigation. \n",
      ".Figure 4(A) Mouse zygotes were microinjected with sterile ddH2O (control group) or Ralgps1, Cd74, Tnip1, Psat1, or Tab1 mRNA. During 96-100 h culturing, the percentages of embryos at various stages were counted under microscopy. The results show that overexpression of Tab1 impaired the early embryonic development. Three independent experiment replicates were performed, error bars represent s.d.; *:P<0.05, ***:P<0.001 in unpaired two-tailed t-test. (B) TAB1 reduced with mouse early embryonic development, while abnormally accumulated at 2-cell stage once treated with Rnf114 siRNAs. TAB1 was detected by western blotting with anti-TAB1 antibody. The relative level of TAB1 protein after Rnf114 siRNAs treatment were presented as mean±s.e.m. (*:P<0.05 in unpaired two-tailed t-test, four independent experiment replicates). (C) Detection of endogenous TAB1 ubiquitination at 1-cell or 2-cell stages. Endogenous TAB1 proteins were immunoprecipated by anti-TAB1 antibody at either 1-cell or 2-cell stages, the immunoprecipates were detected by either anti-TAB1 antibody or anti-ubiquitin antibody. The results show that more Ub chains on TAB1 were detected in the two cell stage embryos than in the zygotes, when the TAB1 proteins were at comparable level. (D) RNF114-dependent TAB1 degradation during early embryo development. (a-c) Representive time lapse embryo images were shown for each treatment. Scale bar = 20um. (a'-c') Quantification of the GFP fluorescent signals for respective treatments in (a-c). n=14, 9 or 12 for the NC, Rnf114 siRNA1# or Rnf114 siRNA2# treatment group respectively. Gray traces: Tab1-GFP fluorescent signals were normalized to the first time point; Black trace: average signal of all embryos for each treatment. (E) Tab1 knockdown partly rescued the early embryo development defect observed with Rnf114 knockdown. Both Rnf114 and Tab1 siRNAs were injected into the GV oocytes, then the oocytes matured, and were fertilized by IVF, and the fertilized eggs were transferred into KSOM media, cultured for 48-50 h. The percentages of embryos arrested at 2-cell stage were counted. Three independent experiment replicates were performed, error bars represent s.d.; **:P<0.01 in unpaired two-tailed t-test. \n",
      ".Figure 5(A) NF-κB activation during MZT. Immunofluorescence staining of p65 shows nuclear translocation of p65 at the two-cell stage and thereafter. Scale bar = 25um. (B) The protein levels of IκBα and its phosphorylated form (p-IκBα) during mouse early embryonic development were detected by western blotting, representative western blotting of IκBα and p-IκBα was shown on the left and the quantitation analysis (right panel) indicated that the ratio of p-IκBα/total IκBα was gradually increased from MII oocytes to 4-cell stage embryos (three independent experiment replicates were performed, error bars represent s.e.m.). (C) NF-κB pathway inhibition using the specific inhibitor Bay11-7082 resulted in two cell stage arrest. Three independent experiment replicates were performed, error bars represent s.d.; ***:P<0.001 in unpaired two-tailed t-test. (D, E) Representive immunofluorescence staining of p65 after knockdown of Rnf114 by Rnf114 siRNA #1 or siRNA #2 or overexpression of Tab1 by Tab1 mRNA injection (shown in D. Scale bar = 25um.), the percentages of two-cell stage embryos with p65 nuclear translocation (shown in E.) were significantly decreased after Rnf114 knockdown or Tab1 overexpression (chi-square test was performed for statistical analysis). (F) Western blot analysis at 2-cell stage showed the levels of p-IκBα were downregulated in the Rnf114 siRNA #1, siRNA #2 injected groups. (G) Western blot analysis at 2-cell stage showed the levels of p-IκBα were downregulated in the exogenous Tab1 mRNA injected groups, ddH2O was injected as control. Dotted line indicates a cropped lane. \n",
      "Fig.Figure 1A) Schematic diagram of the USP1 subfamily and the USP1 deletion mutants tested in this study. B) Single point activity assays of USP1WT and deletion mutants on Ub-Rho show signifi­cantly increased activity in mutants where insert L1 and L3 are both deleted. C) Single point activity assays of USP1WT and deletion mutants (± UAF1) on Ub-Rho show the loss in hyper-activation of USP1ΔL1L3 upon addition of UAF1. D) Michaelis-Menten analysis of USP1 deletion mutants against ubiquitin-rhodamine (Ub-Rho) shows that USP1ΔL1L3 has significantly higher activity compared to WT and other mutants (n=2, biological replicates). E) Michaelis-Menten analysis of USP1 deletion mutants (+UAF1) against Ub-Rho shows that all deletions mutants have similar catalytic activity (n=2, biological replicates). F) Comparing activity of USP1WT and USP1ΔL1L3 on a peptide substrate i.e. LRGG-AMC (100µM). USP1ΔL1L3 cleaves the peptide substrate more efficiently compared to USP1WT (n=2, technical replicates). G) Comparing activity of USP1WT and USP1ΔL1L3 (+UAF1) on a peptide substrate i.e. LRGG-AMC (100µM). Addition of UAF1 allows cleavage of LRGG-AMC by both USP1WT and USP1ΔL1L3 (n=2, technical replicates). \n",
      "Fig.Figure 4A) Gel based quantification of USP1-UAF1 activity on PCNA-Ub (DNA loaded and free) for three different concentration of enzyme and substrate. B) FP based activity assays on free PCNA-Ub (three concentrations of USP1 and PCNA-Ub) and activity of USP1 on increasing concentrations of Ub-Rho. C) Kinetic model of USP1-UAF1 activity on PCNA-Ub and DNA-loaded PCNA-Ub (Ub-Rho not shown here, see EV 4c for full model), constants with the same colour were linked during the fitting and they share same values. \n",
      "Fig.Figure 2A) Quantification of gel-based activity assays where the activity of USP1WT and deletion mutants (+UAF1) on PCNA-UbTAMRA is compared. The USP1ΔL1 mutant has reduced activity on PCNA-UbTAMRA compared to the other USP1 variants tested (n=2, biological replicates). B) Multiple sequence alignment of USP1 insert L1 which highlights the conservation of the PIP and APIM motif in USP1 across species, the residues mutated in this region are indicated with an asterisk. C) Quantification of gel-based activity assays where the activity of USP1 PIP mutants (+UAF1) on PCNA-UbTAMRA is compared. Both the PIP mutants have reduced activity on PCNA-Ub compared to USP1WT (n=2, biological replicates). D) FP based binding assays of USP1 mutants (+ UAF1) with PCNA-UbTAMRA show reduced binding of USP1ΔL1 and USP1PIP1 compared to USP1WT (n=2, technical replicates). E) Quantification of gel-based activity assays where the activity of USP1WT+UAF1 on PCNA-UbWT is compared with PCNA-UbPIM1 shows that PCNA-UbWT is cleaved much faster compared to PCNA-UbPIM1 (n=2, technical replicates (example shown here from 1 of 2 biological replicates)) F) Comparing activity of USP1WT and USP1PIP1 (+UAF1) on PCNA-UbWT and PCNA-UbPIM1 in gel-based assays shows that the activity of USP1WT is severely reduced by the PIM1 mutation in PCNA, whereas USP1PIP1 activity is not further affected indicating that PIP1 and PIM mutation affect the same interaction (n=2, technical replicates (example shown here from 1 of 2 biological replicates)). \n",
      "Fig.Figure 3A) Schematic representation of loading and purification of PCNA-Ub on nicked circular DNA. The DNA-loaded PCNA-Ub elutes in the void of the SEC column and this sample is collected and used for studying activity of USP1 on DNA-loaded PCNA-Ub. B) Michaelis-Menten analysis of USP1+UAF1 with and without DNA (65bp dsDNA) shows that DNA binding alone has no effect on USP1 activity (n=2, technical replicates (example shown here from 1 of 2 biological replicates)). C) Coomassie stained gel of in vitro activity assay showing increased activity of USP1-UAF1 on DNA-loaded PCNA-Ub compared to PCNA-Ub (-DNA) and PCNA-Ub (+DNA, +RFC and not ATP). D) Quantification of gel-based activity assays showing enhanced activity of USP1+UAF1 on DNA-loaded PCNA-Ub compared to free PCNA-Ub (n=3, technical replicates (example shown here from 1 of 2 biological replicates)). E) Quantification of gel-based activity assays showing enhanced activity of USP1 on DNA-loaded PCNA-Ub compared to free PCNA-Ub (n=3, technical replicates (example shown here from 1 of 2 biological replicates)). \n",
      "Fig.Figure 5A) Comparison of USP1WT and deletion mutants (+UAF1) for activity on DNA-loaded PCNA-Ub (solid lines) and free PCNA-Ub (dashed lines) shows no increase in USP1ΔL1 activity on the loaded substrate. Left panel: Quantification of gel-based activity assays showing percentage of cleaved PCNA-Ub at the mentioned time points (n=2, biological replicates), Right panel: Quantification of the activation fold observed in USP1WT and mutants on DNA-loaded PCNA-Ub versus free PCNA-Ub (n=2, biological replicates). B) Multiple sequence alignment of USP1 insert L1 which highlights the DNA binding region and the conservation of the positively charged residues across species. Three groups of positively charged residues (***) are mutated separately and tested for binding to double stranded DNA (65bp) using SPR which shows the importance of this region for DNA binding of USP1. C) Comparison of USP1DM1, USP1PIP1 and USP1PIP1+DM1 (+UAF1) activity on DNA-loaded PCNA-Ub (solid lines) and free PCNA-Ub (dashed lines) shows reduced increase in activity of these mutants compared to USP1WT. Left panel: Quantification of gel based activity assays showing percentage of cleaved PCNA-Ub at the mentioned time points (n=2, technical replicates), Right panel: Quantification of the activation fold observed in USP1WT, USP1PIP1, USP1DM1 and USP1PIP1+DM1 on DNA-loaded PCNA-Ub versus free PCNA-Ub (n=2, technical replicates). D) EMSA based DNA binding experiment shows that USP1PIP1 and USP1WT has similar DNA binding capability while USP1DM1 has lost its ability to bind DNA. E) Schematic model for the role of Insert L1 in USP1 activity regulation by UAF1 and DNA loaded PCNA-Ub. \n",
      ": FKK5 and FKK6 inhibit NF-κB activation in a PXR-dependent manner.Figure 2A, PXR (luciferase) reporter assay in HEK293T cells transiently transfected with PXR plasmids (wild-type and ligand binding domain mutant C285I/C301A). RLU, relative light units are shown normalized to beta-galactosidase (β-Gal) expression. The histogram represents one experimental data (of n > 2 independent experiments each performed in quadruplicate) mean (95 %CI); n.s. not significant; * p < 0.05, two-way ANOVA. B, Histogram represents fold change in mRNA expression, normalized to GAPDH, by RT-qPCR from Caco-2 cells exposed to compounds. DMSO, 0.1% DMSO vehicle; 9.3 is 9.3-fold expression. The data is one representative experiment of two independent experiments (each n = 3 biologic replicates, n = 4 technical replicates). Points and error represent mean (95 %CI) values for control (TNFα 5 ng/ml) normalized NF-κB (luciferase) reporter activity in LS180 cells after C, Passage 13 (P13) and D, Passage 8 (P8) and C, D, Line graph depicts one representative experiment of a series of experiments (n > 4) performed in four consecutive passages of cells. Data was expressed as mean with 95% CI of four technical replicates. E, LS174T PXR-KO (knockout) cells transiently transfected with pNL3.2.NF-κB-RE vector. mean (+ SD). *, ** p < 0.05, two way ANOVA with Tukey's multiple comparisons test. \n",
      ": FKK5 and FKK6 inhibit cytokine and salmonella infection induced NF-κB nuclear translocation.Figure 3A In Caco-2 monolayer cells, Log10 relative expression of IL8 mRNA with or without cytokines (50 ng/ml IL-1, 10 ng/ml IFNγ, 10 ng/ml TNF α), and cytokine cocktail plus FKK drugs (10 and 25 μM) with 2 h or 12 h incubations. B, Salmonella invasion in colony forming units per ml (CFU/ml) at the different time periods of incubation. C, confocal images of NFĸB (red) and the nucleus (blue). Scale bars = 80 µm. Arrows represent localization of NFĸB (red, cytoplasmic; purple, nuclear). Confocal images shown are representative images from (n = 3) biological replicates. Log10 relative expression of IL8 mRNA with or without cytokines (50 ng/ml IL-1, 10 ng/ml IFNγ, 10 ng/ml TNF α), and cytokine cocktail plus FKK drugs (10 and 25 μM) with 2 h or 12 h incubations. In human intestinal organoids (HIO), Salmonella invasion in colony forming units per ml (CFU/ml) at the different time periods of incubation. In human intestinal organoids (HIO), confocal images of NFĸB (red) and the nucleus (blue). Scale bars = 80 µm. n human intestinal organoids (HIO), \n",
      ": FKK6 inhibits cytokine mediated IL8 induction in human colonic organoids and induces PXR target genes in mice.Figure 4Quantitative gene expression by RT-qPCR of IL8 mRNA from human colonic enteroids (over 40 enteroids per individual sample per well) exposed to control (vehicle) or TNF α. with or without FKK5 (n = 9 paired samples). Data shown as mean fold change relative to vehicle control (con) for each paired sample (n = 3 replicates). Each paired sample data set represents biopsies from an individual patient. *, ## p < 0.05, Pair-wise one way ANOVA with Tukey's post hoc test. Quantitative gene expression by RT-qPCR of IL8 mRNA from human colonic enteroids (over 40 enteroids per individual sample per well) exposed to control (vehicle) or TNF α. with or without FKK6 (n = 3 paired samples; one removed due to poor inflammatory response). Data shown as mean fold change relative to vehicle control (con) for each paired sample (n = 3 replicates). Each paired sample data set represents biopsies from an individual patient. *, ## p < 0.05, Pair-wise one way ANOVA with Tukey's post hoc test. Quantitative gene expression by RT-qPCR of IL8 mRNA from human colonic enteroids (over 40 enteroids per individual sample per well) exposed to control (vehicle) or TNF α. with or without FKK9 (n = 5; 3 removed due to poor response and tissue integrity). Data shown as mean fold change relative to vehicle control (con) for each paired sample (n = 3 replicates). Each paired sample data set represents biopsies from an individual patient. *, ## p < 0.05, Pair-wise one way ANOVA with Tukey's post hoc test. Fold induction (mRNA) of genes in C57BL/6 mice gavaged with vehicle (10% DMSO; n = 3) or FKK6 (500 μM in 10% DMSO; n = 3) every 12h for 3 total doses. The entire experiment was repeated two independent times and one representative experiment is shown. Each mouse (each organ) was studied in quadruplicate assays and normalized to internal control, GAPDH. The histograms show mean (95% CI) values for gene expression. The dotted line marks basal fold expression Fold induction (mRNA) of genes in pxr -/- mice gavaged with vehicle (10% DMSO; n = 3) or FKK6 (500 μM in 10% DMSO; n = 3) every 12h for 3 total doses. The entire experiment was repeated two independent times and one representative experiment is shown. Each mouse (each organ) was studied in quadruplicate assays and normalized to internal control, GAPDH. The histograms show mean (95% CI) values for gene expression. The dotted line marks basal fold expression Fold induction (mRNA) of genes in hPXR mice (mice expressing the human PXR gene) gavaged with vehicle (10% DMSO; n = 3) or FKK6 (500 μM in 10% DMSO; n = 3) every 12h for 3 total doses. The entire experiment was repeated two independent times and one representative experiment is shown. Each mouse (each organ) was studied in quadruplicate assays and normalized to internal control, GAPDH. The histograms show mean (95% CI) values for gene expression. The dotted line marks 2-fold gene induction. \n",
      ": FKK5 and FKK6 activate PXR.Figure 1(A) Histogram (mean, 95 %CI) of fold induction of PXR activity reporter assay (luc, luciferase) in LS180 cells transiently transfected with wild-type PXR and p3A4-luc reporter plasmids. Chemical structures of FKK5, FKK6 and FKK999 compounds are overlaid. The bar graph(s) depicts one representative experiment of a series of experiments (n > 3) performed in four consecutive passages of cells. *p < 0.05, one-way ANOVA with Dunnett's post hoc test. *significant over vehicle (DMSO) control. (B), same as (A), in HepG2 cells stably expressing pCYP1A1 luciferase plasmid (AhR reporter). Chemical structures of FKK5, FKK6 and FKK999 compounds are overlaid. Indole is colored blue. The bar graph(s) depicts one representative experiment of a series of experiments (n > 3) performed in four consecutive passages of cells. *p < 0.05, one-way ANOVA with Dunnett's post hoc test. *significant over vehicle (DMSO) control. C, Histogram (mean, 95 %CI) of fold mRNA expression, CYP1A1 (top panel), CYP3A4 (middle panel) and MDR1 (bottom panel) in LS180 cells with or without (mock) transfected PXR plasmid, HepaRG hepatic progenitor cells (PXR-knockout, PXR-KO; AhR-knockout, AhR-KO; parental control 5F clone) and primary human hepatocytes (HEP) from four donors is shown. The bar graph represents one experiment of a series of experiments (n > 3) performed in four consecutive passages of LS180 cells; n = 2 independent experiments with one well/compound and RT-PCR performed in triplicate for each HepaRG genotype; for each donor hepatocyte, n = 1 well/compound and RT-PCR performed in triplicate. *, #p < 0.05, two-way ANOVA with Tukey's post hoc test. *significant over vehicle control. #significant over the same treatment in corresponding mock transfected or knock-out cells. D, Chromatin Immunoprecipitation (ChIP) assay in LS174T cells. Top panel, PCR products cells exposed to vehicle or FKK6 and run on a 2% agarose gel. DNA 100 base pair marker; Vehicle,10% DMSO; FKK6 (10 μM); 1/10 Input - 0.2 million cells before IP; IgG - IP with polyclonal rabbit IgG; PXR - IP with PXR antibody. Bottom panel, quantitative PCR from the ChIP assay for compounds tested with the gene specific PCR amplicon normalized to GAPDH (fold expression). Dash line, 2-fold expression. The data is one representative experiment of two independent experiments (each n = 3 biological replicates, n = 4 technical replicates). Mean fold expression is expressed above each histogram (mean + SD). E, PXR TR-FRET assay. TR-FRET ratio 520/495 nm are plotted against concentration of compound(s). Half-maximal inhibitory concentrations IC50 were obtained from interpolated standard curves (sigmoidal, 4PL, variable slope); error bars show standard deviation of n = 2 independent experiments each with four technical replicates. F, Microcalorimetric titrations of PXR ligand binding domain (LBD) with FKK5 and FKK6. A single-site binding model was used for fitting the data. The upper panel shows the output signal, dQ/dt, as a function of time. The middle panel shows the integrated heats as a function of the ligand/PXR molar ratio in the cell. The solid line represents the best non-linear least-squares fit of the data. The lower panel shows respective thermodynamic signatures of binding to the PXR ligand binding domain. \n",
      ": FKK6 does not abrogate Dextran Sulfate Sodium induced colitis in pxr -/- mice.Figure 6As in Fig 5, pxr -/- mice underwent the identical experimental procedure and analysis. The entire experiment was repeated three independent times and one representative experiment is shown. As in Fig 5, pxr -/- mice underwent the identical experimental procedure and analysis. The entire experiment was repeated three independent times and one representative experiment is shown. As in Fig 5, pxr -/- mice underwent the identical experimental procedure and analysis. The entire experiment was repeated three independent times and one representative experiment is shown. As in Fig 5, pxr -/- mice underwent the identical experimental procedure and analysis. The entire experiment was repeated three independent times and one representative experiment is shown. As in Fig 5, pxr -/- mice underwent the identical experimental procedure and analysis. The entire experiment was repeated three independent times and one representative experiment is shown. \n",
      "The LRR RLK PRK5 is required for GRI‐peptide‐induced ion leakageInfiltration of GRIp65–84 induced cell death similar to GRIp31–96.Figure 1A Infiltration of GRIp65-84 induced cell death similar to GRIp31-96. Bacterially produced 37 nM GST, GST‐GRIp31-96 or biochemically pure GRI‐peptides (GRIp31-96, GRIp31-51, GRIp47-68, GRIp65-84, GRIp80-96) were infiltrated into leaves of Col‐0 plants. B Infiltration of increasing concentrations of GRIp65-84 into Col‐0leaves, electrolyte leakage was measured after 12 h. Background (red line) shows ion leakage from infiltration of leaves with increasing concentrations of (inactive) GRIp80-96. C Infiltration of leaves with 37 nM GRIp31-96 induced elevated ion leakage in Col‐0 and prk4, but not in prk5‐1 or prk5‐2. Infiltration with GST caused the same background effect for all lines. D Genomic complementation of prk5 rescues the insensitivity to induction of elevated ion leakage by GRIp31-96. E Enzymatic superoxide production from xanthine/xanthine oxidase (XXO) induced more electrolyte leakage in gri compared to Col‐0 or prk5‐1, prk5‐2 and prk4 after infiltration into leaves. Infiltration with xanthine buffer (X) was used as a control. \n",
      ": FKK6 abrogates Dextran Sulfate Sodium induced colitis in mice expressing the human PXR gene.Figure 5After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. A, % weight loss from baseline (day 1 vs day 10) After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. B, colon length (cm) After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. C, Inflammation score After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. D, Fecal lipocalin 2 (pg/ml) After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. E, serum FITC-dextran (μg/ml). After coin-toss randomization, hPXR mice (mice expressing the human PXR gene) were allocated to treatment with vehicle (0.8% DMSO) (n = 3/genotype) or FKK6 (200 micromolar) (n = 3/genotype), by simultaneous oral gavage and intra-rectal delivery starting day 1 - 10 of DSS administration. F, fold expression of mRNA as illustrated in hPXR mouse colon tissue exposed to DSS (n = 3/group; each PCR performed in quadruplicate). \n",
      "PRK5 is an atypical, enzymatically inactive RLK A PRK5 domain structure: SP signal peptide (aa 1–39), LRR leucine‐rich repeat, RPT internal repeat, LC region of low complexity, TM transmembrane domain (aa 282–304), EM Y‐based sorting/endocytosis motif (YSSM; aa 670–673).Figure 2A PRK5 domain structure: SP signal peptide (aa 1-39), LRR leucine‐rich repeat, RPT internal repeat, LC region of low complexity, TM transmembrane domain (aa 282-304), EM Y‐based sorting/endocytosis motif (YSSM; aa 670-673). B PRK5‐CFP localized to the cell periphery in Col‐0mesophyll protoplasts.C-F Co‐localization of PRK5‐CFP and PM localized CAAX‐YFP in Col‐0mesophyll protoplasts. (C) overlay; (D) PRK5‐CFP (465-510 nm); (E) CAAX‐YFP (521-587 nm); (F) chloroplast (636-711 nm). G Alignment of subdomains VIb and VII of the catalytic core of the kinase domains of active (BRI1, FLS2, EFR, CRK7) and inactive RLKs (PRK4, PRK5, BIR2, SUB). Residues marked in green highlight conservation of the consensus of active protein kinases while residues highlighted in red indicate deviations from the consensus sequence. An alignment of the full kinase domains for the RLKs used in this figure can be found in Supplementary Fig S11A. H Kinase activity of PRK5 in in vitro phosphorylation assays using γ32P‐ATP and myelin‐basic protein (MBP) as a substrate in the presence of 10 mM MnCl2. GST‐PRK5 and GST‐PRK4 did not show kinase activity. Mutation of conserved residues in kinase subdomains VIb and VII to reconstitute the consensus kinase domain motif restored GST‐PRK5H500DA520G kinase activity. GST‐CRK7 was used as a positive control. Upper panel shows autoradiograph, and lower panel shows the Coomassie‐stained 15% SDS-polyacrylamide gel. \n",
      "A 20‐aa peptide binds to the extracellular domain of PRK5In vitro‐produced 35S‐labeled ectodomains of PRK5 (PRK540–281) or PRK4 (PRK440–279) were incubated with bacterially produced GST, GST‐GRI or GST‐GRI31–96 and purified.Figure 3A In vitro‐produced 35S‐labeled ectodomains of PRK5 (PRK540-281) or PRK4 (PRK440-279) were incubated with bacterially produced GST, GST‐GRI or GST‐GRI31-96 and purified. GST‐GRI31-96 but not GST directly bound to the ectodomain of PRK5. Binding of GST‐GRI to PRK5 ectodomain and binding of GST‐GRI31-96 and GST‐GRI to the ectodomain of PRK4 were strongly reduced. Upper part: autoradiograph, lower part: Coomassie‐stained 12% SDS-polyacrylamide gel. Asterisks in the Coomassie‐stained gel indicate GST, GST‐GRI and GST‐GRI31-96, respectively. Supplementary Fig S12 shows a Western blot of the GST‐tagged proteins. B Infiltration of 37 nM GST, GRIp31-96, GRIp65-84 or Y‐GRIp65-84 into Col‐0 or prk5‐2 leaves. Tyrosine‐labeled GRIp65-84 still induced cell death in Col‐0 but not in prk5‐2 plants. C 125I‐labeled Y‐GRIp65-84 (0.46 nM) bound specifically to Col‐0membrane fractions (light gray bars), the binding was significantly reduced in prk5 plants. Excess of non‐radioactive Y‐GRIp65-84 (10 μM) reduced binding to background levels (dark gray bars; all bars show the average of three samples, triangles show individual data points). D Immunoprecipitation of PRK5‐c‐myc, PRK4‐c‐myc or YFP expressed in protoplasts with rabbit polyclonal anti‐c‐myc antibody, immunoblotted with mouse monoclonal anti‐c‐myc antibody. E Binding of 125I‐Y‐GRIp65-84 (0.46 nM) to immunoprecipitates (using anti‐c‐myc antibodies) from prk5‐2 protoplasts transfected with PRK5‐c‐myc, PRK4‐c‐myc or YFP, respectively. Binding was competed out with 10 μM unlabeled Y‐GRIp65-84. Bars show the average of two samples, and triangles show individual data points. Western blot is shown in panel (D). F Analysis of 125I‐Y‐GRIp65-84 (0.46 nM) binding competed out with increasing amounts of unlabeled Y‐GRIp65-84 to Col‐0membrane extracts. Fifty percent inhibition (IC50) occurred at 25.2 nM. Red line shows binding average competition according to a sigmoid curve, dotted lines show 95% confidence intervals, and circles show data points. G Excess of GRIp65-84 (10 μM) but not of GRIp31-96 or other peptides (GRIp31-51, GRIp47-68, GRIp80-96) competed the binding of 0.46 nM 125I‐Y‐GRIp65-84 to membrane extracts from Col‐0 (all bars show the average of two samples, triangles show individual data points). \n",
      "Processing of GRI‐peptide by METACASPASE‐9 is required for induction of elevated ion leakageRecombinant AtMC9 (rAtMC9) cleaved GRI25–168 in vitro.Figure 4A Recombinant AtMC9 (rAtMC9) cleaved GRI25-168in vitro. Bacterially produced MBP‐GRI25-168 (from 0 to 2 pmol left to right) was incubated with 1 pmol rAtMC9 or inactive rAtMC9mut (rAtMC9C147AC29A), and cleavage products were analyzed by Western blot with anti‐MBP and anti‐AtMC9 antibodies. Arrowheads in the anti‐MBP blot (from top to bottom) indicate MBP‐GRI25-168 (63.5 kDa), MBPMCS (53.5 kDa), rAtMC9‐cleaved MBP‐GRI25-168 (46 kDa). Arrowheads in the anti‐AtMC9 blot indicate: 1: N‐terminal domain + p20 + p10 subunits of AtMC9, 2: p20 + p10 subunits of AtMC9, 3: N‐terminal domain + p20 subunit of AtMC9, 4: p20 subunit of AtMC9. B Infiltration of wild‐type, prk5‐1 and atmc9‐1 leaves with 37 nM GRIp31-96, GRIp65-84, GRIp68-78 or GST. GRIp65-84 and GRIp68-78 but not GRIp31-96 were able to induce elevated ion leakage in the atmc9‐1 mutant. C Schematic representation of the GRI and the cleavage sites for rAtMC9. The cleavage product detected in Western blot analysis with anti‐MBP antibody is shown as black bar. Mass spectrometric analysis of MBP‐GRI25-168 cleavage with rAtMC9 provided evidence for cleavage after SKTR and KANK; further analysis of GRIp31-96 cleavage by rAtMC9 provided evidence for cleavage after SK, SKTR and after KKIKK. The position of the resulting 11‐aa‐long peptide (68LLVSHYKKIKK78) is indicated by a white inset. D Infiltration of Col‐0leaves with 37 nM of GRIp65-84, GRIp68-78, Y‐GRIp68-78, GRIp31-96 or GST. Y‐GRIp68-78 showed similar activity in cell death induction compared to the other GRI‐derived peptides. E 125I‐labeled Y‐GRIp68-78 (0.46 nM) bound specifically to Col‐0membrane fractions (light gray bars), the binding was significantly reduced in prk5‐1 and prk5‐2 plants. Excess of non‐radioactive Y‐GRIp65-84 (10 μM) reduced binding to background levels (dark gray bars; all bars show the average of four samples, triangles show individual data points). F Excess of GRIp68-78 and GRIp65-84 (10 μM) but not of GRIp31-96 or other peptides (GRIp31-51, GRIp47-68, GRIp80-96) competed the binding of 0.46 nM 125I‐Y‐GRIp68-78 to membrane extracts from Col‐0 (all bars show average of four samples, triangles show individual data points). G Saturation binding curve for 125I‐Y‐GRIp68-78 to Col‐0membrane extracts. Specific binding was calculated by subtracting non‐specific binding from the total binding. The affinity of 125I‐Y‐GRIp68-78 to the receptor (Kd = 1.9 nM) was calculated by non‐linear regression analysis. Scatchard plot is shown in Supplementary Fig SI7. \n",
      ": CHK1 is an essential survival factor for Hoxb8-FL cells.Figure 1Hoxb8-FL cells generated from bone marrow of mice of the indicated genotypes were treated for 24h with graded doses of the CHK1-inhibitors PF-477736 (PF) or CHIR-124 (CHIR). Representative histograms of CHK1i-treated (grey) or DMSO-treated (black) cells, analyzed for cell cycle distribution and cell death using Nicoletti staining and flow cytometry are shown. WT Hoxb8-FL cells were treated with CHIR-124 in the presence or absence of the pan-caspase inhibitor QVD. Cell death was assessed after 24h using flow cytometry as in (A). Western blot of CHIR-treated Hoxb8-FL WT and Bax-/-Bak-/- cells using the indicated antibodies. WT and Bax-/-Bak-/- Hoxb8-FL cells were treated for up to 8h with 500 nM CHIR and p21 mRNA levels were compared to Hprt in response to CHK1-inhibition by real-time RT-qPCR. Bars represent means ± S.D. of independent experiments (n=4-6), performed in duplicates. 50.000 Hoxb8-FL cells of the indicated genotypes were seeded to assess cell growth in response to CHK1-inhibition. Cells were counted 48h post seeding. \n",
      ": CHK1-inhibition kills primary murine HSPCs.Figure 2LSK (Lin- Sca1+ cKit+) cells of the indicated genotypes were isolated by cell sorting from the fetal liver at embryonic day E13.5 or the bone marrow of adult mice and treated for 48h with PF-477736 [1.5 μM]. Survival was assessed using AnnexinV-staining and flow cytometry. Bars represent means ± S.E.M. from n=6 wild type and n=3 Vav-BCL2 fetal liver LSK cells (left) and n=3 bone marrow LSK cells (right). WT and Vav-BCL2 derived E14.5 total fetal liver cells (B) were treated for 72h (B) with different doses of CHK1i. Survival was assessed using Nicoletti staining and flow cytometry. Bars represent means ± S.E.M. (n=3 biological replicates per genotype). WT and Vav-BCL2 derived E14.5 total bone marrow cells (C) were treated for up to 48h with different doses of CHK1i. Survival was assessed using Nicoletti staining and flow cytometry. Bars represent means ± S.E.M. (n=3 biological replicates per genotype). \n",
      ": CHK1-inhibition kills human cord-blood derived CD34+ HSPCs in a BCL2 regulated manner.Figure 3MACS-purified CD34+ human cord-blood derived HSPC were treated with PF-477736 or CHIR-124 ± the caspase-inhibitor QVD (50 μM). Cell death was assessed using AnnexinV/7-AAD staining and flow cytometry. CD34+ cells were transduced with a BCL2-encoding lentivirus or a control virus, along with IRES-GFP with a MOI of 10 for two consecutive days, followed by CHK1i application for 48 h. Survival was assessed using Nicoletti staining and flow cytometry. Percentage of living GFP+ transduced human CD34+ HSPCs, cultured in the absence or presence of CHK1i for 48 hours. Colony formation potential of human CD34+ HSPCs transduced with empty control or BCL2 encoding virus. Colonies were counted 10 days post seeding of 1000 CD34+ cells per plate/sample. \"Myeloid colonies\" include granulocyte/monocyte, monocyte, granulocyte and GEMM (granulocyte, erythroid, monocyte/macrophage and megakaryocyte) colonies. \n",
      ": Deletion of Chk1 in HSC prevents fetal hematopoiesis.Figure 4Representative pictures of E13.5 and E18.5 embryos of the indicated genotypes. Scale bar E13.5 = 1 mm, E18.5 = 5 mm. Quantification of fetal liver cell number at E13.5. Bars represent means ± S.E.M. (N=4 for Chk1fl/+, N=3 for Chk1fl/+ Vav-Cre and N=4 for Chk1fl/- Vav-Cre). Representative pictures of isolated fetal livers at E13.5. Scale bar = 1 mm. Representative H&E-staining of fetal liver sections from E13.5 embryos. Black arrows indicate lack of erythroid cells in the blood vessels of Chk1fl/- Vav-Cre fetal livers. Scale bar = 500 µM. Single cell suspensions of fetal livers from Chk1fl/+, Chk1fl/+ Vav-Cre and Chk1fl/- Vav-Cre embryos were stained with antibodies recognizing ckit, CD71, Ter119 or CD45 to assess primitive and definitive erythropoiesis by flow cytometry. Large 'EryA' erythroblasts (CD71high Ter119high FSChigh), smaller, more mature 'EryB' erythroblasts (CD71high Ter119high FSClow); most mature erythroblast subset is EryC (CD71low Ter119high FSClow). Bars represent means ± S.E.M. ctrl N=5 (pooled N=4 Chk1fl/+ and N=1 Chk1fl/+ Vav-Cre), N=3 for Chk1fl/- Vav-Cre. \n",
      ": CHK1-deficiency causes stem cell loss in the fetal liver.Figure 5Representative dot-plots of the LSK cell phenotype of Chk1fl/+ and Chk1fl/- Vav-Cre embryos observed in E13.5 fetal livers and relative distribution / absolute numbers of LKhi and LSKhi cells, as well as LKlow and LSKlow cells, found in the fetal liver on E13.5. Bars represent means ± S.E.M. N=4 for Chk1fl/+ and N=4 for Chk1fl/- Vav-Cre. Colony formation potential of FACS-sorted LSK cells (left panel, N=4 for Chk1fl/+ and N=5 for Chk1fl/- Vav-Cre) and total fetal liver cells (right panel, N=3 for Chk1fl/+ and N=6 for Chk1fl/- Vav-Cre) in methyl-cellulose assays. Bars represent means ± S.E.M. Flow cytometric analysis of LSKhi or LSKlow cells stained with antibodies specific for CD34 and Flt3 (N=4 for Chk1fl/+ and N=4 for Chk1fl/- Vav-Cre), to discriminate LT- and ST-HSC from MPP within the LSK-subset. Bars represent means ± S.E.M. \n",
      "-deficient HSPCs accumulate DNA-damage during compensatory proliferation triggering cell death.Figure 6Terminal deoxynucleotidyl transferase dUTP nick end labeling (TUNEL) staining of fetal liver cryosections of the indicated genotypes at E13.5 and quantification. Scale bar 100x = 200 µM, 400x = 50 µM. The percentage of TUNEL+ area/field was assessed across three random fields from two control (Chk1fl/+) and two knock-out embryos (Chk1fl/- Vav-Cre) using ImageJ, as reported by us before (60). Bars represent means ± S.D. Total fetal liver cell suspensions of Chk1fl/+ (+/+), Chk1fl/+ Vav-Cre (+/-) and Chk1fl/- Vav-Cre (-/-) E13.5 embryos were processed for Western blot analysis using the indicated antibodies. Each lane represents samples isolated from an individual littermate embryo. DAPI-negative viable LK- and LSK-cells of the indicated genotypes (littermates) were sorted from E13.5 fetal livers and immediately fixed in 70% EtOH. Fixed cell suspensions were stained for ɣH2A.X and DAPI intracellularly. Shown here are representative dot-plots and quantification of N=4 embroys per genotype; Bars represent means ± S.E.M. Total E13.5 fetal liver cells were processed for intracellular Ki67 staining in combination with cell surface antibody staining to discriminate LK (top panel) from LSK (lower panel) cells. Shown here are representative dot-plots of Ki67 and DAPI staining in LK and LSK cells, quantified in (E). Bars represent means ± S.E.M. N=4 for Chk1fl/+ and N=5 for Chk1fl/- Vav-Cre. \n",
      "Loss of p53 hastens EMT in NFATc1‐driven pancreatic cancer modelsGenome‐wide expression and GSEA analysis show p53‐enriched signatures with “EMT” and “stem cell transcript” in KNC tumor cells.Figure 2A Genome‐wide expression and GSEA analysis show p53‐enriched signatures with \"EMT\" and \"stem cell transcript\" in KNCtumor cells. B Heat map showing p53‐dependent regulation of \"EMT\"‐ and \"stemness\"‐related genes in KNCtumor cells. Fold change relative to control cells is displayed in a green red color scheme for selected genes with FClog2 1.0 or FClog2 > −1.0. C Statistical analysis of wound healing assay performed in primary KNCtumor cells following p53 depletion. Time‐lapse analyzer software was used to quantify the wound closure rate. Means ± SD are shown from one out of three independent experiments. D Western blot analysis revealing protein expression levels upon p53 depletion in KNCtumor cells. E X‐fold mRNA expression of EMT‐related marker genes after p53 depletion in KNCtumor cells. Means ± SD from at least three independent experiments are shown. Asterisks display significance (*P 0.05). F Representative protein expression of EMT inducers in pancreatic lysates prepared from KNCmice with different tumor gradings. \n",
      ": Apoptosis inhibition cannot restore hematopoiesis in the absence of CHK1.Figure 7Representative dot-plots and quantification of E13.5 fetal liver LSKhi/LSKlow cell phenotypes observed in embryos of the indicated genotypes. N=21 for Chk1fl/+, N=5 for Chk1fl/- Vav-Cre, N=16 for Chk1fl/+ Vav-BCL2, N=5 for Chk1fl/- Vav-Cre Vav-BCL2, N=2 for p53-/- Chk1fl/+ and N=4 for p53-/- Chk1fl/- Vav-Cre. Flow cytometry-based analysis of LSK cells stained with antibodies specific for CD34 and Flt3, to discriminate LT- and ST-HSC from MPP within the LSK subset in fetal livers of embryos of the indicated genotypes. Left: N=25 for Chk1fl/+, N=7 for Chk1fl/+ Vav-Cre and N=7 for Chk1fl/- Vav-Cre. Right: N=16 for Chk1fl/+ Vav-BCL2, N=6 for Chk1fl/+ Vav-Cre Vav-BCL2 and N=5 for Chk1fl/- Vav-Cre Vav-BCL2. Analysis of the colony formation potential of E14.5 total fetal liver cells of the indicated genotypes in methylcellulose. N=3 for controls (pooled; 1 Chk1fl/+, 2 Chk1fl/+ Vav-Cre), N=3 for Chk1fl/- Vav-Cre and N=3 for Chk1fl/- Vav-Cre Vav-BCL2. Western blot analysis of fetal liver cells of the indicated genotypes (each lane represents an embryo) at E14.5 using the indicated antibodies. \n",
      "Loss of p53 tumor suppressor function and constitutive activation of NFATc1 correlates with highly metastatic and dedifferentiated PDACStatistical analysis of NFATc1 expression with respect to tumor grading in KPC mice tissues.Figure 1A Statistical analysis of NFATc1 expression with respect to tumor grading in KPCmice tissues. B Immunohistochemistry detection of NFATc1 and p53, illustrating G1‐G2 and G3 in human PDAC tissues. Scale bars, 100 μm. C Statistical illustration of TMA analysis demonstrating NFATc1 strong nuclear positive tumors (n = 161 patients) with respect to functional status of p53 expression in human PDAC tissues. D Representative H&amp;amp;E‐stained sections as well as immunohistochemistry detection of HA‐NFATc1 and p53 in well‐ and poorly differentiated pancreatic tissues of KNCmice. Scale bars, 100 μm.E Quantification of p53 expression with respect to the tumor grading by evaluating both the intensity of immunostaining and the number of cells with overexpression by means of an immunoreactivity score (IRS) in KNCmice (every dot represents one mouse). F Incidence of livermetastases in KNCmice (mean values ± SD). \n",
      ": Acute depletion of Chk1 in the hematopoietic system leads to cell loss in vitro and retention of CHK1 expression in vivo.Figure 8Three independent Chk1fl/fl mT/mG (coloured) and mT/mG (grey) Hoxb8-FL progenitor cell lines were transduced with a retroviral construct carrying Cre (pMIP-CRE-IRES-PURO) and monitored over time for GFP-expression. Shown here are three technical replicates per genotype. Additionally, Chk1fl/fl mT/mG Hoxb8-FL progenitor cells were FACS-sorted for GFP+ and dsRED+ cells 48h after transduction with the Cre-construct. Sorted cells were processed for western blotting using the indicated antibodies. Chk1fl/fl Hoxb8-FL progenitor cells were transduced with a retroviral construct carrying Cre and GFP (pMIG-CRE-IRES-GFP). Cells were FACS-sorted 72h after transduction with the retroviral construct and processed for western blotting using the indicated antibodies. Percentage of GFP+ cells indicative for CRE expression and recombination in the peripheral blood after tamoxifen administration by gavage (5 x 2 mg, 200 μl/mouse; daily). Bars represent means ± S.E.M. (Day 0 N=9/10, WT/Chk1fl/fl, Day 7 N=8/7, Day 14 N=3/3). Quantification of Chk1-del PCR-product enrichment in DNA from GFP+ vs. Tomato+ bone marrow cells sorted on day 8 after first TAM administration. Transgenic Cre levels were quantified in parallel for reference. Bars represent means ± S.E.M. from n=5 mT/mG Vav-CreERT2 Chk1fl/fl mice. Percentage of living LK or LSK-cells and (F) percentage of GFP+ LK or LSK-cells found in the bone marrow of mT/mG Vav-CreERT2 (WT) and mT/mG Vav-CreERT2 Chk1fl/fl (Chk1fl/fl) mice on day 8 or 15 post first TAM administration. Bars in (E)/(F) represent means ± S.E.M. (Day 8 N=8/8, WT/Chk1fl/fl, Day 15 N=7/8). GFP+ (CRE) and Tomato+ (CRE-) total bone marrow cells were sorted on day 8 after first TAM-administration and processed for Western Blot using the indicated antibodies. Sorted cells from three animals - mT/mG Vav-CreERT2 (WT) and mT/mG Vav-CreERT2 Chk1fl/fl (Chk1fl/fl) - were pooled per lane. Thymocytes of the indicated genotypes were sorted based on GFP+ and Tomato+ on day 15 after the first TAM administration for western analysis using the indicated antibodies. \n",
      "NFATc1 is required for the promotion of EMT in PDACQuantitative RT‐PCR analysis of EMT‐related markers obtained from tumors derived from mice of the indicated genotypes (n = 3, means ± SD).Figure 4A Quantitative RT‐PCR analysis of EMT‐related markers obtained from tumors derived from mice of the indicated genotypes (n = 3, means ± SD). B Relative mRNA expression of NFATc1‐ and EMT‐related marker genes on administration of CsA (1 μM) in KPNC cells. Data represents means ± SD from at least three independent experiments. Asterisks show significance (*P 0.05). C Western blot analysis revealing protein expression levels in KPNC cells from the indicated proteins following transfection of siRNA control or siRNA toward NFATc1, respectively. \n",
      "p53 inactivation enables NFATc1 to drive dedifferentiated PDAC A Kaplan–Meier curves showing survival of KNC, KPNC, and KP∆NC mice under the control of p48‐Cre pancreas‐specific promoters (P  0.Figure 3A Kaplan-Meier curves showing survival of KNC, KPNC, and KPΔNC mice under the control of p48‐Cre pancreas‐specific promoters (P 0.0001 for KPNC or KPΔNC cohorts versus KNC, log‐rank test, for pairwise combination). B Pathological features of KPNC and KP∆NC mice before tumor extraction. Pancreatic tissue is encircled. C, D HE and corresponding immunohistochemistry stainings of (C) KPNC and (D) KP∆NCmouse tumors. Scale bars, 100 μm. E Tumor grading of PDAC from KPNC mice (n = 10). \n",
      "The NFATc1–Sox2 complex promotes EMT and cancer stem cell‐like properties in PDACNFATc1 and Sox2 protein expression in L3.Figure 6A NFATc1 and Sox2 protein expression in L3.6 cells (adherent versus spheres). B Relative mRNA expression of NFATc1 and Sox2 in L3.6 cells as well as in KPNCmicetumor cells (adherent versus spheres). Graph represents data from three independent experiments (means ± SD). Asterisks display significance (*P 0.05). C Cell sorting was performed to collect CD44 high and low populations in KPNC spheres (left panel), which were analyzed for NFATc1 and Sox2 mRNA expression (right panel). D Sox2 mRNA expression upon CsA (1 μM) treatment in L3.6 cells (adherent versus spheres) (*P 0.05). E Bright‐field light microscopy images as well as quantification of KPNC spheres after transient NFATc1 depletion. \n",
      "NFATc1 promotes Sox2 transcription in PDAC A Sox2 mRNA expression of pancreatic lysates from KNC, KPNC, and KPC mice, analyzed by real‐time PCR in three independent experiments (means ± SD).Figure 5A Sox2 mRNA expression of pancreatic lysates from KNC, KPNC, and KPC mice, analyzed by real‐time PCR in three independent experiments (means ± SD). B, C Western blot analysis shows Sox2 expression in pancreatic lysates of (B) KNC mice with indicated tumor grading and in (C) KPNC and KPC mice. D Protein expression levels after siRNA‐mediated p53 depletion in KNC tumor cells. E, F NFATc1 and Sox2 protein expression after (E) NFATc1 depletion or (F) upon CsA (1 μM, 24 h) treatment in Panc1 cells. G Sox2 and E‐cadherin mRNA expression after NFATc1 depletion in L3.6 cells. H Sequence alignment of 5′ and 3′ enhancer regions show highly conserved NFATc1 binding in different species; putative NFAT binding sides within the enhancer regions of the Sox2 promoter are highlighted in red. I, J ChIP assays show NFATc1 binding to Sox2 enhancer and promoter in KNC tumor cells (I) I, J ChIP assays show RNA polymerase II binding to Sox2 promoter upon CsA (1 μM, 24 h) treatment in L3.6 cells (J). K Co‐immunoprecipitation of endogenous HA‐NFATc1 and Sox2 was performed in KPNC cells. L ChIP experiments show NFATc1 binding along with H3K27 acetylation (H3K27ac) mark at the selected Snai1 enhancer region and RNA polymerase II binding at Snai1 promoter in the presence or absence of Sox2 in KPNC tumor cells. Representative results from at least three independent experiments are shown (means ± SD; P 0.05). M Relative mRNA expression levels of EMT‐related marker genes upon Sox2 depletion in KPNC tumor cells (P 0.05). N Zeb1 and Snail mRNA levels in L3.6 cells after shRNA‐mediated depletion of Sox2 and transient overexpression of NFATc1 shows that loss of Sox2 significantly rescues NFATc1‐mediated induction of EMT genes (*P 0.05). \n",
      "Proposed model of antithetical NFATc1–Sox2 and p53–miR‐200c network in pancreatic cancer plasticityA proposed model shows an antidromic NFATc1–Sox2 and p53–miR200c transcriptional network controlling EMT and stemness in pancreatic cancer.Figure 8A proposed model shows an antidromic NFATc1-Sox2 and p53-miR200c transcriptional network controlling EMT and stemness in pancreatic cancer. \n",
      "p53 constrains NFATc1‐dependent promotion of EMT phenotype and cancer stem cell‐like properties through induction of miRNA‐200c A Expression analysis of indicated miRNAs upon adenoviral re‐expression of p53 in KP∆NC tumor cells, detected by real‐time PCR (P  0.Figure 7A Expression analysis of indicated miRNAs upon adenoviral re‐expression of p53 in KPΔNC tumor cells, detected by real‐time PCR (P 0.05). B Protein expression of EMT marker genes after adenoviral re‐expression of GFP‐tagged p53 in KPΔNC tumor cells. C, D Protein (C) expression levels of EMT and stemness marker genes upon miRNA‐34a and miRNA‐200c overexpression using specific mimics in KPNC tumor cells. Representative results from at least three independent experiments are shown (means ± SD; P 0.05). C, D mRNA (D) expression levels of EMT and stemness marker genes upon miRNA‐34a and miRNA‐200c overexpression using specific mimics in KPNC tumor cells. Representative results from at least three independent experiments are shown (means ± SD; P 0.05). E Western blot analysis of EMT marker expression in KNC cells following depletion of miR‐200c. F ChIP experiments showing NFATc1 binding along with H3K27 acetylation mark at the selected Snai1 enhancer region as well as Sox2 and RNA polymerase II binding at Snai1 promoter after miR‐200c overexpression in KPNC tumor cells. Means ± SD are shown from one out of three independent experiments. G ChIP analysis in KP∆NC tumor cells after adenoviral GFP‐tagged p53 overexpression. NFATc1 and H3K27ac binding are illustrated on Snai1 enhancer, and Sox2 and RNA polymerase II binding are demonstrated on Snai1 promoter. Means ± SD are shown from one out of three independent experiments. H Quantification of KP∆NC tumour cell-derived spheres after adenoviralGFP‐tagged p53 overexpression. \n",
      ".Figure 1(A) Confocal images of representative eye cryosections immunostained with anti-Lamp1 antibody from 2-month-old miR-211-/- and control mice sacrificed 3h after light on, at 10 AM (diurnal condition). Enlarged boxes highlight Lamp1-positive structures (white arrowheads) in the RPE. Autofluorescence from lipofuscin granules are visible in all cryosections of miR-211-/- compared to control littermates. Nuclei were counterstained with DAPI (blue). At least n = 6 mice per group. Scale bar 10 μm. Data information: (RPE) Retinal Pigment Epithelium; (OS) outer segment; (ONL) outer nuclear layer. (B) RPE were isolated at 10 AM (diurnal condition) from 2-month-old WT and miR-211-/- mice. Representative Western blot analysis of the Lamp1, Cln5, Trpml1, Sqstm1/p62, Cathepsin D (CtsD) and LC3 proteins from WT and miR-211-/- mice. Note a decrease of both proCtsD and its maturation CtsD heavy chain (hc). The plots show the quantification of the indicated proteins normalized to the Gapdh loading control. Bar graphs represent mean values ± s.e.m. of independent experiments (n=6 mice) Mann and Whitney test (miR-211-/- vs WT), *p ≤ 0.05, **p ≤ 0.01. Data information: (RPE) Retinal Pigment Epithelium (C) Cathepsin B (CtsB) activity in RPE lysates from 2-month-old miR-211-/- and control mice sacrificed 3h after light on at 10 AM (diurnal condition). CtsB was reduced in miR-211-/- compared to control mice. Bar graphs represent percentage of CtsB activity ± s.e.m. of independent experiments (n=3 mice) Mann and Whitney test (miR-211-/- vs WT), *p ≤ 0.05. Data information: (RPE) Retinal Pigment Epithelium (D) Representative images of conventional TEM analysis of RPE of both 2-month-old and 3-month-old WT and miR-211-/- mice. miR-211-/- mice show accumulation of phagolysosomes (red arrows) in the RPE. Scale bar 2 μm. Enlarged boxes highlight phagolysosome-like structures containing poorly processed POS. Scale bar 1.5 μm. Representative images of conventional TEM analysis of RPE shown at higher magnification also highlighted accumulation of lipofuscin-like granules (blue arrows) in 3-month-old miR-211-/- compared to control WT mice. Melanosomes are indicated by white asterisks. Scale bar 500 nm (n=3 mice). (E) Graphs showing number of lipofuscin-like granules (n/100 μm2) from the RPE of 3-month-old miR-211-/- mice as in (D). Bar graphs represent mean values ± s.e.m. Mann and Whitney test (miR-211-/- vs WT) ***p ≤ 0.005 (n=6 mice). Data information: (RPE) Retinal Pigment Epithelium \n",
      ".Figure 3(A) Representative images from ARPE-19 cells treated with DMSO and NSC668394. All cells were fixed and stained with anti-LC3 (green) and anti-LAMP1 (red) antibodies and nuclei were counterstained with DAPI (blue). Scale bar 5 μm. (B) The plot shows the quantification of the numbers of LC3/LAMP1-positive vesicles per cell (n ≥ 100). Values represent means ± s.e.m from 3 independent experiments. Student's t-test (NSC668394 vs DMSO) ***p ≤ 0.005. (C) Representative Western blots of LAMP1, EZRIN, EZRIN-pT567, CLN5, TRPML1, SQSTM1/p62, CTSD and LC3 proteins from DMSO or NSC668394 treated cells. Note an increase of both proCTSD and its maturation CTSD heavy chain (hc). The plot shows the quantification of the indicated proteins normalized to the GAPDH loading control. Bar graphs represent mean values ± s.e.m. of at least n = 6 independent experiments. Mann and Whitney test (NSC668394 vs DMSO) *p ≤ 0.05, ***p ≤ 0.005. (D) Representative images of RFP-GFP-LC3 assay in ARPE-19 cells transiently transfected with RFP-GFP-LC3 and treated with DMSO or NSC668394. Nuclei were counterstained with DAPI (blue). Scale bars: 5 μm. (E) Box plots showing quantitative analysis of RFP+GFP+ puncta (Autophagosome) and RFP+GFP- puncta (Autolysosome ) (n ≥ 100 cells) from 3 independent experiments. Box limits represent 25th percentile and 75th percentile; horizontal lines represent medians; whiskers display min. to max. values. Student's t-test (NSC668394 vs DMSO) *p ≤ 0.05. (F) Endolysosomal Cathepsin activity was revealed by incubation with cathepsin B (CTSB) Magic Red substrate for 20 min and images of the living cell collected on the HC analysis. Scale bar 10 μm. (G) Graph showing the mean values of intensity of cresyl violet fluorescence ± s.e.m. of representative data (n ≥ 100 cells) from 3 independent experiments. Mann and Whitney test (NSC668394 vs DMSO) *p ≤ 0.05. (H) Cathepsin B (CTSB) activity in lysates from NSC668394-treated and DMSO-control ARPE-19 cells. Graph showing the percentage of values ± s.e.m. of CTSB activity from 4 independent experiments. Mann and Whitney test (NSC668394 vs DMSO) *p ≤ 0.05. \n",
      ".Figure 5(A) Confocal images of representative eye cryosections immunostained with anti-Ezrin (green) /anti-ZO-1 (red), anti-Ezrin-pT567 (green) /anti-ZO-1 (red), anti-Lamp1 (green) and anti-Sqstm1/p62 (green) antibodies from 6-month-old miR-211-/- sacrificed five months after DMSO or NSC668394 treatment. Nuclei are counterstained with DAPI (blue). At least n = 6 mice per group. Scale bar 100 μm. Data information: (RPE) Retinal Pigment Epithelium; (OS) outer segment; (ONL) outer nuclear layer. (B) RPE was isolated 5 months after DMSO or NSC668394 treatment from 6-month-old miR-211-/- mice. Representative Western blots proteins from these tissues were performed to determine the expression levels of Lamp1, Ezrin, Ezrin-pT567, Sqstm1/p62 and LC3. The plots show the quantification of the indicated proteins normalized to the β-Actin loading control. Bar graphs represent mean values ± s.e.m. of independent experiments (n=6 mice). Mann and Whitney test (DMSO vs WT; NSC668394 vs DMSO), *p ≤ 0.05, ***p≤0.005. (C) Cathepsin B (CtsB) activity in RPE lysates from miR-211-/- and WT mice sacrificed 3h after light on at 10 AM (diurnal condition). RPE was isolated 1 week after DMSO or NSC668394 treatment from 2-month-old miR-211-/- mice. CtsB was rescued in NSC668394-treated miR-211-/- compared to DMSO-control miR-211-/- mice. Bar graphs represent percentage of CtsB activity ± s.e.m. of independent experiments (n=3 mice) Mann and Whitney test (DMSO-miR-211-/- vs WT; NSC668394-miR-211-/- vs DMSO-miR-211-/-), *p ≤ 0.05. (D) GUSB activity from retina from miR-211-/- and control mice sacrificed 3h after light on at 10 AM (diurnal condition). Retina was isolated 1 week after DMSO or NSC668394 treatment from 2-month-old miR-211-/- mice. GUSB was rescued in NSC668394-treated miR-211-/- compared to DMSO-control miR-211-/- mice. Bar graphs represent percentage of GUSB activity ± s.e.m. of independent experiments (n=3 mice). Mann and Whitney test (DMSO-miR-211-/- vs WT; NSC668394-miR-211-/- vs DMSO-miR-211-/-). \n",
      ".Figure 4(A) Representative images from HC assay of HeLaTFEB-GFP cells transfected with control (siCTRL) or siPPP3CB, and subjected to the indicated conditions. Analysis of TFEB nuclear translocation in HeLaTFEB-GFP cells transfected with siCTRL or siPPP3CB and treated with DMSO or NSC668394, silenced for EZRIN (siEZR) or serum-starved (stv). Both pharmacological inhibition and silencing of Ezrin induced TFEB nuclear localization in stable HeLaTFEB-GFP cells. Starvation (stv) was used as control. Nuclear translocation of TFEB in stable HeLaTFEB-GFP cells subjected to the indicated conditions is rescue after silencing of PPP3CB. Scale bar 5 μm. (B) The graph shows the mean ± s.e.m. of the percentage of nuclear TFEB translocation in Ezrin-inhibited cells compared to control; n=3 independent experiments were performed. Mann and Whitney test (NSC668394 and stv vs DMSO; siEZR vs siCTRL) *p ≤ 0.05. (C) The graph shows the mean ± s.e.m. of the percentage of nuclear TFEB translocation in Ezrin-inhibited cells and subjected to the silencing of PPP3CB. At least 3 independent experiments were performed. Mann and Whitney test (siPPP3CB vs siCTRL) *p ≤ 0.05. (D) qRT-PCR assay for PPP3CB from HeLaTFEB-GFP cells transfected with siPPP3CB and siCTRL. The graph shows the reduction in PPP3CB expression level normalized to the Hprt. Bar graphs represent mean values ± s.e.m. Mann and Whitney test (siPPP3CB vs siCTRL), ***p ≤ 0.005 (n=6 independent experiments). (E) Both pharmacological inhibition and silencing of EZRIN induce downshift of endogenous TFEB electrophoretic mobility in Western blot analysis. Torin treatment was used as control. (F) Representative traces of ML1-GCaMP3 normalized fluorescence recorded in transiently transfected ARPE-19 cells. During time-lapse recording, cells were stimulated with ML-SA1 (20 µM) after addition (arrowhead) of DMSO or NSC668394. Where indicated, the specific ML1 inhibitor ML-SI3 (10 µM) was added. Bar graph reports the mean values ± s.e.m. of the time required by fluorescence to decay to half of the ML-SA1-induced peak. n = 50 cells from 3 independent experiments. ***p<0.005 by Mann and Whitney test. (G) Nuclear translocation of TFEB in stable HeLaTFEB-GFP cells subjected to the indicated conditions is reduced after Ca2+ chelator BAPTA treatment. Scale bar 5 μm. H) The graph shows the mean ± s.e.m. of the percentage of nuclear TFEB translocation in Ezrin-inhibited cells compared with DMSO under Ca2+ chelator BAPTA treatment. At least 3 independent experiments were performed. Mann and Whitney test (stv, stv+BAPTA, BAPTA, NSC668394, NSC668394+BAPTA vs DMSO) *p ≤ 0.05. (I) qRT-PCR analysis for TFEB target genes (TRPML1, BECLIN, MAPLC3B and LAMP1) was performed on ARPE-19 cells treated with DMSO or NSC668394. Bar graphs represent mean values ± s.e.m. of at least 3 independent experiments. Mann and Whitney *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.005. \n",
      ".Figure 2(A) Representative confocal images of eye cryosections immunostained with anti-Ezrin (green) and anti-ZO-1 (red) antibodies from 2-month-old WT mice sacrificed 3h after light on at 10 AM (diurnal condition - LIGHT) and 3h after light off at 10 PM (nocturnal condition - DARK), nuclei were counterstained with DAPI (blue). Enlarged boxes highlight Ezrin staining in both basal infoldings (BI) and apical extension (AE) of the RPE. At least n = 6 mice per group. Scale bar 10 μm. Data information: (RPE) Retinal Pigment Epithelium; (OS) outer segment; (ONL) outer nuclear layer; (BI) RPE basal infoldings (AE) RPE apical extensions. (B) Graph representing the kinetic expression pattern of Ezrin levels in the RPE from 2-month-old WT mice during dark/light and light/dark transitions. Mice were sacrificed at determined time points as indicated on the X axis. Scheme at the top describes light/dark adaptation regime. Values normalized to Hprt, represent means ± s.e.m. Mann and Whitney test *p ≤ 0.05 (n=3 mice for each time point). Values from 12h dark-adapted mice (T0) were set to one. (C) Confocal images of representative eye cryosections immunostained with anti-Ezrin (green) and anti-ZO-1 (red) antibodies from 2-month-old miR-211-/- and WT control mice sacrificed 3h after light on at 10 AM (diurnal condition). Enlarged boxes highlight Ezrin staining in both basal infoldings (BI) and apical extension (AE) of the RPE. At least n = 6 mice per group. Scale bar 10 μm. Data information: (RPE) Retinal Pigment Epithelium; (OS) outer segment; (ONL) outer nuclear layer; (BI) RPE basal infoldings (AE) RPE apical extensions. (D) qRT-PCR assay for Ezrin from RPE 2-month-old WT and miR-211-/- mice sacrificed 3h after light on at 10 AM. The graph shows the expression level of Ezrin normalized to Hprt. Bar graphs represent mean values ± s.e.m. Mann and Whitney test (miR-211-/- vs WT), ***p ≤ 0.005 (n=6 mice). (E) RPE were isolated at 10 AM (diurnal condition) from 2-month-old WT and miR-211-/- mice. Representative image at low exposure of Western blot analysis of the Ezrin protein from WT and miR-211-/- mice. The graph shows the quantification of Ezrin normalized to the Gapdh loading control. Bar graphs represent mean values ± s.e.m. of independent experiments (n=5 mice) Mann and Whitney test (miR-211-/- vs WT), **p ≤ 0.01. (F) Graph representing the kinetic expression pattern of Ezrin levels in the RPE from 2-month-old miR-211-/- mice during dark/light and light/dark transitions. Mice were sacrificed at determined time points as indicated in X axis. Scheme at the top describes light/dark adaptation regime. Values normalized to Hprt, represent means ± s.e.m. (n=3 mice for each time point). Values from 12h dark adapted mice (T0) were set to one. (G) Graph representing the kinetic expression pattern of Trpm1 levels in the RPE from 2-month-old WT mice during dark/light and light/dark transitions. Mice were sacrificed at determined time points as indicated in X axis. Scheme at the top describes light/dark adaptation regime. Values normalized to Hprt, represent means ± s.e.m (n=3 mice for each time point). Values from 12h dark adapted mice (T0) were set to one. (H) qRT-PCR assay for Mtmr10 and Klf13 from RPE 2-month-old WT and miR-211-/- mice sacrificed 3h after on at 10 AM. The graph shows the expression level of these genes normalized to the Hprt. Bar graphs represent mean values ± s.e.m. (n=3 mice) \n",
      ".Figure 6(A) Representative images of conventional TEM analysis of RPE of WT, DMSO-treated miR-211-/- mice and NSC668394-treated miR-211-/- three-month-old mice. NSC668394-treated miR-211-/- mice show rescue of accumulation of double-membrane phagolysosome-like structures containing poorly processed POS (red arrows) compared to DMSO-treated miR-211-/- mice. Scale bar 1 mm. Data information: (OS) outer segment (RPE) Retinal Pigment Epithelium. (B) Confocal images of representative eye cryosections immunostained with anti-Rhodopsin (green) antibody from WT and miR-211-/- mice at three months of age after DMSO or NSC668394 treatment, which are sacrificed 3h after light on at 10 AM (diurnal condition). Nuclei are counterstained with DAPI (blue). At least n = 6 mice per group. Scale bar 100 μm. The enlarged box highlights rhodopsin accumulation in the RPE (green spots). Scale bar 5 μm. Autofluorescence from lipofuscin granules from WT and miR-211-/- mice at three months of age after DMSO or NSC668394 treatment. Representative images of retina cryosections immunostained with anti-Cone Arrestin (green) antibody from WT and miR-211-/- mice at three months of age after DMSO or NSC668394 treatment. Nuclei are counterstained with DAPI (blue). N = at least 6 mice per group. Scale bar 100 μm. Data information: (OS) outer segment; (ONL) outer nuclear layer; (RPE) Retinal Pigment Epithelium. (C) Graphs showing number of lipofuscin granules from the RPE of mice as in (B). Bar graphs represent mean values ± s.e.m. Mann and Whitney test (miR-211-/- DMSO vs WT and miR-211-/- NSC668394 vs DMSO treated mice) ***p ≤ 0.005. *p ≤ 0.05 (n=6 mice). (D) Graphs show cone percentage (cones/area) from the retina of mice treated as in (B). Error bars represent s.e.m. Mann and Whitney test (DMSO vs WT and NSC668394 vs DMSO treated mice) ***p ≤ 0.005 (n=6 mice). (E) Representative flicker traces at three months of age show the rescue of flicker responses of NSC668394-treated miR-211-/- mice (green lines) compared to DMSO-treated miR-211-/- control mice (red lines). WT mice were used as a control (black lines). Flicker recordings were performed with light intensities ranging from 10−4 to 15 cd s/m2 in steps of 0.6 logarithmic units at 6 Hz frequency. (F) Flicker responses, plotted as a function of stimulus intensity, from WT (black lines), DMSO-treated miR-211-/- (red lines) and NSC668394-treated miR-211-/- (green lines) mice, at three months of age. The amplitude of the recordings from NSC668394 miR-211-/- treated mice is significantly rescued compared to DMSO-treated miR-211-/- mice. WT mice were used as a control. Error bars represent s.e.m. ANOVA test (DMSO vs WT and NSC668394 vs DMSO treated mice) **p ≤ 0.01, ***p ≤ 0.005. \n",
      ".Figure 7Under light phase conditions Ezrin is repressed by miR-211. The inhibition of Ezrin releases a Ca2+ microdomain flux into cells thorough the TRPML1 channel. This leads to calcineurin (Cn) activation and lysosomal biogenesis via TFEB nuclear translocation. Under night phase conditions miR-211 is downregulated, this in turn induces Ezrin upregulation and repression of TFEB-mediated lysosomal biogenesis. Both miR-211 and Ezrin may converge on and integrate into the mTOR pathway. \n",
      ".Figure 2Representative western blot showing level of IKKε in IKBKE (IKKε)-silenced (A) T47D breast cancer cell lines. Representative western blot showing level of IKKε in IKBKE (IKKε)-silenced (B) MDA-MB-468 breast cancer cell lines. (C-D) Heatmap and hierarchical clustering of metabolite concentrations in (C) IKBKE (IKKε)-silenced T47D cells and (D) IKBKE (IKKε)-silenced MDA-MB-468 cells (n=5 technical replicates). metabolite levels were normalised to the internal standard HEPES. (E) Glycine production, representative of serine production, from glutamine (glycine m+1, 15N-glutamine labelling) and glucose (glycine m+2, 13C6-glucose labelling), and contribution of pyruvate and glucose-derived carbon to TCA cycle metabolites (citrate m+2, malate m+2, 13C6-glucose labelling) in IKBKE (IKKε)-silenced T47D cells. (n=5 technical replicates). metabolite levels were normalised to the internal standard HEPES. (F) Serine production from glutamine (serine m+1, 15N-glutamine labelling), and serine and glycine production from glucose (glycine m+2, serine m+3, 13C6-glucose labelling) as well as contribution of pyruvate and glucose-derived carbon to TCA cycle metabolites (malate m+2, 13C6-glucose labelling) in IKBKE (IKKε)-silenced MDA-MB-468 cells. \n",
      ".Figure 5(A) qRT-PCR analysis of PHGDH, PSAT1 and PSPH mRNA levels in a panel of IKBKE (IKKε)-silenced breast cancer cell lines. Data are expressed as fold changes, relative to levels in a non-silenced control of each cell line and normalised to β-Actin (n≥3 biological replicates). (B) Representative western blot showing levels of IKKε and the SBP enzymes in IKBKE (IKKε)-silenced ZR-75-1, T47D, MDA-MB-468 and MCF7 breast cancer cell lines. (C-E) Levels of the SBP enzymes in a panel of IKBKE (IKKε)-silenced breast cancer cell lines. (C) PHGDH, (D) PSAT1 and (E) PSPH levels in indicated cell lines normalised to Vinculin. Densitometry analysis quantified single sample density as a percentage of total blot density per cell line prior to vinculin normalisation (n≥3 biological replicates). (F) qRT-PCR analysis of PHGDH, PSAT1 and PSPH mRNA levels in ATF4-silenced ZR-75-1, T47D and MDA-MB-468 breast cancer cell lines. Data are expressed as fold changes, relative to levels in non-silenced control cells and normalised to β-Actin (n=3 biological replicates). (G) Representative western blot showing levels of PHGDH, PSAT1 and PSPH in ATF4-silenced ZR-75-1, MDA-MB-468, MDA-MB-231, T47D and HCC1143 breast cancer cell lines. \n",
      ".Figure 1(A) Top panel: Scheme illustrating the tetracycline inducible Flp-In 293 system that controls the expression of HA-IKKε or HA-GFP. Bottom panel: Representative western blot showing induced expression of HA-IKKε in Flp-In 293 cells treated with doxycycline (Dox, 50 ng/ml) for 16 hours compared to endogenous IKKε in T47D, MDA-MB-231 and MDA-MB-468 breast cancer cell lines. (B) Heatmap and hierarchical clustering of metabolite concentrations in Flp-In 293 HA-GFP and Flp-In 293 HA-IKKε cells treated with doxycycline (Dox, 50 ng/ml, 16 hours) (n=5 technical replicates). (C) Serine production from glucose (serine m+3, 13C6-glucose labelling, left panel) and glutamine (serine m+1, 15N2-glutamine labelling, right panel) in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours) metabolite levels were normalised to the internal standard HEPES. (D) Schematic representation of the 13C6-glucose and 15N2-glutamine labelling strategy to assess the effect of HA-IKKε induction on glycolysis, the TCA cycle and serine metabolism. (E) Contribution of pyruvate and glucose-derived carbon to TCA cyle metabolites in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours). (n=5 technical replicates). metabolite levels were normalised to the internal standard HEPES. (F) Fractional enrichment of serine, malate and citrate 13C-isotopologues in Flp-In 293 HA-GFP and Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours) (n=5 technical replicates). (G) Fractional enrichment of the serine 15-N-isotopologue in Flp-In 293 HA-GFP and Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours). m+1 shows the naturally occurring 13C isotopologue (n=5 technical replicates). metabolite levels were normalised to the internal standard HEPES. \n",
      ".Figure 4(A) Representative western blot showing level of ATF4 in ATF4-silenced Flp-In 293 HA-IKKε cells treated with doxycycline (16 hours). (B) qRT-PCR analysis of PHGDH, PSAT1 and PSPH mRNA levels in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (Dox) for 16 hours. Data are expressed as fold changes, relative to levels in non-treated Flp-In 293 HA-GFP cells and normalised to β-Actin (n=4 biological replicates) (C) Representative western blot showing levels of IKKε, IRF3, phosphorylated IRF3 (S396) and SBP enzymes in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (Dox) for 16 hours. (D) qRT-PCR analysis of PHGDH, PSAT1 and PSPH mRNA levels in ATF4-silenced Flp-In 293 HA-IKKε cells treated with doxycycline (Dox) for 16 hours. Data are expressed as fold changes, relative to levels in a non-silenced, non-treated control, and normalised to β-Actin (n=5 biological replicates). (E) Representative western blot showing levels of IKKε, PHGDH, PSAT1 and PSPH in ATF4-silenced Flp-In 293 HA-IKKε cells treated with doxycycline (Dox) for 16 hours. \n",
      ".Figure 3(A) Basal oxygen consumption in Flp-In 293 HA-IKKε wt, Flp-In 293 HA-IKKε KD-m and Flp-In 293 HA-IKKε UbLD-m cells following treatment with doxycycline (Dox, 16 hours), measured using Oroboros high resolution respirometry. Data are normalised to non-treated control cells. (B) Average TMRM staining intensity in Flp-In 293 HA-IKKε wt, Flp-In 293 HA-IKKε KD-m and Flp-In 293 HA-IKKε UbLD-m expressing cells induced by doxycycline (Dox, 16 hours). Data are normalised to non-treated control cells. (C) Basal mitochondrial oxygen consumption rate (OCR) in a panel of IKBKE (IKKε)-silenced breast cancer cell lines, measured using Seahorse XF96e or XF24 analysis. (D) OCR in mitochondria isolated from Flp-In 293 HA-GFP or HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours), measured using Oroboros high resolution respirometry. (E) Principal component analysis of differentially phosphorylated substrates in three independent single cell clones of Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (100 ng/ml, 16 hours). The phosphoproteomes in the three clones were analysed by mass spectrometry as described in material and methods. (F) Representative western blot showing l­evel of IKKε in three independent single cell clones of Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells following treatment with doxycycline (100 ng/ml, 16 hours). (G) Relative pyruvate dehydrogenase (PDH) activity in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml, 16 hours). (H) Average TMRM staining intensity in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (Dox) and DCA (both for 16 hours). Data are normalised to non-treated Flp-In 293 HA-IKKε cells. (I) Basal OCR in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml) in combination with dichloroacetate (DCA) for 16 hours, measured using Oroboros high resolution respirometry. (J) Basal OCR in Flp-In 293 HA-GFP or Flp-In 293 HA-IKKε cells treated with doxycycline (50 ng/ml) in combination with pyruvate deprivation for 16 hours, measured using Oroboros high resolution respirometry. \n",
      "Regulatory connections present in model.Figure 1Regulatory connections present in modelOur previous model of the photoperiod pathway (inset top; Song et al, 2012) included the circadian clock model of Locke et al (2005) (orange, inset top). The new model includes the circadian clock model of Pokhilko et al (2012) (orange, inset bottom). Circadian clock components are represented in red, flowering time pathway components in green and hypocotyl elongation pathway components in blue. Dashed lines represent forms of regulation that have been modelled previously; solid lines represent new regulatory connections. Protein components are represented by rectangles and mRNA by parallelograms. Model components are denoted by their abbreviated names (see text). PIF-interacting proteins (e.g. DELLAs, PAR1, HFR1) are designated \"Interactors\" in the hypocotyl elongation pathway. \n",
      "Modelling the circadian regulation of CDF1 and FKF1 mRNA.Figure 2Schematic of proposed circadian regulators of FKF1 and CDF1 transcription. Experimental validation for CCA1 regulation of the flowering pathway. ChIP data showing CCA1 enrichment in regions containing CCA1 elements (CBS or EE; white bars) in GI, FKF1 and CDF1 promoters, compared to regions of their respective exons (dark bars). Locations of primers for GI (GI-a, GI-N), FKF1 (FKF1-a, FKF1-N) and CDF1 (CDF1-a, CDF1-N) are shown in Supplementary Fig S3. Seedlings were grown for 14 days 12L:12D cycles at 22°C and harvested at ZT2. Statistical analysis performed using Welch tests, *P < 0.005. Error bars represent standard error of technical replicates. CDF1 mRNA in LDs (C) and FKF1 mRNA in SDs (D), from WT (data: black lines, filled squares; simulation: black lines), and the cca1;lhy mutant (data: green lines, open circles; simulation: dashed green lines) (data sets used for parameter optimisation). CDF1 (E) and FKF1 (F) mRNA data in LDs, from WT (as in C, D) and the prr9;7 mutant (blue-green line, open circles). CDF1 (G) and FKF1 (H) mRNA simulations in LDs, from WT (as in C, D) and the prr9;7 mutant (dashed blue-green line). \n",
      "Modelling the regulation of CO and FT mRNA.Figure 3Schematic of the role of GI in destabilising CDF1 through an FKF1-dependent and FKF1-independent mechanisms, with resulting effects on CO and FT mRNA abundance. CDF1 protein data and simulations in a 35S:3HA-CDF1 line (CDF1ox; black lines, open squares) and 35S:HA-CDF1;fkf1 mutant (CDF1ox;fkf1; blue lines, open squares) in LDs. Data from Imaizumi et al (2005). CDF1 protein data and simulations in a 35S:HA-CDF1 line (CDF1ox; as in B) and 35S:HA-CDF1;gi-2 mutant (CDF1ox;gi; red lines, open squares) in LDs. Plants were grown for 10 days in 16L:8D cycles. CO data and simulations in 16L:8D in WT (data: black lines, filled squares; simulation: black lines), the gi-2 mutant (data: red lines, open circles; simulation: dashed red lines) and fkf1 mutants (data: blue lines, open circles; simulation: dashed blue lines). Data from Sawa et al (2007). CO and FT mRNA data and simulations in LDs in WT (as in D, E) and the prr9;7 mutant (data: blue-green lines, open circles; simulation: dashed blue-green lines). Data from Nakamichi et al (2007). CO and FT mRNA data and simulations in 10L:14D in WT (as in D) and the cca1;lhy mutant (data: green lines, open circles; simulation: dashed green lines). Data from Nakamichi et al (2007). \n",
      "Photoperiodic regulation of PIF activity.Figure 4Schematic of transcriptional and post-transcriptional regulation of PIF activity. PIF4, IAA29 and ATHB2 mRNA levels in SDs in WT (data: black lines, filled squares; simulation: black lines). As in (B-D), for LDs. Data from Nomoto et al (2012b). ATHB2 mRNA at dawn (ZT0) across a range of photoperiods. Data from Kunihiro et al (2011). Comparison of model simulations with microarray time course data from the two largest clusters of PIF-induced targets (Cluster 1: 107 genes, Cluster 2: 84 genes), in SDs (I, K) and LDs (J, L) [data from Michael et al (2008b)]. \n",
      ".Figure 6(A) Correlation of change in OCR (ΔOCR) in a panel of IKBKE (IKKε)-silenced breast cancer cell lines (from Fig. 3C) and the change in cell confluency (Δconfluency) upon treatment of the panel of cell lines with NCT502 (from Fig. EV5A). (B) Correlation of ΔOCR in a panel of IKBKE (IKKε)-silenced breast cancer cell lines (from Fig. 3C) and the Δconfluency upon treatment of the panel of cell lines with 6-Diazo-5-oxo-L-norleucine (DON) (from Fig. EV5C). (C) Association between IKBKE (IKKε) and PSAT1 mRNA overexpression evaluated by a chi-squared independence test. The + sign indicates samples with significant (p<0.05) overexpression of IKBKE or PSAT1. Number and percentage of samples, as well as the Chi-square values are shown. Expression of IKBKE (IKKε) and the SBP enzymes PSAT1, in the METABRIC dataset. Samples were stratified by Pam50 intrinsic subtypes and ER status. Brown-Forsythe and Welch ANOVA test with unpaired t-test with Welch correction were applied. *p<0.05, **p< 0.01, ***p<0.001, ****p<0.0001. Expression of SBP enzymes PHGDH and PSPH in the METABRIC dataset. The expression of a proliferation related gene set and ATF4 are also shown. Samples were stratified by Pam50 intrinsic subtypes and ER status. Brown-Forsythe and Welch ANOVA test with unpaired t-test with Welch correction were applied. *p<0.05, **p< 0.01, ***p<0.001, ****p<0.0001. (J) Association between IKBKE (IKKε) and ATF4 mRNA overexpression evaluated by chi-squared independence test. The + sign indicates samples with significant (p<0.05) overexpression of IKBKE or PSAT1. Number and percentage of samples, as well as the Chi-square values are shown. (K, L) Correlation of IKBKE (IKKε) and PSAT1 (K) or IKBKE (IKKε) and ATF4 (L) expression in the ER negative sample subset. \n",
      "Model describing the photoperiod response of flowering and hypocotyl elongation in WT plants.Figure 5Simulated levels of FT and ATHB2 mRNA (as calculated by taking the area under the curve) across multiple photoperiods. Using FTAREA values, the number of days to flower was calculated and compared to data from Corbesier et al (1996) (see Supplementary Information). Error bars represent standard deviation. Simulated flowering: solid black line, filled squares; data: dashed black lines, empty squares. Same as in (C) using ATHB2AREA to calculate hypocotyl length. Data taken from Kunihiro et al (2011) (error bars were unavailable). \n",
      "Coherent feed-forward networks coordinate the regulation of FT by the circadian clock.Figure 7Coherent feed-forward networks coordinate the regulation of FT by the circadian clockAs highlighted in the discussion, the flowering system is based on a combination of coherent feed-forward pathways, allowing a single component to play multiple reinforcing roles in the system (Mangan Alon,2003). This is highlighted here in the case of the EC and the PRRs.The EC can repress FT expression through at least three partially redundant pathways, involving coordinated control of CDF1, FKF1 and GI levels.Similarly, the PRRs can activate FT expression through at least three pathways. Rectangles denote protein species, and parallelograms denote transcript species. Solid and dashed lines indicate direct and indirect forms of regulation, respectively. \n",
      "Coordinated regulation of ATHB2 and FT by PIF4.Figure 6Schematic of the regulation of ATHB2 and FT expression by PIF4 and temperature. Comparison of data and model simulations for temperature response of PIF4 (B, C) and ATHB2 (D, E) mRNA in LDs (16L:8D) in the WT. Data from Nomoto et al (2012a) were normalised such that data and simulation have equal peaks at 28°C (22°C data: black lines, filled squares; 22°C simulation: black lines; 28°C data: red lines, filled squares; 28°C simulation: red lines). CO mRNA data in LDs in WT and the pif4;pif5 mutant at 22 and 27°C (WT 22°C data/simulation: as in (B-E); pif4;pif5 22°C data: dashed black lines, open squares; WT 27°C data: red lines, filled circles; pif4;pif5 27°C data: dashed red lines, open circles). CO protein data in LDs (16L:8D) in WT at 22 and 27°C (as in F). FT mRNA data in LDs in WT and the pif4;pif5 mutant at 22 and 27°C (as in F) FT mRNA simulation in LDs in WT and the pif4;pif5 mutant at 22 and 27°C (WT 22°C simulation: as in B-E; pif4;pif5 22°C simulation: dashed black lines; WT 27°C simulation: red lines; pif4;pif5 27°C simulation: dashed red lines). \n",
      ".Figure 1Schematic of model substrates and translocation assay. Polypeptide substrates consist of truncated variants of proOmpA (orange), including the signal sequence (SS) (black). Methionines and cysteines are indicated in green and purple, respectively. Substrate was incubated with SecA (tan), proteoliposomes containing SecYEG (blue), and ATP. When the cysteines are oxidized, they form a disulfide, which is too large to allow complete translocation of the substrate, generating a translocation intermediate (IM) (bottom left). Depending on the added nucleotide, the polypeptide can slide back and forth in the channel or make a single step into the channel. When the cysteines are reduced, translocation of the full-length (FL) substrate can be completed with hydrolysable nucleotides (bottom right). Substrate was synthesized in reticulocyte lysate in the presence of 35S-methionine and mixed with SecA and SecYEG proteoliposomes. After incubation with ATP, the samples were treated with proteinase K to degrade any non-translocated material. Where indicated, the reaction was performed in the absence of ATP and proteolysis in the presence of Triton X-100 (TX-100). Translocation reactions were performed in the presence of the reducing agent DTT or oxidizing agent NaTT, generating protease-protected full-length (FL) substrate or a translocation intermediate (IM), respectively. All samples were analyzed by SDS PAGE followed by autoradiography. Lane 1 shows 10% of the input sample. The gel is representative of three replicates. Molecular weight markers are estimated from Coomassie-stained gels run in parallel. Translocation intermediates (IM) were incubated in the presence of different nucleotides (Nuc.) for different time periods and then treated with proteinase K (Prot. K). One sample received hexokinase and glucose (HKG) to generate ADP, other samples ADP•BeFx or ATPγS. The gel is representative of two replicates.   Quantification of (C). Points show IM band intensities normalized to the 0-min time point for each replicate. Lines show the average of replicates. Colors correspond to different nucleotides as shown in the inset box.   As in (C), but DTT was added concurrently with HKG or nucleotides. The gel is representative of two replicates.   Quantification of (E) as in (D), except intensities were calculated for the FL bands.   \n",
      ".Figure 2Schematic showing replacement of wild-type SecA in a translocation intermediate by Walker B (WB) SecA mutants (green). WB SecA with additional mutations (WT, Y794G, R342E) was added to preformed intermediates (IM) and incubated for the indicated times before proteolysis. The gel is representative of two replicates.   Quantification of (B). Points show IM band intensities normalized to the 0-min time point for each replicate. Lines show average of replicates. Colors correspond to WB SecA carrying additional mutations, as shown in the inset box.   \n",
      "Application of yeast-based TAR cloning to generate viral cDNA clones and the recovery of recombinant MHV-GFP.Figure 1.Application of yeast-based TAR cloning to generate viral cDNA clones and the recovery of recombinant MHV-GFP.a, Schematic representation of the general workflow of TAR cloning and virus rescue. In-yeast assembly require the delivery of overlapping DNA fragments covering the viral genome and a TAR vector into yeast in one transformation event. Transformed DNA fragments are joined by homologous recombination in yeast generating a YAC-cloned full-length viral genomic cDNA. The in vitro production of infectious capped viral RNA starts with the isolation of the YAC, followed by plasmid linearization to provide DNA template for run-off T7 polymerase-based transcription. Virus rescue starts with electroporation of BHK-21 and/or BHK-21 cells expressing the corresponding coronavirus N protein, followed by co-cultivation of electroporated cells with susceptible cell line for virus production and amplification. Schematic representation of MHV-GFP genome organisation with 9 viral subgenomic overlapping cDNA fragments used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. Recovery of infectious recombinant MHV-GFP. Cell culture supernatants containing viruses produced after virus rescue from two MHV-GFP YAC clones (Clone1, Clone 2) were used to infect 17Cl-1 cells. At 48 hours post-infection, infected cells were visualised for GFP expression (left panels) and by bright field microscopy (right panels). Mock represents 17Cl-1 cells inoculated with supernatant from BHK-MHV-N cells electroporated without viral RNAs. Replication kinetics of parental MHV-GFP and recombinant MHV-GFP Clone 1 and Clone 2. L929 cells were infected at an MOI of 0.1. Cell culture supernatants were harvested at indicated time points post-infection and titrated by plaque assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. pfu/ml, plaque forming units per millilitre. \n",
      "Generation of viral cDNA clones and recovery of recombinant MERS-CoV and MERS-CoV-GFP.Figure 2.Generation of viral cDNA clones and recovery of recombinant MERS-CoV and MERS-CoV-GFP.a, Schematic representation of MERS-CoV (upper panel) and MERS-CoV-GFP (lower panel) genome organization with 8 and 10 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. Rescue of recombinant MERS-CoV and MERS-CoV-GFP. Following delivery of viral RNAs into BHK-21 cells via electroporation the cells were co-cultivated with VeroB4 cells and supernatants containing recombinant viruses produced were used to infect new VeroB4 cells. Infected cells were visualised by bright field microscopy for recombinant MERS-CoV (upper panel; 5 days post infection), and for GFP expression for recombinant MERS-CoV-GFP (lower panel, 3 days post infection). Mock represents VeroB4 cells inoculated with supernatant from BHK-21 cells electroporated without viral RNAs. Replication kinetics of the MERS-CoV-EMC (laboratory adapted isolate), rMERS-CoV and rMERS-CoV-GFP. VeroB4 cells were infected at an MOI of 0.01. Cell culture supernatants were harvested at indicated time points post infection and titrated by TCID50 assay. Data are representative of three independent experiments with two replicates per virus in each experiment. Error bars show standard deviation (SD) from the mean. TCID50/ml, 50% tissue culture infectious dose per millilitre. \n",
      ".Figure 3Schematic showing the generation of a translocation intermediate with SecA dN20::H6 (purple) recruited to SecYEG liposomes containing Ni-modified lipids (cyan circles). Addition of imidazole dissociates SecA dN20::H6, which can then be replaced by WT or mutant SecA (tan). Imidazole (imi) and either WT or mutant SecA were added to translocation intermediates (IM) pre-formed with SecA dN20::H6. The samples were incubated for the indicated times before proteolysis with proteinase K (Prot. K). The gel is representative of two replicates, except for the experiment with the R792G/K797G mutant and the 30-min time points. Quantification of (B). Points show IM band intensities normalized to the 0-min time point for each replicate. Lines show average of replicates. Colors correspond to different SecA mutants as shown in the inset box. As in (B), but ATPγS was added together with imidazole. The gel is representative of two replicates. Quantification of (E) as in (C). As in (D), but DTT was added with ATPγS ten minutes after the addition of imidazole and WT or mutant SecA. The gel is representative of two replicates.   Quantification of (G) as in (F), except that both the intensities of the IM (darker colors) and FL (lighter colors) bands were measured.   Schematic of the residues at the tip of the SecA two-helix finger. The conserved tyrosine Y794 is indicated in blue, and the conserved basic residues R792 and K797 are in red. \n",
      ".Figure 3Scatter plots showing the correlation between FOXA2 ChIP-seq biological duplicates in CFPAC1 (top panel) and PANC1 cells (bottom panel). Pearson's correlation (r) between samples is indicated. Scatter plot showing the correlation between FOXA2 ChIP-seq peaks in CFPAC1 and PANC1 cells. Green and blue dots indicate FOXA2 peaks that are significantly (FDR < 0.001) different between the two cell types and with log2 fold enrichment ≥ 2 as determined by DiffBind. All other peaks are indicated as grey dots. The boxplots on the right show the intensity of CFPAC1-specific, PANC1-specific and other FOXA2 peaks. Central values represent the median, the bars the 25th and 75th percentile and the vertical lines the lower and upper whiskers. Distribution of the positions of the FOXA motif relative to the summit of different sets of FOXA2 peaks. The PWM used to perform CentriMo analysis and the E-value for each dataset are shown. Representative FOXA2 ChIP-seq snapshot showing FOXA2 peaks classified as CFPAC1-specific, shared or PANC1-specific. GO terms retrieved by GREAT analysis associated with distal FOXA2 peaks specific of CFPAC1 (left) or PANC1 (right). Terms are ranked by the negative log10 p-value from binomial correct test. \n",
      ".Figure 1FOXA1 and FOXA2 expression in human PDAC areas with distinct differentiation grade. Immunohistochemistry analysis of FOXA1 (top) and FOXA2 (bottom) in human PDAC. Representative G1, G2 and G3 areas from one patient are shown. Red dotted lines indicate the borders of poorly differentiated (G3) tumor areas. Percent of FOXA1- and FOXA2-positive nuclei in PDAC cells from areas of different grade. Tissue microarrays were automatically acquired and quantified using QuPath. The number (n) of analyzed areas of each grade is indicated. Expression of FOXA1 and FOXA2 in low-grade and high-grade PDAC cell lines. Vinculin is shown as loading control. \n",
      "Generation of viral cDNA clones and recovery of recombinant SARS-CoV-2 and SARS-CoV-2-GFP.Figure 3.Generation of viral cDNA clones and recovery of recombinant SARS-CoV-2 and SARS-CoV-2-GFP.a, Schematic representation of SARS-CoV-2 (upper panel) and SARS-CoV-2-GFP (lower panel) genome organisation with 12 and 14 viral subgenomic overlapping cDNA fragments, respectively, used for TAR cloning. pT7, T7 RNA polymerase promoter; UTR, untranslated region; An, poly (A) tail; J represents assembly junction between two overlapping DNA fragments. b, Representation of predicted RNA stem-loop (SL) secondary structures formed in the 5'-UTR of SARS-CoV-2. The SARS-CoV-2 RNA secondary structures were manually adjusted based on RNA structure predictions by Chan et al.22. Black letters and numbering represents the SARS-CoV-2 5'-terminal sequence. Red letters depict nucleotides that are different within the SARS-CoV 5'-terminal sequence (\"-\" indicates nucleotide deletion in SARS-CoV versus SARS-CoV-2). N20 indicates 20 nucleotides. SL structures 1, 2, and 4 are shown according to Chan et al.22. Note that recombinant virus construct 1 (rSARS-CoV-2 clone 1) and construct 4 (rSARS-CoV-GFP clone 4) contain the SARS-CoV-2 5'-terminus with nucleotides 3-5 (UAA) exchanged by nucleotides 3-5 (AUU) of SARS-CoV; recombinant virus construct 2 (rSARS-CoV-2 clone 2) and construct 5 (rSARS-CoV-2-GFP clone 5) contain a 5'-terminus with the first 124 nucleotides from SARS-CoV; and recombinant virus construct 3 (rSARS-CoV-2 clone 3) and construct 6 (rSARS-CoV-GFP clone 6) contain the authentic SARS-CoV-2 5'-terminus. Rescue of recombinant rSARS-CoV-2 and rSARS-CoV-2-GFP. The experimental setting is illustrated in the upper panel. Two clones of each construct 1-6 were used to prepare YAC DNAs and in-vitro transcribed viral genome RNAs were electroporated into BHK-21 cells and/or BHK-SARS-N together with an mRNA encoding the SARS-CoV-2 N protein. Electroporated cells were co-cultivated with VeroE6 cells in a T25 flask and cell culture supernatants were transferred to 12-well plates to perform plaque assay for rSARS-CoV-2 constructs 1-3 and fluorescence microscopy for rSARS-CoV-2-GFP constructs 4-6. Plaque assays are shown in the lower panel for SARS-CoV-2 clones 1.1, 2.2, and 3.1. VeroE6 cells were infected with 10−1, 10−2, and 10−3 dilutions of 1 ml supernatant from the individual rescue experiments and are compared to non-infected cells (Mock). The expression of GFP in SARS-CoV-2-GFP infected VeroE6 cells is shown for clones 4.1, 5.2, and 6.1. Mock represents non-infected VeroE6 cells. Summary of the timeline of events of the SARS-CoV-2 outbreak and the work leading to the reconstruction and recovery of rSARS-CoV-2. \n",
      ".Figure 2Western blot showing siRNA-mediated depletion of FOXA1 and/or FOXA2 in the low-grade CFPAC1 cell line. The three biological replicates used in the subsequent experiments are shown. Heatmap showing row-normalized expression of differentially expressed genes in CFPAC1 cells depleted of FOXA1/2 (FDR ≤ 0.05 and absolute log2 FC ≥ 1). Blue and orange depict down- and up-regulation, respectively. FOXA2 and FOXA1 expression in wild type and genome-edited high-grade PANC1 cells. Three different bulk populations were used for RNA-seq experiments. Heatmap showing row-normalized expression of differentially expressed genes in PANC1 cells deleted of FOXA2 (FDR ≤ 0.05 and absolute log2 FC ≥ 1). Blue and orange depict down- and up-regulation, respectively. Venn diagram showing the overlap between FOXA1/2-regulated genes in CFPAC1 and PANC1 cells. Representative Gene Ontology (GO) terms enriched in genes differentially expressed upon FOXA1/2 depletion in CFPAC1 cells. Terms are ranked by the negative log10 p-value from the hypergeometric test. Panels on the right show the effects of FOXA1/2 depletion on cell morphology and adhesion to multiple substrates. Three replicates per condition were used: circularity index was measured on 100 different cells per condition while adhesion is reported as mean of three different fields per replicate for a total of nine observations per condition. Representative GO terms enriched in genes differentially expressed upon FOXA2 deletion in PANC1 cells. Terms are ranked by the negative log10 p-value from the hypergeometric test. Panels on the right show the effects of FOXA2 deletion on cell morphology and adhesion to multiple substrates. Three different bulk populations were used as biological replicates: circularity index was measured on 60 different cells per bulk population while adhesion is reported as mean of six different fields per bulk for a total of eighteen observations per condition. \n",
      ".Figure 4The five top transcription factor motifs over-represented in TSS-distal CFPAC1-specific FOXA2 peaks. TFs were grouped into families or subfamilies and ranked based on the best PWM q-value. The POU domain TF family includes HNF1β. Top scoring motifs are shown on the right. Bar plot showing the percentage of different sets of FOXA2 peaks overlapping with HNF1β ChIP-seq peaks. Distribution of the positions of the best HNF1β motif relative to the summit of the CFPAC1-specific FOXA2 peaks. The graph indicates the average density in 20 bp bins in a window of 500 bp. Bar plot showing the percentage of different sets of FOXA2 peaks overlapping with FOS (left) and JUNB (right) ChIP-seq peaks. Distribution of the positions of the AP-1 motif relative to the summit of the TSS-distal, CFPAC1-specific FOXA2 peaks. The graph indicates the average density in 20 bp bins in a window of 500 bp. Two representative ChIP-seq snapshots indicating the genomic distribution of FOXA2 and partner transcription factors in CFPAC1 peaks. The FOXA2 ChIP-seq track in PANC1 cells is also shown for comparison. \n",
      ".Figure 5Representative areas from human PDAC tissue showing expression of HNF1β in G1, G2 and G3 areas. The red dotted line indicates the borders of a G3 area. The percentage of HNF1β-positive cells in areas of different grade from a PDAC tissue microarray is shown. Colocalization of HNF1β and FOXA2 in human PDAC areas. PDAC sections from three different patients were stained with the anti-HNF1β antibody, stripped and re-stained with the anti-FOXA2 antibody. The red dotted areas on the left represent poorly differentiated (G3) tumor regions. \n",
      ".Figure 6FOXA2 peaks in CFPAC1 cells were divided into 5 clusters based on their association with HNF1β and the effects of HNF1β loss on FOXA2 binding and chromatin accessibility (determined by ATAC-seq). Clusters 1-4 were HNF1β- positive while cluster 5 was HNF1β-negative. The heat-maps on the left show FOXA2, HNF1β, ATAC-seq and H3K27Ac profiles at FOXA2-bound genomic regions. The three heat-maps on the right show JunB ChIP-seq data and ATAC-seq data in wild type or Jun/JunB double knockout (DKO) CFPAC1 cells. Data are shown in +/- 3 kb regions centered on the summit of FOXA2 peaks. Characterization of clusters 1-3. Left: quantification of the indicated signals at FOXA2 bound regions in wild-type (WT) and HNF1β knock-out (KO) cells. Central values represent the median, the bars the 25th and 75th percentile and the vertical lines the lower and upper whiskers. Middle: distribution of FOXA and HNF1β motifs relative to the summit of FOXA2 peaks. Right: over-represented motifs in clusters 1-3 and inferred binding models. Quantification of JunB ChIP-seq signals in clusters 1 to 5. (Right) Quantification of ATAC-seq signals in clusters 1 to 5 in wild type and Jun/JunB double KO (DKO) CFPAC1 cells. Central values represent the median, the bars the 25th and 75th percentile and the vertical lines the lower and upper whiskers. Representative snapshot showing changes in FOXA2 occupancy, chromatin accessibility and histone acetylation in HNF1β knock-out CFPAC1 cells. The FOXA2 ChIP-seq track in PANC1 cells is also shown for comparison. \n",
      ".Figure 7Western blot showing HNF1β over-expression after lentiviral transduction of PANC1 cells. Expression of FOXA2 in wild type and over-expressing (O.E.) cells is shown. Vinculin: loading control. Scatter plot showing the genomic distribution of FOXA2 in PANC1 cells transduced with the HNF1β-expressing lentivirus (HNF1B O.E.) and in their matched mock-infected control. The fraction of gained, unchanged and lost FOXA2 peaks overlapping HNF1β ChIP-seq peaks in transduced PANC1 cells is shown in the pie-charts on the right. TF motif over-represented in gained or lost FOXA2-bound genomic regions. Representative snapshot from PANC1 cells transduced with the HNF1β-expressing lentivirus. HNF1B O.E.: HNF1B over-expression. \n",
      ".Figure 8Distribution of the positions of the best HOXB motif relative to the summit of the PANC1-specific FOXA2 peaks. The best PWM identified in the enrichment motif analysis and used to perform the CentriMo analysis is indicated. The graph indicates the average density in 20 bp bins in a window of 600 bp. Percentage of different sets of FOXA2 peaks overlapping HOXB8 ChIP-seq peaks. Representative snapshots showing the overlap between HOXB8 and FOXA2 ChIP-seq peaks in PANC1 cells. The FOXA2 ChIP-seq track in CFPAC1 cells is also shown for comparison. FOXA2 ChIP-seq in CFPAC1 cells transduced with a HOXB8-expressing lentivirus and in their matched mock-infected control. Gained (brown) and lost (yellow) FOXA2 peaks are shown. Pie-charts on the right show the fraction of gained, unchanged and lost peaks overlapping HOXB8 ChIP-seq peaks in transduced CFPAC1 cells. TF motif over-represented in gained or lost FOXA2-bound genomic regions. Expression of HNF1β protein in wild-type and HOXB8-expressing PANC1 cells. Representative snapshots from CFPAC1 cells transduced with the HOXB8-expressing lentivirus. \n",
      " - Correlation of root Na+/K+ ratio with salt resistance during tomato domestication.Figure 1A Distribution of 369 accessions, including PIM (36 S. pimpinellifolium accessions), CER (118 S. lycopersicum var. cerasiforme accessions), and BIG (215 S. lycopersicum accessions). B The Na+ and K+ contents and Na+/K+ ratio in roots and shoots of three groups of 369 accessions after treatment with 150 mM NaCl for 1 d. C Linear regression of root Na+/K+ ratio under salt stress (150 mM NaCl for 1 d) and fruit weight in 195 accessions from three groups. R2, coefficient of determination. n is the accession number of the three groups. D The Na+/K+ ratios in roots of TS-21, TS-422, TS-577, and TS-670 after treatment with 150 mM NaCl for the indicated time (n = 3 biological repeats). E, F Salt-resistance assay of two PIM accessions (TS-21 and TS-422) and two BIG (TS-577 and TS-670). 20-day-old seedlings grown in soil were treated with 200 mM NaCl for 3 weeks (E). The survival rates were obtained from at least 12 plants in three repeated experiments (F). \n",
      " - Identification of SlHAK20 using GWAS of root Na+/K+ ratio.Figure 2A Manhattan plot displaying the GWAS results of Na+/K+ ratio in root. The red-dashed line indicates the Bonferroni-adjusted significance threshold (P = 1.0 × 10-7). Red arrow indicates the significant SNP signal of Na+/K+ ratio associated with SlHAK20. B The nucleotide diversity ratios between PIM and CER, and between CER and BIG on chromosome 4. The black-dashed horizontal lines indicate top 10% threshold for entire chromosome 4 (1.82 πPIM/πCER for domestication and 4.27 πCER/πBIG for improvement). The red arrows indicate the position of SlHAK20 in the sweeps. π, nucleotide diversity. C SlHAK20-based association mapping and pairwise LD analysis. Dots represent SNPs. Indel48 is highlighted in red. The indel-2381, indel-1818 in the promoter region and five nonsynonymous variants are marked in blue. These eight variants are related to the pairwise LD diagram with dashed lines. D Haplotypes of SlHAK20 among tomato natural variations. The distribution of Na+/K+ ratio in each haplotype group is exhibited by a box plot. The n indicates the number of accessions belonging to each haplotype. In the box plots, the middle line indicates the median, the box indicates the range of the percentiles of the total data using Turkey method, the whiskers indicate the interquartile range and the outer dots are outliers. Significant difference was determined by Student's t-test. E Allele distribution of the SlHAK20 locus at position in PIM, CER, and BIG groups. The n indicates the accession number. \n",
      " - Phenotype of the Hap2 accession overexpressing SlHAK20Hap1.Figure 6A The transcript levels of SlHAK20 in TS-670 and three independent SlHAK20Hap1-YFP transgenic lines grown under normal growth conditions. B The protein levels of SlHAK20Hap1-YFP in TS-670 and transgenic plants. ACTIN was used as loading control. Phenotypes of transgenic lines in soil under salt stress. The survival rates were obtained from at least 8 plants in four repeated experiments 38-day-old plants grown in soil as control. 20-day-old TS-670 and Hap1OE-1 to -3 were treated with 175 mM NaCl for 3 weeks and recovered for 2 weeks The Na+ in shoots and roots of TS-670 and three independent SlHAK20Hap1-YFP transgenic lines during salt stress. The 21-day-old TS-670, Hap1OE-1, Hap1OE-2 and Hap1OE-3 plants grown in 0.25× Hoagland which were then treated with 50 mM NaCl for additional 0 and 7 days. The K+ in shoots and roots of TS-670 and three independent SlHAK20Hap1-YFP transgenic lines during salt stress. The 21-day-old TS-670, Hap1OE-1, Hap1OE-2 and Hap1OE-3 plants grown in 0.25× Hoagland which were then treated with 50 mM NaCl for additional 0 and 7 days. The Na+/K+ ratio in shoots and roots of TS-670 and three independent SlHAK20Hap1-YFP transgenic lines during salt stress. The 21-day-old TS-670, Hap1OE-1, Hap1OE-2 and Hap1OE-3 plants grown in 0.25× Hoagland which were then treated with 50 mM NaCl for additional 0 and 7 days. \n",
      " - Role of SlHAK20 in the salt-tolerant tomato.Figure 4A Generation of mutations in SlHAK20Hap1 by CRISPR/Cas9 using two independent single-guide RNA (sgRNA1 and sgRNA2). Sequences of SlHAK20Hap1 in wild type (TS-21), slhak20-1 and slhak20-2 mutant tomato are shown. The sgRNA-targeted sequences are underlined and the protospacer-adjacent motif (PAM) sequences are in blue. The deletion is indicated by a dashed line and the insertion is labeled with red. Na+/K+ ratio contents in roots of TS-21 wild type and slhak20 mutants. Data are shown as means ± SD (n = 4). Na+ contents in roots of TS-21 wild type and slhak20 mutants. Data are shown as means ± SD (n = 4). K+ contents in roots of TS-21 wild type and slhak20 mutants. Data are shown as means ± SD (n = 4). Na+ contents in the xylem sap of TS-21 and slhak20 mutants after 50 mM NaCl treatment for the indicated days. Data represent means ± SD (n = 4). K+ contents in the xylem sap of TS-21 and slhak20 mutants after 50 mM NaCl treatment for the indicated days. Data represent means ± SD (n = 4). G Shoot height of TS-21, slhak20-1, and slhak20-2. Plant height of 24-day-old TS-21, slhak20-1, and slhak20-2 were measured after growing under normal conditions for 14 days, followed by salt treatment for 14 days, and recovery for 10 days. Values are means ± SD (n = 10 plants of each genotype). H Salt resistance of TS-21 wild type and slhak20 mutants. TS-21, slhak20-1, and slhak20-2 grown under normal growth conditions for 24 days, followed by a treatment with 200 mM NaCl for 14 days, and then recovered for 10 days. Control plants grown under normal growth conditions for 38 days. Scale bars, 5 cm. \n",
      " - Functional characterization of SlHAK20.Figure 3A Phylogenetic tree of the HAK/KUP/KT transporter family clade IV. The phylogenetic tree was constructed based on the amino acid sequences using the neighbor-joining method with MEGA5 in dicots and monocots. The expression pattern of SlHAK20 in tomato. The GUS activity was examined in seedling (left) and root (right) of 21-days-old SlHAK20TS-21pro:GUS transgenic tomato (B). Bars, 2 cm. The expression pattern of SlHAK20 in tomato. Cross-sections of hypocotyl (C) of SlHAK20TS-21pro:GUS transgenic plants. The scale bar is 400 μm in (C) The expression pattern of SlHAK20 in tomato. Cross-sections of root he scale bar is 200 μm in (D). E Subcellular localization of SlHAK20-YFP in tomato. The SlHAK20Hap1-YFP (left) and SlHAK20Hap2-YFP (right) were localized in the plasma membrane. YFP, yellow florescence protein. FM4-64, a lipophilic styryl compound as a red fluorescent marker of plasma membrane. Bar, 20 μm. F SlHAK20 complements the K+ uptake-defective yeast mutant assayed using a dose-response method. The K+ uptake-defective yeast mutant R5421 transformed with SlHAK20Hap1, SlHAK20Hap2, and vector (p416-GPD) were cultured in AP medium containing indicated concentrations of K+ for 24 h. The data points represent three replicates. G Na+ uptake analysis of yeast cells expressing SlHAK20. Shown are the Na+ concentrations in the medium over the time of culture of the ANT3 yeast cells transformed with SlHAK20Hap1, SlHAK20Hap2, or empty vector. The initial NaCl concentration in the medium was 60 μM. The data are shown as the means ± SD (n = 4). H Na+ uptake kinetic analysis of SlHAK20 in yeast cells. The curves show concentration- dependent Na+ inﬂux in yeast expressing SlHAK20Hap1 or SlHAK20Hap2 in the presence of 0, 5, 10, 20, 30, 40, 50, 60, 100, 150 or 200 μM NaCl in the medium. The data are expressed as mean ± SD (n = 3). \n",
      " - Salt tolerance of slhak20 from the salt-sensitive tomato and oshak4/17 from rice.Figure 5Generation of SlHAK20Hap2 mutations using CRISPR/Cas9 system. The sequences of SlHAK20Hap2 in wild type (TS-670), slhak20-3 and slhak20-4 mutant tomato are shown. Salt-tolerance assay of TS-670 and slhak20 mutant plants. TS-670, slhak20-3, and slhak20-4 grown under normal growth conditions for 23 days, and then treated with 150 mM NaCl for 5 days. Control plants grown under normal growth conditions for 28 days. Plant height and fresh weight of TS-670, slhak20-3, and slhak20-4 under normal and salt stressPlant height and fresh weight of TS-670, slhak20-3, and slhak20-4 under normal and salt stress conditions. resh weight f 23-day-old TS-670, slhak20-3, and slhak20-4 were quantified after growing under normal conditions for 5 days, or salt treatment with 150 mM NaCl for 5 days. Values are means ± SD (n = 10 plants of each genotype). Generation of oshak4/17 mutants using CRISPR/Cas9 method. Sequences of OsHAK4 and OsHAK17 in wild type (NIP), oshak4/17-1, oshak4/17-2 and oshak4/17-3 mutants are shown. Phenotype of NIP and oshak4/17 double mutant plants under normal and salt stress conditions. 20-day-old seedlings of NIP and oshak4/17 mutants were treated with 100 mM NaCl for 8 days. The pictures show representative plants ere analyzed among these genotypes. ontrol plants were grown under normal growth conditions for 28 days Phenotype of NIP and oshak4/17 double mutant plants under normal and salt stress conditions. 20-day-old seedlings of NIP and oshak4/17 mutants were treated with 100 mM NaCl for 8 days. The pictures show representative fresh weight of shoots were analyzed among these genotypes. Control plants were grown under normal growth conditions for 28 days and the fresh weights of shoots were measured. Values are means ± SD (n = 12 plants of each genotype for fresh weight Phenotype of NIP and oshak4/17 double mutant plants under normal and salt stress conditions. 20-day-old seedlings of NIP and oshak4/17 mutants were treated with 100 mM NaCl for 8 days. The pictures show representative survival rate were analyzed among these genotypes. \n",
      ".Figure 3A. Confocal microscopy shows that active Akt (phospho-S473Akt) is localized in the basal body of primary cilia (right side panels show higher magnification).B. INVS is localized in the proximal region of the INVS compartment in primary cilia (INVS compartment, right side panels show higher magnification).C. INVS co-localizes with γ-tubulin, a centrosome marker.D. Anti-phospho-Akt substrates stain positively at the basal region of the primary cilia (arrows), as determined by anti-acetylated tubulin (right side panels show higher magnification). E. Silver-intensified immunogold electron microscopy shows that both phospho-Akt (left side panels) and INVS (right side panels) were localized to the ciliary pocket of primary cilia in the presence of PDGF-AA (white arrow). F. Schematic representation of the view from an electron microscope. \n",
      "Fig.Figure 2A. The indicated recombinant INVS proteins (lanes 1-4) were used for IVK. Akt phosphorylates WT (lane5), 1-970 (lane 6), and 1-898 (lane 7) INVS, but failed to phosphorylate 1-670 (lane 8) INVS. The indicated recombinant INVS proteins (lanes 1-10) were used for IVK. Akt phosphorylates WT (lane 6) and 1-898 (lane7) INVS, but failed to phosphorylate other subfragments of INVS (1-822, 1-746, and 1-670, in lane 8, 9, and 10, respectively), as detected using a phospho-Akt substrate antibody.. C. Using the indicated subfragments of INVS for IVK, we confirmed that Akt phosphorylates 675-1065 (lane 4), but not 675-822 (lane 5), or 675-746 (lane 6) INVS. D. Amino acid alignment of 824-898. Motif scan analysis using the Scansite3 database identified three amino acids, T864, S865, and T866, as putative Akt phosphorylation targets. E. Purified recombinant WT or 3A INVS were generated (top panel, lanes 1-4). In IVK, Akt phosphorylated the WT fragment (824-898, middle panel, lane 3), but failed to phosphorylate 3A INVS (middle panel, lane 4), as detected by using a phospho-Akt substrate antibody. F. Akt phosphorylated WT (lane 10), 1-898 (lane 12) and 675-1065 (lane 15) INVS, but failed to phosphorylate both full-length and 675-1065, ∆727-896, 1-857, or 1-822, 3A INVS (lanes 16, 11, 14, and 15, respectively), all of which lacked the T864/S865/T866 target amino acid. \n",
      ".Figure 4PDGF-AA stimulation enhances localization of active Akt (phospho-S473Akt) at the basal body of primary cilia (right side panels show higher magnification). Quantification of co-localization of phosphorylated Akt (green) and INVS (red) at the indicated time points after PDGF-AA stimulation revealed a time-dependent increase. Colocalization was measured by counting yellow pixels (colocalized area) using Imaris software (Bitplane AG). Results presented are means ±SE (n=31 at time 0, n=35 at 1 min., and n=33 at 3 min, respectively). Three independent experiments were analyzed with similar results. Statistical significance was determined by student's t test. Co-immunoprecipitation assays using HEK293T cells show that INVS preferentially binds to WT-Akt (second panel, lane1) or phospho-mimetic Akt (T308D/S473D, lane2), but fails to interact with phosphorylation-defective forms of Akt (T308A/S473A, lane 3).PDGF-AA stimulation of HEK293T cells results in augmented interaction of endogenous INVS with Akt (top panel, comparison between lane 1 and lane 2, untreated and PDGF-AA-stimulated conditions, respectively). Both WT and 3A-INVS, which are localized at the INVS compartment of primary cilia under serum-starved condition (left side panels, lower panels show higher magnification), translocate to the basal body of the cilium after PDGF-AA stimulation (right side panels, lower panels show higher magnification). Results presented are means ± SE of yellow pixels demonstrating colocalization of EGFP-INVS (green) with γ-tubulin (red) in hTERT-RPE1 cells (n=28). Three independent experiments showed similar results. Statistical significance was determined by student's t test. Fluorescence intensities of INVS (green), acetylated-α-tubulin (blue), and γ-tubulin (red) along the line (a-b) are plotted underneath. \n",
      ".Figure 1Flag-INVS interacted with HA-Akt (lanes 1-3), but not with HA-PDK1 (lanes 4-6) or HA-PrKA (lanes 7-9) in HEK293T cells. Expression levels of HA-Akt, PDK1, PrKA, and Flag-INVS were similar, as determined by immunoblotting (HA: anti-HA antibody; F: anti-Flag antibody; C: control antibody). The results presented (A-E) are representative of at least two independent experiments. B. Flag-tagged INVS interacted with HA-tagged Akt1 (lane 1-3), Akt2 (lane 4-6), and Akt3 (lane 7-9) in co-immunoprecipitationassays. Expression of each Akt isoform and INVS were similar. C. INVS was co-immunoprecipitated with endogenous Akt from HEK293 cells, in which INVS is also endogenously expressed (lane 2). D. To determine the Akt domain responsible for interacting with INVS, Akt subfragments were generated. The C-terminus of Akt is the primary domain for INVS interaction (lanes 7-9). Similar levels of each Akt fragment were immunoprecipitated using anti-HA antibody. E. Schematic showing the structure and the functional domains of Flag-tagged full-length and fragmented INVS in mammalian expression vectors (N-Term: 1-421, Intermediate domain: 421-678, and C-Term: 678-1065) used in the current study. Intermediate (lanes 6-9) and C-terminal (lanes 10-12) fragments of INVS were important for interaction with Akt. \n",
      ".Figure 8Figure 8. Model of Akt-Inversin Interaction in Nephronophthisis type IIWe found that Akt interacts with and phosphorylates the ciliary proteinINVS, which is mutated in human NPHP2. Lack of phosphorylation of INVS leads to misalignment of the spindle axis and distorted luminal formation, underscoring the physiological significance of the Akt-INVS interaction in the development of abnormal cysts in NPHP2. \n",
      "Fig.Figure 6Schematic describing the measurement of spindle angle (θ) (Kikuchi et al., 2010). Z-stack images were obtained from 0.45 mm-thick sections of metaphase cells and the angle between the axis of metaphase spindle and that of the substrate surface, which is termed the spindle angle, was calculated. Results presented are scatter plots of the angle (θ) of the mitotic spindle (n=28 for all samples of vector, HA-WT, -3D, -3A, -R899X, -Q671X, and -R603X INVS stably transfected MDCK cells). Three independent experiments showed similar results. Statistical significance was determined by Mann-Whitney's non-parametric median test. Spindle angles were measured in MDCK cells transfected with the indicated INVS plasmids (+). The spindle axis was misaligned in cells expressing mutant forms of INVS as compared to cells expressing WT INVS or sham-transduced cells. Expression of 3A INVS and of clinically relevant truncated forms of INVS but not WT INVS, resulted in disorientation of the spindle axis during metaphase. Similar expression of all INVS constructs was verified by immunoblotting (Appendix figure S4A). WT INVS robustly inhibited the transactivation of INVS in a luciferase reporter assay, which could be reverted by C-terminal truncated INVS (R899X, Q671X, or Q603X), but not by a phospho-mimetic mutant or 3A INVS. Results presented are mean ± SE of the ratio of triplicates between sequentially measured firefly TCF/LEF signal and Renilla luciferase signal in HA-INVS transfected 293T cells. Dual-Luciferase reporter® Assay System (Promega) was used and experiments were performed in triplicates. Two independent experiments showed similar results. Statistical significance was determined by student's t test. The expression of HA-Dvl (upper panels) and the indicated forms of Flag-INVS (lower panels) are shown by immunoblots; α-tubulin was used as an internal control. \n",
      "Fig.Figure 7WT INVS- transduced MDCK cells in 3D cultures form uniform round tubules. However, expression of 3A or NPHP2-related truncated INVS mutants, which lack Akt phosphorylation site (s), disrupts this uniformity. Results presented are representative of three independent experiments and show mean ± SE of percent normal acini in vector- or WT INVS-transduced MDCK cells (n=422 for Vector, n=398 for HA-WT, n=401 for -3D, n=365 for -3A, n= 344 for -R899X, n=331 for -Q671X, and n=325 for -R603X INVS stably transfected MDCK cells, respectively). Three independent experiments showed similar results. Statistical significance was determined by student's t test. WT transduced MDCK cells form a relatively smaller lumen; however, expression of 3A INVS or the indicated NPHP2-related truncation mutants result in relatively larger acini. Both truncation mutants (Q671X or R603X) lack the Akt phosphorylation site and exhibit a larger lumen size. Three independent experiments showed similar results. C. Quantification of normal lumen in cells transduced with WT or mutant INVS. Results presented are mean ±SE of percent normal acini in vector- or WT INVS-transduced MDCK cells (n=166 for Vector, n=204 for HA-WT, n=233 for -3D, n=150 for -3A, n=164 for -R899X, n=128 for -Q671X, and n=137 for -R603X INVS stably transfected MDCK cells, respectively). Three independent experiments showed similar results. Statistical significance was determined by student's t test. D. Size of lumen in cells transduced with WT or mutant INVS is shown. Results presented are scatter plots of acini size (n=205 for Vector, n=143 for HA-WT, n=131 for -3D, n=167 for -3A, n=144 for -R899X, n=128 for -Q671X, and n=144 for -R603X INVS stably transfected MDCK cells, respectively). Two independent experiments showed similar results. Statistical significance was determined by Mann-Whitney's non-parametric median test. \n",
      ".Figure 5A. PDGF-AA stimulation resulted in the phosphorylation of WT INVS (top panel, lane 2), but not 3A INVS (top panel, lane 6). PDGF-AA-stimulated INVS phosphorylation is inhibited by Akt inhibitors (top panel, LY294002, MK2206, and GSK690693, lanes 3, 4, and 5, respectively). B. Myr-Akt, a constitutively active form of Akt, was expressed together with WT or 3A INVS in HEK293 cells and INVS phosphorylation was quantified. WT, but not 3A INVS, can be phosphorylated in the presence of Myr-Akt. C. 3A INVS exhibited a weaker interaction with Akt as compared to WT INVS (top panel, compare lanes 1 and 2, WT and phospho-defective mutant, respectively). Expression of Akt and INVS are shown underneath.D. 3A INVS exhibited weaker dimerization as compared to WT INVS (top panel, compare lanes 1 and 2, WT and 3A, respectively). Expression of Akt and INVS are shown below each. E. Length of primary cilia in confluent hTERT-RPE cells was analyzed after siRNA-mediated knock down of Akt1/2. Ciliary length decreased in cells expressing siRNA. Results presented are means ±SE of the longitudinal length of acetylated-tubulin (red), a marker of primary cilia, in siRNA-transfected (green) hTERT-RPE1 cells (n=175 for control siRNA and Akt siRNA, and n=195 for Akt siRNA plus siRNA resistant Akt, respectively). Three independent experiments showed similar results. Statistical significance was analyzed by student's t test. Note that re-introduction of siRNA-resistant Akt (Matsuda-Lennikov et al., 2014) in Akt knockdown cells rescued the effect on ciliary length. F. Expression of 3A INVS (bottom panels) but not WT INVS (middle panels) inhibited the development of primary cilia as compared to control cells (top panels). Quantification of ciliary length. Results are mean ± SE of longitudinal length of acetylated-tubulin (blue), a marker of primary cilia, in EGFP-INVS (green) transfected in hTERT-RPE1 cells (n=58 for EGFP vector and for EGFP-INVS WT, and n=52 for EGFP-INVS 3A mutant, respectively). Three independent experiments showed similar results. Statistical significance was determined by student's t test. \n",
      ".Figure 1(A) Expression in E. coli and purification of human GS (hGS). Protein was separated by SDS-PAGE and stained with Coomassie Blue dye. M: marker Precision Plus Protein Dual Color Standard (Biorad). From right Lanes 1-4: E. coli Bl21(DE3) cells containing the expression vector without (lanes 1 and 2) and with (lanes 3 and 4) the coding sequence of hGS. Samples were taken immediately before (lanes 1 and 3) and 3,5 h later (lanes 2 and 4) the induction of expression with isopropil-β-D-1-tiogalattopiranoside (IPTG) 0.7mM. The same number of bacteria was analyzed in each sample. Lane 5: isolated and purified inclusion bodies (4 μg). Adiacent boxed lane: Western Blotting analysis of GS. Kinetic study of the reaction catalyzed by recombinant hGS. The reaction, started by adding ATP, was linear for at least 10 min at 25°C (B). Kinetic study of the reaction catalyzed by recombinant hGS. Lineweaver-Burk plot reporting the hGS activity at the indicated concentrations of glutamate in the absence (), or in the presence of glufosinate (C) Symbols- (): 0.025 glufosinate; or 0.050 mM of glufosinate; or 0.065 mM of glufosinate; 0.075 mM of glufosinate. Kinetic study of the reaction catalyzed by recombinant hGS. Lineweaver-Burk plot reporting the hGS activity at the indicated concentrations of glutamate in the absence (), or in the presence of MSO Symbols- (): 2 mM MSO ; () 3 mM of MSO ; () 4 mM of MSO ; () 5 mM of MSO Kinetic study of the reaction catalyzed by recombinant hGS. The insets represent the secondary plot of the slopes of Lineweaver-Burk plot obtained at the indicated concentrations of glufosinate (C) or MSO (D) used for determining the inhibitor constant Ki. GS inhibition (%) in presence of increasing concentrations of glufosinate () or MSO () results from the average of at least three independent experiments (E). The control value for uninhibited hGS activity is 0.51 ±0.08 μmol/(min x mg protein). (F-K) Comparative analysis of GS structures from several organisms. Lateral (F) and top view (G) of the entire GS decameric (a dimer of pentamers) structure are reported in pink cartoon representation. (H) Cartoon representation showing the top view of the monomer-monomer interface hosting cofactors, substrates or inhibitors (Dataset EV1) participating to or inhibiting the conversion of glutamate to glutamine. In particular ADP (cyan), Phosphate ions (brown), MSO phosphate (P3S, yellow), phosphoaminiphosphonic acid-adenylate ester (ANP, white), glutamate (magenta), citrate (black) and the imidazopyridine inhibitor ((4-(6-bromo-3-(butylamino)imidazol(1,2-a)pyridin-2-yl) phenoxy) acetic acid, green) are reported in sticks representation, whereas Mn2+ ions are reported in pale blue spheres and Mg2+ in pale green spheres (involved in the coordination of the imidazopyridine inhibitor). (I) Superimposition of all the sampled 17 crystallized structures (Dataset EV1). (J) 2D representation of glutamate, P3S, glufosinate and glyphosate. (K) Zoomed view of the crystallized P3S binding region and the docked glufosinate and glutamate binding regions within the superimposed human (white cartoon) and Z.mays (pink cartoon) GS structures. \n",
      ".Figure 5(A) Experimental plan. Mice were subcutaneously injected with LLC cells. When the tumor reached 300-400 mm3 size, mice were tumor resected. 14 days after resection mice were treated by gavage with glufosinate (20mg/kg) for 14 days. (B) Evaluation of body weight in vehicle and glufosinate (20mg/kg) treated mice (n=8). (C) Representative images of metastasis at the beginning and end of therapy. Evaluation of the metastatic burden following glufosinate treatment by H&E staining (n=8). (D) Evaluation of lung total metastatic area and of lung metastatic lesions per size by H&E staining in vehicle and glufosinate (20mg/kg) treated mice (n=8). (E) Experimental plan for macrophage depletion setting. Mice were preconditioned for 3 days by i.p. injection with anti-CSF1R antibody (or isotype IgG control) before LLC (CD90.1+) tumor cell injection and then injected 3 times a week until sacrifice. When the tumor reached 80-100 mm3 mice were treated by gavage with vehicle or glufosinate (20mg/kg) for 14 days. Quantification (F) of total metastatic area in LLC-tumor bearing mice preconditioned with IgG or anti-CSF1R antibody and treated with vehicle or glufosinate (20mg/Kg). representative images (G) of total metastatic area in LLC-tumor bearing mice preconditioned with IgG or anti-CSF1R antibody and treated with vehicle or glufosinate (20mg/Kg). \n",
      ".Figure 4Quantification of total length of vessels (A) in vehicle and glufosinate (20mg/kg) treated mice (n=6). Quantification of CD31+ tumor vessel area (B) and of αSMA+ pericyte-covered vessels over the total number of CD31+ vessels (with representative images) (C) in vehicle and glufosinate (20mg/kg) treated mice (n=6). (D) Quantification and representative images of Lectin+ tumor vessel area over the total number of CD31+ vessels in vehicle and glufosinate (20mg/kg) treated mice (n=6). (E) Quantification and representative images of PIMO+ tumor hypoxic areas in vehicle and glufosinate (20mg/kg) treated mice (n=6). FACS quantification of TCRbeta+ CD8+ cytotoxic T cells in tumors of vehicle and glufosinate-treated mice (F). Evaluation of CD69 (G) and CD25 (H) proteins, expressed as Mean Fluorescence Intensity (MFI), was determined by flow cytometry on the surface of responders CD8+ T cells (n=4). (I) Quantification of secreted IFNγ in interstitial tumoral fluid of vehicle and glufosinate-treated mice (n=6). \n",
      ".Figure 3(A) Experimental plan. Mice were subcutaneously injected with LLC cells. When the tumor reached 80-100mm3 size, mice were treated by gavage with glufosinate (10, 20 and 40 mg/kg) for 14 days. Quantification of glufosinate (B) levels in wet tumor tissues of vehicle and glufosinate-treated mice (n=8), by liquid chromatography-mass spectrometry (LC-MS). Quantification of glutamine (GLN) (C) levels in wet tumor tissues of vehicle and glufosinate-treated mice (n=8), by liquid chromatography-mass spectrometry (LC-MS). Subcutaneous LLC tumor growth over the time (D) (pool of 2 independent experiments; 10 mice per condition in total). number of lung metastatic nodules in vehicle and glufosinate (10 and 20mg/kg) treated mice (E) (pool of 2 independent experiments; 10 mice per condition in total). Quantification and representative images of F4/80+ CD11c+ cells in the lung metastasis (F), F4/80+ CD206+ (G) cells infiltration in tumors of vehicle and glufosinate-treated mice (n=6). Quantification and representative images of F4/80 CD11c+ (H) cells infiltration in tumors of vehicle and glufosinate-treated mice (n=6). RT-PCR quantification of M2 (Ccl22, Arg1 and Ccl17) (I) markers in vehicle and glufosinate treated mice (n=4). RT-PCR quantification of M1 (Tnfa, Cxcl9, Nos2, Cd86 and Cd80) (J) markers in vehicle and glufosinate treated mice (n=4). \n",
      ".Figure 2Evaluation of M1 markers in macrophages by real-time PCR. Fold change of TNFA, CD80, CXCL9, and CXCL10 mRNA in IL10, MSO- and glufosinate (10 and 20µM)- stimulated IL10 macrophages (n=3). Evaluation of M2 markers in macrophages by real-time PCR. Fold change of MRC1, MSR1, mRNA in IL10, MSO- and glufosinate (10 and 20µM)- stimulated IL10 macrophages (n=3). Evaluation of M2 markers in macrophages by real-time PCR. Fold change of CCL17 and CCL18 mRNA in IL10, MSO- and glufosinate (10 and 20µM)- stimulated IL10 macrophages (n=3). Evaluation of M1 markers in macrophages following HIF1α inhibition by real-time PCR. Fold change of TNFA, CXCL10, CD86 mRNA in IL10 alone or glufosinate (10 and 20µM)- and acriflavine/glufosinate (10 and 20µM)- IL10 macrophages (n=3). Evaluation of M1 markers in macrophages following HIF1α inhibition by real-time PCR. Fold change of CD80 and CXCL9 mRNA in IL10 alone or glufosinate (10 and 20µM)- and acriflavine/glufosinate (10 and 20µM)- IL10 macrophages (n=3). Evaluation of M2 markers in macrophages following HIF1α inhibition by real-time PCR. Fold change of MRC1, MSR1, and CCL18 mRNA in IL10, glufosinate (10 and 20µM)- and acriflavine/glufosinate (10 and 20µM)- IL10 macrophages (n=3). (Q) Quantification of cancer cell motility through a matrigel coated membrane in presence of IL10, MSO/IL10 and glufosinate (10 and 20µM)- IL10 treated macrophages after 24 h of incubation (n=6). (R) Evaluation of the capillary network formation in presence of macrophages pretreated for 24 h with IL10 or MSO/IL10, and glufosinate (10 and 20µM)/IL10 after 4 h of incubation with HUVEC cells (n=6). (S) CD8+ T cell suppression by macrophages treated with IL10 or MSO/IL10, and glufosinate (10 and 20µM)/IL10 for 24 h (n=4). Proliferation was evaluated by reading radioactivity as cpm (counts per minute), after incubation with 1 μCi / well tritiated thymidine. The proliferation of T cells cultured without macrophages was used as control. (T) CD8+ T cell recruitment in a transwell system by macrophages treated with IL10 or MSO/IL10, and glufosinate (10 and 20µM)/IL10 for 24 h versus macrophages treated with LPS/IFNγ; the migration of T cells cultured without macrophages (Mφ-) in presence of CXCL10 was used as positive control (n=4). (U) Representative image of Western blotting analysis of HIF1α, REDD1, 4E-BP1, S6 and P70S6K (in their phosphorylated and unphosphorilated form) to test mTOR activation in normoxic (NRX) and hypoxic (HYP) IL10 macrophages treated with glufosinate (20µM), rapamycin (20nM) and a combination of both (n=3). (V) CD8+ T cell suppression by normoxic and hypoxic IL10 macrophages treated with glufosinate (20µM), rapamycin and a combination of both for 24 h (n=4). Proliferation was evaluated by reading radioactivity as cpm (counts per minute), after incubation with 1 μCi / well tritiated thymidine. The proliferation of T cells cultured without macrophages was used as control. \n",
      ".Figure 7Quantification of total length of vessels (A) Six images per tumor were analysed. Scale bar: 50µm. CD31+ tumor vessel area (B) and of αSMA+pericyte-covered vessels over the total number of CD31+ vessels (with representative images) (C) in vehicle and glufosinate (20mg/kg) treated mice (n=6). Six images per tumor were analysed. Scale bar: 50µm. (D) Quantification and representative images of CA9+ tumor hypoxic areas in vehicle and glufosinate (20mg/kg) treated mice (n=6). Six images per tumor were analysed. Scale bar: 50µm. (E, F) FACS quantification of CD8+ cytotoxic T cells (E) and CD69+ CD8+ T cells (F) over the total number of CD45+ in vehicle or glufosinate (20mg/kg) treated mice (n=6). \n",
      ".Figure 6(A) Experimental plan. Mice were intradermally injected with YUMM 1.7 (CD90.1+) cells. When the tumor reached 80-100mm3 size, mice were treated by gavage with glufosinate (20mg/kg) or vehicle for 14 days. Intradermal YUMM 1.7 (CD90.1+) tumor growth over the time (B) in vehicle and glufosinate (20mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). number of metastatic lesions per lung (C) in vehicle and glufosinate (20mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). (D) Representative images of metastatic lesions checked by DAB-H stain for CD90.1 positivity in vehicle and glufosinate (20mg/kg) treated mice (n=6). Six images per lung were analysed. Scale bar: 2mm. Zoomed area scale bar: 50µm. (E, F) FACS quantification of M1-like MHC class IIhigh (E) and MHC class IIlow (F) CD11c+ CD206- TAMs over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). (G) Quantification and representative images of CD11c+ cells over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). Six images per tumor were analysed. Scale bars: 50 µm. FACS quantification of M2-like MHC class IIhigh CD11c- CD206+ TAMs over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). FACS quantification of M2-like MHC class IIlow (I) CD11c- CD206+ TAMs over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). (J) Quantification and representative images of CD206+ cells over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). Six images per tumor were analysed. Scale bars: 50 µm. FACS quantification of M1-like MHC class IIhigh (K) TAMs over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). FACS quantification of M1-like M2-like MHC class IIlow (L) TAMs over the total number of F4/80+ cells in vehicle and glufosinate (20mg/kg) treated mice (n=6). (M) Quantification of MHC-II protein, expressed as Mean Fluorescence Intensity (MFI), on F4/80+ cells, was obtained by flow cytometry (n=6). (N) RT-PCR quantification of M2 (Arg1 and Cxcr4) and M1 (Cxcl9 and Nos2) markers in vehicle and glufosinate (20mg/kg) treated mice (n=6). \n",
      ": Size-dependent recognition of defined chitin oligomers in humans, mice and plants.Figure 1 (A) Electron micrograph of a human MoMac engulfing a crude chitin particle (n=2) (B) Size comparison (to scale) of typical TLR ectodomain and a chitin 7-mer (seven NAG subunits) I one representative of 'n' (given in brackets for each panel) independent experiments is shown (mean+SD) Release of IL-6 and/or TNF release (ELISA) from (C) human MoMacs (n=6-8) Data information represent data (mean+SEM) combined from 'n' (given in brackets for each panel) technical or biological replicates (human donor * p<0.05 according to Wilcoxon signed rank sum ( Release of IL-6 and/or TNF release (ELISA) fro (D) murine BMDMs (n=6 Data information represent data (mean+SEM) combined from 'n' (given in brackets for each panel) technical or biological replicate mic * p<0.05 according to Wilcoxon signed rank sum (C, D Release of IL- (ELISA) fro THP-1 cells (n=3) Data information I one representative of 'n' (given in brackets for each panel) independent experiments is shown (mean+SD) Student's t-tes  (f) GFP in vivo fluorescence post intradermal injection of chitin oligomers into LysMEGFP/+ C57BL/6 mice (n=3/group) Data information represent data (mean+SEM) combined from 'n' (given in brackets for each panel) technical or biological replicate mic (G) Representative in vivo imaging results I one representative of 'n' (given in brackets for each panel) independent experiments is shown (mean+SD) Leukocyte infiltration in vivo after intratracheal administration assessed by (H) BALF flow cytometry (n=3-4/group Data information represent data (mean+SEM) combined from 'n' (given in brackets for each panel) technical or biological replicate mic ), one-way ANOVA (H Leukocyte infiltration in vivo after intratracheal administration assessed b (I) histology analysis (n=2), scale bar = 200 μm, arrows show cellular infiltrate I one representative of 'n' (given in brackets for each panel) independent experiments is shown (mean+SD) A. thaliana seedling (J) target gene mRNA induction (n=3 I one representative of 'n' (given in brackets for each panel) independent experiments is shown (mean+SD) Student's t-tes A. thalian leaf piece (K) ROS production (0-45 min, n=6) Data information represent data (mean+SEM) combined from 'n' (given in brackets for each panel) technical or biological replicate plant leave \n",
      ".Figure 9(A) Time course evaluation of murine body weight during treatment in vehicle and glufosinate (10 and 20 mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). (B) Blood counts of WBC and RBC in vehicle and glufosinate (10 and 20 mg/kg) treated mice (n=8). Evaluation of liver toxicity following glufosinate treatment. Quantification of liver glufosinate (C) in serum of vehicle and glufosinate (10 and 20 mg/kg) treated mice (n=8). Evaluation of liver toxicity following glufosinate treatment. hepatotoxicity markers ALT (Alanine Aminotransferase) and AST (Aspartate aminotransferase) (D) in serum of vehicle and glufosinate (10 and 20 mg/kg) treated mice (n=8). Evaluation of neuronal toxicity following glufosinate treatment. Quantification of brain glufosinate (E) Evaluation of neuronal toxicity following glufosinate treatment. grip strength for front paws (triangular bar) (F), as an indicator of the development of tactile allodynia in vehicle and glufosinate-treated mice (n=4), before the treatment (baseline), after 1 day (acute Toxicity) and after 15 days (chronic toxicity) (pool of 2 independent experiments; 10 mice per condition in total). Evaluation of neuronal toxicity following glufosinate treatment. grip strength for all paws (rectangle grid) (G), of the latency to fall off an accelerating rotarod (H), and of the mean paw withdrawal force that caused animals' response in the von Frey test (left hindpaw, I; right hindpaw, J) as an indicator of the development of tactile allodynia in vehicle and glufosinate-treated mice (n=4), before the treatment (baseline), after 1 day (acute Toxicity) and after 15 days (chronic toxicity) (pool of 2 independent experiments; 10 mice per condition in total). \n",
      ".Figure 8(A) Experimental plan. Mice were injected with 4T1 cells in the mammary fat pad. When the tumor reached 80-100mm3 size, mice were treated by gavage with glufosinate (10 mg/kg) or vehicle for 14 days. Evaluation of 4T1 tumor growth over the time (B) , in vehicle and glufosinate (10mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). mice body weight (C) in vehicle and glufosinate (10mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). number of metastatic nodules (D), in vehicle and glufosinate (10mg/kg) treated mice (pool of 2 independent experiments; 10 mice per condition in total). Total metastatic area (E) in vehicle and glufosinate (10mg/kg) treated mice (n=8). Six images per lung were analysed. Scale bar: 2mm. number of lung metastatic lesions per size by H&E staining (F) with representative images (G) in vehicle and glufosinate (10mg/kg) treated mice (n=8). Six images per lung were analysed. Scale bar: 2mm. (H, I) Quantification of F4/80+ macrophage density (H), of F4/80+ CD206+ cells infiltration in the lung metastasis and representative images (I), in vehicle and glufosinate-treated mice (n=6). Six images per lung were analysed. Scale bar: 20µm. (J) Experimental plan for intravasation setting. Mice were injected with 4T1-GFP+ cells in the mammary fat pad and immediately treated by gavage with glufosinate (10mg/kg) or vehicle for 14 days (neoadjuvant therapeutic regimen). After 14 days of treatment, at the time of tumor resection, blood was collected and analyzed for GFP+ circulating tumor cells (CTCs). Survival studies were performed until the natural death of the post-surgical mice. GFP+ CTCs amount in blood, expressed as GFP expression levels, was determined by qPCR (n=8) (K). Blood was drawn from treated and untreated tumor-bearing mice at the time of tumor resection and plated in culture dishes. The formation of tumor cell colonies was traced over time. Graph shows the number of GFP+ tumor cell colonies (n=8) (L). After treatment and tumor resection overall survival studies were performed until the natural death of the post-surgical mice (n=11) (M). (N) Experimental plan for extravasation setting. GFP-expressing 4T1 were injected intravenously into Balb/c mice pre-conditioned with glufosinate (10mg/kg) or vehicle for 2 days. After 2 days of treatment, the overall lung metastatic breast cancer cell burden was determined. (O) GFP+ cancer cells amount in lungs, expressed as GFP expression levels, was determined by qPCR in vehicle and glufosinate (10mg/kg) treated mice (n=8). The results were normalized to the vehicle group. \n",
      ".Figure 1Domain boundaries and sequence motifs mapped to the sequence of CNFY. Cartoon representation of CNFY, colored according to domain boundaries determined with PiSQRD (Aleksiev et al, 2009). Dark blue: domain D1, cyan: domain D2, dark green: domain D3, yellow: ADP-ribosyltransferase-like domain D4, pink: deamidase domain D5. Other colors indicate the position of sequence motifs that have been identified in E. coli CNF1, namely light blue: p37LRP/67LR receptor-binding motif, red: hydrophobic stretches predicted to form membrane-inserting α-helices, orange: cleavage site, magenta: main Lu/BCAM receptor-binding motif. The positions of N- and C-terminus are indicated by N and C, respectively. Surface representation of CNFY as seen from two different orientations with respect to B. Note that the cleavage site between D3 and D4 (orange) as well as the deamidase active site in D5 are partially blocked in the structure of full-length CNFY. The C-terminal domain D5 interacts mainly with D3 (610 Å­2), which partially hides the catalytic site of D5, but it interacts only weakly with D4 (380 Å2), which itself establishes an extensive interface with D1 (1390 Å2) by mainly hydrophilic interactions (17 hydrogen bonds and 6 salt bridges). \n",
      ": Chitin is a unique fungal TLR2 ligand.Figure 3 (A) NF-κB activity in TLR2-transfected HEK293T cells (n=3) Data information I one representative of 'n' independent experiments is shown (mean+SD) Student's t-tes  (B) IL-8 release and ROS production from primary PMNs (n=7) Data information represent data (mean+SEM) from 'n' technical or biological replicates (donors) * p<0.05 according to Wilcoxon signed rank su  (C) NF-κB activity in hTLR2 HEK-Dual cells pretreated with TLR1, TLR2 or TLR6 blocking Abs (n=2) I one representative of 'n' independent experiments is shown (mean+SD). ), one-way ANOVA with Sidak correction (C  (D) IRAK4 S186 and T345 phospho-site containing IRAK4 peptides quantified in proteome-wide phospho-screen in THP-1 cells (n=1) Data informatio I one representative of 'n' independent experiments is shown (mean+SD) for details on data acquisition and analysis for panel see Expanded View Information  (E) Principal component analysis of microarrays conducted on stimulated whole blood samples from healthy donors (n=3-5) Data informatio represent data (mean+SEM) from 'n' technical or biological replicates (donors)  (F) Microarray intensities (excerpt) of chitin-specific signature genes (n=5) Data information represent data (mean+SEM) from 'n' technical or biological replicates (donors) for details on data acquisition and analysis for panels (D-F) see Expanded View Information  (G) Relative cytokine mRNAs in whole blood stimulation (n=5-13) Data information represent data (mean+SEM) from 'n' technical or biological replicates (donors) * p<0.05 according to Wilcoxon signed rank su \n",
      ": TLR2 is the mediator of chitin oligomer-mediated immune activation.Figure 2TNF released from WT or Myd88 (A KO BMDM (total n=6-9/group combined from 3 experiments) Data information: I represent data (mean+SEM) combined from 'n' technical or biological replicate mic Student' t-test TNF released from W or Tlr KO BMDM (total n=6-9/group combined from 3 experiments) Student's t-tes  (C) IL-6 released from primary MoMacs (n=5) treated with non-targeting (NT), TLR2- or MyD88-specific siRNA relative to NT condition one-sample t-tes  (D) CD62L shedding from human PMNs (n=4-8) with or without isotype control or anti-TLR2 blocking antibody Data information: I represent data (mean+SEM) combined from 'n' technical or biological replicate (human donor * p<0.05 according to Mann-Whitney  (E) IL-6 protein released from mock or TLR2 Cas9-CRISPR-edited THP-1 cells (n=3) I one representative of 'n' independent experiments is shown (mean+SD) ), Student's t-tes  (F) NF-κB activity in empty vector (EV) or TLR2-transfected HEK293T cells (n=3) I one representative of 'n' independent experiments is shown (mean+SD)  (G) MPO release from intradermally injected WT or Tlr2 KO mice (n=5-15 biopsies each) Data information I represent data (mean+SEM) combined from 'n' technical or biological replicate mic ), Student's t-tes  (H) BALF IL-6 and TNF from intratracheally treated WT or Tlr2 KO mice (n=4-5 mice each) one-way ANOVA with Dunnett's multiple compariso  (I) Relative IL10 mRNA in whole blood of WT or heterozygous TLR2 R753Q carriers (n=5 each) Data information: I represent data (mean+SEM) combined from 'n' technical or biological replicate (human donor * p<0.05 according to Mann-Whitney \n",
      ": Direct chitin TLR2 binding and immune activation can be blocked by SSL3 or short chitin oligomers.Figure 4Flow cytometric quantificatio of Alexa647-labeled C10-15 interaction with mTLR2-Fc Protein (n=4 each) Data information: Quantification sub-panel represent data (mean+SE combined from 'n' biological replicate mic I one representative of 'n' independent experiments is show p<0.05 according to Mann-Whitney U (A microscale thermophoresis (MST) analysis (B) of Alexa647-labeled C10-15 interaction with mTLR2-Fc Protein (n=4 each) I one representative of 'n' independent experiments is show (C) Flow cytometr of Candida albicans stained with control IgG or mTLR2-Fc anti-Fc-Alexa594, together with ConA-Alexa488 and CFW (n=4 each). Scale bar = 5 μm Data information: Quantification sub-panel represent data (mean+SE combined from 'n' biological replicate mic I one representative of 'n' independent experiments is show one-sample t-test (C (D) fluorescence microscopy of Candida albicans stained with control IgG or mTLR2-Fc anti-Fc-Alexa594, together with ConA-Alexa488 and CFW (n=4 each). Scale bar = 5 μm I one representative of 'n' independent experiments is show (E) Docking of chitin 10 (magenta) into the TLR2 (orange) Pam2 (green) lipopeptide binding pocket (close-up, pdb: 2z7x (F) overlayed with SSL3 from a SSL3-inhibited TLR2 complex (pdb: 5d3i)  (g) MST analysis of Alexa647-labeled C10-15 and mTLR2-Fc protein with SSL3 titration (n=2) Data information I one representative of 'n' independent experiments is show (H) NF-κB activation in TLR2-transfected HEK293T cells (n=3 Data information I one representative of 'n' independent experiments is shown (for asmean+SD) ), Student's t-tes IL-8 release from primary PMNs (n=2) without or with SSL3 Data information I one representative of 'n' independent experiments is show asmean+SD) Student's t-tes  (J) BALF TNF in C57BL/6 mice (n=5/group) upon C10-15 administration without or with SSL3 Data information: Quantificatio -panel represent data (mean+SE combined from 'n' biological replicate mic one-way ANOVA with Dunnett's multiple comparison (J)  (K) ROS production between 0 and 45 min post chitin application in A. thaliana leaf pieces (n=6) Data information: Quantificatio panels ( represent dat mean±SD in K combined from 'n' biological replicate plant leave  (L) MST analysis of Alexa647-labeled C10-15 and mTLR2-Fc Protein with C5 titration (n=2) Data informatio I one representative of 'n' independent experiments is show \n",
      ".Figure 2Schematic overview of marker-tagged CNFY deletion variants. 3xFlag-tagged CNFY deletion variants were expressed in Y. pseudotuberculosis YP147 (ΔcnfY) from plasmids under control of their own promoter and were detected in whole cell extracts using an anti-Flag antibody. To test secretion of the CNFY variants, full-length CNFY and different N- and C-terminally deleted variants fused to beta-lactamase (TEM) were expressed in Y. pseudotuberculosis YP147 (ΔcnfY). Beta-lactamase activity in the culture supernatant was subsequently measured using nitrocefin as substrate. The data represent the mean ± SD of three independent experiments, carried out in triplicates. HEp-2 cells remained untreated or were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing full-length CNF1, CNFY or the N- or C-terminally deleted toxin variants at 37°C for 4 h. The cells were thoroughly washed, pelleted, lysed and the toxin variants bound to the cells were identified by western blotting using an anti-Flag antibody. \n",
      ".Figure 4HEp-2 cells were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing full-length CNFY, N- or C-terminal deletion variants fused to GFP (green) for 90 or 180 min. Cells were fixed and processed for fluorescence microscopy. The red fluorescent signal represents late endosomes (CellLight Late Endosomes-RFP (Rab7a)). Nuclei were stained with DAPI (blue). A merged image of the different channels is shown, and smaller images are magnified views of boxed areas. White scale bar is 10 µm. \n",
      ".Figure 3HEp-2 cells were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing full-length CNFY or the N- or C-terminally deleted toxin variants fused to β-lactamase (TEM) at 37°C for 4 h. Cleavage of the reporter dye CCF4-AM was used to visualize toxin delivery. After cell entry CCF4-AM is rapidly converted into the negatively charged form CCF4, which is retained in the cytosol and emits a green fluorescence signal (520 nm). In the presence of trans­located β-lactamase fusion proteins, CCF4-AM is cleaved, and disruption of FRET results in blue fluorescence (447 nm). Scale bar: 20 µm. Left upper and right panel: HEp-2 cells re­mained untreated or were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing full-length CNFY or the N- or C-terminally deleted toxin variants for 4 h. Cells were lysed and the de­amidation of RhoA was analyzed by the shift of the modified Rho GTPase band in SDS PAGE gels; left lower panel: HEp-2 cells were lysed and the cell extracts were incubated with full-length CNFY or the N-ter­minally deleted toxin variants for 4 h. The de­amidation of RhoA in the cell extracts was analyzed by the mobility shift of the modified Rho GTPase on SDS PAGE after detection with anti-RhoA antibodies. HEp-2 cells were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing full-length CNFY or the N- or C-terminally deleted toxin variants for 24 h. The cell nuclei were strained with DAPI (blue) and the actin cytoskeleton was stained using FITC-phalloidin (green). The formation of large, multinuclear cells was ob­served by fluorescence micro­­scopy and the formation of thick actin stress fibers and membrane actin folding were only observed with CNFY-treated cells. The white scale bar is 40 µm. Cells incubated with extracts of YP147 (ΔcnfY) harboring the empty expression vector were used as negative controls. \n",
      ".Figure 5Side-by-side comparison of CNFY and nigritoxin. Nigritoxin is a toxin of crustaceans and insects. The translocation domain of nigritoxin (PDB entry 5M41, (Labreuche et al, 2017)) and domain D1 of CNFY show partial structural similarity (highlighted areas). This similarity was identified with DALI (Holm & Rosenström, 2010) which was also used to align both structures. The ART-like domain D4 of CNFY. Essential residues of canonical ARTs are not conserved in CNFY (RSE-ARTs exemplified by C. perfringens iota toxin, PDB entry 4H03 (Tsurumura et al, 2013); HYE-ARTs exemplified by P. aeruginosa ExoA, PDB entry 2ZIT (Jørgensen et al, 2008); carbon atoms of NAD+ shown in black). The deamidase domain D5 of CNFY. C866 and H881 form a conserved catalytic dyad also found in the deamidase domain of E. coli CNF1 (PDB entry 1HQ0, (Buetow et al, 2001)) and Burholderia pseudomallei lethal factor BLF1 (PDB entry 3TU8, (Cruz-Migoni et al, 2011)). \n",
      ".Figure 6HEp-2 cells were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing CNFY, the toxin variant mut1: CNFYI535L/P536A/V537G or mut2: CNFYI535L/P536A/­V537G/F539L/­D541A/K542A fused to TEM or no CNFY protein (empty vector) for 4 h. Cells were lysed and the binding of the different CNFY proteins to HEp-2 cells was analyzed by immunoblotting. HEp-2 cells were incubated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing CNFY, the toxin variant mut1: CNFYI535L/P536A/V537G or mut2: CNFYI535L/P536A/­V537G/F539L/­D541A/K542A or no CNFY protein (empty vector). The cell nuclei were stained with DAPI (blue) and the actin cytoskeleton was stained using FITC-phalloidin (green). The results indicated the formation of polynucleated cells and stress fibers only in cells treated with CNFY and CNFYI535L/­P536/V537G. The white scale bar is 20 µm. Nitrocefin (2 mM) was added to the supernatant from 25°C overnight Yersinia cultures expressing the indicated CNFY derivatives to determine β-lactamase activity by measuring changes in absorbance at 390 nm (yellow) and 486 nm (red). The data represent the mean ± SD of three independent experiments, carried out in triplicates. The viability of Y. pseudotuberculosis YPIII expressing the indicated CNFY derivatives was assessed in equalized bacterial cultures using the BacTiter-Glo Microbial Cell Viability Assay kit (Promega). The data represent the mean ± SD of three independent experiments, carried out in triplicates. HEp-2 cells treated with 20 µg/ml of whole cell extract of Y. pseudotuberculosis expressing indicated CNFY variants for 4 h were lysed and the deamidation of RhoA was analyzed by the mobility shift of the modified RhoA GTPase detected by immunoblotting. The activity of the CNFY derivatives was tested by analyzing the deamidation of RhoA in HEp-2 cell lysates by the mobility shift of the modified GTPase detected by immunoblotting. \n",
      ".Figure 8Shown are the architecture of CNFY, Pasteurella multocida toxin PMT and two uncharacterized proteins from Pseudomonas syringiae. The released fragment of PMT contains three domains of which C1 is required for membrane binding, the C2 domain has an unknown function and the C3-domain activates heterotrimeric G-proteins by deamidation. The two Pseudomonas syringiae proteins A0A0P9UH04 and A0A0N8SZE6 represent two uncharacterized toxins that encode catalytic domains of the indicated type. While sequence alignments unequi­vocally reveal a CNF-like imperfect β-barrel in PMT, the presence of this domain in the P. syringiae toxins is less obvious. \n",
      ".Figure 7Crystal structure of the free D4-5 subunit. Note the different relative orientations of domains D4 and D5 with respect to the structure of full-length CNFY (top, thin grey lines). The domain D4 forms a large interface area (1100 Å2) with the catalytic domain D5 involving several polar interactions (8 hydrogen bonds and 8 salt bridges), whereby the active crevice is extended and fully solvent-exposed as can be seen in the right surface plot at the bottom of the panel. The hypothetical NAD+ binding site of the ART-like D4 domain is located on the opposite face (left surface plot). Note that the deamidase active site of domain D5, unlike in the full-length structure (Fig 1), is fully accessible and that its extended shape is also determined by domain D4. Comparative analysis of RhoA activation in HEp-2 cell lysate by CNFY and the recombinant D4-5 protein. Purified CNFY or the D4-5 fragment (1 µM) was added to extracts of HEp-2 cells and incubated for 10, 20, 30 or 60 min. De­amidation of RhoA was analyzed by the shift of the modified Rho GTPase band in SDS PAGE gels after detection with anti-RhoA antibodies. Comparative analysis of recombinant RhoA deamidation by CNFY and the D4-5 fragment. Recombinant RhoA was incubated with purified CNFY or the D4-5 fragment and samples were separated by SDS-PAGE after the indicated times before subjecting to trypsin digestion and quantification of deamidation of Q63 by mass spectrometry. Error bars represent standard deviations of triplicate measurements. \n",
      "DAPk family proteins do not induce cell death via the mitochondrial pathway.Figure 2.(A) Scoring loss of mitochondrial membrane potential (ΔΨm). 293 cells expressing luciferase (black), DRP-1 Δ73 (pink), DAPk ΔCaM (purple), and Bax (green) were assayed for loss of mitochondrial membrane potential 24 h after transfection using flow cytometric analysis of TMRM fluorescence. The fraction of TMRM-negative cells calculated from three experiments is shown in the graph below. The small increase in TMRM fluorescence in the kinase-transfected cells is not statistically significant. (B) Scoring the release of cytochrome C from mitochondria. 293 cells cotransfected with luciferase (1 and 2), Bax (3 and 6), DRP-1 Δ73 (7 and 8), or DAPk ΔCaM (9 and 10) and GFP constructs were incubated for 24 h and immunostained using anti-cytochrome C antibodies. Cells were visualized for cytochrome C (left) or GFP (right) fluorescence. Bax-transfected spread (3 and 4) or blebbed (5 and 6) cells are shown. In fields that also contain nontransfected cells, arrows point to GFP-positive cytochrome C-stained cells. (C) Assessing the effects of Bcl-2 and Bcl-XL. 293 cells were cotransfected with luciferase, DRP-1 Δ73, DAPk ΔCaM, or Bax, and either Bcl-2 or Bcl-XL together with GFP. GFP-positive cells were visualized by fluorescent microscopy and scored for the appearance of cell death morphology 24 h after transfection. Total cell death refers to the sum of all morphological changes observed. \n",
      "Morphological changes induced by DAPk family proteins.Figure 1.(A) Quantification of cell death in 293 cells24 h after transfection with luciferase, p55/TNFR1, DRP-1 Δ73 or DAPk ΔCaM, and GFP. Graphs show the percentage of GFP-positive cells with altered cell morphology. (B) Photographs of GFP-positive cells cotransfected with luciferase (1), p55/TNFR1 (2), DRP-1 Δ73 (3), or DAPk ΔCaM (4). (C) Photographs showing the overlay of GFP and phase-contrast images of 293 cells24 h after transfection with GFP and luciferase (1) or with GFP- DAPk (2 and 3); both a blebbing GFP-DAPk-expressing cell (2, arrows indicate blebs) and spread nontransfected cells are shown (3). (D) Hoechst staining (4-6) of GFP-positive 293 cells (1-3) 60 h after cotransfection with GFP and DAPk ΔCaM (1 and 4), DRP-1 Δ73 (2 and 5), or p55/TNFR1 (3 and 6). Arrows point to condensed nuclei resulting from DRP-1 Δ73 or DAPk ΔCaM transfections (4 and 5). \n",
      "DRP-1 induces autophagy in various cell lines.Figure 4.(A) Quantification of autophagy in 293 and MCF-7 cells. Graphs show the percentage of MDC-positive cells resulting from luciferase or DRP-1 Δ73 transfections (mean ± SD calculated from triplicates of 100 cells each; note that total cell number comprises transfected and nontransfected cells). These experiments were repeated three times with reproducible results. (B) Photographs of MDC-stained MCF-7 cells from the same experiment shown in A. Arrows point to MDC-positive cells. Transfections: luciferase (1a and 1b) and DRP-1 Δ73 (2a and 2b) at low (a) and high (b) magnifications. \n",
      "High resolution morphology of DAPk family protein-induced cell death.Figure 3.(A) Transmission electron micrographs of 293 cells transfected with luciferase (1) p55/TNFR1 (2a-2c), DRP-1 Δ73 (3a and 3b; part of fig. 3b is magnified on its right), and DAPk ΔCaM (4). The different stages of autophagic development are depicted as follows: immature autophagic vesicles (double arrow), mature autophagic vesicles and autolysosomes (black arrow), and residual bodies (dashed arrow). Empty vacuoles (white arrow) are also shown. Arrowheads show condensed and fragmented chromatin. m, mitochondria; g, Golgi apparatus. Bar, 1 μm. (B) Transmission electron micrographs of MCF-7 cells transfected with luciferase (1), p55/TNFR1 (2a-2c), DRP-1 Δ73 (3a and 3b), and DAPk ΔCaM (4). Immature autophagic vesicles (double arrow), autophagic vesicles (black arrow), and residual bodies (dashed arrow) are shown. Insets in B, 2b and 2c, correspond to empty vacuoles, whereas those in B, 3a and 4, correspond to autophagic vesicles. Arrowheads show condensed and fragmented chromatin. m, mitochondria; dm, darkened mitochondria; n, nucleus; g, Golgi apparatus; v, vacuoles; ly, lysosomes; er, endoplasmic reticulum. \n",
      "DRP-1 and DAPk dominant negative mutants reduce membrane blebbing resulting from p55/TNFR1 transfections in 293 cells.Figure 6.(A) 293 cells were cotransfected with p55/TNFR1 in combination with DAPk death domain fused to GFP or DRP-1 K42A and pEGFP-N1 vector. A control luciferase vector was also used. GFP-positive cells were visualized by fluorescent microscopy and scored for the appearance of membrane blebbing 24 h after transfection. (B) Photographs of the various categories of p55/TNFR1-transfected 293 cells: spread cells (1), blebbed cells (2), condensed cells (3), cells displaying a fragmented cytoplasm (4), and cells carrying fragmented nuclei within a noncondensed cytoplasm (5). (C) Quantification of each death morphology observed 24 h after transfection with GFP, p55/TNFR1, and either luciferase or DRP-1 K42A. \n",
      "DAPk family proteins induce caspase-independent cell death in various cell lines.Figure 5.(A) Quantification of cell death-associated morphologies in 293 cells. Graphs show the percentage of cells harboring all spectrum of morphological changes occurring as a result of luciferase, p55/TNFR1, DRP-1 Δ73, or DAPk ΔCaM transfections in the presence or absence of the caspase inhibitors BD-fmk or z-VAD-fmk (100 μM; mean ± SD calculated from triplicates of 100 cells each). This experiment was repeated three times with reproducible results. (B) Quantification of cell death-associated morphologies in MCF-7 cells. Experimental conditions were performed as in A in the presence or absence of BD-fmk (100 μM). (C) Graphs show the percentage of cells harboring the complete spectrum of death-associated morphological changes occurring as a result of cotransfections of luciferase, p55/TNFR1, DRP-1 Δ73, or DAPk ΔCaM with the caspase inhibitor crmA or control luciferase vector (mean ± SD calculated from triplicates of 100 cells each). Immunoblot shows the expression levels of transfected HA-tagged DAPk and DRP-1 Δ73, confirming that cotransfection with crmA did not alter expression of the death-inducing proteins. (D) Transmission electron micrographs of 293 cells transfected with p55/TNFR1 (1) or DRP-1 Δ73 (2a and 2b; parts of these figures are magnified at the right) and treated with the caspase inhibitor BD-fmk (100 μM). Immature autophagic vesicles (double arrow), autophagic vesicles (black arrow), and autolysosomes (white arrow) are shown. m, mitochondria; g, Golgi apparatus. (E) Western blots of cell lysates prepared from HeLa cells transfected with luciferase, p55/TNFR1, DRP-1 Δ73, or DAPk ΔCaM and GFP and incubated for 24 h. Blots were reacted with anti-PARP antibodies, anti-caspase-8 antibodies, anti-caspase-3 antibodies, anti-HA antibodies for DRP-1 Δ73 and DAPk ΔCaM detection, and anti-β-tubulin antibodies to quantitate protein loading. \n",
      "DRP-1 K42A protects from autophagy induced by steroid withdrawal and amino acid starvation.Figure 7.(A and B). Quantification of autophagy in MCF-7 cells. MCF-7 cells were transfected with luciferase or FLAG-DRP-1 K42A followed by steroid withdrawal (serum starvation plus 10−6 M tamoxifen treatment) (A) or by serum and amino acid starvation (B) (mean ± SD calculated from triplicates of 100 cells each). These experiments were repeated three times with reproducible results. Proteins extracted from the transfected cells were subjected to Western blot analysis with anti-FLAG and anti-β-tubulin antibodies. (C) Photographs of the MDC-stained steroid withdrawn MCF-7 cells taken from the same experiment shown in A. Transfections: luciferase (1a and 1b) and DRP-1 K42A (2a and 2b) at low (a) and high (b) magnifications. \n",
      "Autophagy induced by steroid withdrawal or by amino acid starvation is insensitive to caspase inhibitors.Figure 8.(A) Photographs of MDC-stained MCF-7 cells. (Top) Steroid withdrawal. Cells were either grown in complete medium (1), steroid depleted (2), or steroid depleted in the presence of the broad caspase inhibitor BD-fmk (100 μM) (3) for 3 d. (Bottom) amino acid starvation. Cells were either grown in complete medium (1-3) or under amino acid starvation condition (4-6) without (1 and 4) or with (2 and 5) the presence of the broad caspase inhibitor BD-fmk (100 μM) or the general inhibitor of autophagy 3-methyladenine (10 mM) (3 and 6) for 3 d. (B) PARP cleavage in MCF-7-starved cells. Proteins extracted from 4-d steroid-depleted (left) or 12-h TNF-α-treated (100 ng/ml) cells (right) were subjected to Western blot analysis with anti-PARP antibodies. The proteins were prepared from the same experiment shown in A. (C) Graph showing the cumulative phase microscopy scores of dead (shrunk and detached) MCF-7 cells in tamoxifen-treated (filled) or nontreated (unfilled) cultures in the presence (▴) or absence (•) of the indicated caspase inhibitors (25 μM). \n",
      "DRP-1 is localized to the lumen of autophagic vesicles.Figure 10.DRP-1 is localized to the lumen of autophagic vesicles. Immunogold staining of 293 cells expressing HA-DRP-1 reveals a specific staining of DRP-1 in the lumen of double membrane autophagosomes containing cytoplasmic material (A and B) in autophagosomes containing organelles and vacuoles (C) and inside the body of autolysosomes containing digested matter (D). The gold particles do not overlap with the phagocytosed organelles in C or the lysosomes in D. \n",
      "DAPk antisense RNA protects from IFN-γ–induced autophagy of HeLa cells.Figure 9.(A) Quantitation of autophagy in HeLa cells. Cells were either grown in complete medium (1), treated with IFN-γ (1,000 U/ml) (2), or treated with IFN-γ in the presence of the broad caspase inhibitor BD-fmk (100 μM) (3) for 3 d. (B) Photographs of the MDC-stained HeLa cells corresponding to the experiment in A. Treatments: complete medium (1), IFN-γ (1,000 U/ml) (2), and IFN-γ (1,000 U/ml) + BD-fmk (3). (C) PARP cleavage in HeLa IFN-γ-treated cells. Proteins extracted from IFN-γ-treated (left) or TNF-α-treated (100 ng/ml) cells (right) were subjected to Western blot analysis with anti-PARP antibodies. The proteins were prepared from the same experiment shown in A. (D) Transmission electron micrographs of nontreated (top) and IFN-γ-treated (bottom) HeLa cells. Arrows point to autophagic vesicles. (E) DAPk antisense-transfected and control DHFR-transfected polyclonal populations of HeLa cells were incubated with hygromycin B (200 μg/ml) in the presence or absence of IFN-γ (1,000 U/ml) and BD-fmk (50 μM). Detection of autophagic vesicles was performed 2 d later using the MDC dye. Graphs represent mean ± SD of MDC-positive cells calculated from triplicates of 100 cells each. \n",
      ": Identification of new LMF1-interacting partners.Figure 1 A). Experimental schematic of DSP crosslinking experiment  B). Schematic of pBPA crosslinkin  C). Western blot of LMF1-partners after DSP crosslinking and affinity tag purification using LMF1's C-terminal His tag. Tetracycline induces LMF1 expression. Loading dye with 50mM DTT was used to break the disulfide bonds between LMF1 and its interacting partners. LMF1 complexes are labeled with asterisks and an arrow points to LMF1  D). Western blot of LMF1-partners after pBPA photo-crosslinking and affinity tag purification. The DMSO panel does not include pBPA and serves as a negative control. Fractions were eluted with increasing concentrations of imidazole. An arrow points to LMF1  E). SYPRO orange protein stain of DSP crosslinked samples. Samples without tetracycline but with DSP and DTT (negative control), and with tetracycline, DSP and DTT were sent for LC MS/MS. LMF1 complexes are labeled with asterisks and an arrow points to LMF1  F). SYPRO orange stained gel of pBPA samples. An arrow points to LMF1 \n",
      ": siRNA knockdown of ERp44, ERp72, ERdj5, UGGT1, UGGT2 and TRX decreases LPL secretion.Figure 2 A-B). HEK293 cells stably expressing LPL-V5 or PL-V5 were transfected with scrambled negative control siRNA (NC) or siRNA against LMF1-interacting partners. Media fractions were probed for the lipase V5 tags, and lysate fractions were probed for V5 and the LMF1 interacting-partners. GAPDH is a loading control siRNA knockdowns were performed in triplicate, and lipase secretion was quantified relative to NC siRNA. Error bars indicate the standard error. Quantification of the percent of each protein remaining from three independent experiments. Error bars indicate the standard error.  E) HEK293 cells stably expressing LPL-V5 or PL-V5 were transfected with NC siRNA or siRNA against TRX. Lipase and TRX levels were tested as for Fig 1A  F) Txnip overexpression in HEK293 cells stably expressing LPL-V5 or PL-V5 hinders LPL, but not PL, secretion \n",
      " LPL forms disulfide-bonded aggregates when LMF1 is missing.Figure 3 A). A Western blot against the C-terminal V5 tags of both lipases shows that LPL, but not PL, forms intermolecular disulfide bonds in cells. Lipase monomers are marked with a single arrow, and aggregates are marked with double and triple arrows. The LPL aggregates are not present when samples are treated with DTT. There is no PL in the pellet fraction because PL does not aggregate  B). A Western blot of a non-reducing gel of the lysate and pellet fraction of LPL grown in cells lacking LMF1 (HEK293∆LMF1), normal HEK293 cells or cells with additional LMF1 (+LMF1). LPL monomers and aggregates are marked as above  C). Co-translational LPL folding was carried out in the presence of SP HEK293 or HEK293∆LMF1 cells. At the indicated time points, AMS was added to reactions to differentiate reduced (R) and oxidized (O) LPL. The position of the stacker layer is indicated The amount of reduced/total LPL intensity was quantified. Each point is the average of three independent experiments, and error bars represent the standard error. Significance was determined by a 2-tailed student's t-test. the amount of oxidized/total LPL (E) and the intensity of the stacker/total LPL (F) was quantified. Each point is the average of three independent experiments, and error bars represent the standard error. Significance was determined by a 2-tailed student's t-test  G and H). Data from (D-F) was combined to show the trends in LPL folding over time for reactions using HEK293 or HEK293∆LMF1 cells. Each point is the average of three independent experiments, and error bars represent the standard deviation \n",
      ": Conserved cysteines in LMF1 are important to LPL secretion A).Figure 5). The LMF1 protein sequence from multiple species was aligned using Clustal Omega (Sievers et al., 2011) ). Schematic of the membrane topology of LMF1 showing the location of all the cysteines. ). Secretion of LPL-V5 from HEK293∆LMF1 cells transiently transfected with LMF1 bearing the indicated cysteine to alanine mutations. Lipase levels in the lysate and pellet, and LMF1 levels in the pellet (α-HIS), are also probed. GAPDH serves as a loading control. \n",
      ": Analysis of LMF1's redox-dependent interactions.Figure 6α-His Western blot to detect the redox state of LMF1. In the first two lanes samples were untreated and resolved in reducing or non-reducing loading dye. The next 4 lanes were subject to the indicated treatments prior to cell lysis then treated with AMS prior to non-reducing SDS-PAGE. α-His Western blot to detect acid trapped partners of LMF1. Samples were TCA precipitated, treated with NEM, and LMF1 was purified via its C-terminal His tag. Samples were resolved by reducing or nonreducing SDS-PAGE. The lanes marked -LMF1 are HEK293∆LMF1 cells included as a negative control. Samples were TCA precipitated, treated with NEM, and LMF1 was purified via its C-terminal His tag. Samples were resolved by reducing or nonreducing SDS-PAGE. The lanes marked -LMF1 are HEK293∆LMF1 cells included as a negative control. Samples were probed with an antibody for TRX. \n",
      ": LMF1 contributes to redox homeostasis in the ER.Figure 4Differential sensitivity of HEK293 cells containing and lacking LMF1 (A) to drugs that perturb cellular redox homeostasis. Cells were treated with BSO, DTT, or tunicamycin as described in materials and methods. Treated cell and untreated cells were counted, and the fraction of surviving cells was calculated. Three wells were counted and averaged for each data point, and three independent trials were carried out per condition. Significance was determined by a 2-tailed student's t-test. Differential sensitivity of cld/cld and wt/cld MEFs (B) to drugs that perturb cellular redox homeostasis. Cells were treated with BSO, DTT, or tunicamycin as described in materials and methods. Treated cell and untreated cells were counted, and the fraction of surviving cells was calculated. Three wells were counted and averaged for each data point, and three independent trials were carried out per condition. Significance was determined by a 2-tailed student's t-test.  C). The redox sensor ERroGFP-S4 shows that HEK293∆LMF1 cells have a more oxidized ER than HEK293 and HEK293+LMF1 cells both untreated and when stressed by expression of LPL-mCherry. A two-tailed student's t-test was used to calculate the significance of the difference in the ratio of oxidized/reduced GFP fluorescence intensity between cell types. Each data point is one of three spots measured per cell, 17 cells were measured per condition. Significance was determined by a 2-tailed student's t-test \n",
      ": Fibronectin and LDLR depend on LMF1 for secretion.Figure 7Bottom, center and top slices from a z-stack of a cld/wt or cld/cld MEF cell. YPet-tagged fibronectin is in green and mCherry-Sec61β, an ER marker, is in red. Note fibronectin assembly into the extracellular matrix in cld/wt cells. cld/wt cells had significantly higher amounts of fibronectin outside of the ER than cld/cld cells by a two-tailed student's t-test. Bottom, center and top slices show that LDLR is retained in the ER in cells HEK293∆LMF1 cells. LDLR-GFP is in green and mCherry-Sec61β is in red. Note the clumping of LDLR in the ER in HEK293∆LMF1 cells. The ratio of LDLR within the ER/total LDLR was significantly higher for HEK293∆LMF1 cells than HEK293 or HEK293+LMF1 cells by a 2-tailed student's t-test. \n",
      ".Figure 1 IDR-Screen consists of a modular set of stages that can broadly be grouped into the experimental and computational phases. A library of random or designed sequences is inserted into a genome, expressed as a part of a protein that is used for selection (survival or other readouts such as fluorescence). The library is then screened to discover sequences that are functional/non-functional based on the designed assay to infer functionality. Upon data processing, this dataset of experimentally validated functional and non-functional sequences are analyzed to learn the rules using Machine Learning (ML) approaches \n",
      ".Figure 3 Schematic describing the sequence space explored by the random library (left). Table listing the top 10 most important features. The relative feature importance is given as relative percentages in the last four columns. The size of the circles is scaled per method (lasso, ridge, xgboost, stacked). The direction column denotes the direction of enrichment of the given feature for functional sequences compared to non-functional sequences (up, positive direction and down, negative direction) \n",
      ".Figure 2  A. Enrichment and depletion of different amino acids in the random library (log2 of frequencies of functional over non-functional sequences)    B   In the boxplots, the central line shows the median. Statistical significance was assessed using Wilcoxon test. Boxplots of the distribution of the values of: B. length C. pI D. hydrophobicity E. disorder content and F. helicity for sequences that are functional (green) and non-functional (red  .-G Statistical significance was assessed using Wilcoxon test  G. Enrichment of 9aa TAD motif in functional vs. non-function sequences; ratio of with to without 9 aa TAD in functional to non-functional sequences ((219/520)/(13384/50001)  \n",
      ".Figure 4 Heatmap of the tolerance to amino acid substitutions in WT transactivation domain sequences. The tolerance of a mutation is defined as the fraction of functional sequences over all the sequences when a specific substitution was performed. The columns (amino acid in a WT TAD that is substituted) are ordered according to decreasing tolerance (from left to right) and the rows (amino acid into which a residue in the WT TAD is substituted for) are ordered according to decreasing tolerance (from bottom to top). The cells are colored on a green to red gradient for high to low tolerance respectively. Empty tiles represent data points not present in the library  Schematic describing the sequence space explored by the design library (left). Table listing the top 10 most important features. The relative feature importance is given as relative percentages in the last four columns. The size of the circles is scaled per method (lasso, ridge, xgboost, stacked). The direction column denotes the direction of enrichment of the given feature for functional sequences compared to non-functional sequences (up, positive direction and down, negative direction) \n",
      ".Figure 6 Spot assay of designed TAD constructs . The spot assay was performed at 30ºC (left) and during heat shock at 37ºC (right panel) \n",
      ".Figure 5 Schematic describing the sequence space explored by the combined library (left). Table listing the top 10 most important features. The relative feature importance is given as relative percentages in the last four columns. The size of the circles is scaled per method (lasso, ridge, xgboost, stacked). The direction column denotes the direction of enrichment of the given feature for functional sequences compared to non-functional sequences (up, positive direction and down, negative direction) \n",
      ".Figure 7 Transcription factors (TFs) interact with DNA via the DNA binding domain (DBD, blue triangle) and to their interaction partners (grey) via their transactivation domains (TAD, green rectangle). The enrichment for negatively charged residues leads to a local extended conformation of the TAD via intra-chain repulsions, providing the appropriate context for aromatic residues to be exposed and to bind to their interaction partners in hydrophobic binding pocket (circular inset). The aromatic residues could fit the pocket in a stochastic manner, binding in different configurations. This would result in the formation of a \"fuzzy\" complex. In this case, the negative charges could furthermore contribute to the affinity of binding given that the TAD-interaction surfaces often expose positively charged patches. The absence of positively charged residues, compositional bias and particular spacing of negatively charged and aromatic residues could hence be considered as giving rise to a collection of short mini motifs that collectively contributes to TAD functionality \n",
      ".Figure 2Scatter plot elucidating fold change (FC) in gene expression between samples treated with JNKi compared to vehicle (Y-axis) and expression of constitutively active JNK (MKK7-JNK) versus vector control (X-axis) from biological triplicates. Representative examples of genes that are specifically induced by JNK in metastatic breast cancer cells are highlighted in red. Biological functions regulated by the JNK pathway in MDA231-LM2 cells determined by gene ontology using DAVID analysis. False discovery rate (FDR) < 0.1. BP, biological process; CC, cellular compartment; BH, Benjamini-Hochberg. GSEA graphs showing (C) enrichment of a mammary stem cell signature (Pece et al, 2010) in MDA231-LM2 cancer cells expressing MKK7-JNK NES, normalized enrichment score. GSEA graphs showing under-representation of the same mammary stem cell signature in cancer cells treated with JNKi. NES, normalized enrichment score. GSEA showing significant enrichment of four distinct signatures of mammary stem cells within data sets from MDA231-LM2 cells either expressing MKK7-JNK or treated with JNKi. Mammary stem cell signatures are indicated as follows: a, up in mammary stem cells b, up in fetal mammary stem cells ; c, up in adult mammary stem cells and d, up in mammary repopulating units Heat map of the JNK signature composed of 68 genes that are induced by expression of MKK7-JNK and repressed by JNKi, Proportion of JNK signature-positive tumors in breast cancer patients from the METABRIC data set classified according to PAM50 intrinsic biological subtypes JNK-S, JNK signature as in panel F. Subtype patient numbers: Luminal, n = 1213; HER2, n = 240; Basal-like, n = 331. \n",
      ".Figure 1Kaplan-Meier analysis of overall survival in breast cancer patients with recurrent lethal disease, classified according to p-JNK expression. p-JNK was determined by immunostaining of breast cancer samples in a tissue microarray; p-JNK low, n = 31; p-JNK high, n = 10. P value and hazard ratio (HR) were calculated by log-rank test. Immunostaining of p-c-Jun (arrows) in patient-matched breast tumor and lung metastasis. Scale bar, 50 μm. Quantification of p-c-Jun positive (p-c-Jun+) cells per tumor area in matched primary tumors and lung metastases. Boxes depict median from 12 patients with upper and lower quartiles. Quantification of p-c-Jun positive (p-c-Jun+) cancer cells within mammary tumors and matched metastases in mice injected with MDA231-LM2 cells. Immunofluorescence analysis of p-c-Jun in micrometastatic (day 7) and macrometastatic (day 21) nodules in a xenograft mouse model injected intravenously with MDA231-LM2 cells. DAPI was used to stain cell nuclei, and cancer cells express GFP as a marker. Scale bar, 50 μm. Quantification of p-c-Jun+ cancer cells Experimental setup to address the role of JNK signaling in mammary tumor progression and metastasis. MDA231-LM2 or SUM159-LM1 cells were implanted bilaterally into the fourth mammary fat pads of NSG mice. In vivo treatment with the JNK inhibitor CC-401 (JNKi) commenced at day 5 post implantation and was repeated every three days thereafter until analysis. Mammary tumor growth curves from mice described in panel G. Each value represents the mean ± SEM. For MDA231-LM2, n = 12 tumors in 6 mice for each experimental group. For SUM159-LM1, n = 16 tumors in 8 mice for each experimental group. Lung metastatic burden in MDA231-LM2 or SUM159-LM1 tumor-bearing mice treated with JNKi. Quantification of lung metastases. FOV, field of view; LS, lung section. Each value represents the mean ± SEM. For metastatic foci per FOV, 8-10 fields were quantified per mouse. MDA231-LM2 tumor-bearing mice: n = 6 mice per group. SUM159-LM1 tumor-bearing mice, vehicle n = 8 mice; JNKi n = 7 mice. Lung metastatic burden in MDA231-LM2 or SUM159-LM1 tumor-bearing mice treated with JNKi. Representative histology examples of metastasis in lungs, detected by expression of human vimentin. Scale bar, 100 μm. Experimental setup to determine lung colonization of in vivo vehicle-treated and JNK-inhibited MDA231-LM2 cancer cells after intravenous injection. In vivo treatment with JNKi commenced at the time of implantation and was repeated every 3 days thereafter until analysis at day 17. Metastatic burden determined by photon flux in lungs of MDA231-LM2-injected NSG mice treated with vehicle and JNKi. Boxes show the median with upper and lower quartiles. Whiskers represent minimum and maximum values. Vehicle control, n = 8; JNKi-treated, n = 8. Examples of bioluminescence in mice \n",
      ".Figure 3Gene expression profile of MDA231-LM2 cells grown as oncospheres or as a monolayer. Examples of genes that are both upregulated in spheres and induced by JNK signaling are highlighted in red. GSEA graph showing enrichment of JNK signature in MDA231-LM2 cancer cells grown as oncospheres. NES, normalized enrichment score. Immunoblot of phosphorylated JNK (p-JNK), JNK, phosphorylated c-Jun (p-c-Jun) and c-Jun in oncospheres from SUM159-LM1 cells. Tubulin was included as a loading control. GSEA of a basal-like breast cancer signature induced in MDA231-LM2 oncospheres. Normalized enrichment score (GSEA) of mammary stem cell signatures in oncospheres from MDA231-LM2 cells. Generation of oncospheres in SUM159 cells expressing constitutively active JNK or inactive mutated JNK. Values are mean from 10 wells per group ± SD. Oncosphere formation in SUM159-LM1 cells following treatment with JNKi or vehicle control. Values are mean from 10 wells per condition ± SD. Representative images of spheres Oncosphere formation in JNKi-treated cancer cells from breast cancer patients (pleural effusions or ascites). (I) Quantification of oncospheres per well in 4 different patients. Values are mean from 10 wells ± SD. (J) Representative oncospheres from patient samples. Scale bar, 50 μm. Subcutaneous injection of limiting numbers of JNKi-treated MDA231-LM2 cells in NSG mice. Analysis of tumor initiation ability of vehicle- and JNKi-treated cells from experiment described in panel (K); n = 3 mice per group, two implantations per mouse. Matrigel invasion assay of MDA231-LM2 cells treated with JNKi. Left, examples of invading cancer cells stained with crystal violet (arrows). Scale bar, 50 μm. Right, quantification of invading cells per field of view (FOV) showing mean from 8 fields ± SEM. \n",
      ".Figure 7Experimental scheme to determine whether JNK signaling promotes resistance to chemotherapy. MDA231-LM2 or SUM159-LM1 cells were implanted bilaterally into the fourth mammary fat pads of NSG mice. Treatment with JNKi was initiated at day 5 post implantation and repeated every three days; paclitaxel chemotherapy was started at day 10 post implantation and repeated every five days. Mammary tumor growth curves in NSG mice implanted with MDA231-LM2 or SUM159-LM1 cells and treated with paclitaxel as single agent or in combination with JNKi. Each value represents the mean ± SEM. MDA231-LM2: n = 12 tumors per group. SUM159-LM1: vehicle, JNKi and PAX n = 12 tumors per group; PAX + JNKi n = 10 tumors. **P < 0.01, ***P < 0.001. Metastatic burden in lungs of MDA231-LM2 and SUM159-LM1 tumor-bearing mice treated with paclitaxel as single therapy or in combination with JNKi. Each value represents the mean ± SEM. MDA231-LM2 tumor-bearing mice: n = 6 mice per group. SUM159-LM1 tumor-bearing mice: vehicle, JNKi and PAX n = 6 mice per group; PAX+ JNKi n = 5. Ten fields were quantified per lung section. FOV, field of view, LS, lung section. PAX, paclitaxel. Representative histological images of metastatic foci (arrows) in lungs of the MDA231-LM2-implanted mice described in panel C, stained for human vimentin expression in differentially treated groups. Scale bar, 100 μm. Apoptosis was determined by quantification of TUNEL-positive cells per field of view (FOV) in MDA231-LM2 tumors (from the animals described in (C)) treated with JNKi and/or PAX. Values are means from 6 mice per condition ± SEM. GSEA showing enrichment of the JNK signature in tumor samples from breast cancer patients from the TOP trial (GSE16446), stratified according to pathological response to neoadjuvant chemotherapy. NES, normalized enrichment score. pCR, pathological complete response. Kaplan-Meier analyses of chemotherapy-treated breast cancer patients associating JNK signature (JNK-S) with metastasis-free survival (TOP trial data set, JNK-S low n = 54 patients; JNK-S high n = 53 patients (G)) or overall survival (METABRIC data set, JNK-S low n = 135 patients; JNK-S high n = 135 patients (H)). HR, hazard ratio. P values were determined by log-rank test. Model illustrating the link between stress-induced ECM, stem cell and wound healing network and metastatic progression and therapy resistance in breast cancer. \n",
      ".Figure 5Gene expression profile of MDA231-LM2 cells treated with paclitaxel (PAX) for 48 hours. Highlighted are examples of JNK signature genes that are induced by PAX treatment. Venn diagram showing overlap between genes that were significantly (P < 0.05) induced more than 1.5-fold in response to paclitaxel treatment of MDA231-LM2 cells and genes that were significantly (P < 0.05) induced in MDA231-LM2 cells after formation of oncospheres. Significant fold changes were determined by two-sample empirical Bayer tests with Benjamini-Hochberg correction for multiple testing. PAX. GSEA normalized enrichment scores of four independent mammary stem cell signatures that arise in response to paclitaxel treatment of MDA231-LM2 cells. Immunofluorescence analysis of p-c-Jun+ cancer cells in lung metastases from mice injected intravenously with MDA231-LM2 cells and treated with PAX at week 3 post injection. Lungs were harvested 72h after PAX treatment. Cancer cells express GFP as a marker. DAPI was used for nuclear staining. Scale bar, 50 µm. Quantification of cancer cells expressing p-c-Jun in panel D. Values are mean from three mice ± SEM. Nine metastatic foci were analyzed per mouse. Western blot analyzing phosphorylated JNK (p-JNK), JNK, phosphorylated c-Jun (p-c-Jun) and c-Jun in MDA231-LM2 cells treated with PAX for 48 hours. Tubulin was used as a loading control. SPP1 and TNC expression in SUM159-LM1 cells treated with increasing concentrations of PAX qPCR triplicates ± SD. SPP1 and TNC expression in SUM159-LM1 cells treated with increasing concentrations of doxorubicin (DOXO). qPCR triplicates ± SD. SPP1 and TNC expression in SUM159-LM1 cancer cells treated with 4 μM 5-Fluorouracil or 4 μM methotrexate. Cells were treated for 48 hours. qPCR triplicates ± SD. SPP1 and TNC expression in SUM159-LM1 cells treated with paclitaxel and JNKi. qPCR triplicates ± SD. SPP1 and TNC repression by JNKi, P < 0.05 for both vehicle and PAX-treated cells. \n",
      ".Figure 6Experimental treatment schedule of mice after cancer cell implantation in the mammary gland to address the role of SPP1 in resistance to PAX. Growth curves of mammary tumors in vehicle- or PAX-treated Spp1+/- mice implanted with the indicated cells expressing control shRNA, and Spp1-/- mice implanted with the indicated cells expressing SPP1 shRNA. MDA231-LM2: control, PAX and SPP1 def. + PAX n = 16 tumors per group, SPP1 def. n = 14 tumors. SUM159-LM1: control, PAX and SPP1 def. + PAX n = 16 tumors per group, SPP1 def. n = 15 tumors. Quantification of lung metastasis from the animals described in panels A and B. The number of metastatic foci per field of view (FOV) or per lung section (LS) was determined. MDA231-LM2 tumor-bearing mice: control, PAX and SPP1 def. + PAX n = 8 mice per group, SPP1 def. n = 7 mice. SUM159-LM1 tumor-bearing mice: control, PAX and SPP1 def. + PAX n = 8 mice per group, SPP1 def. n = 7 mice. Tumor weight in mice at day 40 after implantation of MDA231-LM2 breast cancer cells in a control or SPP1 deficient setting undergoing combination treatment with doxorubicin (Adriamycin) and cyclophosphamide (AC regimen). Control, AC and SPP1 def. n = 16 tumors per group, SPP1 def. + AC n = 14 tumors. Lung metastasis was quantified in mice that were implanted with MDA231-LM2 tumors and subjected to AC chemotherapy. Each value represents the mean ± SEM; n = 5 mice per group. Ten random fields per lung section were analyzed for percentage of metastatic area. Representative histological images of metastatic nodules in lungs of mice from the experiment described in panel E. Metastases were determined by expression of human vimentin (arrows). Scale bar, 100 µm. Treatment schedule after cancer cell implantation to address the role of TNC in PAX resistance in a xenograft mouse model for breast cancer metastasis to lungs. Mice were implanted with MDA231-LM2 cells expressing either control shRNA (control) or a TNC shRNA (shTNC), treated with paclitaxel or vehicle. Mammary tumor growth curves in PAX-treated NSG mice bearing control or TNC knockdown MDA231-LM2 tumors. Values are means ± SEM, n = 16 tumors per group. **** P < 0.0001. Lung metastases quantified in tumor bearing mice Representative histology of human vimentin-positive lung metastases (arrows) \n",
      ".Figure 4SPP1 and TNC expression in the indicated breast cancer cells treated with increasing concentrations of JNKi. SPP1 and TNC expression in MDA231-LM2 cells transduced with pLVX-puro lentivirus without a transgene (control) or expressing constitutively-active JNK (MKK7-JNK) or mutated inactive JNK (MKK7-JNK(mut)). SPP1 and TNC expression in pleural effusion (1 and 2) and ascites (3 and 4) samples from four different patients treated with JNKi (4 μM, 48h) in culture. Values are mean ± SD from triplicates. The results from all four patient samples were included to calculate significance of JNKi on gene expression. SPP1, P = 0.0079; TNC, P = 0.0002. Analysis of SPP1 and TNC mRNA levels in SUM159-LM1 cells grown as monolayer or spheres and treated with vehicle or JNKi. For panels A-D, values represent the mean of qPCR triplicates ± SD. ChIP analysis of c-Jun binding to sequences in SPP1 or TNC promoters. (E) Diagrams of SPP1 and TNC promoters showing positions of primer pairs, flanking c-Jun/AP-1 consensus sites, that were used for qPCR following ChIP with an antibody against c-Jun. ChIP analysis of c-Jun binding to sequences in SPP1 or TNC promoters. Analysis of SPP1 and TNC promoter fragments pulled down by c-Jun and IgG antibodies. Values represent the mean of qPCR triplicates ± SD. Correlation analysis of SPP1 and TNC with JUN expression in dissected metastatic nodules from breast cancer patients (GSE14020). N = 65 metastasis samples. Lung metastasis in Spp1+/- mice implanted with MDA231-LM2 cells expressing control shRNA (control), and Spp1-/- mice implanted with MDA231-LM2 cells independently expressing one of two different SPP1 shRNAs (SPP1-deficient). Ex vivo lung luminescence was analyzed, panel H. Boxes show the median value with upper and lower quartiles. Whiskers represent minimum and maximum values. Control n = 16 mice, SPP1 deficient (1) n = 14 mice and SPP1 deficient (2) n = 15 mice. P values were determined by a two-tailed Mann-Whitney test. (I) Representative histological images of metastatic foci (top) and ex vivo lung bioluminescence (bottom). In histological samples, immunohistochemistry was used to determine metastatic nodules based on expression of human vimentin (arrows). Scale bar, 100 µm. SUM159-LM1 cells expressing either control shRNA (control), or one of two different TNC shRNAs (shTNC 1 and 2), were injected intravenously into NSG mice. (J) Metastatic colonization was quantified by lung bioluminescence. Control and shTNC (1) n = 8 mice per group, shTNC (2) n = 7 mice. P value was determined by a two-tailed Mann-Whitney test. (K) Representative images of metastatic foci (top) and ex vivo lung bioluminescence (bottom). Arrows denote vimentin-expressing cancer cells in metastases. Kaplan-Meier analysis of breast cancer patients (combined GSE2603 and GSE5327 datasets) showing a link between high SPP1 and TNC expression and poor lung metastasis-free survival. Patient samples, SPP1 and TNC low n = 70; SPP1 and TNC high n = 70. HR, Hazard ratio. \n",
      ".Figure 2A. Crossover frequency normalised by the number of F2 individuals, and SNP density in 100 kilobase (kb) adjacent windows were calculated for each population and ranked into percentiles according to SNP density. Centromeric regions were excluded from analysis (Underwood et al, 2018). The Spearman's rank correlation coefficient (r) between SNP density and crossover frequency is printed inset. Trend lines were fitted in ggplot using a generalized additive model (GAM) with the formula y~poly(x,2). B. Generalized Linear Model (GLM) plots showing observed Col/Ler crossovers per megabase (red) for SNP intervals grouped into hexiles or quintiles, by increasing values (left to right along the x-axis) of a given explanatory variable. Data were modeled with the glm2 function in R, using the binomial family with the logit link function. The formula used for the model was Crossovers ~ (SPO11-1 + nucleosomes + H3K4me3 + DNA methylation + SNPs/kb + width)^2. Predicted crossovers per megabase for each group of intervals were derived by sampling from the binomial distribution based on the probabilities of intervals within each group overlapping a crossover. The black box plots represent 100 samples, where the central band represents the median, the box represents the interquartile range (IQR), and the whiskers represent the minimum and maximum values that are no more than 1.5 times the IQR from the box. The final stepAIC-selected model was: \n",
      ".Figure 1A. Histograms of crossovers (blue) per F2 individual in the indicated populations, with the Poisson expectation plotted in red. Mean values are indicated by the black dotted lines. The genome average SNPs/kb for each cross are printed above the plots. B. Crossovers per 200 kb per F2 plotted along the Arabidopsis chromosomes. Mean values are shown by horizontal dashed lines. SNPs per 200 kb are plotted and shaded in colour (Col×Ws (purple), Col×Bur (Lawrence et al, 2019) (dark green), Col×Ct (light blue), Col×Ler F2 populations with 17,077 (red) and 1,840 (blue) crossovers (Serra et al, 2018b; Rowan et al, 2019) and Col×CLC (pink/light green, where pink indicates Cvi SNPs and green indicates Ler SNPs)). The positions of telomeres (TEL) and centromeres (CEN) are indicated. The number of crossovers analysed is printed inset. C. Data as for B, but analyzing crossovers (lines) or SNPs (shading) along proportionally scaled chromosome arms, orientated from telomere (TEL) to centromere (CEN). \n",
      ".Figure 3A. SNPs/kb in physical windows of increasing size (kb) around crossover midpoints (red), or for matched randomly chosen positions (grey). Printed above the plot for each window are circles coloured green if crossover SNPs/kb values are significantly different to random (P<0.05), or red if not (P>0.05) (Bonferroni adjusted t-tests). The population (Col×Ws, Col×Bur (Lawrence et al, 2019), Col×Ct, Col×Ler (Serra et al, 2018b; Rowan et al, 2019) and Col×CLC) is printed above each plot, and the number of positions analysed is printed inset. B. As for A, but analyzing the indicated Col/Ler transition and transversion polymorphisms around Col×Ler crossovers (Rowan et al, 2019) (red), or the same number of random positions (grey). C. As for A, but analysing Col/Ler SNPs/kb around crossovers from wild type, HEI10-OE, recq4a recq4b or HEI10-OE recq4a recq4b (Serra et al, 2018b). All populations were generated from Col×Ler hybrids. Printed beneath are the estimated average numbers of Class I and Class II crossovers per meiosis for each genotype, measured via genotyping-by-sequencing (Serra et al, 2018b; Ziolkowski et al, 2017). To the right are plots showing the significance of SNPs/kb differences between crossover sets, across the windows tested (Bonferroni adjusted t-tests). \n",
      ".Figure 4A. Histograms of observed crossovers per F2 individual in the indicated wild type and msh2 populations, from Col×Ct, Col×Ler and Col×CLC crosses (blue). The Poisson expectation is plotted in red. Data from HEI10-OE, recq4a recq4b and HEI10-OE recq4a recq4b F2 populations are also shown for comparison (Serra et al, 2018b; Ziolkowski et al, 2017). Mean values are indicated by black dotted lines. B. Crossovers per 200 kb per F2 plotted along the Arabidopsis chromosomes, with mean values shown by horizontal dashed lines. Data are shown for wild type (red) and msh2 (blue) crossovers generated from Col×Ct, Col×Ler and Col×CLC hybrids. SNPs per 200 kb are shaded in colour (blue=Col/Ct, green=Col/Ler, pink=Col/Cvi). The positions of telomeres (TEL) and centromeres (CEN) are labeled. The number of crossovers analysed is printed inset. C. Data as for B, but analyzing crossovers in wild type (red) and msh2 (blue), or SNPs along proportionally scaled chromosome arms orientated from telomeres (TEL) to centromeres (CEN). Crossover data from HEI10-OE, recq4a recq4b and HEI10-OE recq4a recq4b Col×Ler F2 populations are also shown (Serra et al, 2018b; Ziolkowski et al, 2017). \n",
      ".Figure 5A. Histograms showing the mean distance (megabases, Mb) of observed double crossovers (DCO, red vertical line), compared to 2,000 matched sets of randomly generated distances (RAN, black). The mean distance of the random sets is shown in by the vertical grey line. B. SNPs/kb values were calculated in physical windows of increasing size (kb) around crossover midpoints in wild type (red) and msh2 (blue). The same values were calculated for matched randomly chosen positions (grey). Printed above the plot for each window are circles that are coloured green if crossover SNPs/kb values are significantly different to random (P<0.05), or red if not (P>0.05) (Bonferroni adjusted t-tests). The population analysed (Col×Ct, Col×Ler, Col×CLC) and its genome average SNP frequency (SNPs/kb) is printed above the plots and the number of crossover positions analysed is printed inset. Beneath are plots showing the significance of SNPs/kb differences around crossovers between the indicated genotypes. \n",
      ".Figure 7A. Crossover frequency (black) and SNP density (purple) per 200 kb measured from the Col×Ct F2 population. The position of the centromere (CEN, vertical dashed line) and 420 FTL transgenes (red and green vertical lines) used to measure crossover frequency are indicated. B. Plots of chromosome 3 showing positions of Col/Col (black) or Col/Ct (purple) homozygous or heterozygous genotypes at the indicated marker positions (circles). The positions of the 420 FTL transgenes are indicated (red and green vertical lines). C. 420 crossover frequency (cM) in the HOM-HOM, HET-HET, HET-HOM, HOM-HET genotypes shown in B, in either wild type or msh2. Values for replicate individuals are plotted, in addition to the mean (red). D. As for C, but comparing genotypes in wild type or HEI10-OE backgrounds. Values for replicate individuals are plotted, in addition to the mean (red). \n",
      ".Figure 8A. Diagram representing interhomolog strand invasion between red and blue homologous chromosomes during meiotic prophase I. The resulting displacement loop contains a mismatch (bulge), which is recognized by a MSH2 MutS heterodimers. This causes recruitment or activation of the Class I (ZMM) repair pathway and increases the chance of crossover repair at, or in proximity to, the mismatched intermediate. B. Diagram representing the same scenario as in A., but here MSH2 dependent recognition of the mismatch inhibits progression of recombination. This triggers feedback signaling that recruits SPO11-1 complexes to this region and increases meiotic DSB and crossover frequency. C. Comparison of the budding yeast and Arabidopsis physical maps (blue) and genetic maps (red) (Serra et al, 2018b; Mancera et al, 2008), shown in megabases and centiMorgans, respectively. D. Plots of SNP density per 10 kb along the chromosomes for Arabidopsis Col×Ler and budding yeast S288c×SK1 crosses (Cooper et al, 2018). Telomere positions are shown by vertical lines, and centromeres by vertical dashed lines. Centromeres are also labeled above the plots in red, as 'CEN' or 'C'. \n",
      ".Figure 6A. Representative DAPI-stained spreads of pachytene, diakinesis, metaphase-I, dyad and tetrad meiotic stages, for wild type and msh2, in Col×Ler or Col×CLC hybrid backgrounds. B. Male meiocytes immunostained for MSH2 (red) and ASY1 (green), and stained for DAPI (blue) in wild type (Col) or msh2. C. As for B, but immunostaining wild type male meiocytes for MSH2 (red), MSH4 (green) and staining chromatin with DAPI (blue). \n",
      ".Figure 3Fluorescence spectroscopy analyses of the interaction between β-catenin and KRAS using purified recombinant proteins, intact β-catenin and KRAS. Molar ratios of β-catenin and KRAS are presented. A.U., absorbance units Nuclear magnetic resonance (NMR) titration for 15N-labeled KRAS (full-length, 1-188) with β-catenin (C-terminal fragment, 661-781). The titration points are indicated in red (1:0), green (1:5), and blue (1:10). A gradual perturbation in chemical shift is indicated by the arrow A molecular docking model for NMR spectra data represented by a ribbon diagram (left panels of C and D) and structure-based surface area (right panels of C and D) of KRAS (C; chemical shift residues, red; phosphorylation sites T144/T148, green; GTP/GDP loading site, blue). The α-interface area is presented as yellow (D) E The interface (red circle) for interaction between C-terminal domain of β-catenin (helix C) and α-interface of KRAS represented by a ribbon diagram \n",
      ".Figure 4Effects of KYA1797K on the interaction and phosphorylation of β-catenin and pan-RAS. WT- or MT-HCT116 cells were treated with 20 µM of KYA1797K in the presence of MG132. WCLs were immunoprecipitated with pan-RAS antibody and subjected to IB analyses Effects of Wnt3a-CM treatment on the formation of β-catenin destruction complex and phosphorylation of β-catenin and RAS in HEK293 cells treated with L-CM or Wnt3a-CM in the presence of MG132 for 8 hr. WCLs were immunoprecipitated with β-catenin or pan-RAS antibodies and subjected to IB analyses IB analyses were performed to detect β-catenin and pan-RAS in DLD1 cells cotransfected with FLAG-β-catenin-WT or -MT and Myc-KRAS-WT or -2A for 24 hr and then treated or transfected with 20 µM of KYA1797K or HA-GSK3β for 24 hr, respectively \n",
      ".Figure 1Immunoblot (IB) analyses of the levels of β-catenin and RAS in DLD1, WT-HCT116, MT-HCT116, HepG2, and SW48 cells transfected with Myc-axin for 36 hr Effects of β-catenin knock-down on polyubiquitination-dependent degradation of RAS. IB analyses of MT-HCT116 cells transfected with control or two independent β-catenin siRNAs for 48 hr Polyubiquitination assays of MT-HCT116 cells cotransfected with two independent β-catenin siRNAs and HA-ubiquitin (Ub) for 72 hr in the presence of the proteasomal inhibitor MG132, and WCLs were immunoprecipitated with an anti-pan-RAS antibody IB analyses of HEK293 cells transfected with WT- or S33Y-Flag-β-catenin with or without HA-GSK3β for 36 hr IB analyses of HEK293 cells transfected with β-catenin siRNA or control siRNA for 48 h and then treated with L-cell-conditioned medium (L-CM) or Wnt3a-CM for 8 hr Effects of KYA1797K or XAV939 on degradation of β-catenin and RAS in isogenic HCT116 cells harboring WT- or MT-CTNNB1. IB analyses of β-catenin and RAS levels in WT- or MT-HCT116 cells treated with 5 or 20 µM of KYA1797K or XAV939 for 24 hr ICC analyses of WT- or MT-HCT116 cells treated with 20 µM of KYA1797K for 24 hr. Scale bars, 10 µm For polyubiquitination assays, WT- or MT-HCT116 cells were transfected with FLAG-Ub for 24 hr and then treated with 20 µM of KYA1797K for 24 hr in the presence of MG132. WCLs were immunoprecipitated with β-catenin or pan-RAS antibody and subjected to IB analyses to detect each protein The 3-(4,5-dimethylthiazol-2yl)-2-5-diphenyltetrazolium bromide (MTT) assays were performed for measurements of growth of WT- or MT-HCT116 cells treated with the indicated concentrations of KYA1797K. Data are presented as the mean ± SD (n = 3) \n",
      ".Figure 2Analyses of the interaction between β-catenin and RAS. WCLs of DLD1 cells were immunoprecipitated with β-catenin or pan-RAS antibody and subjected to IB analyses IB analyses of indicated proteins in the samples immunoprecipitated with anti-β-catenin antibody with WCLs of HEK293 cells treated with Wnt3a-CM Left panel: MT-HCT116 cell extracts were fractionated as cytosolic and nuclear portions and subjected to IB analyses, respectively. Right panel: The cytosolic and nuclear fractions were immunoprecipitated with anti-β-catenin antibody and subjected to IB analyses β-Catenin-RAS interaction was visualized by in situ proximity ligation assay in HEK293 cells treated with L-CM or Wnt3a-CM. Scale bars, 20 µm. The proximity ligation signal was quantified in 10 randomly distributed microscopic images that contained > 100 cells In vitro pulldown analyses to identify β-catenin binding region of RAS. Upper panel: Schematic of the HRAS full-length (FL) or two deletion mutants. Lower panel: Purified GST-HRAS-FL, -ΔHVR, or -Δ131-165 proteins were pulled-down with purified His-β-catenin-FL protein and subjected to IB analyses IB analyses of IP with FLAG antibody in WCLs of HEK293 cells cotransfected with FLAG-β-catenin and Myc-HRAS-FL, -ΔHVR, or -Δ131-165 for 32 hr IB analyses of IP with FLAG antibody in WCLs of HEK293 cells cotransfected with FLAG-β-catenin and Myc-HRAS-WT, -2A, or -2E for 32 hr Upper panel: Schematic of the β-catenin-FL or two deletion mutants. Lower panel: Purified His-β-catenin-FL, ΔN86, or ΔC110 proteins were pull-down with purified GST-HRAS-FL protein and subjected to IB analyses IB analyses of IP with Myc antibody with WCLs of HEK293 cells cotransfected with Myc-HRAS and GFP-β-catenin-FL, -ΔN86, or -ΔC110 for 32 hr \n",
      ".Figure 7Left panel: In a cell that expresses degradable WT-β-catenin, the active GSK3β phosphorylates β-catenin at the Ser-33 and Ser-37 residues (number 1), followed by the recruitment of β-TrCP, which leads to polyubiquitination-dependent proteasomal degradation of β-catenin (number 2 and 3). As a second step (II), the released RAS by prior β-catenin degradation is subjected to phosphorylation by GSK3β at Thr-144 and Thr-148 (number 4), recruits β-TrCP (number 5), and is subsequently degraded through proteasome (number 6). Right panel: In a cell that expresses nondegradable MT-β-catenin, RAS protein masked by MT-β-catenin at the phosphorylation sites cannot be phosphorylated by GSK3β and costabilized with β-catenin. Blue stars represent mutation sites (Ser-33 and Ser-37) of β-catenin. Expanded View Figure legend \n",
      ": Human CEACAM1 binds to group B Streptococcus (GBS) via the surface-expressed β protein.Figure 1Binding of recombinant (r)CEACAM1 (CC1)-HIS (10 μg/mL) to (A) a panel of Gram-positive bacteria, namely Enterococcus faecalis, Enterococcus faecium, Group A Streptococcus (GAS), Group B Streptococcus (GBS), Group C Streptococcus (GCS), Group G Streptococcus (GGS), Streptococcus pneumoniae, Staphylococcus aureus, (B) an expanded collection of GBS isolates, with serotype and carriage of bac gene shown. rCC1 binding to live bacteria was quantified by a secondary anti-His-FITC monoclonal antibody (mAb). Mean and standard deviation (SD) values are reported for n = 3 independent experiments. In (B), GBS strains from left to right are A909, BS39, 515, BS22, H36B, BS29, 18RS21, BM110, COH1, SBL3066, NCTC10/84, SB35, SB10, SB20, BS26. Correlation between rCC1-binding capacity and β protein expression in GBS strains, respectively. β protein expression was quantified using rabbit anti-β protein serum and a secondary PE-conjugated goat anti-rabbit-IgG. rCC1 binding to live bacteria was quantified by a secondary anti-His-FITC mAb. Mean binding values from n = 3 independent experiments. Concentration-dependent binding of rCC1-HIS to wild type (WT), ∆bac and ∆bac + pLZ.bac complemented A909 strains. Mean and SD values are reported for n = 3 independent flow cytometry experiments. Pull-down of human rCC1-HIS (30 μg/mL) by GBS strain A909 strain, detected using Western blot analysis. Human rCC1-HIS (30 μg/mL) binding to GBS A909 and A909∆bac strains analyzed by pull-down experiments followed by Western blot analysis. Western blot analysis of rCC1-HIS (30 μg/mL) binding to purified β protein, but not to control GBS surface proteins α and Rib. Panel on the left show staining of proteins after separation by SDS-PAGE. Binding of purified β protein to dynabeads (DB) coated with rCC1 (DB.CC1) or control. Binding of β protein was detected using mouse anti-β serum or normal mouse serum (NMS), and PE-conjugated goat anti-mouse-IgG. (H) shows mean and SD values compiled from n = 3 independent replicates, and (I) shows representative flow cytometry plots using 10 μg/ml rCC1. \n",
      ".Figure 6Effects of KYA1797K and PTD-BBR on RAS degradation IB assay of MT-HCT116 or SW48 cells treated with indicated conditions for 24 and 96 hr, respectively Effects of KYA1797K and PTD-BBR o cell growth MTT assays of MT-HCT116 or SW48 cells treated with indicated conditions for 24 and 96 hr, respectively. Data are presented as the mean ± SD (n = 3) Effects of overexpression of GFP-BBR-WT or -2E mutant on KYA1797K-induced growth and transformation of CRC cells. MT-HCT116 or SW48 cells were transfected with GFP-BBR-WT, -2E, or GFP-Mock for 24 hr and then treated with KYA1797K via indicated conditions MTT assays were performed for measurements of cell growth Effects of overexpression of GFP-BBR-WT or -2E mutant on KYA1797K-induced growth and transformation of CRC cells. MT-HCT116 or SW48 cells were transfected with GFP-BBR-WT, -2E, or GFP-Mock for 24 hr and then treated with KYA1797K via indicated conditions foci formation assay were performed to detect cellular transformation Effects of cotreatments with KYA1797K and PTD-BBR on xenografted tumor growth. MT-HCT116 cells were subcutaneously injected in nude mice with subsequent i.p. injection of vehicle, 20 mg/kg of KYA1797K, or a cotreatment of PTD-BBR (25 mg/kg) and KYA1797K (20 mg/kg) for 20 d. Tumor volumes (upper panel) were measured every 4 days, and tumor weights (lower panel) were measured at the times of sacrifice. Data are presented as the mean ± SD (four mice per group) WCLs prepared from tumor tissues were immunoprecipitated with anti-β-catenin antibody. WCLs or co-IP samples were subjected to IB analyses IHC analyses of tissue sections incubated with the indicated antibodies and then counterstained with DAPI. Scale bar, 20 µm \n",
      ".Figure 5IB analyses to determine the interaction between β-catenin and BBR peptide by in vitro pull-down assays using purified His-β-catenin and GST-BBR proteins IB analyses of IP with FLAG antibody and WCLs from HEK293 cells cotransfected with FLAG-β-catenin and GFP-BBR or -Mock in the presence of proteasome inhibitor, ALLN, for 24 hr Effects of BBR overexpression on KYA1797K-induced RAS degradation. I analyses to detect levels of RAS in MT-HCT116 cells transfected with GFP-Mock or -BBR for 24 hr and then treated with 20 µM of KYA1797K for 24 hr Effects of BBR overexpression on KYA1797K-induced RAS degradation ICC (D; Scale bars, 10 µm) analyses to detect levels of RAS in MT-HCT116 cells transfected with GFP-Mock or -BBR for 24 hr and then treated with 20 µM of KYA1797K for 24 hr MT-HCT116 cells were cotransfected with GFP-Mock or -BBR and FLAG-Ub for 24 hr and then treated with 20 µM of KYA1797K in the presence of MG132 for 8 hr, and WCLs were immunoprecipitated with pan-RAS antibody IB analyses of IP with β-catenin antibody in MT-HCT116 cells transfected with GFP-BBR-WT, -2E, or GFP-Mock in the presence of ALLN for 24 hr IB analyses of MT-HCT116 cells transfected with GFP-Mock, -BBR-WT, or -BBR-2E for 24 hr and then treated with 20 µM of KYA1797K for 24 hr MT-HCT116 cells were transfected with GFP-Mock, -BBR-WT, or -BBR-2E together with FLAG-Ub with or without 20 µM of KYA1797K for 24 hr and MG132 was treated for 8 hr before harvesting cells. WCLs were immunoprecipitated with pan-RAS antibody and subjected to IB analyses \n",
      ": The Ig-like domain of β protein represents a new IgI fold subtype A) Dimer co-crystal structure of β-IgSF (blue) and CEACAM1-N (CC1-N; red).Figure 5Dimer co-crystal structure of β-IgSF (blue) and CEACAM1-N (CC1-N; red). Molecular structure of β-IgSF, in which each β-strand is coloured differently. The 1.5-turn α-helix shown in grey is located between the C strand and the D strand. Illustration of the secondary structure organisation of the V-set, C1-set, C2-set, I1-set, I2-set and I3-set domains. Each secondary structure is composed of varying combinations of the β-strands A, A′, B, C, C′, C′′, D, E, F, G. Disulfide bridges betwen the B strand and F strand are represented as a black dashed lines. The C-set domains lack the A′ and C′′ strands, and are subtyped into C1-set and C2-set based on presence and absence of the D strand, respectively. The I-set domains lack C′′ strand, and are subtyped into C1-set and C2-set based on presence and absence of the D strand, respectively. The IgSF fold from β protein has the I3-set domain, characterized by absence of disulfide bridge, presence of the D strand, and presence of a truncated C strand followed by a 1.5-turn α-helix. Comparison of β-IgI3 (blue) with the most similar folds; the IgI1 domain of MCS-F (turquoise) and the IgI1 domain 1 of ICAM-3 (magenta,). The molecular surface mesh of β-IgI3 residues showing protruding residues located between the C and D strands in blue. \n",
      ".Figure 3A) Representative isothermal titration calorimetry (ITC) binding curves of CEACAM-N domains and β-IgSF duplicates. Experiments were performed using β-IgSF and N domains of (r)CEACAM1 (CC1), CEACAM3 (CC3), CEACAM5 (CC5), CEACAM6 (CC6) and CEACAM8 (CC8). B) Representative flow cytometry plots analyzing the interaction of rCC1, CC3, CC5, CC6 or CC8 (10 μg/ml) with GBS A909. Fluorescence of bacteria was measured by flow cytometry. Representative data from n = 3 independent experiments. \n",
      ": GBS exploit human CEACAM for enhanced adhesion to epithelial cells.Figure 4Adherence of GBS A909 to human CEACAM1 (CC1)-, CEACAM3 (CC3)-, CEACAM5 (CC5)-, CEACAM6 (CC6)- or CEACAM8 (CC8)-expressing, or an empty vector control, HeLa transfectants at MOI of 10 for 30 mins. Mean and SD for n = 4 independent replicates are shown. Confocal imaging of FITC-labelled GBS A909 strain adhesion to human CC1-expressing or empty vector control HeLa transfectants. Cell membranes were stained with AF-647 conjugated wheat germ agglutinin (WGA) (red) and nuclei with DAPI (blue). Scale bar: 26 μM. Adherence of A909 to human CC1-, CC3-, CC5-, CC6- or CC8-expressing, or an empty vector control, CHO transfectants at MOI of 10 for 30 mins. Mean and SD from n = 5 independent replicates is displayed. Adhesion of WT, ∆bac and ∆bac + pLZ.bac complemented A909 strains to human CC1- or CC5-expressing CHO transfectants, or empty vector control cells. Mean and SD from n = 5 independent replicates is displayed. Adhesion of FITC-labelled A909 to human CC1- -expressing, or control, CHO transfectants at an MOI of 10. Cell lines were detached and then incubated with FITC-labelled A909 for 30 mins at 4oC. The percentage of fluorescent cells was calculated for each population. Representative flow cytometry plots are shown (E), and the integrated results displaying the mean and SD from 3 independent replicates are reported in (F). Adhesion of FITC-labelled A909 strains to human CC1- expressing or control CHO transfectants at MOI of 10. Cell lines were detached and then incubated with FITC-labelled A909 strains for 30 mins at 4oC. Mean and SD for each population for n = 6 independent replicates is displayed \n",
      ": The IgSF domain in the β protein of GBS binds to the N-terminal domain of CEACAMs.Figure 2Schematic representation of the β protein (Hedén et al, 1991). The first residue of each domain is indicated. XPZ is a proline-rich repeat region (Areschoug et al, 2002a). Known human ligand binding partners are indicated above each respective domain. IgA binds to a region denoted IgABR. Factor H binds the β75K region, but does not bind via XPZ. Staining of recombinant β protein domains (B6N, IgABR, B6C, β-IgSF and β75KN) after separation by SDS-PAGE. Recombinant proteins were expressed in E. coli. β protein purified from GBS is used as a control. Dotted boxes signal proteins of expected size. Binding of rCC1-His or rSiglec5-Fc to dynabeads (DB) coated with β protein domains. Mean and SD values are reported for n = 2 replicates. Binding of varying concentrations of β-IgSF or human serum albumin (HSA) coupled to streptavidin-PE (SPE), to DB coated with CC1 (DB.CC1) or control protein (DB). Mean and SD values are reported for n = 6 replicates. Structure of the relevant (Siglec-like sialic acid-binding) domain of the highest-scoring template 5KIQ (SrpA binding region), and predicted three-dimensional model of the IgSF domain from the β protein, constructed from the Hhpred alignment using MODELLER.(Webb & Sali, 2016) Inhibition of β-IgSF binding to DB.CC1 by anti-CC1-N mAb (clone CC1/3/5-Sab), but not by anti-CC1-A1B1 mAb (clone B3-17). β-IgSF-biotin was coupled to SPE. Mean and SD values are reported for n = 6 replicates. Concentration-dependent binding of β-IgSF to DB coated with 100 μg/ml CC1 (DB.CC1), CC1-N (DB.CC1-N) or CC1-A1B1A2 (DB.CC1-A1B1A2). Assays utilized proteins coupled to SPE. Mean and SD values are reported for n = 6 replicates. \n",
      ".Figure 6Molecular structure of CEACAM1 (CC1)-N, in which each strand of the IgV structure is coloured differenty. Molecular structure of the β-IgI3 & CC1 N-terminal domain co-complex. Residues that form the β-IgI3-CC1 interface are highlighted. Close-up of the β-IgI3 (light blue) interface with CC1-N (pink), including highlighting of residues important for forming β-IgI3 (blue) & CC1-N (red) interactions. The GFCC′C′′ strands of CC1-N are shown and labelled, whilst the C to D strands of β-IgI3 are shown and labelled. Binding of rCC1-N to dynabeads (DB) coated with β-IgI3 wild type (WT) or mutants. Mutation of β-IgI3 residues 42, 46 and 53 lead to significant changes in CC1-N binding capacity. Binding of β-IgI3 coupled to Streptavidin-PE to DB coated with rCC1-N wildtype or variants. Mutation of CC1-N residues 29, 44, 89, 91 and 95 lead to significant changes in β-IgI3 binding capacity. \n",
      ": The unique IgI3 structure is present in Gram-positive bacterial species and evidence for binding CEACAM1.Figure 7Amino acid based Maximum Likelihood tree of β-IgI3 homologs. Clades of related IgI3 domains are shown, with clade I representative of β-IgI3 domain sequences. Branches are coloured by bacterial species from which the sequence originates, except those with no species designated. Maximum likelihood values bootstrap values > 60 are shown as a circle, where larger circles indicate a higher bootstrap support value. Superposition of the IgI3 domain from β protein (blue) onto predicted IgI3 domain structure of clade II sequence of S. pyogenes R28 protein (red). Close-up of the truncated C strand, α-helix and D strand of β-IgI3 (blue) and R28-IgI3 (red), indicating the corresponding R28-IgI3 residues present at sites in β-IgI3 that bind CEACAM1 (CC1). Binding of recombinant (r)CC1-HIS to dynabeads (DB) coated with β-IgI3, R28-IgI3 or control. Mean and SD is shown for n = 6 independent replicates. Binding of β-IgI3, R28-IgI3 or human serum albumin (HSA) coupled to PE-conjugated streptavidin (SPE), to DB coated with CC1 (DB.CC1). Mean and SD is shown for n = 6 independent replicates. Binding of recombinant rCC1-HIS (10 μg/mL) to R28-expressing GBS strain H1147 and R28-expressing S. pyogenes strain AL368. Data representative of n = 3 replicates. The per residue free energy contribution spectrums of β-IgI3/CC1-N interface and R28-IgI3/CC1-N interface. The residues with binding free energy contributions lower than -2.0 kcal/mol are identified as key (favourable) residues and coloured blue (for β-IgI3) or red (for R28-IgI3). Free energy contributions at -2.0 kcal/mol and 2.0 kcal/mol are indicated by dashed lines. The position of the truncated C strand, α-helix and D strand is shaded for each protein. The mean and standard deviation is representative of estimates from 50 docked models.. \n",
      ".Figure 1(A) BMDMs from WT (Pml+/+) and Pml-/- mice were pre-treated with zVAD (20 μM), Nec-1 (40 μM) or GSK872 (10 μM), as indicated, for 30 min followed by co-treatment with AT-406 (1 μM). Cell viability was evaluated by ATP release after 18 h. Data show mean ± SD of four biological replicates. (B) WT and Pml-/- BMDM were treated with TNF (100 ng/ml), zVAD (20 μM) and Nec-1 (40 μM) or GSK872 (10 μM), as indicated, for 16 h. Cell viability was determined by ATP release. Data show mean ± SD of three biological replicates. (C) Normal expression of RIPK1, RIPK3 and MLKL in Pml-/- BMDMs. Lysates from WT and Pml-/- BMDMs were analysed for their levels of RIPK1, RIPK3 and MLKL. (D) WT and Pml-/- immortalized MEFs were treated with TNF (100 ng/ml), zVAD (20 μM), Nec-1 (40 μM), and GSK872 (10 μM), as indicated, for 8 h. Cell viability was determined by ATP assay. Data show mean ± SD of three biological replicates. (E) WT and Pml-/- immortalized MEFs were treated with TNF (100 ng/ml),IKK inhibitor BMS-345541 (10 mM) and Nec-1, as indicated, for 6 h and then cell death was assayed. Data show mean ± SD of five biological replicates, except the Nec-1-containing experiment (n=2). \n",
      ".Figure 2(A, B) Pml+/+ and Pml-/- BMDMs were treated with zVAD + AT406 (zA) (A) or TNF + zVAD (Tz) (B), as in Fig. 1, for the indicated times. Contents of RIPK1, p-RIPK1[S166], RIPK3, p-RIPK3, MLKL, and p-MLKL in cell lysates was analysed by immunoblots. Bottom panels, quantitation of p-RIPK1[S166] from three biological replicates using normalized intensity of p-RIPK1[S166] in WT cells at 4 h as 1. Data show mean ± SD. (C) Pml+/+ and Pml-/- BMDMs were treated with AT-406 at the concentration indicated for 24 h, and cell viability was then determined by ATP assay. Data show mean ± SD of three biological replicates. (D) BMDMs were treated with zVAD (20 μM) +AT-406 (1 μM) for 18 h, and TNF generated was then analyzed by ELISA. Data show mean ± SD of four biological replicates. (E) BMDMs were treated with zVAD + AT-406 (1 μM), with or without anti-TNF for 18 h, and cell viability quantitated. Data show mean ± SD of three biological replicates. \n",
      ".Figure 3(A) Pml+/+ and Pml-/- BMDMs were treated with zVAD for 30 min, followed by AT-406 (zA) treatment at the indicated time-points, before preparing total cell lysates (TCL). TCL were immunoprecipitated with anti-FADD and the amounts of p-RIPK1[S166], RIPK1 and caspase-8 in the precipitates and TCL were analysed. (B) Pml+/+ and Pml-/- BMDMs were treated with zVAD + AT406 (zA) as in (A), and TCL were then harvested. TCL were immunoprecipitated with anti-caspase-8, and the contents of p-RIPK1[S166] and RIPK1 in the precipitates and TCL was analysed. \n",
      ".Figure 4(A, B) WT (Pml+/+) and Pml-/- mice (n = 12 in each group) were injected with mouse TNF (1.5 μg/g body weight), and rectal body temperature (A) and survival (B) were determined at the indicated time-points. #, the temperature of only six mice was measured at the indicated time-points. Mean ± S.E.M is shown. *P < 0.05 as determined by two-way ANOVA (A), or by Log-rank (Mantel-Cox) test (B). (C, D) WT and Pml-/- mice were injected with TNF and serum samples from live mice were collected 30 h later. Levels of LDH (C) and GPT/ALT (D) were quantitated. Mean ± S.E.M. is shown. P = 0.0095 (C, D) by unpaired t-test with Welch's correction. Serum levels of TNF (E) from WT and Pml-/- mice 12 h after TNF injection. Serum levels of IL-6 (F) and IL-1α (G) from WT and Pml-/- mice 12 h after TNF injection. \n",
      ".Figure 5(A) Pml+/+ and Pml-/- BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) for the indicated timeframes. The contents of p-IKKα/β, IKKα/β, IκBα, p-MK2 and MK2 was analysed by immunoblotting. (B) Pml+/+and Pml-/- BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) for the indicated times, and levels of p-RIPK1[S321], RIPK1, p-JNK, JNK, p-ERK, ERK, p-p38, and p38 at the indicated time points were then determined. (C, D, E, F) WT (Pml+/+) and Pml-/- mice were treated with DMSO or Nec-1s (6 μg/g) or MK2 inhibitor PF-3644022 (75 μg/mouse) by intraperitoneal injection 15 min before and 60 min after intravenous TNF (1.5 μg/g body weight) injection. Body temperature (C, E) and survival (D, F) were monitored. Experiments C-F were conducted concurrently. Mean ± S.E.M is shown (C, E). \n",
      ": Hemocytes display distinct properties at E16 and WL stage.Figure 1A) Transcriptome comparison of hemocytes from stage 16 embryos (E16) and wandering 3rd instar larvae (WL). The x-axis is the average genes expression levels (n=3) and the y-axis the Log2 Fold Change WL/E16. P-values are indicated with the color code. B) Gene Ontology (GO) term enrichment analysis in E16 (green) and WL (red) hemocytes. The fold enrichments for a subset of significant GO terms are displayed, the number of genes and the p-value of the GO term enrichment are indicated in brackets. Scatter plots as in (A) highlighting in black subsets of known genes expressed in hemocytes Scatter plots as in (A) highlighting in black subsets of known genes associated with the GO terms extracellular matrix Scatter plots as in (A) highlighting in black subsets of known genes expressed in phagocytosis F-G) Phagocytosis assay on E16 (F) and WL hemocytes (G) srp(hemo)-moesin-RFP. The beads (in green) are phagocytosed by the hemocytes (in red). The WL hemocytes show greater phagocytic capacity compared to the embryonic ones after 5 min of exposure. Full stacks are displayed and the scale bars represent 20 µm. \n",
      ".Figure 6(A, B) WT and Mk2-/- BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) (A) or with zVAD (20 μM) + AT-406 (1 μM) (B), and then the contents of p-RIPK1[S321], RIPK1, p-p38, p38 was determined at the indicated time-points. (C) WT, Pml-/-, Mk2-/- and Pml-/-Mk2 -/- BMDMs were treated with zVAD (20 μM), AT-406 (1 μM), Nec-1 (40 μM) and GSK872 (10 μM), as indicated, for 18 h. Cell viability was determined by ATP release. Data show mean ± SD of three biological replicates. *P < 0.05 for multiple t-tests according to the Holm-Sidak method. (D) WT, Pml-/-, Mk2-/- and Pml-/-Mk2 -/- BMDMs were treated with TNF (100 ng/ml), zVAD (20 μM), Nec-1 (40 μM) and GSK872 (10 μM), as indicated, for 16 h. Cell viability was determined by ATP release. Data show mean ± SD of three biological replicates. *P < 0.05 for multiple t-tests according to the Holm-Sidak method. (E) WT, Pml-/-, Mk2-/- and Pml-/-Mk2 -/- BMDMs were treated with zVAD (20 μM) and AT-406 (1 μM) for the indicated timeframes. The levels of RIPK1, p-RIPK1[S166], RIPK3, p-RIPK3, MLKL, and p-MLKL in cell lysates were analysed by immunoblots. Results are representative of three biological replicates. (F) WT, Pml-/-, Mk2-/- and Pml-/-Mk2 -/- mice were injected i.v. with TNF (1.5 mg/kg), and survival was monitored for 24 h. Data represent a combination of two independent experiments (n=10). *P < 0.05 for Log-rank (Mantel-Cox) test. (G, H) WT, Pml-/-, Mk2-/- and Pml-/-Mk2 -/- mice (n = 5) were injected i.v. with TNF (1.5 mg/kg), before recording body temperature for the next 6 h (G), and collecting serum samples at 3 h to quantitate LDH release (H). *P < 0.05 as determined by two-way ANOVA (G), or P values calculated by unpaired t-test with Welch's correction (H). \n",
      ".Figure 7(A, C) WT BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) for the indicated times. Cell lysates were immunoprecipitated with anti-MK2, and levels of PML were determined. Mk2-/- BMDMs served as a negative control. (B) WT BMDMs were treated with TNF (100 ng/ml) for the indicated times, before immunoprecipitating p38 MAPK and determining the levels of PML, p38, MK2 in the precipitates and total cell lysates. (C) WT BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) for the indicated times. Cell lysates were immunoprecipitated with anti-PML, and levels of p38 and MKK3 were determined. Pml-/- BMDMs served as a negative control. (D) WT and Pml-/- BMDMs were treated with TNF (100 ng/ml) + zVAD (20 μM) for the indicated times. p38 MAPK was immunoprecipitated, and its association with MK2 was then analysed. (E) Recombinant Flag-PML-I, GST-MK2 (60 ng) and His-p38 (60 ng) were incubated, as indicated, at 4 °C for 2 h. p38 was pulled down by anti-p38 and then PML, MK2 and p38 contents in the precipitate and reaction mixture (input) were detected by immunoblotting. (F) Recombinant Flag-PML-I, GST-MK2 (100 ng) and His-p38 (200 ng) were incubated, as indicated, at 30 °C in in vitro kinase assay buffer (20 mM HEPES, 10 mM MgCl2, 100 nM ATP, 1 mM DTT) for 20 min. The levels of p-MK2, MK2, p38, and PML (FLAG) in the reaction mixture were then determined. \n",
      ": 14 hemocyte populations can be distinguished in WL by single cell RNA seq.Figure 2A) UMAP projection representing the 14 clusters of cells identified in the hemocyte pools from OregonR WL (NI dataset). B) Number of cells and proportion of each cluster in the NI dataset. C) GO terms enrichment analysis for each cluster. The x-axis is the GO term enrichment, the color gradient (black to light blue) indicates the p-value, the number of genes and the GO term category (CC: cellular compound, BP: biological process, MF: molecular function) are indicated between brackets. D) Top 5 markers of each cluster. The expression levels is represented by the gradient of purple levels) and the percentage of cells with the size of the dots. \n",
      ": Identification of the cluster specific molecular pathways and of the filiation between the NI clusters.Figure 4A) Heatmap representing the z-score for the top 5 regulons of each cluster determined with SCENIC. The dendrogram on the left side of the panel indicates the correlation between the regulons across the dataset. The z-score is indicated with a gradient from magenta (z-score < 0, the regulon is repressed) to yellow (z-score > 0, the regulon is active). B) Dendrogram representing the distance among the clusters. The tree was built on the correlation (Spearman) calculated on the regulon matrix from the NI dataset. C) Lineage prediction using RNA Velocity. The arrows and lines on the UMAP predict the \"direction\" taken by the cells of each cluster, based on the comparison between the levels of mRNA and pre-mRNA. Note that the clusters CC and PL-ImpL2 (dashed lines) have been moved from their original position to fit in the graph. D) Single cell trajectory reconstructed with monocle 2 on the NI dataset. The first panel shows the overlap of all clusters and subsequent panels show restricted number of clusters for which the RNA velocity analysis suggested a filiation. \n",
      ": Two additional hemocyte populations are produced after wasp infestation.Figure 5A) UMAP projection representing the 16 clusters of cells identified in the WI sample. Note the presence of two additional clusters, LMLM-1 and LM-2, compared to the NI sample. The two clusters correspond to the lamellocytes. B) Estimation of the number of cells of each cluster per larva in normal condition (NI, orange) and after wasp infestation (WI, blue). For each cluster, the cell number was deduced from the total number of plasmatocytes and lamellocytes numerated and the proportion of each cluster in the singe cell datasets. C) GO terms enrichment analysis for each cluster of the WI dataset. Represented as in Figure 2C. D) Dot plot representing the expression levels (gradient of red levels) and the percentage of cells (size of the dots) that express the top 5 markers of each cluster in the WI dataset. \n",
      ": Localisation of the NI hemocytes clusters.Figure 3A) Identification of the position (circulating/resident) of the clusters within the larva by qPCR. The left panel indicates the distribution of each marker across all clusters (as in Figure 2D), the right panel indicates the Log2 of the ratio between the expression level in the circulating versus the resident compartment. Positive values indicate an enrichment in the circulating compartment and negative values in the resident compartment (n=5, mean +/- SD are represented on the graph). The p-values are estimated by bilateral paired student test B) Circulating and resident hemocytes (top and bottom panels, respectively) from Mimic-Lsp1beta-MI05460 WL, which express Lsp1beta tagged with GFP. The immunolabelling was done with anti-GFP (in gray) and the nuclei were marked with DAPI (blue). Full stacks are displayed, the left panels show the overlay of DAPI and GFP, the right panels show the GFP alone. The scale bars represent 10 µm. C) Quantification of the GFP intensity in circulating and resident hemocytesD) Resident hemocytes located around the oenocytes or along the muscles (top and bottom panels, respectively) from WL srp(hemo)-moesin-RFP, that express moesin tagged with RFP in hemocytes. The immunolabelling was done with anti-RFP (in red), anti-Pendulin (Pen, in gray) and the nuclei were marked with DAPI (blue). Full stacks are displayed, the left panels show the overlay of DAPI, Pen and RFP, the middle panels RFP alone and the right panels Pen alone. Arrow heads in the top panels indicate cells co-expressing RFP and Pen. The scale bars represent 50 µm. \n",
      ": Identification of the filiation between the WI clusters.Figure 7A) Lineage prediction for the clusters from the WI sample using RNA velocity, as in Figure 4C. B) Single cell trajectory reconstructed with monocle 2 on the WI dataset. The panels show restricted number of clusters for which the RNA velocity analysis suggested a filiation. \n",
      ": Characterisation of the two lamellocyte clusters.Figure 6A) Scatter plot comparing the transcriptome of the two lamellocyte clusters, deduced from the WI dataset. The x-axis is the average expression levels of the genes and the y-axis the log2 fold change (LM-1/LM-2). The genes significantly enriched in the LM-1 cluster are highlighted in red (p<10-6), those significantly enriched in the LM-2 cluster in orange (p<10-6). B) GO term enrichment analysis on the genes enriched in lamellocytes compared to all other clusters (top panel) and on the genes specific to LM-1 (middle panel) or LM-2 (lower panel). The bars represent the fold enrichment and the color gradient indicates the p-value of the GO term enrichment C) Expression levels (gradient of color) and the percentage of cells (size of the dots) for the top 10 markers of lamellocytes (in black), of markers specific to LM-1 compared to LM-2 (in red) and specific to LM-2 compared to LM-1 (in orange). Note that most markers enriched in LM-1 compared to LM-2 are exclusively expressed in LM-1 while most markers enriched in LM-2 compared to LM-1 are also expressed in the plasmatocyte and crystal cell clusters. D) Hemocytes from WL OregonR infested by wasp. The immunolabelling was done with antibodies targeting the new lamellocyte markers identified in this study Rhea (in gray, top panels) and Shot (in gray, lower panels). Phalloidin-FITC (in green) labels the actin filament particularly abundant in lamellocytes (Tokusumi et al., 2009) and the nuclei were marked with DAPI (blue). Full stacks are displayed, the left panels show the overlay of DAPI, FITC and the lamellocyte markers, the middle panels show the FITC alone and the right panels the lamellocyte markers alone. The scale bars represent 20µm. \n",
      ": Timing of the production of the different clusters in the hemolymph of WI larvae A) Expression levels of the cluster markers in the hemolymph of WI larvae compared to NI larvae.Figure 8A) Expression levels of the cluster markers in the hemolymph of WI larvae compared to NI larvae. The left panel indicates the distribution of each marker across all clusters (as in Figure 2D), the graph on the right panel, indicates the Log2 of the ratio between the expression level in WI versus NI. Positive values indicate an up-regulation upon wasp infestation and negative values a down-regulation (n=3, pool of 10 larvae per replicate, mean +/- SD are represented on the graph). The p-value are estimated by bilateral student test and indicated as follow: ns = not-significant (>0.05), \"*\" = p[0.05 - 0.01[, \"**\" = p[0.01-0.001[. B,C) Expression levels of the cluster markers increasing in WI compared to NI according to (A) during the progression of the immune response to wasp infestation. Collections were carried out at t0 (time of infestation, L2), 24 h (early L3), 48 h (mid L3) and 72 h (WL) after wasp infestation (n=3). The time line is on the x-axis and the normalised expression levels on the y-axis. The expression levels were normalised to the levels at t0. The dotted lines indicate t0 relative level (=1). The markers that do not display strong variability during the timeline are presented in (B) and the ones showing a strong up-regulation are in (C).E) Weighted dot plot representing the enrichment (Log2 fold change) of the markers in hemocytes originating from the lymph gland (DotGFP-WI) compared to embryonic hemocytes (srpGFP-WI) from WI larva. The lymph gland hemocytes and embryonic hemocytes were traced using the lymph gland driver DotGal4 and the embryonic driver Srp(hemo)Gal4 respectively combined with a lineage tracing construct. The relative expression levels are indicated by the size of the dots and the pvalue of the log2 fold change with the gradient dark blue (not significant) to light blue (p<10-3). The p-value are estimated by bilateral student test and indicated as follow: ns = not-significant (>0.05), \"*\" = p[0.05 - 0.01[, \"**\" = p[0.01-0.001[, \"***\" = p <0.001]. \n",
      " - Sporozoite budding is impaired in α1-tubulin(-) parasites A.Figure 2Electron micrographs showing different stages of development in WT and α1-tubulin(-) parasites. Key structural elements are highlighted in the following colors: blue: plasma membrane; yellow: inner membrane complex; green: microtubules; light blue: micronemes; lilac: rhoptries; brown: rootlet fiber; cyan: nucleus. Note the undulating shapes of the elongated mutants and that micronemes are hard to distinguish from other small vesicles in the α1-tubulin(-) parasites. The rootlet fiber in panel II starts at the apical tip and in panel V is associated with the centriolar plaque (dark structure in nuclear envelope). Panel III corresponds to Fig 1D. Scale bars: 200 nm. 3D reconstruction from serial sectioning reveals that growing α1-tubulin(-) 'sporozoites' lack apical polarity and fail to pull in nuclei. Microtubuli: green; rootlet fiber (connecting apical pole to centriolar plaque): red; rhoptries: purple; endomembraneous vesicles (Golgi, premicronemes): brown; nucleus: cyan; sporozoite plasma membrane and inner membrane complex: blue; dotted lines show location of sporoblast membrane. 17 and 10 sections of 100 nm and 300 nm thickness were processed for the tomograms in WT and α1-tubulin(-) parasites, respectively. Scale bar: 1 µm. Tomography of serial sections from budding α1-tubulin(-) parasites at day 13 (same as in panel B) and day 17 to highlight the increase of surface area (blue) with time. The oocyst wall at day 17 is highlighted by cyan lines (top), the sporoblast membrane is highlighted by red lines (bottom). The sporoblast membrane at day 13 is indicated by a dashed line (red). 10 and 11 sections of 300 nm thickness were processed for the tomograms at day 13 and day 17, respectively. A single continuous parasite was highlight at day 17. Scale bar: 1 µm. α1-tubulin(-) parasites show multiple attachments (arrowheads) with the sporoblast membrane at day 17 post infection. Note the highly branched nature of the budding parasite (same parasite as in panel C) shown in two views in I and II. Right: slices through the tomogram in grey scale with the outlines used for the 3D model indicated. Two connections with the sporoblast are indicated by 1 and 2. Scale bars: 1 µm. \n",
      " - Genome replication and nuclear multiplication is proceeding in the absence of microtubules A-D Life imaging of wild-type (WT) and α1-tubulin(-) oocysts expressing ef1α:GFP (early and late oocysts) and csp:mCherry (late oocysts) to locate the oocyst cytoplasm.Figure 3Life imaging of wild-type (WT) and α1-tubulin(-) oocysts expressing ef1α:GFP (early and late oocysts) and csp:mCherry (late oocysts) to locate the oocyst cytoplasm. Microtubules were labeled with SiR-tubulin (green) and DNA with Hoechst (blue). Oocysts are shown in a chronological order from early (day 4) to late (day 12) development. (A, B) Early oocyst development with remaining subpellicular microtubules of the preceding ookinete stage (A) and subsequent DNA replication (B), where some but not all DNA is co-stained by microtubules in the mutant. (C) Nuclear alignments to the invaginated plasma membrane and budding of sporozoites. Note the strong SiR-tubulin signal only seen in sporozoites of WT oocysts and complete absence from the mutant. Sb: sporoblast. (D) Oocyst ready to burst with fully formed sporozoites in WT oocysts. Note that α1-tubulin(-) oocysts retained sporozoite nuclei predominantly within the sporoblast (Sb) during budding. Scale bars: 5 μm, magnification box in B: 1 µm,. 3D reconstructions from spinning disc confocal microscopy z-stacks across an early oocyst labelled with the DNA dye Hoechst (left). 50 and 19 images with a z-distance of 0.5 µm and 1 µm were collated for WT and α1-tubulin(-) oocysts, respectively. Right: 3D reconstructions from late WT and α1-tubulin(-) oocysts labelled with Hoechst (blue) and SiR-tubulin (green). Note that the mutant does not form proper sporozoite shapes during budding and the few peripheral nuclei that were uptaken into budding sporozoites. 10 and 28 images with a z-distance of 1,5 µm and 1 µm were collated for WT and α1-tubulin(-) oocysts, respectively. Sb: sporoblast; scale bars: 5 µm. TEM cross sections of WT and α1-tubulin(-) oocysts during nuclear division. Spindle microtubules (arrowheads) emerging from centriolar plaques (M) were seen in WT oocysts. Although individual nuclei (N) were visible in α1(-) oocysts, microtubules were absent even in dividing centriolar plaques (M). Scale bars: 0.2 μm. \n",
      " - Lowering expression of α1-tubulin leads to fewer microtubules and aberrant sporozoite shapes A.Figure 4Complementation of α1-tubulin(-) parasites with different α1-tubulin constructs lacking introns (α1∆introns) or featuring only codon-modified exons (α1cm&∆introns). Scanning electron micrographs show the aberrant shape of the α1cm&∆introns-expressing sporozoites. See also Fig EV5. Scale bar: 1 μm. Fluorescence images showing the SiR-tubulin stained microtubules, nuclei (Hoechst) and cytoplasmic shape (mCherry) of the parasite lines. Note the small amount of DNA (black arrowhead) and the reduced level of SiR-tubulin fluorescence in the α1cm&∆introns-expressing sporozoites (lower graph). See also Fig S6. Quantitative data was derived from 96-120 sporozoites per line. Box plots represent 50% (boxes) and 95% (bars) of data with horizontal bar showing the median. Scale bar: 5 μm. **** indicates p<0.0001, Kruskal-Wallis-test. Note that α1cm&∆introns-expressing sporozoites do not express mCherry; their autofluorescence (auto-fluo) indicates an aberrant shape. qRT-PCR analysis of the three parasite lines at day 7 and 12 after infection. Error bars correspond to the standard deviation from two biological replicates. TEM reveals α1cm&∆introns-expressing sporozoites with smaller (A and B) and larger (C) diameters than wild type sporozoites. See also Fig S7. Scale bar: 0.2 µm. α1cm&∆introns-expressing sporozoites show a reduced number of subpellicular microtubules (green arrowheads) by TEM. Cartoon showing that sporozoites cannot form in the absence of microtubules, while lower numbers allowed formation of aberrantly shaped parasites. Note that subpellicular microtubules are exclusively polymerized from the apical polar ring. \n",
      " - α1-tubulin is essential for sporozoite formation A.Figure 1Cartoon of a simplified Plasmodium life cycle indicating in red the stages important for our work. Note that transfection is performed in blood stage parasites. Schematic cartoon of a sporozoite with an indicated apical cross section. Subpellicular (cytoplasmic) microtubules (green) can be found below the three membrane layers (plasma membrane and inner membrane complex) of the pellicle (bottom). Isolated sporozoites from the hemolymph (HL) and salivary glands (SG) of infected mosquitoes stained with Hoechst and SiR-tubulin. Note the similar staining of WT and α1-tubulin(-) complemented lines. Scale bars: 5 µm. SiR-tubulin staining (green, I and II) and TEM (III) show that microtubules shrink after wild type sporozoite formation. During budding microtubules can be up to 15 µm long (I), while after formation they measure on average only 6 µm (graph). Inset in II shows sporozoite cross-section at the sporoblast membrane; note the SiR-tubulin fluorescence (green, arrowheads) next to the nuclear Hoechst stain (blue). (III) TEM longitudinal section of budding sporozoites with an indicated subpellicular microtubule (white line, length). Green arrowheads point to microtubules close to the sporoblast. Rh: rhoptry; N: nucleus; ER: endoplasmic reticulum; Mi: mitochondrion; Sb: sporoblast. **** indicate p<0.0001; Kruskal-Wallis-test. Scale bars: I&II: 5 µm, magnification box in II: 1 µm, III: 1 µm, magnification box in III: 0.1 µm. Comparison of sporozoite development in oocysts of WT, α1-tubulin(-) and complemented lines. Hoechst (blue) and SiR-tubulin (green) staining reveal that the complemented line generates sporozoites indistinguishable from WT controls, while SiR tubulin staining is absent from α1-tubulin(-) oocysts. mCherry stains the cytoplasm of the parasite. Scale bars: 5 µm. Transmission electron micrographs of developing sporozoites in WT and α1-tubulin(-) parasite lines. Note the slender sporozoites in the WT oocysts, which are absent in the mutant. Sb: sporoblast; scale bars: 5 µm. TEM cross sections of WT and α1-tubulin(-) parasites showing the aberrant structure of the latter. Subpellicular and centriolar plaque (spindle pole body) associated (hemispindle) microtubules are readily recognizable in the WT sporozoites (green arrowheads in insert) but absent in the mutant, while the inner membrane complex is formed below the plasma membrane of both parasites (white arrowheads). Note the different scale chosen to highlight the malformed shapes in the mutant. N: nucleus; scale bars: 0.2 µm, magnification boxes: 0.1 µm,. \n",
      ".Figure 1A. Experimental scheme of antagomiR-132 and miR-132 mimic injections into the lateral ventricle of 2-month old APPPS1 mice. B. Semi-quantitative PCR of miR-132 and negative control miRNAs in the hippocampus of antagomiR-132-injected mice (ant-132) in comparison to control-injected animals (aCSF and scramble) at six months of age. Sample size, n=9 per group. C. FISH of miR-132 and negative control miR-124 in the hippocampus of ant-132- and scramble-injected mice. Scramble probes were used as FISH negative controls. Scale bar, 500μm. D. Semi-quantitative PCR of miR-132 and miR-212 in the hippocampus of 3-month old miR-132 mimic-injected mice (miR-132) compared to animals injected with a negative control oligonucleotide (Ctr). Sample size, n=6 per group. \n",
      " - Microtubule length and numbers determine sporozoite curvature and infectivity A.Figure 7Quantitative analysis of salivary gland sporozoite lengths from the different lines. n indicates numbers of investigated sporozoites. Box plots represent 50% (boxes) and 95% (bars) of data with horizontal bar showing the median. *** and **** indicate p<0.001, and p<0.0001, respectively; ns: not significant; Kruskal-Wallis-test; scale bar: 1 µm. Quantification of SiR-tubulin fluorescence from 69-127 salivary gland derived sporozoites. Note the decrease in the fluorescence intensity of sporozoites expressing α1-tubulin without introns as well as the decreased intensity and length of the chimeras. Linear correlation of sporozoite length and microtubule length reveals a R2=0.94. *, **, and **** indicate p<0.05, p<0.01, and p<0.0001, respectively; ns: not significant; Kruskal-Wallis-test. Shorter sporozoites exhibit stronger curvatures and hence tighter circular motility on glass. Scale bar: 20 µm. Box plots represent 50% (boxes) and 95% (bars) of data with horizontal bar showing the median. * and **** indicate p<0.05, and p<0.0001, respectively; ns: not significant; Kruskal-Wallis-test. Sporozoites with fewest microtubules (α2+++) show a motility defect. n indicates numbers of investigated sporozoites; Numbers on grey bars indicate percentage of partially and persistently moving sporozoites. Sporozoites from lines with fewer microtubules move more frequently in apparent clockwise (CW) direction. Numbers on black bars indicate percentage of CW moving sporozoites; numbers in movie stills indicate time in seconds. CW moving sporozoites move at lower speed than CCW moving ones. Salivary gland sporozoites contain at least 10 microtubules as revealed by TEM of α2++ infected salivary glands. Green arrowheads point to microtubules. Scale bar: 0.2 µm. Error bars show the full range and boxes 50% of values. Red crosses indicate the mean, red lines the median. Fisher's exact test (two-tailed). Infectivity of sporozoites to mice decreases with decreasing numbers of microtubules and shorter microtubules. The prepatency is prolonged in parasites with a reduced sporozoite length and gliding circle diameter. NMRI mice were exposed to ten mosquito bites of the respective parasite line. Numbers indicate infected mice / mice exposed to mosquito bites (left) and infected mice / mice injected with 10.000 sporozoites (right). Cartoon illustrating the various consequences of different microtubule numbers and lengths during Plasmodium sporozoite formation and tissue invasion. Sporozoites with more than 10 microtubules can enter salivary glands while salivary glands sporozoites with shorter microtubules are least infectious. \n",
      " - Mathematical model explains average number and length of microtubules at different tubulin expression levels A.Figure 6A cartoon depicting four possible states of a microtubule nucleation site considered in the mathematical model, namely unbound, bound, nucleating and microtubule growing, as well as the transition between them. The transition rates depend on the dimer concentration c. An unbound nucleation site transitions to a bound state when a tubulin dimer binds to it with rate kmcp. A bound nucleation site starts nucleation with rate kncn and adds tubulin dimer sequentially with rate kmc to grow the microtubule. Here p and n (where p+n=12 being the size of the critical nucleus, see Methods) represents the cooperativity of tubulin dimer binding to a free and a bound nucleation site, respectively. The minimum value of MSE in MT number (shown by black points) increases with the increase in binding cooperativity (in curves from left to right). The MSE in MT length at binding rates corresponding to MSE minimum in MT number (shown by open black circles) decreases with increase in binding cooperativity. The parameter set shown by star symbol is selected for comparing the modelling results with the experiments. The x-values are multiplied by a factor of (10 − 5 )p to present the plots in a common range for easy comparisons. The microtubule number increases with the increase in the concentration of tubulin dimers. Microtubule number scales with tubulin concentration at low expression. At high expression, microtubule number is limited by the number of nucleation sites. Shown are mean +/- standard deviation from all determined values. The average microtubule length remains close to the maximum length 6 μm. The circles are obtained by averaging results from 200 stochastic simulations of the model equations and stars are the experimental data (blue stars are for (α2+, α2++, α2+++) strains). The error bar shows the standard deviation in the data and simulation results. For the WT expression level (c0 = 36.6 (m0+p+n)), the rate of nucleation is faster than the microtubule growth. Then the microtubule number is determined by the number of nucleation sites. The average microtubule length is around 1 μm (shown by dashed arrow) when the last nucleus forms. At intermediate tubulin numbers (c0 = 16 (m0+p+n)), corresponding to the α1c-term mutant, the microtubule formation rate is comparable to nucleation rate. The average microtubule length is around 4 μm (shown by dashed arrow) when the last nucleus forms. \n",
      " - α1-tubulin specific amino acids modulate microtubule length while modulating expression level tunes microtubule numbers A.Figure 5Expression of different α1-α2 chimeric α-tubulins instead of α1-tubulin leads to sporozoite shape changes as revealed by SEM. See also Fig EV5. Scale bar: 1 µm. α1∆c-term lacks the last 3 amino acids of α1-tubulin. α2+ parasites contain an α1-tubulin with all amino acids that are different between the two α-tubulins changed into those of α2-tubulin; see also Fig S4. α2++ parasites contain α2-tubulin (exons, introns, 3'UTR) and α2+++ parasites express the exons of α2-tubulin and introns of α1-tubulin from the α1-tubulin promoter. SiR-tubulin staining reveals differences in microtubule lengths and intensity. Between 69 and 156 sporozoites were analysed per line. Box plots represent 50% (boxes) and 95% (bars) of data with horizontal bar showing the median. ***, and **** indicate p<0.001 and p<0.0001, respectively; ns: not significant; Kruskal-Wallis-test. qRT-PCR analysis of the parasite lines at day 12 after infection shows different expression level. Note the subtly higher expression of α2++ compared to α2+++. Error bars show standard deviation from the mean of two biological replicates. Schematic cartoon of sporozoite indicating the apical, central and nuclear sections as analysed in panels (E-G). TEM along sporozoites shows a decrease of microtubule numbers towards the nucleus in parasites expressing α-tubulin chimeras. Green arrowheads point to microtubules. See also Fig S8 and S9. Note that the WT control panel is shown already in Figure 4E to indicate that here we perform an extended analysis to the dataset generated earlier. Scale bars: 0.2 µm. TEM reveals different numbers of microtubules at the apical end of sporozoites for the different lines. Each dot represents the numbers of microtubules counted per section. Red lines indicate median from 23 to 40 analysed sections per line. Microtubule quantification along the sporozoite reveals the presence of fewer microtubules in central and nuclear sections of α2+ and α1∆c-term parasites compared to wild type sporozoites. Cartoon summarizing the effect of α2-tubulin expression on microtubule numbers and lengths. Numbers indicate the median number of observed microtubules per line. Semilog correlation of gene expression as determined by qRT-PCR with microtubule numbers as determine by TEM. Bars show range of values in x and standard deviation in y around the mean (dots) of two biological replicates (x) and all determined values (y). \n",
      ".Figure 2A. Amyloid staining (6E10) combined with miR-132FISH in ant-132- and scramble-injected mice at six months of age. Scale bar, 500 μm. B. Quantification of amyloid burden in hippocampus and cortex of ant-132-injected and control animals (see Materials and Methods). Sample size, n=4 per group. C, D. ELISA of soluble (TBS) and insoluble (Formic acid) Aβ40 and Aβ42 levels in the hippocampus of ant-132- (C) and miR-132-injected (D) animals at six and three months of age, respectively. Sample size, n=6 per group. \n",
      ".Figure 4A. Correlation analysis process: miRNA and mRNA profiling datasets were cross-checked for miR-132 predicted targets that are significantly anti-correlated to miR-132 expression and significantly upregulated at late stages of AD. B. Positive hits were validated by semi-quantitative qPCR in ant-132-injected mousehippocampus. Black line indicates mean of scramble control set to 1. Dashed line indicates chosen threshold of 1.3-fold. Sample size, n=9 per group. Values were normalized to scramble-injected group and presented as mean ± SEM. \n",
      ".Figure 3Western blot analysis of total TAU and pTAU (AT8, AT270) levels upon miR-132 knockdown (A) or overexpression (B) at six and three months of age, respectively. Sample size, n=9 per group (A) and n=6 per group (B). Values were normalized to the respective control groups and presented as mean ± SEM. Student's t-test was used. \n",
      ".Figure 5A. Luciferasereporter assay of wild-type (wt) and mutant (mut) ITPKB 3'UTR in HEK293 cells co-transfected with a synthetic miR-132 (miR-132) or a negative control (Neg Ctr) oligonucleotide. B. ELISA of Aβ40 and Aβ42 levels in HEK293-APPswecells transfected with a miR-132 antisense oligonucleotide (miR-132 inh), an siRNA against ITPKB (ITPKB siRNA) or both. The assays in A and B were performed in three independent experiments, each in triplicates. C. Western blot analysis of ITPKB knock down in APPPS1hippocampus using an siRNA oligonucleotide against ITPKB. Sample size, n=6 per group. Values were normalized to control-injected group (Ctr) and presented as mean ± SEM. D. ELISA of insoluble (Formic acid-soluble) Aβ40 and Aβ42 levels in the hippocampus of ITPKB siRNA- and control-injected animals at three months of age. Sample size, n=6 per group. Values were normalized to control group and presented as mean ± SEM. E, F. Western blot analysis of ITPKB levels upon miR-132 down- (E) or up-regulation (F) at six and three months of age, respectively. Sample size, n=9 per group for miR-132 down regulation and n=6 per group for miR-132 overexpression. Values were normalized to control groups and presented as mean ± SEM. \n",
      ".Figure 6Western blot analysis of CTFβ (A) levels upon miR-132 down- (ant-132) (left panel) or upregulation (miR-132) (right panel) in APPPS1hippocampus at six and three months of age, respectively. Sample size, n=9 per group for miR-132 down regulation and n=6 per group for miR-132 overexpression. Values were normalized to respective control groups and presented as mean ± SEM. The Student's t-test was used. Western blot analysis of sAPPβ-swe (B) levels upon miR-132 down- (ant-132) (left panel) or upregulation (miR-132) (right panel) in APPPS1hippocampus at six and three months of age, respectively. Sample size, n=9 per group for miR-132 down regulation and n=6 per group for miR-132 overexpression. Values were normalized to respective control groups and presented as mean ± SEM. The Student's t-test was used. Western blot analysis of phosphorylated ERK1/2 (pERK1/2) (C) levels upon miR-132 down- (ant-132) (left panel) or upregulation (miR-132) (right panel) in APPPS1hippocampus at six and three months of age, respectively. Sample size, n=9 per group for miR-132 down regulation and n=6 per group for miR-132 overexpression. Values were normalized to respective control groups and presented as mean ± SEM. The Student's t-test was used. \n",
      ".Figure 8Semi-quantitative PCR of miR-132 and western blot analysis of ITPKB, phosphorylated (pERK1/2) and total ERK1/2, phosphorylated (AT8 , AT270) and total TAU, phosphorylated sphingosine kinase 1 (pSphK1), BACE1, full length APP (flAPP), APP CTFs and sAPPβ in human AD hippocampi (AD) compared to non demented control samples (ND). Sample size, n=39 for AD and n=15 for ND. Values were normalized to ND and presented as mean ± SEM. The Student's t-test was used. \n",
      ".Figure 7A. Double immunofluorescence staining of amyloid plaques (6E10) and ITPKB in AD prefrontal cortex. Scale bar, 50 μm. Arrowheads indicate ITPKB immunopositivity. B. miR-132FISH coupled with double immunofluorescence against hyperphosphorylated TAU (AT8)-containing neurofibrillary tangles (NFTs) and ITPKB in AD prefrontal cortex. miR-124 was used as a control. Scale bar, 50 μm. C. Quantification of miR-132, ITPKB and hyperphosphorylated TAU (pTAU) signal in single neurons. Integrated intensity values of each signal per cell were normalized to the mean integrated density of each signal across all the cells analyzed (see Materials and Methods). Sample size (AD patients), n=3. Values are presented as mean ± SEM. Two-way Anova was used. Quantifications are summarized in the table provided. Arrow directions refer to the comparison of each normalized signal to the same signal in the other group (in the \"low miR-132\" group comparisons are made to the \"high miR-132\" group and vice versa). \n",
      " - Analysis of HEK293 mutant MRPS5 V336Y and HEK293 MRPS5 wild-type cells.Figure 3A Generation time (n=8 clones, ± SD). B Whole cell in-vivo mitochondrial translation as determined by 35S-Met incorporation (n=4 clones, ± SEM). C Autoradiography of in-organello translation using 35S-Met labelling. D Top five GO terms (cellular component) associated with genes upregulated in HEK293 mutant MRPS5 V336Y compared to HEK293 wild-type MRPS5. Corresponding GO terms associated with genes upregulated in mutant MT-RNR1 A1555G lymphocytes (Hollis et al, 2015) were plotted. The bars represent the percentage of genes that map to a GO term and which are upregulated. Fisher exact t-test, +p<10-6, ++p<10-7, +++p<10-8 (MRPS5 V336Y n=4, MRPS5 wild-type n=6); *p<10-30, **p<10-40, ***p<10-50 (A1555G n=28, wild-type n=15). \n",
      " - Noise-induced hearing damage.Figure 6Noise exposure and ABR measurements. Exposure of mice to broadband noise (2-10 kHz) of either 108 or 110 dB SPL resulted in elevations of auditory thresholds. Thresholds in Mrps5V338Y/V338Y mutant mice (black bars) were significantly higher after exposure than thresholds in littermates (open bars), (p<0.0001, two-way ANOVA for combined 108 and 110 dB data). Frequencies of 12, 24 and 32 kHz were selected for measurement to cover the mid-frequency to high-frequency regions of the cochlea. n=6 for baseline littermates; n=6 for baseline mutants; n=3 for all noise exposures. \n",
      " - Mutation-related mitochondrial function in brain.Figure 4Oxygen consumption rate (OCR) was measured in freshly isolated mitochondria from wild-type and mutant mouse cortex. A OCR in brain mitochondria from 19-months mutant mice (black circle) compared with OCR from wild-type mice (white circle). The sequential injection of mitochondrial substrates and inhibitors is indicated by dotted lines Values corresponding to the different respiratory states are shown. Oxygen consumption rate (OCR) was measured in freshly isolated mitochondria from wild-type and mutant mouse cortex. Baseline OCR, effect of age (p<0.0001) and genotype (p<0.05). Oxygen consumption rate (OCR) was measured in freshly isolated mitochondria from wild-type and mutant mouse cortex. C State 3 OCR (after addition of ADP), effect of age (p<0.001) and genotype (p<0.0001). Oxygen consumption rate (OCR) was measured in freshly isolated mitochondria from wild-type and mutant mouse cortex. D State 3 uncoupled OCR (state 3u after addition of FCCP), effect of genotype (p<0.0001) and age-genotype interaction (p<0.0001). E Ratio of mitochondrial complex IV and complex II activities. F ATP levels. G Superoxide anion radicals. \n",
      " - Exploration and anxiety (A-D), home cage learning (E-F), and place learning in the water-maze (G-I).Figure 5A Average distance to center as measure of anxiety in the open field (ANOVA: genotype F1,24=16.24 p=0.0005, age F2,24=5.217 p=0.0131). Mutants show increased anxiety levels at 9- and 14-months. B Estimated frequency of rearing activity as measure of exploratory drive in the open field (ANOVA: genotype F1, 24=39.40 p<0.0001, age x genotype F2,26=8.204 p=0.0017). Mutants showed little exploratory activity at 9- and 14-months. C Light-dark transition test, % time spent in the bright compartment (ANOVA: genotype F1,16=7.393 p=0.0152). Reduced scores at 9-months old animals indicate increased anxiety. D Exploratory corner visits in dark and light phase during adaptation to the IntelliCage (ANOVA: phase F1,18=617.2 p<0.0001). Corner visits during the dark phase as a correlate of exploratory activity showed premature reduction in 9-months old animals. E Learning in the hippocampus-dependent IntelliCage serial reversal task (ANOVA: day F2,56=26.71 p<0.0001). Scores provided no evidence for a mutation effect on learning rate or performance. F Learning in the hippocampus-dependent IntelliCage chaining task (ANOVA: day F2,54=450.1 p<0.0001). Scores provided no evidence for a mutation effect on learning rate or performance. G Water-maze place navigation, swim path during acquisition and reversal (ANOVA: day F4,104=21.14 p<0.0001, genotype F1,26=13.11 p=0.0013, age x genotype F2,26=8.204 p=0.0017). All groups learned: controls showed reduced performance with increasing age, mutants showed poor performance already at 9- and 14-months. H Number of wall approaches as indicator of poor coping with the test and non-spatial strategy choice (ANOVA: genotype F1,26=18.31 p=0.0002, age F2,26=4.486 p=0.0212, age x genotype F2,26=7.068 p=0.0035). Number of wall approaches increased with age in controls and was prematurely increased in mutants at 9- and 14-months. Water-maze probe trial, spatial retention score for target zones 1-3 comprising 5, 10 and 10% of the pool surface versus remaining 75% of surface (ANOVA: zone F3,78=36.04 p<0.0001, genotype x zone F3,78=7.807 p=0.0001). Spatial selectivity declined with age in controls and was prematurely reduced in mutants at 9- and 14-months. \n",
      "Nurr1 and Foxa2 interact physically and functionally to protect mDA neurons from toxic insult.Figure 1A-C Co-localization of Nurr1 and Foxa2 proteins in mDA neurons of the adult mousemidbrain. Representative confocal sections of the ventral midbrain (A) stained with antibodies specific for TH, Nurr1, and Foxa2. The midbrains of mice at 10 weeks were cryosectioned, and stained images were taken using a confocal microscope with z-stacks through the section thickness (12 μm). Shown in (B) are individual and merged images of TH, Nurr1, and Foxa2 staining of the boxed area in (A) at higher magnification. Representative images of a single nucleus from a TH+mDA neuron of an adult midbrain section (C) co-stained with Nurr1 (red) and Foxa2 (green). Co-localization of Nurr1 and Foxa2 in the merged images was assessed by Pearson's correlation and overlap coefficient values (n = 4, right/low quadrant in C). D Immunoprecipitation (IP) assay for Nurr1 and Foxa2 protein binding. The ventral part of midbrains was dissected from 10-week-old mice, lysed, and subjected to IP. Nurr1/Foxa2 protein binding was detected by WB analysis using an anti-Foxa2 antibody in immunoprecipitates generated with anti-Nurr1 antibody (left), as well as by anti-Nurr1 WB assay in immunoprecipitates with anti-Foxa2 antibody (right). E Physical interaction between Nurr1 and Foxa2 was further assessed by a proximity ligation assay (PLA). The SN area of a midbrain section (10 weeks old) was subjected to the PLA reaction and counterstained for TH. The boxed area in the left panel exhibiting physical Nurr1/Foxa2 interaction (red) in TH+ DA neurons (green) is enlarged in the right panel. F, G Knockdown of Nurr1 and Foxa2 synergistically aggravates H2O2-induced cell death of mDA neurons in culture. Representative image for Nurr1- and Foxa2-co-expressing mDA neurons (F) used in the loss-of-function study. Shown in right panels are the individual Nurr1-, Foxa2-, and TH-stained cells of the boxed area. Nurr1- and Foxa2-co-expressing mDA neurons were formed after 9 days of differentiation in VM-NPC cultures in vitro and transduced with lentiviruses expressing shNurr1 (shN), shFoxa2 (shF), shN + shF (shNF), or shControl (shC). Three days later, the cultures were treated with H2O2 (500 μM) for 8 h and cells positive for TH (upper) and cleaved (activated) caspase-3 (lower) were counted the following day (G). Insets, high-power TH+ cell images of the boxed areas. Significantly different from the control (shC)*, shN#, shF‡ at P < 0.05, n = 5 culture wells in each group. P-values: 0.038 (shN*), 0.026 (shF*), 0.013 (shNF*), 0.036 (shNF#), and 0.042 (shNF‡) for the % TH+ cells; 0.043 (shF*), 0.037 (shNF*), 0.019 (shNF#), and 0.029 (shNF‡) for the percent cleaved caspase-3-positive cells; one-way ANOVA followed by Bonferroni post hoc test. \n",
      " - Assessment of mitoribosomal misreading in in-organello translation.Figure 2A Autoradiography of in-organello mitochondrial translation products derived from HEK293 wild-type cells in the presence of tobramycin, incorporation of 35S-Met and 35S-Cys in MT-CO1 protein. Following in-organello translation proteins were immunoprecipitated and analyzed by autoradiography. Lane 1, 3: controls; lanes 2, 4: in-organello translation in the presence of 1000 µM tobramycin. MT-CO1 bands were quantified and ratio of 35S-Cys / 35S-Met calculated, the 35S-Cys / 35S-Met ratio in the absence of tobramycin was set as 1 (n=3), error bars indicate SEM. B Ratios of 35S-Cys / 35S-Met labelled MT-CO1 and MT-CO2 proteins synthesized in in-organello translation and effect of tobramycin. Following in-organello translation in the presence of tobramycin (1000 µM, 2000 µM) proteins were immunoprecipitated and analyzed by autoradiography. MT-CO1 and MT-CO2 bands were quantified and ratio of 35S-Cys / 35S-Met incorporation calculated (n=3), error bars indicate SEM; *p<0.05, ns, not significant (Student's t-test). Ratio of 35S-Cys and 35S-Met labelled MT-CO1 protein synthesized in in-organello translation using mitochondria from HEK293 MRPS5 wild-type cells and HEK MRPS5 mutant V336Y cells. Following in-organello translation, proteins were immunoprecipitated and analyzed by autoradiography. MT-CO1 bands quantified and ratios calculated. The ratio of the MRPS5 wild-type was set as 1 (n=8 clones), error bars indicate SD; ***p<0.0001 (Student's t-test). D Autoradiography of in-organello mitochondrial translation products derived from HEK293 MRPS5 wild-type cells in the presence of tobramycin (0-750 µM). Proteins were 35S-Met labeled and loaded either directly on SDS-PAGE (total, top) or following poly-lysine immunoprecipitation (bottom). * MT-CO1 and ** MT-CO1 with poly-lysine extension. E Quantification of MT-CO1 proteins in HEK293 MRPS5 wild-type cells and HEK293 MRPS5 mutant V336Y cells. In-organello mitochondrial translation in the presence of 35S-Met and 750 µM tobramycin. Proteins were loaded either directly on SDS-PAGE (total) or following poly-lysine immunoprecipitation and analyzed by autoradiography. For total protein samples the MT-CO1 band was quantified, for poly-lysine immunoprecipitated samples the extended MT-CO1 band was quantified and the ratio of extended MT-CO1/total MT-CO1 calculated. The ratio of MRPS5 wild-type was set as 1 (MRPS5 wild-type, n=5 clones, MRPS5 V336Y, n=7 clones), error bars indicate SD; ***p<0.0001 (Student's t-test). \n",
      " - Determination of translational accuracy in cell-free translation assays.Figure 1A, Misreading (A) were measured by dual luciferase gain-of-function assays using wild-type mitohybrid ribosomes (n=8 misreading, mutant A1555G mitohybrid ribosomes (n=8 misreading, E. coli wild-type ribosomes (n=9), E.coli mutant A127Y ribosomes (n=9), M. smegmatis merodiploid wild-type ribosomes (n=9), and M. smegmatis merodiploid mutant S152Y ribosomes (n=9). Mistranslation was determined by assessing the misreading at His245 (CAC codon) of the F-luc gene that was replaced by the near-cognate codon CGC or the non-cognate codon AGA, both encoding for Arg. Results are derived by calculating mutant hFluc/hRluc activity related to wild-type hFluc/hRluc activity. read-through (B) were measured by dual luciferase gain-of-function assays using wild-type mitohybrid ribosomes n=4 read-through), mutant A1555G mitohybrid ribosomes n=4 read-through), E. coli wild-type ribosomes (n=9), E.coli mutant A127Y ribosomes (n=9), M. smegmatis merodiploid wild-type ribosomes (n=9), and M. smegmatis merodiploid mutant S152Y ribosomes (n=9). To determine read-through the Asp357 (GAC codon) of the F-luc gene was replaced by the nonsense TGA stop codon. Results are derived by calculating mutant hFluc/hRluc activity related to wild-type hFluc/hRluc activity. C Ratio of 35S-Cysteine and 35S-Methionine incorporation by in-vitro translation using wild-type mitohybrid ribosomes, mutant A1555G mitohybrid ribosomes, and MT-CO1 mRNA. Following immunoprecipitation, the MT-CO1 band was quantified by autoradiography, wild-type mitohybrid ribosomes were set as 1 (n=3), error bars indicate SEM; ****p<0.001 (Student's t-test). D Ratio of 35S-Cysteine and 35S-Methionine incorporation by in-vitro translation in the presence of tobramycin using wild-type mitohybrid ribosomes, mutant A1555G mitohybrid ribosomes, and MT-CO1 mRNA; the MT-CO1 band was quantified by autoradiography. E Misreading of wild-type mitohybrid ribosomes and mutant A1555G mitohybrid ribosomes in the presence of tobramycin. Misreading was assessed by dual luciferase assay, and results derived by calculating mutant hFluc/hRluc activity related to wild-type hFluc/hRluc activity. F In-vitro translation of MT-CO1-TGA mRNA and MT-CO1-AGA mRNA using mutant A1555G mitohybrid ribosomes in the presence of tobramycin (0 - 20 µM). Autoradiography of immunoprecipitated MT-CO1 proteins; top: 35S-Met labelling, bottom: 14C-Lys labelling. In-organello translated 35S-Met labelled MT-CO1 was used as a marker. * MT-CO1, ** MT-CO1 extended by read-through, MT-CO1 AGA-polyA and MT-CO1 TGA-polyA constructs used for in-vitro translation are schematized. G Ratio of 14C-Lys and 35S-Met labelled immunoprecipitated MT-CO1 proteins. In-vitro translation using MT-CO1-TGA mRNA, MT-CO1-AGA mRNA, and mutant A1555G mitohybrid ribosomes in the presence of tobramycin (0 - 20 µM). The corresponding MT-CO1 band was quantified by autoradiography. \n",
      "Schematic representation of mDA neuron survival promoted by forced Nurr1 + Foxa2 gene delivery.Figure 6Nurr1 and Foxa2 are endogenously expressed in mDA neurons and promote cellsurvival in a cell-autonomous manner (left). Expression of Nurr1 and Foxa2 is reduced in mDA neurons during aging and degenerative processes. In addition, glia come to be polarized toward the harmful M1-type. DA neurons with reduced Nurr1/Foxa2 expression are more vulnerable to the toxic environment in the aged or degenerating midbrain (middle). Forced expression of Nurr1 + Foxa2 in mDA neurons promotes their survival. In addition, Nurr1 + Foxa2 transgene expression in glial cells (M2-type) creates a neuroprotective environment by suppressing the secretion of pro-inflammatory cytokines as well as promoting neurotrophic factor release (right). \n",
      "Mechanisms for the paracrine neuroprotective roles by Nurr1- and Foxa2-expressing glial cells.Figure 4A Real-time PCR analysis for pro-inflammatory cytokines and neurotrophic factors. BV2 microglia and primarily cultured astrocytes were transduced with C, N, F, and NF. Real-time PCR analyses were carried out 12 h after LPS (100 ng/ml, for pro-inflammatory cytokine expressions), IL-4 (10 ng/ml, other M2 markers), or without the treatment (neurotrophic factors). Significantly different from the control (C)*, N#, F‡ at P < 0.05, n = 3-6 PCRs; one-way ANOVA followed by Bonferroni post hoc test. B, C NO levels released (B) and arginase activity (C) were measured in the media and cells for BV-2 microglia cultures, respectively. Significantly different from the control (C)*, N#, F‡ at P < 0.05, n = 3 reactions, P-values: 0.027 (N-CM*), 0.033 (F-CM*), 0.018 (NF-CM*), and 0.033 (NF-CM‡) for the NO level, 0.011*, 0.025#, and 0.036‡ for the arginase activity; one-way ANOVA followed by Bonferroni post hoc test. \n",
      ": Generation and characterisation of recombinant CSPSIINFEKL and UIS4SIINFEKL P.Figure 1(A) Schematic overview of transgenic parasite lines. To generate CSPSIINFEKL, SIINFEKL replaced amino acids SYIPSAEK in CSP. To generate UIS4SIINFEKL, SIINFEKL was adjoined to the carboxyl-terminus of the UIS4 protein. (B) Sporozoite immunofluorescent antibody staining of WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites after gliding on BSA-coated glass slides. Shown are microscopic images of the respective sporozoites that were stained with anti-CSP (green), anti-UIS4 (red) and nuclear stain Hoechst 33342 (blue). Scale bars, 10μm. The numbers show mean percentage (±SD) of sporozoites with trails assessed from ≥220 sporozoites. (C) Fluorescent-microscopic images of EEF-infected Huh7 hepatoma cells. 24 and 48 hours after infection with WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites, the cells were fixed and stained with anti-UIS4 (red), anti-HSP70 (green) and the nuclear stain Hoechst (blue). Scale bars: 10µm. The numbers show mean numbers (±SD) of intracellular parasites counted per well of 8-well Labtek slides. Data information: The data shown is representative from one of two independent experiments. \n",
      ": Sporozoite surface antigen induces a higher CD8+ T cell response than EEF vacuolar membrane antigen in the spleen and liver.Figure 3(A) Schematic overview of experimental design. C57BL/6 mice received 2x106 OT-I cells alone (n=2-4) or were additionally immunised with 10,000 γ-radiation attenuated WT (n=3-6), CSPSIINFEKL (n=4) or UIS4SIINFEKL (n=4) sporozoites intravenously. See Appendix Table S2 and S3 for the number of mice used for pentamer staining and peptide restimulation. Spleens and livers were harvested either at day 14 or day 42. (B Proportions and numbers of (B- Kb-SIINFEKL+ CD8+ T cells were enumerated, or (E- IFN-γ-secreting CD8+ T cells following restimulation ex vivo with SIINFEKL peptide were quantified. Flow cytometry plots show representative percentages of CD8+ T cells co-stained with CD11a and (B) Kb-SIINFEKL or (E) IFN-γ. Proportions and numbers of Kb-SIINFEKL+ CD8+ T cells were enumerated, or IFN-γ-secreting CD8+ T cells following restimulation ex vivo with SIINFEKL peptide were quantified. The upper panel of bar charts (C show the percentage of co-stained CD8+ T cells and the lower panel (D the absolute cell counts. Bar charts show mean values (±SEM) from representative experiments (*, p<0.05; **, p<0.01; ***, p<0.001; one-way ANOVA with Tukey's multiple comparison test). See Appendix Table S4 for exact p-values. Proportions and numbers of IFN-γ-secreting CD8+ T cells following restimulation ex vivo with SIINFEKL peptide were quantified. The upper panel of bar charts F) show the percentage of co-stained CD8+ T cells and the lower panel G) the absolute cell counts. Bar charts show mean values (±SEM) from representative experiments (*, p<0.05; **, p<0.01; ***, p<0.001; one-way ANOVA with Tukey's multiple comparison test). See Appendix Table S4 for exact p-values. \n",
      ": Higher magnitude of endogenous SIINFEKL-specific CD8+ T cell responses induced by a sporozoite antigen.Figure 4(A) Schematic overview of experimental design. (B- C57BL/6 mice received 10,000 γ-radiation attenuated WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites, either intravenously (n=6-8 mice per group) or (E- intradermally (n=4 mice per group). Additional control mice did not receive sporozoites (n=6-8 intravenously, n=4 intradermally). Spleens and livers were harvested either at day 14 (n=5 livers and n=6 spleens for intravenous immunisation) or day 42 (n=8 for intravenous immunisation), and IFN-γ-secreting lymphocytes following restimulation ex vivo with SIINFEKL peptide were quantified. Flow cytometry plots show representative percentages of CD8+ T cells co-stained with IFN-γ and CD11a (B, E). C57BL/6 mice received 10,000 γ-radiation attenuated WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites, either intravenously (n=6-8 mice per group) Additional control mice did not receive sporozoites (n=6-8 intravenously, n=4 intradermally). Spleens and livers were harvested either at day 14 (n=5 livers and n=6 spleens for intravenous immunisation) or day 42 (n=8 for intravenous immunisation), and IFN-γ-secreting lymphocytes following restimulation ex vivo with SIINFEKL peptide were quantified. The upper panel of bar charts (C, show the percentage of CD11ahi IFN-γ+ CD8+ T cells and the lower panel (D the absolute cell counts. Bar charts show mean values (±SEM) from representative experiments (*, p<0.05; **, p<0.01; ***, p<0.001; one-way ANOVA with Tukey's multiple comparison test). See Appendix Table S4 for exact p-values. C57BL/6 mice received 10,000 γ-radiation attenuated WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites intradermally (n=4 mice per group). Additional control mice did not receive sporozoites (n=6-8 intravenously, n=4 intradermally). Spleens and livers were harvested either at day 14 (n=5 livers and n=6 spleens for intravenous immunisation) or day 42 (n=8 for intravenous immunisation), and IFN-γ-secreting lymphocytes following restimulation ex vivo with SIINFEKL peptide were quantified. The upper panel of bar charts show the percentage of CD11ahi IFN-γ+ CD8+ T cells and the lower panel the absolute cell counts. Bar charts show mean values (±SEM) from representative experiments (*, p<0.05; **, p<0.01; ***, p<0.001; one-way ANOVA with Tukey's multiple comparison test). See Appendix Table S4 for exact p-values. \n",
      ": Increasing antigen dose does not improve antigen-specific CD8+ T cell responses to an EEF vacuolar membrane protein.Figure 5(A) Schematic overview of experimental design. C57BL/6 mice (n=4 per group) received an intravenous dose of 8,000 γ-radiation attenuated CSPSIINFEKL or UIS4SIINFEKL sporozoites or 64,000 γ-radiation attenuated UIS4SIINFEKL sporozoites. Spleens and livers were harvested at day 12 and IFN-γ-secreting lymphocytes following restimulation ex vivo with SIINFEKL peptide were quantified. (B) Flow cytometry plots show representative CD8+ T cells co-stained with IFN-γ and CD11a. (C) The upper panel of bar charts show the percentage of CD11ahi IFN-γ+ CD8+ T cells and the lower panel the absolute cell counts. Bar charts show mean values (±SEM) from representative experiments (***, p<0.001; one-way ANOVA with Tukey's multiple comparison test). See Appendix Table S4 for exact p-values. \n",
      ": Sporozoite antigen exposure elicits superior peripheral blood CD8+ T cell responses.Figure 2(A) Schematic overview of experimental design. C57BL/6 mice received 2x106 OT-I cells alone or were additionally immunised with 10,000 γ-radiation attenuated parasites intravenously. (B) Flow cytometry plots show the gating strategy for identifying Kb-SIINFEKL+ CD11ahi CD8+ T cells. (C) Temporal analysis of antigen-specific CD8+ T cell responses. Peripheral blood was obtained on days 4, 7, 14, 21, 42 and 88 post-immunisation with WT (triangles; n=3-9), CSPSIINFEKL (orange squares; n=4-8) or UIS4SIINFEKL (blue circles; n=4-10) sporozoites, or no parasites (diamonds; n=2-5) and stained for Kb-SIINFEKL+ CD11ahi CD8+ T cells. Line graph shows mean values (±SEM) from representative experiments. (*, p<0.05; **, p<0.01; ***, p<0.001; Welch's t-test comparing CSPSIINFEKL and UIS4SIINFEKL). See Appendix Table S1 for the number of mice used per timepoint and per group. See Appendix Table S4 for exact p-values. (D-H) C57BL/6 mice (n=4 per group), which received 2x106 CFSE-labelled OT-I splenocytes, were immunised with 10,000 γ-radiation attenuated WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites intravenously. 5 days later, mice were sacrificed, spleens harvested and splenocytes assessed for (D) CFSE dilution of CD8+ T cells, (E) CFSE dilution of antigen experienced Kb-SIINFEKL+ CD11ahi CD8+ T cells and stained ex vivo (F-H) for effector CD8+ T cell surface markers. Shown are flow cytometry plots of Kb-SIINFEKL co-staining with markers of effector phenotypes: (F) CD11ahi, (G) CD62Llo and (H) CD49dhi. \n",
      "Reduction of Nurr1 and Foxa2 in mDA neurons during aging and degeneration.Figure 2A-C Nurr1 and Foxa2 protein levels in mDA neurons decrease in the midbrains of old mice in vivo (A) Nurr1 and Foxa2 protein levels were compared in individual mDA neurons of the midbrains of young (10 weeks) and old mice (18 months) of the same mouse strain (C57BL/6, male). All the midbrain sections were immunofluorescently co-stained with Nurr1/TH (upper) and Foxa2/TH (lower) under identical conditions, and levels of Nurr1 and Foxa2 proteins were determined in individual TH+ mDA neurons by measuring mean fluorescence intensities (MFI) using LAS image analysis (Leica). Dots in the graphs represent the Nurr1 and Foxa2 MFI values of individual TH+ DA neurons in the SN of each animal. The average MFI values (indicated by horizontal lines) of three animals from each group were compared (***P = 5.25E-88 for Nurr1 intensity, 5.57E-40 for Foxa2 intensity, one-way ANOVA followed by Bonferroni post hoc test). A-C Nurr1 and Foxa2protein levels in mDA neurons decrease in vitro after long-term culture (B and C).Nurr1 and Foxa2protein levels were also quantified in cultured mDA neurons over 6-34 days in vitro by Western blotting (B) A-C Nurr1 and Foxa2protein levels in mDA neurons decrease in vitro after long-term culture (B and C).Nurr1 and Foxa2protein levels were also quantified in cultured mDA neurons over 6-34 days in vitro immunocytochemical analysis (C). Significantly lower MFI values on day 24 of culture (D24) compared to D12 at *P = 0.027 (Nurr1), *P = 0.012 (Foxa2), n = 60-70 TH+ cells from two cultures in each group, unpaired Student's t-test. D, E Loss of Nurr1 and Foxa2 expression in mDA neurons after treatment with the neurotoxin MPTP (or MPP+). Mice (10 weeks old) were treated with MPTP for 5 days as described in Materials and Methods. Three days after the last MPTP injection, Nurr1 and Foxa2protein levels in the TH+mDA neurons of the MPTP-treated SN were compared with in the mDA neurons of untreated mice (D) (***P = 5.47E-103 for Nurr1 intensity, 1.53E-111 for Foxa2 intensity, one-way ANOVA followed by Bonferroni post hoc test.). The effects of neurotoxin treatment were also determined in mDA neuron cultures treated with MPP+ (250 μM, 8 h, E). *P = 0.015 (% Nurr1+/TH+ cells), *P = 0.018 (% Foxa2+/TH+ cells), unpaired Student's t-test. \n",
      "Therapeutic effects of AAV-mediated gene delivery of Nurr1 + Foxa2 in the mouse PD model.Figure 5A Experimental scheme. As shown, PD model mice were generated by IP injection of MPTP, and AAVs expressing Nurr1 and Foxa2 (NF-AAV) or AAVs expressing empty vector (control, C-AAV) were stereotaxically injected 3 days prior to the initial MPTP treatment. B-E Cytoprotective and behavioral effects of NF expression were assessed in PD mice unilaterally injected with NF-AAVs on the right side of midbrains. C-AAVs were injected into the left midbrains of the same mice. TH+mDA neuron numbers in the midbrain (B) and TH+fiber intensities in the striatum (C) of the NF-AAV-injected right sides were compared with those of the corresponding left sides after 8 weeks. Insets of (B), high-powered images of the areas indicated by red arrowheads in the C-AAV (left) and NF-AAV-injected (right) sides. TH+cell numbers, *P = 0.000078, n = 7, paired Student's t-test. The sectioned striatal tissues were immunofluorescently stained for TH, and TH+fiber intensities were estimated from mean fluorescence intensities (MFI). n = 56 microscopic fields at a magnification of 200× each for the left and right striatum. *P = 0.000003, paired Student's t-test. Behavioral asymmetry in the mice unilaterally injected with NF-AAVs (right side) was assessed using the apomorphine-induced rotation test (D) after 4 and 8 weeks. The cylinder test (E) was carried out after 8 weeks to determine left forelimb movement. As a control, the behaviors were tested in PD mice bilaterally injected with C-AAV. *P = 0.028 (rotation test), 0.043 (cylinder test); one-way ANOVA followed by Bonferroni post hoc test. F-L Behaviors of the PD mice injected with NF-AAVs bilaterally at both sides of the midbrain were assessed using the pole test (J), beam test (K), and locomotor test (L). *P = 0.004 (pole), 0.044 (beam), and 0.045 (locomotor); one-way ANOVA followed by Bonferroni post hoc test. Shown in (F-I) is a representative TH-stained midbrain (F, H) and striatum (G, I) of mice bilaterally injected with C-AAVs (F, G) and NF-AAVs (H, I). M, N Sustained mDA neuroprotective effects. Shown in (M) is a confocal image of TH/HA-stained cells 6 months after NF-AAV injection (Foxa2 gene tagged with HA). Inset, z-stacked image of the boxed area along the y-axis (right) and x-axis (lower). TH+ cells in the right (NF-AAV) and left (C-AAV injected) midbrains were counted 6 and 12 months after AAV injection (N). *P = 0.000019 (6M), 0.000066 (12M), n = 6-7, paired Student's t-test. O, P NF-AAV-mediated mDA neuroprotective effects were further assessed with the experimental schedule (O), in which AAV injections were subjected after three consecutive MPTP injections. Shown in (P) is TH+cell counts of the left and right sides of the midbrains. *P = 0.000012 (D14), 0.000036 (1M), 0.000027 (2M), n = 6-7, paired Student's t-test. \n",
      "Forced expression of Nurr1 and Foxa2 protects mDA neurons from toxic insults in vitro.Figure 3A Representative image for TH+ (mDA neuron), GFAP+ (astrocyte), and Iba-1+ (microglia) cells in mDA neuron-glia cultures used in the gain-of-function experiments. B mDA neurons in the cultures transduced with Nurr1 and/or Foxa2 are resistant to toxic stimuli induced by H2O2 (50-500 μM), 6-OHDA (50-1,000 μM), or MPP+ (50-500 μM). The mDA neuron-glia cultures were transduced with lentiviruses expressing Nurr1 (N), Foxa2 (F), Nurr1 + Foxa2 (NF), or control (C) and treated with the toxin for 8 h. Viable TH+ cells were counted on the following day. Shown in the graphs are percent TH+ cells relative to the respective toxin-untreated cultures. TH+ cells of C, N, F, and NF were compared at the same concentrations of the toxins. Significantly different from the control (C)*, from N#, and from F‡ at P < 0.05, n = 10 cultures each; one-way ANOVA followed by Bonferroni post hoc test. C, D Forced expression of Nurr1 and Foxa2 in glia exerts neuroprotective roles on mDA neurons in a paracrine mode. Experimental scheme (C) to test the effects of paracrine factors released from Nurr1-/Foxa2-expressing glia. Mixed astrocytes + microglia cultures (derived from VM tissue of mousepups on postnatal day 1) were transduced with N, F, NF, or C, and conditioned media (CM) were prepared from the transduced glia and added to mDA neuron cultures. Two days after the CM treatment, H2O2 (500 μM, 8 h)-mediated cell death was measured by counting viable TH+ cells (D). Significantly different from the control (C-CM)*, N-CM#, F-CM‡ at P < 0.05. P-values: 0.031 (N-CM*), 0.021 (F-CM*), 0.017 (NF-CM*), 0.022 (NF-CM#), and 0.027 (NF-CM‡); one-way ANOVA followed by Bonferroni post hoc test. E-G NF-expressing glia secrete factors that reduce oxidative stress by inducing Nrf2-mediated antioxidant gene expressions. Oxidative stress measured by DCF (oxidative stress indicator) staining (E). Primary cultured mDA neurons were pre-treated with C-CM (upper) or NF-CM (lower) for 2 days and then exposed to H2O2 (250 μM) in the presence of the CM. Four hours later, DCF staining was carried out. E-G NF-expressing glia secrete factors that reduce oxidative stress by inducing Nrf2-mediated antioxidant gene expressions. Oxidative stress measured by DCF (oxidative stress indicator) staining. (F) Expression array for antioxidant genes. mDA neuron cultures were pre-treated with C-CM, N-CM, F-CM, or NF-CM for 2 days before exposure to 500 μM H2O2 for 8 h, and mRNA expression levels of 39 antioxidant genes were estimated using an RT2 PCR Profiler ArrayR. Volcano plot (left) demonstrating a tendency for increased antioxidant gene expression in mDA neuron cultures with NF-CM treatment. The pink lines indicate the threshold of 1.5-fold changes in gene expression. The 23 antioxidant genes up-regulated 1.5-fold (circled) by NF-CM treatment relative to C-CM treatment are listed on the bar graph (right). E-G NF-expressing glia secrete factors that reduce oxidative stress by inducing Nrf2-mediated antioxidant gene expressions. Oxidative stress measured by DCF (oxidative stress indicator) staining. (G) NF-CM treatment of mDA neurons activates cytosolic Nrf2proteins (0 min) by inducing nuclear translocation (30 min). Shown in the graph are cytosolic and nuclear Nrf2protein levels (MFI) 30 min after C-CM, N-CM, F-CM, and NF-CM treatments. *P < 0.05, n = 50-60 cells in each group. P-values: 0.032 (N-CM*), 0.029 (F-CM*), 0.019 (NF-CM*) for the Nrf2 intensity in nucleus, 0.029 (F-CM*), 0.039 (NF-CM*) for the Nrf2 intensity in cytosol; one-way ANOVA followed by Bonferroni post hoc test. \n",
      ": Both sporozoite surface and EEF vacuolar membrane antigens are susceptible to vaccine-induced CD8+ T cells for killing, resulting in sterile protection.Figure 6(A) Schematic overview of experimental design. Mice received 2x106 OT-I splenocytes, and one day later they were either vaccinated with 1x108 ifu recombinant AdHu5 expressing whole ovalbumin (AdOVA) (n=5 per challenge group) or left untreated (n=4 per challenge group). (B, C) Flow cytometry (B) and scatter plots (C) represent CD8+ T cells derived from peripheral blood co-stained with IFN-γ and CD11a, following ex vivo restimulation with SIINFEKL. Data information: Data are representative of two experiments performed with scatter plots showing mean values (±SEM). (D) Protective efficacy as measured by quantitative real-time PCR. Groups of mice were vaccinated as described and challenged 19 days later with 10,000 WT (n=7 non-vaccinated, n=9 vaccinated), CSPSIINFEKL (n=6 non-vaccinated, n=4 vaccinated) or UIS4SIINFEKL sporozoites (n=3 non-vaccinated, n=11 vaccinated). 42 hours later livers were removed, and parasite load was assessed by qPCR. Plots show the relative parasite load of mice in each condition (**, p<0.01; Mann-Whitney U test). See Appendix Table S4 for exact p-values. Data information: Data are representative of two experiments performed with scatter plots showing mean values (±SEM). (E) Proportion of sterile protection after immunisation. Mice (vaccinated group, n=8; control group, n=4) were vaccinated as described and were challenged with 800 WT, CSPSIINFEKL or UIS4SIINFEKL sporozoites. Daily blood smears were taken from day 3-14 post challenge to check for parasitaemia. Data information: Data are representative of two experiments performed with scatter plots showing mean values (±SEM). \n",
      ".Figure 2Three epithelial/resistant non-small cell lung cancer cell lines were treated with 5 ng/ml TGF-β for 14 days to induce a mesenchymal phenotype, which was confirmed with Western blot (left) and qPCR (right) analysis of epithelial-to-mesenchymal transition (EMT) markers (A). Parental and TGF-β isogenic cell lines were treated with 50nM volasertib for 72 hours. Cells were then harvested, and lysates were immunoblotted for cleaved PARP and γH2AX proteins were subsequently quantitated and normalized with β-actin (B). Apoptosis was measured by the Apo-BrdU assay (C) DNA damage was measured by the Comet assay (D). \n",
      ".Figure 1Spearman correlations between protein expression and sensitivity to Plk1 inhibitors (BI2536, GSK461364, BRD-K70511574, and GW-843682X), based on data from the Cancer Therapeutics Response Portal v2 database and protein expression data derived from the MD Anderson Cell Line Project database (Li et al, 2017). Proteins with expression levels that correlate with area under the curve (AUC) values for at least two drugs are included in the graph. The color and size of the data point represent the direction and significance (p value) of the correlation. Red indicates a positive correlation with drug resistance and green a positive correlation with drug sensitivity (i.e., proteins in red are more highly expressed in resistant cell lines). Spearman correlations between protein expression and sensitivity to Plk1 inhibitors (BI2536 GSK461364 AUC values for two Plk1 inhibitors are compared with expression of selected proteins (cMet, E-cadherin, β-catenin, and Snail); each data point represents an individual non-small cell lung cancer cell line. E-cadherin, β catenin, and cMet protein expression correlated with drug resistance. \n",
      ".Figure 3Three pairs of TGF-β-treated isogenic NSCLC cell lines (H1975, HCC4006, and HCC366) and two mesenchymal NSCLC cell lines (Calu6 and H1792) were incubated with 50nM volasertib for 24 hours, lysed, and subjected to reverse phase protein array analysis. Experiments were done in triplicate. Thirty-three proteins were differentially regulated between epithelial and mesenchymal NSCLC cell lines after treatment with volasertib, including those involved in the cMet/FAK/Src signaling axis (blue text) and those involved in the PI3K/Akt signaling axis (orange text). The same cell lines treated identically were subjected to immunoblotting for the indicated proteins (upper) with densitometric quantification normalized with β-actin (lower). Epithelial (red text) and mesenchymal (blue text) NSCLC cell lines transfected with 10nM siRNA as indicated for 48 hours. NT, non-targeting control and subjected to Western blotting with the indicated antibodies. Data are means ± standard error of the mean from three independent experiments. Significant differences using two-sided Student t test are indicated by *p < 0.05. H1975 and Calu6 cells were treated with indicated inhibitors for 4 hours or 24 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. \n",
      ".Figure 4Schematic of the experimental plan for Plk1 and cMet inhibition/silencing for viability and apoptosis studies in mesenchymal and epithelial NSCLC cell lines. Cell viability was measured using CellTiter-Glo in NSCLC cell lines treated with volasertib, tepotinib, or a 1:2 ratio of both for 72 hours. Left: representative cell viability graph of cells treated with the indicated drugs. Right: heatmap depicting the calculated combination indices at Fa = 0.25 and Fa = 0.5. Apoptosis was measured using the Apo-BrdU assay in the indicated cell lines treated with the indicated drugs for 24 hours. Immunoblots from cells treated with indicated drugs for 24 hours (left) with densitometric quantification normalized with β-actin (right). All tested cell lines were treated as indicated for 24 hours and allowed to grow in drug-free medium for 15-20 days to form colonies, which were counted using ImageJ. Apoptosis was measured using the Apo-BrdU assay in the indicated cell lines transfected with 10nM siRNA as indicated for 48 hours. NT, non-targeting control. Immunoblots from cells transfected with siRNA as indicated for 48 hours (left) with densitometric quantification normalized with β-actin (right). Data are means ± standard error of the mean from three independent experiments. Significant differences using two-way analysis of variance with Bonferroni or Benjamini-Hochberg (BH) correction for multiple comparison are indicated (*p < 0.01). Mesenchymal and epithelial NSCLC cell lines are indicated in blue and red text, respectively. \n",
      ".Figure 6Calu6, H157, and H1355 TPR-Met-expressing or vector control (pBABE)-expressing cells were treated as indicated for 24 hours and apoptosis was measured using the Apo-BrdU assay. Parental and TPR-Met-expressing cell lines were treated with the indicated drugs for 24 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. β-actin was used as a loading control. Parental and TPR-Met-expressing cell lines were treated as indicated for 24 hours and allowed to grow in drug-free medium for 15-20 days to form colonies, which were counted using ImageJ. Data are means ± standard error of the mean from three independent experiments. Significant differences using two-way analysis of variance with Bonferroni or Benjamini-Hochberg correction for multiple comparisons are indicated. Mesenchymal NSCLC cell lines with the noted MET alterations were incubated with 50nM volasertib for four hours and subjected to immunoblotting with the indicated antibodies. *p < 0.05. **p < 0.01. \n",
      ".Figure 7Cell viability measured by CellTiter-Glo in Calu6 parental and Calu6-VAR cells treated with different concentrations of volasertib for 72 hours. Micrograph shows acquisition of a mesenchymal-to-epithelial transition phenotype in VAR cells. Basal protein and mRNA expression in parental and VAR cells for the indicated proteins was measured using immunoblotting (upper) and reverse-transcriptase PCR (lower), with GAPDH expression used for normalization. Parental and VAR cells were treated as indicated for 48 hours and apoptosis was measured using the Apo-BrdU assay. Parental and VAR cells were treated with the indicated drug concentrations for 48 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Parental and VAR cells were treated as indicated for 24 hours and allowed to grow in drug-free medium for 15-20 days to form colonies, which were counted using ImageJ. \n",
      ".Figure 5Mice bearing epithelial (red text) and mesenchymal (blue text) non-small cell lung cancer tumors were treated with vehicle control, volasertib (30 mg/kg per week intravenously), tepotinib (25 mg/kg per day orally), or the combination for 5 weeks to generate tumor growth curves of patient-derived xenografts (PDX; A) and cell line xenografts The percent change in tumor volume at the end of treatment (normalized to day zero) was calculated with each data point representing an individual mouse, and the bar is the mean value ± standard error of 10 mice for each group Tumor growth curve of mesenchymal PDX TC424 (top) and cell line xenograft Calu6 (bottom) upon treatment for 5 weeks and after cessation of treatment (recovery phase). Data are means ± standard error of the mean from five mice for each group. Kaplan-Meier survival curves for mesenchymal PDX TC424 (top) and cell line xenograft Calu6 (bottom) with death due to excessive tumor burden as the endpoint. Quantification of TUNEL-positive cells in paraffin-embedded xenograft tissue sections. \n",
      ".Figure 8Epithelial and mesenchymal non-small cell lung cancer (NSCLC) cell lines after treatment with 50nM volasertib for 24 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Epithelial and mesenchymal NSCLC cell lines after Plk1 silencing with siRNA for 48 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Calu6 and H1792 mesenchymal NSCLC cell lines after VIM silencing with siRNA for 48 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Calu6 (mesenchymal) and H1975 (epithelial) NSCLC cell lines after Plk1 and INTB1 silencing with siRNA for 48 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Calu6 (mesenchymal) and H1975 (epithelial) NSCLC cell lines after treatment with fibronectin for 24 hours. Cells were then harvested, and lysates were immunoblotted for the indicated proteins. Calu6 and H1975 NSCLC cell lines after treatment with 50nM volasertib for 24 hours. Cells were then harvested, and lysates were immunoprecipitated (IP) using β1-integrin, vimentin, and Plk1 antibodies, then immunoblotted for the indicated proteins. Schematic summary of the proposed mechanism by which cMet drives primary and acquired resistance to volasertib in NSCLC and possible treatment combinations to overcome this resistance. \n",
      ".Figure 1Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. Serum SUA level Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. body weight gain Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. fat Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. lean mass Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. gastrocnemius index. Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. Gastrointestinal muscle fiber immunofluorescent laminin staining Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. frequency histogram of fiber cross-sectional area. \n",
      ".Figure 3Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. mRNA expression of MyHC I, MyHC IIa, PGC-1α, myoglobin, TnnT1 MyHC IIb, MyHC IIx and TnnT3 in the gastrocnemius muscle (n=5-6). Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. Immunoblots and quantification of MyHC-I, IIa and IIb protein expression in gastrocnemius (n=3-4). Male C57BL/6J mice were fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. Representative images and quantification of Laminin (green), MyHC-I and IIb immunofluorescent staining (red) in gastrocnemius (n=3). Scale bar in (C) represents 100 µm. \n",
      ".Figure 2male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. The muscle grip strength male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. running time in low speed male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. four-limb handing time male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. running time in high speed. male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. serum concentration of RBC male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. HGB in whole blood male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. ex vivo gastrocnemius muscle force male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. ex vivo gastrocnemius muscle force male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. fatigability male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. glucose consumption male C57BL/6J mice fed with chow diet supplemented with 0, 0.5%, 1% SUC for 8 weeks. lactate production were tested. \n",
      ".Figure 4Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. The O2 consumption (VO2) and respiratory exchange ratio (RER) Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. Serum concentration of (E) NEFA in whole blood. Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. The enzymes activity of (F) SDH, (G) HK, and (H) LDH in gastrocnemius. Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. Immunoblots and quantification of of p-AMPK，PGC-1α and myoglobin in gastrocnemius. Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. Quantification of mitochondrial and electron transport chain (ETC) related genes expression in gastrocnemius. Male C57BL/6J mice were fed with chow diet supplemented with 0 and1% SUC for 6 weeks. OCRs were measured under basal condition in gastrocnemius. \n",
      ".Figure 5C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. Representative images of MyHC-I and IIb immunofluorescent staining (green) in C2C12 cells (n=16). C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. quantification of MyHC-I and IIb immunofluorescent staining (green) in C2C12 cells (n=16). C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. The enzymes activity of (C) SDH, (D) LDH C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. lactate production in C2C12 cells. C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. Quantification of mitochondrial DNA contents in C2C12 cells. C2C12 cells were treated with 0, 0.5 and 2 mM SUC for 48 h. Mitochondrial electron microscopy showed the (H) mitochondrial density, (I) mitochondrial coverage, and (J) average mitochondrial area in C2C12 cell. Scale bar in (A) represents 50 μm, scale bar in (G) represents 0.5 μm. \n",
      ".Figure 6SUNCR1 protein expression in the gastrocnemius from sedentary or post-running mice (n=4). The mRNA level of SUNCR1 in gastrocnemius and soleus (n=7-8). [Ca2+]i in C2C12 cells treated with 0 or 2 mM SUC (n=18-20). NFAT protein expression in nucleus and cytoplasm of gastrocnemius 0.5 h or 3 h after i. p. injection of 15 mg/kg succinate in C57BL/6J mice (n=4). SUNCR1 protein expression in C2C12 cells transfected with vector or siSUNCR1 (n=3). [Ca2+]i, in vector or siSUNCR1 transfected C2C12 cells treated with 0 or 2 mM SUC (n=5-6). enzymes activity (n=9-10) of (J) HK, (K) LDH, and (L) SDH in vector or siSUNCR1 transfected C2C12 cells treated with 0 or 2 mM SUC (n=5-6). Representative images and quantification of MyHC-I and IIb immunofluorescent staining (green) in C2C12 cells (n=3). Scale bar in (M) represents 50 μm. \n",
      ".Figure 7Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. The O2 consumption (VO2), RER Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. muscle grip strength Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. four-limb handing time Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. low speed running time. Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. The enzymes activity of (H) HK, (I) LDH, and (J) SDH in gastrocnemius. Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. Immunoblots and quantification of MyHC-I, IIb, NFAT, and PGC-1α protein in gastrocnemius. Male C57BL/6J or SUNCR1 KO mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. Representative images and quantification of Laminin (green), or MyHC-I and IIb (red) immunofluorescent staining in gastrocnemius muscle (n=3). Scale bar in (M) represents 100 µm. \n",
      ".Figure 8Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. Timeline of the experimental protocol. Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. SUNCR1 protein expression in gastrocnemius from mice transfected with shSUNCR1 lentivirus or LV-shScrambled (n=3). Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. The running time in low speed Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. four-limb handing time Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. muscle grip strength of both control and gastrocnemius-specific SUNCR1 knockdown mice. Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. The enzymes activity of (F) HK, (G) LDH, (H) SDH in gastrocnemius. Male C57BL/6J mice were injected with LV-shScrambled or shSUNCR1 lentivirus specifically into the gastrocnemius at 6 weeks of age. After two weeks of recovery, mice were fed with chow diet supplemented with 0 or 1% SUC for 6 weeks. Immunoblots and quantification of MyHC-I, IIb, NFAT, and PGC-1α protein in gastrocnemius (n=3). \n",
      ".Figure 1A Affinity measurements by biolayer interferometry (BLI) of bipNbs NM1267 and NM1268. NM1267 and NM1268 were applied with concentrations ranging from 5-0.625 nM and 20‑2.5 nM, respectively (illustrated with gradually lighter shades) on immobilized RBD derived from SARS-CoV-2 B.1 (RBDB.1). Global 1:1 fits are illustrated as dashed lines. Global 1:1 fits are illustrated as dashed lines and binding affinity (KD), association (kon) and dissociation constant (koff) determined for the individual bipNbs are summarized. B Stability analysis of bipNbs NM1267 and NM1268 was performed at time points T0 and T10 after storage at 37°C for ten days to induce accelerated aging. Protein unfolding was determined by fluorescence emission wavelengths shifts illustrated as fluorescence ratios (350 nm/ 330 nm) as first derivative. Protein aggregation status was measured by light intensity loss due to scattering illustrated as first derivative. Melting (Tm) and aggregation (TAgg) temperature are summarized as table for both time points. \n",
      ".Figure 2A-P Affinity measurements by BLI of bipNb NM1267 and NM1268 on recently identified RBD variants B.1.1.7 (Alpha) (A, I), B.1.351 (Beta) (B, J), P1 (Gamma) (C, K), B.1.617.2 (Delta) (D, L), B.1.429 (Epsilon) (E, M), P3 (Theta) (F, N), B.1.617.1 (Kappa) (G, O) and A.23.1 (H, P). NM1267 and NM1268 were applied with concentrations ranging from 5-0.625 nM and 20‑2.5 nM, respectively (illustrated with gradually lighter shades) on immobilized RBD variants. Global 1:1 fits are illustrated as dashed lines. Q Tabular summary of binding affinity (KD), association (kon) and dissociation constant (koff) determined for the individual RBD variants. \n",
      ".Figure 3A-F Neutralization potency of NM1267 and NM1268 were analyzed in Caco‑2 cells using the SARS-CoV-2 B.1 (A, D) SARS-CoV-2 B.1.351 (Beta) (B, E) and SARS-CoV-2 B.1.617.2 (Delta) (C, F). Infection normalized to virus-only infection control is illustrated as percent of infection (infection [%]). Data are presented as mean ± SEM of three (n = 3) biological replicates. G Tabular summary of IC50 values, calculated from a four-parametric sigmoidal model. \n",
      ".Figure 4A Schematic illustration of treatment scheme. Hemizygous K18-hACE2 mice were treated intranasally with 20 µg of NM1251 (n = 15) or NM1267 (n = 12) seven hours prior to infection with 3*103 PFU SARS-CoV-2 B.1. Weight loss were monitored for 14 days. Dashed line indicates humane endpoint and symbols represent mean ± SEM ****P < 0.0001, by two-way ANOVA with Sidak's multiple comparison test in C Hemizygous K18-hACE2 mice were treated intranasally with 20 µg of NM1251 (n = 15) or NM1267 (n = 12) seven hours prior to infection with 3*103 PFU SARS-CoV-2 B.1. survival were monitored for 14 days. , ****P < 0.0001, by log-rank test D Nasal swabs were collected from six mice per group (n = 6) at the indicated time points and viral load was determined by plaque-assay. Bars represent mean ± SEM. **P < 0.01, by unpaired t test \n",
      " - USP26 variants might be responsible for the origin of Klinefelter syndrome.Figure 1A The karyotype of KS patients and controls. B The cohorts used in this study. The first cohort that contained 108 unrelated KS patients with the classical 47, XXY karyotype was selected to perform WES, the second cohort that contained 354 unrelated KS patients was selected to perform Sanger sequencing in the coding regions of USP26, and the third cohort that contained 558 unrelated KS patients was selected to perform Sanger sequencing in the mutated haplotype (c.370-371insACA/494T>C/1423C>T) of USP26. C WES data analysis pipeline. A total of 179,579 sequence variants in 20,800 genes were identified in 108 unrelated KS patients, and among of them nonsense, missense, frameshift, and essential splice-site variants were further investigated. These variants were divided into four subtypes (high, likely high, medium, low), and most of the medium to high variants were localized on the X chromosome. The genes on the X chromosome that contained more than two types of medium to high variants were selected. As the extra paternal-origin X chromosome in KS might be caused by impaired recombination between sex chromosomes during meiosis I, only the testis-specific genes ESX1, LOC101059915, and USP26 were further investigated. The variant frequencies of these three genes in the KS patients were compared to the variant frequencies of these three genes in the 1000 Genomes Project. The frequency of USP26 variants had the highest fold increase. D The genomic localization of sequence variants in 108 KS patients. E Genes with more than two medium- to high-level variants on the X chromosome. The testis-specific genes are labeled in red. F The frequency difference between KS patients and the 1000 Genomes Project. \n",
      ".Figure 6A Schematic illustration of treatment scheme. B, Hemizygous K18-hACE2 mice were treated intranasally with 20 µg of NM1251 (n = 8), NM1267 (n = 9) or NM1268 (n = 9) seven hours prior to infection with 3*103 PFU SARS-CoV-2 B.1.617.2 (Delta). Weight loss were monitored for 14 days. Dashed line indicates humane endpoint and symbols represent mean ± SEM ****P < 0.0001 in orange between NM1251 and NM1267 and in purple between NM1251 and NM1268, *P < 0.1 in black between NM1267 and NM1268, by two-way ANOVA with Sidak's multiple comparison test C Hemizygous K18-hACE2 mice were treated intranasally with 20 µg of NM1251 (n = 8), NM1267 (n = 9) or NM1268 (n = 9) seven hours prior to infection with 3*103 PFU SARS-CoV-2 B.1.617.2 (Delta). survival were monitored for 14 days. ***P < 0.001 in orange between NM1251 and NM1267 and ****P < 0.0001 in purple between NM1251 and NM1268, by log-rank test \n",
      ".Figure 5Histopathological analysis of mice, which were intranasally treated with bipNb NM1267 or control Nb NM1251 and subsequently infected with 3*103 PFU SARS-CoV-2 B.1. A-B Serial tissue sections revealed severe inflammation (H&E) and numerous widespread SARS-CoV-2 RNA positive alveolar epithelia cells and macrophages (in situ hybridization (ISH)) in lungs of infected, control-treated mice. In infected and NM1267-treated animals no inflammation or only small focal areas with inflammation and a few SARS-CoV-2 RNA positive cells were observed. Scale bars represent 1 mm and 200 µm, respectively. Histopathological analysis of mice, which were intranasally treated with bipNb NM1267 or control Nb NM1251 and subsequently infected with 3*103 PFU SARS-CoV-2 B.1. C Quantification of lung damage was done in n = 3 animals per group and grading score of individual animals is presented as mean ± SD with **P < 0.01, by unpaired t test. \n",
      " - Fertile men with USP26 mutations produce XY aneuploid spermatozoa.Figure 7A, B Quantification of different types of spermatozoa in fertile men with or without USP26 mutations. The spermatozoa from fertile men without USP26 mutations (n = 10 independent experiments); fertile men contained the USP26 c.463C>T mutation (The sample was analyzed in three repeated experiments), the USP26 c.125T>C mutation (n = 2 independent experiments, one sample was analyzed in two repeated experiments), the USP26 c.1044T>A mutation (n = 5 independent experiments), and the USP26 c.370_371insACA/494T>C/1423C>T mutation (n = 10 independent experiments) were collected to perform Chr X and Chr Y FISH. P-values are shown for comparison to fertile men without USP26 mutations. C The fertile men with the USP26 mutated haplotype produced XY and O aneuploid spermatozoa. FISH assay of Chr X (green) and Chr Y (red) was performed in fertile men with or without the USP26 mutated haplotype. Nuclei were stained with DAPI (blue). The arrows indicate the Y chromosome, and the arrowheads indicate the X chromosome. D Squares denote male family members, circles denote female family member; a solid symbol indicates affected KS member, open symbols represent unaffected KS family members. The X labeled with red color indicates the X chromosome with the USP26 mutated haplotype (c.370_371insACA/494T>C/1423C>T). All family members were Sanger sequenced. \n",
      " - Usp26-deficient mice produce 41, XXY offspringA Sequences of the WT and Usp26 mutant alleles in mice.Figure 2A Sequences of the WT and Usp26 mutant alleles in mice. B The USP26 protein was absent in Usp26-/Y testes. Immunoblotting of USP26 was performed in Usp26+/Y and Usp26-/Y testes. Histone 3 served as the loading control. C, D The fertility of Usp26-/Y mice was reduced with increasing age. The fertility assessment experiments were performed in 2-month-old Usp26+/Y, Usp26-/Y mice (n =5 independent experiments) and 6-month-old Usp26+/Y, Usp26-/Y mice (n=5 independent experiments) (C). The litter sizes were observed in 2-month-old Usp26+/Y, Usp26-/Y mice (n=14, 22 independent experiments for Usp26+/Y and Usp26-/Y, respectively) and 6-month-old Usp26+/Y, Usp26-/Y mice (n=10, 8 independent experiments for Usp26+/Y and Usp26-/Y, respectively) (D). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. E Genotyping of Usp26-/Y mice offspring. The upper subpanel indicates the genotyping of Usp26-/Y mice male offspring by using Usp26 and Sry primers. The lower subpanel indicates the genotyping of Usp26-/Y, Usp26+/Y male mice and Usp26+/-, Usp26+/+ female mice by using Usp26 and Sry primers. F The wild-type (WT) and Usp26 knock out alleles in XUsp26+XUsp26-Y mice. G The proportion of 41, XXY mice among the male offspring of 2-month-old Usp26+/Y, Usp26-/Y and 6-month-old Usp26+/Y, Usp26-/Y mice (n=14, 22, 10 and 8 independent experiments for the offspring of 2-month-old Usp26+/Y, Usp26-/Y mice, 6-month-old Usp26+/Y, and Usp26-/Y mice, respectively). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. Two pair of X chromosomes could be detected in the 2-month-old XUsp26+XUsp26-Y spermatocytes. FISH analysis of Chr X-FISH (green) and Chr Y-FISH (red) was performed in WT and XUsp26+XUsp26-YY spermatocytes (H). The arrows indicate the Y chromosome and the arrowheads indicate the X chromosome. Nuclei were stained with DAPI (blue). Immunofluorescence analysis of Chr X-FISH (green), Chr Y-FISH (red), and SYCP3 (white) was performed in WT and XUsp26+XUsp26-YY spermatocytes (I). The arrows indicate the Y chromosome and the arrowheads indicate the X chromosome. Nuclei were stained with DAPI (blue). Three sex chromosome axes could be detected in the 2-month-old XUsp26+XUsp26-Y spermatocytes. Immunofluorescence analysis of SYCP3 (red) and SYCP1 (green) in (J), ATR (red) and SYCP3 (green) in (K) was performed in 2-month-old WT and XUsp26+XUsp26-Y spermatocytes. Nuclei were stained with DAPI (blue).The arrows indicate the Y chromosome. The arrowheads indicate the X chromosome. Three sex chromosome axes could be detected in the 2-month-old XUsp26+XUsp26-Y spermatocytes. Immunofluorescence analysis of BRCA1 (red) and SYCP3 (green) in (L) was performed in 2-month-old WT and XUsp26+XUsp26-Y spermatocytes. Nuclei were stained with DAPI (blue).The arrows indicate the Y chromosome. The arrowheads indicate the X chromosome. \n",
      ".Figure 1Scheme of SIRF assay. Nascent DNA was pulse-labelled with EdU. Upon fork stalling, proteins located at stalled forks are in close proximity to EdU. EdU was then biotinylated using click reaction, followed by incubating with biotin and target protein antibodies. PLA amplification was used to visualize the localization of target proteins at EdU-labeled stalled forks. B, C. Detection of CST at stalled and active forks with SIRF assay in U2OS (B) and HCT116 (C) cells. To detect CST at stalled forks, cells were pulse-labeled with EdU for 8 min, followed by 4 mM HU treatment for 3 h. To detect CST at active forks, SIRF was performed without HU treatment. Representative SIRF images of CTC1, STN1 and PCNA on normal or stalled forks are shown. Scale bars: 10 µm. Relative SIRF fluorescence intensity was calculated by normalizing to untreated samples and then to their respective EdU-SIRF signals. Mean value in each sample is labeled in the graph. N: the number of cells analyzed in each sample. \n",
      " - The disruption of Usp26 impairs spermatogenesis.Figure 3A The size of the testes was smaller in the 6-month-old Usp26-/Y mice. B Quantification of body weight in 6-month-old Usp26+/Y and Usp26-/Y mice (n=6 independent experiments). P=0.2442. Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. C Quantification of testis weight in 6-month-old Usp26+/Y and Usp26-/Y mice (n=6 independent experiments). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. D Histological analysis of the seminiferous tubules and caudal epididymis of the 6-month-old Usp26+/Y and Usp26-/Y mice. E The sperm counts in the caudal epididymis were measured in 6-month-old Usp26+/Y and Usp26+/-/Y mice (n= 6 and 5 independent experiments for Usp26+/Y and Usp26-/Y, respectively). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. F The deletion of Usp26 causes pachytene and meiotic division arrest. Representative PAS-hematoxylin staining in 6-month-old Usp26+/Y and Usp26-/Y seminiferous tubules. Parafﬁn sections from Usp26+/Y and Usp26-/Y testes were stained with PAS-hematoxylin. P: pachytene spermatocyte, D: diplotene spermatocyte, rST: round spermatid, eST: elongating spermatid, M: meiotic spermatocyte, aP: apoptotic pachytene spermatocytes, aM: abnormal meiotic divisions. \n",
      " - USP26 participates in sex chromosomes pairing by regulating the stability of TEX11.Figure 6A USP26 interacts with TEX11. pRK-FLAG-Usp26 and pEGFP-Tex11 were co-transfected into HEK293T cells. 24h after transfection, cells were collected for immunoprecipitation (IP) with anti-FLAG antibody and analyzed with anti-FLAG and anti-GFP antibodies, respectively. B USP26 modulates the ubiquitination of TEX11. pRK-FLAG-Usp26, pEGFP-Tex11, pCMV-HA-Ub were transfected into HEK293T cells. At 24h after transfection, cells were collected for immunoprecipitation (IP) with anti-GFP antibody and analyzed with anti-HA and anti-GFP antibodies, respectively. C The TEX11 protein was greatly reduced in the Usp26-/Y testis. Immunoblotting of TEX11 was performed in 6-month-old Usp26+/Y and Usp26-/Y testes. Actin served as the loading control. The X and Y chromosomes were unpaired in Tex11-/Y spermatocytes. Immunofluorescence analysis of SYCP3 (green), TRF1 (red), ATR (white) (D); SYCP3 (green), MLH1 (red), ATR (white) (E) was performed in in Tex11+/Y and Tex11-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrowheads indicate the X chromosome. The X and Y chromosomes were unpaired in Tex11-/Y spermatocytes. Immunofluorescence analysis of ; Chr X-FISH (green), Chr Y-FISH (red) and SYCP3 (white) (F) was performed in in Tex11+/Y and Tex11-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrowheads indicate the X chromosome. G Quantification of unpaired X and Y chromosomes in Tex11+/Y and Tex11-/Y mice (n= 4 independent experiments). Red dots indicate Tex11+/Y mice and green dots indicate Tex11-/Y mice. H Laggard chromosomes were observed in the metaphase I of Tex11-/Y spermatocytes. The arrowhead indicates the laggard chromosome. I The proportion of metaphase I spermatocytes exhibiting laggard chromosomes in Tex11+/Y and Tex11-/Y mice spermatocytes (n= 3 independent experiments). Red dots indicate Tex11+/Y mice and green dots indicate Tex11-/Y mice. J Tex11-deficient mice produced XY aneuploid spermatozoa. FISH assay of Chr X (green) and Chr Y (red) was performed in Tex11+/Y and Tex11-/Y spermatozoa. Nuclei were stained with DAPI (blue). The arrows indicate the Y chromosome, and the arrowheads indicate the X chromosome. \n",
      " - USP26 participates in sex chromosome pairing.Figure 4A USP26 was predominately expressed in the testis. Immunoblotting of USP26 was performed in the heart, liver, spleen, lung, kidney, intestine, brain, ovary, and testis. Histone 3 served as the loading control. B The localization of USP26 during meiosis. Immunofluorescence analysis of SYCP3 (red) and USP26 (white) was performed in wild-type spermatocytes. Nuclei were stained with DAPI (blue). The circles indicate the XY body. C, D The sex chromosomes failed to synapse in the 6-month-old Usp26-deficient spermatocytes. Immunofluorescence analysis of SYCP3 (green), ATR (white), SYCP1 (red) (C); SYCP3 (green), ATR (white), TRF1 (red) (D) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrows indicate the sex chromosomes. E The X and Y chromosomes were unpaired in Usp26-/Y spermatocytes. Immunofluorescence analysis of Chr X-FISH (green), Chr Y-FISH (red), and SYCP3 (white) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. The arrowheads indicate the X chromosome. F Quantification of unpaired X and Y chromosomes in 6-month-old Usp26+/Y and Usp26-/Y mice (n= 5 independent experiments). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. Sex chromosome recombination was perturbed in 6-month-old Usp26-/Y mice. Immunofluorescence analysis of SYCP3 (green), MRE11 (red) (G); was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrows indicate the sex chromosomes. Sex chromosome recombination was perturbed in 6-month-old Usp26-/Y mice. Immunofluorescence analysis of ; SYCP3 (green), γH2AX (red) (H); SYCP3 (green), ATR (red), p-ATM (pink) (I) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrows indicate the sex chromosomes. Sex chromosome recombination was perturbed in 6-month-old Usp26-/Y mice. Immunofluorescence analysis of DMC1 (green), SYCP3 (red) (J) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrows indicate the sex chromosomes. K Quantiﬁcation of the numbers of DMC1 foci in 6-month-old Usp26+/Y (n=50, Zygotene, n=31, Early-Pachytene) and Usp26-/Y (n=31, Zygotene, n=24, Early-Pachytene) spermatocytes. P=0.1893 for Zygotene. Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. L Crossover on sex chromosomes was impaired in 6-month-old Usp26-/Y mice. Immunofluorescence analysis of SYCP3 (green), MLH1 (red), and ATR (white) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). \n",
      " - Usp26-deficient mice produce XY aneuploid spermatozoa.Figure 5A, B Laggard chromosomes were observed in the metaphase I of 6-month-old Usp26-/Y spermatocytes. Immunofluorescence analysis of tubulin (green) was performed in 6-month-old Usp26+/Y and Usp26-/Y spermatocytes. Nuclei were stained with DAPI (blue). The arrowheads indicate the laggard chromosomes. (C) The proportion of metaphase I spermatocytes exhibiting laggard chromosomes in 6-month-old Usp26+/Y and Usp26-/Y mice spermatocytes (n= 5 independent experiments). Red dots indicate Usp26+/Y mice and green dots indicate Usp26-/Y mice. D Representative TUNEL results in Usp26+/Y and Usp26-/Y testes. Parafﬁn sections from 6-month-old Usp26+/Y and Usp26-/Y testes were stained with TUNEL (green) and DAPI (blue) to show dead cells in stage IV, VIII and XII tubules with pachytene and metaphase spermatocytes, respectively. The arrowheads indicate the laggard chromosomes. P: pachytene spermatocytes, rSt: round spermatid, spz: spermatozoa, M: meiotic spermatocyte, aP: apoptotic pachytene spermatocytes, aM: abnormal meiotic divisions, arSt: apoptotic round spermatid. E The protein level of three known SAC proteins was reduced in 2-month-old and 6-month-old Usp26−/Y mouse testes. Immunoblotting of MAD2, BUBR1, PLK1 and USP26 was performed in 2-month-old and 6-month-old Usp26+/Y and Usp26-/Y testes. H3Ser10p and Tub served as loading control. F Usp26-deficient mice produced XY aneuploid round spermatids. FISH analysis of Chr X (green) and Chr Y (red) was performed in 2-month-old and 6-month-old Usp26+/Y and Usp26−/Y round spermatids. Nuclei were stained with DAPI (blue). The arrows indicate the Y chromosome, and the arrowheads indicate the X chromosome. G Quantification of different types of round spermatids in 2-month-old Usp26 +/Y, Usp26-/Y mice (n= 3 independent experiments) and 6-month-old Usp26+/Y (n= 3 independent experiments), Usp26-/Y mice (n= 4 independent experiments). P=0.0026 for XY spermatozoa in 6-month-old Usp26+/Y and Usp26−/Y mice. P=0.0216 for O spermatozoa in 6-month-old Usp26+/Y and Usp26−/Y mice. H Usp26-deficient mice produced XY aneuploid spermatozoa. FISH assay of Chr X (green) and Chr Y (red) was performed in 2-month-old and 6-month-old Usp26+/Y and Usp26−/Y spermatozoa. Nuclei were stained with DAPI (blue). The arrows indicate the Y chromosome, and the arrowheads indicate the X chromosome. I Quantification of different types of spermatozoa in 2-month-old Usp26+/Y, Usp26-/Y mice (n= 3 independent experiments) and 6-month-old Usp26+/Y, Usp26-/Y mice (n= 5 independent ex experiments). P=0.0195 for XY spermatozoa in 6-month-old Usp26+/Y and Usp26−/Y mice. Data are presented as means ± SD. J Usp26-deficient mice could not produce Chr 8, Chr 10 aneuploid spermatozoa. FISH assay of Chr 8 (green) and Chr 10 (red) was performed in 6-month-old Usp26+/Y and Usp26−/Y spermatozoa. Nuclei were stained with DAPI (blue). K Quantification of different types of spermatozoa in 6-month-old Usp26+/Y, Usp26-/Y mice (n= 3 independent experiments). Data are presented as means ± SD. L The disruption of Usp26 produces XY aneuploid spermatozoa and results in the production of 41, XXY offspring. \n",
      ".Figure 3SIRF detection of MRE11 at stalled forks in STN1-deficient U2OS cells. Representative SIRF images of MRE11 at normal or stalled replication forks are shown. Scale bars: 10 µm. ssDNA accumulation analysis in STN1-deficient U2OS cells with and without mirin treatment. Representative images of native BrdU staining are shown. \n",
      ".Figure 4Coomassie blue stained SDS-PAGE gel (15%) of purified human MRE11. (i) Scheme shows the nuclease activity of MRE11 in degrading 5' Cy3-labeled substrates (25 nt + 60 nt with phosphorothioate bonds on both ends). (ii) The 5' Cy3-labeled substrates (80 nM) were incubated with the indicated concentration of MRE11 with or without the Mn2+ cofactor at 37°C for 20 min. Reactions were stopped by SDS and proteinase K and resolved on 27% denatured polyacrylamide gel. (iii) The results are graphed, and error bars represent the standard deviation (± SD) calculated from at least three independent experiments. Coomassie blue stained SDS-PAGE gel (15%) of purified human CST complex (CTC1 wild- type and ∆700N). DNA binding activity of wild-type CST complex and Δ700N mutant complex. The 5' Cy3- labeled substrates (80 nM) were incubated with the indicated concentrations of CST. Samples were analyzed by 0.8 % agarose gel. MRE11 degradation analysis. 5' Cy3-labeled substrates (80 nM) were pre-incubated with indicated concentrations of CST wild-type (i) or ∆700N mutant variant (ii) at 37°C for 5 min. Reactions were completed by adding MRE11 (200 nM) for an additional 20 min incubation and then stopped by SDS and proteinase K. Samples were resolved in 27% denatured polyacrylamide gel. The results are graphed and error bars represent the standard deviation (± SD) calculated from at least three independent experiments. \n",
      ".Figure 2Western blot showing STN1 knockdown. shLUC (siRNA targeting luciferase) was used as the control. CST deficiency causes ssDNA accumulation upon fork stalling. HeLa cells with STN1 knockdown were incubated with 10 µM BrdU for 48 h, followed by 2 mM HU treatment for 3 h, and subsequently stained with BrdU antibody under a non-denaturing condition to detect ssDNA. N, the number of cells analyzed in each condition. Relative BrdU fluorescence intensity was quantitated by Image J. Scheme of DNA fiber analysis and representative fiber images. DNA fiber analysis of U2OS cells with STN1 knockdown and its RNAi-resistant Flag-STN1 WT re-expression with and without mirin treatment. DNA fiber analysis of BJ cells with STN1 knockdown by two siRNA sequence with and without mirin treatment. DNA fiber analysis of HCT116 cells with STN1 knockdown with and without mirin treatment. DNA fiber analysis of U2OS cells showing that SMARCAL1-mediated fork reversal is needed for nascent-stand degradation observed in CST-deficient cells. DNA fiber analysis of U2OS cells showing that ZRANB3-mediated fork reversal is needed for nascent-strand degradation observed in CST-deficient cells. \n",
      ".Figure 7SIRF analysis of RAD51 binding to forks. Representative RAD51 SIRF images at normal or stalled forks are shown. Scale bars: 10 µm. Two independent treatments were performed and scatter plot of RAD51-bound replication sites from one experiment is shown. Mean values in each sample are listed at the top of the graph. N, number of cells analyzed in each sample in one experiment. RAD51 foci formation in U2OS CTC1 knockdown cells was rescued by RNAi-resistant WT but not by ∆700N. Cells were treated with HU (2 mM, 16 h) and stained with anti-RAD51. Representative images are shown. Scale bars: 10 µm. Scatter plot of RAD51 foci number with and without HU treatment is shown. Mean values in each condition are listed at the top of the graph. DNA fiber analysis of nascent-strand DNA degradation in U2OS cells with STN1 and RAD51 co-depletion. Three independent treatments were performed and results from one biological replicate are shown. N, the number of DNA fibers analyzed in each sample in one experiment. Western blots showing protein knockdown in DNA fiber assays are included. Model for CST in protecting reversal forks stability. Upon fork stalling, the stalled fork is reversed by SMARCAL1, ZRANB3, or others. CST binds to ssDNA (5' overhang) at reversed forks to directly block MRE11 access and may also recruit RAD51. Without CST, nascent-strand DNA is degraded by MRE11, leading to the generation of excessive ssDNA and genome instability. \n",
      "Full-length TLN (but not APP) accumulates in a distinct compartment in PS1−/−hippocampal neurons.Figure 1.Full-length TLN (but not APP) accumulates in a distinct compartment in PS1−/−hippocampal neurons. Double-immunofluorescence staining for endogenous TLN (green, B36.1) and APP (red, mAb C1 6.1) in wild-type (top) and PS1−/− (bottom) hippocampal neurons (15-d culture). The inset shows an overview of the neuron. In wild-type neurons TLN labeling is essentially localized to the somatodendritic plasmalemma in contrast to APP, which is found in intracellular compartments. In addition, PS1 deficiency causes TLN, but not APP, to cluster in large somatic accumulations. Bars, 10 μm. \n",
      ".Figure 5Deletion of the N-terminal 700 aa of CTC1 abolishes CTC1 localization at telomeres. U2OS stably expressing vector (V), WT Myc-CTC1 and Myc-∆700N were treated with or without HU and co-stained with Myc (red) and TRF2 (green) antibodies. Boxed areas are amplified in inserts to indicate CTC1/TRF2 colocalization (yellow).Scale bars: 10 μm. Quantification of percent of cells with >3 CTC1/TRF2 colocalization foci were from three independent experiments. In each experiment, >150 cells from each sample were analyzed.    ∆700N does not affect CST complex formation. HEK293T cells were co-transfected with Myc- CTC1 or Myc-∆700N, His6-STN1 and HA-TEN1. Co-IP was performed with Myc antibody to pulldown His6- STN1 and HA-TEN1. Full-length Myc-CTC1 was prone to degradation during immunoprecipitation.      ∆700N retains RAD51 interaction. HEK293T cells were co-transfected with Myc-CTC1 or Myc-∆700N, His6- STN1, HA-TEN1 and Flag-RAD51 or vector control (V), and treated with HU (2 mM, 16 h). Co-IP was performed with Flag antibody to pulldown Myc-CTC1.   DNA fiber analysis of CTC1 depleted U2OS cells with RNAi-resistant WT or ∆700N re-expression. Mean values in each sample are listed at the top of the graph. N, the number of cells analyzed in each condition, *** \n",
      ".Figure 6Micronuclei (MN) formation in BRCA2- and STN1-deficient cells. Co-depletion was achieved by transfecting siBRCA2 in U2OS cells expressing shSTN1. Representative DAPI staining images are shown. Arrows point to MN. Scale bars: 20 µm. Average percentages of nuclei containing MN from three independent experiments are presented. In each experiment, >300 nuclei were analyzed per sample. Anaphase bridges (arrows) in BRCA2- and STN1-deficient U2OS cells. Scale bar: 10 µm. Average percentages of anaphase bridges from three independent experiments are presented.    γH2AX induced by BRCA2 knockdown and STN1 knockdown in U2OS cells. Nuclei containing ≥ 5 foci were considered as positive γ-H2AX staining. Results were from three independent knockdown experiments. In each experiment, >80 cells were analyzed per sample.   Co-depletion of STN1 and BRCA2 significantly impairs DNA replication. Scale bar: 50 µm. Results were from three independent knockdown experiments. In each experiment, >180 nuclei were analyzed per sample. Co-depletion of STN1 and BRCA2 increases chromosome instabilities. U2OS cells with siBRCA2 and/or shSTN1 knockdown were treated with HU (2 mM, 3 h). Representative metaphase images show aberrant chromosomes (red arrows). Boxed areas are amplified and shown at the bottom of images. Scale bars: 20 µm. Two independent knockdown and chromosome spread experiments were performed. N, the number of metaphase spreads analyzed in each sample. \n",
      "Correlative light immuno-EM analysis of TLN accumulations.Figure 5.Correlative light immuno-EM analysis of TLN accumulations. (A) Insert: overview of the selected neuron as seen by immunocytochemistry. The bright signal for TLN reflects the numerous membranous accumulations seen at the ultrastructural immuno-EM level (star). (B) Higher magnification of the boxed area in A. (C) High magnification of the boxed structure seen in B. 10-nm gold particles label TLN. Note the complex composition of the internal membranes of the TLN-positive compartments. (D) Example of a TLN (15-nm gold)-containing structure obtained by the conventional cryosectioning technique. Note some heterogeneity in the internal membranes between C and D. Lamp-1 (10-nm gold) is absent from the TLN-containing compartment. Arrows point to invaginations of the cytoplasm. N, nucleus; P, plasma membrane. Bars: A, 5 μm; B, 2 μm; C, 500 nm; D, 200 nm. \n",
      "TLN accumulation occurs in a compartment distinct from classic biosynthetic and endosomal pathways.Figure 4.TLN accumulation occurs in a compartment distinct from classic biosynthetic and endosomal pathways. (A-F) Early compartments. No colocalization was observed between TLN (green in all panels) and nuclei (A, TOPRO-3), ER-markers BIP (B) and calnexin (C), ERGIC-53 (D), β-COP (E) or GM130 (F), and labeling cis-Golgi. (G-L) Late compartments. TLN accumulations are not accessible for exogenous biotin (G). Early and recycling endosomes (H, EEA1) and transferrin receptor (I, TFR), late endosomes (J, LBPA), and lysosomes (K, Lamp-2; L, catD) also stained negative for TLN. Vertical sections (x-z, arrowheads in I and J) clearly distinguish TLN accumulations from recycling (I, TFR) and late endosomes (J, LBPA). Bars, 10 μm. \n",
      "Turnover of full-length TLN is altered in PS1−/−neurons.Figure 3.Wild-type and PS1−/− cortical neurons were transduced with SFV-TLN (A) or -APP (B), pulse labeled for 15 min with [35S]methionine, and chased for different time periods. ImmunoprecipitatedTLN and APP were treated with EndoH and analyzed by SDS-PAGE and phosphorimaging. The accumulation of an EndoH-resistant band indicates progressive maturation during Golgi passage; however, the ratio of EndoH-resistant to -sensitive TLN (A, top) and APP (B, top) revealed no difference (mean ± SEM, n = 3). Instead, the half-life of overexpressed TLN (A, bottom), but not full-length APP (B, bottom), is significantly prolonged in neurons−/−neurons (mean ± SEM, n = 3). (C) Wild-type and PS1−/− hippocampal neurons (12 d) were scraped, pelleted, and treated with 10 mU EndoH, followed by Western blot detection of APP, TLN, PS1, nicastrin (NCT), synaptophysin, and ductin. TLN levels are increased almost fourfold in PS1−/− neurons. No EndoH-sensitive TLN was detected in PS1−/− neurons, indicating a complete maturation. APP-CTF shows a dramatic accumulation (44-fold; mean ± SEM, n = 3). \n",
      "TLN lacks the characteristics of a γ-secretase substrate.Figure 2.(A) Western blot analysis of endogenous TLN and APP in wild-type and PS1−/− hippocampal neurons. PS1 deficiency results in the accumulation of APP-CTF. No TLN-CTF (estimated ± 7 kD) was detected, neither was it detected after SFV-induced exogenous overexpression. Also, overexpression of a TLNΔE fragment did not generate a PS1-dependent TLN intracellular domain. (B) Different constructs fused to the Gal4VP16 domain were used in a γ-secretase reporter assay. Control transfections (pIPspAdApt empty vector and pIPspAdApt-APP without Gal4VP16) show no luciferase activity. APP-Gal4VP16 and NotchΔE-Gal4VP16 activate the luciferase gene upon cleavage and are inhibited by DAPT, L658,458, or compound 32 (125 nM). No luciferase activity was detected using TLNΔE or APP in which the transmembrane domain was replaced by that of TLN (APP/TLNTMR chimaera), indicating that this region is not cleaved by γ-secretase. (C) Left: γ-secretase inhibitors do not induce the formation of TLN accumulations. Wild-type hippocampal neurons were treated with γ-secretase inhibitors (125 nM, daily from d 7-14 post-plating), fixed, and immunostained for TLN and the percentage of neurons with TLN accumulations was scored. Culturing neurons for 25 d increases the size and number (insets), but not the frequency of TLN accumulations (mean ± SEM, n = 3, 200-700 neurons per time point). Right: Western blot showing that only APP-CTF accumulates after chronic treatment with γ-secretase inhibitors. (D) Adenoviral reintroduction of human PS1 suppresses TLN accumulations. 30-35% of PS1−/− neurons display TLN accumulations, in contrast to <5% in PS1+/+ and +/− cultures as well as in cultures expressing human PS1 in a PS1−/− background. However, suppression of TLN accumulations was obtained by adenoviral introduction of human PS1. A clear dose-dependent inhibition was seen with increasing MOI of adenoviral (AV) human PS1, but also the familial Alzheimer's disease-linked PS1G384A and dominant-negative PS1D257A mutants. Control adenoviral infections with eGFP did not have a significant effect. After 9-11 d post-infection (15 d post-plating), neurons displayed significant PS1 protein expression as demonstrated by double staining for TLN and human PS1(mAb 5.2). Bars, 20 μm. \n",
      "TLN accumulations are distinct from lysosomes.Figure 6.TLN accumulations are distinct from lysosomes. A TLN (15-nm gold)-containing structure as obtained by the conventional cryosectioning technique double labeled with Lamp-1 (10-nm gold). The TLN-positive compartment is clearly distinct in size and from lysosomes (L) and lacks Lamp-1. P, plasma membrane. Bar, 200 nm. \n",
      "TLN accumulations have characteristics of autophagic vacuoles.Figure 7.(B and C) Double immunostaining of wild-type and PS1−/− hippocampal neurons for TLN (green) and autophagic vacuole-associated Apg12p (B) and LC3 (C). To detect both sets of pAbs, an alternative method was developed using biotinylated anti-TLN (see Materials and methods). Each experiment included controls for specificity (not depicted). In wild-type neurons, both Apg12p and LC3 (B and C, top row) were broadly distributed in cell bodies and neurites, but did not colocalize with TLN. Colocalization was only observed in multiple TLN accumulations in PS1−/− hippocampal neurons (second row in B and C, respectively). For Apg12p, coimmunostaining was also noticed in small discrete structures in the proximal dendrites (B, bottom row). Bars, 10 μm. \n",
      "CatD−/−hippocampal neurons as a model system for autophagic vacuole accumulation.Figure 8.(A) Intracellular sorting and maturation of catD. Wild-type and PS1−/− neurons were metabolically labeled and chased for the indicated time periods. Phosphorimaging of immunoprecipitated radiolabeled catD did not reveal a difference in the ratio of proteolytically processed 46-kD intermediate vs. 52-kD precursor (mean ± SEM, n = 3). (B-D) TLN localizes to autophagic vacuoles of catD−/−hippocampal neurons. Lysotracker staining revealed sparsely distributed small-sized organelles in wild-type and PS1−/− neurons in contrast to catD−/−neurons where high numbers of large Lysotracker-positive organelles were found (B, arrowheads). Although TLN accumulations in PS1−/− were fully negative for Lysotracker, some acidic organelles tended to closely associate (C, arrowheads, middle). In catD−/− neurons, TLN was clearly detected in the large acidic organelles (C, bottom). Bars, 20 μm. (D) At the immuno-EM level, the large acidic organelles seen in catD−/−neurons represent dense autophagic vacuole-like structures (star) that label positive for TLN (10-nm gold). M, mitochondrion. Bars, 200 nm. \n",
      "TLN mediates phagocytic uptake of microbeads in hippocampal neurons.Figure 9.TLN mediates phagocytic uptake of microbeads in hippocampal neurons. (A) Microbeads align along neurites of hippocampal neurons as demonstrated by phase contrast (DIC). Already after 4 h, many beads stain positive for TLN (arrowheads, top). At 48 h (bottom), most TLN immunoreactivity was associated with microbeads at the expense of its typical plasma membrane staining. *, cell body. Note that microbeads were not found associated with preexisting TLN accumulations (asterisk in top panels, 4 h). (B) Microbeads (24 h) accumulate TLN and actin, as shown by phalloidin-Alexa 568. An overview of the neuron is given in each top left inset, next to two detailed areas (white boxes). The differential interference contrast (DIC) clearly demonstrates a complete colocalization of microbeads with TLN and phalloidin (arrowheads). TOPRO-3 (blue) marks the nucleus (see also Fig. S1, available at http://www.jcb.org/cgi/content/full/jcb.200406060/DC1). (C) Similar as in B, but immunostained for TLN and endogenous PIP2. The overview demonstrates the overall recruitment of PIP2 to microbeads (arrows). The parallel insets show a detailed area (white box) with PIP2 being colocalized with TLN on individual microbeads (arrowheads). Note that PIP2 occasionally colocalizes with TLN at the plasma membrane (asterisk). Bars (A-C), 10 μm. \n",
      " | AP-2 regulates BACE1 recycling and is mostly dispensable for BACE1 endocytosis.Figure 1(A) BACE1 internalization kinetics after 20 minutes (WT: 19.79±1.89%, KO: 13.28±1.57%, p WT vs KO=0.017) and 40 minutes of HA antibody chase (WT: 15.99±2.04, KO: 9.29±1.33, pWT vs KO=0.010) is significantly decreased in AP-2µ KO neurons (27 WT and 29 KO neurons). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (B) Representative examples of internalized BACE1 after 5 min of HA antibody pulse in HA-BACE1-eGFP-overexpressing WT and KO neurons. Scale bar: 5µm. (C, D) Internalized BACE1 levels after 20 min of HA antibody pulse in HA-BACE1-eGFP-overexpressing WT and KO neurons, calculated as the HA (internalized)/eGFP (total) signal intensity ratio (WT: 0.50±0.052, KO: 0.23±0.036, p=0.000, 30 WT and 36 KO neurons, N=4 biological replicates). Scale bar: 5 µm. Data information: Experimental designs are depicted in the top panels where analyzed BACE1 fraction is marked with the red square. All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (E, F) Recycled BACE1 levels after 20 min of HA antibody pulse in HA-BACE1-eGFP-overexpressing WT and KO neurons, calculated as the HA(recycled)/eGFP(total) signal intensity ratio (WT: 0.29±0.031, KO: 0.53±0.040, p<000, 30 WT and 32 KO neurons, N=4 biological replicates). Scale bar: 5 µm. Data information: Experimental designs are depicted in the top panels where analyzed BACE1 fraction is marked with the red square. All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Recycled levels of BACE1 after 20 min of HA antibody pulse in HA-BACE1-mCherry-transfected WT and KO neurons, co-expressing either eGFP-RAB4 or eGFP-RAB4-S22N pWTRAB4 vs pWTRAB4S22N>0.999, 34-38 neurons, N=4 biological replicates). Scale bar: 5 µm. Data information: Experimental designs are depicted in the top panels , where analyzed BACE1 fraction is marked with the red square. Recycled levels of BACE1 after 20 min of HA antibody pulse in HA-BACE1-mCherry-transfected WT and KO neurons, co-expressing either eGFP-RAB4 or eGFP-RAB4-S22N, calculated as the HA(recycled)/mCherry(total) signal intensity ratio (WTRAB4: 0.14±0.02, KORAB4: 0.20±0.02, WTRAB4-S22N: 0.14±0.02, KORAB4S22N:0.14±0.01, pWTRAB4 vs pKORAB4=0.033, pKORAB4 vs pKORAB4S22N=0.035, pWTRABS22N vs pKORABS22N=0.999, pWTRAB4 vs pWTRAB4S22N>0.999, 34-38 neurons, N=4 biological replicates). Scale bar: 5 µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA in n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Recycled levels of BACE1 after 20 min of HA antibody chase in HA-BACE1-mCherry-transfected WT and KO neurons, co-expressing either eGFP-RAB11 or eGFP-RAB11-S22N, calculated as the HA(recycled)/mCherry(total) signal intensity ratio (WTRAB11: 0.11±0.01, KORAB11: 0.29±0.02, WTRAB11A-S25N: 0.14±0.02, KORAB11A-S25N: 0.19±0.02, pWTRAB11 vs pKORAB11<0.0001, pWTRAB11 vs pWTRAB11A-S25N=0.761, pKORAB11 vs pKORAB11A-S25N=0.001, pWTRAB11A-S25N vs pKORAB11A-S25N=0.175, 28-37 neurons per condition, N=3 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Internalized levels of BACE1 after 20 minutes of HA antibody uptake in HA-BACE1-mCherry transfected WT and KO neurons, co-expressing either eGFP-RAB4 or eGFP-RAB4-S22N pWTRAB4 vs pWTRAB4S22N=0.450, 27-37 neurons per condition, N=3 biological replicates). Scale bar: 5 µm. Data information: Experimental designs are depicted in the top panels where analyzed BACE1 fraction is marked with the red square. Internalized levels of BACE1 after 20 minutes of HA antibody uptake in HA-BACE1-mCherry transfected WT and KO neurons, co-expressing either eGFP-RAB4 or eGFP-RAB4-S22N, calculated as the HA(internalized)/mCherry(total) signal intensity ratio (WTRAB4: 0.54±0.05, KORAB4: 0.33±0.02, WTRAB4-S22N: 0.63±0.06, KORAB11A-S25N: 0.50±0.04, pWTRAB4 vs pKORAB4=0.004, pKORAB4 vs pKORAB4S22N=0.046, pWTRABS22N vs pKORABS22N=0.182, pWTRAB4 vs pWTRAB4S22N=0.450, 27-37 neurons per condition, N=3 biological replicates). Scale bar: 5 µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by ANOVA n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (L, M) BACE1 surface levels are increased in AP-2µ KO neurons, when compared to the WT set to 100% (KO: 129.65±5.30%, p=0.005, N=4 biological replicates). (N) Surface versus total BACE1 ratio is not altered in AP-2µ KO neurons when normalized to the WT set to 100% (KO: 83.02±10.93%, p=0.109, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test in (M,N). n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (O, P) Accumulation of endogenous BACE1 in RAB5-positive early endosomes, quantified using Pearson's colocalization coefficient (WT: 0.41±0.01, KO: 0.46±0.02, pWT vs pKO=0.021, 58 WT and 54 KO neurons, N=4 biological replicates). Scale bar: 10 µm. Arrows indicate BACE1 accumulation within RAB5-positive endosomes. Data information All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (Q) Schematic illustration depicting the role of AP-2 in BACE1 endosomal trafficking. \n",
      " | Hypothetical model explaining the role of AP-2 in intracellular trafficking of BACE1.Figure 6In WT neurons, AP-2μ is required for transport of BACE1- containing autophagosomes and late endosomes en route to lysosomes. Absence of AP-2μ causes defective axonal trafficking of BACE1 leading, which results in increased amyloidogenic processing of APP. \n",
      " | AP-2 is required for lysosomal targeting of BACE1.Figure 2(A, B) BACE1 protein levels in cortical lysates of WT and AP-2µ KO mice. Protein levels in the KO were normalized to the WT set to 100% (KO: 447.64±48.78%, p<0.000, N=7 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (C, D) Immunofluorescence levels of BACE1 are significantly increased in the cortex of AP-2µ KO mice when normalized to the WT set to 100% (KO: 162.714±11.082%, p=0.005, N=4, biological replicates). Scale bar: 100µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (E) Localization of BACE1 in cultured WT and AP-2µ KO neurons. Scale bars: 20 µm, 5 µm (inserts). (F) Line plot of BACE1 immunofluorescence along the neurites Percentage of BACE1 puncta colocalizing with the LC3b (WT: 21.75±2.88, KO: 30.28±2.97, p=0.043, WT=41, KO=45 neurons, N=4 biological replicates). Scale bar: 2µm. Arrowheads indicate BACE1 accumulation within LC3b-positive organelles. Percentage of BACE1 puncta colocalizing with the LC3b (WT: 21.75±2.88, KO: 30.28±2.97, p=0.043, WT=41, KO=45 neurons, N=4 biological replicates). Scale bar: 2µm. Arrowheads indicate BACE1 accumulation within LC3b-positive organelles. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (I, J) Percentage of BACE1 puncta colocalizing with the eGFP-RAB7 (WT:19.54±1.91, KO: 37.09±3.13%, p<0.000, WT=44 and KO=43 neurons, N=4 biological replicates). Scale bar: 2µm. Arrowheads indicate BACE1 accumulation within RAB7-positive organelles. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (K, L) Colocalization of BACE1 and RAB7A in cortical sections of WT and AP-2μ ΚΟ brains, quantified using Pearson's colocalization coefficient (WT: 0.18±0.04, KO: 0.35±0.05, p=0.038, N=4 biological replicates). Scale bar: 10 µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (M Representative fluorescence images of WT neurons transfected with HA-BACE1-mCherry and immunostained for Cathepsin D. Channels were false color-coded to better illustrate the colocalization. Scale bars: 10µm. Arrowheads indicate BACE1 accumulation in lysosomes in the WT Dotted line outlines the neuron. N) Representative fluorescence images of AP-2µ KO neurons transfected with HA-BACE1-mCherry and immunostained for Cathepsin D. Channels were false color-coded to better illustrate the colocalization. Scale bars: 10µm. Arrowheads indicate BACE1 absence in the KO. Dotted line outlines the neuron. (O) Lysosomal localization of BACE1 is decreased in KO neurons (WT: 73.05±3.80, KO: 57.68±4.28, p=0.008, 45 WT and 44 KO neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (P) Schematic illustration of BACE1 delivery to lysosomes monitored via BACE1-mKeima-Red. (Q, R) Representative fluorescence images of WT and AP-2μ KO neurons transfected with BACE1-mKeima-Red and co-transfected with eBFP to reveal the neuronal morphology. Scale bars: 10µm, 2µm in zoomed images. (S) The 550nm/438nm signal intensity ratio of BACE1-mKeima-Red quantified over the entire cell is significantly reduced in the KO condition compared to the WT (WT: 0.59±0.04, KO: 0.48±0.04, p=0.041, 28 WT and 25 KO neurons, N=3 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. \n",
      " | AP-2 prevents amyloidogenic processing of APP in neurons.Figure 4(A, B) The eGFP/mCherry signal intensity ratio is significantly increased in mCherry-APP-P1-eGFP-expressing KO neurons comparing to the WT (WT: 0.94±0.06%, KO: 1.65±0.18, p=0.000, WT=39 and KO=41 neurons, N=5 biological replicates). Scale bars: 2μm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Confocal images of WT and AP-2μ ΚΟ entorhinal cortex immunostained with antibody recognizing total APP and APP-CTFs. Scale bars: 100 µm (C) Confocal images of WT and AP-2μ ΚΟ entorhinal cortex immunostained with antibody recognizing total APP and APP-CTFs. Scale bars: 20µm (D). (E-G) Levels of APP-CTFs are significantly increased in AP-2µ KO cortex compared to the WT set to 100% (KOC99: 156.65±8.37%, p=0.004, N=4 biological replicates; KOC83: 154.79±15.6%, p=0.012; N=5 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (H) Levels of full-length (FL) APP are unaltered in AP-2µ KO cortex (KOAPP FL: 106.64±12.96%, p=0.315, N=6 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (I) Levels of Nicastrin (NCT) are unaltered in the AP-2µ KO cortex (J, K) Intracellular levels of Aβ1-40 are significantly increased in AP-2µ KO axons compared to the WT (WTAβ1-40: 1.82±0.12, KOAβ1-40: 2.45±0.12, p=0.000, WT=24 and KO=24 neurons, N=3 biological replicates). Scale bars, 5µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Intracellular levels of Aβ1-42 are significantly increased in AP-2µ KO axons (WTAβ1-42: 0.65±0.08, KOAβ1-42: 0.91±0.10, p=0.046, WT=38 and KO=38 neurons, N=3 biological replicates). Scale bar: 5 µm. Intracellular levels of Aβ1-42 are significantly increased in AP-2µ KO axons (WTAβ1-42: 0.65±0.08, KOAβ1-42: 0.91±0.10, p=0.046, WT=38 and KO=38 neurons, N=3 biological replicates). Scale bar: 5 µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (N) Significantly increased ratio of Αβ1-42/ Αβ1-40 measured by ELISA in the media of cultured AP-2μ KO neurons compared to the WT set to 100% (KO:124.57±10.97%, p=0.037, N=6 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Accumulation of intracellular Aβ1−42 in neurons expressing the AP2-α−WT (WT: 1.14±0.18 , WT=28 and Mut=33 neurons, N=4 biological replicates). Scale bars: 20µm, 2µm (inserts). Accumulation of intracellular Aβ1−42 in neurons overexpressing the AP2-α-Mut comparing to neurons expressing the AP2-α−WT (WT: 1.14±0.18, Mut: 2.06±0.29, p=0.011, WT=28 and Mut=33 neurons, N=4 biological replicates). Scale bars: 20µm, 2µm (inserts). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Accumulation of intracellular Aβ1-42 is rescued upon BACE1 knock-down (WTScr: 2.76±0.20 WTshBACE1: 2.52±0.12 27 to 39 neurons per condition, N=3 biological replicates). Scale bar: 10µm. Accumulation of intracellular Aβ1-42 is rescued upon BACE1 knock-down in AP-2µ KO neurons (WTScr: 2.76±0.20, KOScr: 3.94±0.21; WTshBACE1: 2.52±0.12; KOshBACE1: 2.74±0.21; pWTScr vs pKOScr=0.000, pKOscr vs pKOshBACE1=000; pWTscr vs pWTshBACE1=0.826; pWTshBACE1 vs pKOshBACE1=0.863; 27 to 39 neurons per condition, N=3 biological replicates). Scale bar: 10µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA in (T) n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (U, V) Levels of intracellular Aβ1−42 are significantly upregulated in neurons overexpressing the HA-BACE1-LL/AA-GFP comparing to neurons expressing the HA-BACE1-GFP (BACE1: 0.78±0.90, BACE1-LL-AA: 1.99±0.33, p=000, 28-29 neurons per condition, N=3 biological replicates). Scale bar, 50µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. \n",
      " | AP-2 regulates axonal transport of BACE1.Figure 3(A, B) Representative fluorescence images (A) and corresponding kymographs (B) from time-lapse videos of control neurons transfected with HA-BACE1-eGFP and co-transfected with AP-2μ-mCherry. Scale bars: (A) 2μm, 1.6µm inserts, (B) x=2μm, y=5s. Arrowheads in (A) correspond to BACE1 carriers indicated by the same arrowheads in (B). (C) Representative fluorescence images and corresponding kymographs from time-lapse videos of WT and AP-2μ KO neurons transfected with HA-BACE1-eGFP. Scale bar: 5µm top panels, x=2μm, y=5s bottom panels. (D) Loss of AP-2μ significantly decreases the BACE1 motility (WT: 37.42±3.79%, KO: 23.07±3.07%, p=0.003, WT=31, KO=32 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (E) Loss of AP-2μ significantly decreases the anterograde and retrograde BACE1 velocity (WTAntero: 0.37±0.02 μm/s, KOAntero: 0.22±0.03 μm/s, pWTAntero vs pKOAntero<0.000; WTRetro: 0.35±0.02 μm/s, KORetro: 0.25±.03 μm/s, pWTRetro vs pKORetro =0.006, WT=35 and KO=32 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (F) Representative fluorescence images and corresponding kymographs from time-lapse videos of WT and AP-2μ KO neurons co-transfected with HA-BACE1-GFP and AP-2μ-mRFP. Scale bar: 5µm top panels, x=2μm, y=5s bottom panels. (G) Bi-directional mobility of BACE1 carriers in WT and AP-2μ ΚΟ neurons, co-expressing the AP-2μ-mRFP (WT: 37.60±2.20%, KO: 25.61±2.67%, WTAP-2μ-mRFP: 45.04±3.27%, KOAP-2μ-mRFP: 40.07±2.37%, pWT vs KO=0.005, pKO vs KO+AP-2μ-mRFP<0.000, pWT vs WT+AP-2μ-mRFP=0.186, pWT+AP-2μ-mRFP vs KO+AP-2μ-mRFP=0.570, 43-53 neurons per condition, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA in (G). n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (H) Representative kymographs of WT and AP-2μ KO neurons co-expressing the HA-BACE1-GFP and tagRFP-LC3B. Scale bars: x=2μm, y=5s. (I) Reduced bi-directional mobility of BACE1-containing autophagosomes in AP-2µ KO neurons. (WT: 33.53±4.19%, KO: 19.45±5.14%, p=0.000, WT=37 and KO=35 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (J) Representative kymographs of WT neurons co-expressing HA-BACE1-GFP and AP2-αΑ WT or AP2-αΑ Μut. Scale bar: 5µm top panels, x=2μm, y=5s bottom panels. (K) Mobility of BACE1 carriers is significantly decreased in neurons overexpressing the AP2-αΑ Mut compared to neurons expressing the AP2-αΑ WT (WT: 52.23±4.16%, Mut: 27.95±3.31, p=<0.000, WT=26 and Mut=29 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (L) Representative fluorescence images and corresponding kymographs from time-lapse videos of WT neurons transfected with either HA-BACE1-GFP or HA-BACE1-LL/AA-GFP. Scale bar: 5µm top panels, x=2μm, y=5s bottom panels. (M) Reduced mobility of BACE1-LL/AA-positive vesicles, when compared to WT BACE1 (WT: 38.97±1.40%, LL/AA: 19.89±1.77%, p<0.000, WT=31 and LL/AA=33 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (N) Reduced velocity of BACE1-LL/AA-positive vesicles, when compared to WT BACE1 (WTAntero: 0.38±0.03 μm/s, LL/AAAntero: 0.15±0.03 μm/s, pWTAntero vs pLL/AAAntero<0.000; WTRetro: 0.39±0.03 μm/s, LL/AARetro: 0.25±.03 μm/s, pWTRetro vs pLL/AARetro =0.000, WT=31 and LL/AA=33 neurons, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant. * indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (O) Schematic illustration of how AP-2 regulates BACE1 intracellular trafficking and transport in neurons. \n",
      " | Synaptic dysfunctions in AP-2µ KO mice are mitigated by BACE1 haploinsufficiency.Figure 5(A, B) AP-2α levels normalized to GAPDH are significantly decreased in lysates of iPSC-derived neurons from late-onset AD patients carrying the TREM2 p.R47H (AD-TREM2-2 and AD-TREM2-4) variant compared to healthy controls (CON8 and CON9) set to 100% (ADAP-2α: 52.42±8.04%, p=0.020, N=3 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by one-sample Student's t‐test n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (C) Confocal images of either AP-2 WT (AAV-GFPCamkIIα-labelled) or AP-2 KO (AAV-Cre-GFPCamkIIα-labelled) granule neurons in the dentate gyrus of 13-week-old mice. Scale bars: 20µm, 2µm (inserts). (D) Loss of AP-2µ significantly decreases the number of spines on granule neurons (WT: 12.63±0.88, KO: 8.07±0.45, p=0.004, N=4 biological replicates). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (E, F) Overlap coefficient between the presynaptic marker Bassoon and the postsynaptic marker PSD95 is significantly decreased in the entorhinal cortex of AP-2µ KO mice compared to the WT (WT: 0.45±0.02, KO: 0.33±0.06, p=0.034, N=3 biological replicates). Scale bar, 750nm. Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Lost preference for the novel object (nov) comparing to the familiar object (fam) in AP-2µ KO mice tested 24h after training in NOR Representative heatmaps are shown in (G). Lost preference for the novel object (nov) comparing to the familiar object (fam) in AP-2µ KO mice tested 24h after training in NOR (WTfam: 27.51±6.77, WTnov: 72.49±6.77, pWTfam vs WTnov=0.000; KOfam: 43.9±13.12, KOnov: 59.91±13.12, pKOfam vs KOnov=0.469, N=7 WT and 8 KO mice). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (I, J) AP-2µ KO mice are significantly more active in the open field compared to the WT (WT: 1663±139.08%, KO: 3976±802.30%, p=0.020, N=7 WT and 8 KO mice). Representative heatmaps are shown in (I). Data information: All graphs show mean ± SEM, statistical analysis was performed by unpaired two-tailed Student's t‐test n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (K, L) Overlap coefficient between Bassoon and PSD95 is rescued upon BACE1 KD in AP-2µ KO neurons (WTscr: 0.39±0.02; KOscr: 0.24±0.01; WTshBACE1: 0.37±0.02; KOshBACE1: 0.33±0.02; pWTscr vs pKOscr<0.000; pKOscr vs pKOshBACE1=0.010; pWTscr vs pWTshBACE1=0.813; pWTshBACE1 vs pKOshBACE1=0.623, 22-24 neurons for each condition, N=3). Scale bars: 10µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. Neuronal loss is rescued in the entorhinal cortex of AP-2µ mice haploinsufficient for BACE1 (AP-2WT/BACEWT: 2.25±0.08 N=9 sections from N=3 for each genotype). Scale bar: 100 μm. Neuronal loss is rescued in the entorhinal cortex of AP-2µ KO mice haploinsufficient for BACE1 AP-2KO/BACEWT: 1.51±0.08, AP-2WT/BACEHET: 1.80±0.07, AP-2KO/BACEHET: 2.00±0.07; pAP-2WT/BACE1WT vs pAP-2KO/BACE1WT<0.000; pAP-2WT/BACE1WT vs pAP-2WT/BACE1HET=0.001; pAP-2KO/BACE1WT vs pAP-2KO/BACE1HET=0.000; p AP-2WT/BACE1HET vs AP-2KO/BACE1HET=0.268, N=9 sections from N=3 for each genotype). Scale bar: 100 μm. Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. (P, Q) APP-CTF levels are rescued in the entorhinal cortex of AP-2µ KO mice haploinsufficient for BACE1 (AP-2WT/BACEWT:427574±24021, AP-2KO/BACEWT: 538895±34662, AP-2WT/BACEHET: 389870±19842, AP-2KO/BACEHET: 395389±21853; pAP-2WT/BACE1WT vs pAP-2KO/BACE1WT=0.017; pAP-2KO/BACE1WT vs pAP-2KO/BACE1HET=0.001; p AP-2WT/BACE1HET vs AP-2KO/BACE1HET=0.979; 13-15 sections, N=3 mice for each genotype). Scale bars: 40µm. Data information: All graphs show mean ± SEM, statistical analysis was performed by two-way ANOVA n.s.-non-significant.* indicates p≤0.05, ** indicates p≤0.01, *** indicates p≤0.001. \n",
      ".Figure 1(A) Genome-wide screen and internalization assay workflow. Library-transduced Ramos Cas9 cells were incubated with fluorescent, biotinylated surrogate antigen, anti-IgM F(ab')2 (total antigen) and subsequently stained with fluorescent streptavidin (surface antigen). Flow cytometry plots show a control sample incubated on ice preventing anti-IgM F(ab')2 endocytosis and an experimental sample from which cells displaying high and low anti-IgM F(ab')2 internalization were flow sorted and analyzed for gRNA abundance. (B) GeCKO library results showing mean internalization score per gene versus statistical significance. Whole-genome data come from two independent repeats. Data information: genes with annotation related to endocytosis or intracellular trafficking are highlighted. Red denotes examples of positive regulators; green denotes examples of negative regulators of internalization. (C) Brunello library results. Whole-genome data come from two independent repeats. Data information: genes with annotation related to endocytosis or intracellular trafficking are highlighted. Red denotes examples of positive regulators; green denotes examples of negative regulators of internalization. (D) Minilibrary validation results from four independent experiments. Validated candidates with internalization score <-0.4 or >0.35 and q<0.05. Data information: genes with annotation related to endocytosis or intracellular trafficking are highlighted. Red denotes examples of positive regulators; green denotes examples of negative regulators of internalization. (E) Selected validated candidates grouped by functional annotation. \n",
      "Fig.Figure 2A-J The indicated Sox14 or Mical constructs were tested for their ability to suppress pruning defects induced by PP2A-29B dsRNA expression. C4da neurons were labeled by CD8::GFP expression under ppk-GAL4, and dendrite pruning defects were assessed at 18 h APF. Panels A - E show effects of Sox14/Mical pathway manipulations alone, panels F - J show neurons coexpressing PP2A-29B dsRNA. The scale bar in A is 50 μm. (A, F) C4da neurons (co-) expressing lacZ. (B, G) C4da neurons (co-) overexpressing Sox14. (C, H) C4da neurons (co-) overexpressing Mical. (D, I) C4da neurons (co-) expressing Mical∆redox. (E, J) C4da neurons (co-) expressing Mical∆CH. K Penetrance of pruning defects in panels A - J. * P<0.05, *** P<0.0005, Fisher's exact test. Numbers of neurons (animals) for each genotype are given in the figure the numbers of animals are given below in parentheses. L Severity of pruning defects in panels A - J as assessed by number of primary and secondary dendrites attached to soma at 18 h APF. Data are mean +/- s. d., * P<0.05, *** P<0.0005, Wilcoxon's test. Numbers of neurons (animals) for each genotype are given in the figure. \n",
      "Fig.Figure 1A - F' Upper panels (A - F) show morphology of c4da neurons of the indicated genotypes at the third instar larval stage, lower panels (A' - F') show c4da neuron morphology at 18 h after puparium formation (APF). Neurons were labeled by CD8::GFP expression under ppk-GAL4 or by tdtomato expression in MARCM clones. (A, A') C4da neurons expressing control Orco dsRNA. Scale bars are 100 μm in A (for larval images) and 50 μm in A' (for pupal images). (B, B') C4da neurons expressing PP2A-29B dsRNA. (C, C') Homozygous PP2A-29BGE16781 mutant c4da neuron MARCM clones. (D, D') Homozygous PP2A-29BGE16781 mutant c4da neuron MARCM clones expressing UAS-HAPP2A-29B. (E, E') C4da neurons expressing dominant-negative Mts. (F, F') Homozygous mtss5286 mutant c4da neuron MARCM clones. G Penetrance of pruning defects at 18 h APF in A' - E'. ** P<0.005, *** P<0.0005, Fisher's exact test. Number of neurons for each genotype are given in the figure, the number of animals is given below in parentheses. H Severity of pruning defects at 18 h APF in A' - E' as assessed by number of primary and secondary dendrites attached to soma at 18 h APF. Data are mean +/- s. d., ** P<0.005, *** P<0.0005, Wilcoxon's test. Numbers of neurons (animals) for each genotype are given in the figure. \n",
      ".Figure 3(A) CRISPR-targeted bone marrow chimera workflow, showing flow cytometry detection of donor-derived (GFP+), sgRNA-containing (mCherry+) cells. (B) Development of B cell populations in chimeras' bone marrow and spleen from Cd4- or Sh3gl1-targeted HSCs. Percentage of mCherry+ cells at each stage is normalized to initial infection rate of transferred lineage- cells, and shown as mean + SEM. N =11-12 mice. *p=0.0286, ***p<0.0001 using two-way ANOVA. (C) MZ development in mCherry+ B cell compartment. (D) Representative stain of surface CD86 and CD69 on adoptively transferred HEL-specific follicular B cells 24 hours post SRBC-HEL immunization. (E) Quantification of surface CD86 and CD69 mean fluorescence intensity (MFI). Data show mean and SEM from 6 mice across 2 experiments. (F) Fractions of mCherry+ cells in GC, PC, IgG1+ class-switched and follicular B cells at 4-, 8- and 14-days post SRBC-HEL immunization. N = 8-10 mice; data shows mean and SEM. *p<0.05; ***p<0.001; ****p<0.0001 using 2-way ANOVA with multiple comparisons. \n",
      ".Figure 4(A) Characterization of B cell populations in bone marrow, spleen and lymph node at steady state. Data show mean and SEM from N = 5 mice. P, significance in t tests performed for follicular (FO) and MZ cell counts. (B) Spleen size in Sh3gl1-/- and WT littermates at 12 weeks of age. (C) Surface levels of MHCII, BAFFR, CD40 and LPAM1 in Sh3gl1-/- and WT littermates. N = 4-5 mice. P, statistical significance from unpaired t tests. Data show mean ± SEM. (D) Quantification of B220+Fas+CD38- GC population as percentage of total splenic B cells and NP-specific cells as a percentage of the GC population 14 days after NP-CGG immunization in alum. Data show mean ± SEM from N = 6 mice, p, significance in unpaired t tests. (E-F) Serum antibodies collected pre-immunization and 7- or 14-days post NP-CGG immunization, detected by NP7-BSA ELISA using isotype-specific secondary antibodies. Data show mean ± SEM from N = 3 experiments with statistical significance from 2-way ANOVA with multiple comparisons. (G) Ratio of binding to NP7 and NP25, as measured by ELISA. Data show mean ± SEM; N = 4-6 mice. Statistical significance from 2-way ANOVA with multiple comparisons. (H) Numbers of Tfh cells 14 days post-immunization, identified as a CD4+CD44+PD-1+CXCR5+ population. Mean and SEM of N = 4 mice. \n",
      ".Figure 2(A) Western blot showing loss of endophilin A2 protein after SH3GL1 targeting in Ramos cells using 2 independent sgRNAs, compared to 2 non-targeting sgRNAs. (B) Soluble anti-IgM F(ab')2 antigen internalization at 20 min in Ramos cells targeted with indicated sgRNAs. P, significance in one-way ANOVA versus control. Plot shows mean and SEM of 7 independent sgRNA infections. (C) Validation of gene candidates affecting soluble antigen uptake in primary mouse B cells. Follicular B cells isolated from CRISPR-targeted bone marrow chimera mice internalize anti-mouse IgM F(ab')2. Data show mean and SEM of N = 3 mice. P, statistics from 2-way ANOVA; at 5 min all 3 genotypes show p<0.0005 versus control; at 15 min p values are indicated in corresponding color. (D) Antigen internalization from PMS in SH3GL1-targeted Ramos cells. ****p<0.0001 in one-way ANOVA. N = 547, 389 and 136 cells respectively, from 2 experiments. (E) Internalization of soluble Fab or F(ab')2 fragments in Ramos cells targeted with indicated sgRNAs. *p=0.0124 **p=0.0092 ***p=0.0004 using 2-way ANOVA. Data show mean ± SEM, from 6 biological replicates across 2 experiments. (F) TIRF images of Ramos cells transduced with endophilin A2-GFP and mCherry-ClathrinLC on antigen-loaded PMS. Scale bar = 5 μm. White arrowheads highlight endophilin A2 colocalization with antigen clusters. (G) Percent of endophilin A2 and clathrinLC spots colocalizing with each other. Levels of random spot colocalization were calculated using randomized spot coordinates. Data show mean and SEM from 107 cells analyzed from 4 experiments; statistics from one-way ANOVA. (H) Percentage of surrogate antigen spots colocalizing with endophilin A2, clathrin LC or both markers. Levels of random spot colocalization were calculated using randomized spot coordinates. Data show mean and SEM from 107 cells analyzed from 4 experiments; statistics from one-way ANOVA. (I) Colocalization of surrogate antigen clusters with endophilin A2-GFP following deletion of clathrin adaptors EPN1 or PICALM1. Data show mean and SEM from 23-29 cells. (J) IgM-BCR immunoprecipitation to investigate recruitment of endophilin A2 to activated BCRs. Anti-endophilin A2 blots detect overexpressed endophilin A2-GFP in streptavidin immunoprecipitates and total supernatants from anti-IgM-biotin-stimulated Ramos and DG75 cells. Representative experiment and quantification from 3 independent pulldowns in WT, BLNK- or GRB2-knockout (KO) DG75 cells. Pulldown intensity is calculated as a percentage of total endophilin A2 protein and normalized to WT in each experiment. Data show mean and SEM; p values calculated using one-way ANOVA. (K) Recruitment of endophilin A2 to antigen clusters quantified as mean antigen intensity in endophilin A2 clusters relative to mean antigen intensity in the entire synapse. N = 30, 29 and 36 cells respectively, from 2 experiments. P, significance in Kruskal-Wallis test with multiple comparisons. \n",
      ".Figure 6(A) CRISPR-targeted follicular B cell proliferation after 3 days of CD40L stimulation. mCherry+ populations were quantified within individual DDAO peaks. N = 3 mice. P, statistical significance calculated by two-way ANOVA reached p<0.0001 for all proliferation peaks. Data information: All data show means ± SEM. (B) CD80 mRNA levels in B cells following 24 hours of CD40L culture, relative to untreated B cells. Data from RNAseq of N = 3 mice. (C-D) Isolated naïve B cells from CRISPR-targeted chimeras after 3-day culture in stated cytokines. (C) mCherry percentage in targeted primary cell cultures. N = 15 mice. *p<0.05, **p<0.01, ****p<0.0001 using two-way ANOVA. (D) AnnexinV surface stain in targeted cells. N = 3-4 mice. Data information: All data show means ± SEM. (E) mCherry+ cell percentage in CRISPR-targeted Ramos cells over time. N = 6 independent experiments. (F) Cell counts at passage 3-6 of CRISPR-targeted puromycin-selected Ramos cells. N = 4 independent infections. P, statistical significance from unpaired t test. Data information: All data show means ± SEM. (G) Oxygen consumption rate (OCR) in CRISPR-targeted Ramos cells. N = 2 independent infections. P, two-way ANOVA of basal and maximum OCR. (H) OCR in B cells isolated from Sh3gl1-/- and WT littermates. N = 9 mice. P, two-way ANOVA of basal and maximum OCR. (I) OCR in T cells isolated from Sh3gl1-/- and WT littermates. N = 4 mice. Cell cultures with full RPMI media supplemented with 50 μg/ml FeSO4 or 5 μM hemin chloride. (J) SH3GL1-targeted Ramos cells. N = 4 independent infections. P, statistical significance from two-way ANOVA from both treatments versus RPMI. (K) Control sgRNA-targeted Ramos cells. Cell cultures with full RPMI media supplemented with 50 μg/ml FeSO4 or 5 μM hemin chloride. (L) Sh3gl1-/- isolated B cells, maintained in CD40L culture. (M) B cells isolated from WT littermates, maintained in CD40L culture. (N) OCR in Sh3gl1-/- or WT B cells after overnight culture in CD40L, supplemented with 5 μM hemin chloride. N = 6 mice. P, two-way ANOVA with matched samples for basal and maximum OCR (O) TfR internalization assay in Ramos cells using biotinylated anti-CD71. 4 independent CRISPR infections in 2 experiments. P, significance calculated using two-way ANOVA and indicated in corresponding color. (P,Q) Mouse transferrin internalization in B cells from SRBC-immunized WT and Sh3gl1-/- littermates. N = 7 mice per genotype in 2 experiments. P, statistical significance analyzed using two-way ANOVA. (P) Naïve B cells. (Q) GC B cells. \n",
      "Fig.Figure 3C4da neurons were labeled by CD8::GFP expression under ppk-GAL4, and expression of Sox14 was assessed by immunofluorescence. A - C Sox14 stainings at 2 h APF. (A) Control c4da neurons. (B) C4da neurons expressing PP2A-29B dsRNA. (C) C4da neuron expressing dominant-negative Mts. D Quantification of Sox14 expression in (A - C). C4da neuron Sox14 expression was normalized to Sox14 expression in neighboring ddaE c1da neurons (identified by characteristic cell body shape). Data are mean +/- s. d.. The numbers of neurons (animals) for each genotype are given in each graph (data were not derived from independent experiments). n. s., not significant, *** P<0.0005, Wilcoxon's test. C4da neurons were labeled by CD8::GFP expression under ppk-GAL4, and expression of Mical was assessed by immunofluorescence. E - H Mical stainings at the indicated time points. (E) Mical expression in control c4da neurons at 2 h APF. (F) Mical expression in c4da neurons expressing dominant-negative Mts at 2 h APF. (G) Mical expression in c4da neurons expressing PP2A-29B dsRNA at 2 h APF. (H) Mical expression in c4da neurons expressing Orco dsRNA at 0 h APF. (I) Mical expression in c4da neurons expressing PP2A-29B dsRNA at 0 h APF. J Quantification of Mical expression in (E - I). C4da neuron Mical expression was normalized to Mical expression in neighboring ddaE c1da neurons (marked by asterisks). Data are mean +/- s. d.. The numbers of neurons (animals) for each genotype are given in each graph (data were not derived from independent experiments). n. s., not significant, ** P<0.005, *** P<0.0005, Wilcoxon's test. Data information: Scale bars in A and E are 10 μm. \n",
      ".Figure 5(A) Soluble anti-IgM internalization in CRISPR-targeted follicular B cells. N = 11- 12 mice. Data show mean ± SEM. P, statistical significance from two-way ANOVA. (B) Soluble anti-IgD internalization. N = 3 mice. Data show mean ± SEM. P, statistical significance from two-way ANOVA. (C) Soluble anti-IgG internalization in GC B cells from SRBC-immunized mice. N = 4 mice. Data show mean ± SEM. P, statistical significance from two-way ANOVA. (D) Internalization of anti-IgM from PMS quantified in fixed naïve B cells at 10 min post synapse formation. N = 172 and 97 cells analyzed in one out of 2 representative experiments. Data show mean ± SEM. P = 0.0001 in unpaired t test. (E) Surface IgM or IgD MFI relative to WT in naïve splenic follicular B cells. N = 3 mice. Data show mean ± SEM. (F) IgM degradation following total cell surface biotinylation, measured as loss of biotin-labelled membrane IgM (mIgM) in cell lysates over time. (G) Western densitometry quantifying IgM degradation assay. Proportion of biotin detected is normalized to the initial WT condition (left) or to the initial amount for each genotype (right). Data show mean ± SEM from three independent tests. P, statistical significance from two-way ANOVA. (H) MHCII antigen presentation in SWHEL B cells 24 hours post adoptive transfer and immunization with Ea-HEL-SRBC conjugate as detected by anti-Y-Ae surface stain. (I) Relative anti-Y-Ae MFI in CRISPR-targeted B cells normalized to HEL-only immunized mice. N = 10 immunized mice across 3 experiments. Data show mean ± SEM. (J) In vitro antigen presentation in CRISPR-targeted B cells measured by anti-Y-Ae. N = 8 independent cultures in 2 experiments. Data show mean ± SEM. (K) Proliferation (CFSE dilution) of OTII CD4 cells in co-cultures with anti-Igκ-OVA pulsed B cells; and quantification from 6-8 mice. Data show mean ± SEM. (L) Relative anti-Y-Ae MFI in GC from immunised wildtype and Sh3gl1-/- mice. N = 11 from 2 experiments. Data show mean ± SEM. \n",
      "Fig.Figure 6Our data indicate a two-pronged effect of PP2A on pruning: firstly, PP2A is required for timely expression of Mical via an unknown mechanism. Secondly, PP2A appears to affect actin dynamics in neurons during both the larval and pupal stages. \n",
      ".Figure 1A Bacterially purified recombinant EDC3 (0.2μg) was incubated in the presence and absence of biotinylated recombinant Pim1 (0.5 μg) protein with Streptavidin-coated magnetic beads. The beads were precipitated and subjected to Western blotting with EDC3 and Pim1 antibodies (Abs). B Pim1, 2, or 3 and GFP-EDC3 expression vectors were transfected into HEK-293T. GFP-Trap Nano-body beads (GFP-Trap) were used to immunoprecipitate (IP) GFP-EDC3, and the associated proteins were analyzed by Western blotting with the indicated Abs. C Western blot of endogenous immunoprecipitation (IP) with EDC3 in MDA-MB-231 cells using Protein G Mag Sepharose with either control IgG or EDC3 primary antibody. D Pim1 kinase dead (KD) and GFP-EDC3 plasmids were co-transfected into HEK-293T cells. The lysates were precipitated with GFP-Trap beads. The IP products were subjected to Western blotting with the indicated Abs. \n",
      ".Figure 2A Schematic of the functional domains of EDC3 and the fragments used to construct GFP fusion proteins. The phosphorylation site on EDC3 serine 161 is shown. B HEK-293T cells were transfected with Pim1 and each EDC3 domain (Fig. 2A) fused to the C-terminus of the GFP-tagged vector. The individual fragments were immunoprecipitated with GFP-Trap beads and analyzed by Western blotting with the indicated Abs. EDC3 wild type (W), F1, F2, F3, F4 were EDC3 fragments vector fused to GFP. The C1 lane was only transfected with the GFP Flag vector while C2 contained EDC3 wild type fused to GFP and no Pim1 but empty vector control (pcDNA3). C 2D [1H,15N] HSQC spectrum of EDC3104-197 (100 µM EDC3104-197; 20 mM HEPES pH 6.2, 500 mM NaCl, 0.5 mM TCEP; 298 K; 14.1 T). HSQC spectrum shows a single cross peak for each N-HN pair, i.e. one per amino acid (other than prolines). The lack of chemical shift dispersion in the 1H chemical shift dimension is typical for intrinsically disordered protein/region (IDR), as they lack the hydrogen bonding network in secondary structure elements of folded proteins. D 2D [1H,15N] HSQC spectra of EDC3104-197 with (red; 1:2 molar ratio) and without (blue) Pim1. We titrated Pim1 into 15N-labeled EDC3, and the 2D [1H,15N] HSQC spectrum of EDC3 (residues 104-197) when bound to Pim1 was measured. Direct comparison with the unbound 2D [1H,15N] HSQC spectrum of EDC3 showed that while the majority of EDC3 peaks are unchanged, ~30 peaks become either broadened beyond detectability demonstrating they directly bind to Pim1 or show small chemical shift perturbations. Upon completion of the sequence-specific backbone assignment of the EDC3 IDR, we identified that EDC3 residues 137 to 167 are necessary for the interaction with Pim1. \n",
      "Fig.Figure 5A - H Genetic interactions between PP2A-29B and actin regulators during dendrite pruning. The indicated manipulations of actin regulators were tested for their ability to suppress pruning defects induced by PP2A-29B dsRNA expression. Suppression tests were performed as in Figure 2. (A) C4da neuron coexpressing PP2A-29B dsRNA and tdtomato as suppression control. (B) C4da neuron coexpressing PP2A-29B dsRNA and GFP::tsr. (C) C4da neuron coexpressing PP2A-29B dsRNA and RacN17. (D) C4da neuron coexpressing PP2A-29B dsRNA and RhoN19. (E) C4da neuron expressing PP2A-29B dsRNA in ena210/+ background. (F) C4da neuron coexpressing PP2A-29B dsRNA and Orco dsRNA as suppression control. (G) C4da neuron coexpressing PP2A-29B dsRNA and Chicadee dsRNA. (H) C4da neuron coexpressing PP2A-29B dsRNA and moesin dsRNA. I Penetrance of pruning defects in panels E - L. n. s., not significant, Fisher's exact test. Numbers of neurons (animals) for each genotype are given in the figure. J Severity of pruning defects in panels E - L as assessed by number of primary and secondary dendrites attached to soma at 18 h APF. Data are mean +/- s. d., * P<0.05, ** P<0.005, *** P<0.0005, Wilcoxon's test. Numbers of neurons (animals) for each genotype are given in the figure. \n",
      ".Figure 4A Immunofluorescence (IF) staining of PC3-LN4 cells following 6 h treatment with DMSO (Control), PIMi (PIM447, 3µM), AKTi (GSK690693, 5μM), and both inhibitors (AKTi + PIMi) and stained with anti-EDC3 and anti-DCP1a antibodies as well as DAPI. Data information: IF Images were subsequently merged using FIJI image analysis software. The yellow box shows a zoom of the merged image. Images were captured under 40x magnification; scale bar: 10µm. P-body number was counted by FIJI. Quantification of EDC3/DCP1a foci number from Control (Ctrl), PIMi, AKTi and the AKTi + PIMi combination in (A). Data information: The quantification data of (B are presented as mean +/- SEM (n = 30 cells/treatment group). p values<0.05, calculated using unpaired student t-test, are considered significant. p values>0.05 not significant (n.s.). The analyses done included multiple independent fields. C Immunofluorescence (IF) staining of PC3-LN4 EDC3 knock out cells following 6 h treatment with DMSO (Control), PIMi (PIM447, 3µM), AKTi (GSK690693, 5μM), and both inhibitors (AKTi + PIMi) and stained with anti-EDC3, and anti-DCP1a antibodies (Abs) as well as DAPI. Data information: IF Images were subsequently merged using FIJI image analysis software. The yellow box shows a zoom of the merged image. Images were captured under 40x magnification; scale bar: 10µm. P-body number was counted by FIJI. D Quantification of DCP1a foci number for Control (Ctrl), PIMi, AKTi, and AKTi + PIMi combination in (C). Data information: The quantification data of D) are presented as mean +/- SEM (n = 30 cells/treatment group). p values<0.05, calculated using unpaired student t-test, are considered significant. p values>0.05 not significant (n.s.). The analyses done included multiple independent fields. \n",
      "Fig.Figure 4A - G The indicated UAS-tsr constructs were tested for their ability to suppress pruning defects induced by PP2A-29B dsRNA expression. Suppression tests were performed as in Figure 2. Panels A - C show effects of the expression of wild type or phosphomutant tsr variants on dendrite pruning, panels D - G show neurons coexpressing PP2A-29B dsRNA. (A, E) C4da neuron (co-) overexpressing wild type tsr. (B, F) C4da neuron (co-) overexpressing tsr S3A. (C, G) C4da neuron (co-) overexpressing tsr S3E. The arrow in panel G denotes a characteristic varicosity at the tip of an unpruned dendrite. (D) C4da neuron co-expressing PP2A-29B dsRNA and lacZ as a UAS titration control.J - L GFP-tagged Drosophila cofilin (GFP::tsr) was expressed in c4da neurons under ppk-GAL4, and its localization was assessed at the indicated developmental stages by live imaging. Panels J - L show GFP::tsr localization in c4da neurons at the third instar stage, panels J' - L' show GFP::tsr localization in c4da neurons at 6 h APF. (J, J') Control c4da neurons expressing Orco dsRNA. (K, K') C4da neurons expressing Mical dsRNA. (L, L') C4da neurons expressing PP2A-29B dsRNA. M Quantification of GFP::tsr punctae in samples J - L'. Data are mean +/- s. d., the numbers of neurons (animals) analyzed for each genotype are given in the figure (data were not derived from independent experiments). ** P<0.005, *** P<0.0005, n. s., not significant, Wilcoxon's test. \n",
      ".Figure 3A In vitro kinase assays were performed as outlined in Materials and Methods with purified recombinant EDC3 (0.2μg) and Pim1 (0.2μg). Samples were analyzed by Western blotting with the indicated Abs. B In vitro kinase assay with EDC3 (0.2μg) and AKT (0.1μg). Samples were analyzed by Western blotting with the indicated Abs. C PC3-LN4 cells containing a doxycycline (DOX) inducible Tripz-Pim1 vector were treated with 50 or 100 ng/ml DOX for 48 h and subsequently incubated with DMSO or PIMi (PIM447, 3 µM) for 24 h. Western blots were probed with the indicated Abs. D PC3-LN4 cells was treated with PIMi (PIM447) at increasing doses (0, 3, 5, and 10μM) for 24 h. Extracts were subjected to Western blotting with the indicated Abs. Relative values of phospho-signals were indicated below p-EDC3 blot by quantification of intensity for p-EDC3 S161 levels using ImageJ (NIH). p-EDC3 S161 levels were normalized to levels of total EDC3 within the respective samples and compared to the p-EDC3 S161 levels in the untreated samples. E PC3-LN4 was treated with PIMi (PIM447, 3μM), AKTi (GSK690693, 5μM), or the combination for 24 h. Extracts were subjected to Western blotting with the indicated Abs. F PC3-LN4 CRISPR-Ctr and Pim1 KO cells were treated with PIMi (PIM447, 3μM), AKTi (GSK690693, 5μM), or the combination for 24 h. Extracts were subjected to Western blotting with the indicated Abs. \n",
      ".Figure 5A, B Western blot of whole-cell (WCL) and P-body (PB) purified from PC3-LN4 cells stably expressing GFP-DCP1a used as a P-body tag. The P-bodies (PB) were isolated with GFP-Trap beads and analyzed by Western blotting with the indicated Abs. C Immunofluorescence (IF) of PC3LN4 cells under normal culture condition with anti-phopsho-EDC3, anti-EDC3 Abs, and DAPI. Images captured under 40x magnification; scale bar: 10µm. D Immunofluorescence of PC3-LN4-EDC3 S161A (SA) cells under normal culture condition with anti-DCP1a and anti-EDC3 Abs as well as DAPI. Images captured under 40x magnification; scale bar: 10µm. \n",
      ".Figure 6A Comparison of P-body RNA and total RNA levels with and without AKT and PIM inhibitor treatment by anota2Seq. B Specific RNA subtypes found in P-body RNAs purified from DMSO treated GFP-EDC3 containing PC3-LN4 cells. \n",
      "Humanization of the Ccnd1 exon 4/5 locus results in exclusive production of cyclin D1b.Figure 1Representative schematic of the targeting construct generated to humanize the Ccnd1 exon 4/5 genomic locus to produce cyclin D1b. Top: Schematic of primer pairs designed to discriminate between wild-type and knock-in alleles. Bottom: Representative genotyping of Ccnd1+/+, Ccnd1+/KI, and Ccnd1KI/KI mice validating specificity of the primer pairs and somatic insertion of the targeting construct. PCR analysis of transcript b expression in organs harvested from Ccnd1+/+ and Ccnd1KI/KI mice, demonstrating production of transcript b specifically in KI mice. Gapdh serves as a control (NTC, Non-template control). Immunoblot from parallel samples in (C) utilizing antisera specific to the 33 amino acids generated by human CCND1 intron 4. Lamin B serves as a control. Arrow indicates the cyclin D1b band. \n",
      ".Figure 8A Two independently derived CRISPR/Cas9 PC3-LN4 EDC3 S161A mutants (EDC3 SA#1, #2) and wild type cells were injected subcutaneously in the flank (5X106) of SCID mice (n=8/group), and tumor growth was monitored until the control group reached 1000 mm3. The subcutaneous tumor volume measurement (mm3) was done as outlined in Materials and Methods. Data are presented as mean ± SEM, (n = 8/group). B Phosphorylated EDC3 levels were examined in normal immortalized breast cell lines, MCF10A and MCF12A, human breast cancer cell lines HCC38, HCC1954, MDA-MB-231, MDA-MD-468, BT-549, MRKNU1, and mouse 4T1 by Western blotting with the indicated Abs. C Dual immunohistochemistry staining of human breast tumor with p-EDC3 Ab (Fredriksson et al.) and total EDC3 (brown). Scale bar: 50µm. D Dual immunohistochemistry staining of human normal resected breast tissue sections with p-EDC3 Ab (Fredriksson et al.) and total EDC3 (brown). Scale bar: 50µm. \n",
      "Cyclin D1b selectively rescues Ccnd1−/− phenotypes.Figure 2Neonates from 2 independent litters of Ccnd1+/KI × Ccnd1+/KI crosses were sacrificed at birth and genotyped. Mice were organized by genotype and total size measured.Mass of neonates was calculated at birth and average mass quantified (n = at least 4 mice per group).Mice were weighed weekly, and growth is plotted as the average mass of each genotype/week (left: male, right: female, n > 5 per gender and genotype). Mice were held by the tails and analyzed for the leg clasping phenotype. Representative images of Ccnd1+/+ and Ccnd1KI/KI age-matched mice are shown (n > 10 for each group). Top: Individual whole-cell lysates were generated from the eyes of mice of each genotype and analyzed for expression of +/+ cyclin D1 (cyclin D1a) and KI/KI cyclin D1b via immunoblot using antisera specific to each isoform. Bottom: Representative H&amp;amp;E staining of retinal tissue from Ccnd1+/+ and Ccnd1KI/KI mice showing individual layers of the retina. The ganglion cell layer and inner plexiform layer (G + IPL), inner nuclear layer (INL), and outer nuclear layer (ONL) were quantified (right). Images were taken at 400× magnification. Comparison of the phenotypes described in the Ccnd1−/− mouse with those observed in the Ccnd1KI/KI mouse, previously described (Sicinski et al, 1995). \n",
      ".Figure 7A PC3-LN4 Naïve and CRISPR/Cas9 genome-edited EDC3 S161A (SA), S161D (Fredriksson et al.), and EDC3 knock-out (KO) cell lines were plated (4000 cells/well in 24 well plates) and imaged every 12 h using phase contrast in the IncuCyte ZOOM platform. The graph shown was generated with IncuCyte Basic Software following 5 days of growth. The experiments were done in triplicates. Values are mean +/- S.E.M. p values<0.05, calculated using unpaired student t-test. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. B Quantification of transwell migration assays performed for PC3-LN4 EDC3 wild type, EDC3 S161A, EDC3 KO, and EDC3 S161D cell lines. The number of migrated cells (40 h) was counted as outlined in Materials and Methods. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. C Quantification of transwell migration assays performed for PC3-LN4 EDC3 KO cells containing Doxycycline (DOX)-inducible EDC3-S161A with and without DOX (100ng/ml) induction. The number of migrated cells (40 h) was counted as outlined in Materials and Methods. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. D Quantification of transwell invasion assay comparing PC3-LN4 EDC3 wild type, EDC3 S161A, EDC3 KO, and EDC3 S161D cell lines. Cells were plated in trans-wells coated with Matrigel and incubated for 72 h. The cells that invaded the matrix were counted as outlined in Materials and Methods. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. E Quantification of transwell invasion assay performed for PC3-LN4 EDC3 KO cells containing DOX- inducible EDC3-S161A were either treated with or without DOX (100 ng/ml). The number of cells that invaded were counted as outlined in Materials and Methods. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. F Relative mRNA expression of ITGA6, ITGB1, and KLF4 in PC3-LN4 WT and EDC3 S161A mutant cell lines. Data information: Real-time PCR data (F are presented as mean ± SEM, (n = 3/group). Unpaired student's t-test was used for statistical analysis. *P < 0.05, **P < 0.01 and NS, not significant. G Western blot analysis of proteins extracted from PC3-LN4 EDC3 wild type, EDC3 S161A, EDC3 knockout (KO, C60 cell line), and EDC3 S161D cell lines using the specified antibodies. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. H Relative mRNA expression of ITGA6, ITGB1, and KLF4 in PC3-LN4 EDC3 KO-C60 cells, containing reexpressed EDC3 WT or EDC3 S161A mutant cDNAs Data information: Real-time PCR data I Western blot analysis of proteins extracted from PC3-LN4-EDC3-KO cells expressing GFP, EDC3 WT or the EDC3 S161A mutant using the specified antibodies. Data information: All invasion and migration assays were performed in triplicates and 6 different fields at 10x magnification were counted in 3 separate inserts. The data are presented as the mean +/- S.D. p values<0.05, calculated using unpaired student's t-test. J Western blot analysis of proteins extracted from PC3-LN4 cells treated with DMSO or the combination of Pim447 and AZD5363 for 96 h using the specified antibodies. \n",
      "Cyclin D1b cooperates with h-Ras to drive tumorigenesis.Figure 4Passage-matched Ccnd1+/+ or Ccnd1KI/KI MAF lines were infected with lentivirus containing vector control or h-Ras constructs. Cells were selected with puromycin for 14 days and assayed for β-galactosidase activity. DAPI serves as a control for cell number. Images were taken at 200× magnification with insets taken at 400× magnification. The indicated cell lines were grown in complete media and assayed for h-Ras expression via immunoblot. Lamin B serves as a loading control. Cells were plated in soft agar and allowed to grow for a period of 3 weeks. Plates were then washed, fixed, and stained with 0.01% crystal violet. Colonies greater than 50 μm were counted and are plotted (top). Representative images of colony growth are shown for each cell line. Images were taken at 40× with insets taken at 200× magnification. Indicated MAF lines were injected subcutaneously into the flanks of nude mice as in (A), and tumor incidence is plotted as % of tumor-free mice over time (n = 10). Indicated tumors were harvested, fixed, and stained for the proliferative marker mKi67. Three random fields from each tumor were counted for mKi67 positivity, and are plotted as a % of total cell number, n = 5 (right: representative images). Images were taken at 400× magnification. \n",
      "Genetic switch to cyclin D1b promotes serum independence.Figure 5Ccnd1+/+ and Ccnd1KI/KIMAF lines were arrested in G1, S, and G2/M phases of the cell cycle and expression of cyclin D1 isoforms determined by immunoblot. cyclin A2 and cyclin B1 serve as phase-specific cell cycle controls. Cells were plated in serum-proficient media and protein lysates generated from each genetic line. Individual D1 cyclins were immunoblotted to verify genetic identity along with other essential cell cycle components. Lamin B serves as a control. Indicated cells lines were immunoprecipitated for CDK4 and analyzed for their ability to incorporate 32P-ATP into full-length RB substrate. Radioactive counts were normalized to CDK4 activity of +/+ cell lines and adjusted for efficiency of CDK4 pull down as determined by densitometry. Data is representative of three independent biological replicates. Indicated cell lines were plated in serum-proficient (10%) or serum-deficient (1%) media and total cell number counted at 24, 48, and 72 h in biological triplicate. Ccnd1+/+ and Ccnd1KI/KI MAF lines were transfected with a validated pool of siRNA directed against the N-terminus of the murine cyclin D1 transcript or control siRNA in full serum. 48 h post-transfection cells were harvested and analyzed for biochemical markers of cell cycle kinetics via immunoblot. GAPDH and Lamin B serve as controls. Cells were treated as (E), and incubated in the indicated serum concentration for 48 h. Cells were treated with BrdU for 1 h prior to harvesting and then stained for BrdU incorporation. Three random fields from each of three biological triplicates were counted for BrdU incorporation via immunofluorescence and data is represented as percent positive/total cell number. \n",
      "Genetic evidence for cyclin D1b as a credentialed oncogene.Figure 3Murine adult fibroblasts (MAFs) were harvested from the peritoneum of Ccnd1+/+ and Ccnd1KI/KI mice, and stable cell lines were generated utilizing a 3T3 protocol. Indicated passage-matched MAF lines were grown to 70% confluency and then harvested for RNA and protein extraction. Top: qPCR analysis of cyclin D1 transcript using primer pairs common to both transcript a and transcript b. Bottom: Immunoblot of cyclin D1 levels in Ccnd1+/+ and Ccnd1KI/KMAF lines using an antibody common to both isoforms. Percentage of tumor-free mice post-injection of 1 million cells mixed with Matrigel (1:1 ratio) into the flanks of nude mice over a period of 10 months (n = 10 per genotype). Indicated tumors were harvested 3 weeks after detectable tumor formation, fixed in formalin, and stained with H&amp;amp;E. Tumors were analyzed for features of malignancy by a board-certified veterinary pathologist. Arrows indicate the specific tumor-associated features noted. Top panels were taken at 40× magnification, and lower panels were taken at 400× magnification. Ccnd1+/+ or Ccnd1KI/KI MAF lines were stably transfected with either cyclin D1a (+/+), cyclin D1b (KI/KI) or vector control constructs. After selection with puromycin, individual lines were assessed for the induction of cyclin D1a in the +/+ line and cyclin D1b in KI/KI lines via immunoblot. One million cells from the indicated stable cell lines were injected subcutaneously into the flanks of nude mice and monitored for tumor formation over a period of 365 days. Indicated time points represent time of palpable tumor detection and are plotted as % of tumor-free mice over time. Sections from KI/KI vec and KI/KI-D1b tumors were stained for the proliferative marker Ki67. Three random fields from each tumor were quantified for Ki67 positivity, and are plotted as the mean ratio of Ki67-positive cells/total cell number for each individual tumor. \n",
      "Cyclin D1b expression promotes double-strand breaks and PARP-1 activity.Figure 6Ccnd1+/+ and Ccnd1KI/KI cells were transfected with a pool of siRNA directed against the N-terminus of the murine cyclin D1 transcript or scramble control for 48 h (as in Figure5). Cells were then fixed and stained for markers of double-strand breaks (p-H2AX and 53BP1, 400× objective). Total number of foci for each cell was quantified and represented as the % of cells with > 10 foci/cell (p-H2AX) or % of cells with > 2 foci/cell (53BP1). Error bars represent ± SEM. Ccnd1+/+ and Ccnd1KI/KIMAF lines were grown in serum-proficient media for 24 h and probed for expression of auto-modified PARP1 via immunoblot. Gapdh and cyclin D1b serve as controls. Cells were plated as in (B) and treated with 0.5 μM etoposide for 3 h. Cells were harvested and then probed for markers of DNA damage via immunoblot. Lamin B serves as a control. Indicated MAF lines were treated as in (A) and harvested 48 h post-transfection. Cells were analyzed for total PAR levels via immunoblot. cyclin D1 levels serve as siRNA validation controls. cyclin D1b expression was induced in the prostate cancer cell line LNCaP (previously described (Augello et al, 2014)) and stained for p-H2AX and 53BP1 foci as in (A) via immunofluorescence (400× objective). Total number of foci/cell is reported for LNCaP vector control and cyclin D1b-expressing isogenic pairs in biological triplicate. Isogenic pairs from (E) were grown in serum-proficient media for 24 h, harvested, and probed for total PAR levels via immunoblot. \n",
      "Model of cyclin D1b function in development and tumorigenesis.Figure 8Top: Comparison of the observed phenotypes noted in the Ccnd1KI/KI and Ccnd1−/− mice (Sicinski et al, 1995). Bottom: Summation of the pro-oncogenic phenotypes associated with cyclin D1b expression in cell-based models. \n",
      "Cyclin D1b expression sensitizes cells to senescence induced by therapeutic challenge.Figure 7Ccnd1+/+ and Ccnd1KI/KIMAF lines were grown in serum-proficient media for 24 h. Cells were then treated with control DMSO or 2.5 μM of the PARP inhibitor ABT-888. One hour post-treatment, the indicated lines were treated with 5 Gy of radiation. Cells were then allowed to recover for 48 h, after which BrdU was added for 1 h, and then harvested for bivariate flow cytometry. Representative traces for each condition are shown (left) and BrdU incorporation of biological triplicates was quantified (right). Cells were plated and treated as in (A). Forty-eight hours post-treatment, cells were fixed and stained for markers of senescence (β-galactosidase activity). Cells positive for the staining (blue) were quantified for each condition (400× magnification) and reported as a percentage of the total population (right). Plates treated in parallel with both IR and ABT-888 were harvested at 48 h via 1× trypsin and re-plated in serum-proficient media. Cells were allowed to grow for 96 h and were then stained with crystal violet (4× objective, with inset at 200× magnification, boxes highlight area of magnified images). \n",
      ".Figure 2GC-MS/MS profile of volatiles produced by 24 h old lawn of P. aeruginosa PA14. Chemotactic response of N2 worms to varying dilutions of 1-undecene. n ≥ 3 assays. Error bars indicate SEM. Omega turn in N2 worms under 1-undecene exposure. Scale bar = 1 mm. Schematic of PA14 (24 h) and undA mutant (24h) bacterial lawn choice assay in a tripartite plate and choice index. n ≥ 3 assays. Error bars indicate SEM. (i) Schematic of setup used to record the movement of 5 worms each under control and 1-undecene odor exposure. (ii) Movement trajectories of N2 worms on E. coli OP50 lawn exposed to 1-undecene odor for 30 min and 60 min, each color represents the trajectory of single worm. Scale bar = 10 mm. \n",
      ".Figure 1Time course of avoidance of P. aeruginosa lawn by N2, odr-3(n2150) and odr-3(n2046) worms. n ≥ 3 assays. *** P ≤ 0.001, **** P ≤ 0.0001 as determined by two-tailed unpaired t-test. Error bars indicate SEM. Images of P. aeruginosa lawn at different time intervals showing avoidance response of N2 and odr-3(n2150) mutant worms. Scale bar = 5 mm. Kaplan Meier survival curve of N2, odr-3(n2150) and odr-3(n2046) worms on partial lawn of P. aeruginosa at 25°C.   Kaplan Meier survival curve of N2, odr-3(n2150) and odr-3(n2046)   worms on full lawn of P. aeruginosa at 25°C. Time required for 50% of worms to die (TD50) on partial lawn and full lawn of P. aeruginosa. Individual data points indicate replicates with ~100 worms each. **** P ≤ 0.0001, ns (not significant) P > 0.05 as determined by two-tailed unpaired t-test. n ≥ 3 assays. Error bars indicate SEM. Schematic of young (8 h) and old (24 h) lawn choice assay in a tripartite plate and choice index. n ≥ 3 assays. Individual data points indicate replicates with ~40 worms each. * P ≤ 0.05 as determined by two-tailed unpaired t-test. Error bars indicate SEM. Representative image of intestinal lumen width, indicated by arrows, in N2 worm exposed to P. aeruginosa for 12 h. Scale bar = 50 µm. Quantification for the width of intestinal lumen in N2 exposed to E. coli and P. aeruginosa for 12 h. ns (not significant) P > 0.05 as determined by two-tailed unpaired t-test. n = 3. Error bars indicate SEM. \n",
      ".Figure 5A non-heme oxidase UndA is required for the production of a 11-carbon olefin, 1-undecene, a volatile. This volatile induces calcium signaling in AWB odor sensory neurons leading to aversion response (FLIGHT) as well as upregulation of pathogen-specific immune response genes (FIGHT) and resistance in C. elegans. \n",
      ".Figure 3Chemotactic dose-response of N2 and odr-3(n2150) worms to 1-undecene. n ≥ 3 assays. ns (not significant) P > 0.05, *** P ≤ 0.001, **** P ≤ 0.0001 as determined by one-way ANOVA, followed by Dunnett's multiple comparison test. Error bars indicate SEM. Chemotaxis of N2, odr-3(n2150), odr-3(n2046), AWC(-), lim-4(ky403), lim-4(yz12), odr-7(ky4), AWA(-) and ASH(-) worms to 1-undecene at 1:10 dilution. n ≥ 3 assays. ns (not significant) P > 0.05, **** P ≤ 0.0001 as determined by one-way ANOVA, followed by Dunnett's multiple comparison test. Error bars indicate SEM. Average calcium responses of AWB::GCaMP3 worms exposed to 1-undecene at indicated concentrations. Each worm was recorded for 180 s. Worms were under stimulus between 11 s-130 s, window shown in grey, and stimulus withdrawal beyond 130 s. Shaded regions around the curves represent SEM. n = 3. Two dashed box represents 10 s time window after stimulus addition and stimulus removal respectively. Average percentage change in ∆F/F for 10 s time window after stimulus addition and withdrawal (dashed boxes in C), each data point represents average data of ≥ 13 worms. ** P ≤ 0.01, **** P ≤ 0.0001 as determined by one-way ANOVA, followed by Dunnett's multiple comparison test. Error bars indicate SEM. Representative image of GCaMP3 fluorescence in AWB neuron during 1-undecene exposure (129 s on x- axis of fig. 3C) and after 1-undecene withdrawal (131.2 s on x-axis of Fig. 3C). Scale bar = 10 µm. Heat map of ratio ∆F/F (change in fluorescence to total fluorescence) in AWB::GCaMP3 neuron upon the addition (10 s) or the removal (130 s) of 1-undecene. Each row represents an individual worm recorded for 180 s under 1:100 dilution of 1-undecene, in 11 s-130 s window. \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ".Figure 4Real time PCR analysis of P. aeruginosa specific immune response genes in naive and 1-undecene odor exposed N2 worms. n = 3. * P ≤ 0.05, ** P ≤ 0.01 as determined by two-tailed unpaired t-test. Error bars indicate SEM. The negative values are arrived at by representing FC value less than 1 as (-1/FC). Real time PCR analysis of irg-1, irg-2 and irg-3 genes in N2 and zip-2(tm4248) worms exposed to 1-undecene odor upon respective naive worms. n = 3. ns (not significant) P > 0.05, * P ≤ 0.05, ** P ≤ 0.01 as determined by two-tailed unpaired t-test. Error bars indicate SEM. irg-1p::GFP induction in naive worms and worms exposed to P. aeruginosa (6 h) and 1-undecene odor (2 h). Scale bar = 500 µm. Real time PCR analysis of irg-1, irg-2 and irg-3 genes in N2, odr-3(n2150) and odr-3(n2046) worms exposed to 1-undecene odor upon respective naive worms. n = 3. * P ≤ 0.05, ** P ≤ 0.01 as determined by two-tailed unpaired t-test. Error bars indicate SEM. Real time PCR analysis of irg-1, irg-2 and irg-3 genes in N2, lim-4(ky403) and lim-4(yz12) worms exposed to 1-undecene odor upon respective naive worms. n = 3. * P ≤ 0.05, ** P ≤ 0.01, *** P ≤ 0.001 as determined by two-tailed unpaired t-test. Error bars indicate SEM. Kaplan Meier survival curve on the full lawn of P. aeruginosa for N2 (naive) worms and N2 worms pre exposed to 1-undecene odor. Survival assay was performed at 20°C. Time required for 50% of worms to die (TD50) on lawn of P. aeruginosa. Each data point indicates replicates with ~100 worms each. n = 3 assays. ** P ≤ 0.01 as determined by two-tailed unpaired t-test. Error bars indicate SEM. Real time PCR analysis of irg-1, irg-2 and irg-3 genes in N2 worms exposed to odor of undA mutant upon PA14. n = 3. * P ≤ 0.05, ** P ≤ 0.01 as determined by two-tailed unpaired t-test. Error bars indicate SEM. \n",
      ".Figure 3A. Domain organization of Sec62. The N-terminal cytoplasmic domain (Sec62 domain) is colored in rainbow. B. Two views on the crystal structure of the Sec62 domain consisting of α-helical bundle and a β-barrel domain. C. Fit of the Sec62 domain crystal structure into the respective cryo-EM density of the ss-bound heptameric The map is low-pass filtered to 15 Å. D. Surface representation of the Sec62 domain colored according to the electrostatic potential. The viewing angle is the same as in the right panel in (B). \n",
      ".Figure 1A. Schematic diagram depicting the recombinant ppαF-mEGFP substrate (upper panel) and purification strategy for the signal sequence-bound Sec complex (lower panel): Detergent-solubilized heptameric Sec complex (TM domains: green; cytosolic domains: cyan; detergent micelles: blue) FLAG-tagged on Sec62 was immobilized on anti-FLAG beads and incubated with molar excess of purified recombinant ppαF-mEGFP substrate. Substrate-bound Sec complex was eluted with 3C protease. B. SDS-PAGE of the reconstituted signal sequence-bound Sec complex stained with Coomassie Blue. C. Cryo-EM reconstruction as obtained after focused refinement of TM region. The map was segmented and color-coded by the individual Sec complex components. For clarity density for the detergent micelle was masked and the contour levels for the for Sec62 TMs and Sec71/72 densities are slightly lower. For the Sec62 cytosolic domain the map is low-pass filtered to 15 Å. D. Molecular model of the signal sequence-bound Sec complex. The model for the Sec62 N-terminal cytoplasmic domain (Sec62 domain) is derived from the crystal structure as determined in this work. \n",
      ".Figure 2A. Cut side view of the cryo-EM density of the signal sequence-bound (up) and the apo (bottom) Sec complex highlighting the ppαF signal sequence (ss; red) and Sec62 cytosolic and transmembrane domains (orange). Both maps were low-pass filtered to 8 Å. The density of the Sec62 cytosolic C-terminal domain connected to the Sec62 TM2 is marked by an arrowhead. B. Two views showing the conformations of the Sec61 complex (green) with the signal sequence (ss; red) bound to the lateral gate (upper panels) and in the apo state. The two Sec62 TMs (orange) are stabilized in ss-bound state. The plug is indicated by \"P\" in the apo state. C-E. Comparison of Sec61 complex conformations in the ss-bound (pale green) with the apo (dark green) Sec complex from this study (C), with the apo Sec complex from Itskanov (PDB 6N3Q, D) and with ss-opened ribosome-bound Sec61 complex (PDB 3JC2, E). Alignments of structures were based on TMs 7-9. The black arrows indicate the movement of Sec61α TM helices of the Sec61 complex. The plug is indicated by \"P\". \n",
      ".Figure 5In the apo state of Sec complex, the Sec61 channel is closed by the plug while the lateral gate is already open. Sec62 flexibly associates with the Sec complex mainly through a basic surface on its Sec62 domain interacting with the acidic C-terminus of Sec63. During the insertion of a post-translational client, its signal sequence (ss) binds to the groove at lateral gate supported by the Sec62 TM2. At the same time, the Sec62 TM1 pushes the Sec61α TM3 outwards leading to an even more open lateral gate. This also leads to removal of the plug away from the pore ring, allowing, the translocating peptide to be gated through the Sec61α central pore. \n",
      ".Figure 4A. Domain organization of Sec63. The sequence of the acidic patch on the C-terminus - divided into C1 and C2 regions - is shown in the lower panel. The two phosphorylated threonines (T652 and T654) are marked in the C2 region. The J domain of Sec63 is indicated by \"J\". B. Pull-down assay of Sec62 domain with different biotin-labeled Sec63 C-terminal peptides. Sec63-C2P denotes di-phosphorylated Sec63-C2. C. ITC-thermogram obtained from ITC after titration of Sec62 domain with increasing amounts of Sec63-C2P. D. Difference in HDX between the Sec62 domain in complex with Sec63-C2P and the Sec62 domain alone. The graph depicts the summed means of differences ± 3 SD (n = 3 technical replicates) for each peptide plotted according to the midpoint of each peptide. E. Crystal structure of Sec62 domain color coded according the difference in HDX The critical residues constituting the interaction interface between Sec63-C2P and Sec62 are labelled. F. Pull-down assay of the Sec62 domain and a mutant in the interaction interface (Sec62R51E) with biotin-labeled Sec63-C2P. \n",
      ": SARS-CoV-2 Spike protein induces IL-1β secretion in macrophages from COVID-19 patients.Figure 1SDS PAGE of the recombinant SARS-CoV-2 spike protein (S-protein). SARS-CoV-2 spike binding assay of IgGs isolated from COVID-19 patients (n = 3; red circles) or SARS-CoV-2 naïve individuals (SC-naïve) (n = 2; blue circles). Experimental scheme: After PBMCs isolation, CD14+ cells were enriched by positive selection. Subsequently, 5x104 CD14+ cells/well were seeded and incubated in the presence of M-CSF for 5 days. Differentiated macrophages were stimulated with/without recombinant SARS-CoV-2 spike protein or lipopolysaccharide (LPS) for 4h. To activate IL-1β secretion nigericin or ATP was added for 2h. Finally, IL-1β secretion was quantified by ELISA. Quantification of IL-1β concentration (pg/ml) in the supernatant of primary macrophage cultures from COVID-19 patients (n = 44; red bars) or SC-naïve individuals (n = 24; blue bars) stimulated with LPS or S-protein (0.1 µg/ml). To activate IL-1β secretion nigericin was added for 2h. For statistical analysis, two-way ANOVA with tukey post hoc test was used. TNF-α (pg/ml) secreted from macrophages derived from SC-naïve and COVID-19 patients as described above. Cells were stimulated with LPS (SC-naïve/COVID-19 n = 4), LPS with nigericin (SC-naïve n = 4; COVID-19 n = 6), S-protein (SC-naïve n = 5; COVID-19 n = 4), S-protein with nigericin (SC-naïve n = 3; COVID-19 n = 4) or left unstimulated (SC-naïve n = 3; COVID-19 n = 4). Significances shown in the figure are always in comparison to the unstimulated control. \n",
      ": S-protein stimulated macrophages from convalescent individuals secrete IL-1β and show distinct gene expression signatures.Figure 3Subgroup analysis of IL-1β secretion of macrophages after stimulation with S-protein and nigericin from COVID-19 patients with severe (red/black; n=23) or mild (white/red; n=21) disease. Student's t-test with Welsh corrections was used to calculate statistical differences. Graph shows mean ± SEM. CD14+ cells of two convalescent individuals were isolated sequentially at d18, d26 and d56 after confirmation of SARS CoV2 infection by PCR. Following M-CSF differentiation, macrophages were stimulated with LPS (red triangle) or S-Protein (red circles) and IL-1β concentration (pg/ml) was quantified upon incubation with nigericin. Blue dashed line indicates mean IL-1β level of SC-naïve individuals stimulated with S-protein and nigericin. Quantification of IL-1β concentration (pg/ml) in the supernatant of primary macrophage cultures from convalescent COVID-19 patients (SC-conv) (n = 4; red bars) or SARS-CoV-2 naïve individuals (SC-naïve) (n = 4; blue bars) stimulated with S-protein (1 µg/ml) and nigericin. Students t-test with Welsh corrections was used to calculate statistical differences as indicated. Graph shows mean ± SEM. *p<0.05. Venn diagram showing number of differentially expressed genes (DEGs) in healthy/naïve (SC-naïve; blue) and convalescent COVID-19 patient-derived macrophages (SC-conv; red) upon stimulation with LPS (left) or S-Protein (right). Circle sizes of the venn diagram correspond to the number of genes. KEGG-gene enrichment analysis, based on RNA-Seq data, showing activated signaling pathways in S-protein stimulated macrophages compared to unstimulated in convalescent COVID-19 patient-derived macrophages (SC-conv). Padj values are indicated (color code) and size of the circles represent number of DEGs (count). P-values are calculated using Wald test and p-adjusted values using the FDR/Benjamini-Hochberg approach. Volcano plot showing DEGs in S-protein stimulated macrophages from convalescent COVID-19 patients (SC-conv) compared to unstimulated cells. Negative log10 adjusted p-values are plotted against the log2 fold-change. Select genes are labeled. P-values are calculated using Wald test and p-adjusted values using the FDR/Benjamini-Hochberg approach. Heat-map indicating DEG patterns comparing LPS or S-protein stimulation of SC-conv macrophages. Z-score is indicated in a color score. \n",
      ": SARS-CoV-2 Spike protein primes NLRP3 inflammasome and induces pyroptosis.Figure 2Chemical inhibition of IL-1β (pg/ml) secretion after pre-incubation of macrophages from COVID-19 patients (red) for 2h with DMSO (n = 21), MMC950 (n = 21) or VX-765 (n = 8), followed by 4h incubation with S-protein and additional 2h with nigericin. For statistical analysis, one-way ANOVA with tukey post hoc test was used. Western blot analysis of total cell lysates of macrophages from a SARS-CoV-2 naïve individual (SC-naïve) (top) and COVID-19 infected individual (bottom). Macrophages were stimulated with LPS or S-protein (4h) followed by 2h of nigericin treatment. Control cells were left unstimulated. Antibodies against NLRP3 (110kDa) and β-actin (loading control) were used for detection. Representative example of three individual experiments. Representative confocal microscope pictures of macrophages from SC-naïve (top) and COVID-19 patients (bottom). Nuclei are stained with DAPI (blue) and ASC speck are labeled by immunofluorescence staining (green). Macrophages were stimulated with LPS (left) and S-Protein (right) for 4h followed by stimulation with nigericin in both cases. Pictures were taken with a 60x objective using the same microscope settings. Quantification of ASC speck from SC-naïve (n=3, blue) and COVID-19 patients (n=8, red) derived macrophages. Percentage of ASC specks was calculated per DAPI stained nuclei. For statistical analysis, two-way ANOVA with tukey post hoc test was used. IL-1β gene expression of macrophages from COVID-19 patients (n = 5; red bars) or SC-naïve individuals (n = 3 (S-protein) n = 4 (control); blue bars) stimulated with/without S-Protein were determined by qRT-PCR. Data are normalized to β-actin. Statistical significance was analyzed using the Kolmogorov-Smirnov test. Western blot analysis of total cell lysates of macrophages from SARS-CoV-2 naïve individuals (SC-naïve) (top) and COVID-19 infected individuals (bottom). Macrophages were stimulated with LPS or S-protein (1h). Control cells were left unstimulated. Antibodies against pro-IL-1β (31kDa) and β-actin (loading control) were used for detection. Representative example of three experiments performed with individual patients and SARS-CoV-2 naïve controls. Western blot analysis comparing macrophage precipitated supernatants from a SC-naïve (top) and a COVID-19 positive patient (bottom). Macrophages were treated with DMSO or MMC950 (2h) and further stimulated with LPS or S-protein (4h) followed by 2h of nigericin treatment. Antibodies binding cleaved IL-1β (17kDa) were used for detection. Representative example of three individual experiments. Western blot analysis of total cell lysates of macrophages from a SARS-CoV-2 naïve individuals (SC-naïve) (top) and COVID-19 infected individual (bottom). Macrophages were stimulated with LPS or S-protein (4h) followed by 2h of nigericin treatment. Control cells were left unstimulated. Antibodies targeting cleaved GSDMD (110kDa) and β-actin (loading control) were used for detection. Representative example of three individual experiments. \n",
      ": TLR-2 is involved in IL-1β secretion of S-protein stimulated macrophages.Figure 4Mean fluorescence intensity (MFI) comparing surface TLR2 expression on healthy/naïve (SC-naïve; blue; n = 7) and COVID-19 patient (SC-conv; red; n = 10) derived macrophages. Students t-test with Welsh corrections was used to calculate statistical differences as indicated. IL-1β concentration (pg/ml) in supernatants of COVID-19 patient derived macrophages pretreated with DMSO (n = 44), KINK-1 (n = 11) and MMG-11 (n = 18) and subsequent stimulation with S-protein and nigericin. For statistical analysis, one-way ANOVA was used. IL-1β concentration (pg/ml) in supernatants of COVID-19 patient derived macrophages pretreated with DMSO (n = 44), KINK-1 (n = 11) and MMG-11 (n = 19) and subsequent stimulation with LPS and nigericin. For statistical analysis, one-way ANOVA was used. IL-1β concentration (pg/ml) in supernatants from macrophages of SC naïve (blue; n = 7) and COVID-19 patients (red; n = 11). Macrophages were pre-treated with DMSO (SC-naïve n = 7; COVID-19 n = 11), KINK-1 (SC-naïve n = 6; COVID-19 n = 7) and MMG-11 (SC-naïve n = 4; COVID-19 n = 4) for 2h and then stimulated with zymosan (4h) and nigericin (2h) as indicated. For statistical analysis, two-way ANOVA with tukey post hoc test was used. IL-1β concentration (pg/ml) in supernatants from COVID-19 patients which were stimulated for 2h with a blocking anti-TLR2 monoclonal antibody in different concentrations as indicated. Subsequently, macrophages were stimulated with LPS (1 ng/ml: n = 6; 10 and 100 ng/ml: n = 7) and S-Protein (1 ng/ml: n = 12; 10 and 100 ng/ml: n = 13) for 4h and both groups for 2h with nigericin. For statistical analysis, one-way ANOVA was used \n",
      ": SchemeTable 1.Figure 7  Figure 7: Scheme Table 1. Summary of selected macrophage RNA-Seq and miRNA data generated in this study. Key pathways and key genes differentially regulated in different groups and conditions. Full datasets have been uploaded as supplementary tables. NOD-like signalling; inflammatory responses  \n",
      ": Epigenetic immune memory in stimulated macropghages.Figure 6Volcano plot showing differentially expressed miRNAs from unstimulated COVID-19 patient-derived macrophages (SC-conv) compared to those from naïve individuals (SC-naïve). Negative log10 adjusted p-values are plotted against the log2 fold-change. Genes of interest are indicated. For miRNA analyses data with a padj value of smaller 0.05 and log2-fold change of ≤-0.85 (blue) and ≥ 0.85 (red) were included. Average profile plot showing distribution of histone modifications identified in CUT&RUN experiments displayed as normalized read counts per million around known transcription start site (TSS) of known genes. Color code for H3K4me3 (SARS-CoV-2 naïve/green; COVID-19/red) and for H3K27ac (SARS-CoV-2 naïve/yellow; COVID-19/purple). GO biological process analysis performed on differentially enriched loci of H3K27ac peaks (enhancer) showing associated pathways from SC-conv monocytes compared to SC-naïve monocytes. KEGG enrichment analysis performed on differentially enriched loci of H3K27ac peaks (enhancer) showing involved pathways from SC-conv monocytes compared to SC-naïve monocytes. Adjusted p-values are indicated (color code) and size of the circles corresponds to the number of detected genes associated with each pathway (count). CNET-plot (derived from analysis shown in D) illustrating genes involved in tuberculosis and phagosome KEGG pathways. Representative gene loci (TLR2 (top) and S100A7A/8/9/12 (bottom) H3K27ac peaks) comparing SC-naïve (blue) and SC-conv (red) monocytes. Red peaks reach higher values than blue peaks (representative examples of four samples for each condition). Quantification of IL-1β concentrations (pg/ml) in the supernatant of primary macrophage cultures from patients with active/untreated tuberculosis (n = 6; red/black bars) or healthy/SARS-CoV-2 naïve individuals (n = 3; blue bars) or the same patients after 6 months of anti-tuberculous treatment; n = 6; white/red bars) stimulated with LPS or S-protein (0.1 µg/ml) for 4h and subsequent incubation with nigericin for 2h. For statistical analysis, two-way ANOVA with tukey post hoc test was used. \n",
      ": Transcriptomic changes are associate with innate immune responses and redox-status of stimulated macrophages.Figure 5Heat-map indicating DEG patterns comparing SC-conv and SC-naïve macrophages stimulated with the S-protein. Z-score is indicated in a color score. BP enrichment analysis, based on RNA-Seq data, showing activated signaling pathways upon S-protein stimulation in SC-conv macrophages compared to SC-naïve macrophages. Padj values are indicated (color code) and circle size corresponds to the number of DEGs contributing to the pathways (count). Heat-map indicating DEG patterns comparing SC-conv and SC-naïve macrophages without any stimulation. Z-score is indicated in a color score. Volcano plot showing DEGs in unstimulated COVID-19 patient-derived macrophages (SC-conv) compared to macrophages from SARS-CoV-2 naïve individuals (SC-naïve). Negative log10 padj values are plotted against the log2 fold-change. Selected genes of interest are labeled. IL-1β concentration (pg/ml) in supernatants of SC-naïve individuals (n = 7) which were pre-treated for 24h with FCCP (1µg/ml) and H2O2 (10µg/ml). Macrophages were then incubated with LPS or S-Protein (4h) and nigericin (2h) as indicated in the figure. Control cells were left unstimulated or stimulated with LPS or S-protein without pre-treatment with FCSP or H2O2 (n = 6). For statistical analysis, Brown-Forsythe and Welsh ANOVA test was used. \n",
      "Fig.Figure 1A. Root tip of Arabidopsis with distinct growth zones marked: meristematic (MZ, green), transition (TZ, blue), elongation (EZ, pink) and differentiation (DZ, yellow). Double arrow indicates the first expanded cortex cell considered as a start point of the TZ, which encompasses epidermal cells smaller than 45 µm. Epidermal cells reaching a length of more than 45 µm prior termination of the elongation are in the EZ. Scale bar 25 µm. B. CMTs visualized by the MAP4-GFP reporter (left) and scheme of the CMT orientations (right) in epidermal cells of distinct growth zones (TZ, EZ and DZ). Individual cells monitored at two time points (0 and 60 min). Dark blue arrows mark the cell expansion direction. Scale bar 10 µm. C. Histograms of the CMT orientation distributions. Orientation was measured as an angle between CMTs and the longitudinal root growth axis, with 0º, 45º and 90º corresponding to longitudinal, oblique and transversal orientations, respectively. The proportion of CMTs in a certain orientation is calculated per cell (n = 15-20 cells per root growth zone with 6-9 roots in 4 independent replicates). D. Analysis of the CMT plus-end growth with the EB1b-GFP reporter. CMT plus-end growth rates (μm/min) measured by tracking the EB1b-GFP marker for 20-30 min in epidermal root cells of three growth zones. In the boxplots, the center lines show medians; box limits indicate the 25th and 75th percentiles as determined by the Graphpad software; whiskers span minimum to maximum values, individual data points are represented by dots. ** P < 0.01, **** P < 0.0001by the Student's t-test. The number of CMT plus-end events (EB1b-GFP) tracked per cell was 20-100/min, of which the growth rate average is represented as a single dot; 3-5 cells per root growth zone, 3-5 roots analyzed per biological replicate in 3 independent experiments. On the right, single trajectories of EB1b-GFP signal tracked over 60 sec in root epidermal cells at different growth zones. Scale bar 0.5 µm. E. Z-stack maximum image projection of the EB1b-GFP plus-end reporter tracked over 30 sec, in epidermal cells at different growth zones. Scale bars 10 µm. F. CMTs visualized by MAP4-GFP in root epidermal cells at the TZ, the EZ and the DZ prior (0) and 60 min after treatment with 1 µm oryzalin. Scale bar 10 µm. G. Quantification of the MAP4-GFP CMTs reporter signal in wild-type root epidermal cells at the TZ, the EZ and the DZ zone treated with mock (DMSO, white box), oryzalin (1 µM, light grey box) and cytokinin (CK; 10 µM 6-benzylaminopurine, BAP) and oryzalin (1 µM) (dark grey box). For double treatments, roots pretreated for 60 min with CK prior to transfer to medium supplemented with both compounds. Boxplots represent ratio between mean fluorescence intensity (arbitrary units) measured in epidermal cells at 60 and 0 min. \n",
      "Fig.Figure 2A. Arabidopsis root expressing the dual reporter TCSn::dndT-tNOS DR5v2:3nGFP that is sensitive to cytokinin (CK) and auxin. CK (red), auxin (green) and overlay of both signals detected in nuclei of cells at the root tip (top, scale bar 50 µm). Green, pink, and blue arrowheads point at epidermal cells located in the meristematic (MZ), transition (TZ) and elongation (EZ) zones, respectively. Magnification of the TZ shown (bottom, scale bar 10 µm). B. Relative fluorescence intensity of TCSn::ntdT:tNOS signal (red) and DR5v2:3nGFP signal (green) measured in epidermal cells along the longitudinal root growth axis. Cell number 1 corresponds to a meristematic cell placed at position -4 before the beginning of TZ (as marked by green arrow at 2A). Mean ± s.d., * P < 0.05, ** P < 0.01, *** P < 0.001, **** P < 0.0001 by Student's t-test, n= 18 roots. C-F. CMTs visualized with MAP4-GFP in root epidermal cells of the TZ and the EZ at time point 0 and 60 min after CK (C), CK and oryzalin (D), auxin (E) and CK and auxin (F) treatments, and in DZ cells at time point 0 and 60 min after auxin treatment (E). As CK and auxin sources 10 µM BAP and 0.1 µM NAA were used, respectively, and 1 µM oryzalin. For the double (CK/Oryzalin and CK/Auxin) treatments, roots were pretreated with CK for 60 min prior to transfer to medium supplemented with both compounds. Scale bar 10 µm. G. Histograms of CMT orientation distributions in epidermal cells of the TZ and the EZ at time point 0 and 60 min after hormonal treatments as described above (C, E and F). n = 15-20 cells per root growth zone with 5-8 roots per condition in 4 independent replicates. H. Analysis of the CMT plus-end growth with the EB1b-GFP reporter. Z-stack maximum image projections of EB1b-GFP tracked over 30 sec in epidermal cells without (mock, DMSO) and with CK, auxin or CK and auxin treatment (60 min). Scale bars 10 µm.\n",
      "Fig.Figure 4A. Relative root growth of wild-type (Col-0) (grey lines) and 35S::ARR1∆DDK-GR (purple lines) seedlings grown on mock (DMSO; continuous line) or with dexamethasone (DEX 10 µM; dashed line) supplemented medium. Five days old seedlings (Day 0) were monitored for 3 days. Mean ± s.d.; ** P < 0.01, **** P < 0.0001 by Student's t-test, referred to wild-type - mock. n = 12-14 roots. B. Immunostaining of α-tubulin in epidermal cells of the transition (TZ) and the elongation (EZ) zone of 35S::ARR1∆DDK-GR roots after non-, 3h or 16h of treatment with dexamethasone (DEX 10 µM). Scale bar 10 µm. C. Histograms present the CMT orientation distributions (%) in epidermal cells of the TZ and the EZ treated as indicated in (B). Mean ± s.d., (* P < 0.05, ** P < 0.01, and **** P < 0.0001 by Student's t-test, referred to same root zone without DEX induction). n = 16-24 cells per growth zone in 8-9 roots per condition were analyzed. D, E. Analysis of root sensitivity to oryzalin in wild-type (Col-0) and 35S::ARR1∆DDK-GR. Seedlings were grown for 5 days on mock (Murashige and Skoog) medium and then transferred to medium supplemented with 1 µM oryzalin, CK with oryzalin (10 µM BAP and 1 µM oryzalin), with or without DEX 10 µM, for 3 days. For the double CK and oryzalin treatment, seedlings were pretreated with 10 µM BAP for 60 min prior to transfer to medium supplemented with both compounds. Representative images; white arrowheads indicate root length at day of transfer. Scale bar 1 mm (D). Quantifications calculated as percentage (%) of root tips exhibiting swelling (dark grey bars), and weak and strong resistance to oryzalin (grey and white bars, respectively). On the right representative images of root phenotype categories. n=10-25 roots per treatment were evaluated. (E). F. Relative root growth of wild-type (Col-0) (grey lines) and 35S::ARR1∆DDK-GR (purple lines) seedlings grown on oryzalin 1µM (continuous line) or on oryzalin 1µM and DEX 10 µM (dashed line) supplemented media monitored for 3 days. Day 0, day of transfer. Mean ± s.d.; n = 12-14 roots. \n",
      "Fig.Figure 3A-D. CMTs visualized with MAP4-GFP in root epidermal cells of the cre1-12 mutant at the transition (TZ) and the elongation (EZ) zones at time point 0 and 60 min after mock (DMSO) (A), cytokinin (CK) (B), auxin (C) and CK plus auxin (D) treatments. As CK and auxin sources 10 µM BAP and 0.1 µM NAA were used, respectively. For the double CK plus auxin treatment roots were pretreated for 60 min with CK, whereafter they were transferred to medium supplemented with both compounds. Scale bar 10 µm. E. Histograms of CMT orientation distributions at time points 0 and 60 min in epidermal cells of cre1-12 at the TZ and the EZ treated as for (A-D). n = 14-29 cells per root growth zone in 4-8 roots per condition in 3 independent replicates. F, G. Analysis of the CMT plus-end growth with the EB1b-GFP reporter in cre1-12. F. Z-stack maximum image projection of EB1b-GFP tracked for 30 seconds (on the left) and single trajectories of EB1b-GFP signal tracked over 60 sec (on the right) in root epidermal cells of the EZ in the wild type and cre1-12 treated with CK for 60 min. Scale bars 10 µm and 0.5 µm, respectively. G. CMT plus-end growth rates (µm/min) quantified from the EB1b-GFP trajectories tracked during 5 min in epidermal cells of the EZ treated with mock (DMSO) or CK. In the boxplots, the center lines show the medians; box limits indicate the 25th and 75th percentiles as determined by the Graphpad software; whiskers span minimum to maximum values, individual data points are represented by dots. (ns = non-significant, *** P < 0.001 by Student's t-test, n= 7-9 cells with 7 roots per condition in 3 independent experiments). Concentrations and treatment conditions (F, G) were as for (A, B). H, I. CMTs visualized with MAP4-GFP in root epidermal cells of the cre1-12 mutant. Cells at the TZ, the EZ and the DZ were monitored at time point 0 and 60 min after treatment with oryzalin (1 µM) (H), and CK plus oryzalin (I). As CK source 10 µM BAP was used and 1 µM oryzalin. For double (CK and oryzalin) treatment roots were pretreated for 60 min with CK and then transferred to medium supplemented with both compounds. Scale bar 10 µm.\n",
      "Fig.Figure 5A. Time lapse imaging of Arabidopsis root expressing the plasma membrane marker (PM) pUB10::EYFP-NPSN12 (wave line W131Y)(Geldner et al, 2009). Root growth recorded for 2 min and overlay of root tip images at time point 0 (magenta) and 2 min (cyan) used for the calculation of the root growth rate (RGR) and the cell elongation rate (CER). Scale bar 25 µm.C-E. Analysis of CK and auxin effects on CER of epidermal cells. Epidermal cells of the TZ and the EZ in the wild type and cre1-12 with PMs visualized with the YFP-NPSN12 marker. Overlay of images of epidermal cells at time 0 (magenta) and 2 min (cyan) is presented. Scale bar 10 µm (C). CER (µm/min) of root epidermal cells of the TZ (D) and the EZ (E) of the wild type (grey bars) and cre1-12 (blue bars) mutant. Hormone concentrations and conditions of treatment (D, E) were as described for (C). Mean ± s.d., (* P < 0.05, ** P < 0.01, *** P < 0.001 and **** P < 0.0001 by Student's t-test, n=15-20 cells per root growth zone in 3-6 roots per condition, in 3 independent). \n",
      "Genotoxic or metabolic stress‐induced UVRAG expression in tumour cells.Figure 1 (A,B) Expression of UVRAG under cell stress. HL60 and HCT116 cells were treated with doxorubicin (1 μg/ml), cisplatin (10 μm) and UV irradiation (5 min after 50 mJ/cm2) for the indicated time, and then apoptosis, autophagy and fluorescence intensity of UVRAG was assayed. Scale bar, 10 μm. (C) In parallel, the total protein extracts were used for western blot analysis. (D,E) Pharmacological inhibition (for example, 3‐methyladenine (3‐MA), 10 mM) or genetic deletion (for example, ATG5−/−) of the autophagy pathway decreases the protein and mRNA expression of UVRAG and increases apoptosis at 12 h in HL60 cells treated with doxorubicin (Doxo) and UV irradiation (n=3, *P0.05). AU, arbitrary units; C‐PARP, cleaved poly‐ADP ribose polymerase; LC3, light chain 3; mRNA, messenger RNA; PI, propidium iodide; UV, ultraviolet; UVRAG, ultraviolet irradiation resistance‐associated gene. \n",
      "Fig.Figure 6 A. MT plus-end growth rates (µm/min) in leukocytes expressing the EB3-mCherry plus-end marker before (dark grey) and after (light grey) treatment with mock (DMSO), CK (10 µM BAP or 10 µM trans-zeatin) and auxin (0.1 µM NAA). MT plus-end growth rates were monitored for 10 min before and for 10 min after each treatment. The number of MT plus-end events (EB3-mCherry) tracked per cell was 100-200/min. Mean ± s.d., (*** P < 0.001 by Student's t-test, n = 1-3 cells per condition in 4 independent replicate).C-E. Immunostaining of α-tubulin (green) in leukocytes mounted in media containing DAPI (blue, nuclear marker) and quantification of immunolabeled MTs. Cells were not or treated with 300 nM nocodazole (N) for 10 min (C). Leukocytes pretreated for 30 min with mock (DMSO), CK (10 µM BAP or 10 µM trans-zeatin), adenine (10 µM) or auxin (10 µM NAA) prior to a 10-min treatment with 300 nM nocodazol (D). Scale bar 50 µm (C, D). E. Relative fluorescence intensity of MTs in leukocytes treated with nocodazole, or nocodazole and hormones as described for (D) compared to untreated (black bar) cells. Mean ± s.d., (P < 0.05 by two-way Anova, n = 18-33 cells per condition in 3 independent replicates). \n",
      "Inhibition of UVRAG expression increases tumour‐cell sensitivity to chemotherapy and radiation therapy in vivo and in vitro.Figure 2(A-C) After transfection with UVRAG shRNA or control shRNA for 48 h, HL60 (A), HCT116 (B) and HeLa (C) cells were treated with doxorubicin (Doxo, 1 μg/ml), cisplatin (Cis, 10 μm) and UV irradiation (5 min after 50 mJ/cm2) for 12 h or starvation (HBSS) for 3 h, and then apoptosis and autophagy were assayed (n=3, *P0.05). (D) Analysis of LC3 processing by autophagy in the presence or absence of lysosomal protease inhibitors pepstatin A (10 μg/ml) and E64D (10 μg/ml) after doxorubicin (1 μg/ml) treatment for 12 h. *P0.05 compared with control shRNA group. (E) After transfection with UVRAG shRNA or control shRNA for 48 h, ATG5+/+ and ATG5−/− murine embryonic fibroblasts (MEFs) were treated with doxorubicin (1 μg/ml), with or without Z‐VAD‐FMK (20 μM) for 12 h, and then apoptosis was assayed (n=3, *P0.05). (F) Nude mice were inoculated with 4 × 106 HL60 tumour cells following transfection of control (shControl) or UVRAG (shUVRAG)‐specific shRNA and treated with doxorubicin (10 mg/kg) beginning at day 7. Tumours were measured twice weekly, and volumes were calculated for 21 days (n=10, *P0.05; shControl+Doxo compared with shUVRAG+Doxo). (G) On day 21 in the experiments described in (F), apoptosis and UVRAG expression in tumour samples were assayed by TUNEL or western blotting, respectively (n=10, *P0.05). BAX, BCL2‐associated X protein; HBSS, Hank's Buffered Salt Solution; LC3, light chain 3; shRNA, short‐hairpin RNA; TUNEL, terminal deoxytransferase uridine triphosphate nick end labelling; UV, ultraviolet; UVRAG, ultraviolet irradiation resistance associated gene. \n",
      "UVRAG inhibits BAX‐induced apoptosis.Figure 3(A) Control shRNA or UVRAG shRNA cells were transfected with BAX plasmid, and apoptosis was assayed (n=3, *P0.05). (B) In parallel, the mitochondrial membrane potential, cytochrome c release from mitochondria, and caspase‐3 and ‐9 activation in HL60 cells were analysed (n=3, *P0.05). (C) After transfection with UVRAG vector or control vector for 48 h, cells were transected or treated with BAX (1 μg), BAD (1 μg), BID (1 μg), FAS antibody (10 μg/ml) or TRAIL (100 ng/ml) for 24 h, then cell apoptosis was analysed (n=3, *P0.05). (D) Knockdown of UVRAG in BAX−/− HCT116 cells restored the sensitivity to UV treatment‐induced apoptosis (n=3, *P0.05). AU, arbitrary units; BAX, BCL2‐associated X protein; shRNA, short‐hairpin RNA; UV, ultraviolet; UVRAG, ultraviolet irradiation resistance associated gene. \n",
      "UVRAG interacts with Bax.Figure 4(A,B) HL60 and HCT116 cells were treated with UV irradiation for 24 h and then assayed for protein expression levels as indicated by co‐immunoprecipitation (RIPA or CHAPS buffer) or western blotting. (C) Proteins in the pellet after GST pull‐down assays in HCT116 cell lysate were analysed by SDS-PAGE and immunodetected with UVRAG or GST antibodies. (D) UVRAG C2 domain deletion terminates BAX binding. HCT116 cells were transfected with Flag‐UVRAG or its mutants (ΔC2, ΔCCD), and then assayed for protein expression levels as indicated by IP or western blot. (E) Apoptosis was assayed in indicated HCT116 cells after treatment with doxorubicin (Doxo, 1 μg/ml), UV irradiation (5 min after 50 mJ/cm2) or transfected with BAX (1 μg) for 24 h. BAX, BCL2‐associated X protein; CCD, coiled‐coil domain; CHAPS, 3‐[(3‐cholamidopropyl)‐dimethylammonio]‐1‐propane sulphonate; GST, glutathione‐S‐transferase; IB, immunoblotting; IgG, immunoglobulin G; IP, immunoprecipitation; RIPA, radioimmunoprecipitation assay; SDS-PAGE, SDS-polyacrylamide gel electrophoresis; UV, ultraviolet; UVRAG, ultraviolet irradiation resistance associated gene. \n",
      "UVRAG inhibits mitochondrial translocation of BAX.Figure 5 UVRAG inhibits mitochondrial translocation of BAX. After transfection with indicated shRNA or cDNA for 48 h, HL60 cells were treated with doxorubicin (Doxo) and UV irradiation for 12 h, and then BAX in cytosol (Cyt) and mitochondria (Mit) was assayed by western blotting (A-C) and activation of BAX (D-F) was assayed by western blotting after IP using BAX monoclonal antibody (clone 6A7) *P0.05. AU, arbitrary units; BAX, BCL2‐associated X protein; CHAPS, 3‐[(3‐cholamidopropyl)‐dimethylammonio]‐1‐propane sulphonate buffer; Cyt c, cytochrome c; IP, immunoprecipitation; shRNA, short‐hairpin RNA; UV, ultraviolet; UVRAG, ultraviolet irradiation resistance associated gene. \n",
      ".Figure 2M1-linked ubiquitin co-localizes with Htt-polyQ aggregates in cultured SH-SY5Y cells (A) Autofluorescent lipofuscin appears in the green channel in human brain. Scale bar, 20 µm (A) M1-linked ubiquitin co-localizes with Htt-polyQ aggregates in cultured R6/2 mouse cortex and striatum (B) Scale bar 10 µm M1-linked ubiquitin co-localizes with Htt-polyQ aggregates in cultured human HD frontal cortex (C). Autofluorescent lipofuscin appears in the green channel in human brain. Scale bar, 10 µm (D) Htt-Q25- or Htt-Q97-expressing HEK293T were analyzed by filter retardation assays. As indicated, the M1 ubiquitin antibody was preincubated with recombinant tetra-ubiquitin linked via M1 (4 x M1 ubiquitin) or K63 (4 x K63 ubiquitin). \n",
      ".Figure 1(A) HOIP, HOIL-1L, and SHARPIN co-localize with Htt-Q97 aggregates. SH-SY5Y cells co-expressing Htt-Q25-GFP or Htt-Q97-GFP (green) and HOIP, HOIL-1L, SHARPIN, or HHARI (red); DAPI (blue). Scale bar, 10 µm. HOIP co-localizes with Htt inclusion bodies in human HD frontal cortex. Perinuclear Htt aggregates visualized by immunofluorescence (B) in human HD frontal cortex. Scale bar, 10 µm (B) HOIP co-localizes with Htt inclusion bodies in human HD frontal cortex. Perinuclear Htt aggregates visualized by DAB immunohistochemistry (C) in human HD frontal cortex. Scale bar, 15 µm (C). \n",
      ".Figure 3(A) Scheme of WT HOIP and HOIP mutants. (B) Quantification of the co-localization of WT HOIP or HOIP mutants and Htt-Q97 aggregates. Data are displayed as mean ± SD and were analyzed by Mann-Whitney U-test, n = 3. Data information: *p ≤ 0.05, ***p ≤ 0.001. (C) Htt-Q97-GFP (green) was co-expressed with HA-tagged WT or HOIP mutants (red) in SH-SY5Y cells and analyzed by immunocytochemistry using an antibody against HA. DAPI (blue). Scale bar, 10 µm. (D) Recruitment of HOIP to Htt-Q97 is decreased in p97/VCP-deficient cells. p97/VCP-deficient SH-SY5Y cells co-expressing Htt-Q97 and HA-HOIP were reconstituted with either WT VCP or VCP∆PIM and analyzed by immunocytochemistry. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9. Data information: *p ≤ 0.05, ***p ≤ 0.001. (E) Recruitment of p97/VCP to Htt-Q97 is decreased in HOIP KO cells. WT HAP1 cells (control) or HOIP KO HAP1 cells reconstituted with either WT HOIP or HOIP∆PUB were analyzed by immunocytochemistry. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Bonferroni Multiple Comparison Test, n = 5. Data information: *p ≤ 0.05, ***p ≤ 0.001. (F) The interaction of soluble Htt-Q60 with p97/VCP is dependent on HOIP. WT HAP1 cells or HOIP KO HAP1 cells were transfected with Htt-Q60-HA. 48 h after transfection cells were lysed with 1% Triton X-100, followed by an immunoprecipitation of Htt-Q60 via the HA tag. Immunopurified proteins were detected by Western blotting using an anti-VCP and anti-HOIP antibody. \n",
      ".Figure 5(A) The interaction between Htt-Q97 and Sp1 is decreased by linear ubiquitination. HEK293T cells were co-transfected with Htt-Q97-HA and the siRNAs indicated. Htt-Q97-GFP was transfected as a control for specific immunoprecipitation of HA-Htt-Q97. Nuclear (left panel) and cytosolic fractions (middle panel) were subjected to an anti-HA immunoprecipitation followed by Western blotting using an anti-Sp1 antibody. Right panel: Input controls (whole cell lysates). (B) Quantification of the interaction between Htt-Q97 and Sp1 in nuclear fractions. Shown is the relative amount of Sp1 co-purifying with Htt-Q97. Signal intensities were normalized to nuclear Sp1. Data are displayed as mean ± SD and were analyzed by 2 independent Mann-Whitney U-tests, n = 4. *p ≤ 0.05, ***p ≤ 0.001. (C, D) Impairment of Sp1-mediated transcription by Htt-Q97 is modified by HOIP. (C) Shown is the relative luciferase activity in HEK293T cells expressing Htt-Q97-HA, a Sp1 luciferase reporter construct and HOIP cDNA. Data are displayed as mean ± SD and were analyzed by Kruskal-Wallis test followed by Dunn's Multiple Comparison Test, n = 9-11. (D) Shown is the relative luciferase activity in HEK293T cells expressing Htt-Q97-HA, a Sp1 luciferase reporter construct and HOIP siRNA. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9. *p ≤ 0.05, ***p ≤ 0.001. (E) Sp1 silencing decreases HOIP and SHARPIN protein expression. SH-SY5Y cells were transfected with control or Sp1-specific siRNAs and analyzed for protein expression 2 days after transfection by immunoblotting (left panel). Quantification of normalized Sp1, HOIP and SHARPIN protein expression levels (right panel). Data are displayed as mean ± SEM with n=5 from unpaired two-tailed Student's t-tests. *p ≤ 0.05, ***p ≤ 0.001. (F) HOIP-, HOIL-1L- and SHARPIN-specific mRNA is decreased in R6/2 striatum. Quantification of mRNA from the striatum of 9-12 weeks old R6/2 mice and non-transgenic littermates. Data are displayed as mean ± SD with n = 6-9 biological replicates from unpaired two-tailed Student's t-tests. *p ≤ 0.05, ***p ≤ 0.001. (G) HOIP protein levels are decreased in human HD brain. Total protein lysates from human brain (anterior cingulate cortex) analyzed by immunoblotting using antibodies against HOIP and βIII-tubulin. HOIP-positive signals were normalized to βIII-tubulin. To verify the specificity of the HOIP antibody, WT and HOIP KO HAP1 cells were analyzed in parallel on a different gel. Data are displayed as mean ± SEM with n=5 for control and n=7 for HD patients from unpaired two-tailed Student's t-tests. *p ≤ 0.05, ***p ≤ 0.001. \n",
      ".Figure 4(A) Increased expression of HOIP decreases nuclear phospho-c-Jun in primary neurons. Primary mouse hippocampal neurons expressing Htt-Q25 or Htt-Q97 +/- HOIP were analyzed by immunocytochemistry using an antibody against phospho-c-Jun. Data are displayed as mean ± SD and were analyzed by 2 independent Mann-Whitney U-tests, n = 3. *p ≤ 0.05, ***p ≤ 0.001. (B) Htt-Q97-induced toxicity is increased in cells silenced for HOIP. SH-SY5Y cells transiently transfected with Htt-Q25 or Htt-Q97 and either control siRNA (CO) or HOIP siRNA were analyzed by immunocytochemistry using an antibody against phospho-c-Jun. For rescue experiments, HOIP was co-expressed 24 h after silencing. Quantification of Htt-expressing cells with nuclear phospho-c-Jun is based on Student's t test from 3 independent experiments. Data are displayed as mean ± SD unless otherwise stated. *p ≤ 0.05, ***p ≤ 0.001. OTULIN silencing decreases the toxicity of Htt-Q97. SH-SY5Y cells transiently transfected with either control siRNA (CO) or OTULIN siRNA and Htt-Q25 or Htt-Q97 were analyzed by immunocytochemistry using antibodies against phospho-c-Jun (C) Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9-11. *p ≤ 0.05, ***p ≤ 0.001. OTULIN silencing decreases the toxicity of Htt-Q97. SH-SY5Y cells transiently transfected with either control siRNA (CO) or OTULIN siRNA and Htt-Q25 or Htt-Q97 were analyzed by immunocytochemistry using antibodies against cleaved caspase-3 (D). Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9-11. *p ≤ 0.05, ***p ≤ 0.001. (E) The protective effect of HOIP is independent of NF-κB signaling. SH-SY5Y cells transiently transfected with Htt-Q97-GFP and either control vector (CO) or HOIP ± the NF-κB super-repressor IκB-2S/A were analyzed by immunocytochemistry using an antibody against phospho-c-Jun. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9. *p ≤ 0.05, ***p ≤ 0.001. (F) SDS-soluble Htt-polyQ species are modified by linear ubiquitin chains. HEK293T cells were co-transfected with Htt-Q60-GFP (as a control for specific immunoprecipitation of HA-tagged Htt-Q60), Htt-Q60-HA or Htt-Q60-3R (3 lysine residues replaced by arginines) and the plasmids indicated. Cells were lysed under denaturing conditions (1% SDS), followed by immunoprecipitation of Htt via the HA tag (1% Triton X-100, 0.1% SDS). Immunopurified proteins were detected by Western blotting using the M1 ubiquitin-specific antibody 1E3 and anti-HA. (G) Silencing of OTULIN increases linear ubiquitination of Htt-Q97. A linear ubiquitination assay was performed as described in (F) with HEK293T cells transfected with Htt-Q97-HA and control siRNA or OTULIN siRNA. (H) SDS-insoluble Htt-polyQ species are modified by linear ubiquitin chains. HEK293T cells expressing Htt-Q97-HA or Htt-Q97-GFP were lysed under denaturing conditions in 1.5% SDS. After centrifugation, the pellets containing the SDS-insoluble aggregates (SDS-insoluble fraction) were dissolved in formic acid. Formic acid-dissolved aggregates were analyzed by immunoblotting using the M1 ubiquitin-specific 1F11/3F5/Y102L antibody. (I) Endogenous NEMO co-localizes with Htt-Q97 aggregates. NEMO (red) and Htt-Q97-GFP (green) were analyzed by immunocytochemistry in SH-SY5Y cells. DAPI (blue). Scale bar, 10 µm. (J) Endogenous Optineurin co-localizes with Htt-Q97 aggregates. Optineurin (red) and Htt-Q97-GFP (green) were analyzed by immunocytochemistry in SH-SY5Y cells. DAPI (blue). Scale bar, 10 µm. \n",
      ".Figure 6(A) HOIP is recruited to disease-associated protein aggregates formed by Ataxin-3-Q84, SOD1 G85R, TDP-43 Q331K or Optineurin R96L. SH-SY5Y cells co-expressing disease-associated proteins (green) and HOIP (red) were analyzed by immunocytochemistry; DAPI (blue). Scale bar, 20 µ Linear ubiquitin accumulates at disease-associated protein aggregates under endogenous LUBAC expression. M1-linked ubiquitin (red) and disease-associated proteins (green) were analyzed by immunocytochemistry in SH-SY5Y cells using the 1E3 antibody (Millipore) (B) DAPI (blue). Scale bar, 20 µm. Linear ubiquitin accumulates at disease-associated protein aggregates under endogenous LUBAC expression. M1-linked ubiquitin (red) and disease-associated proteins (green) were analyzed by immunocytochemistry in primary striatal neurons using the 1F11/3F5/Y102L antibody (Genentech) (C). DAPI (blue). Scale bar, 20 µm. (D) SDS-soluble SOD G85R and TDP-43 Q331K species are modified by linear ubiquitin chains. HEK293T cells were co-transfected with disease-associated proteins and either the plasmids indicated or with control siRNA or OTULIN siRNA. 48 h after transfection cells were lysed under denaturing conditions (1% SDS), followed by immunoprecipitation of SOD G85R and TDP-43 Q331K via the HA tag (1% Triton X-100, 0.1% SDS). Immunopurified proteins were detected by Western blotting using the M1 ubiquitin-specific antibody 1E3. \n",
      "Features of PC Loss in npc1 Mice.Figure 2(A) Sections from the cerebellar vermis of a 70-d-old npc1−/− mouse were stained with anti-Calbindin (green) to visualize PCs and 7AAD (red) for nuclei. Although there are scant PCs remaining in lobule II, the PCs in lobule X have not decreased. (B) Quantification of progressive anterior-to-posterior PC loss. PC densities from 30-d npc1+/+ and 30-, 50-, and 70-d npc1−/− mice were quantified from five sections from two mice each. Error bars show standard deviation. (C) Bergmann glia visualized in 70-d npc1+/+ and npc1−/− mice using anti-S100β (green) have normal morphology. Note that the glial cell bodies near the PC layer and the radial processes extending into the molecular layer are intact in the npc1−/− mouse despite the loss of PCs. (D) Oligodendrocyte cell bodies stained with anti-CC1 (green) have a similar distribution and morphology in the cerebellarwhite matter of 70-d npc1−/− and wild-type mice. TOTO-3 (red) was used as a nuclear counterstain. Bar = 5 μm. (E) Axonal spheroids in npc1−/− mice. Spheroids (arrows), seen as Calbindin-positive (red) swellings surrounded by myelin (anti-myelin basic protein; green), are numerous in the npc1−/− cerebellum (50-d-old, lobule X shown) but are not seen in the wild-type control. Bar = 5 μm. (F) Sections from wild-type and npc1 mice were stained with filipin to visualize free cholesterol. PCs from npc1 mice show an accumulation of intracellular cholesterol at 30 d regardless of their lobular location. Images are oriented such that the PCs (arrowheads) are in the center of the field, with the molecular layer above and the inner granule cell layer below. \n",
      "Other Histological Characteristics in the npc1 Mutant and Chimeric Mice.Figure 4(A-C) Microglia characteristics. Sections from 50-d-old npc1 mutants were stained with anti-Calbindin (green) to visualize PCs and anti-F4/80 (red) to mark microglia. (A) Very few microglia are present in lobule X, where no PC degeneration has yet occurred. (B) Lobule VIII demonstrates an infiltration of microglia in the granule cell layer and white matter tract. (C) In lobule II, nearly all PCs are gone and numerous microglia are present throughout the cerebellum. The dashed line indicates the edge of the granule cell layer. (D) Microglia infiltration is not sufficient to induce PC degeneration. C1.4 demonstrates numerous microglia marked by anti-F4/80 (red) in all layers of the cerebellum, including immediately adjacent to wild-type (GFP-positive) PCs (arrows). Despite this close approximation, no loss of wild-type PCs was detected by cell counting. The GFP transgene is apparently poorly expressed in microglia, as all microglia appear GFP-negative regardless of their genotype. E) Mutant PCs are lost even when surrounded by wild-type Bergmann glia. A stretch of the PC layer in C4.8 is shown where no npc1−/− (GFP-positive) PCs remain despite the presence of numerous wild-type (GFP-negative) glia (arrowheads) marked by anti-S100β (red). The space occupied by two wild-type PCs is indicated (arrows) with two npc1−/− (GFP-positive) glia directly underneath.(F) Wild-type PCs do not degenerate even when surrounded by mutant Bergmann glia. Three wild-type PCs (GFP-positive; arrows) in C1.4 have not degenerated despite only mutant glia (arrowheads) being in the immediate vicinity. (G) Oligodendrocytes in chimeric mice are a mixture of wild-type and mutant cells. Oligodendrocyte cell bodies are stained with anti-CC1 (red). Wild-type (GFP-positive; arrows) and npc1−/− (GFP-negative; arrowheads) oligodendrocytes are interspersed in C1.4. \n",
      ".Figure 7(A) Linear ubiquitination reduces the fraction of cells with Htt-Q97 aggregates. SH-SY5Y cells were transfected with Htt-Q97 and either control siRNA, HOIP siRNA or OTULIN siRNA. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 9. Expression levels of HOIP and OTULIN were analyzed by Western blotting. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (B) Linear ubiquitination reduces the fraction of cells with Htt-Q97 aggregates. SH-SY5Y cells were transfected with Htt-Q97 and either WT HOIP or the catalytically inactive HOIP mutant C885A. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey s Multiple Comparison Test, n = 5. Expression levels of WT and mutant HOIP and OTULIN were analyzed by Western blotting. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (C) The effect of HOIP on Htt-Q97 aggregation is independent of NF-κB signaling. SH-SY5Y cells transiently transfected with Htt-Q97 and either control vector (CO) or WT HOIP plus the NF-κB super-repressor IκB-2S/A were analyzed by immunocytochemistry. Data are mean ± SD with n=5 from an unpaired two-tailed Student's t-test. Expression levels of WT HOIP and IκB-2S/A were analyzed by Western blotting. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (D) Catalytically active HOIP decreases the number of cells with Htt-Q97 aggregates independently of autophagy. WT and ATG5 KO MEFs were transiently transfected with Htt-Q97-GFP and either control vector (CO), HOIP or HOIP C885A. Cells were treated 24 h after transfection with the proteasomal inhibitor MG132 (1 µM, 16 h). Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Tukey's Multiple Comparison Test, n = 7. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (E) Linear ubiquitination reduces the fraction of cells with TDP-43 Q331K aggregates. SH-SY5Y cells were transfected with TDP-43 Q331K and either control siRNA, HOIP siRNA or OTULIN siRNA. Data are displayed as mean ± SD and were analyzed by Kruskal-Wallis test, followed by Dunn's Multiple Comparison Test, n = 7. Expression levels of HOIP and OTULIN were analyzed by Western blotting. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (F) Catalytically active HOIP decreases the number of cells with TDP-43 aggregates dependent on proteasomal degradation. SH-SY5Y cells were transfected with TDP-43 Q331K and either control vector (CO), HOIP or HOIP C885A. Cells were treated 48 h after transfection with the proteasomal inhibitor MG132 (1 µM) or the p97/VCP inhibitor NSM-873 (1 µM) for 3 h. Data are displayed as mean ± SD and were analyzed by one-way ANOVA followed by Dunnett's Multiple Comparison Test, n = 5. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. (G) Inhibition of the proetasome and p97/VCP do not have additive effects on the degradation of misfolded TDP-43 species. SH-SY5Y cells expressing TDP-43 Q331K were treated 48 h after transfection with MG132 (1 µM) and/or NSM-873 (1 µM) for 3 h. All data are mean ± SD, data were analyzed by one-way ANOVA followed by Bonferroni's Multiple Comparison Test, n = 5. Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001. \n",
      "Cell-Autonomous Degeneration of PCs in Chimeric Mice.Figure 3(A) Genotyping of chimeric mice. Upper gel: PCR to amplify a fragment of the npc1 gene from npc1−/−↔GFP, npc1+/−↔GFP, and npc1+/−mice results in npc1− (1,067 bp) and npc1+ (947 bp) bands. Lower gel: The three genotypes can be distinguished by ApaLI digestion of the PCR products. The npc1− allele gives rise to 937- and 130-bp bands. The wild-type allele varies depending on the source: the BALB/c allele is digested to 477-, 340-, and 130-bp bands while the allele from the GFP mouse is digested to 817- and 130-bp bands. Note that the 477- and 340-bp bands are absent from the homozygous mutant chimera. (B) Quantification of npc1−/−PC density in chimeric mice. For each mouse, the actual density observed (the average of the mean densities counted in each lobule) is compared to the number expected if no PC loss occurs (calculated from the density at 30 d in npc1−/−mice and the chimerism percentage). Five sections separated by at least 100 μm were analyzed for each mouse. For the three mice sacrificed at 70 d, only lobules I-IX were included in the analysis because of the lack of degeneration in lobule X in npc1−/−mice at this age. A value for the number of npc1−/−PCs expected if loss does occur (calculated from the density at 70 d in npc1−/−mice and the chimerism percentage) is also included for these three mice. No rescue of mouse−/−PCs was observed for any of the chimeric mice (p < 0.0001 for each mouse comparing observed versus expected if no loss occurs). (C) Quantification of npc1−/− PC density by lobule in C1.4 at 70 d. Observed and expected densities are shown for npc1−/− PCs in each lobule. No clear rescue of npc1−/− PCs was observed in any lobule (p > 0.05 for all lobules comparing observed versus expected if loss occurs, except II and III, where p-value is between 0.01 and 0.05; p < 0.001 for all lobules comparing observed versus expected if no loss occurs, except lobule X, where p > 0.05). D and E) Images of lobules II and X taken from C1.4 at 70 d. GFP is green and Calbindin staining is red. In lobule II, the number of PCs is clearly reduced and of the five remaining PCs, four are wild-type (GFP-positive). In lobule X (where no degeneration occurs during the normal npc1−/− life span), PC density is normal and the majority of PCs (23 of 32; 72%) are mutant, as expected based on npc1−/− contribution to this mouse (67%). (F) Quantification of wild-type PC density in chimeric mice. Observed and expected densities are shown for wild-type PCs. For mice with more than 15% wild-type contribution, the density of wild-type PCs is similar to the number expected if these cells have not degenerated (p > 0.05 for each mouse comparing observed versus expected if no loss occurs, except C5.2, where p = 0.02, and C5.6, where p = 0.0005). (G) Quantification of wild-type PC density by lobules in C1.4 at 70 d. No loss of wild-type PCs was observed in any of the lobules (p > 0.05 for all lobules comparing observed versus expected if no loss occurs; p < 0.001 for all lobules comparing observed versus expected if loss occurs, except for lobule X, where p > 0.05). \n",
      "Levels of the Autophagosomal Marker LC3-II Are Increased in Degenerating npc1 Cerebellum.Figure 7(A) Immunoblots of 70 μg of protein resolved by 15% SDS-PAGE show increased levels of LC3-II in 50- and 67-d-old npc1 cerebellar extracts. Two mice of each age and genotype were analyzed, and relative band intensities were quantified. p-Values derived from comparing wild-type and npc1−/− cerebella at each age are 0.3, 0.01, and 0.001 for 22-, 50-, and 67-d-old mice, respectively. (B) Immunoblots of CHO cell extracts resolved by 15% SDS-PAGE do not show increased levels of LC3-II in CT60 cells compared to wild-type CHO-KI cells or CT60 cells stably transfected with NPC1-YFP. No difference is noted with 10 μM U18666A, a drug that mimics npc1 loss of function. Rapamycin treatment (1 μM) of CHO-KI cells demonstrates robust activation of autophagy with increased LC3-II levels. \n",
      "Increased Autophagy in npc1 PCs.Figure 6Increased Autophagy in npc1PCsEM of npc1−/− and wild-type PCs from anterior vermis of 48-d-old mice.(A) Low-power field of a typical npc1PC.(B) Magnification of the red box in (A) showing accumulated multivesicular and multilamellar organelles.(C) Magnification of the blue box in (A) showing a large AV (arrowheads) containing ribosomes and membranes.(D) Another AV (arrowhead) with ER cisternae within the lumen.(E) An npc1PC probably in the late stages of degeneration, with many AVs, some with apparently degrading mitochondria (arrowheads).(F) Low-power field of two wild-type PCs.(G and H) Magnification of the red (G) and blue (H) boxes in (F) showing no AVs and very few multivesicular organelles (arrow).(I) Quantification of area occupied by AVs and other multivesicular/multilamellar organelles expressed as a percentage of total cytoplasmic area for seven wild-type and seven npc1−/−PCs; p = 0.0009 for comparing AV areas and p = 0.0003 for comparing other multivesicular/multilamellar areas, indicating both types of organelles occupy significantly more area in npc1−/−PCs.Bar = 5 μm for (A) and (F); for all others, bar = 1 μm. \n",
      "Organismal Phenotypes in Chimeric Mice.Figure 5(A) Chimeric mice with more than 50% wild-type contribution do not exhibit wasting. Weekly weights for each of the mice were recorded from 4 wk until the mice were sacrificed. For wild-type and npc1−/− homozygous mice, the curve shown is the mean weight from three to four mice. A progressive decrease in weight of npc1 mice begins to show around 7 wk. The chimeric mice display a delay in, or even absence of, wasting depending on the amount of wild-type contribution. (B) Prevention of ataxia in chimeric mice. Mice were assessed for ataxia weekly during two 3-min trials on a balance beam (see Materials and Methods). (C) Gait of chimeric mice. Front and back paws of mice were dipped in red or green paint, and mice walked across a box lined with paper. The npc1−/− mouse displays shorter stride length and a smearing of the footprints as the paws are not as well lifted between steps. Chimeric mice with high amounts of wild-type contribution (C4.8; C4.4 not shown) exhibited a gait indistinguishable from wild-type. (D) Quantification of stride length. For wild-type and mutant controls, three mice were assessed. For chimeric mice, gait was measured prior to sacrifice at the ages noted. Error bars indicate the standard deviation. Comparison with the GFP mice demonstrates that npc1+/+, C1.4, C4.4, and C4.8 do not have significantly different stride lengths (p > 0.05) while npc1−/−, C5.2, C5.6, C2.1, and C5.8 have significantly shorter stride lengths (p < 0.001) \n",
      "Colloidal osmotic lysis and membrane permeabilization caused by the GBS pigment/lipid toxin.Figure 1The GBS pigment also known as granadaene is an ornithine rhamnopolyene (Rosa-Fraile et al, 2006). Human red blood cells (RBCs) were incubated with 400 nM pigment or control ΔcylE extract, and kinetics of K+ and Hb release was monitored. Data shown are the average and SEM of six independent experiments. The time to 50% K+ and 50% Hb release with pigment was 4.8 min and 8.4 min, respectively; n = 6, P < 0.0001, extra sum-of-squares F test. Role of osmoprotectants in pigment-treated RBC. HumanRBCs were pre-incubated with GBSpigment for 2 min at RT, centrifuged, and resuspended in the presence and absence of 30 mM osmoprotectant with hydrodynamic radius of 0.40 nm (PEG200), 0.56 nm (PEG400), 0.89 nm (PEG1000), 1.1 nm (PEG1500), or 1.6 nm (PEG3000), respectively. Release of Hb was measured after 1 h of incubation at 37°C. Data shown are the average and SEM of three independent experiments. Characteristics of membrane permeabilization by the GBSpigment in artificial lipid bilayers. Lipid bilayers were generated using diphytanoylphosphatidylcholine (DPhPC) and treated with either 2 μM pigment or an equivalent amount of the control ΔcylE extract. In the pigment-treated sample, channel conductance indicating disruption of the membrane is seen within 45 s. Erratic and non-discrete fluctuations in current are observed, suggesting the formation of multiple, small membrane defects. The bilayer eventually breaks at 120 s. In lipid bilayers treated with the control ΔcylE extract, the mean ionic current trace remains constant at 0 pA, showing no membrane disruption. Data shown are representative of three independent experiments. \n",
      "The GBS pigment toxin/lipid toxin is pro-inflammatory and cytotoxic to primary human macrophages.Figure 2A, B PBMC-derived macrophages were treated with GBS WT, ΔcylE, ΔcovR, or ΔcovRΔcylE at an MOI of 1 and incubated for 4 h. Cytotoxicity was measured by LDH release (A), and IL1β release in supernatants was measured by ELISA (B).Data information: Data shown are the average of four independent experiments performed in triplicate, error bars ± SEM. Significance was determined using Bonferroni's multiple comparison test following ANOVA. (A) n = 4, *P = 0.021 for WT versus ΔcovR; *P = 0.01 for ΔcovR versus ΔcovRΔcylE. (B) n = 4, *P = 0.031 for WT versus ΔcovR; *P = 0.036 for ΔcovR versus ΔcovRΔcylE. (C) n = 4, ****P < 0.0001, **P = 0.002. (D) n = 4, **P = 0.005. C, D PBMC-derived macrophages primed with 100 ng/ml LPS for 3 h were incubated with various concentrations of GBSpigment or control ΔcylE extract for 4 h. Cytotoxicity was measured by alamar blue assay (C), and IL1β release from pigment- or ΔcylE extract-treated cells was measured by Luminex assay (D). (C) n = 4, ****P < 0.0001, **P = 0.002. (D) n = 4, **P = 0.005. \n",
      "The GBS pigment toxin/lipid toxin is pro-inflammatory and cytotoxic to immortalized THP-1 monocyte-derived macrophages.Figure 3A, B WT THP-1macrophages were treated with GBS WT, ΔcylE, ΔcovR, or ΔcovRΔcylE at an MOI of 1 and incubated for 4 h. Cytotoxicity was measured by LDH release (A), and IL1β release in supernatants was measured by Luminex assay (B).Data information: Data from three independent experiments performed in triplicate are shown, error bars ± SEM. Significance was determined using Bonferroni's multiple comparison test following ANOVA. (A) n = 3, **P = 0.0080 for WT versus ΔcovR; **P = 0.0016 for ΔcovR versus ΔcovRΔcylE. (B) n = 3, *P = 0.03 for WT versus ΔcovR; *P = 0.02 for ΔcovR versus ΔcovRΔcylE. (C) n = 3, ****P < 0.0001. (D) n = 3, *P = 0.01. C, D WT THP-1macrophages were incubated with various concentrations of GBSpigment or control ΔcylE extract for 4 h. Cytotoxicity was measured by alamar blue assay (C), and IL1β release from pigment- or ΔcylE extract-treated cells was measured by Luminex assay (D).(C) n = 3, ****P < 0.0001. (D) n = 3, *P = 0.01. \n",
      "The GBS pigment induces NLRP3 inflammasome-dependent cell death in human macrophages.Figure 4A, B THP-1macrophages transfected with empty vector, scrambled control, shASC, or shNLRP3 were treated with GBS WT, ΔcylE, ΔcovR, or ΔcovRΔcylE at an MOI of 1 and incubated for 4 h. Cytotoxicity was measured by LDH release (A), and IL1β release in supernatants was measured by Luminex assay (B). Data from three independent experiments performed in triplicate are shown.Data information: Data were analyzed using Bonferroni's multiple comparison test following ANOVA, error bars ± SEM. (A) n = 3, for WT: **P = 0.001 (vector versus shNLRP3), **P = 0.0016 (vector versus shASC); for ΔcovR: ***P = 0.0008 (vector versus shNLRP3), ***P = 0.0001 (vector versus shASC), *P = 0.047 (scrambled versus shNLRP3), **P = 0.0054 (scrambled versus shASC). Data obtained from THP-1/vector was not significantly different from THP-1/scrambled, P = 0.18. (B) n = 3, *P = 0.025. Data obtained from THP-1/vector were not significantly different from THP-1/scrambled, P = 0.89. C, D The shRNA THP-1macrophages were incubated with various concentrations of GBSpigment or ΔcylE extract for 4 h. Cytotoxicity was measured by alamar blue assay (C), and IL1β release in supernatants was measured by Luminex assay (D). Pigment-mediated cytotoxicity is dependent on the NLRP3 inflammasome components, suggesting that pigment is inducing a programmed cell death. Data shown are the average of at least three independent experiments performed in triplicate.Data information: Data were analyzed using Bonferroni's multiple comparison test following ANOVA, error bars ± SEM. (C) n = 3, for 4 μM pigment: **P = 0.0063 (vector versus shNLRP3), **P = 0.0035 (vector versus shASC), *P = 0.01 (scrambled versus shNLRP3), **P = 0.009 (scrambled versus shASC); for 2 μM pigment: **P = 0.0088 (vector versus shNLRP3), **P = 0.0066 (vector versus shASC), *P = 0.01 (scrambled versus shNLRP3 or shASC); for 1 μM pigment: *P = 0.02 (vector versus shNLRP3), *P = 0.01 (vector versus shASC). Data obtained from THP-1/vector were not significantly different from THP-1/scrambled, P = 0.99. (D) n = 3, ****P < 0.0001, *P = 0.015. \n",
      "Proposed model on GBS pigment-mediated host cell lysis and preterm birth.Figure 8The GBSpigment intercalates into host cell membranes leading to membrane permeability and ion flux. In RBC, this leads to colloidal osmotic lysis. In macrophages, the loss of intracellular potassium serves as a trigger for NLRP3 inflammasome formation. The activated inflammasome then activates caspase 1, which cleaves pro-IL-1β and pro-IL-18 to their active forms. Finally, caspase 1 activation causes cell death by pyroptosis. While NLRP3-dependent pyroptosis is the major form of cell death observed in macrophages exposed to purified pigment and hemolytic/pigmented GBS strains, hyperpigmented GBS strains induced low levels of cell death in NLRP3-deficient and ASC-deficient macrophages that were independent of inflammasome and caspase 3/7. In host cells that are either resistant or recover from pigment-induced membrane damage (e.g., NLRP3-deficient macrophages), we predict that lysosomal degradation of pigment or shedding of vesicles that contain pigment-membrane complexes and the turnover of plasma membrane prevents cell death. The GBSpigment exacerbates fetal injury and preterm birth through the combined action of colloidal osmotic lysis and pyroptosis. \n",
      "The GBS hemolytic pigment/lipid toxin induces caspase 1 activation and pyroptosis.Figure 6THP-1macrophages were treated with GBSpigment, and caspase 1 activation was measured by flow cytometry using a FLICA reagent. Pigment treatment of the scrambled shRNA control cell line induces more caspase 1 activation compared to the shNLRP3cell line, demonstrating that the pigment activates caspase 1 exclusively through the NLRP3 inflammasome. Results are representative of three independent experiments. WT THP-1macrophages were treated with the caspase 1 inhibitor Z-YVAD-FMK, the caspase 3/7 inhibitor Z-DEVD-FMK, or DMSO only prior to treatment with the GBSpigment. YVAD is able to significantly decrease cytotoxicity in cells treated with GBSpigment, demonstrating that caspase 1 is required for GBSpigment-mediated cell death, characteristic of pyroptosis, while DEVD had no effect. Data are average of three independent experiments, error bars ± SEM. Significance was determined using Bonferroni's multiple comparison test following ANOVA (n = 3, ****P < 0.0001, ***P = 0.0002, *P = 0.014). \n",
      "GBS pigment induces membrane permeabilization and K+ efflux independently of the NLRP3 inflammasome.Figure 5A, B THP-1macrophages proficient for NLRP3 (transfected with scrambled control, A) or deficient for NLRP3 (shNLRP3, B) were treated with 1 μM pigment or ΔcylE extract for 20 min, and propidium iodide (PI) was added during the final 10 min. PI uptake was measured by flow cytometry, and data shown are representative of two independent experiments. C, D Intracellular potassium concentration was measured by ICP-AES. THP-1macrophages transfected with the scrambled control (C) or shNLRP3 (D) were treated with GBSpigment (1 μM) or an equivalent amount of the ΔcylE extract. At various time points, cells were lysed and intracellular [K+] was measured relative to untreated cells (see bars and left y-axis), and percent cell death was quantified by alamar blue (see squares, dotted connecting lines and right y-axis). Both NLRP3-proficient and NLRP3-deficient macrophages initially lose intracellular K+ due to GBSpigment (compare t = 0 min to t = 30 min), but the NLRP3-deficient cells (shNLRP3) are able to recover, while the scrambled control do not, demonstrating that initial K+ loss occurs independently of NLRP3. Data are average of three independent experiments performed with independent pigment preparations in triplicate and were analyzed using Dunnett's multiple comparison test following ANOVA; all data were compared to control at t = 0, error bars ± SEM. (C) n = 3, ***P = 0.0002, *P = 0.019, **P = 0.0032 for 120 min, **P = 0.0072 for 180 min. (D) n = 3, **P = 0.0043, *P = 0.031. (E) WT THP-1macrophages were incubated with pigment in media containing either 5 mM or 50 mM potassium chloride, and cytotoxicity was measured by alamar blue assay. The addition of potassium chloride is able to protect the macrophages from cytolysis, demonstrating that K+ efflux is essential for this process. Data shown are three independent experiments performed in triplicate and were analyzed using Bonferroni's multiple comparison test following ANOVA, error bars ± SEM (n = 3, ****P < 0.0001, **P = 0.0036). \n",
      "The GBS pigment causes fetal injury by NLRP3 inflammasome-dependent and NLRP3 inflammasome-independent mechanisms.Figure 7Female pregnant wild-type (CD-1, C57BL6) or NLRP3-deficient mice were injected in utero with 106-7CFU of GBS WT, ΔcovR, or ΔcovRΔcylE and monitored for preterm birth. Surgery and GBS inoculation for each pregnant mouse were performed independently. Data shown are representative of experiments with 6 animals per group for each GBS strain and two animals were used for saline controls.Scheme of pup numbering in utero and injection site between fetuses P1 and P2 is shown. In utero fetal death in wild-type CD-1mice due to infection with GBS WT, hyperhemolytic ΔcovR, and non-hemolytic ΔcovRΔcylE. Fetal death is represented by the number of dead fetuses/total number of fetuses obtained from six pregnant mice per group. 'n' indicates total number of pups (both live and dead); ****P < 0.0001, Fisher's exact test. H&amp;amp;E staining of uterine tissue. Open arrow indicates the presence of few mononuclear cells, whereas filled arrows indicate increased infiltration of inflammatory cells and necrotic debris. Fetal death due to infection with hyperhemolytic GBS ΔcovR and non-hemolytic ΔcovRΔcylE in WT C57BL6 and NLRP3 inflammasome-deficient mice; fetal death is represented by the number of dead fetuses/total number of fetuses obtained from 6 pregnant mice per group. 'n' indicates total number of pups (both live and dead); *P = 0.011, **P = 0.0015, ****P < 0.0001, Fisher's exact test. Fetal death due to ΔcovRΔcylE in WT C57BL6 and NLRP3KO mice was not significant and is indicated as NS; P = 0.31, Fisher's exact test. Female pregnant wild-type (CD-1, C57BL6) or NLRP3-deficient mice were injected in utero with 106-7CFU of GBS WT, ΔcovR, or ΔcovRΔcylE and monitored for preterm birth. Surgery and GBS inoculation for each pregnant mouse were performed independently. Data shown are representative of experiments with 6 animals per group for each GBS strain and two animals were used for saline controls.Bacterial burden in fetal pups and uterine horns from mice infected with the various GBS strains (n = 6/pup; of note, pups that were delivered preterm were excluded from CFU enumeration. Scheme of pup numbering is shown in (A). RUH and LUH indicate right uterine horn and left uterine horn, respectively. CFUs are not significantly different between any of the groups (ANOVA, P = 0.6, error bars ± SEM). Female pregnant wild-type (CD-1, C57BL6) or NLRP3-deficient mice were injected in utero with 106-7CFU of GBS WT, ΔcovR, or ΔcovRΔcylE and monitored for preterm birth. Surgery and GBS inoculation for each pregnant mouse were performed independently. Data shown are representative of experiments with 6 animals per group for each GBS strain and two animals were used for saline controls.IL-1β levels in GBS-infected tissues (placenta and fetus, n = 6/group) was measured by Luminex assay (*P = 0.025, ***P = 0.0002, ****P < 0.0001, Bonferroni's multiple comparison test following ANOVA). IL-1β levels was not significantly different in NLRP3KO mice infected with ΔcovR compared to NLRP3KO mice infected with ΔcovRΔcylEmice and is indicated as NS; P = 0.99. \n",
      ".Figure 4(A) Immunofluorescence microscopy analysis for colocalization of M.tb with Lamp-1 in WT BMMs pre-treated with EVs from macrophages infected with WT, ∆secA2 or secA2-complemented (∆secA2-C) M.tb CDC 1551 strains. The cells were pre-treated with EVs supplemented with recombinant mouse IFN-γ for 5 hr and subsequently infected for 24 hr with GFP-expressing M.tb. (B) quantitative analysis for the colocalization of M.tb with Lamp-1. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM ; * p < 0.05 and ** p < 0.01 by two-tailed Student's t-test. (C) M.tb CFU in WT BMMs pre-pretreated with recombinant mouse IFN-γ and EVs from macrophages infected with WT, ∆secA2 or secA2-complemented (∆secA2-C) M.tb CDC 1551 strains. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. n.s., not significant; * p < 0.05 and ** p < 0.01 by two-tailed Student's t-test. (D) and (E) liposomes containing RNA isolated from EVs released from BMMs infected with WT M.tb were also added to cells 5 hr prior to infection. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM n.s., not significant (F) liposome-encapsulated RNA isolated from EVs released from BMMs infected with WT M.tb was included during EV and IFN-γ pre-treatment. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. n.s., not significant; * p < 0.05 and ** p < 0.01 by two-tailed Student's t-test. \n",
      ".Figure 2(C) Yield of purified EVs from BMMs infected with various M.tb CDC 1551 strains based on Nanosight analysis. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. (B) EVs from cells infected with WT, ∆esxA or esxA-complemented (∆esxA-C) M.tb Erdman strains. (C) Yield of purified EVs from BMMs infected with various M.tb CDC 1551 strains based on Nanosight analysis. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. (D) EVs from cells infected with WT, ∆esxA or esxA-complemented (∆esxA-C) M.tb Erdman strains. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. qRT-PCR analysis for M.tb RNA in EVs from BMMs infected with various M.tb CDC 1551 strains (E) ND, not detected. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). qRT-PCR analysis for M.tb RNA in EVs from BMMs infected with various M.tb Erdman strains (F). ND, not detected. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). qRT-PCR analysis for IFN-β mRNA (G) in WT BMMs treated with EVs for 4 hr and 24 hr respectively. The EVs were isolated from BMMs that were infected with the different M.tb CDC 1551 strains. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). IFN-β protein (H) levels in WT BMMs treated with EVs for 4 hr and 24 hr respectively. The EVs were isolated from BMMs that were infected with the different M.tb CDC 1551 strains. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (I) ELISA analysis for IFN-β secreted by BMMs treated with EVs plus liposomes containing M.tb RNA. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. qRT-PCR analysis for IFN-β mRNA (J) in WT BMMs treated with EVs for 4 and 24 hr respectively. The EVs were isolated from BMMs that were infected with the different M.tb Erdman strains. Data shown are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. IFN-β protein (K) levels in WT BMMs treated with EVs for 4 and 24 hr respectively. The EVs were isolated from BMMs that were infected with the different M.tb Erdman strains. Data shown in K) are the mean ± SD (n = 3 per group) and all data shown are representative of three independent experiments. \n",
      ".Figure 5Immunofluorescence microscopy and quantitative analysis for colocalization of M.tb with autophagosome marker LC3 in control siRNA -treated BMMs that were untreated or pre-treated for 5 hr with recombinant mouse IFN-γ and EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) macrophages followed by a 24 hr infection with GFP-expressing M.tb. Mock, untreated. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy and quantitative analysis for colocalization of M.tb with autophagosome marker LC3 (C and D) in RIG-I siRNA-treated BMMs. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. (E and F) Immunofluorescence microscopy and quantitative analysis for colocalization of M.tb with markers LC3 in WT BMMs that were left untreated or pre-treated for 5 hr with recombinant mouse IFN-γ and EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) macrophages, followed by a 24 hr infection with GFP-expressing M.tb. Mock, untreated. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy and quantitative analysis for colocalization of M.tb with markers LC3 in Mavs -/- BMMs. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy quantitative analysis for colocalization of M.tb with NOX2 (I and J) as described above in WT BMMs. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy quantitative analysis for colocalization of M.tb with p47phox (K and L) as described above in WT BMMs. Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy quantitative analysis for colocalization of M.tb similar to ( I -L), but using Mavs -/- BMMs. NOX2 (M and N) Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy quantitative analysis for colocalization of M.tb similar to ( I -L), but using Mavs -/- BMMs. p47phox (O and P) Quantitative data are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM. n.s., not significant; * p < 0.05 , ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. \n",
      ".Figure 1(A) Quantitative RT-PCR analysis of M.tb RNA in EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) BMMs. ND, not detected. Scale bar, 20 μM. Data shown in (A) are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). Quantitative RT-PCR analysis for IFN-β mRNA (B) levels in wild type BMMs 5 and 24 hr post-treatment with EVs respectively (1 μg/ml equals approximately 3 x 108 vesicles/ml). Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). IFN-β protein (C) levels in wild type BMMs 5 and 24 hr post-treatment with EVs respectively (1 μg/ml equals approximately 3 x 108 vesicles/ml). Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (D) IFN-β mRNA level was quantified in wild type BMMs treated with EVs at a concentration of 10μg/ml for the times indicated. Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). Quantitative RT-PCR analysis for IFN-β mRNA (E) levels in either WT, MAVS- and/or RIG-I-knockdown BMMs treated for 4 hr (E) with 10µg/ml EVs from uninfected or M.tb-infected macrophages. (Control) negative control siRNA. Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). IFN-β protein (F) levels in either WT, MAVS- and/or RIG-I-knockdown BMMs treated for 24 hr (F) with 10µg/ml EVs from uninfected or M.tb-infected macrophages. (Control) negative control siRNA. Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (G) Similar to above except using WT and Mavs -/- BMMs. Data shown are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (H) Similar to above except using WT and Mavs -/- BMMs. Scale bar, 20 μM. Data shown in (H) are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (I) Schematic of transwell assay for measuring IFN-β induction. BMMs were uninfected or infected with WT M.tb at a MOI of 5. Twenty-four hr post-infection, cells were transferred into the top chamber and co-cultured with naïve BMMs (Bottom chamber). The IFN-β mRNA levels in BMMs (bottom) were quantified by qRT-PCR after 24 hr. (J) Top chamber: WT BMMs; Bottom chamber: WT or Mavs -/- BMMs. Data shown in (J) are the mean ± SD (n = 3 wells per group) and all data shown are representative of three independent experiments (biological replicates). * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (K) Western blot analysis for phospho-TBK1 (Ser172) in WT and MAVS-knockdown BMMs treated for 4 hr with EVs from uninfected or M.tb-infected macrophages. β-actin served as a loading control. Densitometry of the western blots for (K) are shown. (L) Western blot analysis of IRF3 under the conditions described above for TBK1. Histone H3 (H3) and β-actin were used as loading controls for nuclear fraction and whole-cell lysate (WCL) respectively. Densitometry of the western blots are shown. (M) Fluorescence microscopy analysis for nuclear translocation of IRF3 in wild type BMMs untreated (Mock) or treated with EVs from uninfected or M.tb-infected macrophages. Scale bar, 20 μM. (N) IRF3 nuclear translocation was analyzed in RIG-I- or MAVS-knockdown BMMs. Scale bar, 20 μM. \n",
      ".Figure 3M.tb CFU in WT mouse BMMs pre-treated with EVs minus co-treatment with IFN-ɣ. BMMs were treated with EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) cells for 0, 24, 48 and 72 hr following a 1 hr M.tb infection. Mock, no EV treatment. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. ; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. M.tb CFU in WT mouse BMMs pre-treated with EVs plus (B) co-treatment with IFN-ɣ. BMMs were treated with EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) cells for 0, 24, 48 and 72 hr following a 1 hr M.tb infection. Mock, no EV treatment. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. ; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy analysis for colocalization of M.tb (GFP) with lysosome marker Lamp-1 in WT mouse BMMs. Cells were pre-treated for 5 hr with EVs from uninfected or M.tb-infected macrophages plus IFN-γ, and then infected with GFP-expressing M.tb for 24 hr prior to immunostaining. quantitative analysis of M.tb colocalization with Lamp-1 in WT mouse BMMs Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy analysis for colocalization of M.tb (GFP) with lysosome marker Lamp-1 in Mavs-/- (E and F) mouse BMMs. Cells were pre-treated for 5 hr with EVs from uninfected or M.tb-infected macrophages plus IFN-γ, and then infected with GFP-expressing M.tb for 24 hr prior to immunostaining. quantitative analysis of M.tb colocalization with Lamp-1 in Mavs-/- mouse BMMs Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy as described above except using control siRNA-treated (G and H) BMMs. quantitative analysis of M.tb colocalization with Lamp-1 in control siRNA-treated mouse BMMs Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. Immunofluorescence microscopy as described above except using RIG-I siRNA-treated (I and J) BMMs. quantitative analysis of M.tb colocalization with Lamp-1 in RIG-I siRNA-treated mouse BMMs Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. Scale bar, 5 μM n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. M.tb CFU in infected Mavs-/- (K) mouse BMMs pre-treated with IFN-ɣ plus EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) BMMs. CFU was determined immediately after the 1 hr infection or 24 and 72 hr post-infection. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. ; * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. M.tb CFU in control siRNA-treated (L) or RIG-I siRNA-treated (M) mouse BMMs pre-treated with IFN-ɣ plus EVs from uninfected (EVs_Control) or M.tb-infected (EVs_M.tb) BMMs. CFU was determined immediately after the 1 hr infection or 24 and 72 hr post-infection. Data shown are the mean ± SD of three independent infections and all data shown are representative of at least three independent experiments. * p < 0.05, ** p < 0.01 and *** p < 0.001 by two-tailed Student's t-test. \n",
      ".Figure 6(A and B) Quantitative analysis of immunofluorescence microscopy images for colocalization of M.tb with Lamp-1 in WT (A) and Mavs -/- (B) BMMs infected with M.tb for 24 hr and then treated for an additional 24 hr with moxifloxacin (MFX) and/or EVs from M.tb-infected BMMs (EVs+MFX) in the presence of recombinant mouse IFN-ɣ. Mock, no EVs or moxifloxacin treatment; EVs_Control, EVs from uninfected BMMs. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). LC3 (C), colocalization with M.tb were analyzed. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). NOX2 (D) colocalization with M.tb were analyzed. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). p47phox (E) colocalization with M.tb were analyzed. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). Mavs -/- BMMs were used and colocalization of M.tb with LC3 (F) were quantified. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). Mavs -/- BMMs were used and colocalization of M.tb with NOX2 (G) were quantified. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). Mavs -/- BMMs were used and colocalization of M.tb with p47phox (H) were quantified. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). (I and J) M.tb CFU analysis in WT (I) and Mavs -/- (J) BMMs infected with M.tb for 24 hr followed by treatment with EVs, moxifloxacin or combination in the presence of IFN-ɣ for 24 and 72 hr. Data shown are representative of three independent infections. The results are the mean ± SD of three independent experiments. n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). \n",
      ".Figure 3MITOL KO facilitated IRE1α oligomerization. MEFs were transfected with IRE1α-GFP expression vector for 24 hours prior to Tu treatment for indicated periods (A). The right panels show 5-fold magnification images of the boxed regions. Percentages of cells with IRE1α-foci were calculated from 100 cells by visual inspection in each independent experiment (A). To evaluate the oligomerization level of endogenous IRE1α, cells without any transfection were solubilized and separated by sucrose density-gradient centrifugation, followed by immunoblotting with anti-IRE1α antibody (B). IRE1α oligomers were stabilized by MITOL KO. MEFs transfected with IRE1α-GFP were treated with Tu. After 4 hours, these cells were washed with PBS and subsequently re-fed with fresh media for indicated periods. The right panels show 5-fold magnification images of the boxed regions. Percentages of cells with IRE1α-foci were calculated from 100 cells by visual inspection at indicated periods after wash out of Tu in each independent experiment. Scale bar represents 10 µm. Sustained IRE1α autophosphorylation after wash out of Tu in MITOL-KO MEFs. MEFs were treated with Tu for 4 hours and washed with PBS and re-fed with fresh media for indicated periods, following by immunoblotting anti-IRE1α antibody after Phos-tag SDS-PAGE. Error bars represent SD (n=3). MITOL deletion promoted the interaction between IRE1α and BIM under ER stress. MITOL-depleted HEK293 cells (KO) were generated using CRISPR-Cas9 system. Cells were co-transfected with indicated vectors 24 hours prior to Tu treatment for 12 hours. Cells were immunoprecipitated with anti-FLAG antibody, followed by immunoblotting with indicated antibodies. \n",
      ".Figure 7(A) Schematic for EVs-based adjunctive immunotherapy and moxifloxacin-based chemotherapy in M.tb-infected mice. Representative histopathological analysis for lung sections of WT (B) mice that were infected with M.tb and subsequent left untreated (Mock) or treated with EVs from uninfected BMMs (EVs_Control), EVs from M.tb-infected BMMs (EVs), moxifloxacin (MXF), or combination of EVs and MXF (EVs+MXF). Data shown is representative of two independent experiments. The results are the mean ± SD (n = 4 mice per group). Scale bar, 100 μM M.tb CFU in the lung and spleen of WT (C) mice treated with EVs, MXF or combination of both. Data shown is representative of two independent experiments. The results are the mean ± SD (n = 4 mice per group). n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). Representative histopathological analysis for lung sections of Mavs -/- (D) mice that were infected with M.tb and subsequent left untreated (Mock) or treated with EVs from uninfected BMMs (EVs_Control), EVs from M.tb-infected BMMs (EVs), moxifloxacin (MXF), or combination of EVs and MXF (EVs+MXF). Data shown is representative of two independent experiments. The results are the mean ± SD (n = 4 mice per group). Scale bar, 100 μM M.tb CFU in the lung and spleen of Mavs -/- (E) mice treated with EVs, MXF or combination of both. Data shown is representative of two independent experiments. The results are the mean ± SD (n = 4 mice per group). n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). ELISA analysis for, IFN-β, TNF-α, IL-1β and IFN-γ protein level in serum isolate from M.tb-infected WT (F) or Mavs -/- (G) mice treated with EVs, MXF or a combination of both. Data shown is representative of two independent experiments. The results are the mean ± SD (n = 4 mice per group). n.s., not significant; * p < 0.05, ** p < 0.01 and *** p < 0.001 by Student's t-test (two tailed). \n",
      ".Figure 2IRE1α inhibition compensated MITOL-KO MEFs for vulnerability to ER stress. MEFs were transfected with scramble or siIRE1α (#1, #2). After 24 hours, these MEFs were exposed with Tu for 24 hours , followed by immunoblotting with indicated antibodies. Error bars represent SD (n=3). MEFs were treated with 4µ8C (25 µM) for 2 hours prior to Tu treatment for 24 hours (B), followed by immunoblotting with indicated antibodies. Error bars represent SD (n=3). MITOL KO did not affect the autophosphorylation of IRE1α. MEFs were exposed with Tu for indicated periods and phosphorylated IRE1α was detected by immunoblotting after Phos-tag SDS-PAGE. MITOL KO enhanced RIDD and xbp1 splicing. MEFs were incubated with Tu for 4 hours. Expression levels of various UPR associated genes were determined by qRT-PCR MITOL KO enhanced RIDD and xbp1 splicing. MEFs were incubated with Tu for 4 hours. MEFs were transfected with either miR-17 or miR-34 luciferase reporter vector 24 hours prior to Tu treatment for 4 hours, followed by luciferase reporter assay (F). Error bars represent SD (n=3). MEFs were incubated with Tu for 4 hours. Expression levels of various UPR associated genes were determined by qRT-PCR IRE1α-mediated JNK phosphorylation was prolonged by MITOL KO. MEFs were incubated with Tu for indicated periods and phosphorylated JNK was detected by immunoblotting with indicated antibodies. Error bars represent SD (n=3). JNK inhibition reduced ER stress-induced apoptosis of MITOL-KO MEFs. MEFs were treated with 20 µM JNK inhibitor-II (JNK inh) 2 hours prior to Tu treatment for 24 hours, followed by immunoblotting with indicated antibodies. Error bars represent SD (n=3). \n",
      ".Figure 1MITOL-KO MEFs was vulnerable to ER stress. Control MEFs (MITOLF/F) and MITOL-KO MEFs (MITOL-/-) were treated with DMSO as control, 0.8 µM thapsigargin (Tg), 0.7 µg/mL tunicamycin (Tu) or 1.2 µg/mL brefeldin A (Br) for 24 hours. All experiments using Tu below were performed at concentration of 0.7 µg/mL. The relative cell viability shows a ratio between the survival of MEFs in the absence and presence of ER stress inducers. Viable cells were detected by cell viability assay using cell counting kit-8. MITOL KO increased ER stress-induced apoptosis. MEFs were treated with indicated ER stress inducers at same concentration as Figure 1A for 18 hours and stained with Annexin V-FITC for the detection of apoptosis using flow cytometry. MITOL re-expression in MITOL-KO MEFs rescued ER stress vulnerability. MEFs were transfected with MITOL-coding vector (WT), MITOL C65/68S-coding vector (CS), or empty vector (Vec) 24 hours prior to Tu treatment for 18 hours, followed by immunoblotting with indicated antibodies. Cleaved caspase-3 (cC3) and cleaved PARP (cPARP) were used as apoptosis markers. MITOL KO enhanced ER stress-induced Bax translocation and cytochrome c release. Mitochondria or cytosol fraction was isolated from MEFs treated with Tu for 18 hours, followed by immunoblotting with indicated antibodies. MITOL KO induced excessive mitochondrial depolarization by ER stress. MEFs transfected with indicated vectors were treated with Tu for 18 hours and then stained with TMRM, followed by flow cytometric analysis. \n",
      ".Figure 1A. Schematic work-flow for 10X Genomics scRNASeq of pooled SCN slices after three days in culture. For each sequencing run ca. 17 organotypic SCN slices were pooled into a single sample and their cells dissociated using papain to obtain single cell suspension of ~8,000 cells/µL. Dispersed single cells, along with barcoded 10X Genomics Chromium Single Cell 3' v2 technology gel beads, were partitioned into water-in-oil droplets. Within these droplets, reverse transcription and amplification steps generated cDNA libraries for the barcoded single cells. B. t-SNE plot from 13,324 sequenced SCN cells collected at CT7.5, across three independent sequencing runs. Each sequenced cell marked with a unique barcode is represented as a single dot. This dimensionality reduction method aims to maintain both local and global structure of the data by clustering data points of highest similarity nearest to each other. Cell clusters recognised by the graph-based clustering algorithm are further colour-coded to highlight the cell types: putative SCN neurons, extra-SCN hypothalamic neurons, astrocytes, oligodendrocytes, radial glia, microglia, ependymocytes and endothelial cells. C. Heatmap of the top 5 up-regulated genes that most distinguish the transcriptional clusters identified in B. The degree of up-regulation is measured as the log 2-fold change ratio of gene expression calculated for cells of each cluster and normalized to a size factor accounting for the total transcript count of each sequenced cell across each cluster. D. t-SNE plot from 16,996 SCN sequenced cells collected at CT15.5, across two independent sequencing runs. As for B, the cell types: putative SCN neurons, extra-SCN hypothalamic neurons, astrocytes, oligodendrocytes, radial glia, microglia, ependymocytes and endothelial cells, were identified. E. As C, but for cell clusters identified at CT15.5. \n",
      ".Figure 5MITOL added polyubiquitin chains to K481 of IRE1α. Lysates of HEK293 cells transfected with each lysine mutant of IRE1α and indicated vectors were immunoprecipitated with anti-FLAG antibody, followed by immunoblotting with indicated antibodies. 481: K481R, 545: K545R, 568: K568R. IRE1α K481R interacted with MITOL. Lysates of HEK293 cells transfected with indicated vectors were immunoprecipitated with anti-FLAG antibody, followed by immunoblotting with indicated antibodies. Overexpression of IRE1α K481R induced apoptosis. MEFs were transfected with indicated vectors. After 24 hours, these cells were stained with Annexin V-FITC Overexpression of IRE1α K481R induced apoptosis. or subjected by immunoblotting with indicated antibodies (D). Enhanced RIDD activity in IRE1α K481R. MEFs were transfected with indicated vectors 24 hours prior to analysis (E, F). The RIDD activity was measured by qRT-PCR. Hyper-oligomerization of IRE1α K481R. MEFs were transfected with IRE1α-GFP or IRE1α K481R-GFP. After 24 hours, these cells were exposed with Tu for indicated periods and GFP signals were observed. The right panels show 5.5-fold magnification images of the boxed regions. Percentages of cells with IRE1α-foci were calculated from 100 cells by visual inspection in each independent experiment. Increased interaction of IRE1α K481R with BIM. Cells transfected with indicated vectors 24 hours prior to immunoprecipitation, followed by immunoblotting with indicated antibodies. \n",
      ".Figure 4Interaction between MITOL and IRE1α. Lysates of HEK293 cells were subjected to immunoprecipitation with anti-IRE1α antibody, followed by immunoblotting with indicated antibodies (A). Interaction between MITOL and IRE1α. Lysates of HEK293 cells co-transfected with indicated vectors were subjected to immunoprecipitation, followed by immunoblotting with indicated antibodies (B). Interaction between MITOL and IRE1α. GST-pull-down assay of IRE1α or MITOL was performed on lysates of MEFs, followed by immunoblotting with indicated antibodies (C, D). N-ter: 1-90 aa of MITOL, Loop: 158-210 aa of MITOL, C-ter: 241-278 aa of MITOL, 465-977 aa of IRE1α. MITOL ubiquitylated IRE1α. Lysates of HEK293 cells immunoprecipitated with indicated antibody, followed by immunoblotting with indicated antibodies. MITOL ubiquitylated IRE1α. Lysates of HEK293 cells transfected with indicated vectors, were immunoprecipitated with indicated antibody, followed by immunoblotting with indicated antibodies. MITOL ubiquitylated IRE1α. Lysates of MEFs (G) transfected with indicated vectors, were immunoprecipitated with indicated antibody, followed by immunoblotting with indicated antibodies. MITOL adds lysine 63-linked polyubiquitin chains to IRE1α. Lysates of HEK293 cells transfected with indicated vectors were immunoprecipitated with indicated antibody. IRE1α ubiquitylation was detected MITOL adds lysine 63-linked polyubiquitin chains to IRE1α. Lysates of HEK293 cells transfected with indicated vectors were immunoprecipitated with indicated antibody. IRE1α ubiquitylation was detected \n",
      ".Figure 6Accumulations of MITOL and IRE1α in the MAM fraction. Each organelle fraction was isolated from MEFs by percoll density-gradient centrifugation, followed by immunoblotting with indicated antibodies. mito: pure mitochondria, micro: microsome, cyto: pure cytosol. High solubility of MITOL and IRE1α in the low dose digitonin. Raft-like structure including MAM was solubilized from the crude mitochondrial fraction of MEFs with indicated concentration of digitonin. S: supernatant, P: pellet. IRE1α regulation by MITOL occurred in the MAM-rich mitochondrial fraction. Crude mitochondrial fraction containing MAM (Mito plus MAM) and the cytosolic fraction containing ER (Cyto plus ER) were isolated from HEK293 cells transfected with indicated vector and subsequently immunoprecipitation assay was performed to detect the interaction between MITOL and IRE1α IRE1α regulation by MITOL occurred in the MAM-rich mitochondrial fraction. Crude mitochondrial fraction containing MAM (Mito plus MAM) and the cytosolic fraction containing ER (Cyto plus ER) were isolated from HEK293 cells transfected with indicated vector and subsequently immunoprecipitation assay was performed to detect the level of IRE1α ubiquitylation (D). Knockdown of PACS2 impaired the interaction of MITOL with IRE1α. HEK293 cells were transfected with either scramble RNA or siPACS2 and indicated vectors for 24 hours prior to immunoprecipitation assay. Knockdown of PACS2 inhibited IRE1α ubiquitylation by MITOL. HEK293 cells were transfected with either scramble RNA, siPACS2 and indicated vectors 24 hours prior to immunoprecipitation assay. Knockdown of Mfn2 inhibited IRE1α ubiquitylation by MITOL. HEK293 cells were transfected with either scramble RNA, siMfn2 and indicated vectors 24 hours prior to immunoprecipitation assay. \n",
      ".Figure 7MITOL ubiquitylated IRE1α in the spinal cord of mice. Lysates of the spinal cord in mice were immunoprecipitated with anti-IRE1α antibody, followed by immunoblotting with indicated antibodies. WT: MITOLF/F mice, MITOLnestin: MITOLF/F, Nestin-Cre mice. Loss of MITOL led to hyper-oligomerization of IRE1α in the spinal cord under ER stress. 3 month-old WT or MITOLnestin mice were treated with 1 mg/kg Tu for 24 hours. Spinal cord was solubilized with NP-40 lysis buffer and separated by sucrose-density gradient centrifugation. MITOL deletion resulted in hyper-activation of IRE1α RNase in ER stress. WT or MITOLnestin mice were injected with Tu for 24 hours and spinal cord was analyzed by qRT-PCR. Increased ER stress-induced apoptosis in the MITOL-KO spinal cord. 3 month-old WT or MITOLnestin mice were treated with Tu for 24 hours and each spinal cord was stained using TUNEL and hoechst (F) Arrowheads indicate the representative neurons. Increased ER stress-induced apoptosis in the MITOL-KO spinal cord. 3 month-old WT or MITOLnestin mice were treated with Tu for 24 hours and each spinal cord was stained using cresyl violet (G). Arrowheads indicate the representative neurons. \n",
      ".Figure 2A. Pooled SCN cells generate a combined dataset of 17,363 cells. The sequenced SCN cells carry explicit barcodes which were used to back-reference sequencing run identity. This yields a clear-cut segregation into two transcriptionally distinct cell groups. Left: 98.6% of cells in the yellow cluster are CT7 5 cells. Right: 98.8% of cells in the grey cluster are CT15.5 cells. B. Volcano plot showing the magnitude of differential gene up-regulation, as measured by the respective log 2-fold change in expression level, at day (positive) or night (negative) versus statistical significance, as quantified by a negative binomial exact test (red dots, dashed line p<0.05 adjusted for multiple testing using Benjamin-Hochberg). Known circadian-regulated genes are labelled. \n",
      ".Figure 3A. Sub-clustering of eight neuronal populations using unsupervised graph-based clustering within the transcriptomes of CT7.5 (top) and CT15.5 (bottom) SCN cells. The heatmaps show the respective up-regulation of three representative marker genes in each sub-population. When compared, three subpopulations showed strong transcriptional similarity between the time-points of sequencing (Groups A, B and C), two showed partial similarity (Groups D and E) and three were transcriptionally distinct to either CT7.5 or CT15.5 sequenced cells, giving rise to the identification of eleven SCN neuronal sub-populations. B. Violin plots depicting the distribution of the quantified average cellular expression of Prok2, ProkR2, Vip, Vipr2, Avp, Avpr1a, Grp and Grpr in each sub-cluster in CT7.5 (left) and CT15.5 (right) datasets. C. Visualisation of the location within the pooled t-SNE plot of cells positively expressing >90% of the overall expression range of Prok2, ProkR2, Vip, Vipr2, Avp, Avpr1a, Grp and Grpr. This shows a contrasting expression pattern of the respective genes in cells collected at CT7.5 versus CT15.5. D. Venn diagram summarising the degree of non-/overlapping signalling axes: Prok2 (purple) - ProkR2 (brown), Vip (green) - Vipr2 (red), Avp (blue) - Avpr1a (orange) and Grp (pink) - Grpr (grey). The number SCN neurons collected at CT7.5 positively expressing each gene, which is associated to the respective signalling axis, is further annotated within the population circles. \n",
      ".Figure 4Schematic of inferred topology of neuropeptidergic signalling axes between SCN neuronal sub-populations in circadian day. Top: clusters are annotated in the overview t-SNE for circadian day (A.) and night (B.), respectively. Below: the identified SCN subpopulations are depicted as nodes of each neuropeptide-receptor network, connected by edges that are weighted (shown as differential line thickness) based on the average count of the respective cellular gene expression for each ligand and receptor. The nodes of the inferred topology networks are colour-coded to match those of the overview t-SNE plots. Nodes further highlighted in yellow mark ligand-enriched source clusters. The edges are colour-coded for each neuropeptide-receptor signalling pair: Prok2/ProkR2 - purple, Vip/Vipr2 - green, Avp/Avpr1a - blue. The sum of edge weights within each day and night inferred signalling network is presented below each map to provide a quantification of the overall degree of connectivity. \n",
      ".Figure 6A. Phase response curve for Prok2- (red) and vehicle-treated (grey) SCN slices (p>0.05). The phase shift (h), calculated from the mean of the first three peaks after Prok2 application in Per2::Luciferase rhythms, is plotted against the CT of treatment. Faded circles (grey=vehicle, red=Prok2) show individual SCN datapoints, whilst the mean values across the vehicle-treated or Prok2-treated cohort within 3 hour time-bins are shown bold. B. Application of VIP (blue) delivered during CT7-13 served as a positive control and elicited robust phase delays of ongoing Per2::Luciferase SCN slice oscillations when administered between CT9-12. C. Comparison of phase shifts measured across each treatment (vehicle vs. VIP, *p=0.04) when administered between CT9-12. D. Representative Per2::Luciferase traces before and after administration (arrow) of vehicle (black), Prok2 (red) or VIP (blue) within CT9-12. E. Comparison across treatment cohorts of the relative change in baseline of Per2-driven bioluminescence when vehicle, Prok2 or VIP was applied within CT9-12. Only VIP is statistically shown to significantly raise Per2::Luciferase baseline levels (p=0.0007; vehicle vs. Prok2, ns. p=0.17; vehicle vs. VIP, **p=0.002, Prok2 vs. VIP, *p=0.02). F. The relative change in amplitude of Per2::Luciferase rhythm after treatment with Prok2 or VIP significantly decreases compared to vehicle application (p=0.002; vehicle vs. Prok2, *p=0.01, vehicle vs. VIP, **p=0.001). G. Relative change in baseline Per2::Luciferase measured from the first cycle after treatment across circadian time, binned in 6 hour intervals (treatment factor p<0.0001). Prok2 within CT12-18 significantly increased baseline Per2::Luciferase more than vehicle, *p=0.01. H. Relative change in the amplitude of the first Per2::Luciferase cycle measured after treatment across circadian time, binned in 6 hour intervals (treatment factor p<0.0001). Prok2 within CT6-12 and CT18-24 significantly decreased Per2::Luciferase rhythm amplitude compared to vehicle, CT6-12 *p=0.01, CT18-24 *p=0.04. \n",
      ".Figure 8A. Representative Per2::Luciferase trace of Cry1-/- SCN slice at baseline (black) and after AAV-mediated (arrows) Cry1 complementation in Prok2+ cells (red) or ProkR2+ cells (yellow). B. Corresponding peak-to-peak period plots for baseline (black), Cry1-expressed in Prok2+ (red), and then in ProkR2+ cells (yellow). Individual slices are depicted as grey traces, filled points are their mean±S.E.M. C. Group data (n=5) across three Per2::Luciferase cycles as shown in B. (p=0.006; baseline vs. Prok2.Cre *p=0.04, baseline vs. Prok2.Cre and ProkR2.Cre *p=0.02, Prok2.Cre vs. Prok2.Cre and ProkR2.Cre ns. p=0.95). D. As in A but with order of transfection reversed; green =Cry1 only in ProkR2+ cells (baseline vs. ProkR2.Cre *p=0.04). As in B and C but with transfection order reversed (green =Cry1 expression in ProkR2+ cells only, n=6) (p=0.006; baseline vs. ProkR2.Cre *p=0.03, baseline vs. ProkR2.Cre and Prok2.Cre **p=0.009, ProkR2.Cre vs. ProkR2.Cre and Prok2.Cre ns. p=0.1). F. As in B and C but with transfection order reversed (green =Cry1 expression in ProkR2+ cells only, n=6) (p=0.006; baseline vs. ProkR2.Cre *p=0.03, baseline vs. ProkR2.Cre and Prok2.Cre **p=0.009, ProkR2.Cre vs. ProkR2.Cre and Prok2.Cre ns. p=0.1). G. The final circadian periods of Cry1-/- SCN slice double-transduced in complementary reverse orders did not differ between each other, p=0.7. \n",
      ".Figure 7A. Representative Per2::Luciferase traces before (black) and after vehicle (grey) or Prok2RA (pink) topical application of SCN slices. B. Comparison of the Per2::Luciferase rhythm amplitude of the cycle preceding and the first and second cycles succeeding vehicle (applied CT23-20 dark grey n=12 SCN slices, CT-20-23 light grey n=6 SCN slices) or Prok2RA (applied CT23-20 purple n=13 SCN slices, CT20-23 pink n=11 SCN slices) treatment (interaction, p<0.0001). A significant decline in amplitude occurred when Prok2RA signalling was blocked between CT20-23 (****p<0.0001). C. Change in SCN period after treatment with vehicle (grey) or Prok2RA (pink) between CT23-20 and CT20-23 (vehicle CT23-20 vs. Prok2RA CT20-23, **p=0.009; vehicle CT20-23 vs. Prok2RA CT20-23, **p=0.001) (n = 6, 12, 13 and 11 independent SCN slices, respectively). D. Prok2RA application resulted in a significant period lengthening effect of ongoing SCN ensemble Per2::Luciferase rhythms, which was reversed upon drug washout (p=0.0002, ***p=0.0009, *p=0.01) (n = 22 independent SCN slices with repeated measure). \n",
      ".Figure 5A. Representative, detrended Cry1-luc traces of emitted bioluminescence (photon count values) from 4 SCN slices, pre-transfected with AAV pCry1-DIO-Luciferase after addition of (left) pProk2.Cre.T2A.mCherry (red, cohort n=10) or (right) pProkR2.Cre.T2A.Venus (green, cohort n=9) transfection. B. Raster plots (yellow=trough, blue=peak) of Cry1-luc bioluminescence oscillations from a representative slice of each cohort. The black arrow marks the time-point of pProk2.Cre.T2A.mCherry or pProkR2.Cre.T2A.Venus AAV transfection, thus resulting in the evolution of rhythmic Cry1-luc bioluminescence. C. The period of Cry1-luc bioluminescence rhythms in SCN slices transfected with either pProk2.Cre.T2A.mCherry or pProkR2.Cre.T2A.Venus does not differ between treatments (p=0.71, n= 10, 9 SCN slices per group, respectively).E. Representative raster plots of single-cell bioluminescence (yellow=minimum, blue=maximum) from Prok2Cry1-luc (left) or ProkR2Cry1-luc (right) cells in single slices. F. Within-slice cellular synchrony depicted in representative Rayleigh plots of the phases of peak Cry1-luc bioluminescence for Prok2Cry1-luc (red) and ProkR2Cry1-luc (green) cells. The Rayleigh plots can be read as the face of a clock with CT0 marking the top and CT12 the bottom of the plot circle. The black arrow depicts the mean peak phase across the individual oscillators. G. The mean Rayleigh vector lengths for Prok2Cry1-Luc and ProkR2Cry1-luc (p>0.99) do not differ between the cohorts (n=5 SCN slices per group). H. Left: confocal images of representative SCN slices dual-transfected with pProk2.Cre.T2A.mCherry and Syn.GCaMP6f (top), or pProkR2.Cre.T2A.Venus and Syn.jRCaMP1a (below) (scale bar=500µm). Right, above: representative detrended plots of signal from GCaMP6f (green) traces and pCry1-DIO-Luc, emitted from Prok2+ SCN cells (red). Right, below: representative detrended plots of jRCaMP1a (purple) and pCry1-DIO-Luc emitted from ProkR2+ cells (light green). Orientation bars depict the horizontal axis of the optic chiasm and the vertical axis of the third ventricle for each SCN. I. After registering the respective peaks of [Ca2+]i fluorescence to the peaks of Cry1-luc bioluminescence in each respective cohort, phase-alignment between the peaks of Prok2Cry1-luc and ProkR2Cry1-luc bioluminescence shows that the intracellular clocks of Prok2+ cells are phase advanced by ~0.6h to those of ProkR2+ cells (**p<0.01) (n=5 SCN slices per group). \n",
      ".Figure 3A) Schematic diagram of full length TBC1D14, indicating the C-terminal TBC domain (shaded), and truncation mutants used in subsequent analyses. B) HEK 293A cell lysates (from approximately 8 × 106 cells per construct) expressing the indicated GFP fusions were subjected to IP and immunoblotting for GFP, TRAPPC12 and TRAPPC4. Data are representative of 3 independent experiments. \n",
      ".Figure 9A. Representative Per2::Luciferase traces of Cry1,2-/- SCN slices at baseline (black, arrhythmic), after Cry1 complementation in Prok2+ cells (red) and further complementation in ProkR2+ cells (yellow, rhythmic) (cohort n=8).C. SCN period after Cry1 expression in both Prok2+ and ProkR2+ cells did not differ with order of transduction (p=0.4). D. Bioluminescence rhythm amplitude did not differ with order of complementation (p=0.96). E. Paired goodness-of-fit values for Per2::Luciferase oscillations of baseline, single-transfection and double-transfection SCN (p=0.02, Prok2.Cre before and after, ns. p=0.17; ProkR2.Cre before and after, ns. p=0.99; both Prok2.Cre and ProkR2.Cre applied before and after ****p<0.0001). F. Binary representation of initiated slice elements of a representative SCN slice (CCD recorded bioluminescence pixels) - black indicates non-initiated and pink indicates initiated, i.e., rhythmic elements (cohort n=12). G. Phase map of Per2::Luciferase bioluminescence in a representative Prok2+/ProkR2+ Cry1-initiated SCN slice showing spatial phase synchronisation across the slices (phases colour-coded relative to the mean phase of each slice). H. Inferred neuronal group interactions in relation to circuit-level contribution as \"Distributor\" or \"Integrator\" hubs. \n",
      ".Figure 2A) HEK293A cells transfected with GFP (upper panels) or GFP-TBC1D14 (lower panels) (green) were labelled with anti-RAB11A (red) and anti-RAB1B (white, blue in merge) antibodies. Arrows depict transfected cells. Inset: GFP-TBC1D14 transfected cell showing triple co-localisation between GFP-TBC1D14, RAB11A and RAB1B (white arrows) and double co-localisation between GFP-TBC1D14 and RAB11A (yellow arrows). B) Co-localisation of GFP-TBC1D14, TRAPPC4 (red) and RAB11A (white, blue in merge). White arrowheads indicate regions of triple co-localisation. C) Co-localisation of GFP-TBC1D14, TRAPPC4 (red) and RAB1B (white, blue in merge). White arrowheads indicate regions of triple co-localisation, yellow arrowheads regions of double co-localisation between GFP-TBC1D14 and TRAPPC4. All scale bars = 20 μm. D) Model for recruitment of RAB1 to tubulated RE. RAB11-GTP recruits TBC1D14 to the ERC membrane, and TBC1D14 in turn recruits TRAPP. TRAPP acts as a GEF to recruit RAB1 aberrantly to ERC membranes. \n",
      ".Figure 1A) GST pulldown used to analyse TBC1D14 interactors. Recombinant GST or GST-TBC1D14 was incubated with or without HEK 293A lysate. Bound proteins were eluted from the beads using Laemmli sample buffer and subjected to SDS-PAGE, with visible bands being excised from the gel and analysed by mass spectrometry. B) Lysates of HEK293A cells expressing either GFP or GFP-TBC1D14 were subjected to immunoprecipitation (IP) with GFP-Trap, and then immunoblotted for GFP, TRAPPC12 and TRAPPC4. C) Lysates of cells expressing GFP or GFP-TRAPPC3 were subjected to IP with GFP-Trap, and then immunoblotted for TRAPPC4, GFP and TBC1D14. D) Cells expressing GFP-TBC1D14, generating a tubulated endosomal compartment, were fed Alexa647 labelled Transferrin (white, blue in merge) for 15 minutes in full medium, fixed, stained for endogenous TRAPPC4 (red) and analysed by confocal microscopy. White arrowheads indicate transfected cells, yellow arrowheads depict regions of colocalisation in inset. Scale bars = 10 μm. E) Cells treated as in D were stained for endogenous TRAPPC12 (red) and analysed by confocal microscopy. Scale bars = 10 μm F) Cells in complete medium were fixed, stained for endogenous TBC1D14 (green), RAB1B (red) and TRAPPC4 (white, blue in merge) and analysed by confocal microscopy. Inset shows protein localisation in juxtanuclear area. Yellow arrowheads show regions of colocalisation Scale bars = 10 μm. Data shown are representative from 3 independent experiments. \n",
      ".Figure 4A) Cells expressing the indicated GFP-fusions (green) were loaded with Alexa647-Transferrin (red; arrowheads indicate transfected cells) for 15 minutes, fixed and the morphology of the transferrin positive recycling endosomes analysed by confocal microscopy. Scale bars = 10 μm. Arrows in inset indicate the juxtanuclear ERC B) Cells transfected as in (A) were loaded with Alexa-647 transferrin for 15 minutes and the fluorescent transferrin chased out for the indicated time periods. At the end of the timecourse the cells were fixed and the transferrin content of GFP positive cells analysed by flow cytometry. Results are expressed as a percentage of Tfn fluorescence at t=0 and are the mean of 3 independent experiments, ± s.e.m. C) Cells transfected as in (A) were stained for the cis-Golgi marker GM130. Arrowheads depict transfected cells. Scale bars = 10 μm. Bar chart is quantification of fragmented Golgi complex (> 100 cells per condition from three independent experiments, ± s.e.m. ** p < 0.01, one way ANOVA with Sidak's post-test). D) HeLa C1 cells transfected with CFP or CFP-TBR were treated with D/D solubiliser (Clontech) or D/D solubiliser plus 2.5 μg/ml Brefeldin A (BFA) for the indicated time periods, trypsinised, fixed and their GFP content analysed by flow cytometry. Results are expressed as a percentage of GFP levels at t=0, and the mean of 3 independent experiments ± s.e.m. \n",
      ": TBR expression inhibits autophagy at an early stage.Figure 5A) HEK293A cells expressing GFP, GFP-TBC1D14 224-669 or GFP-TBR were treated in duplicate with EBSS, EBSS plus 100 nM BafA1 or not for two hours, lysed and subjected to immunoblotting for LC3B, tubulin and GFP. The amount of LC3B/tubulin for each condition from three independent experiments is shown on the bar graph, ±s.e.m. * p<0.05, ** p<0.01, one way ANOVA with Sidak's multiple comparison test. B) HEK293A GFP-LC3B (2GL9) cells were transfected with mCherry or mCherry-TBR, treated with EBSS for two hours, fixed and analysed by confocal microscopy. C) HEK293A cells were treated as in (B), fixed, stained for WIPI2 and analysed by confocal microscopy. D) HEK293A cells stably expressing GFP-DFCP1 were treated as in (B), fixed and analysed by confocal microscopy. Scale bars = 20 μm. For quantification of (B), (C) and (D), 10 fields of view containing transfected cells were imaged for each of three experiments, and the number of GFP-LC3B, WIPI2 or GFP-DFCP1 puncta per cell enumerated using Imaris software (Bitplane). \n",
      ".Figure 6A) Overview of BioID technique used to determine TRAPP subunit most proximal to TBC1D14. D = denaturing contitions, ND = non-denaturing conditions B) Samples prepared according to (A) (8 × 107 HEK293A cells per transfection) were lysed in TNTE, split equally and denatured with 1% SDS (D) or not (ND) and subjected to 1 hour streptavidin resin pulldown at room temperature. The bound proteins were eluted in 2x Laemmli sample buffer plus 3 mM Biotin and immunoblotted for the indicated proteins. C) siRNA transfected control cells (RISC Free) or TRAPPC8 knockdown (TRAPPC8) (1x10cm plate per condition) were lysed in TNTE and used in pulldown assays with immobilised recombinant GST or GST-TBR. Bound proteins were subjected to immunoblot analysis using TRAPPC8, TRAPPC4 and tubulin antibodies. Ponceau staining was used to verify loading of the GST fusion proteins (GST-TBR indicated with arrowhead). The bar chart expresses the relative amount of TRAPPC4 isolated from siTRAPPC8 compared to siRISC Free control cells from three independent experiments, error bars are ± s.e.m. ** p<0.01, unpaired T-test D) HEK293T cells stably depleted of TRAPPC8 (shC8) or not (shCtrl) were transfected with constructs encoding GFP or GFP-TRAPPC3 and subjected to GFP Trap IP. Precipitated proteins were immunoblotted for GFP, TRAPPC4 and TRAPPC8. Bar chart: the amount of TRAPPC4 isolated was normalised to the amount of GFP-TRAPPC3 precipitated from three independent experiments and plotted ± s.e.m., ns = non-significant, unpaired T-test. \n",
      ".Figure 7A) HEK293A cells treated with transiently transfected with RISC free control siRNA or siRNA duplexes directed against TRAPPC8 and RAB11A and B (knockdown confirmed by immunoblot analysis) were fed with Alexa-647 Transferrin for 15 minutes in full medium, and the fluorescent transferrin chased out for the indicated time periods. At the end of the time course, cells were trypsinised, fixed and the Alexa-647 fluorescence analysed by flow cytometry. Results are plotted as percentage of transferrin fluorescence at t=0 for the RISC free control cells, expressed as the mean of three independent experiments ± s.e.m. B) Cells transiently transfected with RISC Free control siRNA, or siRNA duplexes directed against TRAPPC8 or RAB1A and B were stained for the cis-Golgi markers GM130 or RAB1B and analysed by confocal microscopy. Scale bars = 20 μm in main panels, 10 μm in inset panels. Bar graph: > 100 cells per siRNA were scored for fragmented or normal juxtanuclear Golgi stacks from three independent experiments. Error bars ± s.e.m., * p<0.05, ** p<0.01, one way ANOVA with Sidak's multiple comparison test. C) HeLa C1 cells transiently transfected with RISC free control siRNA or siRNA duplexes directed against TRAPPC8 or RAB1A and B (knockdown confirmed by immunoblot analysis), were treated with D/D solubiliser for the indicated time periods, trypsinised, fixed and their GFP content analysed by flow cytometry. The line graph shows the GFP fluorescence as a percentage of t=0, ± s.e.m. from three independent experiments. \n",
      ".Figure 9A) Cells depleted of ULK1 (siULK1) or not (RISC Free) were transfected with GFP or GFP-TBR (green), starved (EBSS) or not (-EBSS) and stained for ATG9 using hamster anti-ATG9 (red). Arrows indicate transfected cells. Western blot indicates ULK1 knockdown. Scale bars = 20 μm. B) Cells depleted of ULK1 (siULK1) or not (RISC Free) were transfected with myc-RAB1B or myc-RAB1B (S22N) (green), starved (EBSS) or not (-EBSS) and stained for ATG9 using hamster anti-ATG9 (red) Arrows indicate transfected cells. Western blot indicates ULK1 knockdown. Scale bars = 20 μm. C) Model for the role of TBC1D14 and the TRAPP complex in mammalian autophagy and ATG9 traffic. Recycling endosomes harbour a population TBC1D14 bound to RAB11, and a population of ATG9 molecules which may traffic to and from the ATG9 compartment. (i) Upon amino acid starvation, TBC1D14 and RAB11 induce a vesicle trafficking step from RAB11 to RAB1 positive membranes at a tubulo-vesicular transport intermediate (ii). This results in recycling of ATG9 to RAB1-regulated Golgi compartments (iii) from where ATG9 can be trafficked to the ATG9 compartment in an ULK1 dependent manner (iv). This maintains the ATG9 compartment under starvation, which can then both contribute to, and promote autophagosome formation from other membrane sources, including the RAB11 positive recycling endosome (v). \n",
      ".Figure 8A) ATG9 positive membranes were immunoisolated from approximately 2 × 107 fed (F) or starved (S) HEK293A cells using a monoclonal hamster anti-ATG9 (ATG9) or control hamster IgM and immunoblotted for ATG9 using a rabbit polyclonal antibody, TRAPPC8, TRAPPC4 and actin. B) HEK293A cells transfected with GFP or GFP-TBR (green) were stained for ATG9 using hamster anti-ATG9 (red) and analysed by confocal microscopy. Scale bar = 20 μm C) HEK293A cells transiently transfected with RISC Free siRNA or siRNA against TRAPPC8 were stained for RAB1B (green) and ATG9 using hamster anti-ATG9 (red) and imaged using confocal microscopy. D) Stills at t = 30 minutes from time lapse imaging of 9B9 cells (stably expressing GFP-LC3 and mRFP-ATG9) expressing either CFP or CFP-TBR (blue). Scale bars = 20 μm. Inset panels show mRFP-ATG9 and GFP-LC3 contact events at the indicated time points. Bar chart shows number of times a GFP-LC3 spot was within 1 μm of an mRFP-ATG9 spot (a contact event), expressed as total mRFP-ATG9 contacts per GFP-LC3 spot, n = 18 cells per condition pooled from 4 independent experiments. \n",
      ".Figure 1Human first-trimester trophoblast cell line Sw.71 (left panel) and human primary culture (right panel) trophoblast cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media over time. RNA was collected for measuring viral titers by qRT-PCR. Data represent as mean ± SEM; n=4 biological replicates. NT, no treatment group; HPC, human primary culture. (A) ZIKV titers in Sw.71 and HPC trophoblasts. *p < 0.05 by One-way ANOVA. Human first-trimester trophoblast cell line Sw.71 (left panel) and human primary culture (right panel) trophoblast cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media over time. RNA was collected for measuring gene expression by qRT-PCR. Data represent as mean ± SEM; n=4 biological replicates. NT, no treatment group; HPC, human primary culture. (B) IFNα is not highly induced in response to ZIKV infection in trophoblast cells. IFNα mRNA expression was inhibited in the first 24 h.p.i. and maintained at low level during ZIKV infection. Difference not significant (n.s.) by One-way ANOVA. Human first-trimester trophoblast cell line Sw.71 (left panel) and human primary culture (right panel) trophoblast cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media over time. RNA was collected for measuring gene expression by qRT-PCR. Data represent as mean ± SEM; n=4 biological replicates. NT, no treatment group; HPC, human primary culture. (C) ZIKV infection induced a time-dependent increase in IFNβ mRNA expression in Sw.71 and HPC. *p < 0.05, **p < 0.01 by One-way ANOVA. Human first-trimester trophoblast cell line Sw.71 (left panel) and human primary culture (right panel) trophoblast cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media over time. (D) Expression levels of secreted IFNβ detected in trophoblast supernatants by ELLA assay. Supernatants were collected from ZIKV-infected and control trophoblast cultures at different times, and IFNβ protein secretion was quantified by ELLA assay. Note the increase of secreted IFNβ in the ZIKV-infected supernatant in a time-dependent manner. ****p < 0.0001 by Student's t-test against individual time point NT control. \n",
      ".Figure 4(A) Induction of IFNβ mRNA in response to ZIKV infection in Sw.71 and ISG20-/- Sw.71 trophoblast cells. Sw.71 and ISG20-/- Sw.71 cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media for 48 h, and RNA was collected for qRT-PCR. Note that ZIKV infection induces IFNβ mRNA expression in Sw.71 as well as in ISG20-/- Sw.71 cells. Data represent as mean ± SEM; n=3~4 biological replicates; *p < 0.05, **p < 0.01 by Student's t-test. (B) ISGs mRNA expression stimulated by IFNβ in Sw.71 and ISG20-/- Sw.71 trophoblast cells. Sw.71 and ISG20-/- Sw.71 cells were treated with 30 ng/ml IFNβ for 24 h, and RNA was collected for determining the ISGs gene expression by qRT-PCR. Data represent as mean ± SEM; n=3 biological replicates; *p < 0.05, **p < 0.01 by Student's t-test against control. (C) No ISG20 protein expression in ISG20-/- Sw.71 cells after IFNβ treatment. Sw.71 and ISG20-/- Sw.71 cells were treated with 30 ng/ml IFNβ for 24 h, and proteins were collected for western blot analysis. β-actin served as a loading control for western blot. (D) IFNβ pre-treatment significantly prevented trophoblast cells from ZIKV infection, however, this protection was evidently attenuated due to lack of ISG20. Sw.71 and ISG20-/- Sw.71 cells were pre-treated with or without 30 ng/ml IFNβ for 24 h, followed by ZIKV infection (MOI=2) for 1 h and refreshed with regular media for 24 h, and RNA was collected to determine the viral titers by qRT-PCR. Data represent as mean ± SEM; n=4 biological replicates; *p < 0.05 by Student's t-test. \n",
      ".Figure 3(A) ISG20 protein expression was increased in ZIKV infection. Sw.71 cells were infected with ZIKV (MOI=2) over time, and proteins were collected for western blot analysis. β-actin served as a loading control. (B) No ISG20 protein expression in ISG20-/- Sw.71 in ZIKV infection. Sw.71 and ISG20-/- Sw.71 cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media for 48 h, and proteins were collected for western blot analysis. β-actin served as a loading control. (C) Higher ZIKA titer was shown in ISG20-/- Sw.71 cells. Sw.71 and ISG20-/- Sw.71 cells were infected with ZIKV (MOI=2) for 1 h and refreshed with regular media for 48 h, and RNA was collected for determining the viral titer and gene expression by qRT-PCR. Data represent as mean ± SEM; n=4 biological replicates; *p < 0.05 by Student's t-test. (D) More viral shedding in ZIKV-infected ISG20-/- Sw.71 cell culture supernatant. Supernatant from ZIKV-infected Sw.71 and ISG20-/- Sw.71 cells were collected and plaque assay was performed using Vero cells. A representative plaque assay picture is presented. Note that more plaques were formed in Vero cells by incubating with ZIKV-infected ISG20-/- Sw.71 cell culture supernatant. Scale bar=130μm. (E) Plaques were counted and plaque forming-unit (pfu) ratio was calculated for the comparison of the viral titers. Data represent as mean ± SEM; n=3 biological replicates. *p < 0.05 by Student's t-test. \n",
      ".Figure 2Sw.71 (left panel) and HPC cells (right panel) were treated with IFNβ to detect ISG20 mRNA expression. Data represent as mean ± SEM; n=3 biological replicates. NT, no treatment group; HPC, human primary culture. (A) Sw.71 and HPC cells were treated with different doses of IFNβ (3, 30, 300 ng/ml) for 8 h and RNA were collected for determining ISG20 mRNA expressions by qRT-PCR. Note the increase of ISG20 mRNA expression in a dose-dependent manner. *p < 0.05, **p < 0.01 by One-way ANOVA. (B) Sw.71 and HPC cells were treated with 300ng/ml IFNβ over time and RNA was collected for determining ISG20 mRNA expressions by qRT-PCR. Note the increase of ISG20 mRNA expression in a time-dependent manner. *p < 0.05 by One-way ANOVA. (C) Sw.71 (left panel) and HPC cells (right panel) were treated with IFNβ to detect ISG20 protein expression. (C) Sw.71 and HPC cells were treated with different doses of IFNβ (3, 30, 300 ng/ml) for 24 h and protein was collected for determining ISG20 protein expressions by western blot. Note the increase of ISG20 protein expression in a dose-dependent manner. β-actin served as a loading control. ( Sw.71 (left panel) and HPC cells (right panel) were treated with IFNβ to detect ISG20 protein expression. (D) Sw.71 and HPC cells were treated with 300ng/ml IFNβ over time and protein was collected for determining ISG20 protein expressions by western blot. Note that there was no ISG20 protein expression in the no treatment group in both Sw.71 and HPC, only after IFNβ treatment, ISG20 protein expression exhibited a time-dependent manner. β-actin served as a loading control. \n",
      ".Figure 5(A) Structure of the recombinant ISG20-Fc protein. The components are: 1) ISG20 sequence (Pink); 2) Linker (Orange); 3) Fc domain (White); 4) Tev sequence, cleavage between Q and G (Cyan); 5) Signal sequence (ENPP7) (Red); 6) His Tag (Green). (B) Workflow of clone production and selection. (C) Selection and verification of positive clones. In the positive clone, ISG20 protein expressed both in the cytosol (endogenous) and supernatant (secreted) compared to the non-transfected CHO cells and transfected negative clones. CHO, Chinese hamster ovary cell without transfection; β-actin served as a loading control. (D) ISG20 secretion in the positive clone supernatant significantly decreased ZIKV infection in trophoblast cells. Supernatants from negative and positive clones were collected and added to ISG20-/- Sw.71 trophoblast cells together with ZIKA virus (MOI=2) for 1 h, followed by refreshing with new growth media for 48 h. RNA was then collected to determine viral titers by qRT-PCR. Data represent as mean ± SEM; n=3 biological replicates; **p < 0.01 by Student's t-test. (E) Recombinant ISG20-Fc degrades ZIKA viral RNA and HSV2 viral DNA. 50ng purified viral RNA (ZIKV) or DNA (HSV-2) were incubated with increasing concentrations of rISG20-Fc (5, 50, 500 ng) in the presence of RNase inhibitor for 90min at 37°C followed by quantification of viral titers by qRT-PCR, and agarose gel was used to evaluate the RNA degradation by electrophoresis. The representative picture of agarose gel is presented. rISG20-Fc was able to degrade both ZIKV RNA and HSV-2 DNA in a dose dependent manner; however, rISG20-Fc was more efficient in degrading viral RNA than DNA. Data represent as mean ± SEM; n=3 biological replicates; **p < 0.01. ***p < 0.001 and ****p < 0.0001 by One-way ANOVA. \n",
      ".Figure 8When ZIKA virus infects the pregnant woman, the virus can reach the placenta and transmit to the fetus. In the ZIKV replication cycle, it starts with the virus binding to host cell surface receptors, leading to endocytosis of the virus. Internalized viral particles release the viral RNA into the cytoplasm of the host cell and start replication. During its replication, rISG20-Fc degrades ZIKV RNA and inhibits the subsequent transcription and translation. Therefore, less new virus particles are released from the placenta to fetus.  \n",
      ".Figure 6(A) Mouse experiment protocol. Adult (8-12 weeks of age) IFNAR1-/- pregnant mice were infected i.p. with 1*105 pfu ZIKV or 1% FBS DMEM/F12 media (vehicle) on embryo day 8.5 (E8.5). 1 h after ZIKV/vehicle injection, treatment with rISG20-Fc (1mg/kg) or PBS (control) was administrated i.p. to the pregnant mice. On E9.5 and E10.5, same protein/PBS injection was performed on the pregnant mice. On E14.5, the mice were sacrificed and organs were collected for viral titer quantification. N=3 for control PBS group, and n=4 for treatment groups. (B) Macroscopic evaluation of the pregnant uterus and fetus. Compare to the fetal death and reabsorptions in ZIKV-infected mice, rISG20-Fc treatment rescued this phenotype by decreasing pregnancy loss (resorption). (C) Effect of rISG20-Fc treatment on pregnancy loss. rISG20-Fc treatment significantly decreased the pregnancy loss caused by ZIKA infection. The pregnancy loss rate (number of dead fetus + resorptions/ total number of implantations x100) was calculated. Data represent as mean ± SEM; n=3-4 mice per group; *p < 0.05 and **p < 0.01 by One-way ANOVA. (D) rISG20-Fc treatment improves fetal development. Measurement of crown-rump length (CRL) and occipitofrontal diameter (OFD) were determined on E14.5. n=3-4 mice per group and 6-8 fetuses from each pregnant mouse were evaluated; *p < 0.05, **p < 0.01 by One-way ANOVA. \n",
      ".Figure 7(A) rISG20-Fc treatment alleviates maternal viral burden. Note that rISG20-Fc treatment significantly decreased the maternal serum viral titers. Data represent as mean ± SEM; n=3 mice for control PBS group, and n=4 mice for treatment groups; *p < 0.05 by Student's t-test. (B) rISG20-Fc treatment reduces ZIKV titers in the fetal brain. Although there was no difference of ZIKV titer in the placenta, at the fetal side, there was a significant decrease of viral titer in the fetal brain, suggesting rISG20-Fc can block viral transmission from placenta to fetus. Data represent as mean ± SEM; n=4 for each group, and 3-4 placentas/fetal brains from every mouse were analyzed; *p < 0.05 by Student's t-test. ( (C) rISG20-Fc treatment decreases Eotaxin significantly in maternal circulation. Data represent as mean ± SEM; n=2 for PBS control group, and n=3 for every treatment group; *p < 0.05 by One-way ANOVA; difference not significant for IL-9 and IFNγ by One-way ANOVA. (D) Representative hematoxylin and eosin staining of the mouse placenta on E14.5. Labyrinth layers were marked with a solid line. Note the major alteration on placenta structure characterized by multifocal loss of tissue architecture (necrosis) and the defective blood vessel formation of labyrinth in ZIKV+PBS group, and rISG20-Fc treatment improves vascularity and decreases decidual edema and cellular fragmentation at the labyrinth. Scale bar=580μm (upper images); Scale bar=230μm (bottom images). \n",
      " - Interferon signaling in mice bearing heterozygous Stat1Y701F mutation resembles that of human cells with heterozygous Stat1Y701C mutation.Figure 1A Western blot analysis STAT expression and phosphorylation. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F/+ (WT/YF) or Stat1-/- (S1) mice were treated with 250 IU/mL of IFNβ or 5 ng/mL of IFNγ for 0.5, 6, 12 and 24 h. Whole cell extracts were collected and tested in western blot for levels of phosphorylation of STAT1 (Y701) and STAT2 (Y689) and total level of STAT1 and STAT2. The blots are representative of more than 3 independent experiments. B Effect of STAT1Y701Fheterozygosity on the expression of type I IFN-induced genes (ISG). BMDMs of wild type (WT), Stat1Y701F/+ (WT/YF) or Stat1-/- (S1) mice were treated with 250 IU/mL of IFNβ for 4 and 48 h. Gene expression was measured by Q-PCR and normalized to Gapdh and to the expression levels in untreated wild type cells. The bars represent mean values with the standard deviations (SD) of three independent experiments.C Effect of STAT1Y701Fheterozygosity on the expression of IFNγ-induced genes. BMDMs of wild type (WT), Stat1Y701F/+ (WT/YF) or Stat1-/- (S1) mice were treated with 5 ng/mL of IFNγ for 4 and 48 h. Gene expression was measured by Q-PCR and normalized to Gapdh and to the expression levels in untreated wild type cells. The bars represent mean values with the standard deviations (SD) of three independent experiments. \n",
      " - STAT1Y701F mutant reduces IFNβ-stimulated nuclear translocation of STAT2.Figure 4A Analysis of STAT2 nuclear translocation by immunofluorescence. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F (YF), Stat1-/- (S1), Stat2-/- (S2) and IRF9-/- (IRF9) mice were seeded on cover slips and stimulated with 250 IU/mL of IFNβ for 30 min or 24 h. The cells were fixed and stained for STAT2 specific antibody followed by Alexafluor® 488 conjugated secondary antibody (green). Nuclei were stained with DAPI (blue). Studies are representative of more than three independent experiments. The scale bars represent 10 µm.B Quantitative evaluation of STAT2nuclear translocation. The intensity of STAT2-dependent immunofluorescence over DNA staining (DAPI) was quantified using ImageJ software in 20 cells from two independent experiments. Bars represent a mean with standard deviation (SD) and asterisks denote the level of statistical significance (***, p≤ 0.001); p-value was calculated using unpaired t-test. \n",
      " - Presence of the STAT1Y701F mutant reduces IFNβ-stimulated binding of STAT2 to nuclear ISRE sequences.Figure 3A IFNβ-stimulated binding of STAT2 to ISRE sequences of Mx2 and IRF7 promoters. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F (YF), Stat1-/- (S1), Stat2-/- (S2) and IRF9-/- (IRF9) mice were treated with 250 IU/mL of IFNβ for 2 or 24 h. Cells were crosslinked, sonicated and immunoprecipitated with STAT2-specific antibody. The amount of precipitated DNA was measured by Q-PCR. Bars represent a mean value of 3 independent experiments. Error bars represent standard error of the mean (SEM); asterisks denote the level of statistical significance (**, p≤ 0.01); the p-values were calculated using paired ratio t-test. B Impact of STAT2 deficiency on IFNβ -stimulated STAT1 association with the Mx2 ISRE. BMDMs of wild type (WT), Stat1-/- (S1) and Stat2-/- (S2) mice were treated with 250 IU/mL of IFNβ for 2 or 24 h. Cells were crosslinked, sonicated and immunoprecipitated with STAT1-specific antibody. The amount of precipitated DNA was measured by Q-PCR. Bars represent mean values of three independent experiments; error bars represent standard error of mean (SEM). C Simultaneous association of STAT1 and STAT2 with the Mx2 ISRE analyzed by ChIP-reChIP. BMDMs of wild type (WT) mice were treated with 250 IU/mL of IFNβ for 2 or 24 h. Cells were crosslinked, sonicated and immunoprecipitated with either STAT1-specific antibody and re-immunoprecipitated with STAT2-specific antibody or vice versa. The amount of precipitated DNA was measured by Q-PCR. Bars represent mean values of three independent experiments; error bars represent standard deviation (SD). D Impact of STAT1Y701F on delayed, STAT2-mediated expression of IFN-induced genes. Stat1-/- fibroblasts were transfected with plasmids driving expression of the indicated proteins. 24 h after transfection, 250 IU/ml of IFNβ was added to the transfected cells and ISG expression was determined by Q-PCR after 48h of cytokine treatment. Gene expression was measured by Q-PCR and normalized to Gapdh. Bars represent a mean value of 3 independent experiments. Error bars represent standard error of the mean (SEM) and asterisks denote the level of statistical significance (*, p≤ 0.05); the p-values were calculated using paired ratio t-test. \n",
      " - STAT1Y701F mutant counteracts the inhibition of L.Figure 5A Legionella pneumophila growth in unstimulated macrophages. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F, Stat1-/- mice were seeded on in the 24-well plates and infected with L. pneumophila (JR32 Fla-, MOI 0.25). The numbers of colony forming units (CFUs) were determined 24 h, 48 h and 72 h after infection on charcoal yeast extract plates (CYE). The 0 time point was collected 1.5 h after the infection.B Legionella pneumophila growth in IFNβ-treated macrophages. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F, Stat1-/- mice were seeded in 24-well plates, treated with 500 U/mL of IFNβ and then infected with L. pneumophila (JR32 Fla-, MOI 0.25). The numbers of colony forming units (CFUs) were determined 24 h, 48 h and 72 h after infection on charcoal yeast extract plates (CYE). The 0 time point was collected 1.5 h after the infection. \n",
      " - STAT1 expression and interferon signaling in Stat1Y701F mice.Figure 2A Effect of STAT1Y701F homozygosity on STAT1 expression in mouse cells and organs. Spleens, livers or bone marrow derived macrophages (BMDMs) were isolated from wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice. Whole cell extracts were collected and tested for levels of total Stat1 in western blot. The blots are representative of more than 3 independent experiments. B Effect of STAT1Y701F homozygosity on STAT1 phosphorylation at Y701. BMDMs were isolated from wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice and stimulated for 30 min with 250 IU/mL of IFNβ or 5 ng/mL of IFNγ. Whole cell extracts were collected and tested for levels of STAT1 phosphorylation on Y701 in western blot. The blots are representative of more than 3 independent experiments. C Effect of STAT1Y701Fhomozygosity on the expression of IFNβ-induced genes. BMDMs of wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice were treated with 5 ng/mL of IFNγ for 4 or 48 h. Gene expression was measured by Q-PCR and normalized to Gapdh and to the expression levels in untreated wild type cells. Bars represent a mean value of 3 independent experiments. Error bars represent standard error of the mean (SEM); asterisks denote the level of statistical significance (ns, p>0.05); the p-values were calculated using paired ratio t-test. D Effect of STAT1Y701Fhomozygosity on the expression of type I IFN-induced genes (ISG). BMDMs were isolated from wild type (WT), Stat1Y701F, Stat1-/-, Stat2-/- and IRF9-/- mice treated with 250 IU/mL of IFNβ for 4, 8, 12, 24 or 48 h. Gene expression was measured by Q-PCR and normalized to Gapdh and to the expression levels in untreated wild type cells. Bars represent a mean value of 3 independent experiments. Error bars represent standard error of the mean (SEM); asterisks denote the level of statistical significance (ns, p>0.05; *, p≤ 0.05; **, p≤ 0.01); the p-values were calculated using paired ratio t-test. E STAT1 and STAT2 phosphorylation in Stat1-/-, Stat1Y701F, Stat2-/- and Irf9-/- macrophages. BMDMs were isolated from wild type (WT), Stat1Y701F (YF), Stat1-/- (S1), Stat2-/- (S2) and IRF9-/- (IRF9) mice and treated with 250 IU/mL of IFNβ for 30 min or 6, 12 or 24 h. The whole cell extracts were collected and tested in western blot for levels of phosphorylation of STAT1 on Y701 and of STAT2 on Y689. The same cell extracts were tested for total levels of STAT1 and STAT2. The blots are representative of more than 3 independent experiments. \n",
      " - Interferon signaling in L.Figure 6A Western blot analysis of STAT1 tyrosine phosphorylation. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice were infected with L. monocytogenes (LO28, MOI 10) for 5 or 6 h. Whole cell extracts were collected and tested in western blot for levels of total STAT1 and phosphorylation of STAT1 on tyrosine (Y701). The blots are representative of more than 3 independent experiments. B Impact of Stat1Y701F mutation, or of deletion of ISGF3 subunits on the expression of the IFNβ gene. BMDMs of wild type (WT), Stat1Y701F, Stat1-/-, Stat2-/- and IRF9-/- mice were infected with L. monocytogenes (LO28, MOI 10) for 4, 8, 12, 24 or 48 h. Levels of Ifnβ gene expression were determined by Q-PCR. Bars represent mean values of three independent experiments. Error bars represent standard error of mean (SEM). \n",
      " - Liver inflammation in Stat1Y701F and Stat1-/- mice after infection with Listeria monocytogenes.Figure 8A Immunohistochemical analysis of infection and inflammatory infiltrates. Wild type (WT), Stat1Y701F and Stat1-/- mice were infected by intraperitoneal injection of 1x102 viable L. monocytogenes for 48 h. Liver sections were examined by immunohistochemistry with L. monocytogenes or Ly6C/Ly6G specific antibody. The scale bars on 20x magnification images represent 100 µm and 50 µm on the 63x magnification images.B Quantitative evaluation of inflammatory infiltrates. Infiltrates representing the entire surface of sections from five animals per genotype were counted and categorized according to their size. C Liver pathology in infected mice. Wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice were infected by intraperitoneal injection of 1x105 viable L. monocytogenes for 72 h. Serum was collected and tested for ALT activity. Lines represent the median and asterisks denote statistically significant differences (*, p≤ 0.05; ***, p≤ 0.001); p-values were calculated using unpaired t-test. \n",
      " - STAT1Y701F contributes to clearance of L.Figure 7A Impact of Stat1 deficiency or of STAT1Y701F mutation on the growth of Listeria monocytogenes in macrophages. Bone marrow derived macrophages (BMDMs) of wild type (WT), Stat1Y701F and Stat1-/- mice were infected with L. monocytogenes (LO28, MOI 10). Colony forming unit (CFU) numbers were determined 1, 2, 4, 6 or 8 h after infection by plating on brain-heart-infusion (BHI) agar plates. The graph represents biological triplicates and the data are represented as mean values. Error bars represent standard deviation (SD) and asterisks denote statistically significant differences (ns, p>0.05; **, p≤0.01; ***, p≤ 0.001); p-values were calculated using unpaired t-test. B Survival of mice infected with L. monocytogenes. 10 wild type (WT), Stat1Y701F and Stat1-/- mice per group were infected by intraperitoneal injection of 1x 102 viable L. monocytogenes. Survival was monitored over 10 days. The study is representative of more than 3 independent experiments. C Organ pathogen burdens of mice infected with L. monocytogenes. Wild type (WT), Stat1Y701F (YF) and Stat1-/- (S1) mice were infected by intraperitoneal injection of 1x 102 viable L. monocytogenes. Number of colony forming units (CFU) in organs was determined at 48 h, 72 h or at the terminal stage of infection by plating homogenates on brain-heart-infusion (BHI) agar plates or Oxford agar plates (for lungs). Dots represent pooled data of 3 independent experiments. Lines represent the median and asterisks denote statistically significant differences (ns, p>0.05; *, p≤ 0.05; **, p≤ 0.01; ***, p≤ 0.001; ****, p≤ 0.0001); p-values were calculated using Mann-Whitney test. \n",
      " with an increased nuclear translocation in vitro.Figure 1(A, B) Overexpression of wildtype full-length human tau (hTau, also termed tau441 or tau40 or tau2N4R) induced significant alterations of 9 transcription factors screened by using Transcription Factors Activation Profiling Plate Array II, in which 96 transcription factors (Appendix Table S1 and S2) were monitored. The empty vector was transfected as a control (Ctrl). (C-F) Expression of hTau (probed by HT7) increased total and the phosphorylated STAT1 at Tyr701 (pY-STAT1) in cell whole extracts (C, D) and the nuclear fraction (E, F) measured by Western blotting (n=4). Data information: Data were presented as mean ± SD (Mann-Whitney test). *, p<0.05 vs Ctrl. (G) The representative immunofluorescent images and quantitative analysis show significantly increased STAT1 signal in the nuclear fraction of HEK293 cells with overexpression of hTau compared with the empty vector control (eGFP) (n=5). Scale bar, 10 μm. Data information: Data were presented as mean ± SD (Mann-Whitney test). , **, p<0.01 vs Ctrl. (H) Overexpression of hTau most significantly increased STAT1 monomer and dimer formation in nuclear fraction (Nu) measured by Western blotting. (I) Overexpression of hTau increased STAT1 activity in HEK293 cells detected by luciferase assay (n=4). Data information: Data were presented as mean ± SD (Mann-Whitney test). *, p<0.05 , ***, p<0.001 vs Ctrl. (J) Overexpression of hTau increased STAT1-DNA binding activity in HEK293 cells measured by electrophoretic mobility shift assay (EMSA). *, indicates STAT1/DNA complex. \n",
      " in vivo.Figure 2(A, B) AAV-hTau-eGFP (AAV-hTau) or the empty vector AAV-eGFP (1.13×1013 v.g./ml) was stereotaxically injected into hippocampal CA3 of 3-month-old C57 mice. After one month, the increased levels of STAT1 and pY-STAT1 in hippocampal total extracts and the nuclear fraction were detected in hTau group by Western blotting (n=6, Mann-Whitney test). Data information: Data were presented as mean ± SD. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP, wt or Ctrl. (C, D) The increased STAT1 and pY-STAT1 in hippocampal total extracts and the nuclear fraction of 12-month-old hTau transgenic mice measured by Western blotting (n=4, Mann-Whitney test). Data information: Data were presented as mean ± SD. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP, wt or Ctrl. (E, F) The representative images of STAT1 and pY-STAT1 in the brain of AD patients probed by co-immunohistochemical staining and quantitative analysis (hematoxylin stains nuclei, purple; DAB stains the target proteins, brown; n=5-6 slices). Arrowheads indicated typical nuclear staining of STAT1/pY-STAT1. Data information: Data were presented as mean ± SD. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP, wt or Ctrl. (G) The increased AT8 (pS202/pT205), STAT1 and pY-STAT1 in cortex total extracts of AD patients measured by Western blotting (n=3, Student's t test). Data information: Data were presented as mean ± SD. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP, wt or Ctrl. (H) STAT1 mRNA was analyzed by qPCR in AAV-hTau or AAV-eGFP infected hippocampal tissues (n=6, Mann-Whitney test). Data information: Data were presented as mean ± SD. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP, wt or Ctrl. \n",
      "Highlighted TALEN-TBC1D15-L and TALEN-TBC1D15-R indicate the TALEN binding pair that targets exon 9 of the TBC1D15 gene.Figure 1--figure suppleme\n",
      "TBC1D15 is dispensable for mitochondria and peroxisome morphologies.Figure 1.(A) Total cell lysates prepared from the indicated HCT116 cell lines were analyzed by immunoblotting. For comparison, different amounts of proteins (1:3 ratio) were applied. An asterisk indicated non-specific crossreactive bands. (B) The indicated cell lines were analyzed by immunofluorescence microscopy using anti-Cytochrome c antibody for mitochondria and anti-PMP70 antibody for peroxisome staining. Images are displayed as z-stacks of 6 confocal slices. Magnified images are also shown for mitochondrial morphologies. Scale bars, 10 μm. (C) Quantification of mitochondrial morphologies in (B). Percentages of cells harboring fragmented, tubular, or elongated mitochondria are shown. Tubular and elongated denote normal tubular mitochondria seen in WT cells and highly connected mitochondrial network, respectively. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each of three replicate wells. \n",
      " in mice.Figure 4(A, B) Overexpression of AAV-hTau decreased the protein levels of GluN1, GluN2A and GluN2B detected by Western blotting in the hippocampal CA3 of C57 mice, compared with the AAV-eGFP vector control. (C, D) Simultaneous downregulation of STAT1 by infusing AAV-Cre in hippocampal CA3 of STAT1flox/flox mice abolished the hTau-induced inhibition in expression of NMDARs protein. Data information: Data were presented as mean ± SD (n=4; Mann-Whitney test). *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau. (E, F) Overexpression of AAV-hTau or simultaneous downregulation of STAT1 changed the mRNA levels of GluN1, GluN2A and GluN2B detected by qRT-PCR in the hippocampal CA3. Data information: Data were presented as mean ± SD (n=4; Mann-Whitney test). *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau. \n",
      " to NMDAR promoters and inhibits the expression of NMDARs.Figure 5(A) Overexpression of AAV-hTau increased binding of STAT1 to the promoter regions of GluN1, GluN2A and GluN2B gene in hippocampal CA3 extracts measured by chromatin immunoprecipitation assay (ChIP) (n=3 from three independent experiments). Data information: Data were presented as mean ± SD (two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs Ctrl; ###, p<0.001 vs wildtype reporters. (B) Overexpression of hTau or wildtype STAT1 (WT-STAT1) inhibits the transcription activity of NMDARs compared with the empty vector control (Ctrl) measured by luciferase activity assay in HEK293 cells (n=3 from three independent experiments). Data information: Data were presented as mean ± SD (two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs Ctrl; ###, p<0.001 vs wildtype reporters. Diagrams show the predicted GAS promoter element (GASs) for STAT1 in the promoter (-2000-+299bp) of GluN1 (C) The GASs or the mutant (MUT) plasmids were co-transfected respectively with WT-STAT1 or its empty vector (Ctrl) into HEK293 cells for 24 h, and then the luciferase activity was measured (right panels). N=4 for each group. Data information: Data were presented as mean ± SD (two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs Ctrl; ###, p<0.001 vs wildtype reporters. Diagrams show the predicted GAS promoter element (GASs) for STAT1 in the promoter (-2000-+299bp) of GluN2B (E) The GASs or the mutant (MUT) plasmids were co-transfected respectively with WT-STAT1 or its empty vector (Ctrl) into HEK293 cells for 24 h, and then the luciferase activity was measured (right panels). N=4 for each group. Data information: Data were presented as mean ± SD (two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs Ctrl; ###, p<0.001 vs wildtype reporters. Diagrams show the predicted GAS promoter element (GASs) for STAT1 in the promoter (-2000-+299bp) of GluN2A (G, I). The GASs or the mutant (MUT) plasmids were co-transfected respectively with WT-STAT1 or its empty vector (Ctrl) into HEK293 cells for 24 h, and then the luciferase activity was measured (right panels). N=4 for each group. Data information: Data were presented as mean ± SD (two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs Ctrl; ###, p<0.001 vs wildtype reporters. \n",
      " activation mediates hTau-induced STAT1 activation.Figure 6(A, B) Overexpression of hTau in HEK293 cells for 48 h increased the activity-dependent phosphorylation of JAK2, JNK1 and ERK1 compared with the empty vector control (Ctrl) measured by Western blotting (n=3, student's t test). Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. Pharmacological inhibition of ERK1 (C, D) for 24 h did not significantly affect the hTau-induced STAT1 phosphorylation at pY-STAT1 (Tyr701) in total extracts (C and the nuclear fraction (D measured by Western blotting (n=3). Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. Pharmacological inhibition of JNK1 (E, F) for 24 h did not significantly affect the hTau-induced STAT1 phosphorylation at pY-STAT1 (Tyr701) in total extracts E) and the nuclear fraction F) measured by Western blotting (n=3). The alteration of pS-STAT1 (Ser727) confirms the efficacy of JNK1 inhibitors. Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. Pharmacological inhibition of JAK2 (G, H) abolished hTau-induced STAT1 phosphorylation at Tyr701 in total extracts (G and the nuclear fraction (H (n=3). Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. knockdown JAK2 by siRNA (I, J) abolished hTau-induced STAT1 phosphorylation at Tyr701 in total extracts I) and the nuclear fraction J) (n=3). Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. (K, L) The phosphorylated JAK2 level increased in the hippocampus of 12-month-old hTau transgenic mice (K), and the hippocampus of C57 mice infected with AAV-hTau (1.13×1013 v.g./ml) (L) (n=4, Mann-Whitney test). Data information: Data were presented as mean ± SD, *, p<0.05 vs Ctrl, eGFP or WT. \n",
      " ameliorates hTau-induced cognitive and synaptic impairments.Figure 3AAV-Cre (5×1012 v.g./ml) mixed with AAV-hTau or AAV-eGFP (1.13×1013 v.g./ml) were stereotaxically injected into the hippocampal CA3 of 3-month-old STAT1flox/flox mice. One month later, downregulation of STAT1 was confirmed by immunohistochemical staining. AAV-Cre (5×1012 v.g./ml) mixed with AAV-hTau or AAV-eGFP (1.13×1013 v.g./ml) were stereotaxically injected into the hippocampal CA3 of 3-month-old STAT1flox/flox mice. One month later, downregulation of STAT1 was confirmed by Western blotting (C) Downregulation of STAT1 ameliorated hTau-induced spatial learning deficit shown by the decreased escape latency during 5 consecutive days training in Morris water maze (MWM) test (n=9-11 for each group). Data information: Data were presented as mean ± s.e.m. (two-way repeated measures analysis of variance (ANOVA) followed by Bonferroni's post hoc test for C *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP; #, p<0.05, ###, p<0.001 vs hTau. (D-G) Downregulation of STAT1 ameliorated hTau-induced spatial memory deficit shown by the decreased latency to reach the platform quadrant (D), the increased crossing time in the platform site (E) and time spent in the target quadrant (F) measured at day 6 by removed the platform in MWM test; no motor dysfunction was seen (G) (n=9-11 for each group). Data information: Data were presented as mean ± s.e.m. , one-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test for others). *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP; #, p<0.05, ###, p<0.001 vs hTau. (H) Downregulation of STAT1 ameliorated hTau-induced contextual memory deficits measured at 24 h during contextual fear conditioning test (n=8 each group). Data information: Data were presented as mean ± s.e.m. *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP; #, p<0.05, ###, p<0.001 vs hTau. (I-L) One month after the virus infection, paired-pulse ratio (PPR) was recorded in hippocampal CA3 of hTau or STAT1 knockdown mice (I). The IO curve of fEPSP recorded on acute hippocampal slices (J). The slope of fEPSP after HFS recorded on hippocampal slices of hTau or STAT1 knockdown mice (K). LTP magnitude was calculated as the average (normalized to baseline) of the responses recorded 40-60 min after conditioning stimulation. (L). (n=5 slices from 4 mice for each group). Data information: Data were presented as mean ± SD two-way analysis of variance (ANOVA) followed by Bonferroni's post hoc test for I-K, one-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test for others). *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP; #, p<0.05, ###, p<0.001 vs hTau. (M-P) One month after the virus infection, whole cell patch clamp was used to measure the function of NMDA (at +40 mV) and AMPA (at -70 mV) receptors on acute brain slices (400 μm). The insets show representative sample traces of EPSCs in virus infected neurons (M). The reduced NMDA and unchanged AMPA currents with a reduced NMDA/AMPA ratio were seen in hTau infected neurons, while knockdown of STAT1 restored the hTau-induced NMDA currents (N-P). (n=12 neurons from 4 animals for eGFP group; n=11 neurons from 4 animals for hTau group; n=13 neurons from 4 animals for hTau+CRE group). Data information: Data were presented as mean ± SD one-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test for others). *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP; #, p<0.05, ###, p<0.001 vs hTau. \n",
      " activation rescues hTau-induced synapse and memory impairments.Figure 7AAV-eGFP (eGFP) or AAV-hTau-eGFP (hTau) (1.13×1013 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) plus hTau was stereotaxically injected into hippocampal CA3 of 3-month-old C57 mice. After one month, learning and memory were detected by MWM test. (A) The representative fluorescence image confirms expression of AAV-hTau and AAV-Y701F-STAT1. Scale bar, 200 μm, or 100 μm for the enlarged. AAV-eGFP (eGFP) or AAV-hTau-eGFP (hTau) (1.13×1013 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) plus hTau was stereotaxically injected into hippocampal CA3 of 3-month-old C57 mice. After one month, learning and memory were detected by MWM test. (B) Overexpression of Y701F-STAT1 mitigated hTau-induced spatial learning deficits shown by the decreased escape latency during water maze training (n=7-10 each group). Data information: Data were presented as mean ± s.e.m (two-way repeated measures analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. AAV-eGFP (eGFP) or AAV-hTau-eGFP (hTau) (1.13×1013 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) plus hTau was stereotaxically injected into hippocampal CA3 of 3-month-old C57 mice. After one month, learning and memory were detected by MWM test. (C-E) Overexpression of Y701F-STAT1 mitigated hTau-induced spatial memory impairment shown by the decreased latency to reach the platform (C), the increased crossing time in the platform site (D) and time spent in the target quadrant (E) measured at day 6 by removed the platform (n=7-10 each group). Data information: Data were presented as mean ± s.e.m , two-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. AAV-eGFP (eGFP) or AAV-hTau-eGFP (hTau) (1.13×1013 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) or AAV-Y701F-STAT1 (5×1012 v.g./ml) plus hTau was stereotaxically injected into hippocampal CA3 of 3-month-old C57 mice. After one month, learning and memory were detected by MWM test. (F) Expression of Y701F-STAT1 did not change the swimming speed of the mice in water maze task (n=7-10 each group). Data information: Data were presented as mean ± s.e.m two-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. (G-I) Simultaneous expression of Y701F-STAT1 did not induced any further change on basal synaptic transmission (I/O curve) compared with expression of hTau alone, recorded in hippocampal CA3 (G). LTP magnitude was calculated as the average (normalized to baseline) of the responses recorded 40-60 min after conditioning stimulation. (I). (n=5 slices from 4 mice for each group). Data information: Data were presented as mean ± SD , two-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. Simultaneous expression of Y701F-STAT1 rescued the hTau-induced suppression of NMDARs protein expression measured by Western blotting in hippocampal CA3 of C57 mice (n=4). Simultaneous expression of Y701F-STAT1 rescued the hTau-induced suppression of NMDARs protein (K) expression measured by Western blotting in hippocampal CA3 of C57 mice (n=4). Data information: Data were presented as mean ± SD two-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. Simultaneous expression of Y701F-STAT1 rescued the hTau-induced suppression of NMDARs mRNA expression measured by qRT-PCR in hippocampal CA3 of C57 mice (n=4). Data information: Data were presented as mean ± SD , two-way analysis of variance (ANOVA) followed by Bonferroni' s post hoc test *, p<0.05, **, p<0.01, ***, p<0.001 vs eGFP or hTau; #, p<0.05, ##, p<0.01 vs hTau. \n",
      "Loss of Fis1 or TBC1D15 impedes clearance of damaged mitochondria.Figure 4.(A and B) YFP-Parkin stably expressing cells were treated with valinomycin for indicated times. Total cell lysates were subjected to immunoblotting. I and II denote cytosolic and lipidated LC3B, respectively. An asterisk indicates ubiquitinated TOMM20. (C) Indicated protein amounts as in (B) were quantified. The amount of protein without valinomycin treatment was set to 100%. The error bars represent ±SD from three independent experiments. (D) YFP-Parkin stably expressing WT HCT116 cells were treated with or without valinomycin for 40 hr. Total cell lysates were analyzed by immunoblotting. \n",
      "Tubular LC3 expands along microtubules.Figure 3.(A) The indicated cells stably expressing YFP-LC3 and mCherry-Parkin were treated with valinomycin for 3 hr followed by immunofluorescence microscopy with anti-Cytochrome c antibody. Confocal images were acquired as z-stacks comprising 6 sequential sections with 0.8 μm z-intervals. Scale bars, 10 μm. (B) TBC1D15−/− cells prepared as in (A) were subjected to immunoelectron microscopy with anti-GFP antibody. White arrowheads indicate YFP-LC3-labeled tubules. High magnification image is shown in the lower panel. Scale bars, 500 nm. (C) The indicated cells prepared as in (A) were subjected to immunostaining with anti-Tubulin antibody. YFP-LC3 and Tubulin staining are merged in the right panels. Magnified images are shown for TBC1D15−/− cells. Scale bars, 10 μm. (D) The indicated cells stably expressing YFP-LC3 and mCherry-Parkin were treated with valinomycin in the presence or absence of nocodazole for 3 hr. YFP-LC3 and mCherry-Parkin images are merged in the right panels. Scale bars, 10 μm. (E) YFP-LC3 morphologies of cells in (D) were quantified. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each well. \n",
      "Loss of Fis1 or TBC1D15 causes LC3 accumulation during mitophagy.Figure 2.(A) The indicated cell lines stably expressing YFP-LC3 and mCherry-Parkin were treated with valinomycin for 3 hr and subjected to confocal immunofluorescence microscopy with anti-TOMM20 antibody. Scale bars, 10 μm. (B) Quantification of mCherry-Parkin translocation to mitochondria after 3 hr of valinomycin treatment. Partial or complete translocation to mitochondria in each cell was scored as separate phenotypes. Partial and complete denote that Parkin translocates to some of or all mitochondria, respectively. The error bars represent ±SD from three independent experiments. Over 100 cells were counted in each of three separate wells. (C) YFP-LC3 morphologies in (A) were quantified. Percentages of cells harboring diffuse, punctate or accumulated YFP-LC3 are shown. The error bars represent ±SD from three independent replicates. Over 100 cells were counted in each replicate. For the criteria of LC3 morphology, see Figure 2-figure supplement 1. (D) YFP-LC3 and mCherry-Parkin stably expressing TBC1D15−/− cells in the absence or presence of HA-tagged TBC1D15 WT or Δ221-250 mutant were treated with valinomycin for 3 hr. Cells were subjected to immunofluorescence microscopy with anti-HA antibody. Scale bars, 10 μm. (E) The YFP-LC3 morphology of cells in (D) was quantified. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each well. (F and G) YFP-LC3 and mCherry-Parkin stably expressing WT (F) and TBC1D15−/− (G) cells were treated with valinomycin for 3 hr and then subjected to immunoelectron microscopy with anti-GFP antibody. The square in panel a shows enlarged areas in panel b Scale bars, 500 nm. \n",
      "Effect of mitochondrial fission and starvation induction on LC3 accumulation in WT, FIS1−/− and TBC1D15−/− cells.Figure 5.(A) WT, MFF−/− or DRP1−/− cells stably expressing YFP-LC3 and mCherry-Parkin were treated with valinomycin for 3 hr and subjected to immunofluorescence microscopy with anti-TOMM20 antibody. Scale bars, 10 μm. (B) Quantification of mCherry-Parkin translocation to mitochondria after 3 hr of valinomycin treatment. Partial and complete denote that Parkin translocates to some of and all mitochondria, respectively. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each replicate. (C) YFP-LC3 morphologies of cells in (A) were quantified. Percentages of cells harboring diffuse, punctate or accumulated YFP-LC3 are shown. The error bars represent ±SD from three independent replicates. Over 100 cells were counted in each replicate. (D) WT, FIS1−/−, and TBC1D15−/− cells stably expressing YFP-LC3 were grown in starvation media. Z-stacks of confocal images are shown. Scale bars, 20 μm. (E) The number of YFP-LC3 dots in cells under growth or starvation conditions was quantified. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each well. (F) Total cell lysates from cells grown in normal or starvation media for 6 hr were subjected to immunoblotting. LC3-I and LC3-II denote cytosolic and lipidated LC3B, respectively. \n",
      "Identification of TBC1D17 as a Fis1 and TBC1D15 binding protein.Figure 6.(A) TBC1D15−/− cells and those stably expressing HA-TBC1D15 WT and HA-TBC1D15 (Δ221-250), and FIS1−/− cells stably expressing HA-TBC1D15 WT were subjected to immunostaining with anti-TOMM20 and anti-HA antibodies. Scale bars, 10 μm. (B) HA-TBC1D15 WT or HA-TBC1D15 (Δ221-250) with or without Fis1 was transiently overexpressed (OE) in HeLa cells. Cells were subjected to immunostaining with anti-HA and anti-Fis1 antibodies. Scale bars, 20 μm. (C) YFP-TBC1D17 together with pcDNA vector or Fis1 was transiently overexpressed (OE) in HeLa cells. Cells were subjected to immunostaining with anti-Fis1 and anti-Cytochrome c antibodies. Scale bars, 20 μm. (D) YFP or YFP-Fis1 was co-overexpressed with HA-TBC1D15 or HA-TBC1D17 in HEK293 cells. The cell extracts were subjected to pull down assays with GFP-Trap. 5% input and bound fractions were analyzed by immunoblotting with anti-HA (upper panel) and anti-GFP (lower panel) antibodies. (E) YFP, YFP-TBC1D15, or YFP-TBC1D17 was co-overexpressed with HA-TBC1D15 (upper panel) or HA-TBC1D17 (lower panel) in HEK293 cells. The cell extracts were subjected to pull down assays with GFP-Trap. 5% input and bound fractions were analyzed by immunoblotting with anti-GFP and anti-HA antibodies. (F) HA-TBC1D15 (Δ221-250) together with Fis1 and YFP-TBC1D15 WT or YFP-TBC1D17 WT were transiently overexpressed (OE) in HeLa cells. Cells were subjected to immunostaining with anti-HA and anti-Fis1 antibodies. Images of HA and Fis1 staining were merged in the right panels. Scale bars, 20 μm. (G) Schematic model of Fis1, TBC1D15, and TBC1D17 binding. Homo- or hetero-dimer of TBC1D15 can interact with Fis1 dimer on the mitochondrial outer membrane (OMM). \n",
      "TBC1D15 binds ATG8 family proteins.Figure 7.(A) The indicated cells transiently expressing YFP-GABARAPL1 (green) and mCherry-Parkin (red) were treated with valinomycin for 3 hr. Scale bars, 10 μm. (B) YFP-TBC1D15 overexpressed in HEK293 cells was subjected to binding assays with GST-fused proteins (GABA, L1, and L2 represent GABARAP, GABARAPL1, and GABARAPL2, respectively). 5% input and bound fractions were analyzed by immunoblotting with anti-GFP antibody (upper panel). Coomassie brilliant blue (CBB) staining shows GST-fusion proteins in bound fractions (lower panel). (C) Binding assay carried out as in (B) with GST-GABARAPL1 WT (GST-WT) or its Y49A/L50A mutant (GST-YL). Immunoblotting with anti-GFP antibody (upper panel) and CBB staining (lower panel) are shown. (D) Cell extracts from HEK293 overexpressed HA-TBC1D15 and YFP, YFP-GABARAPL1 (YFP-WT), or its Y49AL50A mutant (YFP-YL) were subjected to pull down assays with GFP-Trap. 5% input and bound fractions were analyzed by immunoblotting with anti-HA (upper panel) and anti-GFP (lower panel) antibodies. (E-H) The indicated YFP-tagged TBC1D15 full-length, truncated, or point-mutant protein or YFP-Fis1 overexpressed in HEK293 cells were subjected to binding assays with recombinant GST-GABARAPL1. 5% input and bound fractions were analyzed by immunoblotting with anti-GFP antibody (upper panel) and CBB staining (lower panel). (I) Summary of binding abilities of truncated or point-mutated TBC1D15 constructs. -, +, ++, and +++ indicates binding of recombinant GST-GABARAPL1 to less than 1%, 1-5%, 5-10%, and over 10%, respectively of the total YFP-TBC1D15 fragment. Yellow boxes indicate YFP tags. (J) LC3-LC3 and mCherry-Parkin stably expressing TBC1D15−/− cells in the presence of HA-tagged TBC1D15 WT or F280A mutant were treated with valinomycin for 3 hr. Cells were subjected to immunofluorescence microscopy with anti-HA antibody. Scale bars, 10 μm. (K) The YFP-LC3 morphology of cells in (J) was quantified. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each well. \n",
      "Rab7 is involved in autophagosome fusion during mitophagy.Figure 8.(A) Total cell lysates from HCT116 treated with control (NTC) or RAB7A siRNA were analyzed by immunoblotting. (B) The indicated cells stably expressing YFP-LC3 (green) and mCherry-Parkin were treated with control (NTC) or Rab7_#5 siRNA. After 3 hr valinomycin treatment, cells were analyzed by immunofluorescence microscopy with anti-TOMM20 antibody (red). Z-stacks of confocal images are shown. Magnified images are also shown. Scale bars, 10 μm. (C) YFP-LC3 morphologies of cells in (B) were quantified. Percentages of cells harboring diffuse, punctuate or accumulated/tubulated YFP-LC3 are shown. Data and error bars were obtained from at least 50 cells in each of three independent replicates. (D) The indicated cells stably expressing YFP-LC3 (green), mCherry-Parkin, and 2HA-Rab7 (Red) were treated with or without valinomycin for 3 hr and analyzed by immunofluorescence microscopy with anti-HA antibody. Magnified images are also shown. Scale bars, 10 μm. (E) YFP-LC3 and mCherry-Parkin stably expressing TBC1D15−/− cells in the presence of HA-tagged TBC1D15 WT or the D397A mutant were treated with valinomycin for 3 hr. Cells were subjected to immunofluorescence microscopy with anti-HA antibody. Scale bars, 10 μm. (F) The YFP-LC3 morphology of cells in (E) was quantified. The error bars represent ±SD from three independent replicates. Over 50 cells were counted in each replicate. \n",
      "Effects of FDA-approved drugs on pseudotyped SARS-CoV-2 viral entry in Huh-7 cells versus human Airway Chips.Fig. 4.(A) Graphs showing the inhibitory effects of chloroquine, arbidol, toremifene, clomiphene, amodiaquine, verapamil, and amiodarone when added at 0, 1, or 5 uM to Huh-7 cells infected with SARS-CoV-2pp for 72 h (black bars). The number of pseudoparticles in the infected cells was quantified by measuring luciferase activity; viral entry in untreated cells was set as 100%. VSVpp were tested in parallel to exclude toxic and nonspecific effects of the drugs tested (grey bars). (B) The efficacy of the same drugs in human Airway Chips infected with CoV-2pp. Chloroquine, Arbidol, toremifene, clomiphene, amodiaquine, Verapamil, and Amiodarone were delivered into apical and basal channels of the chip at their respective Cmax in human blood, and ne day later chips were infected with CoV-2pp while in the continued presence of the drugs for 2 more days. The epithelium from the chips were collected for detection of viral pol gene by qRT-PCR; viral entry in untreated chips was set as 100%. *, P < 0.05; **, P < 0.01; ***, P < 0.001. \n",
      "Characterization of the human Airway Chip and its infection with influenza virus.Fig. 1.(A) Schematic diagram of a cross-section through the Airway Chip. (B) Immunofluorescence micrographs showing the distribution of ZO1-containing tight junctions and cilia in the epithelium and VE-cadherin-containing adherens junctions in the endothelium of the Airway Chip in the absence (Control) or presence (+ Virus) of infection with GFP-labeled influenza PR8 (H1N1) virus (MOI = 0.1) for 48 h (blue, DAPI-stained nuclei; bar, 50 μm). (C) Fold changes in gene expression levels of 4 different epithelial cell serine proteases (TMPRSS4, TMPRSS11D, TMPRSS2, DESC1) in the well-differentiated Airway Chip versus MDCK cells (one of the most commonly used cell lines in influenza studies) or undifferentiated primary human lung airway epithelial cells. Immunofluorescence micrographs showing the expression of ACE2 receptor (D) in the well-differentiated primary human lung airway epithelim on-chip (Post) versus the same cells prior to differentiation (Pre). fold changes in mRNA levels of ACE2 (E) and TMPRSS2 (F) in the well-differentiated primary human lung airway epithelim on-chip (Post) versus the same cells prior to differentiation (Pre). (G) Increase in barrier permeability as measured by apparent permeability (log Papp) within the human Airway chip 48 h post-infection with PR8 (H1N1) or HK/68 (H3N2) virus (MOI = 0.1) compared to no infection (Control). \n",
      "Effects of anti-influenza therapeutics in the human Airway Chip.Fig. 3.(A) Graph showing relative plaque titers of progeny virus in the absence (Ctrl) or presence of 1 μM oseltamivir acid (OSV) 48 h post-infection with WSN (H1N1; MOI = 0.1). (B) Barrier permeability (log Papp) measured under control conditions within the human Airway Chip (-Virus) or 48 h post-infection with WSN (+ Virus) with (+) or without (-) OSV. (C) Immunofluorescence micrographs showing the distribution of ZO1-containing tight junctions in airway epithelium under baseline conditions (Ctrl) or infected with WSN alone (Virus) or in the presence of OSV (Virus + OSV) 48 h post-infection (bar, 50 μm). (D) Production of cytokines in human Airway Chip 48 h post-infection with WSN in the presence (+) or absence (-) of OSV. (E) Virus titer detection showing the effects of Nafamostat at 1 μM (grey bars) or 10 μM (white bars) dose on virus replication of H1N1 and H3N2 in Airway chips 48 h post-infection compared to untreated chips (Ctrl, black bars). (F) The effects of Nafamostat, oseltamivir and their combination on relative viral titers when added to H1N1 virus-infected human Airway Chips at indicated times; note the synergistic effects of these two drugs at later times. *, P<0.05; **, P<0.01; ***, P<0.001. \n",
      "Infection with multiple influenza strains in human Airway Chips and resultant immune responses.Fig. 2.(A) Replication kinetics of influenza H1N1 virus strains WSN (square), PR8 (triangle), NL/09 (diamond) (left graph), and of influenza H3N2 virus strains HK/68 (square) and Pan/99 (triangle) (right graph), when infected at MOI = 0.001 in human Airway Chips. (B) Neutrophil responses to influenza infection in human lung Airway Chip. Top, sequential immunofluorescence micrographs showing time-dependent recruitment of neutrophils (white) to the apical surface of the endothelium (unlabeled) within a human Airway Chip infected with influenza PR8 (H1N1) virus (bar, 50 μm). Bottom, immunofluorescence micrographs showing time-dependent recruitment of neutrophils (white) to the epithelium (unlabeled) and clearance of clustered epithelial cells infected with GFP-labeled PR8 (H1N1) virus (green) (bar, 50 μm). (C) Graph showing numbers of neutrophils recruited to the epithelium in response to infection by H1N1 or H3N2. (D) Virus titers of human Airway Chips infected with WSN (H1N1) or HK/68 (H3N2) in the presence (+) or absence (-) of added neutrophils (PFU, plaque-forming units). (E) Production of indicated cytokines and chemokines in the human Airway chip at 48 h post-infection with different clinically isolated influenza virus strains, including NL/09 (H1N1), Pan/99 (H3N2), and HK/97 (H5N1) (MOI = 0.1). *, P<0·05; **, P<0·01; ***, P<0·001. \n",
      ".Figure 2A. The knockdown of endogenous Nox4 resulted in a substantial inhibition of tunicamycin-induced eIF2a phosphorylation in H9c2 cells, with no change in phospho-Thr980-PERK (PERK-P) levels. GADD34 levels were significantly decreased after Nox4 knockdown while there was no change in PP1 protein levels.B. Overexpression of Nox4 in H9c2 cells caused prolongation of tunicamycin-induced eIF2a phosphorylation, with minimal change in phospho-PERK levels.C,D. Mean levels of phosphorylated eIF2a relative to total eIF2a protein in tunicamycin-treated cells after Nox4 knockdown or overexpression, respectively. n=3/group. *, significant compared to baseline; #, significant comparing Nox4 knockdown (Ad.shNox4) or overexpression (Ad.Nox4) versus corresponding controls (Ad.Ctl or Ad.β-Gal, respectively). E,F. Effect of Nox4 knockdown or overexpression, respectively, on okadaic acid-resistant Ser/Thr phosphatase activity in membrane fractions of tunicamycin-treated H9c2 cells. n=4/group. *p<0.05, **p<0.01 cf. baseline; #, p<0.05, ##, p<0.01 comparing Nox4 knockdown (Ad.shNox4) or overexpression (Ad.Nox4) versus corresponding controls (Ad.Ctl or Ad.β-Gal, respectively). G,H. Nox4 knockdown or overexpression, respectively, had no effect on the phosphorylation of glycogen synthase at Ser641 (GS-P) or Histone H3 at Ser57 (H3-P) in H9c2 cells. I. Nox4-/- MEF cells (KO) showed blunted tunicamycin-induced increases in levels of phospho-eIF2a, ATF4 and ER chaperones as compared to wild-type (WT) MEFs, a response that was rescued by re-introduction of Nox4 (KO+Nox4). The latter had no effect on GS-P or H3-P levels. \n",
      ".Figure 1A. Tunicamycin (Tn, 2 µg/ml) increased Nox4 protein levels in H9c2 cells. Tubulin was used as a loading control. n=4-6/group. *, significant compared to baseline. B. Thapsigargin (Tp, 1 µg/ml) increased Nox4 protein levels in H9c2 cells. n=4-6/group. *, significant compared to baseline. C. Effect of Nox4 on the unfolded protein response. Nox4 was depleted in H9c2 cells by shRNA-mediated knockdown (Ad.shNox4) or cells were treated with a control adenovirus (Ad.Ctl). In cells with Nox4 knockdown, tunicamycin treatment resulted in lower increases in protein levels of the ER chaperones Grp94, Grp78 and calreticulin than in conrol cells. Nuclear protein levels of ATF4 were substantially lower in Nox4-depleted cells than control cells but the levels of cleaved ATF6 (ATF6c) were similar. Histone was used as a loading control. The relative mRNA levels of Xbp1-s (a readout of IRE1 signaling) were unaltered after Nox4 knockdown. Mean data are shown in Appendix Fig S1E. Similar results were obtained with an independent siRNA approach (Appendix Fig S2A).D. Effect of adenoviral-mediated overexpression of Nox4 (Ad.Nox4) or a control β-galactosidase protein (Ad.β-Gal) on tunicamycin responses of H9c2 cells. Nox4 enhanced the increase in cellular ER chaperones and nuclear ATF4 levels but did not affect tunicamycin-induced changes in nuclear ATF6c levels and caused minor reduction in Xbp1s mRNA levels. Mean data are shown in Appendix Fig S2B. E,F. Effect of Nox4 knockdown or overexpression, respectively, on the tunicamycin-induced changes in mRNA levels of ATF4 target genes. n=4/group. Psat1, phosphoserine aminotransferase; Phgdh, 3-phosphoglycerate dehydrogenase; Asns, asparagine synthetase; Slc6a9, glycine transporter 1. G. Effect of ATF4 silencing with two different siRNAs on Nox4 protein levels in tunicamycin-treated H9c2 cells. Scrambled siRNAs were used as a control (Ctl). Representative immunoblots shown to the top (captions at bottom of bar graphs refer also to the immunoblots); tubulin was used as a loading control. n=4/group. *, significant compared to baseline; #, significant comparing siATF4 versus corresponding siCtl. H. Effect of ATF4 overexpression on Nox4 mRNA and protein levels. n=3/group. \n",
      ".Figure 3A. Sub-cellular localization of Nox4. Tunicamycin (Tn 2 µg/ml, 6h) increased Nox4 levels in H9c2 cells as assessed by spinning disk confocal microscopy (scale bar, 10 µm).3D SIM images (scale bars, 2 µm) showed localisation of Nox4 (green) to the ER, which was labeled with an anti-KDEL antibody (red). At higher magnification (right), yellow dots denote co-localization of Nox4 and KDEL signals. Cell nuclei were stained with DAPI (blue). 1 Z slice from 3D stack is shown. B. Progressive enrichment of GADD34, PP1, eIF2α and Nox4 in membrane fractions of tunicamycin-treated H9c2 cells. C. After sucrose gradient fractionation of lysates of tunicamycin-treated H9c2 cells, GADD34, PP1 and Nox4 co-eluted in fractions 12 and 13 (F12, F13). D. Immunoprecipitation (IP) of pooled fractions 12/13 with an anti-GADD34 antibody revealed the presence of both PP1 and Nox4. E. The association of Nox4 with GADD34 was validated in HEK293 cells co-transfected with Nox4 and either Flag-tagged or non-tagged GADD34, followed by IP with an anti-Flag antibody. F. Co-transfection of HEK293 cells with GADD34-Flag and different myc-tagged Nox4 constructs, followed by IP with an anti-myc antibody. GADD34 binds to full-length Nox4 (FL) and the Nox4 transmembrane domain (TD) but not the C-terminal domain (CD). G. Representative pseudocolor images of simultaneous ER and cytosolic ROS measurement with HyPer ER and HyPerRed Cyto, respectively, in tunicamycin-treated MEF cells. Redox-insensitive mutant probes were used as negative controls and to exclude pH changes. Extracellular H2O2 (200 nM) was added as a positive control. The pseudocolor scale is shown along the left vertical edge of each image. KO = Nox4-/-. Scale bars, 2 µm. H. Transfection of HEK293 cells with PP1 and GADD34 increased PP1 activity (bar graph) and decreased phospho-eIF2 levels (immunoblots). Co-transfection of full-length Nox4 reduced PP1 activity and increased phospho-eIF2 levels (captions at bottom refer both to the bar graphs and immunoblots). These effects were abrogated when either Nox4 P437H or the Nox4 transmembrane domain (TD) were transfected. Nox4 did not affect phosphorylation of glycogen synthase (GS-P) or histone H3 (H3-P). All experiments were n=3/group. Values below the immunoblots are mean ± SEM levels for phospho-eIF2/total-eIF2. *, p<0.05 comparing third and fourth lanes; Levels of significance for comparisons of PP1 activities are shown above the bar columns. See also Fig EV1 and Appendix Fig S4. \n",
      "Starvation triggers accumulation of nucleosides, bases and S7P.Figure 2Starvation triggers accumulation of nucleosides, bases and S7P. Wild‐type S. cerevisiae cells growing in minimal media were switched to minimal media containing no carbon, no nitrogen or no phosphate. After the indicated duration of starvation, the metabolome was quantified by LC‐MS. For media composition, see Supplementary Table S2. For the list of absolute concentration of metabolties, see Supplementary Table S3. Data are shown in heat map format, with each line reflecting the dynamics of a particular compound in a particular culture condition. Metabolite levels of biological duplicates were averaged, normalized to cells growing steadily in glucose (time zero), and the resulting fold changes log2 transformed. \n",
      "Confirmation that Pnp1 is the physiological purine nucleoside phosphorylase, Urh1 is the pyrimidine hydrolase, and Prm15 (Pgm3) is the phosphoribomutase.Figure 5Confirmation that Pnp1 is the physiological purine nucleoside phosphorylase, Urh1 is the pyrimidine hydrolase, and Prm15 (Pgm3) is the phosphoribomutase. Ratio of metabolite levels in pnp1/urh1, pnp1, urh1 and prm15 (pgm3) strains versus wild‐type strain during carbon starvation. Data are shown in heat map format, with each line reflecting the dynamics of the ratio of the metabolite levels in a particular strain versus wild‐type strain. All reported values are log2 transformed ratios; data are mean of duplicate samples at each time point. \n",
      "In carbon starvation, nucleosides and PPP compounds are produced by RNA degradation via autophagy in response to kinase signals.Figure 3(A) Experimental design for demonstrating metabolite production via macromolecule degradation. Yeast cells were grown on 2% unlabeled glucose, and then switched to U‐13C‐glucose for 70 min, which completely labels free metabolite but only partially labels macromolecules. Thereafter, glucose was removed and the metabolome analyzed by LC‐MS. (B) Fraction of unlabeled nucleosides, nucleic bases and PPP intermediates as a function of starvation time, in wild‐type and autophagy deficient (atg7 deletion) yeast. The x axis represents minutes after carbon starvation, and the y axis represents fraction of unlabeled metabolites (mean±range of N=2 biological replicates). (C) Ratio of metabolite levels in atg7 strain versus wild‐type strain in carbon starvation. (D) Ratio of metabolite levels in bcy1 strain and snf1 strain versus wild‐type strain in carbon starvation. (E) Ratio of metabolite levels in rapamycin treatment versus nitrogen starvation for wild‐type yeast. In (C) to (E), all reported values are log2 transformed ratios; data are mean of duplicate samples at each time point. \n",
      "Phm8 is the physiological yeast nucleotidase.Figure 4(A) Ratio of metabolite levels in sdt1 and phm8 strains versus wild‐type strain in carbon starvation. All reported values are log2 transformed ratios; data are mean of duplicate samples at each time point. (B) Summary of transcripts data of PHM8, SDT1 and other enzymes in the pathway (Gasch et al, 2000; Bradley et al, 2009; Klosinska et al, 2011). All values are log2 transformed fold changes. (C) Screening of Phm8's phosphatase activity against 90 phosphorylated compounds. Phosphatase activity was measured in the presence of 0.5 mM substrate and 5 mM Mg2+ (pH=7.0, 30°C). Compounds with specific activity higher than 0.1 μmol/mg/min are shown. The x axis represents specific phosphatase activity (μmol of phosphate produced per minute per mg of enzyme, mean±range of N=2 replicates). (D) Top table: absolute intracellular concentration of nucleotide monophosphates in carbon starvation. Bottom plots: Phosphatase activity of Phm8 and Sdt1 as a function of CMP concentration. The x axis represents CMP concentration and the y axis represents specific phosphatase activity (μmol of phosphate produced per minute per mg of enzyme, mean±range of N=2 replicates). \n",
      ".Figure 6A. Plasma urea levels were elevated to a greater extent in tunicamycin-treated Nox4 KO mice than WT. Co-treatment with guanabenz (Gbz) reduced urea levels in both groups. Numbers of animals are indicated within bars. B. 48 hours after systemic tunicamycin treatment, kidneys of Nox4 KO mice showed a marked surface pallor (bottom right). C. TUNEL staining revealed a significantly higher number of apoptotic cells in tunicamycin-treated Nox4 KO mice. n=4/group. D. Immunoblotting of kidney homogenates showed significantly elevated cleaved caspase-12 and cleaved PARP levels in tunicamycin-treated Nox4 KO mice compared to WT. E. Survival curves showed that a very high proportion of Nox4 KO mice died after AKI. Guanabenz (Gbz) treatment dramatically improved survival in tunicamycin-treated KO mice. Number of animals as indicated. Levels of significance by Kaplan Meier analysis are reported to the right. F. Schematic depicting the effect of Nox4-generated ROS on PP1 activity and the balance between eIF2α phosphorylation and dephosphorylation. Nox4 is upregulated by ATF4 and binds to GADD34. It inhibits GADD34-bound PP1 through the local generation of H2O2 and oxidation of the metal (M) center of the serine-threonine phosphatase. The consequent prolongation of eIF2α phosphorylation promotes cell survival in the face of acute protein unfolding stress. M = iron or manganese, which are oxidized from the M (II) to the M (III) species. \n",
      ".Figure 5A. H9c2 cells treated with tunicamycin (2 µg/ml, 12h) showed significantly lower survival when endogenous Nox4 was silenced (siNox4) as compared to cells treated with a scrambled siRNA (siCtl). Cell survival was restored by treatment with either guanabenz (Gbz, 5 µM) or salubrinal (Sal, 50 µM) but was unaffected by clonidine (Cld, 5 µM). n=3/group. B. Nox4-depleted cells had lower levels of phospho-eIF2α and ATF4 than control cells but these were restored in the presence of guanabenz (Gbz). C. Schematic representation of the effect of Nox4 and the small molecule inhibitors, guanabenz and salubrinal, on the GADD34/PP1/eIF2 interaction. D. Hearts from Nox4 knockout (KO) mice and WT controls were subjected to global ischemia followed by aerobic reperfusion (I/R). Infarct size assessed by triphenyltetrazolium chloride (TTC) staining was greater in Nox4 KO hearts compared to WT and was significantly reduced by guanabenz (Gbz). In the representative heart sections shown at the top, white denotes infarct area and red the viable area. Scale bars, 1 mm. Numbers of hearts are indicated within the bars. E. Immunoblotting of heart homogenates after I/R showed lower levels of phospho-eIF2a, ATF4 and ER chaperones, and higher levels of cleaved caspase-12, in Nox4 KO compared to WT. Tubulin was used as a loading control. Treatment with guanabenz (Gbz) reversed these changes (blots shown to the right). \n",
      ".Figure 4A. Recombinant PP1 was inhibited by H2O2 (0.2 mM) and activity was not restored by glutathione (GSH), cysteine (Cys) or dithiothreitol (DTT). A Cys127Ser/Cys273Ser PP1 mutant was inhibited by H2O2 similarly to wild-type PP1. Values above bars denote level of significance for the inhibitory effect of H2O2. B. Ascorbate (Asc) dose-dependently restored PP1 activity. #, significant effect of Asc compared to H2O2 alone. C. EPR spectra of PP1 incubated with ascorbate (1 mM) alone (a) or PP1 exposed to H2O2 followed by catalase treatment, then incubation with ascorbate (b). Panel b shows a typical spectrum for the ascorbyl radical (hyperfine splitting constant, aH = 1.8G), similar to the positive control obtained by exposing ascorbate to H2O2 (c). Panel d shows that no ascorbyl radical is detected if H2O2 is degraded by catalase in the absence of PP1, prior to ascorbate addition. D. Cartoon representation of the active site of H2O2-treated PP1 as in Fig EV2D. 2mFo-DFc electron density map at the 2.2 Å resolution is shown in yellow at the 1.1 level. H2O2-treatment causes an overall shrinkage of the PP1 coordination sphere by 0.12 Å compared to ascorbate-treated crystals consistent with the oxidation of the dinuclear center. This increases the energy barrier for the catalytic steps involving -OH- attack on the phosphorous center of the bridging phosphate and rupture of the P-O scissile bond with the assistance of H125 (black arrows). Reported coordination distances in Å are averaged over the two PP1 molecules in the crystallographic asymmetric unit. See also Appendix Table S1. E. Effect of Nox4 on GADD34/PP1-mediated eIF2 dephosphorylation in transfected HEK293 cells. Nox4 increased eIF2 phosphorylation in cells transfected with GADD34 and WT PP1, and resulted in even higher phospho-eIF2α levels in cells transfected with N124D or D64N PP1 variants. F. Effect of ascorbate (Asc, 0.5 mM) on phosphatase inhibition in tunicamycin-treated H9c2 cells with overexpression or knockdown of Nox4 (Ad.Nox4 and Ad.shNox4, respectively). In control cells, ascorbate enhanced tunicamycin-stimulated increases in phosphatase activity. Phosphatase activity was lower in Nox4-overexpressing than control cells but was normalized by ascorbate to the same level as in control cells. In Nox4 knockdown cells, tunicamycin-induced increases in phosphatase activity were enhanced and ascorbate had minimal additional effect. \n",
      "Nucleoside degradation causes S7P accumulation due to depletion of the other transaldolase substrate glyceraldehyde-3-phosphate, and SBP accumulation due to phosphorylation of S7P into SBP by Pfk1.Figure 6(A) Diagram of proposed reactions. (B‐E) DHAP, SBP and S7P levels in wild type, tkl1/tkl2 (B), tal1/nqm1 (C) and pfk1 (E) strains upon carbon starvation, and wild‐type strain upon abruptly switching from glucose to no carbon (WT) versus to dihydroxyacetone (WT+DHA) as the sole carbon source (D). The x axis represents hours after carbon starvation, and the logarithmic y axis represents absolute intracellular concentration (mean±range of N=2 biological replicates). \n",
      "The accumulation of S7P in carbon starvation facilitates survival of subsequent oxidative stress.Figure 8(A) Experimental design. Yeast cells growing on glucose were switched to no carbon media for 30 min. Thereafter, 20 mM H2O2 was added and metabolome analyzed by LC‐MS. (B) Diagram of proposed reactions and their regulation. S7P accumulates during carbon starvation due to low level of glyceraldehyde‐3‐phosphate (GAP). Oxidative stress blocks GAP consumption via glycolysis by inhibiting GAPDH. This provides sufficient GAP to react with S7P, producing F6P via transaldolase. F6P is further converted to G6P which enters the oxidative PPP. The resulting NADPH is utilized for regeneration of reduced glutathione. 1,3‐DPG, 1,3‐diphosphoglycerate; GSSG, glutathione disulfide; GSH, reduced glutathione. (C and D) DHAP, S7P, G6P, and glutathione levels and NADPH/NADP+ ratio in wild type, phm8 and pnp1/urh1 strains in the experiment shown in (A) The x axis represents minutes after oxidative stress, and the y axis represents absolute intracellular concentration or ratio of intracellular concentration (mean±range of N=2 biological replicates). (E) Viability of wild type, phm8 and pnp1/urh1 strains in oxidative stress. The y axis represents percentage of survived cells calculated by dividing the number of colonies formed after oxidative stress by the number of colonies formed under the same starvation condition but without H2O2 treatment (mean±range of N=2 biological replicates). \n",
      "The ribose-salvage pathway is essential for yeast's survival in starvation.Figure 7(A) Level of hallmark metabolites and energy charge in wild type, phm8 and pnp1/urh1 strains in starvation. The x axis represents hours after starvation, and the logarithmic y axis represents energy charge ([ATP]+0.5[ADP]/([ATP]+[ADP]+[AMP]) or absolute intracellular concentration (mean±range of N=2 biological replicates). (B and C) Growth and viability of wild type, phm8 and pnp1/urh1 strains in starvation. (D) Growth of wild type, phm8 and pnp1/urh1 strains in the switch from glucose to glycerol+ethanol. The x axis represents hours after starvation and the logarithmic y axis represents optical density (A600) in (B and D) and percentage of live cells calculated by dividing the number of colonies formed after starvation by the number of colonies formed before starvation in (C) (mean±range of N=2 biological replicates). \n",
      "Fig.Figure 2In the RNA-seq data (GSE145490, normalized read count (RC)>1, >2.6-fold, 3,000 genes), genes downregulated in the VM-glia treated with the SGK1 inhibitor GSK-650394 vs the vehicle (DMSO)-treated control were analyzed.C, Gene set enrichment analysis (GESA) for inflammatory and immune responses. D, Scatterplots for the differentially expressed genes (DEGs) highlighting the inflammatory/immune genes that are most significantly downregulated in the inhibiter-treated cultures. E, Heatmap for selected inflammatory genes in the SGK1 inhibitor-treated and -untreated glial cultures. Data represent the RC values (inside box) and log2[SGK1 inhibitor-treated/control] (color intensity). F, Microglia and astrocyte reactivation-specific marker expressions in the RNA-seq data. \n",
      "Fig.Figure 3mRNA expression of the NLRP3-inflammasome components Nlrp3, Pycard (Asc), and Caspase1. Decreased expression of the components shown in the RNA-seq Data are expressed as mean ± SEM. Significantly different at p=0.0018*, 0.0405**, 0.0039***, 0.0002#, 0.0033##, 0.0099###, 0.0001####, 0.0056+, 0.0398++, 0.0067+++ B, mRNA expression of the NLRP3-inflammasome components Nlrp3, Pycard (Asc), and Caspase1. Decreased expression of the components shown in the q-PCR analyses in other independent VM-glial cultures treated with the inhibitor (GSK-650394) and sh-Sgk1 and si-Sgk1 (vs. DMSO, sh-control, and si-control, respectively) (B). n=3; Student's t-test. Data are expressed as mean ± SEM. Significantly different at p=0.0018*, 0.0405**, 0.0039***, 0.0002#, 0.0033##, 0.0099###, 0.0001####, 0.0056+, 0.0398++, 0.0067+++ in graph B. C, Immunoblot analysis to assess NLRP3 inflammasome activation. To activate the inflammasome pathway, VM-derived glial cells were treated with LPS (0.25 µg/ml, 3h) and then ATP (2.5 mM, 30 min) in the presence or absence of the SGK1 inhibitor GSK-650394. Two days later, intracellular and released levels of pro- and activated caspase-1 and IL-1β proteins were determined in the culture media and cell lysates, respectively. D, Released level of IL-1β (activated) was further quantified using ELISA. n=3; ANOVA. Significantly different at p=0.0002#, 0.0012*, 0.0004** in graph D. Expression of CGAS-STING pathway genes in the RNA-seq data. The graph represents log2 RC ratios of [GSK-650394-treated/DMSO-treated control]. The molecules downregulated by the SGK1 inhibitor are marked with arrows in the CGAS-STING signal pathway schematized in F. G, WB analysis demonstrating suppression of CGAS-STING signaling by the SGK1 inhibitor in VM-glial cultures. Representative blots are shown on the left. The levels of the active forms of CGAS-STING signaling molecules are quantified from 5-10 independent blots (right). Data are represented as mean ± SEM. One-way ANOVA. Significantly different at p=0.001#, 0.014* (CGAS), 0.00004#, 0.0002* (STING), 0.017#, 0.037* (pTBK1), 0.009#, 0.045* (pIRF3), 0.012#, 0.036* (pP65) in graph G. H, Expression of Ifnb mRNA, a final product of the CGAS-STING pathway, determined by real-time PCR analyses. Data are represented as mean ± SEM. n=3; student's t-test. Significantly different at p=0.0459#, 0.0305* in graph H. \n",
      "Fig.Figure 1Nurr1 (N) and Foxa2 (F) synergistically downregulate SGK1 expression in cultured glia. (A) Up- and downregulated genes in the microarray data for the cultured glia transduced with N+F (vs. mock-transduced control). Sgk1 is marked with an arrow in the top downregulated genes. Effect of N and F on SGK1 expression in glial cells shown in the microarray (B), RNA-seq (C), Data in (B and C) represent values of fold change (FC) or read count (RC) with color intensities. n=3. One-way ANOVA. Data are represented as mean ± SEM. Significantly different at p= 0.0493*, 0.0092**, 0.0009***. Effect of N and F on SGK1 expression in glial cells shown in the RT-PCR analyses. One-way ANOVA. Data are represented as mean ± SEM. Significantly different at p= 0.0493*, 0.0092**, 0.0009***. Downregulation of SGK1 is responsible for N+F function to inhibit NFκB-mediated pro-inflammatory cytokine expression. (E) WB analysis exhibiting N+F-induced inhibition of NFκB intracellular signaling in cultured VM-glia. The N+F-mediated downregulation of NFκB signaling is attained by inhibiting IκB phosphorylation WB analysis exhibiting N+F-induced inhibition of NFκB intracellular signaling in cultured VM-glia. The N+F-mediated downregulation of NFκB signaling is attained by inhibiting IκB phosphorylation (E) and thus blocking the release of NFκB from the NFκB-IκB inhibitory complex (F). N+F-mediated downregulation of NFκB signaling (phosphorylation of p65, p-p65, G) were alleviated by forced SGK1 expression in the VM-glial cultures. H) N+F-mediated downregulation of pro-inflammatory cytokine expression (IL-1β, TNFα, H) were alleviated by forced SGK1 expression in the VM-glial cultures. (I) SGK1 inhibition blocks glial NFκB signaling. Data are represented as mean ± SEM. n=4-6. One-way ANOVA. Significantly different at p=0.002*, 0.00004**, 0.00001#, 3.01E-10##, 3.78E-06+, 1.59E-07++ in graph I. Glial pro-inflammatory cytokine expression regulated by SGK1 inhibition Glial pro-inflammatory cytokine expression regulated by SGK1 inhibition (J-M) and overexpression (N). O, Schematic summary to show how SGK1 inhibition mediates the N+F-induced anti-inflammatory action in glia. \n",
      "Fig.Figure 4Ppargc1a (PGC1α) mRNA expression upregulated in VM-glia treated with the SGK1 inhibitors in the RNA-seq data Data are represented as mean ± SEM. n=3; Student's t-test. Significantly different at p=0.0011#, 0.0499* in graph B. B, Ppargc1a (PGC1α) mRNA expression upregulated in VM-glia treated with the SGK1 inhibitors in the q-PCR analysis Data are represented as mean ± SEM. n=3; Student's t-test. Significantly different at p=0.0011#, 0.0499* in graph B. C, Immunoblot for PGC1α protein expression. Data are represented as mean ± SEM. n=3. Student's t-test. Significantly different at p=0.0077* in graph C. D, Mitochondrial biogenesis estimated by Mitotimer reporter gene expression in cultured VM-glia. Green fluorescence indicates newly synthesized mitochondria, and red fluorescence indicates old or damaged mitochondria. MFIs were estimated in 3 independent cultures (>5 randomly selected microscopic fields/culture). Data are represented as mean ± SEM. One-way ANOVA. Significantly different at p=3.78E-08*, 0.00003** in graph D. E, Mitochondrial ROS levels estimated using the MitoSox red probe. Fluorescence intensity was measured using ImageJ, and ROS levels are presented as mean fluorescent intensity (MFI). Mitochondrial damage was induced by treatment with the mitochondrial toxins CCCP (2µM)+H2O2(500 µM) for 4hr. MFIs were estimated in 3 independent cultures (>5 randomly selected microscopic fields/culture). Data are represented as mean ± SEM. One-way ANOVA. Significantly different at p=0.0013*, 0.0273# in graph E. F, Mitochondrial membrane potential (JC-1). The red fluorescence (JC-1 Aggregate) indicates disrupted mitochondrial membrane potential, and green fluorescence (JC-1 Monomer, insets) indicates intact potential. The inset images were taken from the same microscopic fields of the respective JC-1 aggregate images. Mitochondrial damage was induced by treatment with the mitochondrial toxins CCCP (2µM)+H2O2(500 µM) for 4hr. MFIs were estimated in 3 independent cultures (>5 randomly selected microscopic fields/culture). Data are represented as mean ± SEM. One-way ANOVA. Significantly different at p=0.0006*, 0.0093# in graph F. G, Intracellular levels of ATP, an indicator of mitochondrial oxidative phosphorylation. Data are expressed as mean ± SEM. n=3 cultures; one-way ANOVA. Significantly different at p= 0.0009*, 0.00004**, 0.04913#, 0.0024+in graph G. Glial cell senescence rescued by SGK1 inhibition. (H) Expression of genes associated with cell senescence in the RNA-seq data (VM-glia with vs. without GSK-650394 treatment). (I) Intracellular ROS levels estimated by DCF-DA. Data are represented as mean ± SEM. n=3; one-way ANOVA. Significantly different at p=0.0074*, 0.012# in graph I. (J) Senescence-associated β-galactosidase (SA-β-gal) activity. Data are represented as mean ± SEM. n=3; one-way ANOVA. Significantly different at p=2.05E-06*, 0.0042#, 0.0006+ in graph J. (K) Immunoblots for proteins associated with cell senescence. Significantly different at p=0.008*, 0.001# (TNFA), 0.01*, 0.009# (HMGB1), 7.32E-10*, 1.23E-8# (IL6), 9.62E-11*, 1.01E-5# (MMP3), 0.001*, 5.72E-4# (LAMIN B) in graph K. (L) IL-6 levels secreted from cultured VM-glia. The cytokine levels were estimated using ELISA of the media from the VM-glia with and without the SGK1 inhibitor. Data are represented as mean ± SEM. n=3 cultures. One-way ANOVA. Significantly different at p=0.0028*, 0.0062# in graph L. (M) Glial cell viability following treatment with senolytic compounds (azithromycin, fisetin). Data are represented as mean ± SEM. n=3; Student's t-test. Significantly different at p=0.0073*,0.0018**, 0.0286#, 0.0269##, 0.0045### in graph M. \n",
      "Fig.Figure 5A, Schematic of the experimental procedure in B and C. B, Western blot analysis for detecting SNCA oligomers. Data are represented as mean ± SEM. n=4; one-way ANOVA. Significantly different at p=2.5E-07*, 1.65E-07**, 0.022#, 0.029##, 0.046+, 0.015++, 0.0136&, 0.0016&& in graph B. C, Immunocytochemical detection of p-129-SNCA, associated with SNCA aggregation. Data are represented as mean ± SEM. n=6. Significantly different at p=0.0363* in graph C. Bi-FC-based assessment of the inter-neuronal propagation and aggregation of SNCA. (D) Schematic of the experimental procedures with Bi-FC system. E, Bi-FC-based assessment of the inter-neuronal propagation and aggregation of SNCA. Bi-FC+ inclusions (puncta) were counted in E. Data are represented as mean ± SEM. n=5. Significantly different at p=0.0446* in graph E. Glial SGK1 knockdown effect on neuronal SNCA pathology. (F), Schematic of the experimental procedure in (G-H). VM-glial cultures were transduced with sh-Sgk1 (or sh-cont) lentivirus. Medium was condition in sh-Sgk1-glia (or sh-cont-glia), and the conditioned medium (CM) treatment effects on neuronal SNCA aggregation were assessed by WB VM-glial cultures were transduced with sh-Sgk1 (or sh-cont) lentivirus. Medium was condition in sh-Sgk1-glia (or sh-cont-glia), and the conditioned medium (CM) treatment effects on neuronal SNCA aggregation were assessed by WB VM-glial cultures were transduced with sh-Sgk1 (or sh-cont) lentivirus. Medium was condition in sh-Sgk1-glia (or sh-cont-glia), and the conditioned medium (CM) treatment effects on neuronal SNCA aggregation and p-129-SNCA immunocytochemical analyses. Data are represented as mean ± SEM. n=3. Significantly different at p=0.00003*, 0.0003**, 0.0012***, 0.00004****, 0.0321# in graph G and p=0.0052* in graph H. \n",
      "Fig.Figure 6Co-culture experiments. (A) Schematic for the experimental procedure. Glia derived from mouse VM were transduced with sh-Sgk1 (B) or SGK1 (CMV-Sgk1, C), and co-cultured with VM-derived mDA neurons. Seven days after co-culture, cells were exposed to H2O2 (500µM, 3hr) or not (-), and then mDA neuron viability (TH+ cell counts) and neurite degeneration (TH+ fiber lengths) were estimated. Data are represented as mean ± SEM; n=4 culture coverslips. Student's t-test. Significantly different at p=0.033*, 0.0412**, 7.44E-23#,1.39E-23## in graph B and p=0.0489*, 0.0366**, 0.0078## in graph C. Glial neurotrophic functions potentiated by SGK1 downregulation were mediated in a paracrine manner. VM-glia transduced with sh-Sgk1 (or sh-cont as a control) were challenged with H2O2+LPS for 3hr or not. Glial conditioned medium (GCM) was prepared in each glial culture condition and administered to mDA neurons primarily cultured from mouse VM (D). Glial neurotrophic functions potentiated by SGK1 downregulation were mediated in a paracrine manner. VM-glia transduced with sh-Sgk1 (or sh-cont as a control) were challenged with H2O2+LPS for 3hr or not. Glial conditioned medium (GCM) was prepared in each glial culture condition and administered to mDA neurons primarily cultured from mouse VM (E) Cell viability of the neuronal cells was estimated using CCK8 assays and TuJ1+ cell counts. Data are represented as mean ± SEM. n=3 cultures; One-way ANOVA. Significantly different at p=0.0002*, 3.29E-08**, 0.0003***, 3.66E-06#, 0.0029## (TuJ1+ cell counting), 0.0127+, 5.98E-06++, 0.0002+++, 0.0005&, 0.0078&& (CCK8 assays) in graph E. F, Effect of the SGK1 inhibitor treatment in the mixed mDA neuron + VM-glia co-cultures. Data are represented as mean ± SEM. n=3. Student's t-test. Significantly different at p=0.0068*, 0.0368**, 0.0249#, 1.48E-27## in graph F. \n",
      "Fig.Figure 9A-C, Quantitative and morphometric analyses of microglia in the SN area of SNCA-PD mice. Shown in A are representative views of Iba1+ microglia neighboring TH+ mDA neurons in the SNs of PD mice treated with GSK-650394 and vehicle. Insets, enlarged views for the boxed areas. Data are represented as mean ± SEM. n=3 mice per group. Significantly different at p=1.49E-11* in graph B and p=0.000003* in graph C. D and E, The M1 marker CD16/32 expression in Iba1+ microglia. Data are represented as mean ± SEM. n=3 mice per group. Significantly different at p=0.0016* in graph E. F and G, mRNA expression of NLRP3-inflammasome components and pro-inflammatory cytokines in MPTP (F) and SNCA (G) PD mice injected with the SGK1 inhibitor or vehicle (control). Each bar represents the mean ± SEM of 3 PCR values from each animal, n=3 mice per group. Significantly different at p=0.0372*, 0.0173**, 0.0003*** in graph F and p=0.002#, 0.0113##, 0.0053###, 0.0001#### in graph G. H-J, ASC expression in the SNs and SN microglia of SNCA-PD mice. n=5 mice per group. Data are represented as mean ± SEM. Significantly different at p=1.91E-09* in graph I and p=0.0079* in graph J. K and L, β-gal-stained cell senescence in SNCA-PD mouse SN. Data are represented as mean ± SEM. n=5 mice per group. Significantly different at p=0.0001* \n",
      ".Figure 4(A) LPS/ATP treated and non treated wild type and Becn1+/- microglia were immunolabeled for NLRP3 (green) and LC3 (red) and imaged by confocal microscopy. Stimulation resulted in the appearance of many NLRP3-containing aggregates of different sizes (arrows). In non treated cells, only a few overlapping signals with LC3-stained autophagosomes (arrowheads) are visible. Stimulated microglia however, showed multiple colocalizations of LC3 positive vesicles and NLRP3 aggregates; scale bar: 7.5 µm. (B) LPS/ATP treated wild type microglia were immunolabeled for NLRP3 (green) and LC3 (red) and analysed by super resolution microscopy (SIM). NLRP3 aggregates of different sizes coclustering with LC3-positive autophagosomes are clearly visible. 3D volume rendering of large (I) and small (II) NLRP3 aggregates from the magnified ROIs showed engulfment of NLRP3 by autophagosomes; scale bar: 10 µm; magnified ROIs: 1 µm. (C) 3D Radial intensity profiles of NLRP3 and LC3 signals derived from SIM images in wild type microglia, centered on the maxima of NLRP3 clusters. The radial profiles confirm that both proteins colocalize to the same organelle; mean ± SEM, n = 4. \n",
      "Fig.Figure 8A and B, Efficiency of AAV9-mediated gene delivery. AA9 viruses expressing the reporter GFP gene were stereotaxically injected into the SN of mice, and GFP expression in astrocytes, microglia, and neuronal cells was quantified. Data are represented as mean ± SEM. n=3 mice (>5 randomly selected microscopic fields per animal were counted). Significantly different at p=0.006*, 5.1E-09#, 5.36E-09+ in graph B. (C) Schematic of the experimental procedure. SGK1 knockdown effect in vivo by treatment with shSgk1-AAV9. Total SGK1Cell type-specific SGK1 knockdown was estimated by counting SGK1-immuonoreactivity in neuron (MAP2+), astrocyte (GFAP+), and microglia (Iba1+) populations (E). Data are represented as mean ± SEM. n=4 mice in each group. Significantly different at p=0.0004# in graph D and p=0.049*, 0.007**, 0.002*** in graph E. (F-H) Behavior of the PD mice assessed by pole (F), beam (G) at 6-12 weeks, and locomotor activity (H) tests at 12 weeks post- SNCA injection. Significantly different at p=0.0003#, 0.0428*, 7.46E-06##, 0.0012** in graph F and p=8.04E-08#, 0.004*, 1.62E-09##, 3.3E-06** in graph G and p=0.0002#, 0.0194* (total distance), 9.54E-06#, 0.0047* (average speed) in graph H. Histologic assessment of p129-SNCA immunoreactivity ( I,J), TH+ mDA neuron counts (I,K), neurite lengths (L), and cell body sizes (M) in the SN Significantly different at p=0.0001* in graph J, p=2.35E-25#, 1.09E-16##, 4.36E-08* in graph K and p=1.5E-40#, 1.08E-24##, 2.32E-07* in graph L and p=9.07E-20#, 2.59E-16##, 0.05* in graph M and p=4.62E-22#, 5.11E-12##, 3.49E-06* (TH), n= 7 sh-SGK1-AAV9-treated PD mice, 7 sh-cont-AAV9-treated PD mice, and 8 SNCA-untreated mice. Data are represented as mean ± SEM. One-way ANOVA. Scale bar, 100µm. Histologic assessment of TH+ and DA transporter (DAT) fiber intensities in the striatum n= 7 sh-SGK1-AAV9-treated PD mice, 7 sh-cont-AAV9-treated PD mice, and 8 SNCA-untreated mice. Data are represented as mean ± SEM. One-way ANOVA. Scale bar, 100µm. \n",
      ".Figure 1(A) BECN1 expression of microglia from Becn1+/- and wild type mouse pups in full medium without Bafilomycin A1 (Baf) was quantified by Western blotting. Microglia from Becn1+/- mice show a signifcant reduction of BECN1 (48%) compared to microglia from wild type mice; mean ± SEM, wild type n = 3, Becn1+/- n = 7; two-tailed t-test **: p < 0.01. Blot shows representative amounts of BECN1 after various treatments. (B) Microglia from newborn Becn1+/- and wild type mouse pups were kept for 2 h either in full medium or HBSS with or without Bafilomycin A1 (Baf). LC3 and ACTIN levels were determined by Western blot. LC3-II was significantly reduced in microglia from Becn1+/- mice after Baf treatment in full medium compared to wild type microglia; mean ± SEM, wild type n = 3, Becn1+/- n = 4; two-tailed t-test *: p < 0.05. (C) Microglia from newborn Becn1+/- and wild type mouse pups were kept for 2 h either in full medium or HBSS with Bafilomycin A1 (Baf) and stained for endogenous LC3. Exemplary images and the quantification of LC3-positive vesicles/µm2 cell area show reduced presence of LC3-positive vesicles in microglia from Becn1+/- and induction of autophagy by HBSS; mean ± SEM, wild type n = 4, Becn1+/- n = 4, 3-5 fields per view/n and condition and 73-271 cells/n and condition; two-tailed t-test *: p < 0.05; scale bar: 20 µm \n",
      ".Figure 2(A, B) Microglia isolated from newborn Becn1+/- and wild type mouse pups were either non treated or treated with a pro-inflammatory stimulus (LPS followed by ATP). As additional controls cells treated only with LPS or ATP were used in (A). The concentration of the pro-inflammatory cytokines IL-1beta (A) and IL-18 (B) in the cell supernatant was determined by ELISA. Reduction of BECN1 significantly enhanced release of both cytokines after stimulation with LPS/ATP; mean ± SEM, wild type n (A/B) = 9/2, wild type LPS n = 3/0, wild type ATP n = 3/0, wild type LPS/ATP n = 9/9, Becn1+/- n = 23/5, Becn1+/- LPS/ATP n = 23/23, two-tailed t-test *: p < 0.05. (C) Microglia isolated from wild type mouse pups were pre-incubated in full medium or starvation medium HBSS before addition of a pro-inflammatory stimulus (LPS followed by ATP). The autophagy blocker 3-MA was added for the final 1 h 45 min of the experiment. The concentration of the pro-inflammatory cytokine IL-1beta in the cell supernatant was determined by ELISA. Incubation with HBSS reduced IL-1beta release significantly while 3-MA treatment increased it; mean ± SEM, full medium n = 6, full medium LPS/ATP n = 6, full medium LPS/ATP + 3MA n = 3, HBSS n = 6, HBSS LPS/ATP n = 6, HBSS LPS/ATP + 3MA n = 6; two-tailed t-test *: p < 0.05; **: p < 0.01. (D) Microglia isolated from wild type mouse pups were pre-incubated in starvation medium HBSS before addition of a pro-inflammatory stimulus (LPS followed by ATP). The autophagy blocker 3-MA was added for the final 1 h 45 min of the experiment, microglia were stained for LC3 and images taken by confocal microscopy. Representative images show a reduction of LC3-positive vesicles in 3-MA treated cells. Scale bar: 20 µm. (E) Proteins were extracted from brains of wild type, Becn1+/-, APPPS1 and APPPS1 Becn1+/- mice at the indicated ages. IL-1beta was measured by electrochemiluminescence (MesoScale) and compared within the repective age groups. IL-1beta was significantly increased in the TBS and TX fraction; mean ± SEM, wild type: n (per age group) = 2/6/3, Becn1+/-: n = 2/5/2, APPPS1: n = 5/5/6, APPPS1 Becn1+/-: n = 7/9/6; ANOVA with Dunnett´s post hoc test with WT as control group; *: p < 0.05; **: p < 0.01. (F) Proteins were extracted from brains of APPPS1 and of APPPS1 Becn1+/- mice at the age as indicated. Amyloid beta 1-40 and 1-42 were measured by electrochemiluminescence (MesoScale) and compared within the repective age groups. No difference in the amyloid beta load between the genotypes is apparent; mean ± SEM, APPPS1: n = 5/5/6, APPPS1 Becn1+/-: n = 7/9/6; ANOVA with Dunnett´s post hoc test with APPPS1 as control group, ns. \n",
      ".Figure 3(A) The expression of NLRP3 protein in non treated and LPS/ATP treated wild type and Becn1+/- microglia was determined by Western blot and normalised to ACTIN. Reduction of BECN1 significantly enhanced expression of NLRP3; mean ± SEM, wild type n = 4, wild type LPS/ATP n = 9, Becn1+/- n = 4, Becn1+/- LPS/ATP n = 23; ANOVA with Tukey´s post hoc test *: p < 0.05. (B) The expression of Nlrp3 mRNA in non treated and LPS/ATP treated wild type and Becn1+/- microglia was determined by qPCR. No significant differences can be detected between wild type and Becn1+/- microglia; mean ± SEM, wild type: n = 7, Becn1+/- : n = 3; ANOVA with Tukey ´s post hoc test, ns for LPS/ATP treated wild type versus Becn1+/-. (C) LPS/ATP treated and non treated wild type and Becn1+/- microglia were immunolabeled for ASC (red) and imaged by confocal microscopy to investigate the assembly of inflammasomes. In non stimulated cells ASC is distributed diffusely in the cytoplasm (upper panels). However, after stimulation formation of inflammasomes (arrowheads) can be detected in a subpopulation of cells as well as release of inflammasomes after pyroptosis/ejection (arrow). Quantification showed a significantly increased percentage of inflammasomes in or around Becn1+/- microglia; mean ± SEM, wild type LPS/ATP n = 4, Becn1+/- LPS/ATP n = 7; two-tailed t-test *: p < 0.05; scale bar: 50 µm, insert scale bar: 10 µm. (D) The presence of cleaved CASP1 (p10) protein in the supernatant of LPS/ATP treated wild type and Becn1+/- microglia was determined by Western blot and normalised to ACTIN. Reduction of BECN1 significantly enhanced release of CASP1; mean ± SEM, wild type LPS/ATP n = 8, Becn1+/- LPS/ATP n = 18; two-tailed t-test *: p < 0.05. \n",
      "Fig.Figure 7(A) Overview of the experimental procedure. (B-D) Behaviors of MPTP-PD mice assessed by pole (B), beam (C) at 3-6 weeks post-MPTP injection, and locomotor activity (total distance & average speed, D) tests at 6 weeks post-MPTP injection. n= 20 GSK-650394-treated MPTP-PD mice, 14 vehicle-treated MPTP-PD mice, and 8 MPTP-untreated non-PD mice. Data are represented as mean ± SEM. Significantly different at p=0.0012#, 0.0001*, 3.09E-08##, 8.21E-08** in graph B and p=0.0009#, 0.0035*, 1.03E-06##, 0.00003** in graph C and p=0.0001*, 0.0284**, 0.0101# (total distance), 1.49E-08*, 0.0005**, 0.0001# (average speed) in graph D. Histologic assessment of the number of TH+ mDA neurons (E, F), neurite lengths (E, G), and cell body sizes (E, H) Shown are the representative images for TH+ DA neurons in the SN TH+ DA neurons in boxed areas of (E) are enlarged in right. Histologic assessment of the number of TH+ mDA neurons in the SN and TH+ fiber intensities in the striatum Shown are the representative images for TH+ DA neurons in the TH-immunoreactivity in the striatum, Therapeutic effects of GSK-650394 in SNCA-PD model mice. Therapeutic effects of GSK-650394 in SNCA-PD model mice. Behaviors (L-N) of the SNCA-PD mice were assessed at 6-12 weeks post- SNCA injection Significantly different at p=0.008#, 0.021*, 0.00001##, 0.0009** in graph L and p=0.00002#, 0.0344*, 0.0005##, 0.0067** in graph M and p=0.0057#, 0.0345* (total distance), 0.0001#, 0.0375##, 0.0338* (average speed) in graph N. Therapeutic effects of GSK-650394 in SNCA-PD model mice. mDA neuron degeneration in the SN of the SNCA-PD mice were assessed at 12 weeks post- SNCA injection after sacrifice SNCA pathology assessed by p129-α-syn immunoreactivity in TH+ mDA neurons in the SN. n= 5 GSK-650394-treated SNCA-PD mice, 5 vehicle-treated SNCA-PD mice, and 8 SNCA-untreated non-PD mice. Therapeutic effects of GSK-650394 in SNCA-PD model mice. mDA neuron degeneration in the SN (O-U) of the SNCA-PD mice were assessed 12 weeks post- SNCA injection after sacrifice SNCA pathology assessed by p129-α-syn immunoreactivity in TH+ mDA neurons in the SN. n= 5 GSK-650394-treated SNCA-PD mice, 5 vehicle-treated SNCA-PD mice, and 8 SNCA-untreated non-PD mice. \n",
      "a) UVRAG interaction with C-Vps.figf1(a) UVRAG interaction with C-Vps. 293T cells were co-transfected with Flag-UVRAG and HA-Vps16, HA-Vps11, HA-Vps18, HA-Vps33 or HA-Vps39. Whole-cell lysates (WCLs) were used for immunoprecipitation (IP) with an anti-Flag antibody, followed by immunoblotting (IB) with an anti-HA antibody. 1% WCL was used as the input. (b) Interaction between endogenous UVRAG and C-Vps subunits. WCLs of 293T cells were used for IP with control serum (control) or an anti-UVRAG antibody, followed by IB with the indicated antibodies. The right panel shows endogenous protein expression. (c) Confocal microscopy analysis of the colocalization of endogenous UVRAG and Vps16 or Vps18 in HeLa cells. Insets highlight colocalization. (d) Confocal microscopy analysis of the colocalization of UVRAG with C-Vps subunits in HeLa cells transfected with Flag-UVRAG, and epitope-tagged C-Vps subunits as indicated. (e) Colocalization of UVRAG-C-Vps in early endosomes. HeLa cells were co-transfected with EGFP-Vps16 and Flag-UVRAG, stained with antibodies to LAMP1, LAMP2, EEA1 and Rab5, followed by confocal microscopy. All images are representative of at least three independent experiments. Scale bars, 5 μm. The raw data for a and b are shown in Supplementary Information, Fig. S6 \n",
      ".Figure 5(A) LPS/ATP treated wild type microglia were stained for NLRP3 (green) and CALCOCO2 (red) and analysed by confocal microscopy. Colocalizations of NLRP3 aggregates (arrows) and CALCOCO2 (arrowheads) are apparent; scale bar: 10 µm. (B) LPS/ATP treated wild type microglia were stained for NLRP3 (green) and CALCOCO2 (red) and analysed by super resolution microscopy (SIM). NLRP3 aggregates coclustering with CALCOCO2-positive signals are visible. 3D volume rendering of NLRP3 aggregates from the magnified ROIs showed close contact of NLRP3 with the autophagic receptor CALCOCO2; scale bar: 10 µm; magnified ROIs: 1 µm. (C) 3D Radial intensity profiles of NLRP3 and CALCOCO2 signals derived from SIM images in wild type microglia, centered on the maxima of NLRP3 clusters. The radial profiles confirm that both proteins colocalize; mean ± SEM, n = 3. (D) In SIM images from wild type microglia the sub-autophagosomal localization of NLRP3 was analysed in relation to LC3 and CALCOCO2. LC3 and CALCOCO2 intensities from the center of mass of NLRP3 aggregates were normalised to their maximum within the cluster. While the center of NLRP3 aggregates shows near maximal CALCOCO2 intensity, the relative LC3 intensity there was significantly lower; mean ± SEM, LC3: n = 4, CALCOCO2: n = 3; two-tailed t-test **: p < 0.01. (E) Microglia from Becn1+/- mice were transfected with scrambled siRNA or with siRNA for CALCOCO2. 6 d after transfection cells were stimulated with LPS/ATP. The expression of CALCOCO2 protein was determined by Western blot. Application of CALCOCO2 siRNA resulted in significantly reduced expression of CALCOCO2; mean ± SEM, n = 3; two-tailed t-test *: p < 0.05. (F) The amount of IL-1beta in the supernatant released from the cells described in (E) was determined by ELISA. Microglia with a CALCOCO2 knockdown released significantly more IL-1beta; mean ± SEM, n = 3; two-tailed t-test *: p < 0.05. \n",
      ".Figure 2(A) Flow cytometry dose response curve for cell binding of 4D9 mAb (EC50= 0.29nM), 4D9 Fab (EC50=0.17nM), and isotype to HEK cells stably overexpressing mouse TREM2. Data represent the mean ± SEM (n=2). (B) In vitro ADAM17 sheddase activity is blocked by 4D9-effectorless mAb and 4D9 Fab fragment but not an isotype control. Fluorescence polarization of FAM conjugated TREM2 stalk peptide was detected in the presence or absence of ADAM17 and 4D9 mAb, 4D9 Fab and isotype control. Data represent the mean ± SEM (n=6). One-way ANOVA, Tukey's post hoc test; p (4D9 Fab vs 4D9 mAb) = 0.8855; p (4D9 Fab vs uncleaved) < 0.0001; p (4D9 mAb vs uncleaved) < 0.0001; n.s., not significant. (C) ELISA-mediated quantification of sTREM2 in conditioned media from  (D) AlphaLISA-mediated quantification of pSYK levels in HEK293 Flp-In cells stably overexpressing mouse TREM2 and mouse DAP12 treated with a dose titration of 4D9 mAb, 4D9 Fab or isotype for 5 minutes. Data represent the mean ± SEM (n=3). (E) AlphaLISA-mediated quantification of pSYK levels in HEK293 Flp-In cells stably overexpressing mouse TREM2 and mouse DAP12 or empty vector treated with 1mg/ml POPC/POPS liposomes and 20µg/ml 4D9 mAb, 4D9 Fab or isotype for 5 minutes. pSYK levels were also determined for cells treated with liposomes only. Data represent the mean ± SEM (n=3). Two-way ANOVA, Tukey's post hoc test (cell line effect: F1,16=365.7, p=<0.0001; treatment effect: F3,16=39.35, p<0.0001; cell line x treatment effect: F3,16=38.75, p<0.0001); p (no Ab vs isotype) = 0.6218; p (no Ab vs 4D9 mAb) < 0.0001; p (no Ab vs 4D9 Fab) = 0.7301; p (isotype vs 4D9 mAb) < 0.0001; p (isotype vs 4D9 Fab) > 0.9999; p (4D9 mAb vs 4D9 Fab) < 0.0001; n.s., not significant. (F) Schematic representation of the proposed mechanism of action of antibody 4D9. Binding of 4D9 to TREM2 leads to receptor clustering on the cell surface thereby driving downstream pSYK signaling. At the same time, cell surface levels are enhanced by inhibition of ectodomain shedding, potentially because of the inability of proteases to cleave dimeric substrates. \n",
      ".Figure 7(A, B) Overview of the cortex immunohistochemically stained for Aβ plaques in 6 months old APP-NL-G-F mice treated with isotype control (A) and 4D9 antibody (B). Scale bar= 100µm. (C) Higher magnification images of cortical Aβ plaques in APP isotype and 4D9 injected mice. Scale bar= 10µm. (D) Quantification of cortical plaque area from stainings shown in C. Mann-Whitney U test, p=0.0082. (E) Quantification of total plaque coverage of Aβ stainings in the cortex shown in C. Mann-Whitney U test, p=0.014. (F) Immunoblot analysis of soluble (DEA fraction, top) and insoluble Aβ levels (FA fraction, bottom) in brains of APP-NL-G-F mice treated with either isotype control or 4D9 antibody. While a reduction in levels of soluble Aβ is evident from the DEA fractions no change in levels of insoluble Aβ could be detected in the FA fractions. Note that the higher background in the DEA immunoblot is due to a much longer exposure time, which was needed to visualize soluble Aβ. (G) MSD ELISA quantification of soluble Aβ42 in DEA extracts as shown in (F). Mann-Whitney U test, p=0.026. (H) MSD ELISA quantification of insoluble Aβ42 in FA extracts as shown in (F). Unpaired t-test (two-tailed) with Welch's correction; p=0.0877. \n",
      ".Figure 4(A) Flow cytometry detection of cell surface 4D9 (red) binding and isotype control (blue) on primary wild type and Trem2-/- mouse microglia. Data are shown as MFI (mean fluorescent intensity). (B) Uptake assay for fluorescently labelled myelin, Aβ(1-42) and inactivated E.coli particles. The top row shows that myelin and Aβ (pseudocoloured in white) accumulate within the plasma membrane of primary microglia (labeled with dylight-649 isolectin and pseudocoloured in red). Overnight pre-treatment with antibody 4D9 significantly increased the percentage of substrate positive cells (middle row). Pre-treatment with the isotope control did not affect the uptake rate (bottom row). No changes in the uptake of E.coli particles were observed. Substrate uptake in the absence of antibody is shown in the top row. Hoechst 33342 was used to counter stain the nuclei (in cyan) and to assess cell density. Scale bar= 20µm.\n",
      ".Figure 3(A) Flow cytometry detection of cell surface binding of 4D9 (blue) and isotype (green) to wild type and Trem2-/- mouse bone marrow derived macrophages. Representative histograms are shown as MFI (mean fluorescent intensity). Graphs represent the median signal intensity ± SEM (n=3). Student's t-test (two-tailed); Trem2+/+: p = 0.0005; Trem2-/-: p = 0.4435; n.s., not significant. (B) Immunoblot analysis of membrane fractions of bone marrow derived macrophages (BMDM) upon treatment with 4D9 antibody reveals increased levels of membrane-bound TREM2 relative to an isotype antibody. BMDM from TREM2 knockout mice were included to show the specificity of the anti-TREM2 antibody. Calnexin served as a loading control. Levels of membrane-bound TREM2 were quantified by MSD ELISA. TREM2 antibody clone 5F4 was used for detection. Data represent the mean ± SEM (n=5-6). Unpaired t-test (two-tailed) with Welch's correction; p<0.0001. No gender-specific differences could be observed. (C) Immunoblot analysis of sTREM2 in conditioned media from BMDM upon treatment with 4D9 and isotype antibodies. BMDM from TREM2 knockout mice were included to show the specificity of the anti-TREM2 antibody. Soluble APP served as a loading control. Note that heavy and light chains of the antibodies used for treatment are also detected and annotated as follows: Ab hc= antibody heavy chain; Ab lc= antibody light chain. Levels of sTREM2 were quantified by MSD ELISA. TREM2 antibody clone 5F4 was used for detection. Data represent the mean ± SEM (n=6). Unpaired t-test (two-tailed) with Welch's correction; p<0.0001. No gender-specific differences could be observed. (D) Wildtype, Trem2+/-, and Trem2-/- BMDMs were plated in low concentration of M-CSF on 4D9 or isotype coated wells for 5 days and cellular ATP levels were measured by luminescence detection to indicate cell viability. 4D9 and isotype data for each genotype represent the mean ± SEM (n=2 and n=1, respectively). \n",
      ".Figure 1(A) Schematic representation of TREM2 processing by ADAM10/17. Cleavage occurs C-terminal of residue His157. The entire ectodomain (residues 19-171) was used for immunization of rats to generate TREM2 antibodies. sTREM2 (soluble TREM2), CTF (C-terminal fragment). (B) Immunoblot analysis of membrane fractions of HEK293 Flp-In cells stably overexpressing both mouse TREM2 and mouse DAP12 upon treatment with 4D9 antibody reveals increased levels of membrane-bound TREM2 similar to what can be achieved by ADAM protease inhibition using the GM6001 inhibitor. An isotype antibody was used as a negative control. Calnexin served as a loading control. Levels of membrane-bound TREM2 were quantified by MSD ELISA. Data represent the mean ± SEM (n=3). One-way ANOVA, Tukey's post hoc test; p (DMSO vs GM) = 0.0011; p (DMSO vs isotype) = 0.992; p (isotype vs 4D9) = 0.0005; n.s., not significant. (C) Immunoblot analysis of conditioned media from HEK293 Flp-In cells stably overexpressing both mouse TREM2 and mouse DAP12 upon treatment with 4D9 antibody reveals decreased levels of sTREM2 similar to what can be achieved by ADAM protease inhibition using the GM6001 inhibitor. An isotype antibody was used as a negative control. sAPPα served as a loading control. Note that heavy and light chains of the antibodies used for treatment are also detected and annotated. Levels of sTREM2 were quantified by MSD ELISA. Data represent the mean ± SEM (n=3). One-way ANOVA, Tukey's post hoc test; p (DMSO vs GM) < 0.0001; p (DMSO vs isotype) = 0.6372; p (isotype vs 4D9) < 0.0001; n.s., not significant. (D) 4D9 antibody selectively detects TREM2 on the cell surface of HEK293 Flp-In cells stably overexpressing mouse TREM2 and mouse DAP12. An anti-HA antibody was used as a positive control while empty vector transfected HEK293 Flp-In cells were used as a negative control. Scale bar= 10µm. (E) Peptide ELISAs detect anti-mouse TREM2 antibody binding to tiled stalk region peptides, full length stalk peptide or a truncated ADAM cleavage site peptide. The binding epitope of 4D9 antibody is located 12 amino acids N-terminal of the ADAM cleavage site at His 157. (F) Sequence comparison of mouse TREM2 and human TREM2 shows substantial sequence conservation around the 4D9 epitope (upper panel). Immunoblot analysis demonstrates that antibody 4D9 is highly specific for mouse TREM2 and does not detect human TREM2 or mouse TREM1 (lower panel). (G) 4D9 binding to mouse TREM2 ECD is competed off by a stalk region peptide. A competition ELISA demonstrates that a dose titration of stalk peptide reduces binding of 4D9 to TREM2 ECD with an EC50 of 1.3uM. Data represent the mean ± SEM (n=3). (H) Surface plasmon resonance binding kinetics of increasing concentrations of 4D9 antibody to mouse TREM2 ectodomain evaluated by BIAcore, Kon = 5.9 x 105 M-1s-1, Koff = 4.0 x 10-5 s-1, KD = 68 pM. 4D9 binding to human TREM2 or mouse TREM1 was undetectable. \n",
      ".Figure 5(A) 4D9 demonstrates standard IgG pharmacokinetics in vivo. Peripheral clearance rates of 4D9 on a human IgG effectorless backbone compared to an isotype control were determined in wild type mice by hIgG ELISA detection of plasma antibody concentrations 1 and 24 hours, 4 and 7 days after 10 mg/kg intravenous injection of antibody. (B) Brain antibody concentration was measured 24hrs post intravenous dosing of isotype hIgG at 100mg/kg and 4D9-hIgG dosed intravenously at 100, 50 and 10mg/kg. Detection of hIgG levels by ELISA demonstrated dose dependent brain concentrations of 4D9 in the single digit nM range. Animals were perfused to minimize IgG contribution from the plasma. Data represent the mean ± SEM (n=5). (C) Schematic diagram depicting the target engagement assay setup for bound and total sTREM2 detection. For the bound assay, a secondary anti-human IgG detects 4D9 antibody bound to soluble TREM2 in CSF and plasma. For the total assay, a saturating amount of 4D9 antibody is added to eliminate unbound sTREM2. The ratio of 4D9-bound sTREM2 to total sTREM2 is calculated to determine the level of target engagement achieved by the concentration of antibody present. (D) Target engagement timecourse demonstrated near 100% 4D9-bound sTREM2 in CSF of wildtype mice at 24 hrs post dose. Over a 10 day timecourse with timepoints at day 1, 2, 4, 7 and study termination at day 10, the bound/total sTREM2 reduces gradually to reach ~50% by day 10. Animals were dosed intravenously with 50mg/kg of isotype and 4D9 antibodies. Data represent the mean ± SEM (n=5). (E) Target engagement dose response demonstrated saturated bound sTREM2 at 50 and 10mg/kg with >50% bound sTREM2 at 1mg/kg of antibody. 4D9-bound sTREM2 was undetectable at 0.1mg/kg and lower. 4D9 was IV-dosed at 50, 10, 1, 0.1, 0.01, 0.001, 0.0001 mg/kg, and isotype dosed at 50mg/kg. CSF bound: total sTREM2 in wildtype mice was measured at 24hrs. Data represent the mean ± SEM (n=5). (F) 4D9 antibody demonstrates dose dependent increase of total brain TREM2 levels. Quantification of total TREM2 in brain lysates from wildtype mice dosed with 4D9 or isotype control was performed by a MSD-platform based ELISA. Data represent the mean ± SEM (n=5) and are shown as pg TREM2 per ug of total protein. One-way ANOVA, Dunnett's post hoc test, p (isotype vs 4D9 (100mg/kg)) < 0.0001; p (isotype vs 4D9 (50mg/kg)) = 0.0007. \n",
      ".Figure 6(A) Schematic outlining study design and timeline of intraperitoneal injections of either isotype control of 4D9 antibody in 6 months old APP-NL-G-F and age-matched WT mice. (B) Immunohistochemical costainings of TREM2 (magenta) and IBA1 (green) microglia in the cortex of isotype control and 4D9 injected WT and APP-NL-G-F mice. Side panel shows images from each staining at a larger magnification indicated by the dotted white boxes. Scale bar = 10µm; scale bar (inset) = 2.5µm. (C) Quantification of cortical TREM2 stainings shown in B. Two-way ANOVA, Tukey's multiple comparison test (Genotype effect: F1,23=70.63, p=<0.0001; Treatment effect: F1,23=14.33, p=0.0010; Genotype x Treatment effect: F1,23=14.51, p=0.0009); p (WT isotype vs WT 4D9) > 0.9999; p (WT isotype vs APP isotype) = 0.02; p (WT 4D9 vs APP 4D9) < 0.0001; p (APP isotype vs APP 4D9) = 0.0001; n.s., not significant. (D) Quantification of cortical IBA1 staining shown in B. Two-way ANOVA, Tukey's multiple comparison test (Genotype effect: F1,21=215.4, p=<0.0001; Treatment effect: F1,21=0.5994, p=0.4475; Genotype x Treatment effect: F1,21=0.2992, p=0.5902); p (WT isotype vs APP isotype) < 0.0001; p (WT 4D9 vs APP 4D9) < 0.0001; p (APP isotype vs APP 4D9) = 0.9986; n.s., not significant. (E) Confocal images of P2RY12 (red), IBA1 (green), and Aβ (gray) costainings from cortex. Top panel: Dotted white boxes indicate the area in P2RY12 and IBA1 costainings that are magnified as inset. Bottom panel: Insets show P2RY12, IBA1 and Aβ null at a larger magnification. Of note, we did not observe a complete co-localization of P2RY12 and null suggesting different microglial populations. Scale bar= 10µm; scale bar (insets) = 5µm. (F) Quantification of P2RY12 stained cells in the cortex shown in E. Two-way ANOVA, Tukey's multiple comparison test (Genotype effect: F1,23=71.99, p=<0.0001; Treatment effect: F1,23=9.029, p=0.0063; Genotype x Treatment effect: F1,23=5.93, p=0.0231); p (WT isotype vs APP isotype) = 0.0018; p (WT null vs APP 4D9) < 0.0001; p (APP isotype vs APP 4D9) = 0.0051. \n",
      "At an early stage of autophagy, UVRAG targets Beclin1 to facilitate autophagosome formation, whereas at the later stages, UVRAG interacts with C-Vps to promote autophagosome maturation.figf8At an early stage of autophagy, UVRAG targets Beclin1 to facilitate autophagosome formation, whereas at the later stages, UVRAG interacts with C-Vps to promote autophagosome maturation. UVRAG-C-Vps is also involved in the endosome-lysosome transition by activation of Rab7. \n",
      "a) HeLa.figf3(a) HeLa.Vec and HeLa.UVRAG cells transfected with HA-Vps16 and GFP-LC3 were treated with 2 μM rapamycin for 2 h, and processed for confocal microscopy (left panel; scale bars, 5 μm). The percentage of GFP-LC3 punctae positive for HA-Vps16 staining was quantified (right panel; data are mean ± s.e.m., n = 60, *P 0.01). (b) Effects of Vps16 on UVRAG-mediated autophagosome formation. Light microscopic quantification of autophagosomes in HeLa.Vec and HeLa.UVRAG cells (upper left panel) or in HCT116.Vec and HCT116.UVRAG cells (bottom left panel) when transfected with GFP-LC3 together with control siRNA or Vps16 siRNA (data are mean ± s.e.m., n = 60 for HeLa cells; n = 450 for HCT116 cells, *P 0.01; **P 0.001). Immunoblotting of Vps16 and tubulin are shown in the right panel. \n",
      "a) UVRAG enhances the colocalization efficiency of GFP–LC3 with LAMP1.figf4(a) UVRAG enhances the colocalization efficiency of GFP-LC3 with LAMP1. HeLa.Vec and HeLa.UVRAG cells transfected with GFP-LC3 were either untreated or treated with rapamycin (2 μM) in the absence or presence of bafilomycin A1 (0.1 μM) and stained for LAMP1. Insets highlight the colocalization. The percentage of LAMP1-positive autophagosomes was calculated in each setting (data are mean ± s.e.m., n = 200; 5 independent experiments, **P 0.01; *P 0.05). (b) UVRAG enhances the acquisition of CD63 by autophagosomes. The GFP-LC3-transfected HeLa.Vec and HeLa.UVRAG cells were treated as described in a and stained for CD63 and the percentage of CD63-positive autophagosomes was calculated (data are mean ± s.e.m., n = 200, **P 0.01; *P 0.05). (c) UVRAG promotes autophagic degradation of long-lived proteins. HeLa.Vec and HeLa.UVRAG cells were incubated for 16 h with L-3H-Leu (1 μCi ml-1). The degradation of long-lived proteins was measured at the indicated time points in complete medium (NC), EBSS alone (Starv.), or EBSS + 0.1 μM bafilomycin A1 (Starv.+BalfA1). Results (data are mean ± s.e.m. of triplicates) are representative of 2 independent experiments. Scale bars, 5 μm. \n",
      "a) Colocalization of GFP–LC3 and LAMP1.figf5(a) Colocalization of GFP-LC3 and LAMP1. HeLa cells expressing wild-type or mutant UVRAG were transfected with GFP-LC3 and incubated under normal conditions or treated with rapamycin (2 μM). Autophagosome fusion with LAMP+ structures was analysed by confocal microscopy (left panel; rapamycin-treated) and quantified (right upper panel; data are mean ± s.e.m., n = 100). Arrows and insets show the LAMP1+ autophagosomes. The number of GFP-LC3-positive dots per cell was counted using a fluorescence microscope (right bottom panel; data are mean ± s.e.m., n = 60, *P 0.05; **P 0.0001). (b) Vps16 or Vps18 knockdown inhibits UVRAG-mediated autophagosome maturation. HeLa.Vec (left) and HeLa.UVRAG (right) cells were transfected with GFP-LC3 together with control siRNA, Vps16 siRNA, or Vps18 siRNA, followed by confocal microscopy. Arrows in the top panel denote the autophagosomes with LAMP1 staining. Immunoblotting of Vps16, or Vps18 and tubulin are shown in the middle panel. The percentage of LAMP1+ autophagosomes was calculated (data are mean ± s.e.m., n = 50, *P 0.05). Scale bars, 5 μm. The raw data of the immunoblots are shown in Supplementary Information, Fig. S6. \n",
      "a) The N-terminal C2 domain or the C-terminal region of UVRAG interacts individually with Vps16.figf2(a) The N-terminal C2 domain or the C-terminal region of UVRAG interacts individually with Vps16. 293T cells were co-transfected with HA-Vps16, together with vector, Flag-UVRAG or Flag-UVRAG mutants. WCLs were immunoprecipitated with anti-Flag followed by immunoblotting with an anti-HA antibody. (b) HeLa cells were transfected with EGFP-Vps16 together with Flag-UVRAG mutants as indicated and stained with anti-Flag (red), followed by confocal microscopy. Scale bars, 5 μm. (c) Interactions of the UVRAG C-terminal truncated mutants with Vps16. At 48 h post-transfection with HA-Vps16 and Flag-UVRAG C-terminal mutants, 293T WCLs were immunoprecipitated with anti-Flag, followed by immunoblotting with anti-HA. (d) UVRAG C2 domain or C-terminal region deletion abolishes Vps16 binding. At 48 h post-transfection with HA-Vps16 together with Flag-UVRAG or its mutants, 293T WCLs were immunoprecipitated with anti-Flag followed by immunoblotting with an anti-HA antibody. (e) Schematic representation of UVRAG and its deletion mutants. ++, strong binding; +, weak binding; −, no binding. The raw data for a, c and d are shown in Supplementary Information, Fig. S6 \n",
      "a) UVRAG expression promotes the recruitment of Rab7 GTPase to autophagosomes.figf6(a) UVRAG expression promotes the recruitment of Rab7 GTPase to autophagosomes. HeLa.Vec and HeLa.UVRAG cells transfected with GFP-Rab7 and RFP-LC3 were maintained either under normal conditions (NC) or treated with rapamycin, followed by confocal microscopy. Insets highlight the Rab7 acquisition of autophagosomes. The percentage of Rab7+-autophagosomes was quantified (right panel; data are mean ± s.e.m., n = 60, ***P 0.001; **P 0.01). Scale bars, 5 μm. (b) Rab7 GTPase activity is increased with UVRAG expression. At 48 h post-transfection with GFP-Rab7 together with vector, UVRAG or UVRAGΔN(1-147) vector, 293T WCLs were used for immunoprecipitation with anti-GFP, followed by the Rab7 GTPase activity assay. Left panel: Autoradiographs of the GTP hydrolysis products analysed by TLC. Right panel: Quantification of the percentage of GTP hydrolyzed by Rab7 (data are mean n = 2 independent experiments). (c) Increased Rab7-RILP interaction UVRAG is expressed. HCT116.vector, HCT116.UVRAG and HCT116.UVRAGΔN(1-147) cells were transfected with GFP-Rab7 together with the GST-tagged Rab7 binding domain of RILP (GST-RILPRBD, residues 243-318). WCLs were used for GST pulldown, followed by immunoblotting with an anti-GFP antibody. The raw data of the immunoblots are shown in Supplementary Information, Fig. S6. \n",
      "a) Electron micrographs of the IM (black arrowheads) associated with the ER (black arrows) from Atg4BC74A-expressing NIH 3T3 cells.figf1(a) Electron micrographs of the IM (black arrowheads) associated with the ER (black arrows) from Atg4BC74A-expressing NIH 3T3 cells. The IM appears to expand between the two ER cisternae and forms an autophagosome. The white arrowhead indicates vesicular clusters visible in the vicinity of the IM. The white arrow indicates the autophagosome. (b) The ratio of ER-associated autophagic structures (ER-IM complexes) to total autophagic structures. Data are expressed as the mean ± s.d. of triplicates from representative experiments. (c) Localization of GFP-DFCP1 with Atg16L1 in both control and Atg4BC74A-expressing cells. Insets show enlarged images of boxed regions. (d-g) Gold-enhanced immunogold electron microscopy revealed that GFP-DFCP1 localized to ER-IM complexes in both control (d, e) and Atg4BC74A-expressing cells (f). Immunogold particles are indicated by small arrows. No specific labelling was observed on ER-IM complexes in cells expressing GFP (g). Large arrows and arrowheads indicate the ER and the IM, respectively. Scale bars, 200 nm in a and d-g, an d 10 μm in c. \n",
      "a) EGF-stimulated endocytosis and post-endocytic trafficking.figf7(a) EGF-stimulated endocytosis and post-endocytic trafficking. Uptake of Alexa Fluor 488-EGF (green) by HCT116.Vec (left panel) and HCT116.UVRAG (right panel) cells was followed by confocal microscopy. At the indicated times, cells were fixed and stained with antibodies to LAMP1 (red), and Topro-3 (blue, to stain the nucleus). Magnified images of the insets (fourth row) highlight trafficking of EGF towards LAMP1-positive structures. (b) EGFR degradation. HCT116.Vec, HCT116.UVRAG and HCT116.UVRAGΔN(1-147) cells were treated with EGF (200 ng ml−1) at 37 °C for the indicated times. WCLs were subjected to immunoblotting with an anti-EGFR antibody. EGFR at each time point was quantified and indicated as a percentage relative to that at time zero (right panel; data are mean ± s.e.m., n = 3). (c) DQ-green BSA dequenching assay. Representative confocal microscopy images of DQ-BSA proteolysis (left panel) and flow cytometry analysis (right panel) of RAW264.7 cells expressing an empty vector (RAW264.7.Vec; dotted black line), wild-type UVRAG (red line) or UVRAGΔN(1-147) mutant (green line) loaded with 10 μg ml−1 DQ green-BSA for the indicated times. Background (grey line) represents samples without the BSA load. (d) Effect of UVRAG on endosome fusion. RAW264.7.Vec and RAW264.7.UVRAG cells were pulse-labelled with Oregon green 514-avidin and dextran-568 for 10 min and chased for 30 min. Cells were then labelled with biotin-BSA for 10 min, and processed for confocal microscopy (left panel). In the absence of biotin-BSA, Oregon green514-avidin showed very low fluorescence (first row of confocal image). The percentage of dextran-568-labelled endosomes with visible Oregon green staining (denoted by arrows in the left panel) was calculated from approximately 20 different cells in 2 independent experiments (mean, right panel). Scale bars, 5 μm. The raw data of the immunoblots in b are shown in the Supplementary Information, Fig. S6. \n",
      "a–c) 3D models of three different ER–IM complexes.figf3(a-c) 3D models of three different ER-IM complexes. The ER, IM and VTC-like compartments are shown in blue or red, yellow and white, respectively. White arrowheads in b indicate the continuity of VTC with the ER. (d) A 3D model of the boxed region in c, shown in a different orientation, depicts the connection between the IM and the associated ER. Red arrowheads indicate a membrane extension. (e) Images show 1.5-nm tomographic slices of the boxed region in c, and depict the IM (black arrow), whose edge (red arrowhead) is connected to the associated ER (black arrowhead). Numbers indicate the depth of the tomographic slices. Scale bars, 200 nm in a, 100 nm in b and c, and 50 nm in d and e. \n",
      "Although the site of IM formation in the ER remains an open question, two possible models are illustrated based on our electron tomographic analysis of ER–IM complexes.figf5Although the site of IM formation in the ER remains an open question, two possible models are illustrated based on our electron tomographic analysis of ER-IM complexes. \n",
      "A 2-nm tomographic slice of each ER–IM complex, reconstructed from a 200-nm thick section, and their 3D models are depicted in the left panels.figf4A 2-nm tomographic slice of each ER-IM complex, reconstructed from a 200-nm thick section, and their 3D models are depicted in the left panels. Red arrows indicate sites at which an edge of the IM is connected to the ER. Boxed regions are enlarged in the right panels. 3D models of the boxed regions, shown in a different orientation, depict the connection between the IM and ER (red arrows). Tomographic slices of the boxed regions show the edge of the IM (red arrowheads) is connected to the ER. Numbers indicate the depth of the tomographic slices. Scale bars, 100 nm. \n",
      "Atg4BC74A-expressing cells (for Atg16L1) or control cells (for PDI, COP II and β-COP) were labelled with specific antibodies against the indicated markers.figf2Atg4BC74A-expressing cells (for Atg16L1) or control cells (for PDI, COP II and β-COP) were labelled with specific antibodies against the indicated markers. The left and right anti-Atg16L1 and anti-PDI images are from two different ER-IM complexes. Arrows, arrowheads and open arrowheads indicate the ER, IMs and VTCs, respectively. Small arrows in the image from the anti-COP II experiment indicate immunogold particles. Scale bars, 200 nm. \n",
      ".Figure 1FACS analysis of the E3: T-GR line and scheme for the evaluation of reversion efficiency. Reversion of E3 EpiSCs from GFP-positive (left) and -negative EpiSCs (right). Macroscopic view of AP-stained dishes on day 5 (top). Quantification of reversion efficiency based on the number of AP-positive colony (middle) and GFP-positive cell (bottom). Top left: Scheme for the chimera contribution assay. Bottom left: Representative brightfield/mCherry images from chimeras at E7.5. Scale bar: 100um. Right: Percentage of mCherry-positive cells in chimeric embryos measured by FACS. Blue circles and red circles represent the percentage of mCherry-positive cells derived from control and T-induced PSCs (GOF-positive E3 EpiSCs [E3G], reverted E3 EpiSCs [rE3], and 2iLIF ESCs), respectively. Western blot of the indicated proteins in T-inducible PSCs stimulated as indicated for 24 hr. GAPDH was used as a loading control. \n",
      ".Figure 2Scheme for the piggyBac-based vectors to induce the T-V5-GR-2A-H2BmCherry protein in OCT4-YFP ESCs. Bottom: FACS analysis of the induced T fusion protein and the OCT4-YFP protein. Western blot of the indicated proteins in sorted T-induced ESCs (Oct4-GFP ESCs [left] and OCT4-YFP ESCs [right]). GAPDH was used as a loading control. Heatmap derived from mean-centered expression values (log2 scale) of selected key pluripotency genes. See also Fig EV2C, Dataset EV1 for all the clustered DEGs. PCA of global gene expression in the four distinct T-induced cell populations as shown in (A). \n",
      " - Diencephalic astrocyte subtypes share gene expression between ependyma and tanycytesA Violin plot showing scaled log normalized read counts of S100a6 expression for each cluster.Figure 3A Violin plot showing scaled log normalized read counts of S100a6 expression for each cluster. Color-coding corresponds to clusters In vivo validation of S100a6 expressing DIE astrocytes in Aldh1l1-eGFP mouse (3 mo) with immunolabelling for S100a6 (B: in the SEZ, LV: lateral ventricle, CC: corpus callosum. Scale bars: 200 µm , C In vivo validation of S100a6 expressing DIE astrocytes in Aldh1l1-eGFP mouse (3 mo) with immunolabelling for S100a6 C: in the DIE). Inset in left panel of C is depicted in middle and right panels. S100a6/GFP double positive cells are indicated by arrowheads. LV: lateral ventricle, CC: corpus callosum. Scale bars: 200 µm left panel C), 50 µm (middle and right panels C). \n",
      " - Astrocyte diversity in the diencephalon revealed by scRNA-seqA Scheme depicting the improved dissected region (in blue) from 2 - 3 mo old Aldh1l1-eGFP mice (drawing left).Figure 1A Scheme depicting the improved dissected region (in blue) from 2 - 3 mo old Aldh1l1-eGFP mice (drawing left). Schemes to the right depicting the column-based magnetic cell selection using first nano-sized beads for cell depletion of myelin containing material followed by positive cell selection for ACSA-2 cells. Cells isolated by ACSA-2 magnetic associated cell separation were used for scRNA-seq using 10xGenomics strategy. t-SNE projection of 25761 single-cell transcriptomes with clusters color coded and annotated post hoc based on their transcriptional profile identities Cell numbers in the different clusters are: Cluster 0 = 6564, Cluster 1 = 4617, Cluster 2 = 4147, Cluster 3= 4053, Cluster 4 = 1375, Cluster 5 =1033, Cluster 6 = 991, Cluster 7 = 663, Cluster 8 = 633, Cluster 9 = 414, Cluster 10 = 333, Cluster 11 = 222, Cluster 12 = 207, Cluster 13 = 188, Cluster 14 = 154, Cluster 15 = 85, Cluster 16 = 82. For all differentially expressed genes per cluster of scRNA-seq data from diencephalic astrocytes after ACSA2 MACS following an optimized dissection expression of representative astrocyte specific marker genes (numbers reflect scaled log normalized read counts for the specified gene for each cell, levels indicated on the right; purple: high expression, grey: low to no expression). expression of representative ependyma (D) specific marker genes (numbers reflect scaled log normalized read counts for the specified gene for each cell, levels indicated on the right; purple: high expression, grey: low to no expression). \n",
      " - Spatial gene expression mapping of diencephalic astrocyte clustersA-Q Mapping of the single cell data shows different distribution of the respective clusters.Figure 2Mapping of the single cell data shows different distribution of the respective clusters. Astrocyte clusters 0 - 4, 9 and 10 are shown in panels A - E and J, K, respectively. Data information: Color-coding bars represent degree of gene expression similarity between all cells from given cluster and mixture of all cells from given single pixel of Slide-seq data. Scale bars: 300 µm Mapping of the single cell data shows different distribution of the respective clusters. Data information: Color-coding bars represent degree of gene expression similarity between all cells from given cluster and mixture of all cells from given single pixel of Slide-seq data. Scale bars: 300 µm R Schematic of mouse coronal brain section. CP: caudatoputamen; CTX: cerebral cortex; HC: hippocamus; HY: hypothalamus; int: internal capsule; LV: lateral ventricle, TH: thalamus. Data information: Color-coding bars represent degree of gene expression similarity between all cells from given cluster and mixture of all cells from given single pixel of Slide-seq data. Scale bars: 300 µm \n",
      ".Figure 3ChIP-seq tracks of T (in red; input is in blue) around the Oct4 (7.5 kb). DE, distal enhancer; PE, proximal enhancer. The T-binding peak around the Oct4-DE is indicated (arrowhead). Right: ChIP-qPCR for the enrichment of T on the DE and the regions indicated in Fig EV3E. NC, negative control (the locus without T binding). The enrichment is indicated by the percent input with SDs (three independent experiments with two technical replicates). Luciferase assay with T-binding elements (TBEs) around the indicated loci as shown in (A) and Fig EV3E. Relative luciferase activities of sorted cell fractions are shown with SDs. (three independent experiments with two technical replicates). : PCA of global gene expression in low T-inducible ZHBTc4 ESCs stimulated as indicated in Fig EV3G. Right: Heatmap of selected key pluripotency genes derived from mean-centered expression values (log2 scale). Venn diagrams showing the overlap between down- and upregulated DEGs induced by both high T induction and Oct4 deletion in low T-activated condition (fold-change > 1; p < 0.1 by DESeq2). Bottom: Ontology terms enriched in overlapped DEGs. See also Dataset EV3. \n",
      ".Figure 4Heatmap of k-means clustering derived from mean-centered expression values (log2 scale) of DEGs selected from 6000 DEGs (Fig EV4A) obtained from RNA-seq (Fig 2). See also Dataset EV1. FACS analysis of the induced fusion protein (T-V5-GR-2A-H2BGFP) and Blimp1-RFP reporter. Right: Measurements of Blimp1-RFP expression in three different cell fractions are also plotted on the histogram image. Unsupervised hierarchical clustering (correlation-based distance) obtained from RNA-seq in T-induced PGCLCs (T-PGCLCs), in vivo PGCs, and FACS-sorted cell fractions as indicated in Fig 2A-C. PCA of global gene expression in the same dataset as (D). Left: A representative confocal image of a mouse embryo at E7.0 bearing T-GFP and Blimp1-RFP transgenic reporters. Scale bar: 50um. Arrowheads indicate Blimp1-positive nascent germ cells in the posterior proximal epiblast. Right: FACS analysis of dissociated mouse embryos without visceral endoderm. Expression of the indicated genes measured by qPCR in FACS-sorted cells (E) derived from embryos at different developmental stages (E7.0-7.5). Ten FACS-sorted cells in different cell fractions (E) were used for each individual qPCR analysis with two technical replicates. The ∆Ct values from the average Ct values of Arbp and Ppia are shown (n≥3, log2 scale). P values were calculated by two-tailed unpaired t-test. \n",
      ".Figure 5ChIP-seq tracks of T (in red; input is in blue) and core pluripotency factors (in green) around the Blimp1 (100 kb) and Prdm14 (100 kb). The co-bound locus (L#1-4) and the locus exclusively bound by core pluripotency factors but not T (L#5) are indicated by arrow. NC, negative control (the locus without T binding). ChIP-qPCR for the enrichment of the indicated factors on the loci (L#1-5) indicated in (A). The enrichment is indicated by the percent input with SDs (three independent experiments with two technical replicates). Luciferase assay with the specific sequence of the locus bound by T around L#1 and L#3 as shown in (A). Relative luciferase activities of low T-inducible ZHBTc4 ESCs stimulated as indicated are shown with SDs. (three independent experiments with two technical replicates). Expression of Blimp1 and Prdm14 measured by qPCR in PGCLCs derived from indicated ESC lines. The ∆Ct values from the average Ct values of Arbp and Ppia are shown (log2 scale, the mean value of two independent experiments with two technical replicates). Top: Oct4-GFP (green) reporter expression (left), and Blimp1-RFP (red) and Oct4-GFP (BROG) reporter expression (right) in the gonads of E13.5 embryos generated by 4N complementation with the indicated ESC lines. Scale bar: 100um. Bottom: percentage of Oct4-GFP reporter-positive PGCs (left) and BROG reporter-positive PGCs (right) measured by FACS analysis. Blue circles, red squares, and green triangles represent the percentage of gonadal PGCs in the first, second, and third independent mutant line, respectively, of the examined locus (L#1-5). Model indicating the inductive germ cell segregation achieved by a co-regulatory mechanism of residual pluripotency regulators and lineage regulators. \n",
      " - GO term enrichment analysis of astrocyte clustersA - G Graphs showing top 10 enriched GO terms for each astrocyte cluster annotated in B.Figure 4A - G Graphs showing top 10 enriched GO terms for each astrocyte cluster annotated in Figure 1B. For all significantly enriched GO terms (analyzed by Gorilla) \n",
      " - Wide-spread expression of proliferation-associated genes in diencephalic astrocytesA, B t-Distributed stochastic neighbor embedding visualization of ACSA-2-MACSed cell populations of representative well-known proliferation genes (A; numbers reflect scaled log normalized read counts for each cell, Dataset EV3 for A, B) and associated violin plot (B) showing distribution of proliferative genes across all clusters (cluster color coding and annotation see in Fig.Figure 5A, B t-Distributed stochastic neighbor embedding visualization of ACSA-2-MACSed cell populations of representative well-known proliferation genes (A; numbers reflect scaled log normalized read counts for each cell, and associated violin plot (B) showing distribution of proliferative genes across all clusters (cluster color coding as and value per each single cell (A). C, D Velocity analysis based on differences between spliced and unspliced transcripts for all cell clusters (C) and astrocyte clusters (D). \n",
      " - Diencephalic astrocytes give rise to progeny in vivoA-E Micrographs of maximum intensity projections (A - D) or single optical section (E) of forebrain sections stained as indicated for fluorescent proteins in astrocytes labelled in 8 mo (A, B, E) and 3 mo (C, D) GLASTCreERT2/Confetti mice at 21 dpt.Figure 7A-E Micrographs of maximum intensity projections (A - D) or single optical section (E) of forebrain sections stained as indicated for fluorescent proteins in astrocytes labelled in 8 mo (A, B, E) and 3 mo (C, D) GLASTCreERT2/Confetti mice at 21 dpt. Note that clusters of duplets (D) and triplets (C) labelled in the same color can be found only in the DIE. Also note that astrocyte somata are often very close together, as also observed after live in vivo imaging of dividing astrocytes Somata of single cells are highlighted in circles. Data information: CTX: cerebral cortex; DIE: diencephalon; HC: hippocampus. Scale bars: 300 µm (A), 100 µm (B), 50 µm (C, D), 20 µm (E). \n",
      " - Proliferation of diencephalic astrocytes in the adult mouse brainA-G Micrographs of maximum intensity projections of forebrain sections from 3 mo old GLASTCreERT2/eGFP mice 21 days after tamoxifen treatment (A-D) and 3 mo old C57BL/6J mice after EdU application (experimental setting see in E) for 4 weeks in drinking water (F) immunostained as indicated on top of the panels.Figure 6Micrographs of maximum intensity projections of forebrain sections from 3 mo old GLASTCreERT2/eGFP mice 21 days after tamoxifen treatment Double-positive cells are indicated by arrowheads. CTX: cerebral cortex; DIE: diencephalon; HC: hippocampus. Scale bars: 300 µm (A), 20 µm (B), 75 µm (C), 30 µm (D), Histograms showing quantifications (B, n = 6 animals; 5 region of interest (ROI) covering the diencephalon on a total of 3 slides analyzed per animal Data information: data are presented as mean ± SEM. Each data point represents one animal. 3 mo old C57BL/6J mice after EdU application (experimental setting see in E) for 4 weeks in drinking water Micrographs of maximum intensity projections of forebrain sections from 3 mo old C57BL/6J mice after EdU application for 4 weeks in drinking water (F) immunostained as indicated on top of the panels. Double-positive cells are indicated by arrowheads. CTX: cerebral cortex; DIE: diencephalon; HC: hippocampus. Scale bars: 10 µm (F). G) Histograms showing quantifications for n = 5 animals; 5 ROI covering the diencephalon on a total of 2 slides analyzed per animal). Data information: data are presented as mean ± SEM. Each data point represents one animal. \n",
      " - RNA-seq of astrocytes from adult mouse cerebral cortex grey matter and diencephalonA Schematic drawing of Ribotag assisted mRNA isolation from astrocytes isolated from regions dissected as indicated in red (CTX GM) and blue (DIE).Figure 8A Schematic drawing of Ribotag assisted mRNA isolation from astrocytes isolated from regions dissected as indicated in red (CTX GM) and blue (DIE). B Micrographs of forebrain sections of 3 mo old GLASTCreERT2/RPL22HA/eGFP 21 dpt mice stained for HA show maximum intensity projections of HA+ expressing cells with astroglial morphology. CTX GM: cerebral cortex grey matter; DIE: diencephalon. Scale bars: 100 µm. C Micrographs of forebrain sections of 3 mo old GLASTCreERT2/RPL22HA/eGFP mice 21dpt stained for HA and S100β showing the specificity of the mouse line with HA detected only in S100β+ astrocytes. DIE: diencephalon. Scale bar: 70 μm. Data information: CTX GM: cerebral cortex grey matter; DIE: diencephalon; n=3 biological replicates, tissue from 3 mice was pooled D Histogram showing expression levels of the top 10 astrocyte specific markers genes in DIE (blue bars) and CTX GM (red bars) samples. Data information: CTX GM: cerebral cortex grey matter; DIE: diencephalon; n=3 biological replicates, tissue from 3 mice was pooled E Principal Component Analysis (PCA) from DeSeq2 expression data shows distinct clusters of samples from 3 DIE (blue) and 3 CTX GM (red) astrocyte sample preparations. F Venn diagram depicting co-expressed and differentially expressed gene numbers in DIE and CTX GM astrocytes G, H Heatmaps depicting top 40 differentially expressed genes (G, row scaled log normalized read counts) or transcription factors (H, Library size normalized log2 transformed read counts; from DIE and CTX GM astrocytes. Transcription factors were identified according to the content of AnimalTFDB database (http://www.bioguo.org/AnimalTFDB/). I Bar chart depicting significantly enriched GO terms (analyzed first by Gorilla followed by Revigo) based on the 379 differentially expressed transcription factor genes (DeSeq2, cutoff pAdj0.01) comparing DIE and CTX GM astrocytes. Terms related to cell division, cell proliferation etc. are highlighted in red. (P-value threshold was set to 0.01 \n",
      ": Generation of iPSC-derived NCC.Figure 2A. Schematic of the differentiation protocol timeline. Maintenance Medium (MM) = iPSC medium (StemFlex with 1x penicillin/streptomycin), NCC differentiation medium = DMEM-F12, 10% fetal bovine serum, 1 mM sodium pyruvate, 1 mM penicillin/streptomycin, 1 mM non-essential amino acids, 110 µM 2-mercaptoethanol, 10 ng/mL epidermal growth factor. B. Images of iPSC and iPSC-derived NCC at Days 0, 14, and passage 4 following differentiation. Scale bars: 400 µm (Day 0), 200µm (Day 14, passage 4). \n",
      ": Timeline of key NCC-associated genes during differentiation.Figure 3Gene expression analysis across NCC differentiation of unaffected control ALX1165L/165L (green), heterozygous ALX1165L/165F (magenta) and homozygous ALX1165F/165F iPSC: ALX1, neural plate border specifier genes ZIC1, PAX7, PAX3, MSX1, MSX2, DLX5; neural crest specifier genes FOXD3, P75, TFAP2A, SNAI2, TWIST1; and lineage specifier gene HAND2. The RT-qPCR relative expression values were normalized to RPLP0 and GAPDH expression. Data is represented as pooled mean ± SEM of three experiments on three clones from each genotype. Exact p-values are provided in Table EV1. Data from each clone were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant. \n",
      ": ALX1165F/165F NCC show a migration defect and a difference in BMP secretion.Figure 5A. Mutant ALX1165F/165F NCC (blue) exhibited a migration defect in timed coverage of the central clearing of the wound assay when compared with control ALX1165L/165L NCC (black). Data is presented as percent area recovery of the central circular clear area of the wound assay by migrating NCC at the end of a 24-hour period. For fluorescent pictures, cells were stained in serum free media containing 3.6µM CellTracker Green CMFDA (Life Technologies) for 30 min at 37°C and allowed to recover for 30 min before starting the experiment. Images were acquired every 6 hours using a Keyence BZ-X800 microscope. Surface area analyses and percentages of coverage were measured using ImageJ software (NIH). The NCC were monitored over 24 hours. The data are represented as the average of the percentage of closure ± SEM. Scale bar = 200µm. Data obtained of each clone from three independent experiments were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant.*: Significantly different from control NCC (p<0.0001). B. Multiplex analysis of BMP2 and BMP9 in the supernatant of cultured NCC showed that ALX1165F/165F NCC (blue) secrete less BMP2 and more BMP9 compared to control ALX1165L/165L NCC (green). Data are represented as pooled mean ± SEM of three clones from each genotype. Statistical significance was determined with an ANOVA test. A p-value <0.05 was considered significant. *: Significantly different from control NCC (p =0.0424 for BMP2 and p=0.0192 for BMP9). C. Addition of soluble BMP2 or CV2, a BMP9 antagonist, to the culture medium could partially rescue the migration defect of ALX1165F/165F NCC. At the beginning of the assay, 10, 50, or 100 ng/ml of soluble BMP2, CV2, or a combination of the two at 100 ng/ml each were added to the culture medium, and the cells were monitored over the next 24 hours. The data are represented as the average of the percentage of closure ± SEM. Scale bar: 400µm. \n",
      " - Smad4 regulates diencephalic astrocyte proliferation and neurosphere formationA Reactome Pathway analysis for all 174 transcription factor genes with higher expression in DIE (upper) and all 205 genes higher in CTX GM (lower) astrocytes.Figure 9A Reactome Pathway analysis for all 174 transcription factor genes with higher expression in DIE (upper) and all 205 genes higher in CTX GM (lower) astrocytes. B t-SNE visualization of DIE astrocytes isolated by ACSA-2 MACS showing Smad4 expression (scaled log normalized read counts). C Example of a Smad4/GFP double-positive astrocyte (indicated by arrowhead) in a single optical section of DIE from 3 mo old GLASTCreERT2/eGFP mice 21 dpt immunostained as indicated on top of the panel. Orthogonal projections of the area highlighted in left panel of C shown at higher magnification in the right panel. Scale bar: 15 µm. D Example of a proliferating (PCNA+, arrowhead) astrocyte in a single optical section of DIE from 3 mo old GLASTCreERT2/eGFP mice 21 dpt immunostained as indicated on top with single channels shown in middle and right panels. Scale bars: 15 µm. E Histogram depicting percentage of PCNA positive cells of all GFP positive cells in the DIE of the genotype indicated on the x-axis (n = 6 animals per genotype; 5 ROI covering the diencephalon on a total of 3 slides were analyzed per animal; PCNA negative and positive cells comparing wt/wt vs. fl/fl animals, adjusted P-value = 0.002). Data information: data are presented as mean ± SEM. Each dot represents one n. * P < 0.05, ** P < 0.01 nonparametric 2way ANOVA followed by Sidak's multiple comparisons test F, G Histograms depicting percentage of the neurosphere-generating cells obtained from the DIE (F) or SEZ (G) of GLASTCreERT2/Smad4fl/fl/eGFP or GLASTCreERT2/Smad4WT/WT/eGFP, as indicated on the x-axis (n = 3-4 animals for SEZ, 4 to 8 technical replicates analyzed per animal; 4 animals for DIE, 8 technical replicates analyzed per animal). Data information: data are presented as mean ± SEM. F: GFP negative and GFP positive cells comparing wt/wt vs fl/fl animals, adjusted P-value = 0.0408; G: GFP negative and GFP positive cells comparing wt/wt vs fl/fl animals, ns). Each dot represents one n. * P < 0.05, ** P < 0.01 nonparametric 2way ANOVA followed by Sidak's multiple comparisons test H Histogram depicting the number of secondary neurospheres generated from 10000 Smad4f/f neurosphere cells at 7 days after infection with MLV-based retrovirus containing either CAG-NLS-Cre or only CAG-GFP (n = 11 cultures / animals, 3 technical replicates per condition analyzed; control vs. Cre, P-value = 0.0149). Note the decrease in DIE astrocyte proliferation in vivo and in neurosphere formation upon Smad4 deletion Data information: data are presented as mean ± SEM. Each dot represents one n. * P < 0.05, ** P < 0.01 nonparametric Mann-Whitney test). \n",
      ": NCC apoptosis, cell cycle and differentiation.Figure 4A. Homozygous ALX1165F/165F NCC (blue) showed an increase in sensitivity to apoptosis when compared to control ALX1165L/165L NCC (black). The data on the left represents the mean percentage of Annexin V positive cells, indicative of apoptosis, as determined by FACS analysis, with the data on the right being an example of one such experiment. Apoptosis was induced by immersion in a 55°C water bath for ten minutes. Representative experiment for each condition is shown. Data is represented as pooled mean ± SEM of three independent experiments. Data obtained of each clone from three independent experiments were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant. . *: Significantly different from the basal apoptosis rate: p=3x10-12 between control NCC basal apoptosis and induced apoptosis, and between ALX1165F/165F NCC basal apoptosis and induced apoptosis. **: Significantly different from control NCC (p=0.0004). B. Expression levels of cyclins CCNA2 (blue) and CCND1 (orange) in NCC at passages 2 and 3 of ALX1165L/165L and ALX1165F/165F NCC. The RT-qPCR relative expression values were normalized to RPLP0 and GAPDH expression. Data is represented as pooled mean ± SEM of three experiments on three clones from each genotype. *: Significantly different from control NCC at passage 2 (p=0.001 between control and ALX1165F/165F NCC at passage 2 for CCNA2, p=0.0052 for CCND1). **: Significantly different from control NCC at passage 3 (p=0.0494 between control and ALX1165F/165F NCC for CCNA2, p=0.0008 for CCND1). C. Fluorescence Activated Cell Sorting (FACS) experiments showed that control ALX1165L/165L NCC (green) exhibited increased expression of mesenchymal markers CD90, CD105, and CD73 with culture time (passages 1 through 4), whereas homozygous ALX1165F/165F NCC (blue) showed a consistent expression of the markers expressed at passage 1 throughout. Further, control ALX1165L/165L NCC showed a downregulation of CD57 expression with culture time, while ALX1165F/165F NCC maintained the same level of CD57 across passages. Data are presented as the mean percentage of positive-stained cells across passage numbers. Data obtained of each clone from three independent experiments were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant. *: Significantly different from control NCC. For CD90, p=0.0013 at passage 3 and p=0.0207 at passage 4. For CD105, p=0.0016 at passage 2, p=0.00004 at passage 3 and p=0.0021 at passage 4. For CD73, p=0.0060 at passage 2, p=0.00004 at passage 3 and p=0.0114 at passage 4. For CD57, p=0.0026 at passage 2, p=0.000003 at passage 3 and p=0.000007 at passage 4. \n",
      ".Figure 1A. The pedigree family tree includes two unaffected parents, 4 unaffected male siblings, 5 unaffected female siblings, and 2 each female and male affected sibling. Subjects 1-6, indicated in red, were enrolled in the study. Subjects 4-6 show complex FND with ocular involvement. The eldest affected sibling (subject 4) presented with right coloboma, left microphthalmia, and bilateral Tessier 4 oblique facial clefts. Subject 5 presented with bilateral anophthalmia with fused eyelids and shallow orbits, with bilateral oblique facial clefts. Subject 6 presented with bilateral anophthalmia with open shallow orbits, absent upper and lower eyelids, exposed orbital mucosa, bilateral oblique facial clefts, and malformed nasal ala with nodular skin tags. iPSCs were generated using blood samples collected from subjects 1, 5, and 6. B. Whole exome sequencing was carried out and analysis revealed a missense p.L165F variant (c.493 C>T) in the ALX1 homeodomain, heterozygous in the parents (ALX1165L/165F), wildtype in the unaffected sibling (ALX1165L/165L), and homozygous in both affected subjects (ALX1165F/165F). Schematic of the ALX1 protein structure showing the position of the L165F substitution described here (red) and the locations of exon borders affected by two reported pathogenic variants (purple)(Ullah et al., 2016, Uz et al., 2010). Schematic of the ALX1 genomic sequence, showing the locations of the three reported pathogenic variants. The purple bar at the bottom represents a FND-associated homozygous ALX1 deletion previously reported in the literature (Ullah et al., 2016, Uz et al., 2010). E. Immunofluorescence staining for pluripotent markers SSEA4, OCT4, SOX2, and TRA-1-60 and alkaline phosphatase staining of iPSC clones. One representative iPSC clone is shown for each genotype. Scale bar: 400 µm. F. Expression of pluripotent (OCT4, NANOG), endoderm (Endo., AFP, GATA4, FOXA2), ectoderm (Ecto., NESTIN, GFAP, SOX1), and mesoderm (Meso., BRACH. (BRACHYURY), RUNX1, CD34) gene markers for ALX1165L/165L (green), ALX1165L/165F(red), and ALX1165F/165F (blue) iPSC relative to undifferentiated cells (UND). Data are represented as pooled mean ± SEM of three experiments on three clones from each genotype. Significance: p=0.0167 for OCT4, p=0.0005 for NANOG, p= 0.000004 for AFP, p=0.0082 for GATA4, p=0.0137 for FOXA2, p=0.00002 for NESTIN, p=0.0167 for GFAP, p=0.0014 for SOX1, p=0.0117 for BRACHYURY, p=0.0008 for RUNX1 and p=0.0068 for CD34 when comparing undifferentiated and differentiated ALX1165L/165L iPSC. p=0.0013 for OCT4, p=0.0011 for NANOG, p=0.0000003 for AFP, p=0.0003 for GATA4, p=0.0063 for FOXA2, p=0.0001 for NESTIN, p=0.027 for GFAP, p =0.000002 for SOX1, p=0.000009 for BRACHYURY, p=3e-9 for RUNX1 and p=0.000006 for CD34 when comparing undifferentiated and differentiated ALX1165F/165L iPSC. p=0.0201 for OCT4, p=0.006 for NANOG, p=1x10-12 for AFP, p=5x10-13 for GATA4, p=0.0031 for FOXA2, p=0.0292 for NESTIN, p=0.00001 for GFAP, p=6x10-7 for SOX1, p=0.0204 for BRACHYURY, p=0.0009 for RUNX1 and p=0.000003 for CD34 when comparing undifferentiated and differentiated ALX1165F/165F iPSC. Data from each clone were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant. \n",
      ": alx1 function in Zebrafish.Figure 6A. Dissected flatmount wild type and alx1-/- zebrafish larvae craniofacial cartilages after Alcian blue staining, the anterior points to the left of the page in all images. The ventral cartilages appear normal, but the alx1-/- anterior neurocranium (ANC) appears narrow, with the midline element that is derived from the frontonasal NCC being absent. The Meckel's cartilage (arrow, MC) is also diminutive. Scale bar: 200 µm. B. Zebrafish alx1 mutants (blue) show reduced detectable expression of alx1 but increased expression of alx3, alx4a compared to wild type controls (green). alx4b expression levels are similar between wild type and alx1-/- lines. Data is represented as the mean of all pooled embryos from three different clutches. The RT-qPCR relative expression values were normalized to elfa and 18S expression using the ΔΔCT method. Data from each clutch were pooled and the mathematical mean was calculated. SEM was used to determine the standard error. To test statistical significance, an ANOVA test was performed. A p-value <0.05 was considered to be statistically significant. Statistical significance denoted by *; p<0.0001 between WT zebrafish and alx1-/- at all measured time points; at 10 ss, 24 hpf and 36 hpf for alx3; and at 24 hpf and 36 hpf for alx4a. Refer to Table EV2 for p-values. C. Dissected flatmount of zebrafish embryos injected with Alx1DN, after Alcian blue staining. The embryos developed an absence of the frontonasal derived median portion of the anterior neurocranium (ANC) and a profound hypoplasia of the Meckel's and ventral cartilages. In the most severely affected zebrafish, a nearly abrogated ANC was observed. Scale bar: 200 µm. D. Lineage tracing experiments in control and Alx1DN mutant embryos revealed aberrant migration of anterior cranial NCC when alx1 is disrupted. In the control animal, the anterior cranial NCC always migrate to contribute to the median portion of the ANC. In contrast, the anterior cranial NCC labeled in the Alx1DN animals fail to migrate to the median ANC, where the ANC structure is narrower and the labeled cranial NCC are found in an anterior and lateral ectopic location (white asterisks). Scale bar: 250µm. \n",
      ".Figure 5A α-Syn foci in WT cells might constitute active sites for propagon formation. Disaggregation in these sites generates smaller α-Syn fragments that are more toxic and more readily transferred to adjacent tissues. As they are also more seeding competent, this leads to an amplification of the aggregated material. The disaggregation of amyloid-type aggregates by the HSP70 system is an important part of a propagation cycle involving repeated fragmentation of larger fibrils to generate and amplify toxic seeding-competent particles. In contrast, the disaggregation of amorphous aggregates leads to the formation of smaller seeding-incompetent fragments, which are eventually completely dissociated. B. In HSP-110 KD animals, the disaggregation activity is impaired. Impaired fragmentation of amyloid-type aggregates leads to reduced foci formation, toxicity and spreading. In contrast, amorphous aggregates persist and increase over time. \n",
      ".Figure 1A The usually well-organized human tubular adenoma organoids (TA) disintegrate upon TGFβ stimulation (scale bars: 200 μm). B Following TGFβ treatment TA organoid cultures display increased levels of cleaved Caspase-3. C Gene expression profiles of TGFβ-treated TA organoids show enrichment in apoptosis-related genes when compared to the control condition. D The pro-apoptotic molecule BIM (BCL2L11) is upregulatd in the TGFβ-treated condition in all TA cultures, besides in TA1, which carries a KRASG12V mutation. \n",
      ".Figure 2A,B Morphological changes in (A) normal colon (N1-3) and (B) surviving TA organoids upon TGFβ treatment resemble the induction of a mesenchymal phenotype in these usually well-organized epithelial structures (scale bars: 50 μm). C Genes present in an EMT signature (Gröger et al. 2012) are enriched in TGFβ-treated compared to control samples. D,E The EMT-inducing transcription factor ZEB1 is strongly induced upon TGFβ treatment both in (D) normal as well as (E) TA organoids. F The mesenchymal marker fibronectin 1 (FN1) is induced following TGFβ stimulation. \n",
      ".Figure 2A Maximum intensity projections of fluorescent microscopy z-stacks of 5-day-old nematodes expressing the indicated transgenes are shown. White dashed lines outline the borders of muscle cells. Scale bar: 10 µm. The co-expression of HPI and HPII reduces the amount of α-Syn::YFP inclusions. B Quantification of α-Syn::YFP foci in 3- to 6-day-old animals. Product of mean foci fluorescence and foci area relative to muscle area is displayed (relFluoFoci). Data are shown as mean ± SEM (in %) (N = 3). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. ** = p ≤ 0.01, *** p ≤ 0.001. C Motility Assay as a measure for transgene toxicity. Displayed is the mean number of body bends per 30 s ± SEM (N = 3). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. ** p ≤ 0.01, *** p ≤ 0.001. D Maximum intensity projections of fluorescent microscopy z-stacks of 6-day-old animals expressing the indicated transgenes. Scale bar: 100 µm. The co-expression of HPI and HPII reduces the amount of Q35::YFP inclusions. E Quantification of Q35::YFP foci in WT and hsp-110 hairpin background. Shown is the mean ± SEM (N = 3). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. * p ≤ 0.05, ** p ≤ 0.01. F Motility Assay as a measure for transgene toxicity. Displayed is the mean number of body bends per 30 s ± SEM (N = 3). Statistical analysis was done using one-way ANOVA with Dunnett's multiple comparisons test. * p ≤ 0.05, ** p ≤ 0.01. G Maximum intensity projections of fluorescent microscopy z-stacks of of 5-day-old nematodes expressing the indicated transgenes. White dashed lines outline the borders of muscle cells. Scale bar: 10 µm. M: muscle, H: hypodermis. Signal outside of muscle cells reveals spreading of α-Syn. H Quantification of animals showing α-Syn transmission at indicated ages. Displayed is the mean ± SEM (in %) (N = 4). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. *** p ≤ 0.001. \n",
      ".Figure 1Single plane confocal fluorescent microscopy images of 5-day-old animals harboring GFP-tagged endogenous HSP-110 (HSP-110::GFP). Muscle cells are outlined. M: muscle, H: hypodermis. In animals that co-express hairpin constructs targeting hsp-110 in muscle cells (HSP-110;HPI and HSP-110;HPII), GFP fluorescence is strongly reduced in muscle cells in comparison to control animals, indicating HSP-110::GFP depletion specifically in muscle cells. Scale bar: 10 µm. Experimental set-up: Age-synchronized 4-day-old animals were subjected to heat stress for 3 h at 33°C (+HS) then returned to 20°C or left at 20°C (-HS), respectively. Arrows indicate imaging time points. The -HS controls were imaged at the same time point as the +HS animals after 24h at 20°C. Maximum intensity projections of fluorescent microscopy z-stacks of animals expressing the indicated transgenes. HS-induced FLUCSM::GFP foci are cleared after a recovery for 24h at 20°C in the WT background, but persist in the presence of the hsp-110 hairpin construct. Scale bar: 20 µm. C Quantification of FLUCSM::GFP foci disaggregation. % disaggregation is calculated as 100 - ratio of FLUCSM::GFP foci area relative to total muscle area at the +HS 24h time point to the +HS 2h time point. Data are displayed as mean ± SEM (in %) (N = 3). Statistical analysis was done using one-way ANOVA with Dunnett's adjustment for multiple comparisons. ** p ≤ 0.01. Data information: All strains harbor the sid-1(pk3321) allele. \n",
      ".Figure 4A Maximum intensity projections of confocal z-stacks of 4-day-old animals expressing the indicated transgenes or harboring the indicated temperature-sensitive (ts) mutations. Muscle cells were stained with anti-myosin (green) and Alexa Flour 647-phalloidin (purple). Scale bar: 5 µm. Myosin misfolding reflects the exposure of the ts mutant phenotype of myosin(ts) [unc-54(e1301)] at the permissive temperate of 15°C by co-expression of the hsp-110 hairpins. B Quantification of motility as a measure for toxicity. Displayed is the mean ± SEM (N = 4) of body bends/30s of animals expressing the indicated transgenes or harboring the indicated mutations. Statistical analysis was done using one-way ANOVA with Dunnett's multiple comparison test. *** p ≤ 0.001. Co-expression of the hsp-110 hairpins exposed the ts mutant phenotype at the permissive temperate of 15°C, resulting in a significant increase in movement defects. All strains contain the sid-1(pk3321) allele. C Quantification of motility as a measure for transgene toxicity. Displayed is the mean ± SEM (N = 4) amount of body bends/30s of animals expressing the indicated transgenes or harboring the indicated mutations. Statistical analysis was done using two-way ANOVA with Tukey's multiple comparison test. * p ≤ 0.05. Expression of the hsp-110 hairpins caused movement defects with increasing age. D Maximum intensity projections of confocal z-stacks of 12-day-old animals expressing the indicated transgenes. Scale bar: 10 µm. FLUCSM forms foci with increasing age, which is markedly accelerated upon hsp-110 KD. E Quantification of animals with FLUCSM::GFP foci at indicated days of life. Data are displayed as mean ± SEM (in %) (N = 3). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. n.s. = not significant, ** p ≤ 0.01, *** p ≤ 0.001. F Quantification of motility as a measure for transgene toxicity. Displayed is the mean ± SEM (N = 3) amount of body bends/30s of animals expressing the indicated transgenes or harboring the indicated mutations. Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. n.s. = not significant, *** p ≤ 0.001. Expression of the hsp-110 hairpins caused movement defects with increasing age. The beneficial effects of HSP-110 KD on amyloid toxicity is masked in older animals. G Quantification of motility as a measure for transgene toxicity. Displayed is the mean ± SEM (N = 3) amount of body bends/30s of animals expressing the indicated transgenes or harboring the indicated mutations. Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. n.s. = not significant, , ** p ≤ 0.01, *** p ≤ 0.001. Expression of the hsp-110 hairpins caused movement defects with increasing age. The beneficial effects of HSP-110 KD on amyloid toxicity is masked in older animals. Data information: All strains harbor the sid-1(pk3321) allele. \n",
      ".Figure 3A Maximum intensity projections of confocal microscopy z-stacks of 5-day-old animals expressing the indicated transgenes. White dashed lines outline the borders of muscle cells. Scale bar: 10 µm. The expression of the yeast SSE1N575Y E575A mutant cannot reverse the reduction of α-Syn::YFP foci in the hsp-110 hairpin expressing animals. B Quantification of motility as a measure for transgene toxicity. Displayed is the mean ± SEM (N = 3) amount of body bends/30s of animals expressing the indicated transgenes. Statistical analysis was done using two-way ANOVA and with Dunnett's multiple comparison test. n.s. = not significant, ** p ≤ 0.01, *** p ≤ 0.001. The presence of the yeast SSE1N575Y E575A mutant was not able to suppress the rescue of α-Syn::YFP toxicity in hsp-110 hairpin expressing animals. C Maximum intensity projections of fluorescent microscopy z-stacks of 5-day-old nematodes expressing the indicated transgenes. White dashed lines outline the borders of muscle cells. M: muscle, H: hypodermis. Scale bars: 10 µm. Signal outside of muscle cells reveals spreading of α-Syn. D Quantification of animals showing α-Syn transmission into the hypodermis at indicated ages. Displayed is the mean ± SEM (in %) (N = 3). Statistical analysis was done using two-way ANOVA with Dunnett's multiple comparison test. n.s. = not significant, *** p ≤ 0.001. E Experimental set-up: Age-synchronized 4-day-old animals were subjected to heat stress for 3 h at 33°C (+HS) then returned to 20°C or left at 20°C (-HS), respectively. Arrows indicate imaging time points. The -HS controls were imaged at the same time point as the +HS animals after 24h at 20°C. Maximum intensity projections of fluorescent microscopy z-stacks of animals expressing the indicated transgenes. Scale bar: 20 µm. The impaired clearance of HS-induced FLUCSM::GFP foci in the hsp-110 hairpin expressing cells is rescued by co-expression of WT but not by the HSP-70 binding deficient SSE1N575Y E575A mutant. F Quantification of FLUCSM::GFP foci disaggregation. % disaggregation is calculated as 100 - ratio of FLUCSM::GFP foci area relative to total muscle area at the +HS 24h time point to the +HS 2h time point. Data are displayed as mean ± SEM (in %) (N = 3). Statistical analysis was done using one-way ANOVA with Dunnett's multiple comparison test. n.s. = not significant, ** p ≤ 0.01. All strains harbor the sid-1(pk3321) allele. \n",
      ".Figure 3A,B FRMD6 expression is strongly induced and CDX2 expression is downregulated both in (A) normal and (B) TA organoid cultures following TGFβ stimulation. C The reduction in CDX2 levels in the TGFβ-treated condition is also observed on the protein level. D,E Gene expression profiles of TGFβ-treated samples show enrichment of genes highly expressed in (D) the mesenchymal CMS4 of CRC (log2FC > 2) and (E) SSA precursor lesions when compared to control samples. \n",
      ".Figure 4A The five most strongly induced genes (right column) upon TGFβ treatment of TA organoid cultures are higher expressed in SSAs when compared to TAs, whereas the five most strongly reduced genes (left column) follow the opposite pattern (n = 12 for SSA and n = 15 for TA. The horizontal lines represent the mean, the error bars display the SD. P-values are based on unpaired two-tailed Student's t-tests.). B Hierarchical clustering of 15 TA and 12 SSA samples using a TGFβ response signature derived from TGFβ-treated TA organoids (log2FC > 3). Expression values were mean-centered (gene-wise) and cosine similarity was used as the distance measure. C Genes induced upon TGFβ treatment of TA organoids, that are differentially expressed between TA and SSA samples, display significantly higher expression levels in CMS4-SSAs compared with CMS1-SSAs (n = 8 for CMS1-SSAs and n = 4 for CMS4-SSAs. The horizontal lines represent the mean, the error bars display the SD. P-values are based on unpaired two-tailed Student's t-tests.). D Gene expression-based classification reveals that SSAs could progress to CMS1 and CMS4 tumors. The data presented in this manuscript suggest that high activity of the TGFβ signaling pathway directs these precursor lesions to the CMS4 of CRC. \n",
      ".Figure 2A Workflow of the bioinformatic identification of candidate rhomboid substrates. B A library of artificial substrates harbouring TMDs from 44 candidate rhomboid substrates. C Western blot analysis of candidate substrates reproducibly cleaved by GlpG from the screen. Data information: In (C rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red, arrows, respectively. Controls, inactive enzymes and wild-type S. sonnei (Ss). D Western blot analysis of candidate substrates reproducibly cleaved by Rhom7 from the screen. Data information: In D) rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, red, and blue arrows, respectively. Controls, inactive enzymes and wild-type S. sonnei (Ss). E Venn diagram summarising putative rhomboid substrates identified by the screen. \n",
      ".Figure 1A Topology of GlpG, Rhom7, and the artificial substrate with a maltose-binding protein (MBP) domain, triple-FLAG tag (3xFLAG), the TMD of P. stuartii TatA, and a thioredoxin domain (Trx). B Western blot analysis (probing with an anti-FLAG mAb) to detect cleavage of the artificial substrate by wild-type (WT)/inactive (S201A) GlpG encoded on pBAD33 with/without arabinose (Ara.). rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red, arrows, respectively. Controls, empty pBAD33 (empty) C Western blot analysis to detect cleavage of the artificial substrate by wild-type (WT)/modified (S133A or H187A) Rhom7 encoded on pBAD33 with/without arabinose (Ara.). , rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, and blue arrows, respectively. Controls, empty pBAD33 (empty) and wild-type S. sonnei (Ss). D Activity of Rhom7 with/without its 7th TMD and/or C-terminal domain. rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, , and blue arrows, respectively. Controls, empty pBAD33 (empty) and wild-type S. sonnei (Ss). \n",
      ".Figure 5A An organoid culture genetically engineered using the CRISPR-Cas9 system to carry a BRAFV600E mutation represents a model for the serrated path to CRC. Upon TGFβ treatment, these culture gains a mesenchymal appearance (scale bars: 200 μm). B,C TGFβ stimulation does not induce (B) cleaved Caspase-3 or (C) BIM expression (n = 6) in the BRAFV600E-mutated organoid culture. B,C TGFβ stimulation does not induce (C) BIM expression (n = 6) in the BRAFV600E-mutated organoid culture. D,E Markers of the EMT program, (D) ZEB1 (n = 6) are strongly induced in the TGFβ-treated condition. D,E Markers of the EMT program, (E) FN1 are strongly induced in the TGFβ-treated condition. F,G FRMD6 is upregulated and CDX2 expression is reduced both on RNA (n = 6) as well as on protein level following TGFβ stimulation. F,G FRMD6 is upregulated and CDX2 expression is reduced both on RNA (n = 6) as well as on protein level following TGFβ stimulation. H TGFβ target genes downregulated upon TGFβ treatment of the BRAFV600E-mutant organoid culture (TGFβ down) are lower expressed in CMS4-SSAs compared to CMS1-SSAs and genes induced by TGFβ treatment (TGFβ up) are more strongly expressed in CMS4-SSAs than in CMS1-SSAs (n = 4 for CMS4-SSAs and n=8 for CMS1-SSAs; FC = fold change). I The differential expression of TGFβ target genes observed for CMS1-SSAs and CMS4-SSAs is maintained at the carcinoma stage for CMS4 versus CMS1 BRAF-mutant CRCs of the TCGA dataset (n = 4 for BRAF-mutant CMS4 and n = 34 for BRAF-mutant CMS1 CRC samples; FC = fold change). \n",
      ".Figure 3Schematic of HybA and HybO fusions. Western blot analysis (probing with an anti-FLAG mAb) to detect cleavage of HybO with (+)/without (-) chromosomal (native) or pBAD33-encoded rhomboids (plasmid). homboid substrates that are uncleaved, are marked by black, rrows, ild-type S. sonnei, Ss. Wild-type (WT)/inactive (SAHA: alanine substitution of the catalytic serine and histidine residues) enzymes were pBAD33-encoded (plasmid) in S. sonneiΔglpGΔrhom7. Western blot analysis to detect cleavage of HybA with (+)/without (-) chromosomal (native) or pBAD33-encoded rhomboids (plasmid). rhomboid substrates that are uncleaved, cleaved by GlpG, or by Rhom7, are marked by black, red, and blue arrows, respectively. Wild-type S. sonnei, Ss. Wild-type (WT)/inactive (SAHA: alanine substitution of the catalytic serine and histidine residues) enzymes were pBAD33-encoded (plasmid) in S. sonneiΔglpGΔrhom7. Western blot analysis to detect cleavage of HybAG296F by endogenous or pBAD33-encoded rhomboid. Wild-type S. sonnei, Ss. rhomboid substrates that are uncleaved, cleaved by GlpG, or by Rhom7, re marked by black, red, and blue arrows, espectively. ild-type S. sonnei, Ss. Wild-type (WT)/inactive (SAHA: alanine substitution of the catalytic serine and histidine residues) enzymes were pBAD33-encoded (plasmid) in S. sonneiΔglpGΔrhom7. Western blot analysis to detect cleavage of HybAP300A by endogenous or pBAD33-encoded rhomboid. rhomboid substrates that are uncleaved, cleaved by GlpG, or by Rhom7, are marked by black, red, and blue arrows, respectively. Wild-type S. sonnei, Ss. Wild-type (WT)/inactive (SAHA: alanine substitution of the catalytic serine and histidine residues) enzymes were pBAD33-encoded (plasmid) in S. sonneiΔglpGΔrhom7. \n",
      ".Figure 7A Western blot analysis probing the localisation and status of plasmid-encoded N-terminally V5-tagged wild-type (WT) or modified (P300A) HybA in S. sonneiΔrhom7ΔhybB with wild-type (+) or inactive (SAHA) GlpG expressed from pUC19. Whole cell lysate (Whole cell.), Sol (Sol.), Mem (detergent-solubilised, Mem.), and the Aggregate (Agg.) fractions are shown. HybA that is uncleaved, cleaved only by GlpG, and further degraded post GlpG cleavage is marked by black, red, and green arrows, respectively. B Model of rhomboid-mediated quality control by selectively targeting orphan components of multi-protein respiratory complexes. \n",
      ".Figure 3(A) Nuclear fractions from either untreated or irradiated (20 Gy) U2-OS cells, 1.5 hours or 7 days prior to harvest, were analyzed by EMSA with supershift analysis. For supershifting, lysates were incubated with antibodies or water, as indicated. Arrows point to shift location of a given subunit, due to antibody binding. ns, non-specific band. All antibodies were previously tested for supershift compatibility. Representative gel shown from n = 3 replicates. (B) Expression of SASP targets of NF-κB obtained from RNA-seq analysis (Dataset EV1B), was quantitated using RT-qPCR from the shRNA (RELA) stably expressing U2-OS cells. For knockdown efficiency see Appendix Fig S4A. Cells were treated with doxycycline (Dox) to induce knockdown of p65. Heatmap represents targets normalized to the untreated Scrambled control. Expression is shown as log2 change with p values < 0.05. Statistical analysis performed using ANOVA with Bonferroni correction for multiple testing. First-phase and second-phase samples were irradiated (20 Gy) 1.5h and 7 Days prior to harvest, respectively. Analysis based on n = 3 biological replicates. (C) Cell duplication was measured at time points indicated in U2-OS cells bearing either scrambled control or Dox-inducible shRNA against RELA (n = 3 biological replicates). Treatment with Dox was initiated at two days prior to IR (20 Gy) or after IR. Statistical significance in total duplication number between groups at day 6 was determined by ANOVA with Tukey multiple comparisons test. SD shown. * = P < 0.05, *** = p < 0.001. \n",
      ".Figure 5A Schematic of FdoH and FdnH fusions. B Western blot analysis (probing with an anti-FLAG mAb) to detect cleavage of FdoH cleavage in S. sonneiΔrhom7 with (+)/without (-) FdoI. rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red arrows, respectively. C Western blot analysis to detect cleavage of FdnH in S. sonneiΔglpG with (+) or without (-) chromosomal Rhom7 (native), or wild-type (+)/inactive (SAHA) Rhom7 expressed from pBAD33 in S. sonneiΔglpGΔrhom7 with (+) or without (-) FdnI. rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, and blue arrows, respectively. D Alignments of HybA, FdoH, and FdnH in S. sonnei with P. stuartii TatA and with their homologues from phylogenetically diverse organisms (S. sonnei, Salmonella enterica, Yersinia enterocolitica, Proteus mirabilis, Pseudomonas aeruginosa, and Burkholderia pseudomallei) highlighting conserved glycine (green) and proline (orange) residues. E Western blot analysis to detect cleavage of wild-type (WT) or uncleavable (P300A) HybA by active (+) or inactive (S201A) chromosomal GlpG in the presence (+) or absence (-) of HybB. rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red arrows, respectively. F Western blot analysis to detect cleavage of wild-type (WT) or modified (P259A) FdoH by active (+) or inactive (S201A) chromosomal GlpG in the presence (+) or absence (-) of FdoI. rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red arrows, respectively. G Western blot analysis to detect cleavage of wild-type (WT) or modified (P259A) FdnH with active (+) or inactive (SAHA) Rhom7 expressed from pBAD33 with (+) or without (-) FdnI. rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, blue arrows, respectively. \n",
      ".Figure 2(A) Left panel: IκBα western blot of U2-OS cells. Lane 1, untreated (ut); Lanes 2-4, single IR (20 Gy) at indicated time points prior to harvest; Lane 5, IR at 5 days prior to harvest, plus 1.5 hours prior to harvest. Right panel: Samples treated as in the left panel, analyzed by RT-qPCR. Significance from n = 3 replicates per condition determined by ANOVA analysis with Tukey multiple comparisons test. SD shown. *** = p < 0.001. (B) NFKBIA knockdown by dox-inducible shRNA in U2-OS cells. Cells were pre-treated with Dox to induce knockdown. For knockdown efficiency see Fig. S2F. Cells were irradiated (20 Gy) or not, as indicated and described above. NF-κB activity was analyzed by EMSA. ns, non-specific band. A representative gel from n = 3 biological replicates is shown. (C) Venn diagram of RNA-Seq analysis of genes activated following irradiation in the first and second NF-κB activity phases or following shRNA-mediated NFKBIA knockdown in non-irradiated cells (as in B). Numbers of overlapping genes are shown in red. Hypergeometric probability calculation showed significance between overlaps (adjusted with Bonferroni correction for multiple testing). SD shown. *** = p < 0.001. (D) Top GO terms obtained using DAVID Functional Annotation Bioinformatics Microarray 6.7 for first and second phase genes and genes activated upon NFKBIA knockdown, as in (C), as indicated. (E) GSEA was performed comparing the IκBα IEC‐KO gene list (n = 4 villin-Cre x floxed IκBα mice and n = 4 littermate controls (GSE 139251) Mikuda et al, 2020(), with a gene signature based on a list of SASP factors Coppe et al, 2010(). NES: normalized enrichment score. FDR: false discovery rate. (F) RT-qPCR analysis of indicated genes was performed using RNA extracted from the duodenum of 5 and 8 weeks old control villin-Cre (n = 6) or villin-Cre x floxed IκBα mice (n = 6) both male and female. Expression is shown as fold change between littermates, paired t-test. SD shown. * p < 0.05 ** = P < 0.01. \n",
      ".Figure 4A Bacterial two-hybrid analysis with HybA and/or HybB fused chromosomally with the T25 and T18 domains of B. pertussis CyaA, respectively in the absence of endogenous CyaA. Bacteria were grown on LB agar containing 20 µg/ml X-gal at 37°C for 14 hours in the presence/absence of O2. Scale bar, 1 cm. B Western blot analysis to detect HybA cleavage in S. sonneiΔrhom7 by wild-type (+) or inactive (S201A) GlpG expressed chromosomally (native) or from pUC19 (plasmid) with (+)/without (-) HybB. HybA was expressed from its native locus or a plasmid (pHybA). HybA that is uncleaved or cleaved by GlpG is marked by black and red arrows, respectively. C Quantification of the ratio of cleaved/uncleaved HybA in strains with (+) or without (-) HybB. mean ± S.D. of three experiments. *, p<0.05; ****, p<0.0001 (One-way ANOVA). D Western blot analysis to detect GlpG-mediated cleavage of native tagged HybA at indicated times after blocking protein translation by the addition of chloramphenicol at T0 in the absence of HybB (-). , HybA that is uncleaved or cleaved by GlpG is marked by black and red arrows, respectively. E Western blot analysis to detect GlpG-mediated cleavage of C-terminally sfCherry-3xFLAG tagged HybA at times after blocking protein translation by the addition of chloramphenicol at T0 in the presence of HybB (+). , HybA that is uncleaved or cleaved by GlpG is marked by black and red arrows, respectively. F Growth of bacteria lacking hydrogenases (ΔhyaA-F ΔhycE +/- ΔhybO-G) or GlpG (ΔglpG), or expressing uncleavable HybB (hybAG296F) in 5% H2 G H2 consumption by S. sonnei. Bacteria were grown aerobically in LB overnight then diluted into M9 minimal media supplemented with 0.5% fumarate, 12.5 μg/ml nicotinic acid and 0.2% casamino acids in a sealed glass chromatography vial. The headspace was purged with 10% H2/90% argon and cultures were incubated at 37 °C with shaking at 180 rpm for 9 hours. H2 in the headspace was sampled and measured by gas chromatography. H S. sonnei growth in 10% H2 with plasmid expressed wild-type or non-functional (pglpGSAHA) GlpG. \n",
      ".Figure 6A Western blot analysis (probing with an anti-V5 mAb) to detect degradation of N-terminally V5-tagged wild-type (WT) or modified (P300A) HybA in S. sonneiΔrhom7 chromosomally expressing wild-type (+) or inactive (S201A) GlpG with (+) or without (-) HybB. Rhomboid substrates that are uncleaved, cleaved by GlpG, , are marked by black, red, arrows, respectively. Degradation products post rhomboid cleavage are marked by green arrows. B Degradation of V5-tagged HybA at times after blocking protein translation at T0 in the presence (+) or absence (-) of HybB. Rhomboid substrates that are uncleaved, cleaved by GlpG, black, red, arrows, respectively. Degradation products post rhomboid cleavage are marked by green arrows. C Western blot analysis of N-terminally V5-tagged wild-type (WT) or modified (P259A) FdoH in S. sonneiΔrhom7 chromosomally expressing wild-type (+) or inactive (S201A) GlpG with (+)/without (-) FdoI. Rhomboid substrates that are uncleaved, cleaved by GlpG are marked by black, red, arrows, respectively. D Western blot analysis of N-terminally V5-tagged wild-type (WT) or modified (P259A) FdnH in S. sonneiΔrhom7 with wild-type (+) or inactive (SAHA) Rhom7 expressed from pBAD33 with (+)/without (-) FdnI. Rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, blue arrows, respectively. Degradation of N-terminally V5-tagged wild-type (WT) or modified (P259A) FdoH in S. sonneiΔrhom7 with wild-type (+) or inactive (S201A, SAHA) GlpG expressed chromosomally (native) or from pUC19 (plasmid) without FdoI , +/- exposure to 400 µM CuCl2 for 30 min. : Rhomboid substrates that are uncleaved, cleaved by GlpG, are marked by black, red, arrows, respectively. Degradation products post rhomboid cleavage are marked by green arrows. F Degradation of N-terminally V5-tagged wild-type (WT) or modified (P259A) FdnH in S. sonneiΔrhom7 with wild-type (+) or inactive (S201A, SAHA) GlpG expressed from pUC19 (plasmid) without FdnI (-), , +/- exposure to 400 µM CuCl2 for 30 min. Rhomboid substrates that are uncleaved, cleaved by Rhom7, are marked by black, and blue arrows, respectively. Degradation products post rhomboid cleavage are marked by green arrows. \n",
      ".Figure 1(A) SA-β-gal staining of U2-OS cells, harvested at indicated times post irradiation (IR); h, hours; D, days; UT, untreated. n = 3. Scale bar: 50 μm. Bar Graph: Significance was determined by Brown Forsythe ANOVA with Dunnett's 3T test, using n = 3 biological replicates. SD shown. *** p < 0.001, n.s. = not significant. (B) Nuclear fractions from U2-OS cells were harvested at indicated time points post IR (20 Gy) as in (A). Top panel: NF-κB DNA binding was analyzed by EMSA. Lower panels: Western blot of indicated NF-κB subunits and PARP1 as loading control. The two phases of nuclear translocation (C) Gene expression analysis of two NF-κB phases following damage. RNA-Seq analysis (n = 5 biological replicates per group) of U2-OS cells, either untreated or analyzed 1.5 hours or 7 days after irradiation (20 Gy), as indicated. The heatmap shows significantly regulated genes (log2 value > 0.5 and p value < 0.05). Black arrow points to NFKBIA. Significance determined by ANOVA with Bonferroni correction for multiple testing. See also Dataset EV1. (D) Female mice 12-16 weeks old (n = 6 mice per condition (with three technical repeats per mouse), were irradiated (5 Gy) and sacrificed after 1 hour (first phase), 7 days (second phase) or left untreated. Skin or renal tissue RNA was analyzed by RT-qPCR, shown as a mean +/- SD. Significance was confirmed by ANOVA with Tukey multiple comparisons test. SD shown: * = p < 0.05, ** = p < 0.01. (E) In situ hybridization using an anti-sense IκBα/Nfkbia riboprobe on longitudinal skin sections from male and female mice (5-8 weeks of age) treated as in (D) for the time indicated. Representative sections shown from n = 6 mice per condition. Bu, bulge region (dashed lines), HS, hair shaft, IFE, interfollicular epidermis. Scale bar: 50 μm. (F) Quantitation of (E) from n = 6 mice per condition and 3-4 sections per mouse. Bulge regions positive for Nfkbia were counted and presented as percentage of total. One way ANOVA analysis with Tukey multiple comparisons test was performed. SD shown. ** = p < 0.01. (G) Skin sections as in (E) analyzed by immunofluorescence with IκBα (red) or IL-6 antibody (green) and nuclear DAPI staining (blue). Dotted lines delineate hair follicles, Bu, bulge region, HS, hair shaft, IFE, interfollicular epidermis, SG, sebaceous gland. Scale bar: 50 μm. (H) Quantitation of (G) from n = 6 mice per condition and 3-4 sections per mouse. Bulge regions positive for IκBα or IL-6 were counted and presented as percentage of total. One way ANOVA analysis with Tukey multiple comparisons test was performed. SD shown. * = p < 0.05, *** = p < 0.001. \n",
      ".Figure 4(A) Nuclear (Nuc.) and cytoplasmic (Cyto.) fractions of U2-OS cells were analyzed at times indicated after IR exposure by SDS-PAGE western blotting. PARP1 and LDHA are fractionation controls and phospho-specific p65 signals are indicated. Representative gels from n = 3 biological replicates are shown. Right panel: quantitation of nuclear p65 and phosphorylated p65 species (n = 3). Fold changes compared to untreated samples (ut) and time points post IR are indicated on Y and X axes, respectively. (B) U2-OS cells treated with dox to deplete endogenous p65, were irradiated (20 Gy) and transfected with plasmids encoding p65, p65-S276A or p65-S468A, as indicated. Nuclear (N) and cytoplasmic (C) lysates were analyzed by SDS-PAGE at day seven post IR. Representative gel from n = 3 biological replicates is shown. PARP1 and αTubulin serve as fractionation and loading controls. (C) ChIP performed with U2-OS cells irradiated (20 Gy) 1.5h or 7 days prior to assay, or left untreated (UT). Normalized relative enrichment of NFKBIA is shown (relative to input and two non-recruiting reference regions). Left panel: p65 antibody. n = 5 biological replicates with 3 technical repeats per biological replicate. Right: Antibody against p65 pSer468. n = 2 biological replicates with three technical repeats per biological replicate. Statistical significance was determined by ANOVA with Tukey multiple comparisons test. SD shown. * = p < 0.05, ** = p < 0.01, *** = p < 0.001 (D) Kinase proteome array was performed with nuclear fractions of U2-OS cells that were either left untreated (UT) or irradiated (20 Gy) at time points indicated prior to harvest. Kinase binding to membrane was quantitated (n = 2) and only significant results (p < 0.05) are shown. Significance was determined by ANOVA with Tukey multiple comparisons test. (E) U2-OS cells left untreated (ut) or irradiated (20 Gy) 7 days before harvesting, as indicated. Two days prior to harvest cells were transfected with siRNA against GSK3β or scrambled control (Scr). Whole cell lysates were analyzed by SDS-PAGE and western blotting. A representative example from n = 3 experiments is shown. (F) U2-OS cells were left untreated or exposed overnight to 10 mM LiCl, with or without prior irradiation, as indicated, and analyzed as in (E). Representative data from n = 3 experiments are shown. (G) RT-qPCR of U2-OS cells irradiated (20 Gy) either 1.5h or 7 days prior to harvest. CHIR denotes treatment with GSK3 inhibitor CHIR-99021 (10nM overnight). DMSO overnight treatment was used as control. Quantitation was performed and statistical significance obtained from n = 3 replicates, using ANOVA with Tukey multiple comparisons test. SD shown. * = p < 0.05, ** = p < 0.01, *** = p < 0.001. \n",
      ".Figure 6DNA damage triggers two phases of NF-κB activation and two distinct transcriptomes: anti-apoptotic and inflammatory. NF-κB is rapidly activated through the known genotoxic stress-induced IKK cascade, resulting in proteasome-dependent IκBα degradation and expression of first phase targets genes, including IκBα. NF-κB activation in senescence is caused by a loss of IκBα expression through a largely IKK- and proteasome-independent mechanism and drives SASP expression. \n",
      "API delivery is blocked at low temperatures.Figure 1API delivery is blocked at low temperatures. SEY6210 cells were pulse labeled for 5 min and chased for the times indicated. Identical experiments were performed at 10°, 14°, and 30°C. API was recovered by immunoprecipitation followed by SDS-PAGE and detected by a STORM phosphorimager (Molecular Dynamics, Sunnyvale, CA). The positions of prAPI and mAPI are indicated. \n",
      "Precursor API is trapped in a prevacuolar compartment in vps18 cells.Figure 2Precursor API is trapped in a prevacuolar compartment in vps18 cells. (A) Yeast strain JSR18Δ1 with plasmid pJSR9 (vps18 ts) was grown at the permissive temperature of 26°C. Before labeling, the cells were incubated at either 26° or 38°C for 10 min. Pulse labeling was for 5 min, followed by the indicated chase times. (B) vps18 spheroplasts were shifted to 38°C for 5 min, pulse labeled for 5 min, and then chased for either 0 or 60 min. At each time point the spheroplasts were collected by centrifugation and resuspended in PS0 buffer. The Total (T) fraction either received no treatment, was treated with 50 μg/ml proteinase K, or received proteinase K in the presence of 0.2% Triton X-100. The supernatant (S0) and pellet (P0) fractions were collected after centrifugation at 10,000 g for 5 min. \n",
      "A mutant in the API propeptide causes accumulation of prAPI on a nonvacuolar compartment.Figure 3A mutant in the API propeptide causes accumulation of prAPI on a nonvacuolar compartment. (A) Spheroplasts from an ape1Δ strain containing a plasmid bearing P22L API were subjected to differential lysis in PS200 buffer. The Total (T) fraction (after differential lysis) either received no treatment, was treated with 50 μg/ ml proteinase K, or received proteinase K in the presence of 0.2% Triton X-100. (B) The supernatant (S200) and pellet (P200) fractions were collected after centrifugation at 5,000 g for 5 min. The P200 fraction was resuspended in 10% Ficoll, and vacuoles were isolated by flotation through 4% Ficoll. V, vacuole; I, 4/10% Ficoll interface; P, gradient pellet. Proteins were detected by Western blotting. The positions of prAPI, mAPI, and mPrA are indicated. (C) ape1Δ cells containing P22L API on a plasmid were pulse labeled for 5 min, harvested by centrifugation, and subjected to a nonradioactive chase in either nitrogen-containing (SMD) or nitrogen starvation (SD-N) medium. Samples were collected at the times indicated and immunoprecipitated with API antiserum. The resulting SDS-PAGE gels were quantified using a Molecular Dynamics STORM phosphorimager. \n",
      "Precursor API is trapped within a vesicle in cvt17 mutants.Figure 4Precursor API is trapped within a vesicle in cvt17 mutants. (A) Precursor API is in a protease-protected compartment in cvt17 mutants. THY32 (cvt17) spheroplasts were subjected to differential lysis and protease treatment in PS200 buffer as in Fig 3 B) Precursor API is not free in the vacuolar lumen in cvt17 mutants. Differential fractionation was continued by resuspending the P200 fraction in PS0 buffer and separating the supernatant (S0) and pellet (P0) fractions by centrifugation at 10,000 g. The S0 and P0 fractions were treated with proteinase K in the presence and absence of Triton X-100 as indicated. All fractions were precipitated with 10% TCA and resolved by SDS PAGE; proteins of interest were detected by Western blotting. The positions of prAPI, mAPI, and mPrA are indicated. \n",
      ".Figure 5(A) Diagram showing DNA double strand break (DSB) induced first phase NF-κB activation through an ATM/TRAF6 and PARP1/PIASy dependent IKK activation mechanism Hinz et al, 2010Stilmann et al, 2009Wu et al, 2006(; ; ). (B) SDS-PAGE western blot of whole cell lysates of U2-OS cells irradiated (20 Gy) at time points indicated prior to harvest. ut, untreated. LDHA. Representative gel shown from n = 4 biological replicates. (C) U2-OS cells transfected with TRAF6 siRNA or scrambled control siRNA and TRAF6 and IKKβ levels analyzed by western blotting (top panels). NF-κB activity was determined by EMSA in untreated cells (ut) and after IR at indicated time points (lower panel). ns, unspecific band. Representative gels are shown from n = 2 biological replicates. (D) U2-OS cells were exposed to IR at indicated time points prior to harvest and analyzed by western blotting for levels of IκBα, NIK, IL-1α and PARP1. Treatment with proteasomal inhibitor MG132 (10 μM) started 4 hours prior to harvest. NIK serves as positive control for efficient proteasome inhibition. Quantitation from n = 4 biological replicates and SD is shown for IκBα (right panel). Brown-Forsynthe ANOVA test: p = 0.0005. (E) U2-OS CRISPR IKBKB knockout and control cell lines were irradiated (20 Gy) and harvested at indicated time points. NF-κB activity was analyzed by EMSA (lower panel). ns indicates an non-specific band. Upper panel, western blot analysis of actin and IKKβ. Representative gels shown from n = 3 replicates. (F) Irradiated U2-O2 control and IKBKB knockout cells (as above) were analyzed by RT-qPCR at the time points indicated for expression of NF-κB target genes identified by RNA-Seq (Dataset EV1). Significantly regulated targets, cutoff set at p = 0.05 (as determined by ANOVA with Tukey multiple comparison), are shown. \n",
      " Bayesian skyline plot (BSP) and phylogenetic tree of human-associated S.Figure 3Bayesian skyline plot showing the population size through time for human-associated S. suis, which is constructed based on a subsample of HAC isolates (n=174). The Y-axis represents the effective population size, and the X-axis is calendar years. The line shows the median estimate of the population size. Blue shading shows 95% highest posterior density. Timed phylogeny of the subsample of HAC isolates (n=174). Maximum clade credibility tree is produced using strict-clock model in BEAST2. Major sub-lineages in Asia are indicated (I, II, III). Acquisition of the 89-kb pathogenicity island is indicated by a red pentagon. Acquisition of the 78-kb pathogenicity island, and the 127-kb MGE are indicated by black and green pentagon, respectively. The blue and red block of heatmaps represent the presence of MGEs and ARGs, respectively. Gray represents absence. \n",
      "Characterization of API-containing vesicles.Figure 7Characterization of API-containing vesicles. (A) Fractionations were performed as in Fig. 5. The total (T), vacuole (V), vacuolar vesicle (VV), and subvacuolar vesicle (SV) fractions are shown. Proteins were detected by immunoblotting as indicated in the figure. (B) SYPRO orange staining detected by fluorescent scanning mode on a Molecular Dynamics STORM phosphorimager. The relative mobility of the molecular mass standards are indicated on the left. Polypeptides enriched in the subvacuolar vesicles are indicated on the right. \n",
      " Identification of the human-associated clade from Streptococcus suis population.Figure 1Correlation of the Maximum-likelihood tree with genetic and phenotypic classification/clustering. Maximum-likelihood tree was constructed using genome-wide SNPs of 1634 S. suis isolates. Tip nodes are colored based on the host source (human in red; diseased pig in yellow; healthy pig in green). From inside to outside: the first ring represents BAPS clustering; the second ring represents the country source. Human-associated Clade (HAC) are colored in red, with Diseased-pig Clade (DPC) in yellow and Healthy-pig Clade (HPC) in green. Phylogenetic relationship is consistent with BAPS clustering and correlates with host source. Most isolates in BAPS2, BAPS5, BAPS6 were obtained from healthy pigs. The majority of isolates in BAPS7 and BAPS8 were obtained from patients and diseased pigs, respectively. Phylogenetic tree (unrooted) showing three important clades (HAC in red; DPC in yellow; HPC in green). \n",
      "Precursor API is contained within subvacuolar vesicles in cvt17 mutants.Figure 5Precursor API is contained within subvacuolar vesicles in cvt17 mutants. (A) Isolated vacuoles (V) from cvt17 cells were lysed in PS0 buffer, loaded on top of an Optiprep step gradient, and centrifuged at 170,000 g for 60 min as described in Materials and Methods. Fractions VV, 0/1.06 interface; LD, 1.06 region; SV, 1.06/1.12 interface; HD, 1.12 region. (B) An aliquot of fractions VV and SV was treated with proteinase K either in the presence or absence of Triton X-100 as indicated. Because of the presence of protease inhibitors in the vesicle isolation procedure, an intermediate-sized API degradation product results from proteinase K treatment. The V fraction represents one-tenth of the load of the Optiprep gradient. Proteins were detected by Western blotting. The positions of prAPI and mALP are indicated. \n",
      "Immuno-EM of cvt17 cells.Figure 6Immuno-EM of cvt17 cells. Yeast strains SEY6210 (parental wild type) and THY32 (cvt17) were grown in YPD and prepared for microscopy as described in Materials and Methods. (A) Wild-type cells probed with antibody against mature API. (B) A cvt17 section showing a cytosolic vesicle containing prAPI probed with antibody against the API proregion. (C) A cvt17 section showing subvacuolar vesicles containing API probed with antibody against mature API. N, nucleus; V, vacuole; (arrows) API containing vesicles. Bars, 0.5 μm. \n",
      " Isolates from human-associated clade (HAC) are more virulent than isolates from healthy-pig Clade (HPC).Figure 2The violin plot depicting the smaller genome size of HAC isolates (n=820) than that of DPC (n=407) and HPC (n=262). Inside the violins are the box and whisker plots, with the dot showing the median, the box showing the quartiles, and the whisker showing the 95% percentiles. P-values were calculated using unpaired t-test (**, P<0.05). The survival curve for zebrafish inoculated with representative S. suis isolates from HAC and HPC. In total twenty-five representative isolates were tested, including fourteen isolates from HAC and ten healthy-pig isolates from HPC, as well as the strain SC19 in HAC responsible for the 2005 human outbreak as a positive control. The cumulative mortalities were calculated using the combined data from the isolates in each group. As expected, all 14 isolates from HAC displayed a significantly higher mortality than the HPC isolates (P<0.05). Survival curve and statistical analysis were created using Kaplan-Meier and SPSS 23.0. P-values were calculated using Log rank test. P<0.05 was considered significant (**, P<0.05; NS, P>0.05). No significant difference (NS, P=0.072) was found between HAC isolates and SC19. \n",
      " Phylogeographic analysis revealed global transmission patterns of human-associated S.Figure 5 Figure 5 Phylogeographic analysis revealed global transmission patterns of human-associated S. suis. Countries of relevance in the map are colored. Geographic presence, inferred arrivals (arrows) and principal long-distance transmission events (line) are based on phylogeographic analysis. The date are the inferred values for the most recent common ancestor (taken from BEAST2). The origin of most transmission waves was Europe. One of these transmission waves was spread to China leading to two major outbreaks (Jiangsu and Sichuan) (red line). \n",
      "-kb MGE and 78-kb PAI derived from lineage III and II, respectively.Figure 4Two types of 127-kb tandem MGE detected in multi-drug resistant S. suis ST7 isolates in lineage III. The 127-kb MGE encodes eight antibiotic resistance genes. The integrative and conjugative elements (ICE, in orange) and prophage (in purple), both of which are flanked by att sites, are tandemly arranged in two possible orders. The antibiotic-resistance genes are indicated in red. The chromosomal conserved genes around MGE are indicated in black. Comparison of the novel 78-kb PAI reported in this study, with the 89-kb PAI reported in epidemic strains from two outbreaks in China. The key factors for virulence of epidemic strains are indicated in green, including SalKR, NisKR, a Type IV-Like Secretion System and a Tn916 element. The antibiotic-resistance genes are shown in red. Three regions in the 89-kb PAI are absent from the 78-kb PAI. No virulence-related genes were found within these regions. \n",
      ".Figure 1A Determination of a time window for 2a2iL treatment needed to induce a naïve-like pluripotency state in primed hPSCs. One day treatment with 2a followed by the addition of 2a2iL for up to 14 days. Scale bars: 200 µm. B Schematic illustration that outlines induction of the naïve-like state from primed hPSCs. Primed hPSCs were first cultured as single cells in conventional hPSC medium in the presence of ROCK inhibitor (ROCKi) and Y27632 (day 0). The medium was replaced by serum-free medium plus 2a2iL on day 1 without ROCKi. Compact dome-shaped colonies began to appear on day 3. The resultant naïve-like cells were passaged as single cells in the absence of ROCKi on day 5 in 2iL medium. 2a2iL-hPSCs were passaged every 3 to 4 days in 2iL, and kept typical naïve-like morphology for over 50 passages. C Characteristics of 2a2iL-RH6 cells represented by dome-shaped colonies, ALP activity, and a normal male karyotype (46, XY). 2a2iL-RH6 expressed the common pluripotency markers OCT4, NANOG, SOX2, and TRA-1-81, but not SSEA1. Nuclei were stained with 4′, 6-diamidino-2-phenylindole (DAPI) and Propidium iodide (PI). Scale bars: 200, 100, and 50 µm in the inset. \n",
      ".Figure 2A Doubling time in pRH6 versus 2a2iL-RH6. **P<0.01 (t-test). Data presented as mean ± SD (n=3). B Cloning efficiency in the presence (+) and absence (−) of ROCK inhibitor (ROCKi) in pRH6 versus 2a2iL-RH6. ROCKi (−)*P<0.05, ROCKi (+) *P<0.01 (t-test), data presented as mean ± SD (n=3). C Firefly luciferase expression of constructs that contain distal and proximal enhancers of OCT4 after transfection of pRh6 and 2a2iL-RH6. The ratio of firefly luciferase expression to Renilla luciferase expression (expressed from co-transfected plasmid) measured 48 h after transfection showed proximal enhancer activity in pRH6 and distal enhancer activation in 2a2iL-RH6. OCT4 PE-Luc ***P<0.001, OCT4 DE-Luc *P<0.05 (t-test). Data presented as mean ± SD (n=3). D qRT-PCR analysis of pluripotency-related genes in pRH6 and 2a2iL-RH6. *P<0.05, **P<0.01, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). E Immunofluorescence staining for several naïve specific markers in 2a2iL-RH6. Scale bar: 100 µm. F Western blot analysis of naïve-related pluripotency markers in pRH6 and 2a2iL-RH6. **P<0.01, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). G Immunofluorescence staining against pRH5 and 2a2iL-RH5 (46, XX) for H3K27me3. Nuclei were stained with 4′, 6-diamidino-2-phenylindole (DAPI). Reactivation of inactive X-chromosome in female 2a2iL-RH5 was confirmed by the lack of H3K27me3 nuclear foci. Scale bar: 50 µm. \n",
      ".Figure 3A Phase contrast images of embryoid bodies (EBs) from 2a2iL-RH6 cells that differentiated into neurons. Scale bars: 200 and 100 µm in the inset. B qRT-PCR analysis of 20 day-EBs from pRH6 and 2a2iL-RH6 for markers of different embryonic germ layer derivatives. *P<0.05, **P<0.01, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). C Immunofluorescence images of differentiated 2a2iL-RH6 cells that depict expression of markers for neuron-like cells (TUJ-1), cardiac-muscle-like cells (CTNT), and hepatocyte-like cells (SOX17). Nuclei were stained with 4′, 6-diamidino-2-phenylindole (DAPI). Scale bar: 100 µm. D 2a2iL-RH6 cells form teratomas that consist of three embryonic germ layer-derived tissues. (i) Rosette structures (ectoderm), (ii) cartilage (mesoderm), and (iii) primitive gut (endoderm; scale bar 50 µm). E From left to right: Phase contrast and GFP representation in 2a2iL-RH6-eGFP; presence of injected 2a2iL-RH6-eGFP in the mouse blastocyst and integration into the inner cell mass (ICM) after 3, 24 and 48 h incubation periods. Scale bars: 200, 100, and 50 µm in the inset. F Fluorescence images of tissue-sections from human-mouse chimeras in E10.5 showed cells positive for GFP (top) and human nuclear antigen (HNA) (bottom) in different parts of the embryos, including the head (i), trunk (ii), and tail (iii) with the higher magnification in the right panels. Scale bars: 200 and 100 µm in the inset. G Genomic PCR analysis for a human specific mitochondrial sequence (391 bp) in human-mouse chimeras. H Immunofluorescence staining against NANOG shows lack of expression in E10.5 human-mouse chimera tissue sections. Scale bars: 200 and 100 µm in the inset. I Immunofluorescence staining against TUJ1, BRACHYURY (BRA), and SOX17 in E10.5 human-mouse chimera tissue sections represent differentiation-derivatives of 2a2iL-RH6. Scale bars: 200 and 100 µm in the inset. \n",
      ".Figure 5A Pairwise Pearson correlation coefficient heatmap of transcriptome profiles (RNA-seq) from 2a2iL-, pRH5, and pRH6 cell lines. Each cell of the heatmap shows the pairwise Pearson correlation coefficient heatmap of genes with the largest expression variances in a pair of samples. B Principal component analysis (PCA) of the mean-centered transcriptome profiles. The share of variation captured by each principal component is mentioned in the axis labels. C The volcano plot shows a differential expression between 2a2iL- versus primed hPSCs lines. X-axis shows the log2 fold-change, and the y-axis shows the negative log10 scaled adjusted P-value (Benjamini-Hochberg). Red and blue points represent up-regulated genes in 2a2iL- and primed cells, respectively (absolute log-2 fold-change >1, adjusted P-value <0.05). Gray points indicate non-significant genes between these states. D Integrated analysis of six different naïve-primed datasets. Circular and triangular points in the PCA plot represent naïve and primed samples, respectively. Each color indicates one dataset. Black points represent data generated in this study. Each dataset is indicated by the name of the corresponding author and the year of publication (Dataset EV2). E Integrated analysis of transcriptome profiles of primed and naïve hPSCs, including primary human embryonic cells. Each point represents a biological sample; the colors represent the cell type. Naïve and primed samples are depicted in red and blue, respectively. Samples from pre-implantation embryo are shown in different colors. The number inside each circle indicates the study from which the data is obtained. The study of Tang et al., 2013 demonstrated as empty circle. Large dash circle indicates morula stage of human embryo which is almost in line with our data (small dash circle). All empty points are from dataset (9) Tang 2013. F Functional annotation of 2a2iL- and primed up-regulated genes. The most significant over-represented Gene Ontology (GO) terms and their significance level, along with related genes are presented in each row. Adjusted P<0.05 (adjusted P-value was obtained using the Benjamini and Hochberg correction to determine the false discovery rate with a log2 fold-change >1). G Bar plot showing KEGG signaling pathway analysis of genes significantly up-regulated in 2a2iL-hPSCs (Benjamini-Hochberg adjusted P<0.05 and log2 fold-change >1). X-axis shows \n",
      ".Figure 6A Position of TGF-β family members in the volcano plot of differentially expressed genes in 2a2iL- and primed hPSCs. X-axis shows log2 fold-change and y-axis shows negative log10 adjusted P-value <0.05 (Adjusted P-value was obtained using the Benjamini and Hochberg correction to determine the false discovery rate). Red and blue points represent up-regulated genes in 2a2iL- and primed hPSCs, respectively. Genes with no significant expression change are shown in gray. B qRT-PCR analysis of several TGF-β-related genes in pRH6 and 2a2iL-RH6. **P<0.01, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). C Inhibition of the TGF-β pathway by two small molecules, SB43 and A83. Left panel: morphology of naïve-like cells three days after treatment. Right panel: morphology of treated cells after passaging. Scale bar: 100 µm. D Heatmap shows expression of putative LRH-1 target genes in this study's transcriptome profiles. Colors indicate log2 fold change of marker genes in the naïve versus primed hPSCs, with a range from dark blue for lower expression to dark red for higher expression. E Heatmap shows expression of putative RARγ target genes in this study's transcriptome profiles. Colors indicate log2 fold change of marker genes in the naïve versus primed hPSCs, with a range from dark blue for lower expression to dark red for higher expression. F KEGG analysis of putative LRH-1 and RARγ target genes are up-regulated in 2a2iL-hPSCs. Adjusted P<0.05 (adjusted P-value was obtained using the Benjamini and Hochberg correction to determine the false discovery rate with a log2 fold-change >1). \n",
      ".Figure 4A Schematic overview of protocols used to derive naïve-like hESC lines by 2a2iL from human blastocysts. Two protocols were used to generate naïve hESC lines. In protocol 1, isolated ICMs were cultured in hESCs medium that contained basic fibroblast growth factor (bFGF) for 4 days followed by 2a2iL-supplemented medium until day 10. Dome-shaped colonies representative of the naïve state formed after single-cell dissociation. In protocol 2, ICMs were exposed to 2a2iL-supplemented medium at the beginning of seeding. B Morphology of ICM-outgrowths by protocols 1 and 2 during the first 4 days after seeding. nRH11-14 (naïve Royan H11, 12, 13, and 14): naïve hESC lines produced from human embryos using 2a2iL condition. Scale bar: 100 µm. C Comparison of the efficiency of naïve-like hESCs derivation from human blastocysts by two protocols. D Representative example of naïve-like hESCs (nRH13) at passage 33. Dome-shaped colonies were stained for ALP activity and karyotyped. Scale bars: 200 and 100 µm in the inset. E Expression of the common pluripotency markers OCT4, SOX2, and TRA-1-81, and naïve-related pluripotency markers NANOG, KLF17, and TFCP2L1 in nRH13. Nuclei were stained with 4′, 6-diamidino-2-phenylindole (DAPI). Scale bar: 100 µm. F qRT-PCR analysis for pluripotency markers in nRH13 in comparison to a primed hESC line, pRH6. *P<0.05, **P<0.01, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). G Expression of lineage markers after embryoid body (EB) formation and spontaneous differentiation in nRH13 cells. *P<0.05, ***P<0.001 (t-test). Data presented as mean ± SD (n=3). \n",
      "Fig.Figure 1A: Relative expression of ELDR in OSCC patient samples compared to adjacent non-tumor tissues (N=20) analyzed by qRT-PCR. Technical triplicates were used from each sample. 18S rRNA was used as an internal control. Data were analyzed by Student's t test. Small bars indicate standard error (*, p<0.05, **, p<0.01; *** p<0.001). B: OSCC grade wise upregulation of ELDR expression was noted in the patient samples (N=20). Correlation analysis showed signification positive correlation of ELDR expression with tumor grade (R2= 0.926, significant P value = 0.01). Pearson correlation coefficient and p value was calculated using socscistatistics.com. Well: well differentiated, Mod: moderately differentiated, Poor: poorly differentiated carcinoma. Data are represented as the mean ± SD, C: Representative images of FISH analysis with an RNA probe of ELDR (red) and DAPI staining (blue) in formalin fixed paraffin embedded OSCC tissue. An unrelated probe was used as negative control. Magnification: 40X; inset magnification 100X and scale bar 10 µm. D: Relative expression of ELDR in OSCC cell lines compared to normal oral keratinocytes (NOK) analyzed by qRT-PCR. 18S was used as internal control. Data were analyzed by Student's t test. Small bars indicate standard error (**, p<0.01; *** p<0.001). n = 4 biological replicates. Data are represented as the mean ± SD, \n",
      ".Figure 7A qRT-PCR analysis of ligands associated with TGF-β signaling during conversion of primed to naïve-like pluripotency by 2a2iL induction protocol at days 1, 3, and 5. 2iL was selected as a control group. ***P<0.0001. One-way ANOVA. Data presented as mean ± SD (n=3). B Inhibition of TGF-β pathway using SB43 during conversion of primed to naïve pluripotency. Scale bar: 100 µm. C Morphology of TGFβ2iL-treated RH6 cells along with ALP activity. Scale bar: 200 µm. D qRT-PCR analysis of naïve related genes in pRH6, 2a2iL-, and TGFβ2iL-RH6. **P<0.01, ***P<0.001. One-way ANOVA. Data presented as mean ± SD (n=3). E Bar plot representing cell proliferation rates of 2a2iL- and TGFβ2iL-RH6 by cell counts. **P<0.01 (t-test). Data presented as mean ± SD (n=3). F Bar plot of the cloning efficiency of 2a2iL and TGFβ2iL-RH6 based on the number of ALP positive colonies. ***P<0.001 (t-test). Data presented as mean ± SD (n=3). G Immunofluorescence images show the expression of naïve pluripotency markers in TGF-β2iL-RH6. Nuclei were stained with 4′, 6-diamidino-2-phenylindole (DAPI). Scale bar: 100 µm. \n",
      "Fig.Figure 2A: Control or ELDR plasmid DNA stably transfected NOKs were analyzed for cell proliferation using Trypan blue exclusion at the indicated time points. Live cell numbers are presented. n = 4 biological replicates. Data are represented as the mean ± SD, B: Control or ELDR plasmid DNA overexpressed NOK (3×102) were seeded and allowed to form colonies. After two weeks, colonies were stained with crystal violet and counted. Representative images of colonies in control and ELDR overexpressed cells are presented. The right panel shows quantitation. n = 3 biological replicates. Data are represented as the mean ± SD, C: Cal27, JHU029 and JHU022 cells were transfected with control or ELDR siRNA (50nM), and cell proliferation was analyzed at the indicated time points. n = 3 biological replicates. Data are represented as the mean ± SD, D: Cell lysates from control or ELDR overexpressing NOKs, control or Cal27 and JHU029 cells treated with two different ELDR siRNAs were subjected to Western blot analysis for PCNA expression using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control for normalization. The bottom panel shows a quantitative representation of Western blot band intensities. n = 3 biological replicates. Data are represented as the mean ± SD, E: Cal27 and JHU029 cells were transfected with siELDR, ELDR plasmid DNA or both siELDR and ELDR plasmid DNA. Cell lysates were analyzed by Western blot for PCNA expression using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. The right panel shows quantitative representation of Western blot band intensities. n = 2 biological replicates. Data are represented as the mean ± SD, \n",
      "Fig.Figure 4A: Top hits of ELDR interacting proteins analyzed from the ELDR sense vs. antisense RNA pull-down assay followed by mass spectrometry in lysates from JHU022 and Cal27 cells. B: ELDR interacting proteins identified by mass spectrometry classified by GENEONTOLOGY PANTHER classification system based on \"Molecular Function\". C: Cell lysates from Cal27 and JHU022 cells were incubated with biotinylated ELDR sense and antisense RNA, pulled down and subjected to Western blot analysis for the ILF3 using specific antibodies. Bottom panel shows Western blot analysis of PUM-1 (an unrelated RNA binding protein). D: Cal27 and JHU022 cell lysates were immunoprecipitated against control or ILF3 antibody and RNA was isolated from the precipitates. Relative expression of ELDR (left penal) and PCAT1 (right panel) was analyzed by qRT-PCR. n = 3 biological replicates and 3 technical replicates. Data are represented as the mean ± SD, E: Cell lysates from control and ELDR depleted Cal27 and JHU029 cells were subjected to Western blot analysis for ILF3 using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. Right panel shows the quantitative representation of Western blot band intensities. n = 3 biological replicates. Data are represented as the mean ± SD, F: Control, ELDR overexpressed and ELDR-depleted Cal27 cells were probed with an antibody against ILF3 (red) and DAPI (blue). Representative confocal microscopic images show nuclear and cytoplasmic expression of ILF3. Arrows indicate cytoplasmic expression. Magnifications 60X, scale bar 20 μ. \n",
      "Fig.Figure 7A: Body weight was measured in control and siELDR treated mice. n=5 animal per group B: Representative images of tumors in control and treatment groups. C: Tumors were measured using a slide caliper and tumor volumes were calculated. Arrow indicates starting point siELDR treatment. n=5 animal per group and tumor volume presented as the mean ± SD D: ELDR knockdown was examined by analyzing expression of ELDR in treated tumors compared to the control tumors. 18S rRNA gene was used as internal control. n=5 animal per group and 3 technical replicates from each tumor. E: Control or siELDR treated tumor lysates were subjected to Western blot analysis for EGFR and Cyclin E1expression using specific antibodies and representative bands are shown. The blot was reprobed with an antibody against Actin for normalization. Right panels show quantitation. n= 3 biological replicates from each tumor F: OSCC-PDX tissue was implanted subcutaneously into the flank of NSG mice. When the tumor volume reached >70 mm3, mice were divided into two groups. 10µg of siELDR or control oligoes were injected intratumorally as described above. Body weight was measured in control and siELDR treated mice. n=5 animal per group G: Representative images of OSCC-PDX tumors in control and treatment groups. H: PDX tumors were measured using a slide caliper and tumor volumes were calculated. n=5 animal per group and tumor volume presented as the mean ± SD I: ELDR knockdown was examined by analyzing expression of ELDR in treated OSCC-PDX tumors compared to the control tumors. 18S gene was used as internal control. n=5 animal per group and 3 technical replicates presented as the mean ± SD \n",
      "Fig.Figure 3A: Cal27 or JHU029 cells were transfected with control or ELDR siRNA, and EGFR mRNA expression was examined by qRT-PCR. 18S gene was used as internal control. n = 3 biological replicates and 3 technical repeates. B: Cell lysates from control and ELDR depleted Cal27 or JHU029 cells using two different siRNAs to ELDR were subjected to Western blot analysis for the EGFR using specific antibodies. The membrane was reprobed with an antibody against Actin as an internal control. The same actin blot was used in Fig. 2D (for PCNA). The bottom panel shows a quantitative representation of Western blot band intensities. n = 3 biological replicates. Data are represented as the mean ± SD, C: Cal27 and JHU029 cells were transfected with siELDR, ELDR plasmid or both siELDR and ELDR plasmid. Cell lysates were analyzed by Western blot analysis for EGFR expression using specific antibody. The membrane was reprobed with an antibody against Actin as internal control. The same actin blot was used in Fig. 2E (for PCNA). n = 2 biological replicates. D: Cell lysates from control and ELDR overexpressed NOK were subjected to Western blot analysis for the EGFR using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. The right panel shows a quantitative representation of Western blot band intensities. n = 3 biological replicates. Data are represented as the mean ± SD, E: Cell lysates from control and ELDR-depleted Cal27 and JHU029 cells were subjected to Western blot analysis for phsospho-STAT3 (pSTAT3 Y-705) and total STAT3 using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. Right panel shows a quantitative representation of Western blot band intensities. n = 2 biological replicates. Data are represented as the mean ± SD, \n",
      "Fig.Figure 6A: Cal27 and JHU029 cells were transfected with control or ELDR siRNA and relative expression of miR-7 was analyzed by qRT-PCR. U47 was used as an internal control. n = 3 biological replicates. Data are represented as the mean ± SD, B: Cal27 and JHU029 cells were transfected with control or ELDR plasmid and relative expression of miR-7 was analyzed by qRT-PCR. U47 was used as an internal control. n = 3 biological replicates. Data are represented as the mean ± SD, C: Cal27 and JHU029 cells were transfected with control or mimic miR-7 and relative expression of ELDR was analyzed by qRT-PCR. 18S rRNA was used as an internal control. n = 2 biological replicates; n= 3 technical replicates. Data are represented as the mean ± SD, D: Cal27 and JHU029 cells were transfected with miR-7 mimic, ELDR plasmid or both miR-7 and ELDR plasmid. Cell lysates were analyzed by Western blot analysis for EGFR expression using specific antibody. The membrane was reprobed with an antibody against Actin as an internal control. Representative bands were cropped from same gel. Right panels show quantitative representation of Western blot band intensities. n = 3 biological replicates. Data are represented as the mean ± SD, E: Relative expression of miR-7 in Cal27 and JHU029 cells compared to NOK cell line analyzed by qRT-PCR. U47 was used as an internal control. n = 3 biological replicates and 3 technical replicates. Data are represented as the mean ± SD, F: Schematic representation of ELDR and miR-7 mediated regulation of EGFR signaling. Sharp arrow indicates activation and blunt arrow indicates inhibition. \n",
      ".Figure 1A) Alignment of sequences of all human α- and β‑tubulin isotypes. Only the C-terminal tail region of the tubulins are shown. Isotypes with identical amino acid sequence in these regions are pooled. A large number of glutamate residues in the C-terminal tails would, if modified by glutamylation, be detectable with GT335 (yellow), which requires one additional acidic amino acid residue after the modification site (Wolff et al, 1992). By contrast, the GEF motif (red/purple), which is the main glutamylation site in β2‑tubulin (TUBB2) in the brain (Rüdiger et al, 1992), is present only in a few β‑tubulin isotypes, but not in α-tubulin. B) Schematic representation of the synthetic peptide used as antigen to raise the β-monoE antibody. The modified glutamate residue is in red. C) Representative immunoblots of brain extracts of wild-type, Ttll1-/- and Ttll7-/- mice. Polyglutamylation of α-tubulin, detected with polyE and GT335, is specifically lost in Ttll1-/- brain. By contrast, β-tubulin glutamylation, which is specifically detected with β‑monoE, is absent from brains of Ttll7-/- mice. D) Representative immunoblots of extracts of wild-type, Ttll1‑/- and Ttll7-/- primary hippocampal neurons at DIV4. As in brain (C), α-tubulin polyglutamylation is specifically lost in Ttll1-/- neurons, while β-tubulin glutamylation lacks in Ttll7-/- neurons. E E) Schematic representation of the epitopes detected by the anti-glutamylation antibodies used in this study. For clarity, α- and β‑tubulin are shown separately. F,G) Immunocytochemistry of DIV4 primary hippocampal neurons from wild type, Ttll1-/- and Ttll7‑/- mice. Microtubules in Ttll1-/- cells have reduced levels of polyE signal, indicative of lower α-tubulin polyglutamylation (F), Ttll7‑/- neurons are specifically reduced in β‑monoE reactivity, indicating the lack of β‑tubulin glutamylation (G). \n",
      "Fig.Figure 5A: Cal27 and JHU022 cell lysates were immunoprecipitated with control or ILF3 antibodies and RNA was isolated from the precipitates. Relative mRNA expression of EGFR was analyzed by qRT-PCR. n = 3 biological replicates and 3 technical replicates. Data are represented as the mean ± SD, B: Cal27 and JHU029 cells were transfected with control or ILF3 siRNA and after 48hr cell lysates were subjected to Western blot analysis for EGFR and ILF3 using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. n = 3 biological replicates. C: Cal27 and JHU029 cell lysates were transfected with either siELDR or siILF3 and relative mRNA expression of Cyclin E1 was analyzed by qRT-PCR. 18S was used as internal control. n = 3 biological replicates. Data are represented as the mean ± SD, D: Control or ELDR depleted Cal27 and JHU029 cell lysates using two different siRNAs to ELDR were subjected to Western blot analysis for Cyclin E1 using specific antibody. The membrane was reprobed with an antibody against Actin as an internal control. Bottom panels show quantitation. n = 4 biological replicates. Data are represented as the mean ± SD, E: Control or ELDR overexpressed NOK lysates were subjected to Western blot analysis for Cyclin E1 using specific antibody. The membrane was reprobed with an antibody against Actin as internal control. Bottom panel shows quantitation. n = 3 biological replicates. Data are represented as the mean ± SD, F: Control or ILF3depleted Cal27 and JHU029 cell lysates were subjected to Western blot analysis for Cyclin E1 using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. Bottom panels show quantitation. n = 3 biological replicates. Data are represented as the mean ± SD, G: Cal27 and JHU029 cells were transfected with either siELDR or siILF3 and after 48hr cell lysates were subjected to Western blot analysis for phospho-RB and total RB using specific antibodies. The membrane was reprobed with an antibody against Actin as internal control. Bottom panels show quantitation. n = 3 biological replicates. Data are represented as the mean ± SD H: Control or ELDR knockdown synchronized Cal27 and JHU029 cells were harvested, fixed and stained with propidium iodide. DNA content was analyzed by flow cytometry. Results are represented cell population in G1, S, and G2/M phases of the cell cycle. Right panels show percentage of cell population in different phases of cell cycle. n = 4 biological replicates. Data are represented as the mean ± SD, I: Schematic representation of ELDR-ILF3 mediated regulation of Cyclin E1 signaling. The arrow indicates activation. \n",
      ".Figure 2A) Representative histology images of cerebella from 3-months-old mice stained with anti-calbindin antibody, which specifically labels Purkinje cells (Sequier et al, 1988). Note the presence of an intact Purkinje-cell layer in cerebella of wild-type, Ttll1-/-, Ttll7-/- and Ccp1-/-Ttll1-/- mice, in contrast to the complete absence of these cells in Ccp1-/- and Ccp1-/-Ttll7-/- mice. B) Histology of 25-day old mice with partial degeneration of Purkinje Cells. Note that both, Ccp1-/- and Ccp1-/-Ttll7-/- brains, show highly similar pattern of partial Purkinje-cell loss (red arrow heads: gaps in the Purkinje-cell layer, indicating degenerated Purkinje cells; orange arrow heads: swellings of Purkinje-cell axons, indicative of ongoing degeneration). C) Representative immunoblots of cerebellum extracts of wild-type, Ccp1-/-, Ttll7-/-, Ccp1-/-Ttll7-/-, Ttll1-/- and Ccp1-/-Ttll1-/- mice. Loss of CCP1 leads to hyperglutamylation of α- and β-tubulin as revealed with polyE and GT335 antibodies. Concomitant deletion of TTLL7 lead to a loss of glutamylation from β-tubulin (β‑monoE, GT335), while hyperglutamylation of α-tubulin persists (polyE, GT335). Concomitant loss of CCP1 and TTLL1 eliminates hyperglutamylation from both, α- and β-tubulin (polyE, GT335), while β-tubulin glutamylation is unaltered (β-monoE). \n",
      ".Figure 5A) Representation of the differential glutamylation of α- and β-tubulin, and which epitopes can be detected by the glutamylation-specific antibodies used in this study. The most prominent tubulin isotypes in brain, α1- and β2-tubulin are shown with their C-terminal tails represented as amino-acid sequences. The main modification sites, glutamates (E) 445 on α1-tubulin (Eddé et al, 1990) and E435 on β2-tubulin (Rüdiger et al, 1992) are indicated. However, all glutamate residues in these C-terminal tails (light grey circles) could potentially be glutamylated. Glutamylation generated by TTLL1 is colour-coded red, while TTLL7-catalysed glutamylation is orange. Note that under wild-type conditions, TTLL1 and TTLL7 exclusively modify α- and β-tubulin, respectively in brain tissue. B) Representation of glutamylation patterns found in the knockout mouse models studied here, and interpretation of the participation of the TTLL1, TTLL7 and CCP1 in their generation. Based on the immunoblot analysis shown in Fig. 2C, it can be deduced that TTLL1 is able to hyperglutamylate both, α- and β-tubulin in the absence of CCP1. By contrast, loss of CCP1 in the absence of TTLL1 does not lead to hyperglutamylation, suggesting that TTLL7 is not sufficient to generate this phenotype. In this context, our observation that hyperglutamylation of β-tubulin is abolished with TTLL7 indicates that while TTLL1 can hyperglutamylated β-tubulin, it requires prior glutamylation by TTLL7 to do so. We thus conclude that TTLL1 specifically elongates nascent glutamate chains generated by TTLL7. \n",
      ".Figure 4A) Representative images of DIV4 primary hippocampal neurons from wild type, Ttll1-/- and Ttll7-/- mice Cells were incubated with MitoTracker® (black dots) and mitochondria movements were recorded in the longest neurite (orange lines). B) Kymographs of one representative experiment representing mitochondrial movements C - F) Analysis of one single experiment shown in (B). Scatter dot plots of (C) run length and (D) speed distributions, where each point represents a single run extracted from the kymographs shown in (B). The black lines indicate the median of the distribution, with whiskers at interquartile range. Numerical values of medians are indicated below each plot. (E) A bar graph showing the overall motility of mitochondria (the fraction of time mitochondria spent in movement relative to the total number of mitochondria during the recording time of 60 s). (F) A bar graph showing mitochondrial density in analysed axons. Values of the overall motilities and mitochondrial densities are given below the bar plots. G - J) Statistical analyses of mitochondrial transport parameters of five independent experiments shown in (D-G and Fig. EV3A-D). Each bar represents the mean (±SEM) of medians of single experiments for (G) run length and (H) speed distributions, and average (±SEM) (I) overall mitochondrial motility and (J) density of all five experiments. Values of fold change between control and Ttll1‑/- and Ttll7-/- conditions are indicated below the graphs. Full bars are for anterograde, and empty bars for retrograde transport events. Significance was tested using unpaired t-test. Average (G) run length as well as (H) speed showed no significant differences between control and Ttll1-/- and Ttll7-/- conditions. (I) The averages of overall motilities reveal an about 1.7-fold increase in overall mitochondrial motility in Ttll1-/- neurons, while no significant differences were found in Ttll7-/- neurons. (J) Mitochondrial density was unaltered between control and knockout conditions. Total number of mitochondria analysed from five independent experiments: control (1260), Ttll1-/- (1158) and Ttll7-/- (1191). \n",
      " - svb is expressed in ISC/EBs and is required for their maintenanceA Schematic representation of the svb locus, showing location of enhancers as well as functional organization of somatic (Svb) and germline (OvoA, OvoB) protein isoforms.Figure 1Schematic representation of the svb locus, showing location of enhancers as well as functional organization of somatic (Svb) and germline (OvoA, OvoB) protein isoforms. Red and green boxes represent the repressor and activator domains, respectively; purple ovals depict the DNA-binding zinc fingers. The adult intestinal stem cell lineage, with markers of stem cells (Dl), enteroblasts (Su(H)) and enterocytes (Pdm1). Esg is expressed in progenitor cells gathering stem cells (ISC), enteroblasts (EB) and pre-enteroendocrines (preEE). Both pre-EEs and mature enteroendocrine cells (EE) express Prospero (pros). Posterior midgut showing expression of the E3N svb enhancer (GFP, green) in ISC/EBs, as shown by co-staining with esg-lacZ (β-Gal, red). ' Staining for Dl-lacZ (purple) or Su(H)-lacZ (red) in esgts midguts expressing GFP alone (control), or expressing svb-RNAi. Samples were stained for GFP (green) and ß-Gal. Closeups show separate channels for GFP and ß-Gal. The graphs show quantification of the number of ICS (Dl-positive) and EBs (Su(H)-positive). ' Staining for Dl-lacZ (purple) or Su(H)-lacZ (red) in esgts midguts expressing GFP alone (control), or expressing svb-RNAi. Samples were stained for GFP (green) and ß-Gal. Closeups show separate channels for GFP and ß-Gal. The graphs show quantification of the number of ICS (Dl-positive) and EBs (Su(H)-positive). Quantification of the number of YFP-positive cells (left) and GFP-positive cells (right) in control and upon expression of svb-RNAi driven by ISCts and Su(H)ts, respectively. Posterior midguts containing control or svbR9 MARCM clones (GFP, green), and quantification of the number of clones, and of the average number of cells per clone. esgtsF/O midguts expressing GFP alone (control) or expressing svb-RNAi. Samples were stained for GFP (green) and the apoptotic marker cleaved Dcp1(red). Arrows highlight GFP-positive cells that are also positive for Dcp1. Quantification of GFP-positive cells per posterior midgut in esgts expressing GFP alone (ctrl), or expressing DIAP1, svb-RNAi, and svb-RNAi+DIAP1. \n",
      " - Svb activator induces stem cell proliferationA esgts midguts expressing GFP alone (control), or expressing OvoB, and carrying a Dl-lacZ transgene that marks ISCs.Figure 3esgts midguts expressing GFP alone (control), or expressing OvoB, and carrying a Dl-lacZ transgene that marks ISCs. Samples were stained for GFP (green) and ß-Gal (red); the graph shows quantification of ß-Gal-positive cells (ISCs). Quantification of number of mitotic PH3-positive cells/midgut in esgts guts expressing GFP alone (control), or expressing OvoB and svb-RNAi; Y axis is drawn as log(10). esgts midguts expressing GFP alone (control) or expressing SvbACT. Samples were stained for GFP (green), and ß-catenin (purple), DE-Cadherin (white) or Scribble (yellow). Top and bottom pictures show separate channels for a same region. esgts midguts expressing GFP alone (control), or expressing miR8, OvoB, and mir8 + OvoB. Samples were stained for GFP (green) and ß-catenin (purple). The graph shows quantification of GFP-positive cells for each genotype. The Y axis is drawn using a log(10) scale. esgts midguts expressing GFP alone (control), or expressing Notch Intra Cellular Domain (NICD), and NICD + OvoB. In closeups (right), DAPI is shown in purple for improved contrast; the arrow highlights a cell with intermediate phenotype. \n",
      " - Pri/proteasome processing of Svb is required for ISC/EB maintenanceA Schematic representation of the pri locus, with tested enhancers and location of the pri-Gal4 gene trap insertion.Figure 2Schematic representation of the pri locus, with tested enhancers and location of the pri-Gal4 gene trap insertion. Expression of priA, priJ or priH enhancers in the posterior midgut as seen by lacZ reporters (X-Gal staining, blue), and pri-Gal4 gene trap expressing GFP (green). Schematic representation of Svb maturation by proteasome processing, which is triggered by EcR-mediated expression of pri. esgts midguts expressing GFP alone (control), or expressing pri-RNAi, and quantification of GFP-positive cells (green). The graph also shows the number of YFP-positive cells in ISCts midguts expressing YFP alone (ctrl), or pri-RNAi (see EV2A). ISCts midguts expressing YFP alone (control), or expressing two non-overlapping EcR-RNAi, and quantification of the number of YFP-positive cells (yellow). The graph also plots the number of GFP-positive cells in esgts midgut expressing GFP alone (ctrl), or expressing EcR-RNAi#1, EcR-RNAi#2, and mcherry-RNAi as an additional negative control (see EV2B). esgts midguts expressing GFP alone (control), or expressing EcR-DN, and EcR-DN + pri. Samples were stained for GFP (green). The graph shows quantification of the number of GFP-positive cells in the different genotypes. Posterior midguts containing control and Ubr3 null MARCM clones (GFP, green). esgts midguts expressing GFP alone (control), or expressing Ubr3-RNAi, and Ubr3-RNAi + OvoB. Samples were stained for GFP (green). H' quantification of GFP-positive cells from H. \n",
      ".Figure 3Semi-thin cross sections of femoral quadriceps nerves stained with alkaline methylene blue. Note the smaller size of Ccp1-/- and Ccp1-/-Ttll7-/- nerves (A). Semi-thin cross sections of femoral quadriceps nerves stained with alkaline methylene blue. These sections were quantified for (B) number of myelinated axons, (C) nerve area, (D) nerve perimeter (Fig. EV2) of femoral quadriceps nerves from wild-type, Ccp1-/-, Ttll7-/-, Ccp1-/-Ttll7-/-, Ttll1-/- and Ccp1-/-Ttll1-/- mice. Scatter dot plots in which each point represents one nerve of an individual mouse are shown. The black lines indicate means ±SEM. Significance was tested with one-way ANOVA and Holm-Šídák's multiple comparisons test. Note that all three parameters are decreased for Ccp1-/- and Ccp1-/-Ttll7‑/- mice, while Ccp1-/-Ttll1-/- mice showed no difference to the Ttll1-/- control. E,F) Representative electron microscopy images of cross sections of femoral quadriceps nerves. (E) Nerves from Ttll1-/- and Ccp1-/-Ttll1-/- mice show no abnormalities of myelinated axons (green asterisks), indicating that loss of TTLL1 prevents axonal degeneration caused by the absence of CCP1. (F) Ccp1-/- and Ccp1-/-Ttll7-/- mice both show the presence of foamy macrophages (magenta arrowheads), which are signs for myelin phagocytosis during axon degeneration. This indicates that loss of TTLL7 cannot prevent the CCP1-induced degeneration of femoral quadriceps nerves. \n",
      " - Svb acts downstream of Wnt and EGFR mitogenic signaling pathways in the adult midgutA esgtsF/O midguts expressing GFP alone (control), or expressing RasV12, and RasV12 + svb-RNAi (top panels).Figure 4esgtsF/O midguts expressing GFP alone (control), or expressing RasV12, and RasV12 + svb-RNAi (top panels). Bottom panels show esgts midguts expressing GFP alone (control), or expressing EGFR-DN, and EGFR-DN + OvoB. Samples were stained for GFP (green). The graph shows quantification of the number of GFP-positive cells in esgts midguts expressing GFP alone (ctrl), or expressing EGFR-DN, EGFR-DN + OvoB, and EGFR-DN + svb-RNAi. The Y axis is plotted as log(10). esgts midguts expressing GFP alone (control), or expressing ArmS10, ArmS10 + svb-RNAi, ArmS10 + OvoB ,TCF-DN, and TCF-DN + OvoB. Samples were stained for GFP (green). The graph shows quantification of the number of GFP-positive cells in esgts midguts expressing GFP alone (ctrl), or expressing TCF-DN, TCF-DN + OvoB, and TCF-DN + svb-RNAi. The Y axis is plotted as log(10). The drawing at left schematizes the svb locus, with position of the E3N enhancer. Close-up shows E3N sequence that correspond to binding sites for Pnt (red) and TCF (green). Right subpanels are pictures of posterior midguts showing expression of wild-type E3N-lacZ (E3Nwt), or E3N-Pnt-mt, and E3N-TCF-mt, as seen from X-Gal staining (cyan blue). \n",
      "LRRC8D but not LRRC8A correlates with shorter survival of high grade serous ovarian cancer patients treated with platinum‐based drugs A-D Differential survival based on LRRC8A (A, C) or LRRC8D (B, D) gene expression as extracted from the TCGA database (http://cancergenome.Figure 2Low expression of LRRC8D but not LRRC8A correlates with shorter survival of high grade serous ovarian cancer patients treated with platinum‐based drugsA-DDifferential survival based on LRRC8A (A, C) or LRRC8D (B, D) gene expression as extracted from the TCGA database (http://cancergenome.nih.gov/) (A, B) or using the data from Patch et al (2015) (C, D). As cutoff the lower tertile of LRRC8A or LRRC8D gene expression was used. P‐values were determined using the log‐rank test. \n",
      "Loss‐of‐function screen using haploid KBM7 cells identifies LRCC8A and LRRC8D as determinants of carboplatin resistance.Figure 1Outline of the loss‐of‐function screen. Genes enriched for gene‐trap insertions in a carboplatin‐selected cell population compared to unselected control cells. Circles represent genes and their size corresponds to the number of independent insertions identified in the carboplatin‐selected population. Genes are ranked on the x‐axis based on chromosomal position.Location of gene‐trap insertion sites (red arrowheads). White boxes indicate the 5′‐ and 3′‐untranslated regions, and the black boxes show the coding sequence in exons 3 and 4 (LRRC8A) and exon 3 (LRRC8D). Loss of LRRC8D causes resistance to carboplatin and cisplatin, but not to oxaliplatin. Survival of parental, vector‐transduced, or LRRC8D‐deficient GT1 and GT2 KBM7 cells exposed for 96 h to increasing concentrations of cisplatin, carboplatin, and oxaliplatin. The corresponding IC50 values and 95% confidence interval (CI) are given in Appendix Table S2. Data are presented as mean ± SEM. \n",
      ": Svb repressor is required to maintain enterocyte differentiationA Control MyoIAts midguts, and MyoIAts>svb-RNAi or MyoIAts>OvoB midguts.Figure 7Control MyoIAts midguts, and MyoIAts>svb-RNAi or MyoIAts>OvoB midguts. Cyan dye stains the lumen. MyoIAts midguts expressing GFP alone (control), or expressing svb-RNAi, and SvbACT. Samples were stained for GFP (green). Electron micrographs of MyoIAts control midguts, or expressing svb-RNAi and SvbACT. Brush border microvilli are pseudo-colored in green, and high magnification views are shown in inlets. Nuclei are pseudo-colored in purple, and visceral muscles located above the basement membrane are in pink. Arrowheads point to impaired cell contacts, the arrow points to a pyknotic nucleus. Cross sections of control MyoIAts midguts (expressing GFP and mCherry-RNAi), or expressing svb-RNAi, SvbACT, and Pri. Samples were stained for F-actin (white), GFP (green) and Tsp2a (yellow). \n",
      ": SvbREP suppresses stem cell tumors in the gut epitheliumA esgts midguts expressing GFP alone (control), or expressing Notch-RNAi, Notch-RNAi + svb-RNAi, and Notch-RNAi + SvbREP.Figure 6esgts midguts expressing GFP alone (control), or expressing Notch-RNAi, Notch-RNAi + svb-RNAi, and Notch-RNAi + SvbREP. Samples were stained for GFP (green) and Prospero (red). The graph shows quantification of the number of GFP-positive cells. esgts midguts expressing GFP alone (control), or expressing STAT-RNAi, STAT-RNAi + svb-RNAi and STAT-RNAi + SvbREP. Samples were stained for GFP (green) and Prospero (red). The graph shows quantification of the number of GFP-positive cells. esgts midguts expressing GFP alone (control), or expressing Wg, Wg + OvoB, Wg + SvbREP, and Wg + svb-RNAi. In closeup views, nuclei are in purple. \n",
      ": Ectopic Svb processing disrupts enterocyte differentiationA MyoIAts midguts expressing GFP alone (control), or expressing SvbACT.Figure 8MyoIAts midguts expressing GFP alone (control), or expressing SvbACT. Samples were stained for GFP (green), Scribble (yellow), ß-catenin (purple) and DAPI (Blue). A' pictures display cross sections of the regions shown in A. MyoIAts midguts expressing GFP alone (control), or expressing SvbACT. Samples were stained for GFP (green) and PH3 (red). The graph plots number of mitotic PH3-positive cells per midgut of MyoIAts guts expressing GFP alone (ctrl), or expressing SvbACT , and SvbREP + pri. MyoIAts midguts expressing GFP and SvbACT (green), and stained for GFP (green) and PH3 (red). The picture is a single focal plane; the arrow shows a large GFP-positive cell, which is also positive for mitotic PH3. Summary of the role of SvbACT and SvbREP in the control of intestinal stem cell maintenance, proliferation and differentiation. \n",
      ": Svb repressor promotes enterocyte differentiationA Drawing of apical-basal organization of the intestinal epithelium and expression of svb mRNA as revealed by in situ hybridization.Figure 5Drawing of apical-basal organization of the intestinal epithelium and expression of svb mRNA as revealed by in situ hybridization. The inlet shows an enlarged view. esg-Gal4 midguts expressing mCherry and a svb::GFP rescue mini-gene, consisting of svb-cDNA tagged by GFP (green) and driven by E and 7 svb enhancers (see Fig 1). Samples were stained for GFP (green), mCherry (red) and Prospero (white). B' shows closeup views. The drawing at left schematizes the svb locus, with position of E3N and 9CJ2 enhancers. Close-up shows 9CJ2 sequence with binding sites for Pdm1 (orange). The right subpanels are posterior midguts showing expression of wild type 9CJ2 svb enhancer, and expression of 9CJ2-Pdm-mt in which Pdm1 binding sites have been mutated. esgts midguts expressing GFP alone (control), or expressing SvbREP. Samples were stained for GFP (green) and Prospero (red). Closeups correspond to boxed regions, with DAPI shown in purple and GFP-positive cells outlined in yellow. esgts-F/O midguts expressing GFP alone (control), or expressing SvbREP. Samples were stained for GFP (green) and cleaved DCP1 (red). E' shows quantification of the number of GFP-positive cells in esgts guts expressing GFP alone (ctrl), or DIAP, SvbREP, and SvbREP + DIAP. ReDDM lineage tracing in control midguts, or in midguts expressing SvbREP, and quantification of the percentage of progenitors (GFP-positive, RFP-positive) versus differentiated cells (GFP-negative, RFP-positive). esgts midguts expressing GFP alone (control), or expressing SvbREP. Samples were stained for GFP (green) and Pdm1 (red); arrows show enlarged GFP-positive cells which are also positive for Pdm1. \n",
      "Disruption of LRRC8A, but not of LRRC8D, abolishes ICl,vol and blocks volume regulation.Figure 3A, BVRAC currents (ICl,vol) of the HAP1 (A) and KBM7 (B) haploid cell lines. Left panels, example current traces of ICl,vol fully activated by hypotonic cell swelling measured with the voltage‐clamp protocol shown in (A). Dashed lines indicate zero current. Right panels, averaged current/voltage relationships of maximally activated ICl,vol. Consistent with VRAC currents, they needed hypotonic swelling for activation, displayed an I− > Cl− permeability sequence, and were blocked by DCPIB (Appendix Fig S2A-H). The difference in current inactivation between HAP1 and KBM7 cells can be explained by the fact that KBM7 cells hardly express LRRC8E (Fig EV1) which accelerates VRAC inactivation (Voss et al, 2014). At potentials > +100 mV, also KBM7 currents inactivated (Appendix Fig S2I). Data are presented as mean ± SEM; n = 5-10.C CDependence of regulatory volume decrease (RVD) of HEK cells on LRRC8 genes. Cells were exposed to hypotonic medium starting at t = 0, and intracellular calcein fluorescence was followed over ˜1 h as semiquantitative measure of cell volume. Data are presented as mean values ± SEM from sixteen wells. \n",
      "LRRC8 subunit‐ and osmolarity‐dependent caspase induction in HCT116 cells.Figure 4LRRC8 subunit‐ and osmolarity‐dependent caspase induction in HCT116 cellsA, BCisplatin‐induced caspase activity in the continuous presence of 200 μM cisplatin under isotonic conditions (A), or after 1 h exposure to 200 μM cisplatin under iso‐ and hypotonic conditions (B), was followed over time in WT, HAP1−/−, LRRC8D−/−, and LRRC8 −/−HCT116 cells. Results from HAP1−/− and LRRC8D−/− were obtained with two different clonal cell lines each and averaged.CCaspase activation after 1‐h exposure to 4 μM staurosporine under iso‐ or hypotonic conditions of WT, HAP1−/−, LRRC8D−/−, and LRRC8 −/−HCT116 cells.Data information: Data are presented as mean ± SEM, n = 3-6. *P < 0.05; **P < 0.01; and ***P < 0.001. Similar results were obtained in three independent experiments. Fold change in (A) refers to t = 0. Control experiments indicated that hypotonicity per se had no effect (Appendix Fig S4). \n",
      "Cisplatin–DMSO inhibition of ICl,vol depends on the LRRC8D subunit.Figure 7Cisplatin-DMSO inhibition of ICl,vol depends on the LRRC8D subunitUpper panel, example current traces (as in Fig 3A) of fully activated ICl,vol in HEK cells exposed to hypotonic solution containing vehicle (0.3% DMSO) or 200 μM cisplatin in 0.3% DMSO. Dashed lines indicate zero current. Lower panel, ICl,vol current densities (at −100 mV and 100 mV) of WT HEK cells treated with different cisplatin concentrations.No effect of 200 μM cisplatin/DMSO on ICl,vol in LRRC8D−/− HEK cells.Data information: Data are presented as mean ± SEM; the number of experiments is given for each bar; *P < 0.05; **P < 0.01. \n",
      "LRRC8A/LRRC8D‐containing channels transport the cellular osmolyte taurine.Figure 8Swelling‐induced efflux of 3[H]‐taurine from WT and LRRC8D −/− HEK cells (A, B) and partial rescue by transient transfection of LRRC8D (B). Rescue is incomplete due to low transfection/expression efficiency of LRRC8D (Voss et al, 2014). C-ESwelling‐induced efflux of 3[H]‐taurine from LRRC8(B,C,E) −/− (C), LRRC8(B,D,E) −/− (D), or LRRC8(B,C,D) −/− (E) HEK cells compared to WT cells.FCisplatin‐induced taurine efflux (over 30 min) from WT and HAP1−/−HEK cells after preincubation with 200 μM cisplatin for 4 or 12 h, or without cisplatin (0 h). Carbenoxolone (CBX; 100 μM) blocks taurine efflux from WT cells treated for 12 h with cisplatin, excluding taurine flux through unspecific leaks.Data information: Data are presented as mean ± SEM. Dashed lines in (A-E) represent isotonic efflux from the same individual experiments. Red arrows, change to hypotonic solution. Each panel shows the average of two independent experiments (n = 4 for each experiments, i.e., n = 8 in total). \n",
      "Activation of LRRC8 channels by pro‐apoptotic stimuli.Figure 5Activation of LRRC8 channels by pro‐apoptotic stimuliA-CCisplatin‐induced iodide influx into WT (black ■), but not HAP1−/− (green ▼) HEK cells indicates VRAC halide current activation during apoptosis. Cells expressing an iodide‐sensitive YFP variant were exposed to 200 μM cisplatin for periods of 0.5 h (A), 4.5 h (B), or 8.5 h (C) before adding extracellular I− (50 mM final). The difference in slopes of YFPfluorescence quenching between control and cisplatin‐treated cells semiquantitatively reflects VRAC current activation. Note that increased YFP quenching with cisplatin preincubation is not due to large non‐specific leaks resulting from cell morbidity. Such leaks should lead to a fast component of YFP quenching in WT, but not HAP1−/− cells after the pipetting artifact that immediately follows addition of iodide (indicated by arrows).DSwelling‐induced iodide influx into WT (black ■) and HAP1−/− (green ▼) HEK cells for comparison. Iodide (50 mM final) was added in isotonic or hypotonic (230 mOsm final) solution at the time indicated by arrow.E, FTime course of VRAC activation by 200 μM cisplatin (E) or 4 μM staurosporine (F) determined as in (A-C). Averaged maximal slopes of YFP quenching from eight wells (E) or 16 wells (F) each were evaluated to estimate iodide influx rates. WT (black ■) and HAP1−/− (green ▼). Data are presented as mean ± SEM. \n",
      "LRRC8 subunit‐ and osmolarity‐dependent carboplatin/cisplatin transport.Figure 6Carboplatin uptake into control KBM7 cells and two LRRC8D‐deficient clones (n = 6). Hypotonicity‐stimulated cisplatin uptake in HEK WT cells was selectively blocked by 100 μM carbenoxolone (CBX), a non‐specific blocker of VRAC (n = 3). Cisplatin uptake into HEK cells of indicated genotypes using 40 μM cisplatin under long‐term isotonic (C) or 200 μM cisplatin in short‐term hypo‐ and isotonic (D) conditions as function of time (n = 3). Similar results were obtained in HCT116 cells (Fig EV4). Cisplatin uptake (200 μM) into HEK cells of indicated genotypes. LRRC8(B,C,E) −/− and LRRC8(B,D,E) −/− cells express only LRRC8A and LRRC8D, and LRRC8A and LRRC8C, respectively (n = 3 for WT, 6 for LRRC8D −/−, 9 for LRRC8A −/− in E; n = 3 in F). Mean current densities of maximally activated ICl,vol at −80 mV. The number of cells is indicated for each column. Ratio of LRRC8‐dependent swelling‐activated cisplatin uptake (60 min) to mean ICl,vol (as in G) as function of genotype.Data information: Data are presented as mean ± SEM. *P < 0.05; **P < 0.01; and ***P < 0.001 (for (C, E) compared to HAP1−/− cells). Dotted lines in (D) and (F) highlight that there is no significant difference in isotonic 60‐min uptake between the genotypes. \n",
      ".Figure 1Figure 1. Identification of non-canonical p53 binding sites in human subtelomeres. (A and B) ChIP-seq tracks for p53 in camptothecin (CPT) treated or untreated HCT116 cells were aligned with RNAPII, CTCF, RAD21, and SMC1 for human chromosome18q (A) or 13q (B) subtelomeres. Position relative to the centromeric end of the terminal TTAGGG repeat tract are indicated in bp at the top. Subtelomeric DNA paralogy segment tracks, internal telomere-like sequence tracks (red CCCTAA), and annotated RefSeq gene tracks are indicated above the ChIP-seq tracks. (C) Alignment of human subtelomeric elements containing non-canonical p53 recognition sequences. The position of the RepeatMasker-defined LTR10c element is shown; also note the (TG) n microsatellite repeat adjacent to the p53 binding site. (D) Schematic organization of non-canonical p53 binding sites found in human subtelomeres. \n",
      ".Figure 7. CRISPR-deletion of 18qp53-binding site reveals local changes in transcription and γH2AX formation in response to DNA damage. (A) Schematic showing the deletion of ~70 bp region covering non-canonical p53 binding site and adjacent (TG) n repeat in chromosome18q. (B) Western blot analysis of CRISPR deleted cell lines 18q WT and 18q Δp53 treated with DMSO (D) or 50 µM etoposide (E) for 24 hr. Blots were probed with pan-p53, phospho-p53, γH2AX, or GAPDH (as loading control) (C-E) RT-qPCR for p21 (C), PARDG6 (D), or 18q subtelomeric RNA (E) in 18q WT or18q Δp53 treated with DMSO (black) or 50 μM etoposide (red) for 24 hr. (F) ChIP-qPCR assay for γH2AX at 18q subtelomere in 18q WT or 18q Δp53 treated as in panels B-E. \n",
      ".Figure 3. Transcriptional activity of subtelomeric p53-response elements. (A) Luciferase test constructs containing either empty test vector (top), p53-response elements from 18q subtelomere (middle) or MDM2 promoter (lower) were assayed in HCT p53-/- cells transfected with two independent preparations of empty vector or p53 WT expression vector. (B) RT-qPCR measure of p21 (top panel) or PARD6G (lower panel) in HCT116 p53-/- or p53+/+ cells after treatment with DMSO or 50 µM etoposide. (C) RT-qPCR analysis of TERRA transcripts from subtelomere 13q (upper) or 2q (lower). (D) RT-qPCR analysis of TERRA transcripts from 18q using two independent primer sets pr1 (top panel) or pr2 (lower panel) from HCT116 p53+/+ cells cultured with serum (grey bars) or in HBSS lacking serum (black bars). Error bars represent SD and * indicates p value <.05 using student t-test comparing treated and untreated samples. \n",
      ".Figure 6(A) HCT116 p53+/+ or p53-/- cells treated with 50 µM etoposide were assayed by Western blot with antibodies for p53, γH2AX, H3K9Ac, panH3, and GAPDH at times 0, 8, 16, or 24 hr post-treatment. (B) ChIP-qPCR for γH2AX (upper panels) or IgG control (lower panels) for cells treated as indicated in panel A. (C-D) RT-qPCR analysis for cells treated as shown in panel A. C-D) RT-qPCR analysis for cells treated as shown in panel A. (C) RT-PCR for the p21 (top), or PARD6G mRNA (lower). (D) RT-PCR for primer pairs spanning subtelomere18q as indicated below. Lower panel shows the No-RT control for 18qsubtelomere primers. RT-qPCR was quantified as relative to Actin with error bars indicating SD and * indicating p value <.05 using student t-test comparing etoposide (24 hr) relative to untreated for specific primer pairs. (E) Meta-data analysis of published GRO-Seq and ChIP Seq data sets. Gro-Seq Minus (red/maroon) or Plus (purple/pink) RNA strand from Control or Nutlin -3a treated MCF7 cells was mapped to human 18q subtelomere. Histone H3K27ac, H3K4me1, and H3K4me3ChIP-Seq data sets from untreated MCF7 are also shown for these subtelomeric regions. Right panel shows a zoom in of approximately 1000bp (light blue on left panel) surrounding the p53 response element within the subtelomere. (F) Summary of GRO-Seq and ChIP-Seq metadata analyses from human subtelomeres with p53 binding sites. Each p53 response element is located upstream of a region of bidirectional transcription which is modulated in a p53 dependent manner along with local bidirectional transcription at the p53 response element itself. \n",
      ".Figure 5Figure 5. p53-dependent histone modification changes in response to DNA damage stress. HCT116p53-/- (left panels) or p53 +/+ (right panels) were treated with DMSO (black) or 50 µM etoposide (red) for 24 hrs and assayed by ChIP-qPCR at various locations relative to the TTAGGG repeats across the chromosome18qsubtelomere (panel A) or 13qsubtelomere (panel B) or for control regions (p21 promoter, GKN1/2, PARKIN) as indicated below. ChIP- for p53, H3K9Ac, H3K27Ac, γH2AX, or IgG control were quantified as % input. Error bars indicates SD and * indicates p value<.05 using student t-test comparing ChIP values in p53+/+ relative to p53-/- cells for specific primer pairs. (C) Dot blot analysis of telomere repeat or Alu repeat DNA from p53+/+ or p53-/- HCT116 cells treated with DMSO or etoposide (ETP) (as described in panel A), that were subject to ChIP with antibodies to IgG, p53, γH2AX, TRF2, or TRF1. Quantitation of three independent ChIP-assays for TRF1 and TRF2 is shown in the panel to the right. ( (E) ChIP-qPCR for TRF2 is shown for cells treated as in panel A using primers across the subtelomeres of 18q (left) or 13q (right). \n",
      ".Figure 2Stress-induced p53 binding at human subtelomeres. (A) Western blot of HCT116 p53-/- and p53+/+ cells treated with DMSO (D) or 50 µM etoposide (E) for 24 hrs. Cell lysates were assayed for total p53, phospho-Ser15 p53 (p-p53), γH2AX, or GAPDH. (B) ChIP-qPCR was assayed in HCT116 p53-/- or p53 +/+ cells treated with DMSO or etoposide as in panel A, and assayed by ChIP with antibodies to p53 (Ab6) or control IgG, followed by qPCR for p53 binding site in subtelomere 18q (top panel) or 13q (lower panel). (C) HCT116 p53-/- and p53+/+ cells were cultured in DMEM with 10% FBS or Hank's buffered saline solution (HBSS) lacking FBS and then assayed by ChIP with either IgG control (grey) or two different p53-specific antibodies Ab6 (red) or FL(black). ChIP was assayed by qPCR with primers proximal to p53 binding sites at 18q, 13q, or p21 promoter. . (D) Doxycyclin (Dox)-inducible p53 H1299 cells were treated without (-) or with (+) Dox in media containing 10% serum or in serum free HBSS, and then subject to ChIP assay with control IgG or p53-specific Ab6 antibody. ChIP DNA was assayed by qPCR using primers specific for p53 binding site at p21 (top panel), p53-negative control IGX1A (middle panel), or p21 positive control (lower panel). Error bars represent SD, and * indicates p value < .05. p-values were determined by Student t-test. \n",
      ".Figure 4. p53-dependent cell viability and telomere protection in response to DNA damage. (A) Western blot of total p53, γH2AX, and GAPDH in HCT116 p53-/- or p53+/+ cells treated with 1 µM or 50 µM etoposide for 24 hrs. s. (B) Cell cycle analysis of HCT116 p53-/- or p53+/+ cells untreated or treated with 50 µM etoposide for 24 hrs. Cell cycle was determined by FACS analysis of propidium iodide stained cells. (C) Cell clonal survival assay for HCT116 p53-/- or p53+/+ cells treated with DMSO or 1, 1.5 or 2 µM etoposide for 24 hr prior to replating. . (D) Quantification of clonal survival assay relative total cell numbers (top panel) or normalized to DMSO controls (lower panel). E) TIF assay for HCT116 p53+/+ or p53-/- cells treated with 1 µM etoposide for 3 hrs. Telomere DNA FISH with TelC probe (red), γH2AX (green), Dapi (blue). (F) Quantification of TIF assay for % cells with > 4 TIFs per cell for p53+/+ (black) or p53-/- (red) cell. G) Telomere length assay showing p53-/- or p53+/+ cells treated with either DMS0, 1 or 50 µM etoposide. Ethidium bromide (EtBr) stained agarose gels (left panel), Southern blot hybridized with Tel C probe (middle panel) or with α Satellite DNA promote (right panel). Quantification of telomereDNA signal relative to invariant bands in α Sat. Error bars represent SD and * indicates p values <.05 determined by student t-test (TLA), or chi-square (TIF assay). (H) Two-dimensional agarose gel analysis of telomeric DNA from HCT p53+/+ or or p53-/- treated with 50 µM etoposide for 24 hrs. Ethidium gel (top) and Southern blot probed with TelG probe (lower panel). Single stranded telomere DNA (red arrow) and recombination/ replication-associated structures (blue arrow) are indicated. \n",
      ".Figure 8Figure 8. Model of p53 binding and local histone modification as a form of genome protection to DNA damage stress. \n",
      "a) Co-localization between ATG12 and VAMP2-HA in control and CALM knockdown HeLa cells stably.f5(a) Co-localization between ATG12 and VAMP2-HA in control and CALM knockdown HeLa cells stably expressing VAMP2-HA. Confocal pictures are shown with magnified areas on the right showing co-localization between ATG12 and VAMP2 in control cells and no co-localization in CALM knockdown cells. Quantification of ATG12-VAMP2 co-localization is shown on the graph as the Pearson's coefficient (data are mean ±s.d.;*P0.05; two-tailed t-test). Scale bars, 5 μm. (b) Western blot analysis of VAMP2, actin and LC3-II in HeLa cells where VAMP2 was knocked down, as indicated. The cells were starved in Hanks balanced salt solution (HBSS) and treated with Baf A1 as indicated. (SE, short exposure; LE, longer exposure.) Quantification of LC3-II/actin ratio is shown. Data are representative of three independent experiments and shown as mean ±s.d. (*P0.05; NS, not significant, two-tailed t-test). (c) LC3 dot counting in CALM and VAMP2 knockdown cells. CALM or VAMP2 were knocked down in cells expressing GFP-LC3. The cells were kept in full medium or starved in HBSS for 4 h, fixed and subjected to automated fluorescence microscopy to score the number of LC3 dots per cell. The number of LC3 dots per cell (shown as mean ±s.d.) is shown on the graph for each condition (n≥300 cells per condition; BC, basal conditions). (*P0.01; NS, not significant, two-tailed t-test). (d) Number of p62 dots per cell in VAMP2 knockdown. HeLa cells where VAMP2 was knocked down were fixed and subjected to microscopy after labelling endogenous p62 using specific antibody. The data represent the number of p62 dots per cell shown as mean ±s.d. (*P0.05; two-tailed t-test; n≥300 cells per condition). (e) Percentage of huntingtinQ74-expressing cells with aggregates in VAMP2 knockdown HeLa cells. Data are from one representative experiments performed in triplicate, out of three independent experiments and shown as mean ±s.d. (*P0.05; two-tailed t-test). (f) Western blot analysis of VAMP2, actin and LC3-II in HeLa cells stably expressing VAMP2-HA wild type or VAMP2-HA mutated in the CALM-binding site at different levels (wild-type clone 6, wt Cl6: low level; wild-type clone 11, wt Cl11: high level; mutant clone 11, AA Cl11: low level; mutant clone 12, AA Cl12: high level). The cells were treated with Baf A1 as indicated. (SE, short exposure; LE, longer exposure). Quantification of LC3-II/actin ratio is shown. Data are representative of three independent experiments and shown as mean ±s.d. (*P0.05; two-tailed t-test). \n",
      "a) Western blot analysis of CALM, actin, GAPDH and LC3-II in several cell lines.f1(a) Western blot analysis of CALM, actin, GAPDH and LC3-II in several cell lines (HeLa, HEK, CAD and murine embryonic fibroblast (MEF)) where CALM was knocked down using shRNA or siRNA, as indicated, or knockout with rescue experiment. In all experiments in this paper, we used a scramble siRNA or a luciferase shRNA as controls. The cells were starved in Hanks balanced salt solution (HBSS) and treated with Baf A1 as indicated. (BC, basal conditions; SE, short exposure; LE, longer exposure). Quantification of LC3-II/actin or GAPDH ratio is shown in Supplementary Fig. 1A. *Not specific. (b) LC3 dot counting in CALM knockdown cells. CALM was knocked down in HeLa cells expressing GFP-LC3 and with or without an siRNA-resistant form of wild-type CALM (rescue), as indicated. The cells were starved in HBSS for 4 h, fixed and subjected to fluorescence microscopy to score the number of LC3 dots per cell. The knockdown efficiency and the level of the siRNA-resistant form of CALM are shown on the left on the western blotting. The number of LC3 dots per cell (shown as mean ±s.d.) is shown on the graph for each condition (n≥300 cells per condition; BC, basal conditions). The ratio of the number of LC3 dots per cell between starvation and basal conditions is shown on the right (*P0.01; NS, not significant, two-tailed t-test). Pictures obtained from automated microscope are shown. Scale bars, 20 μm. \n",
      "a) p62 vesicle.f2(a) p62 vesicle formation in CALM knockdown HeLa cells. Confocal pictures are shown. #CALM-downregulated cells where p62 vesicles accumulate. Data are representative of three independent experiments and shown as mean ±s.e.m. (n≥500 cells; *P0.01; two-tailed t-test). Scale bars, 5 μm. (b) Western blot analysis of p62, actin and GAPDH in HeLa cells and HEK cells (basal conditions, BC, or without serum for 1 h) where CALM was knocked down using shRNA or siRNA, as indicated. Data are mean ±s.d (n=3 experiments for HeLa cells and HEK cells; *P0.05; two-tailed t-test). (c) Percentage of huntingtinQ74-expressing cells with aggregates in CALM knockdown HeLa cells. Data depict one representative experiments performed in triplicate, out of three independent experiments and shown as mean ±s.d. (*P0.05; two-tailed t-test). (d) Tau-positive tangle formation in CALM knockdown cells. HeLa cells transiently expressing DsRed-tau 4R were treated with Baf A1 for 4 h as indicated. Cells were fixed and analysed by confocal microscopy after immnunostaining for phosphorylated tau using PHF1 antibody (green). Data represent the number of cells with phosphorylated tau-positive tangles as mean ±s.e.m. (n=3 experiments; *P0.01; two-tailed t-test). Scale bars, 5 μm. (b) Western blot analysis of p62, actin and GAPDH in HeLa cells and HEK cells (basal conditions, BC, or without serum for 1 h) where CALM was knocked down using shRNA or siRNA, as indicated. Data are mean ±s.d (n=3 experiments for HeLa cells and HEK cells; *P0.05; two-tailed t-test). \n",
      "a) Formation of endogenous ATG12 vesicles in CALM knockdown HeLa cells in basal (BC) and serum.f3(a) Formation of endogenous ATG12 vesicles in CALM knockdown HeLa cells in basal (BC) and serum starvation conditions. Confocal pictures are shown with arrows indicating ATG12 vesicles in starvation conditions. Data are from one representative experiment, out of three independent experiments. Data shown as mean ±s.e.m. (n≥500 cells; *P0.01; two-tailed t-test). Scale bars, 5 μm. (b) Co-localization between GFP-ATG16L1 and internalized cholera toxin subunit B conjugated with Alexa555 (CTX; 20 min) in CALM knockdown HeLa cells. Confocal pictures are presented with magnified areas showing the co-localization between ATG16L1 and CTX in greater detail. Scale bars, 5 μm. The Pearson's coefficient between ATG16L1 and cholera toxin is shown. Data are representative of three independent experiments and shown as mean ±s.d. (n≥20 cells; *P0.05; two-tailed t-test). (c) Size of endogenous ATG12 vesicles in CALM knockdown HeLa cells in starvation conditions. Confocal pictures are shown with arrows indicating ATG12 vesicles and # indicating CALM knockdown cells. Data are representative of three independent experiments and shown as mean ±s.d. (n≥100 vesicles; *P0.05; two-tailed t-test). (AU, arbitrary unit). Scale bars, 5 μm. (d) Live-cell imaging of ATG16L1-GFP in CALM knockdown HeLa cells. ATG16L1 pictures from various time points of a 5-min movie are shown in inverted grey style. Arrows indicate ATG16L1 vesicles. The number of fusion events per 5 min is shown. Data are representative of five movies and shown as mean ±s.d. (*P0.05; two-tailed t-test). Scale bars, 5 μm. (e) In vitro fusion assay of post-nuclear supernatant from HeLa cells expressing either GFP-ATG16L1 or mStrawberry-ATG16L1 in control and CALM knockdown conditions. Confocal pictures are shown where ATG16L1-mStrawberry signal is shown in purple to enable better visualization. Fused vesicles appear in white. The ATP-negative condition, which prevents SNARE-dependent fusion, is also shown as a control for the reaction. Magnified areas are shown to allow visualization of the vesicles. The percentage of fused vesicles is represented. n=numbers of vesicles scored per field (a minimum of five fields were analysed per condition). Data are representative of two independent experiments and shown as mean ±s.d. (n≥100 vesicles). Scale bars, 5 μm. (*P0.05; two-tailed t-test). \n",
      "a) LC3 levels in CALM knockdown cells.f4(a) LC3 levels in CALM knockdown cells. CALM was knocked down in cells expressing or not expressing an siRNA-resistant form of CALM wild type (wt) or CALM 219 mutant, as indicated. The cells were lysed and subjected to western blotting. The knockdown efficiency, LC3-II levels and the level of the siRNA-resistant form of CALM are shown on the western blotting. (b) LC3 dots counting in CALM knockdown cells. CALM was knocked down in cells expressing GFP-LC3 with or without an siRNA-resistant form of CALM wild type (wt) or CALM mutant (219 mutant) as indicated. The cells were kept in full medium or starved in Hanks balanced salt solution (HBSS) for 4 h, fixed and subjected to microscopy to score the number of LC3 dots per cell. The ratio of the number of LC3 dots per cell (shown as mean ±s.d.) between starvation and basal conditions is shown on the graph (*P0.01; NS, not significant, two-tailed t-test; n≥300 cells per condition; BC, basal conditions). (c) Co-localization between ATG9A and VAMP3-HA or GFP-LC3 and VAMP8-HA in control and CALM knockdown HeLa cells stably expressing VAMP3-HA or VAMP8-HA. Confocal pictures are shown with magnified areas showing co-localization between ATG9A and VAMP3 or GFP-LC3 and VAMP8 in control cells and no co-localization in CALM knockdown cells. Quantification of ATG9A-VAMP3 or GFP-LC3-VAMP8 co-localization is shown on the graph as a Pearson's coefficient (data are mean ±s.d.; *P0.05; two-tailed t-test). Scale bars, 5 μm. \n",
      ".Figure 1Coomassie- stained proteins from bovine retina that were co-immunoprecipitated by the anti-RIBEYE antibody 2D9 (lane 3). Lane 2, control precipitation by irrelevant mouse IgGs; lane 1, input (10% of the total loaded in each lane) Western blot of the immunoprecipitated proteins (as in A, 1% of the total protein was loaded per lane) probed with anti-RIBEYE (Ba), anti-CASPR1 (Bb) and anti-CNTN1 (Bc). The 2D9 antibody immunoprecipitated RIBEYE (Ba, lane 3), CASPR1 (Bb, lane 3) and CNTN1 (Bc, lane 3); these proteins were not detectable in the control immunoprecipitations. Bd) As a loading control, heavy chains of the precipitating antibody (IgG hc) were visualized by the goat anti-mouse IgG antibody conjugated to peroxidase \n",
      "a) Western blotting showing downregulation of lap (i) or increase in.f7(a) Western blotting showing downregulation of lap (i) or increase in Atg8a-II (ii) expression level in adult fly heads on lap downregulation using the UAS-RNAi lines lapGD12732 or lapKK105767, or the heterozygous allele lap1. Quantification of lap/actin and Atg8a-II/actin is shown. Genotypes: Control w; elav-GAL4/+; for the RNAi lines: w; elav-GAL4/ lapKK105767 and w; elav-GAL4/+; lapGD12732/+; for the lap1 allele: w; elav-GAL4/+; lap1/+. (b) Western blotting showing the accumulation of tau in Drosophila adult flyheads on lap downregulation using the UAS-RNAi lines lapGD12732 or lapKK105767, or the heterozygous allele lap1. Quantification of tau/lapGD12732 is shown. Genotypes: Control w; elav-GAL4/+; for tau-WT: w; elav-GAL4/+; UAS-tau-WT/+; for RNAi lines: w; elav-GAL4/ lapKK105767; UAS-tau-WT/+ and w; elav-GAL4/+; lapGD12732/tau-WT; for the lap1 allele: w; elav-GAL4/+; lap1/+ or w; elav-GAL4/+; lap1/UAS-tau-WT. \n",
      "a) Size of endogenous ATG12 vesicles in HeLa cells stably expressing either.f6(a) Size of endogenous ATG12 vesicles in HeLa cells stably expressing either wild-type or mutant VAMP2. Confocal pictures are presented with magnified areas showing ATG12 vesicles. Data are representative of three independent experiments and shown as mean ±s.d. (n≥100 vesicles; *P0.05; two tail t-test). Scale bars, 5 μm. (b) Co-localization between ATG12 and LC3 in HeLa cells stably expressing either wild-type clone 11 or mutant clone 12 VAMP2. Confocal pictures are presented with magnified areas showing ATG12-LC3 co-localization. The Pearson's coefficient between ATG12 and LC3 is shown. Data are representative of three independent experiments and shown as mean ±s.d. (n≥100 vesicles; *P0.05; two-tailed t-test). Scale bars, 5 μm. (c) Size and number of endogenous ATG12 vesicles in VAMP2-knockdown HeLa cells in starvation condition. Confocal pictures are shown with arrows indicating ATG12 vesicles. Data are representative of three independent experiments and shown as mean ±s.d. for the size of ATG12 vesicle. (n≥100 vesicles; *P0.05; based two-tailed t-test). Scale bars, 5 μm. (d) Live-cell imaging of ATG16L1-GFP in VAMP2-knockdown HeLa cells. Confocal pictures from various time points of a 5-min movie are shown in inverted greyscale. Arrows indicate ATG16L1null. The number of fusion events per vesicles is shown. Data are representative of five movies and shown as mean ±s.d. (*P0.05; two-tailed t-test). Scale bars, 5 μm. (e) In vitro fusion assay of post-nuclear supernatants from control and VAMP2 knockdown HeLa cells expressing either GFP-ATG16L1 or mStrawberry-ATG16L1. Confocal pictures are shown where ATG16L1-mStrawberry signal is shown in purple to enable better visualization. Fused vesicles appear in white. The ATP-negative condition, which prevents SNARE-dependent fusion, is also shown as a control of reaction. Magnified areas are shown to allow visualization of the vesicles. The percentage of fused vesicles (white) is represented. Data are representative of two independent experiments and shown as mean ±s.d. (n≥100 vesicles). Scale bars, 5 μm. (*P0.05; two-tailed t-test). n=numbers of vesicles scored per field. \n",
      "a) Histological section to demonstrate the individual cell layers of.f10(a) Histological section to demonstrate the individual cell layers of the zebrafish retina. The photoreceptor layer (PR, comprising rod and cone photoreceptors) lies immediately adjacent to the retinal pigment epithelium (RPE) at the outermost surface of the eye. Thioflavin-S labelling of retinal sections was used to identify neurofibrillary tangles in the photoreceptor layer (marked with yellow dotted lines). No labelling was observed in the retina of rho::GFP at 8 d.p.f., whereas distinct thioflavin-S-positive tangles (arrows) were observed in the photoreceptor layer of rho::GFP-tau fish. Note, the RPE is highly autofluorescent due to the presence of silver pigment. High power regions are shown in the top right of each panel. (b) Unilateral electroporation of CALM into the retina of rho::GFP-tau zebrafish resulted in a marked increase in thioflavin-S positive tangles in the electroporated retina in the photoreceptor layer (PR) compared with the control side. Top panel are lower magnification images to show the retinal cell layers. Thioflavin-S labelling is restricted to the PR layer. Note the RPE is highly autofluorescent due to the presence of silver pigment. Bottom panel are higher magnification images to show individual thioflavin-S tangles the largest of which are indicated by arrows. \n",
      ".Figure 2Localization of CASPR in semi-thin sections of retinas from either MOG/CFA-injected mice (EAE mice) or CFA-injected control mice (9 days after injection). Semi-thin sections were double-immunolabelled with antibodies against CASPR1 and RIBEYE (A-B OPL, outer plexiform layer; INL, nuclear layer; IPL, inner plexiform layer. Scale bars: 20μm Localization o visual-arrestin (v-arrestin) in semi-thin sections of retinas from either MOG/CFA-injected mice (EAE mice) or CFA-injected control mice (9 days after injection). Semi-thin sections were double-immunolabelle with antibodies against v-arrestin and RIBEYE (C-D) V-arrestin, visual arrestin; OPL, outer plexiform layer; INL, nuclear layer; IPL, inner plexiform layer. Scale bars: 20μm \n",
      "ai) Western blot analysis of tubulin, LC3-I and LC3-II in zebrafish.f8(ai) Western blot analysis of tubulin, LC3-I and LC3-II in zebrafish larvae where ATG7 was downregulated, as indicated. (ii) Western blot analysis of CALM-HA, actin and LC3-II in zebrafishlarvae where CALM-HA was expressed, as indicated. The larvae were treated with ammonium chloride (NH4Cl), as indicated. (SE, short exposure; LE, longer exposure). Quantification of LC3-II/actin ratio is shown. Data are representative of three independent experiments and shown as mean ±s.d. (*P0.05; two-tailed t-test). (b) Modulation of autophagy alters Dendra-tau clearance dynamics. The fluorescence intensity of each individual cell was quantified at each timepoint (n≥ 31 cells, ≥9 larvae per treatment group) and mean cell intensity values for each drug treatment at each timepoint were calculated. Images were taken immediately after photoconversion and at 24, 30 and 48 h intervals thereafter. Rapamycin treatment significantly increased the rate of Dendra-tau clearance. Ammonium chloride (NH4Cl) treatment significantly decreases the rate of Dendra-tau clearance (**P0.01, ***P0.001, one-way analysis of variance (ANOVA)). Error bars are ±s.e.m. (c) Dendra-tau clearance in the presence of CALM: representative images of CALM with mosaic expression of Dendra-tau and full-length Dendra-tau taken immediately after photoconversion and at 6, 24, 30, 48 and 54 h after conversion. The fluorescence intensity of individual cells was quantified and mean fluorescent intensity of cells co-expressing Dendra-tau and either full-length CALM or Δ-ANTH CALM (larvae-ANTH-HA) constructs (n≥ 100 cells, ≥9 larvae per treatment group) at each time point was calculated. (i) Expression of full-length HA significantly delayed the clearance of CALM at all time points compared with Δ-ANTH CALM (***P0.001, one-way ANOVA). One graph, representative of three independent experiments, is presented. (CALM-ANTH-HA: 132 cells, 20 fishes; CALM-CALM: 18 cells, 8 fishes). Another two experiments are shown in Supplementary Fig. S5C. Error bars are mean ±s.d. (ii) Treatment with CALM alters the dynamics of CALM clearance. Expression of full-length CALM significantly delayed the clearance of CALM at all time points compared with Δ-ANTH CALM, as observed in i (P0.001, one-way ANOVA). However, treatment of Δ-ANTH CALM injected CALM with CALM slows the CALM clearance by 70% to a level comparable to that observed in larvae injected with full-length Dendra-tau. Ammonium chloride treatment of Dendra-tau injected with full-length null results in a modest (22%) decrease in Dendra-tau clearance, suggesting that CALM overexpression and ammonium chloride treatment have a cumulative effect (n≥25 cells, n≥9 Dendra-tau per treatment group). Error bars are mean ±s.d. Note that i and ii are distinct experiments. \n",
      "a) rho::GFP-tau.f9(a) rho::GFP-tau fishes were incubated from 3 to 9 d.p.f. in either dimethyl sulphoxide (DMSO) or rapamycin (i) or from 3 to 7 d.p.f in EM alone or 10 mM ammonium chloride or 100 nM Wortmannin (ii). Images through the central retina at 9 d.p.f. (top panel; (i)) reveal rod degeneration (arrow) throughout the retina in control (DMSO-treated) larvae, whereas rod photoreceptors are present throughout the retina, particularly in the central region following treatment with rapamycin (arrow). Images taken through the central retina at 7 d.p.f. (bottom panel; (ii)) show normal photoreceptors in the marginal zones (arrows) and only limited numbers in the central region. NH4Cl exacerbates degeneration-photoreceptors are absent from the central retina and reduced/absent from marginal zones (arrows). Sections were stained with anti-rhodopsin (1D1) antibody. GFP labels whole rod photoreceptors, whereas rhodopsin is present in the rod outer segment. GFP co-localizes with the red rhodopsin label in all experimental conditions. Scale bars, 50 μm. Quantification of rodphotoreceptor degeneration (n=10 larvae per group; ***P0.001; ###P0.001, two-tailed unpaired t-test). Error bars are mean ±s.d. (b) Full-length CALM electroporation into rho::GFP larvae did not cause degeneration., while full-length CALM electroporation into rho::GFP-tau larvae exacerbated photoreceptor degeneration. Central retina sections in the region of the optic nerve head are presented. Scale bars, 50 μm. Data represent the ratio (in %) of the intensity of the GFP signal between CALM-HA electroporated eye versus control eye for five individuals per transgenic line. ***P0.05; two-tailed t-test. Error bars are mean ±s.d. (c) NH4Cl treatment of rho::GFP-tau immediately after unilateral CALM electroporation caused photoreceptor degenerationon the control (non-electroporated) side but did not alter the degeneration caused by CALM electroporation. Rapamycin treatment of Rapamycin::GFP-tau immediately after unilateral CALM electroporation rescued photoreceptors on the control (non-electroporated) side but not on the CALM-electroporated side. To demonstrate that loss of GFP corresponds to loss of photoreceptors, sections were stained with anti-rhodopsin (1D1) antibody. Wilcoxon signed rank test was used to compare the left eye versus the right eye of the same fish; Mann-Whitney test was used to compare drug treatment in the right eye of different fishes. *P0.05. Scale bars, 50 μm. Error bars are mean ±s.e.m. \n",
      ".Figure 3Localization of CNTN1 in semi-thin (0.5μm-thin) sections of retinas from either MOG/CFA-injected mice (EAE mice) or CFA-injected control mice (9 days after injection). (A,B) Sections were double-immunolabelled with antibodies against CNTN1 and RIBEYE proteins in the OPL and IPL. ONL, outer nuclear layer; OPL, outer plexiform layer; INL, nuclear layer; IPL, inner plexiform layer Quantification of the immunosignals of the indicated proteins in the OPL and IPL. ONL, outer nuclear layer; OPL, outer plexiform layer; INL, nuclear layer; IPL, inner plexiform layer; n.s., non-significant; ***, p<0.001 (Mann-Whitney U test, actual p-values are given in C) (N=6 independent set of experiments; n=295 images (CFA); n=274 images (MOG/CFA) for quantification of RIBEYE immunosignals; n=95 (CFA); n=92 (MOG/CFA) for quantification of CASPR1 immunosignals; n=91 (CFA), n=82 (MOG/CFA) for CNTN1 quantification; n=109 (CFA); n=100 (MOG/CFA) for quantification of v-arrestin. Scale bars: 20μm \n",
      ".Figure 6Retinal slices from MOG/CFA-injected and CFA-injected SypHy mice were stimulated by applying a 25mM K+-containing depolarization solution for 1min (Fig. 6A). In response to depolarization, we observed a strong increase in fluorescence (Fig. 6A) that can be best fitted by a double-exponential curve (B). Amplitudes of fast and slow release (together with the respective time constants) of the first depolarization response are plotted in C) \n",
      "Interactions between Atg30, Atg8 and Atg11.Figure 2(A1) Atg30 sequences from aa 68-122 of wild‐type (Atg30), the S71A mutant (Atg30S71A) and two different deletions of the sequence between the AIM and phosphosite required for Atg11 binding in Atg30 (Atg30WDILSSS and Atg30WSILSSS). (A2) Pexophagy experiments of Δatg30 cells complemented with appropriate wild‐type or mutant Atg30 proteins described in A1. (A3) Two‐hybrid assays between Atg30 wild‐type or mutants (described in A1) and Atg8. Phosphomimic S71E was included to detect the interaction with Atg8. (B1) Schematic of the two Atg30 molecules, one with an Atg8‐binding site mutated and a second with an Atg11‐binding site (A11‐BS) mutated, used to complement Δatg30 cells by co‐expression. P: indicates phosphorylation in vivo. (B2) Pexophagy experiments of Δatg30 cells complemented with the two Atg30 molecules described in B1. (C1) Schematic of the Atg30 mutations, S71A and S112A that impair Atg8 and Atg11 binding, respectively. (C2) Pexophagy assays of Δatg30 cells complemented with Atg30 wild‐type and mutants. aa, amino acids; AD, activation domain; AIM, Atg8‐family‐interacting motif; BD, binding domain; HA, haemagglutinin. \n",
      "Atg8 and Atg11 localization during pexophagy.Figure 3(A) Large phagophore membrane formation in WT and the Atg30 mutant cells monitored by GFP-Atg8 during pexophagy conditions. (B) Localization of GFP-Atg11 during pexophagy of methanol‐induced peroxisomes in cells expressing WT or mutant Atg30 proteins. White arrows indicate correct localization and yellow arrows indicate mislocalization, or in case of Atg8 localization, indicate absence of phagophore membrane elongation. Peroxisomes were labelled with BFP-SKL and vacuoles with FM4‐64. Scale bar, 5 μm. GFP, green fluorescent protein; WT, wild type. \n",
      ".Figure 5Optic nerve parameters are unaltered in early EAE (9 days after injection). (A,B) Cryostat sections of the optic nerve from MOG/CFA-injected mice (own lab-made suspension; for comparison with commercial MOG/CFA suspensions, see Appendix Fig. S12) and CFA-injected control mice were stained with anti-Nav antibody (1:50 dilution) to label the node of Ranvier and with anti-CASPR1 antibody (1:500 dilution) to stain the paranodal region. Length of the node of Ranvier and length of the paranodal region were quantified in C) D) (E1, E2) Richardson-Blue stained semi-thin sections from the indicated optic nerve samples. (E2, E3, F2, F3) Transmission electron micrographs of the optic nerves from MOG/CFA- and CFA-injected animals. Abbreviations: ax, axon; my, myelin sheath. Axon number and myelin thickness were quantified in G) and H). Scale bars: 1μm (A,B,E,F) \n",
      ".Figure 7The amount of CASPR1 protein in the synapse is also altered in the PLP mouse model of multiple sclerosis. Semi-thin sections of PLP/CFA and CFA-injected mice were probed with the indicated antibodies (A, B; see also Appendix Fig. S13) The synaptic amount of CASPR1 is strongly reduced both in the OPL and IPL. RIBEYE is only reduced in the OPL but not significantly in the IPL. CNTN1 is strongly reduced in both OPL and IPL (C, see also Appendix Fig. S13). N=4 PLP/CFA-injected mice; N=4 CFA-injected mice; n=140 slices (CFA, RIBEYE); n=143 slices (PLP/CFA, RIBEYE); n=47 slices (CFA, CASPR1); n=53 slices (PLP/CFA, CASPR1); n=49 slices (CFA, CNTN1); n=46 slices (PLP/CFA, CNTN1); n=44 slices (CFA, v-arrestin); n=44 slices (PLP/CFA, v-arrestin) (D,E) Cryostat sections of the optic nerve from PLP and control mice were stained with anti-Nav-channel antibody to label the node of Ranvier and with anti-CASPR1 antibody to stain the paranodal region. Length of the node of Ranvier and length of the paranodal region were quantified in G) H). (F) transmission electron micrographs of the optic nerves from PLP/CFA- and CFA-injected animals (F1-F4). Axon numbers and myelin thickness were quantified in I) J) \n",
      "Atg30 interacts with Atg8 through a cryptic AIM, and phosphorylation upstream of the AIM regulates their interaction.Figure 1(A) IP of GFP-Atg8 (α‐GFP), Atg30-Flag (α‐Flag) and Pex3 (α‐Pex3) under pexophagy conditions. The abundant peroxisome matrix protein, AOX, was used as a negative control. Input: total lysate; ϕ: IP without antibody. (B) Two multiple sequence alignments, including 11 Atg30 homologues and 11 Atg32 homologues (identical residues are indicated with black boxes, and similar residues with grey boxes), and a sequence logo of the combined multiple sequence alignments from Atg30 and Atg32 homologues listed currently in GenBank. (C,D) Pexophagy experiments of Δatg30 (ϕ), wild‐type (Atg30) and Atg30 AIM mutant (Atg30W73A F76A) cells were done by fluorescence microscopy, following the degradation of peroxisomes labelled with BFP fused at its C‐terminus to the Ser-Lys-Leu peroxisomal targeting signal 1 (BFP-SKL) and biochemically by monitoring peroxisomal thiolase degradation. Vacuoles were labelled with FM4‐64. Scale bar, 5 μm. (E) AH109 cells were transformed with two yeast two‐hybrid assay plasmids, AD and BD, which encode the indicated domains fused with Atg30, Atg32 and Atg8 or an empty vector, as negative controls and grown on +His and −His+40 mM 3‐AT plates. (F) . Δatg30 (ϕ) and Δatg30 cells complemented with Atg30-HA (Atg30) and several Atg30-HA mutants were immunoprecipitated (α‐HA IP) under pexophagy conditions. In addition, α‐HA IP of Δatg30 cells (ϕ) and Δatg30 cells complemented with Atg30-HA (Atg30) were incubated with (+) and without (−) λPP. Input: total lysate. (G) Pexophagy in atg30 mutants was monitored by following thiolase levels of oleate‐induced peroxisomes after shifting cells to SD‐N. aa, amino acid; AD, activation domain; AIM, Atg8‐family‐interacting motif; AOX, alcohol oxidase; BD, binding domain; GFP, green fluorescent protein; HA, haemagglutinin; IP, immunoprecipitation; λPP, λ protein phosphatase. \n",
      ".Figure 4MOG/CFA- but not CFA-injected mice contain reactive antibodies for retinal proteins in blood and show altered spatial visual behavior. (A) 7-9 days after injection, blood from MOG/CFA-injected (red) but not from CFA-injected mice (black), elicited strong antibody responses in ELISA. Antibodies for RIBEYE and SV2 were used as positive controls (green), secondary antibody only as a negative control (white). Normalized data are given as mean ± S.E.M. (N=4 (CFA); N=5 (MOG/CFA). ** p<0.01 (significance was assessed by Mann-Whitney test, precise p-values given in the figure) Control Western blot of HEK293 protein lysates obtained from cells transfected with mCherry (lane 1) or CASPR1-mCherry cDNA (lane 2) and probed with anti-mCherry antibody (C,D) Western blot of protein lysates as in B) probed by blood samples from mice before and 7 to 9 days after MOG/CFA injection. Only blood samples obtained from MOG/CFA-injected mice revealed immunoreactivity against CASPR1 (lanes 15,16; top panels) whereas samples obtained before immunization (lanes 7,8; top panels) did not. The same filters were re-probed with anti-actin antibodies as loading control (C,D) Western blot of protein lysates as in B) probed by blood samples from mice before and 7 to 9 days after MOG/CFA injection. Only blood samples obtained from MOG/CFA-injected mice revealed immunoreactivity against CASPR1 (lanes 15,16; top panels) whereas samples obtained before immunization (lanes 7,8; top panels) did not. The same filters were re-probed with anti-actin antibodies as loading control. (E) Summary of data from (C,D) shown as mean ± S.E.M. (N=4, before injection; N=5, MOG/CFA-injected). ***, p<0.001; n.s., non-significant (unpaired two-tailed Student's t-test, precise p-values given in the figure) Spatially guided visual behavior, measured as frequency threshold with a virtual optokinetic system 9 days after immunization, is significantly decreased in MOG/CFA-injected mice compared to CFA-injected control litters. Data are shown as median; boxes and whiskers illustrate 25-75 and 5-95 percentiles of values, respectively. Points below and above whiskers are drawn as individual points. At each trial, mice were measured three times. Home-made and commercial MOG/CFA suspensions were compared as well as PLP-injected mice with control-injected mice. Statistical analysis was done by ANOVA (0,05 significance level), mean comparison probabilities were Bonferroni corrected. N=32, n=96 (MOG/CFA); N=24, n=72 (CFA). ***, p<0.001 (Mann-Whitney test). Results from mice injected with home-made and commercial MOG/CFA suspensions as well as results obtained from PLP/CFA-injected mice (and respective control-injected mice) are displayed \n",
      ": eIF4A regulates TORC1 via TSC2.Figure 5(a) Knockdown of eIF4A and TSC2 do not elevate TORC1 activity in an additive manner, indicating they act in the same, and not parallel pathways. Whereas knockdown of eIF4A increases TORC1 activity in control Kc167 cells (lanes 1-2 and 5-6), eIF4A knockdown cannot further increase TORC1 activity in TSC2 knockdown cells (lanes 3-4 and 7-8). Error bars: std dev. n=3 biological replicates. (b) Activation of TSC2 with the p90RSK inhibitor BI-D1870 rescues the elevated TORC1 activity caused by eIF4A knockdown in Kc167 cells, consistent with TSC2 acting downstream of eIF4A. BI-D1870 leads to inhibition of TORC1 (lanes 1-4) in a TSC2-dependent manner (see also Appendix Figure S5A). Representative of two biological replicates. (c) FLAG-eIF4A and TSC2-V5 co-immunoprecipitate in Kc167 cells in the absence of chemical cross-linker. Cells were transfected with FLAG-eIF4A and TSC2-V5 expression vectors and treated with media containing or lacking amino acids for indicated timepoints prior to lysis and anti-FLAG immunoprecipitation. The experiment was performed in absence of the chemical cross-linker DSP. Representative of 3 biological replicates. (d) Expression of TSC2-insensitive (S15H or Q63L), but not wild-type (WT) Rheb causes TORC1 activity to remain high upon amino acid removal. Kc167 cells were transfected to express either wild-type or mutant Rheb and then incubated with Schneider's medium either containing or lacking amino acids for 30 minutes. Elevated TORC1 activity can be observed either by looking at phosphorylation of endogenous S6K, or phosphorylation of an HA-tagged S6K that was co-transfected with the Rheb constructs to assay specifically the transfected cells. Representative of two biological replicates. \n",
      ": The eIF4F complex regulates TORC1.Figure 3(a) Knockdown of eIF4F components (eIF-4E and eIF-4G) partially phenocopy the eIF4A knockdown, leading to elevated TORC1 activity upon amino acid removal (30 min). Knockdown of all other tested translation initiation factors do not cause elevated TORC1 upon amino acid removal (main Figures 1c, 2a' and 2b). Error bars: std. dev. n=3 biological replicates. (b) Quantification of de novo protein synthesis rates by OPP incorporation reveals that eIF4E and eIF4G knockdowns deplete eIF4F function less efficiently than the eIF4A knockdown, explaining why the effects of eIF4E and eIF4G knockdowns on TORC1 activity (panel a) are a bit milder than the eIF4A knockdown. Kc167 cells treated with dsRNA for 4 days then incubated with 20μM Click-it OPP reagent 30min before fixation and fluorescent labeling. Quantification of OPP fluorescence per cell (nuclear count) for two independent experiments is displayed (3 independent images per condition), normalized to the no dsRNA condition. Scale bars: 25µm. \n",
      ": eIF4A is required for complete TORC1 inactivation upon amino acid removal.Figure 1(a-b) eIF4A knockdown in either S2 (a) or Kc167 (b) cells blunts the inactivation of TORC1 upon amino acid removal. Cells were treated with dsRNA targeting GFP as a negative control, or three independent, non-overlapping dsRNAs targeting eIF4A for 5 days and then incubated with complete medium or medium lacking only amino acids for 30 minutes. Representative of 3 biological replicates. (c) Only knockdown of eIF4A, but not of other genes involved in translation, causes elevated TORC1 upon amino acid withdrawal. S2 cells were treated with dsRNAs for 5 days and then incubated with medium containing or lacking amino acids for 30 minutes. After cell lysis, levels of S6K phosphorylation (T398) were measured by dot blot analysis and normalized to total S6K levels. Results are summarized here for all genes involved in translation, and provided in Dataset EV1. (d) Impaired inactivation of TORC1 in response to eIF4A knockdown is most apparent upon partial depletion of amino acids, caused by removal of amino acid subsets. After 5 days of knockdown, S2 cells were treated for 30 min with either complete Schneider's medium (+aa), Schneider's medium lacking all amino acids (-aa) or various subsets of amino acids, as indicated (where EAA \"essential amino acids\" = H,I,L,K,M,T,W,V). Error bars: std dev. n=3 biological replicates. (e) Time course of amino-acid removal reveals that eIF4A knockdown Kc167 cells maintain elevated S6K phosphorylation up to the maximum possible timepoint of 60 minutes when the cells start dying (see drop in S6K and tubulin levels). Representative of two biological replicates. (f) eIF4A mutant larvae have impaired TORC1 inactivation upon shifting to food lacking amino acids. Control (w1118) or eIF4A1006/1013 1st instar larvae were transferred from standard food to plates containing either standard fly food or PBS/1% agarose+2%sucrose for 1h prior to lysis and immunoblot analysis. Two biological replicates are shown. Representative of 3 biological replicates. \n",
      ": Impaired TORC1 inactivation upon amino acid removal is not a general consequence of impaired translation(a-a’) Although knockdown of eIF4A or eIF3-S2 equally blunt translation of EGFP from an inducible plasmid (a), only knockdown of eIF4A but not eIF3-S2 impairs TORC1 inactivation upon amino acid removal (a’).Figure 2(a-a') Although knockdown of eIF4A or eIF3-S2 equally blunt translation of EGFP from an inducible plasmid (a), only knockdown of eIF4A but not eIF3-S2 impairs TORC1 inactivation upon amino acid removal (a'). (a) S2 cells, treated with indicated dsRNAs for 5 days, on day 3 transfected with an inducible EGFP plasmid (pMT-EGFP) and on day 4 induced for 18h. Non-targeting LacZ dsRNA used as a negative control. (a') S2 cells treated with indicated dsRNAs for 5 days, then incubated with medium lacking amino acids for indicated time prior to lysis. Representative of two biological replicates. (b) Knockdown of eIF4A, but not other translation initiation factors, blunts TORC1 inactivation upon amino acid withdrawal. Kc167 cells treated with indicated dsRNAs for 4 days and then incubated with complete Schneider's medium or Schneider's medium lacking the indicated amino acids for 30 minutes prior to lysis. (c) Inhibition of translation with cycloheximide does not block inactivation of TORC1 upon amino acid withdrawal in Kc167 cells. Kc167 cells treated with either eIF4A dsRNA for 4 days, or cycloheximide (50 µg/mL, \"CHX\") for 5 min prior to, as well as during removal of amino acids (-LIVASTQP). CHX treated cells still inactivate TORC1 (compare lanes 8 to 7) whereas eIF4A knockdown cells do not (lanes 14 vs 13). Quantifications of two biological replicates are shown. CHX and eIF4A samples were normalized to their respective control conditions. (d) Knockdown of eIF4A in Kc167 cells does not prevent a drop in intracellular amino acids levels when amino acids are removed from the medium for 30 min. Quantification of total intracellular amino acids, analyzed in a blinded fashion. Levels of individual amino acids shown in Figure S2D. For CHX samples, cycloheximide (50µg/mL) was added 5 minutes prior to, and during treatment with medium containing or lacking amino acids. Statistical significance tested by ANOVA2 using Scheffé's multiple comparisons method (p<0.05). Error bars: std. dev. n=5. \n",
      ": NAT1 regulates TORC1 activity via eIF4A.Figure 6(a) Mass spectrometry analysis of proteins co-immunoprecipitating with eIF4A reveals NAT1 as one of the top interacting proteins. eIF4A immunoprecipitations were performed in triplicate, from cells treated with medium either containing (+aa) or lacking amino acids (-aa), and average values are shown. Peptide counts for each protein in each replicate were normalized to eIF4A peptide counts. Raw peptide counts shown in parentheses.(a') Schematic diagram of eIF4G and NAT1 primary protein structures. Binding sites for other initiation factors are shown. (b) Binding between eIF4A and NAT1 is regulated by amino acid availability in a TORC1-independent fashion. Co-immunoprecipitation of tagged eIF4A and NAT1 in control Kc167 cells, or cells treated with medium lacking amino acids or supplemented with 20 nM rapamycin for indicated times. Representative of 3 biological replicates. (c) Knockdown of NAT1 using 4 independent, non-overlapping dsRNAs leads to reduced TORC1 activity. Kc167 cells, treated with indicated dsRNAs for 4 days and then incubated with medium containing or lacking amino acids for 30 min. Error bars: std. dev. n=3 biological replicates. (d) eIF4A is epistatic to NAT1 for TORC1 regulation. In the absence of amino acids, eIF4A knockdown cells have elevated TORC1 activity, NAT1 knockdown cells have reduced TORC1 activity, and eIF4A NAT1 double-knockdown cells have elevated TORC1 activity compared to control cells. Cells were treated with indicated dsRNAs for 4 days and then treated with medium containing or lacking the indicated amino acids for 30 min. Error bars: std. dev. n=3 biological replicates. (e) TSC2 is epistatic to NAT1 for TORC1 regulation. As in (d), knockdown of TSC2 rescues the reduced TORC1 activity caused by NAT1 knockdown. (f) Schematic diagram showing the signaling relationships between NAT1, eIF4A and TSC2. When amino acids are present, NAT1 binds eIF4A and inhibits it. Upon amino acid removal, NAT1 releases eIF4A, which activates TSC2 to inhibit TORC1. \n",
      "Atg32, ScAtg32 and Atg36 use similar interaction mechanisms as Atg30.Figure 4(A) Mitophagy experiments of WT (PPY12), Δatg5, Δatg32 (ϕ) and Δatg32 complemented with WT Atg32 (Atg32), Atg32 with a deletion of the sequence between the AIM and phosphosite required for Atg11 binding (Atg32WQVLSSS), Atg32 AIM mutant (Atg32W121A V124A), mutants of the Thr upstream of the Atg32 AIM (Atg32T119A) and mutants altered in the Ser required for Atg11 binding (Atg32S159A). The cells were grown in YPL medium and shifted to SD‐N. Mitophagy was followed by the transport of Tom20-mCherry to the vacuole by fluorescence microscopy. Mitophagy was classified as −no mitophagy,+few cells show mitophagy, +++ most cells show mitophagy and ++++ almost all the cells show mitophagy (the intensity and numbers of cells containing Tom20-mCherry in the vacuole was considered). Scale bar, 5 μm. (B) P. pastoris Δatg32 cells (ΔPpatg32) expressing Tom20-GFP and expressing the indicated Atg32 mutants were cultured in YPL medium for 12, 18 and 36 h. Mitophagy was monitored by GFP appearance by immunoblotting with α‐GFP antibodies. (C) N‐terminal sequence of Atg36 manually aligned against the several sequence alignments of Atg30, Atg32 and ScAtg32. Identical residues are indicated with black boxes, and similar residues with grey boxes. (D) Two‐hybrid protein-protein interaction analysis of ScAtg36, ScAtg8 and ScAtg11. The receptors were mutated at the AIM (Atg36F33A L36A), at serine(s) upstream of the AIM (Atg36S31A) and at the Atg11‐binding site (Atg36S97A). (E) S. cerevisiae Δatg36 cells (ΔScatg36) expressing thiolase-GFP and expressing the indicated Atg36 mutants were cultured in oleate medium until mid‐log growth and then shifted to SD‐N. Pexophagy was monitored by GFP appearance by immunoblotting with α‐GFP antibodies. (F) Y3H analysis of ScAtg36 with ScAtg8 and ScAtg11. The Y3H technology is on the basis of the yeast two‐hybrid system but with the co‐expression of third protein as a competitor and indicated in the figure (NLS-ScAtg11 or NLS-ScAtg8). The positive control was the ScAtg36 mutant affected in Atg11 binding (ScAtg36S97A) or in Atg8 binding (ScAtg36S31A), which should be unaffected by the competition of NLS-ScAtg11 or NLS-ScAtg8, respectively. Appropriate auto‐activation and interaction controls were also included. AD, activation domain; AIM, Atg8‐family‐interacting motif; BD, binding domain; GFP, green fluorescent protein; WT, wild type; Y3H, yeast three‐hybrid. \n",
      ": TORC1 interacts physically with translation preinitiation complexes.Figure 4(a) Binding of eIF4G to the Rag GTPases detected by co-immunoprecipitation from Kc167 cells in absence or presence of the chemical cross-linker DSP. Representative of >3 biological replicates. (b) Binding between eIF4G and the Rag GTPase complex is mainly mediated by RagC, detected by expressing and immunoprecipitating either RagA or RagC alone. Representative of two biological replicates. (c) Interaction between eIF4A and Raptor detected via co-immunoprecipitation of epitope-tagged proteins. Proteins were cross-linked with DSP prior to lysis and immunoprecipitation. Representative of 3 biological replicates. (d-d'') eIF4A and Raptor form a complex, detected by proximity ligation assay (PLA). Specificity of the PLA signal was controlled by knocking down either eIF4A or Raptor in Kc167. (d) Representative images. Cells outlined in white. (d') Quantification of number of PLA spots per cell. (n>200. ***p<0.0001, ANOVA1, Dunnett's multiple comparisons test, was performed on log-transformed data, box = +/- 1 quartile, whiskers = max/min). (d'') Immunoblotting to detect efficiency of eIF4A and Raptor knockdowns. (e-f) The eIF4A-Raptor interaction does not take place on lysosomes (e) or the endoplasmic reticulum (ER) (f). Kc167 cells were either incubated with 100 µg/mL Dextran for 1 hour, washed and incubated for 14 hours in normal growth medium to label late endosomes and lysosomes (e), or transfected to express an ER-resident GFP (f). Amino acid removal was for 45 min. Pearson's correlation coefficients for eIF4A-Raptor PLA and Dextran in panel (e) are 0.09 and 0.08 for +aa and -LIVASTQP respectively (calculated for all cells in the acquired fields, which were chosen randomly, n=40). Scale bars: 5µm. \n",
      ".Figure 1(A) Still images of time-lapse imaging of H3.3-mCherry and H3.2-EGFP proteins in wild-type zygotes (n=17). (B) IF of DAXX and ATRX in zygotes at different pronuclear (PN) stages (PN0, n=8; PN2, n=6; PN5, n= 33). Data information: All panels: images of paternal and maternal pronuclei at given stage/condition are from same zygote. (C) Boxplot displaying signal intensity of DAXX at pat- and mat-PCH in mouse zygotes at PN5 stage, relative to mean signal at pat-PCH. In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 3 times. ***P<0.001, n= 28, paired t-test. (D) IF of DAXX and ATRX in PN5 zygotes upon siRNA-mediated knockdown of Daxx (n=14) or with control siRNAs (n=8). Data information: All panels: images of paternal and maternal pronuclei at given stage/condition are from same zygote. Yellow dashed circles represent the contours of the pronuclei. All scale bars, 10µm. (E) IF of DAXX and ATRX in control (n=15) and Atrxm-z+ (n=14) PN5 zygotes. Data information: All panels: images of paternal and maternal pronuclei at given stage/condition are from same zygote. Yellow dashed circles represent the contours of the pronuclei. All scale bars, 10µm. \n",
      ".Figure 2(A) IF of DAXX and EZH2 in paternal pronuclei of Ezh1m-z+ (n=5), Ezh2m-z+ (n=3) and Ezh1m-z+; Ezh2m-z+ (n=4) late zygotes. Yellow dashed circles represent the contours of the pronuclei. Data information: All scale bars, 10µm. (B) IF of DAXX and RNF2 in PN0 zygotes (n=10). Data information: All scale bars, 10µm. (C) IF of DAXX in wild type (n=30), Ring1m-z+ (n=12), Rnf2m-z+ (n=11) and Ring1m-z+; Rnf2m-z+ (n=12) paternal pronuclei of PN5 zygotes. Data information: All scale bars, 10µm. (D) Anti-Flag co-IP of DAXXMycHis, RNF2 and H3K27me3 with CBX23×Flag from extracts of HEK293 cells in presence or absence of benzonase, followed by immuno-blot analyses for indicated antigens. \n",
      ".Figure 7(A) RNA FISH of forward and reverse strand major satellite sequences in Daxxm+z+ and Daxxm-z+ PN5 zygotes. Scale bars, 10µm. (B) Box plots indicating the foci number (left) and total intensity (right) of forward (top) and reverse (bottom) transcription as measured by RNA FISH on 3D paternal pronuclei in Daxxm+z+ and Daxxm-z+ PN5 zygotes. Lower hinge, central line and upper hinge in boxplots represent 25th, 50th (median) and 75th percentiles respectively. Upper/lower whiskers extend to the largest/smallest values no further than 1.5 * IQR from the upper/lower hinge, where IQR is inter-quartile range or distance between 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 3 times. ***P<0.001; t-test. (C, D) MA-plots showing Fold Change (FC) (in log2) in gene expression between indicated samples as a function of averaged expression across all samples. Dotted lines indicated log2(FC) of 1. (E) Histogram showing classification of genes into 3 groups according to distribution of ChIP enrichment (log2) of H3K27me3 around TSS in MII oocytes (F) MA-plot showing FC (in log2) in gene expression between indicated samples as a function of averaged expression across all samples. Dotted lines indicated log2(FC) of 1. (G - I) Boxplots showing FC (log2) in gene expression between indicated samples for 3 groups of genes classified according to H3K27me3 enrichments (as shown in E). Lower hinge, central line and upper hinge represent 25th, 50th (median) and 75th percentiles respectively. Upper/lower whiskers extend to the largest/smallest values no further than 1.5 * IQR from the upper/lower hinge, where IQR is inter-quartile range or distance between 25th and 75th percentiles. Outliers are not displayed. The notches extend $1.5*IQR/\\sqrt{n}$ where n is a number of genes in each H3K27me3 class. Number of genes (n) in \"Low\", \"Medium\" and \"High\" H3K27me3 class used for boxplots are 9824, 2311, 2647 respectively. Expression changes were estimated using 2 or 3 biological replicates for each condition. Statistical significance was estimated using two-sided t-tests. (J) Boxplot showing FC (log2) in gene expression between Daxx deficient and control MII oocytes for five groups of genes classified according to FC (log2) in gene expression between Ring1 Rnf2 deficient and control GV oocytes. Boxplots were created and number of genes used for each boxplot is displayed below. Expression changes were estimated using 3 biological replicates for each condition. Statistical significance was estimated using two-sided t-tests. (K) Schematic diagram showing molecular pathways controlling repression and chromatin integrity at paternal and maternal major satellite sequences in mouse early pre-implantation embryos. \n",
      " - VpT6SS1 is functional in V.Figure 1A Schematic representation of the V. parahaemolyticus RIMD 2210633 T6SS1 gene cluster. Genes are represented by arrows indicating the direction of transcription. Locus numbers (vpxxxx) are denoted above. Notable T6SS components are denoted below. B Expression (cells) and secretion (media) of VgrG1 from V. parahaemolyticus RIMD 2210633 derivative POR1 (Vpara; T6SS1+) and its T6SS1- mutant (Vpara/∆hcp1), and from V. natriegens (Vnat) carrying the indicated plasmids. Samples were treated (+) or not (-) with 20 µM phenamil to activate surface sensing in media containing 3% NaCl at 30 °C for 5 h. Loading control (LC) is shown for total protein lysates. Arrows denote bands corresponding to VgrG1. Asterisks denote non-specific bands detected in Vnat samples. C Viability counts of V. natriegens prey before (0 h) and after (4 h) co-incubation with the indicated attackers, as described in B, on media containing 3% NaCl at 30 °C. Data are shown as the mean ± SD, n = 3 technical replicates. Statistical significance between samples at the 4 h timepoint by an unpaired, two-tailed Student's t-test is denoted above. A significant difference was considered as P < 0.05. DL, assay detection limit. D Illustration of interbacterial competition mediated by VpT6SS1 when introduced into V. natriegens (yellow) on a plasmid (pT6SS1), as shown in C. Prey cells are denoted in blue. \n",
      ".Figure 5(A) IF of DAXX and ATRX in PN5 stage pronuclei of wild-type (n=19), Hp1βm-z+ (n=10) and Suv39h2m-z+ (n=9) zygotes. Data information: Panels Paternal and maternal pronuclei of a given condition originate from same zygotes. All scale bars 10µm. (B) Boxplots displaying signal intensities of DAXX or ATRX at pat- and mat-PCH in wild-type (n=5) and Hp1βm-z+ (n=10) mouse zygotes at PN5. In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 2 times. *P<0.05, **P<0.01, t-test. (C) IF of DAXX in PN5 stage pronuclei of zygotes expressing CBX2-EGFP (n=9) or CBX2F12A-EGFP (n=17). mRNAs (100ng/µl) were injected into mouse PN2 stage zygotes. Yellow dashed circles represent the contours of the pronuclei.. Data information: Panels C: Paternal and maternal pronuclei of a given condition originate from same zygotes. All scale bars 10µm. \n",
      " - A wide-range functionality of the inducible VpT6SS1-based platform in V.Figure 3A, B Viability counts of V. natriegens prey before (0 h) and after (4 or 24 h) co-incubation with VnatReg attackers harboring the indicated plasmids, on media containing 3% NaCl and 0.1% arabinose (A), or on media containing 0.1% arabinose at 28 °C (B). Data information: Data are shown as the mean ± SD, n = 3 technical replicates. Statistical significance between samples at the 4 or 24 h timepoints by an unpaired, two-tailed Student's t-test is denoted above and is color coded to match the relevant samples. A significant difference was considered as P < 0.05. C-F Viability counts of the indicated prey bacteria before (0 h) and after (4 h) co-incubation with VnatReg attackers harboring the indicated plasmids, on media containing 3% NaCl and 0.1% arabinose at 28 °C. Data information: Data are shown as the mean ± SD, n = 3 technical replicates. Statistical significance between samples at the 4 or 24 h timepoints by an unpaired, two-tailed Student's t-test is denoted above and is color coded to match the relevant samples. A significant difference was considered as P < 0.05. \n",
      " - VP1407 can serve as an effective on/off switch for VpT6SS1 in V.Figure 2A Illustration of the VpT6SS1 (vp1386-vp1420) activation cascade. Genes are represented by arrows indicating the direction of transcription. Genes belonging to the same operon are denoted by the same color. Locus numbers (vpxxxx) are denoted above. Positive regulators are denoted by a red frame. Dashed arrows denote transcriptional activation and are color coded according to the relevant induced operon. Numbers in black circles next to the positive regulators TfoY, VP1407, and VP1391 delineate the progression of the cascade. B Viability counts of V. natriegens prey before (0 h) and after (4 h) co-incubation with wild type V. natriegens attackers harboring the indicated plasmids, on media containing 3% NaCl at 30 °C. Data information: data are shown as the mean ± SD, n = 3 technical replicates; statistical significance between samples at the 4 h timepoint by an unpaired, two-tailed Student's t-test is denoted above and is color coded to match the relevant samples. A significant difference was considered as P < 0.05. C Expression (cells) and secretion (media) of VgrG1 from V. natriegens containing an arabinose-inducible vp1407 in the chromosomal dns locus (VnatReg) and harboring an empty plasmid (pEmpty) or plasmids containing VpT6SS1 that lacks vp1407 (pT6SS1Ind) or both vp1407 and hcp1 (pT6SS1Ind/∆hcp1; T6SS-). Samples were grown in media containing 3% NaCl and either supplemented (+) or not (-) with 0.1% arabinose (Ara) at 30 °C. Loading control (LC) is shown for total protein lysates. D Illustration of the engineered VnatReg (yellow bacteria) containing a plasmid-borne, inducible VpT6SS1 (pT6SS1Ind). In the presence of arabinose (red circles), VP1407 is expressed from the chromosome and the plasmid-borne T6SS is induced, resulting in T6SS-mediated intoxication of adjacent prey bacteria (blue). E Viability counts of V. natriegens prey before (0 h) and after (4 h) co-incubation with VnatReg attackers carrying the indicated plasmids on solid media plates, as described in C. Data information: data are shown as the mean ± SD, n = 3 technical replicates; statistical significance between samples at the 4 h timepoint by an unpaired, two-tailed Student's t-test is denoted above and is color coded to match the relevant samples. A significant difference was considered as P < 0.05. \n",
      ".Figure 3(A) Cartoon displaying recruitment of DAXXMycHis by GAL4DBDCBX2 (CBX2.1 or CBX2.2) to a 5×UAS site integrated in the first intron of the NCOA5 gene in HEK2935xUAS cells. (B) Schematic diagram of CBX2.1 and CBX2.2 human isoform genes and proteins. * highlights the alternative chimeric exon 4. (C) Occupancy (% of IP-ed DNA / input DNA) of GAL4DBDCBX2, RNF2, DAXXMycHis, H3.3, and H3 proteins at 5×UAS site as measured by ChIP-qPCR of chromatin from HEK2935xUAS cells. Data are means ± SEM (n = 3). (D) Immunoblots for the indicated proteins in HEK2935xUAS cells (E) ChIP-seq analysis of HEK2935xUAS cells expressing full length GAL4DBDCBX2.1 and DaxxMycHis showing CBX2.1 dependent recruitment of DAXX to promoters of CCND2 and RUNX1 genes. (F) ChIP-seq analysis of GAL4DBDCBX2.1 occupancy in HEK2935xUAS cells expressing DAXXMycHis (X-axis) or GAL4DBDCBX2.1 and DAXXMycHis (Y-axis), showing (log2) enrichment for GAL4DBDCBX2.1 at many TSSs (± 1 kb) in cells expressing the CBX2.1 protein (Y-axis). (G) ChIP-seq analysis of GAL4DBDCBX2.1 versus DAXXMycHis occupancy in HEK2935xUAS cells expressing GAL4DBDCBX2.1 and DAXXMycHis, revealing very comparable enrichment levels (log2) of both proteins at TSSs. TSSs highlighted in red show enrichment with anti-Myc antibody independently of GAL4DBDCBX2.1 occupancy. (H) ChIP-seq analysis of DAXXMycHis occupancy in HEK2935xUAS cells expressing DAXXMycHis (X-axis) versus GAL4DBDCBX2.1 and DAXXMycHis (Y-axis) showing strong enrichment (log2) for DAXXMycHis at many TSSs of genes, but only when both proteins are expressed (Y-axis), indicating that CBX2.1 targets DAXX to chromatin. TSSs highlighted in red show enrichment with anti-Myc antibody independently of GAL4DBDCBX2.1 occupancy. \n",
      ".Figure 4(A) IF of CBX2 and SUMO2/3 in PN0 (n=10) and PN5 (n=12) zygotes. Data information: Scale bars, 10µm. (B) ChIP-qPCR analyses of HEK2935xUAS cells co-expressing GAL4DBDCBX2.1 either with DAXXMycHis or DAXXΔSIM1/2MycHis. Data are means ± SEM (n = 3). (C) IF of EGFP and DAXX in Daxxm-z+ PN5 zygotes micro-injected with Daxx-EGFP or DaxxΔSIM1/2-EGFP mRNA (100ng/µl). Data information: Scale bars, 10µm. (D) Left: Boxplot displaying EGFP signal intensities at PCH relative to euchromatin in Daxxm-z+ zygotes micro-injected with Daxx-EGFP or DaxxΔSIM1/2-EGFP mRNA. Right: Boxplot displaying signal intensity of EGFP at euchromatin in same samples. In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 3 times. ***P<0.001, N.S.: no statistical difference, P>0.05, t-test. (E) Sequence alignment of mouse and human CBX2 proteins with SUMO sites indicated. (F) Ni-pulldown of 6xHisSUMO1/2/3-conjugated proteins from HEK293 extracts expressing 6xHis-tagged SUMO1, SUMO2 or SUMO3 in combination with CBX2-EGFP. CBX2 and SUMO-conjugated CBX2 were detected with an anti-GFP antibody. (G) Ni-pulldown of 6xHis-SUMO2 conjugated proteins from HEK293 extracts transiently co-expressing 6xHis-SUMO2 with CBX2-EGFP or CBX24KR-EGFP. Detection of CBX2 (H) IF of DAXX and EGFP in wild-type PN5 zygotes, micro-injected with Cbx2 or Cbx24KR-EGFP mRNA (100ng/µl). Data information: Scale bars, 10µm. (I) Left: Boxplot displaying signal intensity of DAXX at PCH relative to mean intensity at pat-PCH in wild-type zygotes micro-injected with Cbx2 or Cbx24KR-EGFP mRNA. Right: Absolute intensities of EGFP in same samples. In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 2 times. ***P<0.001, t-test. \n",
      ".Figure 6(A) Developmental progression of Daxxm+z+, Daxxm-z+ and Daxxm-z- early embryos. (B) Examples of morula and blastocyst embryos with normal and large polyploid nuclei. Data information: All scale bars 10µm. (C) Percentages of embryos with normal or polyploid nuclei. (D) Still images of time-lapse imaging of the first cleavage division in wild-type and maternally deficient zygotes expressing H2B-mCherry. Time points represent minutes after prometaphase. Lagging chromosomes and micronuclei are indicated by yellow arrowheads. Data information: All scale bars 10µm. (E) Percentage of zygotes with lagging chromosomes during 1st cleavage division. Data represent mean ± s.e.m. in n≥3 replicate experiments. (F) IF of 5-methyl and 5-hydroxy-methyl cytosine staining of wild-type (n=42), Daxxm-z+ (n=14) and Ring1m-z+; Rnf2m-z+ (n=6) zygotes with lagging chromosomes. Yellow arrowheads indicate lagging chromosomes. Data information: All scale bars 10µm. (G) DNA-FISH for major (green) and minor (red) satellites on cleavage chromosomes of a Daxxm-z+ zygote with a chromosome broken within the major satellite region. Data information: All scale bars 10µm. (H) H3.3-EGFP live-imaging signal in pat- and mat-PCH of control Daxxm+z+ (n=18), Daxxm-z+ (n=8) and Ring1m-z+; Rnf2m-z+ (n=12) zygotes. Scale bar 10µm. (I) Left: IF of endogenous H3 and ATRX at pat-PCH in control and Daxxm-z+ PN5 zygotes. Right: Fluorescence intensity profiles of H3 and ATRX along white line in left panel at pat-PCH of zygotes. Scale bars 5 µm. Data information: All scale bars 10µm. (J) Boxplot displaying H3 signal intensity at pat-PCH of Daxxm+z+ and Daxxm-z+ PN5 zygotes. In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 2 times. ***P<0.001, t-test, n=12 for both groups. (K) IP of DAXX-EGFP and DAXXR244A-EGFP proteins with an anti-GFP antibody from nuclear extract of HEK293 cells. Immuno-blot detection was performed with antibodies recognizing GFP, histone variant H3.3 and CBX2. (L) Percentage of Daxxm-z+ zygotes with lagging chromosomes following injection of Tubulin-EGFP, Daxx-EGFP or DaxxR244A-EGFP mRNAs (20 or 100 ng/µl) at PN2/3 stage. Data represent mean ± s.e.m. (n ≥2 injection experiments). *** P<0.001 High Daxx-EGFP versus other 3 groups; Fisher's exact test (on pooled embryos). (M) Left: IF of EGFP and endogenous H3 at pat-PCH in Daxxm-z+ PN5 zygotes expressing DAXX-EGFP or DAXXR244A-EGFP or nothing exogenously. Right: Fluorescence intensity profiles of H3 and EGFP along white lines in left panels at pat-PCH of zygotes. Scale bars 5 µm. Data information: All scale bars 10µm. (N) Boxplot displaying 3D quantification of H3 signal intensity at pat-PCH in Daxxm-z+ zygotes expressing exogenous DAXX-EGFP (n=14) or DAXXR244A-EGFP (n=13). In boxplots, the enter lines show the medians; box limits indicate the 25th and 75th percentiles as determined by R software; whiskers extend 1.5 times the interquartile range from the 25th and 75th percentiles. Datapoints are indicated by circles. Experiments were replicated 2-3 times. ***P<0.001 Daxx-EGFP versus other 2 groups; t-test. \n",
      " - Manipulating the effector repertoire and target range of the VpT6SS1-based platform.Figure 4Viability counts of V. natriegens (cyan), V. vulnificus (purple), A. jandaei ∆tssB (brown), E. coli (green), and S. enterica (gray) prey before (0 h) and after (4 h) co-incubation with VnatReg attackers (yellow) harboring the indicated plasmids, on media containing 3% NaCl at 28 °C. The left panels show prey survival when competing alone (single prey) against the attacker at a 4:1 (attacker:prey) ratio; the right panels show the results of a competition experiment in which all prey were mixed together with the attacker at a 10:1:1:1:1:1 (attacker:prey) ratio. Data are shown as the mean ± SD, n = 3 technical replicates. Statistical significance between samples at the 4 h timepoint by an unpaired, two-tailed Student's t-test is denoted above and is color coded to match the relevant samples. A significant difference was considered as P < 0.05. \n",
      " - Effector and immunity combinations alter the VpT6SS1-based platform's toxicity range.Figure 5A-D (A, B, D) Viability counts of V. parahaemolyticus RIMD 2210633 ∆hcp1 (A), V. alginolyticus 12G01 ∆hcp1/∆hcp2 (B), and V. natriegens (D) prey bacteria before (0 h) and after (4 h) co-incubation with VnatReg attackers harboring the indicated plasmids, on media containing 3% NaCl and 0.1% arabinose at 28 °C. Data are shown as the mean ± SD, n = 3 technical replicates. Statistical significance between samples at the 4 h timepoint by an unpaired, two-tailed Student's t-test is denoted above. A significant difference was considered as P < 0.05. ( C) Illustration summarizing the competitions shown in (A, B). Effector and immunity modules expressed by VnatReg carrying pT6SSEffectorless (yellow) are colored to match the bacterium from which they were derived. \n",
      ": Structure and characterization of liposomal NPs.Figure 2(A) Schematic presentation of liposomal NP layers and composition. (B) Characterization of the diameter and zeta potential (Z-potential) with the addition of each layer. Graph below presenting NP size distribution. (C) Imaging of fluorescently-labeled NPs using confocal microscopy. The data shown are representative of three independent experiments. \n",
      ": Specific and efficient siRNA delivery to NK cells using NKp46 antibody-coated NPs.Figure 3(A) NK92-NKp46low, NK92-NKp46high, YTS KIR2DL1, K562 and 721.221 HLA-Cw4 cells were stained with anti-NKp46 monoclonal antibody followed by staining with Alexa568-Fluor goat anti-mouse IgG secondary antibody. Cells were then analyzed using flow cytometry. (B) NK92-NKp46low, NK92-NKp46+, YTS KIR2DL1, K562, and 721.221 HLA-Cw4 cells were incubated with 50 µg rhodamine-labeled NPs and analyzed using flow cytometry to confirm the internalization of the coated NPs in a NKp46-specific manner. (C-F) (C) NK92-NKp46low, (D) NK92-NKp46high, (E) YTS KIR2DL1 and (F) K562 cells were incubated with 50 µg rhodamine-labeled NPs and analyzed using confocal microscopy. Upper panel shows wide field images. Lower panel shows single-cell images. The data shown are representative of three independent experiments. (G) YTS KIR2DL1 were incubated with NKp46 antibody coated-NPs loaded with FITC labeled siRNA. After 24 hours, the cells were stained with anti-NKp46 monoclonal antibody followed by staining with Alexa568-Fluor goat anti-mouse IgG secondary antibody. Then, the cells were analyzed by flow cytometry to determine the incorporation of the rhodamine-labeled NPs loaded with FITC-siRNA and NKp46 antibody staining. Cells that were positive for both rhodamine-labeled NPs and NKp46 antibody staining, were analyzed for the presence of fluorescent FITC-siRNA. \n",
      ": NKp46 antibody-coated NPs do not induce spontaneous non-specific NK cell activation or cytokine secretion.Figure 5(A) YTS KIR2DL1 cells were incubated with SHP-1 and Cbls siRNA-loaded NPs (red curve) or empty NPs (purple curve). After 48 hours, cells were loaded with Fluo-3-AM and analyzed for intracellular calcium levels for 20 min at 37°C. YTS KIR2DL1 cells that were treated with PBS served as a negative control (green curve). NK cells that were mixed with 721.221 Cw7 target cells served as a positive control (blue curve). Calcium levels were analyzed by spectrofluorometry. (n=3) (B) Freshly isolated PBMCs were treated with SHP-1 and Cbls siRNA-loaded NPs, empty NPs, PBS as negative control or PHA as a positive control. Following incubation, supernatant from each sample was collected and the levels of human cytokines TNF-α, IL-6, IL-10 and IFN-γ were determined using ELISA. The data represent four independent experiments (n=4). P values were calculated by One way ANOVA following Tukey's post hoc analysis, and are indicated by asterisks. *P≤0.008. Data are shown as mean ± SEM. \n",
      ": Inhibition of Cbl-b, c-Cbl and SHP-1 enhance NK cell function.Figure 1(A) YTS KIR2DL1 cells were mock-transfected or transfected with N.S siRNA or Cbl-b siRNA, c-Cbl siRNA and SHP-1 siRNA, using Amaxa electroporation. After 48 hours, cells were lysed, and subjected to western blot with anti-Cbl-b, anti-c-Cbl or anti-SHP-1 antibodies. Analysis by ImageJ densitometry is summarized in the graph below. Data are means ± SEM of three independent experiments.(n=3) P values were calculated by One-sample t-test, and are indicated by asterisks *P < 0.05. (B) YTS KIR2DL1 cells were transfected with either N.S siRNA (blue curve) or with SHP-1 and Cbls siRNA (red curve). After 48 hours, cells were loaded with calcium-sensitive Fluo-3-AM and analyzed for basal intracellular calcium levels for 1 min. The NK cells were then mixed with 721.221 Cw4 target cells at 37°C and monitoring of calcium levels was continued. The data shown are representative of three independent experiments. (C) YTS KIR2DL1 cells were incubated with either 721.221 Cw4 or Cw7 target cells for 2 hours and analyzed by flow cytometry to determine the expression of CD107a. Expression of CD107a was measured by mean fluorescence Intensity (MFI). Relative expression of CD107a MFI was normalized to the mock-transfected sample following Cw4 incubation. Data are means ± SEM of five independent experiments.(n=5). P values were calculated by One-sample t-tests and One way ANOVA following Tukey's post hoc analysis are indicated by asterisks *P < 0.05, ****P<0.0001 . (D) Granzyme release. YTS KIR2DL1 cells were mock-transfected or transfected with N.S siRNA or SHP-1 and Cbls siRNA. After 48 hours, cells were incubated with either 721.221 Cw4 or Cw7 target cells, the supernatant was then collected and granzyme B levels were evaluated using ELISA sandwich assay. The levels of granzyme B were quantified using standard recombinant granzyme B concentrations Data are means ± SEM of eight independent experiments;(n=8); P values were calculated by One way ANOVA following Tukey's post hoc analysis, and are indicated by asterisks *P < 0.05.. (E) Killing assay. YTS KIR2DL1 cells were mock-transfected or transfected with N.S siRNA or SHP-1 and Cbls siRNA. After 48 hours, cells were incubated with [35S] Met-loaded 721.221 Cw4 or Cw7 target cells. After 5 hours of co-culture, the supernatant was collected, and the radioactive signal was measured. Data are means ± SEM of three independent experiments. P values were calculated by One-way ANOVA following Tukey's post hoc analysis and are indicated by asterisks *P < 0.05, **P < 0.01 . \n",
      ": NKp46 antibody-coated NPs loaded with SHP-1 and Cbls siRNA enhance NK cell function following inhibitory interaction.Figure 4(A) YTS KIR2DL1 and primary NK cells were incubated with NPs loaded with Cbl-b, c-Cbl and SHP-1 siRNA, or NPs loaded with 1500 pmol N.S siRNA. Following 48 hours of incubation, cells were lysed, and the membrane was blotted with anti-Cbl-b, anti-c-Cbl or anti-SHP-1 antibodies. GAPDH served as loading control. Densitometric analysis of the bands was performed using ImageJ and normalized to the GAPDH densitometry values. Analysis by ImageJ densitometry is summarized in the graph below. Data are means ± SEM of four independent experiments. (n=4). P values were calculated by One-sample t-tests and are indicated by asterisks. *P < 0.05, **P < 0.01. (B) YTS KIR2DL1 cells were incubated with SHP-1 and Cbls siRNAs-loaded NPs (red curve), with NS siRNA loaded NPs (green curve) or with empty NPs (blue curve). After 48 hours, cells were loaded with calcium-sensitive Fluo-3-AM and analyzed for basal intracellular calcium levels for 1 min. The NK cells were then mixed with 721.221 Cw4 target cells at 37°C, and monitoring of calcium levels was continued. The data shown are representative of three independent experiments. (C) YTS KIR2DL1 cells were pretreated either SHP-1 and Cbls siRNA or N.S siRNA loaded NPs. After 48 hours, cells were incubated with either 721.221 Cw4 or Cw7 target cells for 2 hours and analyzed by flow cytometry to determine the expression of CD107a. Expression of CD107a was quantitated by mean fluorescence intensity (MFI). Data are means ± SEM of nine independent experiments. .) (n=9). P values were calculated by One-sample t-tests and One way ANOVA following Tukey's post hoc analysis and are indicated by asterisks. **P < 0.01. (D) YTS KIR2DL1 cells were treated with SHP-1 and Cbls siRNAs-loaded NPs or N.S siRNA-loaded NPs. After 48 hours, cells were incubated with either 721.221 Cw4 or Cw7 target cells, the supernatant was then collected, and granzyme B levels were evaluated using ELISA sandwich assay. The levels of granzyme B were quantified based on a standard curve. Data are means ± SEM of five independent experiments. (n=5). P values were calculated by One way ANOVA following Tukey's post hoc analysis, and are indicated by asterisks. **P < 0.01. (E) YTS KIR2DL1 cells were pretreated with either SHP-1 and Cbls siRNA-loaded NPs, or N.S siRNA-loaded NPs. After 48 hours, cells were incubated with [35S] Met-loaded 721.221 Cw4 or Cw7 target cells. After co-culture for 5 hours, the supernatant was collected, and radioactive signal was measured. Data are means ± SEM of seven independent experiments. . (n=7). P values were calculated by One way ANOVA following Tukey's post hoc analysis and are indicated by asterisks **P < 0.01 ****P<0.0001. (F) pNK cells were pretreated with either SHP-1 and Cbls siRNA-loaded NPs or N.S siRNA-loaded NPs. After 48 hours, cells were incubated with either 721.221 Cw4 or Cw7 target cells for 2 hours and analyzed by flow cytometry to determine the expression of CD107a. Expression of CD107a was quantitated by mean fluorescence intensity (MFI). The data represent five independent experiments (n=5) Data are shown as mean ± SEM.) (n=5). P values were calculated by One-sample t-tests and One way ANOVA following Tukey's post hoc analysis, and are indicated by asterisks. *P≤0.05. \n",
      ".Figure 5(A) Experimental design of TCR sequencing. (B) Percent T cells of estimated cells in BrM and CLN of mice from control or WBRT group. (C) Maximal productive frequency of clones in BrM and CLN samples. (D) Productive clonality in BrM and CLN samples. (E) Clonal space homeostasis of BrM samples indicating the relative abundance of clones with a specific frequency. (F) Lorenz curves generated with the reads of all clones in a sample to visualize deviation from perfect equality. (G) Lorenz curves were used to determine the Gini index of each sample, which is a measure of inequality reflecting clonal expansion. (H) Percent of estimated T cells correlated to BrM volume. (r=-0.7319, p=0.0252) (I) Gini indices of BrM samples correlated to BrM volume (r=-0.9453, p=0.0001). \n",
      ".Figure 1(A) Stacked column depicts proportions of cell types in the TME of 99LN-BrM analyzed by flow cytometry (n=4). (B) Stacked column depicts the relative amount of myeloid and lymphoid cells in 99LN-BrM analyzed by flow cytometry (n=5). (C) Stacked column depicts the relative amount of myeloid and lymphoid subpopulations in 99LN-BrM based on three flow cytometry panels: myeloid cells (n=5), dendritic cells (n=7), lymphoid cells (n=6). (D) Principal component analysis of tumor-associated myeloid cells vs blood monocytes and microglia from tumor-free mice. (n=3 per condition) (E) Principal component analysis of tumor-infiltrating lymphocytes (TIL) vs blood lymphocytes from tumor-free mice. (n=3 per condition) (F-I) Functional gene annotation of altered cellular pathways in (F) TAM-MG, (G) TAM-MDM, (H) TIL-CD4 and (I) TIL-CD8 compared to control cell types from tumor-free animals. Cutoffs: basemean >20 and adjusted p-value (padj) <0.05. Adjusted p-values were obtained by Wald test and corrected for multiple testing using the Benjamini and Hochberg method. All DEGs based on padj were subjected to analysis. \n",
      ": NK cells gene silenced for SHP-1 and Cbls siRNA via NKp46 antibody-coated NPs exhibit therapeutic efficacy in human B-cell lymphoma engrafted mice.Figure 6(A) Schematic representation of the experimental timeline. (B) Tumor volumes measured daily in SCID/NOD mice treated with 3x106 irradiated YTS KIR2DL1 and NPs containing Cbl-b, c-Cbl and SHP-1 siRNA (n=13), or mice treated with 3x106 irradiated YTS KIR2DL1, and NPs containing N.S siRNA that served as a negative control (n=12). P values were calculated using two-tailed Student's t-test and are indicated by asterisks. * P < 0.05. Data are shown as mean ± SEM. (C) Average growth rates of tumors from the first treatment to the end point from same experimental groups of mice as indicated in B. P value was calculated by two-tailed Student's t-test and is indicated within the graph P =0.001. Data are shown as mean ± SEM. (D) Images of the tumor bearing mice at the end point of the experiment (upper panel) and excised tumors (lower panels). (E) Following 24 hours after the last treatment, extracted cells from tumors of either SHP-1 and Cbls or N.S siRNA loaded NKp46-NPs treated mice were analyzed by flow cytometry. Expression of CD107a was measured by mean fluorescence Intensity (MFI). Relative expression of CD107a MFI was normalized to the sample treated with N.S siRNA loaded NPs. The data represent three independent experiments. Data are shown as mean ± SEMn=3; P value was calculated by one sample t-test and is indicated within the graph P =0.009. (F) Survival analysis in mice treated with NK cells' gene silenced for SHP-1 and Cbls using siRNA-loaded NPs (n=18) or N.S siRNA-loaded NPs (n=18) as demonstrated by Kaplan-Meier survival curve. P values were calculated using the log-rank test and is indicated within the graph P =0.00009. \n",
      ".Figure 6(A) Experimental design of the combination trial. (B) Representative MRI images of 99LN-BrM from mice at different time points after treatment start. (C) Tumor growth curves of individual mice grouped by treatment. Maximal values of 8000% increase are shown from treatment start to d63. (D) Quantification of relative tumor growth of the individual mice as area under the curve normalized to the survival time in weeks. (E) Stacked columns depict the percentage of animals with progressive disease (PD), stable disease (SD), partial response (PR) and complete response (CR) at the indicated time points. (F) Percentage of mice that were categorized as CD3 T cell high or low stratified into responders and non-responders in the WBRT and WBRT+αPD-1 treatment group. (G) Kaplan-Meier curves show the percentage of symptom-free mice. \n",
      ".Figure 3(A) Surface expression of PD-1 in vivo in 99LN-BrM quantified by flow cytometry (n=3). (B) PD-L1 in 99LN BrM quantified by flow cytometry (n=3). (C) Flow cytometric analysis of PD-L1 in tumor-free brain and 99LN-BrM (n=3). (D) Flow cytometric analysis of PD-L1 expression on 99LN-BrM associated my myeloid cell types. (n=6). (E) Expression of genes associated with antigen presentation and co-regulatory factors in TAM-MG and TAM-MDM. n=3 per condition. Values are depicted as normalized counts. Genes labeled with # can have stimulatory effects depending on the co-regulatory receptor on T cells. (F) Gene expression of co-regulatory receptors in TIL-CD4 and TIL-CD8. n=3 per condition. Values are depicted as normalized counts. (G) Functional gene annotation of altered cellular pathways in TAM-MG vs. TAM-MDM. Left panel depicts pathways enriched in TAM-MG, right panel depicts pathways enriched in TAM-MDM. Adjusted p value (padj) were obtained by Wald test and corrected for multiple testing using the Benjamini and Hochberg method. (H) Representative HALO spatial plot showing the spatial distribution of cell types with indicated marker combination in 99LN-BrM and representative IF images of multiplexed histology. Scale bar; 100 μm. (I) HALO proximity histogram depicts the distance (μm) between the indicated cell types for individual mice (n=3). \n",
      ".Figure 2(A) Experimental design of T cell depletion by αCD4 and αCD8a antibody treatment. (B) Representative flow cytometry blots of blood samples from mice of both groups 3 weeks after treatment start showing successful depletion of CD45+CD3+ T cells in the αCD4+αCD8 treatment group. (C) Representative IHC images of CD3+ T cells and Iba1+ macrophages/MG in the isotype and αCD4+αCD8 group. Scale bar; 100 μm. (D) Quantification of Iba1+ and CD3+ cells in IHC sections of BrM from the isotype (n=9) and αCD4+αCD8 group (n=7). (E) Representative MRI pictures of BrM in early, medium and late stage of both groups. (F) Kaplan-Meier curves depict BrM free survival of mice in the isotype and αCD4+αCD8 group (Isotype n=19, αCD4+αCD8 n=19). (G) Tumor growth curves depict the increase of absolute BrM volume for each mouse in both groups over time (Isotype n=19, αCD4+αCD8 n=19). (H) Heatmap displays the relative expression level of T cell effector and exhaustion marker in TIL-CD4 and TIL-CD8 compared to blood lymphocytes. Values display the average per group based on vst values (BL-CD4 or BL-CD8 with n=3 for each group) vs. BrM-associated TIL-CD4 or TIL-CD8 (n=3 for each group). \n",
      ".Figure 4(A) Experimental design of tumor initiation and application of WBRT. (B) Representative MRI pictures of untreated and irradiated mice at d0 and d14 after treatment start. (C) Quantification of Iba1+ macrophages in IHC sections of 99LN-BrM at trial endpoint (Ctrl n=27, WBRT n=18). (D) Representative IF images of 99LN-BrM stained for the macrophage marker Iba1 (red) and the MG marker Tmem119 (white). Higher magnification images present areas in the peri-tumor region and in the tumor core. DAPI was used as nuclear counterstain. Scale bar; 100 µm. (E) Quantification of different myeloid populations in 99LN-BrM tumors with and without irradiation by flow cytometry at d14 (Ctrl n=5, WBRT n=4). (F) Representative IF images of 99LN-BrM with and without WBRT stained for DCIR2 (red) and EpCAM (green). DAPI (blue) was used as nuclear counterstain. Scale bar; 100 µm. (G) Quantification of CD11c+MHCII+ immune cells by flow cytometry in 99LN-BrM with and without WBRT. (H, I) Relative proportion of cDC1 and cDC2 in 99LN-BrM with and without WBRT. (J) cDC1:cDC2 ratio in 99LN-BrM based on flow cytometry data (H+I). (K) Representative IHC images of CD3+, FoxP3+ and CD8+ T cells in 99LN-BrM sections with and without WBRT. Scale bar; 100 µm. (L) Quantification of CD3+ cells in IHC sections of 99LN-BrM. (Ctrl n=26, WBRT n=18). (M) FoxP3:CD3 ratio in IHC sections of 99LN-BrM. (Ctrl n=25, WBRT n=18). (N) CD8:CD3 ratio in IHC sections of 99LN-BrM. (Ctrl n=25, WBRT n=18). (O-Q) Quantification of CD3+, CD4+ and CD8+ T cells in 99LN-BrM tumors with and without irradiation by flow cytometry. (R) CD4:CD8 ratio in 99LN-BrM based on data (P+Q). \n",
      ": Model of Elongator subunit organization and Elp456 association.Figure 5A. CXMS and EM data, combined with the X-ray structures of Elp1 C-terminus (5CQS), Elp2 (4XFV), Elp456 (4A8J), and homology models of the Elp1 tandem WD-40 domains and Elp3, were used to estimate the relative positions of each subunit to generate a multi-scale model of Elongator subunit organization. B. Side view of the Elongator architectural model highlighting the cavity (red outline) spanning through the centre of the Elp456 hexamer. C. Comparison of the Elp123 conformations of the intact Elongator complex (gold) and the Elp123 subcomplex (cyan). The Elp456-unloaded lobe of Elongator and one lobe of Elp123 were aligned. \n",
      ".Figure 4A. Fitting of Elp123 structures into the 3D reconstruction of Elongator. The homology structure of the Elp1 N-terminal double WD40-domains, the homology structure of Elp3 based on the X-ray structure of D. mccartyi, the X-ray structure of Elp1 C-terminal dimer (5CQS), and the X-ray structure of Elp2 (4XFV). Scale bar corresponds to 50 Å, and applies to panels A-C. B. Fitting of the Elp456 heterohexameric ATPase ring (4A8J) into the six-membered ring from the larger Elongator lobe. C. Size comparison of a representative tRNA molecule (1VTQ) and the cavity observed through the middle of the Elp456 hexamer. D. Alternate views of the Elongator model showing the subunit crosslinking patterns. \n",
      ": Mass spectrometry analysis of Elongator.Figure 2A. Number of peptides detected of each Elongator subunit purified through Elp4-FLAG pulldown from yeast lacking ELP1, ELP2, or ELP3 as a fraction of the number of Elp4 peptides. Individual data points are represented as circles, while the averages are represented as bars. B. Purified Elongator was incubated with the primary amine crosslinker disuccinimidyl suberate (DSS) and analyzed by mass spectrometry. Schematic representation of the major crosslinking connectivity of Elp123 is shown. C. Major crosslinking connectivity of the Elp456 complex. D. Detailed locations of crosslinked residues plotted onto a domain representation of Elongator. Lysines from two residues found crosslinked together are denoted by black lines. \n",
      ".Figure 7(A) CD45+ immune cells infiltrating 99LN-BrM (n=5 for isotype, n=4 for the other groups). (B) Flow cytometric analysis of dendritic cell (CD45+CD11c+CD83+) infiltration (n=4 for isotype, n=3 for all other groups). (C) Quantitative analysis of IHC CD3+ T cells at trial end point. (n=10 for isotype, n=7 for WBRT, n=10 for PD-1, and n=8 for WBRT+αPD-1 group). (D) Flow cytometric analysis of T cell subpopulations in 99LN-BrM (n=4 for isotype, n=3 for all other groups). (E) Quantitative analysis of IHC FoxP3+ T cells at trial end point. (n=10 for isotype, n=7 for WBRT, n=10 for PD-1, and n=8 for WBRT+αPD-1 group). (F) Flow cytometric analysis of PD-1 expression on T cells in response to different treatments (control n=4, all other groups n=3). (G) Composition of the myeloid compartment in 99LN-BrM in the different treatment groups (n=5 for isotype, all other groups n=4). (H) Relative abundance (%) of PD-L1+ myeloid cell types in BrM samples of the four treatment groups (n=5 for isotype, all other groups n=4). (I) Experimental design of the in vitro T cell activation assay. (J) Relative CD69 protein level on T cells cultivated with different cell types, unstimulated or stimulated with tumor conditioned media (Cond.) (n=3). (K) Relative Gzmb and IFNγ protein level on CD4+ and CD8+ T cells cultivated with different cell types, with or without aPD-1 and unstimulated or stimulated with tumor conditioned media (Cond.) (n=4 for conditions including BMDM, n=5 for all other conditions). \n",
      ": Conserved Elp2 loops are important for Elongator function.Figure 3A. Schematic of the conserved residues within four Elp2 loops that were mutated to alanine within an ELP2 plasmid. B. Plasmids containing ELP2 and the four loop mutants were transformed into elp2Δ yeast. The strains were then plated onto -Ura media containing caffeine. The pRS416 vector is used for these experiments. C. Growth curve analysis of loop mutants in -Ura media with and without K. lactis toxin zymocin. Shaded regions correspond to the SEM at each time point. N=3. D. Lysates of Elp4-FLAG-containing yeast expressing the Elp2 loop mutants were incubated with α-FLAG resin and eluted with 3xFLAG peptides then analyzed by Western Blot using α-HA and α-FLAG antibody \n",
      ": Elongator is a two-lobed asymmetric complex.Figure 1A. Coomassie Blue stained SDS-PAGE gel of yeast Elongator isolated by FLAG affinity purification and glycerol gradient ultracentrifugation from a strain encoding C-terminally FLAG-tagged Elp1. B. Representative segment of an electron micrograph of negatively stained Elongator after stabilization by gradient fixation (Lane 6 of Fig 1A). Circled particles correspond to intact Elongator. White scale bar corresponds to 50 nm. C. Representative 2D class averages of the full Elongator complex from a total of 17,074 particles. All class averages shown in this figure have side width of 500 Å. D. Representative 2D class averages of the Elp123 subcomplex prepared through a high salt dissociation procedure. E. Six evenly-sized densities are arranged in one Elongator lobe. A projection of the Elp456 hexamer X-ray structure (PDB code: 4A8J) is shown for comparison. F. Analysis of Elp2 localization using a C-terminal MBP tag. Class averages of untagged Elongator is shown for comparison. White arrowheads point towards additional densities in the tagged complex. G. Elp3 localization analysis using a C-terminal MBP tag, denoted by the white arrowhead. H. Mapping of Elp1 by incubating purified Elongator containing an Elp1 C-terminal FLAG tag with α-FLAG antibody. The additional antibody density is denoted by the white arrowhead. I. Three dimensional reconstruction of the intact Elongator complex at 25Å. J. Schematic of the approximate subunit locations based on 2D EM analysis. K. Inset showing the cavity formed through the centre of the putative Elp456 density. Black scale bars in this figure correspond to 50 Å. \n",
      ".Figure 6A. Silver-stained SDS-PAGE of Elongator glycerol gradient fractions purified using low salt (75 mM NaCl) buffer. The fraction containing the most Elp4, Elp5, and Elp6 is designated by an asterisk (*) and the corresponding fraction from a glutaraldehyde-containing glycerol gradient is used for EM analysis. B. Comparison of class averages of Elongator purified in buffer containing regular or low salt concentrations (150 mM and 75 mM NaCl, respectively). Densities resembling a second copy of Elp456 is indicated by white triangles. C. Scheme of the Elongator and rElp456 mixing experiment. Recombinant Elp456 subcomplex (rElp456) was purified from E. coli and mixed with native yeast Elongator in regular salt conditions. The mixture, as well as rElp456 alone, were then subjected to glycerol gradient ultracentrifugation. D. Coomassie-stained SDS-PAGE of rElp456 glycerol gradient fractions. E. Coomassie-stained SDS-PAGE of the Elongator mixed with rElp456 glycerol gradient fractions. The fraction containing the most intact complexes is indicated with an asterisk (*), and the corresponding fraction from a glutaraldehyde-containing glycerol is used for EM analysis. F. Comparison of class averages of natively purified Elongator with and without rElp456 incubation. Densities resembling a second copy of Elp456 are indicated by white triangles. \n",
      ".Figure 5A. Representative spinning disk confocal live-cell time series of MT-flux in U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red) treated with indicated conditions. White arrowheads follow the poleward motion of the photoactivated regions due to MT-flux. Scale bars, 10 µm. Time, min:sec. B. Quantification of the impact of stable end-on KT-MT attachments on MT-flux in NDC80-depleted cells under indicated conditions and in early prometaphase cells with and without the CENP-E inhibitor GSK923295. MT-flux values are plotted with mean ± SD. N (number of cells, number of independent experiments): siControl (49, 5), siNDC80 (59, 6), siNDC80 + siKIF4A (41, 3), siNDC80 + STLC (bipol.) (28, 3), siNDC80 + siKIF15 (36, 4), siNDC80 + siCLASPs (33, 3), siNDC80 + siKIF2A (38, 4), siNDC80 + GSK923295 (38, 3), early prometaphase (23, 3), early prometaphase + GSK923295 (28, 3). C. Kymograph profiles (left) highlighting that the photoactivated regions in prometaphase (upper panel) split and move towards both poles, whereas metaphase spindles (lower panel) flux uniformly towards individual poles (indicated by red dotted lines). Illustration of the photoactivated regions (right) in prometaphase and metaphase spindles, highlighting that interpolar MTs overlap over a broader region in early mitosis, becoming focused at the spindle equator in metaphase. Scale bar, 30 sec. D. Percentage of cells with parallel and antiparallel flux movements during prometaphase and metaphase. N (number of cells, number of independent experiments): prometaphase (23, 3), metaphase (22, 3). \n",
      ".Figure 3A. Representative spinning disk confocal time series of MT-flux in bipolar spindles in control cells and cells undergoing mitosis with unreplicated genomes (MUGs). U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red), labeled for chromosomes with SiR-DNA (grey) are shown, with 5 mM caffeine used as a control. Scale bar, 10 µm. Time, min:sec. B. Quantification of the MT-flux rates from indicated conditions. MT-flux values with mean ± SD are plotted. N (number of cells, number of independent experiments): Caffeine-only control (30, 3) and MUGs (34, 3). C. Representative point-scanning confocal maximum-intensity projected images of mitotic spindles in U2OS cells subjected to indicated conditions, immunostained with antibodies against KIF4A and α-tubulin. DNA was counterstained with DAPI. KIF4A in cyan and α-tubulin in red in merged image. Scale bar, 10 µm. \n",
      ".Figure 2A. Representative spinning disk confocal live-cell imaging time series images of U2OS PA-GFP-α-tubulin cells conditionally co-expressing KIF4A shRNA and RNAi-resistant mCherry-Kif4A variants induced using doxycycline. Chromosomes were stained using SiR-DNA. White arrowheads highlight poleward flux of the photoactivated regions. Scale bar, 10 µm. Time, min:sec. B. Representative immunoblot of cell lysates obtained before and after doxycycline induction to validate the efficiency of KIF4A shRNA construct and expression of the RNAi-resistant mCherry-KIF4A variants. The anti-KIF4A antibody was used, together with anti-vinculin antibody as a loading control. C. Quantification of MT-flux upon shRNA-mediated depletion of KIF4A alone or in combination with conditional expression of the RNAi-resistant mCherry-KIF4A variants. The bars in graph represent mean ± SD. N (number of cells, number of independent experiments): uninduced shKIF4A (48, 4), shKIF4A (60, 4), uninduced KIF4A WT (36, 5), shKIF4A + KIF4A WT (43, 5), uninduced KIF4A K94A (42, 7), shKIF4A + KIF4A K94A (38, 6), uninduced KIF4A ΔZip1 (32, 4), shKIF4A + ΔZip1 (30, 4). D. Model illustrating chromosome arms-localized KIF4A driving MT-flux. \n",
      ".Figure 4A. Representative spinning disk confocal live-cell image series of MT-flux in U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red) treated with indicated siRNAs. White arrowheads highlight poleward motion of the photoactivated regions due to MT-flux. Scale bars, 10 µm. Time, min:sec. B. Quantification of the impact of the MT-crosslinking proteins on MT-flux in U2OS PA-GFP/mCherry-α-tubulin cells transfected with respective siRNAs. Graph represent MT-flux values with mean ± SD. N (number of cells, number of independent experiments): siControl (49, 5), siPRC1 (45, 3), siNuMA (40, 3), siHSET (37, 3), siHSET + siNuMA (39, 3). C. Quantification of the impact of the motor activity of HSET on MT-flux in U2OS cells stably expressing mEOS-α-tubulin treated with control or HSET 3′UTR siRNAs in presence or absence of the respective RNAi-resistant GFP-HSET constructs. Graph represent MT-flux values with mean ± SD. N (number of cells, number of independent experiments): siControl (23, 3), siHSET 3′UTR (33, 3), siHSET 3′UTR + HSET WT (29, 5), siHSET 3′UTR + HSET N593K (30, 4). D. Representative kymographs of the photoactivated spindles displaying asynchronous flux motion (split of the two red dashed lines towards individual poles) upon RNAi-mediated depletion of MT-crosslinkers. E. Percentage of cells with asynchronous flux movements F. Model illustrating the role of MT-crosslinking activities of NuMA and HSET in uniform distribution of poleward forces across the mitotic spindle. \n",
      ".Figure 1A. Representative spinning disk confocal live-cell imaging time series of U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red), treated with control and KIF4A siRNAs. White arrowheads highlight poleward motion of the photoactivated regions due to MT-flux. Scale bars, 10 µm. Time, min:sec. B. Representative spinning disk confocal live-cell imaging time series images of S-trityl-L-cysteine (STLC)-treated U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red). Note that MT-flux is abrogated in presence of taxol (lower panel). Scale bars, 10 µm. Time, min:sec. C. Illustration of 405nm laser-photoactivated regions in monopolar spindles (blue circles, left). The effect of photoactivation and region selected for kymograph generation (dashed white rectangle, right). Scale bar, 10 µm. D. Corresponding kymograph profiles of the photoactivated regions in bipolar and monopolar spindles used for quantification of the flux rates (red dotted lines highlight MT-flux slopes). Scale bars, 30 sec. Quantification of MT-flux in bipolar spindles subjected to indicated treatments. Graphs represent MT flux of individual cells with mean ± SD. N (number of cells, number of independent experiments): - bipolar spindles: siControl (49, 5), STLC (31, 3), siKIF15 (39, 3), siKID (32, 3), siKIF4A (28, 3), siKIF2A (36, 3), siCLASPs (21, 2), GSK923295 (44, 3) Quantification of MT-flux in monopolar spindles subjected to indicated treatments. Graphs represent MT flux of individual cells with mean ± SD. N (number of cells, number of independent experiments) monopolar spindles: siControl (35, 3), siKIF15 (44, 3), siKID (38, 3), siKIF4A (44, 3), siKIF2A (38, 3), siCLASPs (12, 2), GSK923295 (33, 3). \n",
      ".Figure 6A. Representative maximum-intensity projected coherent-hybrid STED (CH-STED) images of mitotic spindles in U2OS cells treated with control and NDC80 siRNAs and stained for DNA (DAPI), KTs (CENP-C, magenta), α-tubulin (red) and CENP-E (green). Scale bar, 10 µm. B. Model illustrating potential mechanisms for CENP-E mediated MT-flux forces either through its KT- or antiparallel MT- based localization and sliding. CENP-E localized at KTs laterally interacting with MTs (white arrowheads in 1, 2), and CENP-E localized at interpolar MTs (white arrows in 3, 4), in control and NDC80-depleted mitotic cells. Scale bars, 2 µm. C. CH-STED analyses of HeLa cells stably expressing CENP-E-GFP illustrating bipolar spindles in prometaphase, metaphase and just after sister-chromatid separation at anaphase onset. Approximate x-y resolution is 70 nm. The expected/described localization of CENP-E-GFP (green) at KTs is shown. In addition, CENP-E-GFP was found associated with interpolar MTs (ipMTs), including regions of overlapping anti-parallel MTs, and k-fibers. Chromosomes (confocal mode only) were revealed in the larger panels with DAPI (cyan) and MTs (magenta) were detected with an anti-α-tubulin antibody. Arrows indicate examples of clear MT bundles, including regions of overlapping anti-parallel MTs. Scale bar in all panels is 5 μm. D. Representative spinning disk confocal live-cell time series of HeLa cells stably expressing CENP-E-GFP. White arrowheads highlight CENP-E-GFP at KTs, white arrows highlight CENP-E-GFP on interpolar MTs. Scale bar, 10 µm. Time, hour:min. \n",
      ".Figure 3Superposition of IAPV-IRES models corresponding to IAPV-IRES/40S class 1 and 2 after alignments excluding the IRES and the 40S head. A similar conformation can be observed with distinctive relative orientation with respects to the 40S body. The movements characteristic of the 40S head are indicated by arrows in orthogonal views. Ribbon diagram of the IAPV- IRES colored by pairwise root-mean-square deviation displacements observed between the two IAPV-IRES/40S classes. The ASL/mRNA like regions of the PKI and well as the SL-IV shows the lowest degree of displacement (blue) whereas the apical part of SL-III and the L1.1 the highest (red). Superposition of the IAPV-IRES with tRNAs in different configurations indicated at the bottom. IAPV-IRES is depicted as ribbons colored according to the secondary structure elements and tRNAs are represented as grey ribbons. Alignments of the models were computed with the 40S body, excluding from the computation the ligands (IRES/tRNAs) and the 40S head. Detailed view of the refined model for IAPV-IRES/40S class 1 inserted in the post-processed cryo-EM density focused on the decoding center of the 40S. PKI of IAPV-IRES is depicted green and 18S rRNA yellow. Close up view of the refined model for IAPV-IRES/40S class 1 inserted in the post-processed cryo-EM density focused on the SL-IV of the IAPV-IRES (depicted blue). \n",
      ".Figure 9In our MT-flux model, interpolar MTs are slid apart by the collaborative activity of the MT-crosslinking motors EG5 and KIF15, supported by sequential contribution of CENP-E at KTs in prometaphase and KIF4A on chromosome arms in metaphase. CENP-E at laterally attached KTs during early mitosis interacts with antiparallel MTs and slides them apart. During chromosome alignment to the metaphase plate, when CENP-E is partially stripped away from the KTs, KIF4A gradually takes over, promoting MTs to flux towards the poles. \n",
      ".Figure 2Overview of the mammalian 40S in complex with IAPV-IRES. Left, cryo-EM final post-processed map of class 1 with 40S colored yellow and IAPV- IRES maroon. Right, corresponding final refined model with IAPV-IRES colored according to figure 1A. Ribbon diagram of the 40S colored by pairwise root-mean-square deviation displacements observed between the two IAPV-IRES/40S classes. The different position of the 40S head between both classes is a composition of swiveling and tilt movements (indicated by arrows in orthogonal views). Close up view of the ribosomal sites of the 40S for IAPV-IRES/40S class 1 showing cryo-EM unsharpened cryo-EM density. Sequence of the PKI three-way helical junction. Unsharpened cryo- EM density for the PKI region of the IAPV-IRES in class 1 with the SL-III and the tRNA/mRNA mimicking domain indicated. \n",
      ".Figure 1IAPV-IRES diagram colored according to secondary structure motifs. Bottom, a closer view of the IAPV-IRES PKI highlighting its sequence, with base pairs indicated as well as the Variable Loop Region (VLR) and stem loop III (SL-III). Sucrose gradient UV profile of 60S/40S/IAPV-IRES reaction mixture after an overnight run. The peaks corresponding to 80S and 40S were used for RNA extraction and UREA-PAGE shown in the inset. Representative cryo-EM image where roughly half of the particles correspond to 40S (blue) and the other half to 80S (orange). Two classes with robust density for the IAPV-IRES where found in the 40S group. After classification, three classes with clear IAPV-IRES density and small differences in the conformation of the 40S where found in the 80S group. \n",
      ".Figure 7A. Representative spinning disk confocal time series of U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red), with indicated treatments (left). White arrowheads highlight poleward flux of the photoactivated regions. Scale bar, 10 µm. Time, min:sec. Right panels display corresponding kymographs of the photoactivated spindles used for quantification of the flux rates (red dotted lines highlight MT-flux slopes). Scale bars, 2 µm (horizontal) and 30 sec (vertical). B, C. Quantification of MT-flux in prometaphase (B) and metaphase (C) bipolar spindles subjected to indicated treatments. Graphs represent MT flux of individual cells with mean ± SD. siNDC80, siNDC80 + GSK923295, early prometaphase, early prometaphase + GSK923295, siControl, and siKIF4A N (number of cells, number of independent experiments): siNDC80 + KIF15 + STLC (33, 5), siNDC80 + KIF15 + STLC + GSK923295 (25, 3), early prometaphase + KIF15 + STLC + GSK923295 (11, 2), siKIF15 + STLC (29, 3), siKIF4A + siKIF15 + STLC (10, 3). D. Representative spinning disk confocal time series of U2OS cells stably co-expressing PA-GFP-α-tubulin (cyan) and mCherry-α-tubulin (red), with indicated treatments (left) and corresponding kymographs (right) as described in (A). Scale bars, 2 µm (horizontal) and 30 sec (vertical). \n",
      ".Figure 8A-C. Graph depicting a direct correlation between MT-flux and mitotic spindle where mean spindle length is plotted over mean MT-flux rates for all indicated conditions (A), mitotic conditions containing stable end-on attachments (B), and NDC80 depleted conditions lacking stable end-on attachments (C). Solid and dotted lines represent the linear regression and 95% confidence interval, respectively. Pearson correlation coefficient (r), coefficient of determination (r2) and corresponding p values are indicated. The error bars over x and y axis represent the standard deviation of MT-flux and spindle length, respectively. D. Representative images from spinning disk confocal live-cell time series of U2OS PA-GFP-α-tubulin/mCherry-α-tubulin cells under indicated treatments, with SiR-DNA-labelled DNA. Microtubules are shown in red and DNA in cyan in the merged image. Scale bar: 10 µm. E. Quantification of spindle lengths at metaphase following indicated treatments. Graph represents the spindle length of individual cells with mean ± SD. N (number of cells, number of independent experiments): siControl (49, 5), siMCAK (34,3), siKIF4A + siKIF15 + STLC (46, 6), siKIF4A + siKIF15 + STLC + siMCAK (43, 3), siKIF4A + siKIF15 + STLC + siKIF2A (22, 3), siCLASPs (64, 6), siCLASPs + siMCAK (33, 3), siCLASPs + siKIF2A (32, 3). \n",
      ".Figure 4Overall view of the IAPV-IRES/80S complex class 1 with 60S represented as cyan ribbons, the 40S as yellow ribbons and the IAPV-IRES represented as solid Van der Waals surface colored by secondary structures motifs. Close up view of the intersubunit space with the IAPV-IRES depicted as cartoons colored as in (A) inserted in the unsharpened cryo-EM density. Zoomed view of the A site finger in interacting distance with the SL-III (green). The apical loop of SL-III reaches deep into the 60S contacting the ribosomal protein uL16. Superposition of the IAPV-IRES in complex with 80S (class 1) with tRNAs in different configurations indicated at the bottom. Alignments of the models were computed with the 40S body, excluding from the computation the ligands (IRES/tRNAs) and the 40S head. IAPV-IRES PKI components SL-III/ASL-like domain populates a space of the intersubunit space similar to a A/P-tRNA. IAPV-IRES PKIII (red) mimics the elbow region of a hybrid P/E-tRNA. \n",
      ".Figure 5Ribbon diagram of IAPV-IRES/80S complex viewed from the 40S colored by pairwise root-mean-square deviation displacements observed between the classes indicated on the left. Class 1 is unrotated while classes 2 and 3 exhibit a small rotational movement of the 40S. Top, general overview of the non-rotated IAPV-IRES/80S class 1 structure. IAPV-IRES is depicted as solid Van Der Walls surface colored according to the secondary structure motifs. The PKI (green) is solidly anchored to the A site. Bottom, close up view of the IAPV-IRES PKI inserted in unsharpened map. Zoomed view of A3760, a nucleotide belonging to the helix 69 (H69) of the 28S rRNA interacting with PKI. Final refined model inserted in the post-processed map is shown. This interaction is not disrupted along the small fluctuations of the 40S. An apical view along the axis of the PKI of a superposition of class 1 versus class 2 shows the IRES displacements are minimal and are followed by the H69 which constantly interacts with the IRES. \n",
      "Mfa1+Pg up-regulate the expression of DC-SIGN in human MoDCs.Fig 2A) DC-SIGN mRNA expression in P. gingivalis-infected MoDCs at 0.1, 1 and 10 MOIs. The figure shows the gene expression after 12 hours of Pg381 and mutant strains infections. The target gene (DC-SIGN) was normalized using the endogenous control GAPDH (ΔCt) and fold regulations were calculated using 2-(ΔΔCt) method. The statistical analysis was performed using the t-test, which accounts for the clustering of infected and un-infected controls within 3 different experiments (* p<0.001). B) Immuno-electron microscopy of un-infected MoDCs (Cont.) (upper panel), MoDCs infected with Pg381 (middle panel) and Mfa1+Pg mutants (lower panel). Gold particles (marked with red rings) for positive DC-SIGN were detected in the cell membrane and cytoplasm of cells infected with Mfa1+Pg strains. Minimal positive staining for DC-SIGN was detected in the membranes of MoDCs infected with Pg381, while no cytoplasmic gold labeling was detected in these cells. C) Flow cytometry analysis of surface DC-SIGN in human MoDCs after infection with Pg381, Mfa1+Pg and FimA+Pg. The analysis of the intensity used Kruskal-Wallis test analysis of different groups and Dunn's test for multiple comparisons 3 different experiments (* p<0.01). \n",
      ".Figure 6Biochemical strategy employed to trap a post-translocated state of IAPV-IRES in the ribosome. General view of the IAPV-IRES in a post-translocated state on the ribosome: 60S depicted as blue ribbons, 40S as yellow ribbons, IAPV-IRES represented as solid Van Der Waals surface colored according to secondary structure elements described in figure 1A and eRF1* depicted orange. Close up view of the SL-III inserted in the experimental unsharpened cryo-EM density. On the right, refined model with residues from the 60S (blue) in interacting distance with the SL-III (green) indicated. Comparison of the final refined model for IAPV-IRES colored according to the secondary structure described in figure 1A with canonical tRNAs (PDBID: 4V5D) in the pre-translocated state (left) and after translocation (right). Left, overall top view of the inter-subunit space of the 40S for the post-translocated state. Inset, close up view of the 40S tRNA binding sites where it can be appreciated the insertion of PKI of the IAPV-IRES in the P site, projecting the VLR towards the E site. The elements of the 18S rRNA forming the \"P site gate\" are indicated by solid arrows. On the right, equivalent view for canonical tRNAs. \n",
      ".Figure 7(Top) The Cripavirus family of type IV IRESs is able to capture free 40S sub- units and engage them on a pre-translocation complex by recruiting 60S. This complex is extremely dynamic, with the 40S alternating between non-rotated and rotated configurations with respect to the 60S subunit. IRESs belonging to this family, exemplified by the CrPV-IRES, recruit elongation factors by mimicking a rotated stated of the ribosome with tRNAS. (Bottom), The Aparavirus family of IRESs follows a similar pathway in order to assemble a pre-translocation complex, however, specific structural components of this family allow for additional contacts with the 60S, limiting the rotational freedom of the 40S. Elongation factors engagement and thus effective ribosome hijacking is accomplished by mimicking a ribosome state with tRNAs in hybrid configurations. \n",
      "High intracellular content of Mfa1+Pg within human MoDCs.Fig 1A) Transmission electron microscopy (TEM) of MoDCs infected with P. gingivalis for 2, 12 and 24 hours (left, middle and right panels). The sections show the intra-and extra-cellular contents of Cont. (un-infected), Pg381, Mfa1+Pg and FimA+Pg infected MoDCs for the different time points B) The figure shows the intracellular content of P. gingivalis strains after 24 hours of infection. MoDCs infected with Pg381 or isogenic mutants strains were lysed and the survived intracellular bacteria were re-suspended and maintained in anaerobic broth for 5 days. The data represents CFU within MoDCs harvested from three healthy individuals. The means ±standard deviation (in triplicates) were analyzed by One-way ANOVA of different groups and Tukey's test for multiple group comparisons within 3 different experiments (* statistical significance at the p<0.001). B) The figure shows the intracellular content of P. gingivalis strains after 24 hours of infection. MoDCs infected with Pg381 or isogenic mutants strains were lysed and the survived intracellular bacteria were re-suspended and maintained in anaerobic broth for 5 days. The data represents CFU within MoDCs harvested from three healthy individuals. The means ±standard deviation (in triplicates) were analyzed by One-way ANOVA of different groups and Tukey's test for multiple group comparisons within 3 different experiments (* statistical significance at the p<0.001). \n",
      "Lower intracellular content of Mfa1+Pg in MoDCs treated with Rapamycin.Fig 3A) Pg381 and mutant strains count after 24 hr incubation with human MoDCs with/without Rapamycin treatment. The survived bacteria were measured after maintaining the lysed MoDCs suspension in anaerobic broth for 5 days. The plot represents the means ±standard deviation of CFU within MoDCs harvested from three healthy individuals (* P<0.001). The analysis of readings used One-way AVOVA analysis of different groups and Tukey's test for multiple comparisons. B) Epifluorescence microscopy images of MoDCs treated with Rapamycin 1 hour after P. gingivalis infections. LC3-II (red-fluorescent dye) and the bacterial strains (green CFSE) were studied in MoDCs 11 hours after Rapamycin treatment (12 hours after infections). C) Quantifications of the fluorescent intensity of CFSE-labeled P. gingivalis and LC3-II signals within infected MoDCs using NIS-Elements BR software. One-way ANOVA analysis was used to compare the means of intensity of different groups and Tukey's test for multiple comparisons (* P<0.001). \n",
      "Low LC3-II expression in MoDCs infected with P.Fig 7A) Western blot detection of LC3 in MoDCs infected with Pg381 and mutant strains for 12 hours. B) The mean ±standard deviation of LC3-II intensity represents three different experiments and the values were analyzed using Kruskal-Wallis test analysis of different groups and Dunn's test for multiple comparisons. C) Blot shows the autophagy flux test after blocking the lysosomal fusion with Bafilomycin in MoDCs. Cells were treated with Bafilomycin for 4 hours at 4nM. D) LC3-II intensity in MoDCs infected with Pg381 strains with and without Bafilomycin. The analysis of the intensity of three different experiments used Kruskal-Wallis test analysis of different groups and Dunn's test for multiple comparisons. \n",
      "Low LC3-II signals in human MoDCs infected with Mfa1+Pg.Fig 5A) Epifluorescence microscopy images of MoDCs infected with Pg381, Mfa1+Pg, FimA+Pg and MFB strains after 12 hours. LC3-II was detected in red-fluorescent (Texas red) dye and the bacterial strains were pre-labeled with CFSE (green). Co-localization of P. gingivalis and LC3-II showed in the right panels. B) Quantifications of the fluorescent intensity of LC3-II within infected MoDCs using NIS-Elements BR software. One-way ANOVA analysis was used to compare the means of intensity of different groups and Tukey's test for multiple comparisons of three different experiments (* p<0.001, # p<0.01). C) Bacterial uptake by MoDCs was determined by CFSE fluorescent intensity (* p<0.001, # p<0.01). \n",
      "Induction of autophagy impairs the survival of Mfa1+Pg strain within MoDCs.Fig 4A) Survival of P. gingivalis strains within MoDCs after 6, 12, 24 and 48 hours. Blue lines show P. gingivalis strains survival within MoDCs and their survival in anaerobic condition in the absence of DCs are showed in black lines. The effect of rapamycin on Mfa1+Pg, Pg381 and FimA+Pg survival within MoDCs are shown in figures B, C and D respectively. A three-factor repeated measures ANOVA using mixed models was used to test the effect of strain and rapamycin treatment over time on OD reading. The survival curves for the strains are showed in blue, while the effect of rapamycin treatments are in red. Bacterial survivals in the absence of MoDCs with and without rapamycin are plotted in grey and black, respectively. Statistical analysis showed that the strain by rapamycin treatment overtime interaction indicates the pattern of means in each strain (Mfa1+Pg, Pg381 and FimA+Pg) between treated (rapamycin) and untreated were significantly different overtime (p-value <0.=001). \n",
      "Blocking of DC-SIGN inhibits Mfa1+Pg uptake and restores basal LC3-II signal in MoDCs.Fig 6A) Immuno-fluorescence images of LC3-II and P. gingivalis within MoDCs pre-treated with GP120 (DC-SIGN blocker) after 12 hours of infection. LC3-II was detected in red-fluorescent dye and the bacterial strains were pre-labeled with green CFSE. B) Fluorescent intensity of LC3-II and P. gingivalis strains signal (in triplicates) were statistically analyzed (* and # p<0.001). C) Flow cytometry of MoDCs treated with siRNA for DC-SIGN (si-DC-SIGN) after 12 hours. Left panels show the decrease of DC-SIGN in Cont. (un-infected), Pg381 and Mfa1+Pg-infected MoDCs. D) Epifluorescence microscopy images of MoDCs (si-DC-SIGN) infected with Pg381 and Mfa1+Pg. LC3-II was detected in red-fluorescent dye and the bacterial strains were pre-labeled with green CFSE. E) The figure shows the CFU counts of the P. gingivalis strains with MoDCs that lack DC-SIGN (si-DC-SIGN). The analysis of readings used One-way ANOVA analysis of different groups and Tukey's test for multiple comparisons (* p<0.001). \n",
      "TLRs activation restores LC3-II expression and inhibits the growth of Mfa1+Pg within human MoDCs.Fig 8A) Flow cytometry of CD83 on MoDCs after incubation of TLR4 ligand (E. coli LPS) and TLR1 and 2 ligand (Pam3csk4) for 4 hour. B) Immuno-fluorescence images of LC3-II (red) within MoDCs after incubation with TLR4 and TLR1&amp;amp;2 ligands (E. coli LPS and Pam3csk4) C) The plot represents the means ±standard deviation of CFU within MoDCs harvested from three healthy individuals after 24 hours (** p<0.001). \n",
      ": Structural domains of DIA1 and patients with a novel DFNA1 mutation.Figure 1(A) The positions of M1190D, c.3610C>T (p.R1204X), and c.3636+1G>T (p.A1211VfsX22), with the latter being the only mutation previously associated with DFNA1. Bidirectional arrow shows the autoinhibitory intramolecular interaction between the diaphanous inhibitory domain (DID) and the diaphanous autoregulatory domain (DAD). GBD; GTPase-binding domain, FH1 and FH2; formin homology domain 1 and 2. (B) Electropherograms of DNA sequencing, family pedigrees, and audiograms of two unrelated patients. Electropherograms show a heterozygous c.3610C>T substitution resulting in a stop codon (R1204X). Arrows in family pedigrees indicate the patients identified in the present study. In audiograms: red and blue indicate right and left; respectively, solid and dashed lines and square brackets show air conduction hearing and bone conduction hearing; respectively. The bold line (at the age of 48 years) and thin line (at the age of 50 years) in the audiogram of patient 1 indicate progressive hearing loss (green arrow). \n",
      "LAMP1 decreased within MoDCs infected with P.Fig 10A) Epifluorescence microscopy images of MoDCs infected with pre-labeled bacteria (green-CFSE). LAMP1 detected by transduction of red fluorescent protein (RFP) chimera using baculovirus transgenes to MoDCs. B) The mean of fluorescent intensities ±standard deviation of LAMP1 were quantified in three different experiments (* p<0.001). C) The upper scattered plot shows the co-localization of red and green signals in sections of MoDCs infected with Pg381 and mutants strains. The lower plot shows the average of Pearson's correlation measures of three randomly selected region of interest (ROI) within each field within 3 different experiments (* p<0.0001) (Table 4 and 5). All analysis of fluorescence intensity used One-way ANOVA analysis of different groups and Tukey's test for multiple comparisons. \n",
      ": DIA1(R1204X) induces elongation of microvilli in MDCKmCherry-ESPIN1 cells.Figure 2GFP-tagged DIA1, DIA1(M1190D), and DIA1(R1204X) plasmids were transfected into MDCKmCherry-ESPIN1 cells cultured on membrane inserts. Twenty-four hours after transfection, a reconstructed lateral view of fixed cells was generated using a confocal laser fluorescence microscope. Arrows indicate the elongated microvilli visualized by mCherry-ESPIN1 with GFP-hDIA1(R1204X) expression. Scale bars: 5 μm. Representative of 3 experiments. \n",
      "Formation of double-membrane vesicles in P.Fig 9A) Scanning electron microscopy (SEM) of MoDCs of the early interaction of MoDCs with Pg381 and Mfa1+Pg (upper panel 10000x and lower panel 25000x). B) SEM for MoDCs interacting with Pg381 (green stains for bacteria are computer generated). C) Transmission electron microscopy (TEM) of autophagosome like structures within MoDCs infected with Pg381, Mfa1+Pg and FimA+Pg for 12 hours (upper, middle and lower panels, respectively). The right and left sections show the different magnifications of randomly selected sections. Pg381 and FimA+Pg strains are mostly enclosed in the characteristics double-membrane intracellular vesicles (Orange arrows). Contrary, Mfa1+Pg escaped these autophagic (double-membrane) vesicles and enclosed within single membrane structures or freely occupy the cytoplasm (Green arrows). D) The ratio of bacteria included in the double membrane were compared to total number of bacteria within MoDCs and plotted as percentage. Counting of the bacteria included in single or doubled membrane vesicles after 12 hours of infection. Each strain was counted in three randomly selected grids for each sample. The analysis of the bacterial counts used Kruskal-Wallis test of different groups and Dunn's test for multiple comparisons (*p<0.01). \n",
      ": R1204X mutation in the DAD disrupts the interaction between the DID and the DAD.Figure 4(A) Amino acid sequences of the biotin-labeled WT-, R1204X-, and M1190D-DAD. Underlined and red letters indicate the consensus motif (MDxLLxxL, essential for DID binding) and the unstructured basic region (enhances DID binding), respectively (Lammers et al, 2005). (B) Purified GST and GST-tagged DID proteins (50 nM) were mixed with biotin-labeled DADs (WT; R1204X; and M1190D, 50 nM) in binding buffer. After rotation, streptavidin-coupled magnetic beads were added to the solution, and the mixture was agitated. The material absorbed to the beads was eluted in Laemmli's sample buffer, and were subjected to SDS-PAGE, followed by immunoblotting using an HRP-conjugated GST Ab. Comparable levels of input (GSTproteins) are confirmed in the right panel. Representative of 5 experiments. \n",
      ": DIA1(R1204X)-TG mice show progressive hearing loss and a reduction of HC.Figure 8(A), Age-related click and pure tone-burst (8, 16, 24, 32 kHz) ABR thresholds (dB SPL, mean SE) in DIA1(R1204X)-TG (5 weeks, n = 6; 10 weeks, n = 16; 25 weeks, n = 42) and control mice (5 weeks, n = 6; 10 weeks, n = 16; 25 weeks, n = 33). Note the progressive hearing loss of DIA1(R1204X)-TG mice starting at high frequency sounds from 10 weeks. **P= 0.0039 (24 kHz) and **P< 0.0001 (32 kHz) at 10 weeks and **P< 0.0001 (click), **P< 0.0001 (8 kHz), **P= 0.0001 (16 kHz), **P< 0.0001 (24 kHz), and **P< 0.0001 (32 kHz) at 25 weeks by Bonferroni's post hoc test following two-way ANOVA. (B) Representative SEM images of the organ of Corti at the basal turn obtained from DIA1(R1204X)-TG and control mice at 5, 10, and 25 weeks of age. White and red circles show IHC and OHC losses, respectively. Note OHC-dominant HC loss. Scale bars: 5 µm. (C) The percentages of remaining OHC and IHC in each turn in DIA1(R1204X)-TG and control mice at 25 weeks of age (n = 5). Schematic illustration shows the regions of the cochlea divided into three parts: Ap; apical turn, Md; middle turn, Bs; basal turn. **P= 0.0017 (OHC at Md), **P< 0.0001 (OHC at Bs), *P= 0.0117 (IHC at Ap), *P= 0.0330 (IHC at Md), and **P= 0.0003 (IHC at Bs) by Bonferroni's post hoc test following two-way ANOVA. (D) DPOAE (f2 frequency at 8, 12, 16, 20 kHz) amplitude (dB SPL, mean SE) in DIA1(R1204X)-TG (n = 37) and control mice (n = 30) at 25 weeks of age. *P= 0.0127, **P= 0.0029 (at 12 kHz), and **P= 0.0013 (at 16 kHz) by Bonferroni's post hoc test following two-way ANOVA. (E) High magnification SEM images of OHCs and IHCs at the basal turn of cochlea obtained from DIA1(R1204X)-TG and control mice at 25 weeks of age. Note the sparse and short (arrow) stereocilia of OHC, and sparse and fused (arrowheads) stereocilia of IHC in DIA1(R1204X)-TG mice. Scale bars: 1 µm in OHCs and 5 µm in IHCs. \n",
      ": DIA1(R1204X) induces elongated and thick microvilli in HeLa cells.Figure 3Various GFP-tagged DIA1 plasmids were transfected into HeLa cells, and fixed 24 h after transfection. Fixed cells were stained with Alexa568-conjugated phalloidin and DAPI (A, B: representative of 5 experiments).(A) Stress fiber formation was observed with a confocal laser microscope. Note the induction of stress fibers by GFP-DIA1(M1190D) (asterisks). Arrow and arrowheads indicate a plasma membrane (PM) localization of GFP-DIA1(M1190D) and a dot-like localization of GFP-DIA1(R1204X) adjacent to the PM, respectively. Inset in the top panel of DIA1 (R1204X) shows the magnified image of the region indicated by rectangle. Scale bars: 10 μm.. Various GFP-tagged DIA1 plasmids were transfected into HeLa cells, and fixed 24 h after transfection. Fixed cells were stained with Alexa568-conjugated phalloidin and DAPI (A, B: representative of 5 experiments).(B) Microvilli with or without GFP-DIA1(R1204X) expression were observed under a confocal laser microscope. Note elongated and thick microvilli (arrowheads) in GFP-DIA1(R1204X)- transfected cells, but not in non-transfected cells (circles). 3D movies are available in Movie EV1 (R1204X), Movie EV2 (WT), and Movie EV3 (ttaa). Scale bar: 10 μm. \n",
      ": Single-molecule dynamics of the active mutant, hDIA1(R1204X), in XTC cells.Figure 5GFP-tagged DIA1(WT), DIA1(R1204X), and DIA1(M1190D) were transfected into XTC cells. XTC cells were spread on the PLL-coated coverslips and time-lapse images were acquired every 300 ms, illuminating the restricted areas near the cell edges, by using a single-molecule speckle microscope. Representative images are shown for DIA1(WT) (A, Movie EV5), DIA1(R1204X) (B, Movie EV6), and DIA1(M1190D) (C, Movie EV7). Circles indicate speckles showing directional movement over a distance of several micrometers. Trajectories of these speckles are also illustrated, and crosses were plotted at the end of the directional movement. Speckles showing directional movement were frequently observed in cells expressing GFP-DIA1(R1204X) (B), whereas such speckles were scarce in cells expressing GFP-DIA1(WT) (A). However, the density of these speckles in GFP-DIA1(R1204X)-expressing cells appeared lower than that in GFP-DIA1(M1190D)-expressing cells, which is considered a fully active mutant (C). To evaluate the frequency of speckles indicating directional movement, we performed a quantitative analysis (D). The number of speckles showing directional movement during 3 continuous planes was counted and were normalized by the fluorescent intensity of each cell, which corresponds to the expression level of the GFP-tagged protein. *P= 0.0288 and **P= 0.0002 Bonferroni's post hoc test following one-way ANOVA (WT; n = 5, R1204X; n = 6, M1190D; n = 6). Time is in seconds. Scale bars: 5 µm. \n",
      ": Expression levels of the transgenic DIA1(R1204X) mutant in the inner ear of TG mice.Figure 7(A) Illustration of the linearized expression cassette containing the CAG promoter, 3xFLAG-tagged DIA1(R1204X), and SV40 poly(A) sequence used for injection into fertilized eggs. (B) Expression of FLAG-tagged DIA1(R1204X) in the inner ear and heart of TG, but not WT, mice is confirmed by immunoblotting using a FLAG Ab. Representative of 3 experiments. (C) Expression levels of FLAG-tagged DIA1(R1204X) relative to endogenous Dia1 in the inner ear, thymus, and heart were determined by immunoblotting using a Dia1/DIA1 Ab (inner ear; n = 4, thymus and heart; n = 3), and quantified in the inner ear and thymus (*P= 0.0226 and **P= 0.0016 by two-tailed Student's t-test). Lysates of HEK293 cells overexpressing equal amounts of GFP-Dia1 and GFP-DIA1 (determined using a GFP Ab: right end) are detected with equal affinity by the Dia1/DIA1 antibody. Comparable loading of proteins is confirmed using an α−tubulin or GAPDH Ab. \n",
      ": No HC loss or hearing loss in Dia1-KO mice.Figure 6Cochleae from control, heterozygous Dia1-KO (Dia1-/+) and homozygous Dia1-KO (Dia1-/-) mice were fixed at 5 weeks of age (A, B; representative of n=6).(A) Paraffin-embedded 4 μm sections of cochlea were stained with hematoxylin and eosin. No hair cell (HC) loss (either IHC or OHC) is observed at the basal turn of the Dia1-/+ or Dia1-/-cochleae. No obvious differences are observed in the spiral ganglion (SV), spiral ligament (SL), stria vascularis (SV), and Reissner's membrane (RM) in Dia1-/+, or Dia1-/-mice compared with control mice. Lower panels (Scale bars: 20 µm) are magnified images of the region indicated rectangle in upper panels (Scale bars: 100 µm). Cochleae from control, heterozygous Dia1-KO (Dia1-/+) and homozygous Dia1-KO (Dia1-/-) mice were fixed at 5 weeks of age (A, B; representative of n=6).(B) Basal turns of the dissected cochleae were immunostained using a Myo7a Ab followed by an Alexa488 secondary Ab with DAPI as a nuclear counterstain. No HC loss (either IHC or OHC) is observed in Dia1-/-mice. Scale bars: 20 µm. (C) ABR thresholds (10, 20, 40 kHz) in control and Dia1-/-mice at 5 weeks of age (dB SPL, mean SE). No hearing loss was observed in Dia1-/-mice (n = 8) compared with control mice (n = 4). \n",
      ".Figure 6(A) Vinculin was knocked down as described (Figure 5D) in MCF10A cells and cells were stained for LATS1 and TRIP6. Merged images show LATS1 (green), TRIP6 (red) and DNA (blue). Scale bar=20µm. (B) Vinculin was knocked down as described (Figure 5D) and TRIP6 immune complexes and lysates were probed by western blotting for LATS1, TRIP6, and vinculin and the relative levels quantified (Mean ± SD; n=3; **P≤0.01, T-test). (C) MCF10A cells were subjected to different treatments as described (Figure 4A) and TRIP6 immune complexes and lysates were probed by western blotting for vinculin and TRIP6. (D) Quantification of TRIP6-vinculin binding from part (C) is shown. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, ***P≤0.001, T-test). (E, F) MCF10A cells were treated separately (or not) with Latrunculin B (E) and by serum starvation (F) and LATS1 immune complexes and lysates were probed by western blotting for vinculin, LATS1, and TRIP6. Quantification of LATS1-TRIP6-vinculin binding is shown. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, ***P≤0.001, T-test). (G) Wild type and T12 mutant of vinculin were co-expressed with LATS2 and FLAG-TRIP6 and anti-FLAG antibody was used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for vinculin, LATS2 and FLAG-TRIP6. Quantification of LATS2-TRIP6-vinculin (WT or T12) binding is shown. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, ***P≤0.001, T-test). (H) Vinculin was knocked down as described (Figure 5D) in MCF10A cells grown at high density on PDMS membranes. The membranes were stretched at 17% elongation for 2 hours and fixed while under tension and stained for TRIP6. Merged images show TRIP6 (red) and DNA (blue). Scale bar=20µm. \n",
      ".Figure 4(A) MCF10A cells were either not treated or treated separately with Latrunculin B, Blebbistatin, EGTA, serum starvation, and Y27632, and were stained for TRIP6 and LATS1. Merged images show LATS1 (green), TRIP6 (red) and DNA (blue). Scale bar=20µm. (B) Cells were treated as in (A), then lysed and anti-TRIP6 antibodies were used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for LATS1 and TRIP6. (C) Quantification of (B) is shown. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, T-test). \n",
      ".Figure 2(A) LATS2-GFP and Myc-MOB1A were overexpressed in HEK293 cells with or without co-overexpression of full length FLAG-TRIP6 and FLAG-TRIP6 1-277. Myc-MOB1A was immunoprecipitated using anti-Myc antibodies and immune complexes were assayed for Myc-MOB1A and LATS2-GFP levels. Levels of FLAG-TRIP6, FLAG-TRIP6 1-277, Myc-MOB1A, and LATS2-GFP in the lysate are also shown. The levels of LATS2-GFP in immune complexes relative to the level of Myc-MOB1A are shown in the graph (Mean ± SD; n=3; **P≤0.01, T-test) (B) Competitive binding experiments were done using purified recombinant MBP-LATS2, GST-TRIP6, and 6His-MOB1A. MBP-LATS2 bound to maltose beads was incubated with GST-TRIP6 with or without increasing amounts of 6His-MOB1A, and the levels of each protein bound to MBP-LATS2 on the beads at the end of the experiment was determined by western blotting. The levels of input proteins are shown (lysate). The binding of 6His-MOB1A and GST-TRIP6 to MBP alone, and the use BSA as a competitor instead of 6His-MOB1A are shown as controls. The numbers at the bottom are referred to in the text. \n",
      ".Figure 5(A) MCF10A cells were lysed and anti-TRIP6 or control (IgG) antibodies were used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for vinculin and TRIP6. (B) MCF10A cells were either not treated (control) or treated separately by growth to high density, serum starvation, Latrunculin B, Blebbistatin, EGTA, or Y27632 treatment, and stained using anti-vinculin antibodies by immunofluorescence. Scale bar=20µm. (C) MCF10A cells grown at high density on PDMS membranes and were stretched (or not) at 17% elongation for 2 hours and fixed while under tension and stained for vinculin. Scale bar=20µm. (D) Vinculin was knocked down using two different siRNAs or control siRNA in MCF10A cells and the cell lysates were probed by western blotting for phospho-LATS1 (T1079 and S909), phospho-YAP (S127), LATS1, YAP, vinculin, and tubulin antibodies and the relative levels quantified (Mean ± SD; n=3; **P≤0.01, T-test). (E) Vinculin was knocked down as described in (D) and cells were stained for YAP and vinculin. Merged image shows YAP (green), vinculin (red), and DNA (blue). Quantification of at least 100 cells is shown (Mean ± SD; n=3; ***P≤0.001, Fisher's test). Scale bar=20µm. (F) Vinculin was knocked down as described in (D) and the levels of vinculin and YAP target gene expression was analyzed using RT-qPCR (Mean ± SD; n=3; ***P≤0.001, ****P≤0.0001, T-test). (G) Vinculin was knocked down in MCF10A cells as described in (D), grown at high density on PDMS membranes and cells were stretched (or not) at 17% elongation for 2 hours and the levels of vinculin and YAP target gene expression was analyzed using RT-qPCR (Mean ± SD; n=3; ***P≤0.001, ****P≤0.0001, T-test). \n",
      ".Figure 3(A) MCF10A cells were infected with lentivirus carrying control shRNA (shEGFP), or a mix of two different shRNA against TRIP6 (shTRIP6-1 and shTRIP6-4) and were stained for TRIP6 and LATS1. Merged images show LATS1 (green), TRIP6 (red) and DNA (blue) (quantification of LATS1 localization at cell-cell junctions is shown in Figure EV3D). Scale bar=20µm. (B) LATS1 and LATS2 were knocked down using siLATS1 and siLATS2 SMARTPools in MCF10A cells. MCF10A control cells were treated with control siRNA (siControl) against fire fly luciferase. Cells were stained for TRIP6 and LATS1 as in (A) (quantification of TRIP6 localization at cell-cell junctions is shown in Figure EV3E). Scale bar=20µm. (C) MCF10A cells were grown to high density, and were stained for TRIP6 and LATS1 as in (A). Scale bar=20µm. (D) Cells were grown as in (C), then lysed and anti-TRIP6 antibodies were used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for LATS1 and TRIP6. Quantification is shown. (Mean ± SD; n=3; *P≤0.05, T-test). (E) MCF10A cells grown at high density on PDMS membranes and were stretched (or not) at 17% elongation for 2 hours and lysed while under tension. Anti-TRIP6 antibodies were used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for LATS1 and TRIP6. Quantification is shown. (Mean ± SD; n=3; *P≤0.05, T-test). (F) Cells were treated as in (E), fixed while under tension, and stained for TRIP6 and LATS1 as in (A). Scale bar=20µm. (G) MCF10A cells infected with lentivirus carrying control shRNA (shEGFP), or a mix of two different shRNA against TRIP6 (shTRIP6-1 and shTRIP6-4), were grown at high density on PDMS membranes and were stretched or not (only stretched cells shown) at 17% elongation for 2 hours, fixed while under tension, and were stained for TRIP6 and LATS1 as in (A). Scale bar=20µm. (H) Cells were treated as in (G) and YAP target gene (CTGF and Cyr61) and TRIP6 expression were analyzed using RT-qPCR. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, T-test). (I) TRIP6 expression levels in (H) in control (shEGFP) and TRIP6 (shTRIP6) knockdown cells just prior to stretch were analyzed using RT-qPCR. (Mean ± SD; n=3; *P≤0.05, T-test). \n",
      ".Figure 1TRIP6 promotes YAP activity by inhibiting LATS1/2 (A) Full length (WT), the amino-terminal half (1-277), or the carboxy-terminal half (278-476) of TRIP6 were tested for binding to LATS2 by immunoprecipitation. FLAG-TRIP6 variants were co-expressed with LATS2-GFP in HEK293 cells, anti-FLAG or control (IgG) antibodies were used to isolate immune complexes. Immune complexes and lysates were probed by western blotting for LATS2-GFP and FLAG-TRIP6. Schematic diagram depicts TRIP6 domains (NES: Nuclear Export Signal; LIM: LIM domain; PDZ: PDZ domain binding motif). (B) Lysates from MCF10A cells were subjected to immunoprecipitation using anti-TRIP6 or control (IgG) antibodies, and immune complexes and lysates were probed for TRIP6 and LATS1. (C) FLAG-TRIP6 was tested for binding to various LATS2-GFP deletion mutants as described in part (A). Schematic diagram of LATS2 shows MOB1 binding domain, and the autophosphorylation (S872) and MST1/2 phosphorylation sites (T1041) in the kinase domain. The regions marked in green depict TRIP6 binding sites on LATS2 (D) Lysates from HEK293A cells transfected with control or FLAG-TRIP6 plasmid were analyzed by western blotting using the indicated antibodies (quantification is shown in (F)). (E) Lysates from control (WT) or CRISPR generated TRIP6 null (TRIP6-KO) HEK293A cells were analyzed by western blotting using the indicated antibodies (quantification shown in (G)). (F) The relative levels of LATS1 activating phosphorylation (pLATS1-1079, 909) and YAP S127 inhibitory phosphorylation from (D) were measured relative to LATS1 and YAP levels respectively. (Mean ± SD; n=3; **P≤0.01, ***P≤0.001, T-test) (G) The levels of LATS1 activating phosphorylation and YAP inactivating phosphorylation from part (E) were quantified (Mean ± SD; n=3; *P≤0.05, T-test). (H) GFP-MST2 was expressed with or without FLAG-TRIP6 in HEK293 cells and the levels of MST2, MST2 activating phosphorylation (pMST2-T180), and FLAG-TRIP6 were measured by western blotting with the indicated antibodies. (Mean ± SD; n=3; n.s.≥0.05, T-test). (I) TRIP6 was overexpressed in HEK293A cells and the levels of TRIP6, and YAP target gene expression was analyzed using RT-qPCR. (Mean ± SD; n=3; *P≤0.05, **P≤0.01, ****P≤0.0001, T-test). (J) The levels of YAP target gene expression was analyzed using RT-qPCR in control (WT) and TRIP6-KO HEK293A cells. (Mean ± SD; n=3; ***P≤0.001, T-test) (K) Control (WT) and TRIP6-KO HEK293A cells were stained for YAP and TRIP6. Merged image shows YAP (green), TRIP6 (red), and DNA (blue). Quantification of at least 100 cells is shown (Mean ± SD; n=3; ****P≤0.0001, Fisher's test). Scale bar=20µm. \n",
      ".Figure 1A) Oocytes were fixed for chromosome spreads 8 hours after GVBD, and stained for PP2A-c (red), Sgo2 (green), CREST (blue) and DNA (Hoechst, shown in grey). Spreads were classified into metaphase I and anaphase I, depending on whether chromosome segregation had taken place, or not. Arrowhead indicates fused sister kinetochores, arrows indicate separated sister kinetochores. B) Example of a whole mount oocyte staining, in early anaphase I after cold-treatment to visualize only cold-stable microtubule fibers, stained with anti-tubulin antibody (green). Kinetochores are stained with CREST serum (red) and chromosomes with Hoechst (blue). Shown is an overlay of 20 z-sections. Arrows indicate separated sister kinetochores. C) Metaphase I (6 hours after GVBD) and metaphase II (16-20 hours after GVBD) chromosome spreads were stained with H3K9me3 antibody and CREST serum to reveal pericentromeric heterochromatin (pink) and the centromere (green), respectively. Chromosomes were stained with Hoechst (blue). Below, schematic representation of a bivalent in metaphase I and a dyad in metaphase II. Enlargement shows the centromere region and a region that we call \"chromatid junction\" with the corresponding scheme. D) Quantification of total Sgo2 signal at the centromere and the chromatid junction, normalized to CREST at centromeres, on chromosome spreads from oocytes at the indicated stages of meiotic maturation. Each dot shows mean per oocyte; the number of oocytes analyzed is indicated. Below: scheme illustrating how Sgo2 signals (in red) were measured at the indicated meiotic stages. Ifluor stands for a mean fluorescence intensity of the area within the box (dotted lines, see Material and Methods for quantification approach). E) Chromosome spreads as in A), stained for Sgo2 (red), CREST (green), and DNA (Hoechst, in blue). Spreads in early anaphase I and late anaphase I - telophase I were classified, depending on whether dyads of two chromosome sets had been found close to each other as one group, or were scattered apart into two groups. In early anaphase I, Sgo2 is found inbetween sister kinetochores that are already separated.\n",
      " Kinetochore individualization in meiosis I is required for step-wise cohesin removal A) On monopolar spindles, as on bipolar spindles, chromosomes segregate in meiosis I, and sister chromatids in meiosis II.Figure 7A) On monopolar spindles, as on bipolar spindles, chromosomes segregate in meiosis I, and sister chromatids in meiosis II. B) Model of how separase-dependent kinetochore individualization primes bivalents for deprotection of centromeric cohesin. If separase is absent in meiosis I, arm cohesin and not centromeric cohesin is removed in meiosis II. Under these conditions, sister kinetochores individualize in meiosis II instead of meiosis I, demonstrating that step-wise cohesin removal (first arm cohesin, then centromeric cohesin) depends on the kinetochore structure. Without kinetochore individualization, centromeric cohesin cannot be cleaved in the subsequent division. \n",
      ".Figure 5A) Sep+/+ or Sep-/- oocytes were matured in vitro until metaphase II. Sep-/- oocytes were injected with wild-type separase or cleavage mutant separase encoding mRNAs in metaphase II. Oocytes were either fixed in metaphase II or activated and fixed in anaphase II. Spreads were stained with Rec8 antibody (red), CREST serum (green), and DAPI (blue) to label DNA. n indicates the number of oocytes with the shown phenotype (e.g. in anaphase II, after successful activation) and the total number of oocytes analysed B) Representative image of kinetochore individualization in Sep-/- oocytes injected with wild-type separase in metaphase II and fixed after activation, from A). Shown is staining with CREST serum (green), and DAPI (blue) to label DNA. Arrows indicate separated sister kinetochores. The scheme on the left illustrates the individualization of sister kinetochores (green arrows) at anaphase II. n indicates the number of oocytes with the shown phenotype (e.g. in anaphase II, after successful activation) and the total number of oocytes analysed Scale bars: 10 µm, except bottom panel in B), where the scale bar is 2 μm. \n",
      ".Figure 3(A and D) Oocytes in meiosis I (6 hours after GVBD) or meiosis II (left and right panels, respectively) were treated with STLC for 2.5 hours and fixed for whole mount immunofluorescence. Cold stable microtubules were stained with an antibody against tubulin (green), centromeres with CREST (red) and DNA with Hoechst (blue). Scale bar indicates 5 µm in (A and D), and 10 µm in (B, C, E and F). In all panels n indicates the number of oocytes with the shown phenotype and the total number of oocytes analyzed. (B and E) Chromosome spreads at metaphase and anaphase of meiosis I (6 and 9 hours after GVBD) or meiosis II (left and right panels, respectively), in the presence or absence of STLC (for 2.5 hours) and Reversine (+Rev., for 1 hour) as indicated. Spreads were stained with CREST serum (red) and Hoechst (grey). Single chromosomes are present in anaphase I and single sister chromatids in anaphase II. Scale bar indicates 5 µm in (A and D), and 10 µm in (B, C, E and F). In all panels n indicates the number of oocytes with the shown phenotype and the total number of oocytes analyzed. (C and F) Chromosome spreads exactly as described in B and E), except that spreads were additionally stained with an antibody against Rec8 (green). The CREST signal appears in red and DNA in blue. Rev: Reversine. Scale bar indicates 5 µm in (A and D), and 10 µm in (B, C, E and F). In all panels n indicates the number of oocytes with the shown phenotype and the total number of oocytes analyzed. \n",
      ".Figure 2A) Chromosome spreads at the indicated stages of meiosis, stained for endogenous Sgo2 (red), centromeres with CREST serum (green) and DNA with Hoechst (blue). On the right, schemes illustrate the chromosome figures observed, and the corresponding Sgo2 staining (in red). B) Dyads of metaphase II chromosome spreads of control, Bub1KD, Mps1∆N and Reversine-treated oocytes (from GVBD onwards) stained with Sgo2 antibody (red), CREST serum (green) and Hoechst (blue). C) Dyads of metaphase II chromosome spreads of oocytes treated with Reversine from GVBD + 8 hours onwards, corresponding to the time of anaphase I onset. PBE: Polar Body Extrusion. D) Chromosome spreads at the indicated stages of meiosis, stained for endogenous Mps1 (red), centromeres with CREST serum (green) and DNA with Hoechst (blue). For early and late anaphase II spreads oocytes were chemically activated with Strontium and fixed 25min or 1h later, respectively.F) Percentage of oocytes harbouring one or more single sister chromatids when analysed by chromosome spreads stained with CREST and Hoechst. Where indicated, oocytes were treated with Reversine from GVBD onwards, and treatment was renewed every 8 hours. Early CSF corresponds to 8 hours after GVBD (Reversine treated oocytes undergo anaphase I on average 4-5 hours after GVBD), and late CSF corresponds to 36 hours after GVBD (>29 hours in CSF arrest). PSCS: precocious sister chromatid segregation. G) Quantification of total Sgo2 signal on spreads in early or late CSF arrest (16 hours or 30 hours after GVBD, respectively) relative to CREST per kinetochore pair (per single kinetochore in anaphase II) (left dot plot), and Sgo2 signal averaged per oocyte (right dot plot) at the indicated stages. The number of kinetochore pairs and oocytes analyzed is indicated. The scheme of measurements is shown on the right. Meta: metaphase, ana: anaphase. \n",
      ".Figure 4A) Metaphase II oocytes from mice carrying a conditional, oocyte-specific invalidation of separase (Separaseflox/flox Zp3 Cre+, or Sep-/-) and oocytes from litter mates (Separaseflox/flox, or Sep+/+) were obtained after superovulation. In vivo matured control (Sep+/+) and Sep-/- oocytes are found in oviducts at comparable numbers, enclosed by cumulus cells. Example of cumulus enclosed oocytes from the oviduct is shown below. In (A), n indicates the number of mice and a mean number of oocytes per mouse of each genotype, error bars are + s.e.m. B) Whole mount immunofluorescence to stain cold stable microtubule fibers after in vivo maturation. Microtubules were stained with anti-tubulin antibody (green), centromeres with CREST serum (red) and DNA with Hoechst (blue). Bivalents are attached in a monopolar fashion. Approximate number of z-sections used for overlays to visualize the whole spindle (on the left) or individual chromosomes (magnifications on the right) are indicated.D) Metaphase-to-anaphase transition of meiosis II in oocytes of the indicated genotype cultured in vitro and injected with the separase constructs as indicated, in GV. The schemes indicate the chromosome figures observed upon activation. Chromosome spreads are stained with CREST serum (green), and Rec8 antibody (red). DNA was stained with DAPI (blue). In (D) n indicates the number of oocytes with the shown phenotype (e.g., upon successful activation) and the total number of oocytes analyzed. \n",
      "Atg5‐deficient goblet cells accumulate mucin.Figure 1(A) Alcian blue‐stained sections of descending colons from control and Atg5VC mice. The area between the two black dashed lines indicates the crypt base where epithelial progenitors and nascent goblet cells reside. Bars=200 μm. (B) Quantification of average mucin area/goblet cell in control and Atg5VC mice (n=5-7 mice/group from 3 independent experiments; 100 goblet cells measured/mouse). **P0.01 as determined by the Student's t‐test. This analysis was performed on upper crypt goblet cells (above the upper dashed line in A). (C) Quantification of the average number of goblet cells per crypt (n=3 mice/group; over 100 crypts measured/mouse). (D) TEM images of upper crypt goblet cells from control and Atg5VC transverse colons. The yellow dashed lines outline the mucin granules in the apex of the cells. Bars=2 μm. (E, F) Quantification of (E) the average granule density and (F) the average number of mucin granules/goblet cell (n=3 mice/group and 20 goblet cells evaluated/mouse). ***P0.001 as determined by the Student's t‐test. (G, H) Whole‐mount images of the mucosal surface from control and Atg5VC transverse colons. Whole‐mount colonic samples were either stained with Ulex europaeus agglutinin‐1 (UEA) that labelled mucus (G) or analysed by the scanning electron microscopy (H). Yellow arrows indicate the crypt opening orifices. Green arrowheads indicate goblet cells protruding from the surface epithelium of Atg5VC mice (insets in H show higher power image of the latter). Bars=100 μm. \n",
      "Atg5‐deficient colonic epithelial spheroids accumulate cytoplasmic mucin within goblet cells.Figure 2(A) Schematic for the generation of differentiated goblet cells within colonic epithelial spheroid cultures. (B) Representative whole‐mount images of single spheroids cultured in either 50% conditioned media (CM) or 5% CM+5 μM DAPT and stained for Muc2 (green) to visualize goblet cells. Bars=20 μm. (C, D) Graph of relative expression (as measured by qRT‐PCR) of Atoh1 (C) and Muc2 (D) mRNAs comparing spheroids grown in either 5% CM or 5% CM+ DAPT to spheroids grown in 50% CM (n=4-5 samples/group). Error bars indicate s.e.m. *P0.05; **P0.01 as determined by a two‐tailed Student's t‐test. (E, F) Images of colonic epithelial spheroids isolated from control and Atg5VC mice grown in 5% CM+DAPT and stained for Muc2 (green: E) or with the lectin, UEA (red: F). Bars=20 μm. (G) Quantification of average mucin area/goblet cell. Error bars indicate s.e.m. (n=6 samples/group; 100 cells were quantified/sample; **P0.01 as determined by a two‐tailed Student's t‐test). (H) Relative expression of Muc2 mRNA in differentiated spheroids from control and Atg5VC mice as measured by qRT-PCR (n=4-5 samples/group). No statistically significant differences were found in Muc2 expression as determined by the Student's t‐test. (I, K) Images of FIP200f/f (I) and Atg14f/f (K) colonic epithelial spheroids treated with recombinant Tat‐Cre and stained with UEA (red). (J, L) Quantification of average mucin area/goblet cell. Error bars indicate s.e.m. (n=6 samples/group; 100 cells were quantified/sample; **P0.01 and ***P0.001 as determined by a two‐tailed Student's t‐test). \n",
      ".Figure 6A) Live imaging movie of GFP-Sgo2 expressing oocytes (from GV onwards) undergoing meiosis I. The green channel to visualize GFP-Sgo2 localized to chromosomes is shown. Timepoints were taken every 10min, shown is a montage of selected frames such as indicated (hours: time after GVBD) at the metaphase-to-anaphase transition of meiosis I and in metaphase II. Each timepoint is comprised of an overlay of 11 z-sections of 3μm for GFP. The arrow heads indicate bivalents. In (A-C) n indicates the number of control non-injected and GFP-Sgo2-expressing oocytes analyzed B) Representative images of a chromosome spread showing Rec8 localization on a bivalent chromosome in metaphase II of a GFP-Sgo2 expressing oocyte. Spreads were stained with Rec8 antibody (pink), CREST serum (green), Hoechst for DNA (blue), GFP-Sgo2 was visualized by GFP fluorescence (grey). The scheme on the right shows a bivalent with individualized kinetochores and some Rec8 staining on arms. In (A-C) n indicates the number of control non-injected and GFP-Sgo2-expressing oocytes analyzed C) Metaphase I (4hrs after GVBD) and II spreads after GFP-Sgo2 expression (from GV onwards). Spreads were stained with CREST serum (red), Hoechst for DNA (grey), GFP-Sgo2 was visualized by GFP fluorescence (green). Arrowheads mark separated sister kinetochores. On the right, graph showing percentage of metaphase II spreads containing bivalent chromosomes. In (A-C) n indicates the number of control non-injected and GFPD) Chromosome spreads of oocytes expressing GFP-Sgo2 (from GV onwards) in metaphase II, and upon activation. Spreads were labeled with Rec8 antibody (shown in red, far-red secondary antibody was used), CREST serum (pseudo-colored in green), and Hoechst (blue). GFP direct fluorescence was imaged for GFP-Sgo2 detection. Below the corresponding chromosome figures are represented in the schemes.\n",
      "Rb−/− myoblasts transiently differentiate to form short myotubes that do not twitch, express autophagy markers, and slowly degenerate.Figure 1.(A) Western blot analysis for pRb in DM-2 cultures. (B) Immunostaining for MHC in DM-3 and DM-6 differentiating myotubes. Nuclei were stained with DAPI (blue). (C) Mean number of myotubes on the indicated days after differentiation. Each time point is mean ± SD of six fields from six independent experiments. (D) TUNEL staining. Arrowheads indicate TUNEL-positive nuclei (green). (E) LC3-RFP expression in control (ctrl) and Rb−/− myotubes. Insets show nuclear DAPI stain (blue). (F) Representative Western blot (n = 3) for LC3. Chloroquine was added 12 h before cell harvest. (G) Representative Western blot for LC3 in E16.5 control and mgRb:Rb−/− muscles. (H) Quantification of LC3-II/tubulin ratio in E16.5 mgRb:Rb−/− skeletal muscles relative to control as depicted in G (mean ± SD; n = 3). *, P = 0.016 by Student's t test. (I) Representative Western blot for LC3-I and LC3-II expression in whole cell lysates from the indicated myotubes transduced with Ad.GFP or Ad.RbΔK11. (J) Quantification of LC3-II/tubulin ratio by Western blots as shown in I. Data represent mean ± SD (n = 7). *, P = 0.047; **, P = 0.005 by analysis of variance. \n",
      "LC3β is localized to EEA1‐positive multi‐vesicular vacuoles within colonic goblet cells.Figure 3(A) Images of colonic sections stained for LC3β (green) from control and Atg5VC mice. The white‐dashed lines indicate the border between the basal surface of the crypt epitheliumand the mesenchyme. White asterisks denote non‐specific plasma cell staining from the mouse monoclonal anti‐LC3β antibody. Bars=50 μm. (B) Immunogold transmission electron microscopic image for LC3β in a colonic goblet cell from a wild‐type mouse. The inset is a low power image of the entire goblet cell. The black box within the inset indicates the area shown in this panel. Note this area is not associated with mucin granules. Bar=500 nm. (C) Enriched KEGG pathways for Atg5‐deficient epithelial cells obtained from a microarray comparing control and Atg5VC colonic crypt base epithelium. P‐values were determined by modified Fisher's exact test. (D-G) Double label immunogold transmission electron microscopic images from wild‐type mice for LC3β (12 nm particles) and EEA1 (D), Rab7 (E), Rab11 (F), or Lamp1 (G) (18 nm particles). Bars=200 nm. \n",
      "Goblet cell mucin accumulation correlates with diminished ROS production.Figure 6(A) Colonic epithelial cell spheroids from p22phox−/−, Atg5VC and wild‐type mice. Wild‐type spheroids were treated with vehicle (control), 10 μM DPI (NADPH oxidase inhibitor) or 100 μM Dynasore. Intracellular ROS in live cells were labelled by 5 μM DCF. Bars=100 μm. (B) Quantification of fluorescent DCF normalized to control (n=3 independent experiments/group; 30 spheroids quantified/experiment). (C) Quantification of average area of cytoplasmic mucin from wild‐type mice treated with 3% NAC as compared to controls (n=6-7 mice/group). Error bars indicate s.e.m. *P0.05; **P0.01; ****P0.0001 as determined by the Student's t‐test. (D) TEM images of colonic epithelial monolayers treated with cerium chloride. The presence of electron dense precipitate is indicative of ROS production. Bar=500 nm. \n",
      "Inhibition of endocytosis results in goblet cell mucin accumulation.Figure 4(A) Schematic for the generation of differentiated colonic epithelial cells grown in a monolayer. (B) Colonic epithelial monolayers from wild‐type mice treated with either vehicle or 100 μM Dynasore and stained for LC3β (green) and EEA1 (red). Bars=5 μm. (C, D) Quantification of average number of LC3β dots per cell (C) and the percentage of LC3β dots that co‐localized with EEA1 per cell (D) (n=3 experiments/group; 25-30 cells/experiment). (E) LC3 immunoblots of epithelial spheroids treated with 100 μM Dynasore and 100 nM BafA1 as indicated. GAPDH is a loading control. Representative image is shown from n=3 experiments. (F) Images of wild‐type colonic epithelial spheroids labelled with TRITC‐UEA (red) after treatment with either vehicle or 100 μM Dynasore. Bars=20 μm. (G) Quantification of average mucin area/goblet cell (n=6 samples/group; 150 cells were quantified/sample). Error bars indicate s.e.m. ***P0.001 as determined by the Student's t‐test. \n",
      "NADPH oxidases are required for endosome/autophagosome co‐localization and control mucin granule accumulation.Figure 5(A) Colonic epithelial monolayers from a wild‐type and p22 mut mouse stained with LC3β (green) and EEA1 (red). Bar=5 μm. (B, C) Quantification of average number of LC3β dots per cell (B) and the percentage of LC3β dots that co‐localized with EEA1 per cell (C) (n=3 experiments/group; 25-30 cells/experiment). (D) LC3 immunoblots of either wild‐type or p22 mut epithelial spheroids treated with 100 nM BafA1 as indicated. Representative image is shown from n=3 experiments. (E) Double label immunogold transmission electron microscopic images for LC3β (12μnm particles) and p22phox (18 nm particles) showing a p22phox‐positive single membrane cytoplasmic vesicle (D) and a p22phox‐, LC3β‐positive vacuole (E). Bars=200μnm. (F) Colonic epithelial monolayers from a wild‐type mouse stained for LC3β (green) and p22phox (red) and quantification of percent LC3β dots co‐localized with p22phox per cell, Bar=5 μm. (G) Colonic epithelial cellspheroids from control and p22 mut mice stained with TRITC‐UEA to label goblet cellmucin. Bars=50 μm. (H) Quantification of average mucin area/goblet cell (n=6 samples/group; 150 cells were quantified/sample). Error bars indicate s.e.m. **P0.01; ****P0.0001 as determined by the Student's t‐test. \n",
      "Exogeneous ROS can restore mucin granule accumulation defects.Figure 7Exogeneous ROS can restore mucingranule accumulation defects. (A) Quantification of fluorescent DCF normalized to controls (n=3 independent experiments/group; 10-30 spheroids quantified/culture). *P0.05 as determined by ANOVA with Tukey's multiple post‐test comparisons (B) Images of control and Atg5VC colonic epithelial spheroids labelled with TRITC‐UEA (red) after treatment with either vehicle or 300 μM H2O2. Bars=20 μm. (C) Quantification of average mucin area/goblet cell (n=6 samples/group; 150 cells were quantified/sample). ***P0.001, ****P.0001 as determined by ANOVA with Tukey's multiple post‐test comparisons. (D) Relative expression of Muc2 mRNA in differentiated spheroids from control and Atg5VC mice treated with either vehicle or 300 μM H2O2 as measured by qRT-PCR. No statistically significant differences were found in either group as determined by ANOVA. (E, G) Images of p22 mut or Dynasore‐treated colonic epithelial spheroids (mucin is labelled with TRITC‐UEA; red) after treatment with either vehicle or 300 μM H2O2. Bars=20 μm. (F, H) Quantification of average mucin area/goblet cell (n=6 sample/group; 30-60 cells were quantified/sample) corresponding to experiments performed using p22 mut (E) and Dynasore‐treated wild‐type spheroids (G) Atg5VC and p22 mut spheroids were treated with 300 μM H2O2 plus or minus 100 μM BAPTA‐AM (I) or PMA/ionomycin (J). (I, J) Quantification of average mucin area/goblet cell (n=5 sample/group; 150 cells were quantified/sample). Error bars indicate s.e.m. ****P0.0001 as determined by the Student's t‐test (I) or ANOVA with Tukey's multiple post‐test comparisons (J). \n",
      "The Rb−/− muscle defect is characterized by perinuclear mitochondrial/autophagosome aggregates, reduced mitochondrial DNA content, and reduced ATP production.Figure 2.(A, left) MitoTracker staining. (right) MitoTracker staining after bezafibrate treatment. (B) Quantification of myotubes with perinuclear MitoTracker-positive aggregates in untreated (vehicle) or bezafibrate-treated myotubes at DM-2. Data represent mean ± SD (n = 3). *, P < 0.001 by Student's t test. (C) MitoTracker (red) and LC3immunostaining (green) of myotubes at DM-2. Chloroquine and 3-MA (where indicated) were added 12 h before staining. Note the extensive overlap of MitoTracker and LC3 in Rb−/−myotubes (yellow), which is inhibited by 3-MA. (D) Mean percentage of MitoTracker (red) to LC3 (green) overlap in 25 myotubes from two independent cultures in control versus Rb−/−myotubes (total) or Rb−/−myotubes with perinuclear mitochondrial aggregation (high). *, P = 7 × 10−5 by Student's t test. (E) Ultrastructural analysis of myotubes treated or not with 3-MA at DM-2. Arrowheads point to mitochondria. Asterisks indicate electron-dense autolysosomes. N, nucleus. (F) Ratio of mitochondrialDNA/nuclearDNA in the indicated cultures (n = 3). (G) Ratio of mitochondrialDNA/nuclearDNA in E16.5 skeletal muscle (n = 3; *, P = 0.009). (H) Quantification of ATP in E16.5 skeletal muscle (n = 3; *, P = 0.011). (F-H) Error bars represent SD. *, P < 0.001. \n",
      "Endosome types involved in FP+.Figure 6Endosome types involved in FP+. The FP+ were classified according to MVE and VE involved. The actual numbers of FP+ analyzed are given above the bars. \n",
      "Appearance of tracer in AV.Figure 3Appearance of tracer in AV. Cells were allowed to endocytose tracer particles continuously for the designated periods of time. At each time, random sections were searched for AV+, and AV+/500 cell profiles were plotted. The number of cell profiles examined (denominator) and the respective number of AV+ scored (numerator) are listed above the graph. \n",
      "Tracer distribution in AV+ types.Figure 4Tracer distribution in AV+ types. Cells were allowed to endocytose tracer particles continuously for the designated periods of time. At each time, random sections were searched for AV+. The results are categorized according to AV types and plotted for the relative distribution of tracers. The total tracer counts are given on top of each bar. The number in each block represents the actual number of AV profiles, from which tracer counts are scored. \n",
      "ASGP-R labeling in FP+ and endosomes.Figure 7ASGP-R labeling in FP+ and endosomes. Cells exposed for 40 min to 5-nm gold tracer. Section immunolabeled for ASGP-R with 10-nm gold. (a) Low ASGP-R labeling in tracer-carrying endocytic vesicles (arrowheads), adjacent to (top) or fusing with (bottom) an AVi/d. (b) ASGP-R labeling at the plasma membrane and in endosomal elements near the cell surface (ex, extra cellular space). Vesicles of the MVE (asterisk) and VE (arrowhead) type, containing endocytosed gold, are present in the same area. (c) As b. A VE-like vesicle with endocytosed gold and low ASGP-R labeling (arrowhead) is shown connected to an ASGP-R rich element (arrow), possibly a recycling endosome. Bars, 200 nm. \n",
      "Comparison of structural preservation in cryosections picked up with sucrose (a) and MU (b).Figure 1Comparison of structural preservation in cryosections picked up with sucrose (a) and MU (b). In isolated hepatocytes, autophagy was induced for 30 min in suspension buffer before fixation. (a) Cryosection picked up with 2.3 M sucrose and labeled for SOD. Note the presence of large, \"empty\" gaps between sequestered contents of the AV and the surrounding cytoplasm. (b) A similar section as a, but picked up with an MU mixture. The AV are no longer distorted and the overall membranous structures are sharply delineated. The SOD-labeling density of AVi is as that of the cytosol, whereas AVd show much more intense labeling. (Inset) An AVi/d that is distinct from AVi by its single limiting membrane. The highly pleiomorphic AVd are also bound by a distinctive single membrane, but in contrast to AVi/d, their contents are characterized by debris of digested material. Bars, 250 nm. \n",
      "Rescue of Rb-deficient myotube degeneration by Bcl-2 and autophagy inhibitors.Figure 3.(A) Mean number of myotubes per field in indicated myoblast cultures induced to differentiate after transduction with Ad.GFP or Ad.Bcl-2. Each point represents the mean ± SD of six fields from six independent experiments. Bcl-2-rescued, twitching Rb−/− myotubes are shown in Video 2. (inset) Western blot for Bcl-2 in Ad.Bcl2-transduced versus uninfected primary myoblasts. (B) Immunostaining for MCK (green) in Ad.Bcl-2-transduced control (ctrl) and Rb−/− myotubes at DM-6. (C) Western blot for MCK, myogenin, MHC, and troponin T in indicated cultures at DM-6 after transduction with Ad.Bcl-2. Tubulin was used as a loading control. Black lines indicate that intervening lanes have been spliced out. (D) Mean number of Rb−/− myotubes after 3-MA treatment, bezafibrate, or DMSO/vehicle as indicated. Counts are mean ± SD of six fields (n = 4). (E, left and middle) Brightfield images of the indicated myotubes treated or not with 3-MA at DM-8. (right) Immunostaining for MHC (red) at DM-14 in the presence of 3-MA. Arrowheads label 3-MA-rescued Rb−/− myotubes at DM-8. 3-MA-rescued, twitching Rb−/− myotubes are shown in Video 4. (F) Immunostaining for BrdU (green) and MHC (red) of 3-MA-treated cultures at DM-2 or DM-10. Rb−/− myotubes incorporated BrdU at DM-2 (arrowheads) but not DM-10. (G) Brightfield images (left and middle) and MHC staining (right) of the indicated cultures treated or not with bezafibrate. Arrowheads point to bezafibrate-rescued Rb−/− myotubes. \n",
      "Hypoxia rescues muscle degeneration after chronic or acute inactivation of Rb by inducing glycolysis via HIF-1α.Figure 4.(A) Immunostaining for MHC (green) in the indicated myoblasts induced to differentiate under normoxia or hypoxia treated or not with lonidamine. Note the shortened myotubes in the presence of lonidamine. (B) Representative immunoblot (left) and mean expression of myogenin and MCK normalized for MHC expression (right; n = 3). (C) Rbf/f myoblasts transduced with Ad.GFP or Ad.Cre and immunoblotted for pRb. Tubulin was used as a loading control. (D) Rbf/f myoblasts transduced with Ad.GFP or Ad.Cre and immunostained for pRb (red). (insets) DAPI staining for nuclei (blue). (E) Immunostaining for MHC in Rbf/f myoblasts transduced with Ad.Cre and induced to differentiate in normoxia (top and middle) or hypoxia (bottom). (F, top) Brightfield images of Rbf/f myoblasts transduced with Ad.EV or Ad.Cre and induced to differentiate under hypoxia. (bottom) Larger view of the boxed field. Arrowheads point to rescued Rb−/− myotubes. This field is shown in Video 6. (G) Quantification of myotube formation in Rbf/f myoblasts transduced with Ad.EV or Ad.Cre and induced to differentiate under hypoxia. Counts are mean of six representative fields (n = 3). *, P = 0.004. (H) Quantification of myotube formation in the indicated myoblasts induced to differentiate under the indicated conditions. Myotube counts are the mean of six fields (n = 4). (I) MHC staining (red) of the indicated myoblasts transduced with Ad.NF-κB/HIF-1α and induced to differentiate in normoxia. (J) Quantification of myotube formation in the indicated myoblasts transduced with Ad.EV or Ad.NF-κB/HIF-1α and induced to differentiate in normoxia. Myotube counts are the mean of six fields (n = 3). Error bars represent SD. \n",
      "AV types involved in FP+.Figure 8AV types involved in FP+. The same FP+ data as used in Fig. 6 were classified according to the AV types. \n",
      "Fusion profiles.Figure 5Fusion profiles. Cells exposed to 5-nm gold tracers for 20 (a and c) and 25 min (b) before fixation. Sections were picked up by MU and air-dried (c) or underwent immunolabeling with anti-SOD antibodies (a and b). (a) Fusion between a small VE (arrow) and an AVi. A cluster of two gold particles (arrowhead) is also seen at another location. (b) Fusion between an electron lucent MVE (arrow) and an AVd. (c) Fusion between an MVE (arrow) and an AVd. Bars, 200 nm. \n",
      ": Dll4-ligand expression in the IAHC varies with cluster size (A) Detail images of Dll4 staining in Kit+ cells of IAHC of different sizes.Figure 3Detail images of Dll4 staining in Kit+ cells of IAHC of different sizes. Dll4 expression inversely correlates with the size of the cluster: 2-5 cell IAHCs are uniformly Dll4-positive (upper and middle panel), >5 cell IAHCs are partially Dll4 positive (lower panel). Multistack reconstruction of confocal images. Scale bars: 10 μm. Quantification of Kit+ Dll4+ cells using confocal images. Bars represent the percentage of Dll4 expressing cells within a cluster in each category of cluster size. Mean±SD (total 17 clusters analysed, n=2). Overview of Dll4-ligand expression in the embryonic aorta at E10.5. Dll4 is expressed in the majority of the endothelial layer. Scale bar: 30μm \n",
      "Labeling patterns of cathepsin D and CAIII in MU picked-up cryosections.Figure 2Labeling patterns of cathepsin D and CAIII in MU picked-up cryosections. (a) Cells treated as in Fig. 1 b, but the section was immunolabeled for cathepsin D, which is not detectable in the AVi, moderately present in the AVi/d, and abundant in AVd. Cells in b and c had endocytosed 5-nm gold tracer for 20 and 40 min, respectively, before fixation and CAIII labeling. (b) Complex fusion profile of a CAIII-positive AVi, an AVd which is without CAIII labeling, and a VE that contains an endocytosed gold probe (arrow). Close to the AVd is another tracer-bearing VE. (c) Fusion profile of a tracer-bearing MVE and an AVi/d that exhibits a single limiting membrane and a CAIII-labeling concentration similar to that of the cytoplasm. Bars, 200 nm. \n",
      ": IAHCs have a monoclonal, proliferative origin but become polyclonal at later stage.Figure 2Schematic representation of time of induction of recombination in VeCadCreER:Confetti pregnant females and analysis of embryos. Tamoxifen was injected before cluster initiation (E7.5 or E8.5) and embryos collected at the peak of cluster formation (E10.5 or E11.5). Quantification of recombination efficiency at E10.5 and E11.5 using confocal images. Bars represents the mean±SD of the percentage of cells expressing single colored cassettes in CD31+ endothelial cells (n=2). Transversal section of a recombined VeCadCreER:Confetti AGM from an E10.5 embryo and immunostained with CD31 (C) (white in image, black in label) Multistack reconstruction of confocal images. Scale bars 30μm Transversal section of a recombined VeCadCreER:Confetti AGM from an E10.5 embryo and immunostained with Kit (D) (white in image, black in label). Multistack reconstruction of confocal images. Scale bars 20 μm Schematic representation of possible outcomes in the IAHC after recombination: Clusters derived from sequential division of one colored cell will be unicolored (left panel). Clusters will be multicolored (right panel) when derived from several, independently recombined endothelial cells. Analyses of recombination in 40 clusters. Bars representing single clusters are grouped based on the number of Kit+ cells. Recombination induction follows the schema in the black square. (n=12 embryo). Representative confetti recombination on clusters of different sizes. Small cluster are monoclonal (left panel), while larger clusters are polyclonal (right panel). Inlet shows relevant confetti channel. Kit is shown in white in the image, in black in the label. Multistack reconstruction of confocal images. Scale bars 5 μm. \n",
      ": Bulging cells divide to form the monoclonal core of the IAHC (A) Schematic representation of initial events of IAHC formation.Figure 1Schematic representation of initial events of IAHC formation. (i) A single cell protrudes into the lumen of the aorta and starts expressing CD41. (ii) The CD41:YFP+ bulging cell undergoes mitosis (iii), forming the core of the IAHC. Expression of Kit (magenta) largely overlaps with the expression of CD41:YFP (yellow) in IAHC. Multistack reconstruction of confocal images. Scale bars: 30μm overview, 10μm magnification. (C) Snapshots of Movie EV1. Time-lapse of embryonic organotypic slice culture from CD41:YFP (yellow): H2B-GFP (blue) reporter mouse shows the emergence of a single cell protruding from the endothelial layer into the aortic lumen. Scale bars: 50μm (overview), 10μm (magnification). Time expressed in hh:mm:ss. (D) Snapshots of Movie EV2. Time-lapse of embryonic organotypic slice culture from CD41:YFP (yellow): H2B-GFP (blue) reporter mouse shows a single CD41:YFP expressing cell that divides once on the aortic endothelium. Scale bars: 100μm (overview), 20μm (magnification). Time expressed in hh:mm:ss. BrdU incorporation in the Kit+ IAHC upon cumulative administration. Multistack reconstruction of confocal images. Representative picture of a short BrdU pulse (1h). Scale bars: 50μm (overview), 10μm (magnification). (F) Quantification of the percentage of Kit+ cells within IAHCs incorporating BrdU using confocal images. All cells (Labelling Index=100%) can be labelled after 8h, indicating the lack of quiescent cells inside the IAHCs. Graph represents mean±SE. \n",
      ": changes in gene expression and decrease in Notch activity after Dll4 blockage.Figure 5Experimental design: E10.5 embryos were injected with IgG or αDll4, incubated for 5h and then dissected. The indicated IAHC subpopulations (kit+CD45+; kit+CD45-;kit-CD45-CD41+ were purified and RNA was sequenced. PCA of the different subpopulations from IgG or αDll4 treated embryos based on normalized gene counts. PCA considering the DEG according to the treatment specific for each subpopulation. Function enrichment analysis of DEG in the Kit+CD45- population according to the treatment(E) Experimental design and sorting gate for Kit+CD45- cells for single cell RNAseq (n=483 cells). Experimental design and sorting gate for Kit+CD45- cells for single cell RNAseq (n=483 cells). tSNE distribution of IgG and αDll4-treated Kit+CD45- cells from single cell RNA-seq data. (F) cells corresponding to each treatment are indicated and (G) different clusters identified by ICGS (F) are represented.1-3 are endothelial-like cells and 6-7 are hematopoietic-like cells. Violin plots of Notch family elements expression in IgG and αDll4 treated cells (adj p val: *<0.1 and *** p<0.00001). Percentage of cells in each cell cycle phase extracted from single cell RNA-seq analysis. Bars represent the average±SE (n=483). \n",
      ": Mathematical model supports a role for Dll4 in controlling cell recruitment into IAHCs.Figure 7Schematics of the probabilistic model of IAHC formation. The processes considered in the model are (1) initiation of cluster, (2) cell division event, (3) recruitment event, and (4) cluster detachment. Each process in the model is described as a typical time scale (Tcluster lifetime, Tcell division, Trecruitment). In case of αDll4 treatment, the final five hours of the simulation are assumed to have a different recruitment time. See details of the simulation in the methods. Heat map showing the probability (or p-value) that the simulation produces mean cluster size and fraction of colored cells that match those observed experimentally for the IgG treatment. This probability is calculated for different values of cluster lifetimes and mean recruitment times (values in x and y). 4000 simulations were performed for each set of parameters (each point in the heat map). The most likely parameter values (yellow) correspond to a cluster lifetime of 21 hours and mean recruitment time of 11 hours. Red dashed line indicates the cross-section with the most likely cluster lifetime (21 hours). (C) A graph showing the probability that the simulation produces mean cluster size and fraction of colored cells that match those observed experimentally for the αDll4 treated clusters (blue line). Here, the cluster life time is taken to be 21 hours (taken from b) and the probability to match the experimental data is calculated for different recruitment times in the final 5 hours of the simulation (x-axis). The most likely mean recruitment time for the αDll4 treatment is 5 hours (peak of blue line). For comparison, the most likely mean recruitment time for the IgG treatment is 11 hours (Red dashed line, corresponding to the red dashed in (B)). Hotelling's T-Squared test. Examples of simulations of IAHC distributions for the IgG (left) or αDll4 (right) treated samples. Simulations were performed with the most likely parameter values described in b and c. Note that only clusters that have at least one colored cell are shown (i.e clusters with non-colored cells are not represented). Snapshots of Movie EV5. Time Lapse of embryonic organotypic slice from a CD41:YFP: H2B-GFP reporter mouse showing the recruitment of CD41:YFP negative cells (magenta central point-tracking analyses) migrating towards the IAHC (CD41:YFP+ cells). Green central point and relative tracking identify cells with non-directional movement. Scale bar: 5μm. Time expressed in hh:mm:ss. \n",
      ".Figure 1(A) Schematic representation of BRCA1 and BARD1. Highlighted are RING domains (orange), PALB2-interacting coiled-coil region (CC; red), BRCA1 C-terminal domains (BRCT; green) and ankyrin repeats (ANK; pink). (B) Immunoblotting of whole cell extracts obtained from parental and RAP80-/- clones in the RPE-1 hTERT p53-/- Cas9, RPE-1 hTERT p53-/- BRCA1-/- Cas9, U2OS, U2OS-2-6-3 and U2OS Flp-In/TREx cell lines with RAP80 antibodies. Tubulin is used as loading control. Representative of at least two independent immunoblots. See Figure EV1 for further details on the gene editing strategy used to knockout RAP80. (C, D) The indicated parental and RAP80-/- cell lines were processed for immunofluorescence 1 h post-irradiation (10 Gy) and stained with antibodies against ABRAXAS1 and γH2AX. Shown in (C) is the percentage of cells with > 5 ABRAXAS1 foci that colocalize with γH2AX. A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D. (n=3 biological replicates). Representative micrographs of RPE1-hTERT p53-/- Cas9 wild type (WT) and RAP90-/- cells are shown in (D). (E, The indicated cell lines were processed for immunofluorescence 1 h post-irradiation (10 Gy) and stained with antibodies against BRCA1 and γH2AX. Shown in (E) is the percentage of cells with > 5 BRCA1 foci that colocalize with γH2AX. A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D. (n=3 biological replicates for all conditions except n=5 for U2OS WT, n=2 for U2OS 2-6-3 cells, n=7, 5, 4 for U2OS Flp-In/T-REx cells). F) The indicated cell lines were processed for immunofluorescence 1 h post-irradiation (10 Gy) and stained with antibodies against BRCA1 and γH2AX. Representative microgaphs of RPE-1 hTERT p53-/- Cas9 wild type (WT) and RAP80-/- are shown in (F). (G) RPE-1 hTERT p53-/- Cas9 parental (WT) and RAP80-/- cells were untreated (t=0 h) or irradiated with a 10 Gy dose. Samples were collected at the indicated time points, processed for immunofluorescence and stained with antibodies against BRCA1 and γH2AX. Shown is the percentage of cells with > 5 BRCA1 foci. A minimum of 100 cells per replicate were analyzed and the datapoints represent the mean ± S.D. (n=3 biological replicates). All scale bars are 5 μm. \n",
      ": Blockage of Dll4 increases the number of cells in IAHCs (A)Whole mount images of the aorta endothelium stained for CD31 (white in image, black in label) and Kit (IAHC, cyan) upon IgG (left) or αDll4 antibody treatment (right).Figure 4(A) Whole mount images of the aorta endothelium stained for CD31 (white in image, black in label) and Kit (IAHC, cyan) upon IgG (left) or αDll4 antibody treatment (right). Multistack reconstruction of confocal images. Red arrows indicate IAHCs on the ventral part of the aortic wall. Asterisks indicate aspecific autofluorescence. Scale bar: 50μm. Representative IAHCs of IgG and αDll4 treated embryos. Kit is shown in white in the image, in black in the label. Multistack reconstruction of confocal images. Note the increased IAHC size in the right panel. Scale bars: 50μm (overview), and 10μm (magnification). Frequency of IAHC categories based on the number of Kit+ cells forming a cluster. Quantification of confocal images. Bars represent the mean±SE (n=6) Number of Kit+ cells per IAHC increases following αDll4 antibody treatment. Each dot represents one cluster. Quantification on confocal images. Statistical analysis: Mann-Whitney U test. ****p< 0.0001. Total number of IAHCs per 100μm. Quantification of confocal images. Mean±SE. Statistical analysis: t-test. ns p> 0.05; (n=6). (F) Quantification of the number of Kit+ cells per 100 μm using confocal images. Bars represent the mean±SE. Statistical analysis: t-test. ns p> 0.05; ***p< 0.001 (n=6). \n",
      ": Cell recruitment is increased in Dll4-blocked IAHCs (A) Bar graph represents the frequency of IAHCs containing recombined cells per embryo under the indicated conditions.Figure 6Bar graph represents the frequency of IAHCs containing recombined cells per embryo under the indicated conditions. Quantification of confocal images. Mean±SE (n=15 for untreated, n=7 for IgG and αDll4 treated). Representative images of recombined IAHCs following IgG or αDll4 conditions. Kit is shown in white in the image, in black in the label. Multistack reconstruction of confocal images. Scale bar: 5μm. Quantification and classification of IAHC containing recombined cells in TAM-induced VeCadCreER:Confetti embryos after IgG or αDll4 treatment. Embryos treated with αDll4 contain more IAHC with colored cells. Quantification of confocal images (n=7 embryos). Representative images of recombined endothelial cells in transversal sections of the aorta. The distribution and amount of recombined endothelial cells is similar in IgG and αDll4 conditions. CD31 is shown in white in the image, in black in the label. Multistack reconstruction of confocal images. Scale bar: 50μm. Quantification of recombined cells in the aortic endothelium using confocal images. Mean±SE (n=3). \n",
      ": Interference with the Dll4 ligand functionality results in increased hematopoietic potential in embryonic AGM.Figure 8Schematic representation of functional assays for testing the hematopoietic potential of AGM. Embryos were injected intracardiacally and incubated for five hours with αDll4 or IgG control. Explant cultures were either exposed to the relevant treatment for 5h before FACS analyses or incubated for 48h before transplantation into irradiated mice. Colony-forming assays using E10.5 AGMs derived from embryos treated with IgG or αDll4. FACS analyses of Gfi1+ cells derived from explants treated with IgG or αDll4 for 5 hours (C). Analyses of the hemogenic Gfi1+Kit-(D) and the Gfi1+Kit+ (E) within the CD31+CD45- population. Percentage of reconstitution (chimerism determined by CD45.1 and CD45.2 expression) in peripheral blood (PB). Embryos treated with IgG or αDll4 were kept in explant culture for 48hours and then transplanted into irradiated recipients. Peripheral blood was analyzed at different time points. Each dot/square represents a single transplanted mouse. Analysis of progenitors (lin-sca1+kit+) and HSCs (Lin-sca1+kit+CD48-CD150+) in the donor population in the bone marrow after 4 months of transplantation. Model for recruitment of Gfi1+ cells into the IAHC. Recruitment of Gfi1+endothelial cells increases upon Dll4 blockage. \n",
      ".Figure 6Model of ubiquitin-dependent BRCA1 recruitment to DSB sites. BRCA1 localizes at DSB sites via two redundant ubiquitin (ub)-dependent processes. The first mode is mediated by the ANK-BRCT domain of BARD1, also known as BUDR, which recognizes H4K20me0 nucleosomes modified with H2A-K13/15ub, the product of RNF168. Our work is consistent with a role for the E3 ligase activity (green) for promoting this binding event. The second mode on BRCA1 recruitment is dependent on the BRCT domains of BRCA1 binding to BRCA1-A whose RAP80 subunit recognizes Ly63-linked ub chains (ubK63), most likely on histone H1. \n",
      " - Classification of GaMTs and non-GaMTs based on super-resolution imaging.Figure 1Cartoon description of the segmentation of a polarized cell into 4 quadrants. The red dots indicate the center of the nucleus and the cell leading edge membrane, and the black arrow indicates the cell front-rear direction. The red dashed line box indicates the STORM imaging region. Left: representative conventional image of interphase MT network in the 1st quadrant of a Human Retinal Pigment Epithelium (HRPE) cell. Grey: α-tubulin; Yellow: TGN-46; Yellow dashed line: Golgi boundary. Right: the classified MT subgroups. Red: GaMTs; Green: non-GaMTs. Scale bar: 5 μm. Box-whisker plot presents the proportion of GaMT and non-GaMTs. (data were pooled from 3 independent experiments and n= 36 cells). The ends of whiskers set as the 10% and 90% of the entire population. ***p < 0.001, two-tailed Mann-Whitney. Six representative images of MT networks extracted from MT STORM images of HRPE cells under combinative KDs. Cells were rotated to orient their leading edges locating in the 1st quadrant. Upper: control cell; CLASPs KD cell; GCC185 KD cell. Lower: CLASPs + GCC185 KD cell; CAMSAP2 KD cell; AKAP450 KD cell. Red: GaMT; Green: non-GaMT. Scale bar: 5 μm. Box-whisker plot presents the ratio of GaMT/non-GaMT under combinative KDs. (1 representative of 3 independent experiments and n= 12 cells). The ends of the whiskers are set at 10% and 90% of the entire population. *p < 0.05, **p < 0.01, ***p < 0.001, ns, no significant difference, unpaired t-test. Time series show EB1 movement on individual GaMTs. Red: GaMT; Green: non-GaMT; White: EB1 tracks. Scale bar: 1 μm. Time series show EB1 movement on individual non-GaMTs. Red line: GaMT; Green line: non-GaMT; White line: EB1 tracks. Scale bar: 0.5 μm. EB1 velocity distributions of GaMTs and non-GaMTs. Red curve and green curve represent the Gaussian fittings (data were pooled from 3 independent experiments and n= 92 tracks). ***p <0.001, unpaired t-test. \n",
      ".Figure 3(A, B) U2OS Flp-In/T-REx cells with integrated transgenes encoding GFP-BRCA1BRCT or -BRCA1BRCT S1655A were transfected with non-targeting siRNA (CTRL) or siRNA targeting RAP80 or BRCA1. Following doxycycline treatment to induce transgene expression (5 μg/mL, 24 h), cells were irradiated (10 Gy) and processed for immunofluorescence 1 h post-IR for GFP and antibodies against RAP80 and γH2AX. Shown in (A) is the quantitation of a minimum of 100 cells per replicate where the bars represent the mean ± S.D (n=4). Representative micrographs are shown in (B). (C, D) U2OS Flp-In/T-REx cells stably integrated with the indicated transgenes were treated with doxycycline (5 μg/mL, 36 h) to induce protein expression and transfected with an siRNA targeting BRCA1 and also either non-targeting siRNA (CTRL) or siRNAs targeting RAP80. 1 h post-irradiation (10 Gy) cells were processed for immunofluorescence using GFP (to detect the BRCA1 fusions) and antibodies against γH2AX. Shown in (C) is the quantitation of a minimum of 100 cells per replicate where the bars represent the mean ± S.D (n=3 biological replicates). Representative micrographs are shown in (D). All scale bars are 5 μm. \n",
      ".Figure 2(A, B) U2OS Flp-In/T-Rex parental (WT) and RAP80-/- cells were transfected with siRNA pools targeting BRCA1, RNF8, RNF168 or BARD1 or with a non-targeting control siRNA (CTRL). 48 h post-transfection cells were irradiated (2 Gy) and processed for immunofluorescence 1 h post-IR treatment using antibodies against BRCA1 and γH2AX. Quantitation of the percentage of cells with > 5 BRCA1 foci that colocalize with γH2AX is shown in (A). A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D. (n=3 biological replicates). Representative micrographs are shown in (B). (C, D) Parental (WT) RPE1-hTERT p53-/- Cas9, RNF8-/-, and RNF168-/- cells were treated with either a non-targeting siRNA pool (CTRL) or a pool targeting RAP80. 48 h post-transfection cells were irradiated (2 Gy) and processed for immunofluorescence 1 h post-IR treatment using antibodies against BRCA1 and γH2AX. Quantitation of the percentage of cells with > 10 BRCA1 foci that colocalize with γH2AX is shown in (C). A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D. (n=3 biological replicates). Representative micrographs are shown in (D). All scale bars are 5 μm. \n",
      ".Figure 5(A, B) MDA-MB-436 cells transduced with lentivirus expressing the indicated HA fusions were irradiated (10 Gy) and 3 h later were pulsed with EdU to label S phase cells 20 min prior to fixation. Cells were processed for EdU labeling and immunofluorescence using HA and BRCA1 antibodies. Shown in (A) is the quantitation of a minimum of 100 EdU+ cells per replicate and the bars represent the mean ± S.D (n= 3 biological replicates). Representative micrographs are shown in (B). MDA-MB-436 cells transduced with lentivirus expressing the indicated HA fusions were irradiated (10 Gy) and 3 h later were pulsed with EdU to label S phase cells 20 min prior to fixation. Cells were processed for EdU labeling and immunofluorescence using HA and RAD51 antibodies. Shown in (C) is the quantitation of minimum of 100 EdU+ cells per replicate and the bars represent the mean ± S.D (n= 3 biological replicates). MDA-MB-436 cells transduced with lentivirus expressing the indicated HA fusions were irradiated (10 Gy) and 3 h later were pulsed with EdU to label S phase cells 20 min prior to fixation. Cells were processed for EdU labeling and immunofluorescence using HA and RAD51 antibodies. Representative micrographs are shown in (D). (E) Clonogenic survival assays using the indicated dose of olaparib and MDA-MB-436 cells transduced with lentivirus expressing the indicated HA fusions. Data is shown as the mean ± S.D (n=3 biological replicates). Representative clonogenic images are shown in Figure EV5D. \n",
      ".Figure 4(A, B) U2OS 2-6-3 parental (WT) or RAP80-/- cell lines were transfected with siRNAs targeting BRCA1 followed by nucleofection of the indicated GFP fusion proteins. 48 h post-nucleofection, mCherry-LacR-FokI expression was induced for 5 h prior to being processed for fluorescence microscopy for GFP and mCherry. Shown in (A) is the quantitation of GFP fluorescence at the mCherry focus where the bars represent the mean ± S.D (n= 50, 50, 40, 40, 40, 40, 20, 20, 30, 30 cells analyzed from 3 biological replicates). Representative micrographs for the BRCA1 and BRCA1-I26A conditions are shown in (B). Additional micrographs for the other conditions are in Figure EV2C. (C) RPE-1 hTERT p53-/- BRCA1-/- Cas9 cells (WT) or their isogenic RAP80-/- counterparts expressing the indicated GFP fusion proteins were processed 1 h post-irradiation (10 Gy) for immunofluorescence using GFP and antibodies against BRCA1 and γH2AX. A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D (n=3 biological replicates). Representative micrographs are shown in Figure EV3B. (D) RPE-1 hTERT p53-/- BRCA1-/- Cas9 cells or their isogenic RAP80-/- counterparts expressing the indicated GFP fusion proteins were processed 1 h post-irradiation (10 Gy) for immunofluorescence using GFP and antibodies against RAD51 and γH2AX. A minimum of 100 cells per replicate were analyzed and the bars represent the mean ± S.D (n=3 biological replicates). Representative micrographs are shown in Figure EV3C. Scale bar is 5 μm in (B). \n",
      "-dependent cellular auxin efflux.Figure 1A Normalized firefly luciferase luminescence (compared to renilla luciferase) as a function of external IAA concentration. Arabidopsis protoplast cells were transiently transformed with either AtPIN1 or GFP (in the case of the control) both under the control of a constitutive CaMV 35S promoter. Where indicated, 10 µM NPA was added. B Normalized firefly luciferase luminescence (compared to renilla luciferase) as a function of external NPA concentration in the presence of 100nM IAA in control GFP- (blue or red) or PIN1- (green or purple) transformed protoplasts. \n",
      " - GaMTs are more polarized and contain less intersections A.Figure 4A representative image of MT networks extracted from MT STORM image of an HRPE cell. Red line: GaMT; Green line: non-GaMT. Yellow circle: intersection. Blue circle: intersections formed by GaMTs and non GaMTs. Scale bar: 5 μm. Rose-plot presents the orientation of GaMTs and non-GaMTs. Red: GaMTs; Green: non-GaMTs. Box-whisker plot presents the projections of GaMTs and non-GaMTs to the leading edge direction. (Data were pooled from 3 independent experiments and n= 36 cells). The ends of the whiskers are set at 10% and 90% of the entire population, **p < 0.01, unpaired t-test. Box-whisker plot presents the intersection counts of GaMTs and non-GaMTs. (data are pooled from 3 independent experiments and n= 36 cells). The ends of the whiskers are set at 10% and 90% of the entire population, ***p < 0.001, unpaired t-test. \n",
      " - GaMTs efficiently deliver post-Golgi cargos to the cell leading edge.Figure 6Projections of 120 frame movies present leading edge delivery of two different types of cargos. Left: Ecad cargo; Right: Rab5 cargo. Yellow line mark the cell boundary. Scale bar: 20 μm. Examples of cargo delivery towards the leading edge, time series present at 40 s, 80 s and 120 s, respectively. B: control cell; C: CAMSAP2 KD cell; D: CLASPs&GCC185 KD cell. Red arrow heads indicate cargo current position. Size of cropped regions: 30 μm x 5 μm. Box-whisker plot presents the Ecad cargo number on GaMTs and non-GaMTs. (Data were pooled from 3 independent experiments and n= 36 cells). The ends of the whiskers are set at 10% and 90% of the entire population, **p < 0.01, unpaired t-test. Box-whisker plot presents the number of Ecad cargos finally reached the leading edge membrane during 120 s. (1 representative of 3 independent experiments and n= 24 cells). The ends of the whiskers are set at 10% and 90% of the entire population, **p < 0.01, two-tailed Mann-Whitney. Box-whisker plot shows the number of Rab5 cargos that finally reached the leading edge membrane during 120 s. (1 representative of 3 independent experiments and n= 23 cells). The ends of the whiskers are set at 10% and 90% of the entire population, ns, no significant difference, two-tailed Mann-Whitney. \n",
      " - Cargo velocity on GaMTs and non-GaMTs.Figure 2Representative trajectories of E-cad cargos tracked over 120 s in an HRPE cell. Colors represent different average velocity ranges. Red: > 400 nm/s; Purple: 200~400 nm/s; Green: 100~200 nm/s; Yellow: 0~100 nm/s. Scale bar: 20 μm. Total number of cargos within different velocity ranges in the 4 quadrants, respectively (Data were pooled from 3 independent experiments and n= 45 cells). Proportions of cargos with different velocity in the 4 quadrants. Red: the 1st quadrant; Dark grey: the 2nd quadrant; Light grey: the 3rd quadrant; White: the 4th quadrant (Data were pooled from 3 independent experiments and n= 45 cells). Flow chart shows the method of combining STORM imaging with single particle tracking in live cells. Red lines indicate the cargo trajectories. Lower right, MT networks (Grey) were extracted and present in purple. Representative time series show one fast cargo moving on a GaMT and a slow cargo moving on a non-GaMT. Upper, conventional imaging; lower, STORM imaging. Red line: GaMT; Green line: non-GaMT; White circle: cargo position. Scale bar: 2 μm. Marginal distributions show the velocity on GaMT (red dots) or non-GaMTs (green dots). Red curve and green curve, Gaussian fitting curves of velocity distribution on GaMT or non-GaMT, respectively. (Data were pooled from 3 independent experiments and n= 69 MTs). Peak of red curve (431.14 ± 9.7 nm/s) and peak of purple curve (188.96 ± 22.03 nm/s). ***p < 0.001, unpaired t-test. Upper panel: time series show an example of several fast cargos present on GaMT. Lower panel: time series show one slow cargo on non-GaMT switching onto a GaMT and becoming faster. Red line: GaMT; Green line: non-GaMT; White circle: cargo position. White line, cargo trajectory. Scale bar: 2 μm. Box-whisker plot presents the averaged acetylation level of fast or slow MT tracks. (Data were pooled from 2 independent experiments and n= 63 MTs). The ends of the whiskers are set at 10% and 90% of the entire population, ns, no significant difference, unpaired t-test. Marginal distributions of the mean velocity of cargos on segments of Ac-MTs (orange dots) and non-Ac-MTs (purple dots). Orange curve and purple curve represent the Gaussian fittings (Data were pooled from 2 independent experiments and n= 63 MTs). ns, no significant difference, unpaired t-test. \n",
      " core complex.Figure 2A, B Native PAGE separation reveals distinct detergent-sensitive PIN1-complexes. Plasma membrane preparations from dark grown Arabidopsis cell suspension cultures were solubilized with (a) Complexiolyte 27 (a mixture of ionic and non-ionic detergents), and (b) Complexiolyte 47 (lower stringency, non-ionic detergent), (both Logopharm); first dimension BN-PAGE, second dimension SDS-PAGE, blots stained with anti-PIN1 antibody revealing distinct PIN1 complex populations at the indicated positions. Values are given in KDa. c) Molecular abundance (abundancenormspec values calculated as summarized peptide PVs divided by the number of MS-accessible protein specific amino acids) of PIN subunits and ABCB19 in anti-GFP affinity purification of CL47-solubilized PIN1-GFP-expressing roots with a GFP-specific DARPin. Note the significant heteromerization of several endogenous PINs with PIN1-GFP and the absence of other abundant interaction partners (ABC19B shown as an example). \n",
      " - GaMTs are required for persistent cell migration but not for cell to change directions.Figure 5Random migration of HRPE cells during 24 h. Upper: control cell; Middle: CAMSAP2 KD cell; Lower: CLASPs&GCC185 KD cell. Red arrow and white dashed lines indicate the migration direction. Different migration trajectories are distinguished with different colors. Scale bar: 50 μm. Rose-plot presents the representative directionality of an HRPE cell. (B) control cell; (C) CAMSAP2 KD cell; (D) CLASPs&GCC185 KD cell. Box-whisker plot presents the migration persistence of control cell, CAMSAP2 KD cell and CLASPs&GCC185 KD cell. (1 representative of 3 independent experiments and n= 34 cells). The ends of the whiskers are set at 10% and 90% of the entire population, ***p < 0.001, unpaired t-test. Controlled cell migration of 3 representative cells in PA-Rac1 assay. White spot: 440 nm laser spot; Green area: the original position of the cell; Red area: the current position of the cell; Yellow area: the overlapped area. Scale bar: 10 μm. Cartoon presents a controlled migrating HRPE cell with Golgi, cell, nucleus. Black arrow with red dot: laser direction; Black arrow with orange dot: cell body direction; Black arrow with blue dot: nuclear direction; Black arrow and green dot: Golgi direction. Angle change curve of cell, nucleus and Golgi. Red-dashed line: laser angle; Blue line: nucleus angle; Orange line: cell angle; Green line: Golgi angle. H: control cell; I: CAMSAP2 KD cell; J: CLASPs&GCC185 KD cell. Box-whisker plot shows the total time required for the nucleus, Golgi and cell body reaching the given laser angle. Blue: time required for nucleus; Green: time required for Golgi; Orange: time required for cell body. (1 representative of 3 independent experiments and n= 14 cells). The ends of the whiskers are set at 10% and 90% of the entire population, ***p < 0.001, ns, no significant difference, unpaired t-test. \n",
      " - non-GaMT harbors more MT repair sites where cargos frequently pause and reverse.Figure 3Five representative images of MT repair sites in HRPE cells under combinative KDs. Cells were rotated to orient their leading edges locating in the 1st quadrant. From left to right, control cell, CLASPs&GCC185 KD cell, CAMSAP2 KD cell, AKAP450 KD cell and Centrinone-B treated cell. Grey: α-tubulin; red: GTP-tubulin. Scale bar: 20 μm. Box-whisker plot presents intensity analysis of MT repair sites in HRPE cells under combinative KDs. (1 representative of 3 independent experiments and n= 20 cells). ***p < 0.001, unpaired t-test. Conventional image (left) and STORM image (right) of MTs and MT repair sites. Grey: α-tubulin; yellow: GTP-tubulin; green: Golgi. Scale bar: 5 μm. MT repair sites on GaMTs and non-GaMTs extracted from C. Red: GaMTs; Green: non-GaMTs; yellow: GTP-tubulin. MT repair sites on GaMTs and non-GaMTs separately presented in detail. White arrows indicates GTP-tubulin labeling sites. Scale bar: 5 μm. Box-whisker plot presents the ratio of MT repair sites on GaMTs and non-GaMTs separately. (1 representative of 3 independent experiments and n= 8 cells). ***p < 0.001, unpaired t-test. Representative time series show one fast cargo moving on a GaMT and slowing down at the MT repaire site. Red: GaMT; Green: non-GaMT; Yellow: GTP-tubulin; White circle: cargo position. Scale bar: 2 μm. Representative time series show one cargo moving fast between two MT repair sites and slowing down at the MTrepair sites. Red: GaMT; Green: non-GaMT; Yellow: GTP-tubulin; White circle: cargo position. Scale bar: 2 μm. Pause (I) and reverse (J) events of cargos on MT repair segment and non-MTs repair segment, respectively. (Data were pooled from 2 independent experiments and n= 22 cells). ***p <0.001, ns, no significant difference, unpaired t-test. \n",
      ".Figure 3The cartoon represents a regenerating trunk and the squares display the region analyzed at the anterior \"A\" and posterior \"P\" blastemas. The images are TUNEL maximum projections of representative blastemas from 72 hours and 4 days regenerating trunks in starved conditions. The dashed line delimits the body of the planarian. The graph shows the number of TUNEL+ cells per mm2. The differences in cell death are not significant (n.s.) at 72hR (two-tailed Student's test with equal sample variance) and significant (***P < 0.001 using two-tailed Student's test with equal variance) at 4dR. n ≥ 5 planarians per condition. Percentage of stem cells in different mitotic phases and the percentage of defective mitotic figures in 72hR anterior blastemas of cct3A RNAi and controls in starved conditions after double immunostaining with anti-tyrosine-tubulin and anti-H3P (****P < 0.0001, ***P < 0.001, *P < 0.05 using two-sided Chi-square test; the number of cells analyzed is displayed in Appendix Fig S2B); n ≥ 5 planarians. Representative images are shown. Nuclei are stained with DAPI. Arrows indicate abnormal organization or number of spindle poles and the yellow arrowhead indicates chromosome lagging. Asymmetrical karyokinesis is observed in b11 and b12. \n",
      "-PIN4 interaction in vivo a) Summed 2D projections of PLA interactions (left) and co-localisation (right) (n=3) between PIN1 and PIN4 laterally bisected 2D-projected heat map.Figure 4a) Summed 2D projections of PLA interactions (left) and co-localisation (right) (n=3) between PIN1 and PIN4 laterally bisected 2D-projected heat map. Treatments as indicated. b) Quantification of positive PLA interactions. n.t (no treatment) and 10µM NPA, n=4; 10µM IAA and 10µM IAA+ 10µM NPA; n=3. Error bars indicate standard deviation. \n",
      " complex formation.Figure 3a) BN-PAGE was performed with PIN1-containing microsomes prepared from either PIN1-expressing HEK293T cells or a dark-grown Arabidopsis cell suspension culture. Prior to solubilization with 1% dodecyl maltoside, microsomes were incubated with either 10 µM NPA or 10 µM quercetin. b) PIN1 dimer stability induced by NPA after expression in HEK cells. Relative distribution between monomer and dimer after NPA treatment is given relative to distribution of untreated samples after solubilization with 50% (v/v) CL27. Each measurement given (three for each concentration) represents the mean of three gel lanes for wild type (circles) or triple S2523, S253E, S261E phosphomimetic sequences (crosses) (example images are given in the figure inset). \n",
      ".Figure 1The expression heatmap shows that all the somatic cct subunits from TRiC are up-regulated in X1 (stem cells) compared to Xins (differentiated cells) at 30dS. The colour-coded scale indicates Row Z-score of normalized TPMs (transcripts per million) values per replicate. The asterisks indicate that these genes are significant differentially expressed (q-value (FDR) < 0.01). RNAi injections schedule in starved conditions. Planarians are at 14dS when the amputation is performed (indicated by the grey cross at day 14). Live images show that cct3A(RNAi) planarians form a minimal blastema compared to controls at the time points shown. At the bottom are the number of planarians with the phenotype shown. The remaining planarians are dead by the time point of regeneration shown. dR indicates days of regeneration. RNAi injections schedule in feeding conditions. One extra feeding in respect to (B) is introduced one day prior to injections. Planarians are at 8dS when the amputation is performed. The live images show that most of the cct3A(RNAi) planarians regenerate like controls. At the bottom are the number of planarians with the phenotype shown. At 8-10dR, the remaining planarians are either dead (4/124) or have a tiny blastema (23/124) and either died or regenerated later. At 51-68dR, the remaining planarians are either dead (10/93) or have a tiny blastema and died later (5/93). \n",
      ".Figure 2The graph shows the mitotic numbers of trunks during different time points of regeneration in the starved condition for cct3A RNAi and controls. Error bars are s.d. from the mean and asterisks indicate P < 0.001 (three asterisks), P < 0.01 (two asterisks) and n.s. indicates not significant using two-tailed Student's test with equal variance. n ≥ 9 planarians per time point. Also shown are maximum projections of representative trunks labelled with anti-H3P at different time points of regeneration in the starved condition. On the bottom, the number of planarians with the phenotype shown from the total is displayed. Note that at 50dR planarians show few (a4) or no mitoses (a5). The dashed line delimits the body of the planarian. The graph shows the mitotic numbers during 72 hours of regeneration in feeding conditions for cct3A RNAi and controls. Error bars are s.d. from the mean and n.s. indicates not significant using two-tailed Student's test with equal sample variance; n ≥ 8 planarian per time point. Maximum projections of representative trunks after FISH for smedwi-1 at different time points of regeneration under starved and feeding conditions. On the bottom, the number of planarians with the phenotype shown from the total is displayed. At 15dR the expression levels are similar. At 30dR some planarians have almost no expression (c5) and at 50dR almost all planarians show no expression (c10 and c11) in starved conditions. Under the feeding condition the expression levels at 30dS are similar. On the bottom, the number of planarians with the phenotype shown from the total is displayed. The dashed line delimits the body of the planarian. Relative expression of smedwi-1 at 13 and 50 days of regeneration after either cct3A or gfp RNAi under feeding and starved conditions. Error bars are s.d. from the mean and asterisks indicate P < 0.001 (three asterisks), P < 0.05 (one asterisk) and n.s. indicates not significant using two-tailed Student's test with equal variance. n = 3 replicates (5 planarians each) per time point. \n",
      " dimerization.Figure 5a) Microsomes from HEK293 cells expressing PIN1-RFP were separated under native conditions in the presence of NPA or an anti-PIN1 Fab fragment. 1) PIN1-RFP, 2) PIN1-RFP+Fab, 3) PIN1-RFP+10 µM NPA, 4) PIN1-RFP+Fab+10 µM NPA, 5) PIN1-RFP + 10 µM Quercetin (Q), 6) PIN1-RFP+Fab+10 µM Quercetin (Q). Protein complexes were solubilized and separated under native conditions as described before. Western blots were performed with an anti-RFP monoclonal antibody. b) Fourteen-day-old Arabidopsis seedlings (wild type, brown; pPIN1::Mab9B2; blue) grown on AM containing 0.2 µM NPA. Scale bar = 1 cm. c) NPA-affected lateral root density of nine-day-old Arabidopsis seedlings expressing Mab9B2 scFv fragments (wild type sample sizes were between 16 and 21 plants; Mab9B2 sample sizes were between 12 and 31 plants). Bars indicate standard error. d) NPA-affected apical hook angle in dark-grown three-day-old Arabidopsis seedlings expressing Mab9B2 scFv fragments. (wild type sample sizes were between 38 and 50 plants; Mab9B2 sample sizes were between 33 and 45 plants). e) Gravitropic curvature in four-day old Arabidopsis seedlings expressing Mab9B2 scFv fragments. (wild type sample sizes were between 15 and 21 plants; Mab9B2 sample sizes were between 14 and 20 plants). \n",
      ".Figure 5The Kaplan-Meier curve demonstrates increased survival of cct3A RNAi animals when treated with DTT under starved conditions. Percentages indicate the number of survivals at the indicated time points. Three asterisks indicate p < 0.001 with log-rank (Mantel-Cox) test. Images are representative surviving animals for the different conditions at the displayed time point of regeneration. At the bottom are the number of planarians with the phenotype shown. The rest of planarians are dead at the displayed time point. \n",
      ".Figure 6Volcano plots displaying shRNA-Cct3- and shRNA-luciferase-infected LSK cells cultured either in normal glucose (upper graph) or low glucose conditions (lower graph). Dots represent all the NanoString analyzed transcripts. Red dots are UPR and ER stress related genes. The UPR and ER stress genes significantly down-regulated only under low glucose conditions in shCct3 cells compared to controls are indicated near the lower graph. For better visualization two dots with q ≤ 0.001 have been removed from the volcano plot (see them in Dataset EV4). Y-axis indicates the false discovery rate (FDR) (q- value), and the X-axis indicates the log2 fold changes. The FDR-based method of p-value adjustment was conducted to calculate the q-values by nSolver software using Benjamini-Hochberg methods. Significance is established by q-value < 0.05 and indicated by the dotted horizontal line. n= 3 mice. ;UPR and ER stress\" genes are significantly enriched (only down-regulated) in shCct3 cells compared to controls under low glucose conditions (**P < 0.01 using two-sided Chi-square test). The graph shows that the ratio between Xpb1(spliced) / Xpb1(unspliced) is lower (i.e., lower levels of UPR) in LSK which have Cct-3 down-regulated compared to controls only when cultured under low glucose conditions. Error bars are s.d. from the mean. P < 0.001 (three asterisks) and n.s. indicates not significant using two-tailed Student's test with equal sample variance. n = 3 mice. \n",
      " MEK and ERK are required for tight junction formation and function.Figure 216HBE cells were seeded sparsely, treated with DMSO, GSK1120212 (500nM), PD0325901 (500nM), or SCH772984 (1M), and incubated for 3 days. All data are representative of 3 independent experiments. (a) Cell lysates were analysed by western blotting for pERK, total ERK, p-p90RSK or total RSK1. 16HBE cells were seeded sparsely, treated with DMSO, GSK1120212 (500nM), PD0325901 (500nM), or SCH772984 (1M), and incubated for 3 days. All data are representative of 3 independent experiments. (b) Cells were fixed and stained for ZO-1 and DNA. Scale bar represents 20m. 16HBE cells were seeded sparsely, treated with DMSO, GSK1120212 (500nM), PD0325901 (500nM), or SCH772984 (1M), and incubated for 3 days. All data are representative of 3 independent experiments. (c) Quantification of tight junction phenotype. >500 cells were counted per sample/experiment, across n=3 independent experiments (dots indicate individual data points). Error bars denote mean ± SEM. ***, p = 0.0002; **** p < 0.0001. (d) 16HBE cells were transfected with control siRNA or siSOS1, and incubated for 4 days. Cell lysates were analysed by western blotting for SOS1, pERK and total ERK. (e) 16HBE cells were seeded on transwell filters with DMSO, GSK1120212 (500nM), or PD0325901 (500nM). Transepithelial resistance (TER) was measured on day 4 and expressed as % of DMSO control. n=3 independent experiments (dots indicate individual data points), error bars denote mean ± SEM. ***, p = 0.0002; **** p < 0.0001. (f) 16HBE cells were seeded on glass-bottomed dishes with DMSO, GSK1120212 (500nM), or PD0325901 (500nM). On day 4, FM4-64 dye was applied to the media and confocal z-stacks acquired. Representative images are shown, the red dotted lines indicate the basal surface. (g) 16HBE cells were fixed and stained for MEK or pERK. Scale bar represents 20m. \n",
      ".Figure 7ATP measurement on 48hR trunk extracts shows that cct3A RNAi under starved conditions contains less ATP than controls. Error bars are s.d. from the mean and asterisks indicate P < 0.05 (one asterisk) and n.s. indicates not significant using two-tailed Student's test with equal variance. n = 3 replicates (5 planarians per replicate). The Kaplan-Meier curve demonstrates increased survival of starved cct3A(RNAi) animals when treated with palmitic acid/palmitoylcarnitine (PA/PC). Percentages indicate the number of survivals at the indicated time points. Three asterisks indicate P < 0.001 with log-rank (Mantel-Cox) test. Images are representative surviving animals for the different conditions in (B). At the bottom are the number of planarians with the phenotype shown. The rest of planarians are dead at this point. Scale bars indicate 500 µm. In our proposed model the levels of ER stress (UPR) are basal during regeneration in feeding conditions at 72hR and not essential for stem cell proliferation and regeneration. Planarians obtain energy and lipids from administered food, including exogenous fatty acids (exo FA). During starved conditions, ccts increase the levels of the UPR and this allows proper stem cell proliferation and regeneration when exogenous fatty acids are not available. \n",
      " Ras, MEK and ERK control bronchial epithelial gene expression.Figure 3(a) Acute versus chronic MEK inhibition. Acute (panels 1 and 2): Cells were seeded sparsely and incubated for 4 days in normal media, then subjected to a calcium switch and recovery, in the presence of DMSO (panel 1) or 500nM GSK1120212 (panel 2). Chronic (panels 3 and 4): Cells were seeded sparsely and incubated for 4 days in DMSO (panel 3) or 500nM GSK1120212 (panel 4). Cells were subjected to a calcium switch and recovery, in the presence of DMSO (panel 3) or 500nM GSK1120212 (panel 4). Cells were fixed and stained for ZO-1 and DNA. Scale bar represents 20m. (b-d) Microarray analysis. 16HBE cells were stably infected with pQCXIP (control) or pQCXIP expressing DN HRas. Control cells were treated with DMSO, GSK1120212 (500nM), or SCH772984 (1µM) for 4 days. RNA was isolated and analysed using an Illumina gene array; n=3 independent samples were prepared for each condition. (b) Venn diagram representing genes downregulated by >1.6-fold versus control, with an unadjusted p-value <0.05. (c) Relative expression levels of 33 genes downregulated by DN HRas, MEK and ERK inhibition, expressed as % of DMSO control. (d) Relative expression levels of EMP1. Error bars denote mean ± SEM, dots indicate individual data points. **, p < 0.002 (DN Ras = 0.0019, ERKi = 0.0016); ****, p <0.0001. \n",
      " and Ras are required for bronchial tight junction formation.Figure 116HBE cells were stably infected with pSUPER, pSUPER expressing shCdc42, shARHGEF18, shSOS1 hairpins 1-3, All data are representative of 3 independent experiments. (a) Cells were seeded sparsely on glass coverslips, incubated for 3 days, then fixed and stained for ZO-1 (tight junctions) and DNA (nuclei). Scale bar represents 20m. 16HBE cells were stably infected with pSUPER, pSUPER expressing shCdc42, shARHGEF18, shSOS1 hairpins 1-3. All data are representative of 3 independent experiments. (b) Cell lysates were analysed by western blotting for SOS1 and actin; labels as for c. 16HBE cells were stably infected with pSUPER, pSUPER expressing shCdc42, shARHGEF18, shSOS1 hairpins 1-3. All data are representative of 3 independent experiments. (c) Quantification of tight junction phenotype. >500 cells were counted per sample/experiment, across n=3 independent experiments (dots indicate individual data points). Error bars denote mean ± SEM. *, p = 0.0199; **, p = 0.0041; **** p = 0.0001. 16HBE cells were stably infected pQCXIP, or pQCXIP expressing dominant negative myc-HRas N17. All data are representative of 3 independent experiments. (d) Cell lysates were analysed by western blotting for Ras, pERK and total ERK. 16HBE cells were stably infected with pQCXIP, or pQCXIP expressing dominant negative myc-HRas N17. All data are representative of 3 independent experiments. (e) Cells were seeded sparsely on glass coverslips, incubated for 3 days and then fixed and stained for ZO-1 (tight junctions) and DNA. Scale bar, 20μm. (f) Quantification of tight junction phenotype (see c). ***, p = 0.0002. \n",
      ".Figure 4Volcano plot displaying DEGs in starved cct3A(RNAi) planarians compared to controls at 72hR (q-value < 0.05). Y-axis indicates the negative log10 of the false discovery rate (FDR) (q- value). X-axis indicates the beta values (b), a biased estimator of the fold change. For better visualization, 13 dots with -log10 (q-value) > 20 are not displayed. This includes cct3A with q = 0. Notice that when down-regulating cct3A in starved planarians, xbp1, atf6, bip-1 and a repertoire of ER chaperones are down-regulated, whereas dnajb9, a repressor of the UPR, and other ccts are up-regulated. Relative expression of xbp1 and atf6 related to gfp control at 72 hours and 13 days of regeneration during either starving or feeding conditions in cct3A(RNAi) animals. The graph shows that cct3A RNAi down-regulates xbp1 and atf6 specifically during starvation at 72hR. n = 3 replicates (5 planarians each) per time point. The graph shows that both cct3A RNAi down-regulates xbp1, atf6 and bip-1 specifically at 72hR during starvation in X1 stem cells. 50 planarians per replicate (3 replicates) were used to obtain the X1 population (stem cells in S and G2/M). Live images show that xbp1/atf6(RNAi) planarians in starved conditions form a minimal blastema when compared to controls while at the feeding condition most of them can regenerate as controls. The remaining planarians in the starved condition are dead by the time point of regeneration shown. In the feeding condition, the remaining planarians at 8-10dR are either dead (13/145) or have a tiny blastema and died or regenerated later (40/145) while at 21-27dR are dead. Mitotic numbers during different time points of regeneration in starved conditions for xbp1/atf6 RNAi and controls. n ≥ 7 planarians per time point. Mitotic numbers in feeding conditions for xbp1/atf6 RNAi and controls. n ≥ 9 planarians per time point. Maximum projections of representative regenerating trunks after FISH for smedwi-1 in starved conditions. The dashed line delimits the body of the planarian. Relative expression of smedwi-1 related to controls at 13 and 20 days of regeneration of xpb1/atf6 RNAi in starved conditions. n = 3 replicates (5 planarians each) per time point. TUNEL maximum projections of representative blastemas from regenerating trunks in starved conditions. The dashed line delimits the body of the planarian. The graph shows the number of TUNEL+ cells per mm2 at the same time points in anterior (A) and posterior (P) blastemas. n ≥ 6 planarians per time point. Quantification of the percentage of stem cells in different mitotic phases and the percentage of defective mitotic figures in 72hR anterior blastemas of xbp1/atf6 RNAi and controls in starved conditions after double immunostaining with anti-tyrosine-tubulin and anti-H3P (****P < 0.0001, ***P < 0.001, *P < 0.05, n.s. indicates not significant using two-sided Chi-square test); n ≥ 5 planarians. Representative images are shown. Arrows indicate abnormal organization or number of spindle poles. j8 displays asymmetrical distribution of chromosome content. \n",
      ".Figure 4RNA sequencing (RNA-seq) of week 15 COs and transcriptome analysis (WT: n=2, KO: n=4).   Heatmap of 100 upregulated genes (left panel) and 100 downregulated genes (right panel) selected by highest fold change.   RNA sequencing (RNA-seq) of week 15 COs and transcriptome analysis (WT: n=2, KO: n=4). Gene-set enrichment analysis (GSEA) revealed enrichment of genes related to regionalization of the organoids, and decreased expression of genes related to negative regulation of the cell cycle. RNA sequencing (RNA-seq) of week 15 COs and transcriptome analysis (WT: n=2, KO: n=4). Heatmap of Wnt-pathway related genes in 15-weeks COs. RNA sequencing (RNA-seq) of week 15 COs and transcriptome analysis (WT: n=2, KO: n=4). qPCR analysis for selected Wnt target genes validating the results of the RNA- seq. Y-axis indicates relative expression fold change. Data are represented as mean ± SEM. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=4 from 1 batch, KO: n=4 from 1 batch, W-AAV: n=4 from 1 batch). Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001. Week 16 COs were sub-fractionated into a cytoplasmic (C) and nuclear (N) fractions. The experiment was run twice with a total of 2 WT organoids and 4 KOs organoids (2 for each KO line). KAP-1 marks the nucleus and HSP90 marks the cytoplasm. The numbers at the bottom are a quantification of the nuclear fraction of β-catenin band intensities normalized to the cytosolic fraction [β-C (N/C)]. Heatmap showing the expression levels of markers of the six different layers of the human cortex in week 15 organoids from deepest to the most superficial: TBR1, BCL11B (CTIP2), SATB2, POU3F2 (BRN2), CUX1, RELN. IF staining in week 15 COs validating the decreased levels of the deep-layer cortical markers CTIP2 (BCL11B) and TBR1, and superficial-layer marker SATB2 (WT: n=3, KO: n=4, W-AAV: n=4). Scale = 50µm.   Quantification of the cortical markers seen in G, normalized to the total number of nuclei. Y-axis indicates the fold change compared to the average of the WT COs. The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=9 from 3 batches, KO: n=16 from 3 batches, W-AAV: n=4 organoids from 1 batch).   Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001. \n",
      " in human airway basal progenitor-like cells and lung cancer.Figure 5(a-d) BCi-NS1.1 cells were stably infected with pLKO control or shEMP1.1. Data are representative of n=3 independent experiments. (a) Total RNA was isolated and analysed for EMP1 expression using Taqman/qPCR with a GAPDH control. Error bars denote mean ± SEM, ***, p = 0.0003. (a-d) BCi-NS1.1 cells were stably infected with pLKO control or shEMP1.1. Data are representative of n=3 independent experiments. (b) Filter grown cells were fixed and stained for ZO-1 and DNA and analyzed by confocal imaging. (a-d) BCi-NS1.1 cells were stably infected with pLKO control or shEMP1.1. Data are representative of n=3 independent experiments. (c) Quantification of tight junction phenotype. >500 cells were counted per sample/experiment, across n=3 independent experiments (dots indicate individual data points). Error bars denote mean ± SEM. *, p = 0.0276. (a-d) BCi-NS1.1 cells were stably infected with pLKO control or shEMP1.1. Data are representative of n=3 independent experiments. (d) Cells were cultured on filters under ALI conditions and transepithelial resistance (TER) was measured on day 21 post-seeding. Error bars denote mean ± SEM. ****, p < 0.0001. (e) EMP1 mRNA expression in normal lung versus cancer tissue. The Hou Lung dataset was analyzed using Oncomine software and the 213895_at reporter [45]. Box whisker plots are presented. The points represent the minimum/maximum values, bars denote the 10th-90th percentiles, boxes show the 25th-75th percentiles and the horizontal line marks the median. The number of samples analysed (n), the fold-change (bold) and the p-value are indicated. \n",
      ".Figure 2Sample recordings from 7-week-old hESC-derived Cerebral Organoids (COs). Sample traces show visible differences in local field potential, with WWOX-KO COs (Red) showing increased activity compared to WT (Blue) in baseline condition (Left) and in the presence of 100µM 4AP (Right). Sample recordings from 7-week-old hESC-derived Cerebral Organoids (COs). Mean spectral power of week 7 WT and KO COs in baseline conditions. The line marks 0.25-1 Hz frequency range. Statistical significance was determined using two-tailed unpaired Welch's t-test (WT: n=14 slices, 5 organoids, 3 batches. KO: n=14 slices, 8 organoids, 3 batches).   Normalized area under the curve of the mean spectral power in (B) for the 0.25-1 Hz frequency range. Data represented by mean ± SEM. Statistical significance was determined using two-tailed unpaired Welch's t-test. The numerals in all bars indicate the number of analyzed slices and organoids (i.e. slices (organoids)).   CFC analysis shows increased coupling in the δ: HFO frequency pairs. The colour scale represents the modulation index - a measure of the coupling between the phase of low-frequency oscillations and amplitude of high-frequency oscillations in a given signal. Sample spectrogram of WWOX KO slice shows a gradual increase in activity upon addition of 4AP (marked by red arrow) for up to 2 min, and a decrease after 25 min. All traces were filtered with a 60 Hz notch filter and 0.5 Hz high-pass filter. Sample recordings from 7-week-old hESC-derived Cerebral Organoids (COs). WWOX's coding sequence was re-introduced into week 6 WWOX-KO COs using lentiviral transduction (lenti-WWOX) (F) Immunofluorescent staining showing WWOX expression in different populations in WWOX-KO organoids following infection with lentivirus. NT= non-treated. Scale = 50µm . Sample recordings from 7-week-old hESC-derived Cerebral Organoids (COs). WWOX's coding sequence was re-introduced into week 6 WWOX-KO COs using lentiviral transduction (lenti-WWOX) (G) Normalized area under the curve of the mean spectral power of WT line, 2 KO lines and 2 KO lines infected with lenti-WWOX at week 7 in baseline condition, for the 0.25-1 Hz frequency range. Data represented by mean ± SEM. The numerals in all bars indicate the number of analyzed slices and organoids (i.e. slices (organoids)). Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test. Data information: ***p ≤ 0.001, ****p ≤ 0.0001. \n",
      ".Figure 1Week 10 cerebral organoids (COs) stained for the progenitor marker SOX2, neuronal marker β3-Tubulin and WWOX (WT: n=8 from 3 batches, KO: n=8 from 3 batches, W-AAV n=4 from 1 batch). Scale = 100µm (left), 75µm (right). Week 10 COs stained for the ventricular radial glia (vRGs) marker CRYAB, together with WWOX and the pan-radial glia marker SOX2. The images on the right are enlargements of the boxed area. The dashed line defines the ventricular space. Scale= 50µm (left), 25µm (right) (WT: n=8 from 3 batches, KO: n=8 from 3 batches). Quantification of markers that represent the different populations that compose the ventricular-like zone (VZ), sub-ventricular zone (SVZ) and the cortical plate (CP), as depicted in Appendix Figure S2B-C. NeuN marks mature neurons, TBR2 marks intermediate progenitors (IPs) and SOX2 the ventricular radial glia (vRGs). The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=8 from 3 batches, KO: n=8 from 3 batches, W-AAV n=4 from 1 batch). qPCR analysis for the assessment of expression levels of different neural markers in 15 weeks COs: SOX2 and PAX6 (progenitor cells), TUBB3 (pan-neuronal) SLC17A6 and SLC17A7 (VGLUT2 and VGLUT1; glutamatergic neurons) and GAD1 and GAD2 (GAD67 and GAD65; GABAergic neurons). Y-axis indicates relative expression fold change. Data are represented as mean ± SEM. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=4 from 1 batch, KO: n=4 from 1 batch, W-AAV: n=4 from 1 batch). Immunofluorescent (IF) staining for the glutamatergic neurons marker VGLUT1 and GABAergic neurons marker GAD67 (GAD1) in week 10 COs (WT: n=8 from 3 batches, KO: n=8 from 3 batches, W-AAV: n=4 from 1 batch). Scale = 100µm.   Quantification of the images represented in figure 1E. VGLUT1 and GAD67 were quantified as the surface area covered by the staining and were normalized to the number of nuclei in each image. The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. As the samples were not normally distributed, statistical significance was determined using Kruskal-Wallis test with Dunn's multiple comparisons test. Outliers were removed using the ROUT test (Q=1%). Data information: n.s (non-significant), *p ≤ 0.05, ****p ≤ 0.0001.   \n",
      ".Figure 3Week 15 and week 24 COs stained for the astrocytic and radial glia marker GFAP, and the astrocyte-specific marker S100β (WT W15: n=9 from 3 batches, KO W15: n=16 from 3 batches, W-AAV W15: n=4 organoids from 1 batch. WT W24: n=10 from 4 batches, KO W24: n=9 from 3 batches, W-AAV W24: n=4 from 1 batch). Scale = 100µm qPCR analysis of astrocytic markers in COs at week 15. Y-axis indicates relative expression fold change. Data are represented as mean ± SEM. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=4 from 1 batch, KO: n=4 from 1 batch, W-AAV: n=4 from 1 batch)   qPCR analysis of astrocytic markers in COs at week 24. Y-axis indicates relative expression fold change. Data are represented as mean ± SEM. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=4 from 2 batches, KO: n=3 from 2 batches, W-AAV n=3 from 1 batch).   Data information: n.s (non-significant), *p ≤ 0.05, **p ≤ 0.01, ****p ≤ 0.0001. IF staining of week 6 COs for astrocytic markers in the surrounding of VZs (WT: n=6 from 2 batches, KO: n=6 from 2 batches). Scale = 100µm (left) , 50µm (right).   Staining for the DNA damage markers γH2AX and 53BP1 in the nuclei of cells in the VZ of week 6 COs at physiological conditions, together with the pan-radial glia marker SOX2 (WT: n=8 from 3 individual batches, KO: n=12 from 3 individual batches, W-AAV: n=4 from 1 batch). Scale = 50µm (left), 25µm (right).   Quantification of γH2AX (top) and 53BP1 foci (bottom) in the nuclei of cells composing the innermost layer of the VZ, normalized to the total number of nuclei in this layer. The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WT: n=8 organoids from 3 batches, KO: n=12 organoids from 3 batches, W-AAV: n=4 from 1 batch).  Data information: n.s (non-significant), *p ≤ 0.05, **p ≤ 0.01, ****p ≤ 0.0001.  \n",
      " is required for bronchial epithelial tight junction formation and function.Figure 4a-e) 16HBE cells were stably infected with lentiviral vector pLKO alone or expressing EMP1 shRNAs (1 or 2). All data are representative of n=3 independent experiments. (a) Total RNA was isolated and analysed for EMP1 expression using Taqman/qPCR with a GAPDH control. Wild type cells treated with 500nM GSK1120212 or siSOS1 for 4 days were also analysed. Error bars denote mean ± SEM, dots indicate individual data points. ***, p <0.0002 (GSK = 0.0002, siSOS1 = 0.0001); ****, p <0.0001. a-e) 16HBE cells were stably infected with lentiviral vector pLKO alone or expressing EMP1 shRNAs (1 or 2). All data are representative of n=3 independent experiments. (b) Cells were fixed and stained for ZO-1, E-cadherin and DNA. Scale bar represents 20m. a-e) 16HBE cells were stably infected with lentiviral vector pLKO alone or expressing EMP1 shRNAs (1 or 2). All data are representative of n=3 independent experiments. (c) Quantification of tight junction phenotype. >500 cells were counted per sample/experiment, across n=3 independent experiments (dots indicate individual data points). Error bars denote mean ± SEM. *, p = 0.0148; **, p = 0.006. a-e) 16HBE cells were stably infected with lentiviral vector pLKO alone or expressing EMP1 shRNAs (1 or 2). All data are representative of n=3 independent experiments. (d) Transepithelial resistance (TER) was measured in 16HBE on day 4 post-seeding. Error bars denote mean ± SEM, dots indicate individual data points. **, p = 0.0036; ***, p = 0.0005. a-e) 16HBE cells were stably infected with lentiviral vector pLKO alone or expressing EMP1 shRNAs (1 or 2). All data are representative of n=3 independent experiments. (e) Cells were seeded on glass-bottomed dishes for 4 days. FM4-64 dye was applied to the media and confocal z-stacks acquired. (f) 16HBE cells were fixed and stained for endogenous EMP1 and DNA using two different commercial antibodies. Scale bar represents 20m. (g) 16HBE cells were seeded on glass coverslips for 10 days to form a mature, polarised monolayer. Cells were fixed and costained for EMP1 and ZO-1, then analysed by confocal microscopy. A representative z-stack is presented; arrows indicate ZO-1 positive tight junctions. \n",
      "Fission is associated with ultrastructural changes in mitochondria.Figure 4Fission is associated with ultrastructural changes in mitochondria. EM tomography of mitochondria in cultured corticalneurons demonstrates injury associated with mitochondrial fission. Cortical cultures were exposed to aged SNOC control (A-D) or 200 μM SNOC (E-L) for 6 h in the presence of 80 μM zVAD‐fmk methyl ester. A slice from each tilt series is shown (A, E and I), followed by orthogonal views of 3D reconstructions after segmentation of the OMM (blue) and IMM (gray), with cristae (various arbitrary colors) (B-D, F-H and J-L). For clarity, 10 representative cristae are shown in (B, D, F, G, J and K), while all cristae are displayed in (C, H and L). The control mitochondrion (A-D) is elongated with a dark matrix, indicating no swelling, and 34 cristae. The NO‐exposed mitochondrion in (E-H) is dividing, and the lower mitochondrion of the couplet shows severe damage, recognized by an outer membrane rupture, a regionally confined inner membrane blowout, slight matrix swelling and cristae fragmentation. Each of the daughter mitochondria is shorter and rounder than controls. Unlike the upper daughter, few of the cristae extend throughout the lower mitochondrion. Additionally, cristae are considerably smaller and regionally confined. The lower mitochondrion of the couplet contains 82 cristae, while the upper contains only 39, indicating cristae fragmentation in the former. The NO‐exposed mitochondrial triplet in (I-L) displays further damage, including membrane degradation, accumulation of an electron dense mass, greater cristae fragmentation, and apparent mitochondrial fragmentation. All cristae are smaller than controls, with 73 in the top mitochondrion, 112 in the middle, and 47 in the bottom. Scale bars, 400 nm. See Supplementary Video 3, 4 and 5 of tomographic reconstruction. \n",
      "Mitochondrial fission in ischemic stroke in vivo.Figure 3(A) Elongated mitochondrion (arrow) in neuronal process of the unaffected hemisphere (top). Small mitochondrial segments (arrows) are in close proximity to each other in a neuronal process of the ischemic brain region (bottom). Images are representative of three analyzed mice. Scale bars, 500 nm. (B) Histogram comparing mitochondrial cross‐section length of contralateral, control (red) and ischemic (black) hemispheres. The lower bins of the histogram (2000 nm) were compared with χ2‐test and were not different. The table shows the presence of long mitochondrial cross‐sections (>2000 nm) in the control hemisphere and their absence in the ischemic hemisphere (P0.01; Fisher's exact test). (C) Proximities of mitochondria comparing the contralateral (red) and ischemic (black) hemispheres. Bar diagram shows percents of mitochondrial proximities found to be smaller or larger than 400 nm. The same data set is summarized as a contingency table on the right, (P0.001; two‐way Fisher's exact test). \n",
      ".Figure 5Peripheral blood mononuclear cells (PBMCs) were isolated from a WOREE patient and from his healthy parents and were reprogrammed into iPSCs, and subsequently were differentiated into COs.   Week 10 WSM COs stained for the progenitor marker SOX2, neuronal marker β3-Tubulin and WWOX (WSM F1: n=2 from 1 batch, WSM M2: n=2 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch). Scale = 50µm.   Peripheral blood mononuclear cells (PBMCs) were isolated from a WOREE patient and from his healthy parents and were reprogrammed into iPSCs, and subsequently were differentiated into COs. Representative traces of spontaneous spikes recorded from neurons of WSM P (blue), WSM S (red) WSM S W-AAV (purple) week 7 COs. Each recording is 12 seconds long and the zoom-in is of 0.5 seconds (red box on the right). Peripheral blood mononuclear cells (PBMCs) were isolated from a WOREE patient and from his healthy parents and were reprogrammed into iPSCs, and subsequently were differentiated into COs. Average firing rate over 24 neurons from WSM P COs (4 organoids), 41 neurons from WSM S COs (3 organoids) and 40 neurons from WSM S W-AAV organoids (3 organoids). Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test. Bars represents the mean ± SEM. Data information: ns (non-significant), *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001. Peripheral blood mononuclear cells (PBMCs) were isolated from a WOREE patient and from his healthy parents and were reprogrammed into iPSCs, and subsequently were differentiated into COs. GAD67 and VGLUT1 immunostaining in week 10 WSM COs (WSM F1: n=2 from 1 batch, WSM M2: n=2 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch). Scale = 50µm.   Quantification of the data shown in fig 5D. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WSM F1: n=2 from 1 batch, WSM M2: n=2 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch).  Data information: ns (non-significant), *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001.  \n",
      ".Figure 6Week 15 WSM COs stained for astrocytic markers GFAP and S100β. (WSM F1: n=2 from 1 batch, WSM M2: n=3 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch). Scale = 50µm.   Quantification of DNA damage foci, marked by γH2AX (left) and 53BP1 (right), in the nuclei of the cells in the VZs of week 6 WSM COs. The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WSM F1: n=4 from 1 batch, WSM M2: n=4 from 1 batch, WSM S2: n=4 from 1 batch, WSM S5: n=4 from 1 batch, WSM S5 W-AAV3: n=4 from 1 batch, WSM S5 W-AAV6: n=4 from 1 batch).   Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001.  Representative Immunofluorescence staining depicting the data summarized in 6B. SOX2 marks the radial glia in the VZ (WSM F1: n=4 from 1 batch, WSM M2: n=4 from 1 batch, WSM S2: n=4 from 1 batch, WSM S5: n=4 from 1 batch, WSM S5 W-AAV3: n=4 from 1 batch, WSM S5 W-AAV6: n=4 from 1 batch). Scale = 50µm (left), 25µm (right). qPCR analysis for selected Wnt target genes in week 10 WSM COs. Y-axis indicates relative expression fold change. Data are represented as mean ± SEM. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WSM P: n=6 from 1 batch, WSM S: n=4 from 1 batch, WSM S W-AAV: n=4 from 1 batch). Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001.  Week 15 WSM COs stained for the cortical layers' markers CTIP2 and SATB2. (WSM F1: n=2 from 1 batch, WSM M2: n=3 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch). Scale = 50µm.   Quantification of the staining presented in E. The boxplot represents the 1st and 3rd quartile, with its whiskers showing the minimum and maximum points and a central band representing the median. Statistical significance was determined using one-way ANOVA with Tukey's multiple comparisons test (WSM F1: n=2 from 1 batch, WSM M2: n=3 from 1 batch, WSM S2: n=2 from 1 batch, WSM S5: n=2 from 1 batch, WSM S5 W-AAV3: n=2 from 1 batch, WSM S5 W-AAV6: n=2 from 1 batch).  Data information: *p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001, ****p ≤ 0.0001. E  \n",
      "NO triggers mitochondrial fission.Figure 1(A) 3D time‐lapse microscopy of mitochondria undergoing fission in a dendritic arbor of a neuron. Neurons were transfected with Mito‐DsRed2, pretreated with the pan‐caspase inhibitor zVAD‐fmk methyl ester (100 μM), and exposed to SNOC (200 μM). Images were 3D iso‐surface rendered. Frames depict representative time points of the movie demonstrating mitochondrial fragmentation within 3 h of NO exposure (upper panels; scale bar, 15 μm) and closeup views (lower panels; scale bar, 3 μm). See Supplementary Video 1. (B) Mitochondrial fragmentation within the field of view shown in (A) results in increased mitochondrial numbers. (C) NO induces dose‐dependent mitochondrial fission. Neurons expressing Mito‐DsRed2 were exposed to SNOC and fixed at 60 min. The fraction of neurons displaying fragmented mitochondria is shown as the mean±s.e.m. (D) Mito‐DsRed2‐expressing corticalneurons were pretreated with 1 mM nitro‐L‐arginine (20 min) and exposed to 25 μM NMDA. The fraction of neurons exhibiting fragmented mitochondria at 60 min after NMDA treatment is shown as the mean±s.e.m. of quintuplicate samples from a representative experiment (†††significance at P0.001 or n.s.†, not significant compared to control; ***significance at P0.001 as compared to NMDA treatment). \n",
      "Reversible mitochondrial fission and autophagy.Figure 5(A) Projection image of two neurons expressing Mito‐DsRed2 (red) and MyrPalm‐CFP (blue). Images were high‐pass filtered and z‐stacks were projected over maximum intensity. (B) Changes in mitochondrial dynamics were evaluated as mean skeletal length of mitochondria or (C) as mitochondrial numbers. (D) Magnified representative frames of the time‐lapse recording of the area within the white rectangle of (A) indicate mitochondrial morphology at time 0, 20 min and 1 h. These time points are depicted in (B) and (C) by red arrows. Data are representative of 26 neurons, which exhibited reversible mitochondrial fission from a total of 55 analyzed neurons (six experiments). Scale bar, 10 μm. See Supplementary Video 6. (E) Autophagy in neurites at 6 h after SNOC exposure. Arrows indicate autophagosomes engulfing injured mitochondria next to an intact mitochondrion in the left image. The arrowhead in the right panel indicates membrane wrappers characteristic of autophagosomes. White arrowheads in the insets show the double membranes of the autophagosome and those of mitochondria within the autophagosome. Scale bars, 500 nm. \n",
      "Mitochondrial fission occurs before dendritic injury and without AIF or cytochrome c release from mitochondria.Figure 2Mitochondrial fission occurs before dendritic injury and without AIF or cytochrome c release from mitochondria. Neurons were co‐transfected with vectors encoding (A) Mito‐DsRed2 and (B) MyrPalm‐mCFP plus (C) AIF‐GFP and the effects of SNOC were recorded using 3D time‐lapse imaging. Projection images zoomed in onto the dendritic arbor of one neuron (representative of n=42). Arrows indicate dendritic spines. The inset shows the soma at appropriate grayscale levels. The apparent brightening of AIF‐GFP fluorescence in the nucleus at 5 h was due to nuclear and cell body shrinkage rather than nuclear translocation of AIF. Scale bars, 10 μm. See also Supplementary Video 2. (D) 3D time‐lapse image reconstructions of mitochondria and cytochrome c in a dendritic arbor at indicated time points following SNOC addition. Scale bar, 10 μm. Five mitochondria were selected and motion tracked throughout the imaging series. (E) Traces show the ratio of mean GFP to DsRed2 emission intensity from each mitochondrion. Each trace color corresponds to the same colored mitochondrion. (representative of n=9) (F-G) Immunocytochemistry for staining anti‐AIF antibodies (red) anti‐NeuN antibodies (green) and nuclei labeled with Hoechst 33342 dye (blue) after (F) aged SNOC or (G) SNOC (150 μM; 4 h) treatment. Volume rendered 3D reconstruction of confocal image stacks are shown above, and a single confocal plane in grayscale below. Grid, 3.3 μm for (F) and 1.9 μm for (G), scale bars, 5 μm. (H) Ratio of nuclear to somal AIF signal was measured as mean fluorescence intensity in the center of nuclei divided by the mean fluorescence intensity in the cell body, outlined by the NeuN staining (excluding the nucleus). The bar diagram summarizes the mean±s.e.m. of 282 and 262 neurons measured in n=19 and 20 image stacks in three independent experiments for aged SNOC and SNOC, respectively (P∼0.99 not significant by Student's t‐test). (I) DEVDase activity. Purified cortical neurons were exposed to either 200 μM SNOC or 1 μM staurosporine, as positive control. The rate of zDEVD‐AMC caspase substrate cleavage per minute was monitored and expressed in arbitrary fluorescence units. Data show mean±s.e.m. for four independent experiments (*significance at P0.01 by ANOVA). \n",
      "Fission is associated with bioenergetic failure and free radicals.Figure 6(A) Mitochondrial fission is linked to a drop in ATP. Purified corticalneurons were exposed to increasing SNOC concentrations or mitochondrial inhibitors (2 μM rotenone plus 2 μg/ml oligomycin; Rot/Olig). ATP concentrations are shown as the mean±s.e.m. normalized to the plating density of neurons (n=6) (B) [ATP]/[ADP] ratios in purified corticalneurons exposed to increasing SNOC concentrations (n=4). (C) The bar graph depicts ethidine fluorescence density in arbitrary units of aged and fresh SNOC treated neurons at 2.5 h. (D) Cortical neurons expressing Mito‐GFP (green) were exposed to aged or fresh 200 μM SNOC solution. After 2.5 h cultures were loaded with hydroethidine. Images are representative of more that 20 neurons analyzed (in D) for each condition from at least three independent experiments. Scale bar, 10 μm. (E) Reduced glutathione (GSH) partially blocks mitochondrial fission. Neurons were transfected with Mito pretreated for 2 h with 2 mM GSH monoethyl ester, washed once and then exposed to 35 μM SNOC. Mitochondrial fission was scored after 1 h. Data are means±s.e.m. of quintuplicate samples from one representative experiment of a total of three (†††significance at P0.001 or n.s., †not significant compared to control; *significance at P0.05). \n",
      "Mitochondrial fission is required for neuronal cell death and mediated by Drp1 and Mfn1.Figure 7(A) Representative fluorescence micrographs of mitochondrial morphology before and after SNOC (175 μM; 7 h) or rotenone (30 nM; 2 h) treatments and a comparison of the effects of Aβ25-35 versus Aβ35-25(10 μM, 6 h), as indicated. Scale bar, 20 μm. (B) Percentage of mitochondrial fission and cell death at 18 h († and ††† significance at P0.05 and 0.001 as compared to control pcDNA3 transfection; *,** and *** significance at P0.05 , 0.01 and 0.001, respectively, compared to SNOC treated, pcDNA3 transfected neurons; n=3 independent experiments). Dying neurons were recognized by their shrunken and condensed nuclei after Hoechst 33342 staining. (C) Enforced Drp1 or Fis1 expression (60 h) evokes fission and neuronal cell death (*,** and *** significance at P0.05, 0.01 and 0.001, respectively, compared to pcDNA3 transfected neurons; n=3 independent experiments). (D) Rotenone induces dose‐dependent mitochondrial fission. The fraction of neurons displaying fissioned mitochondria is shown as the mean ±s.e.m. (*significance at P0.05, n=4). (E) Mfn1 or Drp1K38A inhibits 100 nM rotenone‐induced mitochondrial fission (4 h) and cell death (48 h). Data indicate means±s.e.m. of triplicate measurements (†† and ††† significance at P0.01 and 0.001 as compared to untreated control; ***significance at P0.001 as compared to rotenone treatment with pcDNA3 transfection; n=3). (F) Mitochondrial fission by Aβ35-25 or Aβ25-35 exposure. The fraction of neurons exhibiting fragmented mitochondria is shown as the mean±s.e.m. (††significance at P0.01 compared to pcDNA3 transfected, Aβ35−25 treated control; * and ** significance at P0.05 and 0.001, respectively, compared to pcDNA3 transfected, Aβ25-35 treated neurons; n=3). \n",
      ".Figure 1Gating strategy for sorting of EpCAMhighCD24lowSca‑1+ cells and rMC from lung homogenate of adult mice. Time-course of epithelial stem/progenitor cell proliferation and differentiation within BALO at days 8, 11 and 21 of co-culture with rMC. Clonal expansion of EpCAMhighCD24lowSca‑1+ cells derived from either GFP-expressing mice or tdTomato-expressing ROSAmT/mG mice at day 21 of co‑culture. Percentages of SCGB1A1+SFTPC+ (BALO), SCGB1A1+SFTPC- (bronchiolospheres) and SCGB1A1-SFTPC+ (alveolospheres) organoids per well at day 21 of culture derived from EpCAMhighCD24lowSca-1+ cells isolated from Scgb1a1mCherrySftpcYFP reporter mice (n=4 biological replicates). Representative confocal images of days 8 to 21 of culture showing endogenous SCGB1A1 and SFTPC expression during BALO formation derived from EpCAMhighCD24lowSca-1+SCGB1A1+SFTPC+ cells isolated from Scgb1a1mCherrySftpcYFP reporter mice. Heat map (left) and tSNE plot (right) (F) of digested day 21 BALO cultures depicting four distinct clusters (C1, airway, blue; C2, alveolar, purple; C3, MYO, orange, and C4, LIF, green). Violin plots of selected genes representing airway associated-genes (G) (Itgb4, Trp63, Krt7, and Sox2) and alveoli associated‑genes (H) (Cxcl15, Lyz2, Sftpc, and Hopx). Each violin plot shows the frequency distribution of the mean transcript level (log2). \n",
      ".Figure 3Representative images of day 21 BALO and tdTomato+ rMC stained with LipidTOX (green). Scale bar represents 100 μm. Fluorescence images of αSMA (green) and neutral lipids (LipidTOX red) staining in WT‑derived rMC and BALO (dotted lines indicate single BALO or insert) at day 21 of culture. Scale bars represent 50 μm (left) and 25 μm (right). Representative flow cytometric dot plots and histograms of PDGFRα expression in rMC (EpCAM-CD45-CD31-Sca-1+) isolated from the lung homogenate of PdgfraGFP reporter mice. Representative images of BALO formation at day 8, 15 and 21 of co-culture. WT BASC were co‑cultivated with sorted rMC (total PDGFRα+ population) or rMC expressing either low or high levels of PDGFRα‑GFP. Scale bars represent 100 μm. tSNE plots and violin plots depicting selected genes representing MYO (Pdgfrα, Tagln, Acta2, Eln, and Axin2) (top panels) and LIF associated-genes (Fgf10, Apoe, Serpina3n, Gsn, and Gas6) (bottom panels). Each violin plot shows the frequency distribution of the mean transcript level (log2). C3 (MYO, orange) and C4 (LIF, green) refer to the scRNA-Seq experiment in Fig. 1G. Fluorescence image of αSMA+PDGFRαhigh MYO (yellow arrows) and LipidTOX+PDGFRαlow LIF (red arrows) from sorted PDGFRα-GFP rMC after 21 days of BALO culture. Scale bars represent 50 μm (left) and 25 μm (right). Fluorescence image containing PDGFRαhigh MYO from sorted PDGFRα-GFP rMC within BALO derived from tdTomato+ mice at day 21 of culture. Scale bars represent 100 μm (left) and 50 μm (right). \n",
      ".Figure 2mRNA expression analysis of epithelial cell differentiation markers Sftpc (AEC II), Hopx (AEC I), Foxj1 (ciliated cells), Muc5ac (secretory cells) and p63 (basal cells) in BALO at days 0, 10, and 21 of culture (n=3 biological replicates with pooled cells from 4 cultures per replicate) Electron microscopy of BALO alveoli showing two cuboidal epithelial cells (AEC II) connected by tight junctions (red square) with numerous mitochondria (M) and lamellar bodies (LB). A LIF with numerous lipid droplets (*) and a MYO with cisterns of rough ER (red arrow) are located at the basal side (left). The alveolar lumen is filled with lamellar surfactant including tubular myelin (white arrow) (lower right). Exocytosis of a lamellar body (*) from an AEC II (upper right). Scale bars indicate 500 nm. Electron microscopy of bronchiolar-like airway depicting columnar ciliated cells (red arrowhead) with basal bodies (black arrowhead) at the left side of the longitudinally sectioned lumen. The boxed area indicates an apical junctional complex between two ciliated cells. 1 = tight junction, 2 = adherens junction. Scale bar indicates 500 nm (in insert: 100 nm). Staining of lamellar bodies in BALO with RFP LysoTracker. Scale bars represent 100 μm. Western blot analysis of the surfactant proteins: pro-SPC, mature SPC, pro-SPB, mature SPB, and SPA in lung homogenate (LH), AEC and day 21 BALO (n=3 biological replicates). F-G. Alveolar diameter (F) and number of alveoli (G) in BALO at days 15, 21, 30 and 40 of culture was measured from n=5 BALO in n=3 biological replicates with tdTomato+ rMC. H-I. Representative scheme and images of day 40 BALO alveoli (H) and airway (I). BALO alveolar-like structures (H) are shown (h') (red arrowheads) in semithin section (0.5 µm) stained with Toluidine blue. Scale bar indicates 100 μm (left). Electron microscopy showing AEC I (h'') ultrastructure within BALO. Scale bars indicate 2500 nm (center) and 1000 nm (right). An airway-like structure (i') is shown with secretory and ciliated cells (red arrowheads) in semithin sections (0.5 µm) longitudinally cut and stained with Toluidine blue. Scale bar indicates 50 μm (far left). Electron microscopy of a bronchiolar-like airway (I) depicting pseudostratified epithelium (i'') with a basal-like cell (BC), not reaching the lumen in which cilia (C) are seen, located between a secretory (SC) and a ciliated cell (Ci). Scale bars indicate 1000 nm (left and right). Mature cilia (i''') in BALO at higher magnification depicting the 9x2+2 structure with central doubled microtubules (insert, red arrowhead), a characteristic for motile cilia. Scale bar indicates 100 nm (far right). \n",
      "Global analysis of transcript profiling.Figure 2(A) Heat maps for three subsets of the pattern clustering analysis generated using the Euclidean Distance metric. Upregulated genes are in red and downregulated genes are in green. The downregulating transcriptional activity increases with time. The main branch in clustering of the different Atro mutants is along time points. See also Supplementary Figure 4. (B) Venn diagrams for the subset of genes affected at the two time points for the different Atro versions with respect to control flies. The overlap between all Atro forms and the two polyQ versions increases with time. The genes in common for all Atro versions or between the polyQ Atro mutants at both times are listed. \n",
      ".Figure 5Representative images of E11.5 lung explants after treatment for 48 h with 100 μM Scra or mo142-3p reveals reduced size and branching morphogenesis after miR142-3p knockdown. Organoid diameter (n=30-50 per group) in µm before addition of 4 μM Scra or mo142-3p at day 6 (control) and 5 days after treatment (at day 11 BALO culture) in n=3 biological replicates Representative images of β-galactosidase staining in TOPGAL epithelium (C) BALO cultures before (day 6, control) or 5 days after treatment with either 4 μM Scra or mo142-3p (day 11 of co-culture). BASC and rMC were isolated form the lung homogenate of TOPGAL mice. β-galactosidase+ rMC at day 11 of culture are indicated with arrows. Representative images of β-galactosidase staining in TOPGAL rMC (D) BALO cultures before (day 6, control) or 5 days after treatment with either 4 μM Scra or mo142-3p (day 11 of co-culture). BASC and rMC were isolated form the lung homogenate of TOPGAL mice. β-galactosidase+ rMC at day 11 of culture are indicated with arrows. Representative transmission and confocal images after LysoTracker staining indicating branching and number of branching points in n=4 BALO 15 days after treatment with 4 μM Scra or mo142-3p (day 21 of co-culture) in n=3 biological replicates. mRNA levels of epithelial and mesenchymal miR142-3p expression in Scra and mo142-3p-treated organoids 5 days after treatment (n=3-4 biological replicates with pooled cells from 4 cultures per replicate). \n",
      "Transcriptional profiling of polyQ Atrophins.Figure 1(A) Illustration of the crossing and ageing scheme used to obtain total RNA extracts from fly heads for the transcriptional profiling and all successive qPCR assays. Expression of different Atrophin forms with the GMR driver was induced, owing to a temperature‐sensitive mutant Gal80 repressor. F1 flies were allowed to develop at 18°C; at this temperature the Gal80 repressor keeps transgenes silent. Newly eclosed flies (0-48 h) were collected and killed immediately (0d) or aged for 2 or 14 days at 29°C. This inactivates Gal80 and transgenes are switched on by GMR‐Gal4. This protocol allows comparing siblings that differ exclusively in their age and transgenes expression. Control flies crossed to GMR‐Gal4; UbiGal80ts in all experiments are from the w1118 stock in which all UAS transgenes have been generated. (B) Tangential eye sections of flies representative of all the different populations used in the microarray analysis at all different time points. Weak degeneration is only visible after 14 days with polyQ Atro; in particular with Atro75QN there is an initial loss of photoreceptors (PR, arrow), 30.7% of the ommatidia has lost at least 1 PR, that is only 5.1% of all neuronal PR have been lost at this stage (N=333). \n",
      ".Figure 6 Representative images of day 21 BALO after microinjection of SC35M-GFP IAV (green) into the central airway-like (*) after 0 and 12 h pi visualizes IVA spread to the alveolar-like regions. Dotted lines illustrate BALO-borders. Scale bars represent 100 μm  Quantification of plaque forming units in supernatants from mock or SC35M IAV infected BALO 48 h after infection (n=5 biological replicates). N.D.: not-detectable  Relative NP expression in PR8 IAV infected (HA+) or non-infected (HA-) epithelial cells isolated from BALO 48 h pi, FACS-sorted according to EpCAM and HA expression (n=3 biological replicates with pooled cells from 4 cultures per replicate)  Representative flow cytometric data showing the percentage of EpCAM+NP+ cells in mock- and SC35M IAV‑infected BALO at 48 h pi in n=3 biological replicates with pooled cells from 4 cultures per replicate  Representative fluorescence images of a day 21 distal region in BALO generated from mTmG reporter mouse and infected by SC35M-Cre IAV after 0, 10, 14 and 25 h. Arrows indicate cell death. Scale bars represent 25 μm and 10 μm in the insert  mRNA expression of Ifnb in mock and PR8 IAV-infected BALO at 48 h pi (n=3 biological replicates with pooled cells from 4 cultures per replicate)  Release of TNFα (6 h pi), IL-6 (48 h pi), IL-1β (48 h pi) detected by Bio-Plex® Multiplex Immunoassay in the supernatant of mock and IAV infected BALO cultures with and without TR-Mac (n=3 biological replicates) \n",
      ".Figure 4 Gating strategy to define TR‑Mac from BAL of adult tdTomato+ mice  Representative images of BALO after microinjection of tdTomato+ TR-Mac at day 14. TR-Mac are preferentially found in alveoli (right). Scale bar represent 100 μm (left) and 50 μm (right)  Fluorescence confocal images of CD206, Siglec-F, and isotype control staining 14 days after tdTomato+ TR‑Mac microinjection in a day 28 BALO. Scale bars represent 100 μm (left) and 25 μm (right)  Electron microscopy depicting filopodium of TR‑Mac (white dashed lines) with a characteristic actin filament bundle (yellow background) (left panel) in contact with AEC I within BALO alveolar-like structures. Uptake of surfactant by TR-mac is depicted by a (*). Scale bar indicates 1000nm (left, in insert: 500 nm) and 500 nm (right, in insert: 250 nm)  Representative confocal images of Cx43 staining in tdTomato+ TR-Mac monoculture in Matrigel and tdTomato+ TR‑Mac microinjected at day 14 and analyzed in a mature day 21 BALO. Scale bars represent 50 μm (overview) and 5 μm (close up)  Heat map (left) and tSNE plot (right) of the comparative analysis of digested day 23 BALO cultures with (+) and without (-) microinjected TR‑Mac depicting six distinct clusters (C1, club/secretory cells, red; C2, basal cells, yellow; C3, rMC, green; C4, AEC II, blue; C5, AEC I, purple; and C6, ciliated cells, pink)  Expression data dot plots of genes found differentially regulated in cluster C1 between day 23 BALO with (C1+) and without (C1-) microinjected TR-Mac. The circle size illustrates the number of cells expressing a specific gene  Percentage of terminally differentiated cells (AEC I and ciliated cells) in day 23 BALO cultures with (+) and without (-) microinjected TR‑Mac \n",
      "Regulation of ft/Hpo levels is functionally relevant to neurodegeneration by Atrophins.Figure 3(A) qPCR analysis of the fold changes±s.d. of ft transcription. Downregulation starts at 2 days and progresses at 14 days. (B) qPCR analysis of the enrichment±s.d. of different regions of the ft regulatory elements in ChIP for Atro from BG3neuronal cell extracts. Enrichment is calculated as the percentage of DNAimmunoprecipitated from extracts of cells in which Atro expression has been induced, with respect to the amount immunoprecipitated from uninduced cell extracts according to the (Atro-No Ab)induced/(Atro−No Ab)uninduced formula. The DNA region from IV chromosome has been previously shown not to be immunoprecipitated in ChIP for Atro ( Haecker et al, 2007) and has been used as a negative control. (C) Histograms showing the number of ommatidia with a full complement (7) of PR in flies expressing Atro75QN with the Rhodopsin1 driver in either a control (w1118) or different mutant backgrounds and aged at 29°C for 28 days. Heterozygosis for two independent ft alleles and a sav allele significantly enhances the loss of PR. Mild overexpression of Wts via a GMR-wts transgene, which does not display any strong phenotype per se, significantly suppresses the loss of PR. No interaction was detected in this assay with wtsx1 and ykiB5 alleles in heterozygosity. N=430-963 from at least four eyes. (D) Tangential eye sections and histograms showing the degeneration of GMR>Atro75QN, Ubi-Gal80ts flies in combination with different UAS transgenes and aged 14 days as in Figure 1A. UAS-EGFP is used as a negative control. Either Ft or Wts overexpression strongly suppresses loss of PR caused by Atro75QN. Lower panels are provided as examples of quantification. A more modest but still significant rescue is observed by overexpressing a Yki RNAi contruct or Hpo; however, in this case it is to be considered that the overexpression of Hpoper se brings about the loss of at least one PR in ∼5% of ommatidia (data not shown). No effect is detected with overexpression of Sav. Two‐tailed t‐test: *P0.05; **P0.01. The wtsEPG4808 is a previously uncharacterised EP element insertion at the 5′ of the wts gene. Rare homozygous escapers display larger eyes, whereas in combination with GMR-GMR this line gives rise to smaller rough eyes, and, finally, GMR analysis of GMR>wtsEPG4808 indicates a 10‐fold increase in the head content of wts mRNA (data not shown). \n",
      "Neurodegeneration by mutations in ft.Figure 4(A) Tangential eye sections through ftfd clones aged 1, 7 and 14 days, of ftfd,dDGC13 double mutant clones and of ftfd clones in a ykiB5 heterozygous background aged 14 days. Clones are marked by the absence of yellow pigment. Arrows point at degenerating photoreceptors, arrowheads point at intact wt photoreceptors in mosaic ommatidia. The circled ommatidium is genetically wt but non‐autonomously flipped in its polarity and has not degenerated. Lower panels are masks that show clonal borders and quantification of PR number per ommatidia. After 14 days, almost all ftfd cells have degenerated, whereas almost all ftfd,dDGC13 and some ftfd,ykiB5/+ neuronalphotoreceptors have not. (B) Histograms showing the quantification of degeneration in ftfd, ftfddDGC13 and ftfdykiB5/+ clones. Loss of ft leads to statistically significant loss of neurons and this is suppressed at 14 days by ykiB5and much more dramatically by dDGC13. ***P0.001, **P0.01; *P0.05 in two‐tailed t‐test. (C) Degeneration in mosaic ommatidia in ftfd clones. At this stage of degeneration approximately half of the non‐pigmented (genotypically mutant) PR cells display a degenerative phenotype, whereas virtually all pigmented (genotypically wt) PR cells are normal. χ2‐test: 82.38; P0.001. Total PR N=310 (pigmented) and 327 (not pigmented). (D) Tangential eye sections through MARCM clone mutants for ftfd that express either no transgene or an RNAi construct against yki (UAS-ykiIR) with Tub-Gal4 and aged 14 days at 29°C. Because the transgene UAS-ykiIR is on chromosomal arm 2 L, the same as ft, clones generated with this system carry two copies of UAS-ykiIR and therefore downregulate yki very effectively. No other cell outside the clones expresses the construct and is therefore wt for yki. Mask panel is presented on the right of each section. A quantification of photoreceptor numbers (far right) displays a very significant increase in the number of wt ommatidia (N=210 and 219 from four eyes). ***P0.001 in two‐tailed t‐test. If all classes of ommatidia are considered, χ2‐test=232.30; P0.001 for 3 degrees of freedom. \n",
      "Neurodegeneration by polyQ Atro partially requires dachs.Figure 7Neurodegeneration by polyQ Atro partially requires dachs. Tangential eye sections through dGC13 clones (marked by the absence of yellow pigment) either in control flies or those expressing Atro75QN with the Rhodopsin1 driver and aged 28 days. Mask panels below each section exemplify clonal boundaries and PR no. for the ommatidia. Early signs of degeneration indicated by reduced complement of photoreceptors are detected specifically in the pigmented area (arrowhead). At this stage, inside d mutant clones all ommatidia are unaffected and display a wt arrangement (arrows). (B) Histograms showing the quantification of cell loss in eyes shown in (A). A statistically significant rescue is obtained both for the number of wt ommatidia (*P0.05 in two‐tailed t‐test) and for all other categories of ommatidia (χ2=44.76; P0.001 for 2 degrees of freedom). \n",
      ".Figure 1(A) Domain architecture of human HSP27: N-terminal domain (N-term), α-crystallin domain (ACD), and C-terminal region (C-term). The conserved IxI/V motif in the C-terminal region is indicated. Missense, nonsense, and frameshift mutations associated with CMT disease and dHMN are shown. The P182L mutation in the IxI/V motif is blue. Residues with multiple mutations are italicized (e.g. P182L, P182S). The asterisk denotes a nonsense mutation and the X indicates a frameshift mutation. (B) Amino acid sequence alignment of IxI/V motifs and adjacent residues from vertebrate HSP27 orthologues: human (Homo sapiens), mouse (Mus musculus), alligator (Alligator mississippiensis), python (Python bivittatus), frog (Xenopus laevis), shark (Callorhinchus milii), gar (Lepisosteus oculatus), rice fish (Oryzias latipes), and lungfish (Protopterus annectens). (C) Domain boundaries of the ACD used in this study, cHSP27, and residues in the C-terminal IxI/V peptide. (D) Three-dimensional structure of the ACD dimer (black) bound to the IxI/V peptide from the C-terminal region (red). The β-strands of the ACD are numbered. (E) Zoomed-in region from D showing the contacts made between the IxI/V peptide and the ACD for the wild-type peptide. (F) Malate dehydrogenase (MDH; 0.2 µM) was incubated at 40 °C in the absence (black) or presence of 0.5 µM WT HSP27 (red) or 0.5 µM P182L HSP27 (blue). (G) Insulin (40 µM) was incubated at 40 °C in the absence (black) or presence of 20 µM WT HSP27 (red) or 20 µM P182L HSP27 (blue). The y-axes depict the absorbance at 340 nm due to the formation of large aggregates. The solid lines represent the average of three replicates with the filled area reflecting ± one standard deviation. \n",
      "Neurodegeneration by mutations in Hippo pathway genes.Figure 5(A) Tangential eye sections through wtsX1 mutant fly eyes. In clones for the null wtsX1 allele, degeneration inside the clones is at severe stages at 14 days. (B) Tangential eye sections through sav3 clones. In sav3 after enclosure many photoreceptors are intact but after 14 days at 29°C most sav mutant photoreceptors have degenerated. (C) Histograms showing the quantification of ommatidia with the full complement of photoreceptors in hpo, wts and sav mutant clones. The progression of loss of photoreceptors is evident and statistically significant in all cases. Mutants for hpoBF33 do not survive for 14 days at 29oC and corresponding sections are shown in Supplementary Figure 12. N=195-268. ***P0.001, **P0.01; *P0.05 in two‐tailed t‐test. \n",
      "Uncoupling of neurodegeneration and overgrowth.Figure 6(A) Regression analysis of the overgrowth versus the neurodegeneration observed in loss of function mutants (left) and overexpression of transgenes (right). Overgrowth quantification is shown in Supplementary Figure 14. For loss of function mutant, the neurodegeneration was quantified at 14 days in clones (as shown in Figures 4 and 5). For UAS‐overexpression transgenes neurodegeneration was quantified at 14 days when expressed with GMRGal4; UbiGal80ts. For CycD+Cdk4, these specific data are missing and have not been plotted; however, Supplementary Figure 13 shows that these mutants do not cause neurodegeneration in a different setup in which they are expressed during both development and adult life. In both cases the regression coefficient r2 is extremely low (0.21 and 0.03), indicating an absence of correlation between the two processes. (B) Tangential eye sections through the eyes of flies expressing with GMR-Gal4, Ubi-Gal80ts either UAS-EGFP or UAS-Yki or UAS-YkiS111A,S168A,S250A and aged 1 or 28 days. Arrows point at missing or degenerated photoreceptors. The UAS-YkiS111A,S168A,S250A is so effective that even the very low expression leaking out with this system is enough to affect development and generate mild eye roughness and overproliferation of lattice cells. Histograms showing the quantification of cell loss in eyes shown in A and also eyes from flies aged at the intermediate 14‐day stage. Mild but significant degeneration is observed for both Yki forms, with respect to the negative control; however, no statistical difference is observed between the two Yki proteins that have dramatically different effects on overgrowth. ***P0.001, **P0.01; *P0.05 in two‐tailed t‐test. \n",
      "Autophagy in sav mutants.Figure 9(A) EM scan of a 7‐day‐old sav3 mutant ommatidium and zoom‐in on autophagosomes containing undigested debris (arrowhead). Scale bar: 1 μm for panel on the left and 0.2 μm for zoom‐in panel on the right. (B) Graph of the quantification of autophagic vesicles (AV) per photoreceptor found in EM sections of 1‐week‐old sav3 clones. Significant accumulation of AV is found in mutant cells. ***P0.001 in one‐tailed t‐test. N=45 pigmented cells versus 42 non‐pigmented cells. (C) Model for the relationship between Atro, Ft/Hippo and neuronal homeostasis. Atro wt overexpression, and more effectively polyQ Atro overexpression, causes a downregulation of ft transcription and of other genes important for photoreceptor neuron function and homeostasis. This results in downregulation of Ft/Hippo signalling, which affects neurodegeneration. Healthy Ft/Hippo signalling contributes to neuronal homeostasis through effective autophagy. Atro may also exert Ft‐independent effects through other factors (like Tetraspanins) and polyQ Atro also affects neurons through polyQ‐specific factors (like misfolding and aggregation), which put additional pressure on the autophagic system. Other polyQ proteins may not feed into the Ft/Hippo pathway, as in the case of Htt‐ex1‐93Q in flies. \n",
      ".Figure 2(A) SEC-MALS data from WT HSP27 and the P182L variant, with cHSP27 included for comparison. All samples were injected at 40 µM concentration (monomer). The respective average molecular masses are 14 MDa, 470 kDa, and 18 kDa for P182L, WT HSP27, and cHSP27. The normalized UV signals are plotted on the left y-axis as solid lines and the molecular masses are plotted on the right y-axis as circles. (B) Fractionation of HeLa cells overexpressing either HSP27 or the P182L mutant by a 10% to 80% sucrose gradient. Western blot against anti-V5 is shown. (C) Expression of V5-epitope-tagged HSP27 (WT or P182L mutant) in HSP27 knock-out HeLa cells and immunostaining of HSP27. The scale bars correspond to the non-expanded dimensions, derived by dividing by the sample expansion factor of 4.3. (D) The P182L mutation decreases the density of HSP27 spots upon detection of local fluorescence intensity maxima and quantification of the number of detected spots in expanded samples (n = 30). The error bar is the standard deviation for the different cytoplasmic regions. (E) Distribution of the intensity of individual spots (n = 3342) after Gaussian fitting of fluorescence intensity in the neighborhood of detected spots. (F) Immunostaining of HSP27 (white) in motor neurons differentiated from CMT patient-derived iPSCs that harbor the heterozygous P182L mutation reveals large, cytoplasmic aggregates (center and right panels). The control motor neurons that contain WT HSP27 do not show evidence of any aggregates. The nucleus is stained with Hoechst (blue). Scale bar, 10 µm. \n",
      "Autophagic modifications in ft mutants.Figure 8(A) EM scan of a wt (left) and ftfd mutant ommatidium (right) of a ftfd clone after 14 days. Scale bar: 1 μm. High‐magnification panels (far right, top to bottom) display autophagosomes with undigested debris, damaged mitochondria and forming phagophores (arrowheads), found in ftfd mutant cells. Scale bar: 0.2 μm for zoom‐in panels. (B) Graphs of the quantification of autophagic vesicles (AV) per photoreceptor found in EM sections of 7‐day (left)‐ and 14‐day (right)‐old ftfd clones. Significant accumulation of AV is found in mutant cells (not pigmented) with respect to genotypically wt (pigmented) cells. **P0.01 and ***P0.001 in one‐tailed t‐test. N=12 pigmented cells versus 20 non‐pigmented cells for the 7‐day graph and 13 pigmented cells versus 21 non‐pigmented cells for the 14‐day graph. (C) Confocal pictures of whole‐mount retinae of a ftfd clone aged 7 days and expressing GFP∷Atg8a ubiquitously with Tub-Gal4. Red is phalloidin marking rhabdomeres, green is GFP and the clone is marked by the absence of β‐gal staining (blue). Small GFP∷Atg8a dots accumulate specifically inside ft mutant cells (arrow). (D) Confocal pictures of whole‐mount retinae of a ftfd clone aged 7 days (left) and 14 days (right). Red is phalloidin marking rhabdomeres, blue is p62 and the clone is marked by the absence of GFP staining (green). p62 starts to gather in small dots specifically inside ft mutant cells (arrow) and then accumulates massively (arrow) in the ft mutant clones as many cells degenerate. \n",
      ".Figure 4(A, B) i. 15N CPMG RD data for residues in the β4 strand (T113) and the β8 strand (S154) in the absence of added peptide. CPMG data are shown for the same residues in the presence of ii. ca. 2% cHSP27-WT peptide complex or iii. ca. 2% of the cHSP27-P182L peptide complex. Uncertainties in R2,eff values are derived from the standard deviations of fitted peak intensities obtained from duplicated νCPMG values. (C) i. The association (kon) and dissociation (koff) rates for WT (red) and P182L (blue) IxI/V peptide binding to the ACD, calculated directly from the CPMG RD data. Residues that are impacted by the presence of a small amount of peptide are indicated by red (ii., WT) and blue (iii., P182L) spheres. In both cases, residues in similar ACD regions show enhanced Rex, indicative of chemical exchange at similar sites. (D) Free energy landscape of the WT (left) and P182L (right) peptides calculated from the MD trajectories as a function of the P182 or L182 φ/ϕ angles. A lower value of ΔG corresponds to a higher relative population. Select IxI/V conformations extracted from the MD simulations are shown for the indicated φ/ϕ angle pairs in red (WT MD) and blue (P182L MD). The conformation of the IxI/V motif in the crystal structure is shown in grey (X-ray) and the P182L reference structure in grey (Reference), as obtained with in silico mutagenesis using PyMol. The black arrows indicate rotations about the φ or ϕ angles. The green circle denotes the φ/ϕ angles of P182 when bound to cHSP27, measured from the crystal structure. (E) Contour plot showing the first and second principal components of a PCA for the MD trajectories for the WT (left) and P182L peptide (right). The larger spread in the P182L peptide suggests a larger sampling of the conformational landscape. \n",
      ".Figure 3(A) Primary sequences of the WT peptide (red) and P182L peptide (blue). The IxI/V motif is indicated with white text. Below: regions of 2D 1H-15N HSQC spectra of 15N-labeled cHSP27 (black) in the presence of increasing amounts of WT (red) or P182L peptide (blue). Resonances in the β4 and β8 strands broaden and disappear upon peptide binding, indicative of intermediate exchange. (B) Zoomed-in regions of resonances during the peptide titrations. The color bars indicate the amount of added WT or P182L peptide. (C) The combined 1HN and 15N chemical shift changes, denoted as chemical shift perturbation (CSP), are plotted as a function of added peptide concentration. (D) CSPs shown as a function of residue number for the WT (red) and P182L peptide (blue). (E) X-ray structure of cHSP27 bound to the WT peptide (PDB: 4mjh), shown in white sticks, with the results from (D) plotted onto the structure. \n",
      ".Figure 5(A) Percentage of [V/I]X[V/I] tripeptides among (top) the total number of all tripeptides (% tripeptides) or (middle) the total number of [V/I]X[V/I] tripeptides in the structured (black) or disordered regions (red) ([V/I]X[V/I] %) shown as a function of the X residue type. (Bottom) The difference between [V/I]X[V/I] % for disordered and structured regions shows the enrichment (positive) or depletion (negative) of specific [V/I]X[V/I] motifs in disordered regions. The grey bars indicate the [V/I]L[V/I] and [V/I]P[V/I] motifs studied in this work, i.e. ILV and IPV. (B) Volcano plot representing the interactors that are enriched for the P182L mutant versus GFP as a negative control, obtained with affinity-enrichment mass spectrometry. Proteins that co-immunoprecipitated significantly more (p<0.05) with the P182L variant are shown in the upper right quadrant. Interactors with one or more [I/V]-x-[I/V] motifs are displayed in red while other significantly enriched interactors are displayed in black. (C-D) Co-immunoprecipitation from HeLa cells stably overexpressing V5-epitope-tagged HSP27 (WT or P182L mutant) using anti-V5 beads. Co-immunoprecipitation was quantified and presented as a percentage relative to the P182L variant (n=3). The mean ± one standard deviation (n=3) is shown. Right: the location of Pro182 (red) in the crystal structure of peptide-bound cHSP27 (PDB: 4mjh) and in silico creation of the P182L mutation (blue). The peptide is shown in white sticks. P182L specifically weakens binding of the HSP27 IxI/V motif. (E) Co-immunoprecipitation from HeLa cells stably overexpressing V5-epitope-tagged HSP27 (WT or P182L mutant) and transiently transfected with BAG3-eGFP (WT or IPV-mutant) using anti-GFP beads. Western blots for anti-GFP (BAG3), V5 (HSPB1), and α-tubulin. NS indicates a non-specific band. Note that detection of BAG3 (anti-GFP) and HSP27 (anti-V5) occurs with different antibodies. The locations of the IPV-to-GPG mutations in BAG3 are depicted below. (F) Co-immunoprecipitation from HeLa cells transiently transfected with V5-epitope-tagged HSP27 (P182L or S155Q/P182L mutant) using anti-V5 beads (n=2). Right: the location of Ser155 (red) in the crystal structure of peptide-bound cHSP27 (PDB: 4mjh) and in silico creation of the S155Q mutation (purple) showing a clash with the bound peptide. The peptide is shown in white sticks.S155Q disrupts binding of all IxI/V motifs. \n",
      ": Structure of Rab-specific PPM1H phosphatase.Figure 1(A) Domain organization of PPM1H, PPM1J, and PPM1A. The annotation of regions (anchor, flap domain) is discussed in the text. The loop deletion (188-226) that was engineered to improve diffraction is indicated. (B) Ribbon model of the enzyme with a view to the catalytic cleft that contains two Mg2+ ions (cyan spheres). The N-terminal region is magenta (33-71), the flap domain is a wheat colour, and the β-sheet motif is green. The loop deletion (188-226) connects α1/α2 on the opposite face relative to the active site. The back view of the enzyme is also shown with a 180° rotation around the axis indicated. Parts of the anchor (RPxFL motif, magenta) that interact with the globular core are shown as stick models, and discussed in the text. The β1 strand is orange to emphasize its non-canonical conformation due to the presence of the preceding anchor. (C) Comparisons of the flap domain of PPM1H with PPM1A (left) and the tudor domain (right). Apart from a conserved loop (dotted circle) which forms an interface with the catalytic domain, the sequences and structures of flaps are diverse among the PPM family. (D) The indicated amounts of recombinant wild-type and mutant PPM1H or PPM1J (with a His-Sumo N-terminal tag, expressed in E. coli) were incubated in vitro with 2.5 µg pThr72 phosphorylated Rab8a (left) or pThr73 phosphorylated Rab10 (right) for 20 min in the presence of 10 mM MgCl2 in 40 mM HEPES pH 7.5 buffer. Reactions were terminated by addition of SDS Sample Buffer and analyzed by Phos-tag gel electrophoresis that separates phosphorylated (slow migrating) and dephosphorylated Rabs. The gel was stained with Instant Blue Coomassie. D288A is a substrate-trapping (inactive) variant of PPM1H and was used as a control. \n",
      ": Crosslinking and docking analysis suggest that the flap domain binds to pRab8a.Figure 2(A) SDS-PAGE analysis of the PPM1H(D288A): pRab8a complex in the presence of DSBU crosslinker. Control migration of proteins are on the left. NP, non-phosphorylated Rab8a; P, phosphorylated Rab8a. The migration of control and crosslinked proteins is marked on the right. (B) Crosslinked peptides from PPM1H and pRab8a are mapped onto the sequence. The flap domain forms extensive crosslinks with pRab8a. In addition, two flexible loops (104-142, 188-226) also have multiple crosslinks with the substrate. (C) Ribbon model of pRab8a (left) and the switch 2 phosphopeptide (right) docked onto the active site of PPM1H. The crosslinks shown between PPM1H and pRab8a are within accepted distance constraints (32 Å) for DSBU (He et al., 2015). (D) Kinetics of phosphate hydrolysis. 25 nM recombinant wild-type PPM1H was incubated with increasing concentrations of pThr72 phosphorylated Rab8a (GTPγS or GDP) as described in Materials and Methods. Initial velocity (V0) was calculated by dividing the concentration of released phosphatase (μM) by time (min) and plotted against substrate concentration for pThr72 phosphorylated Rab8a[GTP bound conformation] (blue) and pThr72 phosphorylated Rab8a[GDP bound conformation] (red). The experiments were repeated twice, and both data points are shown in curves. Line fittings for the left and middle panel were performed using the mean of the two values. Kinetic constants (Kcat, Vmax, Km) were obtained using GraphPad software, and their uncertainties (±) correspond to the SE of mean. (E) Kinetic analysis of PPM1H as in (D) using 50 nM PPM1H and phosphopeptide substrates, as described in Materials and Methods. Initial velocity (V0) was calculated by dividing the concentration of phosphatase (μM) by time (min) and plotted against substrate concentration for pThr72 Rab8a phospho-peptide (blue) and Rab10 pThr73 phospho-peptide (red). Each experiment was performed twice (individual data points shown). (F) Side by side comparison of the catalytic activity against protein and peptide by in vitro malachite green time course analysis. 50 nM recombinant wild-type PPM1H was incubated with 16 μM Rab8a GTPγS pThr72 phosphorylated protein (blue) or Rab8a pThr72 phosphorylated peptide (red) for indicated times and analysed as described in Materials and Methods. The experiments were repeated 3 times. The error bars represent SE of mean of the technical replicates. \n",
      ": Model of PPM1H specificity for Rab GTPases.Figure 7Although PPM1H phosphatase is likely to be dimeric in cells, a 1:1 complex is shown for simplicity. The switch 2 helix is red and the phosphate site is yellow. The flap domain (305-414) encodes the primary specificity for phosphorylated Rabs while the anchor (33-79) contributes to folding of the enzyme. The β-motif (green, residues 480-496) is also indicated. The conserved loop motif (386-396) with a central Leu392 residue may couple substrate binding to catalysis. \n",
      ": Anchor of PPM1H is a folding motif.Figure 6(A) Interactions between the anchor of PPM1H against the electrostatic surface of the core catalytic domain. The region is placed in context with the dotted box on a ribbon model of PPM1H (right). (B) Incremental deletions of the N-terminus from residues 1-37 to 1-44, one residue at a time. Upon deletion of R43 (44-end), a reduced level of soluble PPM1H expression has no significant catalytic activity. (C) Sequence alignment of the N-terminal regions of the evolutionarily related PPM1H/J/M enzymes. The degree of conservation is above the alignment, and residue numbers correspond to PPM1H. A conserved anchor motif (RPxFL) is annotated below the sequences. \n",
      ": Mutagenesis and functional assays identify determinants of catalysis.Figure 5(A) Sites for targeted mutagenesis on the flap domain and active site Lys88 residue. The central ribbon model provides context for the conserved loop (left) and epitopes distant from the active site in the flap domain (right). Residues subjected to mutagenesis are shown as stick models, with dotted envelopes in the central panel for emphasis. Docked pRab8a (yellow ribbons, right) highlights its proximity to mutagenized residues in the flap domain. The sequences of the conserved loop motif are shown below the left panel for several PPMs with known 3-D structures (PPM1H, PPM1K, PPM1B, PPM1B). (B) In vitro malachite green assay time course of recombinant PPM1H mutants against 16 μM pThr72 phosphorylated Rab8a protein (GTPγS, left), 32 μM pThr72 phosphorylated Rab8a peptide (middle), or 32 μM pThr73 phosphorylated Rab10 peptide (right). The experiments were repeated 4 times. The error bars represent SE of mean of the technical replicates. (C) 3 μg of recombinant wild-type or indicated mutant PPM1H proteins were resolved on 4-12% Bis-Tris gradient gel and stained with Instant Blue Coomassie. (D) HEK293 cells overexpressing indicated constructs were treated and analysed as described in Figure 4E. (E) Activity of 25 nM PPM1H flap domain mutants against 16 μM pThr72 phosphorylated Rab8a protein (GTPγS) using the malachite green assay time course. The experiments were repeated 3 times. The error bars represent SE of mean of the technical replicates. Quality of purified proteins (right panel) is shown using the protocol in (C). (F) HEK293 cells overexpressing indicated constructs were treated and analysed as described in (D), with pRab10 analysed in the upper panel, and pRab8a in the lower panel. \n",
      ": The flap domain of PPM1H is a determinant of Rab specificity.Figure 4(A) Domain organization of chimeric variants for in vitro and cellular assays. (B) 3 μg of recombinant proteins were resolved on 4-12% Bis-Tris gradient gel and stained with Instant Blue Coomassie. (C) In vitro malachite green assay time course to determine enzyme activity of recombinant PPM proteins (25 nM for the protein substrate, 50 nM for the peptide substrate) against 16 μM pThr72 phosphorylated Rab8a protein (GTPγS). The experiments were repeated 3 times. The error bars represent SE of mean of the technical replicates. (D) In vitro malachite green assay against 32 μM pThr72-Rab8a peptide (left) and pThr73-Rab10 peptide mimic (right). The experiments were repeated 3 times. The error bars represent standard error (SE) of mean of the technical replicates. (E, F) (E) HEK293 cells overexpressing indicated constructs were treated with ± 200 nM MLi-2 for 90 min and then lysed. 10 μg whole cell lysate was subjected to immunoblot analysis with the indicated antibodies at 1 μg/ml final concentration and membranes were analysed using the OdysseyClx Western Blot imaging system. Left, each lane represents cell extract obtained from a different dish of cells (two biological replicates per condition without MLi-2 treatment, one biological replicate per condition with MLi-2 treatment). Right, the ratio of phospho-Rab8a/total Rab8a was quantified using Image Studio software and data presented relative to the phosphorylation ratio observed in PPM1H wild-type expressing cells. (F) As in (E) assessing phospho-Rab10 levels. For quantitation the two replicates from the presented blots were used. The data points are shown on the graphs with the bar levels representing the mean value of the biological replicates. \n",
      ": PPM1H is a dimer.Figure 3(A) Model of the dimeric form of MnPPM1HWT-LD. Each monomer of PPM1H has an A or B prefix. Flap domains mediate the dimer and are shown in dark blue/wheat colors. The view is down a pseudo two-fold formed by α3 in the flap domain. Dotted lines denote the flexible loop (183-235) that connects α1/α2 in the catalytic domain. The dimeric organization of PPM1H positions this loop next to substrates, shown as docked models of pRab8a in green ribbons (transparent). The phosphoryated switch 2 helix of pRab8a is red. Yellow spheres indicate site of a double mutation in the flap domain (G357E+A359E) that disrupts a contact with α2 of the catalytic domain. (B) SEC-MALS analyses of PPM1HWT-LD and PPM1H2Glu-LD, showing that the WT enzyme is a dimer (104.5±2 kDa) while the double mutant is a monomer (53.6±1 kDa). The calculated molecular weight of His6-tagged PPM1HWT-LD used for the experiment is approximately 52.1 kDa. The errors were calculated from the SE of technical replicates from the data represented as a line across the central peaks in the gel filtration column. Light scattering from 15/18 possible angles (DAWN-EOS, Wyatt Corp) and the refractive index change relative to buffer (Optilab TrEX) were collected every second along these time points. Subsequently, data were processed using Astra 7.1 software (Wyatt Corp) to generate a weight-averaged molecular mass (y-axis) plotted against time (x-axis). There were approximately 130 technical data points for PPM1HWT-LD and 60 technical data points for PPM1H2Glu-LD. (C) His6-SUMO-tagged full-length variants of PPM1H were used for catalytic assays (left panel). 3 μg of recombinant wild-type or indicated mutant PPM1H proteins were resolved on 4-12% Bis-Tris gradient gel and stained with Instant Blue Coomassie. Right, mass photometry histogram for 40nM His6-SUMO-PPM1H WT (blue) and His6-SUMO-PPM1H 2Glu (brown), where WT is 130kDa (±15.8 kDa, with 1004 single molecules counted) and 2Glu is 83kDa (±13.9 kDa, with 4117 single molecules counted). The calculated molecular weight of the fusion protein is approximately 68.5 kDa. (D) In vitro malachite green assay time course of recombinant His6-SUMO-tagged PPM1H wild-type (blue), 2Glu (red), or D288A (grey) against 16 μM pThr72 phosphorylated Rab8a protein (GTPγS, left), 32 μM pThr72 phosphorylated Rab8a peptide (middle), or 32 μM pThr73 phosphorylated Rab10 peptide (right). The experiments were repeated 4 times. The error bars represent SE of mean of the technical replicates. \n",
      ".Figure 1(A) Total lysates from the indicated cell lines cultured in regular conditions at a steady state were subjected to western blotting. s.e., short exposure; l.e., long exposure. The major Merlin band is indicated by an asterisk. The ratio of mono-ubiquitinated to native Merlin in each lane of the short exposure blot was quantified by Image J and is shown under the blot. The same lysates were also used in Figure EV1A. Asterisks indicate native Merlin, arrows show ubiquitinated Merlin. (B) Total lysates from LN229 cells in which Merlin was or was not knocked out (KO) were subjected to western blotting. LN229 (C) cells were detached (denoted by sus., referring to suspension) by trypsinization (see Methods) and reseeded (attach) for two (LN229) hours. Total lysates from these cells were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. Ubiquitinated Merlin is marked with arrows. Met5-A (D) cells were detached (denoted by sus., referring to suspension) by trypsinization (see Methods) and reseeded (attach) for four (Met5-A) hours. Total lysates from these cells were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. Ubiquitinated Merlin is marked with arrows. (E) LN229 cells transduced with a Merlin or scrambled shRNA were treated with DMSO, ionomycin (Iono) or thapsigargin (TG). Total lysates of these cells were subjected to western blotting. The major Merlin band is indicated by an asterisk. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. Ubiquitinated Merlin is marked with an arrow. (F) LN229 cells treated with DMSO or thapsigargin (TG) were lysed and subjected to immunoprecipitation with a Merlin antibody or IgG. The immunoprecipitated products were subjected to western blotting. The number of conjugated ubiquitin (Ub) moieties was marked. Tetra-ubiquitinated Merlin is indicated by an arrow. (G) LN229 cells stably transduced with 6x-Histidine-tagged ubiquitin (His-Ubi) were treated with DMSO or thapsigargin (TG). These cells were lysed and subjected to nickel-charged affinity purification followed by western blotting for endogenous Merlin. Both arrowhead and arrow point to the mono-ubiquitinated Merlin. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (H) LN229 cells stably transduced with 6x-Histidine-tagged ubiquitin (His-Ubi) were detached or reseeded as in (C). These cells were lysed and subjected to nickel-charged affinity purification followed by western blotting for endogenous Merlin. Arrowheads and arrows point to the mono- or di-ubiquitinated Merlin (see a longer exposed blotting results in Figure EV1D). The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. \n",
      ".Figure 3(A) LN229 cells stably transduced with Flag-tagged Merlin or empty vector were treated with DMSO or thapsigargin (TG) and subjected to Flag immunoprecipitation followed by mass spectrometry. Numbers of unique peptides of each protein in the immunoprecipitated products are shown. (B) LN229 cells transduced with a pool of four siRNAs against NEDD4L (+) or a scrambled siRNA (-) were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the short exposure blot was quantified by Image J and is shown under the blot. (C) LN229 cells stably transduced with indicated shRNAs targeting NEDD4L or a scrambled shRNA were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (D) LN229 cells stably transduced with indicated shRNAs targeting NEDD4L or a scrambled shRNA were detached by trypsinization (see Methods) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (E) LN229 cells stably transduced with 6x-Histidine-tagged Ubiquitin (His-Ubi) were then stably transduced with indicated shRNAs targeting NEDD4L or a scrambled shRNA. These cells were detached as in (D), lysed and subjected to nickel-charged affinity purification followed by western blotting for endogenous Merlin. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (F) HEK293T cells were transfected with indicated genes and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (G) HEK293T cells were transfected with indicated genes. The cells were lysed and subjected to immunoprecipitation with a Flag antibody. The immunoprecipitated products were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (H) LN229 cells were lysed and subjected to immunoprecipitation with a Merlin antibody, a NEDD4L antibody, or IgG. The immunoprecipitated products were subjected to western blotting. \n",
      ".Figure 2(A) LN229 cells were detached (denoted by sus., referring to suspension) by trypsinization and reseeded (denoted by attach) for two hours. Total lysates from these cells were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (B) LN229 cells treated with DMSO, ionomycin (Iono) or thapsigargin (TG) were lysed and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (C) LN229 cells treated with DMSO or thapsigargin (TG) were subjected to active RAC1 pulldown assay followed by western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. LN229 cells transduced with vector or EGFP-RAC1(Q61L) (D) were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (E) LN229 cells transduced with vector or Myc-PAK1(T423E) (E) were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (F) LN229 cells stably transduced with wild-type Merlin or the indicated mutants were treated with DMSO or thapsigargin (TG) and analyzed by western blot. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. Native Merlin (asterisk); ubiquitinated Merlin (arrow). (G) Merlin-depleted LN229 cells stably transduced with Flag-tagged wild-type Merlin or the indicated mutants were treated with DMSO or thapsigargin (TG). The cells were lysed and subjected to immunoprecipitation with a Flag antibody. The lysate and immunoprecipitated products were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. \n",
      ".Figure 4(A) LN229 cells transduced with a pool of four siRNAs against AMOTL1 (+) or a scrambled siRNA (-) were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (B) LN229 cells stably transduced with 6x-Histidine-tagged Ubiquitin (His-Ubi) were then stably transduced with indicated shRNAs against AMOTL1 or a scrambled shRNA control (sh-Co.). These cells were detached, lysed and subjected to nickel-charged affinity purification followed by western blotting for endogenous Merlin. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (C) A diagram illustrating that AMOTL1 mediates the NEDD4L-Merlin interaction. (D) Merlin-depleted LN229 cells stably expressing Flag-HA-tagged Merlin were transduced with scrambled shRNA (sh-Co.) or two distinct shRNAs targeting AMOTL1, treated with DMSO or thapsigargin (TG) and subjected to Flag immunoprecipitation followed by western blotting. (E) HEK293T cells were transfected with the indicated constructs. The cells were lysed and subjected to immunoprecipitation with a Flag antibody. The lysate and immunoprecipitated products were subjected to western blotting. PY: PPEY motif; LY: LPTY motif. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (F) Merlin-depleted LN229 cells stably transduced with Flag-tagged wild-type Merlin or the indicated mutants were treated with DMSO or thapsigargin (TG). The cells were lysed and subjected to immunoprecipitation with a Flag antibody. The lysate and immunoprecipitated products were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (G) HEK293T cells were transfected with the indicated constructs. The cells were lysed and subjected to immunoprecipitation with a Flag antibody. The lysate and immunoprecipitated products were subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. \n",
      ".Figure 5(A) LN229 cells stably transduced with Flag-tagged wild-type Merlin or the S518D mutant were treated with thapsigargin and subjected to immunoprecipitation with a Flag antibody and followed by SDS-PAGE. Outlined gel pieces were analyzed by mass spectrometry. (B) Numbers of unique peptides of Merlin or Ubiquitin from each outlined gel pieces in (A) are shown. (C) The percentage of ubiquitinated forms for each peptide from each immunoprecipitated product are shown. (D) and (E) Mass spectrometry spectrums of the indicated peptides from each immunoprecipitated product are shown. (F) Merlin-depleted LN229 cells expressing His-Ubiquitin were stably transduced with the indicated HA-tagged Merlin forms or an empty vector. These cells treated with DMSO or thapsigargin were subjected to nickel affinity purification in a denaturing step followed by western blotting for HA. Arrows indicate mono-ubiquitinated Merlin. The red asterisk indicates native Merlin and black asterisks indicate non-specific signals. The ratio of mono-ubiquitinated to native Merlin in each lane of the input blot was quantified by Image J and is shown under the blot. (G) Merlin-depleted LN229 cells were stably transduced with an empty vector, Flag-tagged wild-type (WT) Merlin or its K396R mutant. These cells were detached (S, suspension) by trypsinization using the procedure described in the Methods section and reseeded (A, attached) for two hours. Cell lysates were subjected to immunoprecipitation with a Flag antibody followed by western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane of the lysate blot was quantified by Image J and is shown under the blot. (H) Merlin-depleted LN229 cells stably transduced with wild-type (WT) Merlin or its K396R mutant were treated with DMSO or thapsigargin (TG) and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (I) Merlin-depleted LN229 cells stably transduced with HA-tagged wild-type (WT) Merlin or its K396R mutant were treated as described in (G) and subjected to western blotting. The ratio of mono-ubiquitinated to native null in each lane was quantified by Image J and is shown under the blot. \n",
      ".Figure 8(A) Met5-A cells stably transduced with the indicated shRNAs targeting Merlin or a scrambled shRNA were subjected to the BrdU incorporation assay. Scale bar = 40 μm. (B) The percentage of BrdU positive cells from (A) was quantified. Mean±s.e.m, N=3 biological repeats, paired t-test. **P<0.01, ***P<0.001. (C) Met5-A cells stably transduced with the indicated shRNAs targeting NEDD4L or a scrambled shRNA were subjected to the BrdU incorporation assay. Scale bar = 40 μm. (D) The percentage of BrdU positive cells from (C) was quantified. Mean±s.e.m, N=3 biological repeats, paired t-test. **P<0.01, ***P<0.001. (E) Meso-33 cells transduced with vector or the indicated Merlin forms were subjected to the BrdU incorporation assay. Scale bar = 40 μm. (F) The percentage of BrdU positive cells from (E) was quantified. Mean±s.e.m, N=3 biological repeats, ordinary one-way ANOVA. *P<0.05, **P<0.01. (G) Meso-33 cells stably transduced with the indicated shRNAs targeting NEDD4L or a scrambled shRNA were then transduced with vector or Merlin prior to the BrdU incorporation assay. The percentage of BrdU positive cells was quantified. Mean±s.e.m, N=3 biological repeats, two-way ANOVA. (H) FC-1801 cells stably transduced with vector or the indicated Merlin forms were subjected to the sphere formation assay. Scale bar = 500 μm. (I) The number of spheres in each well from (H) was quantified. Mean±s.e.m, N=3 biological repeats, paired t-test. **P<0.05. (J) FC-1801 cells stably transduced with vector or the indicated Merlin forms were subcutaneously injected into nude mice. Tumor sizes in each group were quantified and plotted. Mean±s.e.m, NVector=8 mice, NWT=10 mice, NK396R=10 mice, NS518D=8 mice, two-way ANOVA. P values of the last measurements (day 37 after injection) are shown. *P<0.05, **P<0.01. (K) Tumors were collected when the vector group in (J) reached the end point (day 37 after injection). (L) A proposed model showing how Merlin is activated to induce Lats1 phosphorylation on the plasma membrane. \n",
      ".Figure 6(A) LN229 cells were treated with DMSO or thapsigargin (TG) for the indicated time and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (B) Merlin-depleted LN229 cells were reconstituted with wild type (WT) Merlin or its mutants, treated with DMSO or thapsigargin (TG), and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (C) Merlin-depleted LN229 cells were reconstituted with wild type (WT) Merlin or its mutants, treated with DMSO or thapsigargin (TG), and subjected to western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. (D) LN229 cells transfected with a pool of four siRNAs targeting NEDD4L (+) or a scrambled siRNA (-) were treated with DMSO or thapsigargin (TG) and subjected to western blotting. (E) Merlin-depleted LN229 cells were stably transduced with empty vector, Flag-tagged wild-type (WT) Merlin, or its K396R mutant. These cells were detached (S, suspension) by trypsinization using the procedure described in Methods and reseeded (A, attached) for two hours. Cell lysates were subjected to western blotting. (F) LN229 cells transfected with a pool of four siRNAs targeting NEDD4L (+) or a scrambled siRNA (-) were detached (Susp., suspension) by trypsinization using the procedure described in Methods and reseeded (Attach, attached) for two hours. Total lysates from these cells were subjected to western blotting. (G) LN229 cells transfected with indicated siRNAs were seeded for 3 hours. CTGF or CYR61 mRNA in these cells was quantified by q-RT-PCR. *P<0.05, **P<0.01, ****P<0.0001. Mean±s.e.m, N=4 biological repeats, unpaired t-test. \n",
      ".Figure 7(A) LN229 cells stably transduced withHA-Lats1 (two leftmost panels) or vector only (right panel) were treated with DMSO or thapsigargin (TG) and subjected to PLA using HA and Merlin antibodies. Scale bar = 10 μm. (B) PLA signals (dots) in each cell from the results in (A) were quantified. Mean±s.e.m, two-way ANOVA. *P<0.05, ***P<0.001, ****P<0.0001. Each data point represents an image field containing an average of 10 cells. N=4-9 images for each condition as indicated. All images were collected from one experiment. Two independent experiments were performed and gave similar results. (C) Merlin-depleted LN229 cells were stably transduced with HA-tagged wild-type (WT) Merlin or its mutants. These cells were treated with thapsigargin (TG) for 15 minutes and subjected to PLA using HA and Lats1 antibodies. Scale bar = 20 μm. (D) PLA signals (dots) in each cell from the results in (C) were quantified. Mean±s.e.m, Ordinary one-way ANOVA. ****P<0.0001. Each data point represents an image field containing an average of 10 cells. NVector=8, NWT=16, NK396R=18, NK159R=17 images. All images were collected from one experiment. Two independent experiments were performed and gave similar results. (E) Merlin-depleted LN229 cells stably transduced with empty vector, Flag-tagged wild-type (WT) Merlin, or its K396R mutant were subjected to immunoprecipitation with a Flag antibody followed by western blotting. (F) HEK293T cells were transfected with indicated proteins and subjected to immunoprecipitation with a Flag antibody. The lysates and immunoprecipitated products were subjected to western blotting. (G) Merlin-depleted LN229 cells were stably transduced by HA-tagged Merlin. These cells were then transduced with a pool of four siRNAs against NEDD4L or a scrambled control siRNA and treated with DMSO or thapsigargin (TG). PLA was performed in these cells using HA and Lats1 antibodies, and the PLA signals (dots, showed in Figure EV5E) in each cell were quantified. Mean±s.e.m, two-way ANOVA. ****P<0.0001. Each data point represents an image field containing averagely 10 cells. N=12 images in all conditions, except for Nsi-Control:TG=13 images. All images were collected from one experiment. Two independent experiments were performed and showed similar results. (H) LN229 cells treated with DMSO or thapsigargin (TG) were subjected to cytosolic (C)/membrane (M) fractionation followed by western blotting. The ratio of mono-ubiquitinated to native Merlin in each lane was quantified by Image J and is shown under the blot. \n",
      ": Pacs1 deletion warps ER Ca2+ handling.Figure 6(A) Immunoblot of Pacs1, Wdr37, IP3Rs, and ER stress markers in the parental NIH-3T3 cell line and three separate Pacs1-/- clones. (B) Real-time quantitative PCR of IP3R isoform expression WT and Pacs1-/- 3T3 cells. Expression in the Pacs1-/- cells was measured in three independent clones. Data are presented as mean ± SD. (C) Pacs1+/+ and Pacs1-/- NIH-3T3 cells were transfected with cytosolic aequorin and Ca2+ flux was measured after treatment with 1 µM bradykinin. Representative kinetic tracing is shown. (D) Peak cytosolic Ca2+ concentration based on aequorin measurements as performed in (C). Symbols represent individual wells containing ~5 x 105 transfected 3T3 or Pacs1-/- cells from one experiment. Horizontal bars indicate mean. Two-tailed unpaired t test, **P < 0.01. (E) Pacs1+/+ and Pacs1-/- NIH-3T3 cells (C1 and C2 from (A)) were transfected with ER-GCamP6-210. ER Ca2+ was measured before and after treatment with 10 µM ATP. Kinetic traces show the mean 488/405 nm excitation ratio of each cell line with error bars indicating SEM. N=30 cells (3T3), 16 cells (C1), 25 cells (C2). Data are representative of 2 independent experiments. (F) ER Ca2+ release from the NIH-3T3 cell lines imaged in (E). Symbols represent individual cells. Horizontal bars indicate mean value. One-way ANOVA with Tukey's test for multiple comparisons, **P < 0.01, ***P < 0.001. \n",
      ": Pacs1 deletion causes a defect in cytosolic Ca2+ flux after antigen receptor stimulation (A) Pacs1+/+ and Pacs1−/− splenocytes were labeled with Indo-1 and stained for B220, CD21, and CD23 to identify FOB and MZB cells.Figure 2(A) Pacs1+/+ and Pacs1−/− splenocytes were labeled with Indo-1 and stained for B220, CD21, and CD23 to identify FOB and MZB cells. Fluorescence was measured for 30 seconds to establish a baseline and then cells were stimulated with the indicated amounts of anti-IgM (black arrowhead). Cytosolic Ca2+ flux was monitored with FACS analysis by measuring the violet:blue fluorescence emission ratio of Indo-1. Kinetic traces are displayed from five independent Pacs1+/+ and Pacs1−/− pairs and were normalized to baseline (Pacs1+/+ gray traces, Pacs1−/− pink traces). The mean Ca2+ flux for each genotype is overlaid in bold (­Pacs1+/+ black, Pacs1−/− red). (B) Maximum Ca2+ flux (peak height) at each anti-IgM concentration for FOB and MZB cells. Paired t test, ns = not significant, *P < 0.05, and ***P < 0.001. (C) Pacs1+/+ and Pacs1−/− FOB cells were labeled with Indo-1 and stimulated in Ca2+ free buffer with 5 mcg/ml anti-IgM (black arrowhead) to assess ER Ca2+ efflux. Then, 2 mM Ca2+ was added back to assess SOCE. Kinetic traces normalized to baseline from three independent Pacs1+/+ and Pacs1−/− pairs are shown with the mean Ca2+ flux overlaid in bold. (D) Maximum Ca2+ flux after stimulation under Ca2+ free conditions and after Ca2+ was added back. Two-tailed paired t test, ns = not significant, *P < 0.05. \n",
      ": Pacs1 deletion induces ER stress, ROS, and heightened sensitivity to oxidative stress.Figure 4(A) Immunoblot of ER mass, ER stress, and autophagy markers in Pacs1+/+ and Pacs1-/- splenic B cells that were left unstimulated or stimulated overnight with 5 mcg/ml IgM. (B) B cells were purified from Pacs1+/+ and Pacs1-/- spleens and OCR was measured using a Seahorse metabolic flux analyzer. OCR after rotenone/Antimycin A treatment was subtracted from baseline OCR to give basal mitochondrial respiratory rate. Data shown are the mean of 7-10 technical replicates from three independent experiments. Error bars indicate mean ± SD. Two-tailed unpaired t test, *P < 0.05. (C) Determination of cellular ROS in Pacs1+/+ and Pacs1-/- FOB cells with CellRox Green. Quantification shows the mean fluorescence intensity (MFI) from three separate pairs of Pacs1+/+ and Pacs1−/− mice. Two-tailed paired t test, **P < 0.01. (D) Splenocytes from Pacs1+/+ and Pacs1−/− mice were stained with cell surface antibodies to identify FOB cells and treated with 100 µM H2O2 for 35 minutes. Cells were then labelled with TMRE to monitor MMP by FACS analysis. Low TMRE fluorescence indicated susceptibility to H2O2 treatment. Data are presented as mean ± SD of 3-4 replicates, two-tailed unpaired t test, ***P < 0.001. Results are representative of three independent experiments performed on different Pacs1+/+ and Pacs1−/− pairs. \n",
      ": Wdr37 forms a mutually stabilizing complex with Pacs1 (A) Super-pedigree mapping of two mutations in Wdr37 that are linked to peripheral B cell deficiency.Figure 3(A) Super-pedigree mapping of two mutations in Wdr37 that are linked to peripheral B cell deficiency. Insert shows peripheral B cell deficiency in radical (N=12 REF, 18 HET, 4 VAR) and profound (N=14 REF, 15 HET, 6 VAR) pedigrees. Horizontal bars indicate mean values. Ordinary one-way ANOVA with Tukey's multiple comparisons test, ns = not significant, ***P < 0.001. (B) Co-immunoprecipitation of HA-tagged Pacs1 by FLAG-Wdr37 and HA-Wdr37 by FLAG-Pacs1 in co-transfected 293T cells. (C) Western blot for Pacs1 and Wdr37 expression in peripheral blood cells from WT­, Pacs1−/−, and Wdr37−/−­ mice. (D) B and T cell peripheral blood counts in Wdr37−/− mice. Horizontal bars indicate mean values. Two-tailed unpaired t test, ***P < 0.001. (E) Wdr37+/+ and Wdr37−/− splenocytes were labeled with Indo-1, stained for cell surface markers to identify FOB cells, and stimulated with the indicated amounts of anti-IgM (black arrowhead). Normalized traces from three (2.5 mcg/ml anti-IgM) or four independent experiments (10 mcg/ml and 5 mcg/ml anti-IgM) are shown (Wdr37+/+ gray, Wdr37−/− pink). Mean Ca2+ flux for each genotype is overlaid in bold (Wdr37+/+ black, Wdr37−/− red). (F) Maximum Ca2+ flux at each anti-IgM concentration is shown. Two-tailed paired t test, *P < 0.05, **P < 0.01. (G) Wdr37+/+ and Wdr37−/− FOB cells were labeled with Indo-1 and stimulated in Ca2+ free buffer with 5 mcg/ml anti-IgM (black arrowhead) followed by addition of 2 mM Ca2+. Normalized traces from four independent experiments are shown with mean Ca2+ flux overlaid in bold. (H) Maximum Ca2+ flux after stimulation under Ca2+ free conditions and after Ca2+ was added back. Two-tailed paired t test, ns = not significant, *P < 0.05. \n",
      ": Pacs1-/- B cells have reduced IP3R expression (A) Immunoblot of expression of all three IP3R isoforms and SERCA2 in primary splenic B cells from Pacs1+/+ and Pacs1-/- mice.Figure 5(A) Immunoblot of expression of all three IP3R isoforms and SERCA2 in primary splenic B cells from Pacs1+/+ and Pacs1-/- mice. (B) Real-time quantitative PCR of IP3R and SERCA2 transcripts in B cells. Data are presented as mean ± SD from three independent Pacs1+/+ and Pacs1-/- pairs of mice. (C) Pacs1-/- FOB cells were stimulated with 0.625 µM thapsigargin (black arrowhead) under Ca2+-free conditions to measure intracellular Ca2+ stores. Kinetic traces of four independent experiments are shown (Pacs1+/+ gray, Pacs1-/- pink) with the mean overlaid in bold (Pacs1+/+ black, Pacs1-/- red). (D) Plateau of cytosolic Ca2+ flux from intracellular Ca2+ stores in (A) calculated by the mean value over the last 30 seconds of analysis with baseline subtracted. Two-tailed paired t test, *P < 0.05. (E) AUC of cytosolic Ca2+ flux from intracellular Ca2+ stores in (A). Two-tailed paired t test, *P < 0.05. \n",
      ": Spontaneous proliferation and increased cell death of Pacs1−/− B cells in vivo under lymphocyte replete conditions (A) Pacs1+/+ and Pacs1−/− B cells were purified, labeled with CTV dye, and stimulated with the indicated mitogens.Figure 7(A) Pacs1+/+ and Pacs1−/− B cells were purified, labeled with CTV dye, and stimulated with the indicated mitogens. Cell proliferation was assessed after 72 h with FACS analysis based on CTV dilution. (B, C) Pacs1+/+ and Pacs1−/− mice were immunized with alum-ova and one week later with NP-Ficoll. Anti-ova IgG and anti-NP IgM titers were measured at 14 days and 7 days after immunization, respectively. Each symbol represents an individual mouse. (D) Pacs1+/+ and Pacs1ccy/ccy mice were immunized with NP-KLH. Low affinity (anti-NP30) and high affinity (anti-NP2) antibodies were measured 14 days after immunization. (E) B cells were purified from Pacs1+/+ and Pacs1−/− mice and labeled with CTFR and CTV dyes, respectively. Labeled B cells were injected into unirradiated CD45.1 recipients at ~1:1 ratio. Proliferation and survival of adoptively transferred B cells were measured 8 days post-transplant. (F) Fraction of donor B cells that proliferated after adoptive transfer from independent experiments using three different Pacs1+/+ and Pacs1−/− donor pairs. Each symbol represents an individual recipient. Horizontal bars indicate median value. Two-tailed unpaired t test, **P < 0.01, ***P < 0.001. (G) Fraction of donor B cells that were Annexin V positive after adoptive transfer from two independent experiments using two different Pacs1+/+ and Pacs1−/− donor pairs. Each symbol represents an individual recipient. Horizontal bars indicate median value. Two-tailed unpaired t test, **P < 0.01, ***P < 0.001. (H, I) Pacs1+/+ and Pacs1-/- mice were injected with EdU and the fraction of EdU+ FOB and MZB cells were measured in the spleen at 1, 4, and 7 days post-injection. Data are from one experiment. \n",
      ": Pacs1 deletion suppresses abnormal lymphocyte accumulation in models of lymphoproliferation (A) Spleen size and FACS analysis of abnormally expanded B220+CD23+CD21+/low FOB cells in Pacs1+/−;Bcl2TG and Pacs1−/−;Bcl2TG mice.Figure 8(A) Spleen size and FACS analysis of abnormally expanded B220+CD23+CD21+/low FOB cells in Pacs1+/−;Bcl2TG and Pacs1−/−;Bcl2TG mice. (B) Number of circulating B cells in the blood and FOB cells in the spleen of Pacs1+/−;Bcl2TG and Pacs1−/−;Bcl2TG mice. Horizontal bars indicate mean value. Two-tailed Mann-Whitney test, *P < 0.05, **P < 0.01. (C) B cells were purified from the spleens of Pacs1+/−;Bcl2TG and Pacs1−/−;Bcl2TG mice (CD45.2), labelled with CTFR and CTV proliferation dyes, respectively, and transplanted into unirradiated CD45.1 recipients. Donor B cells were measured in the spleen of recipient mice 7 days after B cell transfer based on CD45.2 expression and proliferation dye fluorescence. (D, E) The fraction of proliferating (D) and recovered (E) donor cells from the experiment in (C). Symbols represent individual recipient mice and data are from two independent adoptive transfer experiments. Two-tailed unpaired t test, **P < 0.01, ***P < 0.001. (F) Fraction of apoptotic B cells in the adoptively transferred B cell populations in the experiment in (C). Symbols represent individual recipient mice and data are from one adoptive transfer experiment. Two-tailed unpaired t test, ***P < 0.001. (G) Splenocytes from Pacs1+/−;Bcl2TG and Pacs1−/−;Bcl2TG mice were stained with cell surface antibodies to identify FOB cells and treated with 100 µM H2O2 for 35 minutes. Cells were then labelled with TMRE to monitor MMP. TMRE fluorescence was measured by FACS analysis. Data are presented as mean ± SD of 3-4 replicates, unpaired t test, ns = not significant, ***P < 0.001. Results are from one experiment. (H) Lymph node size and flow cytometry of lymphoproliferative CD3+B220+ cells in Pacs1+/+;Faslpr/lpr­ and Pacs1−/−;Faslpr/lpr mice. (I) Enumeration of CD3+B220+ cells in the peripheral blood and lymph nodes of Faslpr/lpr dependent on Pacs1 expression. Horizontal bars indicate median value. Two-tailed Mann-Whitney test, **P < 0.01. \n",
      ": Pacs1 is required for normal numbers of circulating lymphocytes (A) Super-pedigree mapping of two mutations in Pacs1 that were linked to peripheral B cell deficiency.Figure 1A) Super-pedigree mapping of two mutations in Pacs1 that were linked to peripheral B cell deficiency. Insert shows peripheral B cell deficiency in the endive (N=22 REF, 28 HET, 10 VAR) and chicory (N=24 REF, 15 HET, 3 VAR) pedigrees. Horizontal bars indicate mean values. Ordinary one-way ANOVA with Tukey's multiple comparisons test. Ns = not significant, **P < 0.01, and ***P < 0.001. Protein domain model shows the encoded location of the ENU alleles. B) A 1 bp insertion in Pacs1 using CRISPR/Cas9 leads to loss of Pacs1 protein (red arrowhead). C) Peripheral blood immune cell counts from Pacs1+/+ and Pacs1−/− mice. Horizontal bars indicate mean values. Two-tailed unpaired t test, *P < 0.05, **P < 0.01, and ***P < 0.001. (D) Absolute numbers of lymphocytes subpopulations in the bone marrow, thymus, and spleen. B cell development in the bone marrow was assessed by FACS analysis for surface expression of: B220+CD43+CD19−IgM−IgD− (pre-pro B); B220+CD43+CD19+IgM−IgD− (pro B); B220+CD43−CD19+IgM−IgD− (pre B); CD19+IgM+IgD− (immature); CD19+IgM+IgD+ (mature). T cell development in the thymus was assessed by FACS analysis for surface expression of: CD4−CD8− (double negative, DN); CD4+CD8+ (double positive, DP); CD4+CD8− (CD4 single positive, SP); CD4−CD8+ (CD8 SP). Splenic B cell populations were assessed by FACS analysis for surface expression of: B220+CD21+CD23+ (follicular B cells, FOB); B220+CD21+CD23low (marginal zone B cells, MZB). Each symbol represents an individual mouse. Horizontal bars indicate mean values. Two-tailed Mann-Whitney test, ns = not significant, *P < 0.05, and **P < 0.01. E) Proportion of cell populations derived from Pacs1+/+;CD45.1 and Pacs1−/−;CD45.2 donors during competitive bone marrow reconstitution. Populations were determined based on the same markers as in (C) with the added congenic markers CD45.1 and CD45.2. Each symbol represents an individual recipient. Results are representative of two independent transplant experiments. Two-tailed unpaired t test, ns = not significant, *P < 0.05, **P < 0.01, and ***P < 0.001. F) Measurement of cell death with Annexin V staining in FOB and MZB cells from the spleens of Pacs1+/+ and Pacs1−/− mice. Results are representative of two independent experiments. Horizontal bars indicate mean values. Two-tailed unpaired t test, ns = not significant, **P < 0.01. \n",
      "a) Western blots showing the inhibitory effect of 3-MA on autophagy in I/R.f6(a) Western blots showing the inhibitory effect of 3-MA on autophagy in I/R. Full-size blots are shown in Supplementary Fig. 13. (b) TTC-stained sections showing the infarct area after administration of 3-MA (100 nmol, i.c.v., at reperfusion) in WT and H3R−/− mice (scale bar, 5 mm), and (c) bar graph showing infarct volume (n=6-8 mice per condition; *P0.05, ***P0.001, with analysis of variances (ANOVAs) followed by the Bonferroni/Dunn post hoc test). (d) TTC-stained sections (scale bar, 5 mm) and bar graph showing the infarct area after administration of thioperamide in Atg5+/−mice (n=4-6 mice per condition). (e) In cultured neurons, viability was tested by MTT assay after 3-MA (2.5 mmol l−1 at reperfusion) and thioperamide was administered in OGD/R (n=7 per condition; ***P0.001, with ANOVAs followed by the Bonferroni/Dunn post hoc test). (f) In cultured neurons, viability was tested by MTT after siRNA for Atg7 and OGD/R (n=7 per condition; ***P0.001, with ANOVAs followed by the Bonferroni/Dunn post hoc test). (g) Viability was tested by MTT assay on Atg5+/+and Atg5−/− MEFs after thioperamide administration after OGD/R (n=7 per condition; **P0.01, ***P0.001, with ANOVAs followed by the Bonferroni/Dunn post hoc test). Data are presented as mean±s.e.m. \n",
      "a) Representative immunohistochemical staining of LC3 (green) along with NeuN staining (red) in WT and H3R−/− mice (scale bar, 25 μm).f5(a) Representative immunohistochemical staining of LC3 (green) along with NeuN staining (red) in WT and H3R−/− mice (scale bar, 25 μm). (b,c) When thioperamide was administered in vivo, LC3 expression was assessed by western blot after I/R in WT mice (b) (*P0.05, ***P0.001; n=6 mice per condition, with analysis of variances (ANOVAs) followed by the Bonferroni/Dunn post hoc test), and in H3R−/− mice (c) (***P0.001 versus H3R−/− sham group; n=5-6 mice per condition, with ANOVAs followed by the Bonferroni/Dunn post hoc test.) Data are presented as mean±s.e.m. Full-size blots are shown in Supplementary Fig. 12. \n",
      "a) 2,3,5-Triphenyltetrazolium chloride (TTC)-stained brain sections from WT and HDC−/− mice showing the infarct area in those receiving saline, the H3R antagonist thioperamide (THIO), the H1R antagonist pyrilamine (PYRI; 10 mg kg−1, i.f3(a) 2,3,5-Triphenyltetrazolium chloride (TTC)-stained brain sections from WT and HDC−/− mice showing the infarct area in those receiving saline, the H3R antagonist thioperamide (THIO), the H1R antagonist pyrilamine (PYRI; 10 mg kg−1, i.p., at reperfusion), the H2R antagonist cimetidine (CIME; 10 mg kg−1, i.p., at reperfusion) and α-FMH (50 mg kg−1, i.p., 2 h before ischemia) after 24 h reperfusion (n=6 per condition; scale bar, 5 mm). (b) Infarct volumes and (c) neurological deficit scores (*P0.05, **P0.01, ***P0.001 with analysis of variances followed by the Bonferroni/Dunn post hoc test). Data are presented as mean±s.e.m. \n",
      "a) H3R expression in cultured rat cortical neurons at DIV7 after OGD/R injury (2 h OGD and 24 h reperfusion) shown by western blot (n=6 per condition; **P0.f1(a) H3R expression in cultured ratcortical neurons at DIV7 after OGD/R injury (2 h OGD and 24 h reperfusion) shown by western blot (n=6 per condition; **P0.01 versus control with Student's t-test). Full-size blots are shown in Supplementary Fig. 9. (b) When the H3R antagonist thioperamide (THIO, 10−6 mol l−1 at reperfusion) and the agonist immepip (IMME, 10−6 mol l−1 at reperfusion) were administered, viability was tested in cultured neurons by 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide (MTT) assay after OGD/R (n=7 per condition; ***P0.001 with analysis of variances (ANOVAs) followed by the Bonferroni/Dunn post hoc test). (c) 2,3,5-triphenyltetrazolium chloride-stained brain sections showing the infarct area in WT and H3R−/− mice receiving saline, the H3R antagonists A331440, clobenpropit (CLOB) and thioperamide, (THIO; 5 mg kg−1, i.p., immediately at reperfusion and 6 h later), and the H3R agonist immepip (IMME, 1 μg, i.c.v., at reperfusion) after I/R (1 h MCAO followed by 24 h reperfusion; scale bar, 5 mm). The bar graphs show infarct volume (d) and neurological score (e) (n=6-8 mice per condition, *P0.05, **P0.01, ***P0.001 versus WT saline group with ANOVAs followed by the Bonferroni/Dunn post hoc test). Data are presented as mean±s.e.m. \n",
      "a) TUNEL staining of apoptotic cells in penumbral brain slices in vivo (scale bar, 100 μm) and (c) in cultured neurons when H3R was inhibited with thioperamide (scale bar, 50 μm) after tMCAO and OGD/R.f2(a) TUNEL staining of apoptotic cells in penumbral brain slices in vivo (scale bar, 100 μm) when H3R was inhibited with thioperamide (scale bar, 50 μm) after tMCAO. TUNEL-positive cells are green, and all cells are stained with 4',6-diamidino-2-phenylindole (blue). (b,d) H3R was inhibited by thioperamide, and cleaved caspase-3 protein expression is shown by western blot in WT and H3R−/−mice after tMCAO (b) (n=6-7 per condition; **P0.01, ***P0.001 with analysis of variances (ANOVAs) followed by Bonferroni/Dunn post hoc test) (c) TUNEL staining of apoptotic cells in cultured neurons when H3R was inhibited with thioperamide (scale bar, 50 μm) after OGD/R. TUNEL-positive cells are green, and all cells are stained with 4',6-diamidino-2-phenylindole (blue). (b,d) H3R was inhibited by thioperamide, and cleaved caspase-3 protein expression is shown by western blot in WT and H3R−/−mice in cultured neurons subjected to OGD/R (d) (n=8 per condition; *P0.05, ***P0.001 with ANOVAs followed by the Bonferroni/Dunn post hoc test). Data are presented as mean±s.e.m. Full-size blots are shown in Supplementary Fig. 10. \n",
      "a) Akt/GSK-3β/mTOR/P70S6K phosphorylation assessed by western blots after thioperamide administration and OGD/R.f4(a) Akt/GSK-3β/mTOR/P70S6K phosphorylation assessed by western blots after thioperamide administration and OGD/R. LiCl (5 mmol l−1 immediately at reperfusion) was used to rephosphorylate GSK-3β/mTOR/P70S6K. The digits below represent the semiquantified optidensity of the bands, the control bands were defined as 1.00. (b) Representative western blots showing the phosphorylation of Akt/GSK-3β/mTOR/P70S6K signaling in H3R−/− mice after OGD/R injury. The digits below represent the semiquantified optidensity of the bands, the control bands were defined as 1.00. (c) Representative images showing LC3-positive autophagic vacuoles (red) with 4',6-diamidino-2-phenylindole staining (blue) after administration of thioperamide and LiCl with OGD/R (scale bar, 10 μm). (d) Cell viability assessed by MTT when thioperamide and LiCl were administered after OGD/R (n=7, **P0.01, ***P0.001 with analysis of variances (ANOVAs) followed by the Bonferroni/Dunn post hoc test). (e) Representative western blots and bar graph showing the expression of LC3-I and LC3-II in steady-state autophagy and autophagic flux under OGD/R injury when thioperamide (THIO) and immepip (IMME) were administered (n=6; **P0.01, ***P0.001 with ANOVAs followed by the Bonferroni/Dunn post hoc test). Data are presented as mean±s.e.m. Full-size blots are shown in Supplementary Fig. 11. \n",
      "H3R antagonist binds with H3R and subsequently releases its binding with CLIC4.f8H3R antagonist binds with H3R and subsequently releases its binding with CLIC4. This process inhibits PI3K/Akt/GSK-3β/mTOR signalling, that activates autophagy. \n",
      "a) Representative co-immunoprecipitation results showing the effect of thioperamide on the interaction of H3R with CLIC4 during OGD/R.f7(a) Representative co-immunoprecipitation results showing the effect of thioperamide on the interaction of H3R with CLIC4 during OGD/R. Full-size blots are shown in Supplementary Fig. 14. (b) Cell viability was tested by MTT assay after administration of Tat-H3RCT414-436 (10−6 mol l−1 at reperfusion) under OGD/R (data are presented as mean±s.e.m.; n=7 per condition; *P0.05, **P0.01, with analysis of variances (ANOVAs) followed by the Bonferroni/Dunn post hoc test). (c) Representative western blots showing the effect of Tat-H3RCT414-436 on phosphorylation of Akt/GSK-3β/mTOR/P70S6K signalling and LC3 expression in OGD/R. Full-size blots are shown in Supplementary Fig. 14. (d) Representative images of the effect of Tat-H3RCT414-436 on LC3-positive autophagic puncta (red) in 4',6-diamidino-2-phenylindole-stained (blue) neurons after OGD/R (scale bar,10 μm). (e) Western blots showing the effect of siRNA for CLIC4 on phosphorylation of Akt/GSK-3β/mTOR/P70S6K signalling and LC3 expression in OGD/R. Full-size blots are shown in Supplementary Fig. 14. (f) Representative western blots and bar graph showing the expression of LC3-I and LC3-II on steady-state autophagy and autophagic flux under OGD/R when CLIC4 was knocked down (data are presented as mean±s.e.m.; n=6; *P0.05, **P0.01; , with ANOVAs followed by the Bonferroni/Dunn post hoc test). Full-size blots are shown in Supplementary Fig. 14. (g) Cell viability was assessed by MTT assay after siRNA for CLIC4 under OGD/R (data are presented as mean±s.e.m.; n=8 per condition; ***P0.001, with ANOVAs followed by the Bonferroni/Dunn post hoc test). \n",
      "a) Suppression of USP33 facilitates TBK1–SEC5 complex formation.figf4(a) Suppression of USP33 facilitates TBK1-SEC5 complex formation. Flag-tagged SEC5 was introduced into HEK293T cells expressing empty vector (V) or HA-tagged RALB-G23V together with GFP shRNA or USP33 shRNA as indicated, and then immunoprecipitated with anti-Flag (M2) agarose followed by immunoblotting using antibody specific for TBK1. (c) RALB ubiquitylation at Lys 47 is crucial for formation of the RALB-SEC5 complex in response to poly(I:C) treatment. Flag-tagged SEC5 and the indicated HA-tagged RALB mutants were introduced into 293-hTLR3 cells. At 48 h after transfection cells were treated 100 μg ml−1 of poly(I:C) for 6 h. Flag-tagged SEC5 was immunoprecipitated with anti-Flag (M2), and analysed for co-precipitation by immunoblotting using anti-HA antibody. (c) RALB ubiquitylation at Lys 47 is crucial for formation of the RALB-SEC5 complex in response to poly(I:C) treatment. Flag-tagged SEC5 and the indicated HA-tagged RALB mutants were introduced into 293-hTLR3 cells. At 48 h after transfection cells were treated 100 μg ml−1 of poly(I:C) for 6 h. Flag-tagged SEC5 was immunoprecipitated with anti-Flag (M2), and analysed for co-precipitation by immunoblotting using anti-HA antibody. (d) RALB ubiquitylation at Lys 47 affects activation of TBK1 and IRF3. At 48 h after transfection with empty vector (V), Flag-RALB-G23V and Flag-RALB-G23V-K47R, 293-hTLR3 cells were treated with 10 μg ml−1 of poly(I:C) for 6 h. Immunoblot analysis was performed using the indicated antibodies. Uncropped images of blots are shown in Supplementary Fig. S7. \n",
      "a) Tandem affinity purification of ubiquitylated RALB.figf2(a) Tandem affinity purification of ubiquitylated RALB. 6xHis-ubiquitin and Flag-RALB-G23V were co-transfected into HEK293T cells, and ubiquitylated RALB was purified using anti-Flag resin followed by Co2+ metal affinity chromatography. Isolated RALB was visualized by immunoblotting using anti-RALB antibody. (b) RALB undergoes ubiquitylation on multiple sites. Schematic diagram of the RALB domains (upper panel). RALB ubiquitylation sites were identified in two independent tandem mass spectrometry analyses with a total coverage of 63% of the RALB amino acid sequence (Supplementary Fig. S4a). The major identified ubiquitylation sites, Lys 47 and Lys 160, are located in the effector-interacting domain (green) and the G5 box (red), respectively. (c) Substitution of arginine for lysine at position 47 significantly inhibits ubiquitylation of RALB. ( (d) USP33 regulates RALB ubiquitylation at Lys 47. (e) The GTP-GDP status of RALB affects RALB ubiquitylation at Lys 47. For c-e, 6xHis-tagged ubiquitin and the indicated RALB mutants were introduced into HEK293T cells expressing GFP shRNA or USP33 shRNA. Ubiquitylated RALB was purified by Co2+ metal affinity chromatography and detected by antibodies specific to RALB. WT, wild type. (f) Lack of ubiquitylation at Lys 47 inhibits RALB binding to SEC5. The indicated Flag-tagged RALB mutants were overexpressed in HEK293T cells and then immunoprecipitated with anti-Flag (M2) agarose followed by immunoblotting using anti-SEC5 antibody. g) Ubiquitylation at Lys 47 impairs RALB binding to EXO84. Flag-tagged RALB mutants were overexpressed in HEK293T cells and then immunoprecipitated with anti-Flag (M2) agarose followed by immunoblotting using anti-EXO84 antibody. (h) Ubiquitylation at Lys 47 does not affect RALB activity. The activity of RALB mutants was determined by a RALBP1-RBD binding assay. (i) The Rosetta v3 protocol described in ref. was used for ubiquitin-docking simulations. The plot represents the energies of the produced decoys distributed by the root-mean-square deviation (RMSD) of ubiquitin position relative to RALA from an arbitrarily chosen decoy. (j) Rosetta modelling of ubiquitylated-RALA-SEC5-RBD and ubiquitylated-RALA-EXO84-RBD complexes. Superposition of the top ten refined solutions of ubiquitylated-RALA-SEC5-RBD complex, selected by combined scores of two independent docking simulations of ubiquitylated-RALA-SEC5-RBD and ubiquitylated-RALA (left). Simulated ubiquitin positions linked to Lys 47 of RALA were combined with the RALA-EXO84 complex (right). Uncropped images of blots are shown in Supplementary Fig. S7. \n",
      "a) Schematic outline of the MAPPIT technique.figf1(a) Schematic outline of the MAPPIT technique. The indicated RAL mutants (baits) were fused to a leptin receptor fragment that is deficient in STAT3 recruitment sites. The RAL downstream effectors (preys) were tethered to a GP130 cytokine receptor fragment containing functional STAT3-docking sites. Association of bait and prey proteins restores a functional leptin receptor complex. On leptin stimulation, STAT3 activation can be monitored by a STAT3-responsive luciferase reporter. WT, wild type. (b) MAPPIT screen identified RAL-interacting proteins. A MAPPIT array covering close to 8,500 preys was screened with RALA-G23V as bait. Data are shown in a volcano plot of the log ratio of normalized MAPPIT luciferase activity versus P value Source data can be found in Supplementary Table S1. (c) MAPPIT assay confirmed interaction between the RAL proteins and USP33. RALA-G23V or RALB-G23V baits were introduced into HEK293T cells together with the indicated preys. The results are expressed as a mean of normalized luciferase activity ± s.d (leptin-treated cells versus leptin-untreated cells) for three independent experiments. Statistics source data can be found in Supplementary Table S1. (d,e) USP33 interacts with both RALA and RALB. At 48 h post-transfection with tagged USP33 and RAL expression constructs, the indicated proteins were then immunoprecipitated with anti-Flag (M2) or anti-HA-agarose, followed by immunoblotting using anti-Flag or anti-HA antibodies. WCL, whole-cell lysate; IP, immunoprecipitates. (f) Endogenous USP33 interacts with RALB. Flag-RALB was immunoprecipitated using anti-Flag (M2) agarose and analysed for co-precipitation with endogenous USP33 using anti-USP33 antibody. Cells expressing USP33 shRNA were used as a negative control. (g) GTP binding is not required for interaction between USP33 and the RAL proteins. The GST-tagged RALA and RALB were isolated with glutathione-agarose and mixed with lysates derived from HEK293T cells expressing Flag-USP33 in the presence of GTPγS or excess of GDP followed by immunoblotting with Flag-specific antibody. (h) USP33 affects ubiquitylation of RALB, but not RALA. 6xHis-tagged ubiquitin and Flag-RAL-G23V mutants were introduced into HEK293T cells stably expressing GFP shRNA or USP33 shRNA. Ubiquitylated RAL proteins were purified by Co2+ metal affinity chromatography and detected by antibodies specific to RALA or RALB. Uncropped images of blots are shown in Supplementary Fig. S7. (g) GTP binding is not required for interaction between USP33 and the RAL proteins. The USP33-tagged RALA and RALB were isolated with glutathione-agarose and mixed with lysates derived from HEK293T cells expressing Flag-USP33 in the presence of GTPγS or excess of GDP followed by immunoblotting with Flag-specific antibody. \n",
      "a) USP33 does not affect RALB activity.figf3(a) USP33 does not affect RALB activity. The activity of RALB was determined in HEK293T cells expressing GFP shRNA or USP33 shRNA and empty vector (V) or HA-tagged USP33 by RALBP1-RBD binding assay. (b) USP33 modulates interaction between RALB and SEC5. Flag-tagged RALB was overexpressed in HEK293T cells expressing GFP shRNA, USP33 shRNA, empty vector (V) or HA-tagged USP33. The indicated proteins were then immunoprecipitated with anti-Flag (M2) agarose followed by immunoblotting using anti-Flag or anti-SEC5 antibodies. (c) USP33 regulates RALB binding to EXO84. Flag-tagged RALB was immunoprecipitated using anti-Flag (M2) agarose from HEK293T cells expressing the indicated constructs. The presence of EXO84 in RALB complexes was analysed by immunoblotting using anti-HA or anti-EXO84 antibody. (d) Activated RALB co-localizes with both EXO84 and SEC5. At 24 h after co-transfection with RALB-G23V-EGFP and HA-EXO84, HeLa cells were immunostained with anti-HA and anti-SEC5 antibodies. Scale bars, 10 μm. The outlined areas are shown at higher magnification in the bottom panels. (e) RALB ubiquitylation controls interaction between EXO84 and SEC5. Flag-tagged SEC5 was immunoprecipitated using anti-Flag (M2) agarose from HEK293T cells overexpressing RALB-G23V or RALB-G23V-K47R. The presence of EXO84 in SEC5 complexes was analysed by immunoblotting using anti-EXO84 antibody. (f) USP33 regulates EXO84-SEC5 complex formation. Flag-tagged SEC5 was immunoprecipitated using anti-Flag (M2) agarose from HEK293T cells expressing the indicated constructs followed by immunoblotting using anti-EXO84 antibody. (g) Suppression of SEC5 affects RALB binding to EXO84. HA-tagged RALB-G23V was immunoprecipitated from HEK293T cells expressing the indicated constructs. The presence of EXO84 in RALB complexes was analysed by immunoblotting using anti-EXO84 antibody. For a-c,f,g immunoblotting using antibodies specific for USP33, anti-Flag (M2) or anti-HA determined the levels of USP33 suppression or overexpression. Immunoblot analysis was performed using the indicated antibodies. Uncropped images of blots are shown in Supplementary Fig. S7. \n",
      ".Figure 4(A) Schematic representation of MS2/PP7 stem-loop repeats at the 5'UTR of GCG1/SUT098 sequence, respectively. Upon transcription, respective coat proteins (dark blue and dark purple) with fluorescent proteins (red and green) bind to the MS2/PP7 repeats at the 5' UTR of GCG1 and SUT098 transcript. The ON and OFF state of transcription versus the time determines the duration and frequency (time between transcription events) of transcription. (B) An example trace of SUT098 transcript fluorescence quantified at the transcription site in a single cell (wild type). The track shows an overlay of the raw trace (green) with the binarized trace (black). The binarized peaks represent transcription initiation. (C) A representative image of transcription initiation observed as fluorescence spots in the recorded live-cell movie of wild type strain. The movie output using Python were analyzed and the snapshot was obtained using Image J software. The red and green channel denotes GCG1 and SUT098 transcript initiation, respectively. Images represent maximum intensity projections, and the right and bottom sidebars indicate sideviews in the yz and xz directions, respectively. Scale: 5 µm. (D, E) Quantification of transcription dynamics by the duration and frequency (time between initiation) of transcription. Bar graphs show the transcription parameters of SUT098 in candidate mutants compared to the wild type strain. The error bars represent SEM calculated by live-cell image analysis of 179, 135, 60 and 129 single-cell biological repeats for WT, hda1∆, hda3∆ and cac2∆ strains, respectively. Statistical significance was calculated by random sampling with replacement method using a bootstrapping algorithm in Python (the asterisk * denotes p-value <0.05). \n",
      ".Figure 3. (A) Metagene analysis of NET-seq signal at all DNC loci (n=1517). The genomic intervals were centered at the transcription start site (TSS) of either protein-coding gene (left panel) or DNC transcript (right panel) . (B) NET-seq data at the YPL172C locus. The signal represents NET-seq reads showing nascent transcription at the genomic position of the divergent non-coding strand (black) and the coding strand (grey). The bar graph depicts NET-seq reads as FPKM values for the coding and DNC transcript in the strains . (C) NET-seq data at the YDR261C locus with a novel DNC transcript. Annotations as in (B) \n",
      ".Figure 1(A) NET-seq data of wild type (WT) yeast at the GCG1/SUT098 and ORC2/SUT014 loci. NET-seq reads in black and grey represent the Watson (+) and Crick (-) strands, respectively. The NET-seq track combines remapped data from previous publications (see Material and Methods for more details). (B) Schematic representation of a shared promoter region initiating a coding (blue) and a DNC (purple) transcript in the opposite orientation. Fluorescent protein reporter (FPR) construction replaces the endogenous coding and DNC genomic regions with sequences encoding mCherry and YFP, respectively. (C) SGA and flow cytometry analysis in budding yeast. The array involves the crossing of MATα query strain harboring the FPR construct with the MATa non-essential gene deletion mutant library. The systematic selection as described in Baryshnikova et al, 2010 results in haploids containing the gene deletion and FPR. Quantification of the resultant haploids by high-throughput flow cytometry records fluorescent signals in up to 50,000 cells for each gene deletion mutant. (D, E) Scatter plot visualization of YFP vs mCherry fluorescence signals in the GCG1 and ORC2 promoter FPR screens. Each data point represents the median signal of a deletion mutant normalized to individual plate median. Highlighted data points represent the mutants of Hda1C (purple), SAGA acetylation module (blue), SAGA deubiquitination module (orange), and cac2∆ (yellow). The regression line is marked in black. The mutants favoring DNC transcription are found above the regression line. The mutants favoring coding transcription are below the regression line. Technical repeat = 1, biological observations = 50,000 individual cells before filtering. \n",
      ".Figure 5(A) Flow cytometry data analysis. Bars represent the mCherry/YFP fluorescence ratio of the respective yeast mutants normalized to the signal values of the H3 wild type FPR strain. The error bars show SEM calculated from biological triplicates with three technical repeats. Unpaired two-tailed student's t-test analysis indicates the statistical significance of the mutant compared to the respective wild type. The asterisks *, ** and *** indicate p-values <0.05, <0.01 and <0.001, respectively. (B) Metagene profile of ChIP data (Data ref: Ha et al, 2019). The genomic intervals cover the first 500 bp of the coding gene (scaled to 100 bins), the first 1 kb of the DNC transcript (scaled to 200 bins), and the variable length gap between the coding TSS and the DNC TSS (scaled to 50 bins). The vertical lines represent transcription start site (TSS) of protein-coding gene (left) and the DNC transcript (0 bin). The H3ac ChIP-seq signal was normalized by the H3 signal in the same samples. (C) Working model of Hda1C repressing DNC. The coding (blue) and divergent (purple) transcript arise in opposing directions of a shared nucleosome-free region (black). Nucleosomes (grey) comprise histones capable of modification by histone deacetylase 1 complex (Hda1C). In the normal state, Hda1C limits the frequency of DNC by deacetylating histones. With the loss of Hda1C function, increased acetylation of H3 favors DNC.  \n",
      ".Figure 2(A) Illustration of the expected distribution of data points combined from independent genetic reporter screens. Each data point represents the directionality score of a deletion mutant. Mutants altering directionality positively by increasing divergent YFP levels are expected to be found in quadrant I. Mutants decreasing YFP with negative directionality scores are found in quadrant III. Data points closer to the diagonal (grey region) have high reproducibility between the screens. (B) Distribution of mutants by directionality scores from the GCG1pr and ORC2pr genetic screens. Quadrants I and III reveal the non-essential mutant subunits of several protein complexes (highlighted in colors) and mutants involved in CAF-I pathway (square data points) altering the directionality. The top candidate repressors and activators of DNC are labeled. (C) Flow cytometry quantification of mCherry/YFP fluorescence in the mutants with GCG1pr FPR background. The data are normalized to the signal values of the wild type (FPR) strain. (D) Endogenous transcript analysis by RT-qPCR in the mutants. The bars represent fold gene expression ratio of coding and divergent transcript normalized to the expression of the reference gene, UBC6. \n",
      "a) USP33 depletion inhibits accumulation of LC3–lipid conjugates.figf6(a) USP33 depletion inhibits accumulation of LC3-lipid conjugates. HEK TE cells stably expressing GFP shRNA or USP33 shRNA were incubated with DMEM or HBSS medium for 4 h, in the presence or absence of 50 μM of chloroquine (CQ). Cells were then assayed for the relative accumulation of LC3-I and LC3-II by immunoblotting analysis using anti-LC3 antibody. LC3-II/GAPDH ratios have been calculated using ImageJ densitometric analysis. (b-d) USP33 expression levels affect the accumulation of LC3punctae. Wild-type USP33 or a USP33-shRNA-resistant USP33 mutant (USP33R) was introduced into HEK TE cells stably expressing GFP shRNA or USP33 shRNA. The generated cells were incubated in HBSS medium for 4 h. USP33 was analysed by immunoblotting using anti-USP33 antibody (b). Representative images of LC3immunostaining are shown (c). Endogenous LC3punctae were quantified using the In Cell Analyser 2000 automatic imaging system and IN Cell Investigator software. The mean distribution of LC3punctae per cell is represented as the mean of two independent experiments. Statistics source data for d can be found in Supplementary Table S1 (d). (e) USP33 depletion leads to increased levels of p62. HEK TE cells expressing GFP shRNA or USP33 shRNA were incubated in HBSS medium for the indicated periods of time. Immunoblotting was performed using the indicated antibodies. (f) Overexpression of the RALB-G23V-K47R mutant leads to accumulation of LC3-lipid conjugates. HEK293T cells overexpressing the RALB-G23V or RALB-G23V-K47R mutants were assayed for the relative accumulation of LC3-I and LC3-II by immunoblotting using anti-LC3 antibody. \n",
      "a–d) USP33 inhibits assembly of SEC5–beclin-1 complexes.figf5(a-d) USP33 inhibits assembly of SEC5-beclin-1 complexes. The indicated proteins were immunoprecipitated from cells expressing GFP shRNA or USP33 shRNA with anti-Flag (M2), anti-HA or anti-SEC5 antibody and analysed for co-precipitation by immunoblotting using specific antibodies as indicated. (e) USP33 potentiates the interaction between EXO84 and beclin 1. Endogenous beclin 1 was immunoprecipitated from HEK293T cells, expressing either empty vector (V), HA-USP33 wild type (WT) or the catalytically inactive mutant C194S-H683Q, using anti-beclin 1 antibody. Immunoprecipitates were analysed for co-precipitation by immunoblotting using anti-EXO84 or anti-beclin 1 antibodies. (f,g) RALB but not RALA undergoes deubiquitylation under nutrient deprivation. 6xHis-tagged ubiquitin and the indicated RAL mutants were overexpressed in HEK293T cells. At 48 h after transfection cells were deprived of nutrients and incubated in Hank's buffered salt solution (HBSS) medium for 90 min. Ubiquitylated RALproteins were purified by Co2+ metal affinity chromatography and analysed by immunoblotting using anti-RALA or anti-RALB antibodies. (h) Nutrient starvation results in the accumulation of USP33. HEK TE cells were incubated in HBSS medium for the indicated periods of time. Immunoblotting was performed using anti-USP33 antibody. (i) USP33 co-localizes with RALB-positive vesicles in the absence of nutrients. At 48 h after co-transfection with RALB-G23V-EGFP and USP33-mCherry, HeLa cells were incubated in HBSS medium for 90 min. Cells were imaged for EGFP and mCherry fluorescence. Scale bars, 10 μm. The outlined areas are shown at higher magnification in the bottom panels. Arrows indicate USP33 co-localization to RALB-positive vesicles. (j) Beclin 1 co-localizes with USP33. HeLa cells were transfected with beclin-1-HA and USP33-mCherry. At 48 h after transfection, cells were incubated in HBSS medium for 90 minutes and treated as in i. Scale bars, 10 μm. The outlined areas are shown at higher magnification in the bottom panels. Uncropped images of blots are shown in Supplementary Fig. S7. \n",
      ": Mesenchymal PDAC cells reduce proliferation and alter the morphology of PSCs.Figure 2(A) Brightfield images of PS-1 cells which were exposed for 72 hrs to CM from control (CM of PS-1 cells ), epithelial (blue) or mesenchymal (red) cell lines. Scale bar represents 100 µm. (B) Quantification of bead-calibrated cell count of PS-1 cells after treatment using flowcytometry. n=6 per group, 3 biological replicates for each cell line. (C) EdU assay performed after treatment showing percentage of PS-1 cells in G0/G1 phase, G2/M phase and S phase. n=6 per group, 3 biological replicates for each cell line.. \n",
      ".Figure 1(A) Picrosirius red (PSR) staining of collagens (red) in PDAC tissue following surgical resection. Scale bar represents 200 µm. (B) Quantification of PSR in low-grade (1-2) and high-grade (3) tumors indicated in panel A as percentage of area. n=15 primary PDAC samples. Data information: data are represented as mean±SD, Student's t-test. (C) Quantification of tumor cell percentage as scored by pathologists in low- and high-grade PDAC indicated in panel A. n=15 primary PDAC samples. Data information: data are represented as mean±SD, Student's t-test. (D) PSR and α-SMA staining of indicated PDAC cell lines which were injected in NSG mice. Scale bar represents 200 µm for PSR and 100 µm for α-SMA images. (E) Quantification of stainings indicated in panel D. Three tumor ROIs of a tumor derived from epithelial cell lines (Capan-2 and AsPC-1) and mesenchymal cell lines (MIA PaCa-2 and PANC-1) are displayed and averages were pooled for analysis. Data information: data are represented as mean±SD, Student's t-test. (F) IHC staining of α-SMA in 3D organotypic co-cultures of PS-1 cells and indicated PDAC cell lines. Scale bar represents 100 µm. \n",
      ": High-grade CSF-1-positive PDAC features deactivated stroma.Figure 6(A) IHC staining of CSF-1 in PDAC tissue following surgical resection representative image of CSF-1 expression in grade 1 and grade 3 PDAC is depicted. Scale bar represents 100 µm. (B) For each tumor of panel A, CSF-1 expression in ductal adenocarcinoma cells was quantified. Student's t-test. Shown is mean±SD. n=15 primary PDAC samples. (C) A validation cohort of PDAC patients (n=21) that underwent direct resection was stained for CSF-1 (scale bar represents 100 µm), PSR (scale bar represents 500 µm) and α-SMA (scale bar represents 200 µm). OK8 is low in CSF-1 expression and high in PSR and α-SMA expression, while the opposite accounts for OK7. (D) Correlation of % of PSR in the tumor and CSF-1 expressing tumor cells, as indicated in panel C. R2 and P-value were analyzed with linear regression. n=21 primary PDAC samples (E) Correlation of % of α-SMA in the tumor and CSF-1 expressing tumor cells, as indicated in panel C, R2 and P-value were analyzed with linear regression. n=21 primary PDAC samples \n",
      ": PSC deactivation is mediated by colony stimulating factor 1.Figure 4(A) Brightfield images of PS-1 cells after 72 hrs of treatment with unfiltered, <100 kDa or <30 kDa filtered CM of indicated cell lines. Scale bar represents 100 µm. (B) Gene expression levels of COL1A1 and ACTA2 in PS-1 cells after treatment using qPCR. Data was normalized against control CM. Statistical significance was tested against unfiltered CM samples. Graph shows n=3 technical replicates of an example experiment that was performed three times. Data information: data are represented as mean±SD, Student's t-test. (C) Graphical representation of 64 cyto- and chemokines present in CM of epithelial and mesenchymal PDAC cell lines using forward-phase protein array. Significantly differential expression of proteins (triangles) and genes (green triangles) are identified between epithelial and mesenchymal cell lines. Dotted line indicates P=0.05. Data information: data are represented as mean±SD, Student's t-test. (D) Brightfield images of PS-1 cells after 72 hrs of treatment with CM of control (CM of PS-1s) or epithelial PDAC cells supplemented with recombinant human CSF-1. Scale bar represents 100 µm. (E) Gene expression levels of COL1A1 in PS-1 cells after treatment using qPCR. n=3 biological replicates per group. Data information: data are represented as mean±SD, Student's t-test. (F) PS-1 cells were starved and stimulated with 500 ng/ml rh-CSF-1 at indicated time points. Activation of pathways involved in CSF1R signaling was analyzed using western blot. Samples were derived from the same experiment and blots were processed in parallel. \n",
      ": Mesenchymal-like PDAC cells deactivate PSCs.Figure 3(A) Gene expression level of stromal activation markers COL1A1, ACTA2, SPARC, FN1 and FAP in PS-1 cells which were exposed for 72 hrs to CM from control (CM of PS-1 cells ), epithelial (blue) or mesenchymal (red) PDAC cell lines using qPCR. Values were normalized to control and n=6 per group, 3 biological replicates for each cell line. Data information: data are represented as mean±SD, Student's t-test. (B) Heatmap representation of COL1A1 and ACTA2 relative gene expression levels measured with qPCR in PS-1 cells after treatment using a large cohort of PDAC cell lines and primary cell cultures 67 and 53m. Values were Log2 transformed and normalized to Suit-2 expression. Data information: data are represented as mean±SD, Student's t-test. (C) Brightfield images of PS-1 cells exposed to 10 days of treatment with 1 and 10 µM all-trans retinoic acid (ATRA). Scale bar represents 100 µm. (D) Gene expression levels of COL1A1 and ACTA2 in PS-1 cells after treatment using qPCR, values were normalized to control. n=3 technical replicates per group. Data information: data are represented as mean±SD, Student's t-test. (E) Brightfield images of primary human CAFs exposed to 72 hrs of treatment with CM from control (CM of primary CAFs) or mesenchymal PDAC cell lines. Scale bar represents 100 µm. (F) Relative gene expression levels of COL1A1 and ACTA2 in primary CAFs after treatment using qPCR, values were normalized to control. n=3 technical replicates per group. Data information: data are represented as mean±SD, Student's t-test. \n",
      ": Targeting CSF-1 in mesenchymal-like PDAC cells abolishes PSC deactivation.Figure 5(A) Brightfield images of PS-1 cells after 72 hrs of treatment with CM of control (CM of PS-1s) or mesenchymal PDAC cell lines supplemented with anti-CSF-1. Scale bar represents 100 µm. (B) Gene expression levels of COL1A1 in PS-1 cells after treatment using qPCR. Graph shows n=3 technical replicates of an example experiment that was performed three times. Data information: data are represented as mean±SD, Student's t-test. (C) Brightfield images of PS-1 cells after 72 hrs of treatment with PANC-1 CM of pLKOctrl or shCSF1 (#560-562). Scale bar represents 100 µm. (D) Correlation graph of CSF1 expression in PANC-1 pLKOctrl or shCSF1 (#560-562) and how these various CM affect COL1A1 expression in PS-1 cells using qPCR, values were normalized for RPLPO expression. n=3 biological replicates for all individual experiments, error bars indicated SD. R2 and P-value were analyzed with linear regression. (E) Brightfield images of PS-1 cells after 72 hrs of treatment with CM of Capan-2 cells with control vector or Capan-2 cells overexpressing CSF1. Scale bar represents 100 µm. (F) Gene expression level of CSF1 in Capan-2 cells after transduction with control vector or overexpressing CSF1. n=3 technical replicates per group. Data information: data are represented as mean±SD, Student's t-test. (G) Gene expression levels of COL1A1 in PS-1 cells after treatment indicated using qPCR. Graph shows n=3 technical replicates of an example experiment that was performed three times. Data information: data are represented as mean±SD, Student's t-test. \n",
      " | Liquid-like Properties of the Rotavirus Replication Factories.Figure 1(a) Dynamics of replication factories tagged with EGFP (NSP5-EGFP) and mCherry (NSP2-mCherry) visualized in MA104-NSP5-EGFP and MA104-NSP2-mCherry rotavirus-infected cell lines. Live-cell confocal images (4-16 min) acquired after 4 hours post infection (HPI). Scale bars, 5 µm. (b) Representative electron micrograph of a viroplasm (V) formed in NSP5-EGFP cells infected with rotaviruses 8 hours post infection. Inset - double-layered particles emerging from the surface of a viroplasm. Scale bar = 300 nm. \n",
      " | Changes in Liquid-like properties of viroplasms during the course of infection.Figure 2(a) Fluorescence recovery after photobleaching (FRAP) of EGFP-tagged replication factories after 4 HPI (early infection) and 12 HPI (late infection). Scale bar, 10 μm. (b) Fluorescence Intensities after FRAP of EGFP-tagged replication factories after 4 HPI (blue) and 12 HPI (red) shown in (a). Each data point represents mean±SD intensity values calculated for multiple NSP5-EGFP-tagged granules in 5 RV-infected cells. (c) Sphericity of NSP5-EGFP-containing granules during RV infection. Each data point represents mean±SD sphericity values calculated for NSP5-EGFP-NSP5-tagged granules in cells detected in 15 frames. Data were recorded for 320 min immediately after 4 HPI when multiple NSP5-EGFP granules could be detected in RV-infected cells. (d) Calculated volumes of NSP5-EGFP-tagged granules formed in RV-infected cells after 4 HPI as shown in (c). The mean values decrease due to de novo formation of multiple smaller NSP5-EGFP granules that continuously assemble in cells between 4 HPI (t=0 min) and 9 HPI (t=300 min). The shaded area represents the 95% confidence interval for each point. \n",
      " | 1,6-hexanediol treatments (1,6HD) differentiate early and late viral replication factories (a) Replication factories in MA104-NSP5-EGFP cells infected with RV at 4 HPI dissolve after >30 s post application of 4% (v/v) 1,6HD added to the cell culture medium (middle panel).Figure 3(a) Replication factories in MA104-NSP5-EGFP cells infected with RV at 4 HPI dissolve after >30 s post application of 4% (v/v) 1,6HD added to the cell culture medium (middle panel). Removal of 1,6HD results reassembly of multiple EGFP-NSP5-containing droplets dispersed in the cytosol (right panel). Bottom, replication factories at 12 HPI: before application of 1,6HD (left), 2 min after application (middle), and 15 min after removal of 1,6HD from cell culture medium (left). Note larger viral factories that remain refractory to 1,6HD treatment. Scale bar, 50 µm. (b) Sensitivity of RV replication factories to aliphatic alcohols at 4 HPI. Left to right -1,6-hexanediol (1,6HD); 1,3-propylene diol (1,3PD); 1,2-propylene diol, or propylene glycol (PG); glycerol (Gly); polyethylene glycol 200 (PEG200). Top panels - before application and bottom panels - 1 min after application of these compounds (4.5% v/v). Scale bar, 30 µm. \n",
      " | Rotavirus replication factories are RNA-protein condensates sensitive to propylene glycol.Figure 6(a) RV-infected MA-NSP5-EGFP cells at 6 HPI. NSP5-EGFP-tagged viral factories (green) are dissolved in the presence of 4.7% (v/v) propylene glycol (PG, middle). Viral RNA-protein condensates rapidly reform (<10 min) after replacing the PG-containing cell culture medium (PG removed, right). Seg3 (magenta) and Seg4 (cyan) transcripts are detected by smFISH, and colocalising Seg3 and Seg4 RNA signals (white). Scale bars: 50 µm, zoomed-in regions: 10 µm. (b) Binding of NSP2 to a fluorescently labelled 20-mer ssRNA in the presence of 4.7% propylene glycol (PG), measured by fluorescence anisotropy. Data points represent mean anisotropy values±SD from three measurements (technical replicates, values normalized from 0 to 1). Effects of PG treatments on the viral production. PG-containing medium (2% or 4.7% PG), or standard cell culture medium, were applied to RV-infected cells (MOI of 10) 2 HPI (1 hour after virus absorption). (c) Viral titres (PFU/ml) of samples harvested at 11 HPI that were briefly (15 min) treated with 4.7% PG at 4 HPI (15' PG), and those of untreated RV-infected cells harvested at 11 HPI (control). Virus titres are expressed as mean±SD values estimated for four independent repeats, and were compared by a two-tailed Mann-Whitney test (no significant difference, P = 0.0857). Effects of PG treatments on the viral production. PG-containing medium (2% or 4.7% PG), or standard cell culture medium, were applied to RV-infected cells (MOI of 10) 2 HPI (1 hour after virus absorption). (d) Rotavirus titres measured after 10 h of PG treatments. The 2% PG treatment slightly lowered the titre compared to the control group (ns, P = 0.2282), while the 4.7% PG treatment significantly inhibited viral replication (P = 0.0032). Reported viral titres represent mean±SD values estimated for four independent biological replicates. Effects of PG treatments on the viral production. PG-containing medium (2% or 4.7% PG), or standard cell culture medium, were applied to RV-infected cells (MOI of 10) 2 HPI (1 hour after virus absorption). (e) Effects of PG treatments on virus production at different infection points. 4.7% (v/v) PG was applied between 2-7 HPI ('early infection'), and then diluted to 2% between 7-11 HPI, or 4.7% PG was applied between 7-11 HPI ('late infection'), and diluted prior to harvesting the virus. Each group was compared to the control group (10 h treatment with 2% PG). Each group was compared to the control group (10 h treatment with 2% PG). Application of 4.7% PG for 5 h significantly reduced virus replication (P = 0.0016) between 2-7 HPI compared to the control group. PG treatment between 7-12 HPI did not significantly reduce the viral titre compared to the control group (P = 0.1141). Infectious titres represent mean±SD values estimated for four independent biological replicates. Statistical analyses were performed using a Kruskal-Wallis test, followed by uncorrected Dunn's multiple comparisons test (**P < 0.002). (f, g) 4.7% PG treatment of RV-infected cells results in NSP5 dephosphorylation. (f) Western blot analysis of RV-infected cells harvested at 7 and 12 HPI, treated with 2% or 4.7% PG at different infection time points indicated in the figure. Multiple phosphorylated forms of NSP5 can be seen as higher MW bands. Treatment with 4.7% PG reduces phosphorylation, which is not perturbed in the presence of 2% PG. Treatments administered between 2-7 and 7-12 hpi reduce NSP5 phosphorylation, albeit to a considerably lower degree compared to the 2-12 hpi treatment. (g) Okadaic acid (5 μM) applied to 4.7% PG-treated infected cells blocks NSP5 dephosphorylation restoring its phosphorylation pattern. \n",
      " | NSP5 drives of liquid-liquid phase separation required for viroplasm formation.Figure 5(a) DeePhase analysis of the phase-separating properties of NSP5 (regions with a score of > 0.5, i.e., above the dotted line, indicate residues with high LLPS potential). The CTR region (AA 179-197) is highlighted in red. Note that the amino acid numbering represents an average score with a 30-residue sliding window, and not individual residues. Green boxes denote NSP5 residues of previously shown to be essential for viroplasm-like structure formation when co-expressed with NSP2(Eichwald et al, 2004). (b) EMBOSS protein charge plot shown as an alternating blue (negatively charged) and red (positively charged) regions with a 30-residue sliding window. (c) flDPnn Disorder Propensity Plot of NSP5. Regions with a score above the dotted line are predicted to be disordered. (d) NSP5 model predicted using Alphafold2. Region 179-197 of the CTR within the predicted C-terminal alpha helix is highlighted in red. Top panel, left to right: NSP5-∆CTR labelled with Atto-647 (25 μM) incubated with unlabeled NSP2 (10 μΜ); NSP5-∆CTR in the presence of 10% v/v PEG-20K; and with 5 μΜ of poly-arginine (polyArg). Bottom panel: Atto-647-labelled NSP5-∆CTR (5 μΜ) incubated with unlabeled NSP5 (25 μΜ) and Atto-488 labelled NSP2 (10 μΜ). NSP5/NSP2 droplets containing labelled NSP2 (green) also contain NSP5-∆CTR-Atto647 (magenta), shown along with an image of both 488/647 channels overlaid. Scale bar, 10 µm. \n",
      " | Rotavirus NSP5 and NSP2 undergo phase-separation.Figure 4(a) Recombinantly expressed, Atto647-dye labelled NSP5, Atto488-dye labelled NSP2 and unlabeled NSP5 protein samples were used for investigating their phase-separation properties. All labelled protein samples were mixed with the unlabeled NSP5 (1:10 molar ratio) to minimise the effect of labelling on LLPS. Top panel, left to right: NSP5-647/NSP2-488 droplets formed upon mixing of both proteins (10 μM each). Both channels are shown, along with an image of both channels overlaid. 4% (v/v) 1,6-hexanediol (1,6HD) dissolves these condensates. Scale bar, 10 µm. Middle panel, left to right: NSP5-647 (50 μM) + poly-arginine (polyArg, 5 μΜ); NSP5-polyArg condensate + 1,6-HD; NSP5-polyArg condensates + 0.5 M NaCl; NSP5 + R9 peptide (Arg-9, 5 μΜ). Bottom row, left to right: NSP5-647 sample (35 μM) alone; after addition of 10% v/v PEG-20K; NSP2-488 (25 μM) alone; after addition of 10% v/v PEG-20K. (b) Phase diagrams generated through droplet microfluidics for the coacervation of NSP2 and NSP5, in the presence of 0% v/v (left), 1.5% v/v (middle), and 4% v/v (right) 1,6HD. Phase diagrams were generated from N = 2206, 2035 and 1470 data points for each 1,6-hexanediol concentrations, respectively, and the data were used to construct the LLPS probability plots. (c) The number of viroplasms (black y axis on the left) and estimated cytoplasmic NSP5 concentration ([NSP5] µm, blue y axis on the right) at 2, 4, and 6 hours post infection. Viroplasms were counted in N = 595 ± 85 cells for each time point, and intracellular NSP5 concentration was determined by quantitative Western blotting \n",
      " | Super-resolution DNA-PAINT analysis of RV replication factories.Figure 7(a) DNA-labelling scheme of NSP2 for DNA-PAINT imaging. Low levels of SNAP-tagged NSP2 (a doughnut-shaped NSP2 octamer) are produced in a stable MA104 cell line. A SNAP-tag reactive benzylguanine (BG) DNA derivative (P1 ssDNA docking strand) can form a stable thioether bond with NSP2-SNAP. (b) Detection of NSP2-rich condensates formed in RV-infected cells using DNA-PAINT approach. Transient binding and dissociation of a Cy3B-dye labelled ssDNA probe P1* (complementary to ssDNA P1) generates blinking at the target sites (ON/OFF) used for stochastic super-resolution imaging. (c) Schematics of smFISH/DNA-PAINT approach using orthogonal P2 ssDNA docking sites installed into Seg3-specific RNA FISH probes (d) A combined super-resolved image of NSP2-rich condensates (NSP2-SNAP in red) and Seg3 transcripts (cyan) in RV-infected cells 4 HPI. Scale bars = 2 μm (top), 500 nm (inset i, bottom). (e) 3D DNA-PAINT analysis of Seg3 RNA foci in RV-infected cells at 4 and 6 HPI. Scale bars, 2 µm (left) and 200 nm (zoomed-in, right). (f-g) Distribution of calculated volumes and sphericities of the Seg3 RNA-containing granules in RV-infected cells at 4 HPI (N=704) and 6 hpi (N=698), shown in (e). Box plots represent the 25th/75th interquartile range, with whiskers representing the 5th/95th percentile values. Medians shown as central bands, and means shown as squares. Symbol 'x' denotes 1% and 99% percentile values, and min and max values are shown as '-'. At the 0,001 level, the two distributions are significantly different between 4 and 6 hpi, assessed by the two-sample Kolmogorov-Smirnov test. \n",
      " | Proposed model of LLPS-driven formation of viral replication factories in rotaviruses.Figure 8Multivalent Asp/Glu- and Ser-rich protein NSP5 (pink) is a scaffold that recruits RNA chaperone NSP2 (cyan doughnut-shaped octamers), and other RNA-binding clients. NSP5 and NSP2 undergo coacervation at low micromolar concentrations, forming protein droplets, also known as 'viroplasm-like structures'. RV transcripts undergo enrichment in these condensates via a mechanism distinct from other better characterised RNP granules. Mechanistically, this could be achieved via a specific protein-RNA recognition, e.g., binding of the RNA-dependent RNA Polymerase (RdRP) VP1 that recognises a conserved sequence present in all RV transcripts. Such RNP complexes are then absorbed into the NSP5/NSP2 condensates consistent with low nM affinity of VP1 for both NSP5 and NSP2(Arnoldi et al, 2007; Viskovska et al, 2014). Other similarly sized, non-viral transcripts (red), devoid of these proteins do not partition into viroplasms. Other multivalent RNA-binding proteins (RBPs), i.e., viral capping enzyme (Pizarro et al, 1991), and NSP2 can also assist in partitioning of the RNP complexes into the NSP2/NSP5 condensates, which can be dissolved with aliphatic diols. Upon NSP5/NSP2 condensation, NSP5 undergoes excessive phosphorylation ('hyperphosphorylation'), which can be also reversed by dissolving these condensates. Such RNP condensates may promote RNA-RNA interactions by increasing cognate RNA concentration bound by the RNA chaperone NSP2, thus being conducive to the assembly of eleven distinct transcripts required for packaging of a multi-segmented viral genome. \n",
      "Fig.Figure 1A 3D structure of a HA trimer (PDB 3UBE generated with PyMol). Each monomer consists of a HA1 (dark grey) and HA2 (light grey) subunit. The trimer can be divided into a membrane distal head which contains the RBS (yellow) and a proximal stalk region. B HA and NP represent the Cal/7/9 wildtype proteins. Chimeric HA (cHA) 1 contains the head regions of H2 HA, cHA2 of H10 HA and cHA3 of H13 HA, while they all contain the Cal/7/9 HA-stalk region. Headless HA (HL) contains a deletion in the HA-head region (dashed line). Modified headless HA (mHL1 and mHL2) contain stabilizing mutations (black boxes) and lack additional internal parts. All constructs were codon-optimized and carry a V5-tag at their C-terminus. C Frequency of C179+ or V5-tag+ 293T cells 24h after transfection of AAV-vector plasmids as measured by flow cytometry. Symbols represent single experiments and bars the mean ± SE (n = 3). D Immunoblot of 293T cells 72 h after transduction with AAV-vectors at a MOI of 106. Antigen expression was detected with an anti-V5-tag antibody. Equal loading was controlled with a GAPDH antibody (n = 3). \n",
      "Fig.Figure 2Mice were intranasally given 1011 vg of AAV-vector in 50 µl volume three times in three-week intervals. 20 µg of Cal/7/9 WIV per 50 µl was given i.n. two times. Blood samples were taken at indicated time points (red drops). After influenza challenge, mice were monitored for survival and weight loss for two weeks before necropsy. Total IgG ELISA titers expressed as area under the curve (AUC) against homologous Cal/7/9 virus of pre-immune and pre-challenge sera of individual animals of the indicated vaccine groups (AAV-HA, -cHA, -GFP, WIV n = 18, AAV-NP n = 11). Background reactivity of pooled AAV-GFP pre-challenge serum is shown as dashed line. ELISAs were performed in technical duplicates. IgA ELISA titers in post-challenge lung homogenates against Cal/7/9 of individual mice of the indicated vaccine groups (AAV-HA, -cHA, -GFP, -NP, WIV n = 11). of the Cal/7/9 (blue symbols) and PR8 low dose (red symbols) challenge groups. ELISAs were performed in technical duplicates. D to G Total IgG ELISA titers expressed as log10 of the mean AUC against indicated viruses in pre-challenge sera in AAV-HA (D, n = 18), AAV-cHA (E, n = 18), AAV-NP (F, n = 11) or WIV (G, n = 18) immunized animals (grey area). Reactivity of sera of AAV-GFP immunized animals is indicated as white area in the center of each web diagram. ELISAs were performed in technical duplicates. \n",
      "Fig.Figure 3A Immunoblot analysis of purified H1N1 viruses A/California/7/2009 (Cal/7/9), A/Brevig Mission/1/1918 (BM/1/1918), A/Puerto Rico/8/1934 (PR8) or A/Brisbane/59/2007 (Bris/59/7) separated under reducing and denaturing conditions. Uncleaved HA0 and cleavage products HA1 and HA2 were detected with pooled pre-challenge sera (AAV-HA, -cHA, -GFPB Significant peptides identified in the epitope screen with AAV-HA pooled pre-challenge serum ((AAV-HA, -cHA, -GFP, WIV n = 18, AAV-NP n = 11; data shown in Fig.EV2) were mapped onto HA1 (red) and HA2 (blue) of the trimeric HA (PDB 3UBE generated with PyMol) (n = 3, technical duplicates). The position of the RBS (yellow) and the C179 epitope (dashed line) is marked. C to F Binding of C179 (C), or antibodies present in AAV-HA (D), AAV-cHA (E) or WIV (F) pooled pre-challenge sera (n = 18 mice per group) to Cal/7/9 or PR8 virus after incubation of the virions at pH = 7.2, 5.8, 5.4, 5.0, 4.4 or 4.4+DTT to induce conformational changes in HA or remove the HA1 subdomain. Mean ± SD (n = 3, in technical triplicates). Statistical significance between pH = 7.2 and other conditions was determined using Kruskal-Wallis test with Dunn's multiple comparison testing (*p<0.05, **p<0.01). \n",
      "Fig.Figure 4A and B Influenza virus specific activation of the murine FcγRI, FcγRII, FcγRIII and FcγRIV by antibodies in pooled pre-challenge sera (AAV-HA, -cHA, -GFP, WIV n = 18, AAV-NP n = 11 mice per group). MDCKII cells were infected with Cal/7/9 (A) or PR8 (B) before mouse serum and FcγR-expressing reporter cells were added. Influenza specific Fc-FcγR interaction induces the production of IL-2 in the FcγR-expressing reporter cell, which was measured by anti-IL2 ELISA (n = 3, technical duplicates). Mean ± SE. C HA-specific activation of FcγR by AAV-HA, AAV-cHA or WIV pooled pre-challenge sera (n = 18 mice per group). MDCKII cells were transfected with wildtype HA (pAAV-HA) or a stalk-only HA (pAAV-mHL1) before mouse serum and FcγR-I-expressing reporter cells were added (n = 2, technical duplicates). Mean ± SE. \n",
      "Fig.Figure 5A to F Mice were vaccinated three times with AAV-vectors or two times with WIV and then challenged with a lethal dose of homologous influenza virus (Cal/7/9) (A, B; n = 5) or two different lethal doses of the heterologous PR8 strain (C, D; n = 6 and E, F; n = 7). Survival was monitored over a 14-day period and is depicted as Kaplan-Meier plot (A, C, E). Weight loss was determined during the 14-day period, which is shown in relation to the weight at day 0 (B, D, F). Mean ± SE. Statistical significance between negative control group (AAV-GFP) or WIV group (E) and each vaccine group was determined using Log-Rank (Mantel-Cox) test (*p<0.05, **p<0.01). \n",
      ".Figure 4A) Histoblasts Mean Cell Area (MCA) increases over time for all (black - the whole nest), central (blue) and peripheral histoblasts (red). B) Snapshots at specific time points of segmented movies depicting histoblasts with color-coded cell areas. Color scale on the right. Scale bar = 40 µm. Note the progressive increase of the histoblasts size, specially the peripheral ones. Data information: In (B) the top panel represents the central histoblasts and the bottom panel the peripheral. We employed an arbitrary annotation and consider peripheral those cells present in the 2 outermost rows of cells all throughout the perimeter of the nest at the earliest recorded time point. C) Histoblasts Elongation Ratio decreases over time for all (black - the whole nest), central (blue) and peripheral histoblasts (red). D) Color coded snapshots of cell elongation at different time points. Color scale on the right. Scale bar = 40 µm. Data information: In (D), the top panel represents the central histoblasts and the bottom panel the peripheral. We employed an arbitrary annotation and consider peripheral those cells present in the 2 outermost rows of cells all throughout the perimeter of the nest at the earliest recorded time point. \n",
      ".Figure 2A) Snapshots of the divergence of the velocity field from a representative movie. Note the areas of positive divergence in the nests, areas of positive and negative divergence in the LECs, and an increase of magnitude on the differences between them over time. Scale bar = 40 µm. B) Frequency distribution of the divergence magnitude in the nests and LECs in the different growth phases. Nests are always shown in red and LECs in blue. C) Color coded radial kymographs of divergence for a representative movie. Low divergences are blue, high are red (see scales). Note the overall positive divergence in the nests and negative in the LECs. \n",
      ".Figure 5A) Snapshots of the velocity field generated by PIV from a representative movie. Measured velocity vectors in each point of the grid are represented in red. Data information: Scale bar = 40 µm. B) Snapshots of the computed Divergence field (expansion - red, contraction - blue) at different time points. Data information: Scale bar = 40 µm. C) HR modeled Pressure field (positive pressure - red, negative pressure - blue). D) HR modeled Power density (positive power / active contribution - red, negative power / resisting tissue - blue). Data information: Scale bar = 40 µm. E) Relative surface stress over time (positive tension / high contraction - red, negative tension / high expansion - blue). Data information: Scale bar = 40 µm. \n",
      ".Figure 3A) Cell numbers in nests (blue - left scale) and LECs territories (red - right scale) (profiles are averaged from three segmented movies). B) Mean cell area evolution of histoblasts (blue - left scale) and LECs (red - right scale). Note the progressive increase in mean histoblast area (profiles are average from three segmented movies). C) Net change rate in cell number variation for nests (red line) versus nests growth (black line). Note the temporal correlation in phases, but the reduced magnitude in cell numbers (profiles are collected from single nests and normalized; control black lines are retrieved from Figure 1). D) Net change rate in cell number variation for LECs (blue line) versus tissue growth (black line). Note the full pattern correlation and the oscillations in LECs net rate numbers (profiles are collected from single nests and normalized; control black lines are retrieved from Figure 1). E) Evolution of the net change rate of histoblasts MCA (red line) versus net change rate of nest growth (black line). Note the delay in reaching the sustained growth phase and the reduced magnitude of the MCA rate (profiles are collected from single nests and normalized; control black lines are retrieved from Figure 1). F) Evolution of the net change rate of LECs MCA (blue line) versus net change rate of LECs tissue growth (black line). Note the oscillatory MCA rate around 0 and the tendency toward negative values (MCA decrease) (profiles are collected from single nests and normalized; control black lines are retrieved from Figure 1). \n",
      "Fig.Figure 6A Ferrets (n = 4 per group) were immunized with AAV-vectors or quadrivalent inactivated vaccine (QIV). Serum was obtained at indicated time points (red drops). After challenge with a Cal/7/9-like (H1N1) pdm strain clinical signs, temperature and weight loss were monitored. At 3 dpi, animals were sacrificed and necropsied. B Cal/7/9-specific serum antibody titers in individual sera at indicated time points. Black lines: individual animals, red lines: mean titer. Statistical significance between pre-immune and immune sera was determined using Friedman test with Dunn's multiple comparison testing (*p<0.05, **p<0.01). The lower table indicates Cal/7/9-specific MN50 and HAI mean titers. Assays were performed in technical duplicates. C to E Clinical score (C), body temperature (D), and relative weight (E) during challenge period. Statistical significance between pre-immune and immune sera was determined using Friedman test with Dunn's multiple comparison testing (AAV-HA ##p<0.01; AAV-cHA §§p<0.01; AAV-GFP $p<0.05; $$p<0.01; QIV *p<0.05;). Mean ± SD F Virus titers in nasal turbinates (NT), trachea or lung homogenates. Statistical significance between AAV-GFP and vaccine groups was determined using Kruskal-Wallis test with Dunn's multiple comparison testing (*p<0.05). Lines indicate the GMT. Assays were performed in technical duplicates. G and H IHC staining of submucosal glands (SMG) (G) or bronchi (H) with an anti-H1N1 polyclonal antibody (bar: 100 µm; red: influenza antigen (neufuchsin); blue: nuclei (haematoxylin)). I and J H&E stained histological sections of bronchi (I) and lung parenchyma with alveolar epithelial cells (AEC, J). Arrowheads: areas of suppurative bronchitis/bronchiolitis; arrows: interstitial pneumonia, necrosis of AEC (bar: (I) 100 µm; (J) 50 µm). \n",
      ".Figure 1A) Ventral nests expansion at 0 min (onset); 245 min, when ventral and spiracle nests fuse; 515 min, when intersegment nests fuse; and 750 min, when most LECs are gone. We employed for these analyses the following stock: w; If/CyO; ZCL2207::hh-DsRed/MKRS. The green signal corresponds to ZCL2207 (Septate junction GFP marker) outlining all apical cell membranes and the red signal to nuclei from the posterior compartment expressing RFP under the control of the hedgehog promoter. Arrows point to the spiracular and ventral nests on the top panel and to the junction between these two nests in the second panel. Images of confocal sections have been rotated to maintain consistent alignment of the dorso-ventral axis (dorsal is up). The outlines of the confocal images are marked with a yellow dotted line. Scale bar = 40 µm. B) Comparison of manual vs. PIV surface area kinetics [evolution of nests (red - manual, black - PIV) and LECs (cyan - manual, blue -PIV)] C) Nests rate of growth from a representative single nest. Note the similarity in behavior (red - manual, black - PIV) in three sequential phases: Phase 1 (Ph 1), accelerated growth; Phase 2 (Ph 2), sustained growth; and Phase 3 (Ph 3) decelerated growth). D) Progression of the elongation ratio (red - manual, black - PIV) of nests Note the initial fast decay on the elongated shape of the nests. E) Nest orientation evolution (red - manual, black - PIV). Anterior is π (3.1415 rads, dorso ventral is π/2 (1.57 rads) and posterior is 0. \n",
      ".Figure 6A) Density Power radial kymograph of a representative time-lapse. Each horizontal line displays for each time point (left scale) the mean of the power value at every distance (bottom scale) to the nest-LECs border (vertical black line). Each phase is represented in a colored background with three different densities of purple. Values scale is on the right. Graph representing the mean (line) and standard deviation (shadow) of the power density radial kymographs (left scale) at different distances (bottom scale) by phases (in coded densities of purple) for all samples. Surface Stress radial kymograph of a representative time-lapse Phases are represented in different densities of brown. Graph representing the mean and standard deviation of the surface stress radial kymographs Values color coded in different densities of brown. \n",
      ".Figure 2A-C. Representative photomicrograph of young (6-8 weeks) and aged (>12 months) livers 40 h post PH stained with Ki67 and visualized by IHC and H E for histology. Sections were scanned using Histech Pannoramic MIDI Scanner and Ki67 positive cells were quantitated with Quant Center 2.0 software. A proliferation index (p.i.) was calculated based on the number of Ki67 positive hepatocytes to total hepatocytes and livers were classified as (+) regenerating, p.i.10% or (-) non-regenerating, p.i.10%.D. Number of young (6-8 weeks) and aged (12 months) mice with (-) non-regenerating and (+) regenerating or livers based on p.i., 40 h post PH. Animals deemed non-surviving for ethical reasons were euthanized 24 hours post PH. Representative results of a single experiment with n=9 aged and n=6 young animals per group. \n",
      ".Figure 8A) Laser microsurgery was performed at the different sides of the ventral nests during phase 2 (300 minutes) in parallel cuts [in the LEC tissue at 40 µm of the nests/LEC border (cyan) and within the nests at 10 µm (red) and 40 µm (green) of the edge]. Myo-GFP transgenic pupae were used to visualize the cortex displacements. This example illustrates at the anterior edge the laser cuts positions. Scale bar = 15 µm. B) Cortex recoil velocity (pixels per second) at different time points after laser cutting (+ 10 sec) for a single experiment. Note the exponentially decaying speed. C) Inferred stress differences (brown) and mean recoil velocities (blue) as measured by PIV and in the LECs (cyan - n = 18)), 10 µm (red - n = 16) and 40 µm within the nest (green - n = 20). Error bars represent Standard Deviations. D) Snapshots of a laser cut in the nest area. PIV arrows are shown in red. Scale bar = 5 µm. E) Comparison of the mean laser recoil velocities in phase 3 for Dacapo overexpressing nests (green) with wild type (blue) at different distances (-40 - n = 16, -10 - n =22, +40 - n = 14) Error bars represent Standard Deviations. F) Comparison of the mean laser recoil velocities in phase 3 for P35 overexpressing nests (green) with wild type (blue) at different distances (-40 - n = 17, -10 - n = 20, +40 - n = 15) Error bars represent Standard Deviations. \n",
      ".Figure 7A) Density power radial kymograph of a representative time-lapse movie of the nests expansion process after blocking histoblasts proliferation by overexpressing Dacapo. Each horizontal line displays at each time point (left scale) the mean of the power value at every distance (bottom scale) to the nest-LECs border (vertical black line). Each phase is represented in a colored background with three different densities of purple. Power values color scale is on the right. B) Comparison of the means and standard deviations of the density power radial kymographs in phase 3 for Dacapo overexpressing nests (green) and wild type (purple). C) Surface Stress radial kymograph of a representative time-lapse movie Phases are represented in different densities of brown. Stress values color scale is on the right. D) Comparison of the means and standard deviations of the surface stress radial kymographs in phase 3 for Dacapo overexpressing nests (green) and wild type (brown). E) Density power radial kymograph of a representative time-lapse movie after blocking LECs delamination by overexpressing P35. Power values color scale is on the right. F) Comparison of the means and standard deviations of the density power radial kymographs in phase 3 for P35 overexpressing LECs (green) and wild type (purple). G) Surface Stress radial kymograph of a representative time-lapse movie Phases are represented in different densities of brown. Power values color scale is on the right. H) Comparison of the means and standard deviations of the surface stress radial kymograph in phase 3 for P35 overexpressing LECs (green) and wild type (brown). \n",
      ".Figure 1A. Representative photomicrographs of mouse liver sections following a two-thirds PH of Ki67 positive hepatocytes (brown) visualized by IHC. Images are representative of 3 independent animals per time point. B. Western blot detection of p-MST1, MST1, p-LATS1, LATS1 from proteins isolated from homogenised liver tissue at several time points. TBP was used as a loading control. Results of a single experiment with n = 3 animals per group. Each band is from an independent animal. Protein levels were normalized to TBP and mean value of the ratio of p-MST to total MST and p-LATS to total LATS is plotted. C. Nuclear enriched protein fractions obtained from sham-operated livers and liver 48 h post PH were analyzed by Western blot for YAP expression. Sp1 was used as a loading control. Image is representative of a single experiment with n = 3 animals per group. D. Photomicrograph of YAP1 expression in sham and regenerating liver 48 h post PH visualized by immunoflourescence microscopy. Images are representative of 3 independent animals per condition. E. Livers from sham and 48 h following PH were digested to isolate the hepatocyte population. Protein extracts were analyzed by Western blot for YAP and TAZ expression. β-actin was used as a loading control. Results of a single experiment with n = 2 animals per group. Each band is representative of an independent animal. F. RT-PCR analysis of RNA isolated from mouse liver for Birc5, FoxM1B and cyclin B1. Data are presented as log2 fold change and were calculated using non-operated mouse liver tissue as a control. Unpaired, two-tailed Student t-test was used to determine the significance between the log2 values of livers 6, 24 and 48 h post PH compared to the sham operated controls. Log2 values are plotted on a linear scale as mean ± SD. \n",
      ".Figure 3A: Clustering and enrichment analyses of shRNA reads in D6C (endoderm), D6F (HM/E), D20LS, D20LK, and D20LSK (HSPC) populations compared with ESCs. Specifically enriched/depleted shRNAs are grouped according to their target gene function (I-X). Color coding bar is shown below. B: Heat map of shRNA reads in D6C and D6F populations compared with ESCs. shRNAs specifically depleted in D6C and D6F are labeled as Groups XI and XII, respectively. C: GO pathway analysis showing the top 5 enriched biological processes specific to D6F (XII), D20LS (VI), D20LK (V), and D20LSK (X) populations. The enrichment analysis was conducted using the Metacore platform. D: GO analysis showing several key pathways, including Wnt signaling, cell adhesion, and cell cycle regulation, that are important at all stages of HSPC development. E: Heat map showing depletion of shRNAs targeting Cdc and Cdk genes in all day 20 samples. \n",
      ".Figure 3A. Western blot detection of p-MST1, MST, p-YAP, YAP and TAZ in control and 40 h following PH in young and aged mice. TBP was used as a loading control. Representative results of a single experiment with n = 6 independent animals. Each band is from an independent animal. Western blot quantification ratios are graphically depicted. Proteins levels were normalized to TBP and mean value of the group is plotted. Unpaired, two-tailed Student t-test was used to calculate the significance between 0h (resected liver) and 40h following PH. Error bars represent mean ± SD B. RT-PCR analysis of RNA isolated from mouse liver for FoxM1B, Birc5 and cyclin B1, Cyclin A2, Cyclin D1 and p21. RT-PCR data are presented as log2 fold change and each value post PH was calculated for each mouse by comparison to its resected liver lobe. Circle, with red outline marks the aged non-regenerating livers shown in Figure 2B. Each dot is representing an independent animal. \n",
      ".Figure 1A: Developmental stages of ESC differentiation to HSPCs, showing the populations isolated on days 6 and 20. B: Schematic overview of the experimental strategy. C: FACS plots from a representative experiment showing gates for purification of the cell populations. Upper left: GFP+ cells in the population of ESCs infected with pGIPZ-shRNA library lentiviruses. Upper right: Isolation of mesodermal (SSEA1-Flk1+Cxcr4-) and endodermal (SSEA1-Flk1-Cxcr4+) cells. Lower panels: Isolation of D20LS (Lin-Sca1+c-Kit-), D20LK (Lin-Sca1-c-Kit+), and D20LSK (Lin-Sca1+c-Kit+) cells. \n",
      ".Figure 2A: mRNA analysis of expression of endothelial (Tie2, CD31, Cdh5), epithelial (Cdh1), and early hematopoietic (Scl, CD41) marker genes in the indicated sorted cell populations. Error bars represent standard deviation (SD), n=3. Student's t-test. *p<0.05. B: Confirmation of the purity of sorted populations by qRT-PCR: c-Kit is highly expressed only in the c-Kit+ (D20LSK and D20LK) populations, and Sca1 is highly expressed only in the in Sca1+ (D20LSK and D20LS) populations. Error bars represent standard deviation (SD), n=3. Student's t-test. *p<0.05. C: qRT-PCR analysis of Gata1, Gata2, Lyz2, and Scl expression in the sorted cell populations on day 20. Error bars represent standard deviation (SD), n=3. Student's t-test. *p<0.05. D: FACS analysis of ESC-derived cells confirming that D20LS, D20LK, and D20LSK cell populations express the early hematopoietic marker CD41 and the pan-hematopoietic marker CD45. \n",
      ".Figure 5A. Aged mice (>12 months) were injected i.v. with siScr or siMST. Twenty-four h after siRNA injection a 70% PH was performed and remnant/regenerating liver tissue was harvested 40 h later. p.i. of the livers was calculated by IHC staining of Ki67 positive hepatocytes and based on the percentage of positive cells liver were classified (-) non regenerating or (+) regenerating. Non-surviving animals were sacrificed before the 40 h end time point. \"n\" indicates the number of animals per group.B. Ki67 immunostaining of the regenerating aged mice livers, 40 h post PH. Sections were scanned using Histotech Pannoramic MIDI Scanner and quantitated with Quant Center 2.0 software. p.i. values are provided on the image. C. Representative photomicrograph of an aged liver treated with siMST and stained with H&amp;amp;E and by IHC for CK19 and Ki67. Arrows are indicating ductal regions with no signs of reaction or oval cell expansion. D. Quantitative RT-PCR analysis of RNA isolated from mouse liver for FoxM1B, Birc5 and cyclin B1 at 0 and 40 h post PH. Log2 fold change was calculated using non-transfected/non-resected mouse liver as a control. Representative results of a single experiment with n = 6 animals per group are shown. Paired, two-tailed Student t-test was used to calculate the significant change in signals between liver tissues at the time of resection (0h) compared to 40 h post PH. E. Percentage of Ki67 positive hepatocytes in aged and aged + siMST 40 hours post resection. Representative results of a single experiment with n = 6 independent aged animals for group are shown. Two-tailed Student t-test was used to calculate the significance of percentage of positive hepatocytes of siMST treated animals compared to non-treated controls. F. Endpoint liver weight was taken of the remnant lobe and expressed as a percentage to total body weight. Representative results of 1experiment with n = 6 independent animals for group expect for control group were n = 6 are shown. Estimated liver-to-body weight ratio before PH in mice = 3.85 ± 0.05 (SD). Unpaired, two-tailed Student t-test was used to calculate the significance of each aged group in comparison to the control young group. G. Hepatocyte geometric diameter was calculated (see material and methods). Representative results of 1 experiment with n = 3 independent animals per group. Unpaired, two-tailed Student t-test was used to calculate the significant change in hepatocyte size in liver tissue with and without siMST 40 h post PH. \n",
      ".Figure 4A. Eight weeks old mice were injected with liposomes coupled to scrambled siRNA (siScr) or sequences targeting MST1 MST2 (siMST). Each dot is representing an independent animal. The % of Mst1 Mst2 mRNA remaining in the liver was tested by RT-PCR, 1, 3 and 6 days post injection and calculated compared to control non-transfected livers. Unpaired, two-tailed Student t-test was used to calculate the significance of mRNA remaining in comparison to non-transfected control livers at each time point. B. Western blot detection of MST1, p-LATS1 and β-actin 1, 3 and 6 days post siRNA injection. A representative animal from each time point is shown. C. Western blot detection of YAP1, p-YAP1, H3 and p-H3, 3 days post siRNA injection. Two representative animals from each group are shown. All samples were run on the same blot. The blot is split due to non-consecutive loading of the samples. D. Quantitative RT-PCR analysis of RNA isolated from mouse liver for FoxM1B, Birc5 and cyclin B1, 1, 3 and 6 days post injection. Log2 fold change was calculated using non-transfected mouse liver as control. Unpaired two tailed Student t-test was used to calculate the significance of fold change of in comparison to a panel non-transfected control livers. E. HNF4α and Ki67 were detected by IHC and cells positive for both were classified as proliferating hepatocytes. Ki67 positive hepatocytes are indicated with an asterisk and Ki67 negative bile ducts/biliary cells with an arrow in tissue 3 days post siRNA delivery. The percentages of positive hepatocytes at day 3 and day 6-post injection are graphed. Two-tailed Student t-test was used to calculate the significance of percentage of positive hepatocytes in comparison to non-transfected control livers. \n",
      ": siRNA screen for identifying regulators of AGR2 dimer formation.Figure 2A) The first screen was carried out using HEK293T cells transfected with the ERMIT bait/prey, system, the XBP1 splicing reporter and the siRNA library (n = 5). The counter screen was carried out using the same experimental setup without the ERMIT bait/prey system (n = 4). B) Volcano plot representing the statistical analysis of the data from the screen, where the y axis represents statistical criteria and the x axis the fold change in luminescence units. C) Number of genes associated with the indicated processes, as revealed by functional analysis based on Gene Ontology and Reactome terms, (*): Apoptotic signaling pathway (p=0.0083), ER-associated degradation (p=0.0076), and lipid biosynthetic process (p=0.0059), respectively; (**): ER stress (p=0.0044), Protein folding (p=0.003), Calcium homeostasis (p=0.0022), N-glycosylation (p=0.0018), respectively. D and E) Enrichment of AGR2 homodimer enhancers in functions associated with ER homeostasis (green) and of AGR2 homodimer inhibitors in functions associated with ER homeostasis imbalance (red). \n",
      ".Figure 7In the proposed roadmap of HSPC development, Stage I refers to the differentiation of ESCs into HM/E, which required Group XII genes. The transition from HM/E, which is dependent on Group IV genes, is denoted as Stage II. Group X genes are specifically required for HSPC specification, indicated as Stage III. See text for details. \n",
      ".Figure 5A: GO analysis of biological processes among Group IV target genes, showing enrichment of networks in the endothelial-hematopoietic transition. B: Heat map showing bona fide HSC regulators in Group IV target genes. C: Heat map showing the most depleted shRNAs in pair-wise comparisons between the D20LSK and D20LK populations. D: Validation of the involvement of Group X target genes Lrsam1, Ap2a1, Mettl22, and Hal in zebrafish hematopoiesis by MO-mediated knockdown. Whole-mount in situ hybridization of Runx1 and c-Myb expression is shown at 26 hpf and 30hpf, respectively. Prdm16 knockdown serves as a positive control. Note that nonspecific staining for Runx1 and c-Myb was largely unaffected by targeted MO knockdown, whereas HSC markers Runx1 and c-Myb staining were notably reduced in discrete cells in the ventral region of the dorsal aorta (arrows). Scale bar = 100 µm. \n",
      ".Figure 6A: Representative flow cytometric analysis showed that CD34+cells were significantly decreased at 5dpi with knockdown of AP2A1, METTL22, LRSAM1, or HAL. The human cord blood CD34+cells were transduced with pLKO-shRNAs targeting human AP2A1, METTL22, LRSAM1, or HAL and cultured for 5 days. Then, the flow cytometric analysis was performed to examine the percentage of CD34+cells in culture.B: Quantification of CD34+cells at 5dpi with knockdown of AP2A1, METTL22, LRSAM1, and HALshowed significant decrease of CD34+ cells compared to control. Error bars represent standard deviation (SD) from duplicates, *p<0.05. C: Flow cytometric analysis showed CD34+/CD90+cells were strikingly decreased at 5dpi with knockdown of AP2A1, METTL22, LRSAM1, and HAL.D: Quantification of CD34+CD90+cells at 5dpi with knockdown of AP2A1, METTL22, LRSAM1, and HALshowed significant decrease of CD34+CD90+cells compared to control. Error bars represent standard deviation (SD) from duplicates, *p<0.05. \n",
      ".Figure 4A: The E14tg2a ESCs were transduced with lentiviral shRNAs targeting genes Esrra, S100a8, or Hcfc2 and induced to form EBs. The day 6 EB cells were analyzed by flow cytometry to measure the Mesoderm-Endothelium cells (Flk1+/Cxcr4-) and Endoderm cells (Cxcr4+/Flk1-). The percentages of Mesoderm-Endothelium cells (Flk1+/Cxcr4-) were decreased on day 6 with shRNA knockdown of Group XII genes Esrra, S100a8, and Hcfc2 compared to control as quantified by flow cytometric analysis.B: Knockdown of Esrra, S100a8, and Hcfc2 showed that the ratios of percentage of Mesoderm-Endothelium population vs Endoderm population were decreased as measured by Flk1+/Cxcr4- vs. Flk1-/Cxcr4+ on day 6. The data were generated based on the quantification of flow cytometry analyses. Error bars represent standard deviation (SD), n=2. Student's t-test, *p<0.05. C: The E14tg2a ESCs were transduced with lentiviral shRNAs targeting genes Atp5g3, Bub1b, Eci1, or Dis3l and induced to form EBs. The day 6 EB cells were analyzed by flow cytometry to measure the Mesoderm-Endothelium cells (Flk1+/Cxcr4-) and Endoderm cells (Cxcr4+/Flk1-). Flow cytometric analysis showed that Endoderm cells (Cxcr4+/Flk1-) were deceased while Mesoderm-Endothelium cells (Flk1+/Cxcr4-) were slightly increased in the cells with shRNA knockdown of Group XI genes Atp5g3, Bub1b, Eci1, and Dis3l compared to control.D: Knockdown of Atp5g3, Bub1b, Eci1, or Dis3l led to increase of Mesoderm-Endothelium/Endoderm population ratio as quantified based on the flow cytometry data. Error bars represent standard deviation (SD), n=3. Student's t-test, *p<0.05. E: The E14tg2a ESCs were transduced with lentiviral shRNAs targeting genes Wbp5, Rbm26, Gdpd4, and Nrxn1 and differentiated towards HSPCs. The differentiated cells on day 20 were analyzed by flow cytometry. Silencing of group IV genes including Wbp5, Rbm26, Gdpd4, and Nrxn1 decreased overall LK, LS, and LSK population on day 20.F: Quantification of LK+LSK+LS cell fractions with Wbp5, Rbm26, Gdpd4, and Nrxn1 silencing based on flow cytometric analyses. Error bars represent standard deviation (SD), n=2. Student's t-test, *p<0.05. G: The E14tg2a ESCs were transduced with lentiviral shRNAs targeting genes Ap2a1, Mettl22, Lrsam1, or Hal and differentiated towards HSPCs. The differentiated cells on day 20 were analyzed by flow cytometry. Flow cytometric analysis showed notable reduction of LSK population on day 20 with Group X genes Ap2a1, Mettl22, Lrsam1, or Hal knockdown.H: Quantification of flow cytometric analyses showed knockdown of Ap2a1, Mettl22, Lrsam1, and Hal significantly impaired generation of LSK population on day 20 in vitro compared to control. Error bars represent standard deviation (SD), n=3. Student's t-test, *p<0.05. \n",
      ": AGR2 dimerization with ERMIT assay, principles and validation of the method.Figure 1The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. A) Upper panel: NMR structure of the non-covalent dimer of AGR2 (PDB ID: 2LNS). The dimer domain is highlighted with a yellow circle (residue 54-70 of each monomer). Monomer A is colored green and monomer B in pink. The figure was generated in Chimera. Lower panel: a close-up illustration of the dimer domain, showing that the dimer is stabilized through two salt-bridges between E60 and K64 of each monomer. The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. B) Molecular modeling showing root-mean-square deviation (RMSD) plot for the MD simulation of the AGR2 WT and E60A mutant. The interaction-energies are defined as the sum of the short-range Coulomb interactions and the short-range Lennard-Jones potential between monomer A and B. The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. Western blot showing the expression of AGR2 dimers (D) and monomers (M) in HEK293T subjected to DSP-mediated cross-linking and that were previously transfected with either a control siRNA (siCTL) or a siRNA targeting AGR2 (siAGR2) for 24 h (C) Reduced or non-reduced samples were resolved by SDS-PAGE and immunoblotted using anti-AGR2, anti-ERK1 antibodies (for loading control). The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. Western blot showing the expression of AGR2 dimers (D) and monomers (M) in HEK293T subjected to DSP-mediated cross-linking and that were previously transfected with either a control siRNA (siCTL) or treated or not with tunicamycin (Tun) prior to cross-linking (D). Reduced or non-reduced samples were resolved by SDS-PAGE and immunoblotted using anti-AGR2 anti-calnexin (CANX) antibodies (for loading control). The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. E) Principles of the ERMIT assay. A wild type IRE1 bait is used to report for dimerization, whereas a kinase catalytic mutant (K599A) is used as a control to prove that signal observed with the wild type form is due to IRE1 activation. The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. F) AGR2 dimerization was monitored with ERMIT (wild-type (WT)). The luminescence signal was abrogated when using the kinase dead (KD) constructs and by the constructs exhibiting mutations in the dimerization domain (E60A or C81S or double mutant (DM)). The graph represents average signal normalized on reporter protein expression ± SD (n=5; **: WT/KD (p=0.002), E60/WT (p=0.0034), E60/KD (p=0.001), C81/KD (p=0.004), DM/WT (p=0.0021) and DM/KD (p=0.0017), respectively; *: C81/WT (p=0.0098)). The ERMIT assay relies on the signaling properties of IRE1, one of the three ER stress sensors and reports for a dimerization event occurring in the lumen of the ER. The assay can be applied to heterodimerization or homodimerization events. G) Cells expressing various bait and prey constructs and the XBP1 splicing reporter or the XBP1 splicing reporter alone were exposed to increasing concentrations of DTT. The ERMIT signals obtained with both baits were then normalized to that of XBP1s to obtain results independent of the activation of endogenous IRE1 by the use of chemical ER stressors (n = 4). \n",
      "Fig.Figure 1T-DNA insertion site in the PLDα6 gene and the complementation construct introduced into the T-DNA insertion mutant. Boxes denote exons and lines introns. PLDα6 transcript in WT, pldα6 and complementation line (PLDα6-COM). Leaf samples from 4-leaf stage rice (Dongjin background) were collected and the expression levels of PLDα6 were analyzed by normalizing to that of GAPDH. Values are means ± SD (n = 3 biological repeats). Seedling phenotype under GA treatment. After germination, Five-day-old seedlings with the same growth stage were transferred to 0.5 MS liquid media without or with different concentrations of GA3. PA from soybean was added to the media at a final concentration of 20 μM. Pictures were taken 7 days after transfer. The horizontal red line separates different plants and the vertical red scale bar represents 2 cm. Seedling length of WT, pldα6, COM, and PA-treated plants grown on 0, 0.1, 1 and 10 μM GA3 for 7 days. Values are means ± SD (n = 15 plants) from one representative of three independent experiments. E Fresh weight of 10 seedlings of WT, pldα6, COM, and PA-treated plants grown on 0, 0.1, 1 and 10 μM GA3 for 7 days. Values are means ± SD (n = 15 plants) from one representative of three independent experiments. \n",
      "Fig.Figure 2Immunoblotting of His-tagged PLDα6 (arrow) expressed in E. coli as separated on an 10% SDS-PAGE and blotted to a membrane. Lipid hydrolyzing activity assayed in the presence of different phospholipids using purified PLDα6 from E. coli. Solid bars are activities assayed using purified PLDα6 whereas open bars were empty vector control that used an equal volume of eluents from bacteria containing the empty vector that was identically processed as those expressing PLDα6. Values are means ± SD (n = 3 biological replicates). \n",
      ": AGR2 status alteration impacts on client protein biogenesis.Figure 4A) The ERMIT-based validation of the complex formation between AGR2 and LYPD3 or CD59, but not OS9. The monomeric AGR2 E60A was used as bait, while LYPD3, CD59 and OS9 were used as prays (n=6). B-C) Expression and subcellular localization of CD59-GFP WT and C94S. D and E) Co-immunoprecipitation of CD59-GFP WT or CD59 C94S mutant with AGR2 WT (D) or AGR2 AA mutant (E) in HEK293T cells. F-I) FACS analysis of CD59 WT (F and G) or CD59 C94S (H and I) total (F and H) and membrane (G and I) expression in HEK293T cells silenced for AGR2 (siAGR2) and TMED2 (siTMED2) when compared to control (siGL2) cells (F and H) or silenced for TMED2 (siTMED2) and overexpressing AGR2 WT or AGR2 AA mutant (G and I). Data are representative of three to four independent experiments. Membranous CD59 expression was detected with anti-GFP APC. ns: non-statistically significant; for F (*): p=0.0385, (***): p=0.0003 (left panel) and p=0.0002 (right panel), respectively; for G (*): p=0.0334, (**) p=0.0036 and (***) p=0.0003; for H (*): p=0.0171 and (**): p=0.0084; and for I (**) p=0.0086 (left panel) and p=0.0076 (right panel), respectively. \n",
      ": Validation of AGR2-TMED2 interaction and its impact on AGR2 dimerization.Figure 3A) Co-immunoprecipitation of AGR2 with TMED2 under basal and tunicamycin induced ER stress in intestinal epithelial cells treated in situ. *indicates Immunoglobulin heavy and light chains. B) Structural association of TMED2 and AGR2. The six TMED2 orientations (pink, dark pink, purple, blue, yellow and green) best positioned to stabilize the AGR2 dimer (colored tan with N-termini and dimer interface alpha helix in red). C) Changes in AGR2 dimer and monomer ratio in TMED2 overexpressing cells. DSP-stabilized AGR2 was analyzed under non-reducing (top blot) or reducing conditions (bottom blot). D=dimeric, M=monomeric AGR2. D) Specific interactions between residues in AGR2 dimer (tan) and best oriented TMED2 structure (green). Glu51, Arg74 and Arg97 of TMED2 form ionic interactions with Lys66, Glu59 and Asp97 of AGR2. His28 of TMED2 interacts with Tyr111 of AGR2 through π-π-interactions. E): Co-immunoprecipitation of AGR2 wild-type (WT, left panel) or AGR2 AA mutant (right panel) with TMED2 in HEK293T cells. Immunoprecipitation was performed with mouse anti-Flag antibody to pull-down the ectopic protein tag. F) Western blot analysis (left panel) and quantification (right panel) of AGR2 intracellular expression upon TMED2 overexpression. Data are representative of 4 independent experiments. Tubulin (TUB) was used as a loading control. G) HEK293T cells were transfected with control (ev) or TMED2 plasmid and then used in the ERMIT assay with AGR2 WT (n=3). H) Western blot analysis (left panel) and quantification (right panel) of AGR2 intracellular expression in total cell lysate (TCL) upon TMED2 silencing. Data are representative of 4 independent experiments. Tubulin (TUB) was used as a loading control. I) HEK293T cells were silencing for TMED2 and then used in the ERMIT assay with AGR2 WT (n=3). \n",
      ": Alteration of TMED2 expression and relationships with eAGR2-mediated monocytes attraction.Figure 6A) Representative histological H&E staining of sections of the proximal colon and ileum of WT and TMED2 hypomorph mice. B) Immunohistological analysis of AGR2 and MUC2 expression in the proximal colon of WT and TMED2 hypomorph mice. C) Semi-quantitative analysis of AGR2 and MUC2 expression in the proximal colon of WT (blank bars) and TMED2 hypomorph mice (black bars) (n=6). D) Freshly isolated PBMCs were placed in Boyden chambers towards media conditioned by cells overexpressing AGR2 WT, E60A, ∆45 or AGR2 AA mutants and incubated for 24 h. Migrating cells were then characterized and quantified by flow cytometry using FCS/SSC parameters or CD14 monocyte marker. (**): p=0.0016. E and F) Freshly isolated PBMC were placed in Boyden chambers towards media conditioned by cells overexpressing TMED2 and either AGR2 WT (E) or AGR2 AA mutant (F) and incubated for 24 h. Migrating cells were then characterized and quantified For E (*): p=0.0461 (AGR2WT/TMED2) and 0.018 (AGR2WT/TMED2 + blocking Ab) respectively; (**): p=0.0053 and (***): p=0.0048; for F (***): p=0.0007. G) Freshly isolated PBMC were placed in Boyden chambers towards media conditioned by cells silenced for TMED2 and incubated for 24 h. Migrating cells were then characterized and quantified (*): p=0.0149. H) Freshly isolated PBMC were placed in Boyden chambers towards decreased doses of recombinant AGR2 and incubated for 24 h. CCL2 cytokine was used as positive control for monocyte migration. ns: non-statistically significant, (*): p=0.0171, (**): p=0.0084, (***): p=0.0003. I) Impact of AGR2 blocking antibodies on AGR2-mediated monocytes migration was tested using Boyden chambers as described above. The concentrations of recombinant human AGR2 was of 200 ng/ml and decreasing amounts of antibodies were used from 20 µg to 1 µg. The non-relevant antibody (Isotype) was used at the maximal dose of 20 µg. Data are representative of three independent experiments. ns: non-statistically significant, (*): p=0.0165 (Ab=20μg/ml), 0.0223 (Ab=10μg/ml) and 0.0493 (Ab=5μg/ml), respectively. \n",
      ": Expression levels of AGR2, TMED2 and CD163 in biopsies from healthy controls and patients with IBD.Figure 7A) Selective enrichment of AGR2 modulators in colonic Crohn's disease (CD). Percentage of tested genes with altered expression in colonic biopsies from patients with Ulcerative Colitis (UC, n=8) and colonic CD (CC, n=15) and in ileal biopsies from patients with ileo-colonic CD (IC, n=9). B) Hierarchical clustering of the AGR2 modulator genes significantly deregulated in non-inflamed colonic mucosa from patients with Crohn's Disease (CD) versus healthy controls (C) (dChip software t-Test) C) TMED2 mRNA expression in healthy controls (CT), colonic Crohn's disease (CC) and ulcerative colitis (UC). D) Representative immunohistological analysis of TMED2 in non-inflamed colonic biopsies from healthy controls and patients with active CD. E) Representative immunohistological analysis of CD163 (scavenger receptor present in macrophages), AGR2, and TMED2 in non-inflamed colonic biopsies from healthy controls and patients with active or quiescent CD. F) Semi-quantitative analysis of CD163, AGR2, and TMED2 immunostaining in colonic mucosa from healthy controls (C) and patients with quiescent (Q) or active (A) CD. Three random sections from each patient of the four groups were scored. Final IHC scores (AxB range to 0-9) combine the percentage of positively labelled cells. A: % of IHC positive labeled cells - 0 (0%), 1 (<30%), 2 (30 to 60%), 3 (>60%)) and the intensity of the reaction product. B: 0 (no reaction), 1 (weak), 2 (mild), 3 (strong) in most of the examined fields. All values are mean ± S.E.M. G) Schematic representation of the role of AGR2 dimer alteration in intestinal epithelial cells under basal conditions or in Crohn's disease. In normal cells, the ratio of dimeric vs. monomeric AGR2 defines proteostasis boundaries in the ER, and those are altered in diseased conditions through the alteration of the expression of AGR2 dimer regulators, thereby leading to pro-inflammatory signals comprising the release of AGR2 in the extracellular milieu and to the induction of uncontrolled autophagy and the subsequent alteration of protein secretion. \n",
      ": Cellular mechanisms of AGR2 secretion.Figure 5A) Formation of GFP-LC3 autophagic puncta in TMED2 overexpressing HEK239T cells as monitored using confocal microscopy (right panel). DAPI was used for nuclear staining visualization. Percentage of GFP-LC3 puncta in control (CTL) and TMED2 overexpressing HEK293T cells as quantified from three independent experiments by counting 240 GFP-LC3 positive cells for each condition (left panel). (**): p=0.0032. B) Western blot detection (left panel) and quantification (right panel) of LC3 level in control and TMED2 overexpressing HEK293T cells treated or not with 50 µM chloroquine for 2 hours. Actin (ACT) served as a loading control. (*): p=0.0498. C) Western blot analysis of AGR2 expression in control and TMED2 overexpressing HEK293T cells upon autophagy inhibition with 20 µM and 10 µM chloroquine treatment. Actin (ACT) served as a loading control. D) HEK293T cells were transfected with control (ev) or TMED2 plasmid and then used in the ERMIT assay with AGR2 WT the presence of gradual amounts of chloroquine. E) Western blot analysis of secreted AGR2 upon TMED2 overexpression. eAGR2, extracellular AGR2; iAGE2, intracellular AGR2. F) Representative pictures of extracellular vesicles (EVs) heterogeneity purified from conditioned media of HEK293T cells transfected with control (ev) or TMED2 plasmid and analyzed by cryo-electron microscopy. Right upper panel: western blot analysis of CD63 in total medium (left) and in the purified extracellular vesicles fraction purified from media conditioned by control and TMED2 overexpressing cells (middle) and western blot analysis of AGR2 levels (right) in extracellular vesicles enriched from culture media conditioned by control and TMED2 overexpressing cells. G) Western blot analysis of secreted AGR2 upon TMED2 silencing eAGR2, extracellular AGR2; iAGE2, intracellular AGR2. H) Quantification of extracellular AGR2 level (eAGR2) in cells silenced for TMED2 (siTMED2), compared to control (siGL2). Data are representative of three independent experiments and are presented as extracellular-to-intracellular AGR2 ratio. (**): p=0.0029. I) Secretion of AGR2 wild-type (wt), E60A and ∆45 mutant as determined by Western blot. eAGR2, extracellular AGR2; iAGE2, intracellular AGR2. \n",
      "Fig.Figure 8GA promotes PLDα6 translocation from cytosol to the nucleus, resulting in an increase of PA in nuclear membranes. PA binds to GID1, tethers it to the membrane, facilitates its nuclear translocation and the degradation of the suppressor SLR1, enhancing rice response to GA. PLs, phospholipids. \n",
      "Fig.Figure 7Protoplasts of WT and pldα6 from 12-day old stage leaf sheath were collected and transfected with a pM999-SLR1 construct, incubated for 12 hours and then treated with 10 μM GA3 for 0, 3, and 9 hours. The first column, fluorescence from GFP; the second, red fluorescence from the nucleus marker Ghd7; the third, bright field, and the fourth, overlay of the three channels. All confocal images were scanned using similar laser gain and offset settings. Bars = 10 μm. Immunoblotting of SLR1-GFP proteins from protoplast cells. Equal amounts of proteins from each sample were used for SDS-PAGE, followed by immunoblotting with anti-GFP antibodies. \n",
      "Fig.Figure 3Total lipid levels in WT, pldα6 and COM without and with 10 μM GA3. Leaf samples (15 seedling each) from 4-leaf stage rice (Dongjin background) were collected and lipids were extracted and profiled using ESI-tandem mass spectrometry. Values are means ± SD (n = 3 biological replicates)). MGDG, monogalactosyldiacylglycerol; DGDG, digalactosyldiacylglycerol; PC, phosphatidylcholine; PE, phosphatidylethanolamine; PG, phosphatidylglycerol; PS, phosphatidylserine; PA, phosphatidic acid. Phospholipid species in WT, pldα6 and COM. Leaf samples (15 seedling each) from 4-leaf stage rice (Dongjin background) were collected and the different phospholipid species including carbon number and double bond number were determined. Values are means ± SD (n = 3 biological replicates). GA3 was dissolved in ethanol and seedlings treated with the same ethanol concentration were used as control. \n",
      "Fig.Figure 4Immunoblotting of His-GID1 and His-SLR1 expressed in E. coli. Lipid-protein blotting assay of PA, PC, PE, PG, and PS with GID1 and SLR1. Lipids (0.5 μg) were spotted on nitrocellulose strips. PA, PC, PE, and PG were from egg yolk, and PS was from porcine brain. Purified proteins (GID1 and SLR1, 0.5 mg/ml) were used, followed by immunoblotting with anti-His-tag antibodies and color development. Liposomes were made from di18:1-PC only or di18:1-PA/PC (PA:PC = 1:3 mole ratio). 1× and 10× refer to the concentration of PC or PA/PC liposomes used. NL, no liposome was added to the binding mixture. Schematic diagram showing serial deletions of GID1. GID1 fragments were expressed in E. coli and used for defining the PA binding region. CD1/2 denote conserved domain HGG and GXSXG. Catalytic triad including three conserved amino acids of GID1, S, D, and H (vertical red bars). Immunoblotting of His-GID1 proteins using constructs shown in (D). Proteins were separated by SDS-PAGE, followed by immunoblotting with anti-His-tag antibodies. The PA binding activity of different truncation mutations were analyzed by fat-immunoblotting. The red arrowheads indicate truncated proteins with different molecular weights. Sequence alignment of the PA-binding fragment of GID1 with that of the PA binding motifs in chicken Raf1, abscisic acid insensitive 1 (ABI1), constitutive triple response1 (CTR1) and werewolf (WER) from Arabidopsis. Residues in bold are basic, potentially involved in PA binding and were mutated to Ala in GID1. Immunoblotting of His-GID1 mutants and lipid-immunoblotting of PA binding by GID1 mutant proteins on a filter. Liposomal binding of GID1 proteins to PA. Liposomes were made from di18:1-PA/PC (PA:PC = 1:3 mole ratio). Liposomal associated proteins were subjected to SDS-PAGE and immunoblotting using anti-His antibodies. The band intensity was analyzed by Image J and the intensity of input was set as 100%. \n",
      "Fig.Figure 6Protoplasts from 12-day old rice (ZH11 background) leaf sheath tissue were collected. and protoplasts were transfected with pM999-GID1 and mutation constructs and imaged 12 hours after transfection. All confocal images were scanned using similar laser gain and offset settings. Bars = 10 μm. Subcellular location of GID1 in WT and pldα6 protoplasts with or without GA3 treatments. Cells for 12 hours after transformation were treated with or without 10 μM GA3 for two hours. Bars = 10 μm. Immunoblotting of subcellular fractions of GID1 and mutations expressed in rice protoplasts. Total (T), soluble (S) and nuclear (N) protein fractions were isolated from protoplasts for 12 hours after transformation. Equal amounts of proteins from each sample were loaded. Immunoblotting of subcellular fractions of GID1 expressed in WT and pldα6 protoplasts. After 2-hour treatment with 10 μM GA3, total (T), soluble (S) and nuclear (N) protein fractions were isolated from 10 samples of protoplasts. Equal amount of each sample was loaded for SDS-PAGE and immunoblotting. \n",
      "Fig.Figure 5PLDα6-GFP distribution in rice protoplasts with different concentrations of GA3 or IAA. Protoplasts from 12-day old rice (ZH11 background) leaf sheath tissue were collected. Rice protoplasts were transfected with pM999-PLDα6 for 12 hours. After that, GA3 or IAA was added to protoplasts and 2 hours later, confocal images of protoplasts are shown. pM999-GFP refers to the empty vector with GFP only that was transformed as control and Ghd7-RFP was a nucleus marker. Scale bar = 10 μm. Immunoblotting of PLDα6 in subcellular fractions. Total (T), soluble (S) and nuclear (N) proteins were isolated from PLDα6:GFP expressed in rice protoplasts treated with 10 μM GA3 or IAA. Equal amount of each sample was loaded for SDS-PAGE and blotting. \n",
      ".Figure 6A Bimolecular fluorescence complementation tests using in combination PAP8ΔcTP-NY (P8Δc-NY) with PAP5ΔcTP-YC (P5Δc-YC) or PAP8ΔcTP-YC (P8Δc-YC) with PAP5ΔcTP-NY (P5Δc-NY); PAP10-RFP (P10-RFP) was used as internal positive control for transfection; the ratio in grey depicts the number of green-fluorescent cells over red-fluorescent cells. Arrowheads indicate nuclei. See Appendix Fig. S6 for control experiments; transgenes expressed under CaMV35S promoter. B Overlay of 1H-15N correlation 2D NMR spectra of free 15N-labelled PAP8 alone (blue) or in complex with PAP5 (red). Grey areas depict changes of signals in the PAP8 spectrum. C 15N-Filtered Diffusion Ordered Spectroscopy-NMR measurements to PAP8. Exponential decay curves of PAP8 in absence or in presence of MBP-PAP5 are shown in red and black respectively. The units on the y-axis are normalized values of the integrals of the signal measured in the amide proton region. \n",
      ".Figure 4(A,B) (A) Transient expression of PAP8NLSm5-GFP mutated in the NLS as described in Fig. EV4A. The circle marks the position of the nucleus (N) as observed with DIC in (B); yellow arrowheads show stromules (str). Scale bars equal 20 µm. C-G Pictures of representative genotypes obtained in the functional complementation test of pap8-1 using pP8::PAP8-NLSm5 (C, F) or pP8::PAP8ΔcTP (G) both constructions without GFP tags; ⌀ indicates pap8-1 without any PAP8 transgene. (D-G) Pictures using Keyence technology of plants with genotypes as labelled; transgenes expressed using the 1.1-kb PAP8 promoter. Scale bars equal 1 mm. \n",
      ".Figure 1A Structure of the PAP8 locus, blue boxes: exons, lines: introns. Red box: inserted T-DNA as inverted repeats (IR1/IR2) in the first intron. White box with a diagonal red line: deletion at the left border of IR2 and part of second exon (italicized grey sequence). Green and red arrows represent forward and reverse primers respectively as o1: oPAP8_rtp_F ; o2: oPAP8_E3_R; o3: oPAP8_rtp_R; o4:op8i2_R; oLB: oLBb1.3. B PCR performed on genomic DNA with indicated primers as shown: o1: oPAP8_rtpF, o2: oPAP8_E3R, o3: oPAP8_rtpR, oLB: oLBb1.3, EF1α: ELONGATION FACTOR 1α, WT: wild type, pap8-1: Homozygous albino plant, Ht: Heterozygous green plant; T: T-DNA, arrowhead: 670-bp contaminant amplification product used as loading control. C RT-PCR on wild type and pap8-1 homozygous plants grown in the dark for 3 days followed with 72-h growth under white light to allow greening of the wild type; EF1α used as control. D Sequence alignment of predicted full length orthologous PAP8 protein found in representatives of major phylogenetic clades Arabidopsis thaliana, At1g21600; Oryza sativa Indica, EEC67529.1; Amborella trichopoda, XP_006827378.1; Selaginella moellendorffii, XP_002976643.2 Physcomitrella patens, XP_024396032.1. cTP, chloroplast transit peptide as predicted with ChloroP1.1 (www.cbs.dtu.dk) underlined in yellow; HCR shaded in grey, highly conserved region. (*), (:), (.), conserved, strongly similar, or weakly similar amino acid properties (standards from www.uniprot.org). amino acids colors as in Clustal Omega (Red (AVFPMILW): Small + hydrophobic [includes aromomatic - Y]); Blue (DE): Acidic; Magenta (RHK): Basic; Green (STYHCNGQ): Hydroxly + sulfhydryl + amine + G.) bNLS, bipartite NLS as predicted with NLS mapper E Half-open siliques of a heterozygous plant showing the embryo greening; scale bar equals 250 µm. The given number is the position rank of the silique from top to bottom of the inflorescence presenting the segregation of homozygous and heterozygous seeds based on their ability to transiently develop chloroplasts. F Mutant Rescue: WT and two representative pap8-1 plants were grown in vitro using sucrose and low white light intensity (of 10 µmol.m-2.s-1); scale bar equals 20 mm. \n",
      ".Figure 2A PAP8 promoter deletion strategy. ERI, EcoRI site; HIII, HindIII site; indicated positions are given relative to the transcription start noted as +1; red boxes represent untranslated regions; pink boxes, ORF of an upstream gene; nearly palindromic element is given in blue. -97m3: mutated promoter as described in C with three mutations (m3) indicated with '***'. B Two or six (-97m3) representative primary transformants expressing GUS under the given PAP8 promoter version; FC+, the corresponding promoters were tested positive in functional complementation of the mutant pap8-1. Scale bar equals 3 mm. C Proximal PAP8 promoter region; m3, 3-bp substitutions within the -97-bp promoter; 5'-UTR in red; ATG, start codon of PAP8. D Dual luciferase reporter assay; Renilla luciferase (Rluc) used as internal control and GFPer used as control for the transfected area; the promoters driving Firefly luciferase (Fluc) were transfected in onion epidermis cells without or with constitutively expressed HY5. The Fluc/Rluc activity was set to 1 for the minus-HY5 control; mean ± standard error corresponding to 3 replicates; photon counts are given in source data. *, ε-test = 3,43 > 1.96 corresponding to p-value < 0,001. E Electro-mobility shift assay of a probe corresponding to the near palindromic PAP8 element (GAcGCTC) with recombinant HY5 protein; a probe containing a canonical G-box element (CACGTG) recognized by HY5 was used as cold competitor. F Integrative genomics viewer (IGV) images of the Chromatin immunoprecipitation (ChIP) sequencing data30 at the PAP8 locus; TAIR10, annotation according to the Arabidopsis thaliana information resource orange box indicates the PAP8 locus. ChIP on hy5-ks50; 35S:HY5-YFP exposed to blue light or red light using GFP antibody and compared to mock corresponding to ChIP control experiment done without antibody. Each treatment is presented as track overlay of triplicates: the read count is given within the \"group autoscale\" range in brackets. Close up on the 5'-UTR region centred on the -95-promoter element in yellow. \n",
      ": Temporal resolution of protein content in seedlings during the dark-to-light transition.Figure 7A Immuno-blots showing the levels of PAP8, PAP5, Histone H3 (H3), Rubisco (RbcL), and HY5 ; D, Dark; Ph, photomorphogenic growth conditions from germination on. *, HY5 is detected as a modified form (+ 6 kDa); the arrows indicate the two different post-translational modified forms of HY5 for which accumulation occurs upon light exposure. B Relative protein contents normalized to histone H3 during the transition from skotomorphogenesis to photomorphogenesis, we propose a phase of photo-initiation corresponding to the 5 minutes of light allowing PAP8 and PAP5 to rise to nearly 100% of their maximum. The first macroscopic signs of greening are indicated around 3 hours while the photosynthetic apparatus bursts at 8 hours to be fully accumulated at 48 hours. \n",
      ".Figure 3A Schematic illustration of domain structure of Arabidopsis PAP8 fused to GFP. cTP, chloroplast transit peptide; NLS, nuclear localization signal. B Transiently expressed PAP8FL-GFP (full-length coding sequence of PAP8 fused to GFP) in onion epidermal cells displays a dual localization in the nucleus (white arrowhead) and in plastids (yellow arrowheads). C Immuno-blots for the detection of PAP8 in total protein extracts (Col-0 Tot) the nuclear fraction (N) and the plastidic fractions (EP, etioplast; CP, chloroplast) of etiolated (Dark) or photomorphogenic (Light) Arabidopsis seedlings by immuno-Western blotting using a PAP8 antiserum (PAP8). As control, a mixture of antisera raised against Histone 3 (H3) and the large subunit of ribulose 1, 5-bisphosphate carboxylase/oxygenase (RbcL) was used to evaluate reciprocal contaminations. The lanes are extracted from the same blots provided in the Source Data file. D-G Co-expression analysis of PAP8FL-GFP (D) with PAP8ΔcTP-RFP (E) in onion epidermal cells; (F) green and red channels merged; yellow arrowheads show plastids; white arrowheads show nuclei as observed with DIC, differential interference contrast in (G); FL, PAP8 full length ORF; ΔcTP, deletion of the cTP; scale bars equal 20 µm. H Confocal imaging on Arabidopsis cotyledons expressing PAP8ΔcTP-GFP, details of a few palisade cells (see Fig. S7B for a view in a cross section between the two cotyledons); magenta, auto-fluorescence of the chloroplast; the white arrowhead shows a nucleus; scale bar equals 5 µm. I,J Confocal imaging on Arabidopsis cotyledons stably expressing pP8::PAP8ΔNLS-GFP; ΔNLS, deletion of the NLS (I) during skotomorphogenesis and (J) after 24h light; Yellow arrowheads show the GFP signal; the picture is a merge of different channels: GFP in green, Pchl,: protochlorophyllide, or Chl: chlorophyll in magenta marked with arrowheads, and propidium iodide, showing the waxy cuticle in red, the empty space correspond to the layer of highly vacuolated epidermal cells. \n",
      ": PAP8 is essential for the PHYB-mediated light induction of photomorphogenesis.Figure 5A Phenotypes of given genotypes subjected to 5 days of illumination at 8 µmol.m-2.s-1 660-nm red light. PBG, pCaMV35S::PHYB-GFP transformed in pap8-1/+ (among 25 lines selected for GFP expression see data source; 2 doubly heterozygous pap8-1/+; PBG/- lines #6 and #7 segregated the photobodies alteration with the albinism). B Hypocotyl length, given as the mean ± SD, of plants grown as in (A) (R8, grey bars) or at 30 µmol.m-2.s-1 660-nm red light (R30, pink bars) showing partial insensitivity of pap8-1 to the PBG overexpression. Measurements are given in source data; in the order of the graph n equals (50, 133, 58, 43, 36, 112, 25, 60) δ-test (comparison of the mean) R8: δPBG/PBGp8-1= 20.38 corresponding to p-value < 10-31 δwt/p8-1= 0.5 < 1.96 not significant at α set to 0.05). Nuclear accumulation of PBG observed under GFP excitation in the given genotypes. (C,D) Epi-fluorescence microscopy. (E,F) Confocal microscopy showing the size of the nuclear bodies. Scale bars equal 5 µm. (G) Box plot (Min, 1st quartile, median as the central band, 3rd quartile, max) on the diameter of the nuclear bodies (NBs); n equals the number of records. H Immuno-blots using a GFP antibody or a PAP8 antibody showing respectively the levels of PHYB-GFP and PAP8 in the given genotypes grown in the dark for 3 days or in light; n.a., not applicable as the pap8-1 mutant can only be visually distinguished from wild type after light exposure; 2 lines (L#06 and L#07) for PBG/pap8-1 were tested. Coomassie blue staining presented as loading; signals were quantified using ImageJ. I RT-qPCR analysis on wild type, pap8-1, PBG and PBG pap8-1. Seedlings were grown in the dark (D) or under white light (L, 30 µmol.m-2.s-1); levels of transcripts are given relative to EF1α; error bars correspond to standard errors on technical triplicates and the dark sample is the wild-type PBG line. δ-test (comparison of the mean) ***, P < 10-72. J Immuno-blots showing the levels of PIF1, PIF3, HY5 in given genotypes grown in the Dark or Light condition as noted: p5/+, mix of an heterozygous pap5-2 siblings progeny undistinguishable from wild type; p5-2, pap5-2 and pifq, quadruple pif1-1 pif3-3 pif4-2 pif5-3 mutant; Histone H3 (H3), RbcL and PAP8 were used as controls; n.a., not applicable. \n",
      "R-III reduces CCl4-induced liver fibrosis.Figure 5Hydroxyproline content in the livers from control and CCl4-, CCl4/albumin-, and CCl4/R-III-treated mice (μg/g liver). *P = 0.037, two-sample t-test (n = 10) (CCl4 + R-III compared to CCl4-treated mice). Western blot analysis on liver extracts from the treated mice with antibody against type I collagen. Immunohistochemical analysis of α-SMA on representative liver sections from the treated mice. Scale bar, 200 μm. \n",
      ": Model for the connection of HY5 to the GLK1 genetic pathways through the action of PAP8 during the dark-to-light transition.Figure 8The yellow spark represents the initial exposure of the seedling to light. Photo-initiation is mediated by PAP8 and PAP5 rapid accumulation in early photo-morphogenesis (φMG): the Pr state of phytochrome B (PHYBPr) is converted in the Pfr state (PHYBPfr), which enters the nucleus, accumulates in early photobodies (small green discs) that regroup in late photobodies (large green discs) while promoting the destabilization of PIFs (PIF1 and PIF3 in particular). The red arrow in the timeline represents the albino block observed in pap8-1 that may represent a key feature of the PAP syndrome. The dark induces ubiquitination and degradation of HY5Ub impeding recognition of a cis-regulatory element in the promoter of PAP8 (yellow box). Light induces transcriptional activation of PAP8 in palisade cells likely through the HY5 pathway (dashed yellow arrow on PAP8 promoter); this part of the model is supported by transient assays, in vitro experiments, and in vivo genome-wide ChIP sequencing data. The cTP pre-sequence of PAP8 allows plastid import and then PAP8 assembly within the PEP-PAP complex. Using an unknown trafficking route such as travelling across the plastid envelope (?), part of the processed PAP8 pool is found in the nucleus where its action could be necessary for the PHYB-mediated transcriptional activation of GLK1 directly through HY530 (dashed yellow arrow) or other light responsive factors (LR), which in turn can activate the Photosynthetic Associated Nuclear Genes (PhANGs) concomitantly to the (PEP-PAP)-driven expression of the Photosynthetic Associated Plastid Genes (PhAPGs) essential for the building of the photosynthetic apparatus (PS) in the functional chloroplast. \n",
      "Therapeutic effect of R-III on CCl4-induced liver fibrosis.Figure 4Representative macroscopic pictures of livers from control and CCl4-, CCl4/albumin-, CCl4/RBP-, and CCl4/R-III-treated mice. Liver sections were stained with H&amp;amp;E and Sirius red, and also subjected to immunohistochemistry for type I collagen. Scale bar, 200 μm. \n",
      "Downregulation of RA signaling may contribute to the anti-fibrotic effect of albumin.Figure 2Albumin and R-III downregulate RA signaling. HSCs-P1 were either transfected with RARE Cignal reporter and expression plasmid containing vehicle or albumin (left panel) or treated with R-III (25-250 nM) after transfection with RARE Cignal reporter (right panel). After 36 h, the transfected cells were harvested for luciferase assay. Renilla luciferase activity was normalized to firefly luciferase activity. Ratios were normalized against the cells transfected with vehicle plasmid. **P-value, two-sample t-test (n = 9) (compared to control), albumin: 0.0037, R-III 125 nM: 0.0032. Phenotypic changes of HSCs by the expression of albumin and/or ALDH1A. HSCs-P1 were transfected with plasmids encoding albumin, ALDH1A1, and ALDH1A2, either individually or in combination, and incubated in the presence of all-trans retinoic acid (50 nM) for 18 h. Cells were analyzed by oil red O staining. Scale bar, 20 μm. Western blot analysis of cell lysates from albumin- and/or ALDH1A-transfected HSCs as in (B). **P = 0.002, paired t-test (n = 3) (albumin compared to control). Reverse-phase HPLC analysis of whole-cell lysates from albumin- and/or ALDH1A-transfected HSCs as in (B). RAR agonist counteracts R-III action. HSCs-P1 were treated with His-tagged R-III (final concentration, 0.15 μM) in the presence or absence of RAR agonist (AGN191183, 1 μM) for 18 h and analyzed by Western blotting. **P = 0.0024, paired t-test (n = 3) (R-III compared to control). The separating lines demarcate bands that come from non-consecutive lanes of the same gel. \n",
      "STRA6 may be involved in the cellular uptake of R-III into HSCs.Figure 3Protein expression of STRA6 during the culture activation of HSCs. Cell lysates were prepared from HSCs at 3 days after plating (HSCs-3d), and from HSCs after passage 1 (P1), 2 (P2), and 3 (P3), and analyzed by Western blotting. Lysates from NIH3T3 cells were used as a negative control. The separating lines demarcate bands that come from non-consecutive lanes of the same gel. Real-time PCR analysis for the expression level of STRA6 during the culture activation of HSCs. The PCR data are expressed as the percentage of HSCs-3d. **P-value, paired t-test (n = 3) (compared to HSCs-d3), HSCs-P2: 0.002, HSCs-P3: 0.0022. R-III uptake parallels STRA6 level. HSCs at different stages of activation were incubated with His-tagged R-III (final concentration, 0.15 μM) for 30 min, washed, and subjected to Western blotting for R-III uptake. STRA6-mediated uptake of R-III. HSCs-P1 were transfected with non-targeting siRNA or one of the two different STRA6-specific siRNAs, and after 48 h, cells were incubated with R-III for 30 min and analyzed by Western blotting. \n",
      "Retinoic acid signaling is involved in HSC activation.Figure 1Phenotypic changes of HSCs by citral. Phase-contrast images (left), oil red O-stained images (middle), and autofluorescence images (right) are shown for HSCs after passage 1 with or without the treatment of citral (100 μM) for 24 h. Scale bar = 10 μm. Western blot analysis of cell lysates from the citral-treated HSCs in (A). α-tubulin was used as a loading control. The quantitative densitometric data are expressed as the percentage of untreated control and represent the means (SD) for three independent experiments. **P-value = 0.002, paired t-test (compared to control). Protein expression of ALDH1A1 and ALDH1A2 during the culture activation of HSCs. Cell lysates were prepared from HSCs at 3 days after plating (HSCs-d3) and from HSCs after passage 1 (P1), 2 (P2), and 3 (P3), and analyzed by Western blotting. Real-time PCR analysis for the expression levels of ALDH1A1 and ALDH1A2 during the culture activation of HSCs. The PCR data are expressed as the percentage of HSCs-d3. **P-value, paired t-test (n = 3) (compared to HSCs-d3), ALDH1A1/HSCs-P3: 0.003, ALDH1A2/HSCs-P1: 0.0018, ALDH1A2/HSCs-P3: 0.001. Effect of ALDH1A suppression on HSC activation. HSCs-P1 were transfected with non-targeting siRNA or siRNA specific to ALDH1A1 and/or ALDH1A2, and after 48 h, whole-cell lysates were subjected to Western blotting. **P = 0.0023, paired t-test (n = 3) (siRNA-ALDH1A1/2 compared to control). Reverse-phase HPLC analysis of whole-cell lysates from the siRNA-ALDH1A-treated HSCs in (E). Typical chromatograms of all-trans retinoic acid and 13-cis retinoic acid. The retention time is 14 min for 13-cis-RA and 23 min for all-trans RA. Phenotypic changes of HSCs by RAR antagonist. HSCs-d3 were incubated with AGN193109 (1 μM) ± all-trans RA (10 μM) for 3 days and subjected to Oil red O staining. Scale bar = 10 μm. Western blot analysis of cell lysates from AGN193109 ± RA-treated HSCs in (G). **P = 0.003, paired t-test (n = 3) (AGN193109 compared to control). \n",
      " | ChIP-Seq and RNA-Seq analysis reveals a complex VxrB regulon.Figure 2(A) Overview of VxrB-binding profiles across the V. cholerae genome after β-lactam exposure (PenG 100μg/ml [10x MIC] for 3 hours). Red peaks indicate VxrB binding sites above the significance cutoff (q-value<10-10; MACS). (B) RNA-Seq MA plot is presented as fold change between VxrBD78E overexpressed sample and control sample. Red dots indicate differentially expressed genes after VxrBD78E overexpression (P-value < 0.001). Representative VxrB target genes are marked on both chromosomes and on the MA plot. (C) Venn diagram depicting overlap between RNA-seq and ChIP-Seq datasets. (D) Regulatory modes of representative VxrB-controlled genes in response to PenG. Upper plot shows VxrB binding patterns (assessed via ChIP-Seq), lower panel shows transcriptional response upon overexpression of VxrB D78E (assessed via RNA-Seq). RNA expression was confirmed by S1 nuclease mapping assay in wild type and ΔvxrAB mutant strains after exposure to PenG (100 µg/ml for 3 hours). White demarcation line between time points indicates that these data points were not adjacent in the original experiment but were cropped for presentation purposes. Source data is available as supplemental material. \n",
      "Immunohistochemical staining of liver sections for TGF-β and F4/80.Figure 6Liver sections from control and CCl4- and CCl4/R-III-treated mice were immunohistochemically stained for TGF-β and F4/80. Scale bar, 50 μm. \n",
      "Targeted delivery of R-III to hepatic stellate cells.Figure 7Liver sections from treated mice were immunohistochemically stained for desmin, His-tag, and α-SMA. His-positive signals largely overlapped with desmin/α-SMA staining. Scale bar, 50 μm. \n",
      "Preventive effect of R-III on CCl4-induced liver fibrosis.Figure 8Liver sections from control and CCl4- and CCl4/R-III-treated mice were stained with Sirius red. Scale bar, 200 μm. Hydroxyproline content in the livers (μg/g liver). *P = 0.034, two-sample t-test (n = 10) (CCl4 + R-III compared to CCl4-treated mice). \n",
      "R-III reduces BDL-induced liver fibrosis.Figure 9Liver sections from control, BDL, and BDL/R-III-treated mice were stained with Sirius red. Scale bar, 200 μm.. Hydroxyproline content in the livers (μg/g liver). *P-value, two-sample t-test (n = 10) (compared to BDL-treated mice); BDL + R-III (10 μg): 0.031, BDL+R-III (5 μg): 0.026. \n",
      " | Spheroplast formation in diverse bacterial pathogens.Figure 1(A) Time-dependent killing of a panel of bacterial isolates in the presence of meropenem (MEM10, 10 µg/ml). Cells were grown overnight and sub-cultured for 30 minutes at 37°C before being exposed to meropenem at 10 µg/mL and moved to a 37°C stationary incubator. Samples were taken at each indicated time point and spot-titered onto LB medium to quantify colony forming units (CFU/mL) throughout meropenem treatment. (B) images of cells corresponding to the time points in (A) were taken after labeling with membrane stain FM4-64. E. cloacae, Enterobacter cloacae; K. aerogenes, Klebsiella aerogenes; K. pneumoniae, Klebsiella pneumoniae; E. coli, Escherichia coli \n",
      " | VxrAB controls genes involved in iron homeostasis.Figure 4(A) Venn diagram showing overlap between the VxrB regulon and known V. cholerae Fur-controlled genes. (as per RegPrecise3.0 DB, http://regprecise.lbl.gov/RegPrecise/). 20 out of 47 Fur-regulated genes also have a VxrB binding site and are differentially regulated by VxrBD78E. (B) Genomic organization of iron uptake systems and their putative Fur/VxrB binding sites. (C) Time-course of iron accumulation in PenG-treated WT and ∆vxrAB cells as measured by ICP-MS. Data are average of 3 independent biological replicates, error bars represent standard deviation. ∗∗P=0.001; ∗∗∗P<0.0003 (paired t-test). (D) Streptonigrin-sensitivity of PenG-treated cells. Cultures were grown to a density of ~2 x 108 cfu/ml. Before (\"- PenG\") and after addition of PenG (100 µg/ml, 10x MIC), ~ 5 x 107 cells were spread on an LB agar plate. In the PenG-treated samples (\"+ PenG\"), beta lactamase solution was also added to remove the antibiotic. Streptonigrin (5 µL of 5 mg/ml, in DMSO) was then added on a filter disk, followed by incubation for 24 h at 37 ˚C. Values represent average diameter of clearance zones (+/- standard deviation) of six biological replicates. SNG, streptonigrin. \n",
      " | PenG induces hydrogen peroxide production and the Fur iron starvation response.Figure 5(A) Total intracellular hydrogen peroxide (H2O2) levels were measured using AmplexTM Red Hydrogen Peroxide/Peroxidase Assay Kit and normalized to total cellular protein level (measured by QubitTM). Data are average values from three independent experiments, error bars represent standard deviation. Cat, catalase (1μg/ml). (B). To test PenG-mediated induction of Fur-regulated genes, S1 nuclease mapping assays were performed on the small RNA ryhB, the RyhB target sodB, hutA/B, coding for heme transport systems and vc2212, which encodes a component of a ferric citrate uptake system). Cells were grown to mid-exponential phase followed by exposure to H2O2 or PenG for the indicated duration. Numbers represent expression levels normalized to untreated control (averages from three independent replicates). White demarcation line between time points indicates that these data points were not adjacent in the original experiment but were cropped for presentation purposes. Source data is available as supplemental material. (C) To confirm the induction of ryhB transcript by PenG treatment, the native ryhB promoter was fused with msfGFP and introduced into WT and ΔvxrA mutant cells. Each strain was cultured until mid-exponential phase, when 1mM EDTA or 100μg/ml PenG were added for the indicated duration. Quantified GFP fluorescence intensities are presented with average values from 3 independent biological replicates (cell, n>500), error bars represent standard deviation. ∗∗∗∗ P<0.0001 (paired t-test). \n",
      " | Peptidoglycan synthesis control only partially explains ΔvxrAB lethality.Figure 3(A and B) DNAse I footprinting results for VxrB interaction with promoters controlling cell wall synthesis functions. Red boxes indicate core DNA regions protected by purified VxrB protein (final concentration 0.17 - 1.38 μM) on the murJ (A) or pbp1A (B) promoter. Identified transcriptional start site (TSS; +1) and promoter elements (-10 and -35) were derived from published datasets(Krin et al, 2018; Papenfort et al, 2015), and are presented with the VxrB binding site. VBS, putative VxrB Binding Sequence. Reduction of cell wall synthesis gene expression causes a tolerance defect. (C) Survival of a ∆pbp1b ∆pbp1a Piptg:pbp1a strain after 6 hours of PenG exposure. Pup indicates conditions of high IPTG concentrations (100 µM), Pdown low IPTG concentration (2 µM). Survival fraction is cfu/ml of PenG-treated cultures normalized to untreated cultures grown in and plated on medium containing the same IPTG concentration. (D) Time-dependent killing experiment in the presence of PenG (100 µg/ml, 10 x MIC). All data are means (+/- standard error) of 3 independent biological replicates. (E) Deleting the putative VxrB binding site from the murJ promoter abrogates induction of murJ by PenG exposure. Transcript levels were measured by S1 nuclease mapping. Numbers represent induction as measured by band intensity, normalized to pre-exposure WT. n.d., not determined. \n",
      " | The iron chelator deferoxamine (DFOA) promotes amoxicillin (AMX) tolerance in vivo.Figure 7(A) Schema of experimental design. Briefly, mice were pretreated with streptomycin (5 mg/mL) for 2 days and then intragastrically inoculated with V. cholerae (~1.97x1010 cells in 200 µl PBS). After 24 h, mice in each group received water (H2O), amoxicillin (50 mg/kg), deferoxamine (200 µl of a 100 µM solution), or both amoxicillin and deferoxamine. (B) Excretion of V. cholerae. Fecal samples were collected every two hours and assessed for their viable cell content (Colony Forming Units, CFU) to monitor the level of V. cholerae excretion. Each symbol represents fecal samples from an individual mouse (n=4). The number of bacterial CFU was normalized per milligram of feces (CFU/mg). Error bars represent standard deviation. ∗∗ P<0.005; ∗∗∗ P<0.001 (two-way ANOVA). \n",
      " | Disruption of iron influx systems partially restores tolerance of the ∆vxrAB mutant against PenG exposure.Figure 6(A) Time-dependent killing experiment in the presence of PenG. (B) and (C) Iron uptake mutants restore ∆vxrAB growth on 10% sucrose. The indicated strains were grown to exponential phase (OD600~0.5) in LB medium and spot-plated on 10% Sucrose plates with or without additional iron sulfate. An example is shown in (B), quantification of cfu is shown in (C). Data information: Data are average of 3 independent biological replicates, error bars represent standard deviation. ∗∗∗ P<0.0003; ∗∗∗∗ P<0.0001 (paired t-test). \n",
      " | Proposed model for VxrAB-mediated antibiotic tolerance.Figure 9PenG induces ETC dysfunction during transition of cell shape from rod to spheroplast, resulting in the production of ROS. Hydrogen peroxide illegitimately induces the Fur regulon, resulting in the upregulation of iron acquisition systems, resulting in enhanced iron influx and lethal oxygen radical generation. VxrAB dampens this response by downregulating iron acquisition genes, in addition to its primary role as a positive regulator of cell wall synthesis genes. SO, superoxide. \n",
      " | PenG exposure is associated with ROS production.Figure 8(A) RNA expression profiles of first line ROS defense systems in V. cholerae monitored by S1 nuclease mapping assay. Quantified average values (normalized to WT before PenG exposure) from three independent experiments are presented under the signals of each gene. White demarcation line between time points indicates that these data points were not adjacent in the original experiment but were cropped for presentation purposes. Source data is available as supplemental material. Note that the loading control is the same as in Fig. 2D, as these S1 assays were done on the same RNA preparations. (B and C) Quantification of the SOD and catalase activity values from the gel shown in Appendix Figure S9. Data are average values from three independent experiments normalized to untreated wild type. Data information: Data are mean of three independent biological replicates and error bars represent standard deviation. (D) Detection of protein oxidation after PenG treatment. To measure total cellular protein oxidation, OxyBlot Protein Oxidation Detection Kit (Sigma-Aldrich) was used. For positive protein oxidation control, oxidized standard marker (OXM) was used. Both DNP-derivatized or nonderivatized (negative control) protein samples were separated by polyacrylamide gel electrophoresis followed by Western blotting with-DNP specific antibody. (E) Time-dependent killing of respiratory chain mutants (∆nqrA, encoding NADH dehydrogenase and ∆ubiA, encoding an early step of ubiquinone synthesis) in the presence of PenG (100 µg/ml, 10xMIC). All strains were grown in LB 0.2 % glucose. Data information: Data are mean of three independent biological replicates and error bars represent standard deviation. (F) Growth-phase dependent killing assay of ROS detoxification systems (∆fur, ∆sodB, and ∆oxyR1∆katG∆katB). At designated time-points, a 5 ml aliquot was withdrawn and exposed to 100 µg/ml PenG for 3 h. CFU/mL for each time point was assessed before (solid lines) and after addition of PenG (dotted lines). Data information: Data are mean of three independent biological replicates and error bars represent standard deviation. \n",
      ".Figure 1At 72 h after transfection with siRNA duplexes for BAG6 or control siRNA (10 nM each), the intracellular localization of TfnR in HeLa cells was examined (shown as green). Nuclear DNA was stained with Hoechst 33342 (shown as blue). Control knockdown (left panel), Rab8a knockdown (center panel), and BAG6 knockdown (right panel). Intracellular localization of Ptc1 (green) in HeLa cells. Nuclei were stained with Hoechst 33342 (shown as blue). Knockdown of Rab8a (with Rab8a siRNA#1, #2, and #3) stimulated the accumulation and stabilization of Ptc1 protein in HEK293 cells. Knockdown of BAG6 (with BAG6 siRNA#1) stimulated the accumulation and stabilization of Ptc1 protein in HEK293 cells. \n",
      ".Figure 3Schematic representation of Rab8a protein. To prevent C-terminal geranylgeranylation, the Cys204 residue was substituted with a serine residue. Alternatively, two truncated proteins (N100 and ∆N100 fragments of Rab8a) were prepared for this experiment. C-terminal geranylgeranylation of Rab8a was dispensable for BAG6 recognition. The N-terminal GTPase domain of Rab8a was essential for BAG6 recognition. A series of Flag-tagged truncated fragments of WT Rab8a were expressed in HeLa cells with S-tagged BAG6 and treated with (+) or without (-) protease inhibitors for 4 h. Flag-Rab8a substrates were immunoprecipitated and probed with an anti-BAG6 antibody. Hydrophobicity recognition domain of BAG6 (N465) was critical for Rab8a recognition. A schematic representation of the BAG6-truncated proteins used in this experiment is shown in the upper panel. Numbers denote the corresponding amino acids of mammalian BAG6. Positions of UBL, BUILD, and DUF3538 domains, which are all linked to hydrophobicity recognition by BAG6 [37], are indicated. WT, wild-type; ∆N465, N-terminal 465 residues-deleted mutant; and Ν465, C-terminal 689 residues-deleted mutant. CHX chase experiments show that GDP-bound Rab8a (T22N) was a highly labile protein, while the GTP-bound active mutant (Q67L) was a stable protein (E). Instability of the T22N mutant was not perturbed by the C204S mutation (F). Actin was used as a loading control. N-terminal 100 residue fragment of the Rab8a GTPase domain was sensitive to the proteasome inhibitor, while the ∆N100 fragment was not. Rab8a (T22N) was polyubiquitinated in the presence of the proteasome inhibitor. Asterisks indicate non-specific bands. \n",
      ".Figure 2Protein interaction network suggested by public databases. BAG6 protein co-precipitated Rab8a, while neither Rab7 nor Rab11a were co-precipitated with BAG6. Flag-tagged Rab8a, Rab7, Rab11a, and luciferase-CL1 (Lc-CL1; a positive control) were expressed in HeLa cells and the cells were treated with 10 μM MG-132 for 4 h. Flag-immunoprecipitates were blotted with anti-BAG6 and anti-Flag antibodies, respectively. Note that all cells used were treated with 10 μM MG-132 for 4 h. S-tagged BAG6 pull-down efficiently co-precipitated Rab8a (T22N), a GDP-bound mutant, while BAG6 scarcely co-precipitated Rab8a (Q67L), a constitutively active mutant. Co-precipitation of Rab8a WT with BAG6 was used as a standard. MG-132 (10 μM) was included in the cell culture for 4 h, as indicated. Note that the T22N mutant protein was expressed at lower levels than either WT or Q67L, and that this was partly due to increased degradation, as will be shown later. S-BAG6 stands for N-terminally S-tagged BAG6 protein Anti-Flag signals in (C) were quantified, and relative signal intensities are presented. The value of the WT Rab8a signal with MG-132 was defined as 1.0. Note that all signal intensities of Flag-tag were normalized by that of the Rab8a input signals. A series of Flag-Rab8a mutants were immunoprecipitated and quantified the amount of endogenous BAG6 that were coprecipitated with Flag-Rab8a. Note that all cells used were treated with 10 μM MG-132 for 4 h. Deficiency of Rabin8, a GEF for Rab8a, enhanced the physical interaction between BAG6 and WT Rab8a proteins. Flag-tagged WT Rab8a was expressed in Rabin8 siRNA-treated cells, and Flag immunoprecipitates were probed with an anti-BAG6 antibody. Note that all cells used were treated with 10 μM MG-132 for 4 h. \n",
      ".Figure 4Schematic representation of the two Switch regions (I and II) within the Rab8a GTPase domain. Numbers denote the corresponding amino acids of human Rab8a (upper panel). Amino acid sequence alignments of the Switch I region of Rab family proteins (lower panel). Three conserved hydrophobic residues (Ile38, Ile41, and Ile43) in this region are indicated in red, while the other hydrophobic residues are indicated in orange. The three hydrophobic residues were substituted with serine and this construct was designated as the T22N-3IS mutant. Note that Ile38 and Ile43 of Rab8a are not conserved in Rab7. Kyte-Doolittle hydrophobicity plots of the complete amino acid sequence of human WT Rab8a and T22N-3IS mutant. The hydrophobicity peak within WT Switch I was abolished in the 3IS mutant (indicated within the red box). The numbers on the horizontal axis denote the corresponding amino acid positions in these proteins. Substitution of the hydrophobic residues of Switch I with hydrophilic residues (T22N 3IS) abolished its binding to BAG6 protein. Rab8a (T22N-3IS) mutant protein was highly stable, while Rab8a (T22N) protein was quite unstable in HeLa cells. Actin was used as a loading control. Rab8a (T22N-3IS) mutant was not subject to polyubiquitin co-precipitation, while Rab8a (T22N) protein was. \n",
      "a, HeLa cells were treated with biotin-conjugated peptides (30 &mgrM, 3 h) and proteins bound to peptides were analysed by immunoblot with anti-GAPR-1.f2a, HeLa cells were treated with biotin-conjugated peptides (30 μM, 3 h) and proteins bound to peptides were analysed by immunoblot with anti-GAPR-1. b-T-B, biotin-Tat-beclin 1; b-T-S, biotin-Tat-scrambled. b, Immunoprecipitation of Flag-beclin 1 with GAPR-1-Myc in HeLa cells 24 h post-transfection. c, GFP-LC3-positive dots in GAPR-1 siRNA-transfected peptide-treated HeLa/GFP-LC3 cells (20 &amp;amp;mgr;M, 3 h) with or without 100 nM bafilomycin A1. Bars represent mean ± s.e.m. of triplicate samples (50-100 cells per sample). Similar results were observed in three independent experiments. NC, non-silencing control. d, Localization of beclin 1 and GM130 (a Golgi marker) in HeLa cells stably transduced with empty vector (left panel) or GAPR-1-Myc (middle panel) or transfected with GAPR-1 siRNA (right panel) and treated with peptide (20 μM, 1 h). Scale bar, 20 μm. e, Immunoblot of beclin 1 in post-nuclear supernatant (PNS) and Golgi-enriched fractions in HeLa cells stably transduced with empty vector or GAPR-1-Myc after peptide treatment (20 μM, 2 h). f, WIPI2 dots in cells in the experimental conditions shown in d. Bars represent mean ± s.e.m. for 100-150 cells. *P 0.05, ***P 0.001; t-test. †††P 0.001; two-way ANOVA for comparison of magnitude of changes between groups. \n",
      ".Figure 7BAG6 knockdown induced the abnormal distribution of Golgi apparatus markers. Representative images of the trans-Golgi membrane protein Stx6 (green) in BAG6-suppressed CHO cells with a Chinese hamster-specific siRNA (cBAG6 siRNA#2). Scale bar: 10 μm (A). Fluorescent signals were detected using a laser scanning confocal microscopy system. Nuclei were stained by Hoechst (blue). BAG6 knockdown induced the abnormal distribution of Golgi apparatus markers. Images of the cis-Golgi membrane protein GS28 and the cis-Golgi matrix protein GM130 in BAG6-suppressed CHO cells with another Chinese hamster-specific siRNA (cBAG6 siRNA#5). GS28 (green) and GM130 (red) are indicated by arrowheads. Scale bar: 10 μm. (B). Fluorescent signals were detected using a laser scanning confocal microscopy system. Nuclei were stained by Hoechst (blue). Glycosylation of the IL-2Rα transmembrane protein was not reduced by BAG6 knockdown. Flag-tagged WT IL-2Rα protein was expressed in HeLa cells with (+) or without (-) BAG6 siRNA, and was immunoprecipitated with an anti-Flag antibody. The precipitates were incubated with (+) or without (-) 10 unit of the deglycosylation enzyme PNGase F, and subjected to Western blot analysis with an anti-Flag antibody. Low-mobility (indicated as glycosylated) and high mobility (indicated as non-glycosylated) signals of WT IL-2Rα are indicated. Defects in the distribution of cell surface glycoproteins in BAG6-suppressed cells. Representative image from a cell surface glycoprotein quantification assay with Alexa FluorTM488-conjugated Lectin GS-II as a probe (D). Defects in the distribution of cell surface glycoproteins in BAG6-suppressed cells. The graph quantitatively displays the number of fluorescence counts per cell as the mean ± S.D. calculated from 10 independent biological replicates (E). Defects in the distribution of cell surface glycoproteins in BAG6-suppressed cells. Lectin GS-II-derived cell surface signals were counted using ImageJ software. SRP54 knockdown was used as a positive control for this experiment t-test). Cell surface fluorescent signals were detected by confocal microscopy without plasma membrane permeabilization. BAG6 siRNA down-regulated the cell surface expression of glycoproteins (F). \n",
      ".Figure 6A series of major Rab family proteins were immunoprecipitated from HeLa cell extracts and then probed with an anti-BAG6 antibody. Co-precipitation of Rab8a (T22N) with BAG6 was used as a positive control for this experiment. Quantification of BAG6 co-precipitation with Rab family proteins. The graph shows the quantities of BAG6 protein in the respective anti-Flag precipitates that were normalized by the amount of Flag-Rab bait proteins. The value of BAG6 co-precipitation with Rab8a (T22N) was defined as 1.0. The quantities of the respective immunosignals were also normalized to the actin signal of each sample, and represent the median calculated from at least 3 independent biological replicates. The number of independent biological replicates was as follows: Rab1a, n = 4; Rab1b, n = 5; Rab2b, n = 3; Rab3a, n = 4; Rab4, n = 4; Rab5a, n = 3; Rab5b, n = 4; Rab6a, n = 5; Rab6b, n = 4; Rab7a, n = 3; Rab8a, n = 14; Rab9b, n = 4; Rab10, n = 6; Rab11a, n = 3; Rab11b, n = 3; Rab12, n = 2; Rab13, n = 6; Rab14, n = 3; Rab15, n = 5; Rab35, n = 3. \n",
      "a, Immunoprecipitation of Flag–beclin 1 constructs with Nef–HA in HeLa cells 24 h post-transfection.f1a, Immunoprecipitation of Flag-beclin 1 constructs with Nef-HA in HeLa cells 24 h post-transfection. b, GFP-LC3-positive dots (autophagosomes) in MCF7 cells expressing GFP-LC3 and Flag-beclin 1 constructs grown in either normal medium or starved in EBSS for 2 h. c, Sequences of beclin 1 amino acids 267-284, Tat-beclin 1 (T-B) and Tat-scrambled (T-S) control peptide. Red letters indicate amino acid substitutions to enhance hydrophilicity. d, Biochemical assessment of autophagy (p62 and LC3 immunoblots) in peptide-treated HeLa cells (3 h). e, f, Representative images (e) and quantification of GFP-LC3-positive dots (f) in peptide-treated HeLa/GFP-LC3 cells (30 µM, 3 h). Scale bars, 20 µm. g, GFP-LC3-positive dots in siRNA-transfected peptide-treated HeLa/GFP-LC3 cells (30 µM, 3 h). h, Model of Tat-beclin 1 peptide (left) based on corresponding elements of the beclin 1 evolutionarily conserved domain (ECD) structure (centre). Essential phenylalanine side chains, magenta; positions of solubility mutations, pink; lipid interaction site, yellow. ECD surface representation (right) illustrates exposure of corresponding peptide (cyan). i, p62 and LC3 immunoblots in peptide-treated HeLa cells (3 h). In b, f, g, bars represent mean ± s.e.m. of triplicate samples (50-100 cells per sample). Similar results were observed in three independent experiments. *P 0.05, **P 0.01; t-test. \n",
      ".Figure 5Rab8a (T22N) protein accumulated in BAG6-knockdown cells. HeLa cells were transfected with siRNA duplexes for BAG6 or control siRNA. At 48 h after siRNA transfection, Flag tagged-Rab8a (T22N) was expressed in the cells. At 24 h after Rab8a (T22N) transfection, the cells were chased with 50 μg/mL CHX and harvested at the indicated time after CHX addition. Actin was used as a loading control. Anti-Flag blot signals in the control or BAG6 siRNA-treated cells were quantified, and relative signal intensities after CHX addition were calculated. The value of the Flag signal at 0 h was defined as 1.0. Note that all signal intensities of the Flag-tag were normalized by that of actin, a loading control, in each sample. Polyubiquitin modification of Rab8a was abolished in BAG6-knockdown cells. Flag-Rab8a (T22N) immunoprecipitates were blotted with an anti-polyubiquitin antibody (FK2, left panel). As a negative control, siRNA#1scr was used. Anti-polyubiquitin signals co-precipitated with Rab8a (T22N) (a representative example is shown in the left panel) were quantified (right panel). Note that the intensities of the co-precipitated polyubiquitin signal were normalized both by the input ubiquitin-signal and bait Flag-signal. Defective distribution of the endosomal protein TfnR in HeLa cells with the excess accumulation of the inactive form of Rab8a. Right panel shows a merged image of TfnR staining (shown as green), Rab8a (T22N) staining (magenta), and Hoechst 33342 nuclear staining (blue). \n",
      "Genetic and molecular features of XRCC4-mutant patients.Figure 1Pedigree. Black symbols designate affected subjects. Schematic view of wild-type and mutant (R225*) XRCC4 proteins with their main domains and phosphorylation sites. The numbering refers to amino acids of refseq: NP_071801.1. CTR: C-terminal region. Electropherograms of the XRCC4 genomic region encompassing the nucleotide substitutions in available members of the family. Quantitative real-time PCR of XRCC4 relative to GAPDH mRNA in affected (II-1 and II-2) and control (CT1 and CT2) fibroblasts. Each value refers to the mean of three independent experiments performed in duplicate. Error bars indicate the standard deviation. Immunoblot analysis of total lysates from fibroblasts (fbs) and of in vitro-synthesized proteins (i.v.p). fl: in vitro-translated full-length XRCC4 protein; ret.: reticulocyte lysate used for in vitro protein synthesis; 225*: in vitro-translated XRCC4 R225* truncated protein; Ct: lysate of control fbs; and II-1 and II-2: lysates of fbs from patient II-1 and II-2. α-XRCC4 (Abcam, 1:1,000) and α-GAPDH antibodies were used. Immunoblot analysis of total lysates from fibroblasts using α-LIG4 (GeneTex, 1:1,000) and α-SDHA antibodies. Ct: lysate of control fbs; II-1 and II-2: lysates of fbs from patient II-1 and II-2. The position of the molecular weight (MW) marker proteins is indicated. \n",
      "MHC class II‐restricted presentation of NeoR and NucNeoR is blocked by inhibition of autophagy and lysosomal degradation.Figure 2(A) RCC1.24‐NeoR and ‐NucNeoR cells were incubated for 48 h with IFN‐γ and in addition with 7.5 mM 3‐MA or 200 µM leupeptin or 100 µM chloroquine for the last 20 h. Subsequently, the cells were fixed with 0.5% paraformaldehyde and tested with 20‐4/A4 cells in a GM‐CSF release assay. Both NeoR and NucNeoR were presented by a pathway that involves autophagy and lysosomal processing. (B) LCL1.11 cells transfected with NeoR or NucNeoR expression plasmids were tested likewise. Presentation of endogenous NeoR and NucNeoR was blocked by all inhibitors, indicating that NeoR and NucNeoR presentation is dependent on autophagy and lysosomal processing. (C) To exclude toxic side effects of the inhibitors, LCL1.11 cells were incubated for 20 h with 200 ng/mL recombinant NeoR protein in the presence or absence of the inhibitors. Afterwards, residual protein and inhibitors were removed by washing and the cells were co‐cultured with the NeoR‐specific CD4+ T cell clone 20‐4/A4. The presentation of exogenous NeoR on MHC class II was not affected by 3‐MA, but was impaired when lysosomal processing was blocked with chloroquine or leupeptin. (D) LCL1.11 cells were transfected with siRNA specific for Atg12 or GFP, and Atg12 mRNA levels were determined 24 h later by qPCR. The relative amount of Atg12 mRNA was reduced approximately fourfold. (E) At 24 h after siRNA transfection, NeoR and NucNeoR were expressed in these cells and antigen presentation was assessed 36 h later. In cells transfected with Atg12 siRNA, but not in mock‐transfected cells or cells transfected with GFP siRNA, antigen presentation of both NeoR and NucNeoR was greatly reduced. \n",
      "NucNeoR presentation does not depend on CRM1‐mediated nuclear export.Figure 3(A) RCC1.24 cells were transfected with the NES‐GFP‐NeoR‐GFP‐NLS expression construct and subcellular localization of the fusion protein was monitored by UV fluorescence. As compared to untreated cells (a), cells treated with 8 nM LMB for 20 h (b) showed a strong accumulation of the protein in the nucleus. (a') and (b') are the corresponding daylight photographs of the transfected cells. (B) NeoR‐ and NucNeoR‐transfected LCL1.11 cells were either left untreated or incubated for 20 h with 8 nM LMB. Subsequently, the cells were fixed with 0.5% paraformaldehyde and probed with the NeoR‐specific T cells. LMB treatment had no significant effect on the presentation of NeoR or NucNeoR on MHC class II, indicating that CRM1‐dependent nuclear export is not involved in the presentation of this nuclear antigen on MHC class II. \n",
      "Subcellular localization of NeoR and NucNeoR and T cell recognition of NeoR‐ and NucNeoR‐expressing cells.Figure 1Subcellular localization of NeoR and NucNeoR and T cell recognition of NeoR‐ and NucNeoR‐expressing cells. RCC1.24 cells were transfected with the NeoR (A) and NucNeoR (B) expression constructs. After 2 days, the subcellular localization of the proteins was analyzed by immunofluorescence using the anti c‐myc antibody 9E10. While NeoR was evenly distributed in nucleus and cytoplasm (a), NucNeoR was confined to the cell nucleus (b). Daylight photographs of the stained cells are shown in (a') and (b'). (C) NeoR‐ and NucNeoR‐transfected RCC1.24 cells were treated with 100 U/mL IFN‐γ for 48 h to induce MHC class II expression. Subsequently, the cells were washed and co‐cultured with the CD4 clone 20‐4/A4, which recognizes a peptide derived from NeoR on HLA‐DP3. GM‐CSF secretion by the T cells was measured 20 h later by ELISA. Both transfectants were recognized after IFN‐γ treatment, indicating that directing NeoR into the cell nucleus does not abrogate antigen presentation. (D) LCL1.11 cells were transfected with NeoR or NucNeoR expression plasmids and antigen presentation was assessed 48 h later with the NeoR‐specific T cell clone 20‐4/A4. Antigen‐transfected LCL1.11 cells were recognized irrespectively of NeoR localization, while untransfected cells or cells transfected with a GFP‐expressing control plasmid were not recognized. \n",
      "a, Percentage of cells with small htt103Q aggregates (left) and number of aggregates per cell (right) in HeLa cells expressing doxycycline (Dox)-repressible CFP–htt103Q after daily treatment with doxycycline or peptide (20 &mgrM, 4 h per day) for 2 days.f3a, Percentage of cells with small htt103Q aggregates (left) and number of aggregates per cell (right) in HeLa cells expressing doxycycline (Dox)-repressible CFP-htt103Q after daily treatment with doxycycline or peptide (20 μM, 4 h per day) for 2 days. Bars represent mean ± s.e.m. of triplicate samples (60-120 cells per sample). Similar results were observed in three independent experiments. b, Filter trap assays for htt103Q large and small aggregates in HeLa/htt103Q cells. c, Viral titres in HeLa cells infected with 0.1 plaque-forming units (p.f.u.) per cell of SINV, CHIKV or WNV (strain TX02) and treated with peptide (10 μM, 4-8 h post-infection). Values represent geometric mean ± s.e.m. for triplicate samples of supernatants collected 18 h post-infection (SINV) or 24 h post-infection (CHIKV and WNV). Similar results were observed in three independent experiments. d, Bacteriacolony-forming units (c.f.u.) in primary BMDMs infected with L. monocytogenes ΔactA mutant strain DPL-4029 for 30 min and treated with peptide (10 µM from 0 to 2 h post-infection). Bars represent mean ± s.e.m. of triplicate samples. Similar results were observed in three independent experiments. e, HIV-1 p24 antigen release in primary human MDMs infected with HIV-1 24 h after initiation of daily peptide treatment. Values represent mean ± s.e.m. of triplicate samples. Similar results were observed in MDMs from three independent donors. f, HIV-1p24 antigen release in MDMs transduced with nonspecific scrambled shRNA (shNS) or ATG5 shRNA (shATG5) and treated daily with peptide (5 µM). Values represent mean ± s.e.m. of triplicate wells. Similar results were observed in MDMs from two independent donors. g, LC3 immunoblot of MDMs transduced with the indicated shRNA at day 0 and day 10 after HIV-1 infection. *P  0.05; **P  0.01; ***P  0.001; t-test. \n",
      "a, GFP–LC3-positive dots in tissues of 6-week-old GFP–LC3 mice treated with the indicated peptide (20 mg kg−1 i.f4a, GFP-LC3-positive dots in tissues of 6-week-old GFP-LC3 mice treated with the indicated peptide (20 mg kg−1 i.p., 6 h). A minimum of ten fields was counted per tissue section. Bars represent mean ± s.e.m. for three mice. Similar results were observed in three independent experiments. b, p62 immunoblot of brains of 5-day-old GFP-LC3 mice treated with the indicated peptide (20 mg kg−1 i.p., 6 h). c, Survival curves of 5-day-old C57BL/6J mice infected with CHIKV (106 p.f.u. s.c.) and treated daily with peptide (15 mg kg−1 i.p. beginning 1 day post-infection). d-f, Representative images of WNV envelope antigen and TdT-mediated dUTP nick end labelling (TUNEL) staining (d) (T, Tat alone), quantification of cell death in brain (e) and survival curves (f) for 5-day-oldC57BL/6Jmice infected with WNV (Egypt strain 101, 1 p.f.u. intracerebral (i.c.)) and treated daily with peptide (D-amino acid forms, 20mgkg−1 i.p. beginning 1day post-infection). Images in d are from cerebral cortex day 6 post-infection. Similar results were observed in all regions of the brain for three mice per group. Scale bar, 20μm. Bars in e represent mean±s.e.m. TUNEL-positive cells per unit area of brain for three mice. g d-f, Representative images of WNV envelope antigen and TdT-mediated dUTP nick end labelling (TUNEL) staining (d) (T, Tat alone), quantification of cell death in brain (e), and survival curves (f) for 5-day-oldC57BL/6Jmice infected with WNV (Egypt strain 101, 1 p.f.u. intracerebral (i.c.)) and treated daily with peptide (D-amino acid forms, 20mgkg−1 i.p. beginning 1day post-infection). Images in d are from cerebral cortex day 6 post-infection. Similar results were observed in all regions of the brain for three mice per group. Scale bar, 20μm. Bars in e represent mean±s.e.m. TUNEL-positive cells per unit area of brain for three mice. g g, Geometric mean + s.e.m. viral titres of WNV-infected mousebrains day 6 post-infection. Values represent combined data for 12-20 mice per treatment group from 10 to 12 litters. Data in c and f represent combined survival probabilities for three and four independent litters, respectively, in each group. Similar results were observed in each independent experiment. *P 0.05; **P 0.01; ***P 0.001; NS, not significant; t-test. \n",
      "Analysis of PARP-1 inhibition on DSB repair.Figure 5XRCC4wt and XRCC4m/m cells irradiated with 0.5 Gy of γ-rays were exposed to the PARP1 inhibitor 3′-AB or to solvent (DMSO). The γ-H2AX foci number is the mean ± SD of three independent experiments. *P < 0.05 (XRCC4m/m + 3′-AB versus XRCC4m/m + DMSO), ***P < 0.001 (XRCC4m/m + 3′-AB versus XRCC4wt + 3′-AB), two-way ANOVA, Bonferroni post hoc test. \n",
      "γ-H2AX nuclear foci formation in different cell cycle phases.Figure 3A, B Percentages of γ-H2AX nuclear foci remaining in CENP-F-negative (A) and CENP-F-positive (B) cells at the indicated time-points after irradiation with 0.5 Gy of γ-rays. Values, subtracted of their non-irradiated controls, are the mean ± SD of three independent experiments. ***P < 0.001; *P < 0.05 (XRCC4m/m versus XRCC4wt cells), two-way ANOVA, Bonferroni post hoc test.C Immunofluorescence micrographs of γ-H2AX foci in CENP-F-positive and negative fibroblasts at 2 h after irradiation. Arrow heads indicate the CENP-F-positive cells. \n",
      "Involvement of homologous recombinations in repairing DNA DSBs.Figure 4RAD51 foci in XRCC4wt and XRCC4m/m fibroblasts irradiated with 0.5 Gy of γ-rays. Values are the mean ± SD of three independent experiments. ***P < 0.001; **P < 0.01 (XRCC4m/m versus XRCC4wt cells), two-way ANOVA, Bonferroni post hoc test.Immunofluorescence micrographs of RAD51 foci in XRCC4wt and XRCC4m/mfibroblasts. Arrowheads indicate the CENP-F-positive cells. \n",
      "Quantification of alternative non-homologous end-joining (A-NHEJ) activity.Figure 6A, B Percentages of DSBs repaired in increasing time intervals (0.5-2 h, 0.5-6 h, 0.5-24 h) were calculated in XRCC4wt (A) and XRCC4m/m (B) cells from the number of γ-H2AX foci in presence or absence of a PARP-1 inhibitor, 3′-AB. **P < 0.01 (XRCC4m/m + 3′-AB versus XRCC4m/m + DMSO), two-way ANOVA, Bonferroni post hoc test. \n",
      "Analysis of γ-H2AX nuclear foci.Figure 2γ-H2AX nuclear foci in XRCC4wt and mutant fibroblasts (XRCC4m/m1 and XRCC4m/m2) were determined at different times after irradiation with 0.5 Gy of γ-rays. Values, subtracted of their non-irradiated control (1.3 foci/nucleus in XRCC4wt cells and 1.4 and 0.9 foci/nucleus in XRCC4m/m1 and XRCC4m/m2 cells, respectively), are the mean ± SD of three independent experiments.Percentages of γ-H2AX foci in XRCC4wt and XRCC4m/m fibroblasts (pooled values) remaining at the indicated time-points.Immunofluorescence micrographs of γ-H2AX foci in XRCC4wt and XRCC4m/m fibroblasts.Data information: ***P < 0.001; **P < 0.01; *P < 0.05 (XRCC4m/m vs. XRCC4wt cells), two-way ANOVA, Bonferroni post hoc test. \n",
      ": Overexpressing lipase in ADAM17 mutant retinas rescues both lipid droplet accumulation and retinal degeneration A-D.Figure 3A-D. Fluorescent images of 1 day old retinas, stained with BODIPY (green) and FM-dye (red) to stain lipid droplets and the photoreceptor membranes respectively; (A) wild type; (B) ADAM17-/- mutant; (C) overexpression of lipase in glial cells of ADAM17-/- mutant; (D) overexpression of lipase in neurons of ADAM17-/- mutant. Scale bars: 10μm. Quantitation of the BODIPY signal shown as (E) LD numbers; (F) normalised LD size for the genotypes n=10 for each genotype. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean and the diamond an outlier. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. Quantitation of the BODIPY signal shown as (G) integrated total lipid, for the genotypes n=10 for each genotype. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean and the diamond an outlier. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. H-K. TEM images of 5 week old adult retinas corresponding to (H) wildtype; (I) ADAM17-/- mutant; (J) glial overexpression of lipase in wild-type glial cells; (K) neuronal overexpression of lipase in the ADAM17-/-mutant. Scale bars: 10μm. L. Quantitation of the percentage of normal, abnormal and missing rhabdomeres observed in the TEM images for the genotypes mentioned above, presented as a stacked graph plot; n=180 ommatidia from 3 different fly retinas for each. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. \n",
      ".Figure 2(A) Both K+ channel blockers exert a neuroprotective effect in ALSC9orf72 cultures by increasing the neurite length of mutant MN (one-way ANOVA followed by Dunnett's multiple comparison test). n=3 independent treatments with each hiPSC line. Scale bar: 50 µm Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (B) Representative confocal image showing autophagic targeting of cytotoxic aggresomes in ALSC9orf72 MN upon K+ channel blockade. Scale bar: 5 µm. Dashed line represents the cell soma. (C) Apamin and XE991 decrease the size of aberrant aggresomes, confirming enhanced autophagy degradation (one-way ANOVA followed by Dunnett's multiple comparison test). n=3 independent treatments with the ALSC9orf72 I line. Scale bar: 5 Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (D) K+ channel blockers increase the levels of autophagy-specific phospho-SQSTM1/p62S403 in ALSC9orf72 MN (one-way ANOVA followed by Dunnett's multiple comparison test). n=3 independent treatments with the ALSC9orf72 I line. Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (E) The load of aggregated SQSTM1/p62 is also reduced by the treatments in ALSC9orf72 cultures (one-way ANOVA followed by Dunnett's multiple comparison test). n=3 independent treatments for each cell line. Scale bar: 20 µm. Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (F) Treatment with the K+ channel blockers reduces the number of GGGGCC toxic RNA foci in ALSC9orf72 MN. n=3 independent treatments with each hiPSC line (one-way ANOVA followed by Dunnett's multiple comparison test). Scale bar: 5 µm. Dashed line represents the cell soma. Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. \n",
      ".Figure 1(A) Heatmap representing the differentially expressed genes in ALSC9orf72 (ALSC9orf72 I and ALSC9orf72 II lines) vs Healthy (Healthy I and CorrectedC9orf72 lines) MN transcriptomes. Colour scale represents gene expression relative to the Healthy genotype. GO (biological processes) terms of the top significant down- or up-regulated pathways in ALSC9orf72 MN (hypergeometric test). GO (biological processes) terms of the top significant up-regulated pathways in ALSC9orf72 MN (hypergeometric test). (D) Reactome representation of clustering and distribution of the significantly up- and down-regulated pathways in ALSC9orf72 MN. (E) PASTAA ranking of the top 3 transcription factors involved in the regulation of the differentially expressed genes in ALSC9orf72 cultures. Significance was set with the algorithm published in Roider et al. (2009). (F) Single tube qPCR experiments showing the down-regulation of the synaptic genes NLGN3, SLC6A, TRIM9, SYT1, SYNGR1, and SNAP91 in ALSC9orf72 MN (Welch's t-test). n=3 independent cultures for each hiPSC line. Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (G-H) Time course analysis of nuclear pCREBS133 levels and MN survival in ALSC9orf72 and Healthy cultures (two-way ANOVA). n=3 independent cultures for each hiPSC line (for each time point). Scale bar: 25 µm. Data information: *p<0.05; **p<0.01; ***p<0.001. Error bars represent SEM. Arrow indicates the structure displayed at higher magnification. Exact p-values are reported in Appendix Table S1. (I) Representative confocal image showing colocalization of the CREB interactor CBP within cytotoxic aggresomes. Scale bar: 5 µm. Dashed line represents the cell soma. \n",
      ".Figure 3(A-B) Volcano plots obtained by DESeq2 analysis showing the effect on the transcriptome of ALSC9orf72 MN exerted by Apamin and XE991 in the ALSC9orf72 II and ALSC9orf72 III lines. (C) PASTAA ranking of the top transcription factors activated by Apamin and XE991. Significance was set with the algorithm published in Roider et al. (2009). (D) Venn diagram and showing the shared 263 CREB-controlled genes significantly altered by treatment with K+ channel blockers; the bar plot represents the significantly altered genes involved in neuronal activity and autophagy obtained by DESeq2 analysis. (E) Apamin and XE991 increase the levels of pCREBS133 in all the ALSC9orf72 lines (one-way ANOVA followed by Dunnett's multiple comparison test). n=3 independent treatments with each hiPSC line. Scale bar: 10 µm. Data information: *p<0.05; **p<0.01. Error bars represent SEM. Exact p-values are reported in Appendix Table S1. (F) ALSC9orf72 MN showed an increased number of Homer1b/c+ synaptic contacts after treatment with the K+ channel blockers, which in contrast had no significant effect on Healthy MN. n=3 independent treatments with each hiPSC line (two-way ANOVA). Scale bar: 5 µm Data information: *p<0.05; **p<0.01. Error bars represent SEM. Exact p-values are reported in Appendix Table S1. (G) Venn diagram showing the number of proteins significantly altered upon Apamin and XE991 treatment in ALSC9orf72 MN. The lines ALSC9orf72 II and ALSC9orf72 III were used as representative of the ALS genotype. (H-I) Volcano plot displaying the up- and down-regulation of the proteins altered by Apamin and XE991 (t-test). The blue dots indicate proteins involved in synaptic structure and function. (J) Up-regulated GO (biological processes) terms in ALSC9orf72 MN treated with either K+ channel blockers (Fisher enrichment test). (K) Synapse- and neuronal processes-related proteins significantly up-regulated by both treatments. \n",
      ": Glia specific loss of ADAM17 results in age-associated retinal degeneration A.Figure 1A. Diagram of a tangential (left) and horizontal (right) section of the adult Drosophila retina, with photoreceptors highlighted in violet and the pigmented glial cells (PGCs) in grey. PR: photoreceptor; R: rhabdomere; CB: cell body. B-K. Transmission electron microscopy (TEM) images of 5 week old adult retinas showing the overall structure of groups of ommatidia in (B) wild type; (C) ADAM17-/- mutant; (D) ADAM17-/Deficiency; (E, F) RNAi against ADAM17 and control (lacZ), expressed throughout the retina; (G, H) RNAi against ADAM17 and control (lacZ), expressed in the neurons; (I,J) RNAi against ADAM17 and control (lacZ), expressed in the pigmented glial cells; and (K) a kuz-/- mutant. L. Quantitation of the percentage of normal, abnormal and missing rhabdomeres from TEM images for the genotypes above, expressed as a stacked bar chart; n=180 ommatidia from 3 different fly retinas for each. M. Whole-mount retina stained with anti-ADAM17 (green) and phalloidin to mark the photoreceptor membranes (red). N. Line intensity profiles delineating expression of ADAM17 and phalloidin across individual ommatidia, n=10 fly retinas. Scale bars: 10μm. \n",
      ": Abnormal LD accumulation in young adult retinas upon a loss of ADAM17 in glial cells A-C.Figure 2A-C. TEM images of 1day old adult retinas showing the overall structure of clusters of ommatidia (A1, B1, C1), or a single ommatidium (A2, B2, C2) in wild type (A1, A2), ADAM17-/- mutant (B1, B2) or ADAM17-/Df (C1, C2). Red arrows show lipid droplets in PGCs, green shading highlights PGCs. Scale bars for A2, B2 and C2: 2μm and 10μm for all other panels. D-J. Fluorescent images of 1 day old fly retinas stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and the photoreceptor membranes respectively; (D) wild type; (E) ADAM17-/- mutant; (F) ADAM17-/ Deficiency; (G) knockdown of ADAM17 throughout the retina; (H) knockdown in glial cells; (I) knockdown in neurons; and (J) kuz-/-; n=10 fly retinas. Scale bars 10μm Quantitation of the BODIPY signal shown as integrated total lipid for the genotypes The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001, **p<0.01. Quantitation of the BODIPY signal shown as normalised LD size of lipid droplets for the genotypes The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001, **p<0.01. M-N. Q-PCR analysis of mRNA transcripts of lipogenic genes -Acetyl CoA carboxylase (ACC) and Fatty Acid Synthase (FASN) from heads of wild type and ADAM17-/- mutants; n=4 and n=3 biological replicates for M and N respectively, with 3 technical replicates for each genotype. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. \n",
      ": Regulation of ROS and peroxidated LDs by ADAM17 and components of the TNF pathway.Figure 8Cleavage of full length TNF/Eiger by ADAM17 leads to the production of soluble TNF/Eiger, which can engage the TNF receptor, Grindelwald. TNF signalling inhibits the production of mitochondrial ROS inside pigmented glial cells, through unknown mechanisms. In the absence of TNF signalling, glial generated ROS combines with ROS from neighbouring photoreceptor neurons to trigger production of LDs. Upon their later dispersal, the combination of fatty acids and ROS leads to high levels of toxic peroxidated lipids, which are toxic to the glia. Finally, glial degeneration is followed by neuronal degeneration. \n",
      ".Figure 1Western blotting analysis of Gadd45a protein expression in mouse intestine crypt lysates from WT and G3Terc-/- mice (A). Quantification of Gadd45a protein expression, β-actin protein level was used as the loading control (n=7 mice per genotype), two independent experiments were carried out (B). Quantification of Gadd45a-ChIP values as percentage of total telomeric DNA and ALU elements in WT and G3Terc-/- MEF cells (n=3 for each genotype). Dot-bot assay following Gadd45a-ChIP in WT and G3Terc-/- MEF cells. Kaplan-Meier survival curves of the mice with indicated genotypes (WT, n=44; Gadd45a-/-, n=48; G3Terc-/-, n=37; G3-dKO, n=41). P value comparing the survival of G3Terc-/- with G3-dKO mice is calculated with the log-rank test. The average body weight of mice with indicated genotypes at indicated time points (G3-dKO Female (n=19) vs G3Terc-/- Female (n=17), P=0.004; G3-dKO Male (n=14) vs G3Terc-/- Male (n=14), P=0.0038). P values are calculated with log-rank test. Whole mount images of colon samples from 7-month old mice with indicated genotypes (G). Quantification on density of colon crypts (H) (n=5 mice per group). H&E-stained small intestine sections from 7-month old mice with indicated genotypes (I). Quantification of the number of small intestine basal crypts (J) (n=5-6 mice per group). \n",
      ": Loss of either Eiger or Grindelwald, but not Wengen, leads to abnormal LD accumulation and age-associated retinal degeneration.Figure 4A. Alkaline phosphatase (AP)-shedding assays performed for Eiger in S2R+ cells expressing either GFP, Drosophila ADAM17, or activity dead mutants of Drosophila ADAM17, in the presence of either PMA, or PMA and TAPI-1 (DMSO is used as control; see methods); n=5. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. B-C. Fluorescent images of retinas from 1 day old flies grown on either DMSO or TAPI-1, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively. Scale bars: 10μm. D. Integrated total lipid counts from retinas corresponding to flies reared either on DMSO or TAPI-1 starting from larval stages. n=10 fly retinas. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. E-H. Fluorescent images of 1week old adult fly retinas, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively: (E) wild type; (F) eiger-/-; (G) grindelwald-/-; (H) wengen-/-. Scale bars: 10μm. I. Quantitation of the BODIPY signal shown as integrated total lipid measurements for the genotypes mentioned above. n=10 fly retinas. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean and the diamond an outlier. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. J-K. mRNA level transcripts of ACC and FASN measured from head lysates of one week old wild type, eiger-/- and grindelwald-/- . n=3 biological replicates, with 3 technical replicates for each genotype. The box end points represent the maximum and minimum values respectively, the central band is the median and the square is the mean Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. L-O. TEM images of two week old retinas corresponding to (L) control; (M) eiger-/-; (N) grindelwald-/-; (O) wengen-/-. n=180 ommatidia from 3 different fly retinas for each. Scale bars: 10μm. P. Quantitation of the percentage of normal, abnormal and missing rhabdomeres from 2 week old retinas observed in TEM, corresponding to the genotypes mentioned above; n=180 ommatidia from 3 different fly retinas for each. Data were quantified for significance using Student's t test. ***p<0.001, **p<0.01. \n",
      ": Activity dependence and role of JNK in driving LD accumulation and retinal degeneration in ADAM17-/- mutants.Figure 6A-B. Fluorescent images of 1 day old adult fly retinas, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively: (A) wild type; (B) ADAM17-/-, both reared in the dark. Scale bars: 10μm. C-F. TEM images of 4 week old retinas: (C, D) wildtype, or (E, F) ADAM17-/- flies reared from first larval instar either in total light (C, E) or total dark (D, F). Scale bars: 10μm. G. Quantitation of the BODIPY signal shown as integrated total lipid from 1 day old retinas corresponding to the genotypes described in A-B, all reared in the dark. n=10 for each. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. **p<0.01. H. Percentage of normal, abnormal and missing rhabdomeres in the genotypes and treatments mentioned above. n=180 ommatidia from 3 different fly retinas for each. Data were quantified for significance using Student's t test. **p<0.01. I. Measurement of MDA levels (in nmoles) from 1 day old retinas of wild type or ADAM17-/- , reared either in light or dark. Four biological replicates with 3 technical replicates for each genotype. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. **p<0.01. J. mRNA levels of the puckered transcripts from retinas of untreated or AD4 treated wild type and ADAM17-/- mutant retinas. n=3 biological replicates, with 3 technical replicates for each genotype. The box end points represent the maximum and minimum values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. **p<0.01. K-L. Fluorescent images of 1 day old adult fly retinas, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively: (K) heterozygous mutant of bsk alone; or (L) in combination with ADAM17-/-. Scale bars: 10μm. M. Quantitation of the BODIPY signal shown as integrated total lipid for the genotypes mentioned in K-L, along with w1118 and ADAM17-/-. n=10 flies per genotype. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. **p<0.01. N-O. TEM images of 2 week old adult retinas, showing clusters of ommatidia for a heterozygous mutant of bsk alone (M) or in combination with ADAM17-/- (N); n=3 for each. Scale bars: 10μm. P. Percentage of normal, abnormal and missing rhabdomeres for the genotypes mentioned above; n=180 ommatidia from 3 different fly retinas for each. Data were quantified for significance using Student's t test. **p<0.01. \n",
      ": Reducing mitochondrial ROS levels in ADAM17-/- mutants leads to a rescue of the LD accumulation and retinal degeneration.Figure 5A-D. Fluorescent images of 1 day old adult fly retinas, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively: (A, B) wild type; (C, D) ADAM17-/- mutants, reared either on DMSO or AD4. Scale bars: 10μm. E. Integrated total lipid counts from retinas corresponding to the genotypes and treatments mentioned above; n=10 for each. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001. F-I. TEM images of 2 week aged retinas corresponding to either wildtype or ADAM17-/- mutants grown on either DMSO or AD4. Scale bars: 10μm. J. Percentage of normal, abnormal or missing rhabdomeres from retinas corresponding to the genotypes and treatments mentioned above. n=180 ommatidia from 3 different fly retinas for each. K-N. Fluorescent images of 1 day old adult fly retinas, stained with BODIPY (green) and FM-dye (red) to mark lipid droplets and photoreceptor membranes respectively: (K) ADAM17-/- ; (L) glial specific SOD2 over-expression in an ADAM17-/- mutant; (M) glial specific SOD1 over-expression in an ADAM17-/- mutant; (N) neuronal specific SOD2 overexpression in ADAM17-/- mutant. Scale bars: 10μm. O. Quantitation of the BODIPY signal shown as Integrated total lipid for the genotypes mentioned above, n=10 for each. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean, and the diamond an outlier. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001, **p<0.01, *p<0.05. P-Q. TEM analysis of 5 week aged retinas corresponding to either ADAM17-/- mutants or SOD2 over-expression in an ADAM17-/- mutant. Scale bars: 10μm. R. Percentage of normal, abnormal and missing rhabdomeres in ADAM17-/- mutant retinas, as compared to retinas overexpressing glial specific SOD2 in an ADAM17-/- mutant background observed with TEM; n=180 ommatidia from 3 different fly retinas for each. S. Oxyblot analysis of oxidative modification of proteins from retina lysates of wild type and ADAM17-/-. T. Analysis of Oxyblot intensity normalised to its respective actin controls for each genotype. n=3 biological replicates, with 3 technical replicates for each genotype. The box end points represent the maximum and minimum values respectively, the central band is the median and the square is the mean. Data were quantified for significance using Student's t test. **p<0.01. Scale bars: 10μm. \n",
      ": Inhibiting ADAM17 activity in human iPSC derived microglia cells leads to abnormal LD phenotypes.Figure 7A-C. Fluorescent images of human iPSC derived microglia cells, labelled with 493/503 BODIPY (green) and FM-dye (red) to mark lipids and membranes respectively. iPSC derived microglia cells were obtained from 3 different donors, and treated with either DMSO, GW, TAPI-1 or GI. Scale bars: 10μm Quantitation of the BODIPY signal shown as integrated total lipid for cells corresponding to each treatment and donor n=15 cells for each treatment. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean, and the diamond an outlier. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001. Quantitation of the BODIPY signal shown as normalised LD size (normalised to DMSO) for cells corresponding to each treatment and donor mentioned =15 cells for each treatment. The box end points represent the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median, the square is the mean, and the diamond an outlier. Data were analysed using the Kruskal Wallis test followed by Dunn's test for post hoc analysis for significance due to unequal sample sizes. ***p<0.001. F-G. Q-PCR analysis of mRNA transcripts of lipogenic genes FASN and LDLR in pooled data obtained from 3 different donors of human iPSC derived microglia treated with either DMSO, GW or GI for 24 hours. n=3 biological replicates, with 3 technical replicates for each genotype. The box end points represent the maximum and minimum values respectively, the central band is the median and the square is the mean. all data were quantified for significance using Student's t test. ***p<0.001, **p<0.01, *p<0.05. H-J. Fluorescent images of Mitotracker and MitoSOX labelling of cells from Donor number 2 treated for 24hours with either DMSO, GW or GI. Scale bars: 10μm K. Quantitative measurements of MitoSOX intensity normalised to that of Mitotracker in iPSC derived microglia cells from three different donors, treated with either DMSO, GW or GI; n=10 cells for each treatment per donor. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. all data were quantified for significance using Student's t test. ***p<0.001, **p<0.01, *p<0.05. L-N. Fluorescent images of C-11 BODIPY treated microglia cells from Donor no.1, treated for 24 hours with either DMSO, GW or GI. These representative images display relative amounts of peroxidated and non-peroxidated lipids. Scale bars: 10μm O. Quantification of BODIPY C-11 staining, shown as ratios of peroxidated versus non-peroxidated lipids, depicted as normalised intensity; n=10 cells for each treatment per donor. The box end points are the upper (75%) and lower (25%) quartiles, the whiskers define the maximum 95th percentile and minimum 5th percentile values respectively, the central band is the median and the square is the mean. all data were quantified for significance using Student's t test. ***p<0.001, **p<0.01, *p<0.05. \n",
      ".Figure 1(A) General sequence alignment of SOX9 high affinity CDC4 phosphodegron (CPD) motif against other affinity motifs present in known FBW7α substrates including Cyclin E, cMYC, and c-Jun. (B) Immunoblot analysis of the immunoprecipitated HA-SOX9 or FLAG- FBW7α and their pull-down products. The corresponding Western blotting from the whole cell extract (WCE) of the transfected HEK293 showed the levels of exogenous HA-SOX9 or FLAG-FBW7α proteins in the total input. Cells were treated with 10 µM MG132 for 4 hours prior to harvesting for the immunoprecipitation. Blots are representative from 3 independent experiments. (C) FBW7α immunoblot eluted from the agarose bead-bound peptide, which had been incubated with the recombinant SCF-FBW7α for 1 hour at 37◦C. The agarose bead-bound peptide contains either the non-phosphorylated SOX9 amino acid sequence spanning from residue number 231 to 245 (SOX9 peptide), or those with the threonine corresponding to residue number 236 and 240 being phosphorylated (pSOX9 peptide). The input (10%) showed the level of the supplemented recombinant SCF-FBW7α in the in vitro binding reaction. Shown blot is representative from 2 independent experiments. (D) Representative Western blotting set from 3 independent repeats of pT236-SOX9 from lysates of HEK293 transfected with either HA-EV or HA-SOX9. Each lysate were immediately divided and left untreated or subjected to lambda phosphatase (λ-PPase) treatment for 1 hour at 37◦C prior to gel electrophoresis. SOX9 blot showed the presence of SOX9 protein in both the untreated and the phosphatase-treated SOX9-transfected cell lysates. (E) Bead-immobilized in vitro translated (IVT) HA-SOX9 wild type (WT) or T236/240 mutant were either untreated or subjected to in vitro GSK3 kinase reaction for 90 minutes at 37◦C prior to elution and gel electrophoresis. The SOX9 blot showed total SOX9 protein eluted from the beads from each reaction. Shown immunoblots are single representative experiment from 3 independent repeats. (F) GSK3-mediated phosphorylation of threonine 236 promoted SOX9 interaction with recombinant SCF-FBW7α in vitro. Bead-immobilized IVT HA-SOX9 were either untreated or phosphorylated with GSK3 as described in (E) prior to further incubation with recombinant SCF-FBW7α for in vitro binding assay. Immunoblot showed FBW7α protein in the in vitro binding reaction (input) as well as those which had been eluted from the untreated and GSK3-treated immobilized SOX9. The set of blots shown is representative from 3 independent experiments. \n",
      ".Figure 4CpG island methylation status of subtelomeric regions on chromosome 1q. gDNAs from MEF with indicated genotypes were PCR amplified and analysed by bisulfite sequencing. Yellow and blue bars denote the frequencies of methylated and unmethylated CpG islands, respectively, at each position (A). Quantification of subtelomeric CpG island methylation (B). Telomeric quantification and dot-bot assay of tri-methyl H3K9 (H3K9me3)-ChIP in indicated genotypes Telomeric quantification and dot-bot assay of acetylated H3K9 (H3K9ac)-ChIP in indicated genotypes Telomeric quantification and dot-bot assay of HP1a-ChIP in indicated genotypes \n",
      ".Figure 2Olfm4-positive intestinal stem cells (indicated by arrows) in small intestine sections (A). Quantification of intestinal stem cell (Olfm4+) number per 100 μm (n=4-5 mice per group) of small intestine sections (B). Representative images of small intestine crypt culture (day 2). Arrows mark the surviving cysts (C). Quantification of numbers of cysts per culture well (D)(n=4-5 mice/genotype). In vitro crypt culture (day 8) from small intestine crypts isolated from mice with indicated genotypes. Arrows mark the crypt-like structure (E). The number of organoids per culture wells (n=4-5 mice per group). Two hundred crypts are seeded per well (F). Representative images of TUNEL staining in small intestine sections. Arrows indicate TUNEL positive cells (G). Quantification of TUNEL positive cells per crypt (n=5 mice per group) (H). Representative images of phosphorylated histone 3 (pH3) antibody staining in mouse small intestine. Arrows indicate pH3 positive cells (I). Quantification of pH3 positive crypt cells per 100 μm of small intestines (n=4-6 mice per group) (J). \n",
      ".Figure 6The knockdown efficiency of shRNA-GADD45A in human WI38 fibroblasts. The proliferation curves of human WI38 fibroblasts treated with control shRNA or shRNA against GADD45A. Representative images of β-gal staining in human fibroblast cultures with control and shRNA-GADD45A treatments at passage 54 (C). Quantification of the percentage of β-gal positive cells in human fibroblasts with control and shRNA-GADD45A treatments (duplicates per group) (D). Representative images of GADD45A and γH2AX staining in colon sections from young (35-50 y/o) and old (75-85 y/o) people (E). Quantification of the percentage of GADD45A positive cells in young and old human colon tissues (n=8 per age group) (F). Quantification of the percentage of γH2AX positive cells in young and old human colon tissues (n=6 per age group) (G). Model for the role of Gadd45a in regulation of DNA damage response initiation. In G3Terc-/- mice, Gadd45a promotes the accessibility of telomeric heterochromatin and enhances the DNA damage response initiation (e.g. pATM activation) through a BER-dependent pathway. The initiated DNA damage response further causes upregulation of p53 and p21, which leads to impairment of intestine stem cell maintenance and function. In G3-dKO mice, the telomeric heterochromatin is condensed and damage response initiation is reduced upon Gadd45a deletion, which further leads to decreased expression of p53 and p21. Therefore, the maintenance and function of intestine stem cell is improved. \n",
      ".Figure 3Representative images of immunostaining for p53 on small intestine sections. Arrows indicate p53-positive cells (A). Quantification of the percentage of p53 positive cells per crypt (n=5-6 mice per group) (B). Representative images of γH2AX antibody staining on small intestine paraffin sections from WT, Gadd45a-/-, G3Terc-/-, and G3-dKO mice (7 months old). Arrows indicate γH2AX positive nuclei (C). Quantification of the percentage of γH2AX positive cells per small intestine crypt (n=4-6 mice per group) (D). Representative images of small intestine crypt sections stained with antibody against pATM (Ser-1981)(E). Quantification of frequencies of pATM (Ser-1981) positive crypt cells (n=4-6 of 7-month old mice per genotype) (F). Representative image of the anaphase bridge (indicated with arrows) in the basal crypt of G3Terc-/- mice (G). The percentage of cells with anaphase bridges in pH3 positive basal crypt cell population (n=4-6 mice per genotype) (H). Heat map indicating the relative expression levels of DDR and cell cycle-related gene transcripts in MEFs from indicated genotypes (Data are extracted from RNAseq datasets). \n",
      ".Figure 5Representative images of small intestine organoids from WT and G3Terc-/- mice incubated with or without APE1 inhibitor (CRT0044876, 10μM) treatment. Arrows indicate the crypt-like structure in organoid cultures (A). Quantification of organoid number (per well) and size (in 104 μm2) (n=3-4 mice per genotype/treatment) (B-C). Representative images of γH2AX antibody staining on dissociated organoid cells from WT and G3Terc-/- organoid cultures with or without APE1 inhibitor (CRT0044876, 10μM) (DNAs are stained with DAPI) (D). Quantification of γH2AX positive cells (n=3-4 mice per group) (E). Telomeric DNA quantification and dot-blot assay of H3K9me3-ChIP Telomeric DNA quantification and dot-blot assay of H3K9ac-ChIP Telomeric DNA quantification and dot-blot assay of HP1a-ChIP Representative images of Olfm4 staining in small intestine sections from mice treated with or without APE1 inhibitor (CRT0044876, CRT). Arrows indicate Olfm4 positive intestine stem cells (I). Quantification of Olfm4 positive intestine stem cells (n=4 mice per group) (J). \n",
      ".Figure 2(A) Endogenous SOX9 protein turnover in the presence of cycloheximide following RNAi depletion of FBW7α, β or γ. HEK293 cells were transfected with 20 nM of non-targeting scramble RNA or those specifically targeting FBW7α, β or γ for 72 hours prior to chase with addition of 100 ng /mL cycloheximide. Immunoblot of Cyclin E, an established SCF-FBW7 substrate, was used to assess the efficacy of RNAi-mediated depletion of FBW7 protein with GAPDH served as protein loading control. (B) Immunofluorescence staining of D324MED medulloblastoma cells showing nuclear co-localization of HA-SOX9 WT (AlexaFluor 568; Red) and FLAG-FBW7α (AlexaFluor 488; Green). The cell nuclei were counter-stained with DAPI (blue). Image set is representative for multiple image fields taken at 40x objective magnification. Scale bar indicate 20 µm. (C) Western blotting of HEK293 endogenous SOX9 proteins 24 hours following forced expression of 100, 250, 500, and 1000 ng of plasmid expressing FLAG-FBW7α. Changes in SOX9 protein level in the representative blots were analysed relative to GAPDH using ImageJ. The set of blots shown is representative from 3 independent experiments. (D) Expression of FLAG-FBW7α enhanced HA-SOX9-WT protein turnover, which can be attenuated with addition of 10 µM MG-132 (MG) in HEK293 cells. By contrast, forced expression of FLAG-FBW7α did not enhance HA-SOX9-T236A and -T236A/T240A protein turnover over the course of 4 hours protein chase. Protein loading for each sample was indicated by the level of β-actin protein. The set of blots shown is representative from 4 independent experiments. (E) Treatment of HEK293 cells with increasing concentration of BIO, a GSK3α/β inhibitor, increased HA-SOX9-WT protein level in dose-dependent manner. HA-SOX9-WT and FLAG-FBW7α were co-expressed in HEK293 overnight prior to treatment with various concentration of BIO for 4 hours. Whole cell lysates were collected for Western blotting with protein loading for each sample adjusted to GAPDH protein level. The set of blots shown is representative from 2 independent experiments. (F) The RNAi depletion of GSK3ß decreased FBW7α-induced HA-SOX9-WT turnover in HEK293 cells. The HA-SOX9-WT protein turnover was examined in the presence of 100 ng/mL cycloheximide, 48 hours following si-GSK3ß (20 nM) transfection, 24 hours after co-expression of HA-SOX9-WT and FLAG-FBW7α. Immunoblots of GSK3ß demonstrated depletion of GSK3ß protein with the siRNA. Changes in SOX9 protein level in the representative blots were analysed relative to GAPDH using ImageJ. The set of blots shown is representative from 3 independent experiments. (G) Forced FLAG-FBW7α-WT expression promoted poly-ubiquitylation of endogenous SOX9 in HEK293, which accumulated further in the presence of 10 µM MG132. Expression of non-functional FBW7α lacking the F-box domain (ΔF), or those containing R465A mutation did not induce SOX9 poly-ubiquitylation in vivo. Ubiquitylation assay was performed under denaturing condition to disrupt non-covalently linked ubiquitin as described in the Materials Methods. Expression of various FLAG-FBW7α constructs and endogenous SOX9 protein were examined in the whole cell extract (WCE). GAPDH protein was used to indicate total protein loading with protein loading. The set of blots shown is representative from 4 independent experiments. (H) Reconstitution of SOX9 ubiquitylation reaction by FLAG-FBW7α in vitro. Immobilized-IVT HA-SOX9 WT were incubated with complete ubiquitylation reaction mixture containing either the eluted FLAG-FBW7α-WT or the ΔF mutant for 60 minutes at 37◦C. Reaction mixture lacking the UbE1, an E1 ubiquitin-activating enzyme, or UbcH3, an E2 ubiquiting-conjugating enzyme, served as negative control for the reaction. SOX9 ubiquitylation was assessed following elution of the protein from the immobilized beads under denaturing condition as described in Materials Methods. The set of blots shown is representative from 3 independent experiments. \n",
      ".Figure 8SCFFBW7α together with GSK3 regulate SOX9 protein levels through the ubiquitin-proteasome system. FBW7α mutations, such as those detected in SHH-medulloblastoma, lead to stabilization of SOX9 protein and subsequently promote SOX9-driven EMT-like reprogramming in medulloblastoma. Small molecule inhibitors of the PI3K/ mTOR pathway can be used to promote GSK3 / FBW7α-mediated SOX9 turnover and may provide a novel strategy to target SOX9-driven medulloblastoma. \n",
      ".Figure 1A Gene enrichments analysis on differentially expressed genes between sorafenib-sensitive and sorafenib-resistant human liver tumors. B Venn diagrams show overlapped genes in signaling pathways response to sorafenib. Heatmap of overlapped expressed genes in response to sorafenib-resistance. D The mRNA expression level of METTL3 in sorafenib-sensitive (n=3) and sorafenib-resistant (n=3) human liver tumors. Data information: In all relevant panels, ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. E Expression of METTL3 was significantly down-regulated in advanced stage of liver cancer from TCGA. F METTL3 RNA levels between naïve HepG-2 cells (n=3) and sorafenib-resistant HepG-2 cells (n=10). Data information: In all relevant panels, ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. G The IC50 of sorafenib-resistant HepG-2 cells treated with sorafenib under hypoxia condition (1%O2). Data information: In all relevant panels, ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. H The protein level of METTL3 in sorafenib-resistant HepG-2 cells treated with sorafenib under hypoxia condition (1%O2). I The global RNA m6A level in naïve HepG-2 and sorafenib-resistant HepG-2 determined by dot-blotting assay under hypoxia condition (1%O2). \n",
      ".Figure 5(A) Scatter plot depicting the relationship between transcriptional changes measured as log2 FC between EV to SOX9-WT (x-axis) and EV to SOX9-T236/T240A (y-axis). Each dot represents one transcript, with colored dots indicating transcripts upregulated (log2 FC>0.585; p<0.05; red dots) or downregulated (log2 FC<-0.585; p<0.05; blue dots) in both comparisons. The transcript corresponding to the SOX9 gene is highlighted as a red filled dot. Indicated r-value indicates Spearman's rank correlation coefficient between the two FC profiles. (B) Box-and-Whisker plot displaying the sample specific expression of three FBW7 isoforms; FBW7α (NM_033632), FBW7β (NM_018315) and FBW7γ(NM_001013415'). Each box represents the expression distribution over three replicates. (C) Venn diagram illustrating the overlap of differentially expressed (FC>1.5, p<0.05) transcripts between any of the three comparisons: EV to SOX9-WT (upregulated: 145+308+21; downregulated 146+333+44), EV to SOX9-T236/T240A (upregulated: 294+308+95+21; downregulated 345+333+95+44), or SOX9-WT to SOX9- T236/T240A (upregulated: 112+95+21; downregulated 114+95+44). Each oval represents the number of transcripts either upregulated or downregulated in the respective comparison, where the direction of differential expression is indicated by the inequality sign in the label. Numbers in overlapping regions signify the number of transcripts commonly differentially expressed in all contributing comparisons. (D) Heat map of the row-wise z-scores of 16 EMT hallmark genes in SOX9-WT and -T236/T240A samples. Heat map was generated using the GenePattern software (Reich et al., 2006). (E) Quantitative PCR (qPCR) of SNAI2 (SLUG), VIM and POU3F1 (OCT6) (normalized to GAPDH levels) in MB002 cells stably transduced with EV (set to 1), SOX9-WT and SOX9-T236/240A. Cells were induced for 8 or 24 hours with doxycycline. (F) Western blot of SOX9, HA-SOX9, pSOX9-T236, SNAI2 (SLUG), VIM and POU3F1 (OCT6) in MB002 cells stably transduced with EV, SOX9-WT and SOX9-T236/240A. Cells were induced for 24 and 48 hours with doxycycline and GAPDH were used as a loading control. (G) Heat map of the row-wise z-scores of 14 pro-metastasis genes differentially expressed between SOX9-WT and -T236/T240A samples and previously identified as SOX9 target genes (Kadaja et al., 2014, Larsimont et al., 2015, Oh et al., 2014). Heat map was generated using the GenePattern software (Reich et al., 2006). \n",
      ".Figure 3(A) FBW7 mutations across medulloblastoma molecular subgroup as identified by exome sequencing. In the SHH medulloblastoma subgroup, FBW7 mutations occurred approximately 11 % in a cohort containing 133 SHH medcases (Kool et al., 2014) and can be categorized as stop-gain (n=3), non-synonymous (n=7). One case of splicing error mutation is detected in the Group 3 medulloblastoma. (B) Immunoblotting of HA-SOX9-WT protein following expression of various FLAG-FBW7α constructs including wild type (WT), or those with R465A or R479A mutation in Daoy. The cells were transfected with equal amount of HA-SOX9-WT and FLAG-FBW7α and the whole cell lysates were collected 24 hours following the transfection. GAPDH proteins were shown to indicate total protein loading for each sample. The set of blots shown is representative from 2 independent experiments. (C) Co-immunoprecipitation of HA-SOX9 with either wild type (WT) or R465A- FLAG-FBW7α. Unlike the wild type protein, FLAG-FBW7α containing arginine 465 mutation did not significantly co-immunoprecipitate with HA-SOX9 (IP with anti HA antibody) and vice versa (IP with anti FLAG antibody). The expressions of HA-SOX9 and FLAG- FBW7α- WT or R465A mutant were assessed in the whole cell extract (WCE) of the transfected HEK293 with GAPDH protein used to indicate total protein loading. (D) Comparison of FBW7 mRNA expression across the medulloblastoma subgroups (WNT, SHH, Group 3, and 4) in a cohort containing 423 medulloblastoma cases. Expression of FBW7 in both normal fetal (n= 5) and adult cerebellum (n = 13) were used for comparison against the medulloblastoma. (E) Overall survival analysis of medulloblastoma patient based upon FBW7 expression level. Analysis was performed on 383 out of the 423 cases from which the survival data were available. Cohort was divided into high- (blue line) and low- FBW7 (red line) expressing subgroup using FBW7 median expression as group classifier. Statistical analysis was determined by using a LogRank test. (F) Correlation analysis between SOX9 protein/mRNA and FBW7 mRNA expression level from medulloblastoma tissue microarray consisting of 142 tissue samples. SOX9 protein level was scored following immunohistochemistry staining, while SOX9 and FBW7 mRNAs expression by the RNA Scope as described in Materials Methods. A modest negative correlation was determined by Kendall's correlation test (τ = - 0.223, p0.001) between SOX9 protein/mRNA and FBW7 expression. Affiliation of samples with medulloblastoma subgroups was indicated by different colored circle (blue=WNT, red=SHH, yellow=Group 3, green=Group 4, grey=Undetermined). \n",
      ".Figure 6(A) Cisplatin dose response curves of (i) MB002, (ii) Daoy, and (iii) MIC in the absence (EV) or presence of SOX9 by Alamar blue. Cells were pre-conditioned with doxycycline to induce expression of SOX9 (or EV) prior to treatment with increasing concentrations of cisplatin. The IC50 were calculated following 5 (MB002 and MIC) or 3 days (Daoy) of treatment. Data are mean + standard deviation from 3 independent repeats, each containing 5 technical replicates. (B) Cisplatin dose response curves of SOX9-expressing (i) Daoy and (ii) MIC in the absence or presence of FBW7α. Experiments and data analysis were performed as described in (A) (C) Overall survival analysis of mice bearing Daoy or Daoy-expressing dox-inducible SOX9 treated with cisplatin. The dox-preconditioned cells (105 cells) were orthotopically xenografted to Nude-Foxn1nu mice and left for 1 week to prior to being treated with vehicle control or cisplatin (2mg/kg) intraperitoneally for every other day for a total of 6 doses. (D) Heat map of the row-wise z-scores of 11 genes associated with cisplatin resistance in MB002 expressing Sox9-WT or Sox9-T236/T240A. Heat map was generated using the GenePattern software. (E) Quantitative analysis of ATP7A, DUSP2, and TTK mRNAs in MB002 following expression of SOX9-WT or SOX9-T236/240A. Total RNA were collected 24 hours following doxycycline treatment, from which cDNA were generated for qPCR. Data are mean mRNA level (normalized to B2M transcript) + standard deviation from 3 independent experiments with statistical significance were determined by Multiple comparisons 2-way ANOVA with Bonferroni's post-test. (F) Time course western blotting of HA-SOX9, ATP7A, DUSP2, ERK1/2 pThr202/Tyr204 and total ERK1/2 in MB002 cells following doxycycline induction of either EV, SOX9-WT or SOX9-T236/240A. GAPDH was used as a loading control. \n",
      ".Figure 7(A) Western blotting of pT236- and total SOX9 following treatment of Daoy with a panel of inhibitors targeting PI3K/AKT/mTOR pathway. The whole cell lysates were collected for gel electrophoresis following 6 hours treatment with 1 μM of AZD5363, 8186, or 2014. The set of blots shown is a representative from 3 independent repeats. (B) Western blotting of endogenous SOX9 protein turnover in the presence of PI3K/mTOR dual inhibitor, AZD2014. Daoy were transfected with either siScr or siFBW7 for 48 hours prior to further treatment with 1 µM AZD2014. The cycloheximide chase of SOX9 protein turnover was examined 2 hours following the inhibitor treatment. Immunobloting of Cyclin E protein was used to indicate the efficacy of siFBW7 transfection. Changes in SOX9 protein level in the representative blots were analysed relative to GAPDH using ImageJ. The set of blots shown is representative from 3 independent experiments. (C) Quantification of resazurin-based Daoy and MB002 cell viability following treatment with Cisplatin (blue bars), AZD2014 (red bars), or their combination (green bars). Daoy were treated for 72 hours, while MB002 were subjected to 5 day course of treatment. The cellviability were calculated relative to the vehicle-treated cells. The synthetic lethality combination index (CI) for each treatment was calculated from the mean of 3 independent experiments using the Compusyn software. Error bars indicate standard deviation. (D) Time course western blotting of endogenous SOX9 and apoptotic markers including cleaved PARP and caspase 3 following MB002 treatment with cisplatin, AZD2014, or their combination. β-actin were used as a loading control. The set of blots shown is representative from 3 independent experiments. (E) Western blotting profiling of MB002 expressing EV, SOX9-WT, or SOX9-T236/240A following 48 hours treatment with cisplatin, AZD2014, and their combination. The set of blots shown is representative from 3 independent experiments. (F) Quantification of cell viability of MB002 expressing EV, SOX9-WT, or SOX9-T236/240A following 5 days treatment with either of 10 μM cisplatin (black bars), 0.5 µM AZD2014 (blue bars), or their combination (red bars). Data are mean + standard deviation from 3 independent experiments, from which statistical significance were analysed by 2-way ANOVA multiple comparisons with Bonferroni's post-test. (G) Immunoblots of ATP7A and DUSP2 protein following doxycycline treatment of Daoy-SOX9/T-FBW7α. The cells were either untreated or incubated with 5 µg/mL doxycycline for 72 hours prior to being harvested for gel electrophoresis. The set of blots shown is representative from 2 independent experiments. \n",
      ".Figure 4(A) Overall survival analysis of mice bearing Daoy cell line expressing SOX9 with doxycycline (dox) -inducible FBW7α ( Daoy-SOX9/T-FBW7α). The dox-preconditioned or the control cells (105cells) were orthotopically xenografted to the cerebellum of 6-weeks old Athymic Nude-Foxn1nu mice, which were continuously fed with either the dox-containing chow (red line) or the regular chow food (green line), respectively. Micexenografted with dox-preconditioned Daoy-SOX9/T-FBW7α and fed continuously with dox-containing chow (red line) survived longer (p= 0.0285) than the control group (green line) (B) Representative of Haemotoxylin Eosin (Hamp;amp;E) staining of medulloblastoma tumors arising from the orthotopic xenograft experiment described in (A). Both the control (no dox-treated) and the dox-treated mice developed primary intracerebellar tumors (green and red arrows, respectively). SOX9-expressing Daoy tumors disseminated to the spinal cord (3/4; green asterisks), while the group continuously receiving dox-containing chow showed no metastatic spread (0/4). (C) Representative bright field images of migrating Daoy-SOX9/T-FBW7α in the absence or presence of Dox. Daoy-SOX9/T-FBW7α cells were either maintain in regular culture or pre-conditioned with dox prior to experiments. The cells (2 x 105cells / insert) were seeded and let to migrate across 5 μm filter pore toward the laminin-coated trans-well for 4 hours prior to fixation and staining with crystal violet. Histogram quantification of cell migration were performed by measuring the crystal violet absorbance of the stained-migrating cells and presented as mean + standard deviation from 2 independent experiments each containing 2 technical replicates. (D) Representative H&amp;amp;E staining of primary (arrow) and disseminated tumors (asterisk) arising from orthotopic xenograft of parental and SOX9-expressing medulloblastoma initiating cells (MIC and MIC-Sox9, respectively). Quantification of primary and metastatic tumors arising from both of MIC (n=16) and MIC-SOX9 (n=8) is shown in the histogram (p=0.0000136). (E) Quantitative comparison of MIC and MIC-SOX9 cells to migrate across 5 μm filter pore toward the laminin-coated trans-well for 4 hours. Quantification of cell migration were performed as described in (C) and presented as mean + standard error mean from 6 replicates. (F) Transwell migration of SOX9-expressing MIC with dox-inducible FBW7α or dominant negative ∆F FBW7α. The cells were either maintain in regular culture or pre-conditioned with dox prior to assessment of in vitro transwell migration as described in (C). Data are mean + standard deviation from 2 independent experiments each containing 2 technical replicates. (G) Representative bright field images of (F) (H) Correlation between clinical metastatic staging and SOX9 protein across the TMA of 142 cases of human medulloblastoma. The SOX9 protein levels were determined by IHC in Figure 3F. Tumors with SOX9 protein level frequently presented (p=0.038, Fisher's exact test,) with the M3 metastatic staging, spread to the spinal cord (red bars). Analysis was carried out by comparing patients with high SOX9 protein rank (Rank 5-7) versus patients with low SOX9 protein rank (Rank 1-3). \n",
      ".Figure 7A Graphic illustration of two human liver cancer PDXs mouse model based sorafenib (SOR) treatment regimen. In vivo analyses of tumor (B) in mice that were subcutaneously implanted with tumor tissues from two human liver cancer patients and divided into four groups. In vivo analyses of tumor growth (C) in mice that were subcutaneously implanted with tumor tissues from two human liver cancer patients and divided into four groups. In vivo analyses of tumor weight (D) in mice that were subcutaneously implanted with tumor tissues from two human liver cancer patients and divided into four groups. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. E H&E and immunohistochemical images of Ki67, LC3-B, VEGF-A in randomly selected PDX#1 and PDX#2 tumors. Quantification of Ki67 (F) expression in immunohistochemical images from PDX tumors by Image Pro Plus (IPP) analysis. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. Quantification of LC3-B (G), and VEGF-A (H) expression in immunohistochemical images from PDX tumors by Image Pro Plus (IPP) analysis. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. I Knockdown of METTL3 by siRNA in PDX#1 and PDX#2 tissues were confirmed by Western blot. J Working model. The molecular mechanisms of m6A-dependent sorafenib-resistance in liver cancer. The expression of METTL3 was down-regulated in sorafenib-resistant liver cancer, and caused decrease of FOXO3 by the m6A modification in an YTHDF1-dependent manner. Degradation of FOXO3 thus promoted autophagy-associated sorafenib-resistance in liver cancer. \n",
      ".Figure 1A, Western blot analysis of 293T cells treated with 20 μM erastin for the indicated times. B, H1299 cells transfected with a control siRNA (siCont.) or an RNF20-specific siRNA (siRNF20) and a wild-type H2B (H2BWT) or a K120R-mutated H2B (H2BK120R) were treated with 12 μM erastin (+) or untreated (-) for 24 h. Representative phase-contrast images were recorded (magnification, ×20), and the surviving cells were counted. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. C, Lipid ROS levels were assessed by flow cytometry after C11-BODIPY staining in cells treated Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. D, H1299 cells treated were subjected to transmission electron microscopy. Representative images are shown. E, Control H1299 cells (transfected with siCont. or H2BWT) and H2Bub1-depleted H1299 cells (transfected with siRNF20 or H2BK120R) were treated with erastin either with or without a ferroptosis-specific inhibitor, ferrostatin-1 (2 μM), for 24 h. Representative phase-contrast images were recorded (magnification, ×20), and the surviving cells were counted. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. \n",
      ".Figure 3A Electron micrographs of METTL3-knockdown SMMC-7721 cells under hypoxia for 48 h. Protein levels of LC3 I/II in METTL3-knockdown SMMC-7721 (B), Bel-7402 (C) cells under hypoxia condition for 48h. Protein levels of LC3 I/II in HepG-2 (D) cells under hypoxia condition for 48h. E Protein level of LC3 I/II in HepG-2 and sorafenib-resistant HepG-2 cells under hypoxia condition for 48h. F-H Overexpression of shRNA-resistant wild type METTL3 but not catalytic mutant METTL3 rescue autophagy phenotype in METTL3-knockdown SMMC-7721 cells (F), Bel-7402 cells (G) and sorafenib-resistant HepG2 cells (H) by representative immunostaining images of LC3 under hypoxia condition for 48h. Scale bar, 200μm. Quantification of the numbers of GFP-LC3 puncta/cell by Imaris in F-1, G-1 and H-1, respectively. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. I-K Protein levels of LC3 I/II in METTL3-knockdown SMMC-7721 cells (I), Bel-7402 cells (J) and HepG-2 cells (K) treated with or without 3-MA under hypoxia condition for 24h. L-N The IC50 of sorafenib in SMMC-7721 cells (L), Bel-7402 cells (M) and HepG-2 cells (N) treated with or without 3-MA for 24h under hypoxia condition (1%O2). Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. \n",
      ".Figure 2A-B Clonogenic survival of METTL3-knockdown (A) and METTL3-overexpression (B) in normal liver cell line WRL68 cells for 7 days in normoxia condition (21% O2) and quantification of clusters in A-1 and B-1, respectively. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. C-D The IC50 of METTL3-knockdown SMMC-7721 cells (C) and Bel-7402 cells (D) after treated with sorafenib for 24h under hypoxia condition (1%O2). Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. E Overexpression of wild type METTL3 or catalytic mutant METTL3 in sorafenib-resistant HepG-2 cells. F The global RNA m6A level in sorafenib-resistant HepG-2 cells with wild type METTL3-overexpression or catalytic mutant METTL3-overexpression by dot-blotting assay. G-I Rescue of shRNA-resistant wild type METTL3 but not catalytic mutant METTL3 sensitized METTL3-knockdown SMMC-7721 cells (G), Bel-7402 cells (H) and sorafenib-resistant HepG-2 cells (I) to sorafenib treatment. The IC50 of the cells was measured after treated with sorafenib for 24h under hypoxia condition (1%O2). Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. J Cell survival assay of sorafenib-resistant HepG-2 cells with wild type METTL3 overexpression or catalytic mutant METTL3 overexpression after treated with sorafenib for 24h under hypoxia condition (1%O2). Scale bar, 1mm. K Capillary-like structures in HUVECs treated with the tumor-conditioned medium (TCM) from control HCCs and METTL3-knockdown HCCs cultured under hypoxic conditions for 48 h. Quantification of the number of tubes with image J in K-1. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. L-M The relative mRNA expression levels of angiogenesis genes were detected in SMMC-7721 cells (L) and Bel-7402 cells (M) with METTL3-knockdown by RT-PCR assay. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. N The protein levels of VEGF-A in METTL3-knockdown HCCs. O The protein levels of PDGF-B in METTL3-knockdown HCCs. \n",
      ".Figure 5A Validation of biomarker genes related to autophagy in FOXO3-knockdown Bel-7402 cells by RT-PCR. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. B Protein level of FOXO3 in naïve HepG-2 and sorafenib-resistant HepG-2 cells under hypoxia for 48h. C Protein levels of LC3 I/II in FOXO3-knockdown SMMC-7721 cells. D Overexpression of FOXO3 in sorafenib-resistant HepG-2 cells by Western blot. E Protein levels of LC3 I/II in METTL3-knockdown & FOXO3-overexpression SMMC-7721 cells under hypoxia for 48 h. F Electron micrographs of METTL3-knockdown SMMC7721 cells and METTL3-knockdown & FOXO3-overexpression SMMC7721 cells under hypoxia for 48 h. G Representative Immunostaining images of LC3 in METTL3-knockdown SMMC7721 cells and METTL3-knockdown & FOXO3-overexpression SMMC7721 cells under hypoxia for 48 h. Scale bar, 200μm. The quantification of the numbers of GFP-LC3 puncta/cell by Imaris in G-1. H Representative Immunostaining images of LC3 in naïve HepG-2 cells and sorafenib-resistant HepG-2 cells under hypoxia for 48 h. Scale bar, 200μm. The quantification of the numbers of GFP-LC3 puncta/cell by Imaris in H-1. I The IC50 of METTL3-knockdown SMMC7721 cells with overexpressing FOXO3 after treated with sorafenib for 24h under hypoxia condition. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. J The IC50 of naïve HepG-2 cells and sorafenib-resistant HepG-2 cells with overexpressing FOXO3 after treated with sorafenib for 24h under hypoxia condition. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. K Cell survival assay of METTL3-knockdown SMMC7721 cells with overexpressing FOXO3 after treated with sorafenib for 24h under hypoxia condition. L Cell survival assay of naïve HepG-2 cells and sorafenib-resistant HepG-2 cells with overexpressing FOXO3 after treated with sorafenib for 24h under hypoxia condition. \n",
      ".Figure 2A, qRT-PCR (left) and Western blot (right) analyses of H1299 cells transfected with a control siRNA (siCont.) or an RNF20-specific siRNA (siRNF20) and a wild-type H2B (H2BWT) or a K120R-mutated H2B (H2BK120R) for 24 h. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. B, Chromatin immunoprecipitation (ChIP) assay was carried out with anti-H2Bub1 antibodies in H1299 cells (left) or 293T cells either untreated or treated with 20 μM erastin for 24 h (right). The intergenic region was used as a negative control for the occupancy of H2Bub1. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. C, Intracellular GSH levels were examined in H1299 cells treated as indicated, and bar graphs are shown. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. D, H1299 cells transfected as indicated were treated with 12 μM erastin (+) or untreated (-) for 24 h. Representative phase-contrast images were recorded (magnification, ×20). E, Surviving cells from the assay shown in (D) were counted. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. F, GO analysis with the genes downregulated in H2BK120R (black) or RNF20-sepecific siRNA (siRNF20) (red) transfected 293T cells by employing a previously reported microarray data Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. G, Affected metal ion-binding genes were selected and subjected to cluster analysis. H, Labile iron levels were assessed by flow cytometry with a standard method in H1299 cells. I, Labile iron levels examined were quantified. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05, **: p<0.01. \n",
      ".Figure 4A Gene Set Enrichment Analysis (GSEA) output images of cellular response to stress pathways displaying a correlation of differentially regulated genes in METTL3-knockdown HepG2, SMMC-7721 and Bel-7402 cells with the C5. biological process symbol set. (p < 0.001). The KEGG analysis shows that FOXO3 involved in hypoxia and autophagy. C Protein level of FOXO3 in METTL3-knockdown HCCs under hypoxia for 48h. D Half-life of FOXO3 mRNA in Bel-7402 cells and correspondent METTL3-knockdown Bel-7402 cells treated with actinomycin D. E Half-life of FOXO3 protein in Bel-7402 cells and correspondent METTL3-knockdown Bel-7402 cells treated with cycloheximide. Quantification of the protein optical density by Image J in E-1. F Analysis of FOXO3 mRNA in non-ribosome portion (<40S), 40S, 60S, 80S and polysome for the METTL3-knockdown Bel-7402 cells compare to control cells under hypoxia for 48h. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. G Knockdown of YTHDF1 and YTHDF2 in Bel-7402 cells by lentiviral shRNAs. H Knockdown of YTHDF1 but not YTHDF2 abolished the increase of FOXO3 protein level in METTL3-overexpressing Bel-7402 cells. Quantification of the protein optical density by Image J in H-1 and H-2. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. I RNA level of FOXO3 in YTHDF1-knockdown Bel-7402 cells by RT-PCR. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. J The m6A-IP sequencing under hypoxia proved the m6A modification participated in regulation of FOXO3. The YTHDF1 binding site locates at the 3'UTR of FOXO3. K 800 bp of FOXO3 3'UTR containing the m6A modification site was cloned into luciferase reporter vector. Mutation of m6A consensus sequence was generated by replacing adenosine with thymine. L Relative luciferase activity of the wild-type and mutant form of FOXO3 3'UTR reporter vectors in Bel-7402 cells with various levels of METTL3 and YTHDF1. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. M Knockdown of METTL3 reduced the m6A methylation in FOXO3 mRNA by the m6A MeRIP analysis. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. N Knockdown of METTL3 reduced the m6A methylation in FOXO3 mRNA by the YTHDF1-RIP analysis. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD and are representative of 3 independent experiments. O Correlation analysis of the RNA expression levels of METTL3 and FOXO3 in liver tumor cohort (Kim-90) by Pearson. P Correlation analysis of the RNA expression levels of YTHDF1 and FOXO3 in liver tumor cohort (Kim-90) by Pearson. \n",
      ".Figure 6A Depletion of METTL3 in Hepa1-6 cells by CRISPR-Cas9 knockout enhanced sorafenib-resistance in vivo. Growth curve (B) of Hepa1-6 cells derived xenografts in the subcutaneous implantation mice model in six treatment groups for 12 days (n=5). tumor weight (C) of Hepa1-6 cells derived xenografts in the subcutaneous implantation mice model in six treatment groups for 12 days (n=5). Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. D Knockdown of METTL3 slightly increased liver tumor growth treated with sorafenib in orthotopic xenograft mouse model. Stable METTL3-knockdown Bel-7402 cells and control cells were injected into the liver of each NOD/SCID mouse. 3 weeks after injection, livers were separated for pathological analysis. Red circle indicates tumors in the livers. E Liver tumors in orthotopic xenograft mouse model were confirmed by hematoxylin and eosin staining. F Overexpression of FOXO3 rescued sorafenib-resistance mediated by METTL3-knockdown in xenograft liver tumors. Representative xenograft tumors at endpoint, containing 6 groups. Growth curve (G) of Hepa1-6 cells derived xenografts in the subcutaneous implantation mice model in 6 treatment groups for 10 days (n=5). tumor weight (H) of Hepa1-6 cells derived xenografts in the subcutaneous implantation mice model in 6 treatment groups for 10 days (n=5). Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. I H&E and immunohistochemical images of Ki67, LC3-B, VEGF-A in Hepa1-6 cells derived xenografts in 6 treatment groups. J-L Quantification of Ki67 (K), LC3-B (K), and VEGF-A (L) expression in immunohistochemical images by Image Pro Plus (IPP) analysis. Data information: In all relevant panels, *p<0.05; **p<0.01; ***p<0.001; ****p<0.0001; Two-tailed t-test. Data are presented as mean ± SD. \n",
      ".Figure 5A, Co-IP analysis was performed in 293T cells with antibodies against p53 or H2Bub1. B, peptide sequence synthesized according a previous report and used for peptide pull-down assay. Peptide pull-down assay was performed with synthesized peptides D, Co-IP analysis for H1299 cells transfected with a Flag-tagged p53 or an empty plasmid (Cont.) treated with a USP7-specific inhibitor P5091. E, A proposed working model of the regulation of H2Bub1 by p53. \n",
      ": GBP1, its GTPase activity and prenylation are required for IFNγ-enhanced cell death upon Toxoplasma infection (A) IFNγ enhances macrophage host cell death after type I (RH) and type II (Pru) Toxoplasma gondii (Tg) infection.Figure 1IFNγ enhances macrophage host cell death after type I (RH) and type II (Pru) Toxoplasma gondii (Tg) infection. Graphs show cell death as measured by LDH release assay (left) and viability measured by XTT assay (right) in untreated or IFNγ-primed THP-1 cells infected with type I or type II Tg for 24 h. LDH release assays from THP-1 cells left untreated or primed with IFNγ, transfected with siRNA against indicated GBPs or non-targeting control (CTRL) and infected with indicated strains of Tg for 24 h. LDH release assay from primary monocyte-derived macrophages (MDM) left untreated or treated with IFNγ, transfected with siRNA against GBP1 or non-targeting control (CTRL) and infected with indicated strain of Tg for 24 hours. Mean ± SEM of n = 4 donors shown. LDH release assay from indicated wildtype, GBP1 knockout (∆GBP1) or GBP1 reconstituted (∆GBP1+Tet-GBP1) cells infected with type I or type II Tg for 24 hours. Cells were untreated or treated with IFNγ or additionally treated with doxycycline (Dox) as indicated. Realtime propidium iodide (PI) uptake assay from indicated THP-1 cells infected with type I or type II Tg, and untreated or treated with IFNγ and/or Dox as labelled. AU, arbitrary units. LDH release assay from wildtype THP-1 or ∆GBP1 cells stably reconstituted with Dox-inducible expression plasmids of the indicated mutants of GBP1. Cells were pre-treated with IFNγ and Dox and infected with either type I or type II Tg for 24 h. \n",
      ".Figure 7A, H1299 cells transfected with a Flag-tagged p53 or an empty plasmid together with a USP7-specific siRNA or a control siRNA (siCont.) as were treated with 12 μM erastin (+) or untreated (-) for 24 h as indicated. Representative phase-contrast images were recorded (magnification, ×20), and the surviving cells were counted. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05. B, H1299 cells treated were subjected to transmission electron microscopy. Representative images are shown. Arrow head indicates condensed and marginated chromatin. C, Lipid ROS levels were examined in cells treated Data information: Bars and error bars are mean±s.d., n=3 independent repeats. Two-tailed unpaired Student's t-test was performed. *: p<0.05. D, Working model. Normal cells or cells without p53 show relatively low levels of nuclear USP7. H2Bub1 is maintained at relatively high levels and is required for normal SLC7A11 expression. However, when cells are subjected to ferroptotic stress (e.g., erastin treatment), increasing amounts of p53 interact with USP7 and promote the nuclear translocation of USP7. The translocated USP7 further removes H2Bub1 from the SLC7A11 gene regulatory region and inactivates SCL7A11 expression. Eventually, ferroptosis occurs. \n",
      ".Figure 3Western blot analysis was performed with H1299 or Hep3B cells transfected with a Flag-tagged p53 or an empty plasmid (Cont.) for 48 h. Western blot analysis was performed with H1299 cells transfected with a Flag-tagged p53 or an empty plasmid (Cont.) for 48 h. C, Immunofluorescence (IF) assay for H1299 cells transfected with a Flag-tagged p53 (Flag-p53) or an empty plasmid (Cont.) for 48 h. Representative images are shown (magnification, ×40). Arrowhead indicates a Flag-p53 overexpressing cell. D, Western blot analysis was performed with H1299 transfected with a Flag-tagged p53 or an empty vector (Cont.) for 48 h. E, 293T cells transfected with a control siRNA (siCont.) and three p53-specific siRNAs (sip53) were subjected to Western blot analysis. F, SMMC cells were treated with 100 μM etoposide (+) or untreated (-) for 24 h, and Western blot analysis was performed. , Western blot analysis of A549 cells and H1299 cells. H, H1299 cells transfected with a Flag-tagged p53 or an empty plasmid (Cont.) were analyzed by micrococcal nuclease (MNase) digestion assay. I, Western blot analysis of H1299 cells transfected with a wild-type p53 (p53-WT) or two \"gain-of-function\" mutants (p53-R273H and p53-R175H) for 48 h. \n",
      ".Figure 4A, Coimmunoprecipitation (Co-IP) analysis was performed in A549 cells with antibodies against p53 or USP7. B, H1299 cells were transfected with an empty vector (Cont.), a Flag-tagged p53, a USP7-specific shRNA, or the last two plasmids together for 48 h. Western blot analysis was then performed. C, Nuclear and cytosolic extracts were differentially prepared from H1299 cells transfected with a Flag-tagged p53 or an empty plasmid (Cont.). Western blot analysis was then performed. D, H1299 cells transfected with a Myc-tagged USP7 or a Myc-tagged USP7 together with a Flag-tagged p53 were subjected to IF analysis (upper). A549 cells transfected with a p53-specific siRNA or a control siRNA (siCont.) were examined by IF analysis, and the RNAi efficiency is shown (lower). Representative images are shown (magnification, ×40). E, Diagrams showing the interaction status of USP7 and its truncated mutants (USP7-△N, lacking the N-terminal region; USP7-△M, lacking the middle region; USP7-△C, lacking the C-terminal region) with p53, based on previous reports F, IF analysis of A549 cells transfected with a Myc-tagged wild-type USP7 (Myc-USP7-WT) or three Myc-tagged truncated mutants. Representative images are shown (magnification, ×40). G, H1299 cells were transfected with a Flag-tagged p53 or an empty vector together with a USP7-specific siRNA or a control siRNA as indicated for 48 h. Cells were then harvested and subjected to a MNase digestion assay. \n",
      ".Figure 6A, Western blot analysis of 293T cells treated with 20 μM erastin for the indicated times. B, H1299 cells were transfected with Flag-tagged p53 for 24 h and then treated with 12 μM erastin or untreated for another 24 h. Co-IP analysis was performed with antibodies against Flag. C, H1299 cells transfected with Flag-tagged p53 were treated with erastin or untreated for 24 h. Nuclear and cytosolic extracts were differentially prepared and subjected to Western blot analysis. D, 293T cells transfected with a p53-specific siRNA, a USP7-specific siRNA, or a control siRNA (siCont.) were treated with 20 μM erastin or untreated for 24 h as indicated. Western blot analysis was then performed. , qRT-PCR (above) and Western blot (below) analyses for H1299 cells transfected with a Flag-tagged p53 or an empty plasmid together with a USP7-specific siRNA or a control siRNA (siCont.) as indicated. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. F, ChIP analysis of H1299 cells transfected with a Flag-tagged p53 or an empty plasmid together with a USP7-specific siRNA or a control siRNA (siCont.) as indicated. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. G, H1299 cells transfected with Flag-tagged p53 were treated with 12 μM erastin or untreated for 24 h. ChIP analysis with antibodies against Flag or USP7 was performed. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. H, 293T cells transfected with a p53-specific siRNA, a USP7-specific siRNA or a control were treated with 20 μM erastin or untreated for 24 h. ChIP analysis was performed with anti-H2Bub1 antibodies. Data information: Bars and error bars are mean±s.d., n=3 independent repeats. I, A proposed working model for the p53-USP7-H2Bub1 axis in the regulation of SLC7A11 expression. \n",
      ": Toxoplasma-infected human macrophages die through AIM2-dependent atypical apoptosis (A) AnnV-Glo assay from IFNγ-primed THP-1 transfected with siRNA against indicated human NLR/ALR sensors and infected with type I or type II Toxoplasma gondii (Tg).Figure 3AnnV-Glo assay from IFNγ-primed THP-1 transfected with siRNA against indicated human NLR/ALR sensors and infected with type I or type II Toxoplasma gondii (Tg). Area under the curve (AUC) from real-time assays are plotted as mean ± SEM from n = 3 independent experiments. AnnV-Glo assay from IFNγ-primed primary human MDMs transfected with siRNA against AIM2 or non-targeting control (CTRL) and infected with type I or type II Tg. Area under the curve (AUC) from real-time assays are plotted from n = 4 independent experiments. Matched shapes and color of symbols indicate donors. Representative images (top) from IFNγ-primed THP-1 WT infected with type I Tg for 8 hours and stained for ASC and active caspase-8. Blue, Nuclei; Red, ASC; Green, active caspase-8 (IETD-FITC) and Grey, Tg. Scale bar, 5 μm or 2 μm in the magnified images. Quantification of ASC specks (bottom) in Tg-infected THP-1 WT or ∆CASP1 treated with IFNγ or left untreated at 8 hours post infection, plotted as mean ± SEM from n = 3 independent experiment. Immunoblots from primary MDMs infected with Tg strains for indicated times. Cells were untreated or pre-treated with IFNγ before infection. Images are representative of n = 4 biologically independent experiments. \n",
      ".Figure 2Propidium iodide (PI) uptake in indicated wildtype THP-1 or ∆CASP-1, ∆CASP-4, ∆ASC cells, or THP-1 stably expressing GSDMD-targeting miRNA (GSDMDmiR) infected with the indicated strains of Toxoplasma gondii (Tg) for 24 h. Cells were untreated or treated with IFNγ as indicated. Mean area under the curve (AUC) ± SEM from n = 3 experiments are shown. ns, not significant, from two-way ANOVA following adjustment for multiple comparisons. Real-time apoptosis assay (Annexin V (AnnV)-Glo luminescence) from IFNγ-primed primary human MDMs transfected with non-targeting control (siCTRL; left) or GBP1 siRNA (siGBP1; right) and left uninfected or infected with type I or type II Tg. Depicted is the mean ± SEM of n = 4 donors. AU, arbitrary units. Immunoblots from primary MDMs infected with the indicated Tg strains for 12 h. Cells were untreated or pre-treated with IFNγ before infection. Treatment with TNFα and cycloheximide (CHX) served as positive control for apoptosis. Images are representative of n = 4 biologically independent experiments. Live time-lapse imaging to visualize apoptotic cell death and measure caspase-3/7 activities using a fluorescent caspase substrate. THP-1 WT were primed with IFNγ or left untreated and infected with either type I or type II Tg. Plot in (D) shows cumulative caspase-3/7 activity from an 18-hour time-course (mean ± SEM from n = 3 independent experiments). Representative phase-contrast images overlaid with fluorescence signal for caspase-3/7 activity (green) images are shown in (E) from IFNγ-primed THP-1 cells infected with type I Tg captured at indicated times post-infection. Scale bar 20 μm. AnnV-Glo assay from IFNγ-primed THP-1 transfected with siRNA against indicated human caspases and infected with type I or type II Tg. Area under the curve (AUC) from a real-time AnnV-Glo luminescence assays similar to those in (B) are plotted as mean ± SEM from n = 3 independent experiments. AnnV-Glo assay from IFNγ-primed primary human MDMs transfected with siRNA against CASP8, CASP10 and non-targeting control (CTRL) and infected with type I or type II Tg. Area under the curve (AUC) from real-time assays similar to those in (B) are plotted from n = 4 independent experiments. Matched shapes and color of symbols indicate donors. \n",
      ".Figure 3Immunoblot of conditioned media collected from HEK293T cells transiently transfected with ERdj3WT or ERdj3H53Q and incubated with an affinity resin consisting of native (N) or guanidinium hydrochloride (GdnHCl)-denatured (D) RNAse A covalently coupled to sepharose resin. Resin was incubated with media for 2 h at ambient temperature, followed by washing and elution in Laemmli's reducing buffer.  Plot showing the time-dependent increase in ThioT fluorescence (ex:440 nm, em:485 nm) of A1-40 incubated (10 µM, 37 OC, pH 7.2) with regular agitation in the presence or absence (black) of recombinant ERdj3 (RERdj3). The RERdj3 concentrations and molar ratios of RERdj3 to A1-40 are as follows: Violet: 1.5 nM, 1:6561; light blue: 4.6 nM, 1:2187; dark blue: 14 nM, 1:729; dark green: 41 nM, 1:243; light green: 120 nM, 1:81; yellow: 370 nM, 1:27; orange: 1.1 M, 1:9; red: 3.3 M, 1:3, or maroon: 10 M, 1:1. All traces represent the average of three replicates.  Plot showing the time-dependent increase in ThioT fluorescence of A1-40 incubated (10 µM, 37 OC, pH 7.2) with regular agitation in the presence or absence of BSA (14 µg/mL) or RERdj3 (14 µg/mL; 370 nM, RERdj3: A=1:27). All traces represent the average of three replicates. Immunoblot of A1-40 incubated (10 µM, 37 OC, pH 7.2) with regular agitation for 90 h in the presence or absence of RERdj3 (14 µg/mL; 370 nM, RERdj3: A1-40 =1:27) or BSA (14 µg/mL). The reaction solution in the well was collected (Nonadhered) and separated into soluble and insoluble fractions by centrifugation. These samples, as well as the washed wells of the plate (Adhered), were denatured in 8 M GdnHCl with sonication and analyzed by SDS-PAGE/immunoblotting. \n",
      ".Figure 4 Schematic illustrating the treatment of PK1 cells with ERdj3-conditioned media. HEK293T-Rex (Control) and HEK293T-Rex cells stably overexpressing tet-inducible ERdj3WT on poly-D-lysine coated plates were treated with dox (1 g/mL) overnight to induce ERdj3WT expression. The media was replaced with fresh OptiMEM I + 5% FBS, which was conditioned for 30 h, sterile-filtered, concentrated 5 X against a 3 kD MWCO membrane, and diluted 1:1.5 with fresh OptiMEM I + 5% FBS. PK1 cells were plated in OptiMEM I + 5% FBS, and after 24 h were treated with conditioned media and TPrP (600 ng/mL) for 3 d.   Representative images under 136X magnification of PK1 cells incubated with control conditioned media collected from HEK293T-Rex or conditioned media prepared from HEK293T-Rex cells overexpressing ERdj3WT in the presence or absence of TPrP (600 ng/mL), as described in Figure 4A. Arrows indicate representative cells containing vacuoles. Insets show 544X magnification of a typical region of the image. Quantification of vacuole formation from images as shown in Figure 4B. Equal area images comprising ~1200 cells per well were counted for each sample. Counting of cells and of cells with identifiable vacuoles was done using the Fiji image processing package (Schindelin et al, 2012). *** indicates a p-value < 0.001 (n = 3).  \n",
      ": GBP1 targets Toxoplasma vacuoles in human macrophages (A-B) Representative images (A) and quantification (B) from IFNγ-primed THP-1 ∆GBP1 stably reconstituted with Tet-mCH-GBP1, mCH-GBP1K51A, mCH-GBP1C589A or mCH-GBP1∆589-592 variants or monocyte derived macrophages (MDM) stained for GBP1.Figure 4Representative images (A) and quantification (B) from IFNγ-primed THP-1 ∆GBP1 stably reconstituted with Tet-mCH-GBP1, mCH-GBP1K51A, mCH-GBP1C589A or mCH-GBP1∆589-592 variants or monocyte derived macrophages (MDM) stained for GBP1. Cells were infected with type I or type II Toxoplasma gondii (Tg) for 2 h. Blue, Nuclei; Grey, Tg; Red, mCH-GBP1 or Magenta, GBP1. Scale bar, 5 μm. Graph in (B) shows mean percentage of Tg vacuoles targeted by GBP1 from n = 4 experiments. \n",
      ".Figure 2Immunoblot in lysates and media collected from HEK293DAX cells following 16 h of activation of XBP1s [by 1 g/mL dox] and/or ATF6 [by 10 M TMP], as indicated (Shoulders et al, 2013). Immunoblots of the ATF6 target protein BiP (reactive with the KDEL antibody) and the XBP1s target Sec24D confirm the small molecule activation of these transcription factors.   Representative autoradiogram of [35S]-labeled ERdj3 immunopurified from lysates and media collected from HEK293DAX cells following preactivation of XBP1s [by 1 g/mL dox] and/or ATF6 [by 10 M TMP] (Shoulders et al, 2013). The experimental protocol is shown above. Quantification of relative amounts of [35S]-labeled ERdj3 in media at 8 h collected from cells treated as shown in Figure 2B (n=3). The media [35S]-labeled ERdj3 was measured by densitometry and is normalized to the amount of media [35S]-labeled ERdj3 at 8 h from vehicle-treated cells. Error bars represent standard error from biological replicates (n = 3). * indicates p-value < 0.05; ** indicates p-value < 0.01; *** indicates p-value < 0.001 as compared to the vehicle-treated condition. Quantification of the fraction ERdj3 secreted from autoradiograms as shown in Figure 2B. The fraction of ERdj3 secreted was calculated by normalizing the recovered [35S]-labeled ERdj3 signal in the media at t = 4 or 8 h to the total amount of [35S]-labeled ERdj3 signal in the media and lysates at t = 0 h. Error bars represent standard error from biological replicates (n = 3). * indicates p-value < 0.05; ** indicates p-value < 0.01 as compared to the vehicle-treated condition. Quantification of the fraction ERdj3 remaining from autoradiograms as shown in Figure 2B. The fraction of ERdj3 remaining was calculated by normalizing the sum of the recovered [35S]-labeled ERdj3 signal in the media and in the lysates at t = 4 or 8 h to the total amount of [35S]-labeled ERdj3 signal in the media and lysates at t = 0 h. Error bars represent standard error from biological replicates (n = 3). * indicates p-value < 0.05; ** indicates p-value < 0.01 as compared to the vehicle-treated condition.  \n",
      ".Figure 1 Immunoblot of media and lysate collected from HEK293T cells overexpressing ERdj3WT or ERdj3KDEL. Fresh media was conditioned on cells for 24 h prior to harvest.  qPCR analysis of ERdj3 mRNA in HEK293T cells treated with or without thapsigargin (Tg; 6 h, 100 nM). Error bars represent the mean ± 95% confidence interval as calculated in DataAssist 2.0 (n = 3).   Representative immunoblot of lysates and media collected from HEK293T-Rex cells stably expressing non-silencing (NS) or ERdj3 shRNA. Cells were treated with the indicated concentration of Tg in fresh media for 16 h. Quantification of the fold increase of ERdj3 in lysates and media collected from HEK293T-Rex cells treated with the indicated concentration of Tg in fresh media for 16 h as in Figure 1C. *indicates a pvalue < 0.05; (n = 3).    Representative autoradiogram of [35S]-labeled ERdj3 in lysates and media collected from Huh7 cells incubated in the presence or absence of Tg (1 µM). The experimental paradigm for labeling is shown above. Quantification of fraction intracellular (dark grey) and extracellular (light grey) [35S]-labeled ERdj3 following a 4 h chase relative to total [35S]-labeled ERdj3 at 4 h (as shown in Figure 1E). Error bars represent standard error from biological replicates (n = 3). Quantification of total [35S]-labeled ERdj3 in media collected from cells pretreated with vehicle or Tg, as shown in Figure 1E. The media [35S]-labeled ERdj3 was measured by densitometry and is normalized to the amount of media [35S]-labeled ERdj3 at 4 h from vehicle-treated cells. Error bars represent standard error from biological replicates. *indicates a pvalue < 0.05; (n = 3).  Representative immunoblot and quantification of sera collected at the indicated times from DBA/2J mice fed a high fat (60% fat) diet after a 18 h fast. A schematic of the fasting/refeeding experimental setup is shown above. The quantification was normalized for each mouse to the amount of ERdj3 in the serum draw immediately prior the fast. *indicates a p-value < 0.05; (n = 6). \n",
      ": Transfection of pathogen-derived ligands bypasses GBP1 in inducing macrophage death (A) Cell death measured using Annexin V (AnnV)-Glo assay of THP-1 ∆CASP1 cells transfected with non-targeting control (CTRL) or GBP1 siRNA and treated as indicated.Figure 6Cell death measured using Annexin V (AnnV)-Glo assay of THP-1 ∆CASP1 cells transfected with non-targeting control (CTRL) or GBP1 siRNA and treated as indicated. Cells were primed with IFNγ, transfected with soluble Tg antigen (STAg) or both, or left untreated (UT). Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 3 independent experiments. AnnV-Glo assay from naïve or IFNγ-primed THP-1 ∆CASP-1 cells transfected with soluble Tg antigen (STAg) that was untreated (UT) or treated as indicated. Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 3 independent experiments. Immunoblots from IFNγ-primed THP-1∆CASP-1 or THP-1 ∆ASC cells transfected with non-targeting control (CTRL), GBP1, AIM2, CASP-1 or CASP-8 siRNA and then untreated (-) or transfected with STAg for 12 h. Images are representative of n = 2 independent experiments. AnnV-Glo assay from naïve or IFNγ-primed THP-1 ∆CASP-1 cells transfected with non-targeting control (CTRL), GBP1, AIM2, CASP-1 or CASP-8 siRNA and then untreated (UT) or transfected with STAg as indicated for 18 h. Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 3 independent experiments. Schematic on right shows an overview of pathways leading to caspase-8 activation. Propidium iodide (PI) uptake assay from MDMs transfected with non-targeting control (CTRL) or GBP1 siRNA and then left untreated (UT), treated with LPS or transfection reagent (Lipofectamine) only or transfected with LPS for 4 h. Area under the curve (AUC) from real-time assays plotted as mean ± SEM from n = 4 independent experiments. Schematic on right shows an overview of pathways leading to caspase-4 activation. \n",
      ": GBP1 targets Salmonella vacuoles and mediates caspase-4-dependent pyroptosis (A) LDH release assay from indicated THP-1 cells untreated or treated with IFNγ and infected with Salmonella Typhimurium (STm) WT or a SPI-1 mutant ΔinvA for 4 h at indicated multiplicities of infection (MOI).Figure 5LDH release assay from indicated THP-1 cells untreated or treated with IFNγ and infected with Salmonella Typhimurium (STm) WT or a SPI-1 mutant ΔinvA for 4 h at indicated multiplicities of infection (MOI). Mean ± SEM of n = 3 experiments are shown. Propidium iodide (PI) uptake assay from IFNγ-primed primary MDMs transfected with non-targeting control siRNA (CTRL) or siRNA against GBP1 (left) or indicated lines of THP-1 (right) infected with STm-GFP for 4 h. Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 3 independent experiments. PI uptake assay from IFNγ-primed primary MDMs transfected with indicated siRNA or non-targeting control (CTRL) and infected with STm-GFP for 4 h. Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 4 independent experiments. PI uptake assay from IFNγ-primed or untreated THP-1 ΔCASP-1 cells transfected with indicated siRNA and infected with STm-GFP for 4 h. Area under the curve (AUC) from real-time assay plotted as mean ± SEM from n = 3 independent experiments. Schematic on right shows overview of the pathway leading to caspase-1 and caspase-4 activation. LDH release assay from indicated THP-1 cells treated with IFNγ and Dox and infected with STm-GFP for 4 h. Shown is the mean ± SEM of n = 3 experiments. Representative images from IFNγ-primed THP-1 ∆GBP1 cells stably reconstituted with Tet-mCH-GBP1, mCH-GBP1K51A, mCH-GBP1C589A or mCH-GBP1∆589-592 variants or primary MDMs stained for GBP1. Cells were infected with STm-GFP for 2 h. Blue, Nuclei; Grey, STm; Red, mCH-GBP1 or Magenta GBP1. Scale bar, 5 μm. quantification (G) from IFNγ-primed THP-1 ∆GBP1 cells stably reconstituted with Tet-mCH-GBP1, mCH-GBP1K51A, mCH-GBP1C589A or mCH-GBP1∆589-592 variants or primary MDMs stained for GBP1. Cells were infected with STm-GFP for 2 h. Blue, Nuclei; Grey, STm; Red, mCH-GBP1 or Magenta GBP1. Scale bar, 5 μm. Graph in (G) shows percentage of STm vacuoles targeted by GBP1. Quantification of GBP1 recruitment to STm-GFP containing vacuoles (SCV) in MDMs pre-treated with IFNγ or left untreated (UT). Mean ± 95% CI of GBP1 fluorescence intensity measured on the indicated number (n) of SCVs for each condition from three independent experiments are shown. Representative images from THP-1 ∆GBP1+Tet-mCH-GBP1 cells stably expressing YFP-CASP4C258S. Cells were infected with STm for 2 h. Blue, Nuclei & STm (Hoechst dye); Red, mCH-GBP1; Green, YFP-CASP4C258S. Scale bar, 5 μm. Quantification of THP-1 ∆GBP1+Tet-mCH-GBP1 cells stably expressing YFP-CASP4C258S and infected with STm for 2 h. Graph shows percentage of STm vacuoles targeted by GBP1, CASP4 or both from n = 3 experiments. n.d. not detected. \n",
      ".Figure 7 In response to ER stress, newly-synthesized ERdj3 binds misfolded ER client proteins and delivers them to BiP for chaperoning in the Hsp70 cycle. When free BiP becomes limiting, or if repeated BiP cycling cannot productively deplete the levels of the misfolded client, the stable ERdj3-substrate complex is co-secreted to the extracellular environment, preemptively binding the misfolded protein and preventing the aggregation of the misfolded client protein in the extracellular space. Furthermore, stress-induced ERdj3 can be secreted on its own into the extracellular space where it can bind to misfolded, aggregation-prone client proteins and attenuate pathologic protein aggregation in the extracellular environment. \n",
      "a, BMDMs were infected with mCherry-expressing M.f2a, BMDMs were infected with mCherry-expressing M. tuberculosis for 4 h and immunostained using linkage-specific ubiquitin antibodies. b, Quantification of K63 and K48 ubiquitin-positive M. tuberculosis from a. c, BMDMs were infected with mCherry-expressing M. tuberculosis for 4 h and immunostained for NDP52, p62, phospho-TBK1 or NBR1. d, Quantification of co-localization from c (n = 3 per group, **P  0.001). All errors, s.e.m. \n",
      ".Figure 5 Schematic describing the co-incubation and co-expression experiments used to demonstrate ERdj3 co-secretion with destabilized TTR. For co-incubation, HEK293T cells overexpressing flag-tagged TTR variants were seeded with an equal number of cells overexpressing ERdj3. For co-expression experiments, HEK293T cells are transfected with both flag-tagged TTR and ERdj3. TTR was immunoisolated with M2 anti-FLAG beads from media conditioned overnight on the cells, eluted and separated by SDS-PAGE for immunoblotting. Immunoblot of M2 anti-FLAG immunopurifications from the conditioned media collected from HEK293T cells overexpressing FTTTRWT, FTTTRA25T, and/or ERdj3WT either as a co-incubation or co-expression experiment as shown in Figure 5A. Beads were washed in RIPA buffer prior to elution. Media inputs (1:400) are shown as a control. Immunoblot of M2 anti-FLAG immunopurifications from media collected from HEK293T cells overexpressing ERdj3WT and FTTTR variants as indicated. Media inputs (1:400) are shown as a control. Immunoblot of M2 anti-FLAG immunopurifications from conditioned media of HEK293T cells overexpressing FTTTRA25T and ERdj3WT in the presence of the indicated concentrations of Tafamidis, a kinetic stabilizer of TTR tetramers. Media inputs (1:400) are shown as a control. Immunoblot of 6E10 anti-APP immunopurifications from the conditioned media of CHO cells or CHO-derived APP751-expressing (7WD10) cells overexpressing ERdj3WT as indicated. Cells were replated together as indicated 24 h after transfection at equal stoichiometry, and media conditioned for 36 h. In this case, ERdj3 transfected CHO cells replated with mock-transfected 7WD10 cells serves as the co-incubation condition, while mock-transfected CHO cells replated with ERdj3-transfected 7WD10 cells comprise the co-expression condition. Media inputs (1:150) are shown as a control. \n",
      ".Figure 6  Representative immunoblot of M2 anti-FLAG immunopurification of media conditioned for 24 h on HEK293DAX cells co-overexpressing FTTTRA25T and ERdj3WT or ERdj3H53Q. Media contained vehicle or trimethoprim (TMP, 10 M, 16 h; activates ATF6), as indicated. Media inputs (1:400) are shown as a control. Lysates from these cells are also shown to confirm the increase of the ATF6 target proteins BiP and GRP94 in cells following TMP-dependent activation of ATF6. Bar graph depicting quantification of the change in the amount of ERdj3 co-immunoprecipitating with TTRA25T following activation of ATF6, as in Figure 6A. The ERdj3/TTR ratio is normalized to the ratio in the absence of ATF6 activation. *indicates a p-value < 0.05; n = 3.   Representative immunoblot of M2 anti-FLAG immunopurification of media conditioned for 24 h on HEK293T cells co-overexpressing FTTTRA25T, ERdj3WT, ERdj3H53Q, and/or BiP, as indicated. Media inputs (1:400) are shown as a control. Lysates from these cells are also shown to confirm the increase of BiP afforded by overexpression. Quantification of immunoblots of M2 anti-FLAG immunopurification of media collected from cells co-overexpressing either ERdj3WT or ERdj3H53Q, FTTTRA25T, and BiP, as in Figure 6C. The ERdj3/TTR ratio is normalized to the ratio in the absence of BiP overexpression. **indicates a p-value < 0.01; n = 5. Immunoblot of 6E10 anti-APP immunopurifications from the conditioned media of CHO cells or CHO-derived APP751-expressing (7WD10) cells overexpressing ERdj3WT, ERdj3H53Q, and BiP as indicated. Media inputs (1:100) are shown as a control. Lysates from these cells are also shown to confirm the increase of BiP afforded by overexpression. \n",
      "a, BMDMs were infected with mCherry-expressing M.f3a, BMDMs were infected with mCherry-expressing M. tuberculosis (arrows) for 4 h and immunostained for LC3 (LC3B antibody) or ATG12. b, Quantification of results from a (n = 3 per group, **P  0.001). c, Western blot analysis of LC3 (LC3B antibody) from cell lysates from a. d, BMDMs were infected with mCherry-expressing M. tuberculosis (arrows) for 6 h and immunostained for LAMP1. d, BMDMs were infected with mCherry-expressing M. tuberculosis (arrows) for 6 h and immunostained for LAMP1. e, Quantification of results from d expressed relative to control BMDMs (n = 3 per group, **P  0.001). f, BMDMs were infected with M. tuberculosis and colony-forming units (c.f.u.) at t = 0 and t = 16 h were determined by plating. Results were normalized to t = 0 (n = 3 per group, *P < 0.02). g, RAW264.7 macrophages transduced with lentivirus expressing green fluorescent protein (GFP) or parkin were infected with M. tuberculosis and c.f.u. were determined after 16 h. Results were normalized to GFP-expressing cells (n = 3 per group, **P < 0.0076). h, U937 human macrophages expressing either scrambled or PARK2 shRNAs were infected with M. tuberculosis for 36 h and c.f.u were determined. Results were normalized to t = 0 (N = 3 per group, *P< 0.02). All errors, s.e.m. \n",
      "a, WT and Park2−/− mice were infected with wild-type M.f4a, WT and Park2-/- mice were infected with wild-type M. tuberculosis by aerosol and lung bacterial burdens were determined by plating (means ± s.d., n = 5 per group, *P < 0.02 by Student's t-test). b, Enumeration of liver and spleenc.f.u. from mice infected in a, 21 days post-infection (means ± s.d, n = 5 per group, *P < 0.03 by Student's t-test). c, Survival of M. tuberculosis-infected wild-type and Park2-/- mice (n = 10, **P < 0.001 by log-rank test). d, Immunohistochemistry staining of lung sections from infected mice21 days post-infection, scale bar, 50 µm. e, Immunohistochemistry staining of lung sections from a human patient with active tuberculosis. Scale bars, 500 µm (×4), 50 µm (×40). f, Confocal microscopy images of humanlung sections from e immunostained for M. tuberculosis and parkin; DNA was visualized using DAPI (4′,6-diamidino-2-phenylindole). Scale bars, 25 µm. g, Mice were infected with wild-type L. monocytogenes by i.p. injection and bacterial burdens in livers and spleens were determined by plating (means ± s.d., n = 7 per group, *P < 0.04 by Student's t-test). \n",
      "a, Wild-type (WT) and Park2−/− BMDMs were infected with mCherry-expressing M.f1a, Wild-type (WT) and Park2-/- BMDMs were infected with mCherry-expressing M. tuberculosis (Mtb) for 4 h and immunostained using anti-parkin antibodies. b, Wild-type BMDMs were infected with mCherry-expressing M. tuberculosis for 4 h and immunostained for polyubiquitin. b, Wild-type BMDMs were infected with mCherry-expressing M. tuberculosis for 4 h and immunostained for polyubiquitin. c, Quantification of ubiquitin-positive M. tuberculosis from b. Results are means ± s.e.m. of three independent experiments (**P < 0.001, paired Student's t-test). d, U937 human macrophages expressing a scrambled shRNA (Control) or one of two different shRNAs targeting PARK2 (shRNA no. 1, shRNA no. 2) were infected with mCherry-expressing M. tuberculosis for 12 h and immunostained for polyubiquitin. d, U937 human macrophages expressing a scrambled shRNA (Control) or one of two different shRNAs targeting PARK2 (shRNA no. 1, shRNA no. 2) were infected with mCherry-expressing M. tuberculosis for 12 h and immunostained for polyubiquitin. e, Quantification of ubiquitin positive M. tuberculosis from d, results are means ± s.e.m. of three independent experiments (**P  0.005, Student's t-test). f, The parkin and actin expression in cells from e was determined by western blotting. g, Park2−/− BMDMs were transduced with lentivirus expressing BFP (−), wild-type parkin, or two separate mutant parkin alleles (T240R, P437L). Cells were infected with M. tuberculosis for 4 h and ubiquitin and M. tuberculosis co-localization was quantified and expressed relative to control BMDMs. Results are means ± s.e.m. of three independent experiments (**P  0.005, paired Student's t-test). h, The parkin and actin expression in cells from g was determined by western blotting. \n",
      ".Figure 5Difference in deuterium uptake by sE404r and sE419r trimers (D404-D419). Left panel, deuteration difference for each peptide reported against the 1-p value obtained via Student's t-test between the two different constructs. In the right panel, peptides are reported as short horizontal lines, with length and position corresponding to the peptide length and location along the protein sequence (x-axis) and with their deuteration delta on the y-axis. In both panels, red (positive) and blue (negative) indicate changes above the Student's t-test threshold, and in grey those not reaching this threshold. Four peptides, termed HDX-a, HDX-b, HDX-c and HDX-d, were identified as showing a strong increase in deuteration in sE404r compared to sE419r. The same pattern was observed in the comparison between sE404r and sE428r, whereas no difference deemed \"significant\" by the same test was seen when comparing sE419r with sE428r (see Fig EV3). Middle panel: ribbons representation of the sE trimer (PDB 1URZ) colored according to domains with high HDX regions a and c boxed and highlighted in light grey. The central box marks the domain I-II hinge. Residues 401 and 403, which occupy the \n",
      ".Figure 2A. The structure of the sE-linker* trimer is displayed as ribbons colored by domains as in Figure 1. The previously determined structure of the TBEV sE401v trimer (PDB 1URZ, [15]) is superposed in light grey with fusion loops (FL) in orange, highlighting the disordered domain II tip in the new structure. The two subdomains of domain II, base and tip, are indicated. Disulfide bonds are shown as dark grey sticks. The residues of the stem are displayed as sticks color-coded according to atom type, with carbon atoms green. B. Close-up view of the sE-linker* trimer with one subunit colored as in (A) and the others in grey. The segment connecting helix αA to the η3 3/10 helical turn (see Appendix Fig S1) of the adjacent subunit is shown as sticks colored according to atom-type with carbon atoms light grey. Broken lines mark polar interactions of the stem with surrounding residues (with distances ranging between 2.7Å and 3.4Å). An open arrow marks the C-terminal end of the sE401v trimer structure reported previously (PDB 1URZ). C. For comparison, the same region of the stem is represented for the DENV-1 sE structure (PDB 4GSX, [19]), using the same color scheme. \n",
      ".Figure 1Ribbon diagrams of the (M/E)2 dimer extracted from the structure on virions (PDB:5O6A) [10], in side (left panel) and bottom (right panel) views with M in brown and E colored according to domains: domain I, red; domain II, yellow; domain III, blue; stem, green; transmembrane (TM) segments, grey; fusion loop (FL), orange. The \n",
      ".Figure 4All diagrams are aligned at the left and at the same scale, using the full-length E protein (Ev, top line) and sE401v (second line), derived from virions, as reference. Recombinant constructs are shown below the dashed line: sE400r, sE401r, sE404r (containing 6 N-terminal amino acids of H1), sE412r (containing H1), sE419r (containing H1 and H2), sE428r (containing H1, H2 and CS) and sE448r (containing the whole stem). Abbreviations EK and StT indicate heterologous sequences corresponding to an enterokinase-specific cleavage site and a strep-tag used for affinity purification, respectively, added at the C-terminal end of each construct. Trimer stability expressed as percentage of the fraction detected in the trimer peak and normalized with respect to the corresponding Ev fraction in sedimentation analyses after incubation at 70°C relative to 37°C (100%). Data are from at least three independent experiments. \n",
      ".Figure 3Left panel, (M/E)2 dimer (PDB 5O6A, [10]) colored as in Figure 1. For clarity, residues downstream E Gln404 were deleted, and only residues 1 to 20 of M are shown in tube representation. The framed region is enlarged in the middle panel, with the pre-fusion E/M heterodimer (E: grey; M: brown) and the post-fusion E trimer (color-coded as domains) superimposed on helices αA and αB, illustrating the common site occupied by M and by the E stem at different stages of the flavivirus cycle. E and M run in opposite directions within the αA/B groove, interacting with the same domain II residues. Right panel: close-ups of the interactions with residues displayed as sticks. Top, residue Ile4 of M takes the place of Phe403 of the stem. The bottom panel illustrates the alternative interactions made by the strictly conserved Trp219 side chain with the main chain of M in pre-fusion and of the stem in post-fusion forms. In both, top and bottom panels, stem and M residues are labeled in green and brown, respectively. pH-dependent dissociation of E and M. TBEV preparations treated at pH 8 (left) and pH 6 (right) were solubilized with Triton X-100 and subjected to sucrose gradient centrifugation. The fractions were analyzed by SDS-PAGE and densitometry. Representative examples of at least two independent experiments are shown. Positions of E dimers (D) and E trimers (T) in the gradients are indicated. \n",
      "a, Wild-type and two independent parkin-deficient D.f5a, Wild-type and two independent parkin-deficient D. melanogaster lines (parkc00062, parkf01950) were infected with L. monocytogenes by anterior abdomen injection. Atg8 processing was monitored in whole-fly protein lysates by western blotting. b, L. monocytogenes c.f.u. from a were determined by plating (means ± s.d., n = 3-5, **P < 0.001 by Student's t-test). c, Survival of D. melanogaster was determined (n = 33, **P < 0.001 by log-rank test). d, e, Flies were infected with S. enterica serovar Typhimurium and bacterial c.f.u. (d) (means ± s.d., n = 3-5 per group, **P  0.009 by Student's t-test) and fly survival (e) were determined (n = 59, **P  0.001 by log-rank test). f, Survival of flies infected with M. marinum (n = 37, **P  0.0045 by log-rank test). \n",
      ".Figure 6Binding sites of mabs (highlighted with ovals) were mapped by mutational analysis (A1 and A2) or by neutralization escape (A3 and B4). The residues affecting binding are represented as white patches on the surface of the sE trimer (PDB 1URZ) color-coded as domains. The a-d surface segments identified by the HDX-MS experiments are shown as grey patches. Blocking ELISA with mabs A1, A2, A3 and B4. Serial dilutions of the trimers were pre-incubated with a predetermined concentration of the respective mabs. The fraction of mab not blocked by the antigen was detected in ELISA with TBEV-coated plates (Materials and Methods). The bar charts show the results expressed relative to the reactivity with the sE401v trimer. Data are from at least three independent experiments. \n",
      ".Figure 7One protomer extracted from the post-fusion trimer of TBEV sE (PDB 1URZ [15] + stem residues 402-404 modeled from the sE-linker* structure) displayed next to a protomer extracted from the complete trimer of hairpins of the RVFV Gc (PDB 6EGU [36]) class II fusion protein. Both proteins are color-coded by domains as in Figure 1 but with the two segments that make up domain II distinguished in yellow and brown. An arrow at the top marks a disulfide bond conserved across all class II viral fusion proteins (shown in black). In RVFV Gc, the stem reaches the FL and its C-terminal end is stabilized by interactions with a GPL head group (shown as ball-and-stick) bound in a cavity underneath the FL, with its glycerol moiety interacting with the conserved disulfide bond. The most C-terminal residue visualized in the TBEV sE-linker* structure, Gln404, is about 15 amino acids short of reaching the FL, which would correspond roughly to position 419 at the beginning of the CS. The location of CS within the stem is indicated in the panel below. \n",
      ".Figure 8Postulated organization of the amphipathic H3 helix in the post-fusion trimer. Left panel: proposed arrangement of the H3 helix in our post-fusion trimer model, lying flat on the fused membrane (in light orange) and projecting the TM segments (not drawn) away from the trimer axis. Second panel: potential conformation of the sE448r trimer, with the H3 helices trimerizing via their hydrophobic surfaces. Third panel: the sE-linker construct. As the linker is not long enough (only 8 residues), the H3 helix cannot trimerize as in the full-stem sE448r trimer. It would be exposed on the sides of the trimer, potentially leading to aggregation. Right-hand panel: the sE-linker* did not lead to aggregation, and at high concentrations it crystallized in a form that had the tip disordered. Trimerization of the three H3 helices in the trimer may have caused this effect, as they must displace the trimer tips in order to oligomerize. This is a further measure of the highly dynamic breathing behavior of the E trimers, as the sequence required to complete the stem zipper was absent from this construct. Schematic of the viral surface in the mature particle, with E and M in side view, color coded as in Fig 1. The breathing behavior within and between E dimers, transiently exposing the fusion loop and regions normally buried at the dimer-dimer interface, is indicated by curved lines. A vertical arrow points to the hinge between domains I and II, a region that supports E protein breathing. ii). Dissociation of E dimers in the acidic milieu of an endosome, exposing the FLs and allowing their insertion into the endosomal membrane. iii). Upon fusion loop insertion, the E proteins form \"extended intermediate\" trimers bridging the two membranes. Curved blue arrows indicate the subsequent re-location of domain III, in blue, to the sides of the trimer to initiate the \"hairpin\" conformation of the individual subunits of the post-fusion trimer. iv). Upon relocation of domain III, the upstream part of the stem becomes anchored to the the \n",
      ".Figure 1A CUEDC2protein levels in mouse tissues. Adult (12-week-old) mouse was sacrifices and total protein was extracted from the liver, kidney, lung, spleen, heart, stomach, intestine, prostate, testis, brain, and thymus. B (top) In vivo cardiac ischemia-reperfusion model protocol (30-minute ischemia followed by indicated times of reperfusion). (bottom) Mice were subjected to reversible ischemia in vivo for 30 minutes and reperfused for different time period. Protein was extracted from the area at risk of heart at the indicated time points and subjected to immunoblot analysis of CUEDC2 and CUEDC1. Representative results were shown from 2 mice at each time point, and experiments were repeated for three times. C Isolated primary ratneonatal cardiomyocytes were treated to hypoxia with serum free-medium for 6h and reoxygenated accompanying adding serum back for different time period in vitro and the protein level of CUEDC2 was tested at different time points during reoxygenation. Each experiment was repeated for three times. D CUEDC2 protein level was plotted using the immunohistochemical scores as described in methods. (left) Plot of CUEDC2 scores in each BZ and DZ (n = 9 patients). (right) Representative images from immunohistochemistry staining for CUEDC2 in tissues from patients who suffered acute myocardial infarctions. The boxed areas in the left images are magnified in the middle and right images. DZ, distant zone; BZ, border zone. Scale bars, 250 μm (left), 50 μm (middle and right). * p = 0.001. \n",
      ".Figure 2A Primary mousecardiomyocytes were treated with H2O2 (1 mM) for 3 hours and cardiomyocyteviability was determined by flow cytometry. Representative results of flow cytometry analysis. B Quantitative analysis of PI-positive cells treated with H2O2 was shown (42.9 ± 3.1% in wild-type group vs. 10.1 ± 4.4% in Cuedc2-/- group, * p = 0.0004, n = 5 wells per group and repeated for three times). C Mouse neonatal cardiomyocytes were subjected to hypoxia for 6 hours and then reoxygenation for 30 minutes. Cardiomyocytes were then stained with 5 μM CellROX® Deep Red Reagent and analyzed by flow cytometry. Representative results of flow cytometry analysis are shown, and all experiments were repeated for three times.D ROS levels were quantitated and summarized. * p = 0.0108, n = 3 wells per group and repeated for three times. E HPLC detection of a superoxide probe oxidized dihydroethidium (DHE) product in sham and I/R injury heart tissue (30-min ischemia followed by 30-min reperfusion), 2-hydroxyethidium (EOH), a specific product for superoxide anion radical. *p < 0.0001, n = 5 in each group.F In the same samples of (E), HPLC detection of ethidium (E), oxidized by other reactive oxygen species such as H2O2 (mainly) and ONOO. *p = 0.0001, n = 5 in each group. \n",
      ".Figure 3A Protein was extracted from the area at risk of left ventricle from wild-type or Cuedc2-/- mice with 30-minute ischemia followed by 30-minute reperfusion and subjected to immunoblot analysis. Representative results from 2 mice are shown, and each experiment was repeated for three times. B Neonatal mousecardiomyocytes were subjected to hypoxia for 6 hours, followed by the indicated times of reoxygenation, before protein was extracted and subjected to immunoblot analysis. C Luciferase assay of WT or Cuedc2-/- MEFs transiently transfected with an NF-κB-responsive luciferase reporter and then, 1 d later, treated for 6 h with TNF (10 ng/ml; TNF) or for H/R. * p = 0.0032, n = 3 per group, ** p = 0.013, n = 3 per group. D Wild-type and Cuedc2-/- mice were treated with a sham or I/R operation in vivo. Three hours after the onset of reperfusion, the mRNA levels of tumor necrosis factor α (TNFα), interleukin 6 (IL-6) and interleukin 23 (IL-23) were tested. * p = 0.0005, ** p = 0.0001, n = 3 per group. \n",
      ".Figure 6A HEK293 cells were transfected with increasing amounts of Flag-TRIM33 expression vectors and 48 hours after transfection, cell lysates were subjected to immunoblotting (IB) using antibodies as indicated. B HEK293 cells were transfected with increasing amounts of Flag-FBXW7 expression vectors and 48 hours after transfection, cell lysates were subjected to immunoblotting (IB). C HEK293T cells were transfected as indicated, at 18 h after transfection, the cells were treated with the proteasome inhibitor MG132 (20μM) for 6 hours and then harvested. Cell lysates were immunoprecipitated by Flag-antibody (M2 beads) and Ubiquitin-conjugated GPX1 was detected by Western blotting with anti-Myc antibody. D HEK293 cells were transfected with either control or CUEDC2 siRNA and increasing amounts of Flag-TRIM33 expression vectors, cell lysates were subjected to immunoblotting (IB) using antibodies as indicated. \n",
      ".Figure 7A CUEDC2 ablation reduced I/R injury, which was induced by 30 minutes of ischemia followed by 24 hours reperfusion. AAR, area at risk; LV, left ventricular area, IRI, area of I/R injury. The ratios of AAR to LV, IRI to AAR and IRI to LV are shown. * p < 0.0001 compared with WT, n = 15 mice per group. (top panel) Representative cross-sectional slices derived from the hearts stained by TTC. Area of I/R injury is indicated in light pink. B Representative images of ventricular myocardium sections from WT and Cuedc2-/- mice exposed to sham operation or I/R injury (2 hours after the onset of reperfusion following 30-min ischemia). Green, TUNEL-positive myocytenuclei; red, Troponin I-stained cardiomyocytes; blue, DAPI-stained nuclei. Scale bar, 50 μm. C Quantitative analysis of apoptosis. * p < 0.0001, n = 8 mice per group. D SerumLDH levels in sham-operated mice or those subjected to ischemia and a 4-hour reperfusion. Data from 3 separate experiments was analyzed by one-way analysis of variance. * p < 0.0001, n = 13 mice per group. E Echocardiography analysis of left ventricular dimensions and cardiac function at 1 week post-I/R in mice. ** p < 0.0001, n = 15 mice in the sham-operated group and n = 10 mice in the I/R group. \n",
      ".Figure 4A Protein was extracted from the left ventricle of wild-type or Cuedc2-/- adult mice (12 week-old) and subjected to immunoblot analysis (3 mice per group). B Wild-type or Cuedc2-/- mice were subjected to a heart ischemia for 30-minute in vivo, followed by reperfusion for the indicated times. Protein was extracted from the area at risk of left ventricle and subjected to immunoblot analysis. C GPX1 protein level was plotted using the immunohistochemical scores as described in methods. (left) representative images from immunohistochemistry staining for GPX1 in tissues from patients who suffered acute myocardial infarctions. (right) plot of GPX1 scores in each BZ and DZ (n = 9 patients). DZ, distant area; BZ, border zone. Scale bars, 250 μm (left). * p = 0.0039. D Primary neonatal mousecardiomyocytes stably expressing control or GPX1 shRNA by lentivirus infection were treated with H2O2 (1 mM) for 3 hours. Cell viability was detected by FACS. * p = 0.0001; ** p = 0.0275, n = 3 wells per group. NS, not significant. E Primary neonatal mousecardiomyocytes were transfected with lentivirus carrying control or GPX1 shRNA, and the efficiency of GPX1 knockdown was tested by immunoblotting. F Primary neonatal mousecardiomyocytes stably expressing control or GPX1 shRNA by lentivirus infection were subjected to H2O2 (1 mM) for 30 minutes and then stained with 5 μM CellROX® Deep Red Reagent and analyzed by flow cytometry. * p = 0.0056; ** p = 0.0218, n = 3 wells per group, n = 3 wells per group. NS, not significant. \n",
      ".Figure 5A Lysates from MEF-WT or MEF-Cuedc2-/- cells treated with 20μM cycloheximide (CHX) for the indicated times were subjected to immunoblotting (left). Relative GPX1 levels were quantified by densitometry (right). B MEF-Cuedc2-/- cells were transfected with increasing amounts of FLAG-CUEDC2 (mouse) plasmids (1 μg and 2 μg). At 24 hours after transfection, the cells were treated with the proteasome inhibitor MG-132 (10 μM). Cells were cultured for additional 6 hours and subjected to immunoblotting. C HEK293T cells were transfected with the FLAG-GPX1, HA-CUEDC2, and Myc-ubiquitin plasmids as indicated and treated with MG-132 (10 µM) for 6 hours before harvest. Cell lysates were immunoprecipitated (IP) with anti-FLAG (M2). The immunoprecipitates were analyzed by western blot using an anti-GPX1 antibody. Whole-cell lysates (WCL) were analyzed by Western blots with anti-FLAG or anti-HA antibody to determine the protein of Flag-GPX1 and HA-CUEDC2. D The heart tissue lysates from wide-type and Cuedc2-/- mice were incubated with IgG or anti-GPX1 antibody, and then captured on protein G-Sepharose beads. The immunoprecipitates were analyzed by immunoblotting with an anti-ubiquitin antibody. E HEK293T cells were transfected with the Flag-CUEDC2 or Flag-CUEDC2△CUE (deletion of the CUE domain of CUEDC2), and treated with MG-132 (10 µM) for 6 hours before harvest. Cell lysates were immunoprecipitated (IP) with anti-Flag (M2). The immunoprecipitates were analyzed by western blot using an anti-GPX1 antibody. Whole-cell lysates (WCL) were analyzed by Western blots with anti-Flag or anti-GPX1 antibody to determine the protein of Flag-CUEDC2 and GPX1. v: vector. \n",
      ".Figure 3A Gadd45 proteins significantly improve reprogramming efficiency. The reprogramming efficiencies were compared with SKO or SKOM, respectively. We normalized the numbers of SKO+Factor to SKO+Flag (1 as control); also we normalized the numbers of SKOM+Factor to SKOM+Flag (1 as control). (**p≤0.01; ***p≤0.001; n=3) B The efficiencies of SKO- and SKOM-induced reprogramming were tested in over expression of two or three Gadd45 proteins. The reprogramming efficiencies were compared with SKO or SKOM, respectively. (**p≤0.01; ***p≤0.001; n=3) C SKO-MEFs or SKOM-MEFs infected with DOX-inducible Gadd45a were either treated with DOX immediately after infection, and DOX was removed at different time points (left panel) or treated with DOX from different time points until the end of the experiment (right panel). The reprogramming efficiencies were compared with DOX-free treatment. (n=3) \n",
      ".Figure 1A Scheme of H1-GFPFRAP in heterochromatin foci (with HP1a-mCherry) and euchromatin (without HP1a-mCherry). The relative FRAP curves of euchromatin (Eu) and heterochromatin (Het) are shown divided into two parts-the mobile fraction (MF) and the immobile fraction (IF) after bleaching recovery. (ROI, region of interest) B FRAP curves of H1-GFP in reprogramming with SKO. The mean values of relative fluorescence recovery were shown in the curves. For heterochromatin, the recovery ratio is higher in cells transfected with reprogramming factors than that in the control on day 3. Changes are significantly different. More than 16 cells were analyzed for each group. (*p≤0.05)C The ratio of MF at 120 s post-bleaching in FRAP was compared in the reprogramming stages. For heterochromatin, cells transfected with SKO showed much more rapid recovery than control cells on day 3. More than 16 cells were analyzed for each group. (*p≤0.05) D DNA FISH images showing the localizations of endogenous Oct4 locus and HP1a foci in MEFs infected with SKO or Flag control. More than 72 cells were analyzed for each group. Scale bar: 5 μm.E Summary of percentage of co-localizations at the Oct4 locus and HP1a foci in SKO mediated reprogramming. More than 72 cells were analyzed for each group. \n",
      ".Figure 2A The recovery kinetics of heterochromatin in MEFs infected with Flag or Gadd45a on day 3. The ratio of MF at 120 s post-bleaching is shown at the right panel. More than 20 cells were analyzed for each group. (***p≤0.001) B The recovery kinetics of heterochromatin in MEFs infected with Flag alone or SKO plus Flag or Gadd45a on day 3. The ratio of MF of heterochromatin is shown at the right panel. More than 19 cells were analyzed for each group. (*p≤0.05) C ChIP-PCR analysis of H3K9Ac, H3K9me2, H3K9me3, H3K27Ac, H3K27me2, or H3K27me3 modification levels in Oct4 binding sites of MEFs infected with SKO plus Flag or Gadd45a on day 3. (**p≤0.01; n=3) \n",
      ".Figure 9A HPLC detection of a superoxide probe oxidized dihydroethidium (DHE) product in young (8-week-old) and old mouse (20-month-old) heart tissue, 2-hydroxyethidium (EOH), a specific product for superoxide anion radical. * p = 0.0087, n = 5 in each group. B HPLC detection of ethidium (E), oxidized by other reactive oxygen species such as H2O2 (mainly) and ONOO. * p = 0.0021, n = 5 in each group. C Comparison of 8-hydroxy-2'-deoxyguanosine (8-OHdG) in young and old mice. * p = 0.0164, n = 6 mice per group. D Comparison of heart weight / body weight (HW/BW) in WT and Cuedc2-/- mice. ** p = 0.036, n = 15 in young WT and Cuedc2-/- mice, n = 10 in old WT mice and n = 12 in old Cuedc2-/- mice. E Comparison of the left ventricle function in WT and Cuedc2-/- mice by transthoracic echocardiography. *p = 0.0384, **p = 0.0389. n = 15 in young WT and Cuedc2-/- mice, n = 10 in old WT mice and n = 12 in old Cuedc2-/- mice. F Left ventriclecardiomyocytes apoptosis was tested by TUNEL staining. *p = 0.0017, n = 5 in each group. G Left ventricle myocardial fibrosis was tested by Masson staining in WT and Cuedc2-/- mice. The represent results were shown in (G) and the quantitative analysis of myocardial fibrosis was shown in (H). *p = 0.0177, n = 5 in each group. \n",
      ".Figure 8A Immunoblotting for CUEDC2 protein level. 4 weeks after tail-vein injection of rAAV9-CUEDC2, indicated mice tissues were harvested and subjected to immunoblotting to detect CUEDC2 protein level. Ctl: heart of rAAV9-GFP injected mice. B Animals were injected by rAAV9-CUEDC2 in a different amount of viral genomes (VG). Heart tissues were then analyzed by immunoblotting to detect CUEDC2 transduction. Tubulin protein level was examined for normalization purposes. C The ratios of AAR to LV, IRI to AAR and IRI to LV are shown. n = 10 mice in WT + rAAV-GFP group and Cuedc2-/- + rAAV-GFP group; n = 12 mice in Cuedc2-/- + rAAV-CUEDC2 (1×1010 VG) group and Cuedc2-/- + rAAV-CUEDC2 (1×1011 VG) group. * p = 0.0055, ** p = 0.0146, *** p = 0.0200, **** p = 0.0377. D Effect of rAAV9-CUEDC2 treatment on echocardiography indices of left ventricle function recovery post-I/R measured by fractional shortening (FS) and ejection fraction (EF). WT+ rAAV-GFP (n = 10); Cuedc2-/- + rAAV-GFP (n = 10); Cuedc2-/- + rAAV-CUEDC2 (Low dose) (n = 12); Cuedc2-/- + rAAV-CUEDC2 (High dose) (n = 12). ** p < 0.0001. \n",
      ".Figure 6A Summary of enriched Gene Ontology terms more potently upregulated by Gadd45a. The p-values represent the modified Fisher Exact corrected EASE score. B Heatmaps depicting the relative fold change of gene expression at 8 dpi by DNA microarray. Red and green colors indicate increased and decreased expression, respectively. C qPCR analysis of endogenous Oct4, Nanog or endogenous Sox2expression level during reprogramming with SKO plus Flag, wild-type Gadd45a or G39A Gadd45a. (**p≤0.01; ***p≤0.001; n=3) \n",
      ".Figure 4A Several Gadd45a mutations were tested revealing that only the G39A mutation lost the ability to increase the H1 dynamics of heterochromatin. More than 18 cells were analyzed for each group. (***p≤0.001)B The ratio of MF increased with wild-type Gadd45a or K45E and R34G Gadd45a, but not G39A Gadd45a at day 3. More than 18 cells were analyzed for each group. (***p≤0.001) C Immunofluorescence detection of HP1a foci in MEFs transfected with wild-type Gadd45a or G39A Gadd45a (upper panel). Quantitative analysis of the ratio of the HP1a foci area to the total nuclear area revealed by DAPI staining (dashed lines) shows that wild-type Gadd45a reduces the relative amount of HP1a foci, whereas the G39A Gadd45a does not (lower panel). More than 20 cells were analyzed for each group. Scale bar: 5 μm. (**p≤0.01) D Immuno-FISH images showing the localizations of endogenous Oct4 loci and HP1a foci in MEFs infected with wild-type or G39A Gadd45a. More than 72 cells were analyzed for each group. Scale bar: 5 μm. E Comparison of the reprogramming efficiencies of SKO-MEFs and SKOM-MEFs overexpressing wild-type or mutant Gadd45a proteins. The reprogramming efficiencies were compared with SKO or SKOM, respectively. (**p≤0.01, ***p≤0.001; n=3) \n",
      "Fig.Figure 3(A) Frequency of mitotic phenotypes observed in indicated cell lines with or without DIA treatment (3 repeats, n>50 per repeat, error-bars indicate s. d.). DIA was added for four hours in asynchronous cells followed by 12 hours of imaging. (B) Mitotic duration comparing mock-treated and DIA-treated B1dd/B2ko cells. Time of Entry and Exit was defined by chromosome condensation and decondensation (C) Representative images from live-cell imaging of SiR-Tubulin labelled (red) mitotic B1dd/B2ko cells expressing FusionRed-H2B (green), Mis12-GFP (white); time in mins, scale bar = 5µm. Cells were imaged four hours after DIA treatment. (D) Characterisation of mitotic spindle in proTAME/Apcin (P/A) arrested B1dd/B2ko cells with or without DIA treatment For cell synchronisation (P/A synchronisation) we released from 24-hour Thymidine block into DIA- or PBS-containing medium and added (P/A) 10 hours after release. APC/C inhibition lasted for two to four hours and was followed by fixation and immunofluorescence analysis. (E) Intensities of anti-(pSP) antibody staining in cells treated as in (D) (3 repeats, n>50 per repeat). Key to legend: I-DIA = control interphase cells, M-DIA, M+DIA = control and DIA treated mitotic cells. (F) Cyclin A2 levels were assessed by immunoblotting after 18 hours in 100ng/ml Nocodazole and/or 5µM STLC. 6µM proTAME was added to the indicated samples for the final six hours. (G) Upper panel, confirmation of Cdc27 depletion by immunoblotting 72 hours after siRNA transfection in B1dd/B2ko cells. Lower panel, following 40-hour siRNA transfection B1dd/B2ko cells were treated for 24 hours with Thymidine, released for 10 hours and treated with proTAME and Apcin for additional four hours. At this point, the cells were fixed and stained with tubulin and pericentrin for immunofluorescence analysis. (H) Representative images of mitotic spindles in control- and DIA-treated B1dd/B2ko cells following arrest in mitosis by proTAME/Apcin treatment Tubulin staining is shown in white, pericentrin in green and DAPI in red, scale bar = 5µm. (I) Confirmation of cyclin B1 and B2 depletion 48 hours after siRNA (pool of four different siRNAs) depletion by immunoblotting in WT RPE-1 cells. Asterisks indicate non-specific bands that are not targeted by siRNAs. (J) Additional siRNA depletion of CycB1 and B2 in Ctr and DIA treated B1dd/B2ko cells does not prevent mitotic entry. Cells were released from a 24-hour Thymidine arrest for 10 hours at which point proTAME/Apcin were added for an additional 6 hours before fixation, Hoechst staining and mitotic index scoring. DIA treated mitotic cells. (K) Additional siRNA depletion of CycB1 and B2 in Ctr and DIA treated B1dd/B2ko cells does not change the mitotic phenotypes caused by DIA depletion. and stained with tubulin and pericentrin for immunofluorescence analysis. (L) Immunofluorescence images from mitotic Ctr and DIA treated B1dd/B2ko cells after P/A synchronisation stained with anti-alpha-tubulin (green), anti-gamma-tubulin (red) antibodies and DAPI (blue), scale bar = 5µm. Cells were exposed to ice-cold medium and either fixed immediately after cold exposure, or incubated for 5,10, or 20 minutes at 37° C before fixation. \n",
      ".Figure 5A Recombinant Gadd45a protein and its G39A mutant were added to the histones-oligo DNA mixture. The mixtures were separated by 4% non-denaturing PAGE and stained with ethidium bromide (EB). The oligo DNA probes were amplified from the Oct4 promoter. B Recombinant wild-type and G39A Gadd45a were added to the mixtures of oligo DNA and the recombinant H2A/H2B heterodimer or H3/H4 tetramer. The mixtures were separated by 4% non-denaturing PAGE and stained with EB. C Co-immunoprecipitation of wild-type Gadd45a, G39A Gadd45a and H3 shows that the interaction between Gadd45a and H3 is dependent on G39 residue. D Two step crosslinking method identified Gadd45a could interact with chromatin in the promoter regions of Oct4 and Nanog. Cells were crosslinked with DSG before ChIP-PCR analysis. (**p≤0.01; ***p≤0.001; n=3) E The chromatin compaction of the indicated regions was detected by nuclease accessibility assay. Genomic DNA was purified from MEFs infected with Flag alone and SKO plus Flag, wild-type Gadd45a or G39A Gadd45a on day 8. (*p≤0.05; **p≤0.01; n=3) F ChIP-PCR analysis of the binding of Oct4, Sox2, and Klf4 to their targets individually in MEFs infected with SKO plus Flag, wild-type Gadd45a or G39A Gadd45a on day 8. (***p≤0.001; n=3) \n",
      "Fig.Figure 1(A) Schematic of protein degradation using AID and SMASh systems. The SMASh tag consists of the self-cleaving viral HCV NS3 protease fused in cis to a constitutive destabilising peptide. The tag detaches itself from the target protein in the off condition. Upon inhibition by the protease inhibitor, asunaprevir (Asv), the tag remains on the target protein and exposes it to the cellular degradation machinery, while the old protein is depleted depending on its specific half-life. The AID-tag relies on the expression of the plant F-box protein TIR1 that combines with mammalian Skp1 and Cul1 to form a functional SCF E3 ligase that is activated to ubiquitylate the AID degron tag upon stimulation by the hormone Auxin (IAA). In the case of the double tag combination newly synthesised proteins are depleted by the two independent degradation systems. Old proteins are destabilised by the mAID-tag (mini-AID). (B) Knock-in strategy for inducible OsTIR1 expression at the human Rosa26 locus, and endogenous cyclin A2 and B1 protein tagging. We integrated mAID-SMASh-T2A-neomycin in frame at the C-terminus of the coding sequence of cyclins A2 and B1. The chosen length for the homologous regions for gene targeting was approximately 1kb. (C) To verify gene targeting primers were designed in the genomic locus and in the cassette (indicated in (B)) and used to amplify genomic DNA from different cell lines: wild-type RPE-1, RPE-1 expressing OsTIR1, CycA2dd, CycB1dd, and primers amplifying a region of the Kif23 gene were used as a positive control (D; E) (D) Representative immunoblots and (E) quantitation showing timecourse of induced degradation of double degron tagged cyclin A and cyclin B (A2dd, B1dd). * indicates crossreacting band. (n=3 experiments, s.d. indicated by error bars). Doxycycline/IAA/Asv (DIA was added to asynchronous cells for indicated time before extract preparation and immunoblotting. (F) Comparison of protein degradation efficiency using different degrons. Cells were incubated for indicated time periods with Asv or DOX/IAA or both (DIA, see methods) then collected for western blot analysis. Doxycycline was added 2 hours before addition of IAA or Asv/IAA. To determine protein degradation, either cyclin A2 or cyclin B1 were probed as indicated. Anti-myc antibody was used to check OsTIR1-myc expression. Asterisks indicate non-specific protein crossreacting with the cyclin B1 antibody. (G) Timecourse of DIA induced cyclin A and B degradation, cross checking levels of each mitotic cyclin. Treatments were performed as for DIA treated Cell extracts were checked for both cyclin A2 and B1 levels following DIA addition (H) Comparison of DIA and siRNA depletion. Cells were treated with DIA for four hours or pre-transfected with CycB1 SMARTpool siRNAs 48 hours earlier. Cell extracts were analysed by immunoblotting for cyclin B1 (L.E, long exposure, S.E. short exposure) using mouse monoclonal antibody (I) Assaying cyclin B1 depletion in mitotic cells using a rabbit mono-clonal antibody. Extracts from asychronous or mitotic cells were collected and analysed by immunoblotting using the indicated antibodies. \n",
      "Fig.Figure 2A) Cyclin B2 knock-out and induced degradation by immunoblotting. Indicated cell lines were analysed 24 hours after mock or Dox/IAA/Asv (DIA) treatment using the indicated antibodies to confirm homozygous gene tagging and efficiency of protein degradation. ( (B) Cell cycle analysis 24 hours after DIA treatment. Cells were analysed by EdU labelling, PI staining and FACS analysis. The histograms show the PI intensities while the dot plots show EdU incorporation (Y-axis) vs PI intensity (X-axis). Gating of cell cycle phases is indicated by colour (G1-black, S-phase -green, G2/M-phase-orange) (C) Cell cycle phase frequencies quantified from flow cytometry data (shown in (B)) in indicated cell lines 24 hours after DIA treatment, (n=3 experiments, s. d. indicated by error bars). (D) Cell proliferation of A2dd, CycB1dd and B1dd/B2ko following mock or DIA treatment. 1000 cells were plated in each well (diameter 3.5cm) and incubated for 10 days before methanol fixation and Crystal Violet staining. (E) Kinetics of mitotic entry as measured by time lapse microscopy in A2dd and B1dd/B2ko cells following mock or four-hour DIA treatment of asynchronous cells. The cells were imaged for 16h with 5min intervals using widefield DIC, mitotic entry was manually scored by detecting cell rounding. Curves display the cumulative mitotic index (data from three repeats, n>500 cells per condition, s. d. indicated by shaded area). (F) qPCR analysis of cyclin B3 mRNA levels, following 72 hours depletion in B1dd/B2ko cells. For quantification, we used primers to amplify two control mRNAs, TATA binding protein, and Actin. The plot shows the levels of CycB3 siRNA depleted mRNA relative to Ctr siRNA transfected cells (Bars indicate the mean of three independent experiments, error bars indicate the standard deviation between these three repeats). (G) Mitotic index measurements of B1dd/B2ko cells with the indicated treatments. Cells were transfected with siRNA for 72 hours, after 36 hours they were blocked for 24 hours with Thymidine, and fixed 12 hours after release from Thymidine. ProTAME and Apcin were added for the final two hours before fixation. Mitotic cells were scored based on DAPI staining and on condensed chromosome formation. The bar-plots show mean values of three biological repeats (n=100 per repeat and sample, error bars indicate standard deviation, p-values were calculated using an independent two sample t-test). (H) Cyclin A2 siRNA depletion in MCF7 and MCF10A cells. The cells were transfected with Ctr or cyclin A2 siRNA for indicated length of time and probed for cyclin A2 levels by immunoblotting. (I) Cyclin A2 siRNA depletion causes endoreplication. Following 72 hours of siRNA transfection MCF7 and MCF10A cells were analysed by PI staining and FACS. The histograms show the changes in DNA content (PI Int.) towards >4N following cyclin A2 depletion. (J) Cyclin A2 siRNA and degron depletion in RPE-1 cells. RPE-1 OsTIR1 and RPE-1 A2dd cells were subjected to 72 hours of cyclin A2 siRNA depletion and/or of DIA treatment as indicated and probed for cyclin A2 levels by immunoblotting. The longer exposure (L.E.) reveals incomplete depletion of cyclin 2 by siRNA. (K) Cyclin A degron depletion causes accumulation of cells in G2 phase. Following 72 hours of siRNA transfection or DIA treatment, RPE-1 OsTIR1 and A2dd cells were analysed by PI staining and FACS. The histograms show the changes in DNA content (PI Int.) towards >4N following cyclin siRNA A2 depletion in RPE-1 OsTIR1 cells, while DIA treatment in A2dd cells does cause an increase in the 4N but not >4N peak. \n",
      "Fig.Figure 9(A) Influence diagram of the network controlling the phosphorylation of mitotic substrates. The auto-activation of CycB:Cdk1 and PP2A:B55 is controlled by inhibitory phosphorylation and by the Greatwall-ENSA pathway, respectively. CycA:Cdk1 helps CycB:Cdk1 auto-activation and inhibits PP2:B55 activity through Greatwall phosphorylation. Three different classes of mitotic substrates (early, intermediate and late) are distinguished based on their sensitivity to CycA:Cdk1, CycB:Cdk1 and PP2A:B55. (B) Final level of mitotic regulators and Cdk1-substrates in response to cyclin synthesis. The heat map is based on model simulations starting with initial conditions without cyclins (Interphase, top row) until high and constant cyclin levels caused by APC/C inhibition. (C) Numerical simulation of mitotic substrate phosphorylation network in response to rising cyclin levels. The model simulates mitotic entry experiments where cyclin levels become stabilized by APC/C inhibition. Each row corresponds to one of the experimental conditions used in this work (control, CycA depletion etc.). Left columns: cyclin levels and phosphorylated mitotic substrates (early, intermediate and late). Right columns: temporal changes of some mitotic regulators. Changes of Cdc25-P and Gwl-P (not shown) are similar to phosphorylated ENSA (pENSA). \n",
      "Fig.Figure 4(A) Time-lapse images of PCNA-mRuby tagged A2dd cells. The imaging sequence was started at the time of Doxycycline addition, and degron activity was triggered three hours later by addition of IAA and Asv or PBS (indicated by dashed line). Time is shown as hh:min, scale bar equals 10µm. For further analysis cells were chosen that had dissipated their PCNA foci before the addition of DIA to ensure that cyclin A2 degradation was triggered in G2 phase. (B) Single-cell analysis of 40 cells per condition treated each line represents the timing of G2/M/G1 progression for a single cell. (C) A2dd cells were pulsed with EdU to label S-phase then protein degradation was induced by DIA addition for 5h and before fixation and Edu/CENPF/DAPI staining. Cells that were in G2 at the time of DIA or mock treatment were identified as CENPF positive, EdU negative. Mitotic cells were excluded based on nuclei morphology. The bar plots show mean percentage of EdU negative/CENPF positive cells of three repeats, error bars indicate standard deviation. (D) Images from time-lapse sequence of mitosis showing cell division in controls and after addition of 0.5µM PD-166285 in DIA treated A2dd labelled with SiR-DNA (time is indicated as hh:min, the scale bars represent 20µm). (E) Accumulative index of mitotic entry measured by time-lapse microscopy in A2dd cells following mock or DIA treatment and addition of Wee1 inhibitors MK1775 (1µM) or PD-166285 (0.5µM). Asynchronous cells were DIA treated for four hours and then imaged for six hours in five-minute intervals. (F) Quantification of metaphase duration in A2dd cells treated with DIA and Wee1 inhibitors as indicated (n>50 per repeat, the boxplot indicates median, first and third quartile and minimum/maximum values. (G) A2dd cells were transfected with Ctr or B55α/δ siRNA. 48 hours after transfection the cells were treated with Thymidine for 24 hours and then released. 10 hours later the cells were probed for B55 depletion by immunoblotting using a pan-B55 monoclonal antibody. (H) Lack of B55 partially rescues mitotic entry in DIA treated A2dd cells. Cells transfected with Ctr siRNA or B55α/δ siRNA were released from Thymidine, mock or DIA treated and analysed by widefield live-cell imaging eight hours after the release. The cells were imaged every 10 minutes for 15 hours and manually scored for mitotic index. \n",
      "Fig.Figure 7(A) Quantification of Greatwall, ENSA/ARPP19 and PP1 phosphorylation based on phospho-proteomic data. (B) Confirmation of ENSA/ARPP19 S67 phosphorylation in mitotic extracts from Ctr and DIA treated cells, before and after ENSA/ARPP19 siRNA depletion. (C) Dynamics of mitotic substrate dephosphorylation triggered by Cdk1 inhibition in control and DIA treated B1dd/B2ko cells. The cells were enriched in mitosis by Thymidine release and proTAME/Apcin treatment and treated with 1µM Flavopiridol to trigger mitotic exit. Dephosphorylation was measured by probing the immunoblots with anti Cdk1 substrate (anti-pSP) antibody. (D) Three repeats of the experiment in (C) were quantified and corrected based on the corresponding GAPDH intensity values. The bar plots show the mean of three experiments and the standard deviation is indicated by error bars. (E) Dynamics of Greatwall, PP1, and Cdc27 dephosphorylation following Cdk1 inhibition in proTAME/Apcin treated mitotic cell extracts from Ctr or DIA treated cells. Cell extracts were prepared at the indicated times following treatment with 1µM Flavopiridol and analysed by immunoblotting \n",
      "Fig.Figure 8A) Representative images from live-cell imaging of siRNA transfected B1dd/B2ko cells (FusionRed-H2B in red, SiR-Tubulin in white, scale bar = 10µm). Bar-plot panels on the right show single-cell analysis of 10 cells manually scored for length mitosis pre-NEBD and post-NEBD. Entry into prophase was scored by cell rounding and NEBD was identified by influx of Tubulin in the nucleus. (B) Widefield Imaging. DIC (Grey), SiR-DNA (b/w), of ENSA/ARPP19 siRNA transfected DIA treated B1dd/B2ko cells. Time is indicated in hh:min, scale bar = 10µm. (C) Panels of representative immunofluorescence images of siRNA transfected and P/A treated cells treated as indicated, scale bar = 10µm. (D) Quantification of mitotic index of siRNA transfected P/A synchronised cells (E) Quantification of mitotic B1dd/B2ko cells following siRNA transfection to deplete Greatwall (siGwl) or ENSA/ARPP19 (siE/A) and/or DIA treatment. Frequencies of mitotic cells with intact (blue) and disassembled (red) Lamin A/C staining are plotted (mean value of three repeats, n>50 per repeat, error bars indicate standard deviation). quantification of control and DIA treated mitotic B1dd/B2ko cells following siRNA transfection to deplete Greatwall and B55α/δ. Frequencies of mitotic cells with intact (blue) and disassembled (red) Lamin A/C staining are plotted (mean value of three repeats, n>50 per repeat, error bars indicate standard deviation). (G) Intensity of pSP-antibody staining in siRNA transfected and DIA treated B1dd/B2ko cells. The swarm-blots classify data from mitotic cells with intact (blue) or disassembled (red) Lamin A/C. (H) As in (G) Intensity of anti-Lamin A/C pS22 antibody staining in siRNA transfected and DIA treated B1dd/B2ko cells. (I) Same as in (G) and (H), data for centrosome distance in mitotic cells. Data for G-I are from three repeats, n>50 per repeat, the bars indicate median, first and third quartile as well as minimum and maximum values, data are from three repeats, n=50 cells per repeat). \n",
      "Fig.Figure 5(A) FACS plots from EdU pulse-chase experiment. Cells were DIA or mock-treated for four hours and then labelled with EdU for one hour. The EdU was then washed out and samples were taken every three hours for 12 hours and analysed by EdU/PI staining and FACS. The top left panel shows the gates that were used for analysis. The quantifications shown in the panels to the right are from three independent repeats of this experiment. (B) Quantification of cells progressing into Gate 1 (Top panel, new S-phase cells) and Gate 2 (Bottom panel, G2 cells) over time. The lines represent the mean of three experiments, the coloured areas indicate the standard deviation. (C) Relative changes in the percentage of EdU positive cells in gates 3,4 and 5 (G1/early-S, mid S and late S/G2 phase). The bars indicate the relative mean values from three independent experiments, the error bars represent the standard deviation. (D) Cyclin A2dd cells with inducible CycB1-YFP (B1-WT) and CycB1-YFP-NLS (B1-NLS) were analysed for DIA induced cyclin B1 expression / cyclin A2 depletion. Samples were collected at indicated time points and probed by immunoblotting with cyclin B1 and cyclin A2 antibodies (E) Quantifications of EdU pulse chase in DIA treated cyclin A2 cells expressing B1-WT, B1-NLS or YFP as indicated. (F) Accumulative mitotic index measurement of cyclin A2 cells following cyclin A2 depletions and induction of indicated cDNAs. The lines represent the mean of three experiments, the coloured areas indicate the standard deviation. (G) Quantification of metaphase duration in A2dd cells treated with DIA following induction of indicated proteins (H) FACS plots from EdU pulse-chase experiment in A2dd cells following induction of indicated genes. (I) Immunoprecipitation of YFP from extract from A2dd following induction of YFP (Y), CycB1-YFP (BY) and CycB1-YFP-NLS (BYN). The samples in the odd lanes (IAA/Asv-) were treated with 1µg/ml doxycycline alone, while samples in even lanes (IAA/Asv+) were treated with Auxin and Asv to induce cyclin A2 depletion. Total cell extract and immune-precipitates were probed by immunoblotting with the indicated antisera. \n",
      "Fig.Figure 6(A) Cell synchronisation scheme: to obtain an optimal enrichment in mitotic cells we pre-synchronised B1dd/B2ko cells by serum starvation for 48 hours followed by a release into Thymidine for 24 hours. Cells were then released into S-phase in the presence or absence of DIA and blocked in mitosis by the addition of proTAME. Mitotic cells were collected by shake-off 12 hours following the release. (B) Phosphoproteomics workflow. Protein extracts were prepared from the indicated cells, digested with trypsin and labelled with 6-plex TMT reagents. The TMT-labelled peptides were then mixed and subjected to fractionation by high pH reverse phase into 24 fractions. From each fraction, 95% was taken for phospho-enrichment and 5% retained for analysis of unmodified peptides. For phospho-enrichment, the 24 fractions were combined into a total of 16 fractions. Samples (unmodified and phospho-enriched) were then analysed by LC-MS/MS on an Orbitrap Fusion instrument using synchronous precursor selection (SPS). This analysis resulted in detection of 11,234 phosphorylation sites, of which 5,670 were quantitated. The number of class I sites are indicated in brackets (site localisation probability > 0.75). (C) Quantification of cyclin B specific substrates by phospho-proteomics. Phosphorylation sites deemed significantly changing are shown in red (D) Gene ontology analysis of cyclin B-dependent substrates. (E) Quantification of changes in selected cyclin B-dependent mitotic phosphorylations including Ki-67 (left panel), INCENP/Borealin (middle panel) and Top2B, CDCA2 and Tpx2. (F) Representative images of Ki-67 immunofluorescent staining of chromosome spreads from control (Ctr) and DIA treated P/A synchronised B1dd/B2ko cells and quantification of Ki-67 staining intensity using cross-sections of chromosomes (G) Representative images from live-cell imaging of DIA-treated SiR-Tubulin labelled (red) B1dd/B2ko cells expressing FusionRed-H2B (green) and AurB-GFP (white); time is indicated in minutes, scale bar = 5µm. Bottom panel shows frequencies of aberrant AurB localisation in P/A synchronised B1dd/B2ko cells. (H) CDCA2 is excluded from mitotic chromosomes in control cells while this exclusion is lost in DIA treated B1dd/B2ko cells. The upper panel shows immunofluorescence images, for quantification we analysed fluorescence intensity across chromosomes. (I) Top2B levels in mitotic cells are increased in DIA treated B1dd/B2ko cells. The left panel shows immunofluorescence images, for quantification we segmented mitotic DAPI staining, measured the mean intensity of Top2B staining and plotted the ratio between mitotic and interphase nuclei. (J) Tpx2 shows reduced spread on mitotic chromosomes in DIA treated B1dd/B2ko cells. The top panel shows immunofluorescence images, for quantification we measured the mean intensity of Tpx2 staining on the centrosomes and spindle and plotted the ratio of these values per cell. We also analysed the centrosome/spindle distribution of AurA that does not change following DIA treatment. \n",
      ": Smaug and SMO physically interact and colocalize (A) Coimmunoprecipitation of SMO-HA and Myc-Smaug proteins.Figure 1(A) Coimmunoprecipitation of SMO-HA and Myc-Smaug proteins. Extracts of Cl8 cells expressing combinations of SMOWT-HA or SMOPKA-SD-HA with Myc-SmaugWT in the absence or presence of HH were immunoprecipitated (IP) with anti-HA. The input (lower panel) and the IP complexes (upper panel) were analyzed by western blot with anti-Myc (αMyc) or anti-HA (αHA) antibodies. Here and in the other figures, the names of the proteins detected are indicated on the left and the molecular weights on the right, in kDa; the samples loaded for the input and the supernatants are equivalent to a tenth of the volume loaded for the IPs. * corresponds to higher molecular weight forms of SMO of unknown origin. Here and in panel F, the black arrow indicates unphosphorylated SMOWT-HA and the bracket indicates the phosphorylated forms of SMOWT-HA or SMOPKA-SD. (B) Schematic representation of the domain structure of the Smaug protein. The conserved SSR 1 and 2 are shown in blue. They are separated by a 79 amino-acid-long non-conserved region (here referred to as 'M'). The sterile alpha motif (SAM) domain is in green and the pseudo heat analogous topology domain (PHAT domain, which increases the affinity of the SAM domain for SRE) in yellow. The dashed double-arrow line at the top represents the smallest interacting region (called SID for Smallest Interacting Domain) found according the two-hybrid screen. The truncated constructs used are presented below. The ability to interact with SMOWT-HA is indicated on the right: in green for yes, red for no. The full red double-arrow line below represents the smallest SMO-binding region (BR) region that we could identify. The amino acid numbers correspond to Smaug-PA. (http://flybase.org/reports/FBgn0016070.html). See also Fig EV1A-C. (C) Schematic representation of the C-terminal 'cytotail' of SMO. The PKA/CKI and FU phosphorylated regions are indicated as orange and green boxes, respectively. The FU-binding region (BR) is indicated by a full green double arrow above. The truncated constructs used are presented below and their ability to coIP with Myc-SmaugWT is indicated on the right. The full red double-arrow line at the bottom represents the smallest Smaug binding region (BR) region identified in this work. (D, E) Mapping of the Smaug interaction domain in SMO. Extracts from Cl8 cells transfected with Myc-SmaugWT and various regions of SMO fused to either HA (D) or to GFP (E) were immunoprecipitated with αMyc (D) or αGFP (E) and analyzed by western blotting with αMyc (lower panel in D and upper panel in E), αHA (D, upper panel) or αGFP (E, lower panel). In: Input; Sup: supernatant. See also Fig EV1D-F. (F) Hyperphosphorylated forms of SMO do not interact with Smaug. Left panel: Extracts of Cl8 cells expressing SMOWT-HA with or without Myc-SmaugWT, in the absence or presence of HH were IP with an αMyc antibody before analysis by western blot with αHA (upper panel) or αMyc antibodies (lower panel). Right panel: Cl8 cells expressing SNAP-SMOWT with Myc-SmaugWT in the presence of HH were extracellularly labelled with a membrane-nonpermeable fluorescent SNAP substrate before cell lysis and immunoprecipitation (IP) with anti-Myc. Labelled SNAP-SMO was directly visualized after electrophoresis. Note its presence in the IP fraction with a relative enrichment of the less phosphorylated forms compared to the hyperphosphorylated forms. See also Fig EV1G-J. (G) SMO and Smaug colocalize. Representative fluorescent images of Cl8 cells expressing GFP-SmaugWT alone (1-2) or together with SMOWT-mCherry (3-3'' and 4-4'') either without (1, 3-3\") or with HH (2, 4-4\"). The merge images in 3\" and 4\" show GFP-SmaugWT in green and SMOWT-mCh in red. The same results were seen with different fluorescent tags as well (Fig EV2B). The lack of effect of HH on GFP-SmaugWT in absence of SMOWT-mCh is likely due to limiting amounts of endogeneous SMO. A least 20 cells were assayed for each condition. In absence of HH, all co-transfected cells exhibited greater than 90% colabelled SMOWT-mCherry. Scale bar (shown in G1, identical for all panels):10μm. See also Fig EV2. \n",
      ".Figure 1Schematic representation of NUP58, CHMP1B, NUP85 and NUP153 genes (transcript IDs reported: NUP58 ENST00000381736.8, CHMP1B ENST00000526991.3, NUP85 ENST00000245544.9, NUP153 ENST00000537253.5). Exons shown as black boxes, introns as connecting black lines, and gRNA target sites are indicated with red triangles. Scale bars, 2kb. Diagram detailing experimental method for isolation of mutant clones. After transfection with eGFP Cas9 plasmid carrying dual gRNA, GFP+ HAP1 cells were individually sorted into 96-well plates. A total of 10 plates were prepared for each biological replicate (n=3). Relative cellular viability following Cas9-induced gene editing. Reported number of colonies was normalized based on viability of HAP1 cells transfected with an Empty Vector (EV) control (dots). Error bars indicate mean of SD from three biological replicates (number of colonies for EV rep1=242/rep2=238/rep3=201; CHMP1B rep1=134/rep2=151/rep3=128; NUP58 rep1=11/rep2=13/rep3=15; NUP85 rep1=0 /rep2=0/rep3=5; NUP153 rep1=61/rep2=55/rep3=60)(Ordinary one-way ANOVA ****P ≤ 0.00001; ns = non-significant). Graph indicates the percentage of viable clones with either undetectable, truncated or wild-type-like proteins as assessed by western blot analysis. Percentage of not analysed colonies is reported after CHMP1B targeting. \n",
      ": Model.Figure 6 Figure 6: Model. In absence of HH, Smaug is associated, likely as S-bodies (in red) to inactive SMO present on intracellular vesicles and leads to the repression (blue line) of its target mRNA(s) (called here xmRNA) via the recruitement of proteins (not represented here) that promotes their decay. Upon HH reception, activated and phosphorylated (yellow P circles) SMO accumulates at the plasma membrane where it relocates both the kinase FU and Smaug. This lead to FU activation which in turn phosphorylates (green full circle) SMO and Smaug. This phosphorylation of Smaug would reduce its negative effect (dashed blue line) favoring the release of its target mRNAs - possibly via the dissolution of the S-foci - and leading to their accumulation. Finally, when the highest levels of SMO phosphorylation are reached (red P circle), phosphorylated Smaug is released, leading to attenuation of the effects of HH/SMO/FU on Smaug. EV Figure Legends \n",
      ".Figure 2Determination of maximum growth rate (Vmax) using a label-free cell count-based proliferation assay. Representative growth curves of WT and NUP58 mutant clone F are plotted using a semi-log scale displaying total numbers of cells over time. Growth rates were calculated across the entire duration of the experiment using a sliding window of three timepoints. Representation of the highest growth rates (Vmax) for each independent cell line. Growth rates of mutants clones were compared to the untransfected (WT) cell line seeded in the same plate. Each dot represents a technical replicate (as plotted from left to right: WT n=62, B n=63, C n=61, D n=63, WT n=84, F n=80, E n=82, WT n=90, A n=78) of one independent experimental replicate performed. Quantification of active nuclear import of asynchronous controls or mutant clones using an NLS-3XmCherry as reporter. Ratio of the cytoplasm vs nucleoplasm fluorescence intensity is plotted. Only cells with comparable overall fluorescence intensity are reported Dots represent analysed cells (WT n=39, B n=22, C n=50, D n=34, E n=34, F n=9, A n=42) of one independent experimental replicate performed. Representative images of analysed WT (top) and clone D (bottom) NLS-3xmCherry stable cells. Mean intensity of fluorescence for cytoplasm and nucleoplasm for each cell are labelled. Scale bar, 20 μm. Western blot for NUP58 protein expression in mutant clones (B-E) and control cells (clone A and WT). Whole-cell lysate hybridized with customized antibody 1 (Ab1, top panel) and antibody 2 (Ab2, bottom panel). Actin was used as loading control. \n",
      ".Figure 3Diagrams detailing NUP58 mutations in mutant HAP1 clones. Nucleotide and amino acid sequences of exons 4 and 13 surrounding the gRNA target site are shown for wild-type (WT) and NUP58 mutant clones. Insertions are highlighted as closed red triangles and deletions as open dotted triangles over the nucleotide sequence. Amino acid substitutions and PTC resulting from nonsense mutations are indicated in red. A large-scale deletion (largescale del) present in C2 and affecting part of intron 3-4, exon 4, intron 4-5, and part of exon 5 is represented with dashed lines. Representation of the highest growth rates (Vmax) for each technical replicate for each indicated cell line. Each dot represents a technical replicate (WT n=35, C1 n=27, C2 n=28, C3 n=34, C4 n=32, C5 n=35, C6 n=38) of one independent experimental replicate performed. Analysis of active nuclear import using an NLS-3XmCherry as reporter as described in Figure 2C-D. Dots represent analysed cells (WT n=22, C1 n=15, C2 n=15, C3 n=18, C4 n=20, C5 n=27) of one independent experimental replicate performed. \n",
      ": HH signaling and Smaug control each other in vivo (A) Effect of FUEE-SBR and HH/SMO on endogeneous Smaug target mRNAs in Cl8 cells.Figure 5(A) Effect of FUEE-SBR and HH/SMO on endogeneous Smaug target mRNAs in Cl8 cells. Relative mRNA levels estimated using semi-quantitative RT-PCR for various Smaug's targets in Cl8 cells expressing FUEE-SBR (black) , FUEE (dark grey) , or FUDANA-SBR (light grey) were reported with respect to levels of a GFP control. mRNA levels were set to 1 for the GFP control. The list of genes and the sequences of the primers used are shown in Appendix Table S2 and S3. See also Fig EV5E. Here and in Fig EV5D: RNA encoding mitochondrial proteins are written in blue. N=3 (biological triplicates) and the quantifications were repeated 3 times on each sample.The results are presented as the means ± standard error. The differences between groups were assessed using Student's t-tests with Prism V8.2.1 software. A p value <0.05 was considered to indicate a statistically significant difference (B) Effect of HH on endogeneous targets of Smaug in the wing imaginal disc. Relative levels of Smaug's targets mRNAs in MS1096; UAS-hh or hhts and their respective controls (as in Fig EV5X). N=3, with xx disc for each phenotype. (C) Effect of smaug mutation on [fu1] wing class distribution. Percent distribution of phenotypic classes (defined in Appendix Figure S4) in fu1 males in presence of zero (fu1), one (fu1;;smaug47/+) or two copies (fu1;; smaug47/smaug47) of the smaug47 allele. The wings were double-blindly classified in \"weak\" and \"strong\" according to the LV3-V4 defects (see Fig EV6B). *Chi2 statistical analysis gave a p of 0,015 for fu1 and fu1;;smaug47/+ distributions and of 0,0024 when comparing fu1;;smaug47/+ to fu1;; smaug47/smaug47. \n",
      ": FU downregulates Smaug levels and repressive effect.Figure 4(A and C) Effect of FUEE-SBR on Smaug phosphorylation and levels. Extracts of Cl8 cells transfected with λN-SNAP-smaug (25 ng) and SNAP-GUS-5BoxB (300 ng) plasmids in absence or with different amounts of FUEE expressing plasmid (ranging from 15 to 60 μg, as indicated) were analyzed by electrophoresis before quantification as above. See a reprenstive gel in Fig EV5A. The % of phosphorylated SNAP-Smaug protein (determined as in Fig 3) (A) are represented as a function of the levels of FUEE-SBR. The mean values and (SD) for independent biological triplicates (N=3) are shown in the graph at the bottom. The statistical analysis was done by one tailed bivariate Wilcoxon rank test with a p value= 0.05 (*). Here CLIP-GUS was used as an internal control. (B) Effect of FUEE-SBR on Smaug activity. The relative levels of reporter expression obtained in the same experiment as in (A-C) are represented as a function of the % of phosphorylated Smaug. Color of data points correspond to ng of transfected FUEE-SBR in A-C. (A and C) Effect of FUEE-SBR on Smaug phosphorylation and levels. Extracts of Cl8 cells transfected with λN-SNAP-smaug (25 ng) and SNAP-GUS-5BoxB (300 ng) plasmids in absence or with different amounts of FUEE expressing plasmid (ranging from 15 to 60 μg, as indicated) were analyzed by electrophoresis before quantification as above. See a reprenstive gel in Fig EV5A. The relative levels of of λN-SNAP-Smaug protein (C) (determined as in Fig 2) are represented as a function of the levels of FUEE-SBR. The mean values and (SD) for independent biological triplicates (N=3) are shown in the graph at the bottom. The statistical analysis was done by one tailed bivariate Wilcoxon rank test with a p value= 0.05 (*). Here CLIP-GUS was used as an internal control. (D) Effect of FUEE-SBR on Smaug subcellular localization.Representative fluorescent images of Cl8 cells expressing GFP-SmaugWT (1), FUEE-SBR-mCherry (2), FUDANA-SBR-mCh (3) or expressing GFP-SmaugWT together with FUEE-SBR-mCherry (4-4'') or GFP-SmaugWT together with FUDANA-SBR-mCh (5-5''). The merge images in 4\" and 5\" show GFP-SmaugWT in green and FUEE-SBR-mCh or FUDANA-SBR-mCh in red. (6) In presence of FUEE-SBR, 274 of a total of 294 cotransfected cells show a diffuse localization of Smaug and FU and no foci. In presence of FUDANA-SBR, 100% of the cotransfected cells (n=241) show Smaug and FU distributed in bodies and colocalized. 250 ng of plasmid were used for each construct. Scale bar (shown in D1, identical for all panels): 10μ \n",
      ": SMO/HH regulates Smaug levels and activity.Figure 2(A) Smaug repression assay. The assay is based on the dual expression of a construct encoding a λN-HA-SNAP-Smaug chimeric protein (abbreviated λN-SNAP-Smaug) and a second that is transcribed into a reporter mRNA (called SNAP-GUS-5BoxB) that carries a translational fusion between SNAP and GUS coding regions followed by five Box B hairpins (5BoxB) inserted in the 3'UTR. A plasmid encoding a GFP-SNAP fusion (not shown here) was also used as a transfection and loading control. (B, C) Smaug downregulates reporter expression. Mean values of the relative levels of SNAP-GUS reporter (SNAP-GUS/GFP-SNAP ratio, in arbitrary units) in extracts of transfected Cl8 cells in the absence (black) or presence of λN-SNAP-Smaug (red), λN-SNAP (pale grey) or SNAP-Smaug (dark grey). See Fig EV3A for the representative gel and Fig EV3B which shows that λN-SNAP-Smaug has no effect on a SNAP-GUS construct that lacks the 5BoxB. N=3 (biological replicates). See also Fig EV3A-C. Here and in D-F: SNAP-GUS/GFP-SNAP ratios were set to 1 in cells expressing λN‐SNAP-Smaug. The error bars correspond to the standard deviation of the mean (SD). Statistical analysis was done by a Kruskal-Wallis test by ranks followed by a Dunn test and p values are: black bracket <0.05, red bracket <0.01, green bracket <0.0001. Here and in D-E: 15 ng of pAct.λN-SNAP-smaug plasmid were used per transfection. , E) HH/SMO reduces Smaug levels and its overall negative effect on reporter expression. Mean values of the relative levels of reporter expression (determined as above) (D) or of λN-SNAP-Smaug (ratio λN-SNAP-Smaug/GFP-SNAP) (E) in the absence of SMO constructs (red) or in the presence of SMOWT-HA (green), SMOPKA-SD-HA (blue) or SMO∆958-HA (yellow), without HH (plain boxes) or in presence of HH (striped boxes). N=6 (independent biological replicates), except for \"no SMO, no HH\" and \"SMOWT with HH\" conditions where n=10 (independent biological replicates). A representative gel is shown in Fig EV3D. Note that SMOWT-HA or HH alone had reproducible weak effects that were observed in two independent biological triplicates but were not statistically significant. See also Fig EV3E-F. (F) HH/SMO reduces Smaug's intrinsic repressive ability. A similar experiment was conducted with different amounts of λN-SNAP-smaug expressing plasmid (ranging from 0 to 50 ng) as indicated). The mean values of the relative levels of reporter (y-axis) are plotted against the relative levels of λN-SNAP-Smaug (x-axis). SNAP-GUS/GFP-SNAP ratios were set to 1 in cells expressing λN‐SNAP-Smaug in absence of SMOPKA-SD/HH. The colored area represent the extent of the values observed for the relative amounts of reporter expression obtained with (blue) and without (red) SMOPKA-SD/HH, respectively. N=3 (biological triplicates). See Fig EV3H and also a representative gel in Fig EV3G. \n",
      ": SMO/HH activation promotes the phosphorylation of Smaug.Figure 3(A) HH/SMO promotes slow-migrating forms of Smaug. Western blot analysis of Cl8 cells that transiently express HA-SmaugWT alone, or together with either SMOWT-GFP or SMOPKA-SD-GFP, in the presence or in absence of HH. GMAP serves as a loading control. Here and in the other panels, the antibodies used are indicated on the left. Here and in (B,C), the black arrows indicate the unphosphorylated form of HA-SmaugWT and the brackets indicate the slower migrating phosphorylated forms of HA-SmaugWT. NT: not transfected. See a phosphatase assay in Fig EV4A. (B) Smaug phosphorylation requires its interaction with SMO. Western blot analysis of Cl8 cells expressing HA-SmaugWT, in presence of HH, in combination with SMO∆958-GFP, SMOPKA-SD-GFP or SMOPKA-SD, ∆978-GFP, as indicated. (C) FU controls Smaug phosphorylation. Extracts of Cl8 cells expressing SNAP-Smaug with SMOPKA-SD FU-SD-GFP (with HH) in combination with GFP-FUWT, GAP-GFP-FUWT, Myc-FUEE or GAP-null-FUDANA (as indicated) were labeled for the SNAP tag before electrophoresis. After quantification of the gels, the percentage (%) of phosphorylated Smaug (% of phosphorylated forms of SNAP-Smaug total amounts of SNAP-Smaug) was estimated. In C to E, the mean values and (SD) for independent biological triplicates (N=3) are shown in the graph at the bottom, and the statistical analysis was done by one tailed bivariate Wilcoxon rank test. See also fig EV4C, D. (D) Phosphorylation of Smaug by activated FU requires the co-expression of SMO. Mean values of the percentage of phosphorylated Smaug in extracts of Cl8 cells that express λN-SNAP-Smaug with GAP-GFP-FUWT and SMOWT alone or together (as indicated) N=3 (billogical triplicates). Note that the effect of GAP-GFP-FUWT and SMOWT together is lower than the one observed with GAP-GFP-FUWT and SMOPKA-SD FU-SD-GFP together (in panel C). This could have multiple nonexclusive causes, including the fact that SMOWT is present at much lower levels than SMOPKA-SD FU-SD [38] and/or the involvement of another kinase. See also Fig EV4B. Note that Smaug does not coIP FU unless SMO is present see Appendix Figure S2. (E) Forcing the interaction of Smaug with activated FU leads to a SMO-independent phosphorylation of Smaug. Mean values of the percentage of phosphorylated Smaug in Cl8 cells coexpressing λN-SNAP-Smaug with GFP-FU, FU-SBR, FUEE-SBR or FUDANA-SBR, with or without HH, as indicated. N=3 (biological triplicates). See also Fig EV4C and D. Note that the levels of Smaug phosphorylation observed in presence of FUDANA-SBR are not lower than those seen with GFP-FU. This suggests an incomplete inhibition of endogenous FU, which may be due to the trapping of FUDANA-SBR by Smaug. See a representative blot in Fig EV3C and a phosphatase assay in Fig EV4D. \n",
      ".Figure 1Representative flow cytometry data showing surface phenotypes of DCs sorted from spleens of wild type (WT) or Nlrc3-/- mice and treated with LPS (100 ng/ml) for 48 hours. ELISA of cytokines in culture supernatants of DCs treated The intracellular production of IFN-γ and IL-17 by CD4+ T cells among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated cytokines in culture supernatants among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated thymidine incorporation proliferation assay among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated CFSE proliferation assay among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated \n",
      ".Figure 5Combined median fold-change in RNA (A) levels between NUP58 mutant clones and WT cells (calculated using n=4 RNA-seq and n=3 mass spectrometry datasets). Data were visualized using Cytoscape. White: comparable average expression to WT; red: upregulated relative to WT; blue: downregulated relative to WT. Nodes belonging to the same subcomplex are grouped; edges represent physical interactions Combined median fold-change in protein (B) levels between NUP58 mutant clones and WT cells (calculated using n=4 RNA-seq and n=3 mass spectrometry datasets). Data were visualized using Cytoscape. White: comparable average expression to WT; red: upregulated relative to WT; blue: downregulated relative to WT. Nodes belonging to the same subcomplex are grouped; edges represent physical interactions A stable cell line carrying KPNB1-KPNA4-eGFP and a WT cell line were transfected with eRFP Cas9 plasmid carrying dual gRNA. Cell populations were either treated with doxycycline (doxy +) or left untreated (doxy -) and individually sorted into 384-well plates. A total of 2 plates were prepared for each replicate. Cellular viability expressed as number of viable colonies following Cas9-induced gene editing of NUP58 in WT cell line and in a cell line stably overexpressing KPNB1-KPNA4-eGFP (KAPs). \n",
      ".Figure 4Sashimi plots indicating combined exon usage of NUP58 mRNA in HAP1 WT and mutant clones across four biological replicates. Numbers of reads per exon-exon junction are indicated below introns. WT canonical isoform is colored in black; mutant isoforms in grey. NUP58 gene structure is reported over the sashimi plot; red asterisks and exons highlight the positions of the identified mutations. Western blot for NUP58 protein expression in mutant clones and control cells. Whole-cell lysate hybridized with customized antibody 1 (Ab1, top panel) and antibody 2 (Ab2, bottom panel). Blots were loaded with wild-type HAP1 cell line (WT), empty vector (EV), non-targeting siRNA (siNT) and NUP58 targeting siRNA (siNUP58), followed by NUP58 mutant clones (C1 / C2 / C3 / C4 in left panels, C5 / C6 in right panels). Actin was used as loading control. Asterisks indicate possible NUP58 alternative isoforms. \n",
      ".Figure 7Comparison of NUP58 mRNA abundance between late (P19) and early (P1) passage in wild-type cells (WT, top) and clone C5 (bottom). mRNA was quantified using qPCR and five independent probe sets amplifying different regions of NUP58 mRNA. Annealing of the probe sets is indicated at the bottom of the NUP58 structure. Relative mRNA abundance between P19 and P1 is color-coded for each exon analysed (according to the color bar shown below). Red triangles highlight the position of the identified mutations in C5. Diagram at the top represents G-banding pattern of chromosome 13 and highlights chromosome locations of tested genes (NUP58, GRK1, ADPRHL1). Graph at the bottom reports qPCR results for indicated genes preformed on the gDNA of indicated samples at the indicated passage number. Representative images of NUP58 gene-specific FISH hybridization for control HAP1 cell lines (WT P3-WT P19) and mutant C5 at P19. Each dot represents a NUP58 locus of cluster thereof. Scale bar, 31.3 μm \n",
      ".Figure 6Maximum growth rates of NUP58 mutant clone C5 during long-term culture. Each dot represents a technical replicate (n=22). Improvement in active nuclear import upon cell passaging. Dots represent analysed cells (WT P3 n=16, WT P19 n=25, C5 P1 n=24, C5 P3 n=27, C5 P19 n=23) of one independent experimental replicate performed. Representative images of analysed C5 P1 (left) and C5 P19 (right) NLS-3xmCherry stable cells. Mean intensity of fluorescence for cytoplasm and nucleoplasm for each cell are labelled. Scale bar, 20 μm. Cytoscape visualization of protein abundance changes between C5 P19 and C5 P1. Red nodes indicate proteins that were upregulated upon passaging; blue nodes indicate proteins that were downregulated upon passaging. White nodes display proteins with comparable expression levels at both early and late passage. NUP58 protein expression in C5 at P1 and P19. Blots were performed using Antibody 1 (Ab1, top) or Antibody 2 (Ab2, bottom). Actin was used as loading control. The band corresponding to the full-length NUP58 is indicated by a black arrow with possible alternative isoform by an asterisk. Alternative NUP58 isoform localizes to the NPC. Representative images of HCT116 cell line showing localization of WT NUP58 protein (top WT isoform, z-stack = 10) and of an isoform lacking exon 4 and ending at exon 13 (bottom mutant isoform, z-stack = 11). Scale bar, 20 μm Cellular viability following Cas9-induced gene editing of native NUP58 (left) or NUP85 (right) in WT cell line (black circles) and in a cell line overexpressing NUP58 mutant isoform (grey circles). \n",
      ".Figure 7EAE was induced by immunization of naive B6 mice with MOG (35-55), and the mice were randomly divided into five groups. BMDCs transduced with either lentiviral vector encoding GFP and NLRC3 (LV-NLRC3) or only GFP (LV-Ctrl) were administered i.v. 4 times, once every 4 days, starting at day 10 after EAE induction. Mean clinical scores of EAE (n= 5 mice per group ). Arrows indicate DC vaccine administration. EAE was induced by immunization of naive B6 mice with MOG (35-55), and the mice were randomly divided into five groups. BMDCs transduced with either lentiviral vector encoding GFP and NLRC3 (LV-NLRC3) or only GFP (LV-Ctrl) were administered i.v. 4 times, once every 4 days, starting at day 10 after EAE induction. Effects of therapeutic DC vaccination on B6 EAE. The maximum score: Mean of the maximum scores per mouse in each group. Data shown are the mean ±SD. Determination of statistical differences was performed using by an one-way ANOVA test. EAE was induced by immunization of naive B6 mice with MOG (35-55), and the mice were randomly divided into five groups. BMDCs transduced with either lentiviral vector encoding GFP and NLRC3 (LV-NLRC3) or only GFP (LV-Ctrl) were administered i.v. 4 times, once every 4 days, starting at day 10 after EAE induction. Recall proliferative to MOG (35-55) in splenocytes taken from DCs-treated mice 26 days after EAE induction. EAE was induced by immunization of naive B6 mice with MOG (35-55), and the mice were randomly divided into five groups. BMDCs transduced with either lentiviral vector encoding GFP and NLRC3 (LV-NLRC3) or only GFP (LV-Ctrl) were administered i.v. 4 times, once every 4 days, starting at day 10 after EAE induction. cytokine response (D) to MOG (35-55) in splenocytes taken from DCs-treated mice 26 days after EAE induction. \n",
      ".Figure 2WT and Nlrc3-/- mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Mean clinical scores of EAE in immunized WT (n= 10) and Nlrc3-/- mice (n= 10). Representative flow cytometry data showing intracellular production of IFN-γ and IL-17A by CD4+ T cells from the spinal cord and brain of WT or Nlrc3-/- mice 26 d after EAE induction after restimulation with MOG(35-55) peptide. Pooled data are presented in the right panel. Representative flow cytometry data showing surface phenotypes of DCs from spleens of WT or Nlrc3-/- mice 26 d after EAE induction. Expression of Nlrc3, Il12, Il6, Il23 and Il27 mRNA in DCs sorted from WT and Nlrc3-/- mice 26 d after EAE induction, presented relative to that of Gapdh. \n",
      ".Figure 6DC(Ctrl) and DC(NLRC3-OE) were stimulated with LPS (100 ng/ml) for specified time. DC lysates were probed for p-p38, p38, NLRC3 and GAPDH. Densitometry quantification of band intensity were presented in the right panel. DC(Ctrl) and DC(NLRC3-OE) were stimulated with LPS (100 ng/ml) for specified time. Enzyme-linked immunosorbent assay of cytokines in culture supernatants of DCs treated with LPS for 48 hours. DC(Ctrl) and DC(NLRC3-OE) were stimulated with LPS (100 ng/ml) for specified time. Cytokines in culture supernatants among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated with LPS for 48 hours. DC(Ctrl) and DC(NLRC3-OE) were stimulated with LPS (100 ng/ml) for specified time. CFSE proliferation assay (D) among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs treated with LPS for 48 hours. \n",
      ".Figure 3(A, Effects of the gain- or loss-of-function of Wip1 on TGF-β-induced growth arrest. HEK293T cells were transfected or not with wt Wip1, Wip1(D277A), as indicated and subjected to CCK-8 assays before (0 h) and after stimulation with TGF-β1 (40 ng/ml) (24 h). Data are presented as the mean ± SEM (n=3 biological replicates). *P<0.05, **P<0.01, ***P<0.001 by unpaired Student's t-test. n.s., not significant. B) Effects of the gain- or loss-of-function of Wip1 on TGF-β-induced growth arrest. HEK293T cells were transfected or not with Co siRNA or Wip1 siRNA as indicated and subjected to CCK-8 assays before (0 h) and after stimulation with TGF-β1 (40 ng/ml) (24 h). Data are presented as the mean ± SEM (n=3 biological replicates). *P<0.05, **P<0.01, ***P<0.001 by unpaired Student's t-test. n.s., not significant. Depletion of Wip1 augments TGF-β-induced migration and invasion of MDA-MB231 breast cancer cells. MDA-MB231 cells transfected with Co siRNA or Wip1 siRNA were scratched with a 20-μL pipette tip and then incubated in the absence or presence of TGF-β1 for the indicated times (C, D) Depletion of Wip1 augments TGF-β-induced migration and invasion of MDA-MB231 breast cancer cells. MDA-MB231 cells transfected with Co siRNA or Wip1 siRNA were subjected to transwell assay with or without TGF-β1 for 20 h (E, F). \n",
      " signaling pathway in DC.Figure 4DCs were sorted from spleens of WT or Nlrc3-/- mice. Purified DCs were treated with LPS (100 ng/ml) for specified time. DC lysates were probed for phosphorylated p65 (p-p65), total p65, p-AKT, AKT, p-p38, p38, p-ERK, ERK, p-JNK, JNK and GAPDH. DCs were sorted from spleens of WT or Nlrc3-/- mice. Purified DCs were treated for 48 hours with LPS (100 ng/ml) in the presence or absence of the p38 inhibitor SB203580 (10 μM or indicated concentrations). Concentrations of IL-12, IL-6, IL-23 and IL-27 in supernatants were detected by ELISA. DCs were sorted from spleens of WT or Nlrc3-/- mice. Purified DCs were treated for 48 hours with LPS (100 ng/ml) in the presence or absence of the p38 inhibitor SB203580 (10 μM or indicated concentrations). Cytokines in culture supernatants among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs. NC: negative control. DCs were sorted from spleens of WT or Nlrc3-/- mice. Purified DCs were treated for 48 hours with LPS (100 ng/ml) in the presence or absence of the p38 inhibitor SB203580 (10 μM or indicated concentrations). CFSE proliferation assay (D) among naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs. NC: negative control. \n",
      ".Figure 5Activity phosphorylation of p38 were detected in DCs in the spleens from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction. Pooled data are presented in the right panel. DC(p38-KO) and DC(p38+NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Mean clinical scores of EAE in immunized DC(WT) (n= 5) and DC(NLRC3-KO) mice (n= 5). DC(p38-KO) and DC(p38+NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Frequencies of CD4+ T cells that express IFN-γ and IL-17A in the spinal cord and brain from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction after restimulation with MOG(35-55) peptide. Pooled data are presented in the right panel. DC(p38-KO) and DC(p38+NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Expression of Il12, Il6 and Il23 mRNA in DCs sorted from spleens of DC(WT), DC(NLRC3-KO), DC(p38-KO) and DC(p38+NLRC3-KO) mice 26 d after EAE induction, presented relative to that of Gapdh. \n",
      ".Figure 1Wip1 represses the gene responses induced by Nodal or BMP signaling in animal caps as analyzed by RT-PCR. ODC serves as a loading control. -RT, control in the absence of reverse transcriptase. WE, whole embryo. Co AC, uninjected control animal caps. (-), no injection of wild-type (wt) Wip1 or Wip1(D277A) mRNA. The amount of mRNA injected: wt Wip1 (1 ng), Wip1(D277A) (1 ng), Xnr1 (50 pg) Wip1 represses the gene responses induced by Nodal or BMP signaling in animal caps as analyzed by RT-PCR. ODC serves as a loading control. -RT, control in the absence of reverse transcriptase. WE, whole embryo. Co AC, uninjected control animal caps. (-), no injection of wild-type (wt) Wip1 or Wip1(D277A) mRNA. The amount of mRNA injected: wt Wip1 (1 ng), Wip1(D277A) (1 ng), BMP4 (100 pg). Wip1 represses the gene responses induced by Nodal or BMP signaling in animal caps as analyzed by RT-PCR. ODC serves as a loading control. -RT, control in the absence of reverse transcriptase. WE, whole embryo. Co AC, uninjected control animal caps. (-), no injection of wild-type (wt) Wip1 or Wip1(D277A) mRNA. The amount of mRNA injected: wt Wip1 (1 ng), Wip1(D277A) (1 ng) (D wt Wip1, but not Wip1(D277A), down-regulates the activity of Activin- or BMP-responsive promoters in HEK293T cells. FAST1, were co-transfected as indicated for efficient activation of the promoters. Data are expressed as the mean ± SEM (n=3 biological replicates). *P<0.05 by unpaired Student's t-test. E) wt Wip1, but not Wip1(D277A), down-regulates the activity of Activin- or BMP-responsive promoters in HEK293T cells. Smad1 and Smad4 were co-transfected as indicated for efficient activation of the promoters. Data are expressed as the mean ± SEM (n=3 biological replicates). *P<0.05 by unpaired Student's t-test. (F-Q) Dorsal (F-N) or ventral (O-Q) injection of wt Wip1 abrogates the in vivo expression of mesodermal markers at stage 10.25 as analyzed by in situ hybridization. Embryos are shown in vegetal views with dorsal to the top. Arrowheads denote the absence of expression of mesodermal markers in the injected region of embryos. Control, an embryo injected with LacZ only. The amount of mRNA injected: wt Wip1 (1 ng), Wip1(D277A) (1 ng) and LacZ (100 pg). Scale bar, 150 μm. \n",
      ".Figure 2(A, Nodal-induced gene responses are enhanced further in the absence of Wip1 as analyzed by RT-PCR. Animal caps ere treated with -), no injection of Co MO, Wip1 MO and/or Wip1 mRNA. The dose of reagent injected: Xnr1 (5 pg), Co MO (60 ng), Wip1 MO (60 ng) and Wip1 (1 ng). B) Activin/Nodal-induced gene responses are enhanced further in the absence of Wip1 as analyzed by RT-PCR. Animal caps in (B) were treated with Activin protein (5 ng/ml). (-), no injection of Co MO, Wip1 MO and/or Wip1 mRNA. The dose of reagent injected: Co MO (60 ng), Wip1 MO (60 ng) and Wip1 (1 ng). (C) Knockdown of Wip1 up-regulates the activity of Activin/Nodal-responsive promoter. HEK293T cells transfected with Co siRNA (50 nM) or Wip1 siRNA (50 nM) were stimulated with Activin protein (10 ng/ml) for 10 h and then subjected to luciferase assays. Data are expressed as the mean ± SEM (n=3 biological replicates). ***P<0.001 by unpaired Student's t-test. Depletion of Wip1 promotes epidermal differentiation at the expense of neural fate. (-), no injection of Noggin mRNA. The amount of MO and mRNA injected: Co MO (60 ng), Wip1 MO (60 ng for D, G and J; 30 ng for F and I) and Noggin (5, 10, 20 pg). Scale bar, 150 μm. Depletion of Wip1 promotes epidermal differentiation at the expense of neural fate. Embryos in (E-J) are shown in dorsal views with anterior to the top. The amount of MO and mRNA injected: Co MO (60 ng), Wip1 MO (60 ng for D, G and J; 30 ng for F and I) and Noggin (5, 10, 20 pg). Scale bar, 150 μm. (K, Activin induction of p21 is up-regulated upon Wip1 knockdown. HEK293T cells were transfected with Co siRNA or Wip1 siRNA and 48 h after siRNA transfection, were stimulated with Activin (10 ng/ml) for 3 to 12 h as indicated t=0 in (K) is 48 h after siRNA transfection. P-Smad2 serves as an indicator of active Activin/Nodal signaling. Arrows ndicate non-specific bands. L) TGF-β induction of p21 is up-regulated upon Wip1 knockdown. HEK293T cells were transfected with Co siRNA or Wip1 siRNA and 48 h after siRNA transfection, were stimulated with increasing concentration of TGF-β1 (10, 20 ng/ml) for 3 h. \n",
      ".Figure 3DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Mean clinical scores of EAE in immunized DC(WT) (n= 5) and DC(NLRC3-KO) mice (n= 5). DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Representative flow cytometry data showing intracellular production of IFN-γ and IL-17A by CD4+ T cells in the spinal cord and brain from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction after restimulation with MOG(35-55) peptide. DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Recall response to MOG(35-55) by splenocytes isolated from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction. DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. Expression of Il12, Il6, Il23 and Il27 mRNA in DCs sorted from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction, presented relative to that of Gapdh. DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. CFSE proliferation assay of naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs sorted from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction. DC(WT) and DC(NLRC3-KO) mice were immunized with MOG(35-55) peptide in CFA adjuvant and pertussis toxin to induce EAE. cytokine secretion of naive 2D2 CD4+ T cells stimulated with MOG(35-55) plus DCs sorted from DC(WT) and DC(NLRC3-KO) mice 26 d after EAE induction. \n",
      ".Figure 1(A) Liver with the extrahepatic biliary ducts (EHBDs) and gallbladder (Liver incl. Gb), liver lobes, EHBDs or gallbladder tissues were minced and cultured in Matrigel in the presence of R-spondin 1 and noggin. Phase contrast microscopy images were acquired one week later. Scale bar: 200 m. (B) Liver lobes or gallbladders harvested from 7-day or 1-year old mice were minced and cultured in Matrigel in the presence of R-spondin 1 and noggin. Phase contrast microscopy images were acquired one week later. Scale bar: 200 m. (C) Phase contrast microscopy images of gallbladder organoids cultured for 3, 7 or 10 days in the presence (+) or absence (-) of R-spondin 1 (R) and/or noggin (N), as indicated. The same organoid was photographed at each time point. Scale bar: 200 m. (D) Fraction of EdU-positive (EdU+) cells in organoids cultured for 5 days in media containing R-spondin 1 and/or noggin, as indicated. The results, presented as means and standard errors, are derived from three independent experiments, each examining five organoids per condition. Statistically significant differences: +R+N vs -R-N (P<0.001); +R-N vs -R-N (P<0.05); -R+N vs -R-N (P<0.05). \n",
      ".Figure 4(A-D) Knockdown of Wip1 expands the expression of dorsal mesodermal markers. Embryos injected dorsally with Co MO (40 ng) or Wip1 MO (40 ng) were subjected to in situ hybridization at stage 10.25. Embryos are shown in dorso-vegetal views with dorsal to the top. Scale bar, 150 μm. (E-I) Morphological phenotypes of Wip1 morphants. Embryos were injected radially in the marginal zone with Co MO (80 ng), Wip1 MO (80 ng), wt Wip1 (1 ng) and Wip1(D277A) mRNA as indicated and cultured to tadpole stages. Embryos are shown in lateral views with anterior to the left. Un Co, uninjected control. Scale bar, 1 mm. (J) Quantification of the phenotypes shown in (E-I). Severe defects include microcephaly, shortened and kinked body axis, and no eye. Mild defects indicate normal body axis with malformed eyes. \n",
      ".Figure 2(A) Clusters of gallbladder tissue (GT)-specific, liver tissue (LT)-specific and small intestine tissue (IT)-specific genes (Table EV2). The number of genes and the names of selected genes in each cluster are indicated. Gene expression levels (log2 scale) are shown for gallbladder organoids (GO), liver organoids (LO), intestinal organoids (IO) and gallbladder (GT), liver (LT) and intestinal (IT) tissues. (B) Genes expressed in gallbladder and liver tissues, but not in the intestine, divided into two subclusters, depending on whether the genes were expressed or not in gallbladder organoids (Table EV3). The number of genes and the names of selected genes in each subcluster are indicated. (C) Clusters of organoid (O)-specific and tissue (T)-specific genes (Table EV4). The number of genes and the names of selected genes in each cluster are indicated. (D) Expression levels of selected genes from the clusters shown in panels B and C. G, gallbladder; L, liver; I, intestine; O, organoid; T, tissue. \n",
      ".Figure 3Expression of Prom1, Cldn3, Epcam, Itga6 and Yap1 in gallbladder-derived organoids and liver tissue, as monitored by immunofluorescence (IF). The cell nuclei were counterstained with DAPI. Scale bars: 25 m. \n",
      ".Figure 7(A-J) Embryos injected dorsally with wt Wip1 (1 ng), wt Smad4 (1 ng), Smad4(T277A) (1 ng) and Smad4 (T277E) (1 ng) as denoted were subjected to in situ hybridization against Chordin or Xbra at stage 10.25. Control, control embryo injected with LacZ mRNA only. Scale bar, 150 μm. (K) Quantification of the results from in situ hybridization analysis in (A-J). (L) Wip1-mediated neural induction is inhibited by Smad4 with a phosphorylatable or phospho-mimetic amino acid at the MAPK site. (-), no co-injection of wt or mutated Smad4. (M) MDA-MB231 and MDA-MB468 cells were transfected with wt Wip1 alone or with wt Smad4 or Smad4(T277A) and then subjected to CCK-8 assays to measure cell growth. Data are presented as the mean ± SEM (n=3 biological replicates). *P<0.05 by unpaired Student's t-test. (N) Anchorage-independent colony-formation assays were performed in MDA-MB231 and MDA-MB468 cells transfected with Wip1 or negative control constructs. Scale bar, 100 μm. (O) Quantification of the assays shown in (N). The number of colonies larger than 50 μm in diameter was scored. Data are expressed as the mean ± SEM (n=3 biological replicates). *P<0.05 by unpaired Student's t-test. (P) Proposed model of the mechanism by which Wip1 controls the transcriptional activity and stability of Smad4. See the discussion for details. \n",
      ".Figure 5(A Wip1 and Smad4 form an endogenous protein complex. Cell lysates of HEK293T cells, which were stimulated or not with Activin (10 ng/ml), TGF-β1 (40 ng/ml) or BMP4 (20 ng/ml) for 1 h, were immunoprecipitated with anti-Smad4 antibody followed by western blotting with anti-Wip1 antibody (A) (-), no treatment with protein ligand. IgG, control IgG. An arrow in (A) indicates non-specific bands. B) Wip1 and Smad4 form an endogenous protein complex. Cell lysates of HEK293T cells, which were stimulated or not with Activin (10 ng/ml), TGF-β1 (40 ng/ml) or BMP4 (20 ng/ml) for 1 h, were immunoprecipitated with anti-Smad4 antibody followed by western blotting with anti-Wip1 antibody (A) and vice versa (B). (-), no treatment with protein ligand. IgG, control IgG. (C) Upon ligand stimulation, Wip1 forms a complex with Smad4 in the nucleus. HeLa cells were treated or not with TGF-β1 (40 ng/ml) for 2 h, fixed and then subjected to proximity ligation assays with primary antibodies against Wip1 and Smad4. Blue: DAPI; Red: PLA signal. Scale bar, 20 μm. (D) Quantitative analysis of Wip1/Smad4 interactions shown in (C). Data are represented as the mean ± SEM (n=3 biological replicates). Twenty cells were analyzed per sample. ***P<0.001 by Student's t-test. (E) Wip1 interferes with the ability of Smad4 to bind to Smad2. HEK293T cells co-transfected with the indicated constructs were treated or not with TGF-β1 for 1 h and then harvested. An arrow denotes Myc-tagged Wip1 protein. (F) Wip1 has a negative effect on nuclear accumulation of Smad4. HEK293T cells were transfected with Co siRNA or Wip1 siRNA, treated with TGF-β1 (40 ng/ml, 1 h) 48 h after siRNA transfection and subjected to nuclear/cytoplasmic fractionation and western blotting analysis. β-tubulin and lamin A/C serve as cytoplasmic and nuclear markers, respectively. C, cytoplasm; N, nucleus. The experiment was repeated two times with similar results. (G) Quantification of the levels of nuclear Smad4 normalized to those of cytoplasmic Smad4 shown in (F). \n",
      ".Figure 6(A) Overexpression of Wip1 down-regulates Smad4 phosphorylation at Thr277. MOCK- or Flag-Wip1-transfected HEK293T cells were treated with 1 h pulse of FGF2 (10 ng/ml), followed by addition of U0126 (40 μM) and harvesting for western blotting at the indicated times. - FGF2, no treatment with FGF2. (B) Time course of Smad4 phosphorylation at Thr277 induced by FGF2 in control and Wip1-depleted HEK293T cells, showing that knockdown of Wip1 up-regulates Smad4 phosphorylation at the MAPK site. A bracket denotes phospho-Smad4. (C) Wip1 directly dephosphorylates Smad4 at Thr277 as assayed by in vitro phosphatase assay. Myc-tagged Smad4 was expressed in HEK293T cells, which were then treated or not with FGF2 (40 ng/ml, 2 h), and the Myc-Smad4 was immunoprecipitated using anti-Myc antibody and subsequently incubated with or without recombinant human Wip1 protein (0.5 μg) in the absence or presence of Mg2+ as indicated. (D) FGF-induced destabilization of Smad4 is enhanced further in Wip1-depleted animal cap cells as analyzed by western blotting. The amount of reagent injected: Myc-Smad4 (400 pg), FGF8 (100 pg), Wnt8 (400 pg), Co MO (60 ng) and Wip1 MO (60 ng). (E) Polyubiquitination of wt Smad4 is promoted in Wip1-silenced cells. HEK293T cells were transfected with the indicated combinations of Flag-ubiquitin, Co siRNA, Wip1 siRNA, Myc-Smad4 and Myc-Smad4(T277A) and then treated with FGF2 (10 ng/ml, 4 h) in the presence of the proteasome inhibitor MG132, and Myc-Smad4 and its phosphorylation-resistant mutant were immunoprecipitated with anti-Myc antibody. Ubiquitin-conjugated Smad4 (Smad4-Ubn) was detected by western blotting with anti-Flag antibody. (F) Wip1 prolongs the half-life of Smad4. HEK293T cells were transfected with Co siRNA or Wip1 siRNA and then treated with FGF2 along with cycloheximide (CHX, 100 ng/ml) for the indicated times and harvested for western blotting. An arrow indicates non-specific bands. (G) Diagram of the post-translational Smad4 modifications by modifying enzymes and their downstream functions. P, phosphorylation; Ub, monoubiquitination; Me, methylation. \n",
      ".Figure 4(A) Immunofluorescence analysis of expression of Hnf4α, A6, Sox17 and Krt19. The nuclei were counterstained with DAPI. EHBD, extrahepatic biliary duct.(B) Higher magnification of the regions demarcated by the white rectangles in panel A. Scale bar: 25 m. \n",
      ".Figure 3Tumour diameter before and after doxycycline withdrawal in 4 K and 5 KM breast tumours with cMet amplification. 0 indicates when doxycycline was removed. Each colour represents one tumour. Blue and green squares indicate the timeframe between doxycycline withdrawal and the moment in which tumours resumed growth. Genome-wide log2-ratio plots of chromosome 6 of two primary tumour biopsies and their corresponding non-regressed tumour showing no amplification in the primary tumour (upper panels) and a small amplification in the non-regressed tumours (bottom panels, yellow arrow). Immunostaining of phospho-cMet in 3 biopsied primary tumours (PT) and their corresponding non-regressed tumours (KM5, KM7 and KM8, which are also shown in Fig 2C). Scale bar 100 µm. Representative two-dimensional scatter plots constructed with overlaid dPCR data of the reference (VIC) and cMet (FAM) from one tumour without cMet amplification and one with cMet amplified. Dots represent results of independent PCR reactions in the wells of a digital PCR chip. Reactions in the bottom left corner (yellow) are negative for both targets, while the ones in the top right corner (green) are double positives. Reactions in the top left (blue) and bottom right (red) corners are positive for cMet and the reference targets, respectively. Representative photographs of FISH staining with a probe for Met (red signal) and a probe for a reference gene EML4 (green signals) The upper panel is a negative example for cMet amplification containing 2 red and 2 green dots (white arrows). The lower panel shows an example of a tumour with cMet amplification (several red dots) and 2 green dots (yellow arrow). Scale bar 10 µm. \n",
      ".Figure 1Representative micrographs of a time-lapse microscopy of K and KM non-regressed tumour cells (H2B-GFP green). Top: Mitotic cell with a chromatin bridge (yellow arrow). Bottom: Mitotic cell with misalignment and cytokinesis failure resulting in binucleation (yellow arrow). Percentage of mitotic errors in K and KM primary tumours and in K and KM non-regressed tumours. Percentage of cells in K and KM primary tumours and in K and KM non-regressed tumours with the indicated mitotic errors. Scale bar 20 µm. \n",
      ".Figure 6(A) (Top) Low magnification view showing organoids embedded in Matrigel (left) next to the liver parenchyma (right) two weeks after engraftment. H&amp;amp;E, hematoxylin eosin staining. (Bottom) Immunofluorescence staining for GFP expression to mark the engrafted organoids and DAPI counterstaining of the cell nuclei. The liver parenchyma corresponds to the right half of the images. Scale bar: 200 m. (B) Characterization of engrafted organoids by immunofluorescence (IF). Almost no organoid cell expressed Ki67 (the one positive cell shown in the image was an exception), in contrast to the high levels of cell proliferation and Ki67 expression observed in organoids cultured in vitro. Scale bar: 25 m. \n",
      ".Figure 5(A) Clusters of genes, whose expression changes towards the hepatocyte fate, upon removal of R-spondin 1 and noggin from the organoid culture media (Table EV5). The number of genes and the names of selected genes in each cluster are indicated. Gene expression levels (log2 scale) are shown for non-differentiated (N) and differentiated (D) gallbladder-derived organoids (GO), gallbladder tissue (GT) and liver tissue (LT). Upreg, upregulated; Downreg, downregulated. (B) Expression levels of selected genes from the clusters of panel A. GO, gallbladder organoid; N, non-differentiated; D, differentiated; G, gallbladder; L, liver; T, tissue. (C) Removal of R-spondin 1 and noggin from the media induces expression of red fluorescent protein (RFP) in a subset of gallbladder organoid cells. A transgene expressing tamoxifen-inducible Cre is driven by the endogenous Alb promoter; a second transgene expresses RFP after Cre-mediated rearrangement. Scale bar: 200 m. (D) Glycogen accumulation upon organoid differentiation, as revealed by periodic acid-Schiff (PAS) staining (red). Scale bar: 25 m. \n",
      ".Figure 2Somatic copy number alterations in each individual Kras and Kras/Mad2 non-regressed tumours. 7 K non-regressed and 16 KM non-regressed tumours were sequenced. Focal deletion (DEL, green), Focal amplification (AMP, light green), whole chromosome gain (WCG, dark grey) and loss (WCL, light grey), partial chromosome gain (PCG, dark blue) and loss (PCL, light blue) and gross chromosomal rearrangement (GCR, red). Overlaying of the amplifications in chromosome 6 across multiple regions showing that all amplicons contained the cMet oncogene. Colour of the block corresponds to the segment mean (the degree to which a genome segment is lost (blue) or gained (red). Immunostaining of phospho-cMet in non-regressed tumours carrying or not the amplification in chromosome 6. Yellow numbers indicate the total number of cMet positive cells/total number of cells counted. Scale bar 100 µm. \n",
      ".Figure 4Schematic of the experiment. K and KM animals were set on doxycycline food until mammary tumours developed. Primary tumours were collected and single cells injected into Rag2-/- animals to recapitulate the tumours followed by either no treatment, drug treatment or switched to a normal diet. Relative volume of tumours grown in 12 Rag2-/- animals after treatment with Tepotinib or vehicle control (3K and 3KM tumours for each condition). No statistical significance was found by One-way ANOVA. Mean +/- SEM. Quantification of cMet copy number detected by digital PCR or FISH in primary tumours injected into Rag2-/- and treated with vehicle control or Tepotinib. Quantification of cMet copy number detected by digital PCR or FISH in primary tumours injected into Rag2-/- and fed with doxycycline or after doxycycline withdrawal. \n",
      ": Effect of TMPRSS2 and IFITM on S- or ACE2-expressing cells.Figure 4A. S-expressing cells (Donor cells), ACE2-expressing cells (Acceptor cells) were co-transfected with TMPRSS2, IFITM or control plasmids. Cell fusion was quantified by measuring the GFP+ area after 18h. The indicated combinations were tested. The impact of IFITMs was measured in absence of TMPRSS2 (left panel), in presence of TMPRSS2 in donor (middle panel) or acceptor cells (right panel). Data are mean±sd of 4 independent experiments. Statistical analysis: One-Way ANOVA, ns: non-significant, ∗∗p < 0.01, ∗∗∗∗p < 0.0001. Impact of TMPRSS2 on IFITMs levels. B. TMPRSS2 does not decrease IFITM amounts measured by flow cytometry. 293T cells were transfected with the indicated IFITM plasmids, with or without TMPRSS2, and analyzed 18h later. C. Impact of TMPRSS2 on S and ACE2, measured by Western Blot. 293T cells were transfected with or without IFITM1, TMPRSS2, S or ACE2 plasmids, and analyzed 18h later. Impact of TMPRSS2 on S levels. D. TMPRSS2 decreases S surface levels, measured by flow cytometry. 293T cells were transfected with S plasmid, with or without TMPRSS2, and analyzed 18h later. Murine polyclonal anti-S antibodies, one serum from a convalescent COVID-19 patient (sera n°111), and two monoclonal antibodies (anti-S1 and anti-S2) were tested. Data are representative of three independent experiments. \n",
      ".Figure 5Schematic of the experiment. K and KM animals were set on doxycycline food until mammary tumours developed. Then switched to a normal diet and non-regressed tumours collected. Single cells were then injected into Rag2-/- animals to recapitulate the non-regressed tumours followed by drug treatment or vehicle control treatment. Relative volume of tumours grown in Rag2-/- animals after treatment with Tepotinib or vehicle control. One-way ANOVA, Tukey's multiple comparison. Ns: not significant; ** p<0.024. Mean +/- SEM. A total of 19 Rag2-/- mice injected with cMet expressing tumours were treated with Tepotinib and 14 used as controls. For tumours with no cMet, 13 Rag2-/- mice were treated with Tepotinib and 9 were used as control. Immunostaining of phospho-cMet in the same tumours grown in Rag2-/- mice after treatment with Tepotinib or vehicle control. Scale bar 200 µm. Quantification of pH3 in tumours grown in Rag2-/- animals after 5 days treatment with the cMet inhibitor Tepotinib or vehicle control. Quantification of caspase 3 in tumours grown in Rag2-/- animals after 5 days treatment with the cMet inhibitor Tepotinib or vehicle control. \n",
      ": Impact of TMPRSS2 and IFITMs on the kinetics of fusion by S-expressing 293T cells.Figure 3A. 293T-GFP1-10 and -GFP11 cells (1:1 ratio) were co-transfected with S, ACE2, TMPRSS2, IFITM or control plasmids. Cell fusion was quantified by measuring the GFP+ area by high-content imaging after 18h. (B and C) or analyzed over time by video microscopy (D and E). B. Representative images of cell-cell fusion. Scale bar: 100 µm. C. Quantification of GFP+ areas. Results are mean±sd from 5 independent experiments. D. TMPRSS2 accelerates fusion. Cells were monitored by video microscopy and the GFP area was quantified over time. Left panel: one representative experiment. Results are mean±sd from 3 fields per condition. Right panel: Mean±sd from 7 independent experiments (at 12h post transfection). E. Impact of TMPRSS2 and IFITMs on the kinetics of fusion by S-expressing 293T cells. One representative out of three independent experiment is shown. \n",
      ".Figure 2Cells were infected at the indicated multiplicity of infection (MOI) and analyzed after 20h. A. TMPRSS2 increases fusion and cell mortality. Right panel: Areas of GFP+ cells and PI+ cells Cells were infected at the indicated multiplicity of infection (MOI) and analyzed after 20h. B. IFITM1, but not IFITM2 and 3, inhibits SARS-CoV-2-induced syncytia formation. Right panels: Area of GFP+ cells Cells were infected at the indicated multiplicity of infection (MOI) and analyzed after 20h. C. The fusion of IFITM1+ cells with U2OS-ACE2 syncytia is drastically reduced. Right panel: Area of IFITM+ GFP+ cells Cells were infected at the indicated multiplicity of infection (MOI) and analyzed after 20h. D. IFITM1 decreases the number of infected cells. Right panel: Fraction of cells positive for S, normalized to control cells (transduced with pQCXIP-empty vector ). \n",
      ".Figure 1A. GFP-Split U2OS-ACE2 were co-cultured at a 1:1 ratio and infected with SARS-CoV-2. Syncytia formation and cell death was monitored by video microscopy or at endpoint using confocal microscopy and high content imaging. B. Still images of GFP (syncytia) and Propidium Iodide (PI) (cell death) at different time-points. Scale bar: 100 µm. C. Quantification of U2OS-ACE2 fusion and death by time-lapse microscopy. Results are mean±sd from 3 fields per condition. D. S staining of infected U2OS-ACE2 cells analyzed by immunofluorescence. The Hoechst dye stains the nuclei. Scale bar: 40 µm. E. Surface S staining of infected U2OS-ACE2 cells analyzed by flow cytometry. Results are representative of at least three independent experiments. \n",
      ".Figure 6T22-GFP-H6-FdU prevents metastases in the CXCR4+ patient-derived M5 model by potently reducing the total and mean number of liver, lung and peritoneal mets, as recorded in H&E stained histology sections at the end of treatment, in comparison to free oligo-FdU o Buffer-treatment. In contrast, the number of LN mets is not reduced after T22-GFP-H6-FdU or free oligo-FdU administration (N=6 mice per Buffer group; N=7 mice per Free oligo-FdU group and N=8 mice per T22-GFP-H6-FdU group; 3 samples/mouse). Data expressed as mean±s.e.m. T22-GFP-H6-FdU induces a higher reduction of CXCR4+ cancer cell fraction (CXCR4+ CCF) in liver, lung and peritoneal metastatic tissue, at the end of treatment, than free oligo-FdU, as measured by anti-CXCR4 IHC. In contrast, T22-GFP-H6-FdU or free oligo-FdU does not reduce the CXCR4+ CCF in LN mets or primary tumor tissue after therapy. (N=6 mice per Buffer group; N=7 mice per Free oligo-FdU group and N=8 mice per T22-GFP-H6-FdU group; 3 samples/mouse). Data expressed as mean±s.e.m Representative CXCR4 IHC images of the reduction in CXCR4+ CCF induced by T22-GFP-H6-FdU (or its absence in free oligo-FdU mice) at the end of treatment, in the M5 patient-derived CRC model, which quantitation is reported in panel B. In the M5 model, the highest reduction in foci number and size occurs in liver metastases, which show the highest reduction in CXCR4+ CCF. \n",
      ".Figure 3Representative images of CXCR4 overexpression in subcutaneous tumor tissue, showing similar CXCR4 levels among compared groups (N=5/group; Buffer, T22-GFP-H6-FdU, T22-GFP-H6 and free oligo-FdU) before treatment (upper panels). Representative images of DNA double-strand break induction and caspase 3 activation (measured with anti-ɣH2AX or anti-cleaved caspase 3 by IHC) 5h post-administration (middle panels). Apoptotic induction (Hoescht staining, 24h post-administration, lower panels). Note the higher number of cells positive for DSBs, Caspase 3 activation and apoptosis induction in the T22-GFP-H6-FdU as compared to free oligo-FdU. Black or white arrows indicate dead cells. Quantitation of the number of cells containing DSBs or active Caspase-3 in IHC-stained tumor sections 5h post-treatment and the number of condensated or disaggregated nuclei (by Hoescht staining) 24h post-treatment in tumor sections of 10 high-powder fields (400x magnification) using the Cell D software. (N=50; 10 tumor fields/mice; 5 mice/group). Data expressed as mean±s.e.m. \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ".Figure 4T22-GFP-H6-FdU depletes CXCR4+ cancer cells from SW1417 CRC tumor tissue after a 100 µg single dose administration. Note the reduction in CXCR4+ cell fraction in the tumor 24h after injection, their almost complete elimination at 48h and the re-emergence of CXCR4+ cells 72h post-administration, using anti-CXCR4 IHC. In contrast, the CXCR4+ cancer cell fraction (CXCR4+ CCF) in tumor tissue remains constant along time after free oligo-FdU treatment. The three day time-lapse for CXCR4+ tumor cell re-appearance defines the dosage interval used in a repeated dose schedule of nanoconjugate administration in the experiments to evaluate its antimetastatic effect. (N=5: 5 mice/group; 1 samples/mouse). Scale bar, 50 µm. Data expressed as mean±s.e.m Significant reduction in the number of spheroid formed (C. optical microscope) and their bioluminescence emission (D, IVIS Spectrum 200), generated by 1x106 disaggregated cells (cultured in stem cell conditioned media and low-adhesion plates), obtained from CXCR4+ luciferase+ SW1417 subcutaneous tumors, 24h after 100µg T22-GFP-H6-FdU intravenous doses, for 2 consecutive days, as compared to Buffer-treated or free oligo-FdU-treated mice. D. Quantitation of the bioluminescent signal (BLI) expressed as Average Radiant Intensity, obtained using the IVIS spectrum 200 equipment (N=2 plates/group). \n",
      ".Figure 5Reduction in the number of formed spheroids (optical microscope) generated by 1x106 disaggregated cells (cultured in stem cell conditioned media and low-adhesion plates) obtained from CXCR4+ M5 subcutaneous tumors, 24h after 100µg T22-GFP-H6-FdU intravenous doses, for 2 consecutive days, as compared to Buffer-treated or free oligo-FdU-treated mice. (N=8; 2 mice/group; 4 plates/mouse). (Scale bar, 100µm). Reduction in tumor re-initiation capacity after subcutaneous inoculation of 5x106 cells in NSG mice (N=4 tumors/group) derived from disaggregated tumor cells obtained from SC tumors 10 days after administration of 100µg T22-GFP-H6-FdU intravenous doses, for 2 consecutive days , as compared to Free oligo-FdU-treated or Buffer-treated mice. Recording of the number and size of positive tumors. (N=4; 2 mice/group, 2 injection points/mouse). T22-GFP-H6-FdU-induced reduction in the number of intravasated tumor emboli in peri-tumoral vessels of the M5-orthotopic primary tumor (E. optical images; F. emboli number quantitation) and reduction of CXCR4 expression in these emboli (G), treated 7 days after tumor cell implantation with 100µg T22-GFP-H6-FdU intravenous doses, for 2 consecutive days, as compared to Buffer-treated or free oligo-FdU-treated mice. Tumor emboli counting in 10 high powder field at 200x magnification in H&E stained sections from each tumor. (N=5/group). \n",
      ".Figure 2Approach to achieve targeted drug delivery and selective killing of metastatic stem cells: CXCR4-nanoconjugate interaction triggers CXCR4-mediated internalization in MetSCs, in primary tumors and metastatic foci, followed by FdU release to the cytosol and diffusion to the nucleus to induce double stand breaks leading to selective killing of CXCR4+ cells. Selective T22-GFP-H6-FdU nanoconjugate biodistribution in subcutaneous CXCR4+ SW1417 CRC tumor tissue 5h after a 100µg single intravenous dose, as measured by fluorescence emission using IVIS Spectrum 200 (N=5/group). Biodistribution is similar to that achieved by the T22-GFP-H6 targeting vector and undetectable after Buffer or free oligo-FdU treatment. (N=5 mice/group) Co-localization (yellow merged) of the T22-GFP-H6-FdU (green) and the CXCR4 receptor (red) and release of T22-GFP-H6-FdU into the cytosol in CXCR4+ tumor cells 5h after a 100ug dose of nanoconjugate, as measured by dual anti-GFP/anti-CXCR4 immunofluorescence (IF). DAPI (blue nuclear staining). Fluorescence emission was measured in the green and red channels using the Image J software and expressed as mean Area (A)±s.e.m (µm2) (N=10, 2 tumor fields x 5 mice; 200X). Note the significant (P=0.003) increase in the area occupied by the green dots (nanoconjugate realeased to the cell cytosol) in T22-GFP-H6-FdU-treated tumors, compared to free oligo-FdU-treated control tissues. Scale bar, 50 µm. Administration of the CXCR4 antagonist AMD3100 completely blocks T22-GFP-H6-FdU tumor biodistribution, as measured by fluorescence emission. Fluorescence is not detected in Buffer or free oligo-FdU controls (N=5 tumor fields/group). The uptake of T22-GFP-H6-FdU observed in CXCR4+ SW1417 tumor tissues is almost completely blocked by prior AMD3100 administration, as quantified using the anti-GFP IHC H-score. (N=5 tumor fields/group). Representative images of T22-GFP-H6-FdU uptake and AMD3100 competition by anti-GFP inmunostaining, which quantitation is reported in panel E. Scale bar, 50 µm \n",
      ".Figure 1The nanoconjugate contains a fusion protein [T22-GFP-H6 -composed of the peptide T22 as a CXCR4 ligand, a green fluorescent protein and a Histidine tail - bound to the payload drug Three to four pentameric oligonucleotides (approximately 20 molecules) of the antitumor drug 5-Fluoro-2'-deoxyUridine (FdU), named oligo-FdU, are conjugated to the T22-GFP-H6 targeting vector using a linker. T22-GFP-H6-FdU chemical synthesis: T22-GFP-H6 is first covalently bound to the 6-Maleimidohexanoic acid N-hydroxysuccinimide ester linker through its amino groups in the external lysines (Hermanson et al, 2013). The thiol-functionalized oligo-FdU (oligo-(FdU) 5-SH) (see Appendix Fig S1) is then reacted with T22-GFP-H6 functionalized with maleimide (Michael reaction). High and constitutive expression of CXCR4 in the membrane of SW1417 CRC cells as measured by flow cytometry. Lack of human SDF-1α release from cultured SW1417 CRC cells, as measured by ELISA, whereas human control 1BR3.G fibroblasts express high SDF-1α levels, after 48h or 72h of growth in culture. (N=2 experiment in duplicate) Nanoconjugate internalization in CXCR4 overexpressing (CXCR4+) SW1417 CRC cells after 1 hour exposure at 1µM, as measured by fluorescence emission using flow cytometry. (N=3 experiments in duplicate) Significant difference at p= 0.002 between T22-GFP-H6-FdU and AMD3100+T22-GFP-H6-FdU groups Intracellular trafficking of T22-GFP-H6-FdU in CXCR4+ SW1417 cells by confocal microscopy after exposure at 1µM for 24 h. The green staining corresponds to GFP containing nanoconjugates and the red staining corresponds to plasma cell membranes stained with a red dye (CellMask™), whereas cell nucleus was stained in blue with hoescht. The insets show detail of the intracellular localization of nanostructured, fluorescent entities, in an isosurface representation within a three-dimensional volumetric x-y-z data field Linearized T22-GFP-H6-FdU dose-response trend line representation compared with unconjugated free oligo-FdU exposure. Antitumor effect was measured as CXCR4+ SW1417 cell viability by MTT after 72 h exposure as the described concentrations. (N=3 experiments in duplicate) Reduction of cell viability determined by optical microscope images of SW1417 cells exposed to 1µM T22-GFP-H6-FdU for 48 h, as compared to T22-GFP-H6 or free oligo-FdU. (N=3 experiments in duplicate; Scale bar, 100µm). \n",
      "Sodium butyrate induced endoplasmic reticulum stress in colorectal cancer cells.Fig 5  HCT-116 (A) and HT-29 (B) cells were treated with the indicated concentrations of sodium butyrate (NaB) for 24 h. Representative Western blots show the expression of IRE-1a, BIP, PDI, and CHOP. GAPDH was used as the loading control. \n",
      "Sodium butyrate induced autophagy in colorectal cancer cells.Fig 2HCT-116 (A, C) and HT-29 (B, D) cells were treated with 2mM sodium butyrate (NaB) for 24 h and then stained with acridine orange (AO; A-B) or monodansylcadaverine (MDC; C-D). Flow cytometry measured the percentage of AO+ cells and the results of three independent AO fluorescence experiments for HTC-116 (E) and HT-29 (F) cells were expressed as mean ± SD. One-way ANOVA was used to compare control cells and NaB treatments. *p<0.05, ** p<0.01 when compared to control. \n",
      "Sodium butyrate inhibited proliferation and induced apoptosis in colorectal cancer cells.Fig 1HCT-116 (A) and HT-29 (B) cells were treated with the indicated concentrations of sodium butyrate (NaB) for 24 (grey bars), 48 (pink bars), or 72 (yellow bars) hours and cell proliferation was assessed. HCT-116 (C, E) and HT-29 (D, F) cells were treated with the indicated concentrations of NaB for 24 h. C.D. The percentage of apoptotic cells in HCT-116 (C) and HT-29 (D) cells was quantified in three independent experiments using an annexinV/PI assay (annexinV+/PI-) by flow cytometry and expressed as mean ± SD. One-way ANOVA was used to compare between the control cells and NaB treatments. *p<0.05, ** p<0.01 compared to control. E.F. Representative Western blots of full-length and cleaved PARP products in HCT-116 (E) and HT-29 (F) cells. \n",
      "Sodium butyrate increased expression of LC3-II.Fig 3HCT-116 (A) or HT-29 (B) cells were treated with the indicated concentrations of sodium butyrate (NaB) for 24 h. Representative blots showed expression of Beclin1, ATG3, and LC3 as well as GAPDH, the loading control. C. Electron microscopy images of control and NaB (2mM) treated HCT-116 cells. ER, endoplasmic reticulum; Mt, mitochondrion; AL. \n",
      "3-MA and chloroquine blocked sodium butyrate induced autophagy in colorectal cancer cells.Fig 4HCT-116 or HT-29 cells were treated with 5mM 3-MA or 5μM chloroquine (CQ) for 30 min, and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of LC3-I and LC3-II expression were quantified by densitometry and normalized to GAPDH (ratio of LC3:GAPDH). The fold change from control is indicated as the mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis. ** p<0.01, compared to the respective control group. (A) HCT-116 cells treated with 3-MA and NaB; HCT-116 or HT-29 cells were treated with 5mM 3-MA or 5μM chloroquine (CQ) for 30 min, and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of LC3-I and LC3-II expression were quantified by densitometry and normalized to GAPDH (ratio of LC3:GAPDH). The fold change from control is indicated as the mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis. ** p<0.01, compared to the respective control group. (B) HT-29 cells treated with 3-MA and NaB HCT-116 or HT-29 cells were treated with 5mM 3-MA or 5μM chloroquine (CQ) for 30 min, and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of null and LC3-II expression were quantified by densitometry and normalized to GAPDH (ratio of LC3:GAPDH). The fold change from control is indicated as the mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis. ** p<0.01, compared to the respective control group. (C) HCT-116 cells treated with CQ and NaB; HCT-116 or HT-29 cells were treated with 5mM 3-MA or 5μM chloroquine (CQ) for 30 min, and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of LC3-I and LC3-II expression were quantified by densitometry and normalized to GAPDH (ratio of LC3:GAPDH). The fold change from control is indicated as the mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis. ** p<0.01, compared to the respective control group.(D) HT-20 cells treated with CQ and NaB. \n",
      ".Figure 7Undetectable T22-GFP-H6-FdU emitted fluorescence in normal tissues, except for a transient accumulation 5h after a 100μg dose in the liver, which disappears at 24 h. Liver emitted fluorescence is transient and significantly lower than the one registered in tumor tissue. Tumor/Liver ratio= 7.5 (see tumor intensity in Fig 2B, which was registered in the same experiment) (N=5 mice/group). Scale bar, 1 cm. Representative images depicting the level of DNA double strand break (DSB) induction in histologically normal bone marrow 5h after treatment, as measured by anti-ɣ-H2AX, which is higher in free oligo-FdU-treated mice than in T22-GFP-H6-FdU (P=0.047). Low level of cells containing DSBs in histologically normal kidney after T22-GFP-H6-FdU or free oligo-FdU treatment, a finding occurring in all normal tissues analysed. (N=50, 5 mice/group; 10 fields/mouse). Representative images showing lack of histopathological alterations in H&E stained tissue or apoptotic induction in H&E stained samples of CXCR4+ (bone marrow) and CXCR4- (brain, kidney, liver, lung and heart) normal tissues 24h after the administration of a 100 μg dose of T22-GFP-H6-FdU or an equimolecular dose of free oligo-FdU (N=5/group). Note that the transient nanoconjugate distribution to liver or the DNA damage induced in bone marrow do not lead to cytoxicity on these no-tumor tissues. (N=50, 5 mice/group; 10 fields/mouse) Lack of differences in body weight among groups registered along time in the SW1417-derived CCR model and the regression of metastases protocol (N=10 mice/group). Lack of differences in body mouse weight among groups registered along time in the SW1417 cell line-derived model and the prevention of metastasis protocol (Buffer (N=11; Free oligo-FdU (N=12), T22-GFP-H6-FdU (N=12)). Lack of differences in body mouse weight among groups registered along time in the M5 patient-derived model and the prevention of metastasis protocol (Buffer (N=6); Free oligo-FdU (N=17); T22-GFP-H6-FdU (N=8). Scale bar, 100 μm \n",
      ".Figure 1(A) Representative transmission electron microscope (TEM) image showing the presence of EVs in the CSF in two independent experiments. (B) NanoSightquantification of the amount of particles in the CSF 0, 1, 2, 4 and 6 hours after i.p. LPS injection (n=3-5). (C) Size distribution of the EVsin vivo in the CSF before (black; n=5) and 6 h after (grey; n=3) LPS treatment determined by NanoSight analysis. (D-G) Quantitative real-time polymerase chain reaction analysis of miR-1a (D), miR-9 (E), miR-146a (F), and miR-155 (G) (n=4). RNA was isolated from pooled CSF (50 µl) from different mice (n=3). Data are displayed as mean ± SEM and analyzed by Student's t-test. Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05; **, 0.001 ≤ P < 0.01. \n",
      "Mithramycin blocked sodium butyrate induced endoplasmic reticulum stress in colorectal cancer cells.Fig 7  HCT-116 (A, B) or HT-29 (C,D) cells were treated with 0.1μM mithramycin for 30 min followed by sodium butyrate (NaB) for 24 h. Representative Western blots showing the expression of IRE-1a, BIP, and PDI in HCT-116 (A) or HT-29 (C) cells are shown. GAPDH was used as loading control. Protein expression (IRE-1a, BIP, and PDI) was quantified by densitometry and normalized to GAPDH (ratio of protein:GAPDH). The fold change from control for each protein is expressed as mean ± SD of three independent experiments for HCT-116 (B) and HT-29 (D) cells. One-way ANOVA was used for statistical analysis. * P<0.05, ** p<0.01, compared to the respective control group. \n",
      "Cycloheximide blocked sodium butyrate induced endoplasmic reticulum stress in colorectal cancer cells.Fig 6  HCT-116 or HT-29 cells were treated with 10 μg/mL cycloheximide for 30 min followed by sodium butyrate (NaB) for 24 h. Representative Western blots showed the expression of IRE-1a, BIP, PDI, and GADPH (loading control) in HCT-116 (A) or HT-29 (C) cells. Protein expression was quantified by densitometry and normalized to GAPDH (ratio of protein:GAPDH). The fold change from control for each protein is expressed as mean ± SD of three independent experiments for HCT-116 (B) and HT-29 (D) cells. are shown. One-way ANOVA was used for statistical analysis. * P<0.05, ** p<0.01, compared to the respective control group. \n",
      "Inhibition of autophagy potentiates sodium butyrate-induced apoptotic cell death.Fig 10  HCT-116 (A) and HT-29 (B) cells were exposed to 5mM 3-MA or 5μM chloroquine (CQ) for 30 min and then treated with 2mM sodium butyrate (NaB) for 24 h. The percentage of apoptotic cells (annexinV+/PI-) was quantified by flow cytometry using an annexinV/PI assay and expressed as mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis to compare control cells and NaB treatments. *p<0.05, ** p<0.01 compared to control. (C.D.) Cells were treated as described above and the proteins were harvested for Western blot analysis. Expression of full-length and cleaved PARP in HCT-116 (C) or HT-29 (D) cells was compared to GAPDH, the loading control. The ratio of cleaved PARP to total PARP was calculated and the fold change in normalized PARP expression relative to cells treated with 2mM NaB is shown as the mean ± SD of three independent experiments. One-way ANOVA was used for statistical analysis. *P<0.05, ** p<0.01, compared to the 2mM NaB group.  \n",
      "RNA interference targeting BIP or CHOP blocked sodium butyrate induced autophagy in colorectal cancer cells.Fig 9  HCT-116 cells were transfected with BIP or CHOP specific siRNAs for 48 h and then treated with or without 2mM sodium butyrate (NaB) for 24 h. Negative control (NC) scramble siRNA was used the negative control for the transfection. Representative Western blots are shown for BIP siRNA (A) and CHOP siRNA (C). The expression level of each protein was determined by densitometry and normalized to GAPDH (ratio of protein:GAPDH). (B) Normalized expression levels of BIP and LC3-II in HCT-116 cells treated with BIP specific siRNA. (C) Normalized expression levels of CHOP and LC3-II in HCT-116 cells treated with CHOP specific siRNA. Means and standard deviation (SD) of three independent experiments are shown. One-way ANOVA was used for statistical analysis. * P<0.05, ** p<0.01, compared to the respective control group.  \n",
      "Cyclohexamide and mithramycin blocked sodium butyrate induced autophagy in colorectal cancer cells.Fig 8HCT-116 or HT-29 cells were treated with 10 μg/mL cycloheximide or 0.1μM mithramycin for 30 min and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of the expression of LC3-II are shown. The level of LC3-II expression was quantified by densitometry and normalized to GAPDH (ratio of LC3-II:GAPDH). The fold change from control cells is shown. Means and standard deviation (SD) of three independent experiments are shown. One-way ANOVA was used for statistical analysis. * P<0.05, ** p<0.01, compared to the control group. (A-B) HCT-116 cells treated with cyclohexamide; (E-F) HT-29 cells treated with cyclohexamide; HCT-116 or HT-29 cells were treated with 10 μg/mL cycloheximide or 0.1μM mithramycin for 30 min and then with 2mM sodium butyrate (NaB) for 24 h. Representative Western blots of the expression of LC3-II are shown. The level of LC3-II expression was quantified by densitometry and normalized to GAPDH (ratio of LC3-II:GAPDH). The fold change from control cells is shown. Means and standard deviation (SD) of three independent experiments are shown. One-way ANOVA was used for statistical analysis. * P<0.05, ** p<0.01, compared to the control group. (C-D) HCT-116 cells treated with mithramycin;  (G-H) HT-29 cells treated with mithramycin. cancel \n",
      ".Figure 7(A) Representative confocal image of brain parenchyma, four hours after intracerebroventricular injection of PKH26 labelled EVs (red). Astrocytes are stained with GFAP (white), nuclei with Hoechst (blue), and pan-cytokeratin (panCK; green) was used to label the choroid plexus. The dotted line shows the ventricular border and the white arrow heads point to EVs that crossed the ependymal cell layer. (B-C) Representative confocal images showing the uptake of double-labelled EVs (Ribogreen; green, PKH26, red) by astrocytes stained with GFAP (white) (B) and microglia cells stained with IBA1 (white) (C) incubated on mixed cortical cultures. Boxed areas are shown as zoomed images on right. Cell nuclei are stained with Hoechst (blue). The in vivo and in vitro uptake experiments were performed 3 and 2 times (n=3), respectively. Scalebar 100 = µm. \n",
      ".Figure 6(A) Venn diagrams showing overlap of proteins identified in EVs isolated from CSF of control (blue) and LPS-injected (yellow) mice that were present in the two independent replicates. (B) Venn diagrams showing overlap of the CSF EV proteome (control and LPS; purple) with the Mus musculus proteome list available on the EVpedia website (green). (C) GO enrichment analysis using DAVID of the EV proteome compared with total choroid plexus proteome. (D) GO enrichment analysis using DAVID of the EV proteome following LPS treatment compared with total choroid plexus proteome. The grey bars represent the number of proteins and the black dots the p-values. (E-F) DAVID (E) and Ingenuity pathway analysis (IPA) (F) on the CSF-derived EV proteome, taking into account all proteins that are exclusively detected after LPS stimulation in both replicates. The grey bars represent the number of proteins and the black dots the p-values. \n",
      ".Figure 2(A-B) In vitro quantification (A) and size distribution (B) of EVs isolated from conditioned medium of primary CPE cells grown in transwell system after 12 h in the absence (black) or presence (grey) of LPS (n=5) determined by NanoSight analysis. (C-E) TaqMan qPCR assay for the quantification of miR-9 (C), miR-146a (D) and miR-155 (E) on the exosomal pellet isolated from conditioned medium of primary CPE cells grown in a transwell system and stimulated for 12 h with LPS (n=3). (F-I) Quantification of the miRNAs miR-1a (F), miR-9 (G), miR-146a (H) and miR-155 (I) by TaqMan qPCR assay from primary CPE cells grown in a transwell system without or with LPS stimulation (n=3). Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05; **, 0.001 ≤ P < 0.01. \n",
      ".Figure 3(A) In vitro quantification of EVs isolated from conditioned medium of LPS-stimulated primary CPE cells grown in a transwell system in the absence or presence of the exosome inhibitor GW4869 (n=3). (B-D) TaqMan assay quantification of the miRNAs miR-9 (B), miR-146a (C) and miR-155 (D) in supernatant of LPS-stimulated primary CPE cells grown in a transwell system and either left untreated or pretreated with GW4869 to inhibit exosome secretion (n=3). miR-1a levels were below detection limit. (E-H) TaqMan assay quantification of the miRNAs miR-1a (E), miR-9 (F), miR-146a (G) and miR-155 (H) in cell lysate of LPS-stimulated primary CPE cells grown in a transwell system left untreated or treated with GW4869 to inhibit exosome secretion (n=3). Data are displayed as mean ± SEM and analyzed by Student's t-test. Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05; **, 0.001 ≤ P < 0.01. \n",
      ".Figure 4(A) Representative confocal images of CD63, RAB5 and ANXA2 (red) in the choroid plexus (CP) 0, 4 and 8 h after LPS treatment. Hoechst (blue) was used to stain the nucleus. The dotted line indicates the ependymal cells that line the ventricle and the square boxes indicate the zoomed insert images displayed at the right corner of each image. Scalebar = 100 µm. (B-C) Representative TEM images showing the presence of MVBs in the CPE cells before (B) and 6 hours after (C) LPS administration in vivo. White arrow heads point to exosomes present in MVBs (n=3). Scale bar = 9 µm. (D-F) Quantification of amount of MVBs per cell section (D), amount of exosomes per MVB (E), and amount of exosomes per cell section (F), based on TEM analysis. (G-J) Quantitative real-time polymerase chain reaction (qPCR) analysis of miR-1a (G), miR-9 (H), miR-146a (I), and miR-155 (J). Data are presented as relative expression normalized with housekeeping miRs by TaqMan qPCR assay (0 h, n=4; 1 h, n=5; 6 h, n=5; 24 h, n=3). (K) NanoSight analysis of CSF isolated from LPS-injected mice followed by icv injection of vehicle or GW4869, a neutral sphingomyelinase inhibitor that inhibits exosome secretion (n = 8). (L) qPCR analysis of the expression of miR-1a, miR-9, miR-146a and miR-155 in the choroid plexus of mice injected with LPS and then icv injected with vehicle (black) or GW4869 (grey), a neutral sphingomyelinase inhibitor that inhibits exosome secretion (n = 4). (M) NanoSight analysis of supernatant of choroid plexus explants from PBS- or LPS-injected mice (n = 6). Data are displayed as mean ± SEM and analyzed by Student's t-test. Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05; **, 0.001 ≤ P < 0.01; ***, 0.0001 ≤ P < 0.001. \n",
      ".Figure 5(A-B) Representative confocal images of choroid plexus (CP) on brain sections from naive mice and four hours after LPS injection (n=3). Brain sections were stained for TLR4 (A) and TNFR1 (B) (red), pan-cytokeratine (panCK; green) and the nuclei were stained with Hoechst (blue). The ependymal cells aligning the ventricles are marked with a dotted line. Scalebar = 100 µm. (C) NanoSight analysis of EVs in the CSF in sham-operated (black) relative to CLP-treated (grey) mice 10 hours after surgery. (D) qPCR analysis of the expression of miR-1a, miR-9, miR-146a and miR-155 in the choroid plexus of sham-operated mice (black) and mice subjected to CLP (grey) 10 hours after surgery. (E) qPCR analysis of the expression of miR-1a, miR-9, miR-146a and miR-155 in the CSF of sham-operated mice (black) and mice subjected to CLP (grey) 10 hours after surgery. (F) NanoSight analysis of EVs in CSF of control (black) and TNF-injected (25 µg/20 g; grey) mice 6 hours after TNF challenge. (G) qPCR analysis of the expression of miR-1a, miR-9, miR-146a and miR-155 in the choroid plexus of control mice (black) and in mice injected with TNF (grey) 6 hours after injection. (H) qPCR analysis of the expression of miR-1a, miR-9, miR-146a and miR-155 in the CSF of control mice (black) and on mice 6 hours after TNF injection (grey). Data are displayed as mean ± SEM and analyzed by Student's t-test. Scalebar 100 µm. Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05; **, 0.001 ≤ P < 0.01; ***, 0.0001 ≤ P < 0.001. \n",
      ".Figure 9CPEcells form a tight barrier between the peripheral blood and the brain. CPEcells are highly secretory and secrete exosomes from MVBs into the CSF. Systemic inflammation induces activation of the exosome machinery, resulting in an increased released of miRNA-containing exosomes into the CSF. Next, the exosomes cross the leaky ependymal cell layer that line the ventricle and are taken up by astrocytes and microglia, whereupon they transfer a pro-inflammatory message. \n",
      ".Figure 4(A) Control, CITK or ASPM-depleted HeLa cells immunostained for α-tubulin (red) and DNA (blue). Maximum intensity projections of confocal z-stacks are shown.(B) Quantification of astral MT number and maximal length in HeLa cells transfected with CTRL, CITK or ASPM siRNA (n > 130 cells in 3 independent experiments). (C) Quantification of the rescue of astral MT number after CITK overexpression in ASPM-depleted HeLa cells (n > 50 cells in 3 independent experiments).(D) Quantification of spindle orientation rescue after overexpression of CITK, CITN (lacking the kinase domain) or CKD (Citron kinase dead) mutants in ASPM-depleted HeLa cells (n > 50 cells in 3 independent experiments). (E) Synchronized HeLa transfected with control, ASPM- or CITK-specific siRNAs treated with DMSO or with 250 pM Paclitaxel (taxol) 30 min before immunostaining for α-Tubulin. Spindle angles were measured as above (n > 100 cells in 6 independent experiments). \n",
      ".Figure 2(A) Selected frames from time-lapse imaging experiments (see Movies EV1 and EV2) showing two dividing cells transfected with either control (CTRL) or CITK-specific siRNAs, respectively. Note one of the two daughter cells in the lower panel (red arrow) going out of focus as a consequence of oblique division.(B) Graphical representation of vertical and oblique divisions. The cleavage plane is indicated by a black dashed line.(C) Quantification of divisions showing uneven timing of daughter cell flattening onto the substrate after mitosis (oblique division) in CITK siRNA treated HeLa cells compared to control (n > 50 cells, 3 independent experiments). (D) Control or CITK-depleted cells were immunostained for γ-tubulin (red) and DNA (blue) and imaged in z (0.3 µm-thick sections). Upper panel: maximum intensity projections of confocal z-stacks are shown. Lower panel: cross-section (XZ) through the two poles of the same cell.(E) Distribution of spindle angles (°) in control and in CITK-depleted cells. The values represent the angles between the axis crossing the two poles of metaphase spindles and the coverslip. (n ≥ 150 cells, 6 independent experiments). (F) Quantification of spindle angles average in control cells and in cells depleted of CITK, ASPM or co-depleted of the two proteins (n ≥ 150 cells, in at least 3 independent experiments). \n",
      ".Figure 5(A) Exemplar image of EB3-tdTomatoHeLa cell used for kymograph analyses. To assess astral MT number and dynamics a line (blue) was added to images from the pole to the cortex. (B) Kymographs of pole (bottom)-to-cortex (top) line scans of control and CITK-depleted EB3-tdTomatoHeLa cells. The maximum projection of a 20-pixel-wide line (exemplified above) was plotted at 500 ms time intervals from left to right. MT growth results in diagonal lines. (C) Quantification of astral MT dynamics. MT stability, length, growth speed (n > 250 MT, 30 cells) and nucleation frequency were calculated (n > 30 cells, 3 independent experiments). (D) Immunofluorescence microscopy images showing MT growth at mitotic spindle poles 0, 1, and 2 min after nocodazole washout. Control and CITK-depleted cells were immunostained for α-tubulin (red) and DNA (blue).(E) Percentage of cells showing detectable MT nucleation (aster size > 1 µm) 0, 1, and 2 min after nocodazole washout (n > 100 cells, 4 independent experiments).(F) Quantification of aster size 2 min after nocodazole washout (n > 100 cells, 4 independent experiments). \n",
      ".Figure 1(A) WT and CITK -/- embryonic (E14.5) mousecerebral cortex was stained for γ-tubulin (green) and DNA (gray). The ventricular plane is marked by a red dashed line, and the spindle axis of apical progenitors is indicated by a white dashed line. The angle between these two lines represents the mitotic angle. Scale bars = 5 μm.(B) Quantification of vertical divisions of apical progenitors in WT and CITK -/- mice. n= 3 per each genotype. (C) Pregnant CITK +/- females, crossed with CITK +/- males, were injected with BrdU at E13.5; 24 hours later (E14.5) WT and CITK -/- embryonic mouse cerebral cortex were fixed in 4% PFA, cryosectioned and stained for Ki67 (red), BrdU (green), Tbr2 (blue) and DAPI (gray). The different cortical regions are indicated: CP=cortical plate; IZ= intermediate zone; SVZ= subventricular zone; VZ= ventricular zone. Scale bars = 10 μm. White arrows in the inset indicate two Ki67- cells positive for BrdU and Tbr2. (D) Quantification of cell cycle exit (ratio of cells BrdU+/ Ki67−) in the proliferative regions (VZ/SVZ) and in the neuronal layers (IZ and CP) of sections prepared as in panel C (n=4 per each genotype). (E) Sections prepared as in C were stained for TUBB3 (TuJ) to reveal post-mitotic neurons and with DAPI. Red arrowheads indicate apoptotic neurons (defined by pyknotic nuclei). Quantification of neuronal distribution in VZ-SVZ versus total neurons population is showed on the right. Scale bars = 10 μm. (F) Quantification of the percentage of Tbr2-positive cells in the BrdU-positive population of the same sections analyzed for panel D (n=4 per each genotype). (G) Neuroblasts of wild type and dck alleles immunostained for α-tubulin and Miranda. Note that whereas in wild type cells there is a tight coupling of the mitotic spindle with the polarity axis, in dck alleles the spindle show a more oblique orientation respect to Mira crescent. Scale bar = 5 μm.(H) Distribution of spindle angle amplitude (°) in neuroblasts of wild type and ck alleles (n=58). \n",
      ".Figure 3(A) HeLa cells pre-extracted for 1 min with 0.5% Triton X-100 in PHEM buffer and immunostained for CITK (green), γ-tubulin (red) and DNA (blue). Scale bars, 10μm (B) HeLa cells expressing CITK-GFP (green) from a BAC transgene treated as in (A) and immunostained for γ-tubulin (red) and DNA (blue). Scale bars, 5μm (C) Western blot of centrosome-containing fractions from HeLa cells, showing that CITK co-purifies with γ‑tubulin. (D) HeLa cells pre-extracted 1 min with 0.5% Triton X-100 in PHEM buffer and immunostained for CITK (green), ASPM (red) and DNA (blue). (E) HeLa cells expressing CITK-GFP treated as in (A) and immunostained for GFP (green), ASPM (red) and DNA (blue). (F) Close physical proximity between CITK and ASPM, revealed by PLA on ASPM-GFP-expressing HeLa cells, incubated with GFP and CITK antibodies and developed with PLA-specific secondary reagents. (G) Control or CITK-depleted cells immunostained for ASPM (green), γ-tubulin (red) and DNA (blue). (H) Control or ASPM-depleted cells treated as in (A) and immunostained for CITK (green), γ-tubulin (red) and DNA (blue).(I) Quantification of CITK positive centrosomes and of the ratio of CITKspindle pole intensity versus total cell mean intensity, in control and ASPM-depleted cells. (n ≥ 75 cells, 4 independent experiments). \n",
      ".Figure 8(A-B) qPCR gene expression analysis of miRNA target genes Mapk3, Notch1, Dicer, Tab2, Sox2, Calm2, Smad2, Smad5, Dnmt3 and Irak1 in mixed cortical cultures incubated with EVs isolated from CSF from untreated (black) or LPS treated (grey) mice (A) and in brain tissue (B) before (black) and 8 h after LPS injection (grey) (n=3). (C) qPCR gene expression analysis of inflammatory genes Il-1β, Tnf, Il-6, iNos and iκbα by qPCR in mixed cortical cultures (left; in vitro) incubated with EVs isolated from CSF from untreated (black) or LPS treated (grey) mice and in brain tissue (right; in vivo) before (black) and 8 h after LPS injection (grey) (n=3). (D) Cytokine analysis (IL-6, IL-1β and TNF) of supernatant of mixed cortical cultures (left; in vitro) incubated with EVs isolated from untreated (black; n=3) or LPS treated (grey; n=3) mice and of CSF (right; in vivo) from untreated (black; n=3) and LPS treated mice (grey; n=6). (E-F) qPCR gene expression analysis of inflammatory genes Il-1β, Tnf, Il-6, iNos and iκbα (E) and miRNA target genes Mapk3, Notch1, Dicer, Tab2, Sox2, Calm2, Smad2, Smad5, Dnmt3 and Irak1 (F) in mixed cortical cultures incubated with CD63-depleted EVs isolated from untreated mice (black) and mice treated with LPS for 6 h (grey) (n=3). (G-H) qPCR gene expression analysis of miRNA target genes Mapk3, Notch1, Dicer, Tab2, Sox2, Calm2, Smad2, Smad5, Dnmt3 and Irak1 (G) and inflammatory genes Il-1β, Tnf, Il-6, iNos and iκbα (H) in brain tissue from LPS-injected mice icv injected with vehicle (black) or GW4869 (grey; n=7). Data are displayed as mean ± SEM and analyzed by Student's t-test. Significance levels are indicated on the graphs: *, 0.01 ≤ P < 0.05. \n",
      "MTM‐3 acts downstream of EPG‐8, ATG‐9, ATG‐18, and ATG‐2 in the aggrephagy pathway A–K Confocal fluorescent images of 200‐cell‐stage embryos from the indicated strains stained by both anti‐SEPA‐1 and anti‐LGG‐1 antibodies.Figure 3A-K Confocal fluorescent images of 200‐cell‐stage embryos from the indicated strains stained by both anti‐SEPA‐1 and anti‐LGG‐1 antibodies. Insets show magnified views. Arrows indicate overlapping SEPA‐1 and LGG‐1 puncta. The wild type contains no visible SEPA‐1 aggregates at this stage and is therefore not shown. Scale bars: 5 μm. L The percentage of SEPA‐1 puncta co‐localizing with LGG‐1 in each strain. At least 10 embryos were scored per strain. Data are shown as mean ± SD. **P 0.0001; N.S.: not statistically different. \n",
      "Loss of mtm‐3 impairs autophagy A‐F′ Confocal fluorescent images of wild‐type (A, C, E) and mtm‐3(tm4475) (B, D, F) embryos at the 1.Figure 1A‐F′ Confocal fluorescent images of wild‐type (A, C, E) and mtm‐3(tm4475) (B, D, F) embryos at the 1.5‐fold stage stained by DAPI (A-F) and anti‐SQST‐1 (A′, B′), anti‐SEPA‐1 (C′, D′) or anti‐LGG‐1 (E′, F′) antibodies. Scale bars: 5 μm. G Western blot analysis of LGG‐1‐I and LGG‐1-II (lipidated) in wild type, epg‐6, and mtm‐3. H mtm‐3 and epg‐6 mutant L1 larvae have shortened mean and maximum life span in the absence of food. At least 300 animals were scored each day. \n",
      "Loss of mtm‐3 causes increased autophagic association of ATG‐18 A–H Fluorescent images of embryos at the 4‐fold stage in the indicated strains expressing ATG‐18::GFP (A–D), ATG‐18(FKKG)::GFP (E, F), or ATG‐18(FTTG)::GFP (G, H).Figure 5A-H Fluorescent images of embryos at the 4‐fold stage in the indicated strains expressing ATG‐18::GFP (A-D), ATG‐18(FKKG)::GFP (E, F), or ATG‐18(FTTG)::GFP (G, H). J Number of wild‐type or mutated ATG‐18::GFP puncta in each strain. At least 15 embryos were scored per strain (mean ± SD). **P 0.0001; N.S: not statistically different. I Confocal fluorescent images of 4‐fold mtm‐3 embryos expressing ATG‐18::GFP (I′) and stained by DAPI (I) and anti‐LGG‐1 (I″) antibody. Insets show magnified views. Arrows indicate ATG‐18 puncta labeled by anti‐LGG‐1 antibody. Scale bars: 5 μm. \n",
      "Loss of mtm‐3 impairs autolysosome formation A–G Confocal fluorescent images of 1.Figure 4A-G Confocal fluorescent images of 1.5‐fold‐stage embryos from the indicated strains expressing both SQST‐1::GFP and NUC‐1::mCHERRY. Insets show magnified views. Arrows indicate overlapping or closely associated GFP and mCHERRY. Scale bars: 5 μm. The wild type contains no visible SQST‐1::GFP puncta at this stage and is therefore not shown. H, I The percentage of SQST‐1::GFP‐positive puncta that overlap with (H) or are close to (I) NUC‐1::mCHERRY‐positive structures in each strain. At least 10 embryos were scored per strain. Data are shown as mean ± SD. **P 0.0001; N.S.: not statistically different. J-L TEM analysis of embryos in wild type (I) and mtm‐3(tm4475) (J, K). Double‐membrane autophagosomes (arrows) were observed in mtm‐3 but not wild‐type embryos. Scale bars: 500 nm. \n",
      "GFP::MTM‐3(C459S) associates with autophagic structures A–C Fluorescent images of wild‐type embryos (1.Figure 2A-C Fluorescent images of wild‐type embryos (1.5‐fold stage) expressing GFP::MTM‐3 (A), GFP::MTM‐3(C459S) (B), or GFP::MTM‐3(C459S, del PH‐G) (C). D‐G″ Confocal fluorescent images of wild‐type (D‐D″), mtm‐3 (E‐E″), atg‐2 (F‐F″), and vps‐34 (G‐G″) embryos at the 200‐cell stage expressing GFP::MTM‐3(C459S) and stained by anti‐LGG‐1 antibody. Insets show magnified views. Arrows indicate overlapping GFP and LGG‐1 puncta. Scale bars: 5 μm. H, I Number of GFP::MTM‐3(C459S)‐positive puncta and co‐localization of GFP::MTM‐3(C459S)‐ and LGG‐1‐positive puncta in various strains [mtm‐3(tm4475), atg‐2(bp576), lgg‐1(bp500), vps‐34(h797)]. At least 10 (I) and 15 (H) embryos were scored per strain. Data are shown as mean ± SD. Data from different mutant backgrounds were compared with wild type. **P 0.0001; N.S.: not statistically different. J MTM‐3 but not MTM‐3(C459S) exhibits phosphatase activity toward PtdIns3P and PtdIns(3,5) P2 in vitro. \n",
      "a–c) Temporal changes in the green fluorescence pattern of photoactivated NIH3T3 cells stably expressing KFERQ–PS–CFP2 maintained in the absence (black circles) or presence (white squares) of serum for the indicated times.f2(a-c) Temporal changes in the green fluorescence pattern of photoactivated NIH3T3 cells stably expressing KFERQ-PS-CFP2 maintained in the absence (black circles) or presence (white squares) of serum for the indicated times. (a) Representative images of cells maintained in serum-free media. Quantification of the number of puncta per cell (b) and green fluorescence intensity (c). Values are expressed as percentage of fluorescence at time 0 and half-lives (t1/2) are indicated. Differences in the number of puncta and fluorescence intensity between cells maintained in the presence and absence of serum (*) were significant for P=0.048 (16 h) and P=0.04 (24 h) (t-test). (d) The same cells imaged in the cyan channel, before and at the indicated times after photoconversion (arrow) by exposure to 405 nm light. Top: Representative images of the cyan fluorescence. Bottom: Quantification of the intensity of the cyan fluorescence. Values are expressed as times the intensity before photoconversion that was given an arbitrary value of 1. Values are mean+s.e. of 3 different experiments with >10 cells counted per experiment. Scale bars, 5 μm. \n",
      "a, Mice were infected for 21 days and liver granulomas analysed by transmission electron microscopy (TEM).figf1a, Mice were infected for 21 days and liver granulomas analysed by transmission electron microscopy (TEM). Shown are spacious vacuoles (SLAPs) containing multiple bacteria. Magnification, ×5,200. Region 1 and 2 (white boxes) are enlarged in the lower panels. b, SCID mice were infected as in a and liver sections stained for LAMP1 (green), bacteria (red) and DNA (blue). Shown is a LAMP1+ SLAP. c, The percentage of bacteria in LAMP1+ compartments was quantified and characterized as either spacious or tight-fitting. Mean ± s.e.m. for three mice examined. The image in a (from ref. 2) and the tissue sections were provided by E. Unanue. \n",
      "a) Scheme of the insertion of the CMA-targeting motif in PS–CFP2.f1(a) Scheme of the insertion of the CMA-targeting motif in PS-CFP2. (b) Map of pKFERQ-PS-CFP2. (c) Scheme of the experimental design to monitor CMA activity in cultured cells using the KFERQ-PS-CFP2 reporter. (d) NIH3T3 cells stably expressing PS-CFP2 proteins were exposed to a 405 nm light for the indicated times and photoconverting was monitored by imaging cells in the green and blue channels. (e, f) Mouse fibroblasts (NIH3T3) stably expressing PS-CFP2 or KFERQ-PS-CFP2 were photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h. (e) Representative images in both channels. (f) Quantification of the number of green fluorescent puncta per cell in cells maintained in the presence (white bars) or absence (gray bars) of serum. Values are mean+s.e. of four different experiments with >50 cells counted per experiment. (*) Differences with PS-CFP2 in the green channel (P=0.005, ANOVA-Bonferroni). (§) Differences with samples in the presence of serum (P=0.03, ANOVA-Bonferroni). (g) Green fluorescence intensity at excitation 450 nm and emission 510-550 nm. Fluorescence intensity in each sample was normalized by the number of cells present in the sample and the percentage of intensity at time 0 still remaining after 16 h of incubation in the presence (white bars) or absence (gray bars) of serum was calculated for each condition. Values are expressed relative to the decay observed in cells maintained in the presence of serum and are means+s.e. of 3 independent experiments. Differences with serum (*) were significant for P=0.04 (t-test). Scale bars, 5 μm. \n",
      "a, GFP–LC3-transfected macrophages were infected for 4 h, and the percentage of infected cells exhibiting SLAPs was quantified.figf3a, GFP-LC3-transfected macrophages were infected for 4 h, and the percentage of infected cells exhibiting SLAPs was quantified. Mean ± s.e.m. for three independent experiments. P values for strains with significant differences from wild-type levels are shown. b, The arrow indicates a GFP-LC3+ SLAP with internal LLO expression in cells infected for 4 h. Scale bar, 5 µm. c, Arrowheads indicate GFP-LC3+ SLAPs devoid of Lysotracker Red in cells infected for 4 h. e, The arrow indicates a LAMP1+ SLAP colocalizing with v-ATPase staining in cells infected for 4 h. Scale bar, 5 µm. f, The percentage of v-ATPase+ SLAPs was quantified as in Fig. 2c. Mean ± s.e.m. for three independent experiments. P values for conditions significantly different from PFA-killed bacteria are shown. \n",
      "a, RAW 264.figf2a, RAW 264.7macrophages infected for 4 or 8 h were analysed by TEM. Shown are spacious vacuoles (SLAPs) containing multiple bacteria. The arrow indicates septum of dividing bacteria. Scale bars, 0.5 µm. b, The arrow indicates a LAMP1+ SLAP colocalizing with green fluorescent protein (GFP)-LC3 in cells infected for 4 h.The percentage of GFP-LC3+ (c) d, The arrowhead indicates a LAMP1+ SLAP devoid of cathepsin D in cells infected for 4 h. Scale bars, 5 µm.The percentage of GFP-LC3+ (c) or cathepsin D+ (e) f, GFP-LC3-transfected macrophages were infected with wild-type or D actA bacteria. Where indicated, chloramphenicol (CM) was added to the media at 3 h post infection. The number of bacteria per SLAP was quantified. Brackets indicate significant differences, and corresponding P values are shown. g, Macrophages were infected with wild-type bacteria for 7 h, pulsed with BrdU for 1 h, and stained for LAMP1 (red), bacteria (blue) and BrdU (green). Magnified images and the arrow indicate SLAPs containing actively replicating bacteria (BrdU+). h, The percentage of BrdU+ bacteria in SLAPs, compared to bacteria and PFA-killed bacteria, was quantified as in c. \n",
      "a, Arrows indicate LAMP1– wild-type bacteria with actin ‘comet tails’ in cells infected for 6 h.figf4a, Arrows indicate LAMP1- wild-type bacteria with actin 'comet tails' in cells infected for 6 h. The boxed region is magnified in the bottom left panels. The merged image for this region is shown at the bottom right. Scale bar, 5 µm. b, Macrophages were infected with IPTG-induced iLLO bacteria. Arrowheads indicate actin- iLLO bacteria within LAMP1+ vacuoles. c, Macrophages were infected with wild-type (triangles) or IPTG-induced iLLO (circles) bacteria, and the percentage of LAMP1+ bacteria was quantified. Mean ± s.e.m. for three independent experiments. d, Macrophages were infected as in c with or without IPTG induction. Intracellular bacterial replication was determined using a gentamicin-protection assay. Shown is fold replication compared to 2 h post infection. Clear triangles, wild type; filled triangles, wild type + IPTG; clear circles, iLLO; filled circles, iLLO + IPTG. Mean ± s.e.m. or range for three (wild type, wild type + IPTG, iLLO + IPTG) or two (iLLO-IPTG) independent experiments, respectively. e, Wild-type or Atg5-/- MEFs were infected with iLLO bacteria, and intracellular bacterial replication was determined as in d. Mean ± s.e.m. for three independent experiments. Clear circles, wild-type MEFs; filled circles, wild-type MEFs + IPTG; clear squares, Atg5-/- MEFs; filled squares, Atg5-/- MEFs + IPTG. f, Wild-type (circles) or Atg5-/- (squares) MEFs were infected as in e with IPTG induction. The percentage of LAMP1+ bacteria was quantified as in c. Mean ± s.e.m. for three independent experiments. g \n",
      ".Figure 1(A-B) Diagram of the organization of the tracheal system at the onset of terminal cell branching (A) and one hour later (B). (C-C''') Terminal cell expressing reporters for membrane (C), ER (C') and Golgi apparatus (C''). Red arrowheads: CD4::mIFP vesicles and associated markers at the tip of the cell. (D) Mean and 95% confidence interval of tube and branch length during the first hour of growth. Numbers represent the rate of growth estimated from the slope of the curve +/-SD, p=0.39, Mann-Whitney test, n=6. (E) Distance of the tip of the tube, and the leading ER and Golgi accumulations from the cell junction that defines the base of the cell. Line and shading represent mean and 95% confidence interval, n=4. (F-I) Terminal cells expressing the general membrane marker CD4::mIFP in combination with different membrane or polarity markers: PH::mCherry, a PIP2 sensor commonly used as apical plasma membrane marker (F'), an endogenously GFP-tagged Crb (G'), polarity proteins Par3::YFP (H') and Par6::mCherry (H''). The boxed region in (H''') is shown at higher magnification in (I) over 4 time points. Red arrowheads: CD4::mIFP vesicles and associated markers at the tip of the cell. \n",
      "a) Localization of photoconverted KFERQ–PS–CFP2 (green) in LysoTracker-stained compartments (red) in mouse fibroblasts maintained in the presence or absence of serum.f3(a) Localization of photoconverted KFERQ-PS-CFP2 (green) in LysoTracker-stained compartments (red) in mouse fibroblasts maintained in the presence or absence of serum. (b) Percentage of co-localization of KFERQ-PS-CFP2 with the indicated proteins. Values are mean+s.e. of three different experiments with >50 cells counted per experiment. (c) Co-localization of photoconverted KFERQ-PS-CFP2 (green) with LAMP-1, LAMP-2 or LAMP-2A. (d) Immunofluorescence with antibodies that recognize PS-CFP2 (green) and hsc70 (red) in cells maintained in the absence of serum and fixed with methanol to eliminate the soluble cytosolic fraction. (e, f) Co-localization of photoconverted KFERQ-PS-CFP2 (green) with LC3, CD-M6PR, Rab5 (e) and ubiquitin (f) in mouse fibroblasts maintained in serum-free media. Scale bars, 5 μm. \n",
      "a) Mouse fibroblasts knockeddown for LAMP-2A and transfected with KFERQ–PS–CFP2 were photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h.f4(a) Mouse fibroblasts knockeddown for LAMP-2A and transfected with KFERQ-PS-CFP2 were photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h. Representative images of cells also immunostained for LAMP-1 are shown. (b, c) Mouse fibroblasts stably expressing KFERQ-PS-CFP2 were photoconverted and maintained for 16 h in media supplemented (+) or not (−) with serum in the presence of 3-methyladenine (3-MA) to inhibit macroautophagy. (b) Representative images of the 3-MA-treated cells. (c) Quantification of the number of green fluorescence puncta per cell in 3-MA treated or not cells maintained in the presence (white bars) or absence (black bars) of serum. Values are mean+s.e. of 3 different experiments with >50 cells counted per experiment. No significant differences were detectable between samples supplemented or not with 3MA. Scale bars, 5 μm. \n",
      "a) Scheme of the experimental design to monitor CMA activity in cultured cells using the KFERQ–PA–mCherry1 reporter.f7(a) Scheme of the experimental design to monitor CMA activity in cultured cells using the KFERQ-PA-mCherry1 reporter. (b) Mouse fibroblasts (NIH3T3) stably expressing KFERQ-PA-mCherry1 were photoactivated and maintained in media supplemented (+) or not (−) with serum for 16 h. Left: Representative images. Nuclei are staining with DAPI. Right: Quantification of the number of red fluorescent puncta per cell. Values are mean+s.e. of three different experiments with >20 cells counted per experiment. *P=0.001, ANOVA-Bonferroni. (c) Temporal changes in the mCherry fluorescent intensity in cells stably expressing KFERQ-PA-mCherry1 at the indicated times after photoactivation using FACS analysis. Cells were maintained in media supplemented (white squares) or not (black circles) with serum, and fluorescence by the MFI was calculated on events gated on living cells. Values are expressed as percentage of the intensity at time 0 and are mean+s.e. of 3-4 individual experiments. (d) Representative FACS experiment showing the effect of serum removal at 6 h (top) and 42 h (bottom). PS, photoswitching. Scale bars, 5 μm. \n",
      "a, b) Human neuroblastoma (SH-SY5Y), hepatoma (HuH-7) and breast adenocarcinoma (MCF-7) cell lines were transiently transfected with KFERQ–PS–CFP2, photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h.f6(a, b) Human neuroblastoma (SH-SY5Y), hepatoma (HuH-7) and breast adenocarcinoma (MCF-7) cell lines were transiently transfected with KFERQ-PS-CFP2, photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h. Images show representative cells after immunostaining for LAMP-2 (red). (b) Quantification of the number of green fluorescent puncta per cell under basal conditions (black bars) and after serum removal (gray bars). Values of NIH3T3 are included as reference. Values are mean+s.e. of three-to-five different experiments with >50 cells counted per experiment. Differences with (*) NIH3T3 or (§) with cell supplemented with serum are significant for P0.05 (range 0.002-0.03, ANOVA-Bonferroni). (c) Primary mouse skin fibroblasts were transduced with a lentiviral vector for expression of KFERQ-PS-CFP2, photoconverted and maintained in media supplemented (+) or not (−) with serum for 16 h. Left: representative image in the green channel. Right: Quantification of the number of green fluorescent puncta per cell in cells maintained in the presence (black bars) or absence (gray bars) of serum. Values are mean+s.e. of three different experiments with >50 cells counted per experiment. (P=0.002, ANOVA-Bonferroni). (d) Mouse fibroblasts stably expressing KFERQ-PS-CFP2 were plated in 384-well plates, maintained in the presence (white bar) or absence (black bars) of serum and subjected to treatment with 3-methyladenine (3-MA), the proteasome inhibitor MG-132 or H2O2. In both (d, e), after fixation, images were subjected to high content image analysis and the percent of responders was calculated in >200 cells per condition. Values are mean+s.e. of results in three different experiments. *P0.001 (range 0.0002-0.0007, t-test). Scale bars, 5 μm. (e) Mouse fibroblasts stably expressing KFERQ-PS-CFP2 were plated in 384-well plates, maintained in the presence of serum supplemented media and leave untreated (white bar) or subjected to the indicated treatments (black bars). In both (d, e), after fixation, images were subjected to high content image analysis and the percent of responders was calculated in >200 cells per condition. Values are mean+s.e. of results in three different experiments. *P0.001 (range 0.0002-0.0007, t-test). Scale bars, 5 μm. \n",
      "a) Immunoblot with the antibody that recognizes PS–CFP2 of homogenates (Hom), cytosol (Cyt) and lysosomal membranes (MB) and matrices (Mtx) isolated from mouse fibroblasts stably expressing PS–CFP2 or KFERQ–PS–CFP2 proteins 16 h after serum removal.f5(a) Immunoblot with the antibody that recognizes PS-CFP2 of homogenates (Hom), cytosol (Cyt) and lysosomal membranes (MB) and matrices (Mtx) isolated from mouse fibroblasts stably expressing PS-CFP2 or KFERQ-PS-CFP2 proteins 16 h after serum removal. (b) Immunoblot for the indicated proteins of homogenates (Hom), mitochondria (Mit) and lysosomal membranes and matrices isolated from cells stably expressing KFERQ-PS-CFP2 maintained for 16 h in media supplemented (+) or not (−) with serum. (c) Quantification of the amount of KFERQ-PS-CFP2 associated to lysosomes (matrix and membrane) performed in blots as the one shown in (b). Values are expressed as percentage of the total KFERQ-PS-CFP2 in the cell and are mean+s.e. of three different experiments. Difference in the lysosomal associated KFERQ-PS-CFP2 between serum+ and serum− condition (*) were significant for P=0.033 (t-test). (d) Changes in the intracellular content of KFERQ-PS-CFP2 on serum removal. Values are expressed as percentage of KFERQ-PS-CFP2 present in serum-supplemented cells and area mean+s.e. of three different experiments. Values are mean+s.e. of three different experiments and differences are significant (*) for P=0.007 (t-test). (e-g) Effect of inhibition of macroautophagy by treatment with 3-methyladenine (3-MA) on the lysosomal content of KFERQ-PS-CFP2 in cells maintained in serum-free media for 16 h. (e) Immunoblot for the indicated proteins of the same fractions as described in (b). The amount of KFERQ-PS-CFP2 associated to lysosomes (f) and the changes in the intracellular content of KFERQ-PS-CFP2 were calculated as in (c, d). Values are mean+s.e. of three different experiments and differences are significant (*) for P=0.032 (t-test). \n",
      ".Figure 2TEM tomograms from high-pressure frozen embryos. (A-A') Low magnification (2.5 nm voxels) tomogram and 3D reconstruction from 10 serial 300 nm sections that cover one terminal cell. (A''-A''') 3D reconstructions derived from higher resolution tomograms showing different organelles in the cell. (B) Example of an individual high-resolution (0.78nm voxels) tomograms used to generate 3D reconstructions. (C) Low resolution tomogram of another cell; the dotted box is magnified in (D). (D) High-resolution tomogram. (D') Model derived from 4 serial 3D reconstructions including the one shown in (D). \n",
      ".Figure 3(A) Cell model composed of superimposed 3D reconstructions of high-resolution TEM tomograms covering different regions of the cell. Each square is one 3.2x3.2x0.17 µm 3D reconstruction. The tomograms were assigned to two categories for quantitative evaluation: those including the tip of the tube (orange) and the rest (purple). (B) Distribution of three commonly seen classes of vesicles, expressed as their density per area of cytoplasm screened, in the two regions of the cell. Box plots represent median, interquartile range (IQR), and IQR*1.5 below and above the IQR. n=62 tomograms from two different cells. (C) Representative examples for each vesicle class are shown next to the quantification plots. (D) Distribution of endocytic events at the apical or basal membrane compartment, expressed as density per membrane surface screened. Left shows tomograms corresponding to the cell body and branch, and right shows tomograms corresponding to the tip of the cell. Box plots represent median, interquartile range (IQR), and IQR*1.5 below and above the IQR. n=48 tomograms from two different cells. \n",
      ".Figure 7High time resolution imaging of the membrane marker CD4::mIFP expressed under btl-gal4. (A) Example of a large CD4 vesicle (arrowhead) rapidly moving towards the tip of the terminal cell. (B-D) Analysis of 28 CD4 vesicles in five terminal cells. (B) Trajectories of CD4 vesicles with their original position mapped to the origin of the plot. (C) Rose diagram of the trajectories shown in (B). (D) Speed of CD4 vesicles compared to speed of tube growth measured over 25 min. Data are plotted as mean +/-SD. (E-H) Terminal cells expressing the membrane reporter CD4::mIFP together with markers for vesicles of the endocytic pathway. (E) Rab5::YFP, red arrowheads point to a Rab5-positive vesicle; the white arrowhead shows the same vesicle once it lost the Rab5 signal. (F) FYVE::GFP, a PI3P reporter and Spin::mRFP , a lysosomal marker. Black arrowheads point to small FYVE::GFP vesicles and the squared area shows large ones at the tip. (G) Par6::mCherry and endogenously labelled Rab7::YFP. (H) Percentage of CD4::mIFP vesicles that carry the indicated markers. Data are plotted as mean +/-SD. For each marker we analyzed at least 20 minutes of cell growth with 10-20 timepoints. Number of cells analyzed: Par3, n=2; Rab5, n=3; FYVE::GFP, n=4; FYVE::GFP-Spin::RFP (which includes single- and double-positive vesicles), n=3; dLamp1, n=2; Rab7, n=5. \n",
      ".Figure 5(A-F) TEM tomograms and 3D reconstructions of terminal cells in shibirets embryos. (A) Control. (B-F) Kept at 34ºC for 1 (B, E) or for 2 hours (C, D, F). Boxed regions in (A) and (B) are magnified in (A'') and (B''). (A', B') Reconstructed 3D models. The tube membrane (cyan) is characterized by the electron-dense apical extracellular matrix (aECM). Invaginations with connections that can be traced to the tube membrane are red, invaginations from the basal plasma membrane are green. The basal membrane is light pink. (C-D') shibirets terminal cell after 2 hours at 34ºC. (C') 3D reconstruction, color coding for basal and tube membrane is the same as above. Membrane that is continuous with both the apical and the basal membrane is shown in green, purple and dark blue. (D-D') higher magnification of a region in the same cell at a level where membrane sheets (purple, dark blue) bridge the apical and basal plasma membrane domains. (E-E'') Two focal planes and model of a tomogram from the cell shown in (B) where a basal (pink arrowhead) and an apical invagination (blue) are seen in close proximity (closest distance is marked by cyan and blue arrowheads). (F-F') apical membrane overgrowth regions of the cell shown in (C). The genotype of the embryos was shibirets; btl>KDEL::RFP, Par3::YFP. The cell shown in (C-D) was found and acquired without the CLEM approach. \n",
      ".Figure 6(A-B) Terminal cells expressing the membrane marker CD4::mIFP under btl-gal4 and FGFR::GFP under its own promoter (from the fTRG library). (A) Time lapse imaging of a control cell. Blue arrowheads point to filopodia and basal plasma membrane and red ones point to puncta containing CD4::mIFP and FGFR::GFP. (B) shibirets cell imaged before dynamin inactivation, after 23 and 45 minutes of inactivation, and after 30 minutes of recovery. White arrowheads point to FGFR::GFP accumulation at the apical compartment. (C-E) Single confocal planes of terminal cells expressing PH::mCherry and stained for βPS-Integrin. The outline of the cells was traced using the PH::mCherry signal and is shown as a blue dashed line. (C) Control. Blue arrowheads: βPS-Integrin signal in filopodia. (D) 1 hour at 34ºC, (E) 2 hours at 34ºC. (C''-E'') Orthogonal views of the lines shown in white in (C-E). (F-G) Quantification of fluorescence intensity from stained embryos of FGFR::GFP (F) and of βPS-Integrin (G). Data are plotted as mean +/- SD; significance was assessed using one-way ANOVA with Dunnett's correction for multiple comparisons. **p=0.0022, ***p=0.0003, ****p<0.0001. Number of cells analyzed, for (F): Control n=8, 1hr at 34ºC n=6, Recovery n=8. (G). For (G): Control= 7, 1hr at 34ºC= 8, 2hrs at 34ºC= 8. \n",
      ".Figure 4Distribution of the plasma membrane reporter PH::GFP in control cells (A-A'') and in cells where dynamin activity had been blocked using a temperature-sensitive allele of dynamin (shibirets In (B), was inactivated at the onset of tube formation (C-D) Distribution of PH::GFP fluorescence intensity in control and shibirets cells. Data from 1-2 minute-interval time lapses was collected in windows of 20 minutes each (except for t=0). (C) Proportion of signal in the apical and in the basal membrane compartment over time in control cells (n=4) and also in cells where dynamin was inactivated (n=5), *p=0.015, Mann-Whitney test. (D) Total fluorescence intensity of PH::GFP over time in controls (n=4) and in cells where dynamin was inactivated (n=5), n.s.: p=0.063, Mann-Whitney test. Distribution of the plasma membrane reporter PH::GFP in cells where dynamin activity had been blocked using a temperature-sensitive allele of dynamin (shibirets in (E) it was inactivated ~15 minutes after the tube had begun to form. (F-G) Distribution of the luminal reporter Gasp::GFP in control cells (F) and in cells where dynamin was inactivated (G). Arrowheads point to protrusions sprouting from the lumen of the terminal cell. (H-I) Distribution of the general plasma membrane marker CD4::mIFP in combination with the Clathrin light chain fused to GFP (CLC::GFP, H) and with Dynamin::YFP (I). \n",
      ".Figure 2(A) Representative images of wild-type (WT) and mutant (KO) neurospheres obtained from E15.5 neocortices and cultured in vitro. The culture passage (\"step\") corresponds to 3 days of culture after dissociation and re-plating. KO aggregates are more numerous and larger in size. Data information: Scale bars: 50µm. (B) Graph showing the number of neurospheres per P100 cell plate at different steps, as indicated. Nr2f1KO neurospheres proliferate for a longer time (up to 30 passages tested), whereas WT cells exhaust around step 9. n ≥ 3 culture wells from n = 2 batches. Data information: data are represented as means ± SEM. 2-way ANOVA (B) (**P<0.01, ***P<0.001). (C) Panel of WT and KO isolated neurospheres grown from day 1 to day 10. Blue and red lines represent the diameter of WT and KO neurospheres, respectively. Data information Scale bars: 50µm. (D,E) PH3 (green; dividing cells) IF on cross-sections of day 4 neurospheres. White arrowheads point at mitotic figures, which are quantified in (E) as percentage of dividing cells over DAPI-stained nuclei (blue). n ≥ 3 neurospheres from n = 2 batches. Data information: data are represented as means ± SEM. Student t-test (E) (**P<0.01, ***P<0.001). Scale bars: 50µm. \n",
      ".Figure 8(A) Nr2f1 is expressed along an anteromedial-low to posterolateral-high gradient (red color code) in the developing mouse cortex, spanning from VZ aRGs to SVZ IPs, bRGs and CP neurons. Upon Nr2f1 removal (KO), the NP pool of the posterior cortex expands leading to occipital macrocephaly. (B) Early to late effects on progenitor and neuron behavior upon Nr2f1 deficiency. Early in development (E10.5-E12-5), Nr2f1 loss causes cell cycle acceleration, increased NP self-renewal and sustained Pax6 expression. Neurogenesis (as well as the expression of neurogenetic factors such as Tbr2 and P21) is delayed, thus allowing amplification of the progenitor pool and lateral expansion of the posterior hemispheres. At later stages (E13.5-E14.5), neurogenesis comes into play and neurons are produced at high rate. Persistent Pax6 expression in the expanded NP population leads to abundant basal RG production at late time points (E16.5-E18.5). High neuronal output results in radial expansion and generation of a thick posterior cortex. \n",
      ".Figure 8(A-F) Terminal cells expressing CD4::mIFP and FYVE::GFP under btl-gal4. The outline of the cells was traced using the CD4::mIFP signal and is shown as a blue dashed line. (A-C) Control cell. (D-F) shibirets cell before 34ºC (D), after 30 minutes at 34ºC (E) and after 30 minutes of recovery (F). Red arrowheads: FYVE::GFP puncta. (G-H) Number of FYVE::GFP vesicles in 4 control cells (G) and in 6 shibirets cells (H). Box plots represent median, interquartile range (IQR), and IQR*1.5 below and above the IQR. Significance in (H) was assessed using one-way ANOVA with Geisser-Greenhouse correction for paired data and Dunnett's multiple comparisons test. *p=0.0288, ***p=0.0016. (I) shrbO3/G5 mutant cell expressing CD4::mIFP under btl-gal4. Arrowhead: CD4::mIFP aggregation. (J-K) Terminal cells over-expressing Shrb::GFP under btl-gal4 together with PH::mCherry (J), or together with Spin::RFP and CD4::mIFP (K). Arrowheads: Shrb::GFP accumulations. (L-M) Dorsal trunk cells stained for Serp. In control embryos (L), Serp is seen exclusively in the tracheal lumen. The cells themselves are not visible in these images. In embryos expressing Shrb::GFP under btl-gal4 (M), Serp is also seen inside the cells, usually in association with or surrounded by Shrb::GFP. Boxed regions are magnified in (L'-M') and shown as single confocal planes. \n",
      ".Figure 7(A-C) NR2F1 expression in cerebral organoids during neural induction and differentiation in vitro (day 40 and day 70, respectively), together with the NP markers PAX6 (green in A), SOX2 (green in B,B') and DCX as a neural marker (blue in A,B,B''). Quantification of NR2F1 levels in NPs is shown in (C). n ≥ 4 organoids from n = 2 batches. Data information: the pixel intensity or the number of positive cells were quantified in 100µm-width boxes, randomly placed across the cortex/organoid neuroepithelia In graphs, data are represented as means ± SEM. 2-way ANOVA test (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. (D-G) NR2F1 (red) and PAX6 (green) IF in day 40 organoids. Neuroepithelia with high NR2F1 levels show weak PAX6 expression (E,E'), and vice versa (F,F'). Quantification by pixel intensity analysis (G). n ≥ 4 organoids from n = 2 batches. Data information: Nuclei (blue stained with DAPI. the pixel intensity or the number of positive cells were quantified in 100µm-width boxes, randomly placed across the cortex/organoid neuroepithelia In graphs, data are represented as means ± SEM. 2-way ANOVA test (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. (H) Schematic representation of an organoid electroporation. PX458 plasmids were injected in ventricular-like cavities, organoids electroporated and processed after 7 days of in vitro culture. (I-J) NR2F1 (red) and GFP (green) IF in day 40 human brain organoids upon electroporation of the PX458-αNR2F1 plasmid. In (I,I'), GFP+NR2F1- cells (empty arrowheads) and a GFP+NR2F1+ cell (white arrowhead) are shown; quantification in (J). n ≥ 6 organoids from n = 2 batches. Data information: Nuclei (blue stained with DAPI. the pixel intensity or the number of positive cells were quantified in 100µm-width boxes, randomly placed across the cortex/organoid neuroepithelia In graphs, data are represented as means ± SEM. 2-way ANOVA test (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. (K-M) TUJ1 (red; differentiating neurons) and GFP (green) IF 7 days after electroporation of control (K) and PX458-αNr2f1 plasmids (L). Co-expression of GFP with SOX2 or TUJ1 distinguishes neural progenitors (NPs) from neurons (Ns), as quantified in (M). Arrowheads in (K,L) point to TUJ1+ GFP+ cells. n ≥ 6 organoids from n = 2 batches. Data information: Nuclei (blue stained with DAPI. the number of positive cells was normalized over the total number of GFP+ cells. In graphs, data are represented as means ± SEM. 2-way ANOVA test (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. (N) Red and green arrowheads in (N) point to a SOX2+ (red) progenitor and a TUJ1+ (blue) neuron, respectively. Scale bars: 50µm. \n",
      ".Figure 5(A-B') Pax6 (green; aRGs) and Nr2f1 (red) IF of E12.5 WT (A,A') and KO (B,B') lateral pallia. Data information: nuclei (blue) were stained with DAPI. Scale bars: 50µm. SVZ: subventricular zone; VZ: ventricular zone. (C) Pax6 and Nr2f1 pixel intensity quantification at E12.5 and E14.5 indicating increased Pax6 levels upon Nr2f1 removal. n ≥ 3 brains. Data information: the pixel intensity was quantified in 100µm-width boxes, randomly placed across the lateral pallium. Data are represented as means ± SEM. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). (D) Real time RT-PCR quantification of Nr2f1 and Pax6 expression in E12.5 cortices. n ≥ 3 cortices. Data information: Data are represented as means ± SEM. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). (E-G'') GFP (green; electroporated cells), Pax6 (red) and Nr2f1 (blue) IF of pCIG2-Nr2f1-IRES-GFP electroporated brains at E13.5 (24-hours after IUE). VZ (F-F'') and CP (G-G'') regions are shown at high magnification. Arrowheads point to GFP+ electroporated cells over-expressing Nr2f1 (blue in F',G'), down-regulating Pax6 (F'') and rapidly migrating out of the VZ (G-G''). Scale bars: 50µm. SVZ: subventricular zone; VZ: ventricular zone. (H) Scatter plot showing Pax6 and Nr2f1 pixel intensity of VZ electroporated progenitors, comparing Nr2f1 overexpressing cells (orange dots) with control pCIG2-GFP electroporated cells (blue dots). Average Nr2f1 and Pax6 pixel intensities of the 2 populations were compared by 2-way ANOVA and resulted significantly different (***= <0.0001). n = 2 electroporated brains. Data information Data are represented as means ± SEM. 2-way ANOVA *P<0.05, **P<0.01, ***P<0.001). (I) Real time RT-PCR of cell cycle genes comparing WT (blue line) and KO (orange line) cortices. CyclinD1 and P21 transcripts are down-regulated whereas Dct is up-regulated in mutants. n = 3 cortices. Data information: Data are represented as means ± SEM. 2-way ANOVA *P<0.05, **P<0.01, ***P<0.001). (J-L) GFP (green; electroporated cells) and P21 (red) IF of E14.5 cortex electroporated 48 hours earlier with control PX458 plasmid (J) or CRISPR/Cas9-expressing plasmid directed against Nr2f1 sequence (PX458-αNr2f1; K). Percentage of P21/GFP double positive cells (L). Arrowheads in (J,K) point to P21+ cells. n = 2 electroporated brains. Data information , nuclei (blue) were stained with DAPI. Student t-test (L; *P<0.05 (M) Schematic model of Nr2f1 action on NP cell cycle and neural differentiation. Nr2f1 promotes neurogenesis by repressing Pax6 and Dct expression and thus cell cycle progression, and by activating P21-mediated cell cycle exit. Both actions modulate the G1-phase length. (N) Graph showing the number of differentially expressed genes (DEG up-regulated (UP) or down-regulated (DOWN) in KO compared to WT, as detected by RNA-Seq of E15.5 neocortices. Data information: For statistical analysis of RNA-Seq data , see Methods. (O) Gene ontology (GO) categories significantly enriched among DEG genes (DAVID Gene Ontology software (P) Hierarchical clustering and heatmap of the expression level of DEGs belonging to the \"neuron differentiation\" Heat map color scale indicates normalized gene expression from high (red) to low (white) level. Data information: For statistical analysis of RNA-Seq data see Methods. \n",
      ".Figure 1Cortical malformations observed in six novel BBSOAS patients Patient (P) 1 shows abnormally elongated occipital convolutions (arrowheads in A), whereas abnormal gyrification of supramarginal and angular gyri is present in five out of six individuals (P2-P6 B', C' and C'' highlight the abnormally convoluted regions boxed in B and C. Cortical malformations observed in six novel BBSOAS patients Patient (P) 1 shows abnormal gyrification of supramarginal and angular gyri is present in five out of six individuals (P2-P6; (G) Schematic representation of the human NR2F1 protein sequence, showing novel variants of a new BBSOAS patient cohort. Key aminoacids for the functioning of the DNA-binding domain (DBD) or the ligand-binding domain (LBD) are listed in boxes. (H,H') TBR2 (green in H), SOX2 (red in H) and NR2F1 (yellow in H') immunofluorescence (IF) of a GW11 section of human neocortex Data information: Nuclei (blue) were stained with DAPI. Scale bars: 50µm. CP: cortical plate; IZ: intermediate zone; oSVZ: outer subventricular zone; SVZ: subventricular zone; VZ: ventricular zone. (I) NR2F1 expression (quantified by pixel intensity) at different L-M levels and in different regions along the A-P extent of the cortex, as indicated. n ≥ 3 sections from n = 1 fetal brain. Data information the number of positive cells was quantified in 100µm-width boxes, randomly placed across the cortex. In graphs, data are represented as means ± SEM. 2-way ANOVA (I) (*P<0.05, ***P<0.001). (J-L') NR2F1 (red) IF of human GW14 neocortex, showing expression levels in the CP (K,L) and VZ (K',L') around primary convolutions in the posterior-most cortex High NR2F1 is detected in the neurogenic area of a gyrus (L'). Data information: Nuclei (blue) were stained with DAPI. In Scale bars: 50µm. CP: cortical plate; IZ: intermediate zone; oSVZ: outer subventricular zone; SVZ: subventricular zone; VZ: ventricular zone. (M,M') NR2F1 level as measured on single cells in the VZ (M) and CP (M') of sulci and gyri n ≥ 4 sections from n = 1 fetal brain. (N-O) NR2F1 (red) and TBR2 (green) immunostaining NR2F1 level as measured in single TBR2+ IPs by pixel intensity is shown in graph (O). n ≥ 4 convolutions from n = 1 fetal brain (n = 2 sections). Data information In graphs, data are represented as means ± SEM. Student t-test (M,M',O (*P<0.05, ***P<0.001). (P-R) NR2F1 (red) and HOPX (green) immunostaining Virtually all HOPX+ bRGs are also NR2F1+ (magnification in P'') and their number is greatly increased in gyri (P'') compared to sulci (P'), as quantified in (R). At a single cell level, NR2F1 pixel intensity is higher in oSVZ HOPX+ cells in gyri compared to sulci (Q). n ≥ 4 sections from n = 1 fetal brain. Data information: Nuclei (blue) were stained with DAPI. the number of positive cells was quantified in 100µm-width boxes, randomly placed across the cortex. In graphs, data are represented as means ± SEM. Student t-test Q,R) (*P<0.05, ***P<0.001). oSVZ: outer subventricular zone \n",
      ".Figure 6(A) Real time RT-PCR analysis of Pax6 expression in neurospheres with different genotypes, as indicated. n = 3 cortices. N KO P HET: Nr2f1 KO, Pax6 HET. Data information: Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001). (B) Representative images of Nr2f1 KO Pax6 HET neurospheres obtained from E15.5 neocortices and cultured in vitro. Scale bars: 50µm. (C) Graph showing the number of neurospheres per P100 cell plate, at different steps, as indicated. While Nr2f1 KO neurospheres (orange line) proliferate for a longer time compared to WT ones (blue line), the loss of one Pax6 allele (gray line) almost restores normal proliferation rate and exhaustion time. Complete Pax6 loss is not compatible with stem cell renewal (yellow line). n ≥ 3 culture wells from n = 2 independent batches. Data information: Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001). (D-E) Paired cell analysis after neurosphere dissociation and 24-hour culture at clonal density. Couples of dividing cells were identified by Map2 (red; N, neurons) and BLBP (green; P, progenitors) IF. Pie charts in (E) show the proportion of P-P (proliferative; D), P-N (asymmetric differentiative; D') or N-N (symmetric differentiative; D'') couples in WT, Nr2f1KO and Nr2f1KO Pax6HET animals. n ≥ 3 samples from n = 2 culture batches. Data information: Nuclei (blue) were stained with DAPI. Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001 Scale bars: 50µm. (F-I) Pax6 (green) and Nr2f1 (red) IF in the posterior-most region of E12.5 cortex, showing Pax6 upregulation in Nr2f1 KO animals (G,G') compared to WT (F',F''). Normal Pax6 levels can be restored by loss of one Pax6 allele (Nr2f1 KO, Pax6 HET; H,H'). n ≥ 3 brains. N KO: Nr2f1 KO; P HET: Pax6 HET. Data information: Nuclei (blue) were stained with DAPI. Scale bars: 50µm. Pixel intensity quantification of Nr2f1 (blue) and Pax6 (orange) is shown in (I). n ≥ 3 brains. N KO: Nr2f1 KO; P HET: Pax6 HET. Data information: the pixel intensity or the number of positive cells was quantified in 100µm-width boxes, randomly placed across the LP. Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001). (J) Cell cycle duration at E12.5 in different mutants as indicated, quantified in posterior regions with double EdU/BrdU injection protocol n ≥ 6 sections from n = 2 brains. Data information: the pixel intensity or the number of positive cells was quantified in 100µm-width boxes, randomly placed across the LP. Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001). (K) Insets showing representative γ-Tubulin (green) and Phospho-Vimentin (red) IF of the cortical VZ surface to evaluate the orientation of the cleavage plane of dividing aRGs. Pie charts show the percentage of vertical (green), oblique (yellow) and horizontal (red) division planes of mitotic figures in the E12.5 lateral pallium of animals with different genotypes, as indicated. n ≥ 2 brains per genotype. Data information: Nuclei (blue) were stained with DAPI. Data are represented as means ± SEM. 2-way ANOVA ; *P<0.05, **P<0.01, ***P<0.001). cale bars: 50µm. \n",
      ".Figure 3(A-D) Ki67 (green; mitotically active progenitors) and Tuj1 (red; post-mitotic neurons) IF at E14.5 (A,A'), E16.5 (B,B') and E18.5 (C,C') of lateral pallia of wild-type (WT) versus mutant (KO) brains. Ki67+ proliferating cells (quantified in D) accumulate in mutant cortices. n ≥ 3 brains. Data information: Nuclei (blue) were stained with DAPI. the number of positive cells was quantified in 100µm-width boxes, randomly placed across the LP. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. CP: cortical plate; IZ: intermediate zone; SVZ: subventricular zone; VZ: ventricular zone. (E-F') Tbr2 (green; IPs) and Pax6 (red; radial glia cells) IF of E13.5 (E,E') and E17.5 (F,F') lateral pallia of WT and KO brains. Data information: Nuclei (blue) were stained with DAPI. Scale bars: 50µm. CP: cortical plate; IZ: intermediate zone; SVZ: subventricular zone; VZ: ventricular zone. Quantification of different classes of cortical progenitors: apical radial glia cells (aRGs) as Pax6+ NPs in VZ (G), basal intermediate progenitors (IPs) as Tbr2+ cells in SVZ (H) n ≥ 3 brains. Data information: the number of positive cells was quantified in 100µm-width boxes, randomly placed across the LP. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). Quantification of different classes of cortical progenitors: , basal RGs as Pax6+ cells in outer SVZ (I). n ≥ 3 brains. Data information: the number of positive cells was quantified in 100µm-width boxes, randomly placed across the LP. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). (J-L) E12.5 dorsal brain views (J,J') and representative sections of posterior hemispheres (K,K'), showing a vesicle enlargement in mutant brains (K'). The extension of the ventricular surface was quantified along the A-P axis (L); red color code represents Nr2f1 gradient in WT brains. 12µm-thick sections were collected on series of 10 slides; consecutive measurement levels are 120µm apart from each other. n ≥ 3 brains. Data information: In graphs, data are represented as means ± SEM. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). (M) Schematic representation of different aRG cell divisions, as visualized in vivo after co-in utero electroporation (IUE) of Tis21-RFP and Sox2-GFP plasmids into E12.5-old embryos. Green: symmetric proliferative divisions; Yellow: asymmetric differentiative divisions; Red: symmetric differentiative divisions. (N-P) GFP (green; expressed under Sox2 promoter) and RFP (red; expressed under Tis21 promoter) IF of E13.5 lateral pallia, 18-hours after IUE. The proportion of single or double positive NPs is shown in pie charts in (P). n = 3 electroporated brains. Data information: Nuclei (blue) were stained with DAPI. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). (Q-R') Ki67 (green; progenitors) and EdU (red) IF of E13.5 WT (Q,Q') and KO (R,R') embryos, injected with EdU at E12.5. Differentiating cells (EdU+Ki67-) are located in the IZ/CP; percentages shown in (Q',R') were obtained from n ≥ 4 sections of n ≥ 2 brains. Data information: Nuclei (blue) were stained with DAPI. Student t-test (Q',R'; E13.5, EdU injected at E12.5; WT/KO: *=0.03967 (*P<0.05, **P<0.01, ***P<0.001). Scale bars: 50µm. CP: cortical plate; IZ: intermediate zone; SVZ: subventricular zone; VZ: ventricular zone. \n",
      ".Figure 4(A-D) Graphs showing the average number of PH3+ mitotic figures in the posterior lateral pallium of WT and mutant cortices. The total number of mitoses (A) as well as the number of dividing cells in VZ (Pax6+Sox2+ aRGs; B), SVZ (Tbr2+ IPs; C) and outer (basal-most) region of SVZ (oSVZ/IZ; Pax6+ oRGs; D) are shown. For graphs (B-D), statistical analysis by 2-way ANOVA n ≥ 3 brains per age/genotype. Data information: the number of positive cells was quantified in 100µm-width boxes, randomly placed across the lateral pallium. In graphs, data are represented as means ± SEM. 2-way ANOVA *P<0.05, **P<0.01, ***P<0.001). SVZ: subventricular zone; VZ: ventricular zone. (E) Cell cycle duration at different embryonic ages (from E10.5 to E16.5) in WT (blue bars) or KO embryos (orange line), as quantified upon a double EdU/BrdU injection protocol in posterior-most LP n ≥ 3 brains per age/genotype. Data information: the number of positive cells was quantified in 100µm-width boxes, randomly placed across the lateral pallium. In graphs, data are represented as means ± SEM. 2-way ANOVA *P<0.05, **P<0.01, ***P<0.001). (F-G') Ki67 (green; NPs) and EdU (red) IF in WT (F,F') and KO (G,G') after 6 hours consecutive EdU incorporation. Note the higher number of labelled NPs in KO cortices (G') suggesting faster cell cycle progression and/or longer S-phase. Data information: Nuclei (blue) were stained with DAPI. (H) Best linear fit of EdU+Ki67+ NP percentage after 8 hours EdU cumulative labelling in WT (blue line) and KO (orange line) brains. The x- and y-intercept are proportional to the S-phase length and to the number of cycling cells, respectively, while the steepness of the line is proportional to cell cycle duration. WT: y = 11.66x +24.62; KO: y = 13.54x +32.96). n ≥ 2 brains per time point/genotype. Data information: the number of positive cells was quantified in 100µm-width boxes, randomly placed across the lateral pallium. In graphs, data are represented as means ± SEM. 2-way ANOVA *P<0.05, **P<0.01, ***P<0.001). (I-K) PH3 (green; dividing NPs) and Pax6 (red; aRGs) IF of E13.5 WT (I) and KO (J) lateral pallia. Strong and weak/punctate PH3 patterns show M-phase and G2-phase cells, respectively. G2-phase NP percentage is quantified in (K). As a result of shorter G1-phase and globally shorter cell cycle time, the percentage of G2-phase cells is increased in the KO NP pool. n ≥ 2 brains. Data information: Nuclei (blue) were stained with DAPI. the number of positive cells was quantified in 100µm-width boxes, randomly placed across the lateral pallium. In graphs, data are represented as means ± SEM. Student t-test (K; *P<0.05) Scale bars: 50µm. SVZ: subventricular zone; VZ: ventricular zone. (L) Cell cycle time of E13.5 NPs in the lateral pallia of WT (upper column) and KO cells (lower column), as calculated by EdU cumulative labelling and PH3 staining KO NPs have a 16.1% faster cell cycle time, due to a 30.9% shorter G1-phase compensating a 14.3% longer S-phase. n ≥ 3 brains. In graphs, data are represented as means ± SEM. 2-way ANOVA (*P<0.05, **P<0.01, ***P<0.001). \n",
      ".Figure 6(A) Overlay of right and left wings of a yki2-only-A female showing fluctuating asymmetry. (B) Quantification of fluctuating asymmetry index FA10 in female flies of the indicated genotypes (F-tests: * p-value<0.05; ** p-value<0.01; *** p-value<0.001; **** p-value<0.0001). (C) Distribution of wing size and wing asymmetry among the flies analyzed in panel B. Each circle represents the area of a single wing. Each wing was measured in duplicate. The difference between Right (orange) and Left (blue) sides of a single fly is schematized by a black line. \n",
      ".Figure 1(A) Number of genes identified by MAJIQ that display 20% or more variation in splicing between wild-type and B52 depleted cells, in each dataset. We identified 108 shared alternative splicing events, corresponding to 105 genes. A comprehensive list of the 108 events is given in Appendix Table S1. (B) High confidence protein-protein complexes identified by MIST (Molecular Interaction Search Tool) within the 105 genes identified in (A). Interactions between proteins can be direct or indirect within the complexes. (C) GO term enrichment analysis of the 105 genes identified in a, performed by PANTHER with Bonferroni correction for multiple testing. (D) Schematic representation of yki locus and its two isoforms: Yki2 (includes exon 3 and contains 2 WW domains) and Yki1 (skips exon 3 and contains 1 WW domain). (E and F) RT-PCR and western blot showing the effect of RNAi-mediated B52 depletion in S2R+ cells (E) and in larval wing discs (F). \n",
      ".Figure 2(A) Expression of diap-lacZ and ex-lacZ reporters upon depletion of B52 in the posterior part of the wing disc (hh-Gal4 driver). GFP expression labels the posterior domain. Arrows highlight the reduced expression of the reporters in the posterior domain. Scale bars: 100 µm. (B) Western blot showing overexpression level of V5-tagged Yki1 and V5-tagged Yki2 in total larvae, under the control of da-Gal4 driver, probed with V5 antibody. (C) Wing phenotype induced by depletion of B52 in the posterior domain of the wing disc combined with modulation of Hpo pathway. Dotted lines separate anterior (top) and posterior parts (bottom) of the wing. Flies were grown at 18°C. (D) Quantification of Posterior/Total wing area in male flies of the genotypes illustrated in panel C. UAS-GFP was used as a control. Each point represents a single wing, bars represent mean with standard deviation. *** p-value<0.001, **** p-value<0.0001 (unpaired two-tailed t-tests). Flies were grown at 18°C. \n",
      ".Figure 7(A) In wild type situation, yki alternative splicing allows production of a major fully active isoform Yki2, and a minor isoform Yki1 with reduced activity. This equilibrium contributes to developmental stability. (B) B52 depletion reduces yki exon3 inclusion level and therefore lowers Yki2/Yki1 ratio. The increased amount of Yki1 enhances its competition with Yki2 for binding to Sd, leading to a decrease of Yki target genes transcription. B52 depletion leads to reduced growth and increased developmental instability. (C) In yki2-only flies, absence of yki alternative splicing causes unbalanced expression of Yki2 isoform which leads to premature pupariation and to increased developmental instability. \n",
      ".Figure 5(A) Structure of the yki2-only allele compared to wild type yki locus. Introns surrounding exon 3 are removed. This allele does not contain any exogenous sequence. (B) Western blot of adult males and females showing disappearance of Yki1 isoform in the two yki2-only fly lines. (C) Comparison of posterior/total area ratio of females wings upon depletion of B52 in a yki+ (wild type) or yki2-only background. Each point corresponds to a single wing, bars represent mean with SD (unpaired two-tailed t-tests: **** p-value <0.0001). (D) Analysis of embryonic lethality according to maternal and paternal genotypes. Bars represent mean and error bars represent SD. Numbers of counted embryos are indicated in each column. Unpaired two-tailed t-tests: **** p-value <0.0001. (E and F) Quantification of sibling clones size in the wing pouch. In (D), sibling wild-type clones are labelled with 2xGFP and 2xRFP. In (E), unlabelled clones are homozygous for the yki2-only-A allele whereas sibling clones (RFP/RFP) are wild type. In both graphs, clones pairs are sorted according to RFP/RFP clones size. \n",
      ".Figure 3(A) Comparison of Yki isoforms activity in luciferase assays using UAS-luciferase or 2xSd-Luciferase reporters in S2R+ cells. Error bars represent SD, n=3. *** p-value<0.001; **** p-value<0.0001 (unpaired two-tailed t-tests). (B) Example of immunostaining used to quantify expression of ex-lacZ in wild type or upon Yki1 or Yki2 overexpression in wing disc posterior domain. GFP labels the posterior domain. Scale bars: 50 µm. (C) Quantification of Yki reporter genes expression upon overexpression of Yki isoforms in the wing posterior domain (hh-Gal4 driver). For each target gene, relative expression between posterior and anterior domain in the wing discs was quantified by immunostaining using anti-ßgal antibody (for ex-lacZ and diap1-lacZ) or direct visualization of GFP (ban-sensor, in this case posterior domain was visualized by immunostaining against V5-tag present in UAS-Yki transgenes). In the scatter dot plot, each symbol (circle, square and triangle) represents a single wing disc, bars represent mean with SEM. ** p-value<0.01; **** p-value<0.0001 (unpaired two-tailed t-tests). \n",
      ".Figure 4(A) Principle of dual-luciferase co-IP. In all experiments IP is performed against the Flag-tag fused to the Firefly Luciferase (FFL). FFL (unfused to Yki) is used as reference to normalize each interaction. The prey protein is fused to Renilla Luciferase (RL) in C-term for most proteins or in N-term (for Wbp2 and Mor). (B) Drawing of the monitored protein-protein interactions. Interactions previously shown to involve WW domains are indicated by orange arrows, whereas other interactions are schematized by blue arrows. (C) Comparison of Yki isoforms binding capabilities by dual-luciferase co-IP. Graphs show the normalized co-IP (NcoIP) results for 13 proteins tested against the two Yki isoforms and the negative control (Flag-Firefly). (D) Interaction between Sd and Tgi in absence or presence of Yki isoforms. (E) Interaction between Sd and Wbp2 in absence or presence of Yki isoforms. (F) Interaction between Tgi and Wbp2 in absence or presence of Yki isoforms. In C, D, E and F, bars represent mean and error bars represent SD, n=4. p-values: *P < 0.05, **P < 0.01, ***P<0.001, ****P<0.0001 (unpaired two-tailed t-tests). (G) Luciferase assay using 3xSd-Luciferase reporter in S2 cells, transfected with single or combination of Yki isoforms. Error bars represent SD, n=6. * p-value<0.05; **** p-value<0.0001 (unpaired two-tailed t-tests). (H) Co-overexpression of Yki isoforms using GMR-Gal4 driver provides evidence of competition between Yki isoforms. \n",
      ".Figure 2(a) Growth assay using the ACT1-CUP1 reporter system. Yeast cells carrying the ACT1-CUP1 reporter were deleted of pseudouridylase genes (indicated on the left), and were then assayed for growth at 30°C on media containing various concentrations of [Cu2+] (indicated at the bottom). (b) Growth rescue by restoration of pseudouridylation. The wild-type strain [from (a)] or the snr81Δ pusΔ strain [also from (a)], which exhibited the most severe growth-deficiency phenotype, was transformed with an empty vector or a plasmid carrying a pseudouridylation guide RNA gene targeting either position 42, position 44, or both (indicated). The resulting cells were assayed for growth at 30°C on media containing various concentrations of [Cu2+] (indicated at the bottom). (c) Schematic representation of U2-pre-mRNA interaction. The U2 BSRR sequence and the pre-mRNA branch site sequence are shown. The arrows indicate several point mutations at the pre-mRNA branch site. Stem-loops I, IIa and IIb are also indicated. (d) Splicing assay using the wild-type and mutant ACT1-CUP1 reporters. RNAs isolated from the wild-type and the snr81Δ pusΔ strains (indicated on the top), which carry the wild-type ACT1-CUP1 reporter pre-mRNA or any of the mutant ACT1-CUP1 reporter pre-mRNAs [indicated, also see (c)], were used for splicing assay (primer-extension analysis). The un-spliced pre-mRNA, lariat intermediate, and spliced mRNA are indicated. A primer complementary to U6 was also used (as an internal control) in the assay, and the U6 band is indicated as well. In addition, the growth phenotype of each strain (in various concentrations of [Cu2+]) is also shown. Lanes 1-12, lanes 13 and 14, and lanes 15 and 16 are from separate gels. (e) Quantification of ACT1-CUP1 mRNA levels. Spliced ACT1-CUP1 mRNA levels were calculated relative to U6 in each lane. The quantification was based on three independent experiments. (f) Relative improvement/reduction for the mutant reporters. Relative improvement or reduction in splicing was calculated by normalizing the splicing efficiency (mRNA/U6) of each mutant reporter to that of WT reporter (set at 1). \n",
      "a, COS7 cells co-transfected with GFP–ATG14 and DsRed–ER or RFP–SEC61&bgr (an ER marker) were immunostained for the mitochondria marker TOMM20.f1a, COS7 cells co-transfected with GFP-ATG14 and DsRed-ER or RFP-SEC61b; (an ER marker) were immunostained for the mitochondria marker TOMM20. Scale bars, 2 µm. b, An immunogold electron micrograph of starved HeLa cells expressing GFP-ATG14 probed with anti-GFP antibodies (red arrows). Scale bar, 200 nm. c, Immunoblots of subcellular fractions isolated from HEK293 cells stably expressing GFP-DFCP1. Cyto, cytosol; micro, microsome; mito, mitochondria; PNS, post-nuclear supernatant. \n",
      "a, COS7 cells stably expressing yellow fluorescent protein (YFP)–ATG5 were transfected with RFP–SEC61&bgr and teal fluorescent protein (TFP)-mito, and immunostained for TOMM20.f2a, COS7 cells stably expressing yellow fluorescent protein (YFP)-ATG5 were transfected with RFP-SEC61b; and teal fluorescent protein (TFP)-mito, and immunostained for TOMM20. b, Time-lapse images of COS7 cells transfected with YFP-ATG5, RFP-SEC61b; and TFP-mito under starved conditions. c, d, Distribution of distances of YFP-ATG5 dots fromthe centre to the surface of the ER (c) and to the surface of mitochondria (d). e, f, Distances of YFP-ATG5 dots over their lifetime for data from c (e) and d (f), respectively. All scale bars, 2 µm. \n",
      ".Figure 1(a) Schematic representation of U2-pre-mRNA interaction. Partial sequences/secondary structures of vertebrate and S. cerevisiae U2 are schematically shown. Also shown are the base-pairing interactions between the U2 branch site recognition region (BSRR) and the pre-mRNA branch site (BS). Pseudouridines (Ψ) are indicated. Enzymes responsible for yeast U2 pseudouridylation at the three conserved sites are also indicated. (b) Temperature-sensitive growth assay. Yeast strains (wild-type or deletion mutants, indicated on the left) were transformed with either an empty vector or a plasmid carrying an artificial guide RNA gene targeting U2 pseudouridylation at position 42 and 44 (indicated on the top), and grown at various temperatures (indicated at the bottom). (c) Blockage of pseudouridylation by deletion of specific pseudouridylase genes. RNAs isolated from wild-type and pseudouridylase-deletion strains (indicated on the top) were used for U2 pseudouridylation assay (CMC modification followed by primer-extension). Primer-extension pauses/stops correspond to ψ sites (indicated on the left and right). Note: The above-background signal of ψ42 in the snr81Δ lane (and to some extent, in the snr81Δ pus7Δ lane) is likely caused by the presence of strong ψ44 band. This is a primer-extension artifact, which often occurs when there is a strong ψ signal in the neighboring position. Alternatively, it is possible that having ψ at position 44 allows for an unidentified enzyme other than snR81 to modify U42 (albeit inefficiently) (Perhaps Pus1, which is responsible for ψ44 formation, can be a bit \"processive\"). Lanes 1-8 and lanes 9-16 are from two separate gels. (d) Restoration of U2 pseudouridylation by an artificial box H/ACA RNA. RNAs isolated from various yeast strains (indicated on the top), including the snr81Δ pusΔ strain that was transformed with an artificial guide RNA targeting positions 42 and 44 (lanes 9 and 10), were assayed for pseudouridylation [see legend to (c)]. \n",
      ".Figure 3Figure 3. Synthetic lethality assay. The Prp5 domain structure, and the mutations in some of the domains (indicated) are schematized at the top. Each of these PRP5 genes (wild-type or mutants, plasmid-borne) was used to replace the chromosomal PRP5. The numbers in parentheses are the relative ATPase activities of the wild-type (SAT) Prp5 and some of the Prp5 mutants tested before (Xu Query, 2007). The U2 status, either containing Ψ42 and Ψ44 (WT-U2 and snR81Δ pus1Δ with pΨ42 or Ψ44) or lacking Ψ42 and Ψ44 (snR81Δ pus1Δ), is indicated (on the right). Upon transformation with the plasmids carrying PRP5 genes (wild-type or any of the mutants), cells were grown on the regular medium (-FOA) or on medium containing 5-FOA (+FOA). \n",
      "a, Correlative light and electron microscopy analysis of STX17 knockdown HeLa cells expressing GFP–LC3 (white arrows) infected with GAS (red arrows).f4a, Correlative light and electron microscopy analysis of STX17 knockdown HeLa cells expressing GFP-LC3 (white arrows) infected with GAS (red arrows). DAPI, 4′,6-diamidino-2-phenylindole. b, Electron micrographs of STX17 knockdown cells. White arrows denote autolysosomes, black arrows denote autophagosome, white arrowheads mark isolation membranes. c, Immunogold electron micrographs of STX17 knockdown HeLa cells stained for GFP-LC3. d, GFP-ATG5 dots (arrows) in STX17 knockdown HeLa cells. e, Quantification of staining in d. f, Immunoblot of p62 in STX17 knockdown cells. Tubulin was used as a loading control. g, Degradation of long-lived proteins in STX17 knockdown cells. St, starved; wor, wortmannin. All error bars denote s.d. Scale bars, 1 µm (a, b), 200 nm (c) and 5 µm (d). \n",
      "a, COS7 cells expressing GFP–SEC61&bgr and STX17 were immunostained for TOMM20.f3a, COS7 cells expressing GFP-SEC61b; and STX17 were immunostained for TOMM20. b, HeLa cells expressing GFP-ATG14 were immunostained for STX17. c, Immunoblot of subcellular fractions of HEK293 cells. d, STX17 knockdown COS7 cells expressing GFP-ATG14 and RFP-SEC61b; were immunostained for TOMM20. Error bars denote s.d. e, Immuoblot of subcellular fractions of STX17 knockdown (using two STX17 siRNAs, STX17-1 and STX17-2) HEK293 cells expressing GFP-DFCP1. f, Co-immunoprecipitation of STX17 and ATG14 using HeLa cells expressing Flag-ATG14 and Myc-STX17. g, Pull-down of ATG14 complex from HeLa cells expressing Myc-STX17 or Myc. Scale bars, 5 µm (a, b) and 2 µm (d). \n",
      ".Figure 2Application of selected reaction monitoring (SRM) mass spectrometry to quantify phosphorylated peptides in cell extracts treated with 1 μM flg22. Bars represent the mean ratio of endogenous phosphopeptide versus spiked-in synthetic phosphopeptide normalized to t = 0 with error bars ± SEM (n=3.). Asterisks indicate significant difference level compared to t = 0 (student t-test, *>0.05, **>0.01 and ***>0.001). The color of each bar corresponds to the different time points (0 minutes = dark blue, 5 minutes = red, 10 minutes = green, 20 minutes = purple, 30 minutes = light blue). Above each graph is the protein name and the phosphorylated residue (in brackets) is indicated and below the corresponding phosphopeptide is shown with the Serine (S), Threonine (T) or Tyrosine (Y) phosphorylation site indicated by \"[+80]\". \n",
      ".Figure 7Figure 7. Model for the function of Ψ42 and Ψ44 during spliceosome assembly. In wild-type, Ψ42 and Ψ44, the surrounding nucleotides, and perhaps the whole BSL structure (Perriman Ares, 2010), stimulate the interaction of U2 with Prp5 (perhaps through SF3b) and activation of Prp5's ATPase activity, resulting in a conformational change and Complex A formation. In the snr81Δ pus1Δ mutant, the absence of Ψ42 and Ψ44 leads to a U2 local structural change. The lack of Ψs coupled with the local structural change negatively impacts Prp5 binding and Prp5's ATPase activity. Consequently, Complex A formation or spliceosome assembly becomes less efficient. \n",
      ".Figure 3A) Immunoblot analyses showing MAPK phosphorylation after flg22 induction in Col-0 and in mkkk7. Protein extracts were made from seedlings treated with 1 μM flg22 and samples were taken at t=0, 10 and 30 min post induction. The p44/42 antibody was used to detect phosphorylated MAPKs. Position of the individual phosphorylated MAPKs is indicated at the right. Equal loading of proteins is shown with an α-Actin antibody as a loading control (bottom panel). Three biological replicates were done with identical results B) MPK6 phosphorylation is specifically enhanced in mkkk7. Comparison of phosphopeptide abundances from selected MAP kinases in Col-0 (blue) and mkkk7 (red) seedlings at t = 0 min and t = 10 min after 1 μM flagellin treatment by selected reaction monitoring (SRM) mass spectrometry. Phosphopeptide corresponding to MKKK7 are only detectable in Col-0 seedlings and are non-detectable (ND) in mkkk7. Bars represent means of measured peptide areas (sum of all transition areas) for three biological replicates, with error bars ± SEM (n=3). Asterisks indicate significant difference between Col-0 and mkkk7 at individual time points (student t-test, *>0.05, **>0.01 and ***>0.001). ND indicates integration of an area without transitions significantly above background. Above each graph the protein name and the phosphorylated residue (in brackets) is indicated as well as the corresponding phosphopeptide sequence. Serine (S), Threonine (T) or Tyrosine (Y) phosphorylation is indicated by \"[+80]\". \n",
      ".Figure 1Immunoprecipitation was performed with GFP-binding protein immobilized on magnetic beads using extracts of Arabidopsis seedlings, expressing either YFP-MKKK7 or Lti6B. Seedlings were treated with 1 μM flg22 for the indicated time. YFP-MKKK7 and Lti6B-GFP were detected with an anti-GFP antibody while FLS2 was detected with an FLS-specific antibody. Upper panel shows (co)-immunoprecipated proteins, lower panel shows input levels of protein. Arrowheads indicate the position of proteins of interest. \n",
      ".Figure 6(a) Native gel analysis of pre-splicing complexes. Pre-splicing complex assembly was carried out in the test tube with Prp5-reconstituted cell extracts and labeled pre-mRNA. Lane 1, Prp5-depleted wild-type cell extract. Lane 2, Prp5-depleted wild-type cell extract, plus wild-type Prp5. Lane 3, Prp5-depleted snR81Δ pus1Δ cell extract, plus wild-type Prp5. Lane 4, Prp5-depleted wild-type cell extract, plus mutant Prp5 (GAR). Lane 5, Prp5-depleted snR81Δ pus1Δ cell extract, plus mutant Prp5 (GAR). The pre-splicing complexes A and B are indicated. (b) DMS in vivo probing. After being exposed to DMS, yeast cells were lysed, total RNA collected, and primer-extension analysis carried out (left panel). Lane 1, cells that were not exposed to DMS. Lane 2, wild-type cells exposed to DMS. Lane 3, snR81Δ pus1Δ cells exposed to DMS. Lane 4, snR81Δ pus1Δ cells, transformed with a plasmid carrying an artificial guide RNA gene targeting positions 42 and 44, and exposed to DMS. Lane 5, snR81Δ pus1Δ cells, transformed with a plasmid carrying a control guide RNA gene with random guide sequences, and exposed to DMS. The nucleotides in parentheses are DMS-modified C and A residues that are one nucleotide after the actual primer-extension stops (indicated by the arrows). A partial U2 sequence (along with the BSL structure) is shown (right panel). DMS-modified adenosine and cytosine in the BSL are also indicated. (c) Residue accessibility quantification. Relative accessibility (relative band intensity) was quantified based on three independent experiments. The intensity of every band in each lane was normalized against the C46 band (the least changed band; compare lanes 2-5 in (b)) in its respective lane [(intensity of a band) / (intensity of C46 in the same lane)]. Each C46-normailzed band in lanes 3-5 (refer to (b)) was then further normalized against its respective counterpart in lane 2 [(the value of a C46-normalized band in lanes 3-5) / (the value of its counterpart in lane 2)]. The error bars represent the standard deviations of the measurements. \n",
      ".Figure 4A) Transient expression analysis in Arabidopsis mesophyll protoplasts shows enhanced defense gene expression in mkkk7 protoplasts after flg22 treatment. Protoplasts were isolated from 4 week old plants and transfected with pWRKY29:fLUC (WRKY29) or pFRK1:fLUC(FRK1) constructs together with 35S:rLUC, as indicated in the graph. Protoplasts were treated for 4 hrs with 10 µM flg22 or mock treated as indicated. The horizontal axis indicates the treatment while the vertical axis represents expression levels relative to the mock treated control sample, as fold induction. All measurements were normalized to the rLUC activity. Bars represent means ± STDEV (n=2). Experiment was repeated 6 times with similar results. B) WRKY29 transcripts measured by qRT-PCR in flg22 treated leaf material. Leaf strips of Col-0 and mkkk7 were treated with 1 µM flg22 for t=0, 1, 2, and 4h. WRKY29 transcripts were normalized against Ubiquitin transcript as described before [62]. Bars represent mean value and error bars show SE (n=3). (* p<0.05, ** p<0.01, student t-test). C) FRK1 transcripts measured by qRT-PCR in flg22 treated leaf material. Leaf strips of Col-0 and mkkk7 were treated with 1 µM flg22 fort=0, 1, 2, and 4h. FRK1 transcripts were normalized against Ubiquitin transcript as described before [62]. Bars represent mean value and error bars show SE (n=3). (* p<0.05, ** p<0.01, student t-test). \n",
      ".Figure 5(a) Prp5's ATPase activity assay using synthetic U2. The 5' sequence of S. cerevisiae U2 is shown (top). The three ψs (35, 42 and 44) as well as the branch site recognition sequence (underlined) are indicated. In the ATPase assay (bottom), no RNA (lane 3), or an equal amount of LiCl-precipitated RNA (lane 1) or tRNA (lane 2), a short U2 fragment (nts 1-76) (lane 4), a fully pseudouridylated short U2 fragment (nts 1-76) (lane 5), a long U2 fragment (nts 1-120) (lane 6), or a fully pseudouridylated long U2 fragment (nts 1-120) (lane 7), was used. The un-hydrolyzed ATP and hydrolyzed product ADP are indicated. Lanes 1 and 2, lanes 3-5, and lanes 6 and 7 are from separate gels. (b) Quantification of Prp5's ATPase activity activated by in vitro transcribed U2. Prp5's ATPase activity shown in (a) was calculated using the formula (ADP) / (ADT+ATP). The numbers at the bottom of (b) correspond to the lane numbers of (a). Quantification was based on three independent experiments. (c) U2-Prp5 binding assay. FLAG-Prp5 and a radiolabeled short (nts 1-76) (lanes 1 and 2) or long (nts 1-120) (lanes 5 and 6) U2 fragment, or FLAG-Prp5 and a radiolabeled fully pseudouridylated short (nts 1-76) (lanes 3 and 4) or long (nts 1-120) (lanes 7 and 8) U2 fragment, were incubated under the conditions used for ATPase activity assay. Anti-FLAG IP was then carried out. Co-precipitated (lanes 2, 4, 6 and 8) and un-precipitated (lanes 1, 3, 5 and 7) U2 RNAs were analyzed by electrophoresis. The U2 band is indicated. (d) Quantification of Prp5-U2 binding. Relative intensities of U2 bands shown in (c) were calculated [setting the un-precipitated uridine-containing U2 (lane 1 and 5) to 1]. The numbers at the bottom of (d) correspond to the lane numbers of (c). Quantification was based on three independent experiments. (e) Prp5's ATPase activity assay using cellular U2. U2 RNA isolated from the wild-type strain (lanes 2 and 4) or from the snR81Δ pus1Δ strain (lanes 6 and 8) was used for the ATPase activity assay. In odd-numbered lanes, no RNA was added. Both the wild-type Prp5 (lanes 1-4) and the mutant Prp5 (GAR) (lanes 5-8) were tested. (f) Quantification of Prp5's ATPase activity activated by U2 from cells. Prp5's ATPase activity shown in (e) was calculated using the formula (ADP) / (ADT+ATP). The numbers at the bottom of (f) correspond to the lane numbers of (e). Quantification was based on four independent experiments. (g) U2 level in the ATPase activity reactions. RNAs from lanes 2 and 4 in (c) were recovered, and U2 level was measured using primer-extension. Lanes 1-5, a titration of U2 snRNA. Lane 6, U2 from the wild-type strain [lane 2 of (c)]. Lane 7, U2 from the snR81Δ pus1Δ strain [lane 4 of (c)]. \n",
      ".Figure 4(a) Prp5-U2 co-immunoprecipitation (IP). Wild-type (lanes 1 and 2) and snR81Δ pus1Δ (lane 3) strains, each containing a FLAG-tagged PRP5 gene (indicated on the top), were used for anti-FLAG IP in the absence (lane 1) or presence (lanes 2 and 3) of ATP. The precipitated Prp5 (WT Prp5-FLAG) as well as co-precipitated U1 and U2, detected by primer-extension, are indicated. (b) Quantification of co-precipitated U2 snRNA. For each lane [the WT lane and the snr81Δ pus1Δ lane, corresponding to lane 2 and lane 3 of (a), respectively], Prp5-co-precipitated U2 was normalized against co-precipitated U1 [(intensity of the U2 band) / (intensity of the U1 band)]. The snr81Δ pus1Δ lane was then further normalized against the WT lane [(the value of normalized U2 from the snr81Δ pus1Δ lane) / (the value of normalized U2 from the WT lane)]. Quantification was based on three independent experiments. (c) U2 expression level in wild-type and snR81Δ pus1Δ strains. Total RNA, isolated from the wild-type (lane 1) or snR81Δ pus1Δ (lane 2) strains, was used for primer-extension to measure the levels of U2 and U6. The U2 and U6 bands are indicated. d) Depletion of Prp5. Wild-type (lanes 1, 3 and 4) and snR81Δ pus1Δ (lanes 2 and 5) cells containing a FLAG-tagged PRP5 were lysed, and anti-FLAG IP was performed at a high concentration of salt. Cell extracts, before (lanes 1 and 2), and after [lanes 3 (mock), 4 and 5] anti-FLAG IP, were used for western analysis using anti-FLAG. The Prp5 band is indicated. (e) IP of reconstituted Prp5 (GAR)-U2. An equal amount of FLAG-tagged Prp5 (GAR) was added to the wild-type (lane 1) and snR81Δ pus1Δ (lane 2) cell extracts depleted of endogenous Prp5 [see (c)]. After the salt concentration was brought back to its original level, anti-FLAG IP was carried out. The Prp5 (GAR) band, detected by western, is indicated. The co-precipitated U1 and U2 bands, detected by primer-extension, are also indicated. f) Quantification of U2 that is co-precipitated with mutant Prp5 (GAR). For each lane [including the WT lane and the snr81Δ pus1Δ lane, corresponding to lane 1 and lane 2 of (e), respectively], U2 co-precipitated with Prp5 (GAR) was normalized against co-precipitated U1 [(intensity of the U2 band) / (intensity of the U1 band)]. The snr81Δ pus1Δ lane was then further normalized against the WT lane [(the value of normalized U2 from the snr81Δ pus1Δ lane) / (the value of normalized U2 from the WT lane)]. Quantification was based on three independent experiments. (g) Prp5-U2-pre-mRNA co-IP. Reconstitution was carried out as described in (d). Prior to IP, an equal amount of radiolabeled pre-mRNA was also added to each reaction. Anti-FLAG IP was then performed in the presence of ATP. The precipitated Prp5 (WT and GAR) band, detected by western, is indicated. The co-precipitated pre-mRNA, directly visualized after electrophoresis, is also indicated. The pre-mRNA input is also shown. In lanes 1 and 2, wild-type and snR81Δ pus1Δ cell extracts, depleted of Prp5, were reconstituted with the wild-type FLAG-Prp5. In lanes 3 and 4, wild-type and snR81Δ pus1Δ cell extracts, depleted of Prp5, were reconstituted with mutant FLAG-Prp5 (GAR). (h) Quantification of pre-mRNA co-precipitated with Prp5 and U2. For each lane in (g), pre-mRNA co-precipitated with Prp5 (Wild-type or GAR) was normalized against the Prp5 (WT/GAR) band in the same lane (detected by Anti-FLAG) [(intensity of pre-mRNA) / (intensity of WT/GAR Prp5)]. Lanes 2,3,and 4 (see (g)) were then further normalized against lane 1 (see (g)) [(the value of normalized pre-mRNA from lane 2, 3 or 4 in (g)) / (the value of normalized pre-mRNA from lane 1 in (g))]. The numbers at the bottom of (h) correspond to the lane numbers of (g). Quantification was based on three independent experiments. \n",
      ".Figure 2A. HeLa cell stably expressing GFP-LGN and transiently expressing RFP-H2B was treated with noscapine to induce chromosome misalignments. Cells were imaged every 5 minutes. The time is relative to NEB. The red arrows indicate cortical regions where the chromosomes are in close proximity to the cell boundaries, accompanied by the displacement of LGN from the cortex. B. Similar assay as described in supplementary figure 2D was carried out in the presence of a small molecule inhibitor against Aurora-B (ZM447439) in combination with the proteasome inhibitor MG-132 to keep cells arrested in mitosis and in B. with an inhibitor against Plk1 (BI2536). Drugs were added at the same timing as Hoechst, 30 minutes prior to imaging. \n",
      ".Figure 7A) Effect of 100 nM flg22 treatment on ROS burst measured in 5 week old plants of Col-0 and mkkk7. B) Effect of 100 nM flg22 treatment on the ROS burst measured in 5 week old plants of Col-0 and two independent inducible MKKK7AA transgenic lines. C) Effect of 100 nM flg22 treatment on the ROS burst measured in 5 weeks old plants of Col-0 and two independent inducible MKKK7DD transgenic lines. Graphs represent means with error bars ± SEM (n=24). The vertical axis represents the relative increase in ROS production (photon counts) after PAMP treatment. At least three biological replicate experiments were done with similar results. \n",
      ".Figure 5A) Transient co-expression of MKKK7 in Arabidopsis mesophyll protoplasts shows suppression of flg22-induced WRKY29 gene expression. Protoplasts were transfected with pWRKY29:fLUC, 35S:rLUC and indicated overexpression constructs of MKKK7 (OE-MKKK7, OE-MKKK7AA or OE-MKKK7DD) as indicated on the horizontal axis. Protoplasts were treated with 10 μM flg22 or mock treated for 4 hrs. All measurements were normalized to the rLUC activity and expression is relative to the mock treated control sample, shown as fold induction on the vertical axis. Results shown are means ± STDEV (n=2). At least two biological replicates were done with similar results B) Protein structure of MKKK7 and mutated versions of MKKK7 with the protein kinase domain shown in yellow and an ARM/HEAT repeat domain shown in blue. The position of the phosphorylated Serine residues is indicated with triangles and bold S below the protein structure. The red triangles indicate phosphorylated Serines that were targeted for mutagenesis or the corresponding phospho-mimic Aspartic acid. Blue triangles indicate the substitution with the non-phosphorylatable amino acid Alanine. Amino acid substitute versions of MKKK7 are shown below the wild-type. S, Serine; A, Alanine; D, Aspartic acid. \n",
      ".Figure 6A) Four-week-old seedlings were dipped into a suspension containing virulent Pst DC3000 and 72 h later the disease symptoms were scored. Data represents mean values ± SEM (n=20;***, p<0.001; paired t-test). Three biological experiments were done showing similar results. B) Quantification of bacterial growth in Arabidopsis lines Col-0, mkkk7 and p35S:MKKK7-GFP in the mkkk7 background. Four to five-week-old plants were pressure-infiltrated with virulent Pst DC3000 and at indicated time points 6 samples were harvested and bacteria re-isolated on selective media. The number of colony forming units (cfu/cm2) was determined at t=0, 2 and 3days post inoculation (dpi). Data represents mean values ± SEM (n=6; **, p<0.01; paired t-test). Experiments were done at least twice with similar results. C) Disease symptom development in Pst-infected lines with estradiol inducible constructs of ind-MKKK7AA L8, ind-MKKK7AA L10, ind-MKKK7DD L1 and ind-MKKK7DD L3. Two independent transgenic lines for each construct were grown under short-day conditions and disease symptoms were scored 3 dpi. Data represents mean values ± SEM (n=20; *, p<0.05; **, p<0.01; paired t-test). The vertical axis represents the percentage disease symptoms. Experiments were done at least twice with similar results. \n",
      ".Figure 1A. Histograms of spindle angle of GFP-H2B U2OS cells transfected with either GAPDH or Spindly siRNA and filmed on rectangular micropatterns. Cells were filmed 48 hours post-transfection. B. Mitotic timing of control-depleted U2OS cells or CLIP-170-depleted cells. CLIP-170-depeleted cells are split in two categories; 'aligned' and 'misaligned', relating to the observed chromosome misalignment phenotype C. The histograms of spindle angles for the two different categories of CLIP-170-depleted U2OS cells (Left graph with alignment defect, right graph without alignment defect). D and E. Mitotic timing and spindle angle histograms of control (GAPDH)-depleted U2OS cells and CENP-E-depleted cells. \n",
      ".Figure 3A. Immunofluorescence images and quantifications of KT-localized Plk1 in U2OS cells after PBIP1 depletion. Cells were either Mock transfected or with PBIP1 siRNA for 72 hours. Cells were treated with nocodazole for 1 hour and fixed. Cells were stained with the centromere marker ACA, Plk1 and DAPI. B. Representative images and histograms showing the effect of chromosome position on cortical LGN enrichment after 72 hours of Mock and PBIP1 depletion. C. Live cellimages and histograms of GFP-H2B U2OS cells Mock transfected or with siPBIP1 filmed on rectangular micropatterns in the presence of a CENP-E inhibitor. The spindle angles for both conditions were scored at 32 minutes after NEB. . D. The total spindle angle displacement within the first hour of mitosis was determined by measuring the spindle angle at each time point and calculating the sum of the Δ spindle angle between each time points. \n",
      ".Figure 3A. Immunoblot analysis of HCT116 cells transfected with indicated RADX siRNAs and RADX knockout cell lines (RADXΔ). B. Schematic for DNA fiber assays in (C-E). C. Cells in (A) were labeled with consecutive pulses of CldU and IdU as shown in (B). Replication fork speeds were calculated as length of labeled track divided by pulse time (bars, mean; n=400 fibers, pooled from two independent experiments, per condition). D. Bidirectional replication fork symmetry was calculated as percentage of shorter divided by longer tracks from (C). Concordance is 100%, representing equal rates of bidirectional elongation for both daughter forks (bars, mean; n=50 bidirectional forks, pooled from two independent experiments, per condition). E. Proportion of stalled forks (CldU-only tracks) among DNA fibers from (C) (bars, mean; 200 fibers analyzed per condition; n=2 independent experiments). F. Proportion of new origins (IdU-only tracks) among DNA fibers in (C) (bars, mean; 20 fields of view quantified per condition; n=2 independent experiments). G. Schematic for DNA fiber assays in (H) and (I). H. Proportion of stalled forks (CldU-only tracks) among DNA fibers from cells labeled as in (G) (bars, mean; 200 fibers analyzed per condition; n=2 independent experiments). I. Proportion of restarted forks (CldU- and IdU-positive tracks) among DNA fibers in (G) (bars, mean; 200 fibers analyzed per condition; n=2 independent experiments). \n",
      ".Figure 2A. Immunoblot analysis of chromatin fractions of U2OS cells transfected with indicated siRNAs and treated or not with HU. B. U2OS cells treated with HU in the presence or absence of ATR inhibitor (ATRi) were fractionated and immunoblotted with indicated antibodies. Relative RADX levels on chromatin were quantified and normalized to histone H3. See also Fig EV2A. C. Representative images of U2OS cells stably expressing indicated GFP-RADX alleles that were subjected to laser microirradiation, fixed with methanol/acetone immediately afterwards and immunostained with γH2AX antibody. D. As in (C), except cells were fixed with paraformaldehyde 20 min after DNA damage infliction and immunostained with PCNA antibody. E. Representative images from PLAs with GFP and RPA2 antibodies in U2OS and U2OS/GFP-RADX cell lines labeled with EdU for 30 min and then fixed or exposed to HU or CPT for 4 h before fixation. F. Quantification of data in (E) by QIBC (n≥3000 cells per condition; data from a representative experiment are shown). See also Fig EV2D. All scale bars, 10 μm. \n",
      ".Figure 4A. Replication fork speeds in U2OS or U2OS/GFP-RADX cells transfected with control (−) or RADX(#6) siRNA targeting the 3'UTR and labeled with CldU and IdU as in Fig 3B (bars, mean; n=400 fibers, pooled from two independent experiments, per condition). B. Fork symmetry in cells treated as in (A) (bars, mean; n=50 bidirectional forks, pooled from two independent experiments, per condition). C. Proportion of stalled forks (CldU-only tracks) among DNA fibers in (A) (bars, mean; 200 fibers analyzed per condition; n=2 independent experiments). D. Proportion of new origins (IdU-only tracks) among DNA fibers in (A) (bars, mean; 20 fields of view quantified per condition; n=2 independent experiments). E. Parental U2OS and derivative lines stably expressing GFP-RADX alleles or all three RPA isoforms at near-endogenous levels (U2OS/Super-RPA) (Fig EV4A) were transfected with control (−) or RADX siRNA. Replication fork speeds were determined as in (A) (bars, mean; n=400 fibers, pooled from two independent experiments, per condition). F. Replication fork speeds in HCT116 cells with indicated genotypes transfected with control (−), RPA1 (0.2 nM concentration [17]) or RAD51 siRNAs (Fig EV4B) and labeled with CldU and IdU as in Fig 3B (bars, mean; n=400 fibers, pooled from two independent experiments, per condition). G. Fork symmetry among DNA fibers in (F) (bars, mean; n=50 bidirectional forks, pooled from two independent experiments, per condition). H. New origin firing among DNA fibers in (F) (bars, mean; 20 fields of view quantified per condition; n=2 independent experiments). \n",
      ".Figure 1A. Domain organization of human RADX and RPA1, showing location of OB folds, including three DNA-binding domains (DBDs) in RPA1, and RADX mutants used in this study (see also Fig EV1A). B. Predicted in silico folding of OB folds 2 and 3 of human RADX, modeled by Phyre2, showing similarity to the structure of the region encompassing DBD-A and DBD-B in human RPA1 (Fig 1A). C. RADX interacts with ssDNA. Extracts of U2OS cells stably expressing GFP-RADX WT or mutants were incubated with biotin-coupled ssDNA oligo immobilized on Streptavidin beads, washed extensively and immunoblotted with GFP and RPA1 antibodies. D. ssDNA-bound Streptavidin beads incubated with cell extracts as in (C) were washed with buffer containing indicated salt concentrations prior to immunoblotting. E. Lysates of untransfected HCT116 cells were incubated with immobilized ssDNA or dsDNA probes as in (C) and immunoblotted with RADX and RPA1 antibodies. F. Immunoblot analysis of RADX expression in human cell lines. See also Fig EV1B. G. Representative images from in situ proximity ligation assays (PLAs) in BrdU-labeled U2OS and GFP/GFP-RADX cell lines (Fig EV1E,F) using GFP and BrdU antibodies under native conditions. Scale bar, 10 μm. H. Quantification of data in (F) by quantitative image-based cytometry (QIBC) (n≥3000 cells per condition; data from a representative experiment are shown). See also Fig EV1G. I. Soluble and chromatin-enriched fractions of U2OS cell lines stably expressing GFP-RADX alleles were immunoblotted with indicated antibodies. \n",
      ".Figure 1A. NS1 dimer with one subunit in grey and the other colored by domain (green, β-roll; orange, wing with magenta connector subdomain; blue, central β-ladder). N-linked glycosylation sites are shown in sticks, and the glycans are shown in spheres.B. Topology diagram for the NS1 monomer, colored as the pattern in (A). Glycosylation sites are indicated with red hexagons and disulfide bonds with yellow circles.C. Side views of the NS1 dimer from the wing (left) and the end of the β-ladder (right). The β-roll (green), connector subdomain (orange) and the intertwined loop (yellow) of the wing domain form a discontinuous protrusion on one face of the β-ladder with the spaghetti loop (cyan) on the other face. The wing domain is omitted from the left image for clarity. \n",
      ".Figure 5A. Schematic for replication fork protection assays in (B) and (C). B. Replication fork degradation (IdU/CldU ratio) in U2OS cells transfected with indicated siRNAs and processed as in (A) (bars, mean; n=200 fibers, pooled from two independent experiments, per condition). C. As in (B), except fork degradation was analyzed in U2OS/GFP-RADX cell lines. D. U2OS or U2OS/GFP-RADX cells transfected with indicated siRNAs and exposed to HU for 4 h were co-immunostained with RPA1 and γ-H2AX antibodies and analyzed by QIBC. Proportion of cells displaying maximal RPA chromatin loading accompanied by H2AX hyperphosphorylation (Fig EV5A), reflecting replication fork catastrophe [17], is indicated (bars, mean; n=5 independent experiments (≥3000 cells analyzed per condition); *P≤0.05,**P≤0.01,***P≤0.001,****P≤0.0001, unpaired t-test). E. As in (D), but using U2OS or U2OS/Super-RPA cells exposed or not to HU (n=6 independent experiments). See also Fig EV5B. F. Clonogenic survival of U2OS cells transfected with indicated siRNAs and subjected to different CPT doses for 24 h (mean±SEM;n=3 independent experiments). LQ (Linear Quadratic) model was fitted to the fractional survival data, using non-linear least square method. Overlap between confidence intervals of the fitting coefficients was used to evaluate the statistical difference between cell survival after siRADX and siCTRL treatment (*No overlap of 95% confidence interval with siCTRL). G. As in (F), except cells were treated with indicated doses of HU (mean±SEM;n=3 independent experiments). H. As in (F), using U2OS and U2OS/Super-RPA cells (mean±SEM;n=3 independent experiments). \n",
      ".Figure 2The ZIKV NS1 hexamer is modeled by using the full-length DENV2 NS1 structure (PDB code: 4O6B) (Akey et al, 2014). Each monomer is colored by domains (green, β-roll; orange, wing; magenta, connector subdomain; blue, central β-ladder). The intertwined loop \"spike\" of ZIKV NS1 contributing to membrane association is colored in red and indicated by dotted lines. \n",
      ".Figure 3ABC. Electrostatic surface views of NS1 from DENV2 (PDB: 4O6B, A) and WNV (PDB: 4O6C, B) (Akey et al, 2014), as well as ZIKV©, showing diverse characteristics on certain regions. Surfaces are colored by electrostatic potential at neutral pH from -2 kT (red) to +2 kT (blue) using PyMOL (http://pymol.org/). \n",
      ".Figure 5A. B-factor putty representation of the intertwined loop structure. Shades of blue and a narrower tube indicate regions with lower B-factors, whereas orange to red and a wider tube indicate regions with higher B-factors. The figure was prepared using PyMOL (http://pymol.org/).B. The 2Fo-Fc electron density map for the intertwined loop (yellow) \"spike\" region residues contoured at 1.0 sigma is represented in blue.C. Detailed interactions of the highly conserved residues W115 and W118 with the neighboring residues to stabilize the conformation of the intertwined loop. W115 forms a hydrogen-bonding interaction with E74 from α2 helix. W118 inserts into the hydrophobic interspace between α1 and α2 helix, formed by P36, L42, V71, L75 and I78, and W118 also interact with P36 and D37 by hydrogen-bonding interactions. D. Sequence alignment of the intertwined loop region (residues 91 to 130) of different flavivirus NS1proteins. The intertwined loop \"spike\" residues of ZIKV NS1 contributing to membrane association are indicated by a dark blue arrowhead. Accession codes: ZIKV strain BeH819015, AMA12085; West Nile virus (WNV), Q9Q6P4; Dengue virus 1 Nauru/West Pac/1974(DENV1), P17763; Dengue virus 2 Puerto Rico/PR159-S1/1969 (DENV2), P12823; Dengue virus 3 Singapore/8120/1995 (DENV3), Q5UB51; Dengue virus 4 Thailand/0348/1991 (DENV4), Q2YHF0; Japanese encephalitis strain K94P05 (JEV), AAC02714; St. Louis encephalitis (SLEV), D8L537; Yellow fever 17D vaccine strain (YFV), CAA27332; Murray Valley encephalitis (MVEV), NP_051124; Tick-borne encephalitis (TBEV), NP_043135. \n",
      ".Figure 4A. The NS1 surface is colored using the ConSurf server (Ashkenazy et al, 2016) according to sequence conservation from the most conserved (dark magenta) to the most divergent (dark cyan) based on an alignment of NS1 sequences from 73 flaviviruses. B. Models of intertwined loop \"spike\" interacting with the cellular membrane. The disordered \"spike\" regions are indicated by dash lines in DENV2 and WNV NS1 structures, which consist of hydrophobic or positively charged amino acids. \n",
      "-MICU2 chimaera.Figure 1A, B Representative size-exclusion chromatography profiles of the MICU1-MICU2 complex and chimaera in 2 mM EGTA (A) and 2 mM CaCl2 (B) conditions. C Top (left) and side (middle and right) views of the overall MICU1-MICU2 chimaera structure with four molecules in a crossed arrangement. Ca2+-free and partial Ca2+-bound MICU1 structures are coloured in pink and magenta, respectively. The Ca2+-free and partial Ca2+-bound MICU2 structures are coloured in light blue and blue, respectively. The red, green, and orange spheres represent Ca2+, N-, and C-terminus. \n",
      "-MICU2 Ca2+-free and partial Ca2+-bound states.Figure 2A Close-up view of Ca2+-free MICU1-MICU2 interfaces I (left and middle) and II (right). The atomic interactions at the interface I consist of mainly electrostatic (left) and weak hydrophobic (middle) interactions, while interface II mainly contains hydrophobic interactions (right) B MICU1 interacts with MICU2 at two major interfaces. C Close-up views of the partial Ca2+-bound MICU1-MICU2 interface II. The atomic interactions at interface II consist of weak electrostatic (left) and strong hydrophobic (middle) interactions. The red, green, and orange spheres represent Ca2+, N-, and C-terminus. D The surface-rendered MICU2 hydrophobic pocket shows the interaction at interface II. \n",
      "-MICU2 with different numbers of Ca2+ ions bound.Figure 3A Structural alignment of Ca2+-free and partial Ca2+-bound MICU1-MICU2 heterodimers, as well as critical substructures with conformational changes indicated by numbered dashed circles. Middle and right, close-up view of the MICU2 EF-hand 1 (middle, circle 1) and MICU1 EF-hand pair 2 (right, circle 2) conformational changes upon Ca2+-binding. The important α-helixes are highlighted as cylinders. The red sphere represents Ca2+. B Structural alignment of Ca2+-free, partial Ca2+-bound and fully Ca2+-bound MICU1-MICU2 model heterodimers as well as important substructures with conformational changes are indicated by numbered dashed circles. Three different MICU1 and MICU2 molecules are individually shown in similar colours, and the colours changing from light to dark correspond to an increase in Ca2+ numbers. Circles 1, 2, and 3: close-up views of MICU2 EF-hand 1, MICU1 EF-hand pair 2 and MICU1 EF-hand 1 conformational changes upon Ca2+-binding, respectively. Important α-helixes are highlighted as cylinders. The red sphere represents Ca2+. C Conformational changes of the alkaline groove between T4L fusion Ca2+-free MICU1-MICU2 and partial Ca2+-bound MICU1-MICU2 heterodimers. Inset, KRKK residues in the alkaline groove rotate upon Ca2+ binding. D Electrostatic surface potential of the alkaline groove in partial Ca2+-bound structure. The gradient shown is from -3 (acidic, red) to +3 (basic, blue) kT/e. \n",
      " Model of the uniporter regulatory mechanism.Figure 5A Model of MCU Ca2+ uptake modulatory mechanism by interaction switch. In resting conditions, MICU1-MICU2 heterodimer associates with MCU pore to prevent Ca2+ transporting. In the stimulated condition, MICU1-MICU2 heterodimer enhances the interaction with EMRE by increasing the positive charge of alkaline groove and disassociates form MCU, resulting in opening the gate and activating the Ca2+ channel. Inset shows modelled interaction between the EMRE C-terminal poly-aspartate tail and MICU1 alkaline groove. B Probability model for MICU1-MICU2 heterodimer to recognize and interact with the channel. As the number of EMRE molecules increases, the interaction sites between MICU1-MICU2 heterodimer and channel increase. Consequently, the probability for MICU1 to recognize and interact with the channel is increased, enhancing the regulatory function of MICU1-MICU2. \n",
      " in vitro and EMRE peptide competition in cells.Figure 4A The binding of fluorescently (GFP) labelled MICU1 and mutants to EMRE peptide shown by MST in the presence of EGTA. B The binding of fluorescently (GFP) labelled MICU1 and mutants to EMRE peptide were analysed with MST in the presence of Ca2+. C Comparison of binding of MICU1 and EMRE peptide in both conditions. D MICU1-MICU2 heterodimer interacts with the EMRE peptide at a 1:1 ratio. E Carbachol (CCh)-induced representative traces of mitochondrial calcium uptake in HEK cells treated with mt-EMRE and scramble peptides. Solid lines, mean; shaded regions, ± SEM, n = 3. F Resting mitochondrial Ca2+ levels in mt-EMRE and scramble peptides treated cells, evaluated via ratiometric imaging of the mitochondrial targeted mt-riG6m. Mean ± s.d., student t-test, **p < 0.01, n = 3. Data information: Error bars for some points are too small to be visible. G Mean mitochondrial Ca2+ peaks response to 100 μM CCh. Mean ± s.d., student t-test, ***p ≤ 0.001, n = 3. Data information: Error bars for some points are too small to be visible. H Representative [Ca2+]c clearance traces of U2OS cells treated with PBS and mt-EMRE peptide. I Quantification of mitochondria Ca2+ uptake rate in response to 100 μM histamine. Mean ± s.d., student t-test, n.s., not significant, n = 4. Data information: Error bars for some points are too small to be visible. \n",
      ".Figure 2(a, b). Decreased Drp1S616 phosphorylation in PINK1-null primary neurons. Lysates of DIV6 neurons generated from PINK1KO and PINK1WT control mice were immune-detected with antibodies against either phospho (Ser616)-Drp1 (pS616) or Drp1. Tuj1 was detected as a loading control (a). Quantitation of pS616/Drp1 for each experiment is shown (b). Student's test. **p<0.01. Data was presented as mean ± SEM of three independent experiments. (c, d). TEM analysis of mitochondria in mouse substantial nigra. Sections of substantial nigra from 18-month-old PINK1KO and PINK1WT control mice were analyzed under TEM (c, left panels, scale bar=2 µm). Amplified images of mitochondria from dot line frames gated region are shown (c, right panels, scale bar=1 µm). Asterisks indicate abnormal elongated mitochondria. A quantitative analysis of mitochondria with perimeter >2µm is shown (d). Over 300 identifiable mitochondria (cristae and/or double membrane) per mice were randomly and blindly selected for perimeter measurement. Student's test. *p<0.05. Data was presented as mean ± SEM of three independent experiments. (e, f). Overexpression of Drp1WT, but not phosphorylation mutant Drp1S616A, suppresses mitochondrial fusion in dendrites of PINK1KO primary neurons. (e). DIV4 neurons generated from PINK1 WT control or PINK1KO mice were co-transfected mito-GFP with either pcDNA3.1 (3.1) or Drp1WT or Drp1S616A. Neurons were immuno-detected with an antibody against myc-tag (Myc) (red, to detect exogenous Drp1). Mito-GFP: green, to detect mitochondria. Representative images were shown, scale bar=25 μm. Magnified dendritic shafts gated by a dot line frame are shown (Zoom, right panels, scale bar=5 μm).(f). Quantification of Mito-index in dendrites of neurons from experiments of (e). 10 neurons were randomly selected and analyzed in each experiment. One-way ANOVA followed with Tukey's test. *p<0.05, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. \n",
      ".Figure 4(a-e). ATG5 is dispensable for PINK1-mediated mitochondrial fission. MEF cells derived from ATG5-null (ATG5KO) and their wildtype control mice (ATG5WT) were co-transfected Δ110-PINK1-GFP-FKBP /FRB-MTS with either PINK1WT or a kinase-dead mutant PINK1D384N. Cells were induced with 250 nM rapalog (Rapalog) for 2 hours to activate PINK1 kinase, followed by immunodetecting mitochondria (TOM20, red) and Δ110-PINK1-GFP-FKBP /FRB-MTS (PINK1-FKBP, green). Cells treated with solvent (DMSO) were used as a treatment control. Representative images are shown, scale bar=25 μm (a). Higher magnification images are also included (panels beneath the large cell images, scale bar=10 μm). Mitochondrial morphology in different transfections was quantified (b-e). Student's test. *p<0.05, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. For each condition, >100 cells were analyzed. (f-i). Drp1 is dispensable for recruitment of parkin to mitochondria. Drp1WT and Drp1KO HeLa cells were transfected with GFP-parkin. Cells were treated with 20 μM CCCP for 2 hours (f) followed by immunodetecting mitochondria (TOM20, red) and parkin (GFP-parkin, green). Nuclei were labeled with DAPI (blue). Cells treated with solvent (DMSO) were used as a treatment control. Representative images were shown, scale bar=25 μm. Cell with mitochondrial parkin (%) in different experimental group was quantified (g, i). One-way ANOVA followed with Tukey's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. For each condition, >100 cells were analyzed. (f-i). Drp1 is dispensable for recruitment of parkin to mitochondria. Drp1WT and Drp1KO HeLa cells were transfected with GFP-parkin. Cells were treated with 150 μM actinonin for 6 hours (h), followed by immunodetecting mitochondria (TOM20, red) and parkin (GFP-parkin, green). Nuclei were labeled with DAPI (blue). Cells treated with solvent (DMSO) were used as a treatment control. Representative images were shown, scale bar=25 μm. Cell with mitochondrial parkin (%) in different experimental group was quantified (g, i). One-way ANOVA followed with Tukey's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. For each condition, >100 cells were analyzed. \n",
      ".Figure 1Decreased Drp1S616 phosphorylation in PINK1-null HEK293 cells Lysates of PINK1-null (PINK1KO) and parkin-null (parkinKO) HEK293 cells (a) and their matched wildtype controls (WT) were immuno-detected with antibodies against either phospho (Ser616)-Drp1 (pS616), phospho (Ser637)-Drp1 (pS637) or Drp1. β-actin was detected as a loading control. Quantitation of pS616/Drp1 for each experiment is shown Two independent PINK1KO and parkinKO HEK293 cell lines (1, 2) were shown. Student's test. *p<0.05, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. Decreased Drp1S616 phosphorylation in PINK1-null MEF cells Lysates of PINK1-null (PINK1KO) and parkin-null (parkinKO) MEF cells (c) and their matched wildtype controls (WT) were immuno-detected with antibodies against either phospho (Ser616)-Drp1 (pS616), phospho (Ser637)-Drp1 (pS637) or Drp1. β-actin was detected as a loading control. Quantitation of pS616/Drp1 for each experiment is shown Student's test. *p<0.05, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. Decreased Drp1S616 phosphorylation in PINK1-null mouse substantial nigra tissues. Lysates of PINK1-null (PINK1KO) and parkin-null (parkinKO) mouse substantial nigra tissues (e), and their matched wildtype controls (WT) were immuno-detected with antibodies against either phospho (Ser616)-Drp1 (pS616), phospho (Ser637)-Drp1 (pS637) or Drp1. β-actin was detected as a loading control. Quantitation of pS616/Drp1 for each experiment is shown Two independent PINK1KO and parkinKO substantial nigra from two separate PINK1KO and parkinKO mice (1, 2) were shown. Student's test. *p<0.05, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. (g, h). Overexpression of PINK1, not PINK1 kinase mutants, reverses Drp1S616 phosphorylation in PINK1KO HEK293 cells. Lysates of PINK1KO and PINK1WT control (WT) HEK293 cells expressing PINK1 variants were immuno-detected with an anti-phospho(Ser616)-Drp1 antibody (pS616), an anti-Drp1 antibody (Drp1) and an anti-PINK1 antibody (PINK1). Cells with mock transfection (Blank) and transfected with an empty plasmid (3.1) were included as controls (g). Quantitation of pS616/Drp1 is shown (h). PINK1: wildtype PINK1; D384N: PINK1 kinase-dead mutant; G309D, pathogenic PINK1 mutant; ∆110: PINK1 mito-target deficient mutant. Student's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. (i, j). PINK1 deficiency does not affect expression and kinase activity of CDK1 and ERK1/2. Lysates of PINK1KO and PINK1WT control (WT) HEK293 cells were immuno-detected with an anti-phospho(Ser616)-Drp1 antibody (pS616), an anti-Drp1 antibody (Drp1), an anti-PINK1 antibody (PINK1), an anti-CDK1 antibody(CDK1), an anti-p44/p42 MAPK antibody (ERK1/2), an anti-phospho(Thr161)-CDK1 antibody (pThr161), and an anti-phospho(Thr202/204)-p44/p42 MAPK antibody (pThr202/204). β-actin was detected as a loading control (i). Quantitation of CDK1Thr161 phosphorylation (pThr161) and ERK1/2Thr202/204 phosphorylation (pThr202/204) was shown (j). Student's test. ns: no significance. Data was presented as mean ± SEM of three independent experiments. (k). Alignment of human Drp1S616 in different species. Alignment of human Drp1 (isoform 1) amino acids 612-620 with indicated species. Serine residual (S) corresponding to human Drp1Ser616 was labeled as red. (l, m). PINK1 phosphorylates Drp1S616 in vitro. Recombinant TcPINK1 was incubated with either GST-Drp1518-736 (WT) or GST-Drp1518-736, S616A (S616A) in the presence (+)/absence (-) of ATP. After heat inactivation, reactions were treated with (+)/without (-) shrimp alkaline phosphatase (SAP). Phosphorylation of Drp1S616 (pS616) was immuno-detected (l, upper panel). Total Drp1 (l, middle panel) and protein loading stained with Coomassie blue staining (l, lower panel) were shown. Quantitation of pS616/Drp1 ratio was shown (m). One-way ANOVA followed with Tukey's test. ***p<0.001. Data was presented as mean ± SEM of three independent experiments. (n, o). Label-free relative quantitative mass spectrum analysis of Drp1S616 phosphorylation. Representative spectra for the Drp1 phospho-peptide (S616) SKPIPMPA(p) SPQK and non-phospho-peptide SKPIPMPA(p) SPQK. Spectra obtained for peptides from PINK1WT HEK293 cells are shown (n). The relative ratios of the phospho-peptides containing phosphor-Ser616 to peptides containing Ser616 from PINK1WT and PINK1KO HEK293 samples were calculated and plotted (o). (p, q). Km and Kcat for PINK1-mediated phosphorylation of ubiquitin and Drp1. In vitro kinase assays were performed with different concentrations of ubiquitin (p) or Drp1 (q) with TcPINK1 at fixed ATP concentration (500 μM). Results were analyzed on phos‐tag gels and modeled to the Michaelis-Menten equation. The given graphs represent global fits to data collected from 3 sets of reactions for both Ub and Drp1 performed independently. Km: michaelis constant. Kcat: catalytic constant. Bars represent the mean ± SEM. \n",
      ".Figure 3(a-c). Drp1 is required for mitochondrial fission induced by PINK1. Drp1-null HeLa cells (Drp1KO) and their wildtype controls (Drp1WT) were co-transfected Δ110-PINK1-GFP-FKBP with FRB-MTS. Cells were induced with 250 nM rapalog for 2 hours to activate PINK1 kinase, followed by immunodetecting mitochondria (TOM20, red) and Δ110-PINK1-GFP-FKBP /FRB-MTS (Δ110-PINK1, green). Nuclei were labeled with DAPI (blue). Cells treated with solvent (DMSO) were used as a treatment control. Representative images were shown (a, upper panel, scale bar=25 μm). Higher magnification images are also included (a, lower panel, scale bar=10 μm). Mitochondrial morphology in different transfections was quantified. (b, c). Student's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. For each condition, >100 cells were analyzed. (d-f). Drp1S616 phosphorylation is required for mitochondrial fission induced by PINK1. Drp1KO HeLa cells were co-transfected Δ110-PINK1-GFP-FKBP /FRB-MTS with either a plasmid encoding Drp1WT or a Drp1 dephosphorylation mimic mutant Drp1S616A. Cells were treated with 250 nM rapalog for 2 hours to activate PINK1 kinase, followed by immunodetecting mitochondria (TOM20, blue), exogenous Drp1 (myc, red), Δ110-PINK1-GFP-FKBP /FRB-MTS (PINK1-FKBP, green). Cells treated with solvent (DMSO) were used as a treatment control. Representative images were shown, scale bar=25 μm (d). Higher magnification images are also included (d, Zoom, scale bar=5 μm). Mitochondrial morphology in different transfections was quantified (e, f). Student's test. **p<0.01, ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. For each condition, >100 cells were analyzed. (g, h). Targeting PINK1 kinase domain to OMM increases mitochondrial localization of phosphorylated Drp1S616. HEK293 cells co-transfected FRB-MTS with Δ110-PINK1-GFP-FKBP were treated with 250 nM rapalog (Rapalog). After 2hours treatment, cytosol protion (cyto) and mitochondria (Mito) were fractionated and immunobloted with antibodies against either phospho (Ser616)-Drp1 (pS616), Drp1 (Drp1), or GFP (PINK1). Tom20 and β-actin were detected as mitochondrial loading control and cytosol loading control, respectively (g). Quantitation of pS616/Drp1 in various conditions was shown (h). Student's test. **p<0.01, ns: no significance. Data was presented as mean ± SEM of three independent experiments. (i, j). Targeting PINK1 kinase domain to the OMM increases mitochondrial localization of Drp1. HeLa cells co-transfected FRB-MTS with Δ110-PINK1-GFP-FKBP (FKBP-PINK1) were treated with 250 nM rapalog (Rapalog) for 2 hours to activate PINK1 kinase. Cells treated with solvent (DMSO) were included as a control. Cells were immunostained with antibodies against either Tom20, Drp1 and GFP (PINK1-FKBP). Representative images were shown, scale bar=2 μm (i). Higher magnification images from gated box are shown (bottom panels). Arrows indicate Drp1 puncta. Mitochondria-associated (yellow) and non-associated Drp1(Red) are shown, scale bar=1 μm. Mitochondria-associated Drp1 puncta were analyzed by using an ImageJ plugin-Coloc 2 and expressed as the Manders' Colocalization Coefficients (MCC) (j). Three different regions from each cell and >15 cells for each experimental group were analyzed. Student's test. **p<0.01. Data was presented as mean ± SEM of three independent experiments. (k, l). PINK1 regulates S616 and S637 phosphorylation of Drp1 via independent mechanisms. DRP1KO HEK293 cells co-transfected FRB-MTS/Δ110-PINK1-GFP-FKBP (PINK1WT) with either myc-Drp1WT or myc-Drp1S616A were treated with 250 nM rapalog (Rapalog) for 2 hours to activate PINK1 kinase. Cells treated with solvent (DMSO) were included as a control. Cell lysates were immunoblotting with antibodies against either phospho (Ser616)-Drp1 (pS616), phospho-(Ser637)-Drp1 (pS637), Drp1 and GFP (to detect PINK1-FKBP) (k). β-actin was detected as a loading control. Quantitation of pS637/Drp1 in various conditions was shown (l). One-way ANOVA followed with Tukey's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. \n",
      ".Figure 5(a, b). Drp1WT and Drp1S616D, but not Drp1S616A, rescue PINK1-null induced crushed thorax in Drosophila. Representative thorax images of PINK1WT and PINK1-deficiency (PINK1KO) Drosophila expressing either mhc-gal4 (Mhc/+), mhc-gal4 driven human Drp1wt (Mhc>hDrp1WT) or mhc-gal4 driven human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven human Drp1S616D (Mhc>hDrp1S616D) are shown. Images were taken either under light microscopy (a, upper panels, yellow arrowhead indicates the collapsed throax) or SEM (b, lower panels). Quantitation of crushed thorax is shown (b). > 200 flies for each experiment are analyzed. Scale bar=200µm. One-way ANOVA followed with Tukey's test. ***p<0.001. Data was presented as mean ± SEM of three independent experiments. For each condition, >200 flies were quantified. (c) Drp1WT and Drp1S616D, but not Drp1S616A, restore abnormal ATP production in PINK1-null flies. ATP production was measured using muscle lysates generated from PINK1WT flies expressing mhc-gal4 (Mhc/+) (as a control) and PINK1KO flies expressing either mhc-gal4 (Mhc/+), mhc-gal4 driven human Drp1wt (Mhc>hDrp1wt) or mhc-gal4 driven human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven human Drp1S616D (Mhc>hDrp1S616D). One-way ANOVA followed with Tukey's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. Drp1WT and Drp1S616D, but not Drp1S616A, rescue mitochondrial structures in PINK1KO flies. Representative images of Mito-GFP (green) labeled mitochondria (d) from PINK1WT flies expressing mhc-gal4 (Mhc/+) (as a control) and PINK1KO flies expressing either mhc-gal4 (Mhc/+), Mhc-gal4 driven human Drp1wt (Mhc>hDrp1wt) or Mhc-gal4 driven human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven human Drp1S616D (Mhc>hDrp1S616D) are shown. Scale bar=10 μm (d) Average mitochondrial size was quantified, >3 flies/group and 3-6 pictures of different microscopic fields from each fly were analyzed per repeat. One-way ANOVA followed with Tukey's test. *p<0.05, ***p<0.001. Data was presented as mean ± SEM of three independent experiments (e). Drp1WT and Drp1S616D, but not Drp1S616A, rescue mitochondrial structures in PINK1KO flies. Representative images of TEM analysis (f) of IFMs from PINK1WT flies expressing mhc-gal4 (Mhc/+) (as a control) and PINK1KO flies expressing either mhc-gal4 (Mhc/+), Mhc-gal4 driven human Drp1wt (Mhc>hDrp1wt) or Mhc-gal4 driven human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven human Drp1S616D (Mhc>hDrp1S616D) are shown. Scale bar= 1 μm (f). Average mitochondrial size was quantified, >3 flies/group and 3-6 pictures of different microscopic fields from each fly were analyzed per repeat. Normal and abnormal mitochondrial number per mm2 were quantified. >3 flies/group and 3-6 pictures of different microscopic fields from each fly were analyzed per repeat. Bars represent the mean ± SEM of three independent experiments (g). (h, i). Drp1WT and Drp1S616D, but not Drp1S616A, suppress cell death in PINK1KO flies. Representative TUNEL staining (Red) images of IFM sections from PINK1WT and PINK1KO flies expressing either Mhc-gal4 (Mhc/+), Mhc-gal4 driven human Drp1wt (Mhc>hDrp1WT) or Mhc-gal4 driven human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven human Drp1S616D (Mhc>hDrp1S616D) are shown. Nuclei were counter-stained with DAPI (blue), scale bar=10 μm. Apoptotic cells (%) in different genotype flies were quantified (i). >5 flies/group and 3-6 pictures of different microscopic fields from each fly were analyzed per repeat.One-way ANOVA followed with Tukey's test. ***p<0.001. Data was presented as mean ± SEM of three independent experiments. (j). Expression and phosphorylation of hDrp1 in Drosophila. Fly muscle lysates generated from PINK1WT or PINK1KO flies expressing mhc-gal4 driven hDrp1WT (WT) and hDrp1S616A (S616A) were immunoblotted with antibodies against either phospho(Ser616)-Drp1(pS616) (to detect phosphorylated hDrp1S616), Drp1 (to detect both endogenous and exogenous Drp1), myc-tag (to detect exogenous Drp1). α-tubulin was detected as a loading control. Flies expressing mhc-gal4 (-) were included as an expression control. \n",
      ".Figure 2Permeabilized HEK293T cells expressing wild type (WT) or edited (H112Q or H112Y) OSCP subunit were suspended in KSCN-based media at pH 7.4 (left traces) or pH 6.5 (right traces) and PTP opening was measured as the decrease in absorbance at 540 nm. Where indicated (arrows) 0.1 mM Ca2+ in the absence of further additions (black traces) or in the presence of 0.5 mM DPC (red traces) or 2 µM CsA (blue traces) was added. The bottom panel reports the difference between the absorbance readings before the addition of Ca2+ and at the end of the swelling phase. Data are mean ± s.e. of 5 experiments (triplicate samples) for Ca2+ alone (black bars) and Ca2+ + DPC (red bars) and of 3 experiments (duplicate samples) for Ca2+ + CsA (blue bars). Solid bars, pH 7.4, hatched bars, pH 6.5 Mitochondria were prepared from cells, suspended in sucrose buffer at pH 7.4 (solid bars) or 6.5 (hatched bars) in the absence (black bars) or presence (blue bars) of 1.6 µM CsA, extracted with 1% (w/v) digitonin and immunoprecipitated with an F-ATP synthase antibody. The immunoprecipitates were subjected to 12% SDS-PAGE followed by Western blotting for CyPD and α subunit of F-ATP synthase. Each immunodetected band was analyzed by densitometry, the ratio between CyPD and α subunit was measured and expressed relative to the ratio obtained in absence of CsA at pH 7.4, which was taken as 100%. Data are average ± s.e. of 6 independent experiments. HEK293T cells were incubated in a balanced salt solution containing 2-deoxyglucose at pH 7.4 (solid bars) or 6.5 (hatched bars) and anoxia induced by incubation in a BD GasPak EZ Anaerobe Gas-generating Pouch System (BD Biosciences). Cell death was measured based on LDH release (average ± s.e. from 8 independent experiments). In all panels the p values calculated with the Student t test are shown (*p ≤ 0.05, **p ≤ 0.01, ***p ≤ 0.001). \n",
      ".Figure 7(a-f). Pedigrees of PD families with PINK1 mutations and Sanger sequencing conformation. Pedigrees of three families with four PD patients (a, c, e) and corresponding sequencing chromatogram of four PD patients harboring PINK1 mutations (b, d, f) are shown. Red arrows indicate the mutations in each patient. (g, h). Reduction of Drp1S616 phosphorylation in dermal fibroblasts derived from PD patients with PINK1 mutations. Lysates of cells derived from three unrelated normal control individuals (Control, C) and four familial PD patients with PINK1 mutations (PINK1mutant) were immunoblotted with antibodies recognizing either phosphor-Drp1S616 (pS616) or Drp1 protein. β-actin was detected as a loading control (g). A quantitative analysis of phospho-Drp1S616 (pS616/Drp1) is shown (h). Student's test. **p<0.01, ns: no significance. Data was presented as mean ± SEM of three independent experiments. Drp1S616 phosphorylation in dermal fibroblasts derived from sporadic PD patients. Lysates of cells derived from five unrelated normal control individuals (Control, C) and seven unrelated sporadic PD patients (Sporadic PD, SP) were immunoblotted with antibodies recognizing either phospho-Drp1S616 (pS616) or Drp1 protein. β-actin was detected as a loading control (i). Drp1S616 phosphorylation in dermal fibroblasts derived from sporadic PD patients. A quantitative analysis of phospho-Drp1S616 (pS616/Drp1) in sporadic cases is shown (j). Student's test. *p<0.05. Data was presented as mean ± SEM of three independent experiments. \n",
      ".Figure 3Representative current traces recorded at V(pipette) +10 mV from mitoplasts derived from wild type (WT) or H112Q OSCP mutation-harboring cells. Channel activity (top) and corresponding amplitude histograms (bottom) recorded in wild type mitochondria before (WT pH 7.3) and after acidification to pH 6.5 (WT pH 6.5). Amplitude histograms were obtained from 120 second-long current traces for the corresponding pH values and the panels are from the same experiment. Recordings as in panel B from mitoplasts of cells harboring the OSCP H112Q replacement. Left panel, pH 7.3; middle panel, 440 seconds after acidification to pH 6,5; right panel, 30 seconds after the addition of 5 mM Ba2+ at pH 6.5. Amplitude histograms were obtained from 100 second-long segments and the panels are from the same experiment. Note that both in panels B and C well-visible subconductance states can be detected in the amplitude histograms (peaks between open levels o2 and o3 on left panels). Gaussian fitting has been obtained using the Origin Program set. In all panels c, closed; o1, o2, o3 open states with discrete subconductance. \n",
      ".Figure 1   Bovine heart mitochondria were incubated at the indicated pH in absence or presence of 200 μM DPC, treated with 1% (w/v) digitonin and F-ATP synthase was immunoprecipitated with an anti-F-ATP synthase Ab followed by 15% SDS-PAGE. β subunit and CyPD content were detected by Western blotting. The ratio between CyPD and the corresponding β subunit is reported in the lower part of the panel, referring to pH 7.4 values as 100%. Data are an average ± s.e. of 3 independent experiments.   B,C EDTA-SMP were solubilized at the indicated pH values, extracted with 1% (w/v) digitonin and subjected to BN-PAGE in order to separate dimers (Vd) and monomers (Vm), which were identified by Coomassie blue (panel B) or in-gel activity staining (panel C) and analyzed by densitometry, which is reported in the bottom part of each panel. Values report the dimer/monomer ratio, where the ratio at pH 7.4 was taken as 100%. Data are an average ± s.e. of 3 independent experiments. The p-values calculated with the Student t test are shown * p≤0.05 *** p≤0.001. D Sequence of peptides 95-113 of OSCP before (left panel) and after reaction with DPC (right panel) obtained from tryptic digests of the SDS-PAGE band corresponding to the expected molecular mass of OSCP in immunoprecipitated F-ATP synthase. Fragments of the series b and y identified in the LC-MS/MS analysis are indicated on the sequence of the peptides. H112 and H112CeT are indicated in red boldface. Ions y4-y17 show a mass shift of +72 Da in the modified peptide. \n",
      ".Figure 6(a, b). Drp1 rescues PINK1KO induced collapsed thorax phenotype in dATG7-deficient Drosophila. Representative SEM images of thorax from PINK1WT, (upper panels) and PINK1KO (lower panels) flies combined with dATP7 deficiency (dATG7KO) or their controls (dATG7WT), followed by expressing either mhc-gal4 (Mhc/+) alone, or Mhc-gal4 driven expression of human Drp1 (Mhc>hDrp1WT), mhc-gal4 driven expression of human Drp1S616A (Mhc>hDrp1S616A) or mhc-gal4 driven expression of human Drp1S616D (Mhc>hDrp1S616D) are shown (a). Quantitation of crushed thorax in flies with indicated genotypes is presented (b). > 100 flies for each experiment are analyzed. Scale bar=200µm. One-way ANOVA followed with Tukey's test. ***p<0.001. Data was presented as mean ± SEM of three independent experiments. (c, d) Drp1 rescues PINK1KO induced mitochondrial defects in dATG7KO Drosophila. Representative TEM images of indirect flight muscle from PINK1WT (upper panels) and PINK1KO (lower panels) flies combined with dATG7KO or their controls (dATG7WT), followed by expressing either mhc-gal4 (Mhc/+) alone, or Mhc-gal4 driven expression of human Drp1(Mhc>hDrp1WT), mhc-gal4 driven expression of human Drp1S616A(Mhc>hDrp1S616A) or mhc-gal4 driven expression of human Drp1S616D(Mhc>hDrp1S616D) are shown (c). Scale bar=1 μm. Normal and abnormal mitochondrial number per mm2 from each experimental group were and quantified. >3 flies/group and 3-6 pictures of different microscopic fields from each fly were analyzed per repeat. Bars represent the mean ± SEM of three independent experiments (d). (e). Drp1 rescues PINK1KO induced ATP reduction in Drosophila. ATP contents of thorax muscle tissues from the indicated genotypes were measured and normalized against the protein levels One-way ANOVA followed with Tukey's test. ***p<0.001, ns: no significance. Data was presented as mean ± SEM of three independent experiments. \n",
      ".Figure 4The left panel depicts a cryo-EM based atomic model of monomeric bovine F-ATP synthase (pdb 5ara) (excluding subunits e, f, g, DAPIT, 6.8PL and A6L) with OSCP (cyan) and subunit b (yellow) highlighted on the structure of F-ATP synthase in the lower part of the panel. The region H112-E115 is shown as van der Waals spheres (inflated for better visibility). The backbone of the stretch T130-K135 is shown in red van der Waals spheres. The top part of the panel shows the results of restrained molecular dynamics simulations of the isolated lateral stalk (OSCP and subunits b, d, F6) displaying a large number of conformations in this region. Snapshots taken at 1 ns intervals along the last 15 ns of the trajectory are shown for the fragment 112-HRGE-115 for the deprotonated (HIS) and the protonated form (HIS+). Only for protonated H112 a salt bridge with a well defined turn or helix arrangement at H112-E115 is found in a large number of MD conformations. The larger conformational variability of the deprotonated form can also be appreciated. The right panel reports a sequence comparison of OSCP across 16 out of 197 species (HRGE is boxed in red; Sauria sequences are in the thin black box). Conservation, quality of the aligment and consensus at each position in the alignment of all vertebrate sequences in RefSeq, excluding Sauria and most divergent sequences, are described in the documentation of the program JalView [67]. \n",
      "Increased E-cadherin:β-catenin levels in colonic crypts of wild-type mice.Figure 3Staining of small intestinal and colonic crypts for E-cadherin. Scale bar, 100 μm. Western blot of purified crypts from the small intestine and colon (N = 4 mice). qRT-PCR of whole tissue from small intestine and colon. Expression of mRNA (2(−dCt)) calculated relative to GAPDH (N = 3 mice). Statistics: one-sided Mann-Whitney U-test. Proximity ligation assay for E-cadherin and β-catenin. For each mouse (N = 6), at least 10 crypts per tissue were analysed to calculate the mean. Statistics: one-sided Mann-Whitney U-test; white scale bar, 50 μm. \n",
      "Activation of one allele of β-catenin in Lgr5-positive stem cells does not result in colonic lesions.Figure 2Table shows that only homozygous activation of β-catenin (Lgr5CreER Catnblox(ex3)/lox(ex3)) or loss of Apc (Lgr5CreER Apcfl/fl) in Lgr5-positive stem cells results in colonic lesions in cohorts, sampled when signs of sickness were apparent. There were no lesions observed when only one allele of β-catenin (Lgr5CreER Catnblox(ex3)/+) was mutated (chi-squared test, P < 0.001). Colonic lesions were scored from H&amp;amp;E by identification of disorganised epithelium. Staining for β-catenin and SOX9 confirmed activation of Wnt signalling in the observed lesions. Scale bar, 100 μm. \n",
      "Apc loss, GSK3 loss and homozygous mutation of β-catenin are sufficient to induce a rapid crypt-progenitor phenotype, but not a single β-catenin mutation.Figure 1Wild-type mice have small defined crypts in the small intestine with little nuclearβ-catenin at the bottom of the crypt. The small intestine of AhCreERAPCfl/fl, AhCreERCatnblox(ex3)/lox(ex3) (day 5) or AhCreERGsk3alphafl/fl betafl/fl (day 6) show the crypt-progenitor cell (CPC) phenotype with increased crypt size (red bar) and nuclearβ-catenin (arrows) along the crypt-villus axis. Scale bar, 100 μm. A heterozygous activation of β-catenin (AhCreER Catnblox(ex3)/+) shows no increase in crypt size or nuclear β-catenin at days 5-10. At day 15 post-induction, accumulation of nuclear β-catenin (arrows) becomes evident with a dramatic CPC phenotype at about day 25. Scale bar, 100 μm. Culture of small intestinal crypts of WT and VilCreER Apcfl/fl (or AhCreER Apcfl/fl) mice with/without R-spo1 shows the dependence of the Wnt agonist in WT organoids but not in Apc-deficient spheres. Representative photos were taken at day 5 in culture. Black scale bar, 50 μm. At day 5 post-induction, only crypts from AhCreERCatnblox(ex3)/lox(ex3) but not AhCreERCatnblox(ex3)/+ survive in culture without the addition of R-spo1. At day 10 post-induction, we observed a mixed phenotype of more organoid-like structures in AhCreERCatnblox(ex3)/+ compared to spheres from Catnblox(ex3)/lox(ex3)crypts in the first week of culture. Black scale bar, 50 μm. Quantification of organoids/spheres at day 10 post-induction (N = 2 or 3 mice per genotype, mean of 6 technical replicates per mouse). \n",
      "Human cancers, characterised by β-catenin mutation, are associated with reduction in E-cadherin levels.Figure 6Proximity ligation assay for E-cadherin:β-catenin on a tissue microarray of SPT patients. PLA dots/cells in normal and tumour tissue were counted. Each dot in the boxplot represents the average for a single patient. Staining for β-catenin confirms accumulation of nuclear β-catenin in tumour tissue. Representative pictures are shown. N = 18, statistics: Mann-Whitney U-test, P = 0.0009051. White scale bar, 50 μm. Correlation of several Wnt target genes with the expression of E-cadherin (CDH1) was analysed in 269 HCC patients (TCGA provisional), and linear regression line (blue) and confidence region (shaded) were added. \n",
      "E-cadherin saturates with mutant β-catenin over time.Figure 4Immunoprecipitation (IP) of E-cadherin from VilCreER Catnblox(ex3)/lox(ex3) mice, 24 h (day 1, top) and 48 h (day 2, bottom) after induction. The ratio of wild-type (WT, top lane) and mutant exon 3 β-catenin (Δex3, bottom lane) bound to E-cadherin (IP:Ecad) was calculated for each tissue. Graphs show average of experiments (N = 3 mice for each genotype and time point); error bars represent s.e.m. IP of E-cadherin from VilCreER Catnblox(ex3)/+ mice at day 4 and day 8. Graphs show average of experiments (N = 3 mice for each genotype and time point); error bars represent s.e.m. \n",
      "Model of single β-catenin mutation and interaction with E-cadherin in the small intestine and colon.Figure 7Model of single β-catenin mutation and interaction with E-cadherin in the small intestine and colonA, B Model of a single activating β-catenin mutation in the murine small intestine and the colon. In contrast to the small intestine (A), the increased levels of E-cadherin in the colon complexed with mutant β-catenin prevent its accumulation in the nucleus (B). \n",
      "Haploinsufficiency for E-cadherin in the presence of single allele β-catenin mutation leads to Wnt deregulation.Figure 5AhCreERCatnblox(ex3)/+ compared to AhCreER Catnblox(ex3)/+ Cdh1fl/+ at day 10 post-induction. Note the presence of colonic lesions in the colon of AhCreER Catnblox(ex3)/+ Cdh1fl/+ mice (arrows). Scale bar, 100 μm; red bar indicates proliferative zone (BrdU). Scoring of BrdU-positive cells per half-crypt in the small intestine of wild-type, AhCreERCdh1fl/+, AhCreERCatnblox(ex3)/+ and AhCreERCatnblox(ex3)/+Cdh1fl/+mice at day 10 post-induction. N ≥ 3, at least 25 crypts per mouse were scored. There was significantly higher proliferation in the AhCreERCatnblox(ex3)/+Cdh1fl/+mice (P = 0.028, one-sided Mann-Whitney U-test). In vitro growth of crypts (small intestine) from mice as indicated at day 5 post-induction without R-spo1. Quantification of sphere-forming efficiency of AhCreERCatnblox(ex3)/+AhCreERCatnblox(ex3)/+Cdh1fl/+, day 5 post-induction. N = 3 mice per genotype, mean of 6 technical replicates per mouse, P = 0.04 one-sided Mann-Whitney U-test. Survival of Lgr5CreERCatnblox(ex3)/+ and Lgr5CreERCatnblox(ex3)/+Cdh1fl/+ shows significant acceleration (P = 0.000123, log-rank test) after E-cadherin reduction. About 85% (6/7 mice) had colonic lesions, as identified with β-cateninIHC in contrast to Lgr5CreERCatnblox(ex3)/+mice. \n",
      ".Figure 1A. MS/MS analysis of peptides corresponding to amino acids 380-399 of STAT2. 293T cells expressing flag-tagged STAT2 were used. B. Abundance of T387 phosphorylated STAT2 in 293T cells expressing flag-tagged STAT2. C. Location of T387 in STAT2. D. Conservation of T387. \n",
      ".Figure 3A. HME cells expressing wild-type, T387A, or T387D STAT2 were seeded in 96-well plates at 8000 cells/well. The cells were exposed to vesicular stomatitis virus (VSV) for 2 h, with or without pre-treatment with IFN-β (100 IU/ml). After 20 h, the amounts of the VSV-M and STAT2 proteins were analyzed by the Western method. B, C. HME cells expressing wild-type, T387A, or T387D STAT2 were seeded in 96-well plates at 8000 cells/well. On the second day, the cells were treated with IFN-β (10 IU/ml) for 2 h. The cells were then exposed to different multiplicities of infection of VSV for 2 h, followed by fresh media containing 10 IU/ml of IFN-β. The cells were analyzed by staining with crystal violet (B) after 48 h. B, C. HME cells expressing wild-type, T387A, or T387D STAT2 were seeded in 96-well plates at 8000 cells/well. On the second day, the cells were treated with IFN-β (10 IU/ml) for 2 h. The cells were then exposed to different multiplicities of infection of VSV for 2 h, followed by fresh media containing 10 IU/ml of IFN-β. The cells were analyzed by the MTT assay (C) after 48 h. D. U6A cells expressing wild-type, T387A, or T387D STAT2 were placed into 96 well plates (2000 cells/well) and the cells were treated with IFN-β (1000 IU/ml) for 96 h. Cell survival was analyzed by the MTT assay. Each experiment was carried out two independent times, with results similar to the representative examples that are shown. \n",
      ".Figure 2A. U6A cells expressing wild-type or T387A STAT2 were treated with IFN-β (100 IU/ml). Cells were harvested after 4 h and total RNAs were analyzed by real-time PCR. Values are the means ± SD from three independent experiments. B. U6A cells expressing wild-type, T387A, or T387D STAT2 were treated with IFN-β (100 IU/ml). Cells were harvested after 4 h and total RNAs were analyzed by real-time PCR. Values are the means ± SD from three independent experiments. C, D. U6A cells expressing wild-type or T387A STAT2 were treated with IFN-β for 0, 4, 8, or 24 h. Total RNAs were analyzed by using an Illumina HumanHT-12 v4 Expression BeadChip array. The average signal for each probe was used to determine expression levels. Genes with average signals below 25 and detection P values greater than 0.01 in the untreated or treated cells were excluded from the analysis. Inductions of less than 2-fold were not scored. The Venn diagram shows the numbers of genes induced by 1.2 fold or more in U6A cells expressing T387A STAT2, relative to untreated control cells. The numbers in the table are fold changes calculated from the ratios T387A STAT2 treated / untreated and wild-type STAT2 treated / untreated. \n",
      ".Figure 4A. An immuno-precipitation was performed using whole cell lysates from cells expressing Flag-tagged wild-type STAT2, and samples were analyzed by the Western method. The numbers in the chart are the proportions of T387 phosphorylated STAT2 in each cell type, calculated in comparison with 293T cells. B. HME cells expressing wild-type STAT2 were treated with IFN-β (100 IU/ml). Whole cell lysates harvested at the times indicated were analyzed by the Western method; C. HME cells expressing wild-type STAT2 were seeded in complete medium. On the second day, the cells were re-fed with medium lacking supplements. On the third day, the cells were treated with each supplement for 4 h, and total lysates were analyzed by the Western method. 1. EGF, 2. Insulin, 3. Hydrocortisone, 4. GA-1000, 5. BPE. D. HME cells were pretreated with hydrocortisone (0.04mg/ml and 0.4mg/ml) for 3 h and then treated with IFN-β (100 IU/ml). Cells were harvested after 4 h and total RNAs were analyzed by real-time PCR. Values are the means ± SD from three independent experiments. E. Consensus sequences for CDK-catalyzed phosphorylation, compared to residues 383-391 of STAT2. F. HCC827 and LnCAP cells expressing wild-type STAT2 were treated with CDK inhibitors (1 µM) for 4 h and whole-cell lysates were analyzed by the Western method. 1. Flavopiridol, 2. AT7519, 3. Dinaciclib. \n",
      ".Figure 5A. U6A cells expressing wild-type or T387A STAT2 were treated with IFN-β (100 IU/ml) for 30 min or were untreated. The cells were washed with PBS and the media was replaced with fresh media containing staurosporine (500 nM). Whole-cell lysates were harvested and analyzed by the Western method; B. U6A cells expressing wild-type or T387A STAT2 were treated with IFN-β (100 IU/ml) for 4 h or were untreated. Whole-cell lysates were used for immunoprecipitations of IRF9 and Flag-STAT2. C. U2A cells expressing wild-type or T387A STAT2 were treated with IFN-β (100 IU/ml) for 4 h or were untreated. Whole-cell lysates were used for immunoprecipitation of Flag-STAT2. D. U3A cells expressing wild-type or T387A STAT2 were treated with IFN-β (100 IU/ml) for 4 h or were untreated. Whole-cell lysates were used for immunoprecipitation of Flag-tagged STAT2. E. EMSAs using an ISRE probe. Extracts from U6A cells expressing wild-type or T387A STAT2 treated with IFN-β for 1 h were analyzed. The position of ISGF3 is labeled. \n",
      "a) Non-fasting blood glucose level was monitored weekly in Atg7+/+, Atg7+/−, Atg7+/+-ob/ob and Atg7+/−-ob/ob mice.f2(a) Non-fasting blood glucose level was monitored weekly in Atg7+/+, Atg7+/−, Atg7+/+-ob/ob and Atg7+/−-ob/ob mice. ***P0.001; two-way ANOVA. (b) IPGTT was done by glucose injection at 16 weeks of age as described in the Methods (left), and AUC calculated (right). *P0.05, **P0.01, ***P0.001; Student's t-test (left) and one-way ANOVA (right). (c) HOMA-IR index was calculated as described in Methods. *P0.05; one-way ANOVA. (d) ITT was performed by insulin injection at 16 weeks of age as described in the Methods. *P0.05, **P0.01, ***P0.001; Student's t-test. (e) Tissue lysates from Atg7+/+-ob/w, Atg7+/−-ob/w, Atg7+/+-ob/ob and Atg7+/−-ob/ob mice were prepared before and 7 min after injection of 5 U kg−1 regular insulin (Ins) into the tail vein, and subjected to immunoblot analysis using antibodies specific for phospho-Akt S473 and total Akt. Numbers below immunoblot bands indicate the fold changes normalized to control bands. \n",
      "a) Atg7+/− mice were generated as described in the Methods.f1(a) Atg7+/− mice were generated as described in the Methods. PCR was done employing genomic DNA and primers flanking the floxed Atg7 regions. Arrows indicate wild-type Atg7 band in Atg7+/+ mice, increased size of PCR band due to inserted flox sequence in Atg7F/+ mice and decreased size of PCR band due to Cre-mediated deletion in Atg7+/− mice, respectively. (b) RT-PCR (left) and real-time RT-PCR (right) were performed using total RNA prepared from tissues of Atg7+/− or control Atg7+/+mice, and primers specific for Atg7 or β-actin sequence. Fold changes of the RT-PCR band intensities are shown (left). *P0.05, **P0.01, ***P0.001; Student's t-test, n=3. (c) Tissue lysate from fasted 12-week-old Atg7+/− and control Atg7+/+mice was subjected to immunoblot analysis using anti-LC3 or -p62 antibody. Fold changes of the immunoblot band intensities are shown (middle and right). **P0.01, ***P0.001; Student's t-test, n=3. (d) Cell lysate of Atg7+/+, Atg7+/− and Atg7−/− MEFs was subjected to immunoblot analysis using antibodies specific for LC3 and p62 after rapamycin (Rap) or control solvent (dimethyl sulfoxide, DMSO) treatment for 3 h. Fold changes of the immunoblot band intensities are shown (middle and right). *P0.05, **P0.01, ***P0.001; Student's t-test, n=3. (e) Sections of the liver and muscle tissues from fasted 12-week-old GFP-LC3+-Atg7+/+ and GFP-LC3+-Atg7+/− mice were subjected to fluorescent microscopy to examine GFP-LC3 puncta (left). The number of GFP-LC3 puncta was counted (right). Scale bars, 10 μm. *P0.05; Student's t-test, n=5. (f) Tissue lysates were prepared from the liver of fasted Atg7+/+ and Atg7+/− mice 4 h after leupeptin administration, and immunoblotting was done. Fold changes of the immunoblot band intensities are shown (right). *P0.05; Student's t-test, n=3. \n",
      "a) Tissue lysates from each type of mouse were subjected to immunoblot analysis using antibodies specific for total JNK, phospho-JNK, total IRS-1 and phospho-IRS-1 S307.f3(a) Tissue lysates from each type of mouse were subjected to immunoblot analysis using antibodies specific for total JNK, phospho-JNK, total IRS-1 and phospho-IRS-1 S307. (b) Liver sections from each type of mouse were subjected to immunohistochemistry using anti-nitrotyrosine antibody. Scale bars, 100 μm. (c) Tissue lysates were prepared from 18-week-old mice of each genotype, and subjected to immunoblot analysis using a kit detecting carbonylated proteins. (d) Immunoblot analysis was performed using anti-phospho-p53 and -p53 antibodies. p21 and β-actin mRNA expression was evaluated by RT-PCR. Numbers below immunoblot bands indicate the fold changes normalized to control bands. \n",
      "a) GFP-LC3 puncta were visualized by fluorescent microscopy of tissue from each type of mouse in fed state, and the numbers of GFP-LC3 puncta counted.f4(a) GFP-LC3 puncta were visualized by fluorescent microscopy of tissue from each type of mouse in fed state, and the numbers of GFP-LC3 puncta counted. Scale bars, 20 μm. *P0.05, **P0.01; Student's t-test. (b) Tissues of ob/w and ob/ob mice were subjected to EM, and the number of autophagosomes was counted. ***P0.001; Student's t-test. (c) Immunoblotting using anti-GFP or -p62 antibody. (d) Tissue lysates were prepared from the liver of fed ob/w or ob/ob mice 4 h after leupeptin administration, and immunoblotting was performed. Numbers below immunoblot bands indicate the fold changes normalized to control bands. (e) After labelling SK-Hep1 cells with C14-leucine, cells were treated with FFAs (600 μM PA or 1,200 μM OA). Percent release of radioactivity was measured for 3-6 h as described in the Methods. The difference between the percent proteolysis without E64d/pepstatin A/NH4Cl and that with E64d/pepstatin A/NH4Cl was regarded as lysosomal proteolysis. *P0.05, **P0.01; one-way ANOVA, n=3. (f) Atg7+/+ and Atg7+/− primary MEFs were loaded with a mixture of the indicated concentrations of PA and OA for 48 h, and TG content was estimated by ORO staining. **P0.01, ***P0.001; two-way ANOVA, n=4. \n",
      "a) The number of CLSs was counted in WAT of each type of mouse as described in the Methods.f5(a) The number of CLSs was counted in WAT of each type of mouse as described in the Methods. *P0.05; one-way ANOVA, n=3. (b) Total mRNA was extracted from each tissue, and RT-PCR was done using primers specific for Tnfa, Il6, F4/80 or pro-Il1b. The expression of pro-Il1b was determined using SVF fraction of WAT tissue. (c,d) Lysate of SVF was subjected to immunoblot analysis using anti-IL-1β (c) or anti-caspase-1 antibody (d). (e) Primary peritoneal macrophages were isolated, and treated with PA of the indicated concentrations in the presence or absence of LPS. ELISA was conducted to determine IL-1β concentrations in the culture supernatant. (BSA, bovine serum albumin) ***P0.001; two-way ANOVA, n=4. (f) After incubation of primary peritoneal macrophages with PA of the indicated concentrations with or without LPS for 16 h, NAD+/NADH ratio was determined using a commercial kit. *P0.05, **P0.01, ***P0.001; two-way ANOVA, n=5. (g) After the same treatment of peritoneal macrophages as in f, cells were incubated with MitoSOX to measure mitochondrial ROS by fluorescence-activated cell sorting (FACS) analysis. (h) After the same treatment of peritoneal macrophages as in f, cells were stained with MitoTracker Red and MitoTracker Green for FACS analysis to determine mitochondrial potential. The numbers in g,h indicate the percentages of cells in the designated gate. \n",
      "Lipid overload in autophagy-insufficient state (Atg7+/−) leads to increased accumulation of lipid due to defective lipophagy.f8Lipid overload in autophagy-insufficient state (Atg7+/−) leads to increased accumulation of lipid due to defective lipophagy. Autophagy insufficiency causes delayed turnover of mitochondria and mitochondrial dysfunction, which leads to augmented inflammasome activation when challenged with inflammasome activators such as lipid. Combined effects of increased lipid content, augmented inflammasome activation and their interaction lead to deteriorated insulin resistance and diabetes. \n",
      "a) Atg7+/− and control Atg7+/+ mice were fed HFD or normal chow diet (NCD), and non-fasting blood glucose levels were monitored.f6(a) Atg7+/− and control Atg7+/+ mice were fed HFD or normal chow diet (NCD), and non-fasting blood glucose levels were monitored. *P0.05, **P0.01; Student's t-test. (b) Fasting bloodglucose levels were determined after 18 weeks of HFD. *P0.05; one-way ANOVA. (c) IPGTT was performed after 18 weeks of HFD, (left) and AUC calculated (right). *P0.05; Student's t-test (left) and one-way ANOVA (right). (d) HOMA-IR was calculated after 18 weeks of HFD. ***P0.001; one-way ANOVA. (e) ITT was performed after 18 weeks of HFD (left), and KITT calculated (right). *P0.05; Student's t-test (left) and one-way ANOVA (right). \n",
      "a) One hour after intraperitoneal injection of 30 mg kg−1 leupeptin to C57BL/6 mice, 25 mg kg−1 of imatinib (Ima) was injected intraperitoneally.f7(a) One hour after intraperitoneal injection of 30 mg kg−1 leupeptin to C57BL/6 mice, 25 mg kg−1 of imatinib (Ima) was injected intraperitoneally. Three hours later, tissue lysate was prepared from the liver and subjected to immunoblot analysis. (b) Imatinib (25 mg kg−1), trehalose (2 g kg−1) or PBS was injected intraperitoneally into 12-week-old diabetic Atg7+/−-ob/ob mice 3 times a week, and the blood glucose level was monitored. ***P0.001, ###P0.001; two-way ANOVA. (c) IPGTT were performed after treatment of Atg7+/−-ob/ob mice with imatinib or trehalose for 8 weeks. #P0.05; ##P or **P0.01; ###P or ***P0.001; Student's t-test. (d) ITT were performed after treatment of Atg7+/−-ob/ob mice with imatinib or trehalose for 8 weeks. #P0.05; ##P or **P0.01; ###P or ***P0.001; Student's t-test. (e) Regular insulin (Ins) was injected into the tail vein of Atg7+/−-ob/ob mice that were treated with imatinib or PBS for 8 weeks. Seven minutes later, tissue lysates were prepared and subjected to immunoblotting. (f) Tissue lysates prepared from the same mice without insulin injection were subjected to immunoblotting. Numbers below immunoblot bands indicate the fold changes normalized to control bands. ('*' indicates comparison between imatinib and control: '#' indicates comparison between trehalose and control). \n",
      "Fig.Figure 1A Time-line of experiment procedures in the chronic social defeat stress. B Social interaction ratio in control, susceptible and resilient mice following the CSDS protocol (n = 44, 55, 44 mice). C Sucrose preference in control, susceptible and resilient mice (n = 20, 20, 12 mice. Treatment F2, 49 = 16.05, P < 0.0001, one-way ANOVA with Fisher's LSD test). D The immobility time in TST in susceptible mice, compared to control mice (n = 16, 32 mice. Susceptible vs control, P = 0.002, Student's t test). E The immobility time in FST in susceptible mice, compared to control mice (n = 17, 35 mice. Susceptible vs control, P = 0.0003, Student's t test). F, G The time in the open arms (F) and closed arms (G) in the EPM in susceptible mice, compared to control mice (n = 19, 33 mice. Susceptible vs control, for open arms, P < 0.0001; For closed time, P < 0.0001, Student's t test). H The central distance in the OFT in susceptible mice, compared to control mice (n = 13, 28 mice. Susceptible vs control, P = 0.0089, Student's t test). I The mRNA level of CD39, ENTPD2 and ENTPD3 in the hippocampus of susceptible mice after CSDS (n = 7, 8 mice. CSDS vs control, CD39 P = 0.0002, ENTPD2 P = 0.0025, Student's t test). J The CD39 protein expression in the hippocampus of susceptible mice induced by CSDS (n = 6, 7 mice. CSDS vs control, P = 0.0003, Student's t test). K The ATPase activity in the hippocampus of stressed animals (n = 6 mice/group. CSDS vs control, P =0.0041, Student's t test). \n",
      "Fig.Figure 2A Experimental timelines for apyrase administration and behavioral study. B The effects of apyrase (40, 80 and 160 U/ml) infusion into cerebral ventricle on social interaction time (n = 8, 10, 10, 10 mice. Interaction F3, 34 = 3.917, P = 0.0167; Target F1, 34 = 0.0004, P = 0.9834; Drug F3, 34 = 2.295, P = 0.0954; For target, Apyrase 80 U/ml vs control, P = 0.0008, Two-way ANOVA with Tukey's post-test). C The effects of apyrase (40, 80 and 160 U/ml) infusion into cerebral ventricle on sucrose preference (n = 8, 10, 9, 11 mice. Treatment F3, 34 = 2.87, P = 0.0505; Apyrase 80 U/ml vs control, P = 0.007, one-way ANOVA with Fisher's LSD test). D Experimental timelines for apyrase/ATP administration and behavioral tests. E The effect of apyrase (40 U/ml) infusion into hippocampal on the social interaction time (n = 14, 20 mice. Interaction F1, 32 = 7.0754, P = 0.0121; Target F1, 32 = 0.68447, P = 0.4142; Drug F1, 32 = 18.211, P = 0.0002; Apyrase vs control, P < 0.0001, Student's t test). F The effect of apyrase (40 U/ml) infusion into hippocampal on the sucrose preference (n = 10, 16 mice. Apyrase vs control, P = 0.0003, Student's t test). G The social avoidance behavior in mice with ATP intra-hippocampal infusion after apyrase exposure (n = 14, 10, 10 mice. Interaction F2, 31 = 2.051, P = 0.1457; Target F1, 31 = 2.666, P = 0.1127; Drug F2, 31 = 3.711, P = 0.0359; For Target, Apyrase-ACSF vs Vehicle, P = 0.0387; Apyrase-ATP vs Apyrase-ACSF, P = 0.0099, Two-way ANOVA with Tukey's post-test). H The sucrose preference in mice with ATP intra-hippocampal infusion after apyrase exposure (n = 10, 8, 8 mice. Treatment F2, 23 = 3.32, P = 0.0540; Apyrase-ACSF vs Vehicle, P =0.0466, one-way ANOVA with Fisher's LSD test). \n",
      "Fig.Figure 5A Experimental timelines for CSDS, ARL67156 treatment and immunostaining. B Immunostaining for BrdU (blue) and DCX (green) in the DG region of control, susceptible and ARL67156 treatment mice. Arrows indicate the BrdU+ DCX+ cells. Scale bar: 50 μm. C, D Quantification of BrdU+ cells (C) and BrdU+ DCX+ cells (D) in the DG of susceptible mice with ARL67156 compared with vehicle treatment (n = 4 mice/group. For BrdU+ Treatment F2, 9 = 10.07, P =0.0051; CSDS - ACSF vs Ctrl - ACSF, P = 0.0147; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0018; BrdU+ DCX+ Treatment F2, 9 = 9.005, P = 0.0071; CSDS - ACSF vs Ctrl - ACSF, P = 0.0211; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0024, one-way ANOVA with Fisher's LSD test). E Experimental timelines for neurogenesis examination with LV-siCD39. F Confocal images of BrdU (blue) and DCX (red) in mice treatment with LV-siCD39 in the DG. Arrows indicate the BrdU+ DCX+ cells. Scale bar: 50 μm. G, H Quantitative of BrdU+ (G) and BrdU+ DCX+ (H) cells in the DG of susceptible mice with LV-siCD39 compared with LV-GFP treatment (n = 3, 6, 6 mice. For BrdU+ Treatment F2, 12 = 15.79, P =0.0004; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.0012; CSDS - LV-siCD39 vs CSDS - LV-GFP, P = 0.0003; BrdU+ DCX+ Treatment F2, 12 = 12.79, P = 0.0011; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.05; CSDS - LV-siCD39 vs CSDS - LV-GFP, P = 0.0003, one-way ANOVA with Fisher's LSD test). \n",
      "Fig.Figure 3A Experimental timelines for CSDS, ARL67156 administration and behavioral study. B The ATPase activity of mice exposure to ARL67156 (n = 7, 7, 5, 5 mice. Treatment F3, 20 = 16.81, P < 0.0001; CSDS - ACSF vs Ctrl - ACSF, P < 0.0001; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0405. One-way ANOVA with Fisher's LSD test). C The social avoidance behavior of susceptible mice with intracerebroventricular infusion of ARL67156 (n = 12, 10, 9, 9 mice. Interaction F3, 36 = 5.119, P = 0.0047; Target F1, 36 = 0.005, P = 0.9437; Drug F3, 36 = 3.626, P = 0.022. For Target, CSDS - ACSF vs Ctrl -ACSF, P = 0.0001; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0054. Two-way ANOVA with Tukey's post-test). D The sucrose preference of susceptible mice with intracerebroventricular infusion of ARL67156 (n = 12, 10, 10, 10 mice. Interaction F1, 38 = 5.352, P = 0.0262; Group F1, 38 = 5.711, P = 0.0219; Drug F1, 38 = 9.703, P = 0.0035; CSDS - ACSF vs Ctrl - ACSF, P = 0.0083; Ctrl - ARL67156 vs CSDS - ACSF, P = 0.0027; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0031. Two-way ANOVA with Tukey's post-test). The social avoidance behavior (E) in mice with intra-hippocampal infusion of ARL67156 after CSDS (n = 9, 9, 9, 8 mice. For social behavior, Interaction F3, 31 = 5.579, P = 0.0035; Target F1, 31 = 0.5121, P = 0.4796; Drug F3, 31 = 8.367, P = 0.0003; CSDS - ACSF vs Ctrl - ACSF, P < 0.0001; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0131. sucrose preference (F) in mice with intra-hippocampal infusion of ARL67156 after CSDS (n = 9, 9, 9, 8 mice. Sucrose, Interaction F1, 30 = 0.3342, P = 0.5675; Group F1, 30 = 17.22, P = 0.0003; Drug F1, 30 = 0.05903, P = 0.8097; CSDS- ACSF vs Ctrl - ACSF, P = 0.0761. Two-way ANOVA with Tukey's post-test). G The immobility time in the TST of mice with intra-hippocampal infusion of ARL67156 after CSDS (n = 16, 12, 21, 19 mice. Treatment F 3, 64= 13.61, P < 0.0001; CSDS - ACSF vs Ctrl - ACSF, P = 0.0059; Ctrl - ARL67156 vs Ctrl - ACSF, P = 0.0011; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0002. One-way ANOVA with Fisher's LSD test). H The immobility time in the FST with intra-hippocampal infusion of ARL67156 after CSDS (n = 21, 12, 23, 19 mice. Treatment F 3, 71 = 12.94, P < 0.0001; CSDS - ACSF vs Ctrl - ACSF, P = 0.0036; Ctrl - ARL67156 vs Ctrl - ACSF, P = 0.0428; CSDS - ARL67156 vs CSDS - ACSF, P < 0.0001. One-way ANOVA with Fisher's LSD test).Data information: data are expressed as the means ± SEM. NS P > 0.05, *P < 0.05, **P < 0.01, ***P < 0.001. Ctrl, control. \n",
      "Fig.Figure 4A Experimental timelines for CSDS, virus stereotaxic injections and behavioral study. B Imaging of GFP expression in the hippocampus 3 weeks after virus vector injection. Scale bar: 250 μm. C Expression of CD39 in the hippocampus of mice injected with LV-siCD39 (n = 6 mice/group. LV-siCD39 vs Control, P = 0.0006, Student's t test). D The social avoidance behavior for the stressed mice after knocking down CD39 (n = 15, 15, 14 mice. Interaction F2, 41 = 17.96, P < 0.0001; Target F1, 41 = 3.75e-005, P = 0.9951; Drug F2, 41 = 6.099, P = 0.0048; For Target CSDS - LV-GFP vs Ctrl - LV-GFP, P < 0.0001; CSDS - LV-siCD39 vs CSDS - LV-GFP, P < 0.0001. Two-way ANOVA with Tukey's post-test). E The immobility time in the TST of mice injected with LV- siCD39 (n = 11, 19, 15 mice. Treatment F 2, 42 = 5.807, P = 0.0059; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.0069; CSDS - LV-siCD39 vs CSDS - LV-GFP, P = 0.0065. One-way ANOVA with Fisher's LSD test). F The immobility time spent in the FST of mice treatment with LV-siCD39 (n = 12, 19, 15 mice. Treatment F2, 43 = 5.237, P = 0.0092; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.0095; CSDS - LV-siCD39 vs CSDS - LV-GFP, P = 0.0093. One-way ANOVA with Fisher's LSD test). G The ATPase hyperactivity of mice after knocking down CD39 in the hippocampal (n = 5, 6, 5 mice. Treatment F2, 13 = 8.45, P = 0.0044; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.0016; CSDS - LV-CD39 vs CSDS - LV-GFP, P = 0.0166. One-way ANOVA with Fisher's LSD test). H The ATP level of mice after knocking down CD39 in the hippocampal (n = 5 mice/group. Treatment F2, 12 = 6.04, P = 0.0153; CSDS - LV-GFP vs Ctrl - LV-GFP, P = 0.0444; CSDS - LV-CD39 vs CSDS - LV-GFP, P = 0.0051. One-way ANOVA with Fisher's LSD test). \n",
      ".Figure 4A Simplified photomicrographs of proximity ligation studies in WT and Nox4KO MEFs, showing cell borders, nuclei (blue) and yellow dots corresponding to co-localization of proteins. Quantification of the number of dots/cell in each condition is shown to the right. Proximity was tested for the following protein couples: FACL4/Nox4, InsP3R/Nox4, and FACL4/InsP3R. Scale bars: 10 µm. n=3/group (with >30 cells/individual experiment). B Simplified photomicrographs of proximity ligation staining in rat cardiomyocytes transfected with Nox4 siRNA or a scrambled control (siScr), with quantification as in (A). Proximity was tested for the FACL4/Nox4 protein couple. Scale bars: 10 µm. n=3/group (with >30 cells/individual experiment). C Simplified photomicrographs of proximity ligation staining in hiPSC-CM (human cardiomyocytes) transduced with Nox4 shRNA or scrambled control, with quantification as in (A). Proximity was tested for the FACL4/Nox4 protein couple. Scale bars: 10 µm. n=3/group (with >30 cells/individual experiment). \n",
      ".Figure 1A, Quantification of Nox4 mRNA levels (A) in cultured rat cardiomyocytes after increasing degrees of serum starvation as compared to serum-replete cells (15% serum). n=4 cell preparations/group. B Quantification of Nox4 protein levels (B) in cultured rat cardiomyocytes after increasing degrees of serum starvation as compared to serum-replete cells (15% serum). n=4 cell preparations/group. A representative immunoblot for Nox4 and tubulin as a loading control is shown in panel B. C Cell death after 48 h serum starvation in cardiomyocytes in which Nox4 was depleted by an adenoviral shRNA (Ad.shNox4) as compared to cells infected with a control vector (Ad.Ctl). n=6/group. The representative immunoblot shows reduction in Nox4 protein levels after shRNA-mediated knockdown. D Lactate dehydrogenase (LDH) activity in cell supernatants. Staurosporine (Stauro, 1 μmol/L) was used to induce cell death by apoptosis. n=4 cell preparations/group. E Levels of high mobility group box1 protein (HMGB1) in cell supernatants. n=3/group. A representative immunoblot is shown at the top. F Protein levels of cytochrome c in the cytoplasmic fractions of cardiomyocytes. n=4/group. A representative immunoblot is shown at the top. The cellular membrane fraction (Mem) containing mitochondria was used as a positive control. G Percentage of Nox4-depleted and control cells with depolarized mitochondria, quantified by flow cytometry. n=3 cell preparations/group (100,000 cells per experiment). H Effect of cyclosporin A (CsA, 1 μmol/L) on the percentage of cells with depolarized mitochondria. n=6-8/group. I Effect of cyclosporin A (CsA, 1 μmol/L) on cell death in serum-starved cardiomyocytes. n=4/group. J Cell death in serum starved Nox4 knockout MEFs (Nox4KO) and wild-type MEFs (WT). Cat = PEG-catalase (500 U/mL). Nox4KO MEFs were transfected either with active Nox4 or a catalytic inactive Nox4 mutant, Nox4P437H (Mut). n=6-12/group. K HMGB1 levels in the supernatants of serum-starved MEFs in similar experiments to those in (J). n=4/group. A representative immunoblot is shown to the top. L Percentage of MEFs with depolarized mitochondria. n=6/group. \n",
      ".Figure 3A Immunoblotting of subcellular fractions from serum-starved rat cardiomyoblasts (H9c2 cells). Input: homogenate before fractionation; Cyto: cytosol; ER: endoplasmic reticulum; Crude M: crude mitochondrial fraction, also containing MAM; Pure Mito: mitochondria; MAM: mitochondria associated-ER membranes. Cytochrome c (Cyt c) was used as a mitochondrial marker, VDAC as an outer mitochondrial membrane marker, calnexin as an ER marker, Sigma1R and FACL4 as MAM markers, and tubulin as a cytosolic marker. B Immunoblotting of subcellular fractions from rat heart. C Immunoblotting of subcellular fractions from rat kidney. D Electron micrographs showing subcellular localization of Nox4 in wild-type (WT) and Nox4KO mouse hearts, and human induced pluripotent stem cell-derived cardiomyocytes (hiPSC-CM). hiPSC-CM were depleted of Nox4 using a lentiviral shRNA (Nox4KD) or infected with a control lentivirus. FACL4 was used as a MAM marker. Arrow heads show peri-mitochondrial localization of Nox4 and FACL4, in the proximity of ER cisternae and ribosomes. M=mitochondria, ER=endoplasmic reticulum, T=T-tubule. Scale bars: 100 nm. \n",
      ".Figure 2A Basal mitochondrial calcium levels assessed in serum-starved rat cardiomyocytes using a mitochondrial-targeted cameleon CFP/YFP FRET probe. Nox4 was depleted with silencing RNAs (siRNAs) or cells were transfected with a control scrambled siRNA (siScr). Representative photomicrographs are shown at the top. The spectrum color scale represents the ratio of emitted fluorescence (YFP/CFP). The mean changes in Nox4-depleted cells as compared to scrambled control (dotted line) at the bottom. The representative immunoblot shows depletion of Nox4 protein levels. Scale bars: 10 µm. n=3/group (with >50 cells per individual experiment). B Basal mitochondrial calcium levels in WT and Nox4KO MEFs after serum starvation. Representative photomicrographs are shown at the top. The spectrum color scale represents the ratio of emitted fluorescence (YFP/CFP). Mean data at the bottom. Scale bars: 10 µm. n=3/group (with >100 cells per individual experiment). Changes in mitochondrial calcium levels in serum-starved WT and Nox4KO MEFs after the addition of histamine (100 μmol/L, C) n=3/group (with >30 cells per individual experiment). D Changes in mitochondrial calcium levels in serum-starved WT and Nox4KO MEFs after the addition of ATP (100 μmol/L, D). n=3/group (with >30 cells per individual experiment). E, Changes in ER calcium levels measured with an ER-targeted cameleon probe in serum-starved WT and Nox4KO MEFs after the addition of histamine (100 μmol/L, E) n=3/group (with >30 cells per individual experiment). F Changes in ER calcium levels measured with an ER-targeted cameleon probe in serum-starved WT and Nox4KO MEFs after the addition of ATP (100 μmol/L, F). n=3/group (with >30 cells per individual experiment). G Peak increase in mitochondrial calcium levels response in response to histamine (100 μmol/L) in WT MEFs with or without treatment with PEG-catalase (Cat), and in Nox4KO MEFs with or without transfection with active Nox4 or a catalytically inactive Nox4P437H mutant (Mut). n=3/group (with >30 cells per individual experiment). H Representative time-course of histamine-induced changes in mitochondrial calcium for the experiments shown in (I). \n",
      "Fig.Figure 6Representative confocal z-stack images of LV-infected dendritic segments in the DG granule neurons from control and susceptible mice with different treatment. White arrows indicate stubby spines, yellow arrows indicate mushroom spines, and blue arrows indicate thin spines. B-E Quantification of total (B), stubby (C), mushroom (D) and thin (E) spine density in DG from control and susceptible mice infected with LV-GFP or LV-siCD39 (n = 15 segments from 4 mice/group. For total density: Treatment F2, 42 = 3.12, P = 0.0544; Ctrl - LV-GFP vs CSDS - LV-siCD39, P = 0.0312; CSDS - LV-siCD39 vs CSDS - LV-GFP, P = 0.0425; stubby density: Treatment F2, 42 = 10.93, P=0.0002; P = 0.0026; CSDS - LV-CD39 vs CSDS - LV-GFP, P < 0.0001; mushroom density: Treatment F2, 42 = 0.34, P = 0.7161; thin density: F2, 42 = 1.00, P = 0.3758. One-way ANOVA with Fisher's LSD test). F Representative confocal z-stack images of LV-infected dendritic segments in the DG granule neurons from control and susceptible mice with different treatment. White arrows indicate stubby spines, yellow arrows indicate mushroom spines, and blue arrows indicate thin spines. G-J Quantification of total (G), stubby (H), mushroom (I) and thin (J) spine density in DG from control and susceptible mice treatment with ACSF or ARL67156 (n = 11, 13, 10 segments from 2, 4, 2 mice, respectively. For total density: Treatment F2, 31 = 4.474, P = 0.0196; CSDS - ASCF vs Ctrl - ACSF, P = 0.0366; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0086; stubby density: Treatment F2, 31 = 3.327, P=0.0491; CSDS - ASCF vs Ctrl - ACSF, P = 0.1026; CSDS - ARL67156 vs CSDS - ACSF, P = 0.0179; mushroom density: Treatment F2, 31 = 1.986, P = 0.1543; thin density: F2, 31 = 1.475, P = 0.2445. One-way ANOVA with Fisher's LSD test). \n",
      ".Figure 7During cellular stress, Nox4 at the MAM augments the level of Akt activation secondary to redox inhibition of PP2a. Activated Akt in turn phosphorylates InsP3R and leads to an inhibition of calcium flux from ER to mitochondria. Mitochondrial calcium is maintained at a low level, the mitochondrial membrane potential (ΔΨm) is preserved, and cells remain viable. When Nox4 is deficient (bottom panel), high PP2a activity dephosphorylates Akt and leads to a reduced level of InsP3R phosphorylation. This removes the restraint on calcium transfer from ER to mitochondria resulting in increased mitochondrial calcium levels, triggering of the mPT, loss of ΔΨm and eventual cell death. \n",
      ".Figure 1(A) Domain architecture and putative crosslinking model of ACTN4 antiparallel dimers. Actin-binding-domain (ABD), calponin-homology (CH1, CH2), spectrin-repeats (SR), calmodulin-like domain (CaM), EF-Hand domains. (B) Immunoblots from cytosolic (cyt) and nuclear (nuc) fractionations of RPE-1 cells in early-G1 showing the presence of endogenous ACTN4 in nuclear fractions. Proteins were detected with antibodies against (α) ACTN4, α-Tubulin or Histone 3 as indicated (C) Upper and middle panels: immunoblots showing precipitation of ACTN4 and β-actin from RPE-1 nuclear lysates in early-G1 in the presence or absence of biotin-phalloidin, as indicated. Lower panel: Immunoblot of cytosolic (cyt) and nuclear (nuc) fractions. Proteins were detected with antibodies against (α) ACTN4, β-Actin, α-Tubulin or Histone 3 as indicated. (D) Immmunofluorescence images of endogenous ACTN4 in non-dividing and post-mitotic NIH3T3 and RPE-1 cells in as indicated. Shown are maximum intensity projections. Cells were fixed in 4% PFA and stained for ACTN4 and nuclei (DAPI) as indicated. Scale bars 10 μm. \n",
      ".Figure 6A Schematic of putative mechanism underlying Nox4-mediated regulation of calcium transfer via InsP3R channels to mitochondria. Akti: Akt inhibitor; XeC: Xestospongin C (InsP3R blocker). B Basal mitochondrial calcium levels in serum-starved WT and Nox4KO MEFs in the presence or absence of Akti (1 µmol/L) normalized versus WT MEF control. n=6/group with >20 cells imaged per experiment. C Effect of Akti (1 µmol/L) or XeC (1 µmol/L) on histamine-induced changes in mitochondrial calcium levels in MEFs. The time course of changes in calcium levels is shown on the left and the mean data for peak mitochondrial calcium levels on the right. n=3/group with >40 cells imaged per experiment. D Quantification of cell death in serum-starved WT and Nox4KO MEFs in the absence or presence of Akti or XeC. n=6/group. E Cardiac troponin I (cTnI) levels as a marker of cardiomyocyte necrosis in the perfusate of isolated WT and Nox4KO hearts subjected to ischemia-reperfusion (I/R). XeC was added at a final concentration of 2 µmol/L prior to ischemia. n=6-7/group. F Immunoblotting for the phosphorylation levels of Akt and InsP3R in crude mitochondrial fractions from WT and Nox4KO hearts after I/R. InsP3R was first immunopreciptated and then the precipitate was immunoblotted for total InsP3R and for the phosphorylated Akt-substrate motif RXRXX(pS/T) (p-InsP3R). Mean data shown to the right. n=3/group. G Cardiac left ventricular contractile function in isolated Langendorff-perfused WT and Nox4KO hearts at baseline (Basal) and then after I/R. Hearts were treated with XeC (1 μmol/L) or vehicle control for 20 min prior to ischemia. RPP, heart rate x left ventricular pressure product; DEVP, left ventricular developed pressure. n=6-7/group. \n",
      ".Figure 5A Immunoblotting of subcellular fractions from serum-starved cardiomyoblasts for InsP3R, Akt and PP2a, as in Fig 3A. B Fractionation of crude mitochondrial fractions (containing mitochondria and MAM) from WT MEFs on a sucrose gradient (5-60%) and immunoblotting for relevant proteins. Fraction size: 200 µL. Representative of 3 independent experiments. C Immunoblotting for phosphorylated and total Akt or InsP3R protein in crude mitochondrial fractions from serum-starved WT and Nox4KO MEFs. FACL4 was used as a loading control for the MAM. InsP3R was first immunopreciptated and then the precipitate was immunoblotted for total InsP3R and for the phosphorylated Akt-substrate motif RXRXX(pS/T) (p-InsP3R). Representative immunoblots are shown at the top and mean data at the bottom. Cat = PEG-catalase. Nox4KO MEFs were transfected either with active Nox4 or a Nox4P437H mutant (Mut). n=3/group. D ROS levels indexed in MEFs using H2O2 specific HyPer probes targeted to the MAM (FACL4 HyPer) or the cytosol (Cyto HyPer). Imaging was performed after serum starvation for 12h. Representative photomicrographs of the fluorescence ratio are shown to the right and mean data for changes in fluorescence ratio presented to the left. The signal obtained using corresponding ROS-insensitive SypHer probes was used to correct for any pH-induced changes in fluorescence; the change in fluorescence ratio between HyPer and corresponding SypHer probe (ΔR) for each condition is reported. n=5 independent cell preparations/group, with at least 20 cells imaged/preparation. Scale bars: 10 µm. \n",
      "a) Confocal micrographs of HeLa cells expressing yellow fluorescent protein (YFP) fused to Nap1, Sintbad or Tank and infected with S.figf1(a) Confocal micrographs of HeLa cells expressing yellow fluorescent protein (YFP) fused to Nap1, Sintbad or Tank and infected with S. Typhimurium, stained for ubiquitin (Ub) 2 h after infection. At least 100 cells were analyzed. DAPI, 4,6-diamidino-2-phenylindole. Scale bar, 10 μm; arrowheads, bacteria shown in insets. (b-e) Normalized ratio between luciferase activity bound to beads and present in lysates. The indicated purified GST fusion proteins coupled to beads were incubated with lysates of 293ET cells expressing Nap1, Sintbad or Tank, each fused to luciferase (b) (c), lysates of 293ET cells expressing the indicated Nap1 mutants fused to luciferase (d), lysates of E. coli expressing Nap1(1-85) fused to luciferase (e) lysates of 293ET cells expressing Nap1, NEMO or Optineurin fused to luciferase \n",
      ".Figure 4(A) Live imaging in NIH3T3 cells expressing ACTN4 constructs stained with SNAP488 provided positions of mitotic cells. After fixation, cells were stained with phalloidin-AlexaFluor647, rechecked for proper processing and transferred to dSTORM imaging. (B) Upper panel shows expression of ACTN4 constructs (green, confocal image) and phalloidin-AF647 staining in the same cells (orange, dSTORM image) scale bar 10 µm; lower panel represents zoom of the indicated areas (white boxes); scale bars 5 µm. (C) Phalloidin-AF647 localizations per post-mitotic nuclei; data shown as mean ± 2.5-97.5 percentile: 52 ACTN4-wt nuclei, 55 ACTN4ΔCH1-NLS nuclei and 40 nuclei of non-dividing cells (ACTN4-wt) from 5 biological replicates. ***p(ACTN4ΔCH1-NLS post-mitotic) < 0.001, ****p(ACTN4-wt non-dividing) < 0.0001, *p (ACTN4ΔCH1-NLS post-mitotic vs. ACTN4-wt non-dividing) < 0.05 by One Way ANOVA. (D) Number of filaments in the post mitotic nuclei; as in (C); data shown as mean ± 2.5-97.5 percentile: 52 ACTN4-wt nuclei, 55 ACTN4ΔCH1-NLS nuclei from 5 biological replicates, ***p < 0.001 by t-test. (E) Actin filament widths for ACTN4-wt and ACTN4ΔCH1-NLS in early G1 nuclei; ); data shown as mean ± 2.5-97.5 percentile with 96 filaments (ACTN4-wt) and 36 filaments (ACTN4ΔCH1-NLS) from 5 biological replicates, **p < 0.01 by t-test. \n",
      ".Figure 2(A) Cells in early G1 stably expressing nAC-mCherry and doxycycline-inducible ACTN4-SNAP were analyzed by time-lapse microscopy. ACTN4-SNAP was labeled by SNAP-Cell 647SiR dye (green). Different arrows mark dynamic ACTN4 clusters. The white square 1 is shown as a timeseries on the right. The white square 2 is shown as a magnification below. The white square 3 shows an actin filament decorated with ACTN4 that was analyzed by linescan in Fig 2B. Scale bar overview (nAC) represents 10 µm. Scale bar time series and magnification 5 µm. Cells in early G1 stably expressing nAC-mCherry and doxycycline-inducible ACTN4-SNAP were analyzed by time-lapse microscopy. ACTN4-SNAP was labeled by SNAP-Cell 647SiR dye (green). The white square 3 shows an actin filament decorated with ACTN4 that was analyzed by linescan in Fig 2B. B) Linescan of an actin filament with associated ACTN4 from square 3 in A. (C) Automated tracking (autoregressive motion) of nuclear actin and ACTN4 in nuclei by Imaris software. Cells were treated like in A. Tracks are visualized by red (actin) or green (ACTN4) lines in a representative image (see corresponding Movie EV2). Scale bar is 2 µm. The track length, track displacement (distance between starting- and end point), average speed and maximum speed were quantified from 3 independent experiments. The black line shows the median and grey dashed lines show the quartiles. Quantifications and images show similar motion characteristics for actin and ACTN4. \n",
      ".Figure 3(A) NIH3T3 cells expressing nAC-mCherry were imaged for 120 min. Image frames represent maximum projections of 30 Z planes. Lower panel is a magnification of the indicated area (orange box); scale bars 10 µm. (B) NIH3T3 cells stably expressing H2B-mCherry were transfected with siCtrl or siACTN1 + siACTN4. After mitotic exit z-stacks (30 planes) were acquired every 5 min. Nuclei were 3D-reconstructed and volume was measured by Imaris. Data are mean ± SEM from 3 biological replicates with 22-27 (siCtrl) and 32-42 (siACTN1+4) analyzed nuclei; ***p < 0.001 by Wilcoxon test and ****p < 0.0001 by t-test at 90 min. (C) RPE-1 cells were analyzed as in B. Data are mean ± SEM from 5 biological replicates with 32-37 (siCtrl) and 31-39 (siACTN1+4) analyzed nuclei; ***p < 0.001 by Wilcoxon test and ****p < 0.0001 by Mann Whitney test at 90 min. (D) Cartoon illustrating experimental design of knockdown and reconstitution experiments in E, F and G to achieve cells lacking ACTN4 specifically in the nuclear compartment (left) versus cells expressing ACTN4-wt. (E) NIH3T3 cells stably expressing H2B-mCherry and siRNA-resistant ACTN4-NES were transfected with the indicated siRNAs and analyzed as described in B. Data are mean ± SEM from 5 biological replicates with 63-71 (siCtrl), 37-41 (siACTN1+4), 52-58 (siACTN1) and 52-55 (siACTN4) analyzed nuclei; ****p(siACTN4) < 0.0001; ****p(siACTN1+4) < 0.0001 by One Way ANOVA at 90 min. (F) NIH3T3 cells stably expressing H2B-mCherry and the indicated ACTN4 mutants were transfected with the indicated siRNAs and analyzed as in B. Data are mean ± SEM from 3 biological replicates with 63-69 (ACTN4 wt + siACTN4), 63-71 (ACTN4-NES + siCtrl) and 54-58 (ACTN4-NES + siACTN4) analyzed nuclei; ****p(ACTN4-NES + siACTN4) < 0.0001 by One Way ANOVA at 90 min. (G) NIH3T3 cells expressing siRNA-resistant ACTN4-NES were transfected with siCtrl or siACTN4; chromatin density was calculated from H2B-mCherry fluorescence intensities by total nuclear volumes (intensity/μm3) and normalized to t=0 (mitotic exit); Data are mean ± SEM from 5 biological replicates with 44-48 (siCtrl) and 44-47 (siACTN4) analyzed nuclei; ***p < 0.001 by Wilcoxon test and *p < 0.05 by t-test at 90 min. \n",
      "a) Flowthrough from nickel-agarose column, separated by SDS-PAGE and stained with Coomassie blue.figf2(a) Flowthrough from nickel-agarose column, separated by SDS-PAGE and stained with Coomassie blue. (b) Amino acid sequence of NDP52. Red and underlining indicates peptides identified by mass spectrometry. (c,d) Normalized ratio between luciferase activity bound to beads and present in lysates. The indicated purified GST fusion proteins coupled to beads were incubated with bacterially expressed Nap1(1-85) fused to luciferase. Reactions were complemented with lysates from 293ET cells that had been depleted with the indicated (α-) antibodies (c, top). Immunoblot of lysates after depletion (c, bottom). (c,d) Normalized ratio between luciferase activity bound to beads and present in lysates. The indicated purified GST fusion proteins coupled to beads were incubated with bacterially expressed Nap1(1-85) fused to luciferase.Reactions were complemented with the indicated amounts of lysates from E. coli cultures expressing myelin basic protein (MBP) fused to NDP52 (d). (e) Lysates of 293ET cells were immunoprecipitated (IP) using the indicated antibodies or an isotype control. Lysates and precipitates were blotted (IB) for the presence of TBK1, IKKε and IKKβ. (f) Lysates of 293ET cells were immunoprecipitated using the indicated antibodies or an isotype control. Precipitates were blotted for the presence of TBK1 and NDP52. Fusion proteins used in Figure 2 are further characterized in Supplementary Figure 1. Data in panels c-f are representative of at least two independent experiments. Ub, ubiquitin; PCNA, proliferating cell nuclear antigen. \n",
      "Percentage of NDP52-positive S.figf4Percentage of NDP52-positive S. Typhimurium among ubiquitin-coated bacteria, determined by microscopy. HeLa cells transduced with the indicated NDP52 domain-deletion mutants fused to green fluorescent protein were infected with S. Typhimurium and stained for ubiquitin 2 h after infection. ZnF, zinc finger domain. *P 0.01, Student's t-test. Mean and s.d. of triplicate coverslips from two independent experiments; more than 50 ubiquitin-positive bacteria were evaluated per coverslip. \n",
      "a,b) Confocal micrographs of HeLa cells expressing YFP-Nap1, YFP-Sintbad or YFP-Tank (a) or YFP-TBK1(K38M), a catalytically inactive mutant (b), stained with NDP52 antiserum 2 h after infection with S.figf5(a,b) Confocal micrographs of HeLa cells expressing YFP-Nap1, YFP-Sintbad or YFP-Tank (a) or YFP-TBK1(K38M), a catalytically inactive mutant (b), stained with NDP52 antiserum 2 h after infection with S. Typhimurium. All data represent at least two independent experiments. DAPI, 4,6-diamidino-2-phenylindole. Scale bars, 10 μm; arrowheads, bacteria shown in insets. \n",
      "a) Confocal micrographs of HeLa cells 2 h after infection with S.figf3(a) Confocal micrographs of HeLa cells 2 h after infection with S. Typhimurium, stained with antibodies to Cocoa, NDP52 and ubiquitin (Ub). (b) Colocalization, at the indicated time points after infection, of ubiquitin-positive S. Typhimurium with NDP52 or Cocoa. (c) Confocal micrographs of HeLa cells 2 h after infection with S. Typhimurium, stained with antibodies to NDP52 and LAMP1. (d) Colocalization, at the indicated time points after infection, of NDP52-positive S. Typhimurium with LAMP1. DAPI, 4,6-diamidino-2-phenylindole. All data are representative of at least two independent experiments. Error bars, s.d.; three coverslips per experiment, 50 marker-positive cells evaluated per coverslip. Scale bars, 10 μm; arrowheads, bacteria shown in insets. \n",
      "a) Lysates of cells transfected with the indicated siRNAs blotted for NDP52, TBK1, IKKε and proliferating cell nuclear antigen (PCNA).figf6(a) Lysates of cells transfected with the indicated siRNAs blotted for NDP52, TBK1, IKKε and proliferating cell nuclear antigen (PCNA). Control, a nonspecific band detected by the IKKε antibody. (b,c) Numbers of bacteria recovered from HeLa cells transfected with the indicated siRNAs and infected with S. Typhimurium. At the indicated time points after infection, cells were lysed and bacteria counted on the basis of their ability to form colonies on agar plates. Mean and s.d. of triplicate HeLa cultures and duplicate colony counts. (d) Numbers of bacteria recovered from HCT116 cells expressing the indicated shRNAs and infected with S. Typhimurium. At the indicated time points cells were lysed and bacteria counted by flow cytometry. Mean and s.d. of triplicate cultures. (e) HeLa cells transfected with the indicated siRNAs were infected with S. Typhimurium expressing green fluorescent protein (GFP). At the indicated time points after infection, cells were scored according to the number of bacteria they contained. (f) HeLa cells transfected with the indicated siRNAs were infected with S. Typhimurium expressing GFP. At the indicated time points after infection, cells were stained for ubiquitin and infected cells were scored if they contained ubiquitylated bacteria. All data represent at least two independent experiments. *P 0.05, Student's t-test. \n",
      ".Figure 1A. 7SK was divided into 6 non-overlapping fragments and in vitro transcribed in the presence of 32p-UTP for use in an in vitro modification assay. U250 in fragment 5 is highlighted in red. B. In vitro modification reactions containing the six fragments and HeLa NE were performed. After nuclease P1 digestion, the products were analyzed by 1D-TLC with pU and pψ as controls. C. WT and mutant 7SK fragment 5 containing U250G were analyzed as in B. D. Outline of the biotin-facilitated extraction and analysis of the modification status of site-specifically labeled nucleotides in 7SK by 1D-TLC. E. 7SK RNA, generated by either in vitro transcription (IVT) or purified from HeLa total RNA and then site-specifically labeled at the indicated positions by the procedure in D, was analyzed by 1D-TLC. F. Cyber Gold staining of 7SK obtained by either IVT or immunoprecipitation (IP) with the indicated antibodies. G. 2D-TLC analysis of 32p-labeled mononucleotides derived from IVT or CDK9-bound 7SK RNA, with the first panel depicting a schematic migration pattern of the indicated 5'-phosphorylated mononucleotides. H. 7SK RNA, obtained by either IVT or anti-CDK9 IP and then site-specifically labeled at the indicated positions by the procedure in D, was analyzed by 1D-TLC. \n",
      "a–d) Analysis of HeLa cells stably expressing green fluorescent protein–LC3, infected with S.figf7(a-d) Analysis of HeLa cells stably expressing green fluorescent protein-LC3, infected with S. Typhimurium. (a) Confocal micrographs of cells stained 1 h after infection with NDP52 antiserum. Scale bar, 10 μm; arrowheads, bacteria shown in insets. (b) Colocalization of NDP52 with LC3-positive bacteria. (c,d) Counts of LC3-positive S. Typhimurium 1 h after infection in cells transfected with the indicated siRNAs. In c, cells were stained for ubiquitin (Ub); in d, red fluorescence protein (RFP)-expressing S. Typhimurium were used. Means of duplicate counts from four independent experiments; *P 0.05, Student's t-test. (e) Binding of NDP52 to GST-LC3 and GST-ubiquitin. Beads carrying the indicated purified GST fusion proteins were incubated with lysates of E. coli expressing NDP52. Bound proteins were eluted with glutathione and stained with Coomassie blue after SDS-PAGE. (f) Normalized ratio of luciferase-LC3 binding to beads. The indicated purified GST proteins coupled to beads were incubated with bacterially expressed LC3 fused to luciferase and the indicated volumes of lysed E. coli expressing NDP52. Data in a,b,e and f are representative of at least two independent experiments. \n",
      ".Figure 2A, B, G, and M. Sequence alternations in WT and 7SK RNA mutants mut1, mut2 and mut3, were shown in shaded boxes. U250 is marked in red. The tag refers to a 6-nt sequence replacement to enable discrimination from endogenous 7SK by primer extension. C-F; H-K; and N-Q. The tagged WT or mutant 7SK RNA was co-transfected with the indicated Flag-tagged proteins into HeLa cells. Anti-Flag or immunoprecipitates (IP) were analyzed by immunoblotting for the indicated proteins and primer extension for the bound 7SK RNA. C-F; H-K; and N-Q. The tagged WT or mutant 7SK RNA was co-transfected with the indicated Flag--tagged proteins into HeLa cells. Anti-Flag immunoprecipitates (IP) were analyzed by immunoblotting for the indicated proteins and primer extension for the bound 7SK RNA. C-F; H-K; and N-Q. The tagged WT or mutant 7SK RNA was co-transfected with the indicated Flag-tagged proteins into HeLa cells. Anti-Flag immunoprecipitates (IP) were analyzed by immunoblotting for the indicated proteins and primer extension for the bound 7SK RNA. C-F; H-K; and N-Q. The tagged WT or mutant 7SK RNA was co-transfected with the indicated HA-tagged proteins into HeLa cells. Anti-HA immunoprecipitates (IP) were analyzed by immunoblotting for the indicated proteins and primer extension for the bound 7SK RNA. L. Alignment between the 7SK sequences (black) flanking U250 (red) and a portion of the box H/ACA snoRNA (blue) showing base-pairing between the two. The altered nucleotides in shaded boxes were introduced into 7SK to disrupt the base-pairing. M. Sequence alternations in WT and 7SK RNA mutants mut1, mut2 and mut3, were shown in shaded boxes. U250 is marked in red. The tag refers to a 6-nt sequence replacement to enable discrimination from endogenous 7SK by primer extension. R. The indicated 7SK were in vitro transcribed in the presence of 32p-UTP and subjected to in vitro modification reactions. The nuclease P1-digested products were analyzed by 1D-TLC with pU and pΨ as controls. \n",
      ".Figure 4A. Luciferase activities were measured in HeLa cells transfected with the indicated reporter constructs, an empty vector (-) or the Tat cDNA, and either siDKC1 or a non-target siRNA (siNT). For each reporter, the activity in cells expressing siNT was set to 1. B, C, and E. The mRNA levels of luciferase (B) or Tat (C E) in cells expressing the indicated siRNAs were determined by qRT-PCR and normalized to the GAPDH mRNA and plotted. D. NH1 cells containing the integrated HIV-1 LTR-luciferase reporter and expressing Tat were transfected with siDKC1 or siNT. ChIP-qPCR analysis was performed to determine the levels of the indicated factors bound to the HIV-1 promoter. The signals were normalized to those of input and plotted. B, C, and E. The mRNA levels of Tat (C E) in cells expressing the indicated siRNAs were determined by qRT-PCR and normalized to the GAPDH mRNA and plotted. F. Western analysis of DKC1 and CDK9 levels in a Jurkat 2D10-derived clone inducibly expressing shDKC1 upon exposure to DOX. G and H. The inducible DKC1 KD clone, treated or untreated with DOX for 4 days, was incubated with the indicated concentrations of JQ1 (G) or prostratin (H) for 24 h. The GFP expression was measured by flow cytometry and expressed as percentages of GFP(+) cells of the entire population. The error bars in all panels represent mean ± SD from three independent experiments, and the p values determined by the Student's t-test. \n",
      ".Figure 1Plot showing the average Z-scores of 210 endolysosomal genes plotted against the negative log10 of the P values from the one-sample t-test of whether average Z-score ≠ 0. Green circles represent significantly upregulated genes (Z > 0 and P < 0.05). Magenta circles represent significantly downregulated genes (Z < 0 and P < 0.05). Grey circles represent genes whose expression is not significantly changed (P > 0.05). All comparisons were between cancers and normal cells from the same tissue. Dashed line shows position on Y-axis that corresponds to P = 0.05. Positions of MCOLN1 (red circle), VAC14 (blue circle), and MTM1 (yellow circle) are indicated. Inset, schematic showing that Mtm1 and Vac14 regulate the levels of PI(3,5) P2, and thereby, influence TRPML1 activity. Unsupervised hierarchical clustering of Pearson's coefficients of pairwise correlation of gene expression reveal 4 indicated clusters. Violin plots of average Z-scores of genes that belong to the four clusters derived from hierarchical clustering. The thick horizontal dashed lines in each sample set represent median; thin dashed lines represent the quartiles; polygons represent density estimates of data and extend to extreme values. Statistical tests employed were 1-sample t-test or Wilcoxon signed rank test to examine significance of differences from a hypothetical value of 0. GSEA conducted on endolysosomal genes ranked on the basis of increasing Z-scores. Positions of the CLEAR target genes are indicated. Bar graph showing the relative expression of the indicated genes in the indicated cell types. Values were normalized to respective 'HN31+HRAS shRNA' average and represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Western blots performed on extracts isolated from the indicated cells probed with antibodies against Tfeb and tubulin. Molecular weights are shown on the left. Bar graph shows relative Tfeb levels in the indicated samples. Values were normalized to 'HN31+HRAS shRNA' and represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Bar graph showing the relative expression of the indicated genes in the indicated cell types. Values were normalized to respective 'HN31+HRAS shRNA' average and represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. 'Attribute circle layout' generated from Pearson's coefficients of pairwise correlation of gene expression using CytoScape. Decreasing node size (clockwise from MCOLN1) reflects decreasing closeness centrality; node color represents degree (number of directed edges); edge color represents Pearson's correlations. Bar graph showing the relative expression of MCOLN1 in the indicated cell types. Values were normalized to HT1197 average and represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. \n",
      ".Figure 3A. Anti-V5immunoprecipitates (IP) from HeLa cells expressing nothing (-) or V5-DKC1 were analyzed by immunoblotting for V5-DKC1 and qRT-PCR for the bound 7SK RNA. The error bars represent mean ± SD from three independent experiments. B. Anti-FlagIP from NE of HeLa or the HeLa-derived F1C2 cells expressing CDK9-F were analyzed by Western blotting (WB) for the indicated proteins and primer extension (PEx) for 7SK RNA. C. WB analysis of DKC1 and CDK9 levels in two HeLa clones inducibly expressing shDKC1 upon doxycycline (DOX) treatment for 5 days. D. The 7SK RNA levels in these two clones were detected by qRT-PCR before and after the DOX treatment and normalized to the GAPDH mRNA levels. The error bars represent mean ± SD from three independent experiments. E. 1D-TLC analysis of Ψ250 in 7SK RNA from inducible DKC1 KD clone 1 before and after the DOX treatment. F. NE and anti-CDK9IP from NE of inducible DKC1 KD clone 1, which was untreated or treated with DOX, were analyzed by WB for the indicated proteins and PEx for 7SK RNA. G. Clone 1 was treated with DOX for 3 days and then transfected with the Tat-Flag cDNA (20 ug/150-mm dish). Two days later, NE and anti-FlagIP from NE were analyzed as in F. \n",
      ".Figure 4Representative Western blots generated using extracts from cells of expressing oncogenic HRAS and controls treated with ML-SI1 as indicated. The primary antibodies used are indicated on the right. Bar graph showing quantification of the Western blots shown in (A). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Representative Western blots generated using extracts from the indicated cell types after the indicated perturbations. Primary antibodies used are indicated on the right. Representative Western blots generated using extracts from the indicated cell types after the indicated perturbations. Primary antibodies used are indicated on the right. Representative Western blots generated using extracts from the indicated cell types after the indicated perturbations. Bar graph showing quantification of the Western blots in (C). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Representative Western blots generated using extracts from the indicated cell types after the indicated perturbations. Bar graph showing quantification of the Western blots in (D). All values represent mean±SEM. Data points represent values from biological replicates. Statistical tests employed were Student's t-test and ANOVA. \n",
      ".Figure 2Graphs showing the changes in the fura-2 ratio in cells of the indicated genotypes in response to ML-SA1 and GPN. Arrows serve as event markers that denote the points of time at which the drugs were added. Grey lines on top represent the time at which the bath Ca2+ was removed. All values represent mean±SEM from 3 biological replicates. Graphs showing the changes in the fura-2 ratio in cells of the indicated genotypes in response to ML-SA1 except that the baseline ratios for both genotypes were adjusted to 0 at the start of the experiment. Lines on the right indicate peak amplitudes of Ca2+ release in response to the application of ML-SA1. All values represent mean±SEM from 3 biological replicates. Graphs showing the changes in the fura-2 ratio in cells of the indicated genotypes in response to GPN. the baseline ratios for both genotypes were adjusted to 0 at the start of the experiment. Lines on the right indicate peak amplitudes of Ca2+ release Ca2+ release was evoked using GPN. Peak amplitudes of Ca2+ release in cells of the indicated genotypes in response to ML-SA1 (E) All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Peak amplitudes of Ca2+ release in cells of the indicated genotypes in response to GPN (F). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Ratio of the amplitudes of Ca2+ release in cells of the indicated genotypes in response to ML-SA1 and GPN. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Total Ca2+ release in cells of the indicated genotypes in response to ML-SA1 and GPN. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Schematic representation of the notion that an increase in the expression of MCOLN1 results in a proportional increase in ML-SA1-induced Ca2+ release in HN31 cells. \n",
      ".Figure 3Bar graphs showing the relative cell numbers in the indicated cell types following the indicated treatments. Cell numbers were assessed after a 48-hour period of growth in cultures seeded at equal densities. All values represent mean±SEM. Data points represent values from biological replicates. Bar graphs showing the relative cell numbers in the indicated cell types following the indicated treatments. Cell numbers were assessed after a 48-hour period of growth in cultures seeded at equal densities. All values represent mean±SEM. Data points represent values from biological replicates. Bar graphs showing the relative cell numbers in the indicated cell types following the indicated treatments. Cell numbers were assessed after a 48-hour period of growth in cultures seeded at equal densities. All values represent mean±SEM. Data points represent values from biological replicates. Representative images showing Ki67-labeling in xenografts generated from implantation of HN31 cells stably expressing control shRNA (left) or MCOLN1 shRNA (right). Pseudocolored panels show the representative distribution Ki67 positive or negative nuclei. Scale bars shown (0.5 mm) apply to all panels. Bar graphs showing fraction (left) and total number (right) of Ki67 labeled nuclei in the indicated cell types. All values represent mean±SEM. Values in the bar graph on the right were normalized to control mean. Data points represent values from independent eggs. Statistical test employed was Student's t-test. Bar graphs showing the relative cell numbers in UMSCC-22A-HRASWT cells following the indicated perturbations. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Kaplan-Meier curves showing the survival of patients stratified on the basis of MCOLN1 expression in BLCA and HNSC patients whose tumors were wild-type for HRAS (right) or carried oncogenic HRAS mutations (left). Statistical test employed was the Mantel-Cox log-rank test. \n",
      ".Figure 6Schematic showing the methodology utilized for preparation of plasma membrane sheets from MDCK cells expressing GFP-HRASWT or GFP-HRASG12V. After preparation of plasma membrane sheets, gold conjugated antibodies are used to visualize clusters of GFP-tagged proteins in electron micrographs. Weighted mean univariate K-function curves of immunogold labeled GFP-HRASG12V under indicated conditions, n ≥ 15 for each condition. The gold point patterns were analyzed with univariate K-functions expressed as L(r)-r. The bar graph on the right shows the peak value Lmax of the L(r)-r curve, which represents the extent of GFP-HRASG12V nanoclustering. Values are normalized to DMSO average and represent mean±SEM (n ≥ 15 for each condition). Statistical significances of differences were evaluated in bootstrap tests. Bar graph showing GFP-HRASG12V content in the inner leaflet of the plasma membrane is quantified as the number of gold particles per 1μm2 region from electron micrographs. Values are normalized to DMSO average and represent mean±SEM, n ≥ 15 for each condition. Statistical test employed was Student's t-test with Bonferroni post hoc correction to account for multiple pairwise comparisons. The extent of GFP-HRASG12V nanoclustering quantified by Lmax values in cells exposed to the indicated perturbations. Values are normalized to DMSO average and represent mean±SEM (n ≥ 15 for each condition). Statistical significances of differences were evaluated in bootstrap tests. Bar graph showing GFP-HRASG12V content in the inner leaflet of the plasma membrane is quantified as the number of gold particles per 1μm2 region from electron micrographs. but in cells exposed to the indicated perturbations. Values shown are normalized to DMSO average. Representative Western blots generated using extracts from HN31 cells stably expressing the MCOLN1 shRNA. Samples treated with exogenous cholesterol for 3 hours are indicated at the bottom. Primary antibodies used are indicated on the right. Representative Western blots generated using extracts from HN31 cells stably expressing the MCOLN1 shRNA. Samples treated with exogenous cholesterol for 3 hours are indicated at the bottom. Bar graph showing quantification of the Western blots in (F). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was ANOVA. Schematic summarizing the effects of decreased TRPML1 activity on HRASWT and HRASG12V. Our data indicate that TRPML1 inhibition diminished HRASG12V nanoclustering and plasma membrane levels. Addition of cholesterol restored these parameters. Only the nanoclustering of HRASWT was affected by ML-SI1. \n",
      ".Figure 5Bar graph showing relative levels of trpml mRNA in Drosophila larval brain extracts from control animals and animals expressing dRasG12V in glia. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Representative images of Drosophila 3rd instar larvae expressing the indicated transgenes in glial cells. repo-GAL4 was used to drive the expression of UAS-dRasG12V in glia. Images shown are volumetric reconstructions of confocal stacks. The various anatomical structures of the larval brain are annotated. VNC refers to the 'ventral nerve chord', which is distinct from dorsal hemispheres of the brain. GFP staining represents the total glial membranes in the larval brains. Scale bar, 100μm. Bar graph showing relative glial cell volume in larvae of the indicated genotypes. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Representative Western blots generated using macrophage extracts isolated from 3rd instar larvae of the indicated genotypes. He-GAL4 was used to drive the expression of UAS-dRasG12V in macrophages. The primary antibodies used are indicated on the right. Bar graph showing quantification of the Western blots shown in (D). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Representative Western blots generated using fat-body extracts isolated from 3rd instar larvae of the indicated genotypes. cg-GAL4 was used to drive the expression of UAS-HRASG12V in larval fat-bodies. The primary antibodies used are indicated on the right. Bar graph showing quantification of the Western blots shown in (F). All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. \n",
      "a,b) HeLa cells were treated with 10 μg ml−1 N-Shh peptide for 24 h.f1(a,b) HeLa cells were treated with 10 μg ml−1 N-Shh peptide for 24 h. (a) Endogenous LC3 was detected by immunoblot in total cell lysates, quantified by densitometric analysis and normalized to actin. Relative levels were expressed in percentage with control cells set to 100 and shown in graph. (a,b) HeLa cells were treated with 10 μg ml−1 N-Shh peptide for 24 h.(b) mRNA was obtained and levels of Ptch1 or Gli1 and actin mRNA were amplified by standard PCR and visualized in agarose gels. Graph shows mRNA levels quantified by densitometric analysis and normalized to actin. (c) 8xGli-luciferase construct was transfected in HeLa cells for 24 h, followed by treatment with 10 μM purmorphamine, or DMSO as a control, for another 24 h. Graph represents the mean value of the firefly luciferase activity relative to Renilla transfection control. (d) Endogenous LC3-II was detected in cell lysates from cells treated with 10 μM purmorphamine for 24 h, either in the absence or presence of trehalose (100 mM). Where indicated, bafA1 (400 nM) was added for the last 4 h. Quantification by densitometric analysis relative to actin is shown in the graph. (e) Cells were treated with 10 μM purmorphamine for 24 h under starvation conditions, either in the presence of bafA1 or DMSO as a control, and LC3-II and actin levels were detected. (f) HA-HttQ74 construct was transfected into HeLa cells followed by treatment with 10 μM purmorphamine for 24 h. The percentage of transfected cells with aggregates detected by HA immunofluorescence is shown in the graph. P-values were calculated by odds ratio. (g) U20S cells stably expressing HaloTag-p62 were labelled with HaloTag ligand for 15 min and washed out followed by treatment with purmorphamine. After 48 h, cells were lysed, run on a SDS-polyacrylamide gel electrophoresis and fluorescent HaloTag was visualized using a Typhoon 8600 variable mode imager. Fluorescence was normalized to total protein levels, detected on the same gels by Kryton Fluorescence protein stain. A representative gel and its quantitated normalized levels are shown. In all panels, graphs show mean values and error bars represent s.d. from a triplicate experiment representative of at least three independent experiments. Statistical analyses were performed by two-tail Student's t-test unless indicated: ***P0.001; **P0.01; *P0.05. SDS-PAGE, SDS-polyacrylamide gel electrophoresis. \n",
      ".Figure 7Schematic showing the role of TRPML1 in vesicular trafficking. TRPML1 is active in the membrane of late-endosomes and facilitates fusion of these vesicles with lysosomes or the plasma membrane. Channels delivered to the plasma membrane are redelivered to the late-endosomes via endocytosis. Whereas delivery of vesicles to the lysosomes is required for cholesterol de-esterification in that compartment, vesicular fusion with the plasma membrane is required for cholesterol recycling. Bar graph showing the relative levels of total in HN31 cell extracts exposed to the indicated perturbagens Values represent mean±SEM. Data points represent values from biological replicates. Statistical tests employed were Student's t-test and ANOVA. Bar graph showing the relative levels of free/total cholesterol in HN31 cell extracts exposed to the indicated perturbagens Values represent mean±SEM. Data points represent values from biological replicates. Statistical tests employed were Student's t-test and ANOVA. Bar graph showing the relative levels of free/total cholesterol in larval brain extracts from animals of the indicated genotypes Values represent mean±SEM. Data points represent values from biological replicates. Statistical tests employed were Student's t-test and ANOVA. Confocal images of MDCK cells expressing mCherry-D4H. Colocalization with the cortical actin (phalloidin) indicates that mCherry-D4H is predominantly localized to the plasma membrane in DMSO treated cells. Within 24 hours of ML-SI1 treatment, the mCherry-D4H redistributes to intracellular punctae. Although minimal mCherry-D4H signal remained at the plasma membrane in ML-SI1-treated cells, the phalloidin signal (cortical actin) remained unchanged. Scale bar (10μm) shown in the panel on the top right applies to all panels. Bar graph showing quantification of mCherry-D4H punctae in DMSO or ML-SI1-treated cells. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Confocal images of MDCK cells expressing mCherry-D4H and LAMP1-GFP treated with ML-SI1. Scale bar (10μm) shown in the panel on the top right applies to all panels. Confocal images of control and MCOLN1 shRNA-expressing HN31 cells treated with the cholesterol sensor, filipin. Scale bar (10μm) shown in the panel on the top applies to both panels. Bar graph showing the quantification of the internalized filipin staining in HN31 cells after MCOLN1 knockdown. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. Fraction of phase-separated GPMVs at the indicated temperatures. Colors of the lines indicate the treatment conditions of MDCK cells prior to generation of GPMVs. Dashed vertical lines indicate the miscibility temperature of the indicated populations of GPMVs. Values shown represent mean±SEM. Lines were generated by fitting the data to sigmoidal functions. Bar graph showing miscibility temperatures of GPMVs isolated from MDCK cells exposed to the indicated conditions. All values represent mean±SEM. Data points represent values from biological replicates. Statistical test employed was Student's t-test. \n",
      "a) Smo siRNA-treated cells were transfected with HA-HttQ74 for the last 24 h and the percentage of transfected cells with aggregates scored in HA-positive cells by immunofluorescence.f3(a) Smo siRNA-treated cells were transfected with HA-HttQ74 for the last 24 h and the percentage of transfected cells with aggregates scored in HA-positive cells by immunofluorescence. Graphs show mean values and error bars represent s.d. of a representative experiment performed in triplicate. P-values were calculated by odds ratio. (b) siRNA-targeting Smo was transfected into HeLa cells stably expressing mRFP-GFP-LC3. Representative confocal microscopy images are shown. Bars represent 26 μm. The number of vesicles was analyzed in a Cellomics array scan and percentage was calculated relative to control siRNA-transfected cells. Statistical tests were performed on data from three independent experiments in triplicate. *P0.05; ***P0.001. \n",
      "a) Wild-type Gli+/+ or Gli1−/−, Gli2−/− and Gli3−/− MEFs were treated with 100 mM trehalose alone or in combination with 10 μM of Pmph for 24 h, or left untreated.f4(a) Wild-type Gli+/+ or Gli1−/−, Gli2−/− and Gli3−/− MEFs were treated with 100 mM trehalose alone or in combination with 10 μM of Pmph for 24 h, or left untreated. Total cell lysates were subjected to immunoblot with anti-LC3 and anti-actin antibodies. (b) Gli2−/− MEFs were transfected for 48 h with either empty vector or Gli2 DNA construct and treated as in a. with 100 mM trehalose alone or in combination with 10 μM of Pmph for 24 h, or left untreated. Total cell lysates were subjected to immunoblot with anti-LC3 and anti-actin antibodies. (c) LC3-II levels were detected by western blotting of total lysates from wild-type and Gli2−/− mice embryos (E12.5). For quantification, LC3-II levels from a total of five wild-type and five Gli2−/− embryos were normalized to actin, wild-type was set to 100. Graphs show mean values and error bars represent s.d. Statistical analysis were performed by two-tail Student's t-test **P0.01. Pmph, purmorphamine. \n",
      "a) HeLa cells were transiently transfected with pcDNA (vector), Ptch1 or Ptch2 constructs for 48 h.f2(a) HeLa cells were transiently transfected with pcDNA (vector), Ptch1 or Ptch2 constructs for 48 h. Where indicated, cells were treated with bafA1 for the last 4 h. LC3-II levels were detected by western blotting and levels were quantified by densitometric analysis relative to tubulin and shown in graph. (b) HeLa were transfected with Ptch1 or Ptch2 together with HA-HttQ74 and the percentage of transfected cells with aggregates was assessed by HA immunofluorescence. P-values were calculated by odds ratio. (c) Atg5+/+ or Atg5−/− MEFs were transfected with GFP-HttQ74 for 48 h, cells were fixed and the percentage of transfected cells with aggregates was scored by direct fluorescence. (d,e) Endogenous LC3-II levels were detected in HeLa cells transfected with control, Ptch1 or Ptch2 siRNA and either left untreated or treated with 100 mM trehalose in the last 24 h (d) Quantification and statistical analysis is shown in graphs. (d,e) Endogenous LC3-II levels were detected in HeLa cells transfected with control, Ptch1 or Ptch2 siRNA and either left untreated or treated with400 nM bafA1 for 4 h (e). Quantification and statistical analysis is shown in graphs (f) siRNA-targeting Ptch2 was transfected into HeLa cells stably expressing mRFP-GFP-LC3 and representative confocal microscopy images are shown. Scale bars represent 26 μm. Percentage of GFP-positive vesicles (autophagosomes), RFP-positive vesicles (total number of vesicles) or percentage of autolysosomes (calculated by subtracting numbers of GFP from RFP vesicles), quantified using a Cellomics array scan, was calculated relative to control siRNA-transfected cells. Although an increase in autophagosome biogenesis generally increases both the number of GFP+ vesicles and the number of GFP-/RFP+ vesicles as a result of an enhanced autophagy flux, defects in autophagosome degradation, such as in bafA1-treated cells, increase GFP+ vesicles but not GFP−/RFP+ structures, as GFP remains intact. Graph shows the mean value obtained from four independent experiments in triplicate and with control conditions set to 100. In all panels, unless indicated, graphs show mean values and error bars represent s.d. from a triplicate experiment representative of at least three independent experiments. Statistical analyses were performed by two-tail Student's t-test: **P0.01; *P0.05; NS, not significant. \n",
      "a) Graph representing fold change in mRNA levels in cells treated with Shh against untreated cells.f6(a) Graph representing fold change in mRNA levels in cells treated with Shh against untreated cells. Each circle represents an autophagy gene; those with largest fold-changes are indicated in red. The central line represents no changes in expression; above the central line, genes whose expression is increased: below, those with reduced levels. Grey lines indicate 2-fold increase or decrease. (b) Table showing the 10 genes with higher fold changes in expression and the values obtained. (c) Levels of PERK protein were assessed by western blotting in HeLa cells treated with 10 μM purmorphamine for 24 h. Actin was used as a protein loading control. (d) HeLa cells were treated with purmorphamine for 24 h and either left in rich media or in HBSS for the last 4 h. Levels of S51 phoshorylation and total levels of eIF2α were detected by western blotting using LI-COR infra-red imager. (e,f) Wild-type and PERK−/− (e) or eIF2α S51A/S51A MEFs (f) were treated with DMSO or 10 μM purmorphamine for 24 h and, where indicated, cells were treated with bafA1 for the last 4 h. tubulin-II and tubulin or tubulin, as a loading control, were detected and a representative blot is shown. HBSS, Hank's balanced salt solution. \n",
      "a,b) Control (a) or hh[ts2] (b) larvae were incubated at permissive temperature (18 °C) to the third instar stage, then cultured at non-permissive temperature (29 °C) in well-fed conditions for 24 h.f5(a,b) Control (a) or hh[ts2] (b) larvae were incubated at permissive temperature (18 °C) to the third instar stage, then cultured at non-permissive temperature (29 °C) in well-fed conditions for 24 h. Dissected fat body tissue was incubated in Lysotracker Red and DAPI and imaged unfixed. (c) Lysotracker Red-labelled fat body containing single-cell flipout clones (GFP positive) overexpressing patched. Fat body was dissected from well-fed larvae, incubated in Lysotracker Red/DAPI and imaged unfixed. (d) Confocal image of a GFP-positive patched-expressing clone, with mCherry-Atg8a expressed in all cells. Fat body was dissected from well-fed larvae and imaged after formaldehyde fixation. (e) GFP-marked flipout clones of CiCell-expressing cells. Fat body was dissected from well-fed larvae, incubated in Lysotracker Red/DAPI and imaged unfixed. (f) Confocal image of fixed mCherry-Atg8a expressing fat body containing a GFP-marked clone of CiCell-expressing cells. (g) Cells homozygous for the null costal2[2] allele (marked by lack of GFP) were generated by flp/FRT-mediated recombination in mCherry-Atg8a expressing fat body. Larvae were starved 4 h before dissection and fixation. \n",
      ".Figure 1(A) Representative image of a typical young WT cortico-hippocampal slice (350 µm) from co-culture (14 DIV) stained with DAPI (gray) to visualize nuclei (left). Dentate gyrus (DG), cornus ammonis 1 (CA1) and cortical (CTX) regions are indicated. An example of the co-culture dish is shown on the right. Co-culture includes 2 young WT (*, young co-culture) and 2 old APPPS1 (+, old co-culture) brain slices. (B) Immunofluorescence analysis of young WT and old APPPS1 slices cultured alone (young alone, old alone) or in co-culture (young co-culture, old co-culture) at 7 days in vitro (DIV) stained with propidium iodide (PI, red) reveals reduced cell viability in old brain slices compared to young.(C) Quantitative analysis of PI positive cells in young WT and old APPPS1 slices cultured alone (white and black bars, respectively) or in co-culture (grey and white squared bars, respectively) at 7 and 14 DIV. The values are expressed as percentages of PI positive cells from the total number of DAPI positive cells. The values represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (n.s.= not significant; ***P < 0.001). (D-F) Immunofluorescence analysis of young WT and old APPPS1 slices cultured alone or in co-culture at 7 DIV and immunostained using neuronal (NeuN), astrocytic (GFAP) and microglial (CD68) markers (green) reveals reduced NeuN and GFAP and increased CD68 immunosignal. Scale bar: 75 µm \n",
      ".Figure 2(A) Immunofluorescence analysis of the old APPPS1 slice in co-culture using antibody 6E10 to detect amyloid plaques (green) and Thiazine Red (TR, red) to visualize fibrillar amyloid cores. At 7 DIV, the majority of amyloid plaques contain 6E10 and TR positive dense core surrounded by 6E10 positive halo of diffuse Aβ. In contrast, at 14 DIV, the majority of amyloid plaques are core-only plaques co-stained with 6E10 and TR without detectable 6E10 positive halo. Lower panels are higher magnification images of boxed regions. Scale bars: 50 µm. (B) Quantitative analysis of core-only plaques in old APPPS1 slices cultured alone (white bars) or in co-culture (gray bars) at 7 and 14 DIV reveals an increased number of core-only plaques upon co-culturing of old APPPS1 brain slices together with young WT slices. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques. The values represent mean ± SEM from 3 independent experiments, each experiment including 3 independent slice culture dishes (***P < 0.001). (C) Western blot analysis of Aβ levels (triplicates) in the old APPPS1 tissue cultured alone or in co-culture with the young WT (14 DIV) using 2D8 antibody.(D) Quantification of Western blot signals reveals decreased levels of Aβ upon co-culturing compared to old tissue cultured alone. The values are normalized to the levels of Aβ in the old tissue cultured alone and represent mean ± SEM from 2 independent experiments, each experiment was performed in triplicates (**P < 0.01). \n",
      ".Figure 3(A, B) Immunofluorescence analysis of the old APPPS1 slice in co-culture (14 DIV) using antibodies 6E10 (green) and CD68 (red). Nuclei were counterstained using DAPI (blue). Image of boxed region in A is depicted at higher magnification in B and reveals clustering of several CD68 positive cells around amyloid plaques. Scale bars: 100 µm (A) and 10 µm (B).(C) Quantitative analysis of plaque-associated CD68 positive cells in co-culture samples at 7 and 14 DIV reveals a trend towards increased number of CD68 positive cells. The values represent mean ± SEM from at least 3 independent experiments, including total of at least 6 independent slice culture dishes (n.s.= not significant).(D) Quantitative analysis of co-culture samples at 7 and 14 DIV reveals a decrease in plaque size. The values represent mean ± SEM from at least 3 independent experiments, including total of at least 6 replicates (***P < 0.001).(E) Quantitative analysis of plaques surrounded by CD68 positive cells (clustered plaques) in co-culture samples at 7 and 14 DIV reveals an increase in the number of clustered plaques. The values are expressed as percentages of clustered plaques from the total number of amyloid plaques. The values represent mean ± SEM from at least 3 independent experiments, including total of at least 6 independent slice culture dishes (***P < 0.001). (F) Immunofluorescence analysis of the old APPPS1 slice in co-culture (14 DIV) treated with cytochalasin D (CytoD) and vehicle control (Ctr) and immunostained with M3.2 (green) and CD68 (red). Inhibition of phagocytosis by CytoD blocks amyloid plaque clearance. Scale bar: 50 µm.(G) Quantitative analysis of core-only plaques in the old APPPS1 slice in co-culture (14 DIV) treated with CytoD and Ctr reveals a decreased number of core-only plaques upon CytoD treatment. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques. The values represent mean ± SEM from 3 independent experiments, including total of 6 independent slice culture dishes (***P < 0.001). \n",
      ".Figure 4Either young WT (A, B) or old APPPS1 (C-E) tissue was treated with clodronate (Clo) to remove CD68 positive cells or vehicle Ctr from 1 until 7 DIV. Treatment was stopped and subsequently old (A, B) or young (C-E) tissue was added to the culture as schematically indicated in A and C. and analyzed 14 days after. Removal of CD68 positive cells in either young WT or old APPPS1 tissue prevents amyloid plaque clearance in the co-culture model.(A) Immunofluorescence analysis of the old APPPS1 slice co-cultured with the young WT slice pre-treated with Clo and Ctr and immunostained with CD68 (red) and M3.2 (green). Scale bar: 50 µm.(B) Quantitative analysis of core-only plaques in the old APPPS1 tissue co-cultured with the young WT tissue pre-treated with Clo and Ctr as indicated in A reveals a decreased number of core-only plaques upon Clo treatment. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, including total of 6 independent slice culture dishes (***P < 0.001).(C) Immunofluorescence analysis of the old APPPS1 slice treated with Clo and Ctr and subsequently co-cultured with the young WT slice and immunostained with CD68 (red) and M3.2 (green). Scale bar: 50 µm.(D) Area of CD68 positive cells (CD68 coverage) in the old APPPS1 tissue treated with Clo and Ctr and subsequently co-cultured with the young WT tissue as indicated in C. CD68 coverage is reduced upon Clo treatment. The values are normalized to CD68 coverage of the Ctr and represent mean ± SEM from 3 independent experiments, including total of 6 independent slice culture dishes (***P < 0.001).(E) Quantitative analysis of core-only plaques in the old APPPS1 tissue treated with Clo and Ctr and subsequently co-cultured with the young WT tissue as indicated in C reveals a decreased number of core-only plaques upon Clo treatment. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, including total of 6 independent slice culture dishes (**P < 0.01). \n",
      ".Figure 5(A) Immunofluorescence analysis of the old APPPS1 slice in co-culture with the young CX3CR1+/GFP slice (14 DIV) and immunostained using GFP (green), CD68 (red) and M3.2 (blue) reveals no co-localization of young, GFP positive microglial cells with CD68 positive cells surrounding amyloid plaques. A GFP antibody was used to amplify the signal of GFP-expressing young microglial cells.(B) Immunofluorescence analysis of the old APPPS1/CX3CR1+/GFP slice in co-culture with the young WT slice (14 DIV) and immunostained using GFP (green), CD68 (red) and M3.2 (blue) reveals co-localization of old, GFP positive microglial cells with CD68 positive cells surrounding amyloid plaques. A GFP antibody was used to amplify the signal of GFP-expressing old microglial cells. Scale bar: 10 µm (A and B). (C) 3D reconstruction of the old APPPS1/CX3CR1+/GFP slice in co-culture with the young WT slice (10 DIV) and immunostained using GFP (green) and M3.2 (red). GFP positive old microglial cells surrounding amyloid plaques exhibit intracellular M3.2 positive immunoreactivity indicative of Aβ uptake. Overview at low magnification is shown in the left panel. Scale bar: 15 µm. A detailed plaque is shown at higher magnification in the right panels. Scale bar: 10 µm. (D) Immunofluorescence analysis of young CX3CR1+/GFP and old APPPS1/CX3CR1+/GFP slices freshly cut (0 DIV), cultured alone or in co-culture (10 DIV) and immunostained using GFP (green) shows alterations in microglial morphology of old cells upon culturing. Upper-left panels are enlargements of boxed regions. A GFP antibody was used to amplify the signal of GFP-expressing old microglial cells. Scale bars: 50 µm.(E) Quantitative analysis of amoeboid microglial cells in young CX3CR1+/GFP (white bars) and old APPPS1/CX3CR1+/GFP (green bars) brain slices freshly cut (0 DIV), cultured alone or in co-culture (10 DIV). The values are expressed as percentages of amoeboid microglia from the total number of microglial cells. The values represent mean ± SEM from 3 independent experiments, each experiment including at least 250 cells per condition (*P < 0.05, ***P < 0.001). \n",
      ".Figure 7(A-F) Immunofluorescence analysis of the old APPPS1 slice (14 DIV) treated with vehicle Ctr in A, anti-inflammatory factors IL-10 and TGF-β in B and C and pro-inflammatory factors IL-6, IL-12/p40 and GM-CSF in D, E and F, respectively and immunostained with CD68 (red) and M3.2 (green). In contrast to increased number of core-only plaques upon GM-CSF treatment, other factors tested did not induce amyloid plaque clearance, resulting in a better preservation of the M3.2 positive halos of Aβ surrounding dense core of amyloid plaques. Scale bar: 50 µm. \n",
      ".Figure 6(A) Old APPPS1 tissue was cultured alone (old alone, white bars) or incubated with conditioned media (CM) collected from young WT slices (old alone + young CM, grey bars). Analysis included co-culture of old APPPS1 and young WT brain slices (old co-culture, black bars) as a positive control and old APPPS1 tissue incubated with conditioned media collected from old APPPS1 slices (old alone + old CM, white squared bars) as a negative control. Quantitative analysis at 7, 11 and 14 DIV reveals increased numbers of core-only plaques upon incubation of the old APPPS1 tissue with conditioned media collected from young WT slices, fully recapitulating core-only plaque numbers observed in the co-culture. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, each experiment including at least 2 independent slice culture dishes (n.s.= not significant, **P <0.01, ***P < 0.001). (B) Old APPPS1 tissue was incubated with conditioned media collected from young WT slices previously treated with clodronate (old alone + young CM + Clo, grey bars ) and vehicle control (old alone + young CM + Ctr, white bars). Quantitative analysis at 7, 11 and 14 DIV reveals decreased numbers of core-only plaques upon incubation of the old APPPS1 tissue with conditioned media collected from young WT slices pre-treated with Clo. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, each experiment including at least 2 independent slice culture dishes (n.s.= not significant, ***P < 0.001). (C) Old APPPS1 tissue was incubated with conditioned media collected from cultured WT primary microglial cells (old alone + MG CM, grey bar) or with unconditioned slice culture media as a control (old alone + Ctr, white bar). Quantitative analysis of core-only plaques at 14 DIV reveals enhanced amyloid plaque clearance upon incubation of the old APPPS1 tissue with conditioned media collected from cultured primary microglia. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (*P < 0.05). \n",
      ".Figure 1A. Hcp release assay. HA-tagged Hcp (HcpHA) release was assessed by separating cells (C) and cell-free culture supernatant (S) fractions from 109 wild-type (WT), ΔtssM cells or ΔtssM cells carrying the AHT-inducible FLAG-tagged tssM-borne plasmid (tssM+) treated (bulgecin) or not (NT) with bulgecin A prior to tssM gene induction. Proteins were separated by 12.5%-acrylamide SDS-PAGE and the periplasmic TolB protein (control for cell lysis), HcpHA and FLTssM were immunodetected using anti-TolB (middle panel), anti-HA (lower panel) and anti-FLAG (upper panel) antibodies. Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in duplicate and a representative result is shown. B. Hcp release assay. HA-tagged Hcp (HcpHA) release was assessed by separating cells (C) and cell-free culture supernatant (S) fractions from 109 wild-type (WT) cells before washing cells (before wash) and after washing and growth (after wash) in absence (NT) or presence (bulgecin) of bulgecin A. Proteins were separated by 12.5%-acrylamide SDS-PAGE and the periplasmic TolB protein (control for cell lysis) and HcpHA were immunodetected using anti-TolB (upper panel) and anti-HA (lower panel) antibodies. Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in duplicate and a representative result is shown. \n",
      ".Figure 8(A) CD68 coverage in the old APPPS1 tissue (14 DIV) treated with GM-CSF and vehicle Ctr. CD68 coverage is increased upon treatment with GM-CSF. The values are normalized to CD68 coverage of the Ctr and represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (*P < 0.05). (B) Quantitative analysis of core-only plaques in the old APPPS1 tissue (14 DIV) treated with GM-CSF and Ctr reveals an increased number of core-only plaques upon treatment with GM-CSF. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, each experiment including 3 independent slice culture dishes (***P < 0.001). (C) CD68 coverage in the old APPPS1 tissue (14 DIV) incubated with conditioned media collected from young WT slices (young CM) or with non-conditioned slice culture media (Ctr). CD68 coverage is increased upon incubation of the APPPS1 tissue with the young CM. The values are normalized to CD68 coverage of the Ctr and represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (**P < 0.01). (D) Immunofluorescence analysis of the old APPPS1 slice cultured alone (old alone) or in co-culture with young GM-CSF-/- brain slices (old co-culture) at 14 DIV and immunostained with CD68 (red) and M3.2 (green) reveals an increased number of core-only plaques upon co-culturing with young GM-CSF-/- brain slices. Scale bar: 50 µm.(E) Quantitative analysis of core-only plaques in the old APPPS1 slice cultured alone or in co-culture with young GM-CSF-/- brain slices (14 DIV) reveals an increased number of core-only plaques upon co-culturing of old APPPS1 brain slices together with young GM-CSF-/- brain slices. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques. The values represent mean ± SEM from 3 independent experiments, each experiment including 3 independent slice culture dishes (***P < 0.001).(F) CD68 coverage in the old APPPS1 tissue (14 DIV) cultured alone or in co-culture with young GM-CSF-/- brain slices. CD68 coverage is increased upon co-culturing of old APPPS1 brain slices together with young GM-CSF-/- brain slices. The values are normalized to CD68 coverage of the old slice cultured alone and represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (*P < 0.05). \n",
      ".Figure 9(A) Immunofluorescence analysis of the old APPPS1 slice (14 DIV) treated with GM-CSF and vehicle Ctr and immunostained with CD68 (red), cell proliferation marker Ki67 (green) and M3.2 (blue). Scale bar: 50 µm.(B) Quantitative analysis of Ki67 and CD68 double positive microglial cells in the old APPPS1 tissue (14 DIV) treated with GM-CSF and Ctr reveals increased microglial proliferation upon treatment with GM-CSF. The values are expressed as percentages of CD68 and Ki67 double positive microglial cells from the total number of CD68 positive cells and represent mean ± SEM from 2 independent experiments, each experiment including 4 independent slice culture dishes (***P < 0.001). (C) Immunofluorescence analysis of the old APPPS1 tissue cultured alone or in co-culture with young WT brain slices (14 DIV) and immunostained with CD68 (red), Ki67 (green) and M3.2 (blue) Scale bar: 50 µm.(D) Quantitative analysis of Ki67 and CD68 double positive microglial cells in the old APPPS1 tissue cultured alone or in co-culture with young brain slices (14 DIV) reveals increased microglial proliferation upon co-culturing of old and young brain slices. The values are expressed as percentages of CD68 and Ki67 double positive microglial cells from the total number of CD68 positive cells and represent mean ± SEM from 3 independent experiments, each experiment including 2 independent slice culture dishes (**P < 0.01). (E) Old APPPS1 tissue was incubated with conditioned media collected from young WT slices and in addition treated with proliferation inhibitor AraC (young CM + AraC) or vehicle Ctr (young CM + Ctr). Immunofluorescence analysis was performed at 14 DIV using CD68 (red) and M3.2 (green). Scale bar: 50 µm.(F) Quantitative analysis of core-only plaques in the old APPPS1 tissue (14 DIV) incubated with conditioned media from young WT slices and in addition treated with AraC or Ctr reveals a decreased number of core-only plaques upon AraC treatment. The values are expressed as percentages of core-only plaques from the total number of amyloid plaques and represent mean ± SEM from 3 independent experiments, each including 2 independent slice culture dishes (***P < 0.001). \n",
      ".Figure 2Hcp release assay. FLAG-tagged Hcp (HcpFL) release was assessed by separating cells (C) and cell-free culture supernatant (S) fractions from 109 cells of the indicated strains. Proteins were separated by 12.5%-acrylamide SDS-PAGE and TolB and HcpFL were immunodetected using anti-TolB (upper panel) and anti-FLAG (lower panel) antibodies. Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in triplicate and a representative result is shown. \n",
      ".Figure 5A. Remazol brilliant blue assay. The absorbance of supernatants from the reaction containing purified and RBB-labelled E. coli peptidoglycan and the indicated protein (50 µg) were measured at =595 nm after incubation for 0.5 or 4 hours at 37°C. The results shown are the average and standard deviation from triplicate reactions (n=3). Asterisks indicate significant differences compared to the buffer (NS, non significant; **, p < 0.01; ***, p < 0.001). The supernatant of the reaction after 4 hours of incubation is shown on bottom. B. Peptidoglycan hydrolysis. The decrease of the absorbance of the M. luteus peptidoglycan suspension in presence of the indicated protein (50 µg) was measured at =600 nm at 37°C over time. The experiment was performed in triplicate and a representative result is shown. \n",
      ".Figure 4A-B. Bacterial two-hybrid assay. BTH101 reporter cells producing the indicated proteins or domains fused to the T18 or T25 domain of the Bordetella adenylate cyclase were spotted on X-Gal indicator plates. The blue color of the colony reflects the interaction between the two proteins. TolB and Pal are two proteins known to interact but unrelated to the T6SS or the MltE proteins. The experiment was performed in triplicate and a representative result is shown. C. Co-immunoprecipitation assay. The solubilized lysates from 2×1010 E. coli K-12 W3110 cells co-producing the indicated FLAG-tagged TssMP variants (exported in the periplasm) and VSV-G-tagged MltE protein (Total, T) were subjected to immune precipitation on anti-FLAG-coupled agarose beads. The immunoprecipitated material (IP) was subjected to 12.5%-acrylamide SDS-PAGE and immunodetected with anti-FLAG (upper panel, TssM domains) and anti-VSV-G (lower panel, MltE) antibodies. Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in triplicate and a representative result is shown. \n",
      ".Figure 3A. Hcp release assay. FLAG-tagged Hcp (HcpFL) release was assessed by separating cells (C) and cell-free culture supernatant (S) fractions from 109 WT, ΔmltE cells or ΔmltE cells producing wild-type (mltE+) or E64Q mutant (mltEE64Q) VSV-G-tagged MltE (MltEV) from arabinose-inducible plasmids. Proteins were separated by 12.5%-acrylamide SDS-PAGE and TolB, HcpFL and MltEV were immunodetected using anti-TolB (upper panel), anti-FLAG (middle panel) and anti-VSV-G (lower panel) antibodies, respectively. Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in triplicate and a representative result is shown. B. Anti-bacterial activity. E. coli K-12 prey cells (W3110 gfp+, kanR) were mixed with the indicated attacker cells, spotted onto Sci-1 inducing medium (SIM) agar plates and incubated for 4 hours at 37°C. The image of a representative bacterial spot and the average and standard deviation (n=3) of the relative fluorescence of the bacterial mixture (in arbitrary unit, AU) are shown in the upper graph. The number of recovered E. coli prey cells (counted on selective kanamycin medium) is indicated in the lower graph (in log10 of colony-forming unit (cfu)). The black, dark grey and light grey circles indicate values from three independent assays, and the average is indicated by the bar. The experiment was performed in triplicate and a representative result is shown. Asterisks indicate significant differences compared to the wild-type attacker strain (NS, non significant; ***, p < 0.001). \n",
      ".Figure 7Biogenesis of the TssJLM complex begins with the initial positioning of the TssJ outer membrane (OM) lipoprotein (a) and the sequential recruitment of the TssM (b) and TssL (c) inner membrane (IM) proteins. TssM binds to the TssJ lipoprotein via its C-terminal -domain 3 (dark blue) and recruits the MltE LTG (green ball) via its -domain 1+2 (light blue). The TssM-mediated activation of MltE creates localized degradation in the cell wall (PG) allowing polymerization of the TssJLM complex (d). The crystal and electron microscopy structures are shown (TssJ, PDB:3RX9 64; TssMP-TssJ complex, PDB:4Y7O 30; TssL cytoplasmic domain, PDB: 3U66 87; TssJLM complex, EMDB:2927 30). \n",
      ".Figure 6A-B. Co-immunoprecipitation assay. The solubilized lysates from 2×1010 EAEC wild-type or ∆mltE cells producing FLAG-tagged TssM (FLTssM) and/or HA-tagged TssJ (TssJHA, panel A) or TssL (TssLHA, panel B) (Total, T) were subjected to immune precipitation on anti-FLAG-coupled agarose beads. The immunoprecipitated material (IP) was subjected to 12.5%-acrylamide SDS-PAGE and immunodetected with anti-FLAG (upper panel, TssM) and anti-HA (lower panel, TssJ or TssL) antibodies. Molecular weight markers (in kDa) are indicated on the left. The experiments were performed in triplicate and a representative result is shown. C. BS3 cross-linking assay. 2×109 cells of the indicated strain producing FLAG-tagged TssM (with the exception of ∆tssM cells) were treated (+) or not (-) with the BS3 cross-linker agent. After the cross-linking reaction, cells were boiled in Laemmli buffer and total proteins were subjected to 7%-acrylamide SDS-PAGE and immunodetected with anti-FLAG antibodies. The TssM protein (FLTssM) and its complexes (*, TssM-TssJ; **, TssM-TssL) are indicated on the right as well as the TssM dimer (arrow). Molecular weight markers (in kDa) are indicated on the left. The experiment was performed in triplicate and a representative result is shown. D. Fluorescence microscopy. Recordings showing TssM localization using the chromosomally-encoded sfGFP-tssM fusion in wild-type (WT) or ΔmltE cells or ΔmltE cells producing the wild-type (mltE+) or catalytic variant (mltEE64Q) MltE protein. Scale bars are 1 μm. The experiment was performed in triplicate and a representative result is shown. \n",
      ".Figure 4Live-cell Airyscan image of mitochondrion in HeLa cell, showing different membrane potentials in different mitochondrial regions. Membrane potentials were calculated based on TMRE FI differences between compartments (FIcomp). Regions of interest from cristae and IBM used for ΔΨm calculations are labeled with arrows on the right-hand side of the mitochondrion, using the cytosol as the reference compartment. Labeled with forks on the left-hand side of the mitochondrion are the calculations of ΔΨm between individual cristae and their neighboring IBM. Nernst equation used to calculate different voltages. Scale bar = 500 nm. LUT of mitochondrion in HeLa cell shown in A, color-coding TMRE FIs on scale of white (most intense) to blue (least intense): LUT scale shown in lower left-hand corner. Arrowheads indicate cristae. Scale bar = 500 nm. Note that most-intense pixels (white) only associate with cristae. Quantification of ΔΨm (mV) at cristae and IBM relative to the average ΔΨm of the whole mitochondrion, calculated as in A. Note: ΔΨm at cristae is significantly higher than ΔΨm at IBM, indicating electrochemical boundaries separate these two regions of the IMM. N = 3 independent experiments. Live-cell Airyscan imaging of mitochondria in cultured primary mouse hepatocytes stained with TMRE. Arrowheads denote cristae. D. Image showing TMRE-labeled mitochondria. E. LUT color-coding of TMRE FI. F. Quantification of ΔΨm differences using Nernst equation as shown in A. Scale bar = 500 nm. N = 2 independent experiments. Live-cell STED imaging of mitochondria in HeLa cell stained with TMRM. Arrowheads show cristae. Scale bar = 500 nm. G. Image showing TMRM-labeled mitochondrion. H. LUT color-coding of TMRM FI. I. Quantification of ΔΨm differences using Nernst equation as shown in A. Scale bar = 500 nm. N = 4 independent experiments. Note that calculation of ΔΨm differences between the compartments based on images captured by STED and by Airyscan were very similar. \n",
      ".Figure 2High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. Mitochondria from HeLa cell, co-stained with NAO and TMRE. Area from dashed white box, zoomed to right, shows the red and green intensities of TMRE and NAO colocalized. Scale bars = 500 nm. N = 3 independent experiments. Mitochondria in L6 myoblast, co-stained with NAO and TMRE. Area from dashed white box, zoomed to right, shows colocalizing NAO and TMRE at the cristae membrane (arrowheads). Scale bars = 500 nm. N = 3 independent experiments. High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. Mitochondria in HeLa cells, stained with NAO alone (top row) vs. NAO + TMRE (bottom row), and simultaneously excited with 488- and 561-nm lasers. Note that mitochondria stained with NAO alone do not emit noticeable fluorescence in the red channel; only after adding TMRE does strong signal appear in the red channel, showing negligible bleed-through. Scale bars = 500 nm. N = 2 independent experiments. \n",
      ".Figure 3Time-lapse Airyscan imaging of ΔΨm in living HeLa cell, co-stained with MTG (ΔΨm-insensitive after loading) and TMRE (ΔΨm sensitive). Arrowhead points to a flickering event where a mitochondrion depolarizes (~9 sec) and repolarizes (~16 sec), showing that heterogeneous signal from TMRE (but not MTG) disappears and reappears. Scale bar = 500 nm. N = 4 independent experiments. Quantification of A. Plot shows average TMRE fluorescence intensity (FI) of cristae (dark red line) vs. matrix (light red line) during the time series. The drop in TMRE FI during the depolarization phase of the flickering is the ΔΨm-sensitive component of the TMRE signal. The remaining TMRE FI during depolarization can be considered as the ΔΨm-insensitive portion of TMRE signal. Note that the remaining TMRE FI after depolarization at the cristae and matrix are approximately identical, indicating that differences in TMRE FI between the cristae and matrix prior to depolarization are derived from differences in ΔΨm. N = 4 independent experiments. \n",
      ".Figure 1High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. Images of IMM in living HeLa, L6, and H1975 cells, stained with 10-N-nonyl acridine orange (NAO). NAO preferentially binds phospholipids in the IMM, such as cardiolipin. Arrowheads indicate cristae in the IMM. Scale bar = 500 nm. N ≥ 3 independent experiments for each cell type. High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. Mitochondrion cropped from HeLa cell shown in A (dashed line) and zoomed in to show cristae, inner boundary membrane (IBM), and matrix (arrows). Scale bar = 100 nm. High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. H1975 cells transduced with matrix-targeted-DsRed and stained with NAO. Matrix-targeted Ds-Red differentiates matrix from cristae stained with NAO. Arrowheads point to cristae. Scale bar = 500 nm. N = 1 independent experiment. High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. The structure of the IMM in Mic13-KO cells (HeLa), stained with NAO. The number of cristae is decreased in Mic13-KO compared to control cells shown in panel A, labeled with arrowheads. Scale bar = 500 nm. N = 3 independent experiments. High-resolution imaging of mitochondria in live cells using the Airyscan module of Zeiss LSM880 confocal microscope. Live-cell imaging of the IMM in control and PTPMT1 KD H1975 cells, a model of cardiolipin deficiency. A gallery of mitochondria from various control H1975 cells expressing scrambled shRNA and stained with NAO, showing cristae (arrowheads). Scale bars = 500 nm. N = 3 independent experiments. A gallery of mitochondria from various H1975 cells expressing PTPMT1 shRNA and stained with NAO. Note the derangement of the ultrastructure (arrowheads). Scale bars = 500 nm. N = 3 independent experiments. \n",
      ".Figure 7Studies show that Opa1 interacts with MICOS complex, promoting closure of CJs. Thus, we tested whether Opa1 was required to maintain the difference in ΔΨm between cristae and IBM. Live-cell Airyscan images of MEF control (top row) vs. Opa1 KO (bottom row), co-stained with NAO and TMRE. Arrowheads indicate decreased intensity of TMRE FI at cristae compared to IBM. Note the more even distribution of TMRE staining, indicating the cristae and IBM are relatively equipotential. Scale bar = 500 nm. Quantification of G, showing Opa1-KO MEFs have significantly decreased ΔΨCr-IBM. N = 3 independent experiments. \n",
      ".Figure 11An equipotential representation of the IMM. In this model the IMM functions as a continuous electrical cable where cristae and IBM have similar ΔΨm and where all cristae respond simultaneously to any change in ΔΨm occurring in the mitochondrion where they reside. Transition from a hetero-potential to an equipotential, via cleavage of Opa1, for example, could play a role in normal mitochondrial physiology where conditions favor high conductance and low ATP synthesis, such as thermogenesis or adaptation to excess nutrient. Cartoon showing IMM as equipotential. Model of mitochondrial hetero-potential, showing cristae as individual bioenergetic units. Electrochemical autonomy of cristae protects the mitochondrion confronting a bioenergetic crisis, where dysfunction of one or more cristae would not inevitably cause the entire organelle to fail. In this model, cristae and IBM can have different membrane potentials. Closed CJs generate significant differences in charge between the cristae membranes and IBM, which could be related to the concentration of ETC subunits in the lateral side of the cristae and F1Fo ATP Synthase on the cristae rims. Cartoon showing IMM as hetero-potential. \n",
      ".Figure 8Live-cell Airyscan images of laser-induced depolarization time series of Mic10 KO Hap1 cell stained with TMRE. Mitochondrion exposed to rapid, high 2-photon laser power in region of white box, arrow (~0.3 sec); large, hyperpolarized vesicle (arrowhead) remains polarized after the rest of the mitochondrion loses ΔΨm, indicating electrochemical discontinuity between hyperpolarized vesicles and the rest of the mitochondrion, including the IBM. Scale bar = 500 nm. N = 2 independent experiments. Live-cell Airyscan image of time series showing hyperpolarized vesicles (arrowheads; region cropped from dashed white box) inside mitochondria lacking both Opa1 and Drp1. Scale bar = 500 nm. N = 2 independent experiments. Free movement of hyperpolarized vesicles within the matrix indicate their detachment from the IBM, representing an extreme case of cristae independency within one mitochondrion (see Movie EV 1 for time series). \n",
      ".Figure 10We calculated the ΔΨCr-IBM between neighboring cristae over time as a surrogate measure of cristae individuality and functional independence, using live-cell Airyscan imaging. Stability of cristae ΔΨCr-IBM over time. Representative images from time series of HeLa cells co-stained with MTG and TMRE. Top sequence shows merged green (MTG) and red (TMRE) channels. Bottom sequence shows color-coded LUT of TMRE signal of the same sequence shown above. White arrowheads in the image point to specific cristae that were measured over time. Scale bar = 500 nm. N = 5 independent experiments. Quantification of A, showing average ΔΨCr-IBM of Cristae \"A-D\" over time. Note that Cristae C and D maintain more than double the ΔΨCr-IBM of Cristae A and B. Error bars indicate STDEV. Quantification of A, showing standard deviation in ΔΨCr-IBM (mV) of individual cristae over time compared to the standard deviation in ΔΨCr-IBM (mV) among different cristae along a single mitochondrion. N = 5 independent experiments. Note that there is less variability in ΔΨCr-IBM of a single crista over time within the same mitochondrion compared to the variability of ΔΨCr-IBM among cristae over time, indicating cristae possess unique bioenergetic properties. We calculated the ΔΨCr-IBM between neighboring cristae over time as a surrogate measure of cristae individuality and functional independence, using live-cell Airyscan imaging. Representative live-cell Airyscan image from time series showing merged green (MTG) and red (TMRE) channels. Area marked by dashed box is shown in the time series in E. Scale bar = 500 nm. Time series of cropped region from D, showing tip of mitochondrion (arrowhead) maintaining ΔΨm while neighboring region of same mitochondrion depolarizes. The capacity of some cristae to maintain polarity, while adjacent cristae transiently depolarize, indicates cristae can exhibit a significant amount of electrochemical autonomy. N = 3 independent experiments. (See Movies EV 3, 4 for time series.) Zoomed-in LUT of mitochondrion from E (~2 sec), showing some cristae remaining polarized (red) while adjacent cristae depolarize (blue). (See Movie EV 5 for time series.) Quantification of D-F, showing that during a flickering event some cristae maintain their membrane potentials while other cristae depolarize, resulting in large differences in ΔΨCr-IBM along the individual mitochondrion. N = 3 independent experiments. \n",
      ".Figure 5Response of ΔΨm to the F1Fo ATP Synthase inhibitor, oligomycin (10 µM), compared to the uncoupler, FCCP (10 µM), determined using live-cell Airyscan imaging of HeLa cells co-stained with MTG and TMRE. Quantification of total mitochondrial TMRE FI. N = 3 independent experiments. Quantification of cristae ΔΨm relative to IBM (ΔΨCr-IBM) in response to oligomycin compared to FCCP. F1Fo ATP Synthase is exclusively localized to the cristae and significant increase in ΔΨCr-IBM in response to blocking of F1Fo ATP Synthase with oligomycin indicates that differences in TMRE FI between cristae and IBM are driven by oxidative phosphorylation. N = 3 independent experiments. Response of ΔΨm to the F1Fo ATP Synthase inhibitor, oligomycin (10 µM), compared to the uncoupler, FCCP (10 µM), determined using live-cell Airyscan imaging of HeLa cells co-stained with MTG and TMRE. Representative Airyscan images showing mitochondria from living HeLa cells stained with MTG and TMRE. Cells were first stained for one hour and only then treated with oligomycin or FCCP. Images show hyperpolarization of cristae under oligomycin (middle row; arrowheads) and depolarization of cristae under FCCP (bottom row; arrowheads) compared to control (top row; arrowheads). Scale bar = 500 nm. N = 3 independent experiments. Note that the ΔΨm-independent fraction of TMRE staining is less than 5% of the signal in control conditions. Zoomed-in region of FCCP-treated mitochondria in C. Increased contrast was used to visualize TMRE to compensate for TMRE loss induced by FCCP. These images show diminished heterogeneity of ΔΨCr-IBM values. \n",
      ".Figure 6The role of cristae structure as well as CJ formation and sealing on the generation of the difference in ΔΨm between cristae and IBM were studied Representative images of mitochondria in control (top row) vs. Mic13-KO (bottom row) HeLa cells. Note that TMRE FI in Mic13-KO mitochondria is distributed more evenly along the IMM, so that ΔΨCr-IBM is decreased. Scale bar = 500 nm. N = 3 independent experiments. Quantification of A, showing deletion of Mic13 in HeLa cells results in significant decrease in ΔΨCr-IBM. While Mic13-KO cells show a substantial decrease in cristae number, the loss of cristae does not appear absolute, making it feasible to compare TMRE FI at cristae relative to IBM. N = 3 independent experiments. The role of cristae structure as well as CJ formation and sealing on the generation of the difference in ΔΨm between cristae and IBM were studied by disrupting cristae using Mic10- and Mic60-deficient cells. Mic60 support cristae formation. Mic10 is essential for CJ formation, and, in its absence, cristae tend to remain as vesicles detached from IBM. Live-cell Airyscan imaging of TMRE was used in all figure panels and models. Representative images of Hap1 control (top row) vs. Mic60 KO (middle row) and Mic10 KO (bottom row); Mic60-KO mitochondria show very few cristae structures. TMRE staining along the IMM is relatively homogeneous, indicating decreased ΔΨCr-IBM. In general, Mic10 KO in Hap1 cells results in decreased TMRE signal intensity at cristae relative to IBM (arrowheads) as compared to control cells. Deletion of Mic10 induces detachment of cristae from IBM (see Figure 8). Except for the detached cristae vesicles, the TMRE staining is homogeneously distributed n Mic10-KO cells, indicating decreased ΔΨCr-IBM. Scale bar = 500 nm. N = 3 independent experiments. Quantification of C, showing Mic60 KO in Hap1 cells results in significant decrease in ΔΨCr-IBM. While Mic60-KO cells show a marked loss of normal cristae structures, depletion of cristae does not appear absolute, making it possible to compare TMRE FI at cristae relative to IBM. N = 3 independent experiments. Quantification of C, showing Mic10 KO in Hap1 cells results in a significant decrease in ΔΨCr-IBM. Note: quantification of ΔΨm at cristae relative to IBM was performed only on cristae structures that appeared to maintain attachment to IBM. N = 3 independent experiments. \n",
      ".Figure 9Laser-induced mitochondrial membrane depolarization in L6 myoblasts shows gradual depolarization of elements along the IMM, visualized using live-cell Airyscan microscopy. Representative images of laser-induced depolarization time series from L6 myoblast, stained with ΔΨm-dependent dye, Rho123. Images show LUT color-coded for Rho123 FI. White and blue pixels represent most and least intense ΔΨm, respectively. See legend of LUT colors on the right. Note that at the first time point (~0.15 sec) white arrow points to box marking area of mitochondrion exposed to phototoxic pulse of 2-photon laser (≤5 msec), inducing depolarization. Ensuing frames show wavelike depolarization away from site of perturbation (arrowheads). Scale bar = 500 nm. N = 3 independent experiments. (See Movie EV 2 for time series.) Profile plot showing Rho123 pixel intensity as function of distance along mitochondrion in A. Note that zero value in the X axis corresponds to top area of the mitochondrion shown in A, near site of 2-photon laser exposure (box). Red line corresponds to frame immediately before depolarization (~0.15 sec); green, blue, purple, and dark purple lines correspond to ~0.30, ~0.60, ~1.05, and ~2.10 sec, respectively, after initial depolarization, showing wavelike dissipation of ΔΨm over time. Laser-induced mitochondrial membrane depolarization in living human skin fibroblasts co-stained with TMRE and MTG and imaged with Airyscan, showing gradual depolarization of elements along the IMM. Time-lapse images of merged green (MTG) and red (TMRE) channels following laser-induced mitochondrial membrane depolarization. Note that, at the first time point (0.3 sec), white arrow points to box marking area of mitochondrion exposed to phototoxic pulse of 2-photon laser (≤5 msec), inducing depolarization. Depolarization is marked by the dissipation of the ΔΨm-dependent dye (TMRE) while the green (MTG) signal persists. Scale bar = 1 µm. N = 4 independent experiments. TMRE channel from C, showing gradual of the loss of TMRE along the IMM. Note that although depolarization has largely completed at the area near the phototoxic stimulus already at 0.9 sec, some specific regions at the very top maintain ΔΨm while others depolarize. Quantification of ΔΨm using TMRE FI from C-D. Measurement of TMRE pixel intensities immediately after laser-induced depolarization at site distal (≥ 10 µm) vs. proximal (≤ 1 µm) to box. N = 4 independent experiments; see specific p values in panel. Quantification of C-D. Percentage of mitochondria that depolarize in a wavelike (i.e., gradual) vs. instantaneous manner after laser-induced depolarization. Note: imaging at high temporal resolution (~100-500 msec/frame) reveals wavelike depolarizations predominate, suggesting the ΔΨm is composed of multiple, disparate electrochemical domains along the IMM. The time scale of propagation of depolarization is slower than the propagation of electrical phenomena. N = 4 independent experiments. \n",
      ".Figure 6Flow cytometry analysis of SB (A), 10C (B), DLD1 (C) and HCT116 (D) cells incubated with CFSE-labeled F. nucleatum 12230 (Fn) or its fadA-deletion mutant US1 (US1) at MOI of 10-20:1 for the indicated time and immuno-stained with anti-Annexin A1 antibodies. Shown on the top panels are the density plots. x-axis, CFSE-labeled F. nucleatum or US1 (CFSE-Fn); y-axis, Annexin A1 (ANXA1). Shown on the bottom panels are the percentages of Annexin A1-positive (solid bars) or negative (clear bars) cells bound by F. nucleatum or US1 out of the total number of cells analyzed. Data are mean values ± SD. The experiments were performed in triplicates and repeated 2-3 times. *p<0.05, **p<0.01, ***p<0.001 (two-way ANOVA). \n",
      ".Figure 4(A) Real-time qPCR analysis of ANXA1 mRNA levels in SB, 10C, HCT116 and DLD1 cells incubated with wild-type F. nucleatum 12230 (Fn) or fadA-deletion mutant US1 (US1) at MOI of 50:1 for the indicated time periods. The results were normalized to those obtained with untreated cells and were the mean of 3 independent experiments each performed in triplicates. *p<0.05, **p<0.01 and ***p<0.001 (two-way ANOVA). (B) Flow cytometry analysis of Annexin A1 expression in 10C cells either untreated, or incubated with BSA (1000 µg/ml), or mFadA (1000 µg/ml), or FadAc (100, 300, or 1000 µg/ml) for 1 hr. Data are mean values ± SD. The experiment was performed in triplicates and repeated more than three times. **p<0.001 (one-way ANOVA). (C) Flow cytometry analysis of Annexin A1 in 10C cells transfected with control siRNA (dotted black line) or CDH1-specific siRNA (dotted red line) followed by no treatment, or incubation with BSA (1000 µg/ml) or FadAc (1000 µg/ml) for 1 hr. C, untreated control. Data are mean values ± SD. The experiment was performed in triplicates and repeated twice. ***p<0.001 (two-way ANOVA). (D) Confocal microscopy analysis of SB and 10C cells either untreated or following incubation with CFSE-labeled F. nucleatum 12230 (Fn) for 1 hr at MOI of 5:1. Annexin A1 was stained green and E-cadherin blue. Imagines are 800x magnification. Note the enhanced expression of Annexin A1 in 10C compared to SB and its location on the outer layer of the cell mass. The experiment was repeated three times. Scale bar, 250 nm. \n",
      ".Figure 2Relationship between ANXA1 mRNA expression levels and disease-free survival (DFS) in colon cancer patients. The relationship between ANXA1 mRNA expression levels and DFS was investigated in a database of 466 primary colon carcinomas, assembled by pooling four independent gene-expression array datasets from the NCBI-GEO online repository (GSE14333, GSE17538, GSE31595, GSE37892) The association between ANXA1 expression levels and DFS was tested using Kaplan-Meier survival curves, after patient stratification in groups with high, medium and low ANXA1 expression, using three different methods: (A) based on the median of ANXA1 mRNA expression levels (high 50% vs. low 50%); (B) based on the quartile distribution of ANXA1 mRNA expression levels (high 25% vs. middle 50% vs. low 25%); and (C) based on ANXA1 mRNA expression thresholds calculated using the StepMiner algorithm (low vs. high) Overall, high ANXA1 mRNA expression levels were associated with a statistically significant reduction in DFS (p<0.001, log-rank test), irrespective of the method used for the stratification. Differences in ANXA1 mRNA expression levels did not appear to correlate with differences in each tumor's relative content of epithelial cells in the analyzed biospecimens (i.e. tumor cell density) as revealed by the lack of visual correlations with the epithelial cell marker Desmoplakin (DSP). \n",
      ".Figure 1(A) Lung cancer cells PC-9, prostate cancer cells 22RV1, bladder cancer cells UMUC3, breast cancer cells MCF-7, colonic adenoma-derived non-cancerous cells AA/C1 (aka C1) and AA/C1/SB (aka SB), or cancerous cells AA/C1/SB/10C (aka 10C) were incubated with wild-type F. nucleatum 12230 (Fn), the fadA-deletion mutant US1 (US1), or E. coli DH5α (E. coli) at multiplicity of infection (MOI) of 1,000:1. Cells numbers are mean values ± SEM. The experiment was performed in triplicates and repeated three times. **p<0.01, ***p<0.001, compared to untreated controls; #p<0.05, ### p<0.001, compared to US1-treated cells (two-way ANOVA). (B) Attachment (left panel) and invasion (right panel) of wild-type F. nucleatum 12230 (Fn) to the non-cancerous SB cells, either untreated or transfected with antisense or sense ANXA1, and to the cancerous 10C cells, either untreated or transfected with control or ANXA1- or CDH1-specific siRNA or both (MOI 50:1). F. nucleatum attachment and invasion to the untreated SB cells were designated as 100%, respectively; all other values were expressed as relative to those obtained with untreated SB. Data are mean values ± SEM. The experiment was performed in triplicates and repeated four times. *p<0.05, **p<0.01 and ***p<0.001 (one-way ANOVA). (C) Left panel: qPCR analysis of Villin1 (VIL1) mRNA levels in 10C cells treated with control siRNA or VIL1-specific siRNA, demonstrating knocking down of Villin1. Right panel: attachment of F. nucleatum 12230 to 10C cells treated with control siRNA or VIL1-specific siRNA. Data are mean values ± SD. The experiment was performed in triplicates and repeated twice. *p<0.05 (student's t-test). \n",
      ".Figure 3(A) Real-time qPCR analysis of ANXA1 expression using mRNA extracted from the non-cancerous SB, cancerous 10C, and human CRC cell lines HCT116, DLD1 and RKO, each grown to 50% or 100% confluency. All results were normalized to the ANXA1 mRNA levels in SB cells of 100% confluency, which was designated as 1. Data are mean values ± SEM. The experiment was performed in triplicates and repeated twice. **p<0.01 and ***p<0.001 (student's t-test). (B) Confocal microscopy analysis of SB and 10C cells grown to 20% (top panels) or 100% (bottom panels) confluency followed by immuno-fluorescence staining of Annexin A1 (green). The nuclei were stained with DAPI (blue). A series of 20-50 consecutive images in the z-axis were stacked together to generate the 3-D figure at 400x magnification. Annexin A1 is most abundantly expressed on the outer layer of 20% confluent 10C (pointed by arrows compared to 100% confluent 10C, or SB of either confluency. Scale bars, x = 1 μm, y = 1 μm, z = 1.6 μm. The experiment was repeated at least twice. (C) Cell proliferation assay of adenoma-derived non-cancerous SB and cancerous 10C and human CRC cell lines HCT116, DLD1, SW480, HT29 and RKO either untreated (black lines) or following treatment with control siRNA (gray lines) or ANXA1-specific siRNA (green lines). Data are mean values ± SEM. The experiment was performed in triplicates and repeated three times. *p<0.05, ** p<0.01 and *** p<0.001, compared to the untreated cells (two-way ANOVA). (D) Cell proliferation assay of SB (left panel) and RKO (right panel) cells transfected with ANXA1 (red line), as compared to the control cells (black lines). Data are mean values ± SEM. The experiment was performed in triplicates and repeated three times. *p<0.05, **p<0.01 and ***p<0.001 (two-way ANOVA). (E) Xenografted tumor growth in nude mice following subcutaneous and bilateral inoculation of HCT116 cells transfected with control siRNA or ANXA1-specific siRNA. The tumor volumes were measured after 7 days post injection (left panel, n=4). For each mouse, the tumor resulting from ANXA1-specific siRNA treated cells was normalized to that from control siRNA-treated cells, which was designated 100%. The line represents the average. *p<0.05 (paired t-test). The individual tumor pairs are shown on the right panel: top, tumors arising from control siRNA treated cells; bottom, tumors arising from ANXA1-specific siRNA treated cells. \n",
      ".Figure 5(A) Western blotting analysis of FadA, E-cadherin (CDH1), Annexin A1 (ANXA1) and β-catenin in DLD1 cells following incubation with FadAc for 15 or 120 min. C, untreated cells. β-actin was included as an internal control. The experiment was repeated three times. (B) Co-immunoprecipitation with Annexin A1. DLD1 cell lysates were incubated with FadAc for 15 or 120 min and mixed with agarose beads conjugated with rabbit anti-Annexin A1 polyclonal antibody (α-Annexin A1) or control rabbit IgG. FadA, E-cadherin (CDH1), Annexin A1 (ANXA1) and β-catenin in the eluates were detected by Western blot. C, untreated control. The experiment was repeated three times. (C) Co-immunoprecipiation with FadA. DLD1 cell lysates were incubated with FadAc for 15 or 120 min and mixed with agarose beads conjugated with mouse anti-FadA monoclonal antibody (α-FadA) or control mouse IgG. FadA, E-cadherin (CDH1), Annexin A1 (ANXA1) and β-catenin in the eluates were detected by Western blot. The experiment was repeated three times. (D) Confocal microscopy analysis of 10C cells either untreated (top panel) or following incubation with CFSE-labeled F. nucleatum 12230 (red, bottom panel) for 3 hrs and then immunofluorescent-stained for Annexin A1 (green) and E-cadherin (blue). Images are 1200x magnification. A side view of the enlarged image is shown on the far right. Note the enhanced expression of Annexin A1 in the F. nucleatum-bound cells and the co-localization of Annexin A1, E-cadherin and F. nucleatum on the cell membranes (arrows). The experiment was repeated more than three times. Scale bar, 500 nm. (E) Confocal microscopy analysis of 10C cells following incubation with Alexa Fluor™ 488-conjugated BSA, mFadA, or FadAc (red) for 1 hr and immuno-staining of Annexin A1 (green) and E-cadherin (blue). Images are 1200x magnification. Note the enhanced expression of Annexin A1 and its co-localization with E-cadherin in the presence of FadAc (arrows), but not with BSA and mFadA. The experiment was repeated twice. The side views are shown to the right and bottom of each image. Scale bar, 500 nm. \n",
      ".Figure 7(A) Colorectal tumors generated in Apcmin/+ mice following treatment with PBS, E. coli DH5α (E. coli), fadA-deletion mutant US1 (US1) or F. nucleatum 12230 (Fn). Each symbol represents one mouse. Horizontal lines represent mean values. Representative tumors formed in the mouse colon are shown on the right, pointed by blue arrows on the right. *p<0.05, **p<0.01 (one-way ANOVA). (B) ANXA1 mRNA levels in Apcmin/+ mouse colonic tumors and normal colonic tissues as measured by real-time qPCR. Each symbol represents one mouse. Horizontal lines represent mean values. *p<0.05, **p<0.01 (two-way ANOVA). (C) Positive correlation between fadA gene copy numbers (x-axis) and ANXA1 mRNA levels (y- axis) in F. nucleatum-induced Apcmin/+ mouse colonic tumors (n=34; Pearson's correlation). Each dot represents the average of qPCR results performed in duplicates (D) Abundance of fadA in the paired normal and adenocarcinoma tissues from human CRC patients (n=18) expressed as ratio of fadA over total 16S rRNA genes determined by qPCR. Each symbol shows the average of duplicate qPCR results from one patient. Horizontal lines represent mean values. **p<0.01 (paired t-test). (E) ANXA1 mRNA levels in paired normal and adenocarcinoma tissues from CRC patients (n=18). Each symbol shows the average of duplicate qPCR results from one patient. Horizontal lines represent mean values. **p<0.01 (paired t-test). (F) Positive correlation between fadA gene copy numbers (x-axis) and ANXA1 mRNA levels (y-axis) in human colorectal adenocarcinoma tissues (n=18; Pearson's correlation). Each dot represents the average of qPCR results performed in duplicates. (G) Confocal microscopy analysis of paired normal and carcinoma tissues from two colon cancer patients. The frozen sections were incubated with rabbit anti-AnnexinA1 polyclonal antibodies and 5G11 mouse anti-FadA monoclonal antibodies. The slides were then stained with Alexa Fluor®680-conjugated donkey anti-rabbit and Alexa Fluor®555-conjugated goat anti-mouse, washed, and covered in mounting medium containing DAPI. The scanning confocal microscopy mages were taken with a Nikon Ti Eclipse inverted microscope at 200x magnification for the normal tissues and 400x for the carcinomas. Scale bar, 50 μm. Co-localization of FadA (red) and Annexin A1 (green) was observed in carcinomas but not in the paired normal tissues. \n",
      ".Figure 9In non-cancerous cells (top panel), there is low level of Annexin A1 (ANXA1) and weak binding of FadA to E-cadherin (CDH1). In cancerous cells (bottom panel), Annexin A1 level increases, FadA binding enhances, FadA-E-cadherin-Annexin A1-β-catenin complex forms, β-catenin is activated, resulting in acceleration of cancer progression. CM, cell membrane. \n",
      ": Compartmentalization of DRiPs and polyUb proteins is catalyzed by the self-assembly of PML into nuclear bodies.Figure 4HeLa cells were lipofected with non-targeting siRNA control or a specific siRNA against PML. 72 h post-transfection, total proteins were extracted and expression levels of PML were analyzed by immunoblotting. TUBA4A was used as loading control. HeLa cells were lipofected with non-targeting siRNA control or a specific siRNA against PML. 72 h post-transfection, cells were treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Left panel: automated quantitation of the number of PML-NBs/nucleus is reported. Statistical significance via 2‐tailed Student's t‐test. Cell number analyzed: 334 - 364; p < 10-10. Right panel: automated quantitation of the number of DRiP foci/nucleus. Cell number analyzed: 334 - 364; p < 10-10. Automated DRiP segmentation is based on azide signal. DRiPs and polyUb (FK1) protein distribution in cells treated as described in (A). Automated DRiP segmentation is based on azide signal. Quantitation of the number of DRiP foci enriched for polyUb (>2) is shown; n = 590 - 2114; statistical significance via 2‐tailed Student's t‐test; p < 10-10. Scale bars: 10 µm. Immunoblotting on protein extracts from HeLa cells lipofected with non-targeting siRNA control or a specific siRNA against Ubc9. A dilution curve of the control sample is shown. TUBA4A was used as loading control. Quantitation of the number of PML-NBs/nucleus from cells treated as described in (D). Statistical significance via 2‐tailed Student's t‐test. Cell number analyzed: 192 - 430; p < 10-10. Automated PML-NB segmentation is based on PML signal. Cells lipofected for 72 h with non-targeting siRNA control or a specific siRNA against Ubc9 were treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Automated quantitation of the number of PML-NBs/nucleus is reported. Statistical significance via 2‐tailed Student's t‐test. Cell number analyzed: 151 - 225; p < 10-10 (see panel G). Quantitation of the number of PML-NBs/nucleus from cells treated as described in (D). Statistical significance via 2‐tailed Student's t‐test. Cell number analyzed: 192 - 430; p < 10-10. Automated PML-NB segmentation is based on PML signal. Representative pictures showing that Ubc9-depleted cells cannot efficiently induce the self-assembly of PML-NBs upon treatment with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Scale bars: 10 µm. \n",
      ": DRiPs accumulate in nucleoli of normally growing HeLa cells (see also ).Figure 1DRiP labelling in GFP-NCL HeLa Kyoto cells that were either left untreated or treated with OP-puro (25 µM) for 2 h. Where indicated cells were allowed to recover in drug-free medium (control) or in presence of MG132 (10 µM) for 5 h. Scale bars: 5 µm. GFP-NCL HeLa Kyoto cells were treated as described in A, with the exception that a lower concentration of OP-puro was used (5 µM). The distribution of DRiPs and GFP-NCL was analyzed by STED super-resolution microscopy. Scale bars: 5 µm. Isolation of nucleoli from HeLa Kyoto cells untreated or treated with puromycin (25 µM) for 2 h. Puromycylated proteins were detected by immunoblotting (right panel). The left panel shows the purity of isolated nucleoli by western blotting, where fibrillarin was used as nucleolar marker, FUS as nucleoplasmic marker and TUBA4A as cytoplasmic marker. \n",
      ".Figure 8(A) Flow cytometry analysis of β-catenin expression in 10C, HCT116 and DLD1 cells following transfection with control siRNA (clear bars) or ANXA1-specific siRNA (solid bars) and incubation with F. nucleatum 12230 at MOI of ~20:1 for indicated time periods. The geometric means of cells treated with control siRNA at time 0 was designated as 1. Data are mean values ± SD. The experiment was performed in duplicates or triplicates and repeated 1-3 times. **p<0.01, *** p<0.001 (two-way ANOVA). (B) Immuno-staining of β-catenin in 10C cells following transfection with control siRNA or ANXA1-specific siRNA and incubation with F. nucleatum 12230 (Fn) at MOI of ~100:1 for 2 hrs. β-catenin was stained with Alexa Fluor®680 (red) and the nuclei with DAPI (blue). The images were captured with confocal microscope at 800x magnification. -, no bacteria added. Note the increased expression of β-catenin and its nucleus translocation in response to F. nucleatum in control siRNA-treated cells, but not in ANXA1 siRNA-treated cells. The experiment was repeated twice. Scale bar, 200 nm. (C) Real-time qPCR analysis of CCND1 expression using mRNA extracted from the cancerous 10C, and human CRC cells HCT116 and RKO, each grown to 50% or 100% confluency. Data are mean values ± SEM. The experiment was performed in triplicates and repeated twice. *p<0.05, ***p<0.001 (student's t-test). (D) Western-blot analysis of Cyclin D1, Annexin A1, and β-actin in RKO cells transfected with control vector or ANXA1. Induction of Cyclin D1 was observed in response to increased Annexin A1. The experiment was repeated twice. \n",
      ": PolyUb proteins are cleared from PML-NBs by chaperones (see also and Appendix -5).Figure 5PolyUb protein (FK1) labelling in PML-GFP HeLa Kyoto cells that were either left untreated or treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Cells were either immediately fixed or let to recover for 5 h in drug-free medium (Control), with VER (40 µM) or Eeyarestatin I (5 µM). Scale bars: 10 µm. Quantitation of the % of cells with polyUb (FK1) nuclear foci shown in (A). Number of cells counted/condition: 310 - 1258 in three independent experiments; statistical significance via One-way ANOVA; p = 0.0003 or 10-6, ± s.e.m. Quantitation of the number of PML-NBs enriched for polyUb (>2); n = 611 - 2612; statistical significance via One-way ANOVA; p < 10-10. Automated PML-NB segmentation is based on PML signal. Quantitation of the % of cells with polyUb proteins (FK1) sequestered in nuclear foci. HeLa cells were treated with OP-puro (25 µM) and heat shock (HS) at 42˚C for 2 h. Then, cells were either immediately fixed or let to recover for 3 h in drug-free medium (control), with VER (40 µM) or Eeyarestatin I (5 µM). Number of cells counted/condition: 593 - 1016 in three independent experiments; statistical significance via One-way ANOVA; p = 10-3, ± s.e.m. Quantitation of the number of PML-NBs enriched for polyUb (>2) in HeLa cells treated as described in (D) but with a recovery time of 2 h; n = 1155 - 3283; statistical significance via One-way ANOVA; p < 10-10. Automated PML-NB segmentation is based on PML signal. Amylo-Glo staining of PML-GFP HeLa Kyoto cells that were subjected to the following treatments: untreated (Control); MG132 (10 µM) and OP-puro (25 µM) for 4 h, alone or combined with CHX (50 μg/ml). Where indicated cells were allowed to recover for 5 hrs in drug-free medium (Control) or with VER (40 µM). Confocal images showing Amylo-Glo and PML-GFP. Scale bars: 5 µm. Amylo-Glo staining of PML-GFP HeLa Kyoto cells that were subjected to heat shock (HS) at 42˚C for 2 hrs. Confocal images showing Amylo-Glo and PML-GFP. Scale bars: 5 µm. \n",
      ": Upon heat shock or proteasome inhibition, DRiPs promote amyloidogenesis in NoABs (see also ).Figure 2HeLa cells were left untreated or treated with heat shock (HS) at 42˚C for 3 h, alone or with cycloheximide (CHX; 50 μg/ml), or Actinomycin D (Act. D; 3 µM) followed by staining with the dye Amylo-glo. DRiP labelling in GFP-NCL HeLa Kyoto cells that were either left untreated or treated with HS at 42˚C and OP-puro (25 µM) for 2 h. Where indicated, cells were cotreated with CHX (50 μg/ml). HSPA8 subcellular distribution and recruitment inside nucleoli following exposure to HS at 42˚C for 2 h, alone or combined with CHX, in GFP-NCL HeLa Kyoto cells. DRiP labelling in HeLa cells treated with MG132 (10 µM) alone or with CHX (50 μg/ml) for 8 h; where indicated, OP-puro (25 µM) was added during the last 45 min, prior to cell fixation. Amylo-glo staining of HeLa cells left untreated or exposed to MG132 (10 µM) alone or combined with CHX (50 μg/ml) for 8 or 13 h. Amylo-glo staining of HeLa cells left untreated or subjected to transcriptional stress (MG132 10 µM and Act. D 4 µM) for 6 hrs. Where indicated, translation was concomitantly inhibited with CHX (50 μg/ml). Amylo-Glo staining is also shown in HeLa cells treated with Act.D alone or with CHX for 6 hrs, as control. Amylo-glo staining of HeLa cells exposed to HS at 42˚C for 3 h and let to recover for 3 h in drug-free medium (+ rec. control) or with VER (40 µM; + rec. VER). Where indicated cells were treated with VER alone (VER 3 h). Amylo-glo staining of HeLa cells treated with MG132 (10 µM) for 13 h and let to recover for 5 h in drug-free medium (+ rec. control) or with VER (40 µM; + rec. VER). Where indicated cells were treated with VER alone (VER 5 h). HeLa cells were transfected with vectors encoding for GFP, GFP-GR50 or RPL23a-GFP. 24 hrs post-transfection cells were fixed and stained with Amylo-glo. The percentage of transfected cells with Amylo-Glo positive nucleoli is shown. Number of cells counted/condition: GFP, 412; RPL23a-GFP, 482; GFP-GR50, 359, in three independent experiments; statistical significance via One-way ANOVA; p < 10-7, ± s.e.m. \n",
      ": Upon heat shock or proteasome inhibition, DRiPs, polyUb proteins and 20S proteasomes accumulate in PML-NBs (see also -3, EV6 and EV7).Figure 3DRiPs, polyUb proteins (FK1) and PML labelling in HeLa cells treated with OP-puro (25 µM) and heat shock (HS) at 42˚C for 2 h; where indicated cycloheximide was also added (CHX; 50 μg/ml). Scale bars: 5 µm. Lower panel: STED super-resolution microscopy showing colocalization of DRiPs with PML-NBs in PML-GFP HeLa cells treated with a low concentration of OP-puro (5 µM) and heat shock (HS) at 42˚C for 2 h. PML-NBs were visualized using a PML specific antibody, followed by incubation with alexa fluor 647. For convenience the alexa fluor 647 signal is shown in green. Scale bars: 5 µm. PolyUb proteins (FK1) labelling in HeLa cells left untreated or treated with HS at 42˚C for 2 h, alone or combined with CHX (50 μg/ml). Scale bars: 5 µm. STED super-resolution microscopy showing colocalization of endogenous polyUb proteins (FK1) and PML-NBs in PML-GFP HeLa cells treated with HS at 42˚C for 2 h. Scale bars: 5 µm. Quantitation of cells shown in A-C. Number of cells counted/condition: 712 - 1343 in three independent experiments; statistical significance via One-way ANOVA; p < 10-5, ± s.e.m. DRiPs, polyUb proteins (FK1) and PML labelling in HeLa cells treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h; where indicated, cycloheximide was also added (CHX; 50 μg/ml). Asterisks (*) indicate nucleoli. Scale bars: 10 µm. Lower panel: STED super-resolution microscopy showing colocalization of DRiPs with PML-NBs in PML-GFP HeLa cells treated with a low concentration of OP-puro (5 µM) and MG132 (10 µM) for 4 hrs. Scale bars: 5 µm. Quantitation of cells shown in E. Number of cells counted/condition: 298 - 1713 in three independent experiments; statistical significance via One-way ANOVA; p ≤ 0.001, ± s.e.m. Intranuclear distribution of 20S proteasomes in HeLa cells left untreated or treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Asterisks (*) indicate nucleoli. Scale bars: 10 µm. \n",
      "a, b) Stable Mito-DsRed (pseudo coloured; blue)-expressing HeLa cells were transfected with Flag–Parkin wild-type or pathogenic mutants.figf4a, b) Stable Mito-DsRed (pseudo coloured; blue)-expressing HeLa cells were transfected with Flag-Parkin wild-type or pathogenic mutants. Cells were treated with CCCP before immunostaining of Parkin with anti-Flag (green) and endogenous poly-ubiquitin with a specific antibody FK1 (red). (a) In contrast to pathogenic mutations, wild-type Parkin shows colocalization with poly-ubiquitin after 6 h of CCCP treatment at the site of condensed mitochondria. (b) After 24 h of CCCP treatment, K161N and R275W Parkin mutants colocalize with both poly-ubiquitin and mitochondria. In contrast, translocation-deficient Parkin mutants K211N, T240R and G430D lack the mitochondria-associated poly-ubiquitin signal. Representative images are shown. Scale bars, 10 μm. Complementary Parkin of untreated cells and quantification of colocalization of Parkin, mitochondria and poly-ubiquitin upon CCCP treatment are shown in Supplementary Information, Fig. S3a, b and c, respectively. (c, d) Parkin-directed mitophagy specifically involves linkage of ubiquitin through lysines 27 and 63. HeLa cells were transfected with EGFP-Parkin wild-type and ubiquitin lysine variants and treated with CCCP before immunostaining with anti-HtrA2/Omi, as a mitochondrial marker (pseudo coloured; blue) and anti-His (red) to detect ectopic ubiquitin. Parkin was visualized by its fusion to EGFP (green). (c) Note that only wild-type, K27 and K63 6×His-tagged ubiquitin variants are colocalized with condensed mitochondria and EGFP-Parkin after 6 h of CCCP treatment. All other ubiquitin mutants interfere with Parkin translocation to mitochondria. (d) Only co-expression of K27 or K63 ubiquitin recapitulates mitochondrial clearance similarly to 6×His-tagged wild-type ubiquitin after 24 h of CCCP treatment. Representative images are shown. Scale bars, 10 μm. Immunostaining of untreated cells and all other ubiquitin variant combinations tested are shown in Supplementary Information, Fig. S4a and b, c, respectively. WT, wild-type; PolyUb, poly-ubiquitin. \n",
      "Stable Parkin wild-type or pathogenic mutants expressing SH-SY5Y cell lines were induced with doxycycline (DOX) for 72 h and treated with CCCP for the indicated times.figf2Stable Parkin wild-type or pathogenic mutants expressing SH-SY5Y cell lines were induced with doxycycline (DOX) for 72 h and treated with CCCP for the indicated times. Cells were stained with anti-Parkin (green), anti-TOM20 (red) and Hoechst33342 (blue). (a-b) CCCP treatment of stable SH-SY5Y cells for 0 h. Unstimulated control cells (- DOX) show relatively low levels of endogenous Parkin, compared to cells with induced expression of Parkin wild-type or K161N, R275W and G430D mutants (+ DOX). Immunostaining (a) and western blot analysis (b) of the induced cells demonstrate comparable levels of Parkin expression in the selected clones. β-Actin served as a loading control. (c, d) After 2 h of CCCP treatment, massive colocalization of wild-type Parkin and mitochondrial clusters can be observed in the induced cells when compared with unstimulated controls. Similarly, Parkin K161N and R275W mutants translocate to mitochondria, whereas G430D does not. (e, f) After 24 h of CCCP treatment, induced wild-type Parkin-expressing cells show full clearance of damaged mitochondria. In contrast, the pathogenic mutants analysed fail to execute mitophagy and, therefore, damaged mitochondria remain uncleared. Representative images of three independent experiments are shown. Scale bars, 10 μm (a, c and e). (d, f) Quantifications of the observed mitochondrial effects in Parkin-expressing SH-SY5Y cells (n = 3). Data represent the mean ± s.d., ***P ≤ 0.0005 compared with induced wild-type Parkin. WT, wild-type. \n",
      "a) Schematic representation of human PINK1 (MTS, mitochondrial targeting sequence; TM, transmembrane domain; Ser/Thr-Kinase, Serine/threonine kinase domain).figf3(b) Co-immunoprecipitations of Parkin and PINK1 after mitochondrial depolarization. Endogenous Parkin and PINK1 were co-immunoprecipitated from neuronal SH-SY5Y cells after CCCP treatment with specific antibodies. (c) Western blot of PINK1 siRNA-transfected HeLa cells used in d-h demonstrates the knockdown efficiency and specificity of PINK1 silencing. d-h) PINK1-specific siRNA abrogates Parkin translocation to mitochondria after 2 h, and mitochondrial clearance after 24 h, of CCCP treatment. This can be rescued by re-transfection with wild-type PINK1, but not with a kinase-dead mutant or a PINK1 variant lacking the MTS. Control siRNA- or PINK1 siRNA-silenced HeLa cells were transfected with EGFP-Parkin wild-type and PINK1-V5 wild-type, 3×KD or ΔMTS mutants. Use of appropriate empty vector controls is depicted. Cells were treated with CCCP and stained with anti-TOM20 (pseudo coloured; blue) and anti-V5 (red) to visualize mitochondrial morphology and re-transfected PINK1, respectively. Wild-type Parkin was visualized by EGFP epifluorescence (green). Parkin-positive cells are surrounded by a line. Representative images of four independent experiments are shown. Scale bars, 10 μm. (e) Wild-type Parkin colocalization with impaired mitochondria after 2 h of CCCP treatment was analysed (n = 4). (g) Quantification of results from cells in e. (f) Uncleared mitochondria were analysed after 24 h of CCCP treatment (n = 4). (h) Quantification of results from cells in f. Data represent the mean ± s.d., **P ≤ 0.005, ***P ≤ 0.0005 compared with control siRNA-transfected cells. Immunostaining of control siRNA-transfected cells at all analysed time points is shown in Supplementary Information, Fig. S2. WT, wild-type. \n",
      ": DRiPs and mCherry-VHL convert PML-NBs into an amyloid solid-like state that chaperones can revert (see also -9).Figure 6PML-GFP HeLa Kyoto cells were lipofected with a cDNA encoding for mCherry-VHL. 24 h post-transfection, cells were left untreated or treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h. Where indicated (OP-puro + MG132 + CHX), translation was inhibited during stress with cycloheximide (CHX; 50 μg/ml). In the lower panel, cells were allowed to recover for 5 h in drug-free medium (Control), or with VER (40 µM). The subcellular distribution of PML-GFP and mCherry-VHL was studied by live-cell confocal imaging. Representative pictures are shown. Scale bars: 5 µm. Quantitation of the fluorescence intensity recovery after bleach of PML-GFP. PML-GFP HeLa Kyoto overexpressing mCherry-VHL for 24 h and treated as described in (A) were analyzed. The mean of 19-23 FRAP curves and the fitting curves are shown in black and red respectively. In gray, the SD is shown. Intranuclear distribution of 20S proteasomes, HSP70, VCP and DRiPs in HeLa cells treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h and allowed to recover for 5 h in drug-free medium (control), with VER (40 µM), Eeyarestatin I (5 µM) or MG132 (10 µM). Scale bars: 5 µm. GFP-PSMA7 HeLa Kyoto cells were left untreated or treated as described in A. Cells were subjected to FRAP to analyze GFP-PSMA7 mobility. Quantitation of three populations of GFP-PSMA7 molecules is shown: fast moving, slow moving and immobile; statistical significance via One-way ANOVA; n = 12 - 14 ± SEM. \n",
      ": Sequestration of ubiquitin in solid PML-NBs impairs the recruitment of 53BP1 at fragile chromosomal sites (see also ).Figure 7Subcellular distribution of 53BP1 and DRiPs in HeLa cells that were left untreated or treated with MG132 (10 µM) and OP-puro (25 µM) for 4 h, followed by 5h recovery in drug-free medium (+ rec. control), with VER (40 µM; + rec. VER) or Eeyarestatin I (5 µM; + rec. EeyI). Where indicated, cells were treated with MG132, VER or EeyI alone for 5 h. Scale bars: 20 µm. Quantitation of the % of cells with 53BP1 foci. Cells were treated as described in (A) and divided in three categories, based on the number of 53BP1 foci/nucleus: 0, 1-2 and > 3. Number of cells counted/condition: 1112 - 1537 in three independent experiments; statistical significance via One-way ANOVA; p < 0.01. HeLa cells were lipofected with cDNAs encoding for HA-Ubiquitin, Flag-Ubiquitin or Flag-VCP 24 h post-transfection, cells were either left untreated or treated with MG132 (10 µM) and OP-puro (25 µM). Cells were processed for staining of HA, Flag and endogenous 53BP1. * indicates cells that do not express HA- or Flag-Ubiquitin and show no rescue of 53BP1 foci formation after stress. Scale bars: 20 µm. Quantitation of the % of cells with 53BP1 foci. Cells were treated as described in (C) and divided in three categories, based on the number of 53BP1 foci/nucleus: 0, 1-2 and > 3. Number of cells counted/condition: 710 - 938 in three independent experiments; statistical significance via One-way ANOVA; p < 0.01. Colony formation assay. The number of colonies surviving 10 days after plating is shown. Control cells or cells treated with MG132 (10 µM) and puromycin (5 µg/ml), followed by 5h recovery in drug-free medium (+ rec. control), with VER (40 µM; + rec. VER) or Eeyarestatin I (5 µM; + rec. EeyI) are shown. Where indicated, cells were treated with VER or EeyI alone for 5 h. Number of colonies counted/condition: up to 526 in three independent experiments; statistical significance via One-way ANOVA; p < 0.03, ± s.e.m. \n",
      "a) Schematic view of human Parkin structure (UBL, ubiquitin-like domain; R0, RING0/unique Parkin domain; R1/2, RING finger domains; IBR, in-between-ring domain).figf1(b) Parkin is recruited selectively to depolarized mitochondria and directs mitophagy. HeLa cells transfected with HA-Parkin were treated with CCCP for the indicated times. Mitochondria were stained by anti-TOM20 (pseudo coloured; blue) and a ΔΨm dependent MitoTracker (red). Parkin was stained with anti-HA (green). Without treatment, mitochondria are intact and stained by both mitochondrial markers, whereas Parkin is equally distributed in the cytoplasm. After 2 h of CCCP treatment, mitochondria are depolarized as shown by the loss of MitoTracker staining. Parkin completely translocates to mitochondria clustering at perinuclear regions. After 24 h of CCCP treatment, massive loss of mitochondria is observed as shown by the disappearance of the mitochondrial marker. Only Parkin-positive cells show mitochondrial clustering and clearance, in contrast to adjacent untransfected cells. Scale bars, 10 μm. (c) Pathogenic Parkin mutations abrogate mitochondrial translocation and/or clearance. HeLa cells transfected with Flag-Parkin wild-type or pathogenic mutants were treated with CCCP and stained with anti-Flag (green), anti-TOM20 (red) and Hoechst33342 (blue). Representative merged images are shown. Scale bars, 10 μm. Immunostaining of cells transfected with non-functional cysteine mutants and functional Parkin variants is shown in Supplementary Information, Fig. S1a and S1b, respectively. (d, e) Based on the mitochondrial effects, Parkin mutants can be classified into functional and non-functional classes. (d) Quantification of Parkin translocation to damaged mitochondria after 2 h of CCCP treatment, as defined by complete colocalization of Parkin with the mitochondrial marker (n = 3). (e) Quantification of cells with uncleared mitochondria after 24 h of CCCP treatment, as shown by complete loss of mitochondrial staining (n = 3). Data represent the mean ± s.d., *P ≤ 0.05, **P ≤ 0.005, ***P ≤ 0.0005 compared with wild-type. WT, wild-type. \n",
      "a, b) Stable Mito-DsRed (pseudo coloured; blue)-expressing HeLa cells were transfected with Flag–Parkin wild-type or pathogenic mutants.figf5(a, b) Stable Mito-DsRed (pseudo coloured; blue)-expressing HeLa cells were transfected with Flag-Parkin wild-type or pathogenic mutants. Cells were treated with CCCP for the indicated times before immunostaining with anti-Flag (green) and anti-p62 (red). (a) In contrast to the pathogenic mutants, wild-type Parkin colocalizes with p62 at the site of clustered mitochondria, after 6 h of CCCP treatment. (b) After 24 h of CCCP treatment, wild-type Parkin-transfected cells show complete loss of mitochondria, whereas K161N and R275W Parkin mutants colocalize with both p62 and mitochondria. In contrast, translocation-compromised mutants K211N, T240R and G430D, lacking the mitochondria-associated poly-ubiquitin signal (Fig. 4b), did not colocalize with p62. Representative images of four independent experiments are shown. Data from untreated control cells and quantification of colocalization of Parkin, mitochondria and p62 upon CCCP treatment are shown in Supplementary Information, Fig. 5a and b, c, respectively. (c, d) Control siRNA- or p62 siRNA-silenced HeLa cells were transfected with EGFP-Parkin wild-type. Cells were treated with CCCP for the indicated times and stained with anti-TOM20 (pseudo coloured; blue) to visualize mitochondrial morphology and anti-p62 (red) to monitor silencing efficiency. Parkin wild-type was visualized by EGFP epifluorescence (green). Parkin-positive cells with cleared mitochondria are surrounded by a line. ( (c-f) Knockdown of p62 has no influence on Parkin translocation to mitochondria after 6 h of CCCP treatment (c-e), but completely blocks final clearance after 24 h (c, d, f). (e, f) Relative parkin translocation (e) and mitochondrial clearance (f) after p62 knockdown and CCCP treatment (n = 4). Data represent the mean ± s.d., ***P ≤ 0.0005 compared with control siRNA-transfected cells. Representative images of four independent experiments are shown. Scale bars, 10 μm. Immunostainings of control cells are shown in Supplementary Information, Fig. S5d, e. (g) Western blot of siRNA-transfected HeLa cells used in c-f demonstrates the efficiency of p62 silencing. β-Actin probing served as a loading control. WT, wild-type. \n",
      "a, Samples from animals with salivary gland-specific knockdown of Atg12 (fkh–Gal4/w; UAS-Atg12IR/+), n = 21, and those with salivary gland-specific knockdown of both Atg12 and drpr-I (fkh–Gal4/w; UAS-Atg12IR/+; UAS-drpr-IIR/+), n = 19, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation.figf3a, Samples from animals with salivary gland-specific knockdown of Atg12 (fkh-Gal4/w; UAS-Atg12IR/+), n = 21, and those with salivary gland-specific knockdown of both Atg12 and drpr-I (fkh-Gal4/w; UAS-Atg12IR/+; UAS-drpr-IIR/+), n = 19, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. b, Quantification of data from a. c, GFP-LC3 was expressed in salivary glands of control animals (+/w; UAS-GFP-LC3/+; fkh-Gal4/+) and those with salivary gland-specific knockdown of drpr (w; UAS-drprIR/UAS-GFP-LC3; fkh-Gal4/+). Salivary glands were dissected 6 h and 14 h after puparium formation, imaged for GFP-LC3, and LC3 puncta were quantified using Zeiss Automeasure software. d, Quantification of data from c. Error bars represent s.e.m.; n ≥ 10; p  0.0000001. e, LAMP1-GFP was expressed in control animals (tub-LAMP-GFP/w; +/fkh-Gal4) and those with salivary gland-specific knockdown of drpr (tub-LAMP-GFP/w; UAS-drprIR/+; fkh-Gal4/+). Salivary glands were dissected 14 h after puparium formation, imaged for LAMP-GFP, and LAMP puncta were quantified using Zeiss Automeasure software. f, Quantification of data from e. Error bars represent s.e.m.; n ≥ 10; p  0.05. g, Samples from drpr mutant animals (+/w; +/UAS-Atg16A; drprDgr;5/drprDgr;5), n = 10, and those with salivary gland-specific expression of Atg16A (fkh-Gal4/w; UAS-Atg16A/+; drprDgr;5/drprDgr;5), n = 16, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. h, Quantification of data from g. \n",
      "a) Western blot analysis of parental SH-SY5Y cells treated with CCCP for the indicated times.figf7(a) Western blot analysis of parental SH-SY5Y cells treated with CCCP for the indicated times. Time-course experiments of CCCP-treated neuronal cells reveal increasing modifications of VDAC1 over time. Total lysates were prepared, and western blots were probed with specific anti-Parkin, anti-PINK1, anti-TOM20 and anti-VDAC1 antibodies. All proteins represent endogenous levels. (b) Immunoprecipitations of Parkin and VDAC1 from parental SH-SY5Y cells confirm ubiquitylation of both proteins. CCCP treatment of neuronal cells increases ubiquitylated species of endogenous Parkin and VDAC1 over time. Using specific antibodies, Parkin and VDAC1 were immunoprecipitated from lysates of parental SH-SY5Y cells at different time points, western blotted and probed with anti-ubiquitin, anti-Parkin and anti-VDAC1 antibodies. (c) Pathogenic Parkin mutations fail to poly-ubiquitylate endogenous VDAC1 in response to CCCP treatment. Western blot analysis of a time-course experiment with inducible Parkin SH-SY5Y cells after CCCP treatment. Total lysates were prepared, and western blots were probed with specific anti-Parkin and anti-VDAC1 antibodies. β-Actin probing served as a loading control in a-c. WT, wild-type; Ub, ubiquitin. \n",
      "a) Time-course experiment of HeLa cells transfected with Flag–Parkin wild-type and treated with CCCP for the indicated periods.figf6(a) Time-course experiment of HeLa cells transfected with Flag-Parkin wild-type and treated with CCCP for the indicated periods. Total lysates were prepared, and western blots were probed with specific anti-Parkin, anti-PINK1, anti-TOM20 and anti-VDAC1 antibodies. β-Actin probing served as a loading control. All of the proteins detected are endogenously expressed, except Flag-Parkin. (b) Pathogenic Parkin mutants fail to poly-ubiquitylate endogenous VDAC1 in response to CCCP treatment. HeLa cells were transfected with empty vector (-), Flag-Parkin wild-type or pathogenic mutants along with 6×His-tagged ubiquitin wild-type for 24 h. Cells were treated with CCCP for 6 h and subsequently protein lysates were prepared in urea buffer. Denaturing conditions of Ni-NTA affinity purifications were used to specifically pulldown only proteins covalently modified with 6xHis-tagged ubiquitin. Western blots of protein lysates and purified Ni-NTA agarose elutions were probed with anti-VDAC1, anti-TOM20 and anti-Parkin antibodies. Modification of VDAC1 was observed in cells expressing Parkin wild-type and the functional mutant G328E, whereas non-functional mutants did not have this effect. ( (c) In response to CCCP treatment, Parkin catalyses predominantly K27-linked poly-ubiquitin chains on endogenous VDAC1 and is itself strongly poly-ubiquitylated. HeLa cells were transfected with an empty vector (-) or Flag-Parkin wild-type along with 6×His-tagged ubiquitin variants for 24 h. Treatment of cells, preparation of lysates and Ni-NTA purification were performed as described in c and probed for Parkin, VDAC1 and β-actin. WT, wild-type; Ub, ubiquitin. \n",
      "a, Protein extracts from drpr null (w; drpr&Dgr5/drpr&Dgr5) pupae at puparium formation (0 h) and wild-type (Canton-S) salivary glands 6 h, 12 h and 14 h after puparium formation, analysed by western blotting with anti-Drpr antibody.figf1a, Protein extracts from drpr null (w; drpr∆5/drpr∆5) pupae at puparium formation (0 h) and wild-type (Canton-S) salivary glands6 h, 12 h and 14 h after puparium formation, analysed by western blotting with anti-Drpr antibody. b, Samples from control animals (+/w; +/drpr∆5), n = 12, and drpr null mutants (w; drpr∆5/drpr∆5), n = 47, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. c, Quantification of data from b and d. d, Samples from control animals (+/w; +/UAS-drprIR), n = 11, and those with salivary gland-specific knockdown of drpr (fkh-Gal4/w; UAS-drprIR/+), n = 19, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. e, Control animals (+/w; +/UAS-drpr-IIR), n = 9, and those with salivary gland-specific knockdown of drpr-I (fkh-Gal4/w; UAS-drpr-IIR/+), n = 20, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. f, Quantification of data from e. g, Samples from drpr null animals (+/w; +/UAS-Drpr-I; drpr∆5/drpr∆5), n = 9, and those with salivary gland-specific expression of Drpr-I (fkh-Gal4/w; UAS-Drpr-I/+; drpr∆5/drpr∆5), n = 20, analysed by histology for the presence of salivary gland material (red circles) 24 h after puparium formation. h, Quantification of data from g. \n",
      "a, Protein extracts from drpr null (w; drprDgr;5/drprDgr;5) pupae at puparium formation (0 h) and from the fat bodies of wild-type (Canton-S) third-instar larvae were analysed by western blotting with anti-Drpr antibody.figf4a, Protein extracts from drpr null (w; drprDgr;5/drprDgr;5) pupae at puparium formation (0 h) and from the fat bodies of wild-type (Canton-S) third-instar larvae were analysed by western blotting with anti-Drpr antibody. Third-instar larvae were either fed or starved for 4 h. b, We either fed Wild-type (Canton-S) third-instar larvae or starved them for 4 h, and dissected fat bodies, stained them with anti-Drpr antibody, and imaged them for Drpr (green). We stained nuclei with 4′,6-diamidino-2-phenylindole (DAPI) (blue). c, We starved third-instar larvae expressing GFP-Atg8a in all cells, and drprIR specifically in dsRed-marked clone cells (hsflp/w; UAS-drprIR/+; hsGFPAtg8a, act,cd2, FRT > Gal4, UAS-dsRed/+), for 4 h. We dissected larval fat bodies and imaged them for GFP-Atg8a (green) and dsRed (red). d, Salivary glands of animals expressing GFP-Atg8a in all cells, and drprIR specifically in dsRed-marked clone cells (hsflp/w; UAS-drprIR/+; hsGFPAtg8a, act,cd2, FRT > Gal4, UAS-dsRed/+), dissected 14 h after puparium formation. Imaged for GFP-Atg8a (green) and dsRed (red). We stained nuclei with Hoechst (blue). \n",
      "VDAC1-specific siRNA abrogates Parkin translocation to mitochondria after 2 h, and mitochondrial clearance after 24 h, of CCCP treatment.figf8VDAC1-specific siRNA abrogates Parkin translocation to mitochondria after 2 h, and mitochondrial clearance after 24 h, of CCCP treatment. Both effects can be rescued by re-transfection with wild-type VDAC1. (a) Immunofluorescence and (b) western blot of siRNA-transfected HeLa cells used in c-g demonstrate partial silencing of endogenous VDAC1. (a) Silenced cells were stained with anti-HtrA2/Omi, a mitochondrial marker (pseudo coloured; blue), and anti-VDAC1 (red) antibodies. (b) Lysates from silenced cells were subjected to western blot analysis and probed with an anti-VDAC1 antibody, while TOM20 and β-actin probing served as loading controls. (c-e) Control siRNA- or VDAC1 siRNA-silenced HeLa cells were transfected with the indicated combinations of EGFP-Parkin wild-type, VDAC1-Flag wild-type or appropriate empty vector controls. Cells were treated with CCCP for 2 h and 24 h and stained with anti-HtrA2/Omi (pseudo coloured; blue) and anti-Flag (red) to visualize mitochondrial morphology and re-transfected VDAC1, respectively. Wild-type Parkin was visualized by EGFP epifluorescence (green). Parkin-positive cells are surrounded by a line. Representative pictures of three independent experiments are shown. Scale bars, 10 μm. (f, g) Quantification of d (f) and e (g). Parkin wild-type colocalization with impaired mitochondria after 2 h (f) and uncleared mitochondria after 24 h (g) of CCCP treatment (n = 3). Data represent the mean ± s.d., *P ≤ 0.05 compared with control siRNA-transfected or with VDAC1-Flag wild-type-transfected cells. \n",
      ": IFN-I, Myd88 and IFN-γ activate distinct sets of genes in mouse pDC during MCMV infection (A) Generation of mixed bone marrow chimera mice (MBMC); CTR, control; TST, test.Figure 1 (A) Generation of mixed bone marrow chimera mice (MBMC); CTR, control; TST, test. Recipient CD45.1 (5.1) mice were irradiated, then reconstituted with equal proportions of bone marrow cells isolated from indicated mice. Mice were used at least 8 weeks after reconstitution  (B) Gating strategy. pDC were identified as lineage (TCRb, CD19, Ly6G, NK1.1)- CD11b- CD11clow/int CD317high cells. 5.2 vs 5.1 cells were finally discriminated within the pDC gate 5.2+ pDC were isolated by sorting from MBMC reconstituted with BM cells from WT B6 (CTR), Ifnar1-/- or Myd88-/- deficient (TST) mice. Their 5.1+ WT counterparts were isolated from the same mice. Two independent experiments were performed for each experimental condition (i.e. with Ifnar1-CTR and -TST MBMC on the one hand, versus Myd88-CTR and TST MBMC on the other hand). Samples from uninfected animals are shown as empty symbols and those from MCMV-infected mice as filled symbols. Altogether, this led to 14 biological replicates for WT pDC, irrespective of their CD45 allotypic status and of the type of MBMC they originated from, both for uninfected and MCMV-infected MBMC. pDC isolated from uninfected or MCMV-infected C57BL/6 mice were included as internal control. Unsupervised hierarchical clustering (HC) and principal component analysis (PCA) were performed on all the probe sets of the microarrays. For HC, Pearson correlation distance and Ward's method linkage were used.  (E) Venn diagram showing the overlap between the lists of genes significantly upregulated in WT pDC of infected mice (n=14) as compared to WT pDC of uninfected animals (n=14) (grey circle), or downregulated in Ifnar1-KO (n=2) (red circle) or Myd88-KO (n=2) (green circle) pDC isolated from infected TST MBMC, as compared to WT pDC (n=6) isolated from the same TST or from their matched CTR infected MBMC  (F) Enrichment of the gene modules identified in (E) for independent gene sets previously established from the analysis of other microarray datasets (blue font), belonging to specific signaling pathways (red font), associated to specific functions or diseases (violet font) or regulated by the indicated upstream regulators (black font), as assessed by using Ingenuity Pathway Analysis (IPA). Data are represented as a heatmap, with gene modules identified in panel E as columns and gene sets as rows. The colored cells indicate the significance of the enrichment of each gene module for each gene set, as computed by IPA analysis and according to the color scale shown below the heatmap \n",
      "a, Samples from animals with salivary gland-specific expression of p35 (fkh–Gal4/+; UAS-p35/+), n = 18 (left), drpr null animals (+/w; +/UAS-p35; drpr&Dgr5/drpr&Dgr5), n = 10 (middle), and drpr null animals with salivary gland-specific expression of p35 (fkh–Gal4/w; UAS-p35/+; drpr&Dgr5/drpr&Dgr5), n = 16 (right), analysed by histology for the presence of salivary gland material 24 h after puparium formation.figf2a, Samples from animals with salivary gland-specific expression of p35 (fkh-Gal4/+; UAS-p35/+), n = 18 (left), drpr null animals (+/w; +/UAS-p35; drpr∆5/drpr∆5), n = 10 (middle), and drpr null animals with salivary gland-specific expression of p35 (fkh-Gal4/w; UAS-p35/+; drpr∆5/drpr∆5), n = 16 (right), analysed by histology for the presence of salivary gland material 24 h after puparium formation. Red circles, cell fragments; yellow circle, gland fragments. a, Samples from animals with salivary gland-specific expression of p35 (fkh-Gal4/+; UAS-p35/+), n = 18 (left), drpr null animals (+/w; +/UAS-p35; drpr∆5/drpr∆5), n = 10 (middle), and drpr null animals with salivary gland-specific expression of p35 (fkh-Gal4/w; UAS-p35/+; drpr∆5/drpr∆5), n = 16 (right), analysed by histology for the presence of salivary gland material 24 h after puparium formation. Red circles, cell fragments; yellow circle, gland fragments. b, Quantification of data from a. c, Salivary glands dissected from animals expressing drprIR specifically in GFP-marked clone cells (heat shock (hs) flp/w; UAS-drprIR/+; act,cd2, FRT > Gal4, UAS-GFP/+) 6 h and 14 h after puparium formation. Stained with GFP antibody (green), to label cells expressing drpr IR, and Lamin antibody (red). \n",
      ".Figure 2Representative genes were selected for modules identified on Venn diagram in Fig 1E and their expression pattern across all samples shown as a heatmap. Red font, pDC ISG (module A); green font, pDC MSG (module D); mixed red and green font, pDC ISG and MSG (module E); black font, non-ISG, non-MSG (module B). The relative mRNA expression levels for each gene (lanes) across biological samples (rows) are represented by colors according to the scale shown above the heatmap. For the legend of the symbols depicting the classes of biological samples above each column please refer to the symbol key in Fig 1D. \n",
      ": pDC IFN-I production is promoted by LFA1 engagement and occurs in the vicinity of infected cells during MCMV infection.Figure 7Ratio of CD86 expression or IFN-I production in 5.2+ vs 5.1+ pDC isolated from MCMV-infected MBMC of each indicated type. Black, CTR MBMC; pale pink, Itgal-TST MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with 3 mice per group. Ratio of CD86 expression or IFN-I production in 5.2+ vs 5.1+ pDC isolated from CpG-stimulated (B) MBMC of each indicated type. Black, CTR MBMC; pale pink, Itgal-TST MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with 3 mice per group. (C) EYFP expression in splenocytes isolated from uninfected (NI) vs MCMV-infected (MCMV) IFNbtm1(EYFP) reporter mice. The expression of indicated markers was analyzed on YFP+ cells. Plots are representative of one out of 6 mice from 2 independent experiments. (D-F) Immunohistological analysis of splenic sections stained with mAb against MOMA1/CD169 (blue), GFP/EYFP (green), whose expression correlates with that of IFN-β, and MCMV IE-1 (red). Scale bars, 100µm. The image shown is one representative of 15 obtained from 5 different mice. Optical magnification was 20x, and images were scanned on the microscope at zoom x2 for (E, F). White dotted lines delimit marginal zone. Yellow squares in (D) defines zoomed region in (E, F). \n",
      ": IKKβ is required for pDC IFN-I production (A, B) Ratio of CD86 expression or IFN-I production in 5.Figure 6 (A, B) Ratio of CD86 expression or IFN-I production in 5.2+ vs 5.1+ pDC isolated from MCMV-infected (A MBMC of each indicated type  (A, B) Ratio of CD86 expression or IFN-I production in 5.2+ vs 5.1+ pDC isolated fro CpG-stimulated (B) MBMC of each indicated type  (C) Ratio of IL-12- or TNF-producing cells in 5.2+ vs 5.1+ pDC of indicated MCMV-infected MBMC mice \n",
      ": IRF7 is necessary for IFN-I production in pDC, but low protein levels are sufficient to promote this function during MCMV infection (A) IRF7 protein expression in pDC isolated from Irf7-/- (pink, bottom), C57BL/6 (WT, black, top) and Ifnar1-/- (red, middle) uninfected (empty histograms, Ø, left) vs MCMV-infected mice (filled histograms, MCMV, right).Figure 5 (A) IRF7 protein expression in pDC isolated from Irf7-/- (pink, bottom), C57BL/6 (WT, black, top) and Ifnar1-/- (red, middle) uninfected (empty histograms, Ø, left) vs MCMV-infected mice (filled histograms, MCMV, right). For each condition histograms are representative of one out of 4-6 mice from 2 independent experiments  (B) IRF7 protein expression in pDC (left), cDC (middle) and conventional monocytes (cMo, right) isolated from uninfected (Ø, empty) or MCMV-infected (MCMV, filled) C57BL/6 (black) or Ifnar1-/- (red) mice. Data are shown as net IRF7 MFI, after subtraction of the MFI obtained using isotype control mAbs. Data shown (mean±SEM) are pooled from 2 independent experiments with 5-6 mice for each group  (C-E) The expression of CD86 (C) or of indicated cytokines (D-E) was analyzed, as described in Fig 2. Black, CTR MBMC; pink, Irf7-TST MBMC. Mice were treated with isotype control (IC, square) or blocking α-IFNAR1 mAbs (triangle). Data shown (mean±SEM) are from 2 pooled independent experiments each with 2-3 mice per group \n",
      ": During MCMV infection, the TLR9/Myd88-dependent pathway is necessary for pDC cytokine production, irrespective of their IFN-I responsiveness (A) Dot plots represent IFN-I vs GFP staining in 5.Figure 4 (A) Dot plots represent IFN-I vs GFP staining in 5.2+ vs 5.1+ pDC isolated from indicated MCMV-GFP-infected MBMC. One staining representative of 6 animals from 2 independent experiments is shown  (B, C) Percentages of GFP+ cells within 5.2+ pDC (B) or of GFP+ (green) vs GFP- (white) cells within 5.2+ IFN-I-producing pDC (C) isolated from indicated MCMV-GFP-infected MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with 3 mice per group  (D) qPCR for the expression of the indicated genes was performed on whole mRNA isolated from the spleen of indicated MCMV-infected MBMC. CTR MBMC (black) and Myd88-/- TST MBMC (dark green) were treated with isotype control (IC, square) or blocking α-IFNAR1 mAbs (triangle). Data shown (mean±SEM) are from 2 pooled independent experiments each with 2-3 mice per group  (E-G) CD86 MFI in 5.2+ pDC of each indicated MBMC type (E) and 5.2/ 5.1 ratio of cytokine-producing pDC (F, G). Data (mean±SEM) are representative of 2 pooled independent experiments each with 2-3 mice per group CD86 MFI and percentages of cytokine-producing cells within pDC of each indicated mouse strain. Black, C57BL6; red, Ifnar1-/-; orange, Ifnar1-/-x Myd88-/- CD86 MFI and percentages of cytokine-producing cells within pDC of each indicated mouse strain. Black, C57BL6 green, Tlr9-/- and pale green, Tlr7-/-. In (K-L) indicated mice were treated with IC (square) or blocking α-IFNAR1 mAbs (triangle). Data depicted (mean±SEM) are from 2 pooled independent experiments each with 2-3 mice per group. \n",
      ": IFN-I-responsiveness and AP3 are dispensable for in vivo IFN-I production in pDC during MCMV infection, but required upon stimulation with CpG.Figure 3(A) Histogram representing CD86 expression in 5.2+ vs 5.1+ pDC isolated from indicated MCMV-infected MBMC; grey, 5.2+ and 5.1+ pDC from CTR MBMC; red, 5.2+ Ifnar1-/- pDC isolated from Ifnar1-TST MBMC; purple, 5.1+ WT pDC isolated from Ifnar1-TST MBMC. Dotted line shows isotype control staining. One histogram representative of 6 animals from 3 independent experiments is shown. (B) Dot plots represent intracellular IFN-I staining in 5.2+ vs 5.1+ pDC isolated from indicated MCMV-infected MBMC. Isotype control (IC) staining was performed on a mixture of cells isolated from both types of MCMV-infected MBMC. The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Results were expressed as 5.2/5.1 ratio of Mean Fluorescence Intensity (MFI) values for CD86 or of the percentages of IFN-I-producing cells obtained from MCMV-infected Black, CTR MBMC; red, Ifnar1-TST MBMC (red) The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Results were expressed as 5.2/5.1 ratio of Mean Fluorescence Intensity (MFI) values for CD86 or of the percentages of IFN-I-producing cells obtained from CpG-stimulated mice. Black, CTR MBMC; red, Ifnar1-TST MBMC (red) Data shown (mean±SEM) are from 2 pooled independent experiments each with at least 3 mice per group. The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Results were expressed as 5.2/5.1 ratio of Mean Fluorescence Intensity (MFI) values for CD86 or of the percentages of IFN-I-producing cells obtained from MCMV-infected Black, CTR MBMC pale blue, Ap3b1-TST MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with at least 3 mice per group. The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Results were expressed as 5.2/5.1 ratio of Mean Fluorescence Intensity (MFI) values for CD86 or of the percentages of IFN-I-producing cells obtained from CpG-stimulated mice. Black, CTR MBMC pale blue, Ap3b1-TST MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with at least 3 mice per group. The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Ratio of IL-12- and TNF-producing cells in pDC isolated from indicated MCMV-infected MBMC. Black, CTR MBMC; red, Ifnar1-TST MBMC (red) Data shown (mean±SEM) are from 2 pooled independent experiments each with at least 3 mice per group. The expression of indicated markers was analyzed in 5.2+ vs 5.1+ pDC of each MBMC type. Ratio of IL-12- and TNF-producing cells in pDC isolated from indicated MCMV-infected MBMC. Black, CTR MBMC ; pale blue, Ap3b1-TST MBMC. Data shown (mean±SEM) are from 2 pooled independent experiments each with at least 3 mice per group. \n"
     ]
    }
   ],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_fig/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[2]) + \".txt\"\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "    first_chars = []\n",
    "    for line in lines: \n",
    "        xml_example = fromstring(line)  \n",
    "        xml_encoder = XMLEncoder(xml_example)\n",
    "        text = []\n",
    "        for element in xml_encoder.element.itertext():\n",
    "            text.append(element)\n",
    "\n",
    "        print(\"\".join(text))\n",
    "        "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a2c61112",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "32514ebc",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "1a283e96",
   "metadata": {},
   "source": [
    "## Generating HuggingFace datasets for panelization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 248,
   "id": "ed6c787e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(['(A)', '(B)'], ['(C-D)'])"
      ]
     },
     "execution_count": 248,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "TEXT = \"\"\"Figure 2 Rendering images. Rendering a high resolution figure from a set of subfigures. \n",
    "(A) Whole figure layout (consiting of three subfigures, denoted A, B and C, in the low resolution page space. \n",
    "(B) Actual subfigure dimensions. (C-D) (Reconstructed high resolution figure.\"\"\"\n",
    "\n",
    "\n",
    "import re\n",
    "re.findall(\"\\(.?\\)\", TEXT), re.findall(\"\\(.?-.?\\)\", TEXT)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a46c2869",
   "metadata": {},
   "source": [
    "## Generating HuggingFace datasets for Roles or syntactic segmentation"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "d4ef96bd",
   "metadata": {},
   "outputs": [],
   "source": [
    "from smtag.xml2labels import SourceDataCodes as sdc\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "4dfa36fe",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['CONTROLLED_VAR', 'MEASURED_VAR']"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sdc.SMALL_MOL_ROLES.all_labels"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "3472cd0c",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "SourceDataCodes.BORING\n",
      "SourceDataCodes.ENTITY_TYPES\n",
      "SourceDataCodes.GENEPROD_ROLES\n",
      "['Antigenicity', 'of', 'the', '2019', '-', 'nCoV', 'RBD', '.', 'Figure', '4', '.', 'The', 'SARS', '-', 'CoV', 'RBD', 'is', 'shown', 'as', 'a', 'white', 'molecular', 'surface', '(', 'PDB', 'ID', ':', '2AJF', ')', ',', 'with', 'residues', 'that', 'vary', 'in', 'the', '2019', '-', 'nCoV', 'RBD', 'colored', 'red', '.', 'The', 'ACE2', 'binding', 'site', 'is', 'outlined', 'with', 'a', 'black', 'dotted', 'line', '.', 'A', 'biolayer', 'interferometry', 'sensorgram', 'that', 'shows', 'binding', 'to', 'ACE2', 'by', 'the', '2019', '-', 'nCoV', 'RBD', '-', 'SD1', '.', 'Binding', 'data', 'are', 'shown', 'as', 'a', 'black', 'line', 'and', 'the', 'best', 'fit', 'of', 'the', 'data', 'to', 'a', '1', ':', '1', 'binding', 'model', 'is', 'shown', 'in', 'red', '.', 'Biolayer', 'interferometry', 'to', 'measure', 'cross', '-', 'reactivity', 'of', 'the', 'SARS', '-', 'CoV', 'RBD', '-', 'directed', 'antibodies', 'S230', ',', 'm396', 'and', '80R', '.', 'Sensortips', 'with', 'immobilized', 'antibodies', 'were', 'dipped', 'into', 'wells', 'containing', '2019', '-', 'nCoV', 'RBD', '-', 'SD1', 'and', 'the', 'resulting', 'data', 'are', 'shown', 'as', 'a', 'black', 'line', '.']\n",
      "['O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', 'O', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1', '1']\n"
     ]
    },
    {
     "ename": "NameError",
     "evalue": "name 'stop' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m/tmp/ipykernel_399/2563205499.py\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     55\u001b[0m             \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mwords\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     56\u001b[0m             \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlabel_words\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 57\u001b[0;31m             \u001b[0mstop\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     58\u001b[0m         \u001b[0;31m#label_dict_words[code_map.name] = label_words\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     59\u001b[0m         \u001b[0miob2_labels\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'stop' is not defined"
     ]
    }
   ],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_fig/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[1]) + \".txt\"\n",
    "\n",
    "code_maps = [sdc.BORING, sdc.ENTITY_TYPES, sdc.GENEPROD_ROLES, sdc.PANELIZATION, sdc.SMALL_MOL_ROLES]\n",
    "\n",
    "def innertext(xml):\n",
    "    return \"\".join([t for t in xml.itertext()])\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "    #print(lines[20])\n",
    "    \n",
    "    for line in lines:\n",
    "        xml_example = fromstring(line)\n",
    "\n",
    "        xml_encoder = XMLEncoder(xml_example)\n",
    "        #print(xml_encoder.element.itertext())\n",
    "        inner_text = innertext(xml_encoder.element)\n",
    "\n",
    "        label_dict_chars, label_dict_words = {}, {}\n",
    "        label_dict_chars['text'] = list(inner_text)\n",
    "        if inner_text.startswith(\"Antigenicity\"):\n",
    "            break\n",
    "    \n",
    "    for code_map in code_maps:\n",
    "        # At this point we have a tag for each character.\n",
    "        # It is here where I should put chars together into words\n",
    "        words, label_words = [], []\n",
    "        xml_encoded = xml_encoder.encode(code_map)\n",
    "        label_dict_chars[code_map.name] = xml_encoded['label_ids']\n",
    "\n",
    "        word = ''\n",
    "        label_word = ''\n",
    "        for i, char in enumerate(label_dict_chars['text']):\n",
    "            if char.isalnum():\n",
    "                word += char\n",
    "                label_word += str(label_dict_chars[code_map.name][i]).replace(\"None\", \"O\")\n",
    "            elif char == \" \":\n",
    "                if word not in [\"\"]:\n",
    "                    words.append(word)\n",
    "                    label_words.append(label_word[0])\n",
    "                word = ''\n",
    "                label_word = ''\n",
    "            else:\n",
    "                if word not in [\"\"]:\n",
    "                    words.append(word)\n",
    "                    label_words.append(label_word[0])\n",
    "                words.append(char)\n",
    "                label_words.append(str(label_dict_chars[code_map.name][i]).replace(\"None\", \"O\"))\n",
    "                word = ''\n",
    "                label_word = ''\n",
    "                \n",
    "        if code_map.name == \"panel_start\":\n",
    "            print(words)\n",
    "            print(label_words)\n",
    "            stop             \n",
    "        #label_dict_words[code_map.name] = label_words\n",
    "        iob2_labels = []\n",
    "        print(code_map)\n",
    "        \n",
    "        for idx, label in enumerate(label_words):\n",
    "            if code_map.name == \"panel_start\":\n",
    "                iob2_labels.append(\"O\")\n",
    "            \n",
    "            if code_map.name != \"panel_start\":    \n",
    "                if label == \"O\":\n",
    "                    iob2_labels.append(label)\n",
    "\n",
    "                if (label != \"O\"):\n",
    "                    if idx == 0:\n",
    "                        iob2_labels.append(code_map.iob2_labels[int(label)*2])\n",
    "                    if (idx > 0) and (label_words[idx-1] != label):\n",
    "                        iob2_labels.append(code_map.iob2_labels[int(label)*2])\n",
    "                    if (idx > 0) and (label_words[idx-1] == label):\n",
    "                        iob2_labels.append(code_map.iob2_labels[int(label)*2-1])\n",
    "                \n",
    "        label_dict_words[code_map.name] = iob2_labels\n",
    "        iob2_labels = []\n",
    "       \n",
    "                \n",
    "        #print(words, label_words)\n",
    "    label_dict_words['words'] = words\n",
    "    \n",
    "    print(label_dict_words)\n",
    "        \n",
    "    for key in label_dict_words.keys():\n",
    "        print(len(label_dict_words[key]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "e80a8da5",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Antigenicity O\n",
      "of O\n",
      "the O\n",
      "2019 O\n",
      "- O\n",
      "nCoV O\n",
      "RBD O\n",
      ". O\n",
      "Figure O\n",
      "4 O\n",
      ". O\n",
      "The B-PANEL_START\n",
      "SARS O\n",
      "- O\n",
      "CoV O\n",
      "RBD O\n",
      "is O\n",
      "shown O\n",
      "as O\n",
      "a O\n",
      "white O\n",
      "molecular O\n",
      "surface O\n",
      "( O\n",
      "PDB O\n",
      "ID O\n",
      ": O\n",
      "2AJF O\n",
      ") O\n",
      ", O\n",
      "with O\n",
      "residues O\n",
      "that O\n",
      "vary O\n",
      "in O\n",
      "the O\n",
      "2019 O\n",
      "- O\n",
      "nCoV O\n",
      "RBD O\n",
      "colored O\n",
      "red O\n",
      ". O\n",
      "The O\n",
      "ACE2 O\n",
      "binding O\n",
      "site O\n",
      "is O\n",
      "outlined O\n",
      "with O\n",
      "a O\n",
      "black O\n",
      "dotted O\n",
      "line O\n",
      ". O\n",
      "A B-PANEL_START\n",
      "biolayer O\n",
      "interferometry O\n",
      "sensorgram O\n",
      "that O\n",
      "shows O\n",
      "binding O\n",
      "to O\n",
      "ACE2 O\n",
      "by O\n",
      "the O\n",
      "2019 O\n",
      "- O\n",
      "nCoV O\n",
      "RBD O\n",
      "- O\n",
      "SD1 O\n",
      ". O\n",
      "Binding O\n",
      "data O\n",
      "are O\n",
      "shown O\n",
      "as O\n",
      "a O\n",
      "black O\n",
      "line O\n",
      "and O\n",
      "the O\n",
      "best O\n",
      "fit O\n",
      "of O\n",
      "the O\n",
      "data O\n",
      "to O\n",
      "a O\n",
      "1 O\n",
      ": O\n",
      "1 O\n",
      "binding O\n",
      "model O\n",
      "is O\n",
      "shown O\n",
      "in O\n",
      "red O\n",
      ". O\n",
      "Biolayer B-PANEL_START\n",
      "interferometry O\n",
      "to O\n",
      "measure O\n",
      "cross O\n",
      "- O\n",
      "reactivity O\n",
      "of O\n",
      "the O\n",
      "SARS O\n",
      "- O\n",
      "CoV O\n",
      "RBD O\n",
      "- O\n",
      "directed O\n",
      "antibodies O\n",
      "S230 O\n",
      ", O\n",
      "m396 O\n",
      "and O\n",
      "80R O\n",
      ". O\n",
      "Sensortips O\n",
      "with O\n",
      "immobilized O\n",
      "antibodies O\n",
      "were O\n",
      "dipped O\n",
      "into O\n",
      "wells O\n",
      "containing O\n",
      "2019 O\n",
      "- O\n",
      "nCoV O\n",
      "RBD O\n",
      "- O\n",
      "SD1 O\n",
      "and O\n",
      "the O\n",
      "resulting O\n",
      "data O\n",
      "are O\n",
      "shown O\n",
      "as O\n",
      "a O\n",
      "black O\n",
      "line O\n",
      ". O\n"
     ]
    }
   ],
   "source": [
    "XML_FOLDER = '/app/data/xml/sd_fig/'\n",
    "subsets = [\"train\", \"eval\", \"test\"]\n",
    "source_file_path = os.path.join(XML_FOLDER, subsets[1]) + \".txt\"\n",
    "\n",
    "code_maps = [sdc.BORING, sdc.ENTITY_TYPES, sdc.GENEPROD_ROLES, sdc.PANELIZATION, sdc.SMALL_MOL_ROLES]\n",
    "\n",
    "def innertext(xml):\n",
    "    return \"\".join([t for t in xml.itertext()])\n",
    "\n",
    "with open(source_file_path) as f:\n",
    "    lines = f.readlines()\n",
    "    #print(lines[20])\n",
    "    \n",
    "    for line in lines:\n",
    "        xml_example = fromstring(line)\n",
    "\n",
    "        xml_encoder = XMLEncoder(xml_example)\n",
    "        #print(xml_encoder.element.itertext())\n",
    "        inner_text = innertext(xml_encoder.element)\n",
    "\n",
    "        label_dict_chars, label_dict_words = {}, {}\n",
    "        label_dict_chars['text'] = list(inner_text)\n",
    "        if inner_text.startswith(\"Antigenicity\"):\n",
    "            break\n",
    "    \n",
    "    code_map = sdc.PANELIZATION\n",
    "    # At this point we have a tag for each character.\n",
    "    # It is here where I should put chars together into words\n",
    "    words, label_words = [], []\n",
    "    xml_encoded = xml_encoder.encode(code_map)\n",
    "    label_dict_chars[code_map.name] = xml_encoded['label_ids']\n",
    "        \n",
    "    output = [\"O\"] * len(xml_encoded['label_ids'])\n",
    "    offsets = xml_encoded[\"offsets\"]\n",
    "    for offset in offsets:\n",
    "        output[offset[0]] = \"B-PANEL_START\"\n",
    "        \n",
    "\n",
    "    word, label_word = '', ''\n",
    "    for i, char in enumerate(label_dict_chars['text']):\n",
    "        if char.isalnum():\n",
    "            word += char\n",
    "            label_word += str(output[i])\n",
    "        elif char == \" \":\n",
    "            if word not in [\"\"]:\n",
    "                words.append(word)\n",
    "                if \"B-PANEL_START\" in label_word:\n",
    "                    label_words.append(\"B-PANEL_START\")\n",
    "                else:\n",
    "                    label_words.append(\"O\")\n",
    "            word = ''\n",
    "            label_word = ''\n",
    "        else:\n",
    "            if word not in [\"\"]:\n",
    "                words.append(word)\n",
    "                if \"B-PANEL_START\" in label_word:\n",
    "                    label_words.append(\"B-PANEL_START\")\n",
    "                else:\n",
    "                    label_words.append(\"O\")\n",
    "            words.append(char)\n",
    "            label_words.append(output[i])\n",
    "            word = ''\n",
    "            label_word = ''\n",
    "                               \n",
    "    for word, label in zip(words, label_words):\n",
    "        print(word, label)\n",
    "    \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "80dd9b60",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['O', 'O', 'O']"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[\"O\"] * 3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 51,
   "id": "75f35d10",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'label': 'SMALL_MOLECULE',\n",
       " 'tag': 'sd-tag',\n",
       " 'attributes': {'entity_type': ['molecule']}}"
      ]
     },
     "execution_count": 51,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "sdc.ENTITY_TYPES.constraints.get(1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "6e3bc618",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "223"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ord('ß')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5aac8405",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
